<!DOCTYPE html>
<html>
  <head>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />


    <meta name="og:image" content="https://homepage.univie.ac.at/noichlm94/full/econ/card.jpg" />
<meta name="twitter:card" content="summary_large_image"/>
        <meta name="twitter:image" content="https://homepage.univie.ac.at/noichlm94/full/econ/card.jpg"/>
    <meta name="twitter:description" content=""/>
<meta name="twitter:site" content="@MaxNoichl"/>

    <link rel="icon" href="icon.png" />
    <title>DHD2020</title>
    <script src="https://unpkg.com/lore-engine@1.1.9/dist/lore.min.js"></script>
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>

    <link 
      rel="stylesheet" 
      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.10.2/css/all.css" 
      integrity="sha256-piqEf7Ap7CMps8krDQsSOTZgF+MU/0MPyPW2enj5I40=" 
      crossorigin="anonymous" />
    <link
      href="https://fonts.googleapis.com/css?family=Open+Sans:400,600"
      rel="stylesheet"
    />
    <link href="https://fonts.googleapis.com/css?family=Libre+Baskerville:400,400i&display=swap" rel="stylesheet">


    <style>
      body {
        margin: 0px;
        padding: 0px;
        height: 100%;
        user-select: none;
        overflow: hidden;
        white-space: nowrap;
      }

      #lore {
        position: absolute;
        width: 100%;
        height: 100%;
      }

      #title {
        position: absolute;
        z-index: 9999;
        pointer-events: none;
        text-height: 1;
        opacity: 0.0;
        transition: opacity 0.5s ease-in;
        
          padding-bottom: 20px;
        
          font-size: 1.0em;
        
          color: #888888;
        
          font-family: 'Open Sans';
        
      }

      #x-axis {
        position: absolute;
        z-index: 9999;
        pointer-events: none;
        text-height: 1;
        opacity: 0.0;
        transition: opacity 0.5s ease-in;
        
          padding-top: 20px;
        
          font-size: 0.7em;
        
          color: #888888;
        
          font-family: 'Open Sans';
        
      }

      #y-axis {
        position: absolute;
        z-index: 9999;
        pointer-events: none;
        text-height: 1;
        transition: opacity 0.5s ease-in;
        
          padding-bottom: 20px;
        
          font-size: 0.7em;
        
          color: #888888;
        
          font-family: 'Open Sans';
        
          transform: rotate(-90deg);
        
      }

      #tip {
        position: absolute;
        z-index: 9999;
        font-size: 0.9em;
        padding: 5px;
        font-family: 'Open Sans';
        background-color: rgba(255, 255, 255, 1.0);
        border-radius: 2px;
        border-left: 5px solid #000;
        pointer-events: none;
        opacity: 0.0;
        transition: opacity 0.1s ease-out;
        filter: drop-shadow(0px 0px 10px rgba(0, 0, 0, 1.0));
      }

      #tip.show {
        opacity: 1.0;
        transition: opacity 0.1s ease-out;
      }

      #tip-text {
        position: relative;
        max-width: 650px;
        white-space: normal;
        line-break: normal;
      }

      #hover-indicator {
        display: none;
        position: absolute;
        z-index: 999;
        border: 1px solid #fff;
        background-color: rgba(255, 255, 255, 0.25);
        border-radius: 50%;
        pointer-events: none;
      }

      #hover-indicator.show {
        display: block !important
      }

      .selected-indicator {
        position: absolute;
        z-index: 999;
        pointer-events: none;
      }

      .selected-indicator .crosshair-x {
        position: absolute;
        top: 0; left: 0; bottom: 0; right: 0;
        height: 1.5px;
        width: 100%;
        margin: auto;
        
          background-color: #fff;
        
        background-color: #000;

      }

      .selected-indicator .crosshair-y {
        position: absolute;
        top: 0; left: 0; bottom: 0; right: 0;
        width: 1.5px;
        height: 100%;
        margin: auto;
        
          background-color: #fff;
        
        background-color: #000;
      }

      #legend {
        position: absolute;
        z-index: 9998;
        
          bottom: 10px;
        
          right: 10px;
        
          padding: 10px;
        
          border: 1px solid #262626;
        
          border-radius: 2px;
        
          background-color: #111111;
        
          filter: drop-shadow(0px 0px 10px rgba(0, 0, 0, 0.5));
        
          color: #eeeeee;
        
          font-family: 'Open Sans';
        
		background-color: #000;
      }

      #legend .container {
        display: flex;
        flex: auto;
        align-items: flex-start;
        
        flex-direction: column;
        
      }

      #legend .legend-section {
        width: 100%;
      }

      #legend .legend-section:not(:first-child) {
        
      }

      #legend h2 {
        font-size: 0.9em;
        font-weight: 600;
        padding-top: 0;
        margin-top: 0;
        margin-bottom: 5px;
        text-align: center;
        max-width: 250px;
        white-space: normal;
      }

      #legend h3 {
        font-size: 0.8em;
        margin-top: 10px;
        margin-bottom: 0px;
        max-width: 250px;
        white-space: normal;
      }

      #legend select {
        width: 100%;
        margin-bottom: 5px;
        background: transparent;
        color: #fff;
        border-radius: 2px;
        font-size: 0.75em;
      }

      #legend select option {
        background: #000;
      }

      #legend .legend-element, #legend .legend-element-range {
        position: relative;
        display: flex;
        flex: auto;
        align-items: flex-start;
        padding-bottom: 2px;
        padding-top: 2px;
      }

      #legend .legend-element {
        align-items: flex-start;
      }

      #legend .legend-element-range {
        flex-direction: column;
      }

      #legend .color-box {
        
          width: 15px;
        
          height: 15px;
        
          border: solid 0px;
        
      }

      #legend .color-stripe {
        
          width: 15px;
        
          height: 1px;
        
          border: solid 0px;
        
      }

      #legend .legend-label {
        height: 15px;
        font-size: 0.7em;
        padding-left: 5px;
      }

      #legend .legend-label.max {
        position: absolute;
        top: 0px;
        margin-left: 15px;
      }

      #legend .legend-label.min {
        position: absolute;
        bottom: 2px;
        margin-left: 15px;
      }

      #selected {
        position: absolute;
        display: none;
        z-index: 99999999998;
        width: 600px;
        
          bottom: 10px;
        
          left: 10px;
        
          padding: 0px;
        
          border: 1px solid #262626;
        
          border-radius: 2px;
        
          background-color: #111111;
        
          filter: drop-shadow(0px 0px 10px rgba(0, 0, 0, 0.5));
        
          color: #eeeeee;
        
          font-family: 'Open Sans';
        
      }

      #selected #selected-controls {
        display: inline-flex;
        align-items: center;
        justify-content: flex-end;
        padding: 5px 10px;
        border-bottom: 1px solid #262626;
        font-size: 0.8em;
       /* font-weight: bold;*/
      }

      #selected #selected-controls * {
        display: inline-block;
      }

      #selected #selected-controls #selected-title {
        flex-grow: 1;
        white-space: pre-wrap; 
        overflow: hidden;
        text-overflow: ellipsis;
        color: #aaa;
      }

      #selected #selected-controls a {
        margin: 0px 2px;
        font-size: 1em;
        color: #fff;
        opacity: 0.75;
        transition: 0.2s;
      }

      #selected #selected-controls a:hover {
        opacity: 1.0;
      }

      #selected #selected-container {
        padding: 10px;
        font-size: 0.8em;
      }

      #selected #selected-container .label {
        font-weight: bold;
        color: #aaa;
        font-size: 0.7em;
      }

      #selected #selected-container .label:not(:first-child) {
        margin-top: 2px;
      }

      #selected #selected-container .content {
        white-space: nowrap; 
        overflow: hidden;
        text-overflow: ellipsis;
      }

      #selected #selected-container .content a {
        text-decoration: none;
        color: #3498db;
      }

      #controls {
        position: absolute;
        z-index: 9998;
        
          top: 10px;
        
          right: 10px;
        
          padding: 2px;
        
          border: 1px solid #262626;
        
          border-radius: 2px;
        
          background-color: #111111;
        
          filter: drop-shadow(0px 0px 10px rgba(0, 0, 0, 0.5));
        
          color: #eeeeee;
        
          font-family: 'Open Sans';
        
      }

      #controls a {
        display: inline-block;
        padding: 5px 10px;
        font-size: 1.2em;
        color: #fff;
        opacity: 0.75;
        transition: 0.2s;
      }

      #controls a:hover {
        opacity: 1.0;
      }

      #controls #more-controls {
        display: inline-block;
      }

      #impress {
      	font-family: 'Libre Baskerville';

        position: absolute;
        z-index: 9999;
        top: 10px;
        left: 10px;
        font-size: 0.53em;
        color: #262626;
      }

      #impress a {
        color: #3498db;
        text-decoration: none;
      }

      .show {
        opacity: 1.0 !important;
      }

      .hide {
        display: none !important;
      }
      /*       have the loader div fill the map screen while disabling interaction */
        #loader {
            position: absolute;
            top: 0;
            bottom: 0;
            width: 100%;
            background-color: #ffffff;
            color: #ffffff;
            z-index: 4000000000;
        }
        
/*       since we know the height/width of the loader gif, 
         position absolutely and use top/left, subtracting half of 
         width and heigth using margin */
        #loader iframe {
            width: 200px;
            height: 400px;
            position: absolute;
            top: 50%;
            left: 50%;
            margin: -200px 0 0 -100px;
            border:none;
        }


      @media only screen and (max-width: 600px) {
        #selected {
          position: absolute;
          display: none;
          z-index: 9998;
          width: 125px;
          opacity: 0.8;
          background-filter: blur(10px);
        }

        #tip {
          display: none;
        }
      }

a { background: transparent; 
    color: #010a0f;
    transition: color 0.3s ease;
    }
a:focus { outline: thin dotted; }
a:active, a:hover { outline: 0; }



/* The Modal (background) */
.modal {
  display: none; /* Hidden by default */
  position: fixed; /* Stay in place */
  z-index: 99999; /* Sit on top */
  padding-top: 100px; /* Location of the box */
  left: 0;
  top: 0;
  width: 100%; /* Full width */
  height: 100%; /* Full height */
  overflow: auto; /* Enable scroll if needed */
  background-color: rgb(0,0,0); /* Fallback color */
  background-color: rgba(0,0,0,0.4); /* Black w/ opacity */

}

/* Modal Content */
.modal-content {
  background-color: #fefefe;
  margin: auto;
  padding: 20px;
  border: 1px solid #888;
  /*width: 80%;*/
  user-select: all;
  white-space: normal;
  max-width: 600px;
  flex: 0 1 auto;
  max-height: calc(100% - 196px);
  overflow-y: scroll;

  /*position: absolute;*/
  /*padding: 5px;*/
  font-family: 'Open Sans';
  background-color: rgba(255, 255, 255, 1.0);
  border-radius: 2px;
  border-left: 5px solid #1a2340;
  transition: opacity 0.2s ease-out;
  filter: drop-shadow(0px 0px 10px rgba(0, 0, 0, 1.0));
}

/* The Close Button */
.close {
  color: #aaaaaa;
  float: right;
  font-size: 28px;
  font-weight: bold;
}

.close:hover,
.close:focus {
  color: #000;
  text-decoration: none;
  cursor: pointer;
}
    </style>
  </head>
<!-- style="background-color:rgba(255, 0, 0, 0.4);" -->
  <body>
    
      <div id="title" data-bind="title" >Models & Systems biology</div>
    
    
    
<!--      <div id="impress"><i class="fa fa-info-circle"></i></div>
 -->     
    <div id="tip" data-bind="tip">
      <div id="tip-text" data-bind="tipText"></div>
    </div>
    <div id="selected" data-bind="selected">
      <div id="selected-controls" data-bind="selectedControls">
        <a id="selected-toggle" data-bind="selectedToggle" href="#">
          <i class="fas fa-toggle-on"></i>
        </a>
        <span>&nbsp;&nbsp;</span>
        <span id="selected-title" data-bind="selectedTitle"></span>
        <span>&nbsp;&nbsp;</span>
        <span id="selected-current" data-bind="selectedCurrent" data-bind="selectedCurrent">0</span><span> / </span><span id="selected-total" data-bind="selectedTotal">0</span>
        <span>&nbsp;&nbsp;</span>
        <a id="selected-prev" data-bind="selectedPrev" href="#">
          <i class="fas fa-angle-left"></i>
        </a>
        <span>&nbsp;</span>
        <a id="selected-next" data-bind="selectedNext" href="#">
          <i class="fas fa-angle-right"></i>
        </a>
      </div>
      <div id="selected-container" data-bind="selectedContainer">
      </div>
    </div>

<div id="myModal" class="modal">

  <!-- Modal content -->
  <div class="modal-content">
    <span class="close">&times;</span>
       Here we can have some Information for the reviewers!


</div>

  </div>

</div>



    <div id="controls" data-bind="controls">
      <div id="more-controls" data-bind="moreControls" class="hide">
        <a id="search" data-bind="search" href="#">
          <i class="fas fa-search"></i>
        </a>
        <a id="info" data-bind="info" href="#">
          <i class="fas fa-info"></i>
        </a>
      </div>
      <a id="show-controls" data-bind="showControls" href="#">
        <i class="fas fa-bars"></i>
      </a>
    </div>
    <div id="hover-indicator" data-bind="hoverIndicator"></div>

<!-- 
    <script>
document.getElementById("info").onclick = displayDate;

function displayDate() {
    document.getElementById("information").innerHTML = Date();
}
</script>
 -->
<script>
// Get the modal
var modal = document.getElementById("myModal");

// Get the button that opens the modal
var btn = document.getElementById("info");

// Get the <span> element that closes the modal
var span = document.getElementsByClassName("close")[0];

// When the user clicks the button, open the modal 
btn.onclick = function() {
  modal.style.display = "block";
}

// When the user clicks on <span> (x), close the modal
span.onclick = function() {
  modal.style.display = "none";
}

// When the user clicks anywhere outside of the modal, close it
window.onclick = function(event) {
  if (event.target == modal) {
    modal.style.display = "none";
  }
}
</script>


<!--     <div id='loader'>
        <iframe src='owlstruct.html' alt='loader'></iframe>
  </div>
 -->    <canvas id="lore"></canvas>

    
    <script>
      const data = {
econ: {
x: [547.712,630.927,335.383,562.292,619.558,507.939,477.604,473.927,269.991,468.282,595.48,516.138,629.91,650.665,424.099,620.812,382.668,496.845,527.429,596.876,543.647,579.623,428.271,658.391,569.297,526.275,488.445,550.785,560.284,450.225,397.309,508.027,442.698,399.76,519.332,525.347,559.407,591.699,450.11,588.562,476.894,486.358,490.278,497.932,592.371,395.737,522.244,530.688,629.946,397.746,476.912,329.9,445.672,594.56,417.369,485.405,476.387,525.973,584.266,482.019,460.763,658.472,472.829,579.49,422.04,535.915,658.211,339.487,373.522,516.707,501.163,546.671,572.046,549.089,519.598,307.949,544.501,414.842,512.13,304.452,425.484,336.635,441.393,466.184,593.771,749.673,484.296,464.911,496.948,310.985,541.23,531.33,580.474,582.049,582.82,536.646,342.241,429.812,481.301,576.522,616.312,376.974,579.263,557.579,408.859,721.537,451.278,658.4,530.662,582.782,403.679,590.24,443.325,420.186,518.832,749.73,570.221,449.332,415.716,454.865,415.572,590.696,517.217,621.288,412.387,575.137,272.176,579.333,445.59,613.299,413.014,640.987,307.143,516.036,660.068,514.815,581.416,482.264,336.446,438.501,525.484,524.645,444.653,580.577,443.569,627.796,621.972,406.69,467.277,343.062,547.945,305.742,340.638,524.189,425.793,477.219,519.183,564.793,552.469,519.029,547.485,504.858,474.521,338.012,398.975,434.141,579.349,476.169,546.14,554.208,602.457,636.351,309.725,538.795,580.667,572.5,602.495,305.719,336.198,519.104,564.647,488.326,518.937,482.683,525.473,535.553,336.337,308.036,494.347,500.388,578.5,560.755,531.831,517.686,721.739,464.725,471.887,536.923,377.009,522.348,498.523,497.77,302.662,661.801,429.764,519.316,523.975,401.474,580.877,507.018,502.006,522.483,342.745,443.325,532.532,590.253,592.262,443.647,530.742,425.124,534.656,396.567,633.912,523.372,587.738,492.401,498.929,537.639,530.958,555.175,616.589,497.966,494.521,274.407,498.833,600.219,465.827,414.751,524.107,496.411,567.269,371.82,578.938,545.255,409.426,513.144,475.079,749.955,414.257,662.03,481.041,424.396,633.641,533.996,538.705,481.742,308.184,538.838,749.643,520.938,405.351,545.238,498.293,574.306,423.865,533.484,480.306,566.004,551.957,499.471,593.332,445.103,537.011,444.925,489.796,430.32,484.312,627.017,417.056,469.353,577.883,643.823,596.25,534.792,621.329,578.148,629.506,585.508,571.42,443.232,539.037,637.189,581.103,444.397,583.021,658.789,302.785,582.785,424.736,585.534,615.173,483.812,417.375,444.435,451.748,414.253,574.336,335.005,517.564,548.058,399.942,531.306,455.848,307.025,420.353,525.862,332.252,662.319,526.619,398.482,631.009,653.315,653.804,388.02,375.714,515.514,516.449,473.233,540.929,632.879,501.139,520.64,519.855,455.696,625.779,750.0,662.941,634.917,334.861,420.699,653.964,592.989,637.369,471.998,395.238,342.136,487.55,576.037,548.466,633.921,547.691,559.134,532.228,370.701,630.776,332.625,428.714,232.606,516.941,306.179,583.902,472.832,332.341,395.249,438.729,576.201,622.803,478.382,477.612,418.774,501.203,398.184,410.877,272.585,422.947,544.896,519.016,518.4,509.491,521.42,498.432,654.818,583.84,579.437,436.532,476.062,336.291,497.952,505.436,619.549,637.304,532.708,344.116,583.43,406.996,585.821,579.513,517.562,460.317,495.817,521.73,499.594,397.359,538.861,473.403,443.098,577.21,484.1,423.48,417.456,448.038,637.015,364.87,474.408,424.224,518.926,440.939,307.705,580.323,579.019,411.298,545.175,540.308,658.911,532.014,544.949,580.626,575.193,443.827,344.414,548.007,634.352,561.283,373.717,581.249,582.787,419.25,419.881,658.468,540.753,307.999,536.375,546.292,576.542,543.176,463.606,434.966,466.079,484.502,526.131,394.445,499.666,496.451,414.978,416.621,470.268,646.078,663.693,626.195,405.197,529.812,749.999,500.276,469.346,442.826,465.369,438.076,523.408,520.051,510.168,584.598,749.651,491.232,478.352,547.53,578.96,463.25,730.281,533.675,749.882,448.984,509.614,613.032,578.901,550.611,522.071,553.917,630.083,547.427,342.379,492.706,558.327,609.278,376.985,522.934,568.446,629.447,419.489,436.916,620.799,634.686,521.143,546.025,445.554,491.451,623.526,449.153,419.186,415.428,642.246,499.8,502.613,482.912,557.989,269.976,560.407,468.076,336.308,625.816,632.961,481.347,527.57,533.05,518.742,522.877,374.229,456.226,269.181,588.367,373.848,525.01,384.335,531.886,341.007,651.146,521.154,483.198,476.736,568.781,396.25,268.824,461.753,395.499,548.653,619.135,520.624,504.228,480.79,660.075,454.569,335.1,429.381,406.925,552.402,566.192,485.493,307.491,443.961,655.298,419.396,442.537,566.23,515.537,565.731,447.242,578.516,588.946,444.953,453.15,567.867,401.074,579.674,510.746,395.002,401.406,334.938,637.382,574.786,406.096,509.184,477.369,487.675,539.83,524.174,367.866,663.302,372.071,367.085,498.495,624.829,522.26,501.319,547.771,720.792,500.026,394.072,503.765,463.464,564.272,582.268,365.249,527.302,575.316,550.512,581.962,540.099,531.156,633.836,508.276,477.62,386.237,340.102,417.075,403.128,477.553,454.923,457.422,473.258,428.421,525.072,557.691,582.003,570.058,304.323,533.333,477.742,412.431,536.282,590.78,634.271,477.762,514.395,448.59,545.493,449.6,490.437,637.423,434.625,501.525,486.269,434.036,553.759,617.527,532.861,331.974,399.717,419.259,533.673,578.14,505.42,544.296,477.121,534.122,398.902,532.051,554.477,423.461,606.216,525.699,381.769,520.532,592.36,534.074,457.555,396.381,485.615,577.525,341.087,622.31,419.434,582.943,421.359,443.75,370.358,633.934,532.59,477.411,543.766,475.604,581.457,434.916,331.525,446.282,612.76,552.405,576.852,400.73,479.735,599.15,421.407,420.881,548.379,398.817,466.894,517.55,370.744,599.621,441.366,625.18,488.582,520.361,521.905,542.723,464.533,395.274,485.025,551.348,556.523,298.067,574.894,574.715,465.157,434.885,505.32,518.398,575.176,424.026,602.651,452.465,603.985,540.512,526.08,546.735,657.985,420.808,445.287,501.087,483.148,588.279,445.255,409.23,449.664,450.364,522.151,573.114,522.161,536.149,582.01,568.771,547.915,473.354,321.754,527.518,603.466,631.83,439.407,421.473,573.596,408.88,471.986,416.371,444.539,492.482,549.348,515.15,389.501,406.013,401.491,367.255,636.675,372.927,513.295,385.82,516.008,370.598,564.183,577.941,424.001,661.356,518.113,543.929,553.961,548.593,405.308,490.504,450.412,536.438,526.124,602.752,482.258,576.354,549.531,467.116,583.827,367.578,562.518,544.577,499.004,637.739,549.724,499.564,547.808,557.941,461.815,451.043,580.926,549.019,521.312,613.033,519.155,507.298,442.466,332.276,621.794,374.567,426.06,477.636,536.564,522.567,663.562,524.766,537.917,540.064,581.88,631.667,563.927,653.958,592.752,449.241,514.507,625.763,654.296,582.224,580.836,502.112,519.411,384.811,481.862,420.681,525.713,447.702,464.627,489.045,578.256,578.56,523.74,476.558,621.705,409.723,418.641,543.595,519.385,536.974,657.811,556.258,412.901,581.173,486.659,633.578,540.59,442.382,636.402,363.2,367.405,535.941,462.056,497.52,549.866,323.249,487.839,397.563,408.313,501.091,654.303,570.574,476.92,660.506,532.044,397.265,612.257,546.435,602.501,446.558,478.4,374.193,447.481,377.087,524.414,550.649,589.827,547.14,542.902,471.884,416.762,636.928,613.646,620.609,302.329,522.74,447.814,531.622,730.203,533.975,582.77,544.57,473.171,451.795,449.686,483.974,546.41,475.248,660.172,440.541,419.755,631.856,657.518,464.653,384.228,621.545,661.158,531.699,580.187,388.02,532.578,419.955,513.596,423.613,548.137,596.696,578.825,530.117,462.659,520.561,400.051,478.357,564.952,470.599,524.237,650.977,581.005,493.454,499.784,482.44,475.649,372.561,657.541,553.821,568.975,483.059,384.679,269.15,414.005,395.009,528.52,479.079,531.052,429.546,445.904,582.771,730.108,484.566,654.39,640.275,528.621,553.579,520.046,529.588,562.144,749.683,420.293,630.414,507.788,519.11,490.613,373.025,581.0,232.802,428.173,524.247,476.747,270.35,504.204,424.601,490.614,434.258,620.371,498.307,556.698,408.022,532.805,569.678,546.422,526.597,500.98,434.014,270.403,472.182,546.427,578.634,525.334,406.179,421.211,603.423,546.574,573.535,305.491,536.374,335.301,576.455,544.986,420.123,568.647,366.971,468.882,730.125,437.994,499.259,532.707,270.435,483.467,398.9,578.834,519.472,661.656,516.19,305.402,374.259,501.113,397.044,634.291,584.004,534.873,383.415,311.202,518.632,648.41,302.721,433.122,481.011,635.716,511.387,505.064,398.203,524.638,521.644,749.704,504.037,600.572,422.931,583.511,464.026,520.722,660.333,519.735,443.901,533.46,336.853,415.687,600.496,546.984,637.877,540.588,601.527,500.968,535.845,478.531,582.173,395.863,582.294,596.036,749.451,473.364,537.187,306.008,519.444,523.636,583.052,395.785,562.209,545.837,617.272,535.84,462.851,336.141,396.384,521.606,730.111,578.397,501.897,326.39,407.523,481.552,414.111,413.324,342.829,482.922,524.392,499.825,397.101,491.709,389.514,603.09,569.739,530.666,339.664,661.022,529.373,459.165,415.872,303.533],
y: [297.703,304.32,398.238,377.139,288.262,405.577,131.686,324.447,416.856,278.86,313.0,294.119,201.944,222.812,332.335,217.977,208.567,312.487,131.264,383.974,164.508,291.709,338.666,238.432,231.579,127.799,311.383,399.024,383.971,233.892,126.135,329.909,249.479,107.036,240.098,130.244,331.496,343.181,192.822,358.619,304.276,145.499,404.648,397.022,247.034,297.721,421.82,219.389,204.288,510.718,131.055,395.129,347.432,289.27,276.141,384.083,302.086,217.199,317.352,134.859,198.489,289.817,293.928,175.699,178.975,163.057,238.747,259.57,121.95,272.776,325.852,394.538,208.641,257.358,103.31,422.583,377.364,212.564,410.939,254.068,182.016,398.268,250.978,366.5,288.9,533.926,146.793,233.732,302.918,385.16,268.766,159.337,317.669,363.88,245.866,228.433,260.908,188.342,371.742,216.549,288.503,365.654,288.434,392.708,272.61,208.359,191.171,238.764,300.637,0.012,191.149,344.088,387.776,322.45,149.818,533.99,331.349,358.448,420.42,348.264,420.26,249.149,148.477,217.158,210.748,222.933,414.944,216.567,387.406,286.546,262.511,294.133,252.109,392.381,100.857,179.85,243.1,136.678,397.371,275.367,195.427,414.748,381.649,244.251,384.356,331.804,214.483,275.931,274.093,261.979,355.887,253.114,260.772,282.954,219.511,131.041,104.066,276.952,404.695,152.27,395.782,291.759,241.125,157.445,103.8,294.741,288.692,371.229,345.501,392.495,159.565,312.714,423.645,366.467,289.236,210.292,158.867,253.371,155.89,149.368,226.758,370.077,149.442,136.961,329.543,366.067,155.633,422.963,230.06,244.277,175.01,271.05,180.117,148.473,208.306,233.987,304.025,341.69,365.297,416.412,244.582,247.627,400.108,234.525,259.867,104.242,270.607,121.14,451.343,390.236,288.588,243.277,262.175,276.261,185.026,249.148,315.133,253.063,277.868,181.288,306.468,130.818,310.077,422.021,356.569,312.418,212.198,362.059,219.355,181.745,288.033,312.007,211.23,413.82,351.779,161.619,201.263,419.836,130.452,258.54,334.032,205.714,293.372,279.561,209.373,239.106,304.893,534.222,118.147,233.827,282.465,180.551,304.53,279.125,300.048,375.595,422.452,376.053,533.9,417.782,191.429,366.023,331.999,210.539,492.346,221.553,316.742,228.377,362.356,398.948,314.361,279.413,364.728,388.845,389.465,260.436,315.648,310.562,256.919,325.956,212.433,298.367,315.321,163.3,288.757,270.4,200.886,362.508,325.701,236.114,377.989,357.984,288.315,388.417,454.07,102.355,399.298,0.0,333.907,363.017,287.804,137.678,179.866,247.327,193.08,274.42,210.728,396.721,368.758,321.905,508.17,219.724,364.125,252.823,322.545,129.606,398.526,234.628,270.425,208.071,298.974,226.92,106.523,387.203,373.269,181.431,150.464,287.864,387.036,358.932,394.21,236.345,207.252,270.476,219.444,534.271,97.999,359.704,398.726,495.393,293.913,314.422,311.86,241.046,295.743,261.989,365.001,212.455,322.019,302.403,298.498,333.764,178.843,207.533,304.674,397.564,217.453,380.223,306.707,397.261,454.947,205.092,397.688,294.407,344.751,365.64,292.563,319.508,286.346,184.793,244.194,509.556,213.825,417.434,493.092,392.148,148.19,205.829,289.758,153.204,392.635,292.473,362.582,289.918,257.893,303.971,155.802,353.628,402.128,314.256,361.726,280.18,263.926,363.436,204.705,362.823,265.817,318.049,270.674,394.508,242.438,241.064,126.584,342.869,289.988,346.307,175.224,147.298,218.505,255.926,379.124,360.646,204.348,351.048,492.016,104.792,280.598,421.919,275.337,174.049,118.36,254.581,163.034,290.655,180.576,263.621,451.005,212.114,385.831,263.751,251.418,309.03,294.509,122.075,254.183,0.006,180.224,495.004,289.857,361.879,421.954,280.857,399.765,217.566,387.537,277.197,216.073,234.11,311.878,415.145,513.688,243.293,396.153,212.514,183.914,118.535,289.372,97.283,292.527,143.349,218.925,534.27,322.907,367.951,250.423,233.75,347.792,216.448,103.418,243.434,363.432,533.91,312.667,352.391,403.117,174.552,241.698,205.928,278.575,534.121,220.833,242.782,287.409,288.502,402.86,304.993,220.254,308.807,219.199,261.835,307.577,242.982,365.774,365.253,417.133,228.932,308.192,204.467,344.122,314.818,307.349,416.229,400.235,387.247,383.83,333.916,281.734,180.314,209.607,219.43,241.055,287.526,136.423,332.961,416.998,176.244,361.364,155.64,335.025,306.677,314.874,273.905,309.772,434.553,421.937,386.191,196.681,417.794,359.628,122.189,121.47,209.931,377.454,261.083,219.923,268.795,209.683,321.674,331.262,511.702,417.967,278.245,297.138,379.319,314.953,107.492,269.304,314.821,236.652,283.509,397.979,338.404,277.037,335.755,371.936,302.912,420.629,385.516,292.414,277.595,273.472,334.049,281.415,338.593,222.495,269.61,250.064,271.606,195.09,271.514,507.029,259.412,291.846,296.728,506.747,259.719,313.215,211.645,192.254,184.533,352.619,340.384,294.623,294.437,205.72,97.637,120.902,207.886,229.682,334.295,289.792,337.851,380.37,208.514,394.115,514.209,241.013,235.899,227.825,453.168,204.195,216.632,360.087,291.036,230.639,332.91,180.523,356.92,244.079,207.847,387.579,260.184,276.609,207.233,382.138,347.564,348.521,282.604,338.195,346.505,337.289,322.301,272.829,416.304,183.427,209.29,264.188,367.701,342.09,313.178,349.163,181.881,230.901,299.057,193.16,308.362,355.902,341.77,353.168,311.856,200.007,361.058,290.124,300.428,398.391,102.603,201.261,311.422,335.047,331.42,343.714,230.938,280.531,104.398,415.143,201.122,332.03,162.19,196.208,208.708,237.702,246.112,333.165,347.385,129.624,408.929,173.301,260.976,313.874,204.253,453.95,494.51,338.526,209.116,358.854,178.797,242.367,374.963,303.184,243.267,296.987,397.511,386.294,222.278,396.648,222.189,135.563,133.792,339.359,201.887,495.096,400.876,104.285,202.313,101.842,207.403,161.787,279.395,217.972,300.953,236.202,159.369,342.0,376.649,129.728,145.93,239.05,177.439,408.393,210.753,329.947,325.062,342.044,392.115,284.331,211.191,209.175,158.897,343.227,158.097,268.681,301.171,298.092,102.733,495.123,388.901,271.948,373.803,313.008,271.673,210.207,231.63,390.498,267.533,275.552,213.746,374.038,258.374,369.455,296.558,282.139,394.51,216.573,159.533,310.913,348.616,201.169,209.661,499.658,361.365,419.307,387.522,313.487,249.267,280.938,218.41,192.872,506.688,208.371,361.281,121.466,410.799,210.736,281.546,207.512,374.372,289.669,491.869,99.587,150.629,385.565,244.779,402.271,275.756,387.283,194.981,164.629,128.235,158.126,134.977,179.629,347.279,325.609,363.904,209.027,277.101,165.589,241.457,362.431,313.884,277.106,320.531,332.9,358.7,258.616,245.676,312.564,307.068,222.584,295.962,405.104,250.697,397.917,217.045,122.596,333.673,319.342,166.347,234.274,97.313,416.361,364.503,300.535,368.858,303.186,228.263,106.424,248.036,194.027,272.123,301.85,293.344,171.453,249.658,331.318,103.464,211.096,231.691,323.029,415.609,191.603,242.915,387.994,253.797,174.809,419.352,231.516,216.981,198.861,254.181,353.092,104.248,168.391,290.264,395.733,263.599,245.054,408.203,357.201,378.428,250.719,358.311,202.875,205.941,164.076,358.699,233.307,219.705,256.162,407.014,510.858,275.1,275.355,293.192,330.385,366.287,233.982,221.201,511.649,296.436,375.485,159.945,388.736,379.332,122.508,387.27,364.669,305.818,402.903,350.482,399.846,367.236,369.303,180.862,360.78,315.848,314.571,401.07,215.317,193.739,278.555,205.946,333.319,0.067,165.926,284.74,195.388,391.573,146.512,298.868,351.348,236.22,344.124,203.241,355.328,290.294,255.907,208.931,216.986,235.089,180.214,450.525,387.571,179.032,201.309,406.842,332.195,324.082,383.661,174.733,220.256,235.437,143.249,102.401,319.7,270.381,119.019,205.593,219.954,248.764,252.136,243.423,137.013,284.789,207.518,291.014,244.551,273.943,230.218,211.491,415.999,119.742,513.279,218.656,319.006,331.192,277.224,387.991,0.051,205.962,309.82,228.114,216.816,297.606,379.719,235.264,411.821,334.184,533.937,206.213,310.752,330.677,433.157,299.466,385.965,451.425,380.211,336.596,415.899,318.112,416.615,242.223,333.282,403.547,294.356,315.832,394.57,179.05,195.194,184.194,325.722,379.762,126.239,285.082,294.77,416.564,282.053,271.834,174.703,347.0,276.769,214.076,159.341,273.904,209.631,417.382,275.602,156.623,216.278,344.703,203.615,271.814,206.292,276.876,205.957,197.898,397.66,175.4,416.574,312.02,509.145,266.509,267.202,235.157,149.73,253.609,386.207,264.977,126.561,293.362,317.752,282.983,211.756,385.002,434.307,291.158,401.14,336.837,347.209,323.206,383.371,269.871,509.756,346.874,282.313,533.956,281.014,159.281,207.22,356.437,241.138,142.214,100.577,236.96,253.188,180.567,399.191,420.34,160.174,383.867,293.36,336.161,159.01,392.033,163.872,377.313,452.956,511.422,453.214,317.143,533.717,282.546,365.782,253.13,239.584,205.111,453.929,512.649,376.054,401.235,288.281,163.645,234.09,155.939,511.935,284.17,205.966,291.078,327.421,397.016,500.064,231.616,211.526,211.287,261.375,369.947,414.317,352.765,131.393,253.35,218.686,319.822,331.239,310.088,396.795,234.66,219.367,380.505,323.645,254.787],
z: [219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794,219.794],
labels: ['<b>Title: </b>Meta-Heuristic Algorithms in Car Engine Design: A Literature Survey<br><b>Year: </b>2015<br><b>Author: </b> Mh. Tayarani-N,  X. Yao and  Hm. Xu<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1109/TEVC.2014.2355174"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IEEE TRANSACTIONS ON EVOLUTIONARY COMPUTATION<br><b>Cluster-Keywords: </b><em>systems biology, regulatory network, interaction networks, network model, gene regulatory, network, network biology, networks, gene network, response networks, network dynamics, dynamic model, cell networks, gene expression, biological networks, system dynamics, dynamic network, model, biological systems, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Meta-heuristic algorithms are often inspired by natural phenomena, including the evolution of species in Darwinian natural selection theory, ant behaviors in biology, flock behaviors of some birds, and annealing in metallurgy. Due to their great potential in solving difficult optimization problems, meta-heuristic algorithms have found their way into automobile engine design. There are different optimization problems arising in different areas of car engine management including calibration, control system, fault diagnosis, and modeling. In this paper we review the state-of-the-art applications of different meta-heuristic algorithms in engine management systems. The review covers a wide range of research, including the application of meta-heuristic algorithms in engine calibration, optimizing engine control systems, engine fault diagnosis, and optimizing different parts of engines and modeling. The meta-heuristic algorithms reviewed in this paper include evolutionary algorithms, evolution strategy, evolutionary programming, genetic programming, differential evolution, estimation of distribution algorithm, ant colony optimization, particle swarm optimization, memetic algorithms, and artificial immune system.</div>','<b>Title: </b>Inference of time-delayed gene regulatory networks based on dynamic Bayesian network hybrid learning method<br><b>Year: </b>2017<br><b>Author: </b> B. Yu,  Jm. Xu,  S. Li,  C. Chen,  Rx. Chen,  L. Wang,  Y. Zhang and  Mh. Wang<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.18632/oncotarget.21268"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ONCOTARGET<br><b>Cluster-Keywords: </b><em>regulatory networks, gene regulatory, gene expression, network inference, expression data, network structure, gene networks, networks grns, regulatory genes, boolean networks, bayesian network, network reconstruction, networks gene, network dynamics, gene regulation, inference method, systems biology, time series, infer gene, network<div style="font-size:11px;"><br><b>Abstract: </b><em>Gene regulatory networks GRNs research reveals complex life phenomena from the perspective of gene interaction, which is an important research field in systems biology. Traditional Bayesian networks have a high computational complexity, and the network structure scoring model has a single feature. Information-based approaches cannot identify the direction of regulation. In order to make up for the shortcomings of the above methods, this paper presents a novel hybrid learning method DBNCS based on dynamic Bayesian network DBN to construct the multiple time-delayed GRNs for the first time, combining the comprehensive score CS with the DBN model. DBNCS algorithm first uses CMI2NI conditional mutual inclusive information-based network inference algorithm for network structure profiles learning, namely the construction of search space. Then the redundant regulations are removed by using the recursive optimization algorithm RO, thereby reduce the false positive rate. Secondly, the network structure profiles are decomposed into a set of cliques without loss, which can significantly reduce the computational complexity. Finally, DBN model is used to identify the direction of gene regulation within the cliques and search for the optimal network structure. The performance of DBNCS algorithm is evaluated by the benchmark GRN datasets from DREAM challenge as well as the SOS DNA repair network in Escherichia coli, and compared with other state-of-the-art methods. The experimental results show the rationality of the algorithm design and the outstanding performance of the GRNs.</div>','<b>Title: </b>Quantifying the effects of co-expressing EGFR and HER2 on HER activation and trafficking<br><b>Year: </b>2008<br><b>Author: </b> H. Shankaran,  Y. Zhang,  L. Opresko and  H. Resat<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.bbrc.2008.04.043"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOCHEMICAL AND BIOPHYSICAL RESEARCH COMMUNICATIONS<br><b>Cluster-Keywords: </b><em>signaling network, systems biology, signal transduction, signaling pathways, receptor tyrosine, erbb signaling, erbb receptor, receptor signaling, intracellular signaling, mathematical models, cell signaling, computational model, factor receptor, receptor activation, growth factor, signaling models, networks signaling, receptor system, receptors systems, downstream signaling<div style="font-size:11px;"><br><b>Abstract: </b><em>The human epidermal growth factor receptor HER system is an intricately regulated system that plays critical roles in development and tumorigenesis. Here, we apply integrated experimentation and modeling to analyze HER receptor activation in a panel of cell lines expressing endogenous levels of EGFR/HER1 and different levels of HER2. A mathematical model that includes the fundamental processes involved in receptor activation and trafficking was used to fit the experimental data, and values of the independent parameters for active receptor dimer formation affinities, trafficking rates and relative phosphorylation levels were estimated. Obtained parameter values quantitatively support the existing ideas on the effect of HER2 on EGFR dynamics, and enable us to predict the abundances of various phosphorylated receptor dimers in the cell lines. C 2008 Published by Elsevier Inc.</div>','<b>Title: </b>BOOLEAN MODELS OF BISTABLE BIOLOGICAL SYSTEMS<br><b>Year: </b>2011<br><b>Author: </b> F. Hinkelmann and  R. Laubenbacher<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3934/dcdss.2011.4.1443"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>DISCRETE AND CONTINUOUS DYNAMICAL SYSTEMS-SERIES S<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>This paper presents an algorithm for approximating certain types of dynamical systems given by a system of ordinary delay differential equations by a Boolean network model. Often Boolean models are much simpler to understand than complex differential equations models. The motivation for this work comes from mathematical systems biology. While Boolean mechanisms do not provide information about exact concentration rates or time scales, they are often sufficient to capture steady states and other key dynamics. Due to their intuitive nature, such models are very appealing to researchers in the life sciences. This paper is focused on dynamical systems that exhibit bistability and are described by delay equations. It is shown that if a certain motif including a feedback loop is present in the wiring diagram of the system, the Boolean model captures the bistability of molecular switches. The method is applied to two examples from biology, the lac operon and the phage lambda lysis/lysogeny switch.</div>','<b>Title: </b>A microarray data-based semi-kinetic method for predicting quantitative dynamics of genetic networks<br><b>Year: </b>2005<br><b>Author: </b> K. Yugi,  Y. Nakayama,  S. Kojima,  T. Kitayama and  M. Tomita<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1471-2105-6-299"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>gene regulatory, gene expression, regulatory networks, expression data, regulatory pathway, regulatory genes, network dynamics, microarray data, genetic networks, transcription factor, models network, target gene, upstream regulatory, network topology, pathway network, gene functional, expression profile, transcriptional network, dynamic model, transcriptional regulatory<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Elucidating the dynamic behaviour of genetic regulatory networks is one of the most significant challenges in systems biology. However, conventional quantitative predictions have been limited to small networks because publicly available transcriptome data has not been extensively applied to dynamic simulation.Results: We present a microarray data-based semi-kinetic  MASK method which facilitates the prediction of regulatory dynamics of genetic networks composed of recurrently appearing network motifs with reasonable accuracy. The MASK method allows the determination of model parameters representing the contribution of regulators to transcription rate from time-series microarray data. Using a virtual regulatory network and a Saccharomyces cerevisiae ribosomal protein gene module, we confirmed that a MASK model can predict expression profiles for various conditions as accurately as a conventional kinetic model.Conclusion: We have demonstrated the MASK method for the construction of dynamic simulation models of genetic networks from time-series microarray data, initial mRNA copy number and first-order degradation constants of mRNA. The quantitative accuracy of the MASK models has been confirmed, and the results indicated that this method enables the prediction of quantitative dynamics in genetic networks composed of commonly used network motifs, which cover considerable fraction of the whole network.</div>','<b>Title: </b>The finite state projection approach for the analysis of stochastic noise in gene networks<br><b>Year: </b>2008<br><b>Author: </b> B. Munsky and  M. Khammash<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1109/TAC.2007.911361"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IEEE TRANSACTIONS ON AUTOMATIC CONTROL<br><b>Cluster-Keywords: </b><em>regulatory networks, stochastic simulation, gene networks, reaction networks, systems biology, stochastic model, network models, complex systems, networks stochastic, gene regulatory, biological systems, stochastic dynamics, differential equations, protein networks, cell cycle, continuous model, interaction networks, systems chemistry, time delays, biochemical reaction<div style="font-size:11px;"><br><b>Abstract: </b><em>In order to capture important subcellular dynamics, researchers in computational biology have begun to turn to mesoscopic models in which molecular interactions at the gene level behave as discrete stochastic events. While the trajectories of such Models cannot be described with deterministic expressions, the probability distributions of these trajectories can be described by the set of linear ordinary differential equations known as the chemical master equation CME. Until recently, it has been believed that the CME could only be solved analytically in the most trivial of problems, and the CME has been analyzed almost exclusively with kinetic Monte Carlo KMC algorithms. However, concepts from linear systems theory have enabled the finite state projection FSP approach and have significantly enhanced our ability to solve the CME without resorting to KMC simulations. In this paper, we review the FSP approach and introduce a variety of systems-theory-based modifications and enhancements to the FSP algorithm. Notions such as observability, controllability, and minimal realizations enable large reductions and increase efficiency with little to no loss in accuracy. Model reduction techniques based upon linear perturbation theory allow for the systematic projection of multiple time-scale dynamics onto a slowly varying manifold of much smaller dimension. We also present a powerful new reduction approach, in which we perform computations on a small subset of configuration grid points and then interpolate to find the distribution on the full set. The power of the FSP and its various reduction approaches is illustrated on few important models of genetic regulatory networks.</div>','<b>Title: </b>A model of the immune network with B-T cell co-operation .1. Prototypical structures and dynamics<br><b>Year: </b>1996<br><b>Author: </b> J. Carneiro,  A. Coutinho,  J. Faro and  J. Stewart<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1006/jtbi.1996.0192"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF THEORETICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>immune system, network models, immune response, model, biological systems, models, model reduction, dynamic model, biological networks, system, mathematical models, biochemical network, response model, systems, systems biology, network, immune, biology model, reaction networks, system systems<div style="font-size:11px;"><br><b>Abstract: </b><em>Hitherto, second generation network models of the immune system have all been restricted to B-lymphocytes and the Ig molecules they produce. These models have not so far been able to provide a convincing mechanism for the distinction between a Central Immune System CIS composed of a connected network of lymphocyte clones which couple with self antigens in a tolerant mode, and a Peripheral Immune System PIS composed of clones with little or no supra-clonal organization and which produce classical immune responses when interacting with non-self antigens. Here, we present a new network model which explicitly incorporates B-T cell co-operation. In this model, B-cell. activation is dependent on T-cell help, and activated T-cells are down-regulated by engagement of their TCRs by soluble Ig. We discuss the underlying biology on which we base the system of ordinary differential equations which defines the present network model. We then illustrate some basic features of the model by examining several prototypical situations with a small number of clones. Depending on the idiotypic connectivity structure, the model exhibits two distinct modes of coupling with antigens: an immune response mode in which T- and B-cell clones grow exponentially; and a tolerant mode in which T-cell clones are controlled by inclusion of all TCRs in the repertoire of an idiotypic B-cell network. Finally, we discuss the simplifying assumptions of the present model and argue that its range of validity is indeed the region of the state-space of the system where the discrimination between the CIS and the PIS takes place. C 1996 Elsevier Science Limited</div>','<b>Title: </b>SAGPAR: Structural Grammar-based Automated Pathway Reconstruction<br><b>Year: </b>2012<br><b>Author: </b> S. Tagore and  Rk. De<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s12539-012-0119-8"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>INTERDISCIPLINARY SCIENCES-COMPUTATIONAL LIFE SCIENCES<br><b>Cluster-Keywords: </b><em>gene expression, systems biology, metabolic network, network model, network analysis, dynamic model, signaling pathways, signaling network, regulatory network, dynamic network, biology approach, candidate genes, gene network, gene regulatory, biology analysis, mathematical model, network dynamic, biology network, metabolic pathways, pathway analysis<div style="font-size:11px;"><br><b>Abstract: </b><em>In-silico metabolic engineering is a very useful branch of systems biology for modeling, analysis and prediction of various outcomes of metabolic pathways. It can also be used for detecting interactions and dynamics within a network. Various protocols have been proposed for modeling a pathway. But most of these protocols have various disadvantages and shortcomings with respect to automated pathway modeling and analysis. In the present article, we have proposed a novel algorithm for automated pathway reconstruction. We have also made a comparative study of our algorithm with other standard protocols and discussed its advantages over others. We present StructurAl Grammar-based automated PAthway Reconstruction SAGPAR, a fast and robust algorithm that generates any metabolic pathway using some given structural representations of metabolites. Users can model any pathway based on some pre-required features that are asked as an input by the algorithm. The algorithm also takes into considerations various thermodynamic thresholds and structural properties while modeling a pathway. The given algorithm has been tested on the standard pathway datasets of 25 pathways of Mycoplasma pneumoniae M129 and 24 pathways of Homo sapiens. The dataset is taken from KEGG and PubChem Compound data repositories. SAGPAR performs much better than some already present metabolic pathway analysis tools like Copasi, PHT, Gepasi, Jarnac and Path-A.</div>','<b>Title: </b>Using harmonic analysis and optimization to study macromolecular dynamics<br><b>Year: </b>2006<br><b>Author: </b> Mk. Kim,  Y. Jang and  Ji. JeongLink: No link<br><b>Journal: </b>INTERNATIONAL JOURNAL OF CONTROL AUTOMATION AND SYSTEMS<br><b>Cluster-Keywords: </b><em>normal mode, mode analysis, elastic network, functional motions, functional dynamics, molecular chaperone, structural biology, acidic residues, network models, molecular dynamics, atomic resolution, protein complexes, conserved functional, hsp dynamics, correlated motions, protein folding, large biomolecular, thioredoxin system, harmonic analysis, protein motions<div style="font-size:11px;"><br><b>Abstract: </b><em>Mechanical system dynamics plays an important role in the area of computational structural biology. Elastic network models ENMs for macromolecules e.g., polymers, proteins, and nucleic acids such as DNA and RNA have been developed to understand the relationship between their structure and biological function. For example, a protein, which is basically a folded polypeptide chain, can be simply modeled as a mass-spring system from the mechanical viewpoint. Since the conformational flexibility of a protein is dominantly subject to its chemical bond interactions e.g., covalent bonds, salt bridges, and hydrogen bonds, these constraints can be modeled as linear spring connections between spatially proximal representatives in a variety of coarse-grained ENMs. Coarse-graining approaches enable one to simulate harmonic and anharmonic motions of large macromolecules in a PC, while all-atom based molecular dynamics MD simulation has been conventionally performed with an aid of supercomputer. A harmonic analysis of a macroscopic mechanical system, called normal mode analysis, has been adopted to analyze thermal fluctuations of a microscopic biological system around its equilibrium state. Furthermore, a structure-based system optimization, called elastic network interpolation, has been developed to predict nonlinear transition or folding pathways between two different functional states of a same macromolecule. The good agreement of simulation and experiment allows the employment of coarse-grained ENMs as a versatile tool for the study of macromolecular dynamics.</div>','<b>Title: </b>A Graph Based Framework to Model Virus Integration Sites<br><b>Year: </b>2016<br><b>Author: </b> R. Fronza,  A. Vasciaveo,  A. Benso and  M. Schmidt<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.csbj.2015.10.006"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>COMPUTATIONAL AND STRUCTURAL BIOTECHNOLOGY JOURNAL<br><b>Cluster-Keywords: </b><em>gene regulatory, systems biology, regulatory networks, gene network, stem cell, network model, transcription factors, gene expression, cell transcriptomes, network analysis, data analysis, stem cells, network grnsight, cell transformation, weighted network, gene ontology, networks grns, graph model, graph layout, endothelial cell<div style="font-size:11px;"><br><b>Abstract: </b><em>With next generation sequencing thousands of virus and viral vector integration genome targets are now under investigation to uncover specific integration preferences and to define clusters of integration, termed common integration sites CIS, that may allow to assess gene therapy safety or to detect disease related genomic features such as oncogenes.Here, we addressed the challenge to: 1 define the notion of CIS on graph models, 2 demonstrate that the structure of CIS enters in the category of scale-free networks and 3 show that our network approach analyzes CIS dynamically in an integrated systems biology framework using the Retroviral Transposon Tagged Cancer Gene Database RTCGD as a testing dataset. C 2016 The Authors. Published by Elsevier B.V. on behalf of the Research Network of Computational and Structural Biotechnology.</div>','<b>Title: </b>Global Stabilization of Boolean Networks to Control the Heterogeneity of Cellular Responses<br><b>Year: </b>2018<br><b>Author: </b> Jm. Yang,  Ck. Lee and  Kh. Cho<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3389/fphys.2018.00774"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>FRONTIERS IN PHYSIOLOGY<br><b>Cluster-Keywords: </b><em>boolean network, boolean functions, regulatory networks, canalizing functions, network models, gene regulatory, boolean control, control networks, biological networks, boolean models, update functions, interaction networks, large network, regulatory functions, canalizing boolean, genetic regulatory, systems biology, driver nodes, networks bcns, biological systems<div style="font-size:11px;"><br><b>Abstract: </b><em>Boolean networks BNs have been widely used as a useful model for molecular regulatory networks in systems biology. In the state space of BNs, attractors represent particular cell phenotypes. For targeted therapy of cancer, there is a pressing need to control the heterogeneity of cellular responses to the targeted drug by reducing the number of attractors associated with the ill phenotypes of cancer cells. Here, we present a novel control scheme for global stabilization of BNs to a unique fixed point. Using a sufficient condition of global stabilization with respect to the adjacency matrix, we can determine a set of constant controls so that the controlled BN is steered toward an unspecified fixed point which can then be further transformed to a desired attractor by subsequent control. Our method is efficient in that it has polynomial complexity with respect to the number of state variables, while having exponential complexity with respect to in-degree of BNs. To demonstrate the applicability of the proposed control scheme, we conduct simulation studies using a regulation influence network describing the metastatic process of cells and the Mitogen-activated protein kinase MAPK signaling network that is crucial in cancer cell fate determination.</div>','<b>Title: </b>Systems biology of innate immunity<br><b>Year: </b>2006<br><b>Author: </b> J. Tegner,  R. Nilsson,  Vb. Bajic,  J. Bjorkegren and  T. Ravasi<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.cellimm.2007.01.010"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>CELLULAR IMMUNOLOGY<br><b>Cluster-Keywords: </b><em>systems biology, regulatory network, interaction networks, network model, gene regulatory, network, network biology, networks, gene network, response networks, network dynamics, dynamic model, cell networks, gene expression, biological networks, system dynamics, dynamic network, model, biological systems, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Systems Biology has emerged as an exciting research approach in molecular biology and functional genomics that involves a systematic use of genomic, proteomic, and metabolomic technologies for the construction of network-based models of biological processes. These endeavors, collectively referred to as systems biology establish a paradigm by which to systematically interrogate, model, and iteratively refine our knowledge of the regulatory events within a cell. Here, we present a new systems approach, integrating DNA and transcript expression information, specifically designed to identify transcriptional networks governing the macrophage immune response to lipopolysaccharide LPS. Using this approach, we are not only able to infer a global macrophage transcriptional network, but also time-specific sub-networks that are dynamically active across the LPS response. We believe that our system biological approach could be useful for identifying other complex networks mediating immunological responses. c 2007 Elsevier Inc. All rights reserved.</div>','<b>Title: </b>What makes biochemical networks tick A graphical tool for the identification of oscillophores<br><b>Year: </b>2004<br><b>Author: </b> Bn. Goldstein,  G. Ermakov,  Jj. Centelles,  Hv. Westerhoff and  M. Cascante<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1111/j.1432-1033.2004.04324.x"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>EUROPEAN JOURNAL OF BIOCHEMISTRY<br><b>Cluster-Keywords: </b><em>biological system, network, pattern formation, system, network analysis, systems, turing patterns, patterns, reaction networks, networks, turing system, network topologies, stoichiometric network, networks pattern, network topology, biological, model, gradient formation, pattern, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>In view of the increasing number of reported concentration oscillations in living cells, methods are needed that can identify the causes of these oscillations. These causes always derive from the influences that concentrations have on reaction rates. The influences reach over many molecular reaction steps and are defined by the detailed molecular topology of the network. So-called autoinfluence paths, which quantify the influence of one molecular species upon itself through a particular path through the network, can have positive or negative values. The former bring a tendency towards instability. In this molecular context a new graphical approach is presented that enables the classification of network topologies into oscillophoretic and nonoscillophoretic, i.e. into ones that can and ones that cannot induce concentration oscillations. The network topologies are formulated in terms of a set of uni-molecular and bi-molecular reactions, organized into branched cycles of directed reactions, and presented as graphs. Subgraphs of the network topologies are then classified as negative ones which can and positive ones which cannot give rise to oscillations. A subgraph is oscillophoretic negative when it contains more positive than negative autoinfluence paths. Whether the former generates oscillations depends on the values of the other subgraphs, which again depend on the kinetic parameters. An example shows how this can be established. By following the rules of our new approach, various oscillatory kinetic models can be constructed and analyzed, starting from the classified simplest topologies and then working towards desirable complications. Realistic biochemical examples are analyzed with the new method, illustrating two new main classes of oscillophore topologies.</div>','<b>Title: </b>Hierarchical neuronal modeling of cognitive functions: from synaptic transmission to the Tower of London<br><b>Year: </b>1998<br><b>Author: </b> Jp. Changeux and  S. Dehaene<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/S0764-44699789829-0"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>COMPTES RENDUS DE L ACADEMIE DES SCIENCES SERIE III-SCIENCES DE LA VIE-LIFE SCIENCES<br><b>Cluster-Keywords: </b><em>systems biology, dynamic models, biological systems, model, dynamic systems, computational systems, network model, experimental data, model parameters, systems, systems model, models, pathway systems, data, biochemical pathway, dynamics model, biochemical networks, computational models, network, system<div style="font-size:11px;"><br><b>Abstract: </b><em>Recent progress in the molecular biology of synaptic transmission, in particular of neurotransmitter receptors, offers novel information relevant to realistic modeling of neural processes at the single cell and network level. Sophisticated computer analyses of 2D crystals by high resolution electron microscopy yield images of single neurotransmitter receptor molecules with tentative identifications of ligand binding sites and of conformational transitions. The dynamics of conformational changes can be accounted for by a multistate allosteric network model. Allosteric receptors also possess the structural and functional properties required to serve as coincidence detectors between pre-and post-synaptic signals and, therefore, can be used as building blocks for a chemical Hebb synapse. These properties were introduced into networks of formal neurons capable of producing and detecting temporal sequences. In more elaborate models of pre-frontal cortex functions, allosteric receptors control the selection of transient pre-representations and their stabilization by external or internal reward signals. We apply this scheme to Shallices Tower of London test, and we show how a hierarchical neuronal architecture can implement executive or planning functions associated with frontal areas. C Academie des sciences/Elsevier, Paris.</div>','<b>Title: </b>Zooming in on Yeast Osmoadaptation<br><b>Year: </b>2012<br><b>Author: </b> C. Kuhn and  E. Klipp<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/978-1-4419-7210-117"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ADVANCES IN SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>kinetic models, metabolic networks, kinetic parameters, dynamic models, systems biology, network model, dynamic metabolic, biochemical network, rate constants, enzyme kinetics, kinetic data, adjustable parameters, metabolic model, parameter estimation, dynamic optimization, model selection, mathematical model, kinetic network, model response, model<div style="font-size:11px;"><br><b>Abstract: </b><em>Saccharomyces cerevisiae is considered as a model organism for the investigation of cellular and molecular processes and gene regulation. Specifically, the response of S. cerevisiae to increase in osmolarity of the external medium osmoadaptation is a model adaptation process. The first mathematical model of volume changes in S. cerevisiae due to osmolarity has been proposed as early as 1983 by Schwartz and Diller Cryobiology 205:542-552. Since then, both experimental and computational methods in biology have progressed dramatically. Especially in recent years, the study of response to hyperosmotic stress in S. cerevisiae by systems biology approaches has advanced rapidly. However, a holistic understanding of osmoadaptation combining environmental conditions, cellular preconditions, biophysical processes, molecular and biochemical network dynamics, has not yet been reached. Here, we review recent advances in the investigation of different aspects of osmoadaptation and discuss them with respect to an integrated view. This leads us to critically evaluate how to approach the goal of such an integrated view.</div>','<b>Title: </b>A Graph Partitioning Approach to Predicting Patterns in Lateral Inhibition Systems<br><b>Year: </b>2013<br><b>Author: </b> Asr. Ferreira and  M. Arcak<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1137/130910142"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SIAM JOURNAL ON APPLIED DYNAMICAL SYSTEMS<br><b>Cluster-Keywords: </b><em>biological system, network, pattern formation, system, network analysis, systems, turing patterns, patterns, reaction networks, networks, turing system, network topologies, stoichiometric network, networks pattern, network topology, biological, model, gradient formation, pattern, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>We analyze spatial patterns on networks of cells where adjacent cells inhibit each other through contact signaling. We represent the network as a graph where each vertex represents the dynamics of an identical individual cell and where graph edges represent cell-to-cell signaling. To predict steady-state patterns we find equitable partitions of the graph vertices and assign them into disjoint classes. We then use results from monotone systems theory to prove the existence of patterns that are structured in such a way that all the cells in the same class have the same final fate. To study the stability properties of these patterns, we rely on the graph partition to perform a block decomposition of the system. Then, to guarantee stability, we provide a small-gain-type criterion that depends on the input-output properties of each cell in the reduced system. Finally, we discuss pattern formation in stochastic models. With the help of a modal decomposition we show that noise can enhance the parameter region where patterning occurs.</div>','<b>Title: </b>Stochastic resonance in biology - How noise can enhance detection of weak signals and help improve biological information processing<br><b>Year: </b>2002<br><b>Author: </b> P. Hanggi<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1002/1439-7641200203153:3<285::AID-CPHC285>3.0.CO;2-A"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>CHEMPHYSCHEM<br><b>Cluster-Keywords: </b><em>coupling strength, synchronization transitions, timeperiodic coupling, time delay, logic network, distribution network, network model, oscillatory networks, neuron networks, water distribution, stochastic resonance, gene networks, multiple synchronization, firing synchronization, neuronal networks, firing behaviors, logical networks, results show, regulatory networks, cell polarization<div style="font-size:11px;"><br><b>Abstract: </b><em>Noise is usually thought of as the enemy of order rather than as a,constructive influence. In nonlinear systems that possess some sort of threshold, random noise plays a beneficial role in enhancing the detection of weak information-carrying signals. This phenomenon, termed stochastic resonance, does find useful applications in physical, biological, and biomedical contexts. Certain biological systems may even use this effect for optimizing function and behavior.</div>','<b>Title: </b>Integration of lipidomics and transcriptomics data towards a systems biology model of sphingolipid metabolism<br><b>Year: </b>2011<br><b>Author: </b> S. Gupta,  Mr. Maurya,  Ah. Merrill,  Ck. Glass and  S. Subramaniam<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-5-26"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>gene expression, systems biology, metabolic network, network model, network analysis, dynamic model, signaling pathways, signaling network, regulatory network, dynamic network, biology approach, candidate genes, gene network, gene regulatory, biology analysis, mathematical model, network dynamic, biology network, metabolic pathways, pathway analysis<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Sphingolipids play important roles in cell structure and function as well as in the pathophysiology of many diseases. Many of the intermediates of sphingolipid biosynthesis are highly bioactive and sometimes have antagonistic activities, for example, ceramide promotes apoptosis whereas sphingosine-1-phosphate can inhibit apoptosis and induce cell growth; therefore, quantification of the metabolites and modeling of the sphingolipid network is imperative for an understanding of sphingolipid biology.Results: In this direction, the LIPID MAPS Consortium is developing methods to quantitate the sphingolipid metabolites in mammalian cells and is investigating their application to studies of the activation of the RAW264.7 macrophage cell by a chemically defined endotoxin, Kdo2-Lipid A. Herein, we describe a model for the C-16-branch of sphingolipid metabolism i.e., for ceramides with palmitate as the N-acyl-linked fatty acid, which is selected because it is a major subspecies for all categories of complex sphingolipids in RAW264.7 cells integrating lipidomics and transcriptomics data and using a two-step matrix-based approach to estimate the rate constants from experimental data. The rate constants obtained from the first step are further refined using generalized constrained nonlinear optimization. The resulting model fits the experimental data for all species. The robustness of the model is validated through parametric sensitivity analysis.Conclusions: A quantitative model of the sphigolipid pathway is developed by integrating metabolomics and transcriptomics data with legacy knowledge. The model could be used to design experimental studies of how genetic and pharmacological perturbations alter the flux through this important lipid biosynthetic pathway.</div>','<b>Title: </b>Modeling the Intra- and Extracellular Cytokine Signaling Pathway under Heat Stroke in the Liver<br><b>Year: </b>2013<br><b>Author: </b> M. Rodriguez-Fernandez,  B. Grosman,  Tm. Yuraszeck,  Bg. Helwig,  Lr. Leon and  Fj. Doyle<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0073393"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Heat stroke HS is a life-threatening illness induced by prolonged exposure to a hot environment that causes central nervous system abnormalities and severe hyperthermia. Current data suggest that the pathophysiological responses to heat stroke may not only be due to the immediate effects of heat exposure per se but also the result of a systemic inflammatory response syndrome SIRS. The observation that pro- e.g., IL-1 and anti-inflammatory e.g., IL-10 cytokines are elevated concomitantly during recovery suggests a complex network of interactions involved in the manifestation of heat-induced SIRS. In this study, we measured a set of circulating cytokine/soluble cytokine receptor proteins and liver cytokine and receptor mRNA accumulation in wild-type and tumor necrosis factor TNF receptor knockout mice to assess the effect of neutralization of TNF signaling on the SIRS following HS. Using a systems approach, we developed a computational model describing dynamic changes intra- and extracellular events in the cytokine signaling pathways in response to HS that was fitted to novel genomic liver mRNA accumulation and proteomic circulating cytokines and receptors data using global optimization. The model allows integration of relevant biological knowledge and formulation of new hypotheses regarding the molecular mechanisms behind the complex etiology of HS that may serve as future therapeutic targets. Moreover, using our unique modeling framework, we explored cytokine signaling pathways with three in silico experiments e.g. by simulating different heat insult scenarios and responses in cytokine knockout strains in silico.</div>','<b>Title: </b>Inference of the Xenopus tropicalis embryonic regulatory network and spatial gene expression patterns<br><b>Year: </b>2014<br><b>Author: </b> Zz. Zheng,  S. Christley,  Wt. Chiu,  Il. Blitz,  Xh. Xie,  Kwy. Cho and  Q. Nie<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-8-3"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: During embryogenesis, signaling molecules produced by one cell population direct gene regulatory changes in neighboring cells and influence their developmental fates and spatial organization. One of the earliest events in the development of the vertebrate embryo is the establishment of three germ layers, consisting of the ectoderm, mesoderm and endoderm. Attempts to measure gene expression in vivo in different germ layers and cell types are typically complicated by the heterogeneity of cell types within biological samples i.e., embryos, as the responses of individual cell types are intermingled into an aggregate observation of heterogeneous cell types. Here, we propose a novel method to elucidate gene regulatory circuits from these aggregate measurements in embryos of the frog Xenopus tropicalis using gene network inference algorithms and then test the ability of the inferred networks to predict spatial gene expression patterns.Results: We use two inference models with different underlying assumptions that incorporate existing network information, an ODE model for steady-state data and a Markov model for time series data, and contrast the performance of the two models. We apply our method to both control and knockdown embryos at multiple time points to reconstruct the core mesoderm and endoderm regulatory circuits. Those inferred networks are then used in combination with known dorsal-ventral spatial expression patterns of a subset of genes to predict spatial expression patterns for other genes. Both models are able to predict spatial expression patterns for some of the core mesoderm and endoderm genes, but interestingly of different gene subsets, suggesting that neither model is sufficient to recapitulate all of the spatial patterns, yet they are complementary for the patterns that they do capture.Conclusion: The presented methodology of gene network inference combined with spatial pattern prediction provides an additional layer of validation to elucidate the regulatory circuits controlling the spatial-temporal dynamics in embryonic development.</div>','<b>Title: </b>An interspecies regulatory network inferred from simultaneous RNA-seq of Candida albicans invading innate immune cells<br><b>Year: </b>2012<br><b>Author: </b> L. Tiemey,  J. Linde,  S. Muller,  S. Brunke,  Jc. Molina,  B. Hube,  U. Schock,  R. Guthke and  K. Kuchler<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3389/fmicb.2012.00085"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>FRONTIERS IN MICROBIOLOGY<br><b>Cluster-Keywords: </b><em>sensitivity analysis, systems biology, robustness analysis, network model, network inference, regulatory network, systems analysis, gene expression, parametric uncertainty, biological networks, mathematical models, biochemical networks, biological functions, cellular networks, biological system, parametric sensitivity, dynamical models, dynamic system, networks analysis, gene regulatory<div style="font-size:11px;"><br><b>Abstract: </b><em>The ability to adapt to diverse micro-environmental challenges encountered within a host is of pivotal importance to the opportunistic fungal pathogen Candida albicans. We have quantified C. albicans and M. musculus gene expression dynamics during phagocytosis by dendritic cells in a genome-wide, time-resolved analysis using simultaneous RNA-seq. A robust network inference map was generated from this dataset using NetGenerator, predicting novel interactions between the host and the pathogen. We experimentally verified predicted interdependent sub-networks comprising Hap3 in C. albicans, and Ptx3 and Mta2 in M. musculus. Remarkably, binding of recombinant Ptx3 to the C. albicans cell wall was found to regulate the expression of fungal Hap3 target genes as predicted by the network inference model. Pre-incubation of C. albicans with recombinant Ptx3 significantly altered the expression of Mta2 target cytokines such as IL-2 and IL-4 in a Hap3-dependent manner, further suggesting a role for Mta2 in host pathogen interplay as predicted in the network inference model. We propose an integrated model for the functionality of these sub-networks during fungal invasion of immune cells, according to which binding of Ptx3 to the C. albicans cell wall induces remodeling via fungal Hap3 target genes, thereby altering the immune response to the pathogen. We show the applicability of network inference to predict interactions between host-pathogen pairs, demonstrating the usefulness of this systems biology approach to decipher mechanisms of microbial pathogenesis.</div>','<b>Title: </b>Mathematical modelling in the post-genome era: understanding genome expression and regulation - a system theoretic approach<br><b>Year: </b>2002<br><b>Author: </b> O. Wolkenhauer<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/S0303-26470200004-7"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOSYSTEMS<br><b>Cluster-Keywords: </b><em>information systems, gene expression, gene network, systems theory, system models, systems biology, systems information, network model, genetic network, mathematical models, dynamic systems, dynamic network, systemic model, network topology, information mechanisms, complex systems, systems complex, relational models, models main, information model<div style="font-size:11px;"><br><b>Abstract: </b><em>This paper introduces a mathematical framework for modelling genome expression and regulation. Starting with a philosophical foundation, causation is identified as the principle of explanation of change in the realm of matter. Causation is, therefore, a relationship, not between components, but between changes of states of a system. We subsequently view genome expression formerly known as gene expression as a dynamic process and model aspects of it as dynamic systems using methodologies developed within the areas of systems and control theory. We begin with the possibly most abstract but general formulation in the setting of category theory. The class of models realised are state-space models, input-output models, autoregressive models or automata. We find that a number of proposed gene network models are, therefore, included in the framework presented here. The conceptual framework that integrates all of these models defines a dynamic system as a family of expression profiles. It becomes apparent that the concept of a gene is less appropriate when considering mathematical models of genome expression and regulation. The main claim of this paper is that we should treat model the organisation and regulation of genetic pathways as what they are: dynamic systems. Microarray technology allows Lis to generate large sets of time series data and is, therefore, discussed with regard to its use in mathematical modelling of gene expression and regulation. C 2002 Elsevier Science Ireland Ltd. All rights reserved.</div>','<b>Title: </b>Streamlining the construction of large-scale dynamic models using generic kinetic equations<br><b>Year: </b>2010<br><b>Author: </b> Da. Adiamah,  J. Handl and  Jm. Schwartz<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1093/bioinformatics/btq136"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>kinetic models, metabolic networks, kinetic parameters, dynamic models, systems biology, network model, dynamic metabolic, biochemical network, rate constants, enzyme kinetics, kinetic data, adjustable parameters, metabolic model, parameter estimation, dynamic optimization, model selection, mathematical model, kinetic network, model response, model<div style="font-size:11px;"><br><b>Abstract: </b><em>Motivation: Studying biological systems, not just at an individual component level but at a system-wide level, gives us great potential to understand fundamental functions and essential biological properties. Despite considerable advances in the topological analysis of metabolic networks, inadequate knowledge of the enzyme kinetic rate laws and their associated parameter values still hampers large-scale kinetic modelling. Furthermore, the integration of gene expression and protein levels into kinetic models is not straightforward.Results: The focus of our research is on streamlining the construction of large-scale kinetic models. A novel software tool was developed, which enables the generation of generic rate equations for all reactions in a model. It encompasses an algorithm for estimating the concentration of proteins for a reaction to reach a particular steady state when kinetic parameters are unknown, and two robust methods for parameter estimation. It also allows for the seamless integration of gene expression or protein levels into a reaction and can generate equations for both transcription and translation. We applied this methodology to model the yeast glycolysis pathway; our results show that the behaviour of the system can be accurately described using generic kinetic equations.Availability and implementation: The software tool, together with its source code in Java, is available from our project web site at http://www.bioinf.manchester.ac.uk/schwartz/grapeContact: jean-marc.schwartzmanchester.ac.ukSupplementary information: Supplementary data are available at Bioinformatics online.</div>','<b>Title: </b>Calibration of dynamic models of biological systems with KInfer<br><b>Year: </b>2010<br><b>Author: </b> P. Lecca,  A. Palmisano,  A. Ihekwaba and  C. Priami<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s00249-009-0520-3"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>EUROPEAN BIOPHYSICS JOURNAL WITH BIOPHYSICS LETTERS<br><b>Cluster-Keywords: </b><em>systems biology, dynamic models, biological systems, model, dynamic systems, computational systems, network model, experimental data, model parameters, systems, systems model, models, pathway systems, data, biochemical pathway, dynamics model, biochemical networks, computational models, network, system<div style="font-size:11px;"><br><b>Abstract: </b><em>Methods for parameter estimation that are robust to experimental uncertainties and to stochastic and biological noise and that require a minimum of a priori input knowledge are of key importance in computational systems biology. The new method presented in this paper aims to ensure an inference model that deduces the rate constants of a system of biochemical reactions from experimentally measured time courses of reactants. This new method was applied to some challenging parameter estimation problems of nonlinear dynamic biological systems and was tested both on synthetic and real data. The synthetic case studies are the 12-state model of the SERCA pump and a model of a genetic network containing feedback loops of interaction between regulator and effector genes. The real case studies consist of a model of the reaction between the inhibitor kappa B kinase enzyme and its substrate in the signal transduction pathway of NF-kappa B, and a stiff model of the fermentation pathway of Lactococcus lactis.</div>','<b>Title: </b>Combinatorics of Boolean automata circuits dynamics<br><b>Year: </b>2012<br><b>Author: </b> J. Demongeot,  M. Noual and  S. Sene<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.dam.2011.11.005"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>DISCRETE APPLIED MATHEMATICS<br><b>Cluster-Keywords: </b><em>regulatory networks, network dynamics, gene regulatory, network model, genetic network, gene networks, genetic regulatory, mathematical model, parameter space, network system, interaction networks, system dynamics, network structure, network, dynamic networks, network reduction, models models, biological networks, network motifs, biological systems<div style="font-size:11px;"><br><b>Abstract: </b><em>In line with fields of theoretical computer science and biology that study Boolean automata networks to model regulation networks, we present some results concerning the dynamics of networks whose underlying structures are oriented cycles, that is, Boolean automata circuits. In the context of biological regulation, former studies have highlighted the importance of circuits on the asymptotic dynamical behaviour of the biological networks that contain them. Our work focuses on the number of attractors of Boolean automata circuits whose elements are updated in parallel. In particular, we give the exact value of the total number of attractors of a circuit of arbitrary size n as well as, for every positive integer p, the number of its attractors of period p depending on whether the circuit has an even or an odd number of inhibitions. As a consequence, we obtain that both numbers depend only on the parity of the number of inhibitions and not on their distribution along the circuit. We also relate the counting of attractors of Boolean automata circuits to other known combinatorial problems and give intuition about how circuits interact by studying their dynamics when they intersect one another in one point. c 2011 Elsevier B.V. All rights reserved.</div>','<b>Title: </b>Molecular stripping, targets and decoys as modulators of oscillations in the NF-kappa B/I kappa B alpha/DNA genetic network<br><b>Year: </b>2016<br><b>Author: </b> Zp. Wang,  Da. Potoyan and  Pg. Wolynes<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1098/rsif.2016.0606"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF THE ROYAL SOCIETY INTERFACE<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Eukaryotic transcription factors in the NF-kappa B family are central components of an extensive genetic network that activates cellular responses to inflammation and to a host of other external stressors. This network consists of feedback loops that involve the inhibitor I kappa B alpha, numerous downstream functional targets, and still more numerous binding sites that do not appear to be directly functional. Under steady stimulation, the regulatory network of NF kappa B becomes oscillatory, and temporal patterns of NF-kappa B pulses appear to govern the patterns of downstream gene expression needed for immune response. Understanding how the information from external stress passes to oscillatory signals and is then ultimately relayed to gene expression is a general issue in systems biology. Recently, in vitro kinetic experiments as well as molecular simulations suggest that active stripping of NF-kappa B by I kappa B alpha from its binding sites can modify the traditional systems biology view of NF-kappa B/I kappa B alpha gene circuits. In this work, we revise the commonly adopted minimal model of the NF-kappa B regulatory network to account for the presence of the large number of binding sites for NF-kappa B along with dissociation from these sites that may proceed either by passive unbinding or by active molecular stripping. We identify regimes where the kinetics of target and decoy unbinding and molecular stripping enter a dynamic tug of war that may either compensate each other or amplify nuclear NF-kappa B activity, leading to distinct oscillatory patterns. Our finding that decoys and stripping play a key role in shaping the NF-kappa B oscillations suggests strategies to control NF-kappa B responses by introducing artificial decoys therapeutically.</div>','<b>Title: </b>Networking of WNT, FGF, notch, BMP, and hedgehog signaling pathways during carcinogenesis<br><b>Year: </b>2007<br><b>Author: </b> M. Katoh<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s12015-007-0006-6"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>STEM CELL REVIEWS<br><b>Cluster-Keywords: </b><em>gene expression, systems biology, metabolic network, network model, network analysis, dynamic model, signaling pathways, signaling network, regulatory network, dynamic network, biology approach, candidate genes, gene network, gene regulatory, biology analysis, mathematical model, network dynamic, biology network, metabolic pathways, pathway analysis<div style="font-size:11px;"><br><b>Abstract: </b><em>The biological functions of some orthologs within the human genome and model-animal genomes are evolutionarily conserved, but those of others are divergent due to protein evolution and promoter evolution. Because WNT signaling molecules play key roles during embryogenesis, tissue regeneration and carcinogenesis, the authors group has carried out a human WNT-ome project for the comprehensive characterization of human genes encoding WNT signaling molecules. From 1996 to 2002, we cloned and characterized WNT2B/WNT13, WNT3, WNT3A, WNT5B, WNT6, WNT7B, WNT8A, WNT8B, WNT9A/ WNT14, WNT9B/WNT14B, WNT10A, WNT10B, WNT11, FZD1, FZD2, FZD3, FZD4, FZD5, FZD6, FZD7, FZD8, FZD10, FRAT1, FRAT2, NKD1, NKD2, VANGL1, RHOU/ ARHU, RHOV/ARHV GIPC2, GIPC3, FBXW11/beta TRCP2, SOX17, TCF7L1/TCF3, and established a cDNA-PCR system for snap-sbot and dynamic analyses on the WNT-transcriptome. In 2003, we identified and characterized PRICKLE1, PRICKLE2, DACT1/DAPPER1, DACT2/DAPPER2, DAAM2, and BCL9L. After completion of the human WNT-ome project, we have been working on the stem cell signaling network. WNT signals are transduced to beta-catenin, NLK, NFAT, PKC, JNK and RhoA signaling cascades. FGF20, JAG1 and DKK1 are target genes of the WNT-beta-catenin signaling cascade. Cross-talk of WNT and FGF signaling pathways potentiates P-catenin and NFAT signaling cascades. BMP signals induce IHH upregulation in co-operation with RUNX Hedgehog signals induce upregulation of SFRP1, JAG2 and FOXL1, and then FOXL1 induces BMP4 upregulation. The balance between WNT-FGF-Notch and BMP-Hedgehog signaling networks is important for the maintenance of homoestasis among stem and progenitor cells. Disruption of the stem cell signaling network results in pathological conditions, such as congenital diseases and cancer.</div>','<b>Title: </b>Stochastic modeling and control of biological systems: The lactose regulation system of Escherichia coli<br><b>Year: </b>2008<br><b>Author: </b> Aa. Julius,  A. Halasz,  Ms. Sakar,  H. Rubin,  V. Kumar and  Gj. Pappas<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1109/TAC.2007.911346"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IEEE TRANSACTIONS ON AUTOMATIC CONTROL<br><b>Cluster-Keywords: </b><em>gene expression, gene network, stochastic model, gene regulatory, systems biology, expression systems, gene regulation, network activity, expression dynamics, noise control, regulatory network, deterministic models, network components, stochastic modeling, regulation system, escherichia coli, regulatory system, synthetic gene, regulatory genes, gene<div style="font-size:11px;"><br><b>Abstract: </b><em>In this paper, we present a comprehensive framework for stochastic modeling, model abstraction, and controller design for a biological system. The first half of the paper concerns modeling and model abstraction of the system. Most models in systems biology are deterministic models with ordinary differential equations in the concentration variables. We present a stochastic hybrid model of the lactose regulation system of E. coli bacteria that capture important phenomena which cannot be described by continuous deterministic models. We then show that the resulting stochastic hybrid model can be abstracted into a much simpler model, a two-state continuous-time Markov chain. The second half of the paper discusses controller design,for a specific architecture. The architecture consists of measurement of a global quantity in a colony of bacteria as an output feedback and manipulation of global environmental variables as control actuation. We show that controller design can be performed on the abstracted Markov chain model and implementation on the real model yields the desired result.</div>','<b>Title: </b>A New Model for Investigating the Evolution of Transcription Control Networks<br><b>Year: </b>2009<br><b>Author: </b> Dj. Jenkins and  Dj. Stekel<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1162/artl.2009.Stekel.006"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ARTIFICIAL LIFE<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Biological systems show unbounded capacity for complex behaviors and responses to their environments. This principally arises from their genetic networks. The processes governing transcription, translation, and gene regulation are well understood, as are the mechanisms of network evolution, such as gene duplication and horizontal gene transfer. However, the evolved networks arising from these simple processes are much more difficult to understand, and it is difficult to perform experiments on the evolution of these networks in living organisms because of the timescales involved. We propose a new framework for modeling and investigating the evolution of transcription networks in realistic, varied environments. The model we introduce contains novel, important, and lifelike features that allow the evolution of arbitrarily complex transcription networks. Molecular interactions are not specified; instead they are determined dynamically based on shape, allowing protein function to freely evolve. Transcriptional logic provides a flexible mechanism for defining genetic regulatory activity. Simulations demonstrate a realistic life cycle as an emergent property, and that even in simple environments lifelike and complex regulation mechanisms are evolved, including stable proteins, unstable mRNA, and repressor activity. This study also highlights the importance of using in silico genetics techniques to investigate evolved model robustness.</div>','<b>Title: </b>Systems approaches to coronavirus pathogenesis<br><b>Year: </b>2014<br><b>Author: </b> A. Schafer,  Rs. Baric and  Mt. Ferris<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.coviro.2014.04.007"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>CURRENT OPINION IN VIROLOGY<br><b>Cluster-Keywords: </b><em>interaction network, actin filament, network model, filament network, network structures, actin network, protein interaction, actin dynamics, network, membrane protein, network dynamics, model dynamic, cell, model, network approach, protein complex, computational model, actin, adhesion dynamics, cell dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>Coronaviruses comprise a large group of emergent human and animal pathogens, including the highly pathogenic SARS-CoV and MERS-CoV strains that cause significant morbidity and mortality in infected individuals, especially the elderly. As emergent viruses may cause episodic outbreaks of disease over time, human samples are limited. Systems biology and genetic technologies maximize opportunities for identifying critical host and viral genetic factors that regulate susceptibility and virus-induced disease severity. These approaches provide discovery platforms that highlight and allow targeted confirmation of critical targets for prophylactics and therapeutics, especially critical in an outbreak setting. Although poorly understood, it has long been recognized that host regulation of virus-associated disease severity is multigenic. The advent of systems genetic and biology resources provides new opportunities for deconvoluting the complex genetic interactions and expression networks that regulate pathogenic or protective host response patterns following virus infection. Using SARS-CoV as a model, dynamic transcriptional network changes and disease-associated phenotypes have been identified in different genetic backgrounds, leading to the promise of population-wide discovery of the underpinnings of Coronavirus pathogenesis.</div>','<b>Title: </b>Discovering Regulatory Network Topologies Using Ensemble Methods on GPGPUs With Special Reference to the Biological Clock of Neurospora crassa<br><b>Year: </b>2015<br><b>Author: </b> A. Al-Omari,  J. Griffith,  M. Judge,  T. Taha,  J. Arnold and  Hb. Schuttler<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1109/ACCESS.2015.2399854"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IEEE ACCESS<br><b>Cluster-Keywords: </b><em>circadian rhythms, systems biology, biological clock, circadian clock, biological rhythms, hpa axis, model, clock network, mathematical models, biological systems, dynamic systems, ode models, biology models, single cell, network, model parameters, experimental data, models, systems, dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>Most genetic networks, such as that for the biological clock, are part of much larger modules controlling fundamental processes in the cell, such as metabolism, development, and response to environmental signals. For example, the biological clock is part of a much larger network controlling the circadian rhythms of about 2418 distinct genes in the genome with 11 000 genes of the model system, Neurospora crassa. Predicting and understanding the dynamics of all of these genes and their products in a genetic network describing how the clock functions is a challenge and beyond the current capability of the fastest serial computers. We have implemented a novel variable-topology supernet ensemble method using Markov chain Monte Carlo simulations to fit and discover a regulatory network of unknown topology composed of 2418 genes describing the entire clock circadian network, a network that is found in organisms ranging from bacteria to humans, by harnessing the power of the general-purpose graphics processing unit and exploiting the hierarchical structure of that genetic network. The result is the construction of a genetic network that explains mechanistically how the biological clock functions in the filamentous fungus N. crassa and is validated against over 31 000 data points from microarray experiments. Two transcription factors are identified targeting ribosome biogenesis in the clock network.</div>','<b>Title: </b>The Present and the Future Perspectives of Biological Network Inference<br><b>Year: </b>2012<br><b>Author: </b> P. Lecca and  A. Palmisano<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.4018/978-1-61350-435-2.ch006"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SYSTEMIC APPROACHES IN BIOINFORMATICS AND COMPUTATIONAL SYSTEMS BIOLOGY: RECENT ADVANCES<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Biological network inference is based on a series of studies and computational approaches to the deduction of the connectivity of chemical species, the reaction pathway, and the reaction kinetics of complex reaction systems from experimental measurements. Inference for network structure and reaction kinetics parameters governing the dynamics of a biological system is currently an active area of research. In the era of post-genomic biology, it is a common opinion among scientists that living systems cells, tissues, organs and organisms can be understood in terms of their network structure as well as in term of the evolution in time of this network structure. In this chapter, the authors make a survey of the recent methodologies proposed for the structure inference and for the parameter estimation of a system of interacting biological entities. Furthermore, they present the recent works of the authors about model identification and calibration.</div>','<b>Title: </b>Response of an actin filament network model under cyclic stretching through a coarse grained Monte Carlo approach<br><b>Year: </b>2011<br><b>Author: </b> J. Kang,  Rl. Steward,  Y. Kim,  Rs. Schwartz,  Pr. Leduc and  Km. Puskar<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.jtbi.2011.01.011"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF THEORETICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>interaction network, actin filament, network model, filament network, network structures, actin network, protein interaction, actin dynamics, network, membrane protein, network dynamics, model dynamic, cell, model, network approach, protein complex, computational model, actin, adhesion dynamics, cell dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>Cells are complex, dynamic systems that actively adapt to various stimuli including mechanical alterations. Central to understanding cellular response to mechanical stimulation is the organization of the cytoskeleton and its actin filament network. In this manuscript, we present a minimalistic network Monte Carlo based approach to model actin filament organization under cyclic stretching. Utilizing a coarse-grained model, a filament network is prescribed within a two-dimensional circular space through nodal connections. When cyclically stretched, the model demonstrates that a perpendicular alignment of the filaments to the direction of stretch emerges in response to nodal repositioning to minimize net nodal forces from filament stress states. In addition, the filaments in the network rearrange and redistribute themselves to reduce the overall stress by decreasing their individual stresses. In parallel, we cyclically stretch NIH 3T3 fibroblasts and find a similar cytoskeletal response. With this work, we test the hypothesis that a first-principles mechanical model of filament assembly in a confined space is by itself capable of yielding the remodeling behavior observed experimentally. Identifying minimal mechanisms sufficient to reproduce mechanical influences on cellular structure has important implications in a diversity of fields, including biology, physics, medicine, computer science, and engineering. C 2011 Elsevier Ltd. All rights reserved.</div>','<b>Title: </b>Solving Large Nonlinear Systems of First-Order Ordinary Differential Equations With Hierarchical Structure Using Multi-GPGPUs and an Adaptive Runge Kutta ODE Solver<br><b>Year: </b>2013<br><b>Author: </b> A. Al-Omari,  J. Arnold,  T. Taha and  Hb. Schuttler<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1109/ACCESS.2013.2290623"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IEEE ACCESS<br><b>Cluster-Keywords: </b><em>circadian rhythms, systems biology, biological clock, circadian clock, biological rhythms, hpa axis, model, clock network, mathematical models, biological systems, dynamic systems, ode models, biology models, single cell, network, model parameters, experimental data, models, systems, dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>The adaptive Runge Kutta ARK method on multi-general-purpose graphical processing units GPUs is used for solving large nonlinear systems of first-order ordinary differential equations ODEs with over similar to 10 000 variables describing a large genetic network in systems biology for the biological clock. To carry out the computation of the trajectory of the system, a hierarchical structure of the ODEs is exploited, and an ARK solver is implemented in compute unified device architecture/C CUDA/C on GPUs. The result is a 75-fold speedup for calculations of 2436 independent modules within the genetic network describing clock function relative to a comparable CPU architecture. These 2436 modules span one-quarter of the entire genome of a model fungal system, Neurospora crassa. The power of a GPU can in principle be harnessed by using warp-level parallelism, instruction level parallelism or both of them. Since the ARK ODE solver is entirely sequential, we propose a new parallel processing algorithm using warp-level parallelism for solving 10 000 ODEs that belong to a large genetic network describing clock genome-level dynamics. A video is attached illustrating the general idea of the method on GPUs that can be used to provide new insights into the biological clock through single cell measurements on the clock.</div>','<b>Title: </b>A journey from reductionist to systemic cell biology aboard the schooner Tara<br><b>Year: </b>2012<br><b>Author: </b> E. Karsenti<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1091/mbc.E11-06-0571"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MOLECULAR BIOLOGY OF THE CELL<br><b>Cluster-Keywords: </b><em>network motifs, regulatory network, gene regulatory, systems biology, gene network, bifan motif, complex systems, model comparison, network modeling, complex gene, biological systems, dynamic systems, interaction network, feedforward loop, modeling gene, intelligent systems, modeling approach, biological pathways, motif model, model motifs<div style="font-size:11px;"><br><b>Abstract: </b><em>In this essay I describe my personal journey from reductionist to systems cell biology and describe how this in turn led to a 3-year sea voyage to explore complex ocean communities. In describing this journey, I hope to convey some important principles that I gleaned along the way. I realized that cellular functions emerge from multiple molecular interactions and that new approaches borrowed from statistical physics are required to understand the emergence of such complex systems. Then I wondered how such interaction networks developed during evolution. Because life first evolved in the oceans, it became a natural thing to start looking at the small organisms that compose the plankton in the worlds oceans, of which 98 are ... individual cells-hence the Tara Oceans voyage, which finished on 31 March 2012 in Lorient, France, after a 60,000-mile around-the-world journey that collected more than 30,000 samples from 153 sampling stations.</div>','<b>Title: </b>Elucidation of functional consequences of signalling pathway interactions<br><b>Year: </b>2009<br><b>Author: </b> Aec. Ihekwaba,  Pt. Nguyen and  C. Priami<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1471-2105-10-370"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: A great deal of data has accumulated on signalling pathways. These large datasets are thought to contain much implicit information on their molecular structure, interaction and activity information, which provides a picture of intricate molecular networks believed to underlie biological functions. While tremendous advances have been made in trying to understand these systems, how information is transmitted within them is still poorly understood. This ever growing amount of data demands we adopt powerful computational techniques that will play a pivotal role in the conversion of mined data to knowledge, and in elucidating the topological and functional properties of protein - protein interactions.Results: A computational framework is presented which allows for the description of embedded networks, and identification of common shared components thought to assist in the transmission of information within the systems studied. By employing the graph theories of network biology such as degree distribution, clustering coefficient, vertex betweenness and shortest path measures - topological features of protein-protein interactions for published datasets of the p53, nuclear factor kappa B NF-kappa B and GI/S phase of the cell cycle systems were ascertained. Highly ranked nodes which in some cases were identified as connecting proteins most likely responsible for propagation of transduction signals across the networks were determined. The functional consequences of these nodes in the context of their network environment were also determined. These findings highlight the usefulness of the framework in identifying possible combination or links as targets for therapeutic responses; and put forward the idea of using retrieved knowledge on the shared components in constructing better organised and structured models of signalling networks.Conclusion: It is hoped that through the data mined reconstructed signal transduction networks, well developed models of the published data can be built which in the end would guide the prediction of new targets based on the pathways environment for further analysis. Source code is available upon request.</div>','<b>Title: </b>A Novel Knowledge-Driven Systems Biology Approach for Phenotype Prediction upon Genetic Intervention<br><b>Year: </b>2011<br><b>Author: </b> R. Chang,  R. Shoemaker and  W. Wang<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1109/TCBB.2011.18"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IEEE-ACM TRANSACTIONS ON COMPUTATIONAL BIOLOGY AND BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>boolean networks, regulatory networks, network model, bayesian networks, model networks, systems biology, random boolean, real cells, immune system, genetic regulatory, networks, biochemical network, fluorescence images, cellular network, gene regulatory, cell differentiation, network, biological systems, ensemble approach, cell dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>Deciphering the biological networks underlying complex phenotypic traits, e. g., human disease is undoubtedly crucial to understand the underlying molecular mechanisms and to develop effective therapeutics. Due to the network complexity and the relatively small number of available experiments, data-driven modeling is a great challenge for deducing the functions of genes/proteins in the network and in phenotype formation. We propose a novel knowledge-driven systems biology method that utilizes qualitative knowledge to construct a Dynamic Bayesian network DBN to represent the biological network underlying a specific phenotype. Edges in this network depict physical interactions between genes and/or proteins. A qualitative knowledge model first translates typical molecular interactions into constraints when resolving the DBN structure and parameters. Therefore, the uncertainty of the network is restricted to a subset of models which are consistent with the qualitative knowledge. All models satisfying the constraints are considered as candidates for the underlying network. These consistent models are used to perform quantitative inference. By in silico inference, we can predict phenotypic traits upon genetic interventions and perturbing in the network. We applied our method to analyze the puzzling mechanism of breast cancer cell proliferation network and we accurately predicted cancer cell growth rate upon manipulating anti cancerous marker genes/proteins.</div>','<b>Title: </b>The genomic regulatory control of skeletal morphogenesis in the sea urchin<br><b>Year: </b>2012<br><b>Author: </b> K. Rafiq,  Ms. Cheers and  Ca. Ettensohn<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1242/dev.073049"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>DEVELOPMENT<br><b>Cluster-Keywords: </b><em>gene regulatory, regulatory network, systems biology, cell differentiation, cell fate, model systems, potential landscape, network grn, dynamical systems, melanocyte differentiation, network models, gene expression, network dynamics, differentiation genes, dynamical models, dynamical networks, cellular differentiation, stem cells, attractor network, developmental gene<div style="font-size:11px;"><br><b>Abstract: </b><em>A central challenge of developmental and evolutionary biology is to understand how anatomy is encoded in the genome. Elucidating the genetic mechanisms that control the development of specific anatomical features will require the analysis of model morphogenetic processes and an integration of biological information at genomic, cellular and tissue levels. The formation of the endoskeleton of the sea urchin embryo is a powerful experimental system for developing such an integrated view of the genomic regulatory control of morphogenesis. The dynamic cellular behaviors that underlie skeletogenesis are well understood and a complex transcriptional gene regulatory network GRN that underlies the specification of embryonic skeletogenic cells primary mesenchyme cells, PMCs has recently been elucidated. Here, we link the PMC specification GRN to genes that directly control skeletal morphogenesis. We identify new gene products that play a proximate role in skeletal morphogenesis and uncover transcriptional regulatory inputs into many of these genes. Our work extends the importance of the PMC GRN as a model developmental GRN and establishes a unique picture of the genomic regulatory control of a major morphogenetic process. Furthermore, because echinoderms exhibit diverse programs of skeletal development, the newly expanded sea urchin skeletogenic GRN will provide a foundation for comparative studies that explore the relationship between GRN evolution and morphological evolution.</div>','<b>Title: </b>Identifying key species in ecosystems with stochastic sensitivity analysis<br><b>Year: </b>2011<br><b>Author: </b> Cm. Livi,  F. Jordan,  P. Lecca and  Ta. Okey<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.ecolmodel.2010.09.025"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ECOLOGICAL MODELLING<br><b>Cluster-Keywords: </b><em>network ecology, food webs, network models, ecological systems, food web, systems biology, ecological networks, ecosystem model, signalling networks, model results, interaction networks, dynamic model, ecological interactions, network analysis, model network, network structure, transport networks, systems network, ecosystem dynamics, restoration ecology<div style="font-size:11px;"><br><b>Abstract: </b><em>The development of approaches to estimate the vulnerability of biological communities and ecosystems to extirpations and reductions of species is a central challenge of conservation biology. One key aim of this challenge is to develop quantitative approaches to estimate and rank interaction strengths and keystoneness of species and functional groups, i.e. to quantify the relative importance of species. Network analysis can be a powerful tool for this because certain structural aspects of ecological networks are good indicators of the mechanisms that maintain co-evolved, biotic interactions. A static view of ecological networks would lead us to focus research on highly-central species in food webs topological key players in ecosystems. There are a variety of centrality indices, developed for several types of ecological networks e.g. for weighted and un-weighted webs. However, truly understanding extinction and its community-wide effects requires the use of dynamic models. Deterministic dynamic models are feasible when population sizes are sufficiently large to minimize noise in the overall system. In models with small population sizes, stochasticity can be modelled explicitly. We present a stochastic simulation-based ecosystem model for identification of "dynamic key species" in situations where stochastic models are appropriate. To demonstrate this approach, we simulated ecosystem dynamics and performed sensitivity analysis using data from the Prince William Sound, Alaska ecosystem model. We then compare these results to those of purely topological analyses and deterministic dynamic Ecosim studies. We present the relationships between various topological and dynamic indices and discuss their biological relevance. The trophic group with the largest effect on others is nearshore demersals, the species mostly sensitive to others is halibut, and the group of both considerable effect on and sensitivity to others is juvenile herring. The most important trophic groups in our dynamical simulations appear to have intermediate trophic levels. C 2010 Elsevier B.V. All rights reserved.</div>','<b>Title: </b>Attractor Landscape: A Bridge between Robotics and Synthetic Biology<br><b>Year: </b>2018<br><b>Author: </b> A. Roli and  M. Braccini<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.25088/ComplexSystems.27.3.229"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>COMPLEX SYSTEMS<br><b>Cluster-Keywords: </b><em>gene regulatory, regulatory network, systems biology, cell differentiation, cell fate, model systems, potential landscape, network grn, dynamical systems, melanocyte differentiation, network models, gene expression, network dynamics, differentiation genes, dynamical models, dynamical networks, cellular differentiation, stem cells, attractor network, developmental gene<div style="font-size:11px;"><br><b>Abstract: </b><em>Genetic regulatory networks GRNs model the dynamics and interactions among genes. From a robotics perspective, GRNs are extremely interesting because they are capable of producing complex behaviors. Notably, cell differentiation can be modeled using GRNs, and the dynamics of this process can be studied by means of dynamical systems methods. In a nutshell, the state of a cell is represented by an attractor in the state space of a dynamical system, and the transitions between cell states correspond to transitions between attractors. This view suggests a visionary approach: apply the metaphor of landscape attractor to design specific cell dynamics that can match the attractor landscape required for attaining a target behavior in a robotic system. The constraints prescribed by the robotic application are just the correspondence between behavioral attractors in the robot and cell attractors in the cell, along with specific transitions between attractors. This perspective may lead to applications in biorobotics, and it may also help synthetic biology systems design, which may benefit from methods developed for complex dynamical systems. We believe that this level of abstraction can provide a common vocabulary and a shared set of categories between researchers in robotics and synthetic biology. In this paper, we elaborate on previous results on GRNs-controlled robots and propose some guidelines for making this approach viable, illustrating these concepts with examples and case studies in biorobotics.</div>','<b>Title: </b>Enhancing the role of veterinary vaccines reducing zoonotic diseases of humans: Linking systems biology with vaccine development<br><b>Year: </b>2011<br><b>Author: </b> Lg. Adams,  S. Khare,  Sd. Lawhon,  Ca. Rossetti,  Ha. Lewin,  Ms. Lipton,  Je. Turse,  Dc. Wylie,  Y. Bai and  Kl. Drake<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.vaccine.2011.05.080"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>VACCINE<br><b>Cluster-Keywords: </b><em>gene expression, systems biology, metabolic network, network model, network analysis, dynamic model, signaling pathways, signaling network, regulatory network, dynamic network, biology approach, candidate genes, gene network, gene regulatory, biology analysis, mathematical model, network dynamic, biology network, metabolic pathways, pathway analysis<div style="font-size:11px;"><br><b>Abstract: </b><em>The aim of research on infectious diseases is their prevention, and brucellosis and salmonellosis as such are classic examples of worldwide zoonoses for application of a systems biology approach for enhanced rational vaccine development. When used optimally, vaccines prevent disease manifestations, reduce transmission of disease, decrease the need for pharmaceutical intervention, and improve the health and welfare of animals, as well as indirectly protecting against zoonotic diseases of people. Advances in the last decade or so using comprehensive systems biology approaches linking genomics, proteomics, bioinformatics, and biotechnology with immunology, pathogenesis and vaccine formulation and delivery are expected to enable enhanced approaches to vaccine development. The goal of this paper is to evaluate the role of computational systems biology analysis of host:pathogen interactions the interactome as a tool for enhanced rational design of vaccines. Systems biology is bringing a new, more robust approach to veterinary vaccine design based upon a deeper understanding of the host-pathogen interactions and its impact on the hosts molecular network of the immune system. A computational systems biology method was utilized to create interactome models of the host responses to Brucella melitensis BMEL, Mycobacterium avium paratuberculosis MAP, Salmonella enterica Typhimurium STM, and a Salmonella mutant isogenic Delta sipA,sopABDE2and linked to the basis for rational development of vaccines for brucellosis and salmonellosis as reviewed by Adams et al. and Ficht etal. 1,2. A bovine ligated ileal loop biological model was established to capture the host gene expression response at multiple time points post infection. New methods based on Dynamic Bayesian Network DBN machine learning were employed to conduct a comparative pathogenicity analysis of 219 signaling and metabolic pathways and 1620 gene ontology GO categories that defined the hosts biosignatures to each infectious condition. Through this DBN computational approach, the method identified significantly perturbed pathways and GO category groups of genes that define the pathogenicity signatures of the infectious agent. Our preliminary results provide deeper understanding of the overall complexity of host innate immune response as well as the identification of host gene perturbations that defines a unique host temporal biosignature response to each pathogen. The application of advanced computational methods for developing interactome models based on DBNs has proven to be instrumental in elucidating novel host responses and improved functional biological insight into the host defensive mechanisms. Evaluating the unique differences in pathway and GO perturbations across pathogen conditions allowed the identification of plausible host-pathogen interaction mechanisms. Accordingly, a systems biology approach to study molecular pathway gene expression profiles of host cellular responses to microbial pathogens holds great promise as a methodology to identify, model and predict the overall dynamics of the host-pathogen interactome. Thus, we propose that such an approach has immediate application to the rational design of brucellosis and salmonellosis vaccines. C 2011 Elsevier Ltd. All rights reserved.</div>','<b>Title: </b>Dimensionality Reduction of Bistable Biological Systems<br><b>Year: </b>2013<br><b>Author: </b> A. Zakharova,  Z. Nikoloski and  A. Koseska<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s11538-013-9807-8"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BULLETIN OF MATHEMATICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>immune system, network models, immune response, model, biological systems, models, model reduction, dynamic model, biological networks, system, mathematical models, biochemical network, response model, systems, systems biology, network, immune, biology model, reaction networks, system systems<div style="font-size:11px;"><br><b>Abstract: </b><em>Time hierarchies, arising as a result of interactions between systems components, represent a ubiquitous property of dynamical biological systems. In addition, biological systems have been attributed switch-like properties modulating the response to various stimuli across different organisms and environmental conditions. Therefore, establishing the interplay between these features of system dynamics renders itself a challenging question of practical interest in biology. Existing methods are suitable for systems with one stable steady state employed as a well-defined reference. In such systems, the characterization of the time hierarchies has already been used for determining the components that contribute to the dynamics of biological systems. However, the application of these methods to bistable nonlinear systems is impeded due to their inherent dependence on the reference state, which in this case is no longer unique. Here, we extend the applicability of the reference-state analysis by proposing, analyzing, and applying a novel method, which allows investigation of the time hierarchies in systems exhibiting bistability. The proposed method is in turn used in identifying the components, other than reactions, which determine the systemic dynamical properties. We demonstrate that in biological systems of varying levels of complexity and spanning different biological levels, the method can be effectively employed for model simplification while ensuring preservation of qualitative dynamical properties i.e., bistability. Finally, by establishing a connection between techniques from nonlinear dynamics and multivariate statistics, the proposed approach provides the basis for extending reference-based analysis to bistable systems.</div>','<b>Title: </b>Glycosylation Network Analysis Toolbox: a MATLAB-based environment for systems glycobiology<br><b>Year: </b>2013<br><b>Author: </b> G. Liu,  A. Puri and  S. Neelamegham<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1093/bioinformatics/bts703"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>A: Systems glycobiology studies the interaction of various pathways that regulate glycan biosynthesis and function. Software tools for the construction and analysis of such pathways are not yet available. We present GNAT, a platform-independent, user-extensible MATLAB-based toolbox that provides an integrated computational environment to construct, manipulate and simulate glycans and their networks. It enables integration of XML-based glycan structure data into SBML Systems Biology Markup Language files that describe glycosylation reaction networks. Curation and manipulation of networks is facilitated using class definitions and glycomics database query tools. High quality visualization of networks and their steady-state and dynamic simulation are also supported.</div>','<b>Title: </b>A dynamical model of the spindle position checkpoint<br><b>Year: </b>2012<br><b>Author: </b> Ak. Caydasi,  M. Lohel,  G. Grunert,  P. Dittrich,  G. Pereira and  B. Ibrahim<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1038/msb.2012.15"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MOLECULAR SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>regulatory networks, stochastic simulation, gene networks, reaction networks, systems biology, stochastic model, network models, complex systems, networks stochastic, gene regulatory, biological systems, stochastic dynamics, differential equations, protein networks, cell cycle, continuous model, interaction networks, systems chemistry, time delays, biochemical reaction<div style="font-size:11px;"><br><b>Abstract: </b><em>The orientation of the mitotic spindle with respect to the polarity axis is crucial for the accuracy of asymmetric cell division. In budding yeast, a surveillance mechanism called the spindle position checkpoint SPOC prevents exit from mitosis when the mitotic spindle fails to align along the mother-to-daughter polarity axis. SPOC arrest relies upon inhibition of the GTPase Tem1 by the GTPase-activating protein GAP complex Bfa1-Bub2. Importantly, reactions signaling mitotic exit take place at yeast centrosomes named spindle pole bodies, SPBs and the GAP complex also promotes SPB localization of Tem1. Yet, whether the regulation of Tem1 by Bfa1-Bub2 takes place only at the SPBs remains elusive. Here, we present a quantitative analysis of Bfa1-Bub2 and Tem1 localization at the SPBs. Based on the measured SPB-bound protein levels, we introduce a dynamical model of the SPOC that describes the regulation of Bfa1 and Tem1. Our model suggests that Bfa1 interacts with Tem1 in the cytoplasm as well as at the SPBs to provide efficient Tem1 inhibition. Molecular Systems Biology 8: 582; published online 8 May 2012; doi:10.1038/msb.2012.15</div>','<b>Title: </b>Evolution of early development in dipterans: Reverse-engineering the gap gene network in the moth midge Clogmia albipunctata Psychodidae<br><b>Year: </b>2014<br><b>Author: </b> A. Crombach,  Ma. Garcia-Solache and  J. Jaeger<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.biosystems.2014.06.003"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOSYSTEMS<br><b>Cluster-Keywords: </b><em>network models, gene network, regulatory networks, boolean networks, gene expression, systems biology, neural networks, network dynamics, gene regulatory, biological networks, network structure, gap gene, gene regulation, cell cycle, network, gene interactions, signaling network, quantum systems, system dynamics, networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Understanding the developmental and evolutionary dynamics of regulatory networks is essential if we are to explain the non-random distribution of phenotypes among the diversity of organismic forms. Here, we present a comparative analysis of one of the best understood developmental gene regulatory networks today: the gap gene network involved in early patterning of insect embryos. We use gene circuit models, which are fitted to quantitative spatio-temporal gene expression data for the four trunk gap genes hunchback hb, Kruppel Kr, giant gt, and knirps kni/knirps-like kill in the moth midge Clogmia albipunctata, and compare them to equivalent reverse-engineered circuits from our reference species, the vinegar fly Drosophila melanogaster. In contrast to the single network structure we find for D. melanogaster, our models predict four alternative networks for C. albipunctata. These networks share a core structure, which includes the central regulatory feedback between hb and knl. Other interactions are only partially determined, as they differ between our four network structures. Nevertheless, our models make testable predictions and enable us to gain specific insights into gap gene regulation in C albipunctata. They suggest a less central role for Kr in C albipunctata than in D. melanogaster, and show that the mechanisms causing an anterior shift of gap domains over time are largely conserved between the two species, although shift dynamics differ. The set of C albipunctata gene circuit models presented here will be used as the starting point for data-constrained in silico evolutionary simulations to study patterning transitions in the early development of dipteran species. C 2014 The Authors. Published by Elsevier Ireland Ltd. This is an open access article under the CC BY license http://creativecommons.org/licenses/by/3.0/.</div>','<b>Title: </b>Computational Modeling of Mitochondrial Function<br><b>Year: </b>2012<br><b>Author: </b> S. Cortassa and  Ma. Aon<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/978-1-61779-382-019"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MITOCHONDRIAL BIOENERGETICS: METHODS AND PROTOCOLS<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>The advent of techniques with the ability to scan massive changes in cellular makeup genomics, proteomics, etc. has revealed the compelling need for analytical methods to interpret and make sense of those changes. Computational models built on sound physico-chemical mechanistic basis are unavoidable at the time of integrating, interpreting, and simulating high-throughput experimental data. Another powerful role of computational models is predicting new behavior provided they are adequately validated.Mitochondrial energy transduction has been traditionally studied with thermodynamic models. More recently, kinetic or thermo-kinetic models have been proposed, leading the path toward an understanding of the control and regulation of mitochondrial energy metabolism and its interaction with cytoplasmic and other compartments. In this work, we outline the methods, step-by-step, that should be followed to build a computational model of mitochondrial energetics in isolation or integrated to a network of cellular processes. Depending on the question addressed by the modeler, the methodology explained herein can be applied with different levels of detail, from the mitochondrial energy producing machinery in a network of cellular processes to the dynamics of a single enzyme during its catalytic cycle.</div>','<b>Title: </b>Modeling Avena fatua seedling emergence dynamics: An artificial neural network approach<br><b>Year: </b>2012<br><b>Author: </b> Gr. Chantre,  Am. Blanco,  Mv. Lodovichi,  Aj. Bandoni,  Mr. Sabbatini,  Rl. Lopez,  Mr. Vigna and  R. Gigon<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.compag.2012.07.005"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>COMPUTERS AND ELECTRONICS IN AGRICULTURE<br><b>Cluster-Keywords: </b><em>reaction networks, network model, systems biology, biochemical reaction, stochastic reaction, model reduction, stochastic model, network dynamics, stochastic biochemical, stochastic simulation, system size, chemical reaction, gene network, elementary reactions, biochemical networks, parameter identification, parameter inference, cell populations, small system, linear noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Avena fatua is an invasive weed of the semiarid region of Argentina. Seedling emergence patterns are very irregular along the season showing a great year-to-year variability mainly due to a highly unpredictable precipitation regime. Non-linear regression techniques are usually unable to accurately predict field emergence under such environmental conditions. Artificial Neural Networks ANNs are known for their capacity to describe highly non-linear relationships among variables thus showing a high potential applicability in ecological systems. The objectives of the present work were to develop different ANN models for A. fatua seedling emergence prediction and to compare their predictive capability against non-linear regression techniques. Classical hydrothermal-time indices were used as input variable for the development of univariate models, while thermal-time and hydro-time were used as independent input variables for developing bivariate models. The accumulated proportion of seedling emergence was the output variable in all cases. A total of 528 input/output data pairs corresponding to 11 years of data collection were used in this study. Obtained results indicate a higher accuracy and generalization performance of the optimal ANN model in comparison to non-linear regression approaches. It is also demonstrated that the use of thermal-time and hydro-time as independent explanatory variables in ANN models yields better prediction than using combined hydrothermal-time indices in classical NLR models. The best obtained ANN model outperformed in 43.3 the best NLR model in terms of RMSE of the test set. Moreover, the best obtained ANN predicted accumulated emergence within the first 50 of total emergence 48.3 better in average than the best developed NLR model. These outcomes suggest the potential applicability of the proposed modeling approach in weed management decision support systems design. C 2012 Elsevier B.V. All rights reserved.</div>','<b>Title: </b>Boolean network modeling in systems pharmacology<br><b>Year: </b>2018<br><b>Author: </b> P. Bloomingdale,  Va. Nguyen,  J. Niu and  De. Mager<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s10928-017-9567-4"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF PHARMACOKINETICS AND PHARMACODYNAMICS<br><b>Cluster-Keywords: </b><em>boolean network, network models, qualitative models, systems biology, mathematical modeling, boolean models, signaling networks, dynamic models, dynamic modeling, signaling dynamics, logic models, dynamic network, mathematical models, boolean modeling, signal transduction, regulatory networks, drug resistance, biological networks, network approach, network modeling<div style="font-size:11px;"><br><b>Abstract: </b><em>Quantitative systems pharmacology QSP is an emerging discipline that aims to discover how drugs modulate the dynamics of biological components in molecular and cellular networks and the impact of those perturbations on human pathophysiology. The integration of systems-based experimental and computational approaches is required to facilitate the advancement of this field. QSP models typically consist of a series of ordinary differential equations ODE. However, this mathematical framework requires extensive knowledge of parameters pertaining to biological processes, which is often unavailable. An alternative framework that does not require knowledge of system-specific parameters, such as Boolean network modeling, could serve as an initial foundation prior to the development of an ODE-based model. Boolean network models have been shown to efficiently describe, in a qualitative manner, the complex behavior of signal transduction and gene/protein regulatory processes. In addition to providing a starting point prior to quantitative modeling, Boolean network models can also be utilized to discover novel therapeutic targets and combinatorial treatment strategies. Identifying drug targets using a network-based approach could supplement current drug discovery methodologies and help to fill the innovation gap across the pharmaceutical industry. In this review, we discuss the process of developing Boolean network models and the various analyses that can be performed to identify novel drug targets and combinatorial approaches. An example for each of these analyses is provided using a previously developed Boolean network of signaling pathways in multiple myeloma. Selected examples of Boolean network models of human patho-physiological systems are also reviewed in brief.</div>','<b>Title: </b>The Biological System of the Elements BSE. Part II: a theoretical model for establishing the essentiality of chemical elements. The application of stoichiometric network analysis to the biological system of the elements<br><b>Year: </b>2000<br><b>Author: </b> S. Franzle and  B. Markert<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/S0048-96979900520-3"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SCIENCE OF THE TOTAL ENVIRONMENT<br><b>Cluster-Keywords: </b><em>biological system, network, pattern formation, system, network analysis, systems, turing patterns, patterns, reaction networks, networks, turing system, network topologies, stoichiometric network, networks pattern, network topology, biological, model, gradient formation, pattern, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Stoichiometric Network Analysis SNA, originally developed by the Canadian chemist Bruce L. Clarke during the 1970s, provides a most efficient means of reducing the background topology of complex interaction networks to some skeleton topology around which systems dynamics can be understood without jeopardising insight into complex dynamics by over- or miss-simplification, Since it focuses on the corresponding autocatalytic AC features of a feedback system as those which control overall behaviour to some extent, SNA deals with reaction kinetics in and beyond chemistry, e.g, with nuclear reactions. It is therefore quite straightforward to apply this manner of simplification, which in turn is supported by a number of mathematical theorems on systems behaviour and properties of AC cycles, to biological systems although their full complexity may not even be assessed in the yet rare cases of complete genetic sequencing. Assuming there is a relationship between the kinds of metal or metalloid species and key biological/biochemical transformations to be promoted with their aid - this relationship being the subject of bio-inorganic chemistry - and that biochemistry is, in effect, about systems which can reproduce and thus behave autocatalytically, one can expect SNA to yield formally sound statements on basic features of biology and biochemistry too. if we sum up the facts and considerations concerning essentiality or possible essentiality in a biological system of elements Markert, 1993, this means joining the triangular representation of BSE, including statements on the degree of biological evolution and aggregation levels, to SNA treatment of autocatalysis within hierarchical systems from metalloenzymes to entire biocoenoses. Arguments using preferred cluster sizes and aggregation tendencies from coordination chemistry are then employed to circumscribe possible functions within the BSE. They are also extended to metals hitherto not known to be essential, such as tellurium or scandium. C 2000 Elsevier Science B.V. All rights reserved.</div>','<b>Title: </b>Multi-level model for the investigation of oncoantigen-driven vaccination effect<br><b>Year: </b>2013<br><b>Author: </b> F. Cordero,  M. Beccuti,  C. Fornari,  S. Lanzardo,  L. Conti,  F. Cavallo,  G. Balbo and  R. Calogero<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1471-2105-14-S6-S11"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>petri net, petri nets, bayesian networks, biological networks, systems biology, regulatory networks, dynamic bayesian, net model, gene regulatory, network data, biological systems, biological data, network analysis, network models, biochemical networks, time series, network structure, signal transduction, net models, dbn model<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Cancer stem cell theory suggests that cancers are derived by a population of cells named Cancer Stem Cells CSCs that are involved in the growth and in the progression of tumors, and lead to a hierarchical structure characterized by differentiated cell population. This cell heterogeneity affects the choice of cancer therapies, since many current cancer treatments have limited or no impact at all on CSC population, while they reveal a positive effect on the differentiated cell populations.Results: In this paper we investigated the effect of vaccination on a cancer hierarchical structure through a multi-level model representing both population and molecular aspects. The population level is modeled by a system of Ordinary Differential Equations ODEs describing the cancer populations dynamics. The molecular level is modeled using the Petri Net PN formalism to detail part of the proliferation pathway. Moreover, we propose a new methodology which exploits the temporal behavior derived from the molecular level to parameterize the ODE system modeling populations. Using this multi-level model we studied the ErbB2-driven vaccination effect in breast cancer.Conclusions: We propose a multi-level model that describes the inter-dependencies between population and genetic levels, and that can be efficiently used to estimate the efficacy of drug and vaccine therapies in cancer models, given the availability of molecular data on the cancer driving force.</div>','<b>Title: </b>Computer forensics system based on artificial immune systems<br><b>Year: </b>2007<br><b>Author: </b> J. Yang,  T. Li,  Sj. Liu,  Tf. Wang,  Dg. Wang and  G. LiangLink: No link<br><b>Journal: </b>JOURNAL OF UNIVERSAL COMPUTER SCIENCE<br><b>Cluster-Keywords: </b><em>immune system, network models, immune response, model, biological systems, models, model reduction, dynamic model, biological networks, system, mathematical models, biochemical network, response model, systems, systems biology, network, immune, biology model, reaction networks, system systems<div style="font-size:11px;"><br><b>Abstract: </b><em>The current computer forensics approaches mainly focus on the network actions capture and analysis the evidences after attacks, which always result in the static methods. Inspired by the theory of artificial immune systems AIS, a novel model of Computer Forensics System is presented. The concepts and formal definitions of immune cells are given, and dynamically evaluative equations for self, antigen, immune tolerance, mature-lymphocyte lifecycle and immune memory are presented, and the hierarchical and distributed management framework of the proposed model are built. Furthermore, the idea of biology immunity is applied for enhancing the self-adapting and self-learning ability to adapt continuously variety environments. The experimental results show that the proposed model has the features of real-time processing, self-adaptively, thus providing a promising solution for computer forensics.</div>','<b>Title: </b>Functional Aspects of the EGF-Induced MAP Kinase Cascade: A Complex Self-Organizing System Approach<br><b>Year: </b>2014<br><b>Author: </b> Ek. Kosmidis,  V. Moschou,  G. Ziogas,  I. Boukovinas,  M. Albani and  Na. Laskaris<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0111612"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>signaling network, systems biology, signal transduction, signaling pathways, receptor tyrosine, erbb signaling, erbb receptor, receptor signaling, intracellular signaling, mathematical models, cell signaling, computational model, factor receptor, receptor activation, growth factor, signaling models, networks signaling, receptor system, receptors systems, downstream signaling<div style="font-size:11px;"><br><b>Abstract: </b><em>The EGF-induced MAP kinase cascade is one of the most important and best characterized networks in intracellular signalling. It has a vital role in the development and maturation of living organisms. However, when deregulated, it is involved in the onset of a number of diseases. Based on a computational model describing a "surface" and an "internalized" parallel route, we use systems biology techniques to characterize aspects of the networks functional organization. We examine the re-organization of protein groups from low to high external stimulation, define functional groups of proteins within the network, determine the parameter best encoding for input intensity and predict the effect of protein removal to the systems output response. Extensive functional re-organization of proteins is observed in the lower end of stimulus concentrations. As we move to higher concentrations the variability is less pronounced. 6 functional groups have emerged from a consensus clustering approach, reflecting different dynamical aspects of the network. Mutual information investigation revealed that the maximum activation rate of the two output proteins best encodes for stimulus intensity. Removal of each protein of the network resulted in a range of graded effects, from complete silencing to intense activation. Our results provide a new "vista" of the EGF-induced MAP kinase cascade, from the perspective of complex self-organizing systems. Functional grouping of the proteins reveals an organizational scheme contrasting the current understanding of modular topology. The six identified groups may provide the means to experimentally follow the dynamics of this complex network. Also, the vulnerability analysis approach may be used for the development of novel therapeutic targets in the context of personalized medicine.</div>','<b>Title: </b>Dynamic Modeling of Metabolic and Gene Regulatory Systems toward Developing Virtual Microbes<br><b>Year: </b>2014<br><b>Author: </b> H. Kurata,  K. Maeda and  Y. Matsuoka<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1252/jcej.13we152"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF CHEMICAL ENGINEERING OF JAPAN<br><b>Cluster-Keywords: </b><em>kinetic models, metabolic networks, kinetic parameters, dynamic models, systems biology, network model, dynamic metabolic, biochemical network, rate constants, enzyme kinetics, kinetic data, adjustable parameters, metabolic model, parameter estimation, dynamic optimization, model selection, mathematical model, kinetic network, model response, model<div style="font-size:11px;"><br><b>Abstract: </b><em>In systems biology, computer simulation of the biochemical networks of a microbial cell is a powerful method to predict their function and phenotype under different culture conditions and genetic modifications. Construction of a virtual microbe, a generic mathematical model of cell growth and metabolism, can be an ambitious but realistic project. It would enable the rational design of a biochemical network to enhance the production of useful metabolites and proteins. However, there exist critical problems, such as parameter uncertainty, mechanism complexity and high dimensionality of the model. We would like to review the dynamic models of the biochemical networks with their problems, providing insight into developing a virtual microbe.</div>','<b>Title: </b>Reverse engineering gene networks: Integrating genetic perturbations with dynamical modeling<br><b>Year: </b>2003<br><b>Author: </b> J. Tegner,  Mks. Yeung,  J. Hasty and  Jj. Collins<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1073/pnas.0933416100"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA<br><b>Cluster-Keywords: </b><em>information systems, gene expression, gene network, systems theory, system models, systems biology, systems information, network model, genetic network, mathematical models, dynamic systems, dynamic network, systemic model, network topology, information mechanisms, complex systems, systems complex, relational models, models main, information model<div style="font-size:11px;"><br><b>Abstract: </b><em>While the fundamental building blocks of biology are being tabulated by the various genome projects, microarray technology is setting the stage for the task of deducing the connectivity of large-scale gene networks. We show how the perturbation of carefully chosen genes in a microarray experiment can be used in conjunction with a reverse engineering algorithm to reveal the architecture of an underlying gene regulatory network. Our iterative scheme identifies the network topology by analyzing the steady-state changes in gene expression resulting from the systematic perturbation of a particular node in the network. We highlight the validity of our reverse engineering approach through the successful deduction of the topology of a linear in numero gene network and a recently reported model for the segmentation polarity network in Drosophila melanogaster. Our method may prove useful in identifying and validating specific drug targets and in deconvolving the effects of chemical compounds.</div>','<b>Title: </b>Adaptive Integral Sliding Mode Controller Design for the Control and Synchronization of a Rod-Type Plasma Torch Chaotic System<br><b>Year: </b>2017<br><b>Author: </b> S. Vaidyanathan<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/978-3-319-55598-012"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>APPLICATIONS OF SLIDING MODE CONTROL IN SCIENCE AND ENGINEERING<br><b>Cluster-Keywords: </b><em>chaotic system, control systems, neural networks, jerk chaotic, torch chaotic, mode control, sliding mode, plasma torch, dynamic neural, mitochondrial dynamics, system, systems biology, adaptive integral, model system, rodtype plasma, integral sliding, systems, jerk system, cancer cell, comparison method<div style="font-size:11px;"><br><b>Abstract: </b><em>Chaos has important applications in physics, chemistry, biology, ecology, secure communications, cryptosystems and many scientific branches. Control and synchronization of chaotic systems are important research problems in chaos theory. Sliding mode control is an important method used to solve various problems in control systems engineering. In robust control systems, the sliding mode control is often adopted due to its inherent advantages of easy realization, fast response and good transient performance as well as insensitivity to parameter uncertainties and disturbance. In this work, we first describe the Ghorui jerk chaotic system 2000, which is an important model of a rod-type plasma torch chaotic system. This jerk system describes a strange chaotic attractor of a thermal arc plasma system based on triple convection theory. The phase portraits of the rod-type plasma torch chaotic system are displayed and the qualitative properties of the rod-type plasma torch chaotic system are discussed. We demonstrate that the rod-type plasma torch chaotic system has three unstable equilibrium points on the x1-axis of R-3. The Lyapunov exponents of the rod-type plasma torch chaotic system are obtained as L-1 = 0.3781, L-2 = 0 and L-3 = -1.3781. The Kaplan-Yorke dimension of the rod-type plasma torch chaotic system is derived as D-KY = 2.2744, which shows the complexity of the system. Next, an adaptive integral sliding mode control scheme is proposed to globally stabilize all the trajectories of the rod-type plasma torch chaotic system. Furthermore, an adaptive integral sliding mode control scheme is proposed for the global chaos synchronization of identical rod-type plasma torch chaotic systems. The adaptive control mechanism helps the control design by estimating the unknown parameters. Numerical simulations using MATLAB are shown to illustrate all the main results derived in this work.</div>','<b>Title: </b>Systems Analysis for Peptide Systems Chemistry<br><b>Year: </b>2019<br><b>Author: </b> Ma. Grover,  Mc. Hsieh and  Dg. Lynn<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3390/life9030055"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>LIFE-BASEL<br><b>Cluster-Keywords: </b><em>regulatory networks, stochastic simulation, gene networks, reaction networks, systems biology, stochastic model, network models, complex systems, networks stochastic, gene regulatory, biological systems, stochastic dynamics, differential equations, protein networks, cell cycle, continuous model, interaction networks, systems chemistry, time delays, biochemical reaction<div style="font-size:11px;"><br><b>Abstract: </b><em>Living systems employ both covalent chemistry and physical assembly to achieve complex behaviors. The emerging field of systems chemistry, inspired by these biological systems, attempts to construct and analyze systems that are simpler than biology, while still embodying biological design principles. Due to the multiple phenomena at play, it can be difficult to predict which phenomena will dominate and when. Conversely, there may be no single rate-limiting step, but rather a reaction network that is difficult to intuit from a purely experimental approach. Mathematical modeling can help to sort out these issues, although it can be challenging to build such models, especially for assembly kinetics. Numerical and statistical methods can play an important role to facilitate the synergistic and iterative use of modeling and experiment, and should be part of a systems chemistry curriculum. Three case studies are presented here, from our work in peptide-based systems, to illustrate some of the tools available for model construction, model simulation, and experimental design. Examples are provided in which these tools help to evaluate hypotheses, uncover design principles, and design new experiments.</div>','<b>Title: </b>Terminating rice innate immunity induction requires a network of antagonistic and redox-responsive E3 ubiquitin ligases targeting a fungal sirtuin<br><b>Year: </b><br><b>Author: </b> G. Li,  Xb. Qi,  Gc. Sun,  Ro. Rocha,  Lm. Segal,  Ks. Downey,  Jd. Wright and  Ra. Wilson<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1111/nph.16365"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>NEW PHYTOLOGIST<br><b>Cluster-Keywords: </b><em>gene expression, systems biology, metabolic network, network model, network analysis, dynamic model, signaling pathways, signaling network, regulatory network, dynamic network, biology approach, candidate genes, gene network, gene regulatory, biology analysis, mathematical model, network dynamic, biology network, metabolic pathways, pathway analysis<div style="font-size:11px;"><br><b>Abstract: </b><em>Fungal phytopathogens can suppress plant immune mechanisms in order to colonize living host cells. Identifying all the molecular components involved is critical for elaborating a detailed systems-level model of plant infection probing pathogen weaknesses; yet, the hierarchy of molecular events controlling fungal responses to the plant cell is not clear. Here we show how, in the blast fungus Magnaporthe oryzae, terminating rice innate immunity requires a dynamic network of redox-responsive E3 ubiquitin ligases targeting fungal sirtuin 2 Sir2, an antioxidation regulator required for suppressing the host oxidative burst. Immunoblotting, immunopurification, mass spectrometry and gene functional analyses showed that Sir2 levels responded to oxidative stress via a mechanism involving ubiquitination and three antagonistic E3 ubiquitin ligases: Grr1 and Ptr1 maintained basal Sir2 levels in the absence of oxidative stress; Upl3 facilitated Sir2 accumulation in response to oxidative stress. Grr1 and Upl3 interacted directly with Sir2 in a manner that decreased and scaled with oxidative stress, respectively. Deleting UPL3 depleted Sir2 during growth in rice cells, triggering host immunity and preventing infection. Overexpressing SIR2 in the Delta upl3 mutant remediated pathogenicity. Our work reveals how redox-responsive E3 ubiquitin ligases in M. oryzae mediate Sir2 accumulation-dependent antioxidation to modulate plant innate immunity and host susceptibility.</div>','<b>Title: </b>Logical modelling of regulatory networks with GINsim 2.3<br><b>Year: </b>2009<br><b>Author: </b> A. Naldi,  D. Berenguier,  A. Faure,  F. Lopez,  D. Thieffry and  C. Chaouiya<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.biosystems.2009.04.008"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOSYSTEMS<br><b>Cluster-Keywords: </b><em>boolean network, network models, qualitative models, systems biology, mathematical modeling, boolean models, signaling networks, dynamic models, dynamic modeling, signaling dynamics, logic models, dynamic network, mathematical models, boolean modeling, signal transduction, regulatory networks, drug resistance, biological networks, network approach, network modeling<div style="font-size:11px;"><br><b>Abstract: </b><em>Many important problems in cell biology require the consideration of dense nonlinear interactions between functional modules. The requirement of computer simulation for the understanding of cellular processes is now widely accepted, and a variety of modelling frameworks have been designed to meet this need. Here, we present a novel public release of the Gene Interaction Network simulation suite GINsim, a software designed for the qualitative modelling and analysis of regulatory networks. The main functionalities of GINsim are illustrated through the analysis of a logical model for the core network controlling the fission yeast cell cycle. The last public release of GINsim version 2.3, as well as development versions, can be downloaded from the dedicated website http://gin.univ-mrs.fr/GINsim/, which further includes a model library, along with detailed tutorial and user manual. C 2009 Elsevier Ireland Ltd. All rights reserved.</div>','<b>Title: </b>Regulatory patterns in molecular interaction networks<br><b>Year: </b>2011<br><b>Author: </b> D. Murrugarra and  R. Laubenbacher<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.jtbi.2011.08.015"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF THEORETICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>boolean network, boolean functions, regulatory networks, canalizing functions, network models, gene regulatory, boolean control, control networks, biological networks, boolean models, update functions, interaction networks, large network, regulatory functions, canalizing boolean, genetic regulatory, systems biology, driver nodes, networks bcns, biological systems<div style="font-size:11px;"><br><b>Abstract: </b><em>Understanding design principles of molecular interaction networks is an important goal of molecular systems biology. Some insights have been gained into features of their network topology through the discovery of graph theoretic patterns that constrain network dynamics. This paper contributes to the identification of patterns in the mechanisms that govern network dynamics. The control of nodes in gene regulatory, signaling, and metabolic networks is governed by a variety of biochemical mechanisms, with inputs from other network nodes that act additively or synergistically. This paper focuses on a certain type of logical rule that appears frequently as a regulatory pattern. Within the context of the multistate discrete model paradigm, a rule type is introduced that reduces to the concept of nested canalyzing function in the Boolean network case. It is shown that networks that employ this type of multivalued logic exhibit more robust dynamics than random networks, with few attractors and short limit cycles. It is also shown that the majority of regulatory functions in many published models of gene regulatory and signaling networks are nested canalyzing. Published by Elsevier Ltd.</div>','<b>Title: </b>Understanding and reducing complex systems pharmacology models based on a novel input-response index<br><b>Year: </b>2018<br><b>Author: </b> J. Knochel,  C. Kloft and  W. Huisinga<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s10928-017-9561-x"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF PHARMACOKINETICS AND PHARMACODYNAMICS<br><b>Cluster-Keywords: </b><em>immune system, network models, immune response, model, biological systems, models, model reduction, dynamic model, biological networks, system, mathematical models, biochemical network, response model, systems, systems biology, network, immune, biology model, reaction networks, system systems<div style="font-size:11px;"><br><b>Abstract: </b><em>A growing understanding of complex processes in biology has led to large-scale mechanistic models of pharmacologically relevant processes. These models are increasingly used to study the response of the system to a given input or stimulus, e.g., after drug administration. Understanding the input-response relationship, however, is often a challenging task due to the complexity of the interactions between its constituents as well as the size of the models. An approach that quantifies the importance of the different constituents for a given input-output relationship and allows to reduce the dynamics to its essential features is therefore highly desirable. In this article, we present a novel state- and time-dependent quantity called the input-response index that quantifies the importance of state variables for a given input-response relationship at a particular time. It is based on the concept of time-bounded controllability and observability, and defined with respect to a reference dynamics. In application to the brown snake venom-fibrinogen Fg network, the input-response indices give insight into the coordinated action of specific coagulation factors and about those factors that contribute only little to the response. We demonstrate how the indices can be used to reduce large-scale models in a two-step procedure: i elimination of states whose dynamics have only minor impact on the input-response relationship, and ii proper lumping of the remaining lower order model. In application to the brown snake venom-fibrinogen network, this resulted in a reduction from 62 to 8 state variables in the first step, and a further reduction to 5 state variables in the second step. We further illustrate that the sequence, in which a recursive algorithm eliminates and/or lumps state variables, has an impact on the final reduced model. The input-response indices are particularly suited to determine an informed sequence, since they are based on the dynamics of the original system. In summary, the novel measure of importance provides a powerful tool for analysing the complex dynamics of large-scale systems and a means for very efficient model order reduction of nonlinear systems.</div>','<b>Title: </b>Energy flows in complex ecological systems: a review<br><b>Year: </b>2009<br><b>Author: </b> J. Zhang<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s11424-009-9169-3"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF SYSTEMS SCIENCE  COMPLEXITY<br><b>Cluster-Keywords: </b><em>network ecology, food webs, network models, ecological systems, food web, systems biology, ecological networks, ecosystem model, signalling networks, model results, interaction networks, dynamic model, ecological interactions, network analysis, model network, network structure, transport networks, systems network, ecosystem dynamics, restoration ecology<div style="font-size:11px;"><br><b>Abstract: </b><em>Energy flow drives the complex systems to evolve. The allometric scaling as the universal energy flow pattern has been found in different scales of ecological systems. It reflects the general power law relationship between flow and store. The underlying mechanisms of energy flow patterns are explained as the branching transportation networks which can be regarded as the result of systematic optimization of a biological target under constraints. Energy flows in the ecological system may be modelled by the food web model and population dynamics on the network. This paper reviews the latest progress on the energy flow patterns, explanatory models for the allometric scaling and modelling approach of flow and network evolution dynamics in ecology. Furthermore, the possibility of generalizing these flow patterns, modelling approaches to other complex systems is discussed.</div>','<b>Title: </b>Inverse problem studies of biochemical systems with structure identification of S-systems by embedding training functions in a genetic algorithm<br><b>Year: </b>2016<br><b>Author: </b> Kd. Sarode,  Vr. Kumar and  Bd. Kulkarni<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.mbs.2016.02.014"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MATHEMATICAL BIOSCIENCES<br><b>Cluster-Keywords: </b><em>ssystem model, biological systems, biological networks, parameter estimation, biochemical networks, dynamic biological, proposed algorithm, regulatory networks, structure identification, model structure, genetic algorithm, gene regulatory, gene expression, network inference, network motif, network structure, gene networks, network identification, network inferring, estimation method<div style="font-size:11px;"><br><b>Abstract: </b><em>An efficient inverse problem approach for parameter estimation, state and structure identification from dynamic data by embedding training functions in a genetic algorithm methodology ETFGA is proposed for nonlinear dynamical biosystems using S-system canonical models. Use of multiple shooting and decomposition approach as training functions has been shown for handling of noisy datasets and computational efficiency in studying the inverse problem. The advantages of the methodology are brought out systematically by studying it for three biochemical model systems of interest. By studying a small-scale gene regulatory system described by a S-system model, the first example demonstrates the use of ETFGA for the multifold aims of the inverse problem. The estimation of a large number of parameters with simultaneous state and network identification is shown by training a generalized S-system canonical model with noisy datasets. The results of this study bring out the superior performance of ETFGA on comparison with other metaheuristic approaches. The second example studies the regulation of cAMP oscillations in Dictyostelium cells now assuming limited availability of noisy data. Here, flexibility of the approach to incorporate partial system information in the identification process is shown and its effect on accuracy and predictive ability of the estimated model are studied. The third example studies the phenomenological toy model of the regulation of circadian oscillations in Drosophila that follows rate laws different from S-system power-law. For the limited noisy data, using a priori information about properties of the system, we could estimate an alternate S-system model that showed robust oscillatory behavior with predictive abilities. C 2016 Elsevier Inc. All rights reserved.</div>','<b>Title: </b>Integrative Analyses of Hepatic Differentially Expressed Genes and Blood Biomarkers during the Peripartal Period between Dairy Cows Overfed or Restricted-Fed Energy Prepartum<br><b>Year: </b>2014<br><b>Author: </b> K. Shahzad,  M. Bionaz,  E. Trevisi,  G. Bertoni,  Sl. Rodriguez-Zas and  Jj. Loor<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0099757"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>gene regulatory, systems biology, regulatory networks, gene network, stem cell, network model, transcription factors, gene expression, cell transcriptomes, network analysis, data analysis, stem cells, network grnsight, cell transformation, weighted network, gene ontology, networks grns, graph model, graph layout, endothelial cell<div style="font-size:11px;"><br><b>Abstract: </b><em>Using published dairy cattle liver transcriptomics dataset along with novel blood biomarkers of liver function, metabolism, and inflammation we have attempted an integrative systems biology approach applying the classical functional enrichment analysis using DAVID, a newly-developed Dynamic Impact Approach DIA, and an upstream gene network analysis using Ingenuity Pathway Analysis IPA. Transcriptome data was generated from experiments evaluating the impact of prepartal plane of energy intake overfed OF or restricted RE on liver of dairy cows during the peripartal period. Blood biomarkers uncovered that RE vs. OF led to greater prepartal liver distress accompanied by a low-grade inflammation and larger proteolysis i.e., higher haptoglobin, bilirubin, and creatinine. Post-partum the greater bilirubinaemia and lipid accumulation in OF vs. RE indicated a large degree of liver distress. The re-analysis of microarray data revealed that expression of.4,000 genes was affected by diet x time. The bioinformatics analysis indicated that RE vs. OF cows had a liver with a greater lipid and amino acid catabolic capacity both pre-and post-partum while OF vs. RE cows had a greater activation of pathways/functions related to triglyceride synthesis. Furthermore, RE vs. OF cows had a larger or higher capacity to cope with ER stress likely associated with greater protein synthesis/processing, and a higher activation of inflammatory-related functions. Liver in OF vs. RE cows had a larger cell proliferation and cell-to-cell communication likely as a response to the greater lipid accumulation. Analysis of upstream regulators indicated a pivotal role of several lipid-related transcription factors e.g., PPARs, SREBPs, and NFE2L2 in priming the liver of RE cows to better face the early postpartal metabolic and inflammatory challenges. An all-encompassing dynamic model was proposed based on the findings.</div>','<b>Title: </b>Understanding system dynamics of an adaptive enzyme network from globally profiled kinetic parameters<br><b>Year: </b>2014<br><b>Author: </b> Awt. Chiang,  Wc. Liu,  P. Charusanti and  Mj. Hwang<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-8-4"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>systems biology, biological systems, system dynamics, molecular systems, experimental design, network model, model system, model parameters, experiment design, kinetic parameters, perfect adaptation, experimental data, molecular mechanisms, homeostatic mechanisms, switching dynamics, biological mechanisms, control systems, chemotaxis network, decentralized control, feedback control<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: A major challenge in mathematical modeling of biological systems is to determine how model parameters contribute to systems dynamics. As biological processes are often complex in nature, it is desirable to address this issue using a systematic approach. Here, we propose a simple methodology that first performs an enrichment test to find patterns in the values of globally profiled kinetic parameters with which a model can produce the required system dynamics; this is then followed by a statistical test to elucidate the association between individual parameters and different parts of the systems dynamics.Results: We demonstrate our methodology on a prototype biological system of perfect adaptation dynamics, namely the chemotaxis model for Escherichia coli. Our results agreed well with those derived from experimental data and theoretical studies in the literature. Using this model system, we showed that there are motifs in kinetic parameters and that these motifs are governed by constraints of the specified system dynamics.Conclusions: A systematic approach based on enrichment statistical tests has been developed to elucidate the relationships between model parameters and the roles they play in affecting system dynamics of a prototype biological network. The proposed approach is generally applicable and therefore can find wide use in systems biology modeling research.</div>','<b>Title: </b>Computational modeling of non-linear diffusion in cardiac electrophysiology: A novel porous-medium approach<br><b>Year: </b>2016<br><b>Author: </b> De. Hurtado,  S. Castro and  A. Gizzi<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.cma.2015.11.014"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>COMPUTER METHODS IN APPLIED MECHANICS AND ENGINEERING<br><b>Cluster-Keywords: </b><em>network models, population model, field model, neural population, neural field, iot networks, neuronal networks, neural network, differential equations, parameter neural, lotkavolterra model, model, neuron models, model neurons, model leads, ecosystem model, iot devices, noise model, neuronal models, inhibitory synaptic<div style="font-size:11px;"><br><b>Abstract: </b><em>The electrophysiological behavior of excitable biological media has been traditionally modeled using a nonlinear reaction-diffusion equation commonly known as the cable equation. To account for the propagating nature of electrical waves, virtually all cardiac electrophysiology formulations proposed to date consider a linear diffusion flux, a constitutive relation known in biology as Ficks law. In this work, motivated by the porous nature of intercalated discs in cardiac muscle cells that mediate intercellular communication and ultimately tissue conductivity, we propose a novel formulation of cardiac electrophysiology that incorporates a nonlinear diffusion term of the porous-media kind. To solve the resulting system of non-linear partial differential equations we develop a non-linear implicit finite-element scheme that is suitable to simulations of large-scale cardiac problems. We show that the proposed porous-medium electrophysiology model results in propagating action potentials that have well-defined wavefronts and travel with finite speed. We also show that the proposed model captures the restitution properties of cardiac tissue similar to the cable model. We demonstrate the capabilities of our method by simulating the activation sequence of a three-dimensional human biventricular heart model, where important microstructural features like cardiomyocyte fiber orientation and the His-Purkinje activation network are successfully incorporated into the simulation. C 2015 Elsevier B.V. All rights reserved.</div>','<b>Title: </b>Dynamical analysis of cellular networks based on the Greens function matrix<br><b>Year: </b>2009<br><b>Author: </b> Tm. Perumal,  Y. Wu and  R. Gunawan<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.jtbi.2009.07.037"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF THEORETICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>sensitivity analysis, systems biology, robustness analysis, network model, network inference, regulatory network, systems analysis, gene expression, parametric uncertainty, biological networks, mathematical models, biochemical networks, biological functions, cellular networks, biological system, parametric sensitivity, dynamical models, dynamic system, networks analysis, gene regulatory<div style="font-size:11px;"><br><b>Abstract: </b><em>The complexity of cellular networks often limits human intuition in understanding functional regulations in a cell from static network diagrams. To this end, mathematical models of ordinary differential equations ODEs have commonly been used to simulate dynamical behavior of cellular networks, to which a quantitative model analysis can be applied in order to gain biological insights. In this paper, we introduce a dynamical analysis based on the use of Greens function matrix GFM as sensitivity coefficients with respect to initial concentrations. In contrast to the classical parametric sensitivity analysis, the GFM analysis gives a dynamical, molecule-by-molecule insight on how system behavior is accomplished and complementarily how impulse signal propagates through the network. The knowledge gained will have application from model reduction and validation to drug discovery research in identifying potential drug targets, studying drug efficacy and specificity, and optimizing drug dosing and timing. The efficacy of the method is demonstrated through applications to common network motifs and a Fas-induced programmed cell death model in Jurkat T cell line. C 2009 Elsevier Ltd. All rights reserved.</div>','<b>Title: </b>Network Modeling Reveals Steps in Angiotensin Peptide Processing<br><b>Year: </b>2013<br><b>Author: </b> Jh. Schwacke,  Jcg. Spainhour,  Jl. Ierardi,  Jm. Chaves,  Jm. Arthur,  Mg. Janech and  Jcq. Velez<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1161/HYPERTENSIONAHA.111.00318"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>HYPERTENSION<br><b>Cluster-Keywords: </b><em>systems biology, dynamic models, biological systems, model, dynamic systems, computational systems, network model, experimental data, model parameters, systems, systems model, models, pathway systems, data, biochemical pathway, dynamics model, biochemical networks, computational models, network, system<div style="font-size:11px;"><br><b>Abstract: </b><em>New insights into the intrarenal renin-angiotensin Ang system have modified our traditional view of the system. However, many finer details of this network of peptides and associated peptidases remain unclear. We hypothesized that a computational systems biology approach, applied to peptidomic data, could help to unravel the network of enzymatic conversions. We built and refined a Bayesian network model and a dynamic systems model starting from a skeleton created with established elements of the renin-Ang system and further developed it with archived matrix-assisted laser desorption ionization-time of flight mass spectra from experiments conducted in mouse podocytes exposed to exogenous Ang substrates. The model-building process suggested previously unrecognized steps, 3 of which were confirmed in vitro, including the conversion of Ang2-10 to Ang2-7 by neprilysin, Ang1-9 to Ang2-9, and Ang1-7 to Ang2-7 by aminopeptidase A. These data suggest a wider role of neprilysin and aminopeptidase A in glomerular formation of bioactive Ang peptides and shunting their formation. Other steps were also suggested by the model, and supporting evidence for those steps was evaluated using model-comparison methods. Our results demonstrate that systems biology methods applied to peptidomic data are effective in identifying novel steps in the Ang peptide processing network, and these findings improve our understanding of the glomerular renin-Ang system. Hypertension. 2013;61:690-700.. circle Online Data Supplement</div>','<b>Title: </b>Investigation into the regulation mechanisms of TRAIL apoptosis pathway by mathematical modeling<br><b>Year: </b>2010<br><b>Author: </b> T. Zhang,  M. Wu,  Q. Chen and  Zr. Sun<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1093/abbs/gmp114"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ACTA BIOCHIMICA ET BIOPHYSICA SINICA<br><b>Cluster-Keywords: </b><em>stem cell, stem cells, systems biology, mathematical models, cell biology, network model, signaling pathways, cell fate, computational models, cancer cells, gene expression, network dynamics, embryonic stem, signaling network, prostate cancer, mathematical modeling, psychological stress, apoptosis signaling, computational systems, molecular mechanisms<div style="font-size:11px;"><br><b>Abstract: </b><em>TNF-related apoptosis-inducing ligand TRAIL selectively induces apoptosis in tumor cell lines, suggesting its great potential in cancer therapy. Many components in TRAIL-mediated signaling pathway have been identified, but how they interact with each other to determine the network dynamics and final apoptosis remains elusive. Here we constructed a mathematical model of this pathway, which integrated current available information from related experimental literatures, to make a systematic and quantitative description of the cellular responses to TRAIL stimulation. By applying sensitivity analysis, we identified the key components and reactions that have the highest impact on the network dynamics, and then analyzed the regulatory role of several key players in this pathway. To elucidate the function of TRAIL decoy receptors, we compared the competitive ligand binding hypothesis versus the pre-ligand binding hypothesis. Our results show that the pre-ligand binding hypothesis is more suitable for explaining the fact that over-expression of decoy receptors can inhibit apoptosis potently. These results, together with our investigation on other downstream proteins and feedback loops in this pathway, provide insights into the molecular mechanisms of the TRAIL-mediated apoptosis pathway.</div>','<b>Title: </b>Differentiation between genomic and non-genomic feedback controls yields an HPA axis model featuring Hypercortisolism as an irreversible bistable switch<br><b>Year: </b>2013<br><b>Author: </b> Ca. Zarzer,  Mg. Puchinger,  G. Kohler and  P. Kugler<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1742-4682-10-65"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>THEORETICAL BIOLOGY AND MEDICAL MODELLING<br><b>Cluster-Keywords: </b><em>circadian rhythms, systems biology, biological clock, circadian clock, biological rhythms, hpa axis, model, clock network, mathematical models, biological systems, dynamic systems, ode models, biology models, single cell, network, model parameters, experimental data, models, systems, dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: The hypothalamic-pituitary-adrenal axis HPA axis is a major part of the neuroendocrine system responsible for the regulation of the response to physical or mental stress and for the control of the synthesis of the stress hormone cortisol. Dysfunctions of the HPA axis characterized by either low hypocortisolism or increased hypercortisolism cortisol levels are implicated in various pathological conditions. Their understanding and therapeutic correction may be supported by mathematical modeling and simulation of the HPA axis.Methods: Mass action and Michaelis Menten enzyme kinetics were used to provide a mechanistic description of the feedback mechanisms within the pituitary gland cells by which cortisol inhibits its own production. A separation of the nucleus from the cytoplasm by compartments enabled a differentiation between slow genomic and fast non-genomic processes. The model in parts was trained against time resolved ACTH stress response data from an in vitro cell culture of murine AtT-20 pituitary tumor cells and analyzed by bifurcation discovery tools.Results: A recently found pituitary gland cell membrane receptor that mediates rapid non-genomic actions of glucocorticoids has been incorporated into our model of the HPA axis. As a consequence of the distinction between genomic and non-genomic feedback processes our model possesses an extended dynamic repertoire in comparison to existing HPA models. In particular, our model exhibits limit cycle oscillations and bistable behavior associated to hypocortisolism but also features a second bistable switch which captures irreversible transitions in hypercortisolism to elevated cortisol levels.Conclusions: Model predictive control and inverse bifurcation analysis have been previously applied in the simulation-based design of therapeutic strategies for the correction of hypocortisolism. Given the HPA model extension presented in this paper, these techniques may also be used in the study of hypercortisolism. As an example, we show how sparsity enforcing penalization may suggest network interventions that allow the return from elevated cortisol levels back to nominal ones.</div>','<b>Title: </b>Exploring mitochondrial system properties of neurodegenerative diseases through interactome mapping<br><b>Year: </b>2014<br><b>Author: </b> J. Vlasblom,  K. Jin,  S. Kassir and  M. Babu<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.jprot.2013.11.008"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF PROTEOMICS<br><b>Cluster-Keywords: </b><em>interaction networks, proteinprotein interactions, protein interaction, network model, complex systems, systems biology, protein protein, protein complex, host network, dynamic interactions, network analysis, biological systems, networks network, cellular network, biological network, viral proteins, interactome networks, regulatory networks, immune system, systems network<div style="font-size:11px;"><br><b>Abstract: </b><em>Mitochondria are double membraned, dynamic organelles that are required for a large number of cellular processes, and defects in their function have emerged as causative factors for a growing number of human disorders and are highly associated with cancer, metabolic, and neurodegenerative ND diseases. Biochemical and genetic investigations have uncovered small numbers of candidate mitochondrial proteins MPs involved in ND disease, but given the diversity of processes affected by MP function and the difficulty of detecting interactions involving these proteins, many more likely remain unknown. However, high-throughput proteomic and genomic approaches developed in genetically tractable model prokaryotes and lower eukaryotes have proven to be effective tools for querying the physical protein-protein and functional gene-gene relationships between diverse types of proteins, including cytosolic and membrane proteins. In this review, we highlight how experimental and computational approaches developed recently by our group and others can be effectively used towards elucidating the mitochondrial interactome in an unbiased and systematic manner to uncover network-based connections. We discuss how the knowledge from the resulting interaction networks can effectively contribute towards the identification of new mitochondrial disease gene candidates, and thus further clarify the role of mitochondrial biology and the complex etiologies of ND disease.Biological significanceBiochemical and genetic investigations have uncovered small numbers of candidate mitochondrial proteins MPs involved in neurodegenerative ND diseases, but given the diversity of processes affected by MP function and the difficulty of detecting interactions involving these proteins, many more likely remain unknown. Large-scale proteomic and genomic approaches developed in model prokaryotes and lower eukaryotes have proven to be effective tools for querying the physical protein-protein and functional gene-gene relationships between diverse types of proteins. Extension of this new framework to the mitochondrial sub-system in human will likewise provide a universally informative systems-level view of the physical and functional landscape for exploring the evolutionary principles underlying mitochondrial function. In this review, we highlight how experimental and computational approaches developed recently by our group and others can be effectively used towards elucidating the mitochondrial interactome in an unbiased and systematic manner to uncover network-based connections.We anticipate that the knowledge from these resulting interaction networks can effectively contribute towards the identification of new mitochondrial disease gene candidates, and thus foster a deeper molecular understanding of mitochondrial biology as well as the etiology of mitochondrial diseases.This article is part of a Special Issue: Can Proteomics Fill the Gap Between Genomics and Phenotypes C 2013 Published by Elsevier B.V.</div>','<b>Title: </b>Cooperation through Competition - Dynamics and Microeconomics of a Minimal Nutrient Trade System in Arbuscular Mycorrhizal Symbiosis<br><b>Year: </b>2016<br><b>Author: </b> S. Schott,  B. Valdebenito,  D. Bustos,  Jl. Gomez-Porras,  T. Sharma and  I. Dreyer<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3389/fpls.2016.00912"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>FRONTIERS IN PLANT SCIENCE<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>In arbuscular mycorrhizal AM symbiosis, fungi and plants exchange nutrients sugars and phosphate, for instance for reciprocal benefit. Until now it is not clear how this nutrient exchange system works. Here, we used computational cell biology to simulate the dynamics of a network of proton pumps and proton-coupled transporters that are upregulated during AM formation. We show that this minimal network is sufficient to describe accurately and realistically the nutrient trade system. By applying basic principles of microeconomics, we link the biophysics of transmembrane nutrient transport with the ecology of organismic interactions and straightforwardly explain macroscopic scenarios of the relations between plant and AM fungus. This computational cell biology study allows drawing far reaching hypotheses about the mechanism and the regulation of nutrient exchange and proposes that the "cooperation" between plant and fungus can be in fact the result of a competition between both for the same resources in the tiny periarbuscular space. The minimal model presented here may serve as benchmark to evaluate in future the performance of more complex models of AM nutrient exchange. As a first step toward this goal, we included SWEET sugar transporters in the model and show that their co-occurrence with proton-coupled sugar transporters results in a futile carbon cycle at the plant plasma membrane proposing that two different pathways for the same substrate should not be active at the same time.</div>','<b>Title: </b>Spatial gradients of protein-level time delays set the pace of the traveling segmentation clock waves<br><b>Year: </b>2014<br><b>Author: </b> A. Ay,  J. Holland,  A. Sperlea,  Gs. Devakanmalai,  S. Knierer,  S. Sangervasi,  A. Stevenson and  Em. Ozbudak<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1242/dev.111930"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>DEVELOPMENT<br><b>Cluster-Keywords: </b><em>gene expression, gene network, stochastic model, gene regulatory, systems biology, expression systems, gene regulation, network activity, expression dynamics, noise control, regulatory network, deterministic models, network components, stochastic modeling, regulation system, escherichia coli, regulatory system, synthetic gene, regulatory genes, gene<div style="font-size:11px;"><br><b>Abstract: </b><em>The vertebrate segmentation clock is a gene expression oscillator controlling rhythmic segmentation of the vertebral column during embryonic development. The period of oscillations becomes longer as cells are displaced along the posterior to anterior axis, which results in traveling waves of clock gene expression sweeping in the unsegmented tissue. Although various hypotheses necessitating the inclusion of additional regulatory genes into the core clock network at different spatial locations have been proposed, the mechanism underlying traveling waves has remained elusive. Here, we combined molecular-level computational modeling and quantitative experimentation to solve this puzzle. Our model predicts the existence of an increasing gradient of gene expression time delays along the posterior to anterior direction to recapitulate spatiotemporal profiles of the traveling segmentation clock waves in different genetic backgrounds in zebrafish. We validated this prediction by measuring an increased time delay of oscillatory Her1 protein production along the unsegmented tissue. Our results refuted the need for spatial expansion of the core feedback loop to explain the occurrence of traveling waves. Spatial regulation of gene expression time delays is a novel way of creating dynamic patterns; this is the first report demonstrating such a control mechanism in any tissue and future investigations will explore the presence of analogous examples in other biological systems.</div>','<b>Title: </b>Temporal logic analysis of gene networks under parameter uncertainty<br><b>Year: </b>2008<br><b>Author: </b> G. Batt,  C. Belta and  R. Weiss<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1109/TAC.2007.911330"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IEEE TRANSACTIONS ON AUTOMATIC CONTROL<br><b>Cluster-Keywords: </b><em>regulatory networks, network dynamics, gene regulatory, network model, genetic network, gene networks, genetic regulatory, mathematical model, parameter space, network system, interaction networks, system dynamics, network structure, network, dynamic networks, network reduction, models models, biological networks, network motifs, biological systems<div style="font-size:11px;"><br><b>Abstract: </b><em>The lack of precise numerical information for the values of biological parameters severely limits the development and analysis of models of genetic regulatory networks. To deal with this problem, we propose a method for the analysis of genetic regulatory networks under parameter uncertainty. We consider models based on piecewise-multiaffine differential equations, dynamical properties expressed in temporal logic, and intervals for the values of uncertain parameters. The problem is then either to guarantee that the system satisfies the expected properties for every possible parameter value-the corresponding parameter set is then called valid-or to find valid subsets of a given parameter set. The proposed method uses discrete abstractions and model checking and allows for efficient search of the parameter space. However, the abstraction process creates spurious behaviors in the abstract systems, along which time does not progress. Consequently, the verification of liveness properties, expressing that something will eventually happen, and implicitly assuming progress of time, often fails. A solution to this second problem is proposed using the notion of transient regions. This approach has been implemented in a tool for robust verification of gene networks and applied to the, tuning of a synthetic network built in E. coli.</div>','<b>Title: </b>Structural Invertibility and Optimal Sensor Node Placement for Error and Input Reconstruction in Dynamic Systems<br><b>Year: </b>2019<br><b>Author: </b> D. Kahl,  P. Wendland,  M. Neidhardt,  A. Weber and  M. Kschischo<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1103/PhysRevX.9.041046"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PHYSICAL REVIEW X<br><b>Cluster-Keywords: </b><em>biological networks, dynamic models, biological systems, model predictions, model parameters, systems biology, mathematical models, biological models, equation models, experimental data, molecular network, interaction networks, model errors, biological interaction, molecular interactions, model, dynamic systems, biological parameters, parameter uncertainty, growth model<div style="font-size:11px;"><br><b>Abstract: </b><em>Despite recent progress in our understanding of complex dynamic networks, it remains challenging to devise sufficiently accurate models to observe, control, or predict the state of real systems in biology, economics, or other fields. A largely overlooked fact is that these systems are typically open and receive unknown inputs from their environment. A further fundamental obstacle is structural model errors caused by insufficient or inaccurate knowledge about the quantitative interactions in the real system. Here, we show that unknown inputs to open systems and model errors can be treated under the common framework of invertibility, which is a requirement for reconstructing these disturbances from output measurements. By exploiting the fact that invertibility can be decided from the influence graph of the system, we analyze the relationship between structural network properties and invertibility under different realistic scenarios. We show that sparsely connected scale-free networks are the most difficult to invert. We introduce a new sensor node placement algorithm to select a minimum set of measurement positions in the network required for invertibility. This algorithm facilitates optimal experimental design for the reconstruction of inputs or model errors from output measurements. Our results have both fundamental and practical implications for nonlinear systems analysis, modeling, and design.</div>','<b>Title: </b>Emergence of rules in cell society: Differentiation, hierarchy, and stability<br><b>Year: </b>1998<br><b>Author: </b> C. Furusawa and  K. Kaneko<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1006/bulm.1997.0034"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BULLETIN OF MATHEMATICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>A dynamic model for cell differentiation, where cells with internal chemical reaction dynamics interact with each other and replicate was studied. It led to spontaneous differentiation of cells and determination, as discussed In the isologous diversification. The following features of the differentiation were obtained: 1hierarchical differentiation from a stem cell to other cell types, with the emergence of the interaction-dependent rules for differentiation;; 2 global stability of an ensemble of cells consisting of several cell types, that were sustained by the emergent, autonomous control on the rate of differentiation; 3 existence of several cell colonies with different cell-type distributions. The results provide a novel viewpoint on the origin of a complex cell society, while relevance to some biological problems, especially to the hemopoietic system, is also discussed. C 1998 Society for Mathematical Biology.</div>','<b>Title: </b>Specification, annotation, visualization and simulation of a large rule-based model for ERBB receptor signaling<br><b>Year: </b>2012<br><b>Author: </b> Ms. Creamer,  Ec. Stites,  M. Aziz,  Ja. Cahill,  Cw. Tan,  Me. Berens,  Hy. Han,  Kj. Bussey,  Dd. Von Hoff,  Ws. Hlavacek and  Rg. Posner<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-6-107"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>allosteric proteins, signaling networks, protein interactions, network model, molecular dynamics, protein complexes, allosteric network, signaling proteins, cell network, biochemical networks, free energy, combinatorial complexity, molecular species, biological systems, receptor signaling, systems biology, chemical species, model, dynamics simulations, cell signaling<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Mathematical/computational models are needed to understand cell signaling networks, which are complex. Signaling proteins contain multiple functional components and multiple sites of post-translational modification. The multiplicity of components and sites of modification ensures that interactions among signaling proteins have the potential to generate myriad protein complexes and post-translational modification states. As a result, the number of chemical species that can be populated in a cell signaling network, and hence the number of equations in an ordinary differential equation model required to capture the dynamics of these species, is prohibitively large. To overcome this problem, the rule-based modeling approach has been developed for representing interactions within signaling networks efficiently and compactly through coarse-graining of the chemical kinetics of molecular interactions.Results: Here, we provide a demonstration that the rule-based modeling approach can be used to specify and simulate a large model for ERBB receptor signaling that accounts for site-specific details of protein-protein interactions. The model is considered large because it corresponds to a reaction network containing more reactions than can be practically enumerated. The model encompasses activation of ERK and Akt, and it can be simulated using a network-free simulator, such as NFsim, to generate time courses of phosphorylation for 55 individual serine, threonine, and tyrosine residues. The model is annotated and visualized in the form of an extended contact map.Conclusions: With the development of software that implements novel computational methods for calculating the dynamics of large-scale rule-based representations of cellular signaling networks, it is now possible to build and analyze models that include a significant fraction of the protein interactions that comprise a signaling network, with incorporation of the site-specific details of the interactions. Modeling at this level of detail is important for understanding cellular signaling.</div>','<b>Title: </b>THE HYDRODYNAMIC LIMIT OF A RANDOMIZED LOAD BALANCING NETWORK<br><b>Year: </b>2019<br><b>Author: </b> R. Aghajani and  K. Ramanan<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1214/18-AAP1444"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ANNALS OF APPLIED PROBABILITY<br><b>Cluster-Keywords: </b><em>gene regulatory, systems biology, synthetic biology, regulatory network, gene network, gene expression, system model, synthetic network, network dynamics, systems theory, synthetic gene, genetic network, model predicts, dynamical systems, time delays, translational time, regulation models, oscillatory expression, network structure, biological systems<div style="font-size:11px;"><br><b>Abstract: </b><em>Randomized load balancing networks arise in a variety of applications, and allow for efficient sharing of resources, while being relatively easy to implement. We consider a network of parallel queues in which incoming jobs with independent and identically distributed service times are assigned to the shortest queue among a subset of d queues chosen uniformly at random, and leave the network on completion of service. Prior work on dynamical properties of this model has focused on the case of exponential service distributions. In this work, we analyze the more realistic case of general service distributions. We first introduce a novel particle representation of the state of the network, and characterize the state dynamics via a countable sequence of interacting stochastic measure-valued evolution equations. Under mild assumptions, we show that the sequence of scaled state processes converges, as the number of servers goes to infinity, to a hydrodynamic limit that is characterized as the unique solution to a countable system of coupled deterministic measure-valued equations. As a simple corollary, we also establish a propagation of chaos result that shows that finite collections of queues are asymptotically independent. The general framework developed here is potentially useful for analyzing a larger class of models arising in diverse fields including biology and materials science.</div>','<b>Title: </b>Motif distributions in phase-space networks for characterizing experimental two-phase flow patterns with chaotic features<br><b>Year: </b>2010<br><b>Author: </b> Zk. Gao,  Nd. Jin,  Wx. Wang and  Yc. Lai<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1103/PhysRevE.82.016210"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PHYSICAL REVIEW E<br><b>Cluster-Keywords: </b><em>complex networks, complex systems, network theory, network structure, biological network, social network, network science, dynamic networks, network systems, network model, network topology, systems biology, network, network analysis, system dynamics, networks, realworld networks, network automata, epidemic network, model network<div style="font-size:11px;"><br><b>Abstract: </b><em>The dynamics of two-phase flows have been a challenging problem in nonlinear dynamics and fluid mechanics. We propose a method to characterize and distinguish patterns from inclined water-oil flow experiments based on the concept of network motifs that have found great usage in network science and systems biology. In particular, we construct from measured time series phase-space complex networks and then calculate the distribution of a set of distinct network motifs. To gain insight, we first test the approach using time series from classical chaotic systems and find a universal feature: motif distributions from different chaotic systems are generally highly heterogeneous. Our main finding is that the distributions from experimental two-phase flows tend to be heterogeneous as well, suggesting the underlying chaotic nature of the flow patterns. Calculation of the maximal Lyapunov exponent provides further support for this. Motif distributions can thus be a feasible tool to understand the dynamics of realistic two-phase flow patterns.</div>','<b>Title: </b>pSSAlib: The partial-propensity stochastic chemical network simulator<br><b>Year: </b>2017<br><b>Author: </b> O. Ostrenko,  P. Incardona,  R. Ramaswamy,  L. Brusch and  If. Sbalzarini<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pcbi.1005865"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS COMPUTATIONAL BIOLOGY<br><b>Cluster-Keywords: </b><em>regulatory networks, stochastic simulation, gene networks, reaction networks, systems biology, stochastic model, network models, complex systems, networks stochastic, gene regulatory, biological systems, stochastic dynamics, differential equations, protein networks, cell cycle, continuous model, interaction networks, systems chemistry, time delays, biochemical reaction<div style="font-size:11px;"><br><b>Abstract: </b><em>Chemical reaction networks are ubiquitous in biology, and their dynamics is fundamentally stochastic. Here, we present the software library pSSAlib, which provides a complete and concise implementation of the most efficient partial-propensity methods for simulating exact stochastic chemical kinetics. pSSAlib can import models encoded in Systems Biology Markup Language, supports time delays in chemical reactions, and stochastic spatiotemporal reaction-diffusion systems. It also provides tools for statistical analysis of simulation results and supports multiple output formats. It has previously been used for studies of biochemical reaction pathways and to benchmark other stochastic simulation methods. Here, we describe pSSAlib in detail and apply it to a new model of the endocytic pathway in eukaryotic cells, leading to the discovery of a stochastic counterpart of the cut-out switch motif underlying early-to-late endosome conversion. pSSAlib is provided as a stand-alone command-line tool and as a developer API. We also provide a plug-in for the SBMLToolbox.</div>','<b>Title: </b>Probing the role of stochasticity in a model of the embryonic stem cell - heterogeneous gene expression and reprogramming efficiency<br><b>Year: </b>2012<br><b>Author: </b> V. Chickarmane,  V. Olariu and  C. Peterson<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-6-98"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>stem cell, stem cells, systems biology, mathematical models, cell biology, network model, signaling pathways, cell fate, computational models, cancer cells, gene expression, network dynamics, embryonic stem, signaling network, prostate cancer, mathematical modeling, psychological stress, apoptosis signaling, computational systems, molecular mechanisms<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Embryonic stem cells ESC have the capacity to self-renew and remain pluripotent, while continuously providing a source of a variety of differentiated cell types. Understanding what governs these properties at the molecular level is crucial for stem cell biology and its application to regenerative medicine. Of particular relevance is to elucidate those molecular interactions which govern the reprogramming of somatic cells into ESC. A computational approach can be used as a framework to explore the dynamics of a simplified network of the ESC with the aim to understand how stem cells differentiate and also how they can be reprogrammed from somatic cells.Results: We propose a computational model of the embryonic stem cell network, in which a core set of transcription factors TFs interact with each other and are induced by external factors. A stochastic treatment of the network dynamics suggests that NANOG heterogeneity is the deciding factor for the stem cell fate. In particular, our results show that the decision of staying in the ground state or commitment to a differentiated state is fundamentally stochastic, and can be modulated by the addition of external factors 2i/3i media, which have the effect of reducing fluctuations in NANOG expression. Our model also hosts reprogramming of a committed cell into an ESC by over-expressing OCT4. In this context, we recapitulate the important experimental result that reprogramming efficiency peaks when OCT4 is over-expressed within a specific range of values.Conclusions: We have demonstrated how a stochastic computational model based upon a simplified network of TFs in ESCs can elucidate several key observed dynamical features. It accounts for i the observed heterogeneity of key regulators, ii characterizes the ESC under certain external stimuli conditions and iii describes the occurrence of transitions from the ESC to the differentiated state. Furthermore, the model iv provides a framework for reprogramming from somatic cells and conveys an understanding of reprogramming efficiency as a function of OCT4 over-expression.</div>','<b>Title: </b>Noisy threshold in neuronal models: connections with the noisy leaky integrate-and-fire model<br><b>Year: </b>2016<br><b>Author: </b> G. Dumont,  J. Henry and  Co. Tarniceriu<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s00285-016-1002-8"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF MATHEMATICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>network models, population model, field model, neural population, neural field, iot networks, neuronal networks, neural network, differential equations, parameter neural, lotkavolterra model, model, neuron models, model neurons, model leads, ecosystem model, iot devices, noise model, neuronal models, inhibitory synaptic<div style="font-size:11px;"><br><b>Abstract: </b><em>Providing an analytical treatment to the stochastic feature of neurons dynamics is one of the current biggest challenges in mathematical biology. The noisy leaky integrate-and-fire model and its associated Fokker-Planck equation are probably the most popular way to deal with neural variability. Another well-known formalism is the escape-rate model: a model giving the probability that a neuron fires at a certain time knowing the time elapsed since its last action potential. This model leads to a so-called age-structured system, a partial differential equation with non-local boundary condition famous in the field of population dynamics, where the age of a neuron is the amount of time passed by since its previous spike. In this theoretical paper, we investigate the mathematical connection between the two formalisms. We shall derive an integral transform of the solution to the age-structured model into the solution of the Fokker-Planck equation. This integral transform highlights the link between the two stochastic processes. As far as we know, an explicit mathematical correspondence between the two solutions has not been introduced until now.</div>','<b>Title: </b>Robustness analysis of the detailed kinetic model of an ErbB signaling network by using dynamic sensitivity<br><b>Year: </b>2017<br><b>Author: </b> H. Masunaga,  Y. Sugimoto,  S. Magi,  R. Itasaki,  M. Okada-Hatakeyama and  H. Kurata<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0178250"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>signaling network, systems biology, signal transduction, signaling pathways, receptor tyrosine, erbb signaling, erbb receptor, receptor signaling, intracellular signaling, mathematical models, cell signaling, computational model, factor receptor, receptor activation, growth factor, signaling models, networks signaling, receptor system, receptors systems, downstream signaling<div style="font-size:11px;"><br><b>Abstract: </b><em>The ErbB receptor signaling pathway plays an important role in the regulation of cellular proliferation, survival and differentiation, and dysregulation of the pathway is linked to various types of human cancer. Mathematical models have been developed as a practical complementary approach to deciphering the complexity of ErbB receptor signaling and elucidating how the pathways discriminate between ligands to induce different cell fates. In this study, we developed a simulator to accurately calculate the dynamic sensitivity of extracellular-signal-regulated kinase ERK activity ERK and Akt activity Akt, downstream of the ErbB receptors stimulated with epidermal growth factor EGF and heregulin HRG. To demonstrate the feasibility of this simulator, we estimated how the reactions critically responsible for ERK and Akt change with time and in response to different doses of EGF and HRG, and predicted that only a small number of reactions determine ERK and Akt. ERK increased steeply with increasing HRG dose until saturation, while showing a gently rising response to EGF. Akt had a gradual wide-range response to HRG and a blunt response to EGF. Akt was sensitive to perturbations of intracellular kinetics, while ERK was more robust due to multiple, negative feedback loops. Overall, the simulator predicted reactions that were critically responsible for ERK and Akt in response to the dose of EGF and HRG, illustrated the response characteristics of ERK and Akt, and estimated mechanisms for generating robustness in the ErbB signaling network.</div>','<b>Title: </b>Nephrin Regulates Lamellipodia Formation by Assembling a Protein Complex That Includes Ship2, Filamin and Lamellipodin<br><b>Year: </b>2011<br><b>Author: </b> M. Venkatareddy,  L. Cook,  K. Abuarquob,  R. Verma and  P. Garg<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0028710"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>interaction network, actin filament, network model, filament network, network structures, actin network, protein interaction, actin dynamics, network, membrane protein, network dynamics, model dynamic, cell, model, network approach, protein complex, computational model, actin, adhesion dynamics, cell dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>Actin dynamics has emerged at the forefront of podocyte biology. Slit diaphragm junctional adhesion protein Nephrin is necessary for development of the podocyte morphology and transduces phosphorylation-dependent signals that regulate cytoskeletal dynamics. The present study extends our understanding of Nephrin function by showing in cultured podocytes that Nephrin activation induced actin dynamics is necessary for lamellipodia formation. Upon activation Nephrin recruits and regulates a protein complex that includes Ship2 SH2 domain containing 5 inositol phosphatase, Filamin and Lamellipodin, proteins important in regulation of actin and focal adhesion dynamics, as well as lamellipodia formation. Using the previously described CD16-Nephrin clustering system, Nephrin ligation or activation resulted in phosphorylation of the actin crosslinking protein Filamin in a p21 activated kinase dependent manner. Nephrin activation in cell culture results in formation of lamellipodia, a process that requires specialized actin dynamics at the leading edge of the cell along with focal adhesion turnover. In the CD16-Nephrin clustering model, Nephrin ligation resulted in abnormal morphology of actin tails in human podocytes when Ship2, Filamin or Lamellipodin were individually knocked down. We also observed decreased lamellipodia formation and cell migration in these knock down cells. These data provide evidence that Nephrin not only initiates actin polymerization but also assembles a protein complex that is necessary to regulate the architecture of the generated actin filament network and focal adhesion dynamics.</div>','<b>Title: </b>Efficient Characterization of Parametric Uncertainty of Complex Bio chemical Networks<br><b>Year: </b>2015<br><b>Author: </b> C. Schillings,  M. Sunnaker,  J. Stelling and  C. Schwab<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pcbi.1004457"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS COMPUTATIONAL BIOLOGY<br><b>Cluster-Keywords: </b><em>systems biology, network models, signaling network, parameter estimation, dynamic models, biological systems, mathematical models, metabolic network, parameter sensitivities, computational model, gene circuit, circuit models, experimental data, metabolic models, model system, scalable parameter, model parameters, parameter space, biomolecular networks, model<div style="font-size:11px;"><br><b>Abstract: </b><em>Parametric uncertainty is a particularly challenging and relevant aspect of systems analysis in domains such as systems biology where, both for inference and for assessing prediction uncertainties, it is essential to characterize the system behavior globally in the parameter space. However, current methods based on local approximations or on Monte-Cario sampling cope only insufficiently with high-dimensional parameter spaces associated with complex network models. Here, we propose an alternative deterministic methodology that relies on sparse polynomial approximations. We propose a deterministic computational interpolation scheme which identifies most significant expansion coefficients adaptively. We present its performance in kinetic model equations from computational systems biology with several hundred parameters and state variables, leading to numerical approximations of the parametric solution on the entire parameter space. The scheme is based on adaptive Smolyak interpolation of the parametric solution at judiciously and adaptively chosen points in parameter space. As Monte-Cario sampling, it is "non-intrusive" and well-suited for massively parallel implementation, but affords higher convergence rates. This opens up new avenues for large-scale dynamic network analysis by enabling scaling for many applications, including parameter estimation, uncertainty quantification, and systems design.</div>','<b>Title: </b>Measurement variation determines the gene network topology reconstructed from experimental data: a case study of the yeast cyclin network<br><b>Year: </b>2010<br><b>Author: </b> Ct. Cuong and  J. Vohradsky<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1096/fj.10-160515"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>FASEB JOURNAL<br><b>Cluster-Keywords: </b><em>information systems, gene expression, gene network, systems theory, system models, systems biology, systems information, network model, genetic network, mathematical models, dynamic systems, dynamic network, systemic model, network topology, information mechanisms, complex systems, systems complex, relational models, models main, information model<div style="font-size:11px;"><br><b>Abstract: </b><em>Inference of the topology of gene regulatory networks from experimental data is one of the primary challenges of systems biology. In an example of a genetic network of cyclins in the yeast cell cycle, we analyzed static genome-wide location data together with microarray kinetic measurements using a recurrent neural network-based model of gene expression and a newly developed, unbiased algorithm based on evolutionary programming principles. The modeling and simulation of gene expression dynamics identified cyclin genetic networks that were active during the cell cycle. We document that because there is inherent experimental variation, it is not possible to identify a single genetic network, only a set of equivalent networks with the same probability of occurrence. Analysis of these networks showed that each target gene was controlled by only a few regulators and that the control was robust. These results led to the reformulation of the cyclin genetic network in the yeast cell cycle as previously published. The analysis shows that with the methodologies that are currently available, it is not possible to predict only one genetic network; rather, we must work with the hypothesis of multiple, equivalent networks. Chromatin immunoprecipitation ChIP-on-chip experiments are not sufficient to predict the functional networks that are active during an investigated process. Such predictions must be considered as only potential, and their actual realization during particular cellular processes must be identified by incorporating both kinetic and other types of data.-To, C. C., Vohradsky, J. Measurement variation determines the gene network topology reconstructed from experimental data: a case study of the yeast cyclin network. FASEB J. 24, 3468-3478 2010. www.fasebj.org</div>','<b>Title: </b>Robust synthetic biology design: stochastic game theory approach<br><b>Year: </b>2009<br><b>Author: </b> Bs. Chen,  Ch. Chang and  Hc. Lee<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1093/bioinformatics/btp310"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>gene networks, synthetic gene, synthetic biology, regulatory networks, design method, robust synthetic, nonlinear stochastic, molecular noises, gene regulatory, interaction networks, external disturbances, robust stability, robust gene, host cell, gene circuit, cellular network, design specifications, design procedure, signal transduction, parameter fluctuations<div style="font-size:11px;"><br><b>Abstract: </b><em>Motivation: Synthetic biology is to engineer artificial biological systems to investigate natural biological phenomena and for a variety of applications. However, the development of synthetic gene networks is still difficult and most newly created gene networks are non-functioning due to uncertain initial conditions and disturbances of extra-cellular environments on the host cell. At present, how to design a robust synthetic gene network to work properly under these uncertain factors is the most important topic of synthetic biology.Results: A robust regulation design is proposed for a stochastic synthetic gene network to achieve the prescribed steady states under these uncertain factors from the minimax regulation perspective. This minimax regulation design problem can be transformed to an equivalent stochastic game problem. Since it is not easy to solve the robust regulation design problem of synthetic gene networks by non-linear stochastic game method directly, the Takagi-Sugeno T-S fuzzy model is proposed to approximate the non-linear synthetic gene network via the linear matrix inequality LMI technique through the Robust Control Toolbox in Matlab. Finally, an in silico example is given to illustrate the design procedure and to confirm the efficiency and efficacy of the proposed robust gene design method.</div>','<b>Title: </b>Formulating genome-scale kinetic models in the post-genome era<br><b>Year: </b>2008<br><b>Author: </b> N. Jamshidi and  Bo. Palsson<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1038/msb.2008.8"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MOLECULAR SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>immune system, network models, immune response, model, biological systems, models, model reduction, dynamic model, biological networks, system, mathematical models, biochemical network, response model, systems, systems biology, network, immune, biology model, reaction networks, system systems<div style="font-size:11px;"><br><b>Abstract: </b><em>The biological community is now awash in high-throughput data sets and is grappling with the challenge of integrating disparate data sets. Such integration has taken the form of statistical analysis of large data sets, or through the bottom-up reconstruction of reaction networks. While progress has been made with statistical and structural methods, large-scale systems have remained refractory to dynamic model building by traditional approaches. The availability of annotated genomes enabled the reconstruction of genome-scale networks, and now the availability of high-throughput metabolomic and fluxomic data along with thermodynamic information opens the possibility to build genome-scale kinetic models. We describe here a framework for building and analyzing such models. The mathematical analysis challenges are reflected in four foundational properties, i the decomposition of the Jacobian matrix into chemical, kinetic and thermodynamic information, ii the structural similarity between the stoichiometric matrix and the transpose of the gradient matrix, iii the duality transformations enabling either fluxes or concentrations to serve as the independent variables and iv the timescale hierarchy in biological networks. Recognition and appreciation of these properties highlight notable and challenging new in silico analysis issues.</div>','<b>Title: </b>Hunt for the tipping point during endocrine resistance process in breast cancer by dynamic network biomarkers<br><b>Year: </b>2019<br><b>Author: </b> R. Liu,  Jz. Wang,  M. Ukai,  K. Sewon,  P. Chen,  Y. Suzuki,  Hy. Wang,  K. Aihara,  M. Okada-Hatakeyama and  Ln. Chen<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1093/jmcb/mjy059"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF MOLECULAR CELL BIOLOGY<br><b>Cluster-Keywords: </b><em>systems biology, dynamic network, early differentiation, complex diseases, stem cells, erythroid cells, system dynamics, network biomarkers, immune system, tcm syndrome, dnb genes, gene expression, dnb model, stem cell, expression dynamics, model system, mathematical model, model dynamics, dynamic model, sensitivity analysis<div style="font-size:11px;"><br><b>Abstract: </b><em>Acquired drug resistance is the major reason why patients fail to respond to cancer therapies. It is a challenging task to determine the tipping point of endocrine resistance and detect the associated molecules. Derived from new systems biology theory, the dynamic network biomarker DNB method is designed to quantitatively identify the tipping point of a drastic system transition and can theoretically identify DNB genes that play key roles in acquiring drug resistance. We analyzed time-course mRNA sequence data generated from the tamoxifen-treated estrogen receptor ER-positive MCF-7 cell line, and identified the tipping point of endocrine resistance with its leading molecules. The results show that there is interplay between gene mutations and DNB genes, in which the accumulated mutations eventually affect the DNB genes that subsequently cause the change of transcriptional landscape, enabling full-blown drug resistance. Survival analyses based on clinical datasets validated that the DNB genes were associated with the poor survival of breast cancer patients. The results provided the detection for the pre-resistance state or early signs of endocrine resistance. Our predictive method may greatly benefit the scheduling of treatments for complex diseases in which patients are exposed to considerably different drugs and may become drug resistant.</div>','<b>Title: </b>Toward Time-Resolved Analysis of RNA Metabolism in Archaea Using 4-Thiouracil<br><b>Year: </b>2017<br><b>Author: </b> R. Knuppel,  C. Kuttenberger and  S. Ferreira-Cerca<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3389/fmicb.2017.00286"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>FRONTIERS IN MICROBIOLOGY<br><b>Cluster-Keywords: </b><em>gene expression, systems biology, metabolic network, network model, network analysis, dynamic model, signaling pathways, signaling network, regulatory network, dynamic network, biology approach, candidate genes, gene network, gene regulatory, biology analysis, mathematical model, network dynamic, biology network, metabolic pathways, pathway analysis<div style="font-size:11px;"><br><b>Abstract: </b><em>Archaea are widespread organisms colonizing almost every habitat on Earth. However, the molecular biology of archaea still remains relatively uncharacterized. RNA metabolism is a central cellular process, which has been extensively analyzed in both bacteria and eukarya. In contrast, analysis of RNA metabolism dynamic in archaea has been limited to date. To facilitate analysis of the RNA metabolism dynamic at a system-wide scale in archaea, we have established non-radioactive pulse labeling of RNA, using the nucleotide analog 4-thiouracil 4TU in two commonly used model archaea: the halophile Euryarchaeota Haloferax volcanii, and the thermo-acidophile Crenarchaeota Sulfolobus acidocaldarius. In this work, we show that 4TU pulse labeling can be efficiently performed in these two organisms in a dose-and time-dependent manner. In addition, our results suggest that uracil prototrophy had no critical impact on the overall 4TU incorporation in RNA molecules. Accordingly, our work suggests that 4TU incorporation can be widely performed in archaea, thereby expanding the molecular toolkit to analyze archaeal gene expression network dynamic in unprecedented detail.</div>','<b>Title: </b>Sampling saddle points on a free energy surface<br><b>Year: </b>2014<br><b>Author: </b> A. Samanta,  M. Chen,  Tq. Yu,  M. Tuckerman and  E. Weinan<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1063/1.4869980"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF CHEMICAL PHYSICS<br><b>Cluster-Keywords: </b><em>allosteric proteins, signaling networks, protein interactions, network model, molecular dynamics, protein complexes, allosteric network, signaling proteins, cell network, biochemical networks, free energy, combinatorial complexity, molecular species, biological systems, receptor signaling, systems biology, chemical species, model, dynamics simulations, cell signaling<div style="font-size:11px;"><br><b>Abstract: </b><em>Many problems in biology, chemistry, and materials science require knowledge of saddle points on free energy surfaces. These saddle points act as transition states and are the bottlenecks for transitions of the system between different metastable states. For simple systems in which the free energy depends on a few variables, the free energy surface can be precomputed, and saddle points can then be found using existing techniques. For complex systems, where the free energy depends on many degrees of freedom, this is not feasible. In this paper, we develop an algorithm for finding the saddle points on a high-dimensional free energy surface "on-the-fly" without requiring a priori knowledge the free energy function itself. This is done by using the general strategy of the heterogeneous multi-scale method by applying a macro-scale solver, here the gentlest ascent dynamics algorithm, with the needed force and Hessian values computed on-the-fly using a micro-scale model such as molecular dynamics. The algorithm is capable of dealing with problems involving many coarse-grained variables. The utility of the algorithm is illustrated by studying the saddle points associated with a the isomerization transition of the alanine dipeptide using two coarse-grained variables, specifically the Ramachandran dihedral angles, and b the beta-hairpin structure of the alanine decamer using 20 coarse-grained variables, specifically the full set of Ramachandran angle pairs associated with each residue. For the alanine decamer, we obtain a detailed network showing the connectivity of the minima obtained and the saddle-point structures that connect them, which provides a way to visualize the gross features of the high-dimensional surface. C 2014 AIP Publishing LLC.</div>','<b>Title: </b>Cellular automata with object-oriented features for parallel molecular network modeling<br><b>Year: </b>2005<br><b>Author: </b> H. Zhu,  Yh. Win,  S. Huang,  Y. Sun and  P. Dhar<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1109/TNB.2005.850473"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IEEE TRANSACTIONS ON NANOBIOSCIENCE<br><b>Cluster-Keywords: </b><em>biological networks, dynamic models, biological systems, model predictions, model parameters, systems biology, mathematical models, biological models, equation models, experimental data, molecular network, interaction networks, model errors, biological interaction, molecular interactions, model, dynamic systems, biological parameters, parameter uncertainty, growth model<div style="font-size:11px;"><br><b>Abstract: </b><em>Cellular automata are an important modeling paradigm for studying the dynamics of large, parallel systems composed of multiple, interacting components. However, to model biological systems, cellular automata need to be extended beyond the large-scale parallelism and intensive communication in order to capture two fundamental properties characteristic of complex biological systems: hierarchy and heterogeneity. This paper proposes extensions to a cellular automata language, Cellang, to meet this purpose. The extended language, with object-oriented features, can be used to describe the structure and activity of parallel molecular networks within cells. Capabilities of this new programming language include object structure to define molecular programs within a cell, floating-point data type and mathematical functions to perform quantitative computation, message passing capability to describe molecular interactions, as well as new operators, statements, and built-in functions. We discuss relevant programming issues of these features, including the object-oriented description of molecular interactions with molecule encapsulation, message passing, and the description of heterogeneity and anisotropy at the cell and molecule levels. By enabling the integration of modeling at the molecular level with system behavior at cell, tissue, organ, or even organism levels, the program will help improve our understanding of how complex and dynamic biological activities are generated and controlled by parallel functioning of molecular networks.</div>','<b>Title: </b>Proton transfer events in GFP<br><b>Year: </b>2011<br><b>Author: </b> M. Di Donato,  Ljgw. Van Wilderen,  Ihm. Van Stokkum,  Tc. Stuart,  Jtm. Kennis,  Kj. Hellingwerf,  R. Van Grondelle and  Ml. Groot<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1039/c1cp20387h"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PHYSICAL CHEMISTRY CHEMICAL PHYSICS<br><b>Cluster-Keywords: </b><em>sensitivity analysis, systems biology, robustness analysis, network model, network inference, regulatory network, systems analysis, gene expression, parametric uncertainty, biological networks, mathematical models, biochemical networks, biological functions, cellular networks, biological system, parametric sensitivity, dynamical models, dynamic system, networks analysis, gene regulatory<div style="font-size:11px;"><br><b>Abstract: </b><em>Proton transfer is one of the most important elementary processes in biology. Green fluorescent protein GFP serves as an important model system to elucidate the mechanistic details of this reaction, because in GFP proton transfer can be induced by light absorption. Illumination initiates proton transfer through a proton-wire, formed by the chromophore the proton donor, water molecule W22, Ser205 and Glu222 the acceptor, on a picosecond time scale. To obtain a more refined view of this process, we have used a combined approach of time resolved mid-infrared spectroscopy and visible pump-dump-probe spectroscopy to resolve with atomic resolution how and how fast protons move through this wire. Our results indicate that absorption of light by GFP induces in 3 ps 10 ps in D2O a shift of the equilibrium positions of all protons in the H-bonded network, leading to a partial protonation of Glu222 and to a so-called low barrier hydrogen bond LBHB for the chromophores proton, giving rise to dual emission at 475 and 508 nm. This state is followed by a repositioning of the protons on the wire in 10 ps 80 ps in D2O, ultimately forming the fully deprotonated chromophore and protonated Glu222.</div>','<b>Title: </b>Additive Functions in Boolean Models of Gene Regulatory Network Modules<br><b>Year: </b>2011<br><b>Author: </b> C. Darabos,  F. Di Cunto,  M. Tomassini,  Jh. Moore,  P. Provero and  M. Giacobini<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0025110"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>boolean network, boolean functions, regulatory networks, canalizing functions, network models, gene regulatory, boolean control, control networks, biological networks, boolean models, update functions, interaction networks, large network, regulatory functions, canalizing boolean, genetic regulatory, systems biology, driver nodes, networks bcns, biological systems<div style="font-size:11px;"><br><b>Abstract: </b><em>Gene-on-gene regulations are key components of every living organism. Dynamical abstract models of genetic regulatory networks help explain the genomes evolvability and robustness. These properties can be attributed to the structural topology of the graph formed by genes, as vertices, and regulatory interactions, as edges. Moreover, the actual gene interaction of each gene is believed to play a key role in the stability of the structure. With advances in biology, some effort was deployed to develop update functions in Boolean models that include recent knowledge. We combine real-life gene interaction networks with novel update functions in a Boolean model. We use two sub-networks of biological organisms, the yeast cell-cycle and the mouse embryonic stem cell, as topological support for our system. On these structures, we substitute the original random update functions by a novel threshold-based dynamic function in which the promoting and repressing effect of each interaction is considered. We use a third real-life regulatory network, along with its inferred Boolean update functions to validate the proposed update function. Results of this validation hint to increased biological plausibility of the threshold-based function. To investigate the dynamical behavior of this new model, we visualized the phase transition between order and chaos into the critical regime using Derrida plots. We complement the qualitative nature of Derrida plots with an alternative measure, the criticality distance, that also allows to discriminate between regimes in a quantitative way. Simulation on both real-life genetic regulatory networks show that there exists a set of parameters that allows the systems to operate in the critical region. This new model includes experimentally derived biological information and recent discoveries, which makes it potentially useful to guide experimental research. The update function confers additional realism to the model, while reducing the complexity and solution space, thus making it easier to investigate.</div>','<b>Title: </b>Dynamical modelling of secondary metabolism and metabolic switches in Streptomyces xiamenensis 318<br><b>Year: </b>2019<br><b>Author: </b> Xm. Zhu,  Xx. Zhang,  Rt. Cheng,  Hl. Yu,  Rs. Yuan,  Xl. Bu,  J. Xu,  P. Ao,  Yc. Chen and  Mj. Xu<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1098/rsos.190418"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ROYAL SOCIETY OPEN SCIENCE<br><b>Cluster-Keywords: </b><em>gene regulatory, regulatory network, systems biology, cell differentiation, cell fate, model systems, potential landscape, network grn, dynamical systems, melanocyte differentiation, network models, gene expression, network dynamics, differentiation genes, dynamical models, dynamical networks, cellular differentiation, stem cells, attractor network, developmental gene<div style="font-size:11px;"><br><b>Abstract: </b><em>The production of secondary metabolites, while important for bioengineering purposes, presents a paradox in itself. Though widely existing in plants and bacteria, they have no definite physiological roles. Yet in both native habitats and laboratories, their production appears robust and follows apparent metabolic switches. We show in this work that the enzyme-catalysed process may improve the metabolic stability of the cells. The latter can be responsible for the overall metabolic behaviours such as dynamic metabolic landscape, metabolic switches and robustness, which can in turn affect the genetic formation of the organism in question. Mangrove-derived Streptomyces xiamenensis 318, with a relatively compact genome for secondary metabolism, is used as a model organism in our investigation. Integrated studies via kinetic metabolic modelling, transcriptase measurements and metabolic profiling were performed on this strain. Our results demonstrate that the secondary metabolites increase the metabolic fitness of the organism via stabilizing the underlying metabolic network. And the fluxes directing to NADH, NADPH, acetyl-CoA and glutamate provide the key switches for the overall and secondary metabolism. The information may be helpful for improving the xiamenmycin production on the strain.</div>','<b>Title: </b>Dynamic network-based epistasis analysis: Boolean examples<br><b>Year: </b>2011<br><b>Author: </b> E. Azpeitia,  M. Benitez,  P. Padilla-Longoria,  C. Espinosa-Soto and  Er. Alvarez-Buylla<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3389/fpls.2011.00092"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>FRONTIERS IN PLANT SCIENCE<br><b>Cluster-Keywords: </b><em>network models, gene network, regulatory networks, boolean networks, gene expression, systems biology, neural networks, network dynamics, gene regulatory, biological networks, network structure, gap gene, gene regulation, cell cycle, network, gene interactions, signaling network, quantum systems, system dynamics, networks<div style="font-size:11px;"><br><b>Abstract: </b><em>In this article we focus on how the hierarchical and single-path assumptions of epistasis analysis can bias the inference of gene regulatory networks. Here we emphasize the critical importance of dynamic analyses, and specifically illustrate the use of Boolean network models. Epistasis in a broad sense refers to gene interactions, however, as originally proposed by Bateson, epistasis is defined as the blocking of a particular allelic effect due to the effect of another allele at a different locus herein, classical epistasis. Classical epistasis analysis has proven powerful and useful, allowing researchers to infer and assign directionality to gene interactions. As larger data sets are becoming available, the analysis of classical epistasis is being complemented with computer science tools and system biology approaches. We show that when the hierarchical and single-path assumptions are not met in classical epistasis analysis, the access to relevant information and the correct inference of gene interaction topologies is hindered, and it becomes necessary to consider the temporal dynamics of gene interactions. The use of dynamical networks can overcome these limitations. We particularly focus on the use of Boolean networks that, like classical epistasis analysis, relies on logical formalisms, and hence can complement classical epistasis analysis and relax its assumptions. We develop a couple of theoretical examples and analyze them from a dynamic Boolean network model perspective. Boolean networks could help to guide additional experiments and discern among alternative regulatory schemes that would be impossible or difficult to infer without the elimination of these assumption from the classical epistasis analysis. We also use examples from the literature to show how a Boolean network-based approach has resolved ambiguities and guided epistasis analysis. Our article complements previous accounts, not only by focusing on the implications of the hierarchical and single-path assumption, but also by demonstrating the importance of considering temporal dynamics, and specifically introducing the usefulness of Boolean network models and also reviewing some key properties of network approaches.</div>','<b>Title: </b>Activated Oncogenic Pathway Modifies Iron Network in Breast Epithelial Cells: A Dynamic Modeling Perspective<br><b>Year: </b>2017<br><b>Author: </b> J. Chifman,  S. Arat,  Zy. Deng,  E. Lemler,  Jc. Pino,  La. Harris,  Ma. Kochen,  Cf. Lopez,  Sa. Akman,  Fm. Torti,  Sv. Torti and  R. Laubenbacher<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pcbi.1005352"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS COMPUTATIONAL BIOLOGY<br><b>Cluster-Keywords: </b><em>boolean network, network models, qualitative models, systems biology, mathematical modeling, boolean models, signaling networks, dynamic models, dynamic modeling, signaling dynamics, logic models, dynamic network, mathematical models, boolean modeling, signal transduction, regulatory networks, drug resistance, biological networks, network approach, network modeling<div style="font-size:11px;"><br><b>Abstract: </b><em>Dysregulation of iron metabolism in cancer is well documented and it has been suggested that there is interdependence between excess iron and increased cancer incidence and progression. In an effort to better understand the linkages between iron metabolism and breast cancer, a predictive mathematical model of an expanded iron homeostasis pathway was constructed that includes species involved in iron utilization, oxidative stress response and oncogenic pathways. The model leads to three predictions. The first is that overexpression of iron regulatory protein 2 IRP2 recapitulates many aspects of the alterations in free iron and iron-related proteins in cancer cells without affecting the oxidative stress response or the oncogenic pathways included in the model. This prediction was validated by experimentation. The second prediction is that iron-related proteins are dramatically affected by mitochondrial ferritin overexpression. This prediction was validated by results in the pertinent literature not used for model construction. The third prediction is that oncogenic Ras pathways contribute to altered iron homeostasis in cancer cells. This prediction was validated by a combination of simulation experiments of Ras overexpression and catalase knockout in conjunction with the literature. The model successfully captures key aspects of iron metabolism in breast cancer cells and provides a framework upon which more detailed models can be built.</div>','<b>Title: </b>Systems Modeling of Anti-apoptotic Pathways in Prostate Cancer: Psychological Stress Triggers a Synergism Pattern Switch in Drug Combination Therapy<br><b>Year: </b>2013<br><b>Author: </b> Xq. Sun,  Jg. Bao,  Kc. Nelson,  Kc. Li,  G. Kulik and  Xb. Zhou<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pcbi.1003358"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS COMPUTATIONAL BIOLOGY<br><b>Cluster-Keywords: </b><em>stem cell, stem cells, systems biology, mathematical models, cell biology, network model, signaling pathways, cell fate, computational models, cancer cells, gene expression, network dynamics, embryonic stem, signaling network, prostate cancer, mathematical modeling, psychological stress, apoptosis signaling, computational systems, molecular mechanisms<div style="font-size:11px;"><br><b>Abstract: </b><em>Prostate cancer patients often have increased levels of psychological stress or anxiety, but the molecular mechanisms underlying the interaction between psychological stress and prostate cancer as well as therapy resistance have been rarely studied and remain poorly understood. Recent reports show that stress inhibits apoptosis in prostate cancer cells via epinephrine/beta2 adrenergic receptor/PKA/BAD pathway. In this study, we used experimental data on the signaling pathways that control BAD phosphorylation to build a dynamic network model of apoptosis regulation in prostate cancer cells. We then compared the predictive power of two different models with or without the role of Mcl-1, which justified the role of Mcl-1 stabilization in anti-apoptotic effects of emotional stress. Based on the selected model, we examined and quantitatively evaluated the induction of apoptosis by drug combination therapies. We predicted that the combination of PI3K inhibitor LY294002 and inhibition of BAD phosphorylation at S112 would produce the best synergistic effect among 8 interventions examined. Experimental validation confirmed the effectiveness of our predictive model. Moreover, we found that epinephrine signaling changes the synergism pattern and decreases efficacy of combination therapy. The molecular mechanisms responsible for therapeutic resistance and the switch in synergism were explored by analyzing a network model of signaling pathways affected by psychological stress. These results provide insights into the mechanisms of psychological stress signaling in therapy-resistant cancer, and indicate the potential benefit of reducing psychological stress in designing more effective therapies for prostate cancer patients.Author Summary Psychological stress and anxiety are often experienced by prostate cancer patients, but the underlying mechanisms of interactions between psychological stress and cancer development, as well as drug resistance, are unclear. Here, we employed a systems biology approach to study interactions between stress-activated epinephrine/beta2 adrenergic receptor/protein kinase A signaling and a regulatory network that controls apoptosis in prostate cancer cells. We developed a dynamic network model of signaling pathways that control apoptosis in prostate cancer cells and quantitatively evaluated the effects of stress-activated signaling on apoptosis induced by drug combinations. Experimental data were used to guide modeling, to fit the unknown parameters and validate the model. Based on our model we found that epinephrine/beta2 adrenergic receptor/protein kinase A signaling can decrease drug efficiency, and can shift the effect of drug combination from synergy to antagonism. We also predicted that in addition to BAD phosphorylation Mcl-1 expression could be upregulated by stress/epinephrine signaling to inhibit apoptosis. This study provides insights into the mechanisms of psychological stress signaling in therapy-resistant cancer, and suggests that reducing psychological stress could help to make prostate cancer treatment more effective.</div>','<b>Title: </b>Smart Buildings IoT Networks Accuracy Evolution Prediction to Improve Their Reliability Using a Lotka-Volterra Ecosystem Model<br><b>Year: </b>2019<br><b>Author: </b> R. Casado-Vara,  A. Canal-Alonso,  A. Martin-Del Rey,  F. De La Prieta and  J. Prieto<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3390/s19214642"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SENSORS<br><b>Cluster-Keywords: </b><em>network models, population model, field model, neural population, neural field, iot networks, neuronal networks, neural network, differential equations, parameter neural, lotkavolterra model, model, neuron models, model neurons, model leads, ecosystem model, iot devices, noise model, neuronal models, inhibitory synaptic<div style="font-size:11px;"><br><b>Abstract: </b><em>Internet of Things IoT is the paradigm that has largely contributed to the development of smart buildings in our society. This technology makes it possible to monitor all aspects of the smart building and to improve its operation. One of the main challenges encountered by IoT networks is that the the data they collect may be unreliable since IoT devices can lose accuracy for several reasons sensor wear, sensor aging, poorly constructed buildings, etc.. The aim of our work is to study the evolution of IoT networks over time in smart buildings. The hypothesis we have tested is that, by amplifying the Lotka-Volterra equations as a community of living organisms an ecosystem model, the reliability of the system and its components can be predicted. This model comprises a set of differential equations that describe the relationship between an IoT network and multiple IoT devices. Based on the Lotka-Volterra model, in this article, we propose a model in which the predators are the non-precision IoT devices and the prey are the precision IoT devices. Furthermore, a third species is introduced, the maintenance staff, which will impact the interaction between both species, helping the prey to survive within the ecosystem. This is the first Lotka-Volterra model that is applied in the field of IoT. Our work establishes a proof of concept in the field and opens a wide spectrum of applications for biology models to be applied in IoT.</div>','<b>Title: </b>A model integration approach linking signalling and gene-regulatory logic with kinetic metabolic models<br><b>Year: </b>2014<br><b>Author: </b> A. Ryll,  J. Bucher,  A. Bonin,  S. Bongard,  E. Goncalves,  J. Saez-Rodriguez,  J. Niklas and  S. Klamt<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.biosystems.2014.07.002"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOSYSTEMS<br><b>Cluster-Keywords: </b><em>systems biology, network models, signaling network, parameter estimation, dynamic models, biological systems, mathematical models, metabolic network, parameter sensitivities, computational model, gene circuit, circuit models, experimental data, metabolic models, model system, scalable parameter, model parameters, parameter space, biomolecular networks, model<div style="font-size:11px;"><br><b>Abstract: </b><em>Systems biology has to increasingly cope with large- and multi-scale biological systems. Many successful in silico representations and simulations of various cellular modules proved mathematical modelling to be an important tool in gaining a solid understanding of biological phenomena. However, models spanning different functional layers e.g. metabolism, signalling and gene regulation are still scarce. Consequently, model integration methods capable of fusing different types of biological networks and various model formalisms become a key methodology to increase the scope of cellular processes covered by mathematical models. Here we propose a new integration approach to couple logical models of signalling or/and gene-regulatory networks with kinetic models of metabolic processes. The procedure ends up with an integrated dynamic model of both layers relying on differential equations. The feasibility of the approach is shown in an illustrative case study integrating a kinetic model of central metabolic pathways in hepatocytes with a Boolean logical network depicting the hormonally induced signal transduction and gene regulation events involved. In silico simulations demonstrate the integrated model to qualitatively describe the physiological switch-like behaviour of hepatocytes in response to nutritionally regulated changes in extracellular glucagon and insulin levels. A simulated failure mode scenario addressing insulin resistance furthermore illustrates the pharmacological potential of a model covering interactions between signalling, gene regulation and metabolism. C 2014 Elsevier Ireland Ltd. All rights reserved.</div>','<b>Title: </b>Qualitative simulation of genetic regulatory networks using piecewise-linear models<br><b>Year: </b>2004<br><b>Author: </b> H. De Jong,  Jl. Gouze,  C. Hernandez,  M. Page,  T. Sari and  J. Geiselmann<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.bulm.2003.08.010"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BULLETIN OF MATHEMATICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>regulatory networks, network dynamics, gene regulatory, network model, genetic network, gene networks, genetic regulatory, mathematical model, parameter space, network system, interaction networks, system dynamics, network structure, network, dynamic networks, network reduction, models models, biological networks, network motifs, biological systems<div style="font-size:11px;"><br><b>Abstract: </b><em>In order to cope with the large amounts of data that have become available in genomics, mathematical tools for the analysis of networks of interactions between genes, proteins, and other molecules are indispensable. We present a method for the qualitative simulation of genetic regulatory networks, based on a class of piecewise-linear PL differential equations that has been well-studied in mathematical biology. The simulation method is well-adapted to state-of-the-art measurement techniques in genomics, which often provide qualitative and coarse-grained descriptions of genetic regulatory networks. Given a qualitative model of a genetic regulatory network, consisting, of a system of PL differential equations and inequality constraints on the parameter values, the method produces a graph of qualitative states and transitions between qualitative states, summarizing the qualitative dynamics of the system. The qualitative simulation method has been implemented in Java in the computer tool Genetic Network Analyzer. C 2003 Sociey for Mathematical Biology. Published by Elsevier Ltd. All rights reserved.</div>','<b>Title: </b>Current approaches to gene regulatory network modelling<br><b>Year: </b>2007<br><b>Author: </b> T. Schlitt and  A. Brazma<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1471-2105-8-S6-S9"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>gene regulatory, gene expression, regulatory networks, expression data, regulatory pathway, regulatory genes, network dynamics, microarray data, genetic networks, transcription factor, models network, target gene, upstream regulatory, network topology, pathway network, gene functional, expression profile, transcriptional network, dynamic model, transcriptional regulatory<div style="font-size:11px;"><br><b>Abstract: </b><em>Many different approaches have been developed to model and simulate gene regulatory networks. We proposed the following categories for gene regulatory network models: network parts lists, network topology models, network control logic models, and dynamic models. Here we will describe some examples for each of these categories. We will study the topology of gene regulatory networks in yeast in more detail, comparing a direct network derived from transcription factor binding data and an indirect network derived from genome-wide expression data in mutants. Regarding the network dynamics we briefly describe discrete and continuous approaches to network modelling, then describe a hybrid model called Finite State Linear Model and demonstrate that some simple network dynamics can be simulated in this model.</div>','<b>Title: </b>Reducing the complexity of mathematical models for the plant circadian clock by distributed delays<br><b>Year: </b>2019<br><b>Author: </b> It. Tokuda,  Oe. Akman and  Jcw. Locke<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.jtbi.2018.12.014"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF THEORETICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>circadian clock, biological systems, network models, mathematical models, clock gene, systems biology, model selection, biological networks, network data, experimental data, model parameters, network dynamics, plant circadian, regulatory network, clock model, plant clock, complex network, clock circuit, approximate bayesian, stochastic models<div style="font-size:11px;"><br><b>Abstract: </b><em>A major bottleneck in the modelling of biological networks is the parameter explosion problem - the exponential increase in the number of parameters that need to be optimised to data as the size of the model increases. Here, we address this problem in the context of the plant circadian clock by applying the method of distributed delays. We show that using this approach, the system architecture can be simplified efficiently - reducing the number of parameters - whilst still preserving the core mechanistic dynamics of the gene regulatory network. Compared to models with discrete time-delays, which are governed by functional differential equations, the distributed delay models can be converted into sets of equivalent ordinary differential equations, enabling the use of standard methods for numerical integration, and for stability and bifurcation analyses. We demonstrate the efficiency of our modelling approach by applying it to three exemplar mathematical models of the Arabidopsis circadian clock of varying complexity, obtaining significant reductions in complexity in each case. Moreover, we revise one of the most up-to-date Arabidopsis models, updating the regulation of the PRR9 and PRR7 genes by LHY in accordance with recent experimental data. The revised model more accurately reproduces the LHY-induction experiments of core clock genes, compared with the original model. Our work thus shows that the method of distributed delays facilitates the optimisation and reformulation of genetic network models. C 2018 Elsevier Ltd. All rights reserved.</div>','<b>Title: </b>Complexity, communication between cells, and identifying the functional components of living systems: Some observations<br><b>Year: </b>1996<br><b>Author: </b> Dc. Mikulecky<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/BF00046527"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ACTA BIOTHEORETICA<br><b>Cluster-Keywords: </b><em>information systems, gene expression, gene network, systems theory, system models, systems biology, systems information, network model, genetic network, mathematical models, dynamic systems, dynamic network, systemic model, network topology, information mechanisms, complex systems, systems complex, relational models, models main, information model<div style="font-size:11px;"><br><b>Abstract: </b><em>The concept of complexity has become very important in theoretical biology. It is a many faceted concept and too new and ill defined to have a universally accepted meaning. This review examines the development of this concept from the point of view of its usefulness as a criteria for the study of living systems to see what it has to offer as a new approach. In particular, one definition of complexity has been put forth which has the necessary precision and rigor to be considered as a useful categorization of systems, especially as it pertains to those we call living. This definition, due to Robert Rosen, has been developed in a number of works and involves some deep new concepts about the way we view systems. In particular, it focuses on the way we view the world and actually practice science through the use of the modelling relation. This mathematical object models the process by which we assign meaning to the world we perceive. By using the modelling relation, it is possible to identify the subjective nature of our practices and deal with this issue explicitly. By so doing, it becomes clear that our notion of complexity and especially its most popular manifestations, is in large part a product of the historical processes which lead to the present state of scientific epistemology. In particular, it is a reaction to the reductionist/mechanistic view of nature which can be termed the Newtonian Paradigm. This approach to epistemology has dominated for so long that its use as a model has become implicit in most of what we do in and out of science. The alternative to this approach is examined and related to the special definition of complexity given by Rosen. Some historical examples are used to emphasize the dependence of our view of what is complex in a popular sense on the ever changing state of our knowledge. The role of some popular concepts such as chaotic dynamics are examined in this context The fields of artificial life and related areas are also viewed from the perspective of this rigorous view of complexity and found lacking. The notion that in some way life exists at the edge of chaos is examined from the perspective of the second law of thermodynamics given by Schneider and Kay. Finally, the causal elements in complex systems are explored in relation to complexity. Rosen has shown that a clear difference in causal relations exists between complex and simple systems and that this difference leads to a uniquely useful definition of what we mean by living. Rosen makes it very clear that the class of systems which are complex is a much larger class than those which we call living. For that reason, the focus of this review will be on complexity as a stepping stone towards the deeper question of what makes a system alive.</div>','<b>Title: </b>Cycling expression and cooperative operator interaction in the trp operon of Escherichia coli<br><b>Year: </b>2010<br><b>Author: </b> A. Hernandez-Valdez,  M. Santillan and  Es. Zeron<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.jtbi.2009.12.005"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF THEORETICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>gene expression, gene network, stochastic model, gene regulatory, systems biology, expression systems, gene regulation, network activity, expression dynamics, noise control, regulatory network, deterministic models, network components, stochastic modeling, regulation system, escherichia coli, regulatory system, synthetic gene, regulatory genes, gene<div style="font-size:11px;"><br><b>Abstract: </b><em>Oscillatory behaviour in the tryptophan operon of an Escherichia coli mutant strain lacking the enzyme-inhibition regulatory mechanism has been observed by Bliss et al. but not confirmed by others. This behaviour could be important from the standpoint of synthetic biology, whose goals include the engineering of intracellular genetic oscillators. This work is devoted to investigating, from a mathematical modelling point of view, the possibility that the trp operon of the E coli inhibition-free strain expresses cyclically. For that we extend a previously introduced model for the regulatory pathway of the tryptophan operon in Escherichia coli to account for the observed multiplicity and cooperativity of repressor binding sites. Thereafter we investigate the model dynamics using deterministic numeric solutions, stochastic simulations, and analytic studies. Our results suggest that a quasi-periodic behaviour could be observed in the trp operon expression level of single bacteria. C 2009 Elsevier Ltd. All rights reserved.</div>','<b>Title: </b>A microfluidic-based dynamic microarray system with single-layer pneumatic valves for immobilization and selective retrieval of single microbeads<br><b>Year: </b>2014<br><b>Author: </b> H. Kim and  J. Kim<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s10404-013-1267-9"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MICROFLUIDICS AND NANOFLUIDICS<br><b>Cluster-Keywords: </b><em>chaotic system, control systems, neural networks, jerk chaotic, torch chaotic, mode control, sliding mode, plasma torch, dynamic neural, mitochondrial dynamics, system, systems biology, adaptive integral, model system, rodtype plasma, integral sliding, systems, jerk system, cancer cell, comparison method<div style="font-size:11px;"><br><b>Abstract: </b><em>A simple yet effective dynamic bead-based microarray is necessary for multiplexed high-throughput screening applications in the fields of biology and chemistry. This paper introduces a microfluidic-based dynamic microbead array system using pneumatically driven elastomeric valves integrated with a microchannel in a single polydimethylsiloxane PDMS layer that performs the following functions: single-microbead arraying with loading and trapping efficiencies of 100 , sequential microbead release for selective retrieval of microbeads of interest, and rapid microarray resettability < 1 s. The key feature is the utilization of an elastomeric membrane as a valve for trapping and releasing single microbeads; this membrane is deformable depending on the applied pneumatic pressure, thereby simply providing a dual trap-and-release function. We propose an effective single-microbead-trapping mechanism based on a dynamic flow-change network and a mathematical model as the design criterion of a trapping site. A sequential microbead release technique via a multistep "release-retrap-and-repeat" method was developed for the selective retrieval of trapped microbeads with a simple configuration consisting of a single PDMS layer and a simple macro-to-micro connection. The proposed dynamic microbead array could be a powerful tool for high-throughput multiplex bead-based drug screening or disease diagnosis.</div>','<b>Title: </b>Dynamic Modeling of Cell-Free Biochemical Networks Using Effective Kinetic Models<br><b>Year: </b>2015<br><b>Author: </b> Ja. Wayman,  A. Sagar and  Jd. Varner<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3390/pr3010138"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PROCESSES<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Cell-free systems offer many advantages for the study, manipulation and modeling of metabolism compared to in vivo processes. Many of the challenges confronting genome-scale kinetic modeling can potentially be overcome in a cell-free system. For example, there is no complex transcriptional regulation to consider, transient metabolic measurements are easier to obtain, and we no longer have to consider cell growth. Thus, cell-free operation holds several significant advantages for model development, identification and validation. Theoretically, genome-scale cell-free kinetic models may be possible for industrially important organisms, such as E. coli, if a simple, tractable framework for integrating allosteric regulation with enzyme kinetics can be formulated. Toward this unmet need, we present an effective biochemical network modeling framework for building dynamic cell-free metabolic models. The key innovation of our approach is the integration of simple effective rules encoding complex allosteric regulation with traditional kinetic pathway modeling. We tested our approach by modeling the time evolution of several hypothetical cell-free metabolic networks. We found that simple effective rules, when integrated with traditional enzyme kinetic expressions, captured complex allosteric patterns such as ultrasensitivity or non-competitive inhibition in the absence of mechanistic information. Second, when integrated into network models, these rules captured classic regulatory patterns such as product-induced feedback inhibition. Lastly, we showed, at least for the network architectures considered here, that we could simultaneously estimate kinetic parameters and allosteric connectivity from synthetic data starting from an unbiased collection of possible allosteric structures using particle swarm optimization. However, when starting with an initial population that was heavily enriched with incorrect structures, our particle swarm approach could converge to an incorrect structure. While only an initial proof-of-concept, the framework presented here could be an important first step toward genome-scale cell-free kinetic modeling of the biosynthetic capacity of industrially important organisms.</div>','<b>Title: </b>Parasites in food webs: the ultimate missing links<br><b>Year: </b>2008<br><b>Author: </b> Kd. Lafferty,  S. Allesina,  M. Arim,  Cj. Briggs,  G. De Leo,  Ap. Dobson,  Ja. Dunne,  Ptj. Johnson,  Am. Kuris,  Dj. Marcogliese,  Nd. Martinez,  J. Memmott,  Pa. Marquet,  Jp. Mclaughlin,  Ea. Mordecai,  M. Pascual,  R. Poulin and  Dw. Thieltges<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1111/j.1461-0248.2008.01174.x"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ECOLOGY LETTERS<br><b>Cluster-Keywords: </b><em>network ecology, food webs, network models, ecological systems, food web, systems biology, ecological networks, ecosystem model, signalling networks, model results, interaction networks, dynamic model, ecological interactions, network analysis, model network, network structure, transport networks, systems network, ecosystem dynamics, restoration ecology<div style="font-size:11px;"><br><b>Abstract: </b><em>Parasitism is the most common consumer strategy among organisms, yet only recently has there been a call for the inclusion of infectious disease agents in food webs. The value of this effort hinges on whether parasites affect food-web properties. Increasing evidence suggests that parasites have the potential to uniquely alter food-web topology in terms of chain length, connectance and robustness. In addition, parasites might affect food-web stability, interaction strength and energy flow. Food-web structure also affects infectious disease dynamics because parasites depend on the ecological networks in which they live. Empirically, incorporating parasites into food webs is straightforward. We may start with existing food webs and add parasites as nodes, or we may try to build food webs around systems for which we already have a good understanding of infectious processes. In the future, perhaps researchers will add parasites while they construct food webs. Less clear is how food-web theory can accommodate parasites. This is a deep and central problem in theoretical biology and applied mathematics. For instance, is representing parasites with complex life cycles as a single node equivalent to representing other species with ontogenetic niche shifts as a single node Can parasitism fit into fundamental frameworks such as the niche model Can we integrate infectious disease models into the emerging field of dynamic food-web modelling Future progress will benefit from interdisciplinary collaborations between ecologists and infectious disease biologists.</div>','<b>Title: </b>Reconstructing biochemical pathways from time course data<br><b>Year: </b>2007<br><b>Author: </b> J. Srividhya,  Ej. Crampin,  Pe. Mcsharry and  S. Schnell<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1002/pmic.200600428"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PROTEOMICS<br><b>Cluster-Keywords: </b><em>systems biology, dynamic models, biological systems, model, dynamic systems, computational systems, network model, experimental data, model parameters, systems, systems model, models, pathway systems, data, biochemical pathway, dynamics model, biochemical networks, computational models, network, system<div style="font-size:11px;"><br><b>Abstract: </b><em>Time series data on biochemical reactions reveal transient behavior, away from chemical equilibrium, and contain information on the dynamic interactions among reacting components. However, this information can be difficult to extract using conventional analysis techniques. We present a new method to infer biochemical pathway mechanisms from time course data using a global nonlinear modeling technique to identify the elementary reaction steps which constitute the pathway. The method involves the generation of a complete dictionary of polynomial basis functions based on the law of mass action. Using these basis functions, there are two approaches to model construction, namely the general to specific and the specific to general approach. We demonstrate that our new methodology reconstructs the chemical reaction steps and connectivity of the glycolytic pathway of Lactococcus lactis from time course experimental data.</div>','<b>Title: </b>Network Models for Dissecting Plant Development by Functional Mapping<br><b>Year: </b>2009<br><b>Author: </b> S. Wu,  Js. Yap,  Y. Li,  Q. Li,  Gf. Fu,  Jh. Li,  K. Das,  A. Berg,  Yr. Zeng and  Rl. Wu<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.2174/157489309789071093"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>CURRENT BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>systems biology, regulatory network, interaction networks, network model, gene regulatory, network, network biology, networks, gene network, response networks, network dynamics, dynamic model, cell networks, gene expression, biological networks, system dynamics, dynamic network, model, biological systems, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Understanding the genetic machinery of plant growth and development is of fundamental importance in agriculture and biology. Recently, a novel statistical framework, coined functional mapping, has been developed to study the genetic architecture of the dynamic pattern of phenotypic development at different levels of organization. By integrating mathematical aspects of cellular and biological processes, functional mapping provides a quantitative platform in which a seemingly unlimited number of hypotheses about the interplay between genes and development can be asked and tested. However, plant development involves a series of multi-hierarchical, sequential pathways from DNA to mRNA to proteins to metabolites and finally to high-order phenotypes, and thus it is unlikely that the control mechanisms of plant development can be understood using genetic knowledge alone. Here, we describe a network biology approach for functional mapping of phenotypic formation and progression through their underlying biochemical pathways. The integration of functional mapping with information-rich spectroscopic data sets including transcriptome, proteome, and metabolome can be used to model and predict physiological variation and plant development, and will pave the way for future genetic studies capable of addressing the complex nature of growth and development.</div>','<b>Title: </b>Concurrent heterogeneous neural model simulation on real-time neuromimetic hardware<br><b>Year: </b>2011<br><b>Author: </b> A. Rast,  F. Galluppi,  S. Davies,  L. Plana,  C. Patterson,  T. Sharp,  D. Lester and  S. Furber<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.neunet.2011.06.014"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>NEURAL NETWORKS<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Dedicated hardware is becoming increasingly essential to simulate emerging very-large-scale neural models. Equally, however, it needs to be able to support multiple models of the neural dynamics, possibly operating simultaneously within the same system. This may be necessary either to simulate large models with heterogeneous neural types, or to simplify simulation and analysis of detailed, complex models in a large simulation by isolating the new model to a small subpopulation of a larger overall network. The SpiNNaker neuromimetic chip is a dedicated neural processor able to support such heterogeneous simulations. Implementing these models on-chip uses an integrated library-based tool chain incorporating the emerging PyNN interface that allows a modeller to input a high-level description and use an automated process to generate an on-chip simulation. Simulations using both LIF and Izhikevich models demonstrate the ability of the SpiNNaker system to generate and simulate heterogeneous networks on-chip, while illustrating, through the network-scale effects of wavefront synchronisation and burst gating, methods that can provide effective behavioural abstractions for large-scale hardware modelling. SpiNNakers asynchronous virtual architecture permits greater scope for model exploration, with scalable levels of functional and temporal abstraction, than conventional or neuromorphic computing platforms. The complete system illustrates a potential path to understanding the neural model of computation, by building and breaking neural models at various scales, connecting the blocks, then comparing them against the biology: computational cognitive neuroscience. C 2011 Elsevier Ltd. All rights reserved.</div>','<b>Title: </b>Profile likelihood-based analyses of infectious disease models<br><b>Year: </b>2018<br><b>Author: </b> C. Tonsing,  J. Timmer and  C. Kreutz<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1177/0962280217746444"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>STATISTICAL METHODS IN MEDICAL RESEARCH<br><b>Cluster-Keywords: </b><em>complex networks, complex systems, network theory, network structure, biological network, social network, network science, dynamic networks, network systems, network model, network topology, systems biology, network, network analysis, system dynamics, networks, realworld networks, network automata, epidemic network, model network<div style="font-size:11px;"><br><b>Abstract: </b><em>Ordinary differential equation models are frequently applied to describe the temporal evolution of epidemics. However, ordinary differential equation models are also utilized in other scientific fields. We summarize and transfer state-of-the art approaches from other fields like Systems Biology to infectious disease models. For this purpose, we use a simple SIR model with data from an influenza outbreak at an English boarding school in 1978 and a more complex model of a vector-borne disease with data from the Zika virus outbreak in Colombia in 2015-2016. Besides parameter estimation using a deterministic multistart optimization approach, a multitude of analyses based on the profile likelihood are presented comprising identifiability analysis and model reduction. The analyses were performed using the freely available modeling framework Data2Dynamics data2dynamics.org which has been awarded as best performing within the DREAM6 parameter estimation challenge and in the DREAM7 network reconstruction challenge.</div>','<b>Title: </b>Drug Target Optimization in Chronic Myeloid Leukemia Using Innovative Computational Platform<br><b>Year: </b>2015<br><b>Author: </b> R. Chuang,  Ba. Hall,  D. Benque,  B. Cook,  S. Ishtiaq,  N. Piterman,  A. Taylor,  M. Vardi,  S. Koschmieder,  B. Gottgens and  J. Fisher<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1038/srep08190"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SCIENTIFIC REPORTS<br><b>Cluster-Keywords: </b><em>gene regulatory, regulatory network, systems biology, cell differentiation, cell fate, model systems, potential landscape, network grn, dynamical systems, melanocyte differentiation, network models, gene expression, network dynamics, differentiation genes, dynamical models, dynamical networks, cellular differentiation, stem cells, attractor network, developmental gene<div style="font-size:11px;"><br><b>Abstract: </b><em>Chronic Myeloid Leukemia CML represents a paradigm for the wider cancer field. Despite the fact that tyrosine kinase inhibitors have established targeted molecular therapy in CML, patients often face the risk of developing drug resistance, caused by mutations and/or activation of alternative cellular pathways. To optimize drug development, one needs to systematically test all possible combinations of drug targets within the genetic network that regulates the disease. The BioModelAnalyzer BMA is a user-friendly computational tool that allows us to do exactly that. We used BMA to build a CML network-model composed of 54 nodes linked by 104 interactions that encapsulates experimental data collected from 160 publications. While previous studies were limited by their focus on a single pathway or cellular process, our executable model allowed us to probe dynamic interactions between multiple pathways and cellular outcomes, suggest new combinatorial therapeutic targets, and highlight previously unexplored sensitivities to Interleukin-3.</div>','<b>Title: </b>A systems biology framework for modeling metabolic enzyme inhibition of Mycobacterium tuberculosis<br><b>Year: </b>2009<br><b>Author: </b> X. Fang,  A. Wallqvist and  J. Reifman<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-3-92"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>metabolic network, metabolic model, dynamic flux, growth rate, genomescale metabolic, flux balance, balance analysis, metabolic flux, metabolic pathways, dynamic model, metabolic reactions, cellular growth, regulatory network, systems biology, optimal growth, metabolic, modeling metabolic, growth conditions, dynamic analysis, cell growth<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Because metabolism is fundamental in sustaining microbial life, drugs that target pathogen-specific metabolic enzymes and pathways can be very effective. In particular, the metabolic challenges faced by intracellular pathogens, such as Mycobacterium tuberculosis, residing in the infected host provide novel opportunities for therapeutic intervention.Results: We developed a mathematical framework to simulate the effects on the growth of a pathogen when enzymes in its metabolic pathways are inhibited. Combining detailed models of enzyme kinetics, a complete metabolic network description as modeled by flux balance analysis, and a dynamic cell population growth model, we quantitatively modeled and predicted the dose-response of the 3-nitropropionate inhibitor on the growth of M. tuberculosis in a medium whose carbon source was restricted to fatty acids, and that of the 5-O-N-salicylsulfamoyl adenosine inhibitor in a medium with low-iron concentration.Conclusion: The predicted results quantitatively reproduced the experimentally measured dose-response curves, ranging over three orders of magnitude in inhibitor concentration. Thus, by allowing for detailed specifications of the underlying enzymatic kinetics, metabolic reactions/constraints, and growth media, our model captured the essential chemical and biological factors that determine the effects of drug inhibition on in vitro growth of M. tuberculosis cells.</div>','<b>Title: </b>Non-steady state mass action dynamics without rate constants: dynamics of coupled reactions using chemical potentials<br><b>Year: </b>2017<br><b>Author: </b> Wr. Cannon and  Se. Baker<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1088/1478-3975/aa7d80"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PHYSICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>kinetic models, metabolic networks, kinetic parameters, dynamic models, systems biology, network model, dynamic metabolic, biochemical network, rate constants, enzyme kinetics, kinetic data, adjustable parameters, metabolic model, parameter estimation, dynamic optimization, model selection, mathematical model, kinetic network, model response, model<div style="font-size:11px;"><br><b>Abstract: </b><em>Comprehensive and predictive simulation of coupled reaction networks has long been a goal of biology and other fields. Currently, metabolic network models that utilize enzyme mass action kinetics have predictive power but are limited in scope and application by the fact that the determination of enzyme rate constants is laborious and low throughput. We present a statistical thermodynamic formulation of the law of mass action for coupled reactions at both steady states and non-stationary states. The formulation uses chemical potentials instead of rate constants. When used to model deterministic systems, the method corresponds to a rescaling of the time dependent reactions in such a way that steady states can be reached on the same time scale but with significantly fewer computational steps. The relationships between reaction affinities, free energy changes and generalized detailed balance are central to the discussion. The significance for applications in systems biology are discussed as is the concept and assumption of maximum entropy production rate as a biological principle that links thermodynamics to natural selection.</div>','<b>Title: </b>Elucidating the digital control mechanism for DNA damage repair with the p53-Mdm2 system: single cell data analysis and ensemble modelling<br><b>Year: </b>2006<br><b>Author: </b> Ba. Ogunnaike<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1098/rsif.2005.0077"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF THE ROYAL SOCIETY INTERFACE<br><b>Cluster-Keywords: </b><em>dna damage, signalling networks, network dynamics, pmdm system, systems biology, systems dynamics, biological systems, single cell, kinase inhibitors, signaling network, biological networks, cellular automata, cell cycle, oscillation system, temporal dynamics, signaling pathway, individual cells, signaling processes, dynamic model, damage response<div style="font-size:11px;"><br><b>Abstract: </b><em>Recent experimental evidence about DNA damage response using the p53-Mdm2 system has raised some fundamental questions about the control mechanism employed. In response to DNA damage, an ensemble of cells shows a damped oscillation in p53 expression whose amplitude increases with increased DNA damage-consistent with analogue control. Recent experimental results, however, show that the single cell response is a series of discrete pulses in p53; and with increase in DNA damage, neither the height nor the duration of the pulses change, but the mean number of pulses increase-consistent with digital control. Here we present a system engineering model that uses published data to elucidate this mechanism and resolve the dilemma of how digital behaviour at the single cell level can manifest as analogue ensemble behaviour. First, we develop a dynamic model of the p53-Mdm2 system that produces non-oscillatory responses to a stress signal. Second, we develop a probability model of the distribution of pulses in a cell population, and combine the two with the simplest digital control algorithm to show how oscillatory responses whose amplitudes grow with DNA damage can arise from single cell behaviour in which each single pulse response is independent of the extent of DNA damage. A stochastic simulation of the hypothesized control mechanism reproduces experimental observations remarkably well.</div>','<b>Title: </b>Control Design for Signal Transduction Networks<br><b>Year: </b>2009<br><b>Author: </b> Cl. Lin,  Yw. Liu and  Ch. ChuangLink: No link<br><b>Journal: </b>BIOINFORMATICS AND BIOLOGY INSIGHTS<br><b>Cluster-Keywords: </b><em>gene networks, synthetic gene, synthetic biology, regulatory networks, design method, robust synthetic, nonlinear stochastic, molecular noises, gene regulatory, interaction networks, external disturbances, robust stability, robust gene, host cell, gene circuit, cellular network, design specifications, design procedure, signal transduction, parameter fluctuations<div style="font-size:11px;"><br><b>Abstract: </b><em>Signal transduction networks of biological systems are highly complex. How to mathematically describe a signal transduction network by systematic approaches to further develop an appropriate and effective control strategy is attractive to control engineers. In this paper, the synergism and saturation system S-systems representations are used to describe signal transduction networks and a control design idea is presented. For constructing mathematical models, a cascaded analysis model is first proposed. Dynamic analysis and controller design are simulated and verified.</div>','<b>Title: </b>A proposal for using the ensemble approach to understand genetic regulatory networks<br><b>Year: </b>2004<br><b>Author: </b> S. Kauffman<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.jtbi.2003.12.017"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF THEORETICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>boolean networks, regulatory networks, network model, bayesian networks, model networks, systems biology, random boolean, real cells, immune system, genetic regulatory, networks, biochemical network, fluorescence images, cellular network, gene regulatory, cell differentiation, network, biological systems, ensemble approach, cell dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>Understanding the genetic regulatory network comprising genes, RNA, proteins and the network connections and dynamical control rules among them, is a major task of contemporary systems biology. I focus here on the use of the ensemble approach to find one or more well-defined ensembles of model networks whose statistical features match those of real cells and organisms. Such ensembles should help explain and predict features of real cells and organisms. More precisely, an ensemble of model networks is defined by constraints on the "wiring diagram" of regulatory interactions, and the "rules" governing the dynamical behavior of regulated components of the network. The ensemble consists of all networks consistent with those constraints. Here I discuss ensembles of random Boolean networks, scale free Boolean networks, "medusa" Boolean networks, continuous variable networks, and others. For each ensemble, M statistical features, such as the size distribution of avalanches in gene activity changes unleashed by transiently altering the activity of a single gene, the distribution in distances between gene activities on different cell types, and others, are measured. This creates an M-dimensional space, where each ensemble corresponds to a cluster of points or distributions. Using current and future experimental techniques, Such as gene arrays, these M properties are to be measured for real cells and organisms, again yielding a cluster of points or distributions in the M-dimensional space. The procedure then finds ensembles close to those of real cells and organisms, and hill climbs to attempt to match the observed M features. Thus obtains one or more ensembles that should predict and explain many features of the regulatory networks in cells and organisms. C 2004 Elsevier Ltd. All rights reserved.</div>','<b>Title: </b>Decrease of energy spilling in Escherichia coli continuous cultures with rising specific growth rate and carbon wasting<br><b>Year: </b>2011<br><b>Author: </b> K. Valgepea,  K. Adamberg and  R. Vilu<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-5-106"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Growth substrates, aerobic/anaerobic conditions, specific growth rate mu etc. strongly influence Escherichia coli cell physiology in terms of cell size, biomass composition, gene and protein expression. To understand the regulation behind these different phenotype properties, it is useful to know carbon flux patterns in the metabolic network which are generally calculated by metabolic flux analysis MFA. However, rarely is biomass composition determined and carbon balance carefully measured in the same experiments which could possibly lead to distorted MFA results and questionable conclusions. Therefore, we carried out both detailed carbon balance and biomass composition analysis in the same experiments for more accurate quantitative analysis of metabolism and MFA.Results: We applied advanced continuous cultivation methods A-stat and D-stat to continuously monitor E. coli K 12 MG1655 flux and energy metabolism dynamic responses to change of mu and glucose-acetate co-utilisation. Surprisingly, a 36 reduction of ATP spilling was detected with increasing mu and carbon wasting to non-CO2 by-products under constant biomass yield. The apparent discrepancy between constant biomass yield and decline of ATP spilling could be explained by the rise of carbon wasting from 3 to 11 in the carbon balance which was revealed by the discovered novel excretion profile of E. coli pyrimidine pathway intermediates carbamoylphosphate, dihydroorotate and orotate. We found that carbon wasting patterns are dependent not only on mu, but also on glucose-acetate co-utilisation capability. Accumulation of these compounds was coupled to the two-phase acetate accumulation profile. Acetate overflow was observed in parallel with the reduction of TCA cycle and glycolysis fluxes, and induction of pentose phosphate pathway.Conclusions: It can be concluded that acetate metabolism is one of the major regulating factors of central carbon metabolism. More importantly, our model calculations with actual biomass composition and detailed carbon balance analysis in steady state conditions with -omics data comparison demonstrate the importance of a comprehensive systems biology approach for more advanced understanding of metabolism and carbon re-routing mechanisms potentially leading to more successful metabolic engineering.</div>','<b>Title: </b>A continuous optimization approach for inferring parameters in mathematical models of regulatory networks<br><b>Year: </b>2014<br><b>Author: </b> Zm. Deng and  Th. Tian<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1471-2105-15-256"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: The advances of systems biology have raised a large number of sophisticated mathematical models for describing the dynamic property of complex biological systems. One of the major steps in developing mathematical models is to estimate unknown parameters of the model based on experimentally measured quantities. However, experimental conditions limit the amount of data that is available for mathematical modelling. The number of unknown parameters in mathematical models may be larger than the number of observation data. The imbalance between the number of experimental data and number of unknown parameters makes reverse-engineering problems particularly challenging.Results: To address the issue of inadequate experimental data, we propose a continuous optimization approach for making reliable inference of model parameters. This approach first uses a spline interpolation to generate continuous functions of system dynamics as well as the first and second order derivatives of continuous functions. The expanded dataset is the basis to infer unknown model parameters using various continuous optimization criteria, including the error of simulation only, error of both simulation and the first derivative, or error of simulation as well as the first and second derivatives. We use three case studies to demonstrate the accuracy and reliability of the proposed new approach. Compared with the corresponding discrete criteria using experimental data at the measurement time points only, numerical results of the ERK kinase activation module show that the continuous absolute-error criteria using both function and high order derivatives generate estimates with better accuracy. This result is also supported by the second and third case studies for the G1/S transition network and the MAP kinase pathway, respectively. This suggests that the continuous absolute-error criteria lead to more accurate estimates than the corresponding discrete criteria. We also study the robustness property of these three models to examine the reliability of estimates. Simulation results show that the models with estimated parameters using continuous fitness functions have better robustness properties than those using the corresponding discrete fitness functions.Conclusions: The inference studies and robustness analysis suggest that the proposed continuous optimization criteria are effective and robust for estimating unknown parameters in mathematical models.</div>','<b>Title: </b>ConvAn: A convergence analyzing tool for optimization of biochemical networks<br><b>Year: </b>2012<br><b>Author: </b> A. Kostromins,  I. Mozga and  E. Stalidzans<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.biosystems.2011.12.004"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOSYSTEMS<br><b>Cluster-Keywords: </b><em>kinetic models, metabolic networks, kinetic parameters, dynamic models, systems biology, network model, dynamic metabolic, biochemical network, rate constants, enzyme kinetics, kinetic data, adjustable parameters, metabolic model, parameter estimation, dynamic optimization, model selection, mathematical model, kinetic network, model response, model<div style="font-size:11px;"><br><b>Abstract: </b><em>Dynamic models of biochemical networks usually are described as a system of nonlinear differential equations. In case of optimization of models for purpose of parameter estimation or design of new properties mainly numerical methods are used. That causes problems of optimization predictability as most of numerical optimization methods have stochastic properties and the convergence of the objective function to the global optimum is hardly predictable.Determination of suitable optimization method and necessary duration of optimization becomes critical in case of evaluation of high number of combinations of adjustable parameters or in case of large dynamic models. This task is complex due to variety of optimization methods, software tools and non-linearity features of models in different parameter spaces.A software tool ConvAn is developed to analyze statistical properties of convergence dynamics for optimization runs with particular optimization method, model, software tool, set of optimization method parameters and number of adjustable parameters of the model. The convergence curves can be normalized automatically to enable comparison of different methods and models in the same scale.By the help of the biochemistry adapted graphical user interface of ConvAn it is possible to compare different optimization methods in terms of ability to find the global optima or values close to that as well as the necessary computational time to reach them. It is possible to estimate the optimization performance for different number of adjustable parameters.The functionality of ConvAn enables statistical assessment of necessary optimization time depending on the necessary optimization accuracy. Optimization methods, which are not suitable for a particular optimization task, can be rejected if they have poor repeatability or convergence properties.The software ConvAn is freely available on www.biosystems.lv/convan. C 2011 Elsevier Ireland Ltd. All rights reserved.</div>','<b>Title: </b>Bayesian Parameter Inference by Markov Chain Monte Carlo with Hybrid Fitness Measures: Theory and Test in Apoptosis Signal Transduction Network<br><b>Year: </b>2013<br><b>Author: </b> Y. Murakami and  S. Takada<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0074178"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>When model parameters in systems biology are not available from experiments, they need to be inferred so that the resulting simulation reproduces the experimentally known phenomena. For the purpose, Bayesian statistics with Markov chain Monte Carlo MCMC is a useful method. Conventional MCMC needs likelihood to evaluate a posterior distribution of acceptable parameters, while the approximate Bayesian computation ABC MCMC evaluates posterior distribution with use of qualitative fitness measure. However, none of these algorithms can deal with mixture of quantitative, i.e., likelihood, and qualitative fitness measures simultaneously. Here, to deal with this mixture, we formulated Bayesian formula for hybrid fitness measures HFM. Then we implemented it to MCMC MCMC-HFM. We tested MCMC-HFM first for a kinetic toy model with a positive feedback. Inferring kinetic parameters mainly related to the positive feedback, we found that MCMC-HFM reliably infer them using both qualitative and quantitative fitness measures. Then, we applied the MCMC-HFM to an apoptosis signal transduction network previously proposed. For kinetic parameters related to implicit positive feedbacks, which are important for bistability and irreversibility of the output, the MCMC-HFM reliably inferred these kinetic parameters. In particular, some kinetic parameters that have experimental estimates were inferred without using these data and the results were consistent with experiments. Moreover, for some parameters, the mixed use of quantitative and qualitative fitness measures narrowed down the acceptable range of parameters.</div>','<b>Title: </b>Gap Gene Regulatory Dynamics Evolve along a Genotype Network<br><b>Year: </b>2016<br><b>Author: </b> A. Crombach,  Kr. Wotton,  E. Jimenez-Guri and  J. Jaeger<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1093/molbev/msw013"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MOLECULAR BIOLOGY AND EVOLUTION<br><b>Cluster-Keywords: </b><em>network models, gene network, regulatory networks, boolean networks, gene expression, systems biology, neural networks, network dynamics, gene regulatory, biological networks, network structure, gap gene, gene regulation, cell cycle, network, gene interactions, signaling network, quantum systems, system dynamics, networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Developmental gene networks implement the dynamic regulatory mechanisms that pattern and shape the organism. Over evolutionary time, the wiring of these networks changes, yet the patterning outcome is often preserved, a phenomenon known as "system drift." System drift is illustrated by the gap gene network-involved in segmental patterning-in dipteran insects. In the classic model organism Drosophila melanogaster and the nonmodel scuttle fly Megaselia abdita, early activation and placement of gap gene expression domains show significant quantitative differences, yet the final patterning output of the system is essentially identical in both species. In this detailed modeling analysis of system drift, we use gene circuits which are fit to quantitative gap gene expression data in M. abdita and compare them with an equivalent set of models from D. melanogaster. The results of this comparative analysis show precisely how compensatory regulatory mechanisms achieve equivalent final patterns in both species. We discuss the larger implications of the work in terms of "genotype networks" and the ways in which the structure of regulatory networks can influence patterns of evolutionary change evolvability.</div>','<b>Title: </b>A model for p53 dynamics triggered by DNA damage<br><b>Year: </b>2007<br><b>Author: </b> V. Chickarmane,  A. Ray,  Hm. Sauro and  A. Nadim<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1137/060653925"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SIAM JOURNAL ON APPLIED DYNAMICAL SYSTEMS<br><b>Cluster-Keywords: </b><em>dna damage, signalling networks, network dynamics, pmdm system, systems biology, systems dynamics, biological systems, single cell, kinase inhibitors, signaling network, biological networks, cellular automata, cell cycle, oscillation system, temporal dynamics, signaling pathway, individual cells, signaling processes, dynamic model, damage response<div style="font-size:11px;"><br><b>Abstract: </b><em>Several recent experiments on DNA-damage-induced signaling networks in mammalian cells have shown interesting dynamics in p53 protein expression during the repair cycle. Pulses of p53 are produced, whose frequency and amplitude are fairly independent of the amount of damage, but the probability of a cell exhibiting this pulsatile behavior increases with damage. This phenomenon has been described as a "digital oscillator." We present here a simple model oscillator comprising two species, p53 and Mdm2, which is activated by the Atm kinase. The Atm kinase exhibits bistable switch-like behavior. The network dynamics essentially consists of the core p53 oscillator, which is turned ON/OFF by the Atm switch, which is in turn activated by DNA damage. The complex dynamics are thus explained by the modular nature of the network and are fairly independent of the biological details. A stochastic model of the network dynamics reveals that the pulsatile behavior is robust to intrinsic noise of the protein components and extrinsic noise which arises due to noisy damage signals. The robustness is due to the bistable switch, which makes the system more resilient to stochastic fluctuations in its components. However, the system is more susceptible to noise in the Mdm2 protein production rate.</div>','<b>Title: </b>SIMULATING BIOCHEMICAL SIGNALING NETWORKS IN COMPLEX MOVING GEOMETRIES<br><b>Year: </b>2010<br><b>Author: </b> W. Strychalski,  D. Adalsteinsson and  Tc. Elston<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1137/090779693"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SIAM JOURNAL ON SCIENTIFIC COMPUTING<br><b>Cluster-Keywords: </b><em>biological system, network, pattern formation, system, network analysis, systems, turing patterns, patterns, reaction networks, networks, turing system, network topologies, stoichiometric network, networks pattern, network topology, biological, model, gradient formation, pattern, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Signaling networks regulate cellular responses to environmental stimuli through cascades of protein interactions. External signals can trigger cells to polarize and move in a specific direction. During migration, spatially localized activity of proteins is maintained. To investigate the effects of morphological changes on intracellular signaling, we developed a numerical scheme consisting of a cut cell finite volume spatial discretization coupled with level set methods to simulate the resulting advection-reaction-diffusion system. We then apply the method to several biochemical reaction networks in changing geometries. We found that a Turing instability can develop exclusively by cell deformations that maintain constant area. For a Turing system with a geometry-dependent single or double peak solution, simulations in a dynamically changing geometry suggest that a single peak solution is the only stable one, independent of the oscillation frequency. The method is also applied to a model of a signaling network in a migrating fibroblast.</div>','<b>Title: </b>Complex networks: From graph theory to biology<br><b>Year: </b>2006<br><b>Author: </b> A. Lesne<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s11005-006-0123-1"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>LETTERS IN MATHEMATICAL PHYSICS<br><b>Cluster-Keywords: </b><em>complex networks, complex systems, network theory, network structure, biological network, social network, network science, dynamic networks, network systems, network model, network topology, systems biology, network, network analysis, system dynamics, networks, realworld networks, network automata, epidemic network, model network<div style="font-size:11px;"><br><b>Abstract: </b><em>The aim of this text is to show the central role played by networks in complex system science. A remarkable feature of network studies is to lie at the crossroads of different disciplines, from mathematics  graph theory, combinatorics, probability theory to physics  statistical physics of networks to computer science  network generating algorithms, combinatorial optimization to biological issues  regulatory networks. New paradigms recently appeared, like that of scale-free networks providing an alternative to the random graph model introduced long ago by Erdos and Renyi. With the notion of statistical ensemble and methods originally introduced for percolation networks, statistical physics is of high relevance to get a deep account of topological and statistical properties of a network. Then their consequences on the dynamics taking place in the network should be investigated. Impact of network theory is huge in all natural sciences, especially in biology with gene networks, metabolic networks, neural networks or food webs. I illustrate this brief overview with a recent work on the influence of network topology on the dynamics of coupled excitable units, and the insights it provides about network emerging features, robustness of network behaviors, and the notion of static or dynamic motif.</div>','<b>Title: </b>Identification of all steady states in large networks by logical analysis<br><b>Year: </b>2003<br><b>Author: </b> V. Devloo,  P. Hansen and  M. Labbe<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/S0092-82400300061-2"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BULLETIN OF MATHEMATICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>regulatory networks, network dynamics, gene regulatory, network model, genetic network, gene networks, genetic regulatory, mathematical model, parameter space, network system, interaction networks, system dynamics, network structure, network, dynamic networks, network reduction, models models, biological networks, network motifs, biological systems<div style="font-size:11px;"><br><b>Abstract: </b><em>The goal of generalized logical analysis is to model complex biological systems, especially so-called regulatory systems, such as genetic networks. This theory is mainly characterized by its capacity to find all the steady states of a given system and the functional positive and negative circuits, which generate multistationarity and a cycle in the state sequence graph, respectively. So far, this has been achieved by exhaustive enumeration, which severely limits the size of the systems that can be analysed. In this paper, we introduce a mathematical function, called image function, which allows the calculation of the value of the logical parameter associated with a logical variable depending on the state of the system. Thus the state table of the system is represented analytically. We then show how all steady states can be derived as solutions to a system of steady-state equations. Constraint programming, a recent method for solving constraint satisfaction problems, is applied for that purpose. To illustrate the potential of our approach, we present results from computer experiments carried out on very large randomly-generated systems graphs with hundreds, or even thousands, of interacting components, and show that these systems can be solved using moderate computing time. Moreover, we illustrate the approach through two published applications, one of which concerns the computation times of all steady states for a large genetic network. C 2003 Society for Mathematical Biology. Published by Elsevier Ltd. All rights reserved.</div>','<b>Title: </b>Loop 7 of E2 Enzymes: An Ancestral Conserved Functional Motif Involved in the E2-Mediated Steps of the Ubiquitination Cascade<br><b>Year: </b>2012<br><b>Author: </b> E. Papaleo,  N. Casiraghi,  A. Arrigoni,  M. Vanoni,  P. Coccetti and  L. De Gioia<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0040786"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>normal mode, mode analysis, elastic network, functional motions, functional dynamics, molecular chaperone, structural biology, acidic residues, network models, molecular dynamics, atomic resolution, protein complexes, conserved functional, hsp dynamics, correlated motions, protein folding, large biomolecular, thioredoxin system, harmonic analysis, protein motions<div style="font-size:11px;"><br><b>Abstract: </b><em>The ubiquitin Ub system controls almost every aspect of eukaryotic cell biology. Protein ubiquitination depends on the sequential action of three classes of enzymes E1, E2 and E3. E2 Ub-conjugating enzymes have a central role in the ubiquitination pathway, interacting with both E1 and E3, and influencing the ultimate fate of the substrates. Several E2s are characterized by an extended acidic insertion in loop 7 L7, which if mutated is known to impair the proper E2-related functions. In the present contribution, we show that acidic loop is a conserved ancestral motif in E2s, relying on the presence of alternate hydrophobic and acidic residues. Moreover, the dynamic properties of a subset of family 3 E2s, as well as their binary and ternary complexes with Ub and the cognate E3, have been investigated. Here we provide a model of L7 role in the different steps of the ubiquitination cascade of family 3 E2s. The L7 hydrophobic residues turned out to be the main determinant for the stabilization of the E2 inactive conformations by a tight network of interactions in the catalytic cleft. Moreover, phosphorylation is known from previous studies to promote E2 competent conformations for Ub charging, inducing electrostatic repulsion and acting on the L7 acidic residues. Here we show that these active conformations are stabilized by a network of hydrophobic interactions between L7 and L4, the latter being a conserved interface for E3-recruitment in several E2s. In the successive steps, L7 conserved acidic residues also provide an interaction interface for both Ub and the Rbx1 RING subdomain of the cognate E3. Our data therefore suggest a crucial role for L7 of family 3 E2s in all the E2-mediated steps of the ubiquitination cascade. Its different functions are exploited thank to its conserved hydrophobic and acidic residues in a finely orchestrate mechanism.</div>','<b>Title: </b>Invalidation of the structure of genetic network dynamics: a geometric approach<br><b>Year: </b>2012<br><b>Author: </b> R. Porreca,  E. Cinquemani,  J. Lygeros and  G. Ferrari-Trecate<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1002/rnc.2799"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>INTERNATIONAL JOURNAL OF ROBUST AND NONLINEAR CONTROL<br><b>Cluster-Keywords: </b><em>regulatory networks, network dynamics, gene regulatory, network model, genetic network, gene networks, genetic regulatory, mathematical model, parameter space, network system, interaction networks, system dynamics, network structure, network, dynamic networks, network reduction, models models, biological networks, network motifs, biological systems<div style="font-size:11px;"><br><b>Abstract: </b><em>This work concerns the identification of the structure of a genetic network model from measurements of gene product concentrations and synthesis rates. In earlier work, we developed a data preprocessing algorithm that is able to reject many hypotheses on the network structure by testing certain monotonicity properties for a wide family of network models. Here, we develop a geometric interpretation of the method. Then, for a relevant subclass of genetic network models, we extend our approach to the combined testing of monotonicity and convexity-like properties associated with the network structures. The theoretical aspects and practical performance of the enhanced methods are illustrated by way of numerical results. Copyright c 2012 John Wiley  Sons, Ltd.</div>','<b>Title: </b>Genome-Scale Consequences of Cofactor Balancing in Engineered Pentose Utilization Pathways in Saccharomyces cerevisiae<br><b>Year: </b>2011<br><b>Author: </b> A. Ghosh,  Hm. Zhao and  Nd. Price<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0027316"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>metabolic network, metabolic model, dynamic flux, growth rate, genomescale metabolic, flux balance, balance analysis, metabolic flux, metabolic pathways, dynamic model, metabolic reactions, cellular growth, regulatory network, systems biology, optimal growth, metabolic, modeling metabolic, growth conditions, dynamic analysis, cell growth<div style="font-size:11px;"><br><b>Abstract: </b><em>Biofuels derived from lignocellulosic biomass offer promising alternative renewable energy sources for transportation fuels. Significant effort has been made to engineer Saccharomyces cerevisiae to efficiently ferment pentose sugars such as Dxylose and L-arabinose into biofuels such as ethanol through heterologous expression of the fungal D-xylose and L-arabinose pathways. However, one of the major bottlenecks in these fungal pathways is that the cofactors are not balanced, which contributes to inefficient utilization of pentose sugars. We utilized a genome-scale model of S. cerevisiae to predict the maximal achievable growth rate for cofactor balanced and imbalanced D-xylose and L-arabinose utilization pathways. Dynamic flux balance analysis DFBA was used to simulate batch fermentation of glucose, D-xylose, and L-arabinose. The dynamic models and experimental results are in good agreement for the wild type and for the engineered D-xylose utilization pathway. Cofactor balancing the engineered D-xylose and L-arabinose utilization pathways simulated an increase in ethanol batch production of 24.7 while simultaneously reducing the predicted substrate utilization time by 70. Furthermore, the effects of cofactor balancing the engineered pentose utilization pathways were evaluated throughout the genome-scale metabolic network. This work not only provides new insights to the global network effects of cofactor balancing but also provides useful guidelines for engineering a recombinant yeast strain with cofactor balanced engineered pathways that efficiently co-utilizes pentose and hexose sugars for biofuels production. Experimental switching of cofactor usage in enzymes has been demonstrated, but is a time-consuming effort. Therefore, systems biology models that can predict the likely outcome of such strain engineering efforts are highly useful for motivating which efforts are likely to be worth the significant time investment.</div>','<b>Title: </b>Transcription factor network reconstruction using the living cell array<br><b>Year: </b>2009<br><b>Author: </b> E. Yang,  Ml. Yarmush and  Ip. Androulakis<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.jtbi.2008.09.040"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF THEORETICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>gene regulatory, gene expression, regulatory networks, expression data, regulatory pathway, regulatory genes, network dynamics, microarray data, genetic networks, transcription factor, models network, target gene, upstream regulatory, network topology, pathway network, gene functional, expression profile, transcriptional network, dynamic model, transcriptional regulatory<div style="font-size:11px;"><br><b>Abstract: </b><em>The objective of identifying transcriptional regulatory networks is to provide insights as to what governs an organisms long term response to external stimuli. We explore the coupling of the living cell array LCA, a novel microfluidics device which utilizes fluorescence levels as a Surrogate for transcription factor activity with reverse Euler deconvolution RED a computational technique proposed in this work to decipher the dynamics of the interactions. It is hypothesized that these two methods will allow us to first assess the underlying network architecture associated with the transcription factor network as well as specific mechanistic consequences of transcription factor activation such as receptor dimerization or tolerance.The overall approach identifies evidence of time-lagged response which may be indicative of mechanisms such as receptor dimerization, tolerance mechanisms which are evidence of various receptor mediated dynamics, and feedback loops which regulate the response of an organism to changing environmental conditions. Furthermore, through the exploration of multiple network architectures, we were able to obtain insights as to the role each transcription factor plays in the overall response and their overall redundancy in the organisms response to external perturbations. Thus, the LCA along with the proposed analysis technique is a valuable tool for identifying the possible architectures and mechanisms underlying the transcriptional response. c 2008 Elsevier Ltd. All rights reserved.</div>','<b>Title: </b>dNSP: A biologically inspired dynamic Neural network approach to Signal Processing<br><b>Year: </b>2008<br><b>Author: </b> Jm. Cano-Izquierdo,  J. Ibarrola,  M. Pinzolas and  M. Almonacid<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.neunet.2008.03.015"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>NEURAL NETWORKS<br><b>Cluster-Keywords: </b><em>chaotic system, control systems, neural networks, jerk chaotic, torch chaotic, mode control, sliding mode, plasma torch, dynamic neural, mitochondrial dynamics, system, systems biology, adaptive integral, model system, rodtype plasma, integral sliding, systems, jerk system, cancer cell, comparison method<div style="font-size:11px;"><br><b>Abstract: </b><em>The arriving order of data is one of the intrinsic properties of a signal. Therefore, techniques dealing with this temporal relation are required for identification and signal processing tasks. To perform a classification of the signal according with its temporal characteristics, it would be useful to find a feature vector in which the temporal attributes were embedded. The correlation and power density spectrum functions are suitable tools to manage this issue. These functions are usually defined with statistical formulation. On the other hand, in biology there can be found numerous processes in which signals are processed to give a feature vector: for example, the processing of sound by the auditory system. In this work, the dNSP dynamic Neural Signal Processing architecture is proposed. This architecture allows representing a time-varying signal by a spatial thus statical vector. Inspired by the aforementioned biological processes, the dNSP performs frequency decomposition using an analogical parallel algorithm carried out by simple processing units. The architecture has been developed under the paradigm of a multilayer neural network, where the different layers are composed by units whose activation functions have been extracted from the theory of Neural Dynamic Grossberg, S. 1988. Nonlinear neural networks principles, mechanisms and architectures. Neural Networks, 1, 17-61. A theoretical study of the behavior of the dynamic equations of the units and their relationship with some statistical functions allows establishing a parallelism between the unit activations and correlation and power density spectrum functions. To test the capabilities of the proposed approach, several testbeds have been employed, i.e. the frequencial study of mathematical functions. As a possible application of the architecture, a highly interesting problem in the field of automatic control is addressed: the recognition of a controlled DC motor operating state. C 2008 Elsevier Ltd. All rights reserved.</div>','<b>Title: </b>Regulatory network reconstruction using an integral additive model with flexible kernel functions<br><b>Year: </b>2008<br><b>Author: </b> E. Novikov and  E. Barillot<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-2-8"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>regulatory networks, gene regulatory, gene expression, network inference, expression data, network structure, gene networks, networks grns, regulatory genes, boolean networks, bayesian network, network reconstruction, networks gene, network dynamics, gene regulation, inference method, systems biology, time series, infer gene, network<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Reconstruction of regulatory networks is one of the most challenging tasks of systems biology. A limited amount of experimental data and little prior knowledge make the problem difficult to solve. Although models that are currently used for inferring regulatory networks are sometimes able to make useful predictions about the structures and mechanisms of molecular interactions, there is still a strong demand to develop increasingly universal and accurate approaches for network reconstruction.Results: The additive regulation model is represented by a set of differential equations and is frequently used for network inference from time series data. Here we generalize this model by converting differential equations into integral equations with adjustable kernel functions. These kernel functions can be selected based on prior knowledge or defined through iterative improvement in data analysis. This makes the integral model very flexible and thus capable of covering a broad range of biological systems more adequately and specifically than previous models.Conclusion: We reconstructed network structures from artificial and real experimental data using differential and integral inference models. The artificial data were simulated using mathematical models implemented in JDesigner. The real data were publicly available yeast cell cycle microarray time series. The integral model outperformed the differential one for all cases. In the integral model, we tested the zero-degree polynomial and single exponential kernels. Further improvements could be expected if the kernel were selected more specifically depending on the system.</div>','<b>Title: </b>Zebrafish Pou5f1-dependent transcriptional networks in temporal control of early development<br><b>Year: </b>2010<br><b>Author: </b> D. Onichtchouk,  F. Geier,  B. Polok,  Dm. Messerschmidt,  R. Mossner,  B. Wendik,  Sm. Song,  V. Taylor,  J. Timmer and  W. Driever<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1038/msb.2010.9"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MOLECULAR SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>stem cell, stem cells, systems biology, mathematical models, cell biology, network model, signaling pathways, cell fate, computational models, cancer cells, gene expression, network dynamics, embryonic stem, signaling network, prostate cancer, mathematical modeling, psychological stress, apoptosis signaling, computational systems, molecular mechanisms<div style="font-size:11px;"><br><b>Abstract: </b><em>The transcription factor POU5f1/OCT4 controls pluripotency in mammalian ES cells, but little is known about its functions in the early embryo. We used time-resolved transcriptome analysis of zebrafish pou5f1 MZspg mutant embryos to identify genes regulated by Pou5f1. Comparison to mammalian systems defines evolutionary conserved Pou5f1 targets. Time-series data reveal many Pou5f1 targets with delayed or advanced onset of expression. We identify two Pou5f1-dependent mechanisms controlling developmental timing. First, several Pou5f1 targets are transcriptional repressors, mediating repression of differentiation genes in distinct embryonic compartments. We analyze her3 gene regulation as example for a repressor in the neural anlagen. Second, the dynamics of SoxB1 group gene expression and Pou5f1-dependent regulation of her3 and foxD3 uncovers differential requirements for SoxB1 activity to control temporal dynamics of activation, and spatial distribution of targets in the embryo. We establish a mathematical model of the early Pou5f1 and SoxB1 gene network to demonstrate regulatory characteristics important for developmental timing. The temporospatial structure of the zebrafish Pou5f1 target networks may explain aspects of the evolution of the mammalian stem cell networks. Molecular Systems Biology 6: 354; published online 9 March 2010; doi: 10.1038/msb.2010.9</div>','<b>Title: </b>The effect of coupled stochastic processes in a two-state biochemical switch<br><b>Year: </b>2011<br><b>Author: </b> Gc. Graham and  O. Lipan<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s10867-011-9226-8"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF BIOLOGICAL PHYSICS<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Cell signaling pathways consist of multiple connections of different types of gene, mRNA and protein networks. It is not a trivial task to follow the signals flowing through these networks. The difficulty comes from considering the entire biological structure as a single network without breaking it into connected modules. The study of these networks simplifies if the complex system is reduced to a hierarchy of interconnected modules. Out of many potential modules, a specific one, namely the Goldbeter-Koshland switch, was encountered by the authors during their study of the Mammalian Heat Shock Response Network MHSRN where the switch acts as a stress sensor. Usually, only the steady state behavior of the switch is studied, in which the phosphorylated protein is given as a function of the enzyme concentration. Experimental results show that the heat shock response is still present 20 h after the temperature stress had ended. Thus, it is useful to analyze the transient behavior of the switch that couples the environment to the MHSRN. A stochastic model for the switch is proposed using the Master Equation which is subsequently transformed into an equation for the factorial cumulant generating function. This generating function can be easily read from a graphical representation of the stochastic switch. The second order approximation of the equation for the factorial cumulant generating function is solved and the time dependence of the transient regime of the mean and standard deviation is readily obtained. Using the mean and standard deviation of the switchs output as a function of the stochastic input signals that represent the environment, we classify the switches according to different criteria. The switches differ by the numerical values of the parameters that characterize the switchs chemical reactions. The classifying criteria will distinguish the switches by the levels of the response for a given transition time and by the sensitivity of the response to the enzyme levels. It is also found that the environment can drastically change the response of the switch, which has important biological consequences.</div>','<b>Title: </b>Dynamical thermal dose models and dose time-profile effects<br><b>Year: </b>2019<br><b>Author: </b> Mt. Ladjimi,  D. Labavic,  M. Guilbert,  F. Anquez,  A. Pruvost,  E. Courtade,  B. Pfeuty and  Q. Thommen<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1080/02656736.2019.1633478"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>INTERNATIONAL JOURNAL OF HYPERTHERMIA<br><b>Cluster-Keywords: </b><em>inflammatory response, network model, mechanistic models, lde model, injured patients, critically injured, prediction model, tbi survivors, computational modeling, model abm, impaired astrocytes, model predicts, metabolic dynamics, significant differences, twoprocess model, agent-based model, coupled model, model, inflammatory network, computational models<div style="font-size:11px;"><br><b>Abstract: </b><em>Introduction: Models of dose-effect relationships seek systematic and predictive descriptions of how cell survival depends on the level and duration of the stressor. The CEM43 thermal dose model has been empirically derived more than thirty years ago and still serves as a benchmark for hyperthermia protocols despitethe advent of regulatory network models. Objective: In this paper, we propose and realize a simple experimental test to assess whether mechanistic models can prove more reliable indicators for some protocols. We define two time-asymmetric hyperthermia profiles, faster rise than decay or slower rise than decay, for which the CEM43 model predicts the same survival while a regulatory network model predicts significant differences. Materials: Experimental data both control 37 degrees C and hyperthermia assays were collected from duplicate HeLa cell cultures. Cells were imaged before and 24, 48 and 72 h after the hyperthermia assay double-stained with fluorescein-5-isothiocyanate FITC-labeled annexin V and propidium iodide for detecting cell death. Results: Survival experiments of HeLa cells show that a fast temperature rise followed by a slow decay can be twice more lethal than the opposite, consistently with the prediction of the network model. Conclusions: Using a model reduction approach, we obtained a simple nonlinear dynamic equation that identifies the limited repair capacity as the main factor underlying the dose-asymmetry effect and that could be useful for refining thermal doses for dynamic protocols.</div>','nan','<b>Title: </b>Interlinked nonlinear subnetworks underlie the formation of robust cellular patterns in Arabidopsis epidermis: a dynamic spatial model<br><b>Year: </b>2008<br><b>Author: </b> M. Benitez,  C. Espinosa-Soto,  P. Padilla-Longoria and  Er. Alvarez-Buylla<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-2-98"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>network models, gene network, regulatory networks, boolean networks, gene expression, systems biology, neural networks, network dynamics, gene regulatory, biological networks, network structure, gap gene, gene regulation, cell cycle, network, gene interactions, signaling network, quantum systems, system dynamics, networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Dynamical models are instrumental for exploring the way information required to generate robust developmental patterns arises from complex interactions among genetic and non-genetic factors. We address this fundamental issue of developmental biology studying the leaf and root epidermis of Arabidopsis. We propose an experimentally-grounded model of gene regulatory networks GRNs that are coupled by protein diffusion and comprise a meta-GRN implemented on cellularised domains.Results: Steady states of the meta-GRN model correspond to gene expression profiles typical of hair and non-hair epidermal cells. The simulations also render spatial patterns that match the cellular arrangements observed in root and leaf epidermis. As in actual plants, such patterns are robust in the face of diverse perturbations. We validated the model by checking that it also reproduced the patterns of reported mutants. The meta-GRN model shows that interlinked sub-networks contribute redundantly to the formation of robust hair patterns and permits to advance novel and testable predictions regarding the effect of cell shape, signalling pathways and additional gene interactions affecting spatial cell-patterning.Conclusion: The spatial meta-GRN model integrates available experimental data and contributes to further understanding of the Arabidopsis epidermal system. It also provides a systems biology framework to explore the interplay among sub-networks of a GRN, cell-to-cell communication, cell shape and domain traits, which could help understanding of general aspects of patterning processes. For instance, our model suggests that the information needed for cell fate determination emerges from dynamic processes that depend upon molecular components inside and outside differentiating cells, suggesting that the classical distinction of lineage versus positional cell differentiation may be instrumental but rather artificial. It also suggests that interlinkage of nonlinear and redundant sub-networks in larger networks is important for pattern robustness. Pursuing dynamic analyses of larger genomic coupled networks is still not possible. A repertoire of well-characterised regulatory modules, like the one presented here, will, however, help to uncover general principles of the patterning-associated networks, as well as the peculiarities that originate diversity.</div>','<b>Title: </b>The Balanced Minimum Evolution Problem<br><b>Year: </b>2012<br><b>Author: </b> D. Catanzaro,  M. Labbe,  R. Pesenti and  Jj. Salazar-Gonzalez<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1287/ijoc.1110.0455"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>INFORMS JOURNAL ON COMPUTING<br><b>Cluster-Keywords: </b><em>immune system, network models, immune response, model, biological systems, models, model reduction, dynamic model, biological networks, system, mathematical models, biochemical network, response model, systems, systems biology, network, immune, biology model, reaction networks, system systems<div style="font-size:11px;"><br><b>Abstract: </b><em>A phylogeny is an unrooted binary tree that represents the evolutionary relationships of a set of n species. Phylogenies find applications in several scientific areas ranging from medical research, to drug discovery, to epidemiology, to systematics, and to population dynamics. In such applications, the available information is usually restricted to the leaves of a phylogeny and is represented by molecular data extracted from the analyzed species, such as DNA, RNA, amino acid, or codon fragments. On the contrary, the information about the phylogeny itself is generally missing and is determined by solving an optimization problem, called the phylogeny estimation problem PEP, whose versions depend on the criterion used to select a phylogeny from among plausible alternatives. In this paper, we investigate a recent version of the PEP, called the balanced minimum evolution problem BMEP. We present a mixed-integer linear programming model to exactly solve instances of the BMEP and develop branching rules and families of valid inequalities to further strengthen the model. Our results give perspective on the mathematics of the BMEP and suggest new directions on the development of future efficient exact approaches to solutions of the problem.</div>','<b>Title: </b>Computational modelling of the BRI1 receptor system<br><b>Year: </b>2013<br><b>Author: </b> Gw. Van Esse,  K. Harter and  Sc. De Vries<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1111/pce.12077"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLANT CELL AND ENVIRONMENT<br><b>Cluster-Keywords: </b><em>signaling network, systems biology, signal transduction, signaling pathways, receptor tyrosine, erbb signaling, erbb receptor, receptor signaling, intracellular signaling, mathematical models, cell signaling, computational model, factor receptor, receptor activation, growth factor, signaling models, networks signaling, receptor system, receptors systems, downstream signaling<div style="font-size:11px;"><br><b>Abstract: </b><em>Computational models are useful tools to help understand signalling pathways in plant cells. A systems biology approach where models and experimental data are combined can provide experimentally verifiable predictions and novel insights. The brassinosteroid insensitive 1 BRI1 receptor is one of the best-understood receptor systems in Arabidopsis with clearly described ligands, mutants and associated phenotypes. Therefore, BRI1-mediated signalling is attractive for mathematical modelling approaches to understand and interpret the spatial and temporal dynamics of signal transduction cascades in planta. To establish such a model, quantitative data sets incorporating local protein concentration, binding affinity and phosphorylation state of the different pathway components are essential. Computational modelling is increasingly employed in studies of plant growth and development. In this section, we have focused on the use of quantitative imaging of fluorescently labelled proteins as an entry point in modelling studies.</div>','<b>Title: </b>Spatial and Functional Heterogeneities Shape Collective Behavior of Tumor-Immune Networks<br><b>Year: </b>2015<br><b>Author: </b> Dk. Wells,  Ys. Chuang,  Lm. Knapp,  D. Brockmann,  Wl. Kath and  Jn. Leonard<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pcbi.1004181"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS COMPUTATIONAL BIOLOGY<br><b>Cluster-Keywords: </b><em>neural network, network model, network, neurogenesis model, genomic network, networks, saccade control, droplets networks, understanding networks, gene regulatory, model, tumor-immune networks, networks dynamical, network systems, artificial neurogenesis, regulatory networks, dynamic network, information flow, cancer cells, capillary networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Tumor growth involves a dynamic interplay between cancer cells and host cells, which collectively form a tumor microenvironmental network that either suppresses or promotes tumor growth under different conditions. The transition from tumor suppression to tumor promotion is mediated by a tumor-induced shift in the local immune state, and despite the clinical challenge this shift poses, little is known about how such dysfunctional immune states are initiated. Clinical and experimental observations have indicated that differences in both the composition and spatial distribution of different cell types and/or signaling molecules within the tumor microenvironment can strongly impact tumor pathogenesis and ultimately patient prognosis. How such "functional" and "spatial" heterogeneities confer such effects, however, is not known. To investigate these phenomena at a level currently inaccessible by direct observation, we developed a computational model of a nascent metastatic tumor capturing salient features of known tumor-immune interactions that faithfully recapitulates key features of existing experimental observations. Surprisingly, over a wide range of model formulations, we observed that heterogeneity in both spatial organization and cell phenotype drove the emergence of immunosuppressive network states. We determined that this observation is general and robust to parameter choice by developing a systems-level sensitivity analysis technique, and we extended this analysis to generate other parameter-independent, experimentally testable hypotheses. Lastly, we leveraged this model as an in silico test bed to evaluate potential strategies for engineering cell-based therapies to overcome tumor associated immune dysfunction and thereby identified modes of immune modulation predicted to be most effective. Collectively, this work establishes a new integrated framework for investigating and modulating tumor-immune networks and provides insights into how such interactions may shape early stages of tumor formation.</div>','<b>Title: </b>Computational modeling of drug response with applications to neuroscience<br><b>Year: </b>2014<br><b>Author: </b> R. HerwigLink: No link<br><b>Journal: </b>DIALOGUES IN CLINICAL NEUROSCIENCE<br><b>Cluster-Keywords: </b><em>boolean network, network models, qualitative models, systems biology, mathematical modeling, boolean models, signaling networks, dynamic models, dynamic modeling, signaling dynamics, logic models, dynamic network, mathematical models, boolean modeling, signal transduction, regulatory networks, drug resistance, biological networks, network approach, network modeling<div style="font-size:11px;"><br><b>Abstract: </b><em>The development of novel high-throughput technologies has opened up the opportunity to deeply characterize patient tissues at various molecular levels and has given rise to a paradigm shift in medicine towards personalized therapies. Computational analysis plays a pivotal role in integrating the various genome data and understanding the cellular response to a drug. Based on that data, molecular models can be constructed that incorporate the known downstream effects of drug-targeted receptor molecules and that predict optimal therapy decisions. In this article, we describe the different steps in the conceptual framework of computational modeling. We review resources that hold information on molecular pathways that build the basis for constructing the model interaction maps, highlight network analysis concepts that have been helpful in identifying predictive disease patterns, and introduce the basic concepts of kinetic modeling. Finally, we illustrate this framework with selected studies related to the modeling of important target pathways affected by drugs. C 2014, AICH - Servier Research Group</div>','<b>Title: </b>Global Parameter Identification of Stochastic Reaction Networks from Single Trajectories<br><b>Year: </b>2012<br><b>Author: </b> Cl. Muller,  R. Ramaswamy and  If. Sbalzarini<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/978-1-4419-7210-128"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ADVANCES IN SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>reaction networks, network model, systems biology, biochemical reaction, stochastic reaction, model reduction, stochastic model, network dynamics, stochastic biochemical, stochastic simulation, system size, chemical reaction, gene network, elementary reactions, biochemical networks, parameter identification, parameter inference, cell populations, small system, linear noise<div style="font-size:11px;"><br><b>Abstract: </b><em>We consider the problem of inferring the unknown parameters of a stochastic biochemical network model from a single measured time-course of the concentration of some of the involved species. Such measurements are available, e.g., from live-cell fluorescence microscopy in image-based systems biology. In addition, fluctuation time-courses from, e.g., fluorescence correlation spectroscopy FCS provide additional information about the system dynamics that can be used to more robustly infer parameters than when considering only mean concentrations. Estimating model parameters from a single experimental trajectory enables single-cell measurements and quantification of cell cell variability. We propose a novel combination of an adaptive Monte Carlo sampler, called Gaussian Adaptation GaA, and efficient exact stochastic simulation algorithms SSA that allows parameter identification from single stochastic trajectories. We benchmark the proposed method on a linear and a non-linear reaction network at steady state and during transient phases. In addition, we demonstrate that the present method also provides an ellipsoidal volume estimate of the viable part of parameter space and is able to estimate the physical volume of the compartment in which the observed reactions take place.</div>','<b>Title: </b>Dynamic metabolic profiling and metabolite network and pathways modeling<br><b>Year: </b>2013<br><b>Author: </b> M. Cuperlovic-CulfLink: No link<br><b>Journal: </b>NMR METABOLOMICS IN CANCER RESEARCH<br><b>Cluster-Keywords: </b><em>metabolic network, metabolic model, dynamic flux, growth rate, genomescale metabolic, flux balance, balance analysis, metabolic flux, metabolic pathways, dynamic model, metabolic reactions, cellular growth, regulatory network, systems biology, optimal growth, metabolic, modeling metabolic, growth conditions, dynamic analysis, cell growth<div style="font-size:11px;"><br><b>Abstract: </b><em>Metabolic reactions are dynamic, time-dependent processes that are controlled and influenced by different internal and external factors. Cellular metabolism and behavior cannot be fully described by independently considering only the constitutive elements of a cell at one point in time. A dynamic analysis of networks can provide a model for different aspects of cells behavior in the steady state, or following various stimulations. The creation of these models with the use of appropriate theory and experimental data is the focus of systems biology. Metabolomics provides crucial information in this process, with NMR metabolomics allowing the measurement of concentrations of metabolites both quickly and reliably, whether ex vivo, in vitro, and/or in vivo. Computational simulations of metabolic processes fed with information obtained from metabolomics can be used to reproduce what is physiologically observed, and to understand biological principles. This chapter shows different approaches used for metabolic pathways and networks modeling and ways to use metabolomics for model development.</div>','<b>Title: </b>Hierarchical Differentiation of Myeloid Progenitors Is Encoded in the Transcription Factor Network<br><b>Year: </b>2011<br><b>Author: </b> J. Krumsiek,  C. Marr,  T. Schroeder and  Fj. Theis<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0022649"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>network models, gene network, regulatory networks, boolean networks, gene expression, systems biology, neural networks, network dynamics, gene regulatory, biological networks, network structure, gap gene, gene regulation, cell cycle, network, gene interactions, signaling network, quantum systems, system dynamics, networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Hematopoiesis is an ideal model system for stem cell biology with advanced experimental access. A systems view on the interactions of core transcription factors is important for understanding differentiation mechanisms and dynamics. In this manuscript, we construct a Boolean network to model myeloid differentiation, specifically from common myeloid progenitors to megakaryocytes, erythrocytes, granulocytes and monocytes. By interpreting the hematopoietic literature and translating experimental evidence into Boolean rules, we implement binary dynamics on the resulting 11-factor regulatory network. Our network contains interesting functional modules and a concatenation of mutual antagonistic pairs. The state space of our model is a hierarchical, acyclic graph, typifying the principles of myeloid differentiation. We observe excellent agreement between the steady states of our model and microarray expression profiles of two different studies. Moreover, perturbations of the network topology correctly reproduce reported knockout phenotypes in silico. We predict previously uncharacterized regulatory interactions and alterations of the differentiation process, and line out reprogramming strategies.</div>','<b>Title: </b>Metabolic flux understanding of Pichia pastoris grown on heterogenous culture media<br><b>Year: </b>2014<br><b>Author: </b> Jm. Gonzalez-Martinez,  A. Folch-Fortuny,  F. Llaneras,  M. Tortajada,  J. Pico and  A. Ferrer<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.chemolab.2014.02.003"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>CHEMOMETRICS AND INTELLIGENT LABORATORY SYSTEMS<br><b>Cluster-Keywords: </b><em>metabolic network, metabolic model, dynamic flux, growth rate, genomescale metabolic, flux balance, balance analysis, metabolic flux, metabolic pathways, dynamic model, metabolic reactions, cellular growth, regulatory network, systems biology, optimal growth, metabolic, modeling metabolic, growth conditions, dynamic analysis, cell growth<div style="font-size:11px;"><br><b>Abstract: </b><em>Within the emergent field of Systems Biology, mathematical models obtained from physical-chemical laws the so-called first principles-based models of microbial systems are employed to discern the principles that govern cellular behaviour and achieve a predictive understanding of cellular functions. The reliance on this biochemical knowledge has the drawback that some of the assumptions specific kinetics of the reaction system, unknown dynamics and values of the model parameters may not be valid for all the metabolic possible states of the network. In this uncertainty context, the combined use of fundamental knowledge and data measured in the fermentation that describe the behaviour of the microorganism in the manufacturing process is paramount to overcome this problem. In this paper, a grey modelling approach is presented combining data-driven and first principles information at different scales, developed for Pichia pastoris cultures grown on different carbon sources. This approach will allow us to relate patterns of recombinant protein production to intracellular metabolic states and correlate intra and extracellular reactions in order to understand how the internal state of the cells determines the observed behaviour in P. pastoris cultivations. C 2014 Elsevier B.V. All rights reserved.</div>','<b>Title: </b>List-decoding methods for inferring polynomials in finite dynamical gene network models<br><b>Year: </b>2009<br><b>Author: </b> J. Dingel and  O. Milenkovic<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1093/bioinformatics/btp281"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Motivation: The problem of reverse engineering the dynamics of gene expression profiles is of focal importance in systems biology. Due to noise and the inherent lack of sufficiently large datasets generated via high-throughput measurements, known reconstruction frameworks based on dynamical systems models fail to provide adequate settings for network analysis. This motivates the study of new approaches that produce stochastic lists of explanations for the observed network dynamics that can be efficiently inferred from small sample sets and in the presence of errors.Results: We introduce a novel algebraic modeling framework, termed stochastic polynomial dynamical systems SPDSs that can capture the dynamics of regulatory networks based on microarray expression data. Here, we refer to dynamics of the network as the trajectories of gene expression profiles over time. The model assumes that the expression data is quantized in a manner that allows for imposing a finite field structure on the observations, and the existence of polynomial update functions for each gene in the network. The underlying reverse engineering algorithm is based on ideas borrowed from coding theory, and in particular, list-decoding methods for so called Reed-Muller codes. The list-decoding method was tested on synthetic data and on microarray expression measurements from the M-3D database, corresponding to a subnetwork of the Escherichia coli SOS repair system, as well as on the complete transcription factor network, available at RegulonDB. The results show that SPDSs constructed via list-decoders significantly outperform other algebraic reverse engineering methods, and that they also provide good guidelines for estimating the influence of genes on the dynamics of the network.</div>','<b>Title: </b>Key Features of Turing Systems are Determined Purely by Network Topology<br><b>Year: </b>2018<br><b>Author: </b> X. Diego,  L. Marcon,  P. Muller and  J. Sharpe<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1103/PhysRevX.8.021071"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PHYSICAL REVIEW X<br><b>Cluster-Keywords: </b><em>biological system, network, pattern formation, system, network analysis, systems, turing patterns, patterns, reaction networks, networks, turing system, network topologies, stoichiometric network, networks pattern, network topology, biological, model, gradient formation, pattern, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Turings theory of pattern formation is a universal model for self-organization, applicable to many systems in physics, chemistry, and biology. Essential properties of a Turing system, such as the conditions for the existence of patterns and the mechanisms of pattern selection, are well understood in small networks. However, a general set of rules explaining how network topology determines fundamental system properties and constraints has not been found. Here we provide a first general theory of Turing network topology, which proves why three key features of a Turing system are directly determined by the topology: the type of restrictions that apply to the diffusion rates, the robustness of the system, and the phase relations of the molecular species.</div>','<b>Title: </b>Flat clathrin lattices: stable features of the plasma membrane<br><b>Year: </b>2014<br><b>Author: </b> J. Grove,  Dj. Metcalf,  Ae. Knight,  St. Wavre-Shapton,  T. Sun,  Ed. Protonotarios,  Ld. Griffin,  J. Lippincott-Schwartz and  M. Marsh<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1091/mbc.E14-06-1154"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MOLECULAR BIOLOGY OF THE CELL<br><b>Cluster-Keywords: </b><em>chaotic system, control systems, neural networks, jerk chaotic, torch chaotic, mode control, sliding mode, plasma torch, dynamic neural, mitochondrial dynamics, system, systems biology, adaptive integral, model system, rodtype plasma, integral sliding, systems, jerk system, cancer cell, comparison method<div style="font-size:11px;"><br><b>Abstract: </b><em>Clathrin-mediated endocytosis CME is a fundamental property of eukaryotic cells. Classical CME proceeds via the formation of clathrin-coated pits CCPs at the plasma membrane, which invaginate to form clathrin-coated vesicles, a process that is well understood. However, clathrin also assembles into flat clathrin lattices FCLs; these structures remain poorly described, and their contribution to cell biology is unclear. We used quantitative imaging to provide the first comprehensive description of FCLs and explore their influence on plasma membrane organization. Ultrastructural analysis by electron and superresolution microscopy revealed two discrete populations of clathrin structures. CCPs were typified by their sphericity, small size, and homogeneity. FCLs were planar, large, and heterogeneous and present on both the dorsal and ventral surfaces of cells. Live microscopy demonstrated that CCPs are short lived and culminate in a peak of dynamin recruitment, consistent with classical CME. In contrast, FCLs were long lived, with sustained association with dynamin. We investigated the biological relevance of FCLs using the chemokine receptor CCR5 as a model system. Agonist activation leads to sustained recruitment of CCR5 to FCLs. Quantitative molecular imaging indicated that FCLs partitioned receptors at the cell surface. Our observations suggest that FCLs provide stable platforms for the recruitment of endocytic cargo.</div>','<b>Title: </b>Dissection of DEN-Induced Platelet Proteome Changes Reveals the Progressively Dys-Regulated Pathways Indicative of Hepatocarcinogenesis<br><b>Year: </b>2010<br><b>Author: </b> Th. Leng,  N. Liu,  Y. Dai,  Yb. Yu,  C. Zhang,  Ry. Du and  Xa. Chen<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1021/pr100679t"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF PROTEOME RESEARCH<br><b>Cluster-Keywords: </b><em>gene regulatory, systems biology, regulatory networks, gene network, stem cell, network model, transcription factors, gene expression, cell transcriptomes, network analysis, data analysis, stem cells, network grnsight, cell transformation, weighted network, gene ontology, networks grns, graph model, graph layout, endothelial cell<div style="font-size:11px;"><br><b>Abstract: </b><em>Due to the lack of precise markers indicative of its occurrence, progression, and malignant stages, hepatocellular carcinoma HCC is currently associated with high mortality Given the fact that thrombocytopenia is associated with chronic liver diseases, and the multifunctional nature of platelets we reason that phenotype-specific platelets could be the systemic barometer for hepato-carcinogenesis The mass spectrometry MS-based proteomic efforts to discover novel biomarkers in plasma or serum are largely compromised by a few of the overwhelmingly abundant proteins that comprise over 95 of the total protein mass of plasma or sera Platelets however are free of these MS signal-suppressing proteins On the basis of a HCC animal model where diethyl nitrosamine DEN administration on male rats specifically induces HCC, by using a multiplex quantitative proteomic approach, we profiled the phase-to-phase proteome changes in a series of viable phenotype-specific platelets along with the DEN-induced progressive liver transformation The platelet proteome was found highly responsive to each physiological stage of liver inflammation or pathogenesis Using data-dependent bioinformatics network analysis, we found that certain pathway modules involved in immune response, tissue wound repair, apoptosis, cell proliferation, and catabolism and metabolism were differentially regulated, which were uncovered by the DEN-induced differential expression of the corresponding pathway components The phase-specific presentations of these pathways suggested that the DEN-induced progression of immune suppression and apoptosis resistance is dynamically coordinated in the platelets These novel platelet signatures are interconnected in the dynamic networks along with HCC progression and could be identified noninvasively for HCC prognosis and early diagnosis</div>','<b>Title: </b>Forecasting craniofacial growth in individuals with class III malocclusion by computational modelling<br><b>Year: </b>2014<br><b>Author: </b> P. Auconi,  M. Scazzocchio,  E. Defraia,  Ja. Mcnamara and  L. Franchi<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1093/ejo/cjt036"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>EUROPEAN JOURNAL OF ORTHODONTICS<br><b>Cluster-Keywords: </b><em>stem cell, stem cells, systems biology, mathematical models, cell biology, network model, signaling pathways, cell fate, computational models, cancer cells, gene expression, network dynamics, embryonic stem, signaling network, prostate cancer, mathematical modeling, psychological stress, apoptosis signaling, computational systems, molecular mechanisms<div style="font-size:11px;"><br><b>Abstract: </b><em>To develop a mathematical model that adequately represented the pattern of craniofacial growth in class III subject consistently, with the goal of using this information to make growth predictions that could be amenable to longitudinal verification and clinical use.Materials and methods: A combination of computational techniques i.e. Fuzzy clustering and Network analysis was applied to cephalometric data derived from 429 untreated growing female patients with class III malocclusion to visualize craniofacial growth dynamics and correlations. Four age groups of subjects were examined individually: from 7 to 9 years of age, from 10 to 12 years, from 13 to 14 years, and from 15 to 17 years.Results : The connections between pathway components of class III craniofacial growth can be visualized from Network profiles. Fuzzy clustering analysis was able to define further growth patterns and coherences of the traditionally reported dentoskeletal characteristics of this structural imbalance. Craniofacial growth can be visualized as a biological, space-constraint-based optimization process; the prediction of individual growth trajectories depends on the rate of membership to a specific winner cluster, i.e. on a specific individual growth strategy. The reliability of the information thus gained was tested to forecast craniofacial growth of 28 untreated female class III subjects followed longitudinally. Conclusion: The combination of Fuzzy clustering and Network algorithms allowed the development of principles for combining multiple auxological cephalometric features into a joint global model and to predict the individual risk of the facial pattern imbalance during growth.</div>','<b>Title: </b>Photomorphogenesis for robot self-assembly: adaptivity, collective decision-making, and self-repair<br><b>Year: </b>2019<br><b>Author: </b> Md. Soorati,  Mk. Heinrich,  J. Ghofrani,  P. Zahadat and  H. Hamann<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1088/1748-3190/ab2958"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOINSPIRATION  BIOMIMETICS<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Self-assembly in biology is an inspiration for engineered large-scale multi-modular systems with desirable characteristics, such as robustness, scalability, and adaptivity. Previous works have shown that simple mobile robots can be used to emulate and study self-assembly behaviors. However, many of these studies were restricted to rather static and inflexible aggregations in predefined shapes, and were limited in adaptivity compared to that observed in nature. We propose a photomorphogenesis approach for robots using our vascular morphogenesis model-a light-stimuli directed method for multi-robot self-assembly inspired by the tissue growth of trees. Robots in the role of leaves collect a virtual resource that is proportional to a real, sensed environmental feature. This is then used to build a virtual underlying network that shares a common resource throughout the whole robot aggregate and determines where it grows or shrinks as a reaction to the dynamic environment. In our approach the robots use supplemental bioinspired models to collectively select a leading robot to decide who starts to self-assemble and where, or to assemble static aggregations. The robots then use our vascular morphogenesis model to aggregate in a directed way preferring bright areas, hence resembling natural phototropism growth towards light. Our main result is that the assembled robots are adaptive and able to react to dynamic environments by collectively and autonomously rearranging the aggregate, discarding outdated parts, and growing new ones. In representative experiments, the self-assembling robots collectively make rational decisions on where to grow. Cutting off parts of the aggregate triggers a self-organizing repair process in the robots, and the parts regrow. All these capabilities of adaptivity, collective decision-making, and self-repair in our robot self-assembly originate directly from self-organized behavior of the vascular morphogenesis model. Our approach opens up opportunities for self-assembly with reconfiguration on short time-scales</div>','<b>Title: </b>Dissecting mechanisms of mouse embryonic stem cells heterogeneity through a model-based analysis of transcription factor dynamics<br><b>Year: </b>2016<br><b>Author: </b> M. Herberg,  I. Glauche,  T. Zerjatke,  M. Winzi,  F. Buchholz and  I. Roeder<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1098/rsif.2016.0167"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF THE ROYAL SOCIETY INTERFACE<br><b>Cluster-Keywords: </b><em>stem cell, stem cells, systems biology, mathematical models, cell biology, network model, signaling pathways, cell fate, computational models, cancer cells, gene expression, network dynamics, embryonic stem, signaling network, prostate cancer, mathematical modeling, psychological stress, apoptosis signaling, computational systems, molecular mechanisms<div style="font-size:11px;"><br><b>Abstract: </b><em>Pluripotent mouse embryonic stem cells mESCs show heterogeneous expression levels of transcription factors TFs involved in pluripotency regulation, among them Nanog and Rex1. The expression of both TFs can change dynamically between states of high and low activity, correlating with the cells capacity for self-renewal. Stochastic fluctuations as well as sustained oscillations in gene expression are possible mechanisms to explain this behaviour, but the lack of suitable data hampered their clear distinction. Here, we present a systems biology approach in which novel experimental data on TF heterogeneity is complemented by an agent-based model of mESC self-renewal. Because the model accounts for intracellular interactions, cell divisions and heredity structures, it allows for evaluating the consistency of the proposed mechanisms with data on population growth and on TF dynamics after cell sorting. Our model-based analysis revealed that a bistable, noise-driven network model fulfils the minimal requirements to consistently explain Nanog and Rex1 expression dynamics in heterogeneous and sorted mESC populations. Moreover, we studied the impact of TF-related proliferation capacities on the frequency of state transitions and demonstrate that cellular genealogies can provide insights into the heredity structures of mESCs.</div>','<b>Title: </b>Systems Biology Approaches to the Study of Apoptosis<br><b>Year: </b>2009<br><b>Author: </b> H. Huber,  E. Bullinger and  M. Rehm<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/978-1-60327-381-712"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ESSENTIALS OF APOPTOSIS, SECOND EDITION: A GUIDE FOR BASIC AND CLINICAL RESEARCH<br><b>Cluster-Keywords: </b><em>stem cell, stem cells, systems biology, mathematical models, cell biology, network model, signaling pathways, cell fate, computational models, cancer cells, gene expression, network dynamics, embryonic stem, signaling network, prostate cancer, mathematical modeling, psychological stress, apoptosis signaling, computational systems, molecular mechanisms<div style="font-size:11px;"><br><b>Abstract: </b><em>Today, we can avail of comprehensive information on the molecular mechanisms of apoptosis signaling that was gathered during decades of intense research. This chapter presents how mathematical approaches in the field of cellular signaling are used to integrate this complex and heterogeneous information into computational models with the aim to elucidate the functional proper-ties of apoptotic signaling networks. Mathematical modeling allows one to describe properties of signaling systems that emanate from the interplay of the systems individual components and has a longstanding and successful history in the fields of physics, chemistry, and their applied engineering sciences. Systems analyses can serve to describe and identify signaling dynamics, molecular switches, thresholds, and feedback regulatory mechanisms and allow systems properties such as stability and robustness toward external perturbations to be identified. Crucially, systems analyses can also serve to generate novel qualitative and quantitative research hypotheses, which in turn allow for more focused experimental research approaches. This chapter provides a concise and critical overview on the current state of systems biology in the field of apoptotic signaling and the methodology employed.</div>','<b>Title: </b>Networks and games for precision medicine<br><b>Year: </b>2016<br><b>Author: </b> C. Biane,  F. Delaplace and  H. Klaudel<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.biosystems.2016.08.006"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOSYSTEMS<br><b>Cluster-Keywords: </b><em>systems biology, regulatory network, interaction networks, network model, gene regulatory, network, network biology, networks, gene network, response networks, network dynamics, dynamic model, cell networks, gene expression, biological networks, system dynamics, dynamic network, model, biological systems, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Recent advances in omics technologies provide the leverage for the emergence of precision medicine that aims at personalizing therapy to patient. In this undertaking, computational methods play a central role for assisting physicians in their clinical decision-making by combining data analysis and systems biology modelling. Complex diseases such as cancer or diabetes arise from the intricate interplay of various biological molecules. Therefore, assessing drug efficiency requires to study the effects of elementary perturbations caused by diseases on relevant biological networks. In this paper, we propose a computational framework called Network-Action Game applied to best drug selection problem combining Game Theory and discrete models of dynamics Boolean networks. Decision-making is modelled using Game Theory that defines the process of drug selection among alternative possibilities, while Boolean networks are used to model the effects of the interplay between disease and drugs actions on the patients molecular system. The actions/strategies of disease and drugs are focused on arc alterations of the interactome. The efficiency of this framework has been evaluated for drug prediction on a model of breast cancer signalling. C 2016 Elsevier Ireland Ltd. All rights reserved.</div>','<b>Title: </b>Network Theory in Prebiotic Evolution<br><b>Year: </b>2018<br><b>Author: </b> Si. Walker and  C. Mathis<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/978-3-319-93584-310"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PREBIOTIC CHEMISTRY AND CHEMICAL EVOLUTION OF NUCLEIC ACIDS<br><b>Cluster-Keywords: </b><em>complex networks, complex systems, network theory, network structure, biological network, social network, network science, dynamic networks, network systems, network model, network topology, systems biology, network, network analysis, system dynamics, networks, realworld networks, network automata, epidemic network, model network<div style="font-size:11px;"><br><b>Abstract: </b><em>One of the most challenging aspect of origins of life research is that we do not know precisely what life is. In recent years, the use of network theory has revolutionized our understanding of living systems by permitting a mathematical framework for understanding life as an emergent, collective property of many interacting entities. So far, complex systems science has seen little direct application to the origins of life, particularly in laboratory science. Yet, networks are important mathematical descriptors in cases where the structure of interactions matters more than counting individual component parts-precisely what we envision happens as matter transitions to life. Here, we review a few notable examples of the use of network theory in prebiotic evolution, and discuss the promise of systems approaches to origins of life. The end goal is to develop a statistical mechanics useful to origins of life-that is, one that deals with interactions of system components rather than merely counting them and is therefore equipped to model life as an emergent phenomena.</div>','<b>Title: </b>The development of a fully-integrated immune response model FIRM simulator of the immune response through integration of multiple subset models<br><b>Year: </b>2013<br><b>Author: </b> S. Palsson,  Tp. Hickling,  El. Bradshaw-Pierce,  M. Zager,  K. Jooss,  Pj. OBrien,  Me. Spilker,  Bo. Palsson and  P. Vicini<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-7-95"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>immune system, network models, immune response, model, biological systems, models, model reduction, dynamic model, biological networks, system, mathematical models, biochemical network, response model, systems, systems biology, network, immune, biology model, reaction networks, system systems<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: The complexity and multiscale nature of the mammalian immune response provides an excellent test bed for the potential of mathematical modeling and simulation to facilitate mechanistic understanding. Historically, mathematical models of the immune response focused on subsets of the immune system and/or specific aspects of the response. Mathematical models have been developed for the humoral side of the immune response, or for the cellular side, or for cytokine kinetics, but rarely have they been proposed to encompass the overall system complexity. We propose here a framework for integration of subset models, based on a system biology approach.Results: A dynamic simulator, the Fully-integrated Immune Response Model FIRM, was built in a stepwise fashion by integrating published subset models and adding novel features. The approach used to build the model includes the formulation of the network of interacting species and the subsequent introduction of rate laws to describe each biological process. The resulting model represents a multi-organ structure, comprised of the target organ where the immune response takes place, circulating blood, lymphoid T, and lymphoid B tissue. The cell types accounted for include macrophages, a few T-cell lineages cytotoxic, regulatory, helper 1, and helper 2, and B-cell activation to plasma cells. Four different cytokines were accounted for: IFN-gamma, IL-4, IL-10 and IL-12. In addition, generic inflammatory signals are used to represent the kinetics of IL-1, IL-2, and TGF-beta. Cell recruitment, differentiation, replication, apoptosis and migration are described as appropriate for the different cell types. The model is a hybrid structure containing information from several mammalian species. The structure of the network was built to be physiologically and biochemically consistent. Rate laws for all the cellular fate processes, growth factor production rates and half-lives, together with antibody production rates and half-lives, are provided. The results demonstrate how this framework can be used to integrate mathematical models of the immune response from several published sources and describe qualitative predictions of global immune system response arising from the integrated, hybrid model. In addition, we show how the model can be expanded to include novel biological findings. Case studies were carried out to simulate TB infection, tumor rejection, response to a blood borne pathogen and the consequences of accounting for regulatory T-cells.Conclusions: The final result of this work is a postulated and increasingly comprehensive representation of the mammalian immune system, based on physiological knowledge and susceptible to further experimental testing and validation. We believe that the integrated nature of FIRM has the potential to simulate a range of responses under a variety of conditions, from modeling of immune responses after tuberculosis TB infection to tumor formation in tissues. FIRM also has the flexibility to be expanded to include both complex and novel immunological response features as our knowledge of the immune system advances.</div>','<b>Title: </b>A stochastic model of autocatalytic reaction networks<br><b>Year: </b>2012<br><b>Author: </b> A. Filisetti,  A. Graudenzi,  R. Serra,  M. Villani,  Rm. Fuchslin,  N. Packard,  Sa. Kauffman and  I. Poli<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s12064-011-0136-x"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>THEORY IN BIOSCIENCES<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Autocatalytic cycles are rather widespread in nature and in several theoretical models of catalytic reaction networks their emergence is hypothesized to be inevitable when the network is or becomes sufficiently complex. Nevertheless, the emergence of autocatalytic cycles has been never observed in wet laboratory experiments. Here, we present a novel model of catalytic reaction networks with the explicit goal of filling the gap between theoretical predictions and experimental findings. The model is based on previous study of Kauffman, with new features in the introduction of a stochastic algorithm to describe the dynamics and in the possibility to increase the number of elements and reactions according to the dynamical evolution of the system. Furthermore, the introduction of a temporal threshold allows the detection of cycles even in our context of a stochastic model with asynchronous update. In this study, we describe the model and present results concerning the effect on the overall dynamics of varying a the average residence time of the elements in the reactor, b both the composition of the firing disk and the concentration of the molecules belonging to it, c the composition of the incoming flux.</div>','<b>Title: </b>Towards more biological mutation operators in gene regulation studies<br><b>Year: </b>2004<br><b>Author: </b> J. Watson,  N. Geard and  J. Wiles<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.biosystems.2004.05.016"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOSYSTEMS<br><b>Cluster-Keywords: </b><em>network models, gene network, regulatory networks, boolean networks, gene expression, systems biology, neural networks, network dynamics, gene regulatory, biological networks, network structure, gap gene, gene regulation, cell cycle, network, gene interactions, signaling network, quantum systems, system dynamics, networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Genetic regulation is often viewed as a complex system whose properties emerge from the interaction of regulatory genes. One major paradigm for studying the complex dynamics of gene regulation uses directed graphs to explore structure, behaviour and evolvability. Mutation operators used in such studies typically involve the insertion and deletion of nodes, and the insertion, deletion and rewiring of links at the network level. These network-level mutational operators are sufficient to allow the statistical analysis of network structure, but impose limitations on the way networks are evolved. There are a wide variety of mutations in DNA sequences that have yet to be analysed for their network-level effects. By modelling an artificial genome at the level of nucleotide sequences and mapping it to a regulatory network, biologically grounded mutation operators can be mapped to network-level mutations. This paper analyses five such sequence level mutations single-point mutation, transposition, inversion, deletion and gene duplication for their effects at the network level. Using analytic and simulation techniques, we show that it is rarely the case that nodes and links are cleanly added or deleted, with even the simplest point mutation causing a wide variety of network-level modifications. As expected, the vast majority of simple single-point mutations are neutral, resulting in a neutral plateau from which a range of functional behaviours can be reached. By analysing the effects of sequence-level mutations at the network level of gene regulation, we aim to stimulate more careful consideration of mutation operators in gene regulation models than has previously been given. C 2003 Elsevier Ireland Ltd. All rights reserved.</div>','<b>Title: </b>Metabolic network modelling: Including stochastic effects<br><b>Year: </b>2005<br><b>Author: </b> P. Ao<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.compchemeng.2005.05.007"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>COMPUTERS  CHEMICAL ENGINEERING<br><b>Cluster-Keywords: </b><em>gene expression, gene network, stochastic model, gene regulatory, systems biology, expression systems, gene regulation, network activity, expression dynamics, noise control, regulatory network, deterministic models, network components, stochastic modeling, regulation system, escherichia coli, regulatory system, synthetic gene, regulatory genes, gene<div style="font-size:11px;"><br><b>Abstract: </b><em>We propose to model the dynamics of metabolic networks from a systems biology point of view by four dynamical structure elements: potential function, translocation matrix, degradation matrix, and stochastic force. These four elements are balanced to determine the network dynamics, which gives arise to a special stochastic differential equation supplemented by a relationship between the stochastic force and the degradation matrix. Important network behaviors can be obtained from the potential function without explicitly solving for the time-dependent solution. The existence of such a potential function suggests a global optimization principle. The existence stochastic force corresponds naturally to the hierarchical structure in metabolic networks. We provide. theoretical evidences to justify our proposal by discussing its connections to others large-scale biochemical systems approaches, such as the network thermodynamics theory, biochemical systems theory, metabolic control analysis, and flux balance analysis. Experimental data displaying stochasticity which carries important biological information are also pointed out. c 2005 Elsevier Ltd. All rights reserved.</div>','<b>Title: </b>Dynamical biocontrol systems: insights through mechanistic modelling<br><b>Year: </b>2009<br><b>Author: </b> Rm. Javier and  Jj. Distefano<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1080/13873950802379340"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MATHEMATICAL AND COMPUTER MODELLING OF DYNAMICAL SYSTEMS<br><b>Cluster-Keywords: </b><em>dna damage, signalling networks, network dynamics, pmdm system, systems biology, systems dynamics, biological systems, single cell, kinase inhibitors, signaling network, biological networks, cellular automata, cell cycle, oscillation system, temporal dynamics, signaling pathway, individual cells, signaling processes, dynamic model, damage response<div style="font-size:11px;"><br><b>Abstract: </b><em>Dynamical biological systems can be modelled mathematically at many levels, from relatively macroscopic, i.e. aggregating many processes or components into subsystem blocks and associated equations characterizing overall behaviour, to microscopic, i.e. focusing on physical mechanisms at the level of individual molecules and associated unit-process equations reflecting such couplings. Here we examine some representative examples within this spectrum of approaches, illustrating the ways in which various investigators handle data-driven modelling problems associated with drug effects - from mostly blood-borne data; switch-like responses in cell signalling subsystems - from molecular biochemistry data; gene network regulation - from pharmacogenomics data; and tumour-suppressor protein regulation mechanisms - also from molecular biochemistry data. Special attention is given to the latter, as modelling of the potential mechanisms involved in this system is an ongoing investigation in our lab. The biosystem model representations described here demonstrate to varying degrees the power of more microscopic methods for discriminating between alternative mechanistic hypotheses of biocontrol system connectivity and dynamics.</div>','<b>Title: </b>Bounded noise induced first-order phase transitions in a baseline non-spatial model of gene transcription<br><b>Year: </b>2018<br><b>Author: </b> A. DOnofrio,  G. Caravagna and  S. De Franciscis<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.physa.2017.11.123"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PHYSICA A-STATISTICAL MECHANICS AND ITS APPLICATIONS<br><b>Cluster-Keywords: </b><em>gene expression, gene network, stochastic model, gene regulatory, systems biology, expression systems, gene regulation, network activity, expression dynamics, noise control, regulatory network, deterministic models, network components, stochastic modeling, regulation system, escherichia coli, regulatory system, synthetic gene, regulatory genes, gene<div style="font-size:11px;"><br><b>Abstract: </b><em>In this work we consider, from a statistical mechanics point of view, the effects of bounded stochastic perturbations of the protein decay rate for a bistable biomolecular network module. Namely, we consider the perturbations of the protein decay/binding rate constant DBRC in a circuit modeling the positive feedback of a transcription factor TF on its own synthesis. The DBRC models both the spontaneous degradation of the TF and its linking to other unknown biomolecular factors or drugs.We show that bounded perturbations of the DBRC preserve the positivity of the parameter value and also its limited variation, and induce effects of interest. First, the noise amplitude induces a first-order phase transition. This is of interest since the system in study has neither spatial components nor it is composed by multiple interacting networks. In particular, we observe that the system passes from two to a unique stochastic attractor, and vice-versa. This behavior is different from noise-induced transitions also termed phenomenological bifurcations, where a unique stochastic attractor changes its shape depending on the values of a parameter. Moreover, we observe irreversible jumps as a consequence of the above-mentioned phase transition.We show that the illustrated mechanism holds for general models with the same deterministic hysteresis bifurcation structure. Finally, we illustrate the possible implications of our findings to the intracellular pharmacodynamics of drugs delivered in continuous infusion. C 2017 Elsevier B.V. All rights reserved.</div>','<b>Title: </b>Protein-DNA binding dynamics predict transcriptional response to nutrients in archaea<br><b>Year: </b>2013<br><b>Author: </b> H. Todor,  K. Sharma,  Amc. Pittman and  Ak. Schmid<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1093/nar/gkt659"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>NUCLEIC ACIDS RESEARCH<br><b>Cluster-Keywords: </b><em>systems biology, regulatory network, interaction networks, network model, gene regulatory, network, network biology, networks, gene network, response networks, network dynamics, dynamic model, cell networks, gene expression, biological networks, system dynamics, dynamic network, model, biological systems, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Organisms across all three domains of life use gene regulatory networks GRNs to integrate varied stimuli into coherent transcriptional responses to environmental pressures. However, inferring GRN topology and regulatory causality remains a central challenge in systems biology. Previous work characterized TrmB as a global metabolic transcription factor in archaeal extremophiles. However, it remains unclear how TrmB dynamically regulates its similar to 100 metabolic enzyme-coding gene targets. Using a dynamic perturbation approach, we elucidate the topology of the TrmB metabolic GRN in the model archaeon Halobacterium salinarum. Clustering of dynamic gene expression patterns reveals that TrmB functions alone to regulate central metabolic enzyme-coding genes but cooperates with various regulators to control peripheral metabolic pathways. Using a dynamical model, we predict gene expression patterns for some TrmB-dependent promoters and infer secondary regulators for others. Our data suggest feed-forward gene regulatory topology for cobalamin biosynthesis. In contrast, purine biosynthesis appears to require TrmB-independent regulators. We conclude that TrmB is an important component for mediating metabolic modularity, integrating nutrient status and regulating gene expression dynamics alone and in concert with secondary regulators.</div>','<b>Title: </b>Lentiviral-mediated Knockdown During Ex Vivo Erythropoiesis of Human Hematopoietic Stem Cells<br><b>Year: </b>2011<br><b>Author: </b> Cg. Palii,  R. Pasha and  M. Brand<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3791/2813"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOVE-JOURNAL OF VISUALIZED EXPERIMENTS<br><b>Cluster-Keywords: </b><em>systems biology, dynamic network, early differentiation, complex diseases, stem cells, erythroid cells, system dynamics, network biomarkers, immune system, tcm syndrome, dnb genes, gene expression, dnb model, stem cell, expression dynamics, model system, mathematical model, model dynamics, dynamic model, sensitivity analysis<div style="font-size:11px;"><br><b>Abstract: </b><em>Erythropoiesis is a commonly used model system to study cell differentiation. During erythropoiesis, pluripotent adult human hematopoietic stem cells HSCs differentiate into oligopotent progenitors, committed precursors and mature red blood cells 1. This process is regulated for a large part at the level of gene expression, whereby specific transcription factors activate lineage-specific genes while concomitantly repressing genes that are specific to other cell types 2. Studies on transcription factors regulating erythropoiesis are often performed using human and murine cell lines that represent, to some extent, erythroid cells at given stages of differentiation 3-5. However transformed cell lines can only partially mimic erythroid cells and most importantly they do not allow one to comprehensibly study the dynamic changes that occur as cells progress through many stages towards their final erythroid fate. Therefore, a current challenge remains the development of a protocol to obtain relatively homogenous populations of primary HSCs and erythroid cells at various stages of differentiation in quantities that are sufficient to perform genomics and proteomics experiments.Here we describe an ex vivo cell culture protocol to induce erythroid differentiation from human hematopoietic stem/progenitor cells that have been isolated from either cord blood, bone marrow, or adult peripheral blood mobilized with G-CSF leukapheresis. This culture system, initially developed by the Douay laboratory 6, uses cytokines and co-culture on mesenchymal cells to mimic the bone marrow microenvironment. Using this ex vivo differentiation protocol, we observe a strong amplification of erythroid progenitors, an induction of differentiation exclusively towards the erythroid lineage and a complete maturation to the stage of enucleated red blood cells. Thus, this system provides an opportunity to study the molecular mechanism of transcriptional regulation as hematopoietic stem cells progress along the erythroid lineage.Studying erythropoiesis at the transcriptional level also requires the ability to over-express or knockdown specific factors in primary erythroid cells. For this purpose, we use a lentivirus-mediated gene delivery system that allows for the efficient infection of both dividing and non-dividing cells 7. Here we show that we are able to efficiently knockdown the transcription factor TAL1 in primary human erythroid cells. In addition, GFP expression demonstrates an efficiency of lentiviral infection close to 90. Thus, our protocol provides a highly useful system for characterization of the regulatory network of transcription factors that control erythropoiesis.</div>','<b>Title: </b>Fragmentation alters stream fish community structure in dendritic ecological networks<br><b>Year: </b>2012<br><b>Author: </b> Js. Perkin and  Kb. Gido<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1890/12-0318.1"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ECOLOGICAL APPLICATIONS<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Effects of fragmentation on the ecology of organisms occupying dendritic ecological networks DENs have recently been described through both conceptual and mathematical models, but few hypotheses have been tested in complex, real-world ecosystems. Stream fishes provide a model system for assessing effects of fragmentation on the structure of communities occurring within DENs, including how fragmentation alters metacommunity dynamics and biodiversity. A recently developed habitat-availability measure, the "dendritic connectivity index DCI, allows for assigning quantitative measures of connectivity in DENs regardless of network extent or complexity, and might be used to predict fish community response to fragmentation. We characterized stream fish community structure in 12 DENs in the Great Plains, USA, during periods of dynamic summer and muted fall discharge regimes to test the DCI as a predictive model of fish community response to fragmentation imposed by road crossings. Results indicated that fish communities in stream segments isolated by road crossings had reduced species richness alpha diversity relative to communities that maintained connectivity with the surrounding DEN during summer and fall. Furthermore, isolated communities had greater dissimilarity beta diversity to downstream sites not isolated by road crossings during summer and fall. Finally, dissimilarity among communities within DENs decreased as a function of increased habitat connectivity measured using the DCI for summer and fall, suggesting that communities within highly connected DENs tend to be more homogeneous. Our results indicate that the DCI is sensitive to community effects of fragmentation in riverscapes and might be used by managers to predict ecological responses to changes in habitat connectivity. Moreover, our findings illustrate that relating structural connectivity of riverscapes to functional connectivity among communities might aid in maintaining metacommunity dynamics and biodiversity in complex dendritic ecosystems.</div>','<b>Title: </b>Approximate probabilistic analysis of biopathway dynamics<br><b>Year: </b>2012<br><b>Author: </b> B. Liu,  A. Hagiescu,  Sk. Palaniappan,  B. Chattopadhyay,  Z. Cui,  Wf. Wong and  Ps. Thiagarajan<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1093/bioinformatics/bts166"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>circadian rhythms, systems biology, biological clock, circadian clock, biological rhythms, hpa axis, model, clock network, mathematical models, biological systems, dynamic systems, ode models, biology models, single cell, network, model parameters, experimental data, models, systems, dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>Motivation: Biopathways are often modeled as systems of ordinary differential equations ODEs. Such systems will usually have many unknown parameters and hence will be difficult to calibrate. Since the data available for calibration will have limited precision, an approximate representation of the ODEs dynamics should suffice. One must, however, be able to efficiently construct such approximations for large models and perform model calibration and subsequent analysis.Results: We present a graphical processing unit GPU based scheme by which a system of ODEs is approximated as a dynamic Bayesian network DBN. We then construct a model checking procedure for DBNs based on a simple probabilistic linear time temporal logic. The GPU implementation considerably extends the reach of our previous PC-cluster-based implementation Liu et al., 2011b. Further, the key components of our algorithm can serve as the GPU kernel for other Monte Carlo simulations-based analysis of biopathway dynamics. Similarly, our model checking framework is a generic one and can be applied in other systems biology settings.We have tested our methods on three ODE models of bio-pathways: the epidermal growth factor-nerve growth factor pathway, the segmentation clock network and the MLC-phosphorylation pathway models. The GPU implementation shows significant gains in performance and scalability whereas the model checking framework turns out to be convenient and efficient for specifying and verifying interesting pathways properties.</div>','<b>Title: </b>A deterministic model of growth factor-induced angiogenesis<br><b>Year: </b>2005<br><b>Author: </b> Sy. Sun,  Mf. Wheeler,  M. Obeyesekere and  Cw. Patrick<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.bulm.2004.07.004"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BULLETIN OF MATHEMATICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Understanding the formation and structure of a capillary network is critical for any reparative strategy since the capillary network dictates tissue survival, hemodynamics, and mass transport. Vascular assembly and patterning has largely been studied using a reductionist approach where a particular endothelial cell molecular pathway or cellular mechanism is investigated as a relatively closed system. This trend of research has yielded a staggering wealth of genes, proteins, and cells that play critical roles in angiogenesis and some have resulted in successful targeted angiogenic therapies. However, these genes, proteins, and cells do not exist in discrete closed systems, rather they are intimately coupled across spatial and temporal dimensions. Designing experiments to study a single or group of perturbations is fraught with confounding complications. An integrative tool is required that incorporates gene, protein, and cell information and appropriately describes the complex systems behavior of vascular assembly and patterning.In this paper, we propose a new deterministic mathematical formulation to model growth factor-induced angiogenesis. Conductivity, of the extracellular matrix for the movement/extension of capillary sprouts is a new concept introduced to account for the heterogeneity and anisotropy of the extracellular matrix. The replacement of traditional endothelial cell density by the capillary indicator function enhances the capabilities of capturing the capillary network sharply in a fine scale i.e., tracking the dynamics of the tip in uni-cellular scale. Major mechanisms including cell proliferation, sprout branching, and anastomosis are incorporated directly into this continuous mathematical model and model parameters are perturbed to determine the strength of their effect on angiogenesis. The model is fully deterministic and generates the overall dendritic structure of the capillary network morphologically similar to those observed in vivo. The simulations "capture" significant vascular patterning, such as vascular loops and backward growth. Moreover, the simulations provide a deeper understanding of the influence of extracellular matrix on angiogenesis and vascular patterning. An advantage of this model is that the complex physical, chemical and biological processes in angiogenesis can be described and consequently analyzed by a mathematical system with self-contained information. C 2004 Society for Mathematical Biology. Published by Elsevier Ltd. All rights reserved.</div>','<b>Title: </b>Introduction to information systems theory: concepts, formalism and applications<br><b>Year: </b>2004<br><b>Author: </b> Vs. Lerner<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1080/00207720410001734183"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>INTERNATIONAL JOURNAL OF SYSTEMS SCIENCE<br><b>Cluster-Keywords: </b><em>information systems, gene expression, gene network, systems theory, system models, systems biology, systems information, network model, genetic network, mathematical models, dynamic systems, dynamic network, systemic model, network topology, information mechanisms, complex systems, systems complex, relational models, models main, information model<div style="font-size:11px;"><br><b>Abstract: </b><em>An aim of the proposed information systems theory IST is to build a bridge between the general systems theorys formalism and the world of information and information technologies, dealing with transformation of information as a common non-material substance, whose models informs of computer algorithms and programs could be implemented to different material objects, including a humans thoughts and languages. A new approach to IST is based on a single concept and follows mathematical formalism that uses an information variation principle to build an information systemic model of a specific object. The problems solution, procedure and methodology modelling are illustrated by the application of the information macrodynamics IMD, which reveals the system models main layers: microlevel stochastics, macrolevel dynamics, hierarchical dynamic network IN of information structures, its minimal logic, and optimal code of communication language, generated by the IN hierarchy, dynamics and geometry. The systems complex dynamics originate information geometry and evolution with the functional information mechanisms of ordering, cooperation, mutation, stability, diversity, adaptation, self-organization, the double spirals genetics, the systems generation, decaying, and transferring information to other systems along with the information mechanisms of heredity and replication. The developed IMDs theoretical computer-based methodology and software have been applied to such areas a technology, communications, computer science, intelligent processes, biology, economy, management, and other non-physical and physical subjects with their mutual interactions, informational superimposition, and the information transferred between interactions.</div>','<b>Title: </b>Integrating multi-scale knowledge on cardiac development into a computational model of ventricular trabeculation<br><b>Year: </b>2014<br><b>Author: </b> Ba. De Boer,  Jf. Le Garrec,  Vm. Christoffels,  Sm. Meilhac and  Jm. Ruijter<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1002/wsbm.1285"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>WILEY INTERDISCIPLINARY REVIEWS-SYSTEMS BIOLOGY AND MEDICINE<br><b>Cluster-Keywords: </b><em>systems biology, network models, signaling network, parameter estimation, dynamic models, biological systems, mathematical models, metabolic network, parameter sensitivities, computational model, gene circuit, circuit models, experimental data, metabolic models, model system, scalable parameter, model parameters, parameter space, biomolecular networks, model<div style="font-size:11px;"><br><b>Abstract: </b><em>Insights into the mechanisms of development of the mammalian four-chambered heart are based on biological observations at organ, tissue, cell, and molecular levels, but the full integration of these experimental data awaits a systems biology approach. Such an approach can be employed to formulate and test conceptual models in a computational simulation. To illustrate how this can be applied to heart development, we used the process of trabeculation, which is the formation of muscular strands during chamber development. We selected this process because it is localized, involves a restricted number of cell types, and a range of experimental data is available. Trabeculation of the ventricles is based on the interplay between endocardial and myocardial cells and involves molecular pathways underlying cell-cell interactions and tissue-specific cell behavior. A cellular Potts model was used for the simulation of these multi-scale processes. With fairly simple inputs, of which the relative contributions are unknown, an iterative exploration achieved an outcome that resembles the trabeculation process and allows further investigation of contributing factors. The systems biology pipeline from biological observations and conceptual modeling to a mathematical model and computational algorithms is described and discussed. The multi-level biological observations provide the components and their connections of the conceptual model. However, the true strength of systems biology must be found in the biological test of the predictions that result from an experimental change in the computational model. These validated predictions will ultimately elucidate the functional role of a component or interaction in the process of heart development. WIREs Syst Biol Med 2014, 6:389-397. doi: 10.1002/wsbm.1285 For further resources related to this article, please visit the . Conflict of interest: The authors have declared no conflicts of interest for this article.</div>','<b>Title: </b>Systems science, information systems theory, and informational macrodynamics: review<br><b>Year: </b>2007<br><b>Author: </b> Vs. Lerner<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1108/03684920710741224"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>KYBERNETES<br><b>Cluster-Keywords: </b><em>blood flow, blood cells, flow rate, blood pumps, cells, cell, results, patients, stem cells, methods, blood, patients blood, flow, patients methods, patients objectives, cell viability, objectives, results cells, blood pressure, cell culture<div style="font-size:11px;"><br><b>Abstract: </b><em>Purpose - Science of systems requires a specific and constructive mathematical model and language, which describe jointly such systemic categories as adaptation, self-organization, complexity, evolution, and bring the applied tools for building a system model for each specific object of a diverse nature. This formalism should be connected directly with a world of information and computer applications of systemic model, developed for a particular object. The considered information systems theory IST is aimed at building a bridge between. the mathematical systemic formalism and information technologies to develop a constructive systemic model of revealing information regularities and specific information code for each object.Design/methodology/approach - To fulfill this goal and the considered systems definition, the IST joins two main concepts: unified information description of interacted flows, initiated by the sources of different nature, with common information language and systems modeling methodology, applied to distinct interdisciplinary objects; general systems information formalism for building the model, which allows expressing mathematically the systems regularities and main systemic mechanisms.Findings - The formalism of informational macrodynamics IMD, based of the minimax variational principle, reveals the system models main layers: microlevel stochastics, macrolevel dynamics, hierarchical dynamic network IN of information structures, its minimal logic, and optimal code of communication language, generated by the IN hierarchy, dynamics, and geometry. The systems complex dynamics originate information geometry and evolution with the functional information mechanisms of ordering, cooperation, mutation, stability, diversity, adaptation, self-organization, and the double helixs genetic code.Practical implications - The developed IMDs theoretical computer-based methodology and the software has been applied to such areas as technology, communications, computer science, intelligent processes, biology, economy, management, and other nonphysical and physical subjects.Originality/value - The IMDs macrodynamics of uncertainties connect randomness and regularities, stochastic and determinism, reversibility and irreversibility, symmetry and asymmetry, stability and instability, structurization and stochastization, order and disorder, as a result of micro-macrolevels interactions for an open system, when the external environment can change the models structure.</div>','<b>Title: </b>An externally modulated, noise-driven switch for the regulation of SPI1 in Salmonella enterica serovar Typhimurium<br><b>Year: </b>2011<br><b>Author: </b> M. Bailly-Bechet,  A. Benecke,  Wd. Hardt,  V. Lanza,  A. Sturm and  R. Zecchina<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s00285-010-0385-1"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF MATHEMATICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>gene expression, gene network, stochastic model, gene regulatory, systems biology, expression systems, gene regulation, network activity, expression dynamics, noise control, regulatory network, deterministic models, network components, stochastic modeling, regulation system, escherichia coli, regulatory system, synthetic gene, regulatory genes, gene<div style="font-size:11px;"><br><b>Abstract: </b><em>In this work we consider the regulation system present on the SPI1 pathogenicity island of Salmonella enterica serovar Typhimurium. It is well-known that HilA is the central regulator in the overall scheme of SPI1 regulation and directly binds to virulence operons and activates their expression. The regulation of the expression of HilA is via a complex feed-forward loop involving three transcriptional activators: HilC, HilD and RtsA, and the negative regulator HilE. Our aim is to model this regulation network and study its dynamical behavior. We show that this regulatory system can display a bistable behavior relevant to the biology of Salmonella, and that noise can be a driving force in this system.</div>','<b>Title: </b>Computing weakly reversible linearly conjugate chemical reaction networks with minimal deficiency<br><b>Year: </b>2013<br><b>Author: </b> Md. Johnston,  D. Siegel and  G. Szederkenyi<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.mbs.2012.09.008"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MATHEMATICAL BIOSCIENCES<br><b>Cluster-Keywords: </b><em>reaction networks, reaction systems, metabolic networks, chemical reaction, network structure, systems biology, biological systems, biological networks, biochemical reaction, reaction rates, differential equations, reaction graph, complex networks, ssi modules, networks, life systems, ordinary differential, cellular networks, reaction, network flows<div style="font-size:11px;"><br><b>Abstract: </b><em>Mass-action kinetics is frequently used in systems biology to model the behavior of interacting chemical species. Many important dynamical properties are known to hold for such systems if their underlying networks are weakly reversible and have a low deficiency. In particular, the Deficiency Zero and Deficiency One Theorems guarantee strong regularity with regards to the number and stability of positive equilibrium states. It is also known that chemical reaction networks with distinct reaction structure can admit mass-action systems with the same qualitative dynamics. The theory of linear conjugacy encapsulates the cases where this relationship is captured by a linear transformation. In this paper, we propose a mixed-integer linear programming algorithm capable of determining the minimal deficiency weakly reversible reaction network which admits a mass-action system which is linearly conjugate to a given reaction network. C 2012 Elsevier Inc. All rights reserved.</div>','<b>Title: </b>Differential dependency network analysis to identify condition-specific topological changes in biological networks<br><b>Year: </b>2009<br><b>Author: </b> B. Zhang,  H. Li,  Rb. Riggins,  M. Zhan,  J. Xuan,  Z. Zhang,  Ep. Hoffman,  R. Clarke and  Y. Wang<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1093/bioinformatics/btn660"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>regulatory networks, gene regulatory, gene expression, network inference, expression data, network structure, gene networks, networks grns, regulatory genes, boolean networks, bayesian network, network reconstruction, networks gene, network dynamics, gene regulation, inference method, systems biology, time series, infer gene, network<div style="font-size:11px;"><br><b>Abstract: </b><em>Motivation: Significant efforts have been made to acquire data under different conditions and to construct static networks that can explain various gene regulation mechanisms. However, gene regulatory networks are dynamic and condition-specific; under different conditions, networks exhibit different regulation patterns accompanied by different transcriptional network topologies. Thus, an investigation on the topological changes in transcriptional networks can facilitate the understanding of cell development or provide novel insights into the pathophysiology of certain diseases, and help identify the key genetic players that could serve as biomarkers or drug targets.Results: Here, we report a differential dependency network DDN analysis to detect statistically significant topological changes in the transcriptional networks between two biological conditions. We propose a local dependency model to represent the local structures of a network by a set of conditional probabilities. We develop an efficient learning algorithm to learn the local dependency model using the Lasso technique. A permutation test is subsequently performed to estimate the statistical significance of each learned local structure. In testing on a simulation dataset, the proposed algorithm accurately detected all the genes with network topological changes. The method was then applied to the estrogen-dependent T-47D estrogen receptor-positive ER breast cancer cell line datasets and human and mouse embryonic stem cell datasets. In both experiments using real microarray datasets, the proposed method produced biologically meaningful results. We expect DDN to emerge as an important bioinformatics tool in transcriptional network analyses. While we focus specifically on transcriptional networks, the DDN method we introduce here is generally applicable to other biological networks with similar characteristics.</div>','<b>Title: </b>From word models to executable models of signaling networks using automated assembly<br><b>Year: </b>2017<br><b>Author: </b> Bm. Gyori,  Ja. Bachman,  K. Subramanian,  Jl. Muhlich,  L. Galescu and  Pk. Sorger<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.15252/msb.20177651"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MOLECULAR SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>allosteric proteins, signaling networks, protein interactions, network model, molecular dynamics, protein complexes, allosteric network, signaling proteins, cell network, biochemical networks, free energy, combinatorial complexity, molecular species, biological systems, receptor signaling, systems biology, chemical species, model, dynamics simulations, cell signaling<div style="font-size:11px;"><br><b>Abstract: </b><em>Word models natural language descriptions of molecular mechanisms are a common currency in spoken and written communication in biomedicine but are of limited use in predicting the behavior of complex biological networks. We present an approach to building computational models directly from natural language using automated assembly. Molecular mechanisms described in simple English are read by natural language processing algorithms, converted into an intermediate representation, and assembled into executable or network models. We have implemented this approach in the Integrated Network and Dynamical Reasoning Assembler INDRA, which draws on existing natural language processing systems as well as pathway information in Pathway Commons and other online resources. We demonstrate the use of INDRA and natural language to model three biological processes of increasing scope: i p53 dynamics in response to DNA damage, ii adaptive drug resistance in BRAF-V600E-mutant melanomas, and iii the RAS signaling pathway. The use of natural language makes the task of developing a model more efficient and it increases model transparency, thereby promoting collaboration with the broader biology community.</div>','<b>Title: </b>Computational implementation of a tunable multicellular memory circuit for engineered eukaryotic consortia<br><b>Year: </b>2015<br><b>Author: </b> J. Sardanyes,  A. Bonforti,  N. Conde,  R. Sole and  J. Macia<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3389/fphys.2015.00281"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>FRONTIERS IN PHYSIOLOGY<br><b>Cluster-Keywords: </b><em>gene circuits, synthetic biology, circuit behavior, biological parts, synthetic gene, circuit design, genetic circuits, cell types, metabolite dynamics, mathematical model, designed circuit, control circuit, biological systems, circuit models, synthetic circuit, feedback circuits, biological circuits, design models, metabolic pathways, circuit parameters<div style="font-size:11px;"><br><b>Abstract: </b><em>Cells are complex machines capable of processing information by means of an entangled network of molecular interactions. A crucial component of these decision-making systems is the presence of memory and this is also a specially relevant target of engineered synthetic systems. A classic example of memory devices is a 1-bit memory element known as the flip-flop. Such system can be in principle designed using a single-cell implementation, but a direct mapping between standard circuit design and a living circuit can be cumbersome. Here we present a novel computational implementation of a 1-bit memory device using a reliable multicellular design able to behave as a set reset flip-flop that could be implemented in yeast cells. The dynamics of the proposed synthetic circuit is investigated with a mathematical model using biologically-meaningful parameters. The circuit is shown to behave as a flip-flop in a wide range of parameter values. The repression strength for the NOT logics is shown to be crucial to obtain a good flip-flop signal. Our model also shows that the circuit can be externally tuned to achieve different memory states and dynamics, such as persistent and transient memory. We have characterized the parameter domains for robust memory storage and retrieval as well as the corresponding time response dynamics.</div>','<b>Title: </b>Information systems theory and informational macrodynamics: Review of the main results<br><b>Year: </b>2007<br><b>Author: </b> Vs. Lerner<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1109/TSMCC.2007.905749"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IEEE TRANSACTIONS ON SYSTEMS MAN AND CYBERNETICS PART C-APPLICATIONS AND REVIEWS<br><b>Cluster-Keywords: </b><em>information systems, gene expression, gene network, systems theory, system models, systems biology, systems information, network model, genetic network, mathematical models, dynamic systems, dynamic network, systemic model, network topology, information mechanisms, complex systems, systems complex, relational models, models main, information model<div style="font-size:11px;"><br><b>Abstract: </b><em>The science of systems requires a specific and constructive mathematical model and language which would describe jointly such systemic categories as adaptation, self-organization, complexity, and bring the applied tools for building a system model for each specific object of a diverse nature. This formalism should be connected directly with a world of information and computer applications of a systemic model, developed for a particular object. The aim of the considered information systems theory IST is to build a bridge between the mathematical systemic formalism and information technologies. The objective is to develop a constructive systemic model of revealing information regularities and specific information code for each object. To fulfill the goal and the considered systems definition, the IST joins two main concepts. First is the unified information description of the different natures interacted flows, with a common information language and systems modeling methodology, applied to distinct interdisciplinary objects. Second is the general systems information formalism for building the model, which allows expressing mathematically the systems regularities and main systemic mechanisms. This formalism is represented by informational macrodynamics IMDs, which reveals the system models main layers: microlevel stochastics, macrolevel dynamics, hierarchical dynamic network IN of information structures, its minimal logic, and optimal code of communication language, generated by the IN hierarchy, dynamics, and geometry. The systems complex dynamics originate information geometry and evolution with the functional information mechanisms of ordering, cooperation, mutation, stability, diversity, adaptation, self-organization, and the double helixs genetic code. The developed IMDs theoretical computer-based methodology and software has been applied to such areas as technology, communications, computer science, intelligent processes, biology, economy management, and other nonphysical and physical subjects. The review is written for a broad readers audience familiar with basic physics, calculus, and computers.</div>','<b>Title: </b>Coarse Dynamics for Coarse Modeling: An Example From Population Biology<br><b>Year: </b>2014<br><b>Author: </b> J. Bush and  K. Mischaikow<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3390/e16063379"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ENTROPY<br><b>Cluster-Keywords: </b><em>regulatory networks, network dynamics, gene regulatory, network model, genetic network, gene networks, genetic regulatory, mathematical model, parameter space, network system, interaction networks, system dynamics, network structure, network, dynamic networks, network reduction, models models, biological networks, network motifs, biological systems<div style="font-size:11px;"><br><b>Abstract: </b><em>Networks have become a popular way to concisely represent complex nonlinear systems where the interactions and parameters are imprecisely known. One challenge is how best to describe the associated dynamics, which can exhibit complicated behavior sensitive to small changes in parameters. A recently developed computational approach that we refer to as a database for dynamics provides a robust and mathematically rigorous description of global dynamics over large ranges of parameter space. To demonstrate the potential of this approach we consider two classical age-structured population models that share the same network diagram and have a similar nonlinear overcompensatory term, but nevertheless yield different patterns of qualitative behavior as a function of parameters. Using a generalization of these models we relate the different structure of the dynamics that are observed in the context of biologically relevant questions such as stable oscillations in populations, bistability, and permanence.</div>','<b>Title: </b>Multi-equilibrium property of metabolic networks: SSI module<br><b>Year: </b>2011<br><b>Author: </b> Hb. Lei,  Jf. Zhang and  La. Chen<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-5-S1-S15"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>reaction networks, reaction systems, metabolic networks, chemical reaction, network structure, systems biology, biological systems, biological networks, biochemical reaction, reaction rates, differential equations, reaction graph, complex networks, ssi modules, networks, life systems, ordinary differential, cellular networks, reaction, network flows<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Revealing the multi-equilibrium property of a metabolic network is a fundamental and important topic in systems biology. Due to the complexity of the metabolic network, it is generally a difficult task to study the problem as a whole from both analytical and numerical viewpoint. On the other hand, the structure-oriented modularization idea is a good choice to overcome such a difficulty, i.e. decomposing the network into several basic building blocks and then studying the whole network through investigating the dynamical characteristics of the basic building blocks and their interactions. Single substrate and single product with inhibition SSI metabolic module is one type of the basic building blocks of metabolic networks, and its multi-equilibrium property has important influence on that of the whole metabolic networks.Results: In this paper, we describe what the SSI metabolic module is, characterize the rates of the metabolic reactions by Hill kinetics and give a unified model for SSI modules by using a set of nonlinear ordinary differential equations with multi-variables. Specifically, a sufficient and necessary condition is first given to describe the injectivity of a class of nonlinear systems, and then, the sufficient condition is used to study the multi-equilibrium property of SSI modules. As a main theoretical result, for the SSI modules in which each reaction has no more than one inhibitor, a sufficient condition is derived to rule out multiple equilibria, i.e. the Jacobian matrix of its rate function is nonsingular everywhere.Conclusions: In summary, we describe SSI modules and give a general modeling framework based on Hill kinetics, and provide a sufficient condition for ruling out multiple equilibria of a key type of SSI module.</div>','<b>Title: </b>Network Features and Dynamical Landscape of Naive and Primed Pluripotency<br><b>Year: </b>2018<br><b>Author: </b> B. Pfeuty,  C. Kress and  B. Pain<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.bpj.2017.10.033"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOPHYSICAL JOURNAL<br><b>Cluster-Keywords: </b><em>stem cell, stem cells, systems biology, mathematical models, cell biology, network model, signaling pathways, cell fate, computational models, cancer cells, gene expression, network dynamics, embryonic stem, signaling network, prostate cancer, mathematical modeling, psychological stress, apoptosis signaling, computational systems, molecular mechanisms<div style="font-size:11px;"><br><b>Abstract: </b><em>Although the broad and unique differentiation potential of pluripotent stem cells relies on a complex transcriptional network centered around Oct4, Sox2, and Nanog, two well-distinct pluripotent states, called "naive and "primed, have been described in vitro and markedly differ in their developmental potential, their expression profiles, their signaling requirements, and their reciprocal conversion. Aiming to determine the key features that segregate and coordinate these two states, data-driven optimization of network models is performed to identify relevant parameter regimes and reduce network complexity to its core structure. Decision dynamics of optimized networks is characterized by signal-dependent multistability and strongly asymmetric transitions among naive, primed, and nonpluripotent states. Further model perturbation and reduction approaches reveal that such a dynamical landscape of pluripotency involves a functional partitioning of the regulatory network. Specifically, two overlapping positive feedback modules, Klf4/Esrrb/Nanog and Oct4/Nanog, stabilize the naive or the primed state, respectively. In turn, their incoherent feedforward and negative feedback coupling mediated by the Erk/Gsk3 module is critical for robust segregation and sequential progression between naive and primed states before irreversible exit from pluripotency.</div>','<b>Title: </b>Integrating demographic and genetic effects of connections on the viability of an endangered plant in a highly fragmented habitat<br><b>Year: </b>2013<br><b>Author: </b> F. Noel,  N. Machon and  A. Robert<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.biocon.2012.07.029"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOLOGICAL CONSERVATION<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Classical ecological theories state that the viability of small metapopulations is critically related to: 1 the correlation of local population dynamics, 2 the possibility of extinct patch re-colonisation, and 3 the effects of genetic deterioration. Recent works suggested that these factors are not independent because dispersal might affect both re-colonisation and genetic rescue processes and because connectivity between habitat patches may be related to their environmental synchrony. Close patches may exhibit both high connectivity and synchronisation of their environments.Here, we examined these effects in the endangered plant Ranunculus nodiflorus that inhabits temporary water puddles, where water corridors constitute the vector 1 allowing seed dispersal throughout puddle networks and 2 leading to synchrony of local environmental factors. We utilised a demo-genetic metapopulation model to show how the antagonistic effects of connectivity and synchrony lead to complex interactions between the species biology and network configuration. We specifically demonstrated that 1 when an effective seed bank occurs, viability is maximised either for systems of numerous small independent networks not connected and not synchronised, or for one single large network, while intermediate situations lead to a dramatic increase of the extinction risk; 2 genetic deterioration has notable negative effects on viability only in situations where environmentally driven extinction is buffered by the presence of a seed bank and network configuration; and 3 temporary connections among networks lead to a dramatic reduction of the extinction risk. The implementation of such connections is an efficient management option for the species. C 2012 Elsevier Ltd. All rights reserved.</div>','<b>Title: </b>Analysis of a minimal model for p53 oscillations<br><b>Year: </b>2007<br><b>Author: </b> S. Bottani and  B. Grammaticos<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.jtbi.2007.04.026"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF THEORETICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>dna damage, signalling networks, network dynamics, pmdm system, systems biology, systems dynamics, biological systems, single cell, kinase inhibitors, signaling network, biological networks, cellular automata, cell cycle, oscillation system, temporal dynamics, signaling pathway, individual cells, signaling processes, dynamic model, damage response<div style="font-size:11px;"><br><b>Abstract: </b><em>Oscillatory behaviours in genetic networks are important examples for studying the principles underlying the dynamics of cellular regulation. Recently the team of Alon has reported a surprisingly rich oscillatory response of the p53 tumor suppressor to irradiation stress et al. Lahav, G., Rosenfeld, N., Sigal, A., Geva-Zatorsky, N., Levine, AJ., Elowitz, M.B., Alon, U., 2004. Dynamics of the p53Mdm2 feedback loop in individual cells. Nat. Genet. 36 2, 147-150; Geva-Zatorsky, N., Rosenfeld, N., Itzkovitz, S., Milo, R., Sigal, A., Dekel, E., Yarnitzky, T., Liron, Y., Polak, P., Lahav, G., Alon, U., 2006. Oscillations and variability in the p53 system. Mol. Syst. Biol. 2, 2006.0033. Several models for this system have been proposed by different groups, based essentially on negative feedback loops. In this paper we investigate in detail oscillations and stability in a deterministic time delayed differential model of the core circuit for p53 expression. This model is representative of a class of modelling approaches of this system, based on a "minimal" set of well-established biomolecular regulations. Depending on the protein degradation rates we show the existence of bifurcations between a stable steady state and oscillations both in presence and absence of stress. C 2007 Elsevier Ltd. All rights reserved.</div>','<b>Title: </b>Relations between the dynamics of network systems and their subnetworks<br><b>Year: </b>2017<br><b>Author: </b> Yx. Wang,  K. Chilakamarri,  D. Kazakos and  Mc. Leite<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3934/Math.2017.2.437"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>AIMS MATHEMATICS<br><b>Cluster-Keywords: </b><em>regulatory networks, network dynamics, gene regulatory, network model, genetic network, gene networks, genetic regulatory, mathematical model, parameter space, network system, interaction networks, system dynamics, network structure, network, dynamic networks, network reduction, models models, biological networks, network motifs, biological systems<div style="font-size:11px;"><br><b>Abstract: </b><em>Statistical analysis of the connectivity of real world networks have revealed interesting features such as community structure, network motif and as on. Such discoveries tempt us to understand the dynamics of a complex network system by studying those of its subnetworks. This approach is feasible only if the dynamics of the subnetwork systems can somehow be preserved or partially preserved in the whole system. Most works studied the connectivity structures of networks while very few considered the possibility of translating the dynamics of a subnetwork system to the whole system. In this paper, we address this issue by focusing on considering the relations between cycles and fixed points of a network system and those of its subnetworks based on Boolean framework. We proved that at a condition we called agreeable, if X-0 is a fixed point of the whole system, then X-0 restricted to the phase-space of one of the subnetwork systems must be a fixed point as well. An equivalent statement on cycles follows from this result. In addition, we discussed the relations between the product of the transition diagrams a representation of trajectories of subnetwork systems and the transition diagram of the whole system.</div>','<b>Title: </b>Tellurium notebooks-An environment for reproducible dynamical modeling in systems biology<br><b>Year: </b>2018<br><b>Author: </b> Jk. Medley,  K. Choi,  M. Konig,  L. Smith,  S. Gu,  J. Hellerstein,  Sc. Sealfon and  Hm. Sauro<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pcbi.1006220"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS COMPUTATIONAL BIOLOGY<br><b>Cluster-Keywords: </b><em>systems biology, network models, signaling network, parameter estimation, dynamic models, biological systems, mathematical models, metabolic network, parameter sensitivities, computational model, gene circuit, circuit models, experimental data, metabolic models, model system, scalable parameter, model parameters, parameter space, biomolecular networks, model<div style="font-size:11px;"><br><b>Abstract: </b><em>The considerable difficulty encountered in reproducing the results of published dynamical models limits validation, exploration and reuse of this increasingly large biomedical research resource. To address this problem, we have developed Tellurium Notebook, a software system for model authoring, simulation, and teaching that facilitates building reproducible dynamical models and reusing models by 1 providing a notebook environment which allows models, Python code, and narrative to be intermixed, 2 supporting the COMBINE archive format during model development for capturing model information in an exchangeable format and 3 enabling users to easily simulate and edit public COMBINE- compliant models from public repositories to facilitate studying model dynamics, variants and test cases. Tellurium Notebook, a Python-based Jupyter-like environment, is designed to seamlessly interoperate with these community standards by automating conversion between COMBINE standards formulations and corresponding in-line, human-readable representations. Thus, Tellurium brings to systems biology the strategy used by other literate notebook systems such as Mathematica. These capabilities allow users to edit every aspect of the standards-compliant models and simulations, run the simulations in-line, and re-export to standard formats. We provide several use cases illustrating the advantages of our approach and how it allows development and reuse of models without requiring technical knowledge of standards. Adoption of Tellurium should accelerate model development, reproducibility and reuse.</div>','<b>Title: </b>The p53-MDM2 network: from oscillations to apoptosis<br><b>Year: </b>2007<br><b>Author: </b> I. Bose and  B. Ghosh<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s12038-007-0103-3"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF BIOSCIENCES<br><b>Cluster-Keywords: </b><em>dna damage, signalling networks, network dynamics, pmdm system, systems biology, systems dynamics, biological systems, single cell, kinase inhibitors, signaling network, biological networks, cellular automata, cell cycle, oscillation system, temporal dynamics, signaling pathway, individual cells, signaling processes, dynamic model, damage response<div style="font-size:11px;"><br><b>Abstract: </b><em>The p53 protein is well-known for its tumour suppressor function. The p53-MDM2 negative feedback loop constitutes the core module of a network of regulatory interactions activated under cellular stress. In normal cells, the level of p53 proteins is kept low by MDM2, i.e. MDM2 negatively regulates the activity of p53. In the case of DNA damage, the p53-mediated pathways are activated leading to cell cycle arrest and repair of the DNA. If repair is not possible due to excessive damage, the p53-mediated apoptotic pathway is activated bringing about cell death. In this paper, we give an overview of our studies on the p53-MDM2 module and the associated pathways from a systems biology perspective. We discuss a number of key predictions, related to some specific aspects of cell cycle arrest and cell death, which could be tested in experiments.</div>','<b>Title: </b>A Geometric Method for Model Reduction of Biochemical Networks with Polynomial Rate Functions<br><b>Year: </b>2015<br><b>Author: </b> Ss. Samal,  D. Grigoriev,  H. Frohlich,  A. Weber and  O. Radulescu<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s11538-015-0118-0"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BULLETIN OF MATHEMATICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>immune system, network models, immune response, model, biological systems, models, model reduction, dynamic model, biological networks, system, mathematical models, biochemical network, response model, systems, systems biology, network, immune, biology model, reaction networks, system systems<div style="font-size:11px;"><br><b>Abstract: </b><em>Model reduction of biochemical networks relies on the knowledge of slow and fast variables. We provide a geometric method, based on the Newton polytope, to identify slow variables of a biochemical network with polynomial rate functions. The gist of the method is the notion of tropical equilibration that provides approximate descriptions of slow invariant manifolds. Compared to extant numerical algorithms such as the intrinsic low-dimensional manifold method, our approach is symbolic and utilizes orders of magnitude instead of precise values of the model parameters. Application of this method to a large collection of biochemical network models supports the idea that the number of dynamical variables in minimal models of cell physiology can be small, in spite of the large number of molecular regulatory actors.</div>','<b>Title: </b>Mechanisms for temporal tuning and filtering by postsynaptic signaling pathways<br><b>Year: </b>2002<br><b>Author: </b> Us. Bhalla<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/S0006-34950275205-3"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOPHYSICAL JOURNAL<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Networks of signaling pathways perform complex temporal decoding functions in diverse biological systems, including the synapse, development, ants bacterial chemotaxis. This paper examines temporal filtering and tuning properties of synaptic signaling pathways as a possible substrate for emergent temporal decoding, A mass action kinetic model of 16 synaptic signaling pathways was used to dissect out the contribution of these pathways in linear cascades and when coupled to form a network. The model predicts two primary mechanisms of temporal tuning of pathways: a weighted summation of responses of pathways with different timings and the presence of biochemical feedback loops with emergent dynamics. Regulatory inputs act differently on these two tuning mechanisms. In the first case, regulators act like a gain-control on pathways with different intrinsic tuning. In the case of feedback loops, the temporal properties of the loop itself are changed. These basic tuning mechanisms may underlie specialized temporal tuning functions in more complex signaling systems in biology.</div>','<b>Title: </b>Modular, rule-based modeling for the design of eukaryotic synthetic gene circuits<br><b>Year: </b>2013<br><b>Author: </b> Ma. Marchisio,  M. Colaiacovo,  E. Whitehead and  J. Stelling<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-7-42"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>gene circuits, synthetic biology, circuit behavior, biological parts, synthetic gene, circuit design, genetic circuits, cell types, metabolite dynamics, mathematical model, designed circuit, control circuit, biological systems, circuit models, synthetic circuit, feedback circuits, biological circuits, design models, metabolic pathways, circuit parameters<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: The modular design of synthetic gene circuits via composable parts DNA segments and pools of signal carriers molecules such as RNA polymerases and ribosomes has been successfully applied to bacterial systems. However, eukaryotic cells are becoming a preferential host for new synthetic biology applications. Therefore, an accurate description of the intricate network of reactions that take place inside eukaryotic parts and pools is necessary. Rule-based modeling approaches are increasingly used to obtain compact representations of reaction networks in biological systems. However, this approach is intrinsically non-modular and not suitable per se for the description of composable genetic modules. In contrast, the Model Description Language MDL adopted by the modeling tool ProMoT is highly modular and it enables a faithful representation of biological parts and pools.Results: We developed a computational framework for the design of complex eukaryotic gene circuits by generating dynamic models of parts and pools via the joint usage of the BioNetGen rule-based modeling approach and MDL. The framework converts the specification of a part or pool structure into rules that serve as inputs for BioNetGen to calculate the parts species and reactions. The BioNetGen output is translated into an MDL file that gives a complete description of all the reactions that take place inside the part or pool together with a proper interface to connect it to other modules in the circuit. In proof-of-principle applications to eukaryotic Boolean circuits with more than ten genes and more than one thousand reactions, our framework yielded proper representations of the circuits truth tables.Conclusions: For the model-based design of increasingly complex gene circuits, it is critical to achieve exact and systematic representations of the biological processes with minimal effort. Our computational framework provides such a detailed and intuitive way to design new and complex synthetic gene circuits.</div>','<b>Title: </b>Can the assumption of a non-random search improve our prediction of butterfly fluxes between resource patches<br><b>Year: </b>2004<br><b>Author: </b> P. Kindlmann,  S. Aviron,  F. Burel and  A. Ouin<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1111/j.0307-6946.2004.00614.x"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ECOLOGICAL ENTOMOLOGY<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>1. Understanding dispersal patterns that enable small, spatially isolated populations to survive in fragmented landscapes has become an important issue in conservation biology and landscape management. However, for most of the species of interest it is not known whether dispersing individuals navigate or follow systematic search strategies, as opposed to moving randomly.2. Recently it was shown that individuals of the butterfly species Maniola jurtina do not seek resources by means of random flight. If true, this may be problematic for existing metapopulation models, including those based on the evolution of dispersal rates in metapopulations.3. The study tested to what extent the non-random dispersal patterns described in the literature can explain M. jurtina fluxes in its natural habitat.4. A model based on literature assumptions of M. jurtina movements is presented in the work reported here, and its predictions are compared with 2 years of capture-recapture data on its fluxes in two landscapes.5. The model provides a good fit to the data and gives better predictions than the model based only on patch sizes and distances between patches.6. Thus, if data are available about the actual landscape under consideration, the model should be preferred to simpler approaches; however, in general theoretical considerations the simple approach based on patch size and the degree of its isolation will retain its value.</div>','<b>Title: </b>BioNetGen: software for rule-based modeling of signal transduction based on the interactions of molecular domains<br><b>Year: </b>2004<br><b>Author: </b> Ml. Blinov,  Jr. Faeder,  B. Goldstein and  Ws. Hlavacek<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1093/bioinformatics/bth378"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>allosteric proteins, signaling networks, protein interactions, network model, molecular dynamics, protein complexes, allosteric network, signaling proteins, cell network, biochemical networks, free energy, combinatorial complexity, molecular species, biological systems, receptor signaling, systems biology, chemical species, model, dynamics simulations, cell signaling<div style="font-size:11px;"><br><b>Abstract: </b><em>BioNetGen allows a user to create a computational model that characterizes the dynamics of a signal transduction system, and that accounts comprehensively and precisely for specified enzymatic activities, potential post-translational modifications and interactions of the domains of signaling molecules. The output defines and parameterizes the network of molecular species that can arise during signaling and provides functions that relate model variables to experimental readouts of interest. Models that can be generated are relevant for rational drug discovery, analysis of proteomic data and mechanistic studies of signal transduction.</div>','<b>Title: </b>Model-based selection of the robust JAK-STAT activation mechanism<br><b>Year: </b>2012<br><b>Author: </b> M. Rybinski and  A. Gambin<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.jtbi.2012.04.031"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF THEORETICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>JAM-STAT pathway family is a principal signaling mechanism in eukaryotic cells. Evolutionary conserved roles of this mechanism include control over fundamental processes such as cell growth or apoptosis. Deregulation of the JAM-STAT signaling is frequently associated with cancerogenesis. JAK-STAT pathways become hyper-activated in many human tumors. Therefore, components of these pathways are an attractive target for drugs, which design requires as adequate models as possible. Although, in principle, JAK-STAT signaling is relatively simple, the ambiguities in a receptor activation prevent a clear explanation of the underlying molecular mechanism.Here, we compare four variants of a computational model of the JAK1/2-STAT1 signaling pathway. These variants capture known, basic discrepancies in the mechanism of activation of a cytokine receptor, in the context of all key components of the pathway. We carry out a comparative analysis using mass action kinetics. The investigated differences are so marginal that all models satisfy a goodness of fit criteria to the extent that the state of the art Bayesian model selection BMS method fails to significantly promote one model. Therefore, we comparatively investigate changes in a robustness of the JAK1/2-STAT1 pathway variants using the global sensitivity analysis method GSA, complemented with the identifiability analysis IA. Both BMS and GSA are used to analyze the models for the varying parameter values. We found out that, both BMS and GSA, narrowed down to the receptor activation component, slightly promote the least complex model. Further, insightful, comprehensive GSA, motivated by the concept of robustness, allowed us to show that the precise order of reactions of a ligand binding and a receptor dimerization is not as important as the on-membrane pre-assembly of the dimers in the absence of ligand.The main value of this work is an evaluation of the usefulness of different model selection methods in a frequently encountered, but not much discussed case of a model of a considerable size, which has several variants differing at peripheries. In such situation, all considered variants can reach nearly perfect agreement with respect to their numerical simulations results and, most often, the sufficient experimental data to test against is not available. We argue that in such an adverse setting, the GSA and IA, although not directly corresponding to the model selection problem, can be more informative than the representative, generalizability-based approaches to this task. An additional insight into how the responsibility for the network dynamics spreads among model parameters, enables more conscious, expert-mediated choice of the preferred model. C 2012 Elsevier Ltd. All rights reserved.</div>','<b>Title: </b>The RNA world in the 21st century-a systems approach to finding non-coding keys to clinical questions<br><b>Year: </b>2016<br><b>Author: </b> U. Schmitz,  H. Naderi-Meshkin,  Sk. Gupta,  O. Wolkenhauer and  J. Vera<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1093/bib/bbv061"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BRIEFINGS IN BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>regulatory network, systems biology, systems biological, network dynamics, gene expression, biochemical networks, raft proteins, biological networks, gene circuits, gene regulatory, signalling network, lipid raft, nuclear lamina, protein domain, networks systems, biological approaches, expression pattern, protein interaction, cell differentiation, network motifs<div style="font-size:11px;"><br><b>Abstract: </b><em>There was evidence that RNAs are a functionally rich class of molecules not only since the arrival of the next-generation sequencing technology. Non-coding RNAs ncRNA could be the key to accelerated diagnosis and enhanced prediction of disease and therapy outcomes as well as the design of advanced therapeutic strategies to overcome yet unsatisfactory approaches.In this review, we discuss the state of the art in RNA systems biology with focus on the application in the systems biomedicine field. We propose guidelines for analysing the role of microRNAs and long non-coding RNAs in human pathologies. We introduce RNA expression profiling and network approaches for the identification of stable and effective RNomics-based biomarkers, providing insights into the role of ncRNAs in disease regulation. Towards this, we discuss ways to model the dynamics of gene regulatory networks and signalling pathways that involve ncRNAs. We also describe data resources and computational methods for finding putative mechanisms of action of ncRNAs. Finally, we discuss avenues for the computer-aided design of novel RNA-based therapeutics.</div>','<b>Title: </b>Hybrid centralized/decentralized control of a network of bacteria-based bio-hybrid microrobots<br><b>Year: </b>2019<br><b>Author: </b> Ej. Leaman,  Bq. Geuther and  B. Behkam<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s12213-019-00116-0"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF MICRO-BIO ROBOTICS<br><b>Cluster-Keywords: </b><em>systems biology, biological systems, system dynamics, molecular systems, experimental design, network model, model system, model parameters, experiment design, kinetic parameters, perfect adaptation, experimental data, molecular mechanisms, homeostatic mechanisms, switching dynamics, biological mechanisms, control systems, chemotaxis network, decentralized control, feedback control<div style="font-size:11px;"><br><b>Abstract: </b><em>Engineering microrobotic systems using a bio-hybrid approach that couples synthetic micro/nanoscale components with live cells is a powerful approach to address key shortcomings of micro/nanotechnology such as providing an on-board power source, communication, and decentralized control. In the last decade, a number of centralized control strategies based on native biological mechanisms have been demonstrated; however, decentralized cooperative control of a swarm of bio-hybrid microrobots has rarely been explored. In this work, we use our previously developed agent-based computational model to explore the utility of integrating synthetic biology with bio-hybrid microrobotics with the goal of developing strategies for robust decentralized control of bio-hybrid microrobotic systems. To this end, our Nanoscale Bacteria-Enabled Autonomous Delivery System NanoBEADS, wherein each agent is comprised of an Escherichia coli bacterium conjugated with an ensemble of nanoparticles, was used as a model system. We imparted bacteria with engineered biological circuits to facilitate agent-agent communication and enable predictable and robust cooperative control of a network of NanoBEADS agents. We developed a hybrid control strategy wherein a centralized chemotaxis-basedcontrol scheme is used to direct migration, and a decentralized quorum sensing QS-basedcontrol scheme enables the agents to independently coordinate a cooperative behavior fluorescent protein expression. We then computationally analyzed the role of bacterial subsystem properties and spatiotemporal changes in population organization in the emergent behavior i.e., the activation time of the QS response of the NanoBEADS. We show that the spatial changes in NanoBEADS agents distribution, in response to the centralized control input, and the mass transport boundary conditions are critical factors in determining the dynamics as well as the sensitivity and robustness of the QS response. We analyze the stark differences in feasible QS parameter domains under dissimilar chemoattractant gradients and show that the flow boundary conditions may greatly influence system function. Altogether, we show that a bacteria-based bio-hybrid system must be designed with careful consideration of growth rate, chemotaxisproperties, QS properties, and the environment initial and boundary conditions in the context of its target application. This modeling framework can serve as an insightful tool for the predictive design of bio-hybrid microrobotic swarms with a tunable and robust response.</div>','<b>Title: </b>Theoretical Study of the One Self-Regulating Gene in the Modified Wagner Model<br><b>Year: </b>2018<br><b>Author: </b> C. Guyeux,  Jf. Couchot,  A. Le Rouzic,  Jm. Bahi and  L. Marangio<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3390/math6040058"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MATHEMATICS<br><b>Cluster-Keywords: </b><em>network models, gene network, regulatory networks, boolean networks, gene expression, systems biology, neural networks, network dynamics, gene regulatory, biological networks, network structure, gap gene, gene regulation, cell cycle, network, gene interactions, signaling network, quantum systems, system dynamics, networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Predicting how a genetic change affects a given character is a major challenge in biology, and being able to tackle this problem relies on our ability to develop realistic models of gene networks. However, such models are rarely tractable mathematically. In this paper, we propose a mathematical analysis of the sigmoid variant of the Wagner gene-network model. By considering the simplest case, that is, one unique self-regulating gene, we show that numerical simulations are not the only tool available to study such models: theoretical studies can be done too, by mathematical analysis of discrete dynamical systems. It is first shown that the particular sigmoid function can be theoretically investigated. Secondly, we provide an illustration of how to apply such investigations in the case of the dynamical system representing the one self-regulating gene. In this context, we focused on the composite function fam.x where fa is the parametric sigmoid function and m is a scalar not in 0, 1 and we have proven that the number of fixed-point can be deduced theoretically, according to the values of a and m.</div>','<b>Title: </b>Computational modelling reveals feedback redundancy within the epidermal growth factor receptor/extracellular-signal regulated kinase signalling pathway<br><b>Year: </b>2008<br><b>Author: </b> Rj. Orton,  Oe. Sturm,  A. Gormand,  W. Wolch and  Dr. Gilbert<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1049/iet-syb:20070066"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IET SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>feedback loops, signaling network, positive feedback, network model, negative feedback, biochemical networks, signaling pathways, signal transduction, systems biology, kinase erk, protein kinase, transduction networks, dynamic model, emt network, erk pathway, cell fate, cell death, biochemical reaction, coupled feedback, impulse response<div style="font-size:11px;"><br><b>Abstract: </b><em>The epidermal growth factor receptor EGFR activated extracellular-signal regulated kinase ERK pathway is a central cell signalling pathway that mediates many biological responses including cell proliferation, transformation, survival and motility. Deregulation of the pathway either through mutation of components or overexpression of EGFRs is associated with several forms of cancer. Under normal conditions, EGF stimulates a rapid but transient activation of ERK as the signal is rapidly shutdown, whereas under cancerous conditions, the ERK signal cannot be shutdown and is sustained. Computational modelling techniques have been used to investigate the signalling dynamics of the EGFR/ERK pathway, focusing on identifying the key processes involved in signal termination and what role the ERK to son of sevenless SOS negative feedback loop plays in generating a transient response. This model predicts that this negative feedback loop is not needed to achieve a transient activation of ERK as the process of receptor degradation alone is enough to terminate the signal. Importantly, the behaviour and predictions of this model are verified with laboratory data, as is essential for modern systems biology approaches. Further analysis showed that the feedback loop and receptor degradation were both redundant processes, as each could compensate for the absence of the other. This led to the prediction that in the case of a receptor which is not degraded, such as the insulin receptor, the negative feedback loop to SOS will actually be essential for a transient response to be achieved. Overall, the results shed new light on the role of negative feedback in EGF receptor signalling and suggest that different receptors are dependent on different features within the ERK pathway when relaying their signals.</div>','<b>Title: </b>A synthetic-natural hybrid oscillator in human cells<br><b>Year: </b>2010<br><b>Author: </b> Je. Toettcher,  C. Mock,  E. Batchelor,  A. Loewer and  G. Lahav<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1073/pnas.1005615107"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA<br><b>Cluster-Keywords: </b><em>dna damage, signalling networks, network dynamics, pmdm system, systems biology, systems dynamics, biological systems, single cell, kinase inhibitors, signaling network, biological networks, cellular automata, cell cycle, oscillation system, temporal dynamics, signaling pathway, individual cells, signaling processes, dynamic model, damage response<div style="font-size:11px;"><br><b>Abstract: </b><em>Recent studies have shown that many cell-signaling networks contain interactions and feedback loops that give rise to complex dynamics. Synthetic biology has allowed researchers to construct and analyze well-defined signaling circuits exhibiting behavior that can be predicted and quantitatively understood. Combining these approaches-wiring natural network components together with engineered interactions-has the potential to precisely modulate the dynamics of endogenous signaling processes and control the cell decisions they influence. Here, we focus on the p53 signaling pathway as a template for constructing a tunable oscillator comprised of both natural and synthetic components in mammalian cells. We find that a reduced p53 circuit implementing a single feedback loop preserves some features of the full networks dynamics, exhibiting pulses of p53 with tightly controlled timing. However, in contrast to the full natural p53 network, these pulses are damped in individual cells, with amplitude that depends on the input strength. Guided by a computational model of the reduced circuit, we constructed and analyzed circuit variants supplemented with synthetic positive and negative feedback loops and subjected to chemical perturbation. Our work demonstrates that three important features of oscillator dynamics-amplitude, period, and the rate of damping-can be controlled by manipulating stimulus level, interaction strength, and feedback topology. The approaches taken here may be useful for the rational design of synthetic networks with defined dynamics, and for identifying perturbations that control dynamics in natural biological circuits for research or therapeutic purposes.</div>','<b>Title: </b>Dynamic Modeling of the Human Coagulation Cascade Using Reduced Order Effective Kinetic Models<br><b>Year: </b>2015<br><b>Author: </b> A. Sagar and  Jd. Varner<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3390/pr3010178"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PROCESSES<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>In this study, we present a novel modeling approach which combines ordinary differential equation ODE modeling with logical rules to simulate an archetype biochemical network, the human coagulation cascade. The model consisted of five differential equations augmented with several logical rules describing regulatory connections between model components, and unmodeled interactions in the network. This formulation was more than an order of magnitude smaller than current coagulation models, because many of the mechanistic details of coagulation were encoded as logical rules. We estimated an ensemble of likely model parameters N = 20 from in vitro extrinsic coagulation data sets, with and without inhibitors, by minimizing the residual between model simulations and experimental measurements using particle swarm optimization PSO. Each parameter set in our ensemble corresponded to a unique particle in the PSO. We then validated the model ensemble using thrombin data sets that were not used during training. The ensemble predicted thrombin trajectories for conditions not used for model training, including thrombin generation for normal and hemophilic coagulation in the presence of platelets a significant unmodeled component. We then used flux analysis to understand how the network operated in a variety of conditions, and global sensitivity analysis to identify which parameters controlled the performance of the network. Taken together, the hybrid approach produced a surprisingly predictive model given its small size, suggesting the proposed framework could also be used to dynamically model other biochemical networks, including intracellular metabolic networks, gene expression programs or potentially even cell free metabolic systems.</div>','<b>Title: </b>Discovering a critical transition state from nonalcoholic hepatosteatosis to nonalcoholic steatohepatitis by lipidomics and dynamical network biomarkers<br><b>Year: </b>2016<br><b>Author: </b> Rn. Sa,  Ww. Zhang,  J. Ge,  Xb. Wei,  Yh. Zhou,  Dr. Landzberg,  Zz. Wang,  Xl. Han,  Ln. Chen and  Hy. Yin<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1093/jmcb/mjw016"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF MOLECULAR CELL BIOLOGY<br><b>Cluster-Keywords: </b><em>systems biology, dynamic network, early differentiation, complex diseases, stem cells, erythroid cells, system dynamics, network biomarkers, immune system, tcm syndrome, dnb genes, gene expression, dnb model, stem cell, expression dynamics, model system, mathematical model, model dynamics, dynamic model, sensitivity analysis<div style="font-size:11px;"><br><b>Abstract: </b><em>Nonalcoholic fatty liver disease NAFLD is a major risk factor for type 2 diabetes and metabolic syndrome. However, accurately differentiating nonalcoholic steatohepatitis NASH from hepatosteatosis remains a clinical challenge. We identified a critical transition stage termed pre-NASH during the progression from hepatosteatosis to NASH in a mouse model of high fat-induced NAFLD, using lipidomics and a mathematical model termed dynamic network biomarkers DNB. Different from the conventional biomarker approach based on the abundance of molecular expressions, the DNB model exploits collective fluctuations and correlations of different metabolites at a network level. We found that the correlations between the blood and liver lipid species drastically decreased after the transition from steatosis to NASH, which may account for the current difficulty in differentiating NASH from steatosis based on blood lipids. Furthermore, most DNB members in the blood circulation, especially for triacylglycerol TAG, are also identified in the liver during the disease progression, suggesting a potential clinical application of DNB to diagnose NASH based on blood lipids. We further identified metabolic pathways responsible for this transition. Our study suggests that the transition from steatosis to NASH is not smooth and the existence of pre-NASH may be partially responsible for the current clinical limitations to diagnose NASH. If validated in humans, our study will open a new avenue to reliably diagnose pre-NASH and achieve early intervention of NAFLD.</div>','<b>Title: </b>A Systems Biology Approach to Understanding Alcoholic Liver Disease Molecular Mechanism: The Development of Static and Dynamic Models<br><b>Year: </b>2017<br><b>Author: </b> L. Shafaghati,  Z. Razaghi-Moghadam and  J. Mohammadnejad<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s11538-017-0336-8"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BULLETIN OF MATHEMATICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>gene expression, systems biology, metabolic network, network model, network analysis, dynamic model, signaling pathways, signaling network, regulatory network, dynamic network, biology approach, candidate genes, gene network, gene regulatory, biology analysis, mathematical model, network dynamic, biology network, metabolic pathways, pathway analysis<div style="font-size:11px;"><br><b>Abstract: </b><em>Alcoholic liver disease ALD is a complex disease characterized by damages to the liver and is the consequence of excessive alcohol consumption over years. Since this disease is associated with several pathway failures, pathway reconstruction and network analysis are likely to explicit the molecular basis of the disease. To this aim, in this paper, a network medicine approach was employed to integrate interactome protein-protein interaction and signaling pathways and transcriptome data to reconstruct both a static network of ALD and a dynamic model for it. Several data sources were exploited to assemble a set of ALD-associated genes which further was used for network reconstruction. Moreover, a comprehensive literature mining reveals that there are four signaling pathways with crosstalk TLR4, NF- B, MAPK and Apoptosis which play a major role in ALD. These four pathways were exploited to reconstruct a dynamic model of ALD. The results assure that these two models are consistent with a number of experimental observations. The static network of ALD and its dynamic model are the first models provided for ALD which offer potentially valuable information for researchers in this field.</div>','<b>Title: </b>Deconstructing the core dynamics from a complex time-lagged regulatory biological circuit<br><b>Year: </b>2009<br><b>Author: </b> O. Eriksson,  B. Brinne,  Y. Zhou,  J. Bjorkegren and  J. Tegner<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1049/iet-syb.2007.0028"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IET SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>blood flow, blood cells, flow rate, blood pumps, cells, cell, results, patients, stem cells, methods, blood, patients blood, flow, patients methods, patients objectives, cell viability, objectives, results cells, blood pressure, cell culture<div style="font-size:11px;"><br><b>Abstract: </b><em>Complex regulatory dynamics is ubiquitous in molecular networks composed of genes and proteins. Recent progress in computational biology and its application to molecular data generate a growing number of complex networks. Yet, it has been difficult to understand the governing principles of these networks beyond graphical analysis or extensive numerical simulations. Here the authors exploit several simplifying biological circumstances which thereby enable to directly detect the underlying dynamical regularities driving periodic oscillations in a dynamical nonlinear computational model of a protein-protein network. System analysis is performed using the cell cycle, a mathematically well-described complex regulatory circuit driven by external signals. By introducing an explicit time delay and using a tearing-and-zooming approach the authors reduce the system to a piecewise linear system with two variables that capture the dynamics of this complex network. A key step in the analysis is the identification of functional subsystems by identifying the relations between state-variables within the model. These functional subsystems are referred to as dynamical modules operating as sensitive switches in the original complex model. By using reduced mathematical representations of the subsystems the authors derive explicit conditions on how the cell cycle dynamics depends on system parameters, and can, for the first time, analyse and prove global conditions for system stability. The approach which includes utilising biological simplifying conditions, identification of dynamical modules and mathematical reduction of the model complexity may be applicable to other well-characterised biological regulatory circuits.</div>','<b>Title: </b>Defining the robust behaviour of the plant clock gene circuit with absolute RNA timeseries and open infrastructure<br><b>Year: </b>2015<br><b>Author: </b> A. Flis,  Ap. Fernandez,  T. Zielinski,  V. Mengin,  R. Sulpice,  K. Stratford,  A. Hume,  A. Pokhilko,  Mm. Southern,  Dd. Seaton,  Hg. Mcwatters,  M. Stitt,  Kj. Halliday and  Aj. Millar<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1098/rsob.150042"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>OPEN BIOLOGY<br><b>Cluster-Keywords: </b><em>circadian clock, biological systems, network models, mathematical models, clock gene, systems biology, model selection, biological networks, network data, experimental data, model parameters, network dynamics, plant circadian, regulatory network, clock model, plant clock, complex network, clock circuit, approximate bayesian, stochastic models<div style="font-size:11px;"><br><b>Abstract: </b><em>Our understanding of the complex, transcriptional feedback loops in the circadian clock mechanism has depended upon quantitative, timeseries data from disparate sources. We measure clock gene RNA profiles in Arabidopsis thaliana seedlings, grown with or without exogenous sucrose, or in soil-grown plants and in wild-type and mutant backgrounds. The RNA profiles were strikingly robust across the experimental conditions, so current mathematical models are likely to be broadly applicable in leaf tissue. In addition to providing reference data, unexpected behaviours included co-expression of PRR9 and ELF4, and regulation of PRR5 by GI. Absolute RNA quantification revealed low levels of PRR9 transcripts peak approx. 50 copies cell-1 compared with other clock genes, and threefold higher levels of LHY RNA more than 1500 copies cell-1 than of its close relative CCA1. The data are disseminated from BioDare, an online repository for focused timeseries data, which is expected to benefit mechanistic modelling. One data subset successfully constrained clock gene expression in a complex model, using publicly available software on parallel computers, without expert tuning or programming. We outline the empirical and mathematical justification for data aggregation in understanding highly interconnected, dynamic networks such as the clock, and the observed design constraints on the resources required to make this approach widely accessible.</div>','<b>Title: </b>Backward-stochastic-differential-equation approach to modeling of gene expression<br><b>Year: </b>2017<br><b>Author: </b> E. Shamarova,  R. Chertovskih,  Af. Ramos and  P. Aguiar<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1103/PhysRevE.95.032418"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PHYSICAL REVIEW E<br><b>Cluster-Keywords: </b><em>reaction networks, network model, systems biology, biochemical reaction, stochastic reaction, model reduction, stochastic model, network dynamics, stochastic biochemical, stochastic simulation, system size, chemical reaction, gene network, elementary reactions, biochemical networks, parameter identification, parameter inference, cell populations, small system, linear noise<div style="font-size:11px;"><br><b>Abstract: </b><em>In this article, we introduce a backward method to model stochastic gene expression and protein-level dynamics. The protein amount is regarded as a diffusion process and is described by a backward stochastic differential equation BSDE. Unlike many other SDE techniques proposed in the literature, the BSDE method is backward in time; that is, instead of initial conditions it requires the specification of end-point "final" conditions, in addition to the model parametrization. To validate our approach we employ Gillespies stochastic simulation algorithm SSA to generate forward benchmark data, according to predefined gene network models. Numerical simulations show that the BSDE method is able to correctly infer the protein-level distributions that preceded a known final condition, obtained originally from the forward SSA. This makes the BSDE method a powerful systems biology tool for time-reversed simulations, allowing, for example, the assessment of the biological conditions e.g., protein concentrations that preceded an experimentally measured event of interest e.g., mitosis, apoptosis, etc..</div>','<b>Title: </b>Integrative Analysis of Subcellular Quantitative Proteomics Studies Reveals Functional Cytoskeleton Membrane-Lipid Raft Interactions in Cancer<br><b>Year: </b>2016<br><b>Author: </b> Ad. Shah,  Kl. Inder,  Ak. Shah,  As. Cristino,  Ab. Mckie,  H. Gabra,  Mj. Davis and  Mm. Hill<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1021/acs.jproteome.5b01035"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF PROTEOME RESEARCH<br><b>Cluster-Keywords: </b><em>regulatory network, systems biology, systems biological, network dynamics, gene expression, biochemical networks, raft proteins, biological networks, gene circuits, gene regulatory, signalling network, lipid raft, nuclear lamina, protein domain, networks systems, biological approaches, expression pattern, protein interaction, cell differentiation, network motifs<div style="font-size:11px;"><br><b>Abstract: </b><em>Lipid rafts are dynamic membrane micro domains that orchestrate molecular interactions and are implicated in cancer development. To understand the functions of lipid rafts in cancer, we performed an integrated analysis of quantitative lipid raft proteomics data sets modeling progression in breast cancer, melanoma, and renal cell carcinoma. This analysis revealed that cancer development is associated with increased membrane raft cytoskeleton interactions, with similar to 40 of elevated lipid raft proteins being cytoskeletal components. Previous studies suggest a potential functional role for the raft cytoskeleton in the action of the putative tumor suppressors PTRF/Cavin-1 and Merlin. To extend the observation, we examined lipid raft proteome modulation by an unrelated tumor suppressor opioid binding protein cell-adhesion molecule OPCML in ovarian cancer SKOV3 cells. In agreement with the, other model systems, quantitative proteomics revealed that 39 of OPCML-depleted lipid raft proteins are cytoskeletal components, with microfilaments and intermediate filaments specifically down-regulated. Furthermore, protein protein interaction network and simulation analysis showed significantly higher interactions among cancer raft proteins compared with general human raft proteins. Collectively, these results suggest increased cytoskeleton-mediated stabilization of lipid raft domains with greater molecular interactions as a common, functional, and reversible feature of cancer cells.</div>','<b>Title: </b>Hybrid modeling of the crosstalk between signaling and transcriptional networks using ordinary differential equations and multi-valued logic<br><b>Year: </b>2014<br><b>Author: </b> Fm. Khan,  U. Schmitz,  S. Nikolov,  D. Engelmann,  Bm. Putzer,  O. Wolkenhauer and  J. Vera<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.bbapap.2013.05.007"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOCHIMICA ET BIOPHYSICA ACTA-PROTEINS AND PROTEOMICS<br><b>Cluster-Keywords: </b><em>regulatory network, systems biology, systems biological, network dynamics, gene expression, biochemical networks, raft proteins, biological networks, gene circuits, gene regulatory, signalling network, lipid raft, nuclear lamina, protein domain, networks systems, biological approaches, expression pattern, protein interaction, cell differentiation, network motifs<div style="font-size:11px;"><br><b>Abstract: </b><em>A decade of successful results indicates that systems biology is the appropriate approach to investigate the regulation of complex biochemical networks involving transcriptional and post-transcriptional regulations. It becomes mandatory when dealing with highly interconnected biochemical networks, composed of hundreds of compounds, or when networks are enriched in non-linear motifs like feedback and feedforward loops. An emerging dilemma is to conciliate models of massive networks and the adequate description, of non-linear dynamics in a suitable modeling framework. Boolean networks are an ideal representation of massive networks that are humble in terms of computational complexity and data demand. However, they are inappropriate when dealing with nested feedback/feedforward loops, structural motifs common in biochemical networks. On the other hand, models of ordinary differential equations ODEs cope well with these loops, but they require enormous amounts of quantitative data for a full characterization of the model. Here we propose hybrid models, composed of ODE and logical sub-modules, as a strategy to handle large scale, non-linear biochemical networks that include transcriptional and post-transcriptional regulations. We illustrate the construction of this kind of models using as example a regulatory network centered on E2F1, a transcription factor involved in cancer. The hybrid modeling approach proposed is a good compromise between quantitative/qualitative accuracy and scalability when considering large biochemical networks with a small highly interconnected core:and module of transcriptionally regulated genes that are not part of critical regulatory loops. This article is part of a Special Issue entitled: Computational Proteomics, Systems Biology 82 Clinical Implications. Guest Editor: Yudong Cai. C 2013 Elsevier B.V. All rights reserved.</div>','<b>Title: </b>The mechanism of proton conduction in phosphoric acid<br><b>Year: </b>2012<br><b>Author: </b> L. Vilciauskas,  Me. Tuckerman,  G. Bester,  Sj. Paddison and  Kd. Kreuer<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1038/NCHEM.1329"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>NATURE CHEMISTRY<br><b>Cluster-Keywords: </b><em>allosteric proteins, signaling networks, protein interactions, network model, molecular dynamics, protein complexes, allosteric network, signaling proteins, cell network, biochemical networks, free energy, combinatorial complexity, molecular species, biological systems, receptor signaling, systems biology, chemical species, model, dynamics simulations, cell signaling<div style="font-size:11px;"><br><b>Abstract: </b><em>Neat liquid phosphoric acid H3PO4 has the highest intrinsic proton conductivity of any known substance and is a useful model for understanding proton transport in other phosphate-based systems in biology and clean energy technologies. Here, we present an ab initio molecular dynamics study that reveals, for the first time, the microscopic mechanism of this high proton conductivity. Anomalously fast proton transport in hydrogen-bonded systems involves a structural diffusion mechanism in which intramolecular proton transfer is driven by specific hydrogen bond rearrangements in the surrounding environment. Aqueous media transport excess charge defects through local hydrogen bond rearrangements that drive individual proton transfer reactions. In contrast, strong, polarizable hydrogen bonds in phosphoric acid produce coupled proton motion and a pronounced protic dielectric response of the medium, leading to the formation of extended, polarized hydrogen-bonded chains. The interplay between these chains and a frustrated hydrogen-bond network gives rise to the high proton conductivity.</div>','<b>Title: </b>CONSTRUCTION AND CUSTOMIZATION OF STABLE OSCILLATION MODELS IN BIOLOGY<br><b>Year: </b>2008<br><b>Author: </b> Ww. Yin and  Eo. Voit<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1142/S0218339008002502"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF BIOLOGICAL SYSTEMS<br><b>Cluster-Keywords: </b><em>systems biology, dynamic models, biological systems, model, dynamic systems, computational systems, network model, experimental data, model parameters, systems, systems model, models, pathway systems, data, biochemical pathway, dynamics model, biochemical networks, computational models, network, system<div style="font-size:11px;"><br><b>Abstract: </b><em>Oscillations exist at all levels of biological systems and are often crucial for their proper functioning. Among the various types of oscillations, limit cycles have received particular attention for more than one hundred years. Specifically, theorems have been established that characterize whether a system might have the capability of exhibiting limit cycles. However, the practical application of these theorems is usually cumbersome and there are hardly any guidelines for devising de novo models that exhibit limit cycles of a desired form. In this paper, we propose a simple method for constructing and customizing stable limit cycles in two-dimensional systems according to desired features, including frequency, amplitude, and phase shift between system variables. The method is based on "inverting" a criterion proposed by Lewis for characterizing oscillations in two-dimensional S-system models. First, we execute comprehensive simulations that result in a set of over 2000 prototype limit cycles. Second, we show with examples how these prototypes can be further customized to adhere to predetermined specifications. This two-step process is fast and efficacious, especially when one considers the paucity of alternative methods. Finally, we illustrate how one may create systems with more complex dynamics by modulating the prototypes with external input signals.</div>','<b>Title: </b>BIO-ENTITY NETWORK FOR ANALYSIS OF PROTEIN-PROTEIN INTERACTION NETWORKS<br><b>Year: </b>2011<br><b>Author: </b> Lh. Ren,  Yz. Shen,  Ys. Ding and  Kc. Chou<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1002/asjc.395"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ASIAN JOURNAL OF CONTROL<br><b>Cluster-Keywords: </b><em>interaction network, actin filament, network model, filament network, network structures, actin network, protein interaction, actin dynamics, network, membrane protein, network dynamics, model dynamic, cell, model, network approach, protein complex, computational model, actin, adhesion dynamics, cell dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>Protein-protein interactions PPIs are vitally important for every process in a living cell. Information about these interactions can improve our understanding of diseases and provide the basis to revolutionize therapeutic treatments. However, since PPIs are involved with extremely complicated biological processes, it is necessary to develop novel tools to deal with this kind of network systems. To realize this, a bio-entity network approach is introduced to show the topology structure of dynamic and collective performances of PPI networks and analyze the variance of the protein node that plays an important role in the PPI network. Also, spectrum analysis is used to capture the discrete and stochastic feature of PPIs. The yeast protein interaction network is considered as a paradigm. It is demonstrated that the proposed approach can easily and clearly identify the hub-proteins that have the most impact on the PPI system concerned. It is expected that the bio-entity network approach as presented in this paper might become a useful tool in system biology.</div>','<b>Title: </b>Emergence of regulatory networks in simulated evolutionary processes<br><b>Year: </b>2005<br><b>Author: </b> D. Drasdo and  M. Kruspe<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1142/S0219525905000415"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ADVANCES IN COMPLEX SYSTEMS<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Despite spectacular progress in biophysics, molecular biology and biochemistry our ability to predict the dynamic behavior of multicellular systems under different conditions is very limited. An important reason for this is that still not enough is known about how cells change their physical and biological properties by genetic or metabolic regulation, and which of these changes affect the cell behavior. For this reason, it is difficult to predict the system behavior of multicellular systems in case the cell behavior changes, for example, as a consequence of regulation or differentiation. The rules that underlie the regulation processes have been determined on the time scale of evolution, by selection on the phenotypic level of cells or cell populations. We illustrate by detailed computer simulations in a multi-scale approach how cell behavior controlled by regulatory networks may emerge as a consequence of an evolutionary process, if either the cells, or populations of cells are subject to selection on particular features. We consider two examples, migration strategies of single cells searching a signal source, or aggregation of two or more cells within minimal multiscale models of biological evolution. Both can be found for example in the life cycle of the slime mold Dictyostelium discoideum. However, phenotypic changes that can lead to completely different modes of migration have also been observed in cells of multi-cellular organisms, for example, as a consequence of a specialization in stem cells or the de-differentiation in tumor cells. The regulatory networks are represented by Boolean networks and encoded by binary strings. The latter may be considered as encoding the genetic information the genotype and are subject to mutations and crossovers. The cell behavior reflects the phenotype. We find that cells adopt naturally observed migration strategies, controlled by networks that show robustness and redundancy. The model simplicity allow us to unambiguously analyze the regulatory networks and the resulting phenotypes by different measures and by knockouts of regulatory elements. We illustrate that in order to maintain a cells phenotype in case of a knockout, the cell may have to be able to deal with contradictory information. In summary, both the cell phenotype as well as the emerged regulatory network behave as their biological counterparts observed in nature.</div>','<b>Title: </b>A data integration methodology for systems biology: Experimental verification<br><b>Year: </b>2005<br><b>Author: </b> D. Hwang,  Jj. Smith,  Dm. Leslie,  Ad. Weston,  Ag. Rust,  S. Ramsey,  Pd. Atauri,  Af. Siegel,  H. Bolouri,  Jd. Aitchison and  L. Hood<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1073/pnas.0508649102"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA<br><b>Cluster-Keywords: </b><em>interaction networks, proteinprotein interactions, protein interaction, network model, complex systems, systems biology, protein protein, protein complex, host network, dynamic interactions, network analysis, biological systems, networks network, cellular network, biological network, viral proteins, interactome networks, regulatory networks, immune system, systems network<div style="font-size:11px;"><br><b>Abstract: </b><em>The integration of data from multiple global assays is essential to understanding dynamic spatiotemporal interactions within cells. In a companion paper, we reported a data integration methodology, designated Pointillist, that can handle multiple data types from technologies with different noise characteristics. Here we demonstrate its application to the integration of 18 data sets relating to galactose utilization in yeast. These data include global changes in mRNA and protein abundance, genome-wide protein-DNA interaction data, database information, and computational predictions of protein-DNA and protein-protein interactions. We divided the integration task to determine three network components: key system elements genes and proteins, protein-protein interactions, and protein-DNA interactions. Results indicate that the reconstructed network efficiently focuses on and recapitulates the known biology of galactose utilization. It also provided new insights, some of which were verified experimentally. The methodology described here, addresses a critical need across all domains of molecular and cell biology, to effectively integrate large and disparate data sets.</div>','<b>Title: </b>Synchronizing stochastic circadian oscillators in single cells of Neurospora crassa<br><b>Year: </b>2016<br><b>Author: </b> Zj. Deng,  S. Arsenault,  C. Caranica,  J. Griffith,  Tt. Zhu,  A. Al-Omari,  Hb. Schuttler,  J. Arnold and  Ld. Mao<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1038/srep35828"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SCIENTIFIC REPORTS<br><b>Cluster-Keywords: </b><em>circadian rhythms, systems biology, biological clock, circadian clock, biological rhythms, hpa axis, model, clock network, mathematical models, biological systems, dynamic systems, ode models, biology models, single cell, network, model parameters, experimental data, models, systems, dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>The synchronization of stochastic coupled oscillators is a central problem in physics and an emerging problem in biology, particularly in the context of circadian rhythms. Most measurements on the biological clock are made at the macroscopic level of millions of cells. Here measurements are made on the oscillators in single cells of the model fungal system, Neurospora crassa, with droplet microfluidics and the use of a fluorescent recorder hooked up to a promoter on a clock controlled gene-2 ccg-2. The oscillators of individual cells are stochastic with a period near 21 hours h, and using a stochastic clock network ensemble fitted by Markov Chain Monte Carlo implemented on general-purpose graphical processing units or GPGPUs we estimated that >94 of the variation in ccg-2 expression was stochastic as opposed to experimental error. To overcome this stochasticity at the macroscopic level, cells must synchronize their oscillators. Using a classic measure of similarity in cell trajectories within droplets, the intraclass correlation ICC, the synchronization surface ICC is measured on >25,000 cells as a function of the number of neighboring cells within a droplet and of time. The synchronization surface provides evidence that cells communicate, and synchronization varies with genotype.</div>','<b>Title: </b>Molecular Network Dynamics of Cell Cycle Control: Periodicity of Start and Finish<br><b>Year: </b>2017<br><b>Author: </b> A. Palmisano,  J. Zamborszky,  C. Oguz and  A. Csikasz-Nagy<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/978-1-4939-6603-521"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>CELL CYCLE SYNCHRONIZATION: METHODS AND PROTOCOLS, 2ND EDITION<br><b>Cluster-Keywords: </b><em>cell cycle, mammalian cell, cdk network, cell proliferation, cell, cells cell, regulatory network, detailed model, cell size, molecular network, skeleton model, cycle arrest, cycle progression, cell division, cell density, kinases cdks, cycle, phase transition, cycle control, network driving<div style="font-size:11px;"><br><b>Abstract: </b><em>The cell division cycle is controlled by a complex regulatory network which ensures that the phases of the cell cycle are executed in the right order. This regulatory network receives signals from the environment, monitors the state of the DNA, and decides timings of cell cycle events. The underlying transcriptional and post-translational regulatory interactions lead to complex dynamical responses, such as the oscillations in the levels of cell cycle proteins driven by intertwined biochemical reactions. A cell moves between different phases of its cycle similar to a dynamical system switching between its steady states. The complex molecular network driving these phases has been investigated in previous computational systems biology studies. Here, we review the critical physiological and molecular transitions that occur in the cell cycle and discuss the role of mathematical modeling in elucidating these transitions and understand cell cycle synchronization.</div>','<b>Title: </b>Bridging genetic networks and queueing theory<br><b>Year: </b>2004<br><b>Author: </b> A. Arazi,  E. Ben-Jacob and  U. Yechiali<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.physa.2003.07.009"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PHYSICA A-STATISTICAL MECHANICS AND ITS APPLICATIONS<br><b>Cluster-Keywords: </b><em>regulatory networks, stochastic simulation, gene networks, reaction networks, systems biology, stochastic model, network models, complex systems, networks stochastic, gene regulatory, biological systems, stochastic dynamics, differential equations, protein networks, cell cycle, continuous model, interaction networks, systems chemistry, time delays, biochemical reaction<div style="font-size:11px;"><br><b>Abstract: </b><em>One of the main challenges facing biology today is the understanding of the joint action of genes, proteins and RNA molecules, interwoven in intricate interdependencies commonly known as genetic networks. To this end, several mathematical approaches have been introduced to date. In addition to developing the analytical tools required for this task anew, one can utilize knowledge found in existing disciplines, specializing in the representation and analysis of systems featuring similar aspects. We suggest queueing theory as a possible source of such knowledge. This discipline, which focuses on the study of workloads forming in a variety of scenarios, offers an assortment of tools allowing for the derivation of the statistical properties of the inspected systems. We argue that a proper adaptation of modeling techniques and analytical methods used in queueing theory can contribute to the study of genetic regulatory networks. This is demonstrated by presenting a queueing-inspired model of a genetic network of arbitrary size and structure, for which the probability distribution function is derived. This model is further applied to the description of the lac operon regulation mechanism. In addition, we discuss the possible benefits stemming for queueing theory from the interdisciplinary dialogue with molecular biology-in particular, the incorporation of various dynamical behaviours into queueing networks. C 2003 Elsevier B.V. All rights reserved.</div>','<b>Title: </b>A predictive model for transcriptional control of physiology in a free living cell<br><b>Year: </b>2007<br><b>Author: </b> R. Bonneau,  Mt. Facciotti,  Dj. Reiss,  Ak. Schmid,  M. Pan,  A. Kaur,  V. Thorsson,  P. Shannon,  Mh. Johnson,  Jc. Bare,  W. Longabaugh,  M. Vuthoori,  K. Whitehead,  A. Madar,  L. Suzuki,  T. Mori,  De. Chang,  J. Diruggiero,  Ch. Johnson,  L. Hood and  Ns. Baliga<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.cell.2007.10.053"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>CELL<br><b>Cluster-Keywords: </b><em>systems biology, regulatory network, interaction networks, network model, gene regulatory, network, network biology, networks, gene network, response networks, network dynamics, dynamic model, cell networks, gene expression, biological networks, system dynamics, dynamic network, model, biological systems, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>The environment significantly influences the dynamic expression and assembly of all components encoded in the genome of an organism into functional biological networks. We have constructed a model for this process in Halobacterium salinarum NRC-1 through the data-driven discovery of regulatory and functional interrelationships among similar to 80 of its genes and key abiotic factors in its hypersaline environment. Using relative changes in 72 transcription factors and 9 environmental factors EFs this model accurately predicts dynamic transcriptional responses of all these genes in 147 newly collected experiments representing completely novel genetic backgrounds and environments-suggesting a remarkable degree of network completeness. Using this model we have constructed and tested hypotheses critical to this organisms interaction with its changing hypersaline environment. This study supports the claim that the high degree of connectivity within biological and EF networks will enable the construction of similar models for any organism from relatively modest numbers of experiments.</div>','<b>Title: </b>Systems biology by the rules: hybrid intelligent systems for pathway modeling and discovery<br><b>Year: </b>2007<br><b>Author: </b> Wj. Bosl<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-1-13"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>network motifs, regulatory network, gene regulatory, systems biology, gene network, bifan motif, complex systems, model comparison, network modeling, complex gene, biological systems, dynamic systems, interaction network, feedforward loop, modeling gene, intelligent systems, modeling approach, biological pathways, motif model, model motifs<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Expert knowledge in journal articles is an important source of data for reconstructing biological pathways and creating new hypotheses. An important need for medical research is to integrate this data with high throughput sources to build useful models that span several scales. Researchers traditionally use mental models of pathways to integrate information and development new hypotheses. Unfortunately, the amount of information is often overwhelming and these are inadequate for predicting the dynamic response of complex pathways. Hierarchical computational models that allow exploration of semi-quantitative dynamics are useful systems biology tools for theoreticians, experimentalists and clinicians and may provide a means for cross-communication.Results: A novel approach for biological pathway modeling based on hybrid intelligent systems or soft computing technologies is presented here. Intelligent hybrid systems, which refers to several related computing methods such as fuzzy logic, neural nets, genetic algorithms, and statistical analysis, has become ubiquitous in engineering applications for complex control system modeling and design. Biological pathways may be considered to be complex control systems, which medicine tries to manipulate to achieve desired results. Thus, hybrid intelligent systems may provide a useful tool for modeling biological system dynamics and computational exploration of new drug targets. A new modeling approach based on these methods is presented in the context of hedgehog regulation of the cell cycle in granule cells. Code and input files can be found at the Bionet website: www.chip.ord/similar to wbosl/Software/Bionet.Conclusion: This paper presents the algorithmic methods needed for modeling complicated biochemical dynamics using rule-based models to represent expert knowledge in the context of cell cycle regulation and tumor growth. A notable feature of this modeling approach is that it allows biologists to build complex models from their knowledge base without the need to translate that knowledge into mathematical form. Dynamics on several levels, from molecular pathways to tissue growth, are seamlessly integrated. A number of common network motifs are examined and used to build a model of hedgehog regulation of the cell cycle in cerebellar neurons, which is believed to play a key role in the etiology of medulloblastoma, a devastating childhood brain cancer.</div>','<b>Title: </b>Sig2GRN: a software tool linking signaling pathway with gene regulatory network for dynamic simulation<br><b>Year: </b>2016<br><b>Author: </b> F. Zhang,  Rs. Liu and  J. Zheng<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/s12918-016-0365-1"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>stem cell, stem cells, systems biology, mathematical models, cell biology, network model, signaling pathways, cell fate, computational models, cancer cells, gene expression, network dynamics, embryonic stem, signaling network, prostate cancer, mathematical modeling, psychological stress, apoptosis signaling, computational systems, molecular mechanisms<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Linking computational models of signaling pathways to predicted cellular responses such as gene expression regulation is a major challenge in computational systems biology. In this work, we present Sig2GRN, a Cytoscape plugin that is able to simulate time-course gene expression data given the user-defined external stimuli to the signaling pathways.Methods: A generalized logical model is used in modeling the upstream signaling pathways. Then a Boolean model and a thermodynamics-based model are employed to predict the downstream changes in gene expression based on the simulated dynamics of transcription factors in signaling pathways.Results: Our empirical case studies show that the simulation of Sig2GRN can predict changes in gene expression patterns induced by DNA damage signals and drug treatments.Conclusions: As a software tool for modeling cellular dynamics, Sig2GRN can facilitate studies in systems biology by hypotheses generation and wet-lab experimental design.</div>','<b>Title: </b>Statistical ensemble of gene regulatory networks of macrophage differentiation<br><b>Year: </b>2016<br><b>Author: </b> F. Castiglione,  P. Tieri,  A. Palma and  As. Jarrah<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/s12859-016-1363-4"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>neural network, network model, network, neurogenesis model, genomic network, networks, saccade control, droplets networks, understanding networks, gene regulatory, model, tumor-immune networks, networks dynamical, network systems, artificial neurogenesis, regulatory networks, dynamic network, information flow, cancer cells, capillary networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Macrophages cover a major role in the immune system, being the most plastic cell yielding several key immune functions.Methods: Here we derived a minimalistic gene regulatory network model for the differentiation of macrophages into the two phenotypes M1 pro- and M2 anti-inflammatory.Results: To test the model, we simulated a large number of such networks as in a statistical ensemble. In other words, to enable the inter-cellular crosstalk required to obtain an immune activation in which the macrophage plays its role, the simulated networks are not taken in isolation but combined with other cellular agents, thus setting up a discrete minimalistic model of the immune system at the microscopic/intracellular i.e., genetic regulation and mesoscopic/intercellular scale.Conclusions: We show that within the mesoscopic level description of cellular interaction and cooperation, the gene regulatory logic is coherent and contributes to the overall dynamics of the ensembles that shows, statistically, the expected behaviour.</div>','<b>Title: </b>Use of mathematics to guide target selection in systems pharmacology; application to receptor tyrosine kinase RTK pathways<br><b>Year: </b>2017<br><b>Author: </b> N. Benson,  Ph. Van Der Graaf and  La. Peletier<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.ejps.2017.05.049"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>EUROPEAN JOURNAL OF PHARMACEUTICAL SCIENCES<br><b>Cluster-Keywords: </b><em>feedback loops, signaling network, positive feedback, network model, negative feedback, biochemical networks, signaling pathways, signal transduction, systems biology, kinase erk, protein kinase, transduction networks, dynamic model, emt network, erk pathway, cell fate, cell death, biochemical reaction, coupled feedback, impulse response<div style="font-size:11px;"><br><b>Abstract: </b><em>A key element of the drug discovery process is target selection. Although the topic is subject to much discussion and experimental effort, there are no defined quantitative rules around optimal selection. Often rules of thumb, that have not been subject to rigorous exploration, are used. In this paper we explore the rule of thumb notion that the molecule that initiates a pathway signal is the optimal target. Given the multi-factorial and complex nature of this question, we have simplified an example pathway to its logical minimum of two steps and used a mathematical model of this to explore the different options in the context of typical small and large molecule drugs. In this paper, we report the conclusions of our analysis and describe the analysis tool and methods used. These provide a platform to enable a more extensive enquiry into this important topic.</div>','<b>Title: </b>A quantitative atlas of Even-skipped and Hunchback expression in Clogmia albipunctata Diptera: Psychodidae blastoderm embryos<br><b>Year: </b>2014<br><b>Author: </b> H. Janssens,  K. Siggens,  D. Cicin-Sain,  E. Jimenez-Guri,  M. Musy,  M. Akam and  J. Jaeger<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/2041-9139-5-1"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>EVODEVO<br><b>Cluster-Keywords: </b><em>network models, gene network, regulatory networks, boolean networks, gene expression, systems biology, neural networks, network dynamics, gene regulatory, biological networks, network structure, gap gene, gene regulation, cell cycle, network, gene interactions, signaling network, quantum systems, system dynamics, networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Comparative studies of developmental processes are one of the main approaches to evolutionary developmental biology evo-devo. Over recent years, there has been a shift of focus from the comparative study of particular regulatory genes to the level of whole gene networks. Reverse-engineering methods can be used to computationally reconstitute and analyze the function and dynamics of such networks. These methods require quantitative spatio-temporal expression data for model fitting. Obtaining such data in non-model organisms remains a major technical challenge, impeding the wider application of data-driven mathematical modeling to evo-devo.Results: We have raised antibodies against four segmentation gene products in the moth midge Clogmia albipunctata, a non-drosophilid dipteran species. We have used these antibodies to create a quantitative atlas of protein expression patterns for the gap gene hunchback hb, and the pair-rule gene even-skipped eve. Our data reveal differences in the dynamics of Hb boundary positioning and Eve stripe formation between C. albipunctata and Drosophila melanogaster. Despite these differences, the overall relative spatial arrangement of Hb and Eve domains is remarkably conserved between these two distantly related dipteran species.Conclusions: We provide a proof of principle that it is possible to acquire quantitative gene expression data at high accuracy and spatio-temporal resolution in non-model organisms. Our quantitative data extend earlier qualitative studies of segmentation gene expression in C. albipunctata, and provide a starting point for comparative reverse-engineering studies of the evolutionary and developmental dynamics of the segmentation gene system.</div>','<b>Title: </b>Symbolic dynamics of Boolean control networks<br><b>Year: </b>2013<br><b>Author: </b> G. Hochma,  M. Margaliot,  E. Fornasini and  Me. Valcher<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.automatica.2013.05.004"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>AUTOMATICA<br><b>Cluster-Keywords: </b><em>boolean network, boolean functions, regulatory networks, canalizing functions, network models, gene regulatory, boolean control, control networks, biological networks, boolean models, update functions, interaction networks, large network, regulatory functions, canalizing boolean, genetic regulatory, systems biology, driver nodes, networks bcns, biological systems<div style="font-size:11px;"><br><b>Abstract: </b><em>We consider Boolean control networks BCNs, and in particular Boolean networks BNs, in the framework of symbolic dynamics SD. We show that the set of state-space trajectories of a BCN is a shift space of finite type SFT. This observation allows us to extend two important analysis tools from SD, namely, the Artin-Mazur zeta function and the topological entropy, to BNs and BCNs. Some of the theoretical results are illustrated using a BCN model of the core network regulating the mammalian cell cycle. C 2013 Elsevier Ltd. All rights reserved.</div>','<b>Title: </b>Membrane related dynamics and the formation of actin in cells growing on micro-topographies: a spatial computational model<br><b>Year: </b>2014<br><b>Author: </b> At. Bittig,  C. Matschegewski,  Jb. Nebe,  S. Stahlke and  Am. Uhrmacher<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/s12918-014-0106-2"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>interaction network, actin filament, network model, filament network, network structures, actin network, protein interaction, actin dynamics, network, membrane protein, network dynamics, model dynamic, cell, model, network approach, protein complex, computational model, actin, adhesion dynamics, cell dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Intra-cellular processes of cells at the interface to an implant surface are influenced significantly by their extra-cellular surrounding. Specifically, when growing osteoblasts on titanium surfaces with regular micro-ranged geometry, filaments are shorter, less aligned and they concentrate at the top of the geometric structures. Changes to the cytoskeleton network, i.e., its localization, alignment, orientation, and lengths of the filaments, as well as the overall concentration and distribution of key-actors are induced. For example, integrin is distributed homogeneously, whereas integrin in activated state and vinculin, both components of focal adhesions, have been found clustered on the micro-ranged geometries. Also, the concentration of Rho, an intracellular signaling protein related to focal adhesion regulation, was significantly lower.Results: To explore whether regulations associated with the focal adhesion complex can be responsible for the changed actin filament patterns, a spatial computational model has been developed using ML-Space, a rule-based model description language, and its associated Brownian-motion-based simulator. The focus has been on the deactivation of cofilin in the vicinity of the focal adhesion complex. The results underline the importance of sensing mechanisms to support a clustering of actin filament nucleations on the micro-ranged geometries, and of intracellular diffusion processes, which lead to spatially heterogeneous distributions of active dephosphorylated cofilin, which in turn influences the organization of the actin network. We find, for example, that the spatial heterogeneity of key molecular actors can explain the difference in filament lengths in cells on different micro-geometries partly, but to explain the full extent, further model assumptions need to be added and experimentally validated. In particular, our findings and hypothesis referring to the role, distribution, and amount of active cofilin have still to be verified in wet-lab experiments.Conclusion: Letting cells grow on surface structures is a possibility to shed new light on the intricate mechanisms that relate membrane and actin related dynamics in the cell. Our results demonstrate the need for declarative expressive spatial modeling approaches that allow probing different hypotheses, and the central role of the focal adhesion complex not only for nucleating actin filaments, but also for regulating possible severing agents locally.</div>','<b>Title: </b>Network Analysis Reveals a Common Host-Pathogen Interaction Pattern in Arabidopsis Immune Responses<br><b>Year: </b>2017<br><b>Author: </b> H. Li,  Y. Zhou and  Zd. Zhang<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3389/fpls.2017.00893"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>FRONTIERS IN PLANT SCIENCE<br><b>Cluster-Keywords: </b><em>interaction networks, proteinprotein interactions, protein interaction, network model, complex systems, systems biology, protein protein, protein complex, host network, dynamic interactions, network analysis, biological systems, networks network, cellular network, biological network, viral proteins, interactome networks, regulatory networks, immune system, systems network<div style="font-size:11px;"><br><b>Abstract: </b><em>Many plant pathogens secrete virulence effectors into host cells to target important proteins in host cellular network. However, the dynamic interactions between effectors and host cellular network have not been fully understood. Here, an integrative network analysis was conducted by combining Arabidopsis thaliana protein-protein interaction network, known targets of Pseudomonas syringae and Hyaloperonospora arabidopsidis effectors, and gene expression profiles in the immune response. In particular, we focused on the characteristic network topology of the effector targets and differentially expressed genes DEGs. We found that effectors tended to manipulate key network positions with higher betweenness centrality. The effector targets, especially those that are common targets of an individual effector, tended to be clustered together in the network. Moreover, the distances between the effector targets and DEGs increased over time during infection. In line with this observation, pathogen-susceptible mutants tended to have more DEGs surrounding the effector targets compared with resistant mutants. Our results suggest a common plant-pathogen interaction pattern at the cellular network level, where pathogens employ potent local impact mode to interfere with key positions in the host network, and plant organizes an in-depth defense by sequentially activating genes distal to the effector targets.</div>','<b>Title: </b>A variational method for analyzing limit cycle oscillations in stochastic hybrid systems<br><b>Year: </b>2018<br><b>Author: </b> Pc. Bressloff and  J. Maclaurin<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1063/1.5027077"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>CHAOS<br><b>Cluster-Keywords: </b><em>network models, population model, field model, neural population, neural field, iot networks, neuronal networks, neural network, differential equations, parameter neural, lotkavolterra model, model, neuron models, model neurons, model leads, ecosystem model, iot devices, noise model, neuronal models, inhibitory synaptic<div style="font-size:11px;"><br><b>Abstract: </b><em>Many systems in biology can be modeled through ordinary differential equations, which are piecewise continuous, and switch between different states according to a Markov jump process known as a stochastic hybrid system or piecewise deterministic Markov process PDMP. In the fast switching limit, the dynamics converges to a deterministic ODE. In this paper, we develop a phase reduction method for stochastic hybrid systems that support a stable limit cycle in the deterministic limit. A classic example is the Morris-Lecar model of a neuron, where the switching Markov process is the number of open ion channels and the continuous process is the membrane voltage. We outline a variational principle for the phase reduction, yielding an exact analytic expression for the resulting phase dynamics. We demonstrate that this decomposition is accurate over timescales that are exponential in the switching rate is an element of-1. That is, we show that for a constant C, the probability that the expected time to leave an Oa neighborhood of the limit cycle is less than T scales as Texp -Ca/is an element of. Published by AIP Publishing.</div>','<b>Title: </b>Multiple feedback loops are key to a robust dynamic performance of tryptophan regulation in Escherichia coli<br><b>Year: </b>2004<br><b>Author: </b> Kv. Venkatesh,  S. Bhartiya and  A. Ruhela<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/S0014-57930400310-2"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>FEBS LETTERS<br><b>Cluster-Keywords: </b><em>systems biology, regulatory network, interaction networks, network model, gene regulatory, network, network biology, networks, gene network, response networks, network dynamics, dynamic model, cell networks, gene expression, biological networks, system dynamics, dynamic network, model, biological systems, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Living systems must adapt quickly and stably to uncertain environments. A common theme in cellular regulation is the presence of multiple feedback loops in the network. An example of such a feedback structure is regulation of tryptophan concentration in Escherichia coli. Here, three distinct feedback mechanisms, namely genetic regulation, mRNA attenuation and enzyme inhibition, regulate tryptophan synthesis. A pertinent question is whether such multiple feedback loops are "a case of regulatory overkill, or do these different feedback regulators have distinct functions" Freeman 2000 Nature 295, 313-319. Another moot question is how robustness to uncertainties can be achieved structurally through biological interactions. Correlation between the feedback structure and robustness can be systematically studied by tools commonly employed in feedback theory. An analysis of feedback strategies in the tryptophan system in E. coli reveals that the network complexity arising due to the distributed feedback structure is responsible for the rapid and stable response observed even in the presence of system uncertainties. C 2004 Published by Elsevier B.V. on behalf of the Federation of European Biochemical Societies.</div>','<b>Title: </b>Coupled feedback loops control the stimulus-dependent dynamics of the yeast transcription factor Msn2<br><b>Year: </b>2017<br><b>Author: </b> Yf. Jiang,  Z. Akhavanaghdam,  Ls. Tsimring and  N. Hao<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1074/jbc.C117.800896"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF BIOLOGICAL CHEMISTRY<br><b>Cluster-Keywords: </b><em>circadian rhythms, systems biology, biological clock, circadian clock, biological rhythms, hpa axis, model, clock network, mathematical models, biological systems, dynamic systems, ode models, biology models, single cell, network, model parameters, experimental data, models, systems, dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>Information about environmental stimuli often can be encoded by the dynamics of signaling molecules or transcription factors. In the yeast Saccharomyces cerevisiae, different types of stresses induce distinct nuclear translocation dynamics of the general stress-responsive transcription factor Msn2, but the underlying mechanisms remain unclear. Using deterministic and stochastic modeling, we reproduced in silico the different dynamic responses of Msn2 to glucose limitation and osmotic stress observed in vivo and found that a positive feedback loop on protein kinase A mediated by the AMP-activated protein kinase Snf1 is coupled with a negative feedback loop to generate the characteristic pulsatile dynamics of Msn2. The model predicted that the stimulus-specific positive feedback loop could be responsible for the difference between Msn2 dynamics induced by glucose limitation and osmotic stress. This prediction was further verified experimentally by time-lapse microscopic examinations of the snf1 strain. In this mutant lacking the Snf1-mediated positive feedback loop, Msn2 responds similarly to glucose limitation and osmotic stress, and its pulsatile translocation is largely abrogated. Our combined computational and experimental analysis reveals a regulatory mechanism by which cells can encode information about environmental cues into distinct signaling dynamics through stimulus-specific network architectures.</div>','<b>Title: </b>Prior knowledge driven Granger causality analysis on gene regulatory network discovery<br><b>Year: </b>2015<br><b>Author: </b> S. Yao,  S. Yoo and  Dt. Yu<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/s12859-015-0710-1"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>regulatory networks, gene regulatory, gene expression, network inference, expression data, network structure, gene networks, networks grns, regulatory genes, boolean networks, bayesian network, network reconstruction, networks gene, network dynamics, gene regulation, inference method, systems biology, time series, infer gene, network<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Our study focuses on discovering gene regulatory networks from time series gene expression data using the Granger causality GC model. However, the number of available time points T usually is much smaller than the number of target genes n in biological datasets. The widely applied pairwise GC model PGC and other regularization strategies can lead to a significant number of false identifications when n >> T.Results: In this study, we proposed a new method, viz., CGC-2SPR CGC using two-step prior Ridge regularization to resolve the problem by incorporating prior biological knowledge about a target gene data set. In our simulation experiments, the propose new methodology CGC-2SPR showed significant performance improvement in terms of accuracy over other widely used GC modeling PGC, Ridge and Lasso and MI-based MRNET and ARACNE methods. In addition, we applied CGC-2SPR to a real biological dataset, i.e., the yeast metabolic cycle, and discovered more true positive edges with CGC-2SPR than with the other existing methods.Conclusions: In our research, we noticed a "1  1 > 2" effect when we combined prior knowledge and gene expression data to discover regulatory networks. Based on causality networks, we made a functional prediction that the Abm1 gene its functions previously were unknown might be related to the yeasts responses to different levels of glucose. Our research improves causality modeling by combining heterogeneous knowledge, which is well aligned with the future direction in system biology. Furthermore, we proposed a method of Monte Carlo significance estimation MCSE to calculate the edge significances which provide statistical meanings to the discovered causality networks. All of our data and source codes will be available under the link https://bitbucket.org/dtyu/grangercausality/wiki/Home.</div>','<b>Title: </b>Identification of models of heterogeneous cell populations from population snapshot data<br><b>Year: </b>2011<br><b>Author: </b> J. Hasenauer,  S. Waldherr,  M. Doszczak,  N. Radde,  P. Scheurich and  F. Allgower<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1471-2105-12-125"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>reaction networks, network model, systems biology, biochemical reaction, stochastic reaction, model reduction, stochastic model, network dynamics, stochastic biochemical, stochastic simulation, system size, chemical reaction, gene network, elementary reactions, biochemical networks, parameter identification, parameter inference, cell populations, small system, linear noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Most of the modeling performed in the area of systems biology aims at achieving a quantitative description of the intracellular pathways within a "typical cell". However, in many biologically important situations even clonal cell populations can show a heterogeneous response. These situations require study of cell-to-cell variability and the development of models for heterogeneous cell populations.Results: In this paper we consider cell populations in which the dynamics of every single cell is captured by a parameter dependent differential equation. Differences among cells are modeled by differences in parameters which are subject to a probability density. A novel Bayesian approach is presented to infer this probability density from population snapshot data, such as flow cytometric analysis, which do not provide single cell time series data. The presented approach can deal with sparse and noisy measurement data. Furthermore, it is appealing from an application point of view as in contrast to other methods the uncertainty of the resulting parameter distribution can directly be assessed.Conclusions: The proposed method is evaluated using artificial experimental data from a model of the tumor necrosis factor signaling network. We demonstrate that the methods are computationally efficient and yield good estimation result even for sparse data sets.</div>','<b>Title: </b>An Iterative Genetic and Dynamical Modelling Approach Identifies Novel Features of the Gene Regulatory Network Underlying Melanocyte Development<br><b>Year: </b>2011<br><b>Author: </b> Er. Greenhill,  A. Rocco,  L. Vibert,  M. Nikaido and  Rn. Kelsh<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pgen.1002265"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS GENETICS<br><b>Cluster-Keywords: </b><em>gene regulatory, regulatory network, systems biology, cell differentiation, cell fate, model systems, potential landscape, network grn, dynamical systems, melanocyte differentiation, network models, gene expression, network dynamics, differentiation genes, dynamical models, dynamical networks, cellular differentiation, stem cells, attractor network, developmental gene<div style="font-size:11px;"><br><b>Abstract: </b><em>The mechanisms generating stably differentiated cell-types from multipotent precursors are key to understanding normal development and have implications for treatment of cancer and the therapeutic use of stem cells. Pigment cells are a major derivative of neural crest stem cells and a key model cell-type for our understanding of the genetics of cell differentiation. Several factors driving melanocyte fate specification have been identified, including the transcription factor and master regulator of melanocyte development, Mitf, and Wnt signalling and the multipotency and fate specification factor, Sox10, which drive mitf expression. While these factors together drive multipotent neural crest cells to become specified melanoblasts, the mechanisms stabilising melanocyte differentiation remain unclear. Furthermore, there is controversy over whether Sox10 has an ongoing role in melanocyte differentiation. Here we use zebrafish to explore in vivo the gene regulatory network GRN underlying melanocyte specification and differentiation. We use an iterative process of mathematical modelling and experimental observation to explore methodically the core melanocyte GRN we have defined. We show that Sox10 is not required for ongoing differentiation and expression is downregulated in differentiating cells, in response to Mitfa and Hdac1. Unexpectedly, we find that Sox10 represses Mitf-dependent expression of melanocyte differentiation genes. Our systems biology approach allowed us to predict two novel features of the melanocyte GRN, which we then validate experimentally. Specifically, we show that maintenance of mitfa expression is Mitfa-dependent, and identify Sox9b as providing an Mitfa-independent input to melanocyte differentiation. Our data supports our previous suggestion that Sox10 only functions transiently in regulation of mitfa and cannot be responsible for long-term maintenance of mitfa expression; indeed, Sox10 is likely to slow melanocyte differentiation in the zebrafish embryo. More generally, this novel approach to understanding melanocyte differentiation provides a basis for systematic modelling of differentiation in this and other cell-types.</div>','<b>Title: </b>Identifying mechanisms of regulation to model carbon flux during heat stress and generate testable hypotheses<br><b>Year: </b>2018<br><b>Author: </b> Ah. Hubbard,  Xk. Zhang,  S. Jastrebski,  Sj. Lamont,  A. Singh and  Cj. Schmidt<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0205824"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>gene expression, systems biology, metabolic network, network model, network analysis, dynamic model, signaling pathways, signaling network, regulatory network, dynamic network, biology approach, candidate genes, gene network, gene regulatory, biology analysis, mathematical model, network dynamic, biology network, metabolic pathways, pathway analysis<div style="font-size:11px;"><br><b>Abstract: </b><em>Understanding biological response to stimuli requires identifying mechanisms that coordinate changes across pathways. One of the promises of multi-omics studies is achieving this level of insight by simultaneously identifying different levels of regulation. However, computational approaches to integrate multiple types of data are lacking. An effective systems biology approach would be one that uses statistical methods to detect signatures of relevant network motifs and then builds metabolic circuits from these components to model shifting regulatory dynamics. For example, transcriptome and metabolome data complement one another in terms of their ability to describe shifts in physiology. Here, we extend a previously described linear-modeling based method used to identify single nucleotide polymorphisms SNPs associated with metabolic changes. We apply this strategy to link changes in sulfur, amino acid and lipid production under heat stress by relating ratios of compounds to potential precursors and regulators. This approach provides integration of multi-omics data to link previously described, discrete units of regulation into functional pathways and identifies novel biology relevant to the heat stress response, in addition to generating hypotheses.</div>','<b>Title: </b>Network-Based Analysis of Bortezomib Pharmacodynamic Heterogeneity in Multiple Myeloma Cells<br><b>Year: </b>2018<br><b>Author: </b> V. Ramakrishnan and  De. Mager<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1124/jpet.118.247924"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF PHARMACOLOGY AND EXPERIMENTAL THERAPEUTICS<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>The objective of this study is to evaluate the heterogeneity in pharmacodynamic response in four in vitro multiple myeloma cell lines to treatment with bortezomib, and to assess whether such differences are associated with drug-induced intracellular signaling protein dynamics identified via a logic-based network modeling approach. The in vitro pharmacodynamic-efficacy of bortezomib was evaluated through concentration-effect and cell proliferation dynamical studies in U266, RPMI8226, MM.1S, and NCI-H929 myeloma cell lines. A Boolean logic-based network model incorporating intracellular protein signaling pathways relevant to myeloma cell growth, proliferation, and apoptosis was developed based on information available in the literature and used to identify key proteins regulating bortezomib pharmacodynamics. The time-course of network-identified proteins was measured using the MAGPIX protein assay system. Traditional pharmacodynamic modeling endpoints revealed variable responses of the cell lines to bortezomib treatment, classifying cell lines as more sensitive MM.1S and NCI-H929 and less sensitive U266 and RPMI8226. Network centrality and model reduction identified key proteins e.g., phosphorylated nuclear factor-KB, phosphorylated protein kinase B, phosphorylated mechanistic target of rapamycin, Bcl-2, phosphorylated c-Jun N-terminal kinase, phosphorylated p53, p21, phosphorylated Bcl-2-associated death promoter, caspase 8, and caspase 9 that govern bortezomib pharmacodynamics. The corresponding relative expression normalized to 0-hour untreated-control cells of proteins demonstrated a greater magnitude and earlier onset of stimulation/inhibition in cells more sensitive MM.1 S and NCI-H929 to bortezomib-induced cell death at 20 nM, relative to the less sensitive cells U266 and RPMI8226. Overall, differences in intracellular signaling appear to be associated with bortezomib pharmacodynamic heterogeneity, and key proteins may be potential biomarkers to evaluate bortezomib responses.</div>','<b>Title: </b>Metabolic Feedback Circuits Provide Rapid Control of Metabolite Dynamics<br><b>Year: </b>2018<br><b>Author: </b> D. Liu and  Fz. Zhang<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1021/acssynbio.7b00342"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ACS SYNTHETIC BIOLOGY<br><b>Cluster-Keywords: </b><em>gene circuits, synthetic biology, circuit behavior, biological parts, synthetic gene, circuit design, genetic circuits, cell types, metabolite dynamics, mathematical model, designed circuit, control circuit, biological systems, circuit models, synthetic circuit, feedback circuits, biological circuits, design models, metabolic pathways, circuit parameters<div style="font-size:11px;"><br><b>Abstract: </b><em>Metabolism constitutes the basis of life, and the dynamics of metabolism dictate various cellular processes. However, exactly how metabolite dynamics are controlled remains poorly understood. By studying an engineered fatty acid-producing pathway as a model, we found that upon transcription activation a metabolic product from an unregulated pathway required seven cell cycles to reach to its steady state level, with the speed mostly limited by enzyme expression dynamics. To overcome this limit, we designed metabolic feedback circuits MeFCs with three different architectures, and experimentally measured and modeled their metabolite dynamics. Our engineered MeFCs could dramatically shorten the rise-time of metabolites, decreasing it by as much as 12-fold. The findings of this study provide a systematic understanding of metabolite dynamics in different architectures of MeFCs and have potentially immense applications in designing synthetic circuits to improve the productivities of engineered metabolic pathways.</div>','<b>Title: </b>Automatic Screening for Perturbations in Boolean Networks<br><b>Year: </b>2018<br><b>Author: </b> Jd. Schwab and  Ha. Kestler<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3389/fphys.2018.00431"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>FRONTIERS IN PHYSIOLOGY<br><b>Cluster-Keywords: </b><em>boolean network, network models, qualitative models, systems biology, mathematical modeling, boolean models, signaling networks, dynamic models, dynamic modeling, signaling dynamics, logic models, dynamic network, mathematical models, boolean modeling, signal transduction, regulatory networks, drug resistance, biological networks, network approach, network modeling<div style="font-size:11px;"><br><b>Abstract: </b><em>A common approach to address biological questions in systems biology is to simulate regulatory mechanisms using dynamic models. Among others, Boolean networks can be used to model the dynamics of regulatory processes in biology. Boolean network models allow simulating the qualitative behavior of the modeled processes. A central objective in the simulation of Boolean networks is the computation of their long-term behavior-so-called attractors. These attractors are of special interest as they can often be linked to biologically relevant behaviors. Changing internal and external conditions can influence the long-term behavior of the Boolean network model. Perturbation of a Boolean network by stripping a component of the system or simulating a surplus of another element can lead to different attractors. Apparently, the number of possible perturbations and combinations of perturbations increases exponentially with the size of the network. Manually screening a set of possible components for combinations that have a desired effect on the long-term behavior can be very time consuming if not impossible. We developed a method to automatically screen for perturbations that lead to a user-specified change in the networks functioning. This method is implemented in the visual simulation framework ViSiBool utilizing satisfiability SAT solvers for fast exhaustive attractor search.</div>','<b>Title: </b>Computing enclosures for uncertain biochemical systems<br><b>Year: </b>2012<br><b>Author: </b> E. August and  H. Koeppl<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1049/iet-syb.2011.0091"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IET SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>systems biology, biological systems, system dynamics, molecular systems, experimental design, network model, model system, model parameters, experiment design, kinetic parameters, perfect adaptation, experimental data, molecular mechanisms, homeostatic mechanisms, switching dynamics, biological mechanisms, control systems, chemotaxis network, decentralized control, feedback control<div style="font-size:11px;"><br><b>Abstract: </b><em>In this study, the authors present a novel method that provides enclosures for state trajectories of a non-linear dynamical system with uncertainties in initial conditions and parameter values. It is based on solving positivity conditions by means of semi-definite programmes and sum of squares decompositions. The method accounts for the indeterminacy of kinetic parameters, measurement uncertainties and fluctuations in the reaction rates because of extrinsic noise. This is particularly useful in the field of systems biology when one seeks to determine model behaviour quantitatively or, if this is not possible, semi-quantitatively. The authors also demonstrate the significance of the proposed method to model selection in biology. The authors illustrate the applicability of their method on the mitogen-activated protein kinase signalling pathway, which is an important and reoccurring network motif that apparently also plays a crucial role in the development of cancer.</div>','<b>Title: </b>Dynamics of cellular level function and regulation derived from murine expression array data<br><b>Year: </b>2004<br><b>Author: </b> B. De Bivort,  S. Huang and  Y. Bar-Yam<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1073/pnas.0406707102"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA<br><b>Cluster-Keywords: </b><em>gene regulatory, gene expression, regulatory networks, expression data, regulatory pathway, regulatory genes, network dynamics, microarray data, genetic networks, transcription factor, models network, target gene, upstream regulatory, network topology, pathway network, gene functional, expression profile, transcriptional network, dynamic model, transcriptional regulatory<div style="font-size:11px;"><br><b>Abstract: </b><em>A major open question of systems biology is how genetic and molecular components interact to create phenotypes at the cellular level. Although much recent effort has been dedicated to inferring effective regulatory influences within small networks of genes, the power of microarray bioinformatics has yet to be used to determine functional influences at the cellular level. In all cases of data-driven parameter estimation, the number of model parameters estimable from a set of data is strictly limited by the size of that set. Rather than infer parameters describing the detailed interactions of just a few genes, we chose a larger-scale investigation so that the cumulative effects of all gene interactions could be analyzed to identify the dynamics of cellular-level function. By aggregating genes into large groups with related behaviors megamodules, we were able to determine the effective aggregate regulatory influences among 12 major gene groups in murine B lymphocytes over a variety of time steps. Intriguing observations about the behavior of cells at this high level of abstraction include: i a medium-term critical global transcriptional dependence on ATIP-generating genes in the mitochondria, it a longer-term dependence on glycolytic genes, iii the dual role of chromatin-reorganizing genes in transcriptional activation and repression, iv homeostasis-favoring influences, v the indication that, as a group, G protein-mediated signals are not concentration-dependent in their influence on target gene expression, and vi shortterm -activating/long-term-repressing behavior of the cell-cycle system that reflects its oscillatory behavior.</div>','<b>Title: </b>Modeling of Eicosanoid Fluxes Reveals Functional Coupling between Cyclooxygenases and Terminal Synthases<br><b>Year: </b>2014<br><b>Author: </b> Y. Kihara,  S. Gupta,  Mr. Maurya,  A. Armando,  I. Shah,  O. Quehenberger,  Ck. Glass,  Ea. Dennis and  S. Subramaniam<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.bpj.2014.01.015"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOPHYSICAL JOURNAL<br><b>Cluster-Keywords: </b><em>gene expression, systems biology, metabolic network, network model, network analysis, dynamic model, signaling pathways, signaling network, regulatory network, dynamic network, biology approach, candidate genes, gene network, gene regulatory, biology analysis, mathematical model, network dynamic, biology network, metabolic pathways, pathway analysis<div style="font-size:11px;"><br><b>Abstract: </b><em>Eicosanoids, including prostaglandins PG and leukotrienes, are lipid mediators derived from arachidonic acid. A quantitative and biochemical level understanding of eicosanoid metabolism would aid in understanding the mechanisms that govern inflammatory processes. Here, we present a combined experimental and computational approach to understanding the biochemical basis of eicosanoid metabolism in macrophages. Lipidomic and transcriptomic measurements and analyses reveal temporal and dynamic changes of the eicosanoid metabolic network in mouse bone marrow-derived macrophages BMDM upon stimulation of the Toll-like receptor 4 with Kdo2-Lipid A KLA and stimulation of the P2X7 purinergic receptor with adenosine 5-triphosphate. Kinetic models were developed for the cyclooxygenase COX and lipoxygenase branches of arachidonic acid metabolism, and then the rate constants were estimated with a data set from ATP-stimulated BMDM, using a two-step matrix-based approach employing a constrained least-squares method followed by nonlinear optimization. The robustness of the model was validated through parametric sensitivity, uncertainty analysis, and predicting an independent dataset from KLA-primed ATP-stimulated BMDM by allowing the parameters to vary within the uncertainty range of the calculated parameters. We analyzed the functional coupling between COX isozymes and terminal enzymes by developing a PGH2-divided model. This provided evidence for the functional coupling between COX-2 and PGE2 synthase, between COX-1/COX-2 and PGD2 synthase, and also between COX-1 and thronnboxane A2 synthase. Further, these functional couplings were experimentally validated using COX-1 and COX-2 selective inhibitors. The resulting fluxomics analysis demonstrates that the "multi-omics" systems biology approach can define the complex machinery of eicosanoid networks.</div>','<b>Title: </b>Modeling the Pro-inflammatory Tumor Microenvironment in Acute Lymphoblastic Leukemia Predicts a Breakdown of Hematopoietic-Mesenchymal Communication Networks<br><b>Year: </b>2016<br><b>Author: </b> J. Enciso,  H. Mayani,  L. Mendoza and  R. Pelayo<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3389/fphys.2016.00349"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>FRONTIERS IN PHYSIOLOGY<br><b>Cluster-Keywords: </b><em>network ecology, food webs, network models, ecological systems, food web, systems biology, ecological networks, ecosystem model, signalling networks, model results, interaction networks, dynamic model, ecological interactions, network analysis, model network, network structure, transport networks, systems network, ecosystem dynamics, restoration ecology<div style="font-size:11px;"><br><b>Abstract: </b><em>Lineage fate decisions of hematopoietic cells depend on intrinsic factors and extrinsic signals provided by the bone marrow microenvironment, where they reside. Abnormalities in composition and function of hematopoietic niches have been proposed as key contributors of acute lymphoblastic leukemia ALL progression. Our previous experimental findings strongly suggest that pro-inflammatory cues contribute to mesenchymal niche abnormalities that result in maintenance of ALL precursor cells at the expense of normal hematopoiesis. Here, we propose a molecular regulatory network interconnecting the major communication pathways between hematopoietic stem and progenitor cells HSPCs and mesenchymal stromal cells MSCs within the BM. Dynamical analysis of the network as a Boolean model reveals two stationary states that can be interpreted as the intercellular contact status. Furthermore, simulations describe the molecular patterns observed during experimental proliferation and activation. Importantly, our model predicts instability in the CXCR4/CXCL12 and VLA4/VCAM1 interactions following microenvironmental perturbation due by temporal signaling from Toll like receptors TLRs ligation. Therefore, aberrant expression of NF-kappa B induced by intrinsic or extrinsic factors may contribute to create a tumor microenvironment where a negative feedback loop inhibiting CXCR4/CXCL12 and VLA4NCAM1 cellular communication axes allows for the maintenance of malignant cells.</div>','<b>Title: </b>Modeling general proteostasis: proteome balance in health and disease<br><b>Year: </b>2011<br><b>Author: </b> Dm. Roth and  We. Balch<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.ceb.2010.11.001"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>CURRENT OPINION IN CELL BIOLOGY<br><b>Cluster-Keywords: </b><em>normal mode, mode analysis, elastic network, functional motions, functional dynamics, molecular chaperone, structural biology, acidic residues, network models, molecular dynamics, atomic resolution, protein complexes, conserved functional, hsp dynamics, correlated motions, protein folding, large biomolecular, thioredoxin system, harmonic analysis, protein motions<div style="font-size:11px;"><br><b>Abstract: </b><em>Protein function is generated and maintained by the proteostasis network PN Balch et al. 2008 Science, 319:916. The PN is a modular, yet integrated system unique to each cell type that is sensitive to signaling pathways that direct development and aging, and respond to folding stress. Mismanagement of protein folding and function triggered by genetic, epigenetic and environmental causes poses a major challenge to human health and lifespan. Herein, we address the impact of proteostasis defined by the FoldFx model on our understanding of protein folding and function in biology. FoldFx describes how general proteostasis control GPC enables the polypeptide chain sequence to achieve functional balance in the context of the cellular proteome. By linking together the chemical and energetic properties of the protein fold with the composition of the PN we discuss the principle of the proteostasis boundary PB as a key component of GPC. The curved surface of the PB observed in 3-dimensional space suggests that the polypeptide chain sequence and the PN operate as an evolutionarily conserved functional unit to generate and sustain protein dynamics required for biology. Modeling general proteostasis provides a rational basis for tackling some of the most challenging diseases facing mankind in the 21st century.</div>','<b>Title: </b>Model-Based Analysis for Qualitative Data: An Application in Drosophila Germline Stem Cell Regulation<br><b>Year: </b>2014<br><b>Author: </b> M. Pargett,  Ae. Rundell,  Gt. Buzzard and  Dm. Umulis<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pcbi.1003498"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS COMPUTATIONAL BIOLOGY<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Discovery in developmental biology is often driven by intuition that relies on the integration of multiple types of data such as fluorescent images, phenotypes, and the outcomes of biochemical assays. Mathematical modeling helps elucidate the biological mechanisms at play as the networks become increasingly large and complex. However, the available data is frequently under-utilized due to incompatibility with quantitative model tuning techniques. This is the case for stem cell regulation mechanisms explored in the Drosophila germarium through fluorescent immunohistochemistry. To enable better integration of biological data with modeling in this and similar situations, we have developed a general parameter estimation process to quantitatively optimize models with qualitative data. The process employs a modified version of the Optimal Scaling method from social and behavioral sciences, and multi-objective optimization to evaluate the trade-off between fitting different datasets e. g. wild type vs. mutant. Using only published imaging data in the germarium, we first evaluated support for a published intracellular regulatory network by considering alternative connections of the same regulatory players. Simply screening networks against wild type data identified hundreds of feasible alternatives. Of these, five parsimonious variants were found and compared by multi-objective analysis including mutant data and dynamic constraints. With these data, the current model is supported over the alternatives, but support for a biochemically observed feedback element is weak i.e. these data do not measure the feedback effect well. When also comparing new hypothetical models, the available data do not discriminate. To begin addressing the limitations in data, we performed a model-based experiment design and provide recommendations for experiments to refine model parameters and discriminate increasingly complex hypotheses.</div>','<b>Title: </b>Systems analysis of cellular networks under uncertainty<br><b>Year: </b>2009<br><b>Author: </b> Hm. Kaltenbach,  S. Dimopoulos and  J. Stelling<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.febslet.2009.10.074"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>FEBS LETTERS<br><b>Cluster-Keywords: </b><em>reaction networks, reaction systems, metabolic networks, chemical reaction, network structure, systems biology, biological systems, biological networks, biochemical reaction, reaction rates, differential equations, reaction graph, complex networks, ssi modules, networks, life systems, ordinary differential, cellular networks, reaction, network flows<div style="font-size:11px;"><br><b>Abstract: </b><em>Besides the often-quoted complexity of cellular networks, the prevalence of uncertainties about components, interactions, and their quantitative features provides a largely underestimated hallmark of current systems biology. This uncertainty impedes the development of mechanistic mathematical models to achieve a true systems-level understanding. However, there is increasing evidence that theoretical approaches from diverse scientific domains can extract relevant biological knowledge efficiently, even from poorly characterized biological systems. As a common denominator, the methods focus on structural, rather than more detailed, kinetic network properties. A deeper understanding, better scaling, and the ability to combine the approaches pose formidable challenges for future theory developments.c 2009 Federation of European Biochemical Societies. Published by Elsevier B.V. All rights reserved.</div>','<b>Title: </b>Global Optimization, Local Adaptation, and the Role of Growth in Distribution Networks<br><b>Year: </b>2016<br><b>Author: </b> H. Ronellenfitsch and  E. Katifori<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1103/PhysRevLett.117.138301"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PHYSICAL REVIEW LETTERS<br><b>Cluster-Keywords: </b><em>network ecology, food webs, network models, ecological systems, food web, systems biology, ecological networks, ecosystem model, signalling networks, model results, interaction networks, dynamic model, ecological interactions, network analysis, model network, network structure, transport networks, systems network, ecosystem dynamics, restoration ecology<div style="font-size:11px;"><br><b>Abstract: </b><em>Highly optimized complex transport networks serve crucial functions in many man-made and natural systems such as power grids and plant or animal vasculature. Often, the relevant optimization functional is nonconvex and characterized by many local extrema. In general, finding the global, or nearly global optimum is difficult. In biological systems, it is believed that such an optimal state is slowly achieved through natural selection. However, general coarse grained models for flow networks with local positive feedback rules for the vessel conductivity typically get trapped in low efficiency, local minima. In this work we show how the growth of the underlying tissue, coupled to the dynamical equations for network development, can drive the system to a dramatically improved optimal state. This general model provides a surprisingly simple explanation for the appearance of highly optimized transport networks in biology such as leaf and animal vasculature.</div>','<b>Title: </b>Bayesian Parameter Inference and Model Selection by Population Annealing in Systems Biology<br><b>Year: </b>2014<br><b>Author: </b> Y. Murakami<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0104057"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Parameter inference and model selection are very important for mathematical modeling in systems biology. Bayesian statistics can be used to conduct both parameter inference and model selection. Especially, the framework named approximate Bayesian computation is often used for parameter inference and model selection in systems biology. However, Monte Carlo methods needs to be used to compute Bayesian posterior distributions. In addition, the posterior distributions of parameters are sometimes almost uniform or very similar to their prior distributions. In such cases, it is difficult to choose one specific value of parameter with high credibility as the representative value of the distribution. To overcome the problems, we introduced one of the population Monte Carlo algorithms, population annealing. Although population annealing is usually used in statistical mechanics, we showed that population annealing can be used to compute Bayesian posterior distributions in the approximate Bayesian computation framework. To deal with un-identifiability of the representative values of parameters, we proposed to run the simulations with the parameter ensemble sampled from the posterior distribution, named "posterior parameter ensemble. We showed that population annealing is an efficient and convenient algorithm to generate posterior parameter ensemble. We also showed that the simulations with the posterior parameter ensemble can, not only reproduce the data used for parameter inference, but also capture and predict the data which was not used for parameter inference. Lastly, we introduced the marginal likelihood in the approximate Bayesian computation framework for Bayesian model selection. We showed that population annealing enables us to compute the marginal likelihood in the approximate Bayesian computation framework and conduct model selection depending on the Bayes factor.</div>','<b>Title: </b>Modeling the therapeutic efficacy of NF kappa B synthetic decoy oligodeoxynucleotides ODNs<br><b>Year: </b>2018<br><b>Author: </b> Zp. Wang,  Da. Potoyan and  Pg. Wolynes<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/s12918-018-0525-6"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Transfection of NF kappa B synthetic decoy Oligodeoxynucleotides ODNs has been proposed as a promising therapeutic strategy for a variety of diseases arising from constitutive activation of the eukaryotic transcription factor NF kappa B. The decoy approach faces some limitations under physiological conditions notably nuclease-induced degradation.Results: In this work, we show how a systems pharmacology model of NF kappa B regulatory networks displaying oscillatory temporal dynamics, can be used to predict quantitatively the dependence of therapeutic efficacy of NF kappa B synthetic decoy ODNs on dose, unbinding kinetic rates and nuclease-induced degradation rates. Both deterministic mass action simulations and stochastic simulations of the systems biology model show that the therapeutic efficacy of synthetic decoy ODNs is inversely correlated with unbinding kinetic rates, nuclease-induced degradation rates and molecular stripping rates, but is positively correlated with dose. We show that the temporal coherence of the stochastic dynamics of NF kappa B regulatory networks is most sensitive to adding NF kappa B synthetic decoy ODNs having unbinding time-scales that are in-resonance with the time-scale of the limit cycle of the network.Conclusions: The pharmacokinetics/pharmacodynamics PK/PD predicted by the systems-level model should provide quantitative guidance for in-depth translational research of optimizing the thermodynamics/kinetic properties of synthetic decoy ODNs.</div>','<b>Title: </b>Gene Expression Ratios Lead to Accurate and Translatable Predictors of DR5 Agonism across Multiple Tumor Lineages<br><b>Year: </b>2015<br><b>Author: </b> A. Reddy,  Jd. Growney,  Ns. Wilson,  Cm. Emery,  Ja. Johnson,  R. Ward,  Ka. Monaco,  J. Korn,  Je. Monahan,  Md. Stump,  Fa. Mapa,  Cj. Wilson,  J. Steiger,  J. Ledell,  Rj. Rickles,  Ve. Myer,  Sa. Ettenberg,  R. Schlegel,  Wr. Sellers,  Ha. Huet and  J. Lehar<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0138486"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Death Receptor 5 DR5 agonists demonstrate anti-tumor activity in preclinical models but have yet to demonstrate robust clinical responses. A key limitation may be the lack of patient selection strategies to identify those most likely to respond to treatment. To overcome this limitation, we screened a DR5 agonist Nanobody across >600 cell lines representing 21 tumor lineages and assessed molecular features associated with response. High expression of DR5 and Casp8 were significantly associated with sensitivity, but their expression thresholds were difficult to translate due to low dynamic ranges. To address the translational challenge of establishing thresholds of gene expression, we developed a classifier based on ratios of genes that predicted response across lineages. The ratio classifier outperformed the DR5Casp8 classifier, as well as standard approaches for feature selection and classification using genes, instead of ratios. This classifier was independently validated using 11 primary patient-derived pancreatic xenograft models showing perfect predictions as well as a striking linearity between prediction probability and anti-tumor response. A network analysis of the genes in the ratio classifier captured important biological relationships mediating drug response, specifically identifying key positive and negative regulators of DR5 mediated apoptosis, including DR5, CASP8, BID, cFLIP, XIAP and PEA15. Importantly, the ratio classifier shows translatability across gene expression platforms from Affymetrix microarrays to RNA-seq and across model systems in vitro to in vivo. Our approach of using gene expression ratios presents a robust and novel method for constructing translatable biomarkers of compound response, which can also probe the underlying biology of treatment response.</div>','<b>Title: </b>Inferring the parameters of a Markov process from snapshots of the steady state<br><b>Year: </b>2018<br><b>Author: </b> Sl. Dettmer and  J. Berg<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1088/1742-5468/aaa8ea"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF STATISTICAL MECHANICS-THEORY AND EXPERIMENT<br><b>Cluster-Keywords: </b><em>boolean networks, regulatory networks, network model, bayesian networks, model networks, systems biology, random boolean, real cells, immune system, genetic regulatory, networks, biochemical network, fluorescence images, cellular network, gene regulatory, cell differentiation, network, biological systems, ensemble approach, cell dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>We seek to infer the parameters of an ergodic Markov process from samples taken independently from the steady state. Our focus is on non-equilibrium processes, where the steady state is not described by the Boltzmann measure, but is generally unknown and hard to compute, which prevents the application of established equilibrium inference methods. We propose a quantity we call propagator likelihood, which takes on the role of the likelihood in equilibrium processes. This propagator likelihood is based on fictitious transitions between those configurations of the system which occur in the samples. The propagator likelihood can be derived by minimising the relative entropy between the empirical distribution and a distribution generated by propagating the empirical distribution forward in time. Maximising the propagator likelihood leads to an efficient reconstruction of the parameters of the underlying model in different systems, both with discrete configurations and with continuous configurations. We apply the method to non-equilibrium models from statistical physics and theoretical biology, including the asymmetric simple exclusion process ASEP, the kinetic Ising model, and replicator dynamics.</div>','<b>Title: </b>Dynamics and Control at Feedback Vertex Sets. I: Informative and Determining Nodes in Regulatory Networks<br><b>Year: </b>2013<br><b>Author: </b> B. Fiedler,  A. Mochizuki,  G. Kurosawa and  D. Saito<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s10884-013-9312-7"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF DYNAMICS AND DIFFERENTIAL EQUATIONS<br><b>Cluster-Keywords: </b><em>oscillator networks, network dynamics, network, networks, dynamical systems, dynamics, node dynamics, systems, coupled oscillator, synchronization, dynamics network, complex network, coupled system, network modifications, oscillators, network connectivity, feedback vertex, network topology, synchronization properties, collective dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>We consider systems of differential equations which model complex regulatory networks by a graph structure of dependencies. We show that the concepts of informative nodes Mochizuki and Saito, J Theor Biol 266:323-335, 2010 and determining nodes Foias and Temam, Math Comput 43:117-133, 1984 coincide with the notion of feedback vertex sets from graph theory. As a result we can determine the long-time dynamics of the entire network from observations on only a feedback vertex set. We also indicate how open loop control at a feedback vertex set, only, forces the remaining network to stably follow prescribed stable or unstable trajectories. We present three examples of biological networks which motivated this work: a specific gene regulatory network of ascidian cell differentiation Imai et al., Science 312:1183-1187, 2006, a signal transduction network involving the epidermal growth factor in mammalian cells Oda et al., Mol Syst Biol 1:1-17, 2005, and a mammalian gene regulatory network of circadian rhythms Mirsky et al., Proc Natl Acad Sci USA 106:11107-11112, 2009. In each example the required observation set is much smaller than the entire network. For further details on biological aspects see the companion paper Mochizuki et al., J Theor Biol, 2013, in press. The mathematical scope of our approach is not limited to biology. Therefore we also include many further examples to illustrate and discuss the broader mathematical aspects.</div>','<b>Title: </b>Computational inference of replication and transcription activator regulator activity in herpesvirus from gene expression data<br><b>Year: </b>2008<br><b>Author: </b> A. Recchia,  E. Wit,  V. Vinciotti and  P. Kellam<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1049/iet-syb:20070053"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IET SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>information systems, gene expression, gene network, systems theory, system models, systems biology, systems information, network model, genetic network, mathematical models, dynamic systems, dynamic network, systemic model, network topology, information mechanisms, complex systems, systems complex, relational models, models main, information model<div style="font-size:11px;"><br><b>Abstract: </b><em>One of the main aims of system biology is to understand the structure and dynamics of genomic systems. A computational approach, facilitated by new technologies for high-throughput quantitative experimental data, is put forward to investigate the regulatory system of dynamic interaction among genes in Kaposis sarcoma-associated herpesvirus network after induction of lytic replication. A reconstruction of transcription factor activity and gene-regulatory kinetics using data from a time-course microarray experiment is proposed. The computational approach uses nonlinear differential equations. In particular, the quantitative Michaelis-Menten model of gene-regulatory kinetics is extended to allow for post-transcriptional modi. cations and synergic interactions between target genes and the Rta transcription factor. The kinetic method is developed within a Bayesian inferential framework using Markov chain Monte Carlo. The pro. le of the Rta transcriptional regulator, other post-transcriptional regulatory genes and gene-specific kinetic parameters are inferred from the gene expression data of the target genes. The method described here provides an example of a principled approach to handle a wide range of transcriptional network architectures and regulatory activation mechanisms to reconstruct the activity of several transcription factors and activation kinetic parameters in a single regulatory network.</div>','<b>Title: </b>Parameter identifiability-based optimal observation remedy for biological networks<br><b>Year: </b>2017<br><b>Author: </b> Yl. Wang and  Hy. Miao<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/s12918-017-0432-2"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>biological networks, dynamic models, biological systems, model predictions, model parameters, systems biology, mathematical models, biological models, equation models, experimental data, molecular network, interaction networks, model errors, biological interaction, molecular interactions, model, dynamic systems, biological parameters, parameter uncertainty, growth model<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: To systematically understand the interactions between numerous biological components, a variety of biological networks on different levels and scales have been constructed and made available in public databases or knowledge repositories. Graphical models such as structural equation models have long been used to describe biological networks for various quantitative analysis tasks, especially key biological parameter estimation. However, limited by resources or technical capacities, partial observation is a common problem in experimental observations of biological networks, and it thus becomes an important problem how to select unobserved nodes for additional measurements such that all unknown model parameters become identifiable. To the best knowledge of our authors, a solution to this problem does not exist until this study.Results: The identifiability-based observation problem for biological networks is mathematically formulated for the first time based on linear recursive structural equation models, and then a dynamic programming strategy is developed to obtain the optimal observation strategies. The efficiency of the dynamic programming algorithm is achieved by avoiding both symbolic computation and matrix operations as used in other studies. We also provided necessary theoretical justifications to the proposed method. Finally, we verified the algorithm using synthetic network structures and illustrated the application of the proposed method in practice using a real biological network related to influenza A virus infection.Conclusions: The proposed approach is the first solution to the structural identifiability-based optimal observation remedy problem. It is applicable to an arbitrary directed acyclic biological network recursive SEMs without bidirectional edges, and it is a computerizable method. Observation remedy is an important issue in experiment design for biological networks, and we believe that this study provides a solid basis for dealing with more challenging design issues e.g., feedback loops, dynamic or nonlinear networks in the future.</div>','<b>Title: </b>Resonant response of forced complex networks: The role of topological disorder<br><b>Year: </b>2009<br><b>Author: </b> Hs. Chen,  Y. Shen,  Zh. Hou and  Hw. Xin<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1063/1.3211131"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>CHAOS<br><b>Cluster-Keywords: </b><em>complex networks, complex systems, network theory, network structure, biological network, social network, network science, dynamic networks, network systems, network model, network topology, systems biology, network, network analysis, system dynamics, networks, realworld networks, network automata, epidemic network, model network<div style="font-size:11px;"><br><b>Abstract: </b><em>We investigate the effect of topological disorder on a system of forced threshold elements, where each element is arranged on top of complex heterogeneous networks. Numerical results indicate that the response of the system to a weak signal can be amplified at an intermediate level of topological disorder, thus indicating the occurrence of topological-disorder-induced resonance. Using mean field method, we obtain an analytical understanding of the resonant phenomenon by deriving the effective potential of the system. Our findings might provide further insight into the role of network topology in signal amplification in biological networks.</div>','<b>Title: </b>A Hybrid Petri Net Model of the Akt-Wnt-mTOR-p70S6K Signalling Network in Neurons<br><b>Year: </b>2018<br><b>Author: </b> Mp. Fortin and  Sv. Hardy<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3233/FI-2018-1672"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>FUNDAMENTA INFORMATICAE<br><b>Cluster-Keywords: </b><em>regulatory network, systems biology, systems biological, network dynamics, gene expression, biochemical networks, raft proteins, biological networks, gene circuits, gene regulatory, signalling network, lipid raft, nuclear lamina, protein domain, networks systems, biological approaches, expression pattern, protein interaction, cell differentiation, network motifs<div style="font-size:11px;"><br><b>Abstract: </b><em>Signalling networks in the mammalian cell are complex systems. Their dynamic properties can often be explained by the interaction of regulatory network motifs. Computational modelling is instrumental in explaining how these systems function. To accomplish this task in this paper, we combine hybrid Petri net modelling and simulation, which produce the individual trajectories of protein concentrations and enable structural analysis of the reaction network. In the end, we generate dynamic graphs to get a system view of the signalling network dynamics. We use this methodology on the regulatory network of the proteins mTOR and p70S6K. In neuronal synaptic plasticity, prolonged activation of these proteins is needed to support an increased protein synthesis. However, biologists wonder how two brief calcium influxes of 1 second each can lead to this long activation downstream. With our computational approach and a new model of the Akt-Wnt-mTOR-p70S6K network, we explore the current biological hypothesis for the response of mTOR: the crosstalk between the Akt and Wnt pathways. Simulation results indicate instead that a feedforward motif between Akt, GSK3 and TSC2 acts as a coincidence detector. From the simulation results, we can also make two predictions that can be tested experimentally and indicate where a molecular regulatory mechanism seems to be missing to completely explain the activity in the signalling network.</div>','<b>Title: </b>Dynamic cross-talk analysis among TNF-R, TLR-4 and IL-1R signalings in TNF alpha-induced inflammatory responses<br><b>Year: </b>2010<br><b>Author: </b> Sk. Yang,  Yc. Wang,  Cc. Chao,  Yj. Chuang,  Cy. Lan and  Bs. Chen<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1755-8794-3-19"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC MEDICAL GENOMICS<br><b>Cluster-Keywords: </b><em>gene expression, systems biology, metabolic network, network model, network analysis, dynamic model, signaling pathways, signaling network, regulatory network, dynamic network, biology approach, candidate genes, gene network, gene regulatory, biology analysis, mathematical model, network dynamic, biology network, metabolic pathways, pathway analysis<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Development in systems biology research has accelerated in recent years, and the reconstructions for molecular networks can provide a global view to enable in-depth investigation on numerous system properties in biology. However, we still lack a systematic approach to reconstruct the dynamic protein-protein association networks at different time stages from high-throughput data to further analyze the possible cross-talks among different signaling/regulatory pathways.Methods: In this study we integrated protein-protein interactions from different databases to construct the rough protein-protein association networks PPANs during TNF alpha-induced inflammation. Next, the gene expression profiles of TNFa-induced HUVEC and a stochastic dynamic model were used to rebuild the significant PPANs at different time stages, reflecting the development and progression of endothelium inflammatory responses. A new cross-talk ranking method was used to evaluate the potential core elements in the related signaling pathways of toll-like receptor 4 TLR-4 as well as receptors for tumor necrosis factor TNF-R and interleukin-1 IL-1R.Results: The highly ranked cross-talks which are functionally relevant to the TNF alpha pathway were identified. A bow-tie structure was extracted from these cross-talk pathways, suggesting the robustness of network structure, the coordination of signal transduction and feedback control for efficient inflammatory responses to different stimuli. Further, several characteristics of signal transduction and feedback control were analyzed.Conclusions: A systematic approach based on a stochastic dynamic model is proposed to generate insight into the underlying defense mechanisms of inflammation via the construction of corresponding signaling networks upon specific stimuli. In addition, this systematic approach can be applied to other signaling networks under different conditions in different species. The algorithm and method proposed in this study could expedite prospective systems biology research when better experimental techniques for protein expression detection and microarray data with multiple sampling points become available in the future.</div>','<b>Title: </b>Constructing an integrated genetic and epigenetic cellular network for whole cellular mechanism using high-throughput next-generation sequencing data<br><b>Year: </b>2016<br><b>Author: </b> Bs. Chen and  Cw. Li<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/s12918-016-0256-5"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>gene networks, synthetic gene, synthetic biology, regulatory networks, design method, robust synthetic, nonlinear stochastic, molecular noises, gene regulatory, interaction networks, external disturbances, robust stability, robust gene, host cell, gene circuit, cellular network, design specifications, design procedure, signal transduction, parameter fluctuations<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Epigenetics has been investigated in cancer initiation, and development, especially, since the appearance of epigenomics. Epigenetics may be defined as the mechanisms that lead to heritable changes in gene function and without affecting the sequence of genome. These mechanisms explain how individuals with the same genotype produce phenotypic differences in response to environmental stimuli. Recently, with the accumulation of high-throughput next-generation sequencing NGS data, a key goal of systems biology is to construct networks for different cellular levels to explore whole cellular mechanisms. At present, there is no satisfactory method to construct an integrated genetic and epigenetic cellular network IGECN, which combines NGS omics data with gene regulatory networks GRNs, microRNAs miRNAs regulatory networks, protein-protein interaction networks PPINs, and epigenetic regulatory networks of methylation using high-throughput NGS data.Results: We investigated different kinds of NGS omics data to develop a systems biology method to construct an integrated cellular network based on three coupling models that describe genetic regulatory networks, protein-protein interaction networks, microRNA miRNA regulatory networks, and methylation regulation. The proposed method was applied to construct IGECNs of gastric cancer and the human immune response to human immunodeficiency virus HIV infection, to elucidate human defense response mechanisms. We successfully constructed an IGECN and validated it by using evidence from literature search. The integration of NGS omics data related to transcription regulation, protein-protein interactions, and miRNA and methylation regulation has more predictive power than independent datasets. We found that dysregulation of MIR7 contributes to the initiation and progression of inflammation-induced gastric cancer; dysregulation of MIR9 contributes to HIV-1 infection to hijack CD4 T cells through dysfunction of the immune and hormone pathways; dysregulation of MIR139-5p, MIRLET7i, and MIR10a contributes to the HIV-1 integration/replication stage; dysregulation of MIR101, MIR141, and MIR152 contributes to the HIV-1 virus assembly and budding mechanisms; dysregulation of MIR302a contributes to not only microvesicle-mediated transfer of miRNAs but also dysfunction of NF-kappa B signaling pathway in hepatocarcinogenesis.Conclusion: The coupling dynamic systems of the whole IGECN can allow us to investigate genetic and epigenetic cellular mechanisms via omics data and big database mining, and are useful for further experiments in the field of systems and synthetic biology.</div>','<b>Title: </b>Estimating Network Kinetics of the MAPK/ERK Pathway Using Biochemical Data<br><b>Year: </b>2012<br><b>Author: </b> V. Purutcuoglu and  E. Wit<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1155/2012/752631"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MATHEMATICAL PROBLEMS IN ENGINEERING<br><b>Cluster-Keywords: </b><em>circadian rhythms, systems biology, biological clock, circadian clock, biological rhythms, hpa axis, model, clock network, mathematical models, biological systems, dynamic systems, ode models, biology models, single cell, network, model parameters, experimental data, models, systems, dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>The MAPK/ERK pathway is a major signal transduction system which regulates many fundamental cellular processes including the growth control and the cell death. As a result of these roles, it has a crucial importance in cancer as well as normal developmental processes. Therefore, it has been intensively studied resulting in a wealth of knowledge about its activation. It is also well documented that the activation kinetics of the pathway is crucial to determine the nature of the biological response. However, while individual biochemical steps are well characterized, it is still difficult to predict or even understand how the activation kinetics works. The aim of this paper is to estimate the stochastic rate constants of the MAPK/ERK network dynamics. Accordingly, taking a Bayesian approach, we combined underlying qualitative biological knowledge in several competing dynamic models via sets of quasireactions and estimated the stochastic rate constants of these reactions. Comparing the resulting estimates via the BIC and DIC criteria, we chose a biological model which includes EGFR degradation-Raf-MEK-ERK cascade without the involvement of RKIPs.</div>','<b>Title: </b>Model building and model checking for biochemical processes<br><b>Year: </b>2003<br><b>Author: </b> M. Antoniotti,  A. Policriti,  N. Ugel and  B. Mishra<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1385/CBB:38:3:271"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>CELL BIOCHEMISTRY AND BIOPHYSICS<br><b>Cluster-Keywords: </b><em>systems biology, dynamic models, biological systems, model, dynamic systems, computational systems, network model, experimental data, model parameters, systems, systems model, models, pathway systems, data, biochemical pathway, dynamics model, biochemical networks, computational models, network, system<div style="font-size:11px;"><br><b>Abstract: </b><em>A central claim of computational systems biology is that, by drawing on mathematical approaches developed in the context of dynamic systems, kinetic analysis, computational theory and logic, it is possible to create powerful simulation, analysis, and reasoning tools for working biologists to decipher existing data, devise new experiments, and ultimately to understand functional properties of genomes, proteomes, cells, organs, and organisms. In this article, a novel computational tool is described that achieves many of the goals of this new discipline. The novelty of this system involves an automaton-based semantics of the temporal evolution of complex biochemical reactions starting from the representation given as a set of differential equations. The related tools also provide ability to qualitatively reason about the systems using a propositional temporal logic that can express an ordered sequence of events succinctly and unambiguously The implementation of mathematical and computational models in the Simpathica and XSSYS systems is described briefly. Several example applications of these systems to cellular and biochemical processes are presented: the two most prominent are Leibler et al.s repressilator an artificial synthesized oscillatory network, and Curto-Voit-Sorribas-Cascantes purine metabolism reaction model.</div>','<b>Title: </b>BioTapestry: A Tool to Visualize the Dynamic Properties of Gene Regulatory Networks<br><b>Year: </b>2012<br><b>Author: </b> Wjr. Longabaugh<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/978-1-61779-292-221"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>GENE REGULATORY NETWORKS: METHODS AND PROTOCOLS<br><b>Cluster-Keywords: </b><em>gene regulatory, systems biology, regulatory networks, gene network, stem cell, network model, transcription factors, gene expression, cell transcriptomes, network analysis, data analysis, stem cells, network grnsight, cell transformation, weighted network, gene ontology, networks grns, graph model, graph layout, endothelial cell<div style="font-size:11px;"><br><b>Abstract: </b><em>BioTapestry is an open source, freely available software tool that has been developed to handle the challenges of modeling genetic regulatory networks GRNs. Using BioTapestry, a researcher can construct a network model and use it to visualize and understand the dynamic behavior of a complex, spatially and temporally distributed GRN. Here we provide a step-by-step example of a way to use BioTapestry to build a GRN model and discuss some common issues that can arise during this process.</div>','<b>Title: </b>A Stochastic Mean Field Model for an Excitatory and Inhibitory Synaptic Drive Cortical Neuronal Network<br><b>Year: </b>2014<br><b>Author: </b> Q. Hui,  Wm. Haddad,  Jm. Bailey and  T. Hayakawa<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1109/TNNLS.2013.2281065"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IEEE TRANSACTIONS ON NEURAL NETWORKS AND LEARNING SYSTEMS<br><b>Cluster-Keywords: </b><em>network models, population model, field model, neural population, neural field, iot networks, neuronal networks, neural network, differential equations, parameter neural, lotkavolterra model, model, neuron models, model neurons, model leads, ecosystem model, iot devices, noise model, neuronal models, inhibitory synaptic<div style="font-size:11px;"><br><b>Abstract: </b><em>With the advances in biochemistry, molecular biology, and neurochemistry there has been impressive progress in understanding the molecular properties of anesthetic agents. However, there has been little focus on how the molecular properties of anesthetic agents lead to the observed macroscopic property that defines the anesthetic state, that is, lack of responsiveness to noxious stimuli. In this paper, we develop a mean field synaptic drive firing rate cortical neuronal model and demonstrate how the induction of general anesthesia can be explained using multistability; the property whereby the solutions of a dynamical system exhibit multiple attracting equilibria under asymptotically slowly changing inputs or system parameters. In particular, we demonstrate multistability in the mean when the system initial conditions or the system coefficients of the neuronal connectivity matrix are random variables. Uncertainty in the system coefficients is captured by representing system uncertain parameters by a multiplicative white noise model wherein stochastic integration is interpreted in the sense of Ito. Modeling a priori system parameter uncertainty using a multiplicative white noise model is motivated by means of the maximum entropy principle of Jaynes and statistical analysis.</div>','<b>Title: </b>Effects of Time Point Measurement on the Reconstruction of Gene Regulatory Networks<br><b>Year: </b>2010<br><b>Author: </b> Wy. Yan,  Hq. Zhu,  Y. Yang,  Jj. Chen,  Yy. Zhang and  Br. Shen<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3390/molecules15085354"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MOLECULES<br><b>Cluster-Keywords: </b><em>regulatory networks, gene regulatory, gene expression, network inference, expression data, network structure, gene networks, networks grns, regulatory genes, boolean networks, bayesian network, network reconstruction, networks gene, network dynamics, gene regulation, inference method, systems biology, time series, infer gene, network<div style="font-size:11px;"><br><b>Abstract: </b><em>With the availability of high-throughput gene expression data in the post-genomic era, reconstruction of gene regulatory networks has become a hot topic. Regulatory networks have been intensively studied over the last decade and many software tools are currently available. However, the impact of time point selection on network reconstruction is often underestimated. In this paper we apply the Dynamic Bayesian network DBN to construct the Arabidopsis gene regulatory networks by analyzing the time-series gene microarray data. In order to evaluate the impact of time point measurement on network reconstruction, we deleted time points one by one to yield 11 distinct groups of incomplete time series. Then the gene regulatory networks constructed based on complete and incomplete data series are compared in terms of statistics at different levels. Two time points are found to play a significant role in the Arabidopsis gene regulatory networks. Pathway analysis of significant nodes revealed three key regulatory genes. In addition, important regulations between genes, which were insensitive to the time point measurement, were also identified.</div>','<b>Title: </b>General and craniofacial development are complex adaptive processes influenced by diversity<br><b>Year: </b>2014<br><b>Author: </b> Ah. Brook,  Mb. ODonnell,  A. Hone,  E. Hart,  Te. Hughes,  Rn. Smith and  Gc. Townsend<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1111/adj.12158"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>AUSTRALIAN DENTAL JOURNAL<br><b>Cluster-Keywords: </b><em>interaction networks, proteinprotein interactions, protein interaction, network model, complex systems, systems biology, protein protein, protein complex, host network, dynamic interactions, network analysis, biological systems, networks network, cellular network, biological network, viral proteins, interactome networks, regulatory networks, immune system, systems network<div style="font-size:11px;"><br><b>Abstract: </b><em>Complex systems are present in such diverse areas as social systems, economies, ecosystems and biology and, therefore, are highly relevant to dental research, education and practice. A Complex Adaptive System in biological development is a dynamic process in which, from interacting components at a lower level, higher level phenomena and structures emerge. Diversity makes substantial contributions to the performance of complex adaptive systems. It enhances the robustness of the process, allowing multiple responses to external stimuli as well as internal changes. From diversity comes variation in outcome and the possibility of major change; outliers in the distribution enhance the tipping points. The development of the dentition is a valuable, accessible model with extensive and reliable databases for investigating the role of complex adaptive systems in craniofacial and general development. The general characteristics of such systems are seen during tooth development: self-organization; bottom-up emergence; multitasking; self-adaptation; variation; tipping points; critical phases; and robustness. Dental findings are compatible with the Random Network Model, the Threshold Model and also with the Scale Free Network Model which has a Power Law distribution. In addition, dental development shows the characteristics of Modularity and Clustering to form Hierarchical Networks. The interactions between the genes nodes demonstrate Small World phenomena, Subgraph Motifs and Gene Regulatory Networks.Genetic mechanisms are involved in the creation and evolution of variation during development. The genetic factors interact with epigenetic and environmental factors at the molecular level and form complex networks within the cells. From these interactions emerge the higher level tissues, tooth germs and mineralized teeth. Approaching development in this way allows investigation of why there can be variations in phenotypes from identical genotypes; the phenotype is the outcome of perturbations in the cellular systems and networks, as well as of the genotype.Understanding and applying complexity theory will bring about substantial advances not only in dental research and education but also in the organization and delivery of oral health care.</div>','<b>Title: </b>Closed-loop learning control of bio-networks<br><b>Year: </b>2004<br><b>Author: </b> J. Ku,  Xj. Feng and  H. Rabitz<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1089/cmb.2004.11.642"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF COMPUTATIONAL BIOLOGY<br><b>Cluster-Keywords: </b><em>systems biology, regulatory network, interaction networks, network model, gene regulatory, network, network biology, networks, gene network, response networks, network dynamics, dynamic model, cell networks, gene expression, biological networks, system dynamics, dynamic network, model, biological systems, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>A general goal of systems biology is to acquire a detailed quantitative understanding of the life-sustaining interactions between genes and proteins. There arises an interesting question of whether these network dynamics can be controlled externally. In the open-loop approach to experimental biology, a control design would be chosen based on a desired target re sponse and modeling with all the available knowledge about the system. If the system is not completely understood or disturbances occur, then unexpected deviations from the desired response can arise. A means to circumvent this difficulty is to optimize the controls in A closed-loop operation by modifying successive input controls based on the performance of previous controls. This paper presents a simulation of closed-loop learning control applied to biological systems in order to generate a desired response. The most powerful advantage of this technique is that the controls are deduced based on experimental results and the process can operate without a model for the underlying biochemical network. This feature eliminates the problem of faulty predictions as well as the need for a detailed understanding of the underlying molecular pathways, suggesting that biological systems can be controlled even before the post-systems biology era.</div>','<b>Title: </b>Systems Biology Approaches to Enzyme Kinetics: Analyzing Network Models of Drug Metabolism<br><b>Year: </b>2014<br><b>Author: </b> Na. Finn and  Ml. Kemp<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/978-1-62703-758-715"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ENZYME KINETICS IN DRUG METABOLISM: FUNDAMENTALS AND APPLICATIONS<br><b>Cluster-Keywords: </b><em>systems biology, network models, signaling network, parameter estimation, dynamic models, biological systems, mathematical models, metabolic network, parameter sensitivities, computational model, gene circuit, circuit models, experimental data, metabolic models, model system, scalable parameter, model parameters, parameter space, biomolecular networks, model<div style="font-size:11px;"><br><b>Abstract: </b><em>Intracellular drug metabolism involves transport, bioactivation, conjugation, and other biochemical steps. The dynamics of these steps are each dependent on a number of other cellular factors that can ultimately lead to unexpected behavior. In this review, we discuss the confounding processes and coupled reactions within bioactivation networks that require a systems-level perspective in order to fully understand the time-varying behavior. When converting known in vitro characteristics of drug-enzyme interactions into descriptions of cellular systems, features such as substrate availability, cell-to-cell variability, and intracellular redox state deserve special focus. An example of doxorubicin bioactivation is used for discussing points of consideration when constructing and analyzing network models of drug metabolism.</div>','<b>Title: </b>Machines vs. Ensembles: Effective MAPK Signaling through Heterogeneous Sets of Protein Complexes<br><b>Year: </b>2013<br><b>Author: </b> R. Suderman and  Ej. Deeds<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pcbi.1003278"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS COMPUTATIONAL BIOLOGY<br><b>Cluster-Keywords: </b><em>allosteric proteins, signaling networks, protein interactions, network model, molecular dynamics, protein complexes, allosteric network, signaling proteins, cell network, biochemical networks, free energy, combinatorial complexity, molecular species, biological systems, receptor signaling, systems biology, chemical species, model, dynamics simulations, cell signaling<div style="font-size:11px;"><br><b>Abstract: </b><em>Despite the importance of intracellular signaling networks, there is currently no consensus regarding the fundamental nature of the protein complexes such networks employ. One prominent view involves stable signaling machines with well-defined quaternary structures. The combinatorial complexity of signaling networks has led to an opposing perspective, namely that signaling proceeds via heterogeneous pleiomorphic ensembles of transient complexes. Since many hypotheses regarding network function rely on how we conceptualize signaling complexes, resolving this issue is a central problem in systems biology. Unfortunately, direct experimental characterization of these complexes has proven technologically difficult, while combinatorial complexity has prevented traditional modeling methods from approaching this question. Here we employ rule-based modeling, a technique that overcomes these limitations, to construct a model of the yeast pheromone signaling network. We found that this model exhibits significant ensemble character while generating reliable responses that match experimental observations. To contrast the ensemble behavior, we constructed a model that employs hierarchical assembly pathways to produce scaffold-based signaling machines. We found that this machine model could not replicate the experimentally observed combinatorial inhibition that arises when the scaffold is overexpressed. This finding provides evidence against the hierarchical assembly of machines in the pheromone signaling network and suggests that machines and ensembles may serve distinct purposes in vivo. In some cases, e. g. core enzymatic activities like protein synthesis and degradation, machines assembled via hierarchical energy landscapes may provide functional stability for the cell. In other cases, such as signaling, ensembles may represent a form of weak linkage, facilitating variation and plasticity in network evolution. The capacity of ensembles to signal effectively will ultimately shape how we conceptualize the function, evolution and engineering of signaling networks.</div>','<b>Title: </b>Modeling the tunability of the dual-feedback genetic oscillator<br><b>Year: </b>2020<br><b>Author: </b> Yj. Joshi,  Y. Jawale and  C. Athale<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1103/PhysRevE.101.012417"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PHYSICAL REVIEW E<br><b>Cluster-Keywords: </b><em>gene regulatory, systems biology, synthetic biology, regulatory network, gene network, gene expression, system model, synthetic network, network dynamics, systems theory, synthetic gene, genetic network, model predicts, dynamical systems, time delays, translational time, regulation models, oscillatory expression, network structure, biological systems<div style="font-size:11px;"><br><b>Abstract: </b><em>Oscillatory gene circuits are ubiquitous to biology and are involved in fundamental processes of cell cycle, circadian rhythms, and developmental systems. The synthesis of small, non-natural oscillatory genetic circuits has been increasingly used to test the fundamental principles of genetic network dynamics. While the "repressilator" was used to first demonstrate the proof of principle, a more recently developed dual-feedback, fast, tunable genetic oscillator has demonstrated a greater degree of robustness and control over oscillatory behavior by combining positive- and negative-feedback loops. This oscillator, combining lad negative- and araC positive- feedback loops, was, however, modeled using multiple layers of differential equations to capture the molecular complexity of regulation, in order to explain the experimentally measured oscillations. In the search for design principles of such minimal oscillatory circuits, we have developed a reduced model of this dual-feedback loop oscillator consisting of just six differential equations, two of which are delay differential equations. The delay term is optimized, as the only free parameter, to fit the experimental dynamics of the oscillator period and amplitude tunability by the two inducers isopropyl beta-D-1-thiogalactopyranoside IPTG and arabinose. We proceed to use our reduced and experimentally validated model to redesign the network by comparing the effect of asymmetry in gene expression at the level of a DNA copy numbers and the rates of b mRNA translation and c degradation, since experimental and theoretical work had predicted a need for an asymmetry in the copy numbers of activator araC and repressor lad genes encoded on plasmids. We confirm that the minimal period of the oscillator is sensitive to DNA copy number asymmetry, and can demonstrate that while the asymmetry in the translation rate has an identical effect as the plasmid copy numbers, modulating the asymmetry in mRNA degradation can improve the tunability of the period and amplitude of the oscillator. Thus, our model predicts control at the level of translation can be used to redesign such networks, for improved tunability, while at the same time making the network robust to replication "noise" and the effects of the host cell cycle. Thus, our model predicts experimentally testable principles to redesign a potentially more robust oscillatory genetic network.</div>','<b>Title: </b>A study of soft computing models for prediction of longitudinal wave velocity<br><b>Year: </b>2016<br><b>Author: </b> J. Singh,  Ak. Verma,  H. Banka,  Tn. Singh and  S. Maheshwar<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s12517-015-2115-x"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ARABIAN JOURNAL OF GEOSCIENCES<br><b>Cluster-Keywords: </b><em>gene networks, synthetic gene, synthetic biology, regulatory networks, design method, robust synthetic, nonlinear stochastic, molecular noises, gene regulatory, interaction networks, external disturbances, robust stability, robust gene, host cell, gene circuit, cellular network, design specifications, design procedure, signal transduction, parameter fluctuations<div style="font-size:11px;"><br><b>Abstract: </b><em>Genetic algorithm GA and support vector machine SVM optimization techniques are applied widely in the area of geophysics, civil, biology, mining, and geo-mechanics. Due to its versatility, it is being applied widely in almost every field of engineering. In this paper, the important features of GA and SVM are discussed as well as prediction of longitudinal wave velocity and its advantages over other conventional prediction methods. Longitudinal wave measurement is an indicator of peak particle velocity PPV during blasting and is an important parameter to be determined to minimize the damage caused by ground vibrations. The dynamic wave velocity and physico-mechanical properties of rock significantly affect the fracture propagation in rock. GA and SVM models are designed to predict the longitudinal wave velocity induced by ground vibrations. Chaos optimization algorithm has been used in SVM to find the optimal parameters of the model to increase the learning and prediction efficiency. GA model also has been developed and has used an objective function to be minimized. A parametric study for selecting the optimized parameters of GA model was done to select the best value. The mean absolute percentage error for the predicted wave velocity V value has been found to be the least 0.258  for GA as compared to values obtained by multivariate regression analysis MVRA, artificial neural network ANN, adaptive neuro-fuzzy inference system ANFIS, and SVM.</div>','<b>Title: </b>Computing the Projected Reachable Set of Stochastic Biochemical Reaction Networks Modeled by Switched Affine Systems<br><b>Year: </b>2018<br><b>Author: </b> F. Parise,  Me. Valcher and  J. Lygeros<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1109/TAC.2018.2798800"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IEEE TRANSACTIONS ON AUTOMATIC CONTROL<br><b>Cluster-Keywords: </b><em>reaction networks, network model, systems biology, biochemical reaction, stochastic reaction, model reduction, stochastic model, network dynamics, stochastic biochemical, stochastic simulation, system size, chemical reaction, gene network, elementary reactions, biochemical networks, parameter identification, parameter inference, cell populations, small system, linear noise<div style="font-size:11px;"><br><b>Abstract: </b><em>A fundamental question in systems biology is what combinations of mean and variance of the species present in a stochastic biochemical reaction network are attainable by perturbing the system with an external signal. To address this question, we show that the moments evolution in any generic network can be either approximated or, under suitable assumptions, computed exactly as the solution of a switched affine system. We then propose a new method to approximate the reachable set of such a switched affine system. A remarkable feature of our approach is that it allows one to easily compute projections of the reachable set for pairs of moments of interest, without requiring the computation of the full reachable set, which can be prohibitive for large networks. As a second contribution, we also show how to select the external signal in order to maximize the probability of reaching a target set. To illustrate the method, we study a renowned model of the controlled gene expression and we derive estimates of the reachable set, for the protein mean and variance, that are more accurate than those available in the literature and consistent with experimental data.</div>','<b>Title: </b>Assessing the multi-pathway threat from an invasive agricultural pest: Tuta absoluta in Asia<br><b>Year: </b>2019<br><b>Author: </b> J. Mcnitt,  Yy. Chungbaek,  H. Mortveit,  M. Marathe,  Mr. Campos,  N. Desneux,  T. Brevault,  R. Muniappan and  A. Adiga<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1098/rspb.2019.1159"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PROCEEDINGS OF THE ROYAL SOCIETY B-BIOLOGICAL SCIENCES<br><b>Cluster-Keywords: </b><em>complex networks, complex systems, network theory, network structure, biological network, social network, network science, dynamic networks, network systems, network model, network topology, systems biology, network, network analysis, system dynamics, networks, realworld networks, network automata, epidemic network, model network<div style="font-size:11px;"><br><b>Abstract: </b><em>Modern food systems facilitate rapid dispersal of pests and pathogens through multiple pathways. The complexity of spread dynamics and data inadequacy make it challenging to model the phenomenon and also to prepare for emerging invasions. We present a generic framework to study the spatio-temporal spread of invasive species as a multi-scale propagation process over a time-varying network accounting for climate, biology, seasonal production, trade and demographic information. Machine learning techniques are used in a novel manner to capture model variability and analyse parameter sensitivity. We applied the framework to understand the spread of a devastating pest of tomato, Tuta absoluta, in South and Southeast Asia, a region at the frontier of its current range. Analysis with respect to historical invasion records suggests that even with modest self-mediated spread capabilities, the pest can quickly expand its range through domestic city-to-city vegetable trade. Our models forecast that within 5-7 years, Tuta absoluta will invade all major vegetable growing areas of mainland Southeast Asia assuming unmitigated spread. Monitoring high-consumption areas can help in early detection, and targeted interventions at major production areas can effectively reduce the rate of spread.</div>','<b>Title: </b>Developmental Design of Synthetic Bacterial Architectures by Morphogenetic Engineering<br><b>Year: </b>2016<br><b>Author: </b> J. Pascalie,  M. Potier,  T. Kowaliw,  Jl. Giavitto,  O. Michel,  A. Spicher and  R. Doursat<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1021/acssynbio.5b00246"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ACS SYNTHETIC BIOLOGY<br><b>Cluster-Keywords: </b><em>gene circuits, synthetic biology, circuit behavior, biological parts, synthetic gene, circuit design, genetic circuits, cell types, metabolite dynamics, mathematical model, designed circuit, control circuit, biological systems, circuit models, synthetic circuit, feedback circuits, biological circuits, design models, metabolic pathways, circuit parameters<div style="font-size:11px;"><br><b>Abstract: </b><em>Synthetic biology is an emerging scientific field that promotes the standardized manufacturing of biological components without natural equivalents. Its goal is to create artificial living systems that can meet various needs in health care or energy domains. While most works are focused on the individual bacterium as a chemical reactor, our project, SynBioTIC, addresses a novel and more complex challenge: shape engineering; that is, the redesign of natural morphogenesis toward a new kind of developmental 3D printing. Potential applications include organ growth, natural computing in biocircuits, or future vegetal houses. To create in silico multicellular organisms that exhibit specific shapes, we construe their development as an iterative process combining fundamental collective phenomena such as homeostasis, patterning, segmentation, and limb growth. Our numerical experiments rely on the existing Escherichia coli simulator Gro, a physicochemical computation platform offering reaction-diffusion and collision dynamics solvers. The synthetic bioware of our model executes a set of rules, or genome, in each cell. Cells can differentiate into several predefined types associated with specific actions divide, emit signal, detect signal, die. Transitions between types are triggered by conditions involving internal and external sensors that detect various protein levels inside and around the cell. Indirect communication between bacteria is relayed by morphogen diffusion and the mechanical constraints of 2D packing. Starting from a single bacterium, the overall architecture emerges in a purely endogenous fashion through a series of developmental stages, inlcuding proliferation, differentiation, morphogen diffusion, and synchronization. The genome can be parametrized to control the growth and features of appendages individually. As exemplified by the L and T shapes that we obtain, certain precursor cells can be inhibited while others can create limbs of varying size divergence of the homology. Such morphogenetic phenotypes open the way to more complex shapes made of a recursive array of core bodies and limbs and, most importantly, to an evolutionary developmental exploration of unplanned functional forms.</div>','<b>Title: </b>Spatio-temporal structure of cell distribution in cortical Bone Multicellular Units: A mathematical model<br><b>Year: </b>2011<br><b>Author: </b> Pr. Buenzli,  P. Pivonka and  Dw. Smith<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.bone.2010.12.009"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BONE<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Bone remodelling maintains the functionality of skeletal tissue by locally coordinating bone-resorbing cells osteoclasts and bone-forming cells osteoblasts in the form of Bone Multicellular Units BMUs. Understanding the emergence of such structured units out of the complex network of biochemical interactions between bone cells is essential to extend our fundamental knowledge of normal bone physiology and its disorders. To this end, we propose a spatio-temporal continuum model that integrates some of the most important interaction pathways currently known to exist between cells of the osteoblastic and osteoclastic lineage. This mathematical model allows us to test the significance and completeness of these pathways based on their ability to reproduce the spatio-temporal dynamics of individual BMUs. We show that under suitable conditions, the experimentally observed structured cell distribution of cortical BMUs is retrieved. The proposed model admits travelling-wave-like solutions for the cell densities with tightly organised profiles, corresponding to the progression of a single remodelling BMU. The shapes of these spatial profiles within the travelling structure can be linked to the intrinsic parameters of the model such as differentiation and apoptosis rates for bone cells. In addition to the cell distribution, the spatial distribution of regulatory factors can also be calculated. This provides new insights on how different regulatory factors exert their action on bone cells leading to cellular spatial and temporal segregation, and functional coordination. C 2010 Elsevier Inc. All rights reserved.</div>','<b>Title: </b>Piecewise linear approximations to model the dynamics of adaptation to osmotic stress by food-borne pathogens<br><b>Year: </b>2017<br><b>Author: </b> A. Metris,  Sm. George and  D. Ropers<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.ijfoodmicro.2016.06.022"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>INTERNATIONAL JOURNAL OF FOOD MICROBIOLOGY<br><b>Cluster-Keywords: </b><em>regulatory networks, network dynamics, gene regulatory, network model, genetic network, gene networks, genetic regulatory, mathematical model, parameter space, network system, interaction networks, system dynamics, network structure, network, dynamic networks, network reduction, models models, biological networks, network motifs, biological systems<div style="font-size:11px;"><br><b>Abstract: </b><em>Addition of salt to food is one of the most ancient and most common methods of food preservation. However, little is known of how bacterial cells adapt to such conditions. We propose to use piecewise linear approximations to model the regulatory adaptation of Escherichia coli to osmotic stress. We apply the method to eight selected genes representing the functions known to be at play during osmotic adaptation. The network is centred on the general stress response factor, sigma S, and also includes a module representing the catabolic repressor CRP-cAMP. Glutamate, potassium and supercoiling are combined to represent the intracellular regulatory signal during osmotic stress induced by salt. The output is a module where growth is represented by the concentration of stable RNAs and the transcription of the osmotic gene osmY. The time course of gene expression of transport of osmoprotectant represented by the symporter proP and of the osmY is successfully reproduced by the network. The behaviour of the rpoS mutant predicted by the model is in agreement with experimental data. We discuss the application of the model to food-borne pathogens such as Salmonella; although the genes considered have orthologs, it seems that supercoiling is not regulated in the same way. The model is limited to a few selected genes, but the regulatory interactions are numerous and span different time scales. In addition, they seem to be condition specific: the links that are important during the transition from exponential to stationary phase are not all needed during osmotic stress. This model is one of the first steps towards modelling adaptation to stress in food safety and has scope to be extended to other genes and pathways, other stresses relevant to the food industry, and food-borne pathogens. The method offers a good compromise between systems of ordinary differential equations, which would be unmanageable because of the size of the system and for which insufficient data are available, and the more abstract Boolean methods. C 2016 Elsevier B.V. All rights reserved.</div>','<b>Title: </b>A Non-Homogeneous Dynamic Bayesian Network with Sequentially Coupled Interaction Parameters for Applications in Systems and Synthetic Biology<br><b>Year: </b>2012<br><b>Author: </b> M. Grzegorczyk and  D. Husmeier<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1515/1544-6115.7"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>STATISTICAL APPLICATIONS IN GENETICS AND MOLECULAR BIOLOGY<br><b>Cluster-Keywords: </b><em>petri net, petri nets, bayesian networks, biological networks, systems biology, regulatory networks, dynamic bayesian, net model, gene regulatory, network data, biological systems, biological data, network analysis, network models, biochemical networks, time series, network structure, signal transduction, net models, dbn model<div style="font-size:11px;"><br><b>Abstract: </b><em>An important and challenging problem in systems biology is the inference of gene regulatory networks from short non-stationary time series of transcriptional profiles. A popular approach that has been widely applied to this end is based on dynamic Bayesian networks DBNs, although traditional homogeneous DBNs fail to model the non-stationarity and time-varying nature of the gene regulatory processes. Various authors have therefore recently proposed combining DBNs with multiple changepoint processes to obtain time varying dynamic Bayesian networks TV-DBNs. However, TV-DBNs are not without problems. Gene expression time series are typically short, which leaves the model over-flexible, leading to over-fitting or inflated inference uncertainty. In the present paper, we introduce a Bayesian regularization scheme that addresses this difficulty. Our approach is based on the rationale that changes in gene regulatory processes appear gradually during an organisms life cycle or in response to a changing environment, and we have integrated this notion in the prior distribution of the TV-DBN parameters. We have extensively tested our regularized TV-DBN model on synthetic data, in which we have simulated short non-homogeneous time series produced from a system subject to gradual change. We have then applied our method to real-world gene expression time series, measured during the life cycle of Drosophila melanogaster, under artificially generated constant light condition in Arabidopsis thaliana, and from a synthetically designed strain of Saccharomyces cerevisiae exposed to a changing environment.</div>','<b>Title: </b>Boolean modeling: a logic-based dynamic approach for understanding signaling and regulatory networks and for making useful predictions<br><b>Year: </b>2014<br><b>Author: </b> R. Albert and  J. Thakar<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1002/wsbm.1273"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>WILEY INTERDISCIPLINARY REVIEWS-SYSTEMS BIOLOGY AND MEDICINE<br><b>Cluster-Keywords: </b><em>boolean network, network models, qualitative models, systems biology, mathematical modeling, boolean models, signaling networks, dynamic models, dynamic modeling, signaling dynamics, logic models, dynamic network, mathematical models, boolean modeling, signal transduction, regulatory networks, drug resistance, biological networks, network approach, network modeling<div style="font-size:11px;"><br><b>Abstract: </b><em>The biomolecules inside or near cells form a complex interacting system. Cellular phenotypes and behaviors arise from the totality of interactions among the components of this system. A fruitful way of modeling interacting biomolecular systems is by network-based dynamic models that characterize each component by a state variable, and describe the change in the state variables due to the interactions in the system. Dynamic models can capture the stable state patterns of this interacting system and can connect them to different cell fates or behaviors. A Boolean or logic model characterizes each biomolecule by a binary state variable that relates the abundance of that molecule to a threshold abundance necessary for downstream processes. The regulation of this state variable is described in a parameter free manner, making Boolean modeling a practical choice for systems whose kinetic parameters have not been determined. Boolean models integrate the body of knowledge regarding the components and interactions of biomolecular systems, and capture the systems dynamic repertoire, for example the existence of multiple cell fates. These models were used for a variety of systems and led to important insights and predictions. Boolean models serve as an efficient exploratory model, a guide for follow-up experiments, and as a foundation for more quantitative models. C 2014 Wiley Periodicals, Inc.</div>','<b>Title: </b>A systems biology network analysis of nutriepigenomic changes in endothelial cells exposed to epicatechin metabolites<br><b>Year: </b>2018<br><b>Author: </b> D. Milenkovic,  W. Vanden Berghe,  C. Morand,  S. Claude,  A. Van De Sandt,  S. Gorressen,  Le. Monfoulet,  Cs. Chirumamilla,  K. Declerck,  Ksv. Szic,  M. Lahtela-Kakkonen,  C. Gerhauser,  Mw. Merx and  M. Kelm<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1038/s41598-018-33959-x"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SCIENTIFIC REPORTS<br><b>Cluster-Keywords: </b><em>gene expression, systems biology, metabolic network, network model, network analysis, dynamic model, signaling pathways, signaling network, regulatory network, dynamic network, biology approach, candidate genes, gene network, gene regulatory, biology analysis, mathematical model, network dynamic, biology network, metabolic pathways, pathway analysis<div style="font-size:11px;"><br><b>Abstract: </b><em>Although vasculo-protective effects of flavan-3-ols are widely accepted today, their impact on endothelial cell functions and molecular mechanisms of action involved is not completely understood. The aim of this study was to characterize the potential endothelium-protective effects of circulating epicatechin metabolites and to define underlying mechanisms of action by an integrated systems biology approach. Reduced leukocyte rolling over vascular endothelium was observed following epicatechin supplementation in a mouse model of inflammation. Integrative pathway analysis of transcriptome, miRNome and epigenome profiles of endothelial cells exposed to epicatechin metabolites revealed that by acting at these different levels of regulation, metabolites affect cellular pathways involved in endothelial permeability and interaction with immune cells. In-vitro experiments on endothelial cells confirmed that epicatechin metabolites reduce monocyte adhesion and their transendothelial migration. Altogether, our in-vivo and in-vitro results support the outcome of a systems biology based network analysis which suggests that epicatechin metabolites mediate their vasculoprotective effects through dynamic regulation of endothelial cell monocyte adhesion and permeability. This study illustrates complex and multimodal mechanisms of action by which epicatechin modulate endothelial cell integrity.</div>','<b>Title: </b>An effective network reduction approach to find the dynamical repertoire of discrete dynamic networks<br><b>Year: </b>2013<br><b>Author: </b> Jgt. Zanudo and  R. Albert<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1063/1.4809777"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>CHAOS<br><b>Cluster-Keywords: </b><em>regulatory networks, network dynamics, gene regulatory, network model, genetic network, gene networks, genetic regulatory, mathematical model, parameter space, network system, interaction networks, system dynamics, network structure, network, dynamic networks, network reduction, models models, biological networks, network motifs, biological systems<div style="font-size:11px;"><br><b>Abstract: </b><em>Discrete dynamic models are a powerful tool for the understanding and modeling of large biological networks. Although a lot of progress has been made in developing analysis tools for these models, there is still a need to find approaches that can directly relate the network structure to its dynamics. Of special interest is identifying the stable patterns of activity, i.e., the attractors of the system. This is a problem for large networks, because the state space of the system increases exponentially with network size. In this work, we present a novel network reduction approach that is based on finding network motifs that stabilize in a fixed state. Notably, we use a topological criterion to identify these motifs. Specifically, we find certain types of strongly connected components in a suitably expanded representation of the network. To test our method, we apply it to a dynamic network model for a type of cytotoxic T cell cancer and to an ensemble of random Boolean networks of size up to 200. Our results show that our method goes beyond reducing the network and in most cases can actually predict the dynamical repertoire of the nodes fixed states or oscillations in the attractors of the system. C 2013 AIP Publishing LLC.</div>','<b>Title: </b>Bistability and Asynchrony in a Boolean Model of the L-arabinose Operon in Escherichia coli<br><b>Year: </b>2017<br><b>Author: </b> A. Jenkins and  M. Macauley<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s11538-017-0306-1"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BULLETIN OF MATHEMATICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>The lactose operon in Escherichia coli was the first known gene regulatory network, and it is frequently used as a prototype for new modeling paradigms. Historically, many of these modeling frameworks use differential equations. More recently, Stigler and Veliz-Cuba proposed a Boolean model that captures the bistability of the system and all of the biological steady states. In this paper, we model the well-known arabinose operon in E. coli with a Boolean network. This has several complex features not found in the lac operon, such as a protein that is both an activator and repressor, a DNA looping mechanism for gene repression, and the lack of inducer exclusion by glucose. For 11 out of 12 choices of initial conditions, we use computational algebra and Sage to verify that the state space contains a single fixed point that correctly matches the biology. The final initial condition, medium levels of arabinose and no glucose, successfully predicts the systems bistability. Finally, we compare the state space under synchronous and asynchronous update and see that the former has several artificial cycles that go away under a general asynchronous update.</div>','<b>Title: </b>Spindle assembly checkpoint is sufficient for complete Cdc20 sequestering in mitotic control<br><b>Year: </b>2015<br><b>Author: </b> B. Ibrahim<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.csbj.2015.03.006"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>COMPUTATIONAL AND STRUCTURAL BIOTECHNOLOGY JOURNAL<br><b>Cluster-Keywords: </b><em>regulatory networks, stochastic simulation, gene networks, reaction networks, systems biology, stochastic model, network models, complex systems, networks stochastic, gene regulatory, biological systems, stochastic dynamics, differential equations, protein networks, cell cycle, continuous model, interaction networks, systems chemistry, time delays, biochemical reaction<div style="font-size:11px;"><br><b>Abstract: </b><em>The spindle checkpoint assembly SAC ensures genomefidelity by temporarily delaying anaphase onset, until all chromosomes are properly attached to the mitotic spindle. The SAC delays mitotic progression by preventing activation of the ubiquitin ligase anaphase-promoting complex APC/C or cyclosome; whose activation by Cdc20 is required for sister-chromatid separation marking the transition into anaphase. The mitotic checkpoint complex MCC, which contains Cdc20 as a subunit, binds stably to the APC/C. Compelling evidence by Izawa and Pines Nature 2014; 10.1038/nature13911 indicates that the MCC can inhibit a second Cdc20 that has already bound and activated the APC/C. Whether or not MCC per se is sufficient to fully sequester Cdc20 and inhibit APC/C remains unclear. Here, a dynamic model for SAC regulation in which the MCC binds a second Cdc20 was constructed. This model is compared to the MCC, and the MCC-and-BubR1 dual inhibition of APC core model variants and subsequently validated with experimental data from the literature. By using ordinary nonlinear differential equations and spatial simulations, it is shown that the SAC works sufficiently to fully sequester Cdc20 and completely inhibit APC/C activity. This study highlights the principle that a systems biology approach is vital for molecular biology and could also be used for creating hypotheses to design future experiments. C 2015 Ibrahim. Published by Elsevier B.V. on behalf of the Research Network of Computational and Structural Biotechnology. This is an open access article under the CC BY-NC-ND license http://creativecommons.org/licenses/by-nc-nd/4.0/.</div>','<b>Title: </b>Invertibility and nonsingularity of Boolean control networks<br><b>Year: </b>2015<br><b>Author: </b> Kz. Zhang,  Lj. Zhang and  Lh. Xie<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.automatica.2015.07.016"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>AUTOMATICA<br><b>Cluster-Keywords: </b><em>boolean network, boolean functions, regulatory networks, canalizing functions, network models, gene regulatory, boolean control, control networks, biological networks, boolean models, update functions, interaction networks, large network, regulatory functions, canalizing boolean, genetic regulatory, systems biology, driver nodes, networks bcns, biological systems<div style="font-size:11px;"><br><b>Abstract: </b><em>Invertibility is an interesting and classical control-theoretic problem. However, there has been no result for the invertibility of Boolean control networks BCNs so far. We first adopt the theory of symbolic dynamics to characterize it. First, it is shown that a BCN generates a continuous mapping from the space of input trajectories to the space of output trajectories. Based on it, the concepts of nonsingularity and invertibility of BCNs are first defined as the injectivity and bijectivity of the mapping, respectively. Second, combined symbolic dynamics with the semi-tensor product STP of matrices, an equivalent test criterion for invertibility is given; easily computable algorithms to construct the inverse BCN for an invertible BCN are presented; and it is proved that invertibility remains invariant under coordinate transformations. Third, an equivalent test criterion for nonsingularity is given via defining a novel directed graph that is called weighted pair graph. Lastly, as an application of invertibility to systems biology, we prove that the BCN model proposed in Faure et al. 2006 is not invertible, i.e., we prove that arbitrarily controlling mammalian cell cycles is unfeasible at the theoretical level. C 2015 Elsevier Ltd. All rights reserved.</div>','<b>Title: </b>Biologically inspired calibration-free adaptive saccade control of a binocular camera-head<br><b>Year: </b>1997<br><b>Author: </b> J. Bruske,  M. Hansen,  L. Riehn and  G. Sommer<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s004220050403"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOLOGICAL CYBERNETICS<br><b>Cluster-Keywords: </b><em>neural network, network model, network, neurogenesis model, genomic network, networks, saccade control, droplets networks, understanding networks, gene regulatory, model, tumor-immune networks, networks dynamical, network systems, artificial neurogenesis, regulatory networks, dynamic network, information flow, cancer cells, capillary networks<div style="font-size:11px;"><br><b>Abstract: </b><em>This paper describes fast and accurate calibration-free adaptive saccade control of a four-degrees-of-freedom binocular camera-head by means of Dynamic Cell Structures DCS. The approach has been inspired by biology because primates face a similar problem and there is strong evidence that they have solved it in a similar way, i.e., by error feedback learning of an inverse model. Yet the emphasis of this article is not on detailed biological modeling but on how incremental growth of our artificial neural network model up to a prespecified precision results in very small networks suitable for real-time saccade control. Error-feedback-based training of this network proceeds in two phases. In the first phase we use a crude model of the cameras and the kinematics of the head to learn the topology of the input manifold together with a rough approximation of the control function off-line. In contrast to, for example, Kohonen-type adaptation rules, the distribution of neural units minimizes the control error and does not merely mimic the input probability density. In the second phase, the operating phase, the linear output units of the network continue to adapt on-line. Besides our TRC binocular camera-head we use a Datacube image processing system and a Staubli R90 robot arm for automated training in the second phase. It will be demonstrated that the controller successfully corrects errors in the model and rapidly adapts to changing parameters.</div>','<b>Title: </b>Modeling somatic computation with non-neural bioelectric networks<br><b>Year: </b>2019<br><b>Author: </b> S. Manicka and  M. Levin<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1038/s41598-019-54859-8"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SCIENTIFIC REPORTS<br><b>Cluster-Keywords: </b><em>gene circuits, synthetic biology, circuit behavior, biological parts, synthetic gene, circuit design, genetic circuits, cell types, metabolite dynamics, mathematical model, designed circuit, control circuit, biological systems, circuit models, synthetic circuit, feedback circuits, biological circuits, design models, metabolic pathways, circuit parameters<div style="font-size:11px;"><br><b>Abstract: </b><em>The field of basal cognition seeks to understand how adaptive, context-specific behavior occurs in non-neural biological systems. Embryogenesis and regeneration require plasticity in many tissue types to achieve structural and functional goals in diverse circumstances. Thus, advances in both evolutionary cell biology and regenerative medicine require an understanding of how non-neural tissues could process information. Neurons evolved from ancient cell types that used bioelectric signaling to perform computation. However, it has not been shown whether or how non-neural bioelectric cell networks can support computation. We generalize connectionist methods to non-neural tissue architectures, showing that a minimal non-neural Bio-Electric Network BEN model that utilizes the general principles of bioelectricity electrodiffusion and gating can compute. We characterize BEN behaviors ranging from elementary logic gates to pattern detectors, using both fixed and transient inputs to recapitulate various biological scenarios. We characterize the mechanisms of such networks using dynamical-systems and information-theory tools, demonstrating that logic can manifest in bidirectional, continuous, and relatively slow bioelectrical systems, complementing conventional neural-centric architectures. Our results reveal a variety of non-neural decision-making processes as manifestations of general cellular biophysical mechanisms and suggest novel bioengineering approaches to construct functional tissues for regenerative medicine and synthetic biology as well as new machine learning architectures.</div>','<b>Title: </b>The regulatory network of E. coli metabolism as a Boolean dynamical system exhibits both homeostasis and flexibility of response<br><b>Year: </b>2008<br><b>Author: </b> A. Samal and  S. Jain<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-2-21"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>metabolic network, metabolic model, dynamic flux, growth rate, genomescale metabolic, flux balance, balance analysis, metabolic flux, metabolic pathways, dynamic model, metabolic reactions, cellular growth, regulatory network, systems biology, optimal growth, metabolic, modeling metabolic, growth conditions, dynamic analysis, cell growth<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Elucidating the architecture and dynamics of large scale genetic regulatory networks of cells is an important goal in systems biology. We study the system level dynamical properties of the genetic network of Escherichia coli that regulates its metabolism, and show how its design leads to biologically useful cellular properties. Our study uses the database Covert et al., Nature 2004 containing 583 genes and 96 external metabolites which describes not only the network connections but also the Boolean rule at each gene node that controls the switching on or off of the gene as a function of its inputs.Results: We have studied how the attractors of the Boolean dynamical system constructed from this database depend on the initial condition of the genes and on various environmental conditions corresponding to buffered minimal media. We find that the system exhibits homeostasis in that its attractors, that turn out to be fixed points or low period cycles, are highly insensitive to initial conditions or perturbations of gene configurations for any given fixed environment. At the same time the attractors show a wide variation when external media are varied implying that the system mounts a highly flexible response to changed environmental conditions. The regulatory dynamics acts to enhance the cellular growth rate under changed media.Conclusion: Our study shows that the reconstructed genetic network regulating metabolism in E. coli is hierarchical, modular, and largely acyclic, with environmental variables controlling the root of the hierarchy. This architecture makes the cell highly robust to perturbations of gene configurations as well as highly responsive to environmental changes. The twin properties of homeostasis and response flexibility are achieved by this dynamical system even though it is not close to the edge of chaos.</div>','<b>Title: </b>Quasi-cellular systems: stochastic simulation analysis at nanoscale range<br><b>Year: </b>2013<br><b>Author: </b> L. Calviello,  P. Stano,  F. Mavelli,  Pl. Luisi and  R. Marangoni<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1471-2105-14-S7-S7"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>regulatory networks, stochastic simulation, gene networks, reaction networks, systems biology, stochastic model, network models, complex systems, networks stochastic, gene regulatory, biological systems, stochastic dynamics, differential equations, protein networks, cell cycle, continuous model, interaction networks, systems chemistry, time delays, biochemical reaction<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: The wet-lab synthesis of the simplest forms of life minimal cells is a challenging aspect in modern synthetic biology. Quasi-cellular systems able to produce proteins directly from DNA can be obtained by encapsulating the cell-free transcription/translation system PURESYSTEM TMPS in liposomes. It is possible to detect the intra-vesicle protein production using DNA encoding for GFP and monitoring the fluorescence emission over time. The entrapment of solutes in small-volume liposomes is a fundamental open problem. Stochastic simulation is a valuable tool in the study of biochemical reaction at nanoscale range. QDC Quick Direct-Method Controlled, a stochastic simulation software based on the well-known Gillespies SSA algorithm, was used. A suitable model formally describing the PS reactions network was developed, to predict, from inner species concentrations very difficult to measure in small-volumes, the resulting fluorescence signal experimentally observable.Results: Thanks to suitable features specific of QDC, we successfully formalized the dynamical coupling between the transcription and translation processes that occurs in the real PS, thus bypassing the concurrent-only environment of Gillespies algorithm. Simulations were firstly performed for large liposomes 2.67 mu m of diameter entrapping the PS to synthetize GFP. By varying the initial concentrations of the three main classes of molecules involved in the PS DNA, enzymes, consumables, we were able to stochastically simulate the time-course of GFP-production. The sigmoid fit of the GFP-production curves allowed us to extract three quantitative parameters which are significantly dependent on the various initial states. Then we extended this study for small-volume liposomes 575 nm of diameter, where it is more complex to infer the intra-vesicle composition, due to the expected anomalous entrapment phenomena. We identified almost two extreme states that are forecasted to give rise to significantly different experimental observables.Conclusions: The present work is the first one describing in the detail the stochastic behavior of the PS. Thanks to our results, an experimental approach is now possible, aimed at recording the GFP production kinetics in very small microemulsion droplets or liposomes, and inferring, by using the simulation as a reverse-engineering procedure, the internal solutes distribution, and shed light on the still unknown forces driving the entrapment phenomenon.</div>','<b>Title: </b>Dynamic and collective analysis of membrane protein interaction network based on gene regulatory network model<br><b>Year: </b>2012<br><b>Author: </b> Ys. Ding,  Yz. Shen,  Lh. Ren and  Lj. Cheng<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.neucom.2011.05.047"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>NEUROCOMPUTING<br><b>Cluster-Keywords: </b><em>interaction network, actin filament, network model, filament network, network structures, actin network, protein interaction, actin dynamics, network, membrane protein, network dynamics, model dynamic, cell, model, network approach, protein complex, computational model, actin, adhesion dynamics, cell dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>Membrane protein interactions are vitally important for every process in a living cell. Information about these interactions can improve our understanding of diseases and provide the basis to revolutionize therapeutic treatments. However, current experimental techniques for large-scale detection of protein-protein interactions are biased against membrane proteins, it is necessary to develop novel tools to deal with this kind of bio-network. To realize this, we construct membrane protein interaction network based on gene regulatory network model. Three model forms, basic form, non-dimensionalization form, and more complex form, are proposed to understand the dynamic and collective control of developmental process and the characters of membrane protein interaction network, including small-world network, scale free distributing and robustness, and its significance for biology. Four simulation examples are presented to illustrate the usefulness and flexibility of the GRN model method for the study of membrane protein interaction network. The results show that the proposed approach holds a high potential to become a useful tool in prediction of membrane protein interactions. Moreover, it has biological significance and value for biology and pharmacology. c 2012 Elsevier B.V. All rights reserved.</div>','<b>Title: </b>Attractor landscape analysis of colorectal tumorigenesis and its reversion<br><b>Year: </b>2016<br><b>Author: </b> Sh. Cho,  Sm. Park,  Hs. Lee,  Hy. Lee and  Kh. Cho<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/s12918-016-0341-9"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>gene expression, systems biology, metabolic network, network model, network analysis, dynamic model, signaling pathways, signaling network, regulatory network, dynamic network, biology approach, candidate genes, gene network, gene regulatory, biology analysis, mathematical model, network dynamic, biology network, metabolic pathways, pathway analysis<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Colorectal cancer arises from the accumulation of genetic mutations that induce dysfunction of intracellular signaling. However, the underlying mechanism of colorectal tumorigenesis driven by genetic mutations remains yet to be elucidated.Results: To investigate colorectal tumorigenesis at a system-level, we have reconstructed a large-scale Boolean network model of the human signaling network by integrating previous experimental results on canonical signaling pathways related to proliferation, metastasis, and apoptosis. Throughout an extensive simulation analysis of the attractor landscape of the signaling network model, we found that the attractor landscape changes its shape by expanding the basin of attractors for abnormal proliferation and metastasis along with the accumulation of driver mutations. A further hypothetical study shows that restoration of a normal phenotype might be possible by reversely controlling the attractor landscape. Interestingly, the targets of approved anti-cancer drugs were highly enriched in the identified molecular targets for the reverse control.Conclusions: Our results show that the dynamical analysis of a signaling network based on attractor landscape is useful in acquiring a system-level understanding of tumorigenesis and developing a new therapeutic strategy.</div>','<b>Title: </b>Gene regulatory networks on transfer entropy GRNTE: a novel approach to reconstruct gene regulatory interactions applied to a case study for the plant pathogen Phytophthora infestans<br><b>Year: </b>2019<br><b>Author: </b> Jc. Castro,  I. Valdes,  Ln. Gonzalez-Garcia,  G. Danies,  S. Canas,  Fv. Winck,  Ce. Nustez,  S. Restrepo and  Dm. Riano-Pachon<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/s12976-019-0103-7"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>THEORETICAL BIOLOGY AND MEDICAL MODELLING<br><b>Cluster-Keywords: </b><em>regulatory networks, gene regulatory, gene expression, network inference, expression data, network structure, gene networks, networks grns, regulatory genes, boolean networks, bayesian network, network reconstruction, networks gene, network dynamics, gene regulation, inference method, systems biology, time series, infer gene, network<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: The increasing amounts of genomics data have helped in the understanding of the molecular dynamics of complex systems such as plant and animal diseases. However, transcriptional regulation, although playing a central role in the decision-making process of cellular systems, is still poorly understood. In this study, we linked expression data with mathematical models to infer gene regulatory networks GRN. We present a simple yet effective method to estimate transcription factors GRNs from transcriptional data.Method: We defined interactions between pairs of genes edges in the GRN as the partial mutual information between these genes that takes into account time and possible lags in time from one gene in relation to another. We call this method Gene Regulatory Networks on Transfer Entropy GRNTE and it corresponds to Granger causality for Gaussian variables in an autoregressive model. To evaluate the reconstruction accuracy of our method, we generated several sub-networks from the GRN of the eukaryotic yeast model, Saccharomyces cerevisae. Then, we applied this method using experimental data of the plant pathogen Phytophthora infestans. We evaluated the transcriptional expression levels of 48 transcription factors of P. infestans during its interaction with one moderately resistant and one susceptible cultivar of yellow potato Solanum tuberosum group Phureja, using RT-qPCR. With these data, we reconstructed the regulatory network of P. infestans during its interaction with these hosts.Results: We first evaluated the performance of our method, based on the transfer entropy GRNTE, on eukaryotic datasets from the GRNs of the yeast S. cerevisae. Results suggest that GRNTE is comparable with the state-of-the-art methods when the parameters for edge detection are properly tuned. In the case of P. infestans, most of the genes considered in this study, showed a significant change in expression from the onset of the interaction 0h post inoculum - hpi to the later time-points post inoculation. Hierarchical clustering of the expression data discriminated two distinct periods during the infection: from 12 to 36 hpi and from 48 to 72 hpi for both the moderately resistant and susceptible cultivars. These distinct periods could be associated with two phases of the life cycle of the pathogen when infecting the host plant: the biotrophic and necrotrophic phases.Conclusions: Here we presented an algorithmic solution to the problem of network reconstruction in time series data. This analytical perspective makes use of the dynamic nature of time series data as it relates to intrinsically dynamic processes such as transcription regulation, were multiple elements of the cell e.g., transcription factors act simultaneously and change over time. We applied the algorithm to study the regulatory network of P. infestans during its interaction with two hosts which differ in their level of resistance to the pathogen. Although the gene expression analysis did not show differences between the two hosts, the results of the GRN analyses evidenced rewiring of the genes interactions according to the resistance level of the host. This suggests that different regulatory processes are activated in response to different environmental cues. Applications of our methodology showed that it could reliably predict where to place edges in the transcriptional networks and sub-networks. The experimental approach used here can help provide insights on the biological role of these interactions on complex processes such as pathogenicity. The code used is available at https://github.com/jccastrog/GRNTE under GNU general public license 3.0.</div>','<b>Title: </b>A new comparison method for stability theory of differential systems with time-varying delays<br><b>Year: </b>2008<br><b>Author: </b> Z. Zeng,  P. Yu and  Xx. Liao<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1142/S0218127408020215"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>INTERNATIONAL JOURNAL OF BIFURCATION AND CHAOS<br><b>Cluster-Keywords: </b><em>chaotic system, control systems, neural networks, jerk chaotic, torch chaotic, mode control, sliding mode, plasma torch, dynamic neural, mitochondrial dynamics, system, systems biology, adaptive integral, model system, rodtype plasma, integral sliding, systems, jerk system, cancer cell, comparison method<div style="font-size:11px;"><br><b>Abstract: </b><em>In this paper, a new comparison method is developed by using increasing and decreasing mechanisms, which are inherent in time-delay systems, to decompose systems. Based on the new method, whose expected performance is compared with the state of the original system, some new conditions are obtained to guarantee that the original system tracks the expected values. The locally exponential convergence rate and the convergence region of the polynomial differential equations with time-varying delays are also investigated. In particular, the comparison method is used to improve the 3/2 stability theorems of differential systems with pure delays. Moreover, the comparison method is applied to identify a threshold, and to consider the disease-free equilibrium points of an HIV endemic model with stages of progress to AIDs and time-varying delay. It is shown that if the threshold is smaller than 1, the equilibrium point of the model is globally, exponentially stable. Another application of the comparison method is to investigate the global, exponential stability of neural networks, and some new theoretical results are obtained. Numerical simulations are presented to verify the theoretical results.</div>','<b>Title: </b>Metabolic Network Constrains Gene Regulation of C-4 Photosynthesis: The Case of Maize<br><b>Year: </b>2016<br><b>Author: </b> S. Robaina-Estevez and  Z. Nikoloski<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1093/pcp/pcw034"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLANT AND CELL PHYSIOLOGY<br><b>Cluster-Keywords: </b><em>gene expression, systems biology, metabolic network, network model, network analysis, dynamic model, signaling pathways, signaling network, regulatory network, dynamic network, biology approach, candidate genes, gene network, gene regulatory, biology analysis, mathematical model, network dynamic, biology network, metabolic pathways, pathway analysis<div style="font-size:11px;"><br><b>Abstract: </b><em>Engineering C-3 plants to increase their efficiency of carbon fixation as well as of nitrogen and water use simultaneously may be facilitated by understanding the mechanisms that underpin the C-4 syndrome. Existing experimental studies have indicated that the emergence of the C-4 syndrome requires co-ordination between several levels of cellular organization, from gene regulation to metabolism, across two co-operating cell systems-mesophyll and bundle sheath cells. Yet, determining the extent to which the structure of the C-4 plant metabolic network may constrain gene expression remains unclear, although it will provide an important consideration in engineering C-4 photosynthesis in C-3 plants. Here, we utilize flux coupling analysis with the second-generation maize metabolic models to investigate the correspondence between metabolic network structure and transcriptomic phenotypes along the maize leaf gradient. The examined scenarios with publically available data from independent experiments indicate that the transcriptomic programs of the two cell types are co-ordinated, quantitatively and qualitatively, due to the presence of coupled metabolic reactions in specific metabolic pathways. Taken together, our study demonstrates that precise quantitative coupling will have to be achieved in order to ensure a successfully engineered transition from C-3 to C-4 crops.</div>','<b>Title: </b>Model Rejection and Parameter Reduction via Time Series<br><b>Year: </b>2018<br><b>Author: </b> B. Cummins,  T. Gedeon,  S. Harker and  K. Mischaikow<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1137/17M1134548"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SIAM JOURNAL ON APPLIED DYNAMICAL SYSTEMS<br><b>Cluster-Keywords: </b><em>regulatory networks, network dynamics, gene regulatory, network model, genetic network, gene networks, genetic regulatory, mathematical model, parameter space, network system, interaction networks, system dynamics, network structure, network, dynamic networks, network reduction, models models, biological networks, network motifs, biological systems<div style="font-size:11px;"><br><b>Abstract: </b><em>We show how a graph algorithm for finding matching labeled paths in pairs of labeled directed graphs can be used to perform model invalidation for a class of dynamical systems including regulatory network models of relevance to systems biology. In particular, given a partial order of events describing local minima and local maxima of observed quantities from experimental time series data, we produce a labeled directed graph we call the pattern graph for which every path from root to leaf corresponds to a plausible sequence of events. We then consider the regulatory network model, which can itself be rendered into a labeled directed graph we call the search graph via techniques previously developed in computational dynamics. Labels on the pattern graph correspond to experimentally observed events, while labels on the search graph correspond to mathematical facts about the model. We give a theoretical guarantee that failing to find a match invalidates the model. As an application we consider gene regulatory models for the yeast S. cerevisiae.</div>','<b>Title: </b>Recurrent neural network based hybrid model for reconstructing gene regulatory network<br><b>Year: </b>2016<br><b>Author: </b> K. Raza and  M. Alam<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.compbiolchem.2016.08.002"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>COMPUTATIONAL BIOLOGY AND CHEMISTRY<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>One of the exciting problems in systems biology research is to decipher how genome controls the development of complex biological system. The gene regulatory networks GRNs help in the identification of regulatory interactions between genes and offer fruitful information related to functional role of individual gene in a cellular system. Discovering GRNs lead to a wide range of applications, including identification of disease related pathways providing novel tentative drug targets, helps to predict disease response, and also assists in diagnosing various diseases including cancer. Reconstruction of GRNs from available biological data is still an open problem. This paper proposes a recurrent neural network RNN based model of GRN, hybridized with generalized extended Kalman filter for weight update in backpropagation through time training algorithm. The RNN is a complex neural network that gives a better settlement between biological closeness and mathematical flexibility to model GRN; and is also able to capture complex, non-linear and dynamic relationships among variables. Gene expression data are inherently noisy and Kalman filter performs well for estimation problem even in noisy data. Hence, we applied non-linear version of Kalman filter, known as generalized extended Kalman filter, for weight update during RNN training. The developed model has been tested on four benchmark networks such as DNA SOS repair network, IRMA network, and two synthetic networks from DREAM Challenge. We performed a comparison of our results with other state-of-the-art techniques which shows superiority of our proposed model. Further, 5 Gaussian noise has been induced in the dataset and result of the proposed model shows negligible effect of noise on results, demonstrating the noise tolerance capability of the model. C 2016 Elsevier Ltd. All rights reserved.</div>','<b>Title: </b>An efficient algorithm for identifying primary phenotype attractors of a large-scale Boolean network<br><b>Year: </b>2016<br><b>Author: </b> Sm. Choo and  Kh. Cho<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/s12918-016-0338-4"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>boolean network, boolean functions, regulatory networks, canalizing functions, network models, gene regulatory, boolean control, control networks, biological networks, boolean models, update functions, interaction networks, large network, regulatory functions, canalizing boolean, genetic regulatory, systems biology, driver nodes, networks bcns, biological systems<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Boolean network modeling has been widely used to model large-scale biomolecular regulatory networks as it can describe the essential dynamical characteristics of complicated networks in a relatively simple way. When we analyze such Boolean network models, we often need to find out attractor states to investigate the converging state features that represent particular cell phenotypes. This is, however, very difficult often impossible for a large network due to computational complexity.Results: There have been some attempts to resolve this problem by partitioning the original network into smaller subnetworks and reconstructing the attractor states by integrating the local attractors obtained from each subnetwork. But, in many cases, the partitioned subnetworks are still too large and such an approach is no longer useful. So, we have investigated the fundamental reason underlying this problem and proposed a novel efficient way of hierarchically partitioning a given large network into smaller subnetworks by focusing on some attractors corresponding to a particular phenotype of interest instead of considering all attractors at the same time. Using the definition of attractors, we can have a simplified update rule with fixed state values for some nodes. The resulting subnetworks were small enough to find out the corresponding local attractors which can be integrated for reconstruction of the global attractor states of the original large network.Conclusions: The proposed approach can substantially extend the current limit of Boolean network modeling for converging state analysis of biological networks.</div>','<b>Title: </b>Robust stability and instability of biochemical networks with parametric uncertainty<br><b>Year: </b>2011<br><b>Author: </b> S. Waldherr and  F. Allgower<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.automatica.2011.01.012"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>AUTOMATICA<br><b>Cluster-Keywords: </b><em>sensitivity analysis, systems biology, robustness analysis, network model, network inference, regulatory network, systems analysis, gene expression, parametric uncertainty, biological networks, mathematical models, biochemical networks, biological functions, cellular networks, biological system, parametric sensitivity, dynamical models, dynamic system, networks analysis, gene regulatory<div style="font-size:11px;"><br><b>Abstract: </b><em>Parameter perturbations in dynamical models of biochemical networks affect the qualitative dynamical behaviour observed in the model. Since this qualitative behaviour is in many cases the key model output used to explain biological function, the robustness analysis of the models behaviour with respect to parametric uncertainty is a crucial step in systems biology research. In this paper, we develop a new method for robustness analysis of the dynamical behaviour. As a first step, we provide a characterization of non-robust perturbations as a system of polynomial equalities and inequalities. In the second step, we apply the Positivstellensatz and Handelman representation of polynomials to check for the non-existence of solutions to this system, which can be relaxed to solving a linear program. Thereby, a solution to the linear program yields a robustness certificate for the considered dynamical behaviour. With these robustness certificates, we propose an algorithm to compute a lower robustness bound corresponding to a level of parametric uncertainty up to which no local bifurcations can occur. The applicability of the proposed method to biochemical network models is illustrated by analysing the robustness of oscillations in a model of the NF-kappa B signalling pathway. The results may be used to define a level of confidence in the observed model behaviour under parametric uncertainty, making them valuable for evaluating dynamical models of biological networks. C 2011 Elsevier Ltd. All rights reserved.</div>','<b>Title: </b>Assigning numbers to the arrows: Parameterizing a gene regulation network by using accurate expression kinetics<br><b>Year: </b>2002<br><b>Author: </b> M. Ronen,  R. Rosenberg,  Bi. Shraiman and  U. Alon<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1073/pnas.152046799"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA<br><b>Cluster-Keywords: </b><em>gene regulatory, gene expression, regulatory networks, expression data, regulatory pathway, regulatory genes, network dynamics, microarray data, genetic networks, transcription factor, models network, target gene, upstream regulatory, network topology, pathway network, gene functional, expression profile, transcriptional network, dynamic model, transcriptional regulatory<div style="font-size:11px;"><br><b>Abstract: </b><em>A basic challenge in systems biology is to understand the dynamical behavior of gene regulation networks. Current approaches aim at determining the network structure based on genomic-scale data. However, the network connectivity alone is not sufficient to define its dynamics; one needs to also specify the kinetic parameters for the regulation reactions. Here, we ask whether effective kinetic parameters can be assigned to a transcriptional network based on expression data. We present a combined experimental and theoretical approach based on accurate high temporal-resolution measurement of promoter activities from living cells by using green fluorescent protein GFP reporter plasmids. We present algorithms that use these data to assign effective kinetic parameters within a mathematical model of the network. To demonstrate this, we employ a well defined network, the SOS DNA repair system of Escherichia coli. We find a strikingly detailed temporal program of expression that correlates with the functional role of the SOS genes and is driven by a hierarchy of effective kinetic parameter strengths for the various promoters. The calculated parameters can be used to determine the kinetics of all SOS genes given the expression profile of just one representative, allowing a significant reduction in complexity. The concentration profile of the master SOS transcriptional repressor can be calculated, demonstrating that relative protein levels may be determined from purely transcriptional data. This finding opens the possibility of assigning kinetic parameters to transcriptional networks on a genomic scale.</div>','<b>Title: </b>TOPOLOGY AND DYNAMICS OF BOOLEAN NETWORKS WITH STRONG INHIBITION<br><b>Year: </b>2011<br><b>Author: </b> Yw. Rong,  C. Zeng,  C. Evans,  H. Chen and  Gy. Wang<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3934/dcdss.2011.4.1565"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>DISCRETE AND CONTINUOUS DYNAMICAL SYSTEMS-SERIES S<br><b>Cluster-Keywords: </b><em>network models, gene network, regulatory networks, boolean networks, gene expression, systems biology, neural networks, network dynamics, gene regulatory, biological networks, network structure, gap gene, gene regulation, cell cycle, network, gene interactions, signaling network, quantum systems, system dynamics, networks<div style="font-size:11px;"><br><b>Abstract: </b><em>A major challenge in systems biology is to understand interactions within biological systems. Such a system often consists of units with various levels of activities that evolve over time, mathematically represented by the dynamics of the system. The interaction between units is mathematically represented by the topology of the system. We carry out some mathematical analysis on the connections between topology and dynamics of such networks. We focus on a specific Boolean network model - the Strong Inhibition Model. This model defines a natural map from the space of all possible topologies on the network to the space of all possible dynamics on the same network. We prove this map is neither surjective nor injective. We introduce the notions of "redundant edges" and "dormant vertices" which capture the non-injectiveness of the map. Using these, we determine exactly when two different topologies yield the same dynamics and we provide an algorithm that determines all possible network solutions given a dynamics.</div>','<b>Title: </b>Bistability and oscillations in chemical reaction networks<br><b>Year: </b>2009<br><b>Author: </b> M. Domijan and  M. Kirkilionis<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s00285-008-0234-7"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF MATHEMATICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>biological system, network, pattern formation, system, network analysis, systems, turing patterns, patterns, reaction networks, networks, turing system, network topologies, stoichiometric network, networks pattern, network topology, biological, model, gradient formation, pattern, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Bifurcation theory is one of the most widely used approaches for analysis of dynamical behaviour of chemical and biochemical reaction networks. Some of the interesting qualitative behaviour that are analyzed are oscillations and bistability a situation where a system has at least two coexisting stable equilibria. Both phenomena have been identified as central features of many biological and biochemical systems. This paper, using the theory of stoichiometric network analysis SNA and notions from algebraic geometry, presents sufficient conditions for a reaction network to display bifurcations associated with these phenomena. The advantage of these conditions is that they impose fewer algebraic conditions on model parameters than conditions associated with standard bifurcation theorems. To derive the new conditions, a coordinate transformation will be made that will guarantee the existence of branches of positive equilibria in the system. This is particularly useful in mathematical biology, where only positive variable values are considered to be meaningful. The first part of the paper will be an extended introduction to SNA and algebraic geometry-related methods which are used in the coordinate transformation and set up of the theorems. In the second part of the paper we will focus on the derivation of bifurcation conditions using SNA and algebraic geometry. Conditions will be derived for three bifurcations: the saddle-node bifurcation, a simple branching point, both linked to bistability, and a simple Hopf bifurcation. The latter is linked to oscillatory behaviour. The conditions derived are sufficient and they extend earlier results from stoichiometric network analysis as can be found in Aguda and Clarke in J Chem Phys 87:3461-3470, 1987; Clarke and Jiang in J Chem Phys 99:4464-4476, 1993; Gatermann et al. in J Symb Comput 40:1361-1382, 2005. In these papers some necessary conditions for two of these bifurcations were given. A set of examples will illustrate that algebraic conditions arising from given sufficient bifurcation conditions are not more difficult to interpret nor harder to calculate than those arising from necessary bifurcation conditions. Hence an increasing amount of information is gained at no extra computational cost. The theory can also be used in a second step for a systematic bifurcation analysis of larger reaction networks.</div>','<b>Title: </b>Classification of transient behaviours in a time-dependent toggle switch model<br><b>Year: </b>2014<br><b>Author: </b> B. Verd,  A. Crombach and  J. Jaeger<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-8-43"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>gene regulatory, regulatory network, systems biology, cell differentiation, cell fate, model systems, potential landscape, network grn, dynamical systems, melanocyte differentiation, network models, gene expression, network dynamics, differentiation genes, dynamical models, dynamical networks, cellular differentiation, stem cells, attractor network, developmental gene<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Waddingtons epigenetic landscape is an intuitive metaphor for the developmental and evolutionary potential of biological regulatory processes. It emphasises time-dependence and transient behaviour. Nowadays, we can derive this landscape by modelling a specific regulatory network as a dynamical system and calculating its so-called potential surface. In this sense, potential surfaces are the mathematical equivalent of the Waddingtonian landscape metaphor. In order to fully capture the time-dependent non-autonomous transient behaviour of biological processes, we must be able to characterise potential landscapes and how they change over time. However, currently available mathematical tools focus on the asymptotic steady-state behaviour of autonomous dynamical systems, which restricts how biological systems are studied.Results: We present a pragmatic first step towards a methodology for dealing with transient behaviours in non-autonomous systems. We propose a classification scheme for different kinds of such dynamics based on the simulation of a simple genetic toggle-switch model with time-variable parameters. For this low-dimensional system, we can calculate and explicitly visualise numerical approximations to the potential landscape. Focussing on transient dynamics in non-autonomous systems reveals a range of interesting and biologically relevant behaviours that would be missed in steady-state analyses of autonomous systems. Our simulation-based approach allows us to identify four qualitatively different kinds of dynamics: transitions, pursuits, and two kinds of captures. We describe these in detail, and illustrate the usefulness of our classification scheme by providing a number of examples that demonstrate how it can be employed to gain specific mechanistic insights into the dynamics of gene regulation.Conclusions: The practical aim of our proposed classification scheme is to make the analysis of explicitly time-dependent transient behaviour tractable, and to encourage the wider use of non-autonomous models in systems biology. Our method is applicable to a large class of biological processes.</div>','<b>Title: </b>A Simple Computational Cell: Coupling Boolean Gene and Protein Networks<br><b>Year: </b>2012<br><b>Author: </b> L. Bull<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1162/artla00060"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ARTIFICIAL LIFE<br><b>Cluster-Keywords: </b><em>boolean networks, regulatory networks, network model, bayesian networks, model networks, systems biology, random boolean, real cells, immune system, genetic regulatory, networks, biochemical network, fluorescence images, cellular network, gene regulatory, cell differentiation, network, biological systems, ensemble approach, cell dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>This article presents an abstract, tunable model containing two of the principal information-processing features of cells and explores its use with simulated evolution. The random Boolean model of genetic regulatory networks is extended to include a protein interaction network. The underlying behavior of the resulting two coupled dynamical networks is investigated before their evolvability is explored using a version of the NK model of fitness landscapes.</div>','<b>Title: </b>C-elegans collectively forms dynamical networks<br><b>Year: </b>2019<br><b>Author: </b> T. Sugi,  H. Ito,  M. Nishimura and  Kh. Nagai<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1038/s41467-019-08537-y"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>NATURE COMMUNICATIONS<br><b>Cluster-Keywords: </b><em>interaction network, actin filament, network model, filament network, network structures, actin network, protein interaction, actin dynamics, network, membrane protein, network dynamics, model dynamic, cell, model, network approach, protein complex, computational model, actin, adhesion dynamics, cell dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>Understanding physical rules underlying collective motions requires perturbation of controllable parameters in self-propelled particles. However, controlling parameters in animals is generally not easy, which makes collective behaviours of animals elusive. Here, we report an experimental system in which a conventional model animal, Caenorhabditis elegans, collectively forms dynamical networks of bundle-shaped aggregates. We investigate the dependence of our experimental system on various extrinsic parameters material of substrate, ambient humidity and density of worms. Taking advantage of well-established C. elegans genetics, we also control intrinsic parameters genetically determined motility by mutations and by forced neural activation via optogenetics. Furthermore, we develop a minimal agent-based model that reproduces the dynamical network formation and its dependence on the parameters, suggesting that the key factors are alignment of worms after collision and smooth turning. Our findings imply that the concepts of active matter physics may help us to understand biological functions of animal groups.</div>','<b>Title: </b>Effect of Model Selection on Prediction of Periodic Behavior in Gene Regulatory Networks<br><b>Year: </b>2012<br><b>Author: </b> T. Gedeon,  G. Cummins and  Jj. Heys<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s11538-012-9732-2"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BULLETIN OF MATHEMATICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>gene regulatory, systems biology, synthetic biology, regulatory network, gene network, gene expression, system model, synthetic network, network dynamics, systems theory, synthetic gene, genetic network, model predicts, dynamical systems, time delays, translational time, regulation models, oscillatory expression, network structure, biological systems<div style="font-size:11px;"><br><b>Abstract: </b><em>One of the current challenges for cell biology is understanding of the system level cellular behavior from the knowledge of a network of the individual subcellular agents. We address a question of how the model selection affects the predicted dynamic behavior of a gene network. In particular, for a fixed network structure, we compare protein-only models with models in which each transcriptional activation is represented both by mRNA and protein concentrations. We compare linear behavior near equilibria for both cyclic feedback systems and a general system. We show that, in general, explicit inclusion of the mRNA in the model weakens the stability of equilibria. We also study numerically dynamics of a particular gene network and show significant differences in global dynamics between the two types of models.</div>','<b>Title: </b>Model checking optimal finite-horizon control for probabilistic gene regulatory networks<br><b>Year: </b>2017<br><b>Author: </b> O. Wei,  Zh. Guo,  Y. Niu and  Wy. Liao<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/s12918-017-0481-6"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>boolean networks, regulatory networks, genetic regulatory, gene regulatory, probabilistic boolean, network, networks, model, network model, probabilistic model, model checking, networks pbns, network dynamics, markov chain, regulatory, models, optimal control, boolean, networks models, networks boolean<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Probabilistic Boolean networks PBNs have been proposed for analyzing external control in gene regulatory networks with incorporation of uncertainty. A context-sensitive PBN with perturbation CS-PBNp, extending a PBN with context-sensitivity to reflect the inherent biological stability and random perturbations to express the impact of external stimuli, is considered to be more suitable for modeling small biological systems intervened by conditions from the outside. In this paper, we apply probabilistic model checking, a formal verification technique, to optimal control for a CS-PBNp that minimizes the expected cost over a finite control horizon.Results: We first describe a procedure of modeling a CS-PBNp using the language provided by a widely used probabilistic model checker PRISM. We then analyze the reward-based temporal properties and the computation in probabilistic model checking; based on the analysis, we provide a method to formulate the optimal control problem as minimum reachability reward properties. Furthermore, we incorporate control and state cost information into the PRISM code of a CS-PBNp such that automated model checking a minimum reachability reward property on the code gives the solution to the optimal control problem. We conduct experiments on two examples, an apoptosis network and a WNT5A network. Preliminary experiment results show the feasibility and effectiveness of our approach.Conclusions: The approach based on probabilistic model checking for optimal control avoids explicit computation of large-size state transition relations associated with PBNs. It enables a natural depiction of the dynamics of gene regulatory networks, and provides a canonical form to formulate optimal control problems using temporal properties that can be automated solved by leveraging the analysis power of underlying model checking engines. This work will be helpful for further utilization of the advances in formal verification techniques in system biology.</div>','<b>Title: </b>Elements of a network theory of complex adaptive systems<br><b>Year: </b>2011<br><b>Author: </b> Dg. GreenLink: No link<br><b>Journal: </b>INTERNATIONAL JOURNAL OF BIO-INSPIRED COMPUTATION<br><b>Cluster-Keywords: </b><em>information systems, gene expression, gene network, systems theory, system models, systems biology, systems information, network model, genetic network, mathematical models, dynamic systems, dynamic network, systemic model, network topology, information mechanisms, complex systems, systems complex, relational models, models main, information model<div style="font-size:11px;"><br><b>Abstract: </b><em>The need to understand and manage complex systems is increasing in importance, but complexity theory is still hampered by being highly fragmented in nature. This article argues that many elements for a general theory of complexity now exist and briefly reviews the main features. First, the universal nature of network model of complexity provides a suitable foundation for a general theory. A brief summary of the network model is followed by a discussion of related issues, including simulation, dynamics and self-organisation. Gaps identified include the need for formal methods to describe complexity and to identify structural equivalence. Finally, some important lessons from biology are summarised.</div>','<b>Title: </b>Quantitative modeling of triacylglycerol homeostasis in yeast - metabolic requirement for lipolysis to promote membrane lipid synthesis and cellular growth<br><b>Year: </b>2008<br><b>Author: </b> J. Zanghellini,  K. Natter,  C. Jungreuthmayer,  A. Thalhammer,  Cf. Kurat,  G. Gogg-Fassolter,  Sd. Kohlwein and  Hh. Von Grunberg<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1111/j.1742-4658.2008.06681.x"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>FEBS JOURNAL<br><b>Cluster-Keywords: </b><em>metabolic network, metabolic model, dynamic flux, growth rate, genomescale metabolic, flux balance, balance analysis, metabolic flux, metabolic pathways, dynamic model, metabolic reactions, cellular growth, regulatory network, systems biology, optimal growth, metabolic, modeling metabolic, growth conditions, dynamic analysis, cell growth<div style="font-size:11px;"><br><b>Abstract: </b><em>Triacylglycerol metabolism in Saccharomyces cerevisiae was analyzed quantitatively using a systems biological approach. Cellular growth, glucose uptake and ethanol secretion were measured as a function of time and used as input for a dynamic flux-balance model. By combining dynamic mass balances for key metabolites with a detailed steady-state analysis, we trained a model network and simulated the time-dependent degradation of cellular triacylglycerol and its interaction with fatty acid and membrane lipid synthesis. This approach described precisely, both qualitatively and quantitatively, the time evolution of various key metabolites in a consistent and self-contained manner, and the predictions were found to be in excellent agreement with experimental data. We showed that, during pre-logarithmic growth, lipolysis of triacylglycerol allows for the rapid synthesis of membrane lipids, whereas de novo fatty acid synthesis plays only a minor role during this growth phase. Progress in triacylglycerol hydrolysis directly correlates with an increase in cell size, demonstrating the importance of lipolysis for supporting efficient growth initiation.</div>','<b>Title: </b>From quiescence to proliferation: Cdk oscillations drive the mammalian cell cycle<br><b>Year: </b>2012<br><b>Author: </b> C. Gerard and  A. Goldbeter<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3389/fphys.2012.00413"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>FRONTIERS IN PHYSIOLOGY<br><b>Cluster-Keywords: </b><em>cell cycle, mammalian cell, cdk network, cell proliferation, cell, cells cell, regulatory network, detailed model, cell size, molecular network, skeleton model, cycle arrest, cycle progression, cell division, cell density, kinases cdks, cycle, phase transition, cycle control, network driving<div style="font-size:11px;"><br><b>Abstract: </b><em>We recently proposed a detailed model describing the dynamics of the network of cyclin-dependent kinases Cdks driving the mammalian cell cycle Gerard and Goldbeter, 2009. The model contains four modules, each centered around one cyclin/Cdk complex. Cyclin D/Cdk4-6 and cyclin E/Cdk2 promote progression in G1 and elicit the G1/S transition, respectively; cyclin A/Cdk2 ensures progression in S and the transition S/G2, while the activity of cyclin B/Cdk1 brings about the G2/M transition. This model shows that in the presence of sufficient amounts of growth factor the Cdk network is capable of temporal self-organization in the form of sustained oscillations, which correspond to the ordered, sequential activation of the various cyclin/Cdk complexes that control the successive phases of the cell cycle. The results suggest that the switch from cellular quiescence to cell proliferation corresponds to the transition from a stable steady state to sustained oscillations in the Cdk network. The transition depends on a finely tuned balance between factors that promote or hinder progression in the cell cycle. We show that the transition from quiescence to proliferation can occur in multiple ways that alter this balance. By resorting to bifurcation diagrams, we analyze the mechanism of oscillations in the Cdk network. Finally, we show that the complexity of the detailed model can be greatly reduced, without losing its key dynamical properties, by considering a skeleton model for the Cdk network. Using such a skeleton model for the mammalian cell cycle we show that positive feedback PF loops enhance the amplitude and the robustness of Cdk oscillations with respect to molecular noise. We compare the relative merits of the detailed and skeleton versions of the model for the Cdk network driving the mammalian cell cycle.</div>','<b>Title: </b>Metabolic dynamics in critically injured patients: a prospective cohort study integrated with 1H NMR metabolomics<br><b>Year: </b>2019<br><b>Author: </b> Yb. Zhou,  K. Wang,  J. Zeng,  W. Li,  J. Peng,  Zy. Zhou,  Pc. Deng,  Mw. Sun,  H. Yang,  Sj. Li,  Cd. Lu and  H. Jiang<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.6133/apjcn.201906282.0024"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ASIA PACIFIC JOURNAL OF CLINICAL NUTRITION<br><b>Cluster-Keywords: </b><em>inflammatory response, network model, mechanistic models, lde model, injured patients, critically injured, prediction model, tbi survivors, computational modeling, model abm, impaired astrocytes, model predicts, metabolic dynamics, significant differences, twoprocess model, agent-based model, coupled model, model, inflammatory network, computational models<div style="font-size:11px;"><br><b>Abstract: </b><em>Background and Objectives: By combining the techniques of metabolomics and computational biology, this research aims to explore the mechanism of metabolic dynamics in critically injured patients and develop a new early warning method for mortality. Methods and Study Design: A prospective cohort study was conducted, group plasma samples of critically injured patients were collected for 1H-NMR metabolomics analysis. The data was processed with partial least squares regression, to explore the role of enzyme-gene network regulatory mechanism in critically injured metabolic network regulation and to build a quantitative prediction model for early warning of fast death. Results: In total, 60 patients were enrolled. There were significant differences in plasma metabolome between the surviving patients and the deceased ones. Compared to the surviving patients, 112 enzymes and genes regulating the 6 key metabolic marker disturbances of neopterin, corticosterone, 3-methylhistidine, homocysteine, Serine, tyrosine, prostaglandin E2, tryptophan, testosterone and estriol, were observed in the plasmas of deceased ones. Among patients of different injury stages, there were significant differences in plasma metabolome. Progressing from T0 to T50 stages of injury, increased levels of neopterin, corticosterone, prostaglandin E2, tryptophan and testosterone, together with decreased levels of homocysteine, and estriol, were observed. Eventually, the quantitative prediction model of death warning was established. Cross-validation results showed that the predictive effect was good RMSE=0.18408, R-2=0.87 p=0.036. Conclusions: Metabolomics approaches can be used to quantify the metabolic dynamics of patients with critically injuries and to predict death of critically injured patients by plasma 1H-NMR metabolomics.</div>','<b>Title: </b>Specific ion effects in physicochemical and biological systems: Simulations, theory and experiments<br><b>Year: </b>2007<br><b>Author: </b> P. Koelsch,  P. Viswanath,  H. Motschmann,  Vl. Shapovalov,  G. Brezesinski,  H. Mohwald,  D. Horinek,  Rr. Netz,  K. Giewekemeyer,  Ts. Alditt,  H. Schollmeyer,  R. Von Klitzing,  J. Daillant and  P. Guenoun<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.colsurfa.2007.03.040"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>COLLOIDS AND SURFACES A-PHYSICOCHEMICAL AND ENGINEERING ASPECTS<br><b>Cluster-Keywords: </b><em>allosteric proteins, signaling networks, protein interactions, network model, molecular dynamics, protein complexes, allosteric network, signaling proteins, cell network, biochemical networks, free energy, combinatorial complexity, molecular species, biological systems, receptor signaling, systems biology, chemical species, model, dynamics simulations, cell signaling<div style="font-size:11px;"><br><b>Abstract: </b><em>Charged surfaces and ion-water interactions at an interface play a decisive role in many physico-chemical and biological processes. The classical treatment of ions at charged interfaces is the Poisson-Boltzmann PB theory. Despite severe simplifying assumptions it describes surprisingly well univalent ions not too close to the interface for low electrolyte concentrations in the mmol regime. However, it breaks down in the vicinity of the interface at higher surface charge densities. Consequently the list of decorations and modifications of the original PB equation is long aiming for a more realistic picture. One striking deficiency of the treatment on the pure electrostatic level is the prediction that ions of the same valence produce the same results, independent of their chemical nature. In contrast, experiments reveal pronounced differences between different ions. Specific ion effects can be found everywhere in chemistry and biology and there are many reports of pronounced differences in the properties of charged monolayers, micelles, vesicles, dispersions or polyelectrolyte multilayers using different identically charged counterions. The so-called "counterion effect" is usually discussed in terms of the Hofmeister series for cations or anions which are the result of a subtle balance of several competing evenly matched interactions. The complex interplay of electrostatics, dispersion forces, thermal motion, polarization, fluctuations, hydration, ion size effects and the impact of interfacial water structure makes it hard to identify a universal law. The diversity of specific ion effects is a direct consequence of this subtle interplay of forces and imposes a true challenge for the theories. The decisive information for an assessment of the theories is knowledge of the prevailing ion distribution. Hence a considerable amount of work has been applied to develop suitable model systems and to push surface characterization tools such as resonant X-ray reflectivity, total reflection X-ray fluorescence or X-ray standing waves to new limits. These techniques give direct information on the ions and on the interfacial architecture. A second alternative to complement these studies is infrared-visible sum frequency spectroscopy allowing to record surface vibrational spectra of the water as it is perturbed in the presence of the salts. The paper is organized in sections describing various facets of ion specific effects discussed within the network. c 2007 Elsevier B.V. All fights reserved.</div>','<b>Title: </b>A comprehensive workflow for general-purpose neural modeling with highly configurable neuromorphic hardware systems<br><b>Year: </b>2011<br><b>Author: </b> D. Bruderle,  Ma. Petrovici,  B. Vogginger,  M. Ehrlich,  T. Pfeil,  S. Millner,  A. Grubl,  K. Wendt,  E. Muller,  Mo. Schwartz,  Dh. De Oliveira,  S. Jeltsch,  J. Fieres,  M. Schilling,  P. Muller,  O. Breitwieser,  V. Petkov,  L. Muller,  Ap. Davison,  P. Krishnamurthy,  J. Kremkow,  M. Lundqvist,  E. Muller,  J. Partzsch,  S. Scholze,  L. Zuhl,  C. Mayr,  A. Destexhe,  M. Diesmann,  Tc. Potjans,  A. Lansner,  R. Schuffny,  J. Schemmel and  K. Meier<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s00422-011-0435-9"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOLOGICAL CYBERNETICS<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>In this article, we present a methodological framework that meets novel requirements emerging from upcoming types of accelerated and highly configurable neuromorphic hardware systems. We describe in detail a device with 45 million programmable and dynamic synapses that is currently under development, and we sketch the conceptual challenges that arise from taking this platform into operation. More specifically, we aim at the establishment of this neuromorphic system as a flexible and neuroscientifically valuable modeling tool that can be used by non-hardware experts. We consider various functional aspects to be crucial for this purpose, and we introduce a consistent workflow with detailed descriptions of all involved modules that implement the suggested steps: The integration of the hardware interface into the simulator-independent model description language PyNN; a fully automated translation between the PyNN domain and appropriate hardware configurations; an executable specification of the future neuromorphic system that can be seamlessly integrated into this biology-to-hardware mapping process as a test bench for all software layers and possible hardware design modifications; an evaluation scheme that deploys models from a dedicated benchmark library, compares the results generated by virtual or prototype hardware devices with reference software simulations and analyzes the differences. The integration of these components into one hardware-software workflow provides an ecosystem for ongoing preparative studies that support the hardware design process and represents the basis for the maturity of the model-to-hardware mapping software. The functionality and flexibility of the latter is proven with a variety of experimental results.</div>','<b>Title: </b>Monte-Carlo Modeling of the Central Carbon Metabolism of Lactococcus lactis: Insights into Metabolic Regulation<br><b>Year: </b>2014<br><b>Author: </b> E. Murabito,  M. Verma,  M. Bekker,  D. Bellomo,  Hv. Westerhoff,  B. Teusink and  R. Steuer<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0106453"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>kinetic models, metabolic networks, kinetic parameters, dynamic models, systems biology, network model, dynamic metabolic, biochemical network, rate constants, enzyme kinetics, kinetic data, adjustable parameters, metabolic model, parameter estimation, dynamic optimization, model selection, mathematical model, kinetic network, model response, model<div style="font-size:11px;"><br><b>Abstract: </b><em>Metabolic pathways are complex dynamic systems whose response to perturbations and environmental challenges are governed by multiple interdependencies between enzyme properties, reactions rates, and substrate levels. Understanding the dynamics arising from such a network can be greatly enhanced by the construction of a computational model that embodies the properties of the respective system. Such models aim to incorporate mechanistic details of cellular interactions to mimic the temporal behavior of the biochemical reaction system and usually require substantial knowledge of kinetic parameters to allow meaningful conclusions. Several approaches have been suggested to overcome the severe data requirements of kinetic modeling, including the use of approximative kinetics and Monte-Carlo sampling of reaction parameters. In this work, we employ a probabilistic approach to study the response of a complex metabolic system, the central metabolism of the lactic acid bacterium Lactococcus lactis, subject to perturbations and brief periods of starvation. Supplementing existing methodologies, we show that it is possible to acquire a detailed understanding of the control properties of a corresponding metabolic pathway model that is directly based on experimental observations. In particular, we delineate the role of enzymatic regulation to maintain metabolic stability and metabolic recovery after periods of starvation. It is shown that the feedforward activation of the pyruvate kinase by fructose-1,6-bisphosphate qualitatively alters the bifurcation structure of the corresponding pathway model, indicating a crucial role of enzymatic regulation to prevent metabolic collapse for low external concentrations of glucose. We argue that similar probabilistic methodologies will help our understanding of dynamic properties of small-, medium- and large-scale metabolic networks models.</div>','<b>Title: </b>Potential Landscape and Flux Framework of Nonequilibrium Biological Networks<br><b>Year: </b>2011<br><b>Author: </b> J. Wang<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/B978-0-444-53835-2.00001-8"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ANNUAL REPORTS IN COMPUTATIONAL CHEMISTRY, VOL 7<br><b>Cluster-Keywords: </b><em>gene regulatory, regulatory network, systems biology, cell differentiation, cell fate, model systems, potential landscape, network grn, dynamical systems, melanocyte differentiation, network models, gene expression, network dynamics, differentiation genes, dynamical models, dynamical networks, cellular differentiation, stem cells, attractor network, developmental gene<div style="font-size:11px;"><br><b>Abstract: </b><em>We established a theoretical framework for studying nonequilibrium networks with two distinct natures essential for characterizing the global probabilistic dynamics: the underlying potential landscape and the corresponding curl flux. We applied the idea to two biological examples: biochemical oscillation network and stem-cell fate decision network for differentiation and development.We found that the underlying potential landscape for the oscillation limit cycle has a distinct closed ring valley Mexican hat-like shape when the fluctuations are small. This global landscape structure leads to attractions of the system to the ring valley. On the ring, we found that the nonequilibrium flux is the driving force for oscillations. Therefore, both structured landscape and flux are needed to guarantee a robust oscillating network. The barrier height separating the oscillation ring and other areas derived from the landscape topography is shown to be correlated with the escaping time from the limit cycle attractor and provides a quantitative measure of the robustness for the network. The landscape becomes shallower and the closed ring valley shape structure becomes weaker lower barrier height with larger fluctuations. We observe that the period and the amplitude of the oscillations are more dispersed and oscillations become less coherent when the fluctuations increase. We also found that the entropy production of the whole network, characterizing the dissipation costs from the combined effects of both landscapes and fluxes, decreases when the fluctuations decrease. Therefore, less dissipation leads to more robust networks. Our approach is quite general and applicable to other networks, dynamical systems, and biological evolution. It can help designing robust networks.Differentiation from a multipotent stem or progenitor state to a mature cell is an essential irreversible process. The associated changes in gene expression patterns exhibit time directionality. This "arrow of time" in the collective change of gene expression across multiple stable gene expression patterns attractors is neither explained by the regulated activation, or suppression of individual genes, which are bidirectional molecular processes, nor by standard dynamical models of the underlying gene circuit which only account for local stability of attractors. To capture the global dynamics of this nonequilibrium system and gain insight in the time asymmetry of state transitions, we computed the quasipotential landscape of the stochastic dynamics of a canonical gene circuit that governs branching cell fate commitment. The potential landscape reveals the global dynamics and permits the calculation of "potential barriers" between cell phenotypes imposed by the circuit architecture. The generic asymmetry of barrier heights indicates that the transition from the uncommitted multipotent state to differentiated states is inherently unidirectional. The model agrees with observations and predicts the extreme conditions for reprogramming cells back to the undifferentiated state.</div>','<b>Title: </b>A generalized framework for Network Component Analysis<br><b>Year: </b>2005<br><b>Author: </b> R. Boscolo,  C. Sabatti,  Jc. Liao and  Vp. Roychowdhury<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1109/TCBB.2005.47"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IEEE-ACM TRANSACTIONS ON COMPUTATIONAL BIOLOGY AND BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>gene regulatory, gene expression, regulatory networks, expression data, regulatory pathway, regulatory genes, network dynamics, microarray data, genetic networks, transcription factor, models network, target gene, upstream regulatory, network topology, pathway network, gene functional, expression profile, transcriptional network, dynamic model, transcriptional regulatory<div style="font-size:11px;"><br><b>Abstract: </b><em>The authors recently introduced a framework, named Network Component Analysis NCA, for the reconstruction of the dynamics of transcriptional regulators activities from gene expression assays. The original formulation had certain shortcomings that limited NCAS application to a wide class of network dynamics reconstruction problems, either because of limitations in the sample size or because of the stringent requirements imposed by the set of identifiability conditions. In addition, the performance characteristics of the method for various levels of data noise or in the presence of model inaccuracies were never investigated. In this article, the following aspects of NCA have been addressed, resulting in a set of extensions to the original framework: 1 The sufficient conditions on the a priori connectivity information required for successful reconstructions via NCA are made less stringent, allowing easier verification of whether a network topology is identifiable, as well as extending the class of identifiable systems. Such a result is accomplished by introducing a set of identifiability requirements that can be directly tested on the regulatory architecture, rather than on specific instances of the system matrix. 2 The two-stage least square iterative procedure used in NCA is proven to identify stationary points of the likelihood function, under Gaussian noise assumption, thus reinforcing the statistical foundations of the method. 3 A framework for the simultaneous reconstruction of multiple regulatory subnetworks is introduced, thus overcoming one of the critical limitations of the original formulation of the decomposition, for example, occurring for poorly sampled data typical of microarray experiments. A set of monte carlo simulations we conducted with synthetic data suggests that the approach is indeed capable of accurately reconstructing regulatory signals when these are the input of large-scale networks that satisfy the suggested identifiability criteria, even under fairly noisy conditions. The sensitivity of the reconstructed signals to inaccuracies in the hypothesized network topology is also investigated. We demonstrate the feasibility of our approach for the simultaneous reconstruction of multiple regulatory subnetworks from the same data set with a successful application of the technique to gene expression measurements of the bacterium Escherichia coli.</div>','<b>Title: </b>Automated refinement and inference of analytical models for metabolic networks<br><b>Year: </b>2011<br><b>Author: </b> Md. Schmidt,  Rr. Vallabhajosyula,  Jw. Jenkins,  Je. Hood,  As. Soni,  Jp. Wikswo and  H. Lipson<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1088/1478-3975/8/5/055011"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PHYSICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>immune system, network models, immune response, model, biological systems, models, model reduction, dynamic model, biological networks, system, mathematical models, biochemical network, response model, systems, systems biology, network, immune, biology model, reaction networks, system systems<div style="font-size:11px;"><br><b>Abstract: </b><em>The reverse engineering of metabolic networks from experimental data is traditionally a labor-intensive task requiring a priori systems knowledge. Using a proven model as a test system, we demonstrate an automated method to simplify this process by modifying an existing or related model-suggesting nonlinear terms and structural modifications-or even constructing a new model that agrees with the systems time series observations. In certain cases, this method can identify the full dynamical model from scratch without prior knowledge or structural assumptions. The algorithm selects between multiple candidate models by designing experiments to make their predictions disagree. We performed computational experiments to analyze a nonlinear seven-dimensional model of yeast glycolytic oscillations. This approach corrected mistakes reliably in both approximated and overspecified models. The method performed well to high levels of noise for most states, could identify the correct model de novo, and make better predictions than ordinary parametric regression and neural network models. We identified an invariant quantity in the model, which accurately derived kinetics and the numerical sensitivity coefficients of the system. Finally, we compared the system to dynamic flux estimation and discussed the scaling and application of this methodology to automated experiment design and control in biological systems in real time.</div>','<b>Title: </b>Parameter estimation and identifiability in a neural population model for electro-cortical activity<br><b>Year: </b>2019<br><b>Author: </b> A. Hartoyo,  Pj. Cadusch,  Dtj. Liley and  Dg. Hicks<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pcbi.1006694"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS COMPUTATIONAL BIOLOGY<br><b>Cluster-Keywords: </b><em>network models, population model, field model, neural population, neural field, iot networks, neuronal networks, neural network, differential equations, parameter neural, lotkavolterra model, model, neuron models, model neurons, model leads, ecosystem model, iot devices, noise model, neuronal models, inhibitory synaptic<div style="font-size:11px;"><br><b>Abstract: </b><em>Electroencephalography EEG provides a non-invasive measure of brain electrical activity. Neural population models, where large numbers of interacting neurons are considered collectively as a macroscopic system, have long been used to understand features in EEG signals. By tuning dozens of input parameters describing the excitatory and inhibitory neuron populations, these models can reproduce prominent features of the EEG such as the alpha-rhythm. However, the inverse problem, of directly estimating the parameters from fits to EEG data, remains unsolved. Solving this multi-parameter non-linear fitting problem will potentially provide a real-time method for characterizing average neuronal properties in human subjects. Here we perform unbiased fits of a 22-parameter neural population model to EEG data from 82 individuals, using both particle swarm optimization and Markov chain Monte Carlo sampling. We estimate how much is learned about individual parameters by computing Kullback-Leibler divergences between posterior and prior distributions for each parameter. Results indicate that only a single parameter, that determining the dynamics of inhibitory synaptic activity, is directly identifiable, while other parameters have large, though correlated, uncertainties. We show that the eigenvalues of the Fisher information matrix are roughly uniformly spaced over a log scale, indicating that the model is sloppy, like many of the regulatory network models in systems biology. These eigenvalues indicate that the system can be modeled with a low effective dimensionality, with inhibitory synaptic activity being prominent in driving system behavior.Author summary Electroencephalography EEG, where electrodes are used to measure electric potential on the outside of the scalp, provides a simple, non-invasive way to study brain activity. Physiological interpretation of features in EEG signals has often involved use of collective models of neural populations. These neural population models have dozens of input parameters to describe the properties of inhibitory and excitatory neurons. Being able to estimate these parameters by direct fits to EEG data holds the promise of providing a real-time non-invasive method of inferring neuronal properties in different individuals. However, it has long been impossible to fit these nonlinear, multi-parameter models effectively. Here we describe fits of a 22-parameter neural population model to EEG spectra from 82 different subjects, all exhibiting alpha-oscillations. We show how only one parameter, that describing inhibitory dynamics, is constrained by the data, although all parameters are correlated. These results indicate that inhibitory synaptic activity plays a central role in the generation and modulation of the alpha-rhythm in humans.</div>','<b>Title: </b>Contractility in an extensile system<br><b>Year: </b>2017<br><b>Author: </b> Kt. Stanhope,  V. Yadav,  Cd. Santangelo and  Jl. Ross<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1039/c7sm00449d"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SOFT MATTER<br><b>Cluster-Keywords: </b><em>interaction network, actin filament, network model, filament network, network structures, actin network, protein interaction, actin dynamics, network, membrane protein, network dynamics, model dynamic, cell, model, network approach, protein complex, computational model, actin, adhesion dynamics, cell dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>Essentially all biology is active and dynamic. Biological entities autonomously sense, compute, and respond using energy-coupled ratchets that can produce force and do work. The cytoskeleton, along with its associated proteins and motors, is a canonical example of biological active matter, which is responsible for cargo transport, cell motility, division, and morphology. Prior work on cytoskeletal active matter systems showed either extensile or contractile dynamics. Here, we demonstrate a cytoskeletal system that can control the direction of the network dynamics to be either extensile, contractile, or static depending on the concentration of filaments or weak, transient crosslinkers through systematic variation of the crosslinker or microtubule concentrations. Based on these new observations and our previously published results, we created a simple one-dimensional model of the interaction of filaments within a bundle. Despite its simplicity, our model recapitulates the observed activities of our experimental system, implying that the dynamics of our finite networks of bundles are driven by the local filament-filament interactions within the bundle. Finally, we show that contractile phases can result in autonomously motile networks that resemble cells. Our results reveal a fundamentally important aspect of cellular self-organization: weak, transient interacting species can tune their interaction strength directly by tuning the local concentration to act like a rheostat. In this case, when the weak, transient proteins crosslink microtubules, they can tune the dynamics of the network to change from extensile to contractile to static. Our experiments and model allow us to gain a deeper understanding of cytoskeletal dynamics and provide an new understanding of the importance of weak, transient interactions to soft and biological systems.</div>','<b>Title: </b>The concepts of emergent and collective properties in individual-based models - Summary and outlook of the Bornhoved case studies<br><b>Year: </b>2005<br><b>Author: </b> H. Reuter,  F. Holker,  U. Middelhoff,  F. Jopp,  C. Eschenbach and  B. Breckling<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.ecolmodel.2005.02.014"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ECOLOGICAL MODELLING<br><b>Cluster-Keywords: </b><em>network ecology, food webs, network models, ecological systems, food web, systems biology, ecological networks, ecosystem model, signalling networks, model results, interaction networks, dynamic model, ecological interactions, network analysis, model network, network structure, transport networks, systems network, ecosystem dynamics, restoration ecology<div style="font-size:11px;"><br><b>Abstract: </b><em>Ecology requires the conceptual and technical ability to analyse complex and dynamic systems consisting of a high and variable number of components and relations. These components are part of a variable interaction structure in a spatially heterogeneous context. The components of ecological interaction networks can give rise to self-organised, and scale-dependent interaction patterns and processes, which are the underlying causes of the overall ecological systems states.The individual-based modelling approach provides a widely applicable simulation framework based on a hierarchy theory view of ecological systems.Here, we summarise and generalise the theoretical implications of the modelling studies presented in this volume in the field of terrestrial and aquatic, animal and plant ecology. The case studies cover a representative profile of processes related to ecological applications, such as food web interactions, population dynamics, dispersal, energy physiology, nutrient allocation and mutual impact of morphological and physiological development. The generic approach applied in this context allows a hierarchical representation of ecological systems and their components. Model results are obtained as self-organised structural relation networks and as aggregated quantitative states. In order to address different model characteristics we distinguish collective and emergent properties. Collective properties are those that are attributed equally to different organisation levels of the system. Emergent properties result from the activities of lower level entities on a higher organisation level, while not being present on the lower level. They can be subdivided into aggregational and connective properties. Emergent properties that are aggregational are those which emerge as a result of an aggregation procedure by an observer on the higher level which does not make sense or is not applicable on lower levels. Emergent properties that are connective, however, are based on an interaction network of lower level entities, which brings about the specific system characteristic.This classification of model results will allow to generalise the achievements and potential of the individual-based modelling approach in ecology. c 2005 Elsevier B.V. All rights reserved.</div>','<b>Title: </b>Bioluminescent Assays for Glucose and Glutamine Metabolism: High-Throughput Screening for Changes in Extracellular and Intracellular Metabolites<br><b>Year: </b>2017<br><b>Author: </b> D. Leippe,  M. Sobol,  G. Vidugiris,  Jj. Cali and  J. Vidugiriene<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1177/1087057116675612"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SLAS DISCOVERY<br><b>Cluster-Keywords: </b><em>chaotic system, control systems, neural networks, jerk chaotic, torch chaotic, mode control, sliding mode, plasma torch, dynamic neural, mitochondrial dynamics, system, systems biology, adaptive integral, model system, rodtype plasma, integral sliding, systems, jerk system, cancer cell, comparison method<div style="font-size:11px;"><br><b>Abstract: </b><em>Cancer cell metabolism is a complex, dynamic network of regulated pathways. Interrogation of this network would benefit from rapid, sensitive techniques that are adaptable to high-throughput formats, facilitating novel compound screening. This requires assays that have minimal sample preparation and are adaptable to lower-volume 384-well formats and automation. Here we describe bioluminescent glucose, lactate, glutamine, and glutamate detection assays that are well suited for high-throughput analysis of two major metabolic pathways in cancer cells: glycolysis and glutaminolysis. The sensitivity 1-5 pmol/sample, broad linear range 0.1-100 mu M, and wide dynamic range > 100-fold are advantageous for measuring both extracellular and intracellular metabolites. Importantly, the assays incorporate rapid inactivation of endogenous enzymes, eliminating deproteinization steps required by other methods. Using ovarian cancer cell lines as a model system, the assays were used to monitor changes in glucose and glutamine consumption and lactate and glutamate secretion over time. Homogeneous formats of the lactate and glutamate assays were robust Z = 0.6-0.9 and could be multiplexed with a real-time viability assay to generate internally controlled data. Screening a small-compound library with these assays resulted in the identification of both inhibitors and activators of lactate and glutamate production.</div>','<b>Title: </b>Importance of metabolic coupling for the dynamics of gene expression following a diauxic shift in Escherichia coli<br><b>Year: </b>2012<br><b>Author: </b> V. Baldazzi,  D. Ropers,  J. Geiselmann,  D. Kahn and  H. De Jong<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.jtbi.2011.11.010"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF THEORETICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>regulatory networks, network dynamics, gene regulatory, network model, genetic network, gene networks, genetic regulatory, mathematical model, parameter space, network system, interaction networks, system dynamics, network structure, network, dynamic networks, network reduction, models models, biological networks, network motifs, biological systems<div style="font-size:11px;"><br><b>Abstract: </b><em>Gene regulatory networks consist of direct interactions, but also include indirect interactions mediated by metabolism. We investigate to which extent these indirect interactions arising from metabolic coupling influence the dynamics of the system. To this end, we build a qualitative model of the gene regulatory network controlling carbon assimilation in Escherichia coli, and use this model to study the changes in gene expression following a diauxic shift from glucose to acetate. In particular, we compare the relative variation in the steady-state concentrations of enzymes and transcription regulators during growth on glucose and acetate, as well as the dynamic response of gene expression to the exhaustion of glucose and the subsequent assimilation of acetate. We find significant differences between the dynamics of the system in the absence and presence of metabolic coupling. This shows that interactions arising from metabolic coupling cannot be ignored when studying the dynamics of gene regulatory networks. C 2011 Elsevier Ltd. All rights reserved.</div>','<b>Title: </b>Context-dependent regulation of receptor tyrosine kinases: Insights from systems biology approaches<br><b>Year: </b>2019<br><b>Author: </b> I. Lam,  Cm. Pickering and  F. Mac Gabhann<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1002/wsbm.1437"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>WILEY INTERDISCIPLINARY REVIEWS-SYSTEMS BIOLOGY AND MEDICINE<br><b>Cluster-Keywords: </b><em>signaling network, systems biology, signal transduction, signaling pathways, receptor tyrosine, erbb signaling, erbb receptor, receptor signaling, intracellular signaling, mathematical models, cell signaling, computational model, factor receptor, receptor activation, growth factor, signaling models, networks signaling, receptor system, receptors systems, downstream signaling<div style="font-size:11px;"><br><b>Abstract: </b><em>Receptor tyrosine kinases RTKs are cell membrane proteins that provide cells with the ability to sense proteins in their environments. Many RTKs are essential to development and organ growth. Derangement of RTKs-by mutation or by overexpression-is central to several developmental and adult disorders including cancer, short stature, and vascular pathologies. The mechanism of action of RTKs is complex and is regulated by contextual components, including the existence of multiple competing ligands and receptors in many families, the intracellular location of the RTK, the dynamic and cell-specific coexpression of other RTKs, and the commonality of downstream signaling pathways. This means that both the state of the cell and the microenvironment outside the cell play a role, which makes sense given the pivotal location of RTKs as the nexus linking the extracellular milieu to intracellular signaling and modification of cell behavior. In this review, we describe these different contextual components through the lens of systems biology, in which both computational modeling and experimental "omics" approaches have been used to better understand RTK networks. The complexity of these networks is such that using these systems biology approaches is necessary to get a handle on the mechanisms of pathology and the design of therapeutics targeting RTKs. In particular, we describe in detail three concrete examples involving ErbB3, VEGFR2, and AXL that illustrate how systems approaches can reveal key mechanistic and therapeutic insights. This article is categorized under: Biological Mechanisms > Cell Signaling Models of Systems Properties and Processes > Mechanistic Models Translational, Genomic, and Systems Medicine > Therapeutic Methods</div>','<b>Title: </b>Characterization of a quorum sensing device for synthetic biology design: Experimental and modeling validation<br><b>Year: </b>2013<br><b>Author: </b> N. Saeidi,  M. Arshath,  Mw. Chang and  Cl. Poh<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.ces.2012.12.016"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>CHEMICAL ENGINEERING SCIENCE<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Modeling of biological parts is of crucial importance as it enables the in silico study of synthetic biological systems prior to the actual construction of genetic circuits, which can be time consuming and costly. Because standard biological parts are utilized to build the synthetic systems, it is important that each of these standard parts is well characterized and has a corresponding mathematical model that could simulate the characteristics of the part. These models could be used in computer aided design CAD tools during the design stage to facilitate the building of the model of biological systems. This paper describes the development of a mathematical model that is able to simulate both the dynamic and static performance of a biological device created using standard parts. We modeled an example quorum sensing device that produces green fluorescent protein GM as reporter in the presence of Acyl Homoserine Lactone AHL. The parameters of the model were estimated using experimental results. The simulation results show that the model was able to simulate behavior similar to experimental results. Since it is important that these models and the content in the models can be searchable and readable by machines, standard SBML system biology markup language format was used to store the models. All parts and reactions are fully annotated to enable easy searching, and the models follow the Minimum Information Requested In the Annotation of Models MIRIAM compliance as well as the Minimum Information About a Simulation Experiment MIASE. C 2012 Elsevier Ltd. All rights reserved.</div>','<b>Title: </b>Simulation of the Dynamics of Bacterial Quorum Sensing<br><b>Year: </b>2015<br><b>Author: </b> Ai. Psarras and  Ig. Karafyllidis<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1109/TNB.2014.2385109"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IEEE TRANSACTIONS ON NANOBIOSCIENCE<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Quorum sensing QS is a signaling mechanism that pathogenic bacteria use to communicate and synchronize the production of exofactors to attack their hosts. Understanding and controlling QS is an important step towards a possible solution to the growing problem of antibiotic resistance. QS is a cooperative effort of a bacterial population in which some of the bacteria do not participate. This phenomenon is usually studied using game theory and the non-participating bacteria are modeled as cheaters that exploit the production of common goods exofactors by other bacteria. Here, we take a different approach to study the QS dynamics of a growing bacterial population. We model the bacterial population as a growing graph and use spectral graph theory to compute the evolution of its synchronizability. We also treat each bacterium as a source of signaling molecules and use the diffusion equation to compute the signaling molecule distribution. We formulate a cost function based on Lagrangian dynamics that combines the time-like synchronization with the space-like diffusion of signaling molecules. Our results show that the presence of non-participating bacteria improves the homogeneity of the signaling molecule distribution preventing thus an early onset of exofactor production and has a positive effect on the optimization of QS signaling and on attack synchronization.</div>','<b>Title: </b>Colored Petri net modeling and simulation of signal transduction pathways<br><b>Year: </b>2006<br><b>Author: </b> Dy. Lee,  R. Zimmer,  Sy. Lee and  S. Park<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.ymben.2005.10.001"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>METABOLIC ENGINEERING<br><b>Cluster-Keywords: </b><em>petri net, petri nets, bayesian networks, biological networks, systems biology, regulatory networks, dynamic bayesian, net model, gene regulatory, network data, biological systems, biological data, network analysis, network models, biochemical networks, time series, network structure, signal transduction, net models, dbn model<div style="font-size:11px;"><br><b>Abstract: </b><em>Presented herein is a methodology for quantitatively analyzing the complex signaling network by resorting to colored Petri nets CPN. The mathematical as well as Petri net models for two basic reaction types were established, followed by the extension to a large signal transduction system stimulated by epidermal growth factor EGF in an application study. The CPN models based on the Petri net representation and the conservation and kinetic equations were used to examine the dynamic behavior of the EGF signaling pathway. The usefulness of Petri nets is demonstrated for the quantitative analysis of the signal transduction pathway. Moreover, the trade-offs between modeling capability and simulation efficiency of this pathway are explored, suggesting that the Petri net model can be invaluable in the initial stage of building a dynamic model. c 2005 Elsevier Inc. All rights reserved.</div>','<b>Title: </b>SBML qualitative models: a model representation format and infrastructure to foster interactions between qualitative modelling formalisms and tools<br><b>Year: </b>2013<br><b>Author: </b> C. Chaouiya,  D. Berenguier,  Sm. Keating,  A. Naldi,  Mp. Van Iersel,  N. Rodriguez,  A. Drager,  F. Buchel,  T. Cokelaer,  B. Kowal,  B. Wicks,  E. Goncalves,  J. Dorier,  M. Page,  Pt. Monteiro,  A. Von Kamp,  I. Xenarios,  H. De Jong,  M. Hucka,  S. Klamt,  D. Thieffry,  N. Le Novere,  J. Saez-Rodriguez and  T. Helikar<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-7-135"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>boolean network, network models, qualitative models, systems biology, mathematical modeling, boolean models, signaling networks, dynamic models, dynamic modeling, signaling dynamics, logic models, dynamic network, mathematical models, boolean modeling, signal transduction, regulatory networks, drug resistance, biological networks, network approach, network modeling<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Qualitative frameworks, especially those based on the logical discrete formalism, are increasingly used to model regulatory and signalling networks. A major advantage of these frameworks is that they do not require precise quantitative data, and that they are well-suited for studies of large networks. While numerous groups have developed specific computational tools that provide original methods to analyse qualitative models, a standard format to exchange qualitative models has been missing.Results: We present the Systems Biology Markup Language SBML Qualitative Models Package "qual", an extension of the SBML Level 3 standard designed for computer representation of qualitative models of biological networks. We demonstrate the interoperability of models via SBML qual through the analysis of a specific signalling network by three independent software tools. Furthermore, the collective effort to define the SBML qual format paved the way for the development of LogicalModel, an open-source model library, which will facilitate the adoption of the format as well as the collaborative development of algorithms to analyse qualitative models.Conclusions: SBML qual allows the exchange of qualitative models among a number of complementary software tools. SBML qual has the potential to promote collaborative work on the development of novel computational approaches, as well as on the specification and the analysis of comprehensive qualitative models of regulatory and signalling networks.</div>','<b>Title: </b>Optimal regulatory strategies for metabolic pathways in Escherichia coli depending on protein costs<br><b>Year: </b>2011<br><b>Author: </b> F. Wessely,  M. Bartl,  R. Guthke,  P. Li,  S. Schuster and  C. Kaleta<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1038/msb.2011.46"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MOLECULAR SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>While previous studies have shed light on the link between the structure of metabolism and its transcriptional regulation, the extent to which transcriptional regulation controls metabolism has not yet been fully explored. In this work, we address this problem by integrating a large number of experimental data sets with a model of the metabolism of Escherichia coli. Using a combination of computational tools including the concept of elementary flux patterns, methods from network inference and dynamic optimization, we find that transcriptional regulation of pathways reflects the protein investment into these pathways. While pathways that are associated to a high protein cost are controlled by fine-tuned transcriptional programs, pathways that only require a small protein cost are transcriptionally controlled in a few key reactions. As a reason for the occurrence of these different regulatory strategies, we identify an evolutionary trade-off between the conflicting requirements to reduce protein investment and the requirement to be able to respond rapidly to changes in environmental conditions. Molecular Systems Biology 7: 515; published online 19 July 2011; doi:10.1038/msb.2011.46</div>','<b>Title: </b>Sensitivity and Robustness Analysis for Stochastic Model of Nanog Gene Regulatory Network<br><b>Year: </b>2015<br><b>Author: </b> Qq. Wu,  F. Jiang and  Th. Tian<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1142/S021812741540009X"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>INTERNATIONAL JOURNAL OF BIFURCATION AND CHAOS<br><b>Cluster-Keywords: </b><em>stem cell, stem cells, systems biology, mathematical models, cell biology, network model, signaling pathways, cell fate, computational models, cancer cells, gene expression, network dynamics, embryonic stem, signaling network, prostate cancer, mathematical modeling, psychological stress, apoptosis signaling, computational systems, molecular mechanisms<div style="font-size:11px;"><br><b>Abstract: </b><em>The advances of systems biology have raised a large number of mathematical models for exploring the dynamic property of biological systems. A challenging issue in mathematical modeling is how to study the influence of parameter variation on system property. Robustness and sensitivity are two major measurements to describe the dynamic property of a system against the variation of model parameters. For stochastic models of discrete chemical reaction systems, although these two properties have been studied separately, no work has been done so far to investigate these two properties together. In this work, we propose an integrated framework to study these two properties for a biological system simultaneously. We also consider a stochastic model with intrinsic noise for the Nanog gene network based on a published model that studies extrinsic noise only. For the stochastic model of Nanog gene network, we identify key coefficients that have more influence on the network dynamics than the others through sensitivity analysis. In addition, robustness analysis suggests that the model parameters can be classified into four types regarding the bistability property of Nanog expression levels. Numerical results suggest that the proposed framework is an efficient approach to study the sensitivity and robustness properties of biological network models.</div>','<b>Title: </b>Parameter estimation in tree graph metabolic networks<br><b>Year: </b>2016<br><b>Author: </b> L. Astola,  H. Stigter,  Mvg. Roldan,  F. Van Eeuwijk,  Rd. Hall,  M. Groenenboom and  Jj. Molenaar<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.7717/peerj.2417"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PEERJ<br><b>Cluster-Keywords: </b><em>kinetic models, metabolic networks, kinetic parameters, dynamic models, systems biology, network model, dynamic metabolic, biochemical network, rate constants, enzyme kinetics, kinetic data, adjustable parameters, metabolic model, parameter estimation, dynamic optimization, model selection, mathematical model, kinetic network, model response, model<div style="font-size:11px;"><br><b>Abstract: </b><em>We study the glycosylation processes that convert initially toxic substrates to nutritionally valuable metabolites in the flavonoid biosynthesis pathway of tomato Solanum lycopersicum seedlings. To estimate the reaction rates we use ordinary differential equations ODEs to model the enzyme kinetics. A popular choice is to use a system of linear ODEs with constant kinetic rates or to use MichaelisMenten kinetics. In reality, the catalytic rates, which are affected among other factors by kinetic constants and enzyme concentrations, are changing in time and with the approaches just mentioned, this phenomenon cannot be described. Another problem is that, in general these kinetic coefficients are not always identifiable. A third problem is that, it is not precisely known which enzymes are catalyzing the observed glycosylation processes. With several hundred potential gene candidates, experimental validation using purified target proteins is expensive and time consuming. We aim at reducing this task via mathematical modeling to allow for the pre-selection of most potential gene candidates. In this article we discuss a fast and relatively simple approach to estimate time varying kinetic rates, with three favorable properties: firstly, it allows for identifiable estimation of time dependent parameters in networks with a tree-like structure. Secondly, it is relatively fast compared to usually applied methods that estimate the model derivatives together with the network parameters. Thirdly, by combining the metabolite concentration data with a corresponding microarray data, it can help in detecting the genes related to the enzymatic processes. By comparing the estimated time dynamics of the catalytic rates with time series gene expression data we may assess potential candidate genes behind enzymatic reactions. As an example, we show how to apply this method to select prominent glycosyltransferase genes in tomato seedlings.</div>','<b>Title: </b>Network dynamics determine the autocrine and paracrine signaling functions of TNF<br><b>Year: </b>2014<br><b>Author: </b> Ab. Caldwell,  Z. Cheng,  Jd. Vargas,  Ha. Birnbaum and  A. Hoffmann<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1101/gad.244749.114"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>GENES  DEVELOPMENT<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>A hallmark of the inflammatory response to pathogen exposure is the production of tumor necrosis factor TNF that coordinates innate and adaptive immune responses by functioning in an autocrine or paracrine manner. Numerous molecular mechanisms contributing to TNF production have been identified, but how they function together in macrophages remains unclear. Here, we pursued an iterative systems biology approach to develop a quantitative understanding of the regulatory modules that control TNF mRNA synthesis and processing, mRNA half-life and translation, and protein processing and secretion. By linking the resulting model of TNF production to models of the TLR-, the TNFR-, and the NF kappa B signaling modules, we were able to study TNFs functions during the inflammatory response to diverse TLR agonists. Contrary to expectation, we predicted and then experimentally confirmed that in response to lipopolysaccaride, TNF does not have an autocrine function in amplifying the NF kappa B response, although it plays a potent paracrine role in neighboring cells. However, in response to CpG DNA, autocrine TNF extends the duration of NF kappa B activity and shapes CpG-induced gene expression programs. Our systems biology approach revealed that network dynamics of MyD88 and TRIF signaling and of cytokine production and response govern the stimulus-specific autocrine and paracrine functions of TNF.</div>','<b>Title: </b>Temporal Perturbation of Tyrosine Phosphoproteome Dynamics Reveals the System-wide Regulatory Networks<br><b>Year: </b>2009<br><b>Author: </b> M. Oyama,  H. Kozuka-Hata,  S. Tasaki,  K. Semba,  S. Hattori,  S. Sugano,  J. Inoue and  T. Yamamoto<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1074/mcp.M800186-MCP200"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MOLECULAR  CELLULAR PROTEOMICS<br><b>Cluster-Keywords: </b><em>signaling network, systems biology, signal transduction, signaling pathways, receptor tyrosine, erbb signaling, erbb receptor, receptor signaling, intracellular signaling, mathematical models, cell signaling, computational model, factor receptor, receptor activation, growth factor, signaling models, networks signaling, receptor system, receptors systems, downstream signaling<div style="font-size:11px;"><br><b>Abstract: </b><em>Signal transduction systems are known to widely regulate complex biological events such as cell proliferation and differentiation. Because phosphotyrosine-dependent networks play a key role in transmitting signals, a comprehensive and fine description of their dynamic behavior can lead us to systematically analyze the regulatory mechanisms that result in each biological effect. Here we established a mass spectrometry-based framework for analyzing tyrosine phosphoproteome dynamics through temporal network perturbation. A highly time-resolved description of the epidermal growth factor-dependent signaling pathways in human A431 cells revealed a global view of their multiphase network activation, comprising a spike signal transmission within 1 min of ligand stimulation followed by the prolonged activation of multiple Src-related molecules. Temporal perturbation of Src family kinases with the corresponding inhibitor PP2 in the prolonged activation phase led to the down-regulation of the molecules related to cell adhesion and receptor degradation, whereas the canonical cascades as well as the epidermal growth factor receptor relatively maintained their activities. Our methodology provides a system-wide view of the regulatory network clusters involved in signal transduction that is essential to refine the literature-based network structures for a systems biology analysis. Molecular  Cellular Proteomics 8:226-231, 2009.</div>','<b>Title: </b>Using heterogeneous data sources in a systems biology approach to modeling the Sonic Hedgehog signaling pathway<br><b>Year: </b>2010<br><b>Author: </b> Er. Boykin and  Wo. Ogle<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1039/c0mb00006j"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MOLECULAR BIOSYSTEMS<br><b>Cluster-Keywords: </b><em>systems biology, dynamic models, biological systems, model, dynamic systems, computational systems, network model, experimental data, model parameters, systems, systems model, models, pathway systems, data, biochemical pathway, dynamics model, biochemical networks, computational models, network, system<div style="font-size:11px;"><br><b>Abstract: </b><em>Experimental data from biological pathways come in many forms: qualitative or quantitative, static or dynamic. By combining a variety of these heterogeneous sources of data, we construct a mathematical model of a critical regulatory network in vertebrate development, the Sonic Hedgehog signaling pathway. The structure of our model is first constrained by several well-established pathway interactions. On top of this, we develop a hierarchical genetic algorithm that is capable of integrating different types of experimental data collected on the pathways function, including qualitative as well as static and dynamic quantitative data, in order to estimate model parameters. The result is a dynamical model that fits the observed data and is robust to perturbations in its parameters. Since it is based on a canonical power-law representation of biochemical pathways whose parameters can be directly translated into physical interactions between network components, our model provides insight into the nature and strength of pathway interactions and suggests directions for future research.</div>','<b>Title: </b>Towards a proteome-scale map of the human protein-protein interaction network<br><b>Year: </b>2005<br><b>Author: </b> Jf. Rual,  K. Venkatesan,  T. Hao,  T. Hirozane-Kishikawa,  A. Dricot,  N. Li,  Gf. Berriz,  Fd. Gibbons,  M. Dreze,  N. Ayivi-Guedehoussou,  N. Klitgord,  C. Simon,  M. Boxem,  S. Milstein,  J. Rosenberg,  Ds. Goldberg,  Lv. Zhang,  Sl. Wong,  G. Franklin,  Sm. Li,  Js. Albala,  Jh. Lim,  C. Fraughton,  E. Llamosas,  S. Cevik,  C. Bex,  P. Lamesch,  Rs. Sikorski,  J. Vandenhaute,  Hy. Zoghbi,  A. Smolyar,  S. Bosak,  R. Sequerra,  L. Doucette-Stamm,  Me. Cusick,  De. Hill,  Fp. Roth and  M. Vidal<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1038/nature04209"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>NATURE<br><b>Cluster-Keywords: </b><em>interaction networks, proteinprotein interactions, protein interaction, network model, complex systems, systems biology, protein protein, protein complex, host network, dynamic interactions, network analysis, biological systems, networks network, cellular network, biological network, viral proteins, interactome networks, regulatory networks, immune system, systems network<div style="font-size:11px;"><br><b>Abstract: </b><em>Systematic mapping of protein-protein interactions, or interactome mapping, was initiated in model organisms, starting with defined biological processes1,2 and then expanding to the scale of the proteome3-7. Although far from complete, such maps have revealed global topological and dynamic features of interactome networks that relate to known biological properties8,9, suggesting that a human interactome map will provide insight into development and disease mechanisms at a systems level. Here we describe an initial version of a proteome-scale map of human binary protein-protein interactions. Using a stringent, high-throughput yeast two-hybrid system, we tested pairwise interactions among the products of similar to 8,100 currently available Gateway-cloned open reading frames and detected similar to 2,800 interactions. This data set, called CCSB-HI1, has a verification rate of similar to 78 as revealed by an independent co-affinity purification assay, and correlates significantly with other biological attributes. The CCSB-HI1 data set increases by similar to 70 the set of available binary interactions within the tested space and reveals more than 300 new connections to over 100 disease-associated proteins. This work represents an important step towards a systematic and comprehensive human interactome project.</div>','<b>Title: </b>Interplay between Graph Topology and Correlations of Third Order in Spiking Neuronal Networks<br><b>Year: </b>2016<br><b>Author: </b> S. Jovanovic and  S. Rotter<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pcbi.1004963"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS COMPUTATIONAL BIOLOGY<br><b>Cluster-Keywords: </b><em>complex networks, complex systems, network theory, network structure, biological network, social network, network science, dynamic networks, network systems, network model, network topology, systems biology, network, network analysis, system dynamics, networks, realworld networks, network automata, epidemic network, model network<div style="font-size:11px;"><br><b>Abstract: </b><em>The study of processes evolving on networks has recently become a very popular research field, not only because of the rich mathematical theory that underpins it, but also because of its many possible applications, a number of them in the field of biology. Indeed, molecular signaling pathways, gene regulation, predator-prey interactions and the communication between neurons in the brain can be seen as examples of networks with complex dynamics. The properties of such dynamics depend largely on the topology of the underlying network graph. In this work, we want to answer the following question: Knowing network connectivity, what can be said about the level of third-order correlations that will characterize the network dynamics We consider a linear point process as a model for pulse-coded, or spiking activity in a neuronal network. Using recent results from theory of such processes, we study third-order correlations between spike trains in such a system and explain which features of the network graph i.e. which topological motifs are responsible for their emergence. Comparing two different models of network topology-random networks of Erdos-Renyi type and networks with highly interconnected hubs-we find that, in random networks, the average measure of third-order correlations does not depend on the local connectivity properties, but rather on global parameters, such as the connection probability. This, however, ceases to be the case in networks with a geometric out-degree distribution, where topological specificities have a strong impact on average correlations.</div>','<b>Title: </b>Properties of sparse penalties on inferring gene regulatory networks from time-course gene expression data<br><b>Year: </b>2015<br><b>Author: </b> Lz. Liu,  Fx. Wu and  Wj. Zhang<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1049/iet-syb.2013.0060"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IET SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>regulatory networks, gene regulatory, gene expression, network inference, expression data, network structure, gene networks, networks grns, regulatory genes, boolean networks, bayesian network, network reconstruction, networks gene, network dynamics, gene regulation, inference method, systems biology, time series, infer gene, network<div style="font-size:11px;"><br><b>Abstract: </b><em>Genes regulate each other and form a gene regulatory network GRN to realise biological functions. Elucidating GRN from experimental data remains a challenging problem in systems biology. Numerous techniques have been developed and sparse linear regression methods become a promising approach to infer accurate GRNs. However, most linear methods are either based on steady-state gene expression data or their statistical properties are not analysed. Here, two sparse penalties, adaptive least absolute shrinkage and selection operator and smoothly clipped absolute deviation, are proposed to infer GRNs from time-course gene expression data based on an auto-regressive model and their Oracle properties are proved under mild conditions. The effectiveness of those methods is demonstrated by applications to in silico and real biological data.</div>','<b>Title: </b>A heuristic model for working memory deficit in schizophrenia<br><b>Year: </b>2016<br><b>Author: </b> Z. Qi,  Gp. Yu,  F. Tretter,  O. Pogarell,  Aa. Grace and  Eo. Voit<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.bbagen.2016.04.031"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOCHIMICA ET BIOPHYSICA ACTA-GENERAL SUBJECTS<br><b>Cluster-Keywords: </b><em>systems biology, dynamic models, biological systems, model, dynamic systems, computational systems, network model, experimental data, model parameters, systems, systems model, models, pathway systems, data, biochemical pathway, dynamics model, biochemical networks, computational models, network, system<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: The life of schizophrenia patients is severely affected by deficits in working memory. In various brain regions, the reciprocal interactions between excitatory glutamatergic neurons and inhibitory GABAergic neurons are crucial. Other neurotransmitters, in particular dopamine, serotonin, acetylcholine, and norepinephrine, modulate the local balance between glutamate and GABA and therefore regulate the function of brain regions. Persistent alterations in the balances between the neurotransmitters can result in working memory deficits.Methods: Here we present a heuristic computational model that accounts for interactions among neurotransmitters across various brain regions. The model is based on the concept of a neurochemical interaction matrix at the biochemical level and combines this matrix with a mobile model representing physiological dynamic balances among neurotransmitter systems associated with working memory.Results: The comparison of clinical and simulation results demonstrates that the model output is qualitatively very consistent with the available data. In addition, the model captured how perturbations migrated through different neurotransmitters and brain regions. Results showed that chronic administration of ketamine can cause a variety of imbalances, and application of an antagonist of the D-2 receptor in PFC can also induce imbalances but in a very different manner.Conclusions: The heuristic computational model permits a variety of assessments of genetic, biochemical, and pharmacological perturbations and serves as an intuitive tool for explaining clinical and biological observations.General significance: The heuristic model is more intuitive than biophysically detailed models. It can serve as an important tool for interdisciplinary communication and even for psychiatric education of patients and relatives. This article is part of a Special Issue entitled "System Genetics" C 2016 Elsevier B.V. All rights reserved.</div>','<b>Title: </b>Empowering the impaired astrocytes in the tripartite synapses to improve accuracy of pattern recognition<br><b>Year: </b>2019<br><b>Author: </b> S. Nazari and  K. Faez<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s00500-018-03671-z"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SOFT COMPUTING<br><b>Cluster-Keywords: </b><em>inflammatory response, network model, mechanistic models, lde model, injured patients, critically injured, prediction model, tbi survivors, computational modeling, model abm, impaired astrocytes, model predicts, metabolic dynamics, significant differences, twoprocess model, agent-based model, coupled model, model, inflammatory network, computational models<div style="font-size:11px;"><br><b>Abstract: </b><em>Recent finding has demonstrated that glial cells and especially astrocytes are responsible for some important roles in the central nervous system. Laboratory examination of the roles of astrocytes in information processing of neural system is associated with technical difficulties. Therefore, computational modeling provides a suitable approach in describing cognitive phenomena and neuroscience. In this paper, the role of strong and weak astrocytes in pattern recognition has considered and bio-stimulator was used to compensate the malfunction of the weak astrocyte. Therefore, we designed the tripartite synapses network, which astrocytes dominate the synaptic spaces. Then, the network has been trained on MNIST dataset. After that, the control feedback of astrocyte to pre- and postsynaptic neurons has decreased so that the astrocyte dominance on synaptic transitions decreased and impaired tripartite synapses network has been configured. In the next step, due to the destructive effects of impaired astrocyte, based on the dynamical model of the astrocyte biophysical model, a bio-inspired stimulator was designed. Accordingly, a new concept Stimulating the impaired astrocytes to compensate their malfunction in pattern recognition was introduced. This new mechanism was proposed to stimulate the impaired astrocytes in a population of tripartite synapses for restoration of the normal neural oscillations. Reported results confirmed that the increased error of always firing in pattern recognition caused by astrocytes dysfunction could be compensated by the bio-inspired stimulator.</div>','<b>Title: </b>Mathematical Models for Systems Biology and How to Construct Them<br><b>Year: </b>2012<br><b>Author: </b> Cp. Barnes,  M. Huvet,  N. Harmston and  Mph. StumpfLink: No link<br><b>Journal: </b>SYSTEMS MICROBIOLOGY: CURRENT TOPICS AND APPLICATIONS<br><b>Cluster-Keywords: </b><em>circadian clock, biological systems, network models, mathematical models, clock gene, systems biology, model selection, biological networks, network data, experimental data, model parameters, network dynamics, plant circadian, regulatory network, clock model, plant clock, complex network, clock circuit, approximate bayesian, stochastic models<div style="font-size:11px;"><br><b>Abstract: </b><em>Modelling methodologies in the life and biomedical sciences are hampered by the complexity of the processes and systems at work. Modelling studies into prokaryotic systems require the elucidation of the mechanistic model. In this chapter we introduction modelling methodologies and discuss the problem of model and parameter inference. We comment on state-of-the-art research questions and provide a general discussion on how models can and should be used in order to better understand the structure, function and dynamics of biological systems. The aim is not to provide an introduction to modelling per se, but to provide readers with an overview on the available methodologies. The modelling approach chosen depends on the biological question at hand as well as a range of other factors.</div>','<b>Title: </b>Hybrid regulatory models: a statistically tractable approach to model regulatory network dynamics<br><b>Year: </b>2013<br><b>Author: </b> A. Ocone,  Aj. Millar and  G. Sanguinetti<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1093/bioinformatics/btt069"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>circadian clock, biological systems, network models, mathematical models, clock gene, systems biology, model selection, biological networks, network data, experimental data, model parameters, network dynamics, plant circadian, regulatory network, clock model, plant clock, complex network, clock circuit, approximate bayesian, stochastic models<div style="font-size:11px;"><br><b>Abstract: </b><em>Motivation: Computational modelling of the dynamics of gene regulatory networks is a central task of systems biology. For networks of small/medium scale, the dominant paradigm is represented by systems of coupled non-linear ordinary differential equations ODEs. ODEs afford great mechanistic detail and flexibility, but calibrating these models to data is often an extremely difficult statistical problem.Results: Here, we develop a general statistical inference framework for stochastic transcription-translation networks. We use a coarse-grained approach, which represents the system as a network of stochastic binary promoter and continuous protein variables. We derive an exact inference algorithm and an efficient variational approximation that allows scalable inference and learning of the model parameters. We demonstrate the power of the approach on two biological case studies, showing that the method allows a high degree of flexibility and is capable of testable novel biological predictions.</div>','<b>Title: </b>MODELING AND ANALYSIS OF BIOPATHWAYS DYNAMICS<br><b>Year: </b>2012<br><b>Author: </b> B. Liu and  Ps. Thiagarajan<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1142/S0219720012310014"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF BIOINFORMATICS AND COMPUTATIONAL BIOLOGY<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Cellular processes are governed and coordinated by a multitude of biopathways. A pathway can be viewed as a complex network of biochemical reactions. The dynamics of this network largely determines the functioning of the pathway. Hence the modeling and analysis of biochemical networks dynamics is an important problem and is an active area of research. Here we review quantitative models of biochemical networks based on ordinary differential equations ODEs. We mainly focus on the parameter estimation and sensitivity analysis problems and survey the current methods for tackling them. In this context we also highlight a recently developed probabilistic approximation technique using which these two problems can be considerably simplified.</div>','<b>Title: </b>Input-dependent wave propagations in asymmetric cellular automata: Possible behavior forward loop in biological reaction network<br><b>Year: </b>2008<br><b>Author: </b> A. Awazu<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3934/mbe.2008.5.419"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MATHEMATICAL BIOSCIENCES AND ENGINEERING<br><b>Cluster-Keywords: </b><em>dna damage, signalling networks, network dynamics, pmdm system, systems biology, systems dynamics, biological systems, single cell, kinase inhibitors, signaling network, biological networks, cellular automata, cell cycle, oscillation system, temporal dynamics, signaling pathway, individual cells, signaling processes, dynamic model, damage response<div style="font-size:11px;"><br><b>Abstract: </b><em>Dynamical aspects of the asymmetric cellular automata were investigated to consider the signaling processes in biological systems. As a metamodel of the cascade of feed-forward loop type network motifs in biological reaction networks, we consider the one dimensional asymmetric cellular automata where the state of each cell is controlled by a trio of cells, the cell itself, the nearest upstream cell and the next nearest upstream cell. Through the systematic simulations, some novel input-dependent wave propagations were found in certain asymmetric CA, which may be useful for the signaling processes like the distinction, the filtering and the memory of external stimuli.</div>','<b>Title: </b>Reconstruction of gene regulatory networks reveals chromatin remodelers and key transcription factors in tumorigenesis<br><b>Year: </b>2016<br><b>Author: </b> V. Malysheva,  Ma. Mendoza-Parra,  Mam. Saleem and  H. Gronemeyer<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/s13073-016-0310-3"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>GENOME MEDICINE<br><b>Cluster-Keywords: </b><em>gene regulatory, systems biology, regulatory networks, gene network, stem cell, network model, transcription factors, gene expression, cell transcriptomes, network analysis, data analysis, stem cells, network grnsight, cell transformation, weighted network, gene ontology, networks grns, graph model, graph layout, endothelial cell<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Alterations in genetic and epigenetic landscapes are known to contribute to the development of different types of cancer. However, the mechanistic links between transcription factors and the epigenome which coordinate the deregulation of gene networks during cell transformation are largely unknown.Methods: We used an isogenic model of stepwise tumorigenic transformation of human primary cells to monitor the progressive deregulation of gene networks upon immortalization and oncogene-induced transformation. We applied a systems biology approach by combining transcriptome and epigenome data for each step during transformation and integrated transcription factor-target gene associations in order to reconstruct the gene regulatory networks that are at the basis of the transformation process.Results: We identified 142 transcription factors and 24 chromatin remodelers/modifiers CRMs which are preferentially associated with specific co-expression pathways that originate from deregulated gene programming during tumorigenesis. These transcription factors are involved in the regulation of divers processes, including cell differentiation, the immune response, and the establishment/modification of the epigenome. Unexpectedly, the analysis of chromatin state dynamics revealed patterns that distinguish groups of genes which are not only coregulated but also functionally related. Decortication of transcription factor targets enabled us to define potential key regulators of cell transformation which are engaged in RNA metabolism and chromatin remodeling.Conclusions: We reconstructed gene regulatory networks that reveal the alterations occurring during human cellular tumorigenesis. Using these networks we predicted and validated several transcription factors as key players for the establishment of tumorigenic traits of transformed cells. Our study suggests a direct implication of CRMs in oncogene-induced tumorigenesis and identifies new CRMs involved in this process. This is the first comprehensive view of the gene regulatory network that is altered during the process of stepwise human cellular tumorigenesis in a virtually isogenic system.</div>','<b>Title: </b>Limitations of quantitative gene regulation models: A case study<br><b>Year: </b>2003<br><b>Author: </b> Pm. Kim and  B. Tidor<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1101/gr.1207003"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>GENOME RESEARCH<br><b>Cluster-Keywords: </b><em>gene regulatory, systems biology, synthetic biology, regulatory network, gene network, gene expression, system model, synthetic network, network dynamics, systems theory, synthetic gene, genetic network, model predicts, dynamical systems, time delays, translational time, regulation models, oscillatory expression, network structure, biological systems<div style="font-size:11px;"><br><b>Abstract: </b><em>Understanding the relationship between network structure and behavior is fundamental to the field of computational and systems biology. A particularly important distinction is the extent to which qualitative aspects of network performance are encoded in network topology as opposed to being determined through quantitative details, such as those of kinetics. Here, we develop a general and rigorous mathematical framework for the analysis of genetic networks and apply it to a family of synthetic gene networks. A key feature of our methodology involves determining network behavior that is insensitive to kinetic parameters Such as rate constants and nonlinear functional dependencies of rates oil molecular concentrations. Results indicate that behavior observed in some networks cannot be reconciled with standard gene expression and regulation models. We explore relaxing model assumptions to explain the observed behavior, allowing for both dynamic and stochastic phenomena, and propose an alternative model. Our alternative model includes the Suggestion of a new mechanism by which the counterintuitive behavior could be achieved; central to the model is the assumption that the Clp protein degradation system, which is responsible for the regulatory proteins used in this study, becomes saturated.</div>','<b>Title: </b>Transient Dynamics of Reduced-Order Models of Genetic Regulatory Networks<br><b>Year: </b>2012<br><b>Author: </b> R. Pal and  S. Bhattacharya<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1109/TCBB.2012.37"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IEEE-ACM TRANSACTIONS ON COMPUTATIONAL BIOLOGY AND BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>boolean networks, regulatory networks, genetic regulatory, gene regulatory, probabilistic boolean, network, networks, model, network model, probabilistic model, model checking, networks pbns, network dynamics, markov chain, regulatory, models, optimal control, boolean, networks models, networks boolean<div style="font-size:11px;"><br><b>Abstract: </b><em>In systems biology, a number of detailed genetic regulatory networks models have been proposed that are capable of modeling the fine-scale dynamics of gene expression. However, limitations on the type and sampling frequency of experimental data often prevent the parameter estimation of the detailed models. Furthermore, the high computational complexity involved in the simulation of a detailed model restricts its use. In such a scenario, reduced-order models capturing the coarse-scale behavior of the network are frequently applied. In this paper, we analyze the dynamics of a reduced-order Markov Chain model approximating a detailed Stochastic Master Equation model. Utilizing a reduction mapping that maintains the aggregated steady-state probability distribution of stochastic master equation models, we provide bounds on the deviation of the Markov Chain transient distribution from the transient aggregated distributions of the stochastic master equation model.</div>','<b>Title: </b>Implicit Methods for Qualitative Modeling of Gene Regulatory Networks<br><b>Year: </b>2012<br><b>Author: </b> A. Garg,  K. Mohanram,  G. De Micheli and  I. Xenarios<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/978-1-61779-292-222"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>GENE REGULATORY NETWORKS: METHODS AND PROTOCOLS<br><b>Cluster-Keywords: </b><em>regulatory networks, stochastic simulation, gene networks, reaction networks, systems biology, stochastic model, network models, complex systems, networks stochastic, gene regulatory, biological systems, stochastic dynamics, differential equations, protein networks, cell cycle, continuous model, interaction networks, systems chemistry, time delays, biochemical reaction<div style="font-size:11px;"><br><b>Abstract: </b><em>Advancements in high-throughput technologies to measure increasingly complex biological phenomena at the genomic level are rapidly changing the face of biological research from the single-gene single-protein experimental approach to studying the behavior of a gene in the context of the entire genome and proteome. This shift in research methodologies has resulted in a new field of network biology that deals with modeling cellular behavior in terms of network structures such as signaling pathways and gene regulatory networks. In these networks, different biological entities such as genes, proteins, and metabolites interact with each other, giving rise to a dynamical system. Even though there exists a mature field of dynamical systems theory to model such network structures, some technical challenges are unique to biology such as the inability to measure precise kinetic information on gene gene or gene protein interactions and the need to model increasingly large networks comprising thousands of nodes. These challenges have renewed interest in developing new computational techniques for modeling complex biological systems.This chapter presents a modeling framework based on Boolean algebra and finite-state machines that are reminiscent of the approach used for digital circuit synthesis and simulation in the field of very-large-scale integration VLSI. The proposed formalism enables a common mathematical framework to develop computational techniques for modeling different aspects of the regulatory networks such as steady-state behavior, stochasticity, and gene perturbation experiments.</div>','<b>Title: </b>Statistical model comparison applied to common network motifs<br><b>Year: </b>2010<br><b>Author: </b> N. Domedel-Puig,  I. Pournara and  L. Wernisch<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-4-18"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>network motifs, regulatory network, gene regulatory, systems biology, gene network, bifan motif, complex systems, model comparison, network modeling, complex gene, biological systems, dynamic systems, interaction network, feedforward loop, modeling gene, intelligent systems, modeling approach, biological pathways, motif model, model motifs<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Network motifs are small modules that show interesting functional and dynamic properties, and are believed to be the building blocks of complex cellular processes. However, the mechanistic details of such modules are often unknown: there is uncertainty about the motif architecture as well as the functional form and parameter values when converted to ordinary differential equations ODEs. This translates into a number of candidate models being compatible with the system under study. A variety of statistical methods exist for ranking models including maximum likelihood-based and Bayesian methods. Our objective is to show how such methods can be applied in a typical systems biology setting.Results: We focus on four commonly occurring network motif structures and show that it is possible to differentiate between them using simulated data and any of the model comparison methods tested. We expand one of the motifs, the feed forward FF motif, for several possible parameterizations and apply model selection on simulated data. We then use experimental data on three biosynthetic pathways in Escherichia coli to formally assess how current knowledge matches the time series available. Our analysis confirms two of them as FF motifs. Only an expanded set of FF motif parameterisations using time delays is able to fit the third pathway, indicating that the true mechanism might be more complex in this case.Conclusions: Maximum likelihood as well as Bayesian model comparison methods are suitable for selecting a plausible motif model among a set of candidate models. Our work shows that it is practical to apply model comparison to test ideas about underlying mechanisms of biological pathways in a formal and quantitative way.</div>','<b>Title: </b>Drug Resistance Mechanisms in Colorectal Cancer Dissected with Cell Type-Specific Dynamic Logic Models<br><b>Year: </b>2017<br><b>Author: </b> F. Eduati,  V. Doldan-Martelli,  B. Klinger,  T. Cokelaer,  A. Sieber,  F. Kogera,  M. Dorel,  Mj. Garnett,  N. Bluthgen and  J. Saez-Rodriguez<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1158/0008-5472.CAN-17-0078"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>CANCER RESEARCH<br><b>Cluster-Keywords: </b><em>boolean network, network models, qualitative models, systems biology, mathematical modeling, boolean models, signaling networks, dynamic models, dynamic modeling, signaling dynamics, logic models, dynamic network, mathematical models, boolean modeling, signal transduction, regulatory networks, drug resistance, biological networks, network approach, network modeling<div style="font-size:11px;"><br><b>Abstract: </b><em>Genomic features are used as biomarkers of sensitivity to kinase inhibitors used widely to treat human cancer, but effective patient stratification based on these principles remains limited in impact. Insofar as kinase inhibitors interfere with signaling dynamics, and, in turn, signaling dynamics affects inhibitor responses, we investigated associations in this study between cell-specific dynamic signaling pathways and drug sensitivity. Specifically, we measured 14 phosphoproteins under 43 different perturbed conditions combinations of 5 stimuli and 7 inhibitors in 14 colorectal cancer cell lines, building cell line-specific dynamic logic models of underlying signaling networks. Model parameters representing pathway dynamics were used as features to predict sensitivity to a panel of 27 drugs. Specific parameters of signaling dynamics correlated strongly with drug sensitivity for 14 of the drugs, 9 of which had no genomic biomarker. Following one of these associations, we validated a drug combination predicted to overcome resistance to MEK inhibitors by coblockade of GSK3, which was not found based on associations with genomic data. These results suggest that to better understand the cancer resistance and move toward personalized medicine, it is essential to consider signaling network dynamics that cannot be inferred from static genotypes. C 2017 AACR.</div>','<b>Title: </b>Spatiotemporal Characterization of a Fibrin Clot Using Quantitative Phase Imaging<br><b>Year: </b>2014<br><b>Author: </b> R. Gannavarpu,  B. Bhaduri,  K. Tangella and  G. Popescu<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0111381"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Studying the dynamics of fibrin clot formation and its morphology is an important problem in biology and has significant impact for several scientific and clinical applications. We present a label-free technique based on quantitative phase imaging to address this problem. Using quantitative phase information, we characterized fibrin polymerization in real-time and present a mathematical model describing the transition from liquid to gel state. By exploiting the inherent optical sectioning capability of our instrument, we measured the three-dimensional structure of the fibrin clot. From this data, we evaluated the fractal nature of the fibrin network and extracted the fractal dimension. Our non-invasive and speckle-free approach analyzes the clotting process without the need for external contrast agents.</div>','<b>Title: </b>Pattern dynamics of the reaction-diffusion immune system<br><b>Year: </b>2018<br><b>Author: </b> Qq. Zheng,  Jw. Shen and  Zj. Wang<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0190176"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>biological system, network, pattern formation, system, network analysis, systems, turing patterns, patterns, reaction networks, networks, turing system, network topologies, stoichiometric network, networks pattern, network topology, biological, model, gradient formation, pattern, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>In this paper, we will investigate the effect of diffusion, which is ubiquitous in nature, on the immune system using a reaction-diffusion model in order to understand the dynamical behavior of complex patterns and control the dynamics of different patterns. Through control theory and linear stability analysis of local equilibrium, we obtain the optimal condition under which the system loses stability and a Turing pattern occurs. By combining mathematical analysis and numerical simulation, we show the possible patterns and how these patterns evolve. In addition, we establish a bridge between the complex patterns and the biological mechanism using the results from a previous study in Nature Cell Biology. The results in this paper can help us better understand the biological significance of the immune system.</div>','<b>Title: </b>Measuring information flow in cellular networks by the systems biology method through microarray data<br><b>Year: </b>2015<br><b>Author: </b> Bs. Chen and  Cw. Li<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3389/fpls.2015.00390"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>FRONTIERS IN PLANT SCIENCE<br><b>Cluster-Keywords: </b><em>gene networks, synthetic gene, synthetic biology, regulatory networks, design method, robust synthetic, nonlinear stochastic, molecular noises, gene regulatory, interaction networks, external disturbances, robust stability, robust gene, host cell, gene circuit, cellular network, design specifications, design procedure, signal transduction, parameter fluctuations<div style="font-size:11px;"><br><b>Abstract: </b><em>In general, it is very difficult to measure the information flow in a cellular network directly. In this study, based on an information flow model and microarray data, we measured the information flow in cellular networks indirectly by using a systems biology method. First, we used a recursive least square parameter estimation algorithm to identify the system parameters of coupling signal transduction pathways and the cellular gene regulatory network GRN. Then, based on the identified parameters and systems theory, we estimated the signal transductivities of the coupling signal transduction pathways from the extracellular signals to each downstream protein and the information transductivities of the GRN between transcription factors in response to environmental events. According to the proposed method, the information flow, which is characterized by signal transductivity in coupling signaling pathways and information transductivity in the GRN, can be estimated by microarray temporal data or microarray sample data. It can also be estimated by other high-throughput data such as next-generation sequencing or proteomic data. Finally, the information flows of the signal transduction pathways and the GRN in leukemia cancer cells and non-leukemia normal cells were also measured to analyze the systematic dysfunction in this cancer from microarray sample data. The results show that the signal transductivities of signal transduction pathways change substantially from normal cells to leukemia cancer cells.</div>','<b>Title: </b>Innate Immunity in Disease: Insights from Mathematical Modeling and Analysis<br><b>Year: </b>2014<br><b>Author: </b> N. Azhar and  Y. Vodovotz<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/978-1-4939-2095-211"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SYSTEMS BIOLOGY APPROACH TO BLOOD<br><b>Cluster-Keywords: </b><em>inflammatory response, network model, mechanistic models, lde model, injured patients, critically injured, prediction model, tbi survivors, computational modeling, model abm, impaired astrocytes, model predicts, metabolic dynamics, significant differences, twoprocess model, agent-based model, coupled model, model, inflammatory network, computational models<div style="font-size:11px;"><br><b>Abstract: </b><em>The acute inflammatory response is a complex defense mechanism that has evolved to respond rapidly to injury, infection, and other disruptions in homeostasis. This robust responsiveness to biological stress likely endows the host with increased fitness, but over-robust or inadequate inflammation predisposes the host to various diseases. Importantly, well-compartmentalized inflammation is generally beneficial, but spillover of inflammation into the blood is a hallmark-and likely also a driver-of self-maintaining inflammation. The blood is also a key entry point and immunological interface for vectors of parasitic diseases, diseases that themselves incite systemic inflammation. The complex role of inflammation in health and disease has made this biological system difficult to understand comprehensively and modulate rationally for therapeutic purposes. Consequently, systems approaches have been applied in order to characterize dynamical properties and identify key control points in inflammation. This process begins with the collection of high-dimensional, experimental, and clinical data, followed by data reduction and data-driven modeling that finally informs mechanistic computational models for analysis, prediction, and rational modulation. These studies have suggested that the overall architecture of the inflammatory response includes a multiscale positive feedback from inflammation -> tissue damage -> inflammation, which is often inadequately controlled by negative feedback from anti-inflammatory mediators. Given the importance of the blood interface for the inflammatory response, and the accessibility of this compartment both as an immunological sampling reservoir for vectors as well as for diagnosis and therapy, we suggest that any rational efforts at modulating inflammation via the blood compartment must involve computational modeling.</div>','<b>Title: </b>Steady-state probabilities for attractors in probabilistic Boolean networks<br><b>Year: </b>2005<br><b>Author: </b> M. Brun,  Er. Dougherty and  I. Shmulevich<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.sigpro.2005.02.016"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SIGNAL PROCESSING<br><b>Cluster-Keywords: </b><em>boolean networks, regulatory networks, genetic regulatory, gene regulatory, probabilistic boolean, network, networks, model, network model, probabilistic model, model checking, networks pbns, network dynamics, markov chain, regulatory, models, optimal control, boolean, networks models, networks boolean<div style="font-size:11px;"><br><b>Abstract: </b><em>Boolean networks form a class of disordered dynamical systems that have been studied in physics owing to their relationships with disordered systems in statistical mechanics and in biology as models of genetic regulatory networks. Recently they have been generalized to probabilistic Boolean networks PBNs to facilitate the incorporation of uncertainty in the model and to represent cellular context changes in biological modeling. In essence, a PBN is composed of a family of Boolean networks between which the PBN switches in a stochastic fashion. In whatever framework Boolean networks are studied, their most important attribute is their attractors. Left to run, a Boolean network will settle into one of a collection of state cycles called attractors. The set of states from which the network will transition into a specific attractor forms the basin of the attractor. The attractors represent the essential long-run behavior of the network. In a classical Boolean network, the network remains in an attractor once there; in a Boolean network with perturbation, the states form an ergodic Markov chain and the network can escape an attractor, but it will return to it or a different attractor unless interrupted by another perturbation; in a probabilistic Boolean network, so long as the PBN remains in one of its constituent Boolean networks it will behave as a Boolean network with perturbation, but upon a switch it will move to an attractor of the new constituent Boolean network. Given the ergodic nature of the model, the steady-state probabilities of the attractors are critical to network understanding. Heretofore they have been found by simulation; in this paper we derive analytic expressions for these probabilities, first for Boolean networks with perturbation and then for PBNs. c 2005 Elsevier B.V. All rights reserved.</div>','<b>Title: </b>Coupled Stochastic Spatial and Non-Spatial Simulations of ErbB1 Signaling Pathways Demonstrate the Importance of Spatial Organization in Signal Transduction<br><b>Year: </b>2009<br><b>Author: </b> Mn. Costa,  K. Radhakrishnan,  Bs. Wilson,  Dg. Vlachos and  Js. Edwards<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0006316"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>signaling network, systems biology, signal transduction, signaling pathways, receptor tyrosine, erbb signaling, erbb receptor, receptor signaling, intracellular signaling, mathematical models, cell signaling, computational model, factor receptor, receptor activation, growth factor, signaling models, networks signaling, receptor system, receptors systems, downstream signaling<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: The ErbB family of receptors activates intracellular signaling pathways that control cellular proliferation, growth, differentiation and apoptosis. Given these central roles, it is not surprising that overexpression of the ErbB receptors is often associated with carcinogenesis. Therefore, extensive laboratory studies have been devoted to understanding the signaling events associated with ErbB activation.Methodology/Principal Findings: Systems biology has contributed significantly to our current understanding of ErbB signaling networks. However, although computational models have grown in complexity over the years, little work has been done to consider the spatial-temporal dynamics of receptor interactions and to evaluate how spatial organization of membrane receptors influences signaling transduction. Herein, we explore the impact of spatial organization of the epidermal growth factor receptor ErbB1/EGFR on the initiation of downstream signaling. We describe the development of an algorithm that couples a spatial stochastic model of membrane receptors with nonspatial stochastic model of the reactions and interactions in the cytosol. This novel algorithm provides computationally efficient method to evaluate the effects of spatial heterogeneity on the coupling of receptors to cytosolic signaling partners.Conclusions/Significance: Mathematical models of signal transduction rarely consider the contributions of spatial organization due to high computational costs. A hybrid stochastic approach simplifies analyses of the spatio-temporal aspects of cell signaling and, as an example, demonstrates that receptor clustering contributes significantly to the efficiency of signal propagation from ligand-engaged growth factor receptors.</div>','<b>Title: </b>Improving nonhomogeneous dynamic Bayesian networks with sequentially coupled parameters<br><b>Year: </b>2018<br><b>Author: </b> Ms. Kamalabad and  M. Grzegorczyk<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1111/stan.12136"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>STATISTICA NEERLANDICA<br><b>Cluster-Keywords: </b><em>petri net, petri nets, bayesian networks, biological networks, systems biology, regulatory networks, dynamic bayesian, net model, gene regulatory, network data, biological systems, biological data, network analysis, network models, biochemical networks, time series, network structure, signal transduction, net models, dbn model<div style="font-size:11px;"><br><b>Abstract: </b><em>In systems biology, nonhomogeneous dynamic Bayesian networks NH-DBNs have become a popular modeling tool for reconstructing cellular regulatory networks from postgenomic data. In this paper, we focus our attention on NH-DBNs that are based on Bayesian piecewise linear regression models. The new NH-DBN model, proposed here, is a generalization of an earlier proposed model with sequentially coupled network interaction parameters. Unlike the original model, our novel model possesses segment-specific coupling parameters, so that the coupling strengths between parameters can vary over time. Thereby, to avoid model overflexibility and to allow for some information exchange among time segments, we globally couple the segment-specific coupling strength parameters by a hyperprior. Our empirical results on synthetic and on real biological network data show that the new model yields better network reconstruction accuracies than the original model.</div>','<b>Title: </b>INFERENCE OF S-SYSTEM MODELS OF GENE REGULATORY NETWORKS USING IMMUNE ALGORITHM<br><b>Year: </b>2011<br><b>Author: </b> T. Nakayama,  S. Seno,  Y. Takenaka and  H. Matsuda<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1142/S0219720011005768"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF BIOINFORMATICS AND COMPUTATIONAL BIOLOGY<br><b>Cluster-Keywords: </b><em>ssystem model, biological systems, biological networks, parameter estimation, biochemical networks, dynamic biological, proposed algorithm, regulatory networks, structure identification, model structure, genetic algorithm, gene regulatory, gene expression, network inference, network motif, network structure, gene networks, network identification, network inferring, estimation method<div style="font-size:11px;"><br><b>Abstract: </b><em>The S-system model is one of the nonlinear differential equation models of gene regulatory networks, and it can describe various dynamics of the relationships among genes. If we successfully infer rigorous S-system model parameters that describe a target gene regulatory network, we can simulate gene expressions mathematically. However, the problem of finding an optimal S-system model parameter is too complex to be solved analytically. Thus, some heuristic search methods that offer approximate solutions are needed for reducing the computational time. In previous studies, several heuristic search methods such as Genetic Algorithms GAs have been applied to the parameter search of the S-system model. However, they have not achieved enough estimation accuracy. One of the conceivable reasons is that the mechanisms to escape local optima. We applied an Immune Algorithm IA to search for the S-system parameters. IA is also a heuristic search method, which is inspired by the biological mechanism of acquired immunity. Compared to GA, IA is able to search large solution space, thereby avoiding local optima, and have multiple candidates of the solutions. These features work well for searching the S-system model. Actually, our algorithm showed higher performance than GA for both simulation and real data analyses.</div>','nan','<b>Title: </b>Dynamic deterministic effects propagation networks: learning signalling pathways from longitudinal protein array data<br><b>Year: </b>2010<br><b>Author: </b> C. Bender,  F. Henjes,  H. Frohlich,  S. Wiemann,  U. Korf and  T. Beissbarth<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1093/bioinformatics/btq385"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>regulatory networks, gene regulatory, gene expression, network inference, expression data, network structure, gene networks, networks grns, regulatory genes, boolean networks, bayesian network, network reconstruction, networks gene, network dynamics, gene regulation, inference method, systems biology, time series, infer gene, network<div style="font-size:11px;"><br><b>Abstract: </b><em>Motivation: Network modelling in systems biology has become an important tool to study molecular interactions in cancer research, because understanding the interplay of proteins is necessary for developing novel drugs and therapies. De novo reconstruction of signalling pathways from data allows to unravel interactions between proteins and make qualitative statements on possible aberrations of the cellular regulatory program. We present a new method for reconstructing signalling networks from time course experiments after external perturbation and show an application of the method to data measuring abundance of phosphorylated proteins in a human breast cancer cell line, generated on reverse phase protein arrays.Results: Signalling dynamics is modelled using active and passive states for each protein at each timepoint. A fixed signal propagation scheme generates a set of possible state transitions on a discrete timescale for a given network hypothesis, reducing the number of theoretically reachable states. A likelihood score is proposed, describing the probability of measurements given the states of the proteins over time. The optimal sequence of state transitions is found via a hidden Markov model and network structure search is performed using a genetic algorithm that optimizes the overall likelihood of a population of candidate networks. Our method shows increased performance compared with two different dynamical Bayesian network approaches. For our real data, we were able to find several known signalling cascades from the ERBB signalling pathway.</div>','<b>Title: </b>Progression from a stem cell-like state to early differentiation in the C. elegans germ line<br><b>Year: </b>2010<br><b>Author: </b> O. Cinquin,  Sl. Crittenden,  De. Morgan and  J. Kimble<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1073/pnas.0912704107"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA<br><b>Cluster-Keywords: </b><em>systems biology, dynamic network, early differentiation, complex diseases, stem cells, erythroid cells, system dynamics, network biomarkers, immune system, tcm syndrome, dnb genes, gene expression, dnb model, stem cell, expression dynamics, model system, mathematical model, model dynamics, dynamic model, sensitivity analysis<div style="font-size:11px;"><br><b>Abstract: </b><em>Controls of stem cell maintenance and early differentiation are known in several systems. However, the progression from stem cell self-renewal to overt signs of early differentiation is a poorly understood but important problem in stem cell biology. The Caenorhabditis elegans germ line provides a genetically defined model for studying that progression. In this system, a single-celled mesenchymal niche, the distal tip cell DTC, employs GLP-1/Notch signaling and an RNA regulatory network to balance self-renewal and early differentiation within the "mitotic region," which continuously self-renews while generating new gametes. Here, we investigate germ cells in the mitotic region for their capacity to differentiate and their state of maturation. Two distinct pools emerge. The "distal pool" is maintained by the DTC in an essentially uniform and immature or "stem cell-like" state; the "proximal pool," by contrast, contains cells that are maturing toward early differentiation and are likely transit-amplifying cells. A rough estimate of pool sizes is 30-70 germ cells in the distal immature pool and approximate to 150 in the proximal transit-amplifying pool. We present a simple model for how the network underlying the switch between self-renewal and early differentiation may be acting in these two pools. According to our model, the self-renewal mode of the network maintains the distal pool in an immature state, whereas the transition between self-renewal and early differentiation modes of the network underlies the graded maturation of germ cells in the proximal pool. We discuss implications of this model for controls of stem cells more broadly.</div>','<b>Title: </b>Network Dynamics in Cardiac Electrophysiology<br><b>Year: </b>2014<br><b>Author: </b> Zl. Qu<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/978-3-642-38505-610"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SYSTEMS BIOLOGY OF METABOLIC AND SIGNALING NETWORKS: ENERGY, MASS AND INFORMATION TRANSFER<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>In a network perspective, the heart is a network of cells that are composed of subnetworks of genes, proteins, metabolites, and organelles. In this chapter, we provide an overview of the networks in the heart and a current understanding of the network dynamics in the context of cardiac electrophysiology. We first review current knowledge of the genetic, signaling, and metabolic networks in the heart and their links to arrhythmias. We then review the emergent properties from the mitochondrial and calcium release unit networks, the cellular dynamics arising from integrated subnetworks, and the electrical dynamics arising from the cellular networks to manifest as normal rhythms and arrhythmias. Finally, we discuss future challenges and how systems biology approaches can overcome these challenges to uncover the mechanisms of normal heart rhythms and arrhythmias.</div>','<b>Title: </b>Computational Challenges in Systems Biology<br><b>Year: </b>2010<br><b>Author: </b> Mr. Maurya and  S. SubramaniamLink: No link<br><b>Journal: </b>SYSTEMS BIOMEDICINE: CONCEPTS AND PERSPECTIVES<br><b>Cluster-Keywords: </b><em>stem cell, stem cells, systems biology, mathematical models, cell biology, network model, signaling pathways, cell fate, computational models, cancer cells, gene expression, network dynamics, embryonic stem, signaling network, prostate cancer, mathematical modeling, psychological stress, apoptosis signaling, computational systems, molecular mechanisms<div style="font-size:11px;"><br><b>Abstract: </b><em>The "state" of a cell is defined by its components their concentrations and locations, the interactions between components, which are modulated in space and time, and the complex circuitry, which involves a large number of interacting networks. The state represents a snapshot of the dynamic processes, such as gene expression, cell cycle, transport of components, that characterize the function of the cell. Advances in high-throughput genomic, metabolomic and proteomic technologies now allow study of the cellular components and their interactions in a quantitative manner. These technologies are aiding the development of predictive models by combining legacy knowledge and novel data. There are two paradigms in computational systems biology: i the iterative cycle of biochemical model-mathematical model-computational model, and ii integration of novel data and legacy knowledge to develop context-specific biochemical, mathematical and computational models. Challenges in building biochemical models include a the complexity of proteomic states and interactions, b integration of diverse data to infer biochemical interactions and c the temporal state of biochemical models. Challenges in building mathematical models include i incorporating statistical / probabilistic information into analytical models, ii utilizing qualitative constraints in mathematical models, and iii incomplete knowledge and coarse-graining. Challenges in computational modeling include: a the absence of knowledge about model parameters such as rate constants, b local versus global concentrations of species and multiple scales of distance and time, and c variation among different cell types and subpopulation variability, or variability among biological repeats. This chapter will review these challenges and some of the recent advances in computational systems biology.</div>','<b>Title: </b>MAMMOTh: A new database for curated mathematical models of biomolecular systems<br><b>Year: </b>2018<br><b>Author: </b> F. Kazantsev,  I. Akberdin,  S. Lashin,  N. Ree,  V. Timonov,  A. Ratushny,  T. Khlebodarova and  V. Likhoshvai<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1142/S0219720017400108"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF BIOINFORMATICS AND COMPUTATIONAL BIOLOGY<br><b>Cluster-Keywords: </b><em>systems biology, network models, signaling network, parameter estimation, dynamic models, biological systems, mathematical models, metabolic network, parameter sensitivities, computational model, gene circuit, circuit models, experimental data, metabolic models, model system, scalable parameter, model parameters, parameter space, biomolecular networks, model<div style="font-size:11px;"><br><b>Abstract: </b><em>Motivation: Living systems have a complex hierarchical organization that can be viewed as a set of dynamically interacting subsystems. Thus, to simulate the internal nature and dynamics of the entire biological system, we should use the iterative way for a model reconstruction, which is a consistent composition and combination of its elementary subsystems. In accordance with this bottom-up approach, we have developed the MAthematical Models of bioMOlecular sysTems MAMMOTh tool that consists of the database containing manually curated MAMMOTh fitted to the experimental data and a software tool that provides their further integration.Results: The MAMMOTh database entries are organized as building blocks in a way that the model parts can be used in different combinations to describe systems with higher organizational level metabolic pathways and/or transcription regulatory networks. The tool supports export of a single model or their combinations in SBML or Mathematica standards. The database currently contains 110 mathematical sub-models for Escherichia coli elementary subsystems enzymatic reactions and gene expression regulatory processes that can be combined in at least 5100 complex/sophisticated models concerning more complex biological processes as de novo nucleotide biosynthesis, aerobic/anaerobic respiration and nitrate/nitrite utilization in E. coli. All models are functionally interconnected and sufficiently complement public model resources.</div>','<b>Title: </b>Nonlinear and temporal multiscale dynamics of gene regulatory networks: A qualitative simulator<br><b>Year: </b>2016<br><b>Author: </b> L. Ironi and  Dx. Tran<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.matcom.2015.11.007"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MATHEMATICS AND COMPUTERS IN SIMULATION<br><b>Cluster-Keywords: </b><em>regulatory networks, network dynamics, gene regulatory, network model, genetic network, gene networks, genetic regulatory, mathematical model, parameter space, network system, interaction networks, system dynamics, network structure, network, dynamic networks, network reduction, models models, biological networks, network motifs, biological systems<div style="font-size:11px;"><br><b>Abstract: </b><em>Advanced experimental technologies have made the disclosure of networks of intricate regulatory interactions between genes and gene products feasible and revealed their extreme complexity. Thus, understanding which particular dynamical behaviors derive from specific gene regulatory structures poses a challenging question, at both scientific and application level, that necessarily requires computational tools to be answered. Herein, we discuss the algorithmic aspects and the implementation of a mathematical method, grounded on singular perturbation analysis, for the study of the dynamics of regulated gene networks. This results in a gene regulatory network simulator of the full range of possible dynamics of a specific class of ordinary differential equations adequate to model gene regulatory networks. The considered class of equations represent phenomenological models of the long interaction chains in a network: genes are the main players and the interactions between them are modeled by steep threshold-dependent response functions. The simulator we propose operates in the presence of incomplete knowledge of parameter values. It assumes that threshold-dependent regulation is modeled by continuous steep sigmoid functions, and each transcription factor only regulates one gene at each of its thresholds. Under these assumptions, the simulator derives sound predictions of the nonlinear and temporal multiscale dynamics of a gene regulatory network from an initial state and parameter space, symbolically described by inequalities between parameters. Beside its predictive soundness, it outperforms other qualitative simulators as for characterization of trajectories and possible calculation of the probability of occurrence of each behavior when parameters are assigned stochastic values. C 2015 International Association for Mathematics and Computers in Simulation IMACS. Published by Elsevier B.V. All rights reserved.</div>','<b>Title: </b>A Computational Multiagent Model of Bioluminescent Bacteria for the Emergence of Self-Sustainable and Self-Maintaining Artificial Wireless Networks<br><b>Year: </b>2019<br><b>Author: </b> N. Djezzar,  If. Perez,  N. Djedi and  Y. Duthen<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.31449/inf.v43i3.2381"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>INFORMATICA-JOURNAL OF COMPUTING AND INFORMATICS<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Biology is a rich source of inspiration in designing digital artifacts capable of autonomous, cooperative and distributed behaviors. Particularly, conceptual links can be established between 1 communication networks and 2 colonies of bacteria that communicate using chemical molecules. The goal of this paper is to propose a computational multiagent model of an interspecies bacterial communication system, termed quorum sensing, and analyze its self-sustainability and its self-maintaining ability to cooperatively form artificial wireless networks. Specifically, we propose a bottom-up agent-based approach combined with Ordinary Differential Equations, which abstract the intracellular dynamics, such as a proposed metabolism model that serves as a basis underlying self-sustainable networks. Results show that artificial bacterial cells have regeneration abilities in the light of random cell death and selected area for cell death, and a metabolism allowing them to exploit their own produced energy to cooperate at the population level to exhibit near-optimal self-organizing light-producing behaviors. The resulting artificial networks display several beneficial properties and could be used for the emergence of resistant wireless network topologies without the use of overhead messages.</div>','<b>Title: </b>TimeDelay-ARACNE: Reverse engineering of gene networks from time-course data by an information theoretic approach<br><b>Year: </b>2010<br><b>Author: </b> P. Zoppoli,  S. Morganella and  M. Ceccarelli<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1471-2105-11-154"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>regulatory networks, gene regulatory, gene expression, network inference, expression data, network structure, gene networks, networks grns, regulatory genes, boolean networks, bayesian network, network reconstruction, networks gene, network dynamics, gene regulation, inference method, systems biology, time series, infer gene, network<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: One of main aims of Molecular Biology is the gain of knowledge about how molecular components interact each other and to understand gene function regulations. Using microarray technology, it is possible to extract measurements of thousands of genes into a single analysis step having a picture of the cell gene expression. Several methods have been developed to infer gene networks from steady-state data, much less literature is produced about time-course data, so the development of algorithms to infer gene networks from time-series measurements is a current challenge into bioinformatics research area. In order to detect dependencies between genes at different time delays, we propose an approach to infer gene regulatory networks from time-series measurements starting from a well known algorithm based on information theory.Results: In this paper we show how the ARACNE Algorithm for the Reconstruction of Accurate Cellular Networks algorithm can be used for gene regulatory network inference in the case of time-course expression profiles. The resulting method is called TimeDelay-ARACNE. It just tries to extract dependencies between two genes at different time delays, providing a measure of these dependencies in terms of mutual information. The basic idea of the proposed algorithm is to detect time-delayed dependencies between the expression profiles by assuming as underlying probabilistic model a stationary Markov Random Field. Less informative dependencies are filtered out using an auto calculated threshold, retaining most reliable connections. TimeDelay-ARACNE can infer small local networks of time regulated gene-gene interactions detecting their versus and also discovering cyclic interactions also when only a medium-small number of measurements are available. We test the algorithm both on synthetic networks and on microarray expression profiles. Microarray measurements concern S. cerevisiae cell cycle, E. coli SOS pathways and a recently developed network for in vivo assessment of reverse engineering algorithms. Our results are compared with ARACNE itself and with the ones of two previously published algorithms: Dynamic Bayesian Networks and systems of ODEs, showing that TimeDelay-ARACNE has good accuracy, recall and F-score for the network reconstruction task.Conclusions: Here we report the adaptation of the ARACNE algorithm to infer gene regulatory networks from time-course data, so that, the resulting network is represented as a directed graph. The proposed algorithm is expected to be useful in reconstruction of small biological directed networks from time course data.</div>','<b>Title: </b>Anomalous Diffusion within the Transcriptome as a Bio-Inspired Computing Framework for Resilience<br><b>Year: </b>2017<br><b>Author: </b> W. Seffens<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3390/computation5030032"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>COMPUTATION<br><b>Cluster-Keywords: </b><em>systems biology, regulatory network, interaction networks, network model, gene regulatory, network, network biology, networks, gene network, response networks, network dynamics, dynamic model, cell networks, gene expression, biological networks, system dynamics, dynamic network, model, biological systems, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Much of biology-inspired computer science is based on the Central Dogma, as implemented with genetic algorithms or evolutionary computation. That 60-year-old biological principle based on the genome, transcriptome and proteasome is becoming overshadowed by a new paradigm of complex ordered associations and connections between layers of biological entities, such as interactomes, metabolomics, etc. We define a new hierarchical concept as the Connectosome, and propose new venues of computational data structures based on a conceptual framework called Grand Ensemble which contains the Central Dogma as a subset. Connectedness and communication within and between living or biology-inspired systems comprise ensembles from which a physical computing system can be conceived. In this framework the delivery of messages is filtered by size and a simple and rapid semantic analysis of their content. This work aims to initiate discussion on the Grand Ensemble in network biology as a representation of a Persistent Turing Machine. This framework adding interaction and persistency to the classic Turing-machine model uses metrics based on resilience that has application to dynamic optimization problem solving in Genetic Programming.</div>','<b>Title: </b>Using a Bayesian network to clarify areas requiring research in a host-pathogen system<br><b>Year: </b>2017<br><b>Author: </b> Ds. Bower,  K. Mengersen,  Ra. Alford and  L. Schwarzkopf<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1111/cobi.12950"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>CONSERVATION BIOLOGY<br><b>Cluster-Keywords: </b><em>boolean networks, regulatory networks, network model, bayesian networks, model networks, systems biology, random boolean, real cells, immune system, genetic regulatory, networks, biochemical network, fluorescence images, cellular network, gene regulatory, cell differentiation, network, biological systems, ensemble approach, cell dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>Bayesian network analyses can be used to interactively change the strength of effect of variables in a model to explore complex relationships in new ways. In doing so, they allow one to identify influential nodes that are not well studied empirically so that future research can be prioritized. We identified relationships in host and pathogen biology to examine disease-driven declines of amphibians associated with amphibian chytrid fungus Batrachochytrium dendrobatidis. We constructed a Bayesian network consisting of behavioral, genetic, physiological, and environmental variables that influence disease and used them to predict host population trends. We varied the impacts of specific variables in the model to reveal factors with the most influence on host population trend. The behavior of the nodes the way in which the variables probabilistically responded to changes in states of the parents, which are the nodes or variables that directly influenced them in the graphical model was consistent with published results. The frog population had a 49 probability of decline when all states were set at their original values, and this probability increased when body temperatures were cold, the immune system was not suppressing infection, and the ambient environment was conducive to growth of B. dendrobatidis. These findings suggest the construction of our model reflected the complex relationships characteristic of host-pathogen interactions. Changes to climatic variables alone did not strongly influence the probability of population decline, which suggests that climate interacts with other factors such as the capacity of the frog immune system to suppress disease. Changes to the adaptive immune system and disease reservoirs had a large effect on the population trend, but there was little empirical information available for model construction. Our model inputs can be used as a base to examine other systems, and our results show that such analyses are useful tools for reviewing existing literature, identifying links poorly supported by evidence, and understanding complexities in emerging infectious-disease systems.</div>','<b>Title: </b>Propagation of kinetic uncertainties through a canonical topology of the TLR4 signaling network in different regions of biochemical reaction space<br><b>Year: </b>2010<br><b>Author: </b> J. Gutierrez,  G. St Laurent and  S. Urcuqui-Inchima<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1742-4682-7-7"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>THEORETICAL BIOLOGY AND MEDICAL MODELLING<br><b>Cluster-Keywords: </b><em>feedback loops, signaling network, positive feedback, network model, negative feedback, biochemical networks, signaling pathways, signal transduction, systems biology, kinase erk, protein kinase, transduction networks, dynamic model, emt network, erk pathway, cell fate, cell death, biochemical reaction, coupled feedback, impulse response<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Signal transduction networks represent the information processing systems that dictate which dynamical regimes of biochemical activity can be accessible to a cell under certain circumstances. One of the major concerns in molecular systems biology is centered on the elucidation of the robustness properties and information processing capabilities of signal transduction networks. Achieving this goal requires the establishment of causal relations between the design principle of biochemical reaction systems and their emergent dynamical behaviors.Methods: In this study, efforts were focused in the construction of a relatively well informed, deterministic, non-linear dynamic model, accounting for reaction mechanisms grounded on standard mass action and Hill saturation kinetics, of the canonical reaction topology underlying Toll-like receptor 4 TLR4-mediated signaling events. This signaling mechanism has been shown to be deployed in macrophages during a relatively short time window in response to lypopolysaccharyde LPS stimulation, which leads to a rapidly mounted innate immune response. An extensive computational exploration of the biochemical reaction space inhabited by this signal transduction network was performed via local and global perturbation strategies. Importantly, a broad spectrum of biologically plausible dynamical regimes accessible to the network in widely scattered regions of parameter space was reconstructed computationally. Additionally, experimentally reported transcriptional readouts of target pro-inflammatory genes, which are actively modulated by the network in response to LPS stimulation, were also simulated. This was done with the main goal of carrying out an unbiased statistical assessment of the intrinsic robustness properties of this canonical reaction topology.Results: Our simulation results provide convincing numerical evidence supporting the idea that a canonical reaction mechanism of the TLR4 signaling network is capable of performing information processing in a robust manner, a functional property that is independent of the signaling task required to be executed. Nevertheless, it was found that the robust performance of the network is not solely determined by its design principle topology, but this may be heavily dependent on the networks current position in biochemical reaction space. Ultimately, our results enabled us the identification of key rate limiting steps which most effectively control the performance of the system under diverse dynamical regimes.Conclusions: Overall, our in silico study suggests that biologically relevant and non-intuitive aspects on the general behavior of a complex biomolecular network can be elucidated only when taking into account a wide spectrum of dynamical regimes attainable by the system. Most importantly, this strategy provides the means for a suitable assessment of the inherent variational constraints imposed by the structure of the system when systematically probing its parameter space.</div>','<b>Title: </b>SYNCHRONIZATION OF HETEROGENEOUS OSCILLATORS UNDER NETWORK MODIFICATIONS: PERTURBATION AND OPTIMIZATION OF THE SYNCHRONY ALIGNMENT FUNCTION<br><b>Year: </b>2016<br><b>Author: </b> D. Taylor,  Ps. Skardal and  J. Sun<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1137/16M1075181"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SIAM JOURNAL ON APPLIED MATHEMATICS<br><b>Cluster-Keywords: </b><em>oscillator networks, network dynamics, network, networks, dynamical systems, dynamics, node dynamics, systems, coupled oscillator, synchronization, dynamics network, complex network, coupled system, network modifications, oscillators, network connectivity, feedback vertex, network topology, synchronization properties, collective dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>Synchronization is central to many complex systems in engineering physics e.g., the power grid, Josephson junction circuits, and electrochemical oscillators and biology e.g., neuronal, circadian, and cardiac rhythms. Despite these widespread applications for which proper functionality depends sensitively on the extent of synchronization there remains a lack of understanding for how systems can best evolve and adapt to enhance or inhibit synchronization. We study how network modifications affect the synchronization properties of network-coupled dynamical systems that have heterogeneous node dynamics e.g., phase oscillators with nonidentical frequencies, which is often the case for real-world systems. Our approach relies on a synchrony alignment function SAF that quantifies the interplay between heterogeneity of the network and of the oscillators and provides an objective measure for a systems ability to synchronize. We conduct a spectral perturbation analysis of the SAF for structural network modifications including the addition and removal of edges, which subsequently ranks the edges according to their importance to synchronization. Based on this analysis, we develop gradient-descent algorithms to efficiently solve optimization problems that aim to maximize phase synchronization via network modifications. We support these and other results with numerical experiments.</div>','<b>Title: </b>A novel mutual information-based Boolean network inference method from time-series gene expression data<br><b>Year: </b>2017<br><b>Author: </b> S. Barman and  Yk. Kwon<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0171097"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>regulatory networks, gene regulatory, gene expression, network inference, expression data, network structure, gene networks, networks grns, regulatory genes, boolean networks, bayesian network, network reconstruction, networks gene, network dynamics, gene regulation, inference method, systems biology, time series, infer gene, network<div style="font-size:11px;"><br><b>Abstract: </b><em>BackgroundInferring a gene regulatory network from time-series gene expression data in systems biology is a challenging problem. Many methods have been suggested, most of which have a scalability limitation due to the combinatorial cost of searching a regulatory set of genes. In addition, they have focused on the accurate inference of a network structure only. Therefore, there is a pressing need to develop a network inference method to search regulatory genes efficiently and to predict the network dynamics accurately.ResultsIn this study, we employed a Boolean network model with a restricted update rule scheme to capture coarse-grained dynamics, and propose a novel mutual information-based Boolean network inference MIBNI method. Given time-series gene expression data as an input, the method first identifies a set of initial regulatory genes using mutual information-based feature selection, and then improves the dynamics prediction accuracy by iteratively swapping a pair of genes between sets of the selected regulatory genes and the other genes. Through extensive simulations with artificial datasets, MIBNI showed consistently better performance than six well-known existing methods, REVEAL, Best-Fit, RelNet, CST, CLR, and BIBN in terms of both structural and dynamics prediction accuracy. We further tested the proposed method with two real gene expression datasets for an Escherichia coli gene regulatory network and a fission yeast cell cycle network, and also observed better results using MIBNI compared to the six other methods.ConclusionsTaken together, MIBNI is a promising tool for predicting both the structure and the dynamics of a gene regulatory network.</div>','<b>Title: </b>Features of the reversible sensitivity-resistance transition in PI3K/PTEN/AKT signalling network after HER2 inhibition<br><b>Year: </b>2012<br><b>Author: </b> A. Goltsov,  D. Faratian,  Sp. Langdon,  P. Mullen,  Dj. Harrison and  J. Bown<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.cellsig.2011.09.030"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>CELLULAR SIGNALLING<br><b>Cluster-Keywords: </b><em>signaling network, systems biology, signal transduction, signaling pathways, receptor tyrosine, erbb signaling, erbb receptor, receptor signaling, intracellular signaling, mathematical models, cell signaling, computational model, factor receptor, receptor activation, growth factor, signaling models, networks signaling, receptor system, receptors systems, downstream signaling<div style="font-size:11px;"><br><b>Abstract: </b><em>Systems biology approaches that combine experimental data and theoretical modelling to understand cellular signalling network dynamics offer a useful platform to investigate the mechanisms of resistance to drug interventions and to identify combination drug treatments. Extending our work on modelling the PI3K/PTEN/AKT signalling network SN, we analyse the sensitivity of the SN output signal, phospho-AKT, to inhibition of HER2 receptor. We model typical aberrations in this SN identified in cancer development and drug resistance: loss of PTEN activity, PI3K and AKT mutations, HER2 overexpression, and overproduction of GSK3 beta and CK2 kinases controlling PTEN phosphorylation. We show that HER2 inhibition by the monoclonal antibody pertuzumab increases SN sensitivity, both to external signals and to changes in kinetic parameters of the proteins and their expression levels induced by mutations in the SN. This increase in sensitivity arises from the transition of SN functioning from saturation to non-saturation mode in response to HER2 inhibition. PTEN loss or PIK3CA mutation causes resistance to anti-HER2 inhibitor and leads to the restoration of saturation mode in SN functioning with a consequent decrease in SN sensitivity. We suggest that a drug-induced increase in SN sensitivity to internal perturbations, and specifically mutations, causes SN fragility. In particular, the SN is vulnerable to mutations that compensate for drug action and this may result in a sensitivity-to-resistance transition. The combination of HER2 and PI3K inhibition does not sensitise the SN to internal perturbations mutations in the PI3K/PTEN/AKT pathway: this combination treatment provides both synergetic inhibition and may prevent the SN from acquired mutations causing drug resistance. Through combination inhibition treatments, we studied the impact of upstream and downstream interventions to suppress resistance to the HER2 inhibitor in the SN with PTEN loss. Comparison of experimental results of PI3K inhibition in the PTEN upstream pathway with PDK1 inhibition in the PTEN downstream pathway shows that upstream inhibition abrogates resistance to pertuzumab more effectively than downstream inhibition. This difference in inhibition effect arises from the compensatory mechanism of an activation loop induced in the downstream pathway by PTEN loss. We highlight that drug target identification for combination anti-cancer therapy needs to account for the mutation effects on the upstream and downstream pathways. C 2011 Elsevier Inc. All rights reserved.</div>','<b>Title: </b>The Rate of Change in Evolutionary Systems and Evolutionary Economic Modeling<br><b>Year: </b>2018<br><b>Author: </b> T. Heinrich<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1080/00213624.2018.1469937"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF ECONOMIC ISSUES<br><b>Cluster-Keywords: </b><em>complex networks, complex systems, network theory, network structure, biological network, social network, network science, dynamic networks, network systems, network model, network topology, systems biology, network, network analysis, system dynamics, networks, realworld networks, network automata, epidemic network, model network<div style="font-size:11px;"><br><b>Abstract: </b><em>Evolutionary economics seeks to model socio-economic reality as an evolutionary system. This powerful approach entails the implication of the continuous loss of information through the evolutionary process. The implication corresponds to evolutionary biology, although the systems in evolutionary economics are different from those in evolutionary biology. The issue of the loss of information has not been extensively studied in economics. Many open questions remain: Which knowledge is lost under what circumstances Can loss of information be harmful to the socio-economic system as a whole in the presence of runaway dynamics caused by, for example, network externalities How can the development of knowledge in economic systems be studied The present article examines these questions and more.</div>','<b>Title: </b>Analysis of the Molecular Networks in Androgen Dependent and Independent Prostate Cancer Revealed Fragile and Robust Subsystems<br><b>Year: </b>2010<br><b>Author: </b> R. Tasseff,  S. Nayak,  S. Salim,  P. Kaushik,  N. Rizvi and  Jd. Varner<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0008864"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Androgen ablation therapy is currently the primary treatment for metastatic prostate cancer. Unfortunately, in nearly all cases, androgen ablation fails to permanently arrest cancer progression. As androgens like testosterone are withdrawn, prostate cancer cells lose their androgen sensitivity and begin to proliferate without hormone growth factors. In this study, we constructed and analyzed a mathematical model of the integration between hormone growth factor signaling, androgen receptor activation, and the expression of cyclin D and Prostate-Specific Antigen in human LNCaP prostate adenocarcinoma cells. The objective of the study was to investigate which signaling systems were important in the loss of androgen dependence. The model was formulated as a set of ordinary differential equations which described 212 species and 384 interactions, including both the mRNA and protein levels for key species. An ensemble approach was chosen to constrain model parameters and to estimate the impact of parametric uncertainty on model predictions. Model parameters were identified using 14 steady-state and dynamic LNCaP data sets taken from literature sources. Alterations in the rate of Prostatic Acid Phosphatase expression was sufficient to capture varying levels of androgen dependence. Analysis of the model provided insight into the importance of network components as a function of androgen dependence. The importance of androgen receptor availability and the MAPK/Akt signaling axes was independent of androgen status. Interestingly, androgen receptor availability was important even in androgen-independent LNCaP cells. Translation became progressively more important in androgen-independent LNCaP cells. Further analysis suggested a positive synergy between the MAPK and Akt signaling axes and the translation of key proliferative markers like cyclin D in androgen-independent cells. Taken together, the results support the targeting of both the Akt and MAPK pathways. Moreover, the analysis suggested that direct targeting of the translational machinery, specifically eIF4E, could be efficacious in androgen-independent prostate cancers.</div>','<b>Title: </b>New Measurement Methods of Network Robustness and Response Ability via Microarray Data<br><b>Year: </b>2013<br><b>Author: </b> Ct. Tu and  Bs. Chen<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0055230"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>systems biology, regulatory network, interaction networks, network model, gene regulatory, network, network biology, networks, gene network, response networks, network dynamics, dynamic model, cell networks, gene expression, biological networks, system dynamics, dynamic network, model, biological systems, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>"Robustness, the network ability to maintain systematic performance in the face of intrinsic perturbations, and "response ability, the network ability to respond to external stimuli or transduce them to downstream regulators, are two important complementary system characteristics that must be considered when discussing biological system performance. However, at present, these features cannot be measured directly for all network components in an experimental procedure. Therefore, we present two novel systematic measurement methods - Network Robustness Measurement NRM and Response Ability Measurement RAM - to estimate the network robustness and response ability of a gene regulatory network GRN or protein-protein interaction network PPIN based on the dynamic network model constructed by the corresponding microarray data. We demonstrate the efficiency of NRM and RAM in analyzing GRNs and PPINs, respectively, by considering aging- and cancer-related datasets. When applied to an aging-related GRN, our results indicate that such a network is more robust to intrinsic perturbations in the elderly than in the young, and is therefore less responsive to external stimuli. When applied to a PPIN of fibroblast and HeLa cells, we observe that the network of cancer cells possesses better robustness than that of normal cells. Moreover, the response ability of the PPIN calculated from the cancer cells is lower than that from healthy cells. Accordingly, we propose that generalized NRM and RAM methods represent effective tools for exploring and analyzing different systems-level dynamical properties via microarray data. Making use of such properties can facilitate prediction and application, providing useful information on clinical strategy, drug target selection, and design specifications of synthetic biology from a systems biology perspective.</div>','<b>Title: </b>Development of a yeast-based system to identify new hBRAFV600E functional interactors<br><b>Year: </b>2019<br><b>Author: </b> S. Lubrano,  L. Comelli,  C. Piccirilli,  A. Marranci,  F. Dapporto,  E. Tantillo,  F. Gemignani,  Js. Gutkind,  A. Salvetti,  G. Chiorino,  G. Cozza,  M. Chiariello,  A. Galli,  L. Poliseno and  T. Cervelli<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1038/s41388-018-0496-5"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ONCOGENE<br><b>Cluster-Keywords: </b><em>allosteric proteins, signaling networks, protein interactions, network model, molecular dynamics, protein complexes, allosteric network, signaling proteins, cell network, biochemical networks, free energy, combinatorial complexity, molecular species, biological systems, receptor signaling, systems biology, chemical species, model, dynamics simulations, cell signaling<div style="font-size:11px;"><br><b>Abstract: </b><em>BRAFV600E is a mutant Ser-Thr protein kinase that plays a crucial role in many types of cancer, including melanoma. Despite several aspects of BRAFV600E biology have been already elucidated, the proteins that regulate its expression and activity remain largely unknown, hampering our capacity to control its unrestrained effects. Here, we propose yeast Saccharomyces cerevisiae as a model system that can be used to achieve a better understanding of the regulation of human BRAFV600E.By showing that in osmotic stress conditions hBRAFV600E can rescue the growth of strains carrying a double or triple deletion in MAPKKK belonging to the HOG pathway, we demonstrate that this oncogenic kinase is active in yeast even if it does not have an ortholog. Moreover, we report that, in the yeast ptp3 Delta ptc1 Delta strain that is deleted in the genes encoding for two phosphatases responsible for Hog1 de-phoshorylation, hBRAFV600E mimics the toxicity observed in the presence of constitutive Hog1 activation. Finally, we exploit such a toxicity to perform a functional screening of a human cDNA library, looking for cDNAs able to rescue yeast growth. In this way, we identify SMIM10, a mitochondrial protein that in melanoma cells selectively downregulates BRAFV600E RNA and protein levels, by acting indirectly at the post-transcriptional level. Upon SMIM10 overexpression, BRAFV600E melanoma cells show disrupted mitochondrial structure/function and undergo senescence. They also show decreased ability to proliferate and form colonies, as well as increased sensitivity to the BRAF inhibitor vemurafenib. Interestingly, the analysis of TCGA melanoma samples indicates that patients with higher SMIM10 levels have a better prognosis. Therefore, these data suggest that SMIM10 exerts an oncosuppressive role in melanoma cells.Taken together, our results unveil the potential of S. cerevisiae to study hBRAFV600E, to populate the network of its functional interactors and, in doing so, to uncover new cancer-associated genes with therapeutic potential.</div>','<b>Title: </b>The balance between cell cycle arrest and cell proliferation: control by the extracellular matrix and by contact inhibition<br><b>Year: </b>2014<br><b>Author: </b> C. Gerard and  A. Goldbeter<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1098/rsfs.2013.0075"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>INTERFACE FOCUS<br><b>Cluster-Keywords: </b><em>cell cycle, mammalian cell, cdk network, cell proliferation, cell, cells cell, regulatory network, detailed model, cell size, molecular network, skeleton model, cycle arrest, cycle progression, cell division, cell density, kinases cdks, cycle, phase transition, cycle control, network driving<div style="font-size:11px;"><br><b>Abstract: </b><em>To understand the dynamics of the cell cycle, we need to characterize the balance between cell cycle arrest and cell proliferation, which is often deregulated in cancers. We address this issue by means of a detailed computational model for the network of cyclin-dependent kinases Cdks driving the mammalian cell cycle. Previous analysis of the model focused on how this balance is controlled by growth factors GFs or the levels of activators oncogenes and inhibitors tumour suppressors of cell cycle progression. Supra-threshold changes in the level of any of these factors can trigger a switch in the dynamical behaviour of the Cdk network corresponding to a bifurcation between a stable steady state, associated with cell cycle arrest, and sustained oscillations of the various cyclin/Cdk complexes, corresponding to cell proliferation. Here, we focus on the regulation of cell proliferation by cellular environmental factors external to the Cdk network, such as the extracellular matrix ECM, and contact inhibition, which increases with cell density. We extend the model for the Cdk network by including the phenomenological effect of both the ECM, which controls the activation of the focal adhesion kinase FAK that promotes cell cycle progression, and cell density, which inhibits cell proliferation via the Hippo/YAP pathway. The model shows that GFs and FAK activation are capable of triggering in a similar dynamical manner the transition to cell proliferation, while the Hippo/YAP pathway can arrest proliferation once cell density passes a critical threshold. The results account for the dependence or independence of cell proliferation on serum and/or cell anchorage to ECM. Whether the balance in the Cdk network is tilted towards cell cycle arrest or proliferation depends on the direction in which the threshold associated with the bifurcation is passed once the cell integrates the multiple, internal or external signals that promote or impede progression in the cell cycle.</div>','<b>Title: </b>Stochastic and deterministic multiscale models for systems biology: an auxin-transport case study<br><b>Year: </b>2010<br><b>Author: </b> J. Twycross,  Lr. Band,  Mj. Bennett,  Jr. King and  N. Krasnogor<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-4-34"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>network ecology, food webs, network models, ecological systems, food web, systems biology, ecological networks, ecosystem model, signalling networks, model results, interaction networks, dynamic model, ecological interactions, network analysis, model network, network structure, transport networks, systems network, ecosystem dynamics, restoration ecology<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Stochastic and asymptotic methods are powerful tools in developing multiscale systems biology models; however, little has been done in this context to compare the efficacy of these methods. The majority of current systems biology modelling research, including that of auxin transport, uses numerical simulations to study the behaviour of large systems of deterministic ordinary differential equations, with little consideration of alternative modelling frameworks.Results: In this case study, we solve an auxin-transport model using analytical methods, deterministic numerical simulations and stochastic numerical simulations. Although the three approaches in general predict the same behaviour, the approaches provide different information that we use to gain distinct insights into the modelled biological system. We show in particular that the analytical approach readily provides straightforward mathematical expressions for the concentrations and transport speeds, while the stochastic simulations naturally provide information on the variability of the system.Conclusions: Our study provides a constructive comparison which highlights the advantages and disadvantages of each of the considered modelling approaches. This will prove helpful to researchers when weighing up which modelling approach to select. In addition, the paper goes some way to bridging the gap between these approaches, which in the future we hope will lead to integrative hybrid models.</div>','<b>Title: </b>Molecular modeling of ErbB4/HER4 kinase in the context of the HER4 signaling network helps rationalize the effects of clinically identified HER4 somatic mutations on the cell phenotype<br><b>Year: </b>2013<br><b>Author: </b> Se. Telesco,  R. Vadigepalli and  R. Radhakrishnan<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1002/biot.201300022"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOTECHNOLOGY JOURNAL<br><b>Cluster-Keywords: </b><em>signaling network, systems biology, signal transduction, signaling pathways, receptor tyrosine, erbb signaling, erbb receptor, receptor signaling, intracellular signaling, mathematical models, cell signaling, computational model, factor receptor, receptor activation, growth factor, signaling models, networks signaling, receptor system, receptors systems, downstream signaling<div style="font-size:11px;"><br><b>Abstract: </b><em>In the ErbB/HER family of receptor tyrosine kinases, the deregulation of the EGFR/ErbB1/HER1, HER2/ErbB2, and HER3/ErbB3 kinases is associated with several cancers, while the HER4/ErbB4 kinase has been shown to play an anti-carcinogenic role in certain tumors. We present molecular and network models of HER4/ErbB4 activation and signaling in order to elucidate molecular mechanisms of activation and rationalize the effects of the clinically identified HER4 somatic mutants. Our molecular-scale simulations identify the important role played by the interactions within the juxtamembrane region during the activation process. Our results also support the hypothesis that the HER4 mutants may heterodimerize but not activate, resulting in blockage of the HER4-STAT5 differentiation pathway, in favor of the proliferative PI3K/AKT pathway. Translating our molecular simulation results into a cellular pathway model of wild type versus mutant HER4 signaling, we are able to recapitulate the major features of the PI3K/AKT and JAK/STAT activation downstream of HER4. Our model predicts that the signaling downstream of the wild type HER4 is enriched for the JAK-STAT pathway, whereas downstream of the mutant HER4 is enriched for the PI3K/AKT pathway. HER4 mutations may hence constitute a cellular shift from a program of differentiation to that of proliferation.</div>','<b>Title: </b>Control and Regulation of Substrate Selection in Cytoplasmic and Mitochondrial Catabolic Networks. A Systems Biology Analysis<br><b>Year: </b>2019<br><b>Author: </b> S. Cortassa,  Ma. Aon and  Sj. Sollott<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3389/fphys.2019.00201"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>FRONTIERS IN PHYSIOLOGY<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Appropriate substrate selection between fats and glucose is associated with the success of interventions that maintain health such as exercise or caloric restriction, or with the severity of diseases such as diabetes or other metabolic disorders. Although the interaction and mutual inhibition between glucose and fatty-acids FAs catabolism has been studied for decades, a quantitative and integrated understanding of the control and regulation of substrate selection through central catabolic pathways is lacking. We addressed this gap here using a computational model representing cardiomyocyte catabolism encompassing glucose Glc utilization, pyruvate transport into mitochondria and oxidation in the tricarboxylic acid TCA cycle, beta-oxidation of palmitate Palm, oxidative phosphorylation, ion transport, pH regulation, and ROS generation and scavenging in cytoplasmic and mitochondrial compartments. The model is described by 82 differential equations and 119 enzymatic, electron transport and substrate transport reactions accounting for regulatory mechanisms and key players, namely pyruvate dehydrogenase PDH and its modulation by multiple effectors. We applied metabolic control analysis to the network operating with various Glc to Palm ratios. The flux and metabolites concentration control were visualized through heat maps providing major insights into main control and regulatory nodes throughout the catabolic network. Metabolic pathways located in different compartments were found to reciprocally control each other. For example, glucose uptake and the ATP demand exert control on most processes in catabolism while TCA cycle activities and membrane-associated energy transduction reactions exerted control on mitochondrial processes namely beta-oxidation. PFK and PDH, two highly regulated enzymes, exhibit opposite behavior from a control perspective. While PFK activity was a main rate-controlling step affecting the whole network, PDH played the role of a major regulator showing high sensitivity elasticity to substrate availability and key activators/inhibitors, a trait expected from a flexible substrate selector strategically located in the metabolic network. PDH regulated the rate of Glc and Palm consumption, consistent with its high sensitivity toward AcCoA, CoA, and NADH. Overall, these results indicate that the control of catabolism is highly distributed across the metabolic network suggesting that fuel selection between FAs and Glc goes well beyond the mechanisms traditionally postulated to explain the glucose-fatty-acid cycle.</div>','<b>Title: </b>Extraction of elementary rate constants from global network analysis of E. coli central metabolism<br><b>Year: </b>2008<br><b>Author: </b> J. Zhao,  D. Ridgway,  G. Broderick,  A. Kovalenko and  M. Ellison<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-2-41"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>kinetic models, metabolic networks, kinetic parameters, dynamic models, systems biology, network model, dynamic metabolic, biochemical network, rate constants, enzyme kinetics, kinetic data, adjustable parameters, metabolic model, parameter estimation, dynamic optimization, model selection, mathematical model, kinetic network, model response, model<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: As computational performance steadily increases, so does interest in extending one-particle-per-molecule models to larger physiological problems. Such models however require elementary rate constants to calculate time-dependent rate coefficients under physiological conditions. Unfortunately, even when in vivo kinetic data is available, it is often in the form of aggregated rate laws ARL that do not specify the required elementary rate constants corresponding to mass-action rate laws MRL. There is therefore a need to develop a method which is capable of automatically transforming ARL kinetic information into more detailed MRL rate constants.Results: By incorporating proteomic data related to enzyme abundance into an MRL modelling framework, here we present an efficient method operating at a global network level for extracting elementary rate constants from experiment-based aggregated rate law  ARL models. The method combines two techniques that can be used to overcome the difficult properties in parameterization. The first, a hybrid MRL/ARL modelling technique, is used to divide the parameter estimation problem into sub-problems, so that the parameters of the mass action rate laws for each enzyme are estimated in separate steps. This reduces the number of parameters that have to be optimized simultaneously. The second, a hybrid algebraic-numerical simulation and optimization approach, is used to render some rate constants identifiable, as well as to greatly narrow the bounds of the other rate constants that remain unidentifiable. This is done by incorporating equality constraints derived from the King-Altman and Cleland method into the simulated annealing algorithm. We apply these two techniques to estimate the rate constants of a model of E. coli glycolytic pathways. The simulation and statistical results show that our innovative method performs well in dealing with the issues of high computation cost, stiffness, local minima and uncertainty inherent with large-scale non-convex nonlinear MRL models.Conclusion: In short, this new hybrid method can ensure the proper solution of a challenging parameter estimation problem of nonlinear dynamic MRL systems, while keeping the computational effort reasonable. Moreover, the work provides us with some optimism that physiological models at the particle scale can be rooted on a firm foundation of parameters generated in the macroscopic regime on an experimental basis. Thus, the proposed method should have applications to multi-scale modelling of the real biological systems allowing for enzyme intermediates, stochastic and spatial effects inside a cell.</div>','<b>Title: </b>Supervised Network-Based Fuzzy Learning of EEG Signals for Alzheimers Disease Identification<br><b>Year: </b>2020<br><b>Author: </b> Ht. Yu,  Xy. Lei,  Zx. Song,  C. Liu and  J. Wang<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1109/TFUZZ.2019.2903753"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IEEE TRANSACTIONS ON FUZZY SYSTEMS<br><b>Cluster-Keywords: </b><em>gene regulatory, regulatory network, systems biology, cell differentiation, cell fate, model systems, potential landscape, network grn, dynamical systems, melanocyte differentiation, network models, gene expression, network dynamics, differentiation genes, dynamical models, dynamical networks, cellular differentiation, stem cells, attractor network, developmental gene<div style="font-size:11px;"><br><b>Abstract: </b><em>Accurate identification of Alzheimers disease AD with electroencephalograph EEG is crucial in the clinical diagnosis of neurological disorders. However, the effectiveness and accuracy of manually labeling EEG signals are barely satisfactory, due to lacking effective biomarkers. In this paper, we propose a novel machine learning method network-based Takagi-Sugeno-Kang N-TSK for AD identification which employs the complex network theory and TSK fuzzy system. With the construction of functional network of AD subjects, the topological features of weighted and unweighted networks are extracted. Taken the network parameters as independent inputs, a fuzzy-system-based TSK model is established and further trained to identify AD EEG signals. Experimental results demonstrate the effectiveness of the proposed scheme in AD identification and ability of N-TSK fuzzy classifiers. The highest accuracy can achieve 97.3 for patients with closed eyes and 94.78 with open eyes. In addition, the performance of weighted N-TSK largely exceeds unweighted N-TSK. By further optimizing the network features utilized in the N-TSK fuzzy classifiers, it is found that local efficiency and clustering coefficient are the most effective factors in AD identification. This work provides a potential tool for identifying neurological disorders from the perspective of functional networks with EEG signal, especially contributing to the diagnosis and identification of AD.</div>','<b>Title: </b>Reverse engineering module networks by PSO-RNN hybrid modeling<br><b>Year: </b>2009<br><b>Author: </b> Yj. Zhang,  Jh. Xuan,  Bg. De Los Reyes,  R. Clarke and  Hw. Ressom<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1471-2164-10-S1-S15"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC GENOMICS<br><b>Cluster-Keywords: </b><em>gene regulatory, gene expression, regulatory networks, expression data, regulatory pathway, regulatory genes, network dynamics, microarray data, genetic networks, transcription factor, models network, target gene, upstream regulatory, network topology, pathway network, gene functional, expression profile, transcriptional network, dynamic model, transcriptional regulatory<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Inferring a gene regulatory network GRN from high throughput biological data is often an under-determined problem and is a challenging task due to the following reasons: 1 thousands of genes are involved in one living cell; 2 complex dynamic and nonlinear relationships exist among genes; 3 a substantial amount of noise is involved in the data, and 4 the typical small sample size is very small compared to the number of genes. We hypothesize we can enhance our understanding of gene interactions in important biological processes differentiation, cell cycle, and development, etc and improve the inference accuracy of a GRN by 1 incorporating prior biological knowledge into the inference scheme, 2 integrating multiple biological data sources, and 3 decomposing the inference problem into smaller network modules.Results: This study presents a novel GRN inference method by integrating gene expression data and gene functional category information. The inference is based on module network model that consists of two parts: the module selection part and the network inference part. The former determines the optimal modules through fuzzy c-mean FCM clustering and by incorporating gene functional category information, while the latter uses a hybrid of particle swarm optimization and recurrent neural network PSO-RNN methods to infer the underlying network between modules. Our method is tested on real data from two studies: the development of rat central nervous system CNS and the yeast cell cycle process. The results are evaluated by comparing them to previously published results and gene ontology annotation information.Conclusion: The reverse engineering of GRNs in time course gene expression data is a major obstacle in system biology due to the limited number of time points. Our experiments demonstrate that the proposed method can address this challenge by: 1 preprocessing gene expression data e. g. normalization and missing value imputation to reduce the data noise; 2 clustering genes based on gene expression data and gene functional category information to identify biologically meaningful modules, thereby reducing the dimensionality of the data; 3 modeling GRNs with the PSO-RNN method between the modules to capture their nonlinear and dynamic relationships. The method is shown to lead to biologically meaningful modules and networks among the modules.</div>','<b>Title: </b>Analysis of the steady-state relations and control-algorithm characterisation in a mathematical model of cholesterol biosynthesis<br><b>Year: </b>2013<br><b>Author: </b> A. Belic,  J. Acimovic,  A. Naik and  M. Golicnik<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.simpat.2011.12.010"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SIMULATION MODELLING PRACTICE AND THEORY<br><b>Cluster-Keywords: </b><em>gene expression, systems biology, metabolic network, network model, network analysis, dynamic model, signaling pathways, signaling network, regulatory network, dynamic network, biology approach, candidate genes, gene network, gene regulatory, biology analysis, mathematical model, network dynamic, biology network, metabolic pathways, pathway analysis<div style="font-size:11px;"><br><b>Abstract: </b><em>Elevated levels of cholesterol are known to be a risk factor for cardiovascular diseases. As a result, several treatment strategies and drugs have been developed to control these elevated cholesterol levels, but they are not always successful. Statins are now the most widely used cholesterol-lowering drugs; however, not all the mechanisms of their action are understood and this can sometimes lead to adverse effects. A dynamic mathematical model of the cholesterol biosynthesis network was developed with aim to understand the key mechanisms of cholesterol biosynthesis. In this article we show that in spite of a serious lack of experimental data, the model can be used to study the concepts of possible mechanisms of cholesterol biosynthesis and drug interactions. If only steady-state data is used for the models identification, the model can predict the steady-state relations after perturbation correctly, while the dynamical properties are not necessarily related to the real system. The control mechanism for the cholesterol levels through the SREBF-2 transcription factor was identified as the PI control algorithm. The comparison of model simulations and performed biological experiments indicated that the substances LK-980 and Atorvastatin most likely trigger the same indirect mechanism of cholesterol biosynthesis control, although they interact with the network in different ways. C 2011 Elsevier B.V. All rights reserved.</div>','<b>Title: </b>POMO - Plotting Omics analysis results for Multiple Organisms<br><b>Year: </b>2013<br><b>Author: </b> Jk. Lin,  R. Kreisberg,  A. Kallio,  Am. Dudley,  M. Nykter,  I. Shmulevich,  P. May and  R. Autio<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1471-2164-14-918"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC GENOMICS<br><b>Cluster-Keywords: </b><em>gene regulatory, systems biology, regulatory networks, gene network, stem cell, network model, transcription factors, gene expression, cell transcriptomes, network analysis, data analysis, stem cells, network grnsight, cell transformation, weighted network, gene ontology, networks grns, graph model, graph layout, endothelial cell<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Systems biology experiments studying different topics and organisms produce thousands of data values across different types of genomic data. Further, data mining analyses are yielding ranked and heterogeneous results and association networks distributed over the entire genome. The visualization of these results is often difficult and standalone web tools allowing for custom inputs and dynamic filtering are limited.Results: We have developed POMO http://pomo.cs.tut.fi, an interactive web-based application to visually explore omics data analysis results and associations in circular, network and grid views. The circular graph represents the chromosome lengths as perimeter segments, as a reference outer ring, such as cytoband for human. The inner arcs between nodes represent the uploaded network. Further, multiple annotation rings, for example depiction of gene copy number changes, can be uploaded as text files and represented as bar, histogram or heatmap rings. POMO has built-in references for human, mouse, nematode, fly, yeast, zebrafish, rice, tomato, Arabidopsis, and Escherichia coli. In addition, POMO provides custom options that allow integrated plotting of unsupported strains or closely related species associations, such as human and mouse orthologs or two yeast wild types, studied together within a single analysis. The web application also supports interactive label and weight filtering. Every iterative filtered result in POMO can be exported as image file and text file for sharing or direct future input.Conclusions: The POMO web application is a unique tool for omics data analysis, which can be used to visualize and filter the genome-wide networks in the context of chromosomal locations as well as multiple network layouts. With the several illustration and filtering options the tool supports the analysis and visualization of any heterogeneous omics data analysis association results for many organisms. POMO is freely available and does not require any installation or registration.</div>','<b>Title: </b>Approximating the maximum weight clique using replicator dynamics<br><b>Year: </b>2000<br><b>Author: </b> Im. Bomze,  M. Pelillo and  V. StixLink: No link<br><b>Journal: </b>IEEE TRANSACTIONS ON NEURAL NETWORKS<br><b>Cluster-Keywords: </b><em>network models, population model, field model, neural population, neural field, iot networks, neuronal networks, neural network, differential equations, parameter neural, lotkavolterra model, model, neuron models, model neurons, model leads, ecosystem model, iot devices, noise model, neuronal models, inhibitory synaptic<div style="font-size:11px;"><br><b>Abstract: </b><em>Given an undirected graph with weights on the vertices, the maximum weight clique problem MWCP is to find a subset of mutually adjacent vertices i.e., a clique having the largest total weight, This is a generalization of the classical problem of finding the maximum cardinality clique of an unweighted graph, which arises as a special case of the MWCP when all the weights associated to the vel tices are equal, The problem is known to be NP-hard for arbitrary graphs and, according to recent theoretical results, so is the problem of approximating it within a constant factor. Although there has recently been much interest around neural-network algorithms for the unweighted maximum clique problem, no effort has been directed so far toward its weighted counterpart. In this paper, we present a parallel, distributed heuristic for approximating the MWCP based on dynamics principles de,eloped and studied in various branches of mathematical biology, The proposed framework centers around a recently introduced continuous characterization of the MWCP which generalizes an earlier remarkable result by Motzkin and Straus. This allows us to Formulate the MWCP a purely combinatorial problem in terms of a continuous quadratic programming problem. One drawback associated with this formulation, however, is the presence of "spurious" solutions, and we present characterizations of these solutions. To avoid them we introduce a new regularized continuous formulation of the MWCP inspired by previous works on the unweighted problem, and show how this approach completely solves the problem. The continuous formulation of the MWCP naturally maps onto a parallel, distributed computational network whose dynamical behavior is governed by the so-called replicator equations. These are dynamical systems introduced in evolutionary; game theory and population genetics to model evolutionary processes on a macroscopic scale. We present theoretical results which guarantee that the solutions provided by our clique finding replicator network are actually the ones being sought. Extensive experiments on both randomly generated and standard benchmark graphs have been conducted, end the results obtained confirm the effectiveness of the proposed approach.</div>','<b>Title: </b>A gene regulatory architecture that controls region-independent dynamics of oligodendrocyte differentiation<br><b>Year: </b>2019<br><b>Author: </b> M. Cantone,  M. Kuspert,  S. Reiprich,  X. Lai,  M. Eberhardt,  P. Gottle,  F. Beyer,  K. Azim,  P. Kury,  M. Wegner and  J. Vera<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1002/glia.23569"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>GLIA<br><b>Cluster-Keywords: </b><em>regulatory network, systems biology, systems biological, network dynamics, gene expression, biochemical networks, raft proteins, biological networks, gene circuits, gene regulatory, signalling network, lipid raft, nuclear lamina, protein domain, networks systems, biological approaches, expression pattern, protein interaction, cell differentiation, network motifs<div style="font-size:11px;"><br><b>Abstract: </b><em>Oligodendrocytes OLs facilitate information processing in the vertebrate central nervous system via axonal ensheathment. The structure and dynamics of the regulatory network that mediates oligodendrogenesis are poorly understood. We employed bioinformatics and meta-analysis of high-throughput datasets to reconstruct a regulatory network underpinning OL differentiation. From this network, we identified families of feedforward loops comprising the transcription factors TFs Olig2, Sox10, and Tcf7l2 and their targets. Among the targets, we found eight other TFs related to OL differentiation, suggesting a hierarchical architecture in which some TFs Olig2, Sox10, and Tcf7l2 regulate via feedforward loops the expression of others Sox2, Sox6, Sox11, Nkx2-2, Nkx6-2, Hes5, Myt1, and Myrf. Model simulations with a kinetic model reproduced the mechanisms of OL differentiation only when in the model, Sox10-mediated repression of Tcf7l2 by miR-338/miR-155 was introduced, a prediction confirmed in genetic functional experiments. Additional model simulations suggested that OLs from dorsal regions emerge through BMP/Sox9 signaling.</div>','<b>Title: </b>A structured approach for the engineering of biochemical network models, illustrated for signalling pathways<br><b>Year: </b>2008<br><b>Author: </b> R. Breitling,  D. Gilbert,  M. Heiner and  R. Orton<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1093/bib/bbn026"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BRIEFINGS IN BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>petri net, petri nets, bayesian networks, biological networks, systems biology, regulatory networks, dynamic bayesian, net model, gene regulatory, network data, biological systems, biological data, network analysis, network models, biochemical networks, time series, network structure, signal transduction, net models, dbn model<div style="font-size:11px;"><br><b>Abstract: </b><em>Quantitative models of biochemical networks signal transduction cascades, metabolic pathways, gene regulatory circuits are a central component of modern systems biology. Building and managing these complex models is a major challenge that can benefit from the application of formal methods adopted from theoretical computing science. Here we provide a general introduction to the field of formal modelling, which emphasizes the intuitive biochemical basis of the modelling process, but is also accessible for an audience with a background in computing science and/or model engineering. We show how signal transduction cascades can be modelled in a modular fashion, using both a qualitative approachqualitative Petri nets, and quantitative approachescontinuous Petri nets and ordinary differential equations ODEs. We review the major elementary building blocks of a cellular signalling model, discuss which critical design decisions have to be made during model building, and present a number of novel computational tools that can help to explore alternative modular models in an easy and intuitive manner. These tools, which are based on Petri net theory, offer convenient ways of composing hierarchical ODE models, and permit a qualitative analysis of their behaviour. We illustrate the central concepts using signal transduction as our main example. The ultimate aim is to introduce a general approach that provides the foundations for a structured formal engineering of large-scale models of biochemical networks.</div>','<b>Title: </b>Feedback through graph motifs relates structure and function in complex networks<br><b>Year: </b>2018<br><b>Author: </b> Y. Hu,  Sl. Brunton,  N. Cain,  S. Mihalas,  Jn. Kutz and  E. Shea-Brown<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1103/PhysRevE.98.062312"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PHYSICAL REVIEW E<br><b>Cluster-Keywords: </b><em>complex networks, complex systems, network theory, network structure, biological network, social network, network science, dynamic networks, network systems, network model, network topology, systems biology, network, network analysis, system dynamics, networks, realworld networks, network automata, epidemic network, model network<div style="font-size:11px;"><br><b>Abstract: </b><em>In physics, biology, and engineering, network systems abound. How does the connectivity of a network system combine with the behavior of its individual components to determine its collective function We approach this question for networks with linear time-invariant dynamics by relating internal network feedbacks to the statistical prevalence of connectivity motifs, a set of surprisingly simple and local statistics of connectivity. This results in a reduced order model of the network input-output dynamics in terms of motif structures. As an example, the formulation dramatically simplifies the classic Erdos-Renyi graph, reducing the overall network behavior to one proportional feedback wrapped around the dynamics of a single node. For general networks, higher-order motifs systematically provide further layers and types of feedback to regulate the network response. Thus, the local connectivity shapes temporal and spectral processing by the network as a whole, and we show how this enables robust, yet tunable, functionality such as extending the time constant with which networks remember past signals. The theory also extends to networks composed from heterogeneous nodes with distinct dynamics and connectivity, and patterned input to and readout from subsets of nodes. These statistical descriptions provide a powerful theoretical framework to understand the functionality of real-world network systems, as we illustrate with examples including the mouse brain connectome.</div>','<b>Title: </b>Probing Molecular Mechanisms of the Hsp90 Chaperone: Biophysical Modeling Identifies Key Regulators of Functional Dynamics<br><b>Year: </b>2012<br><b>Author: </b> A. Dixit and  Gm. Verkhivker<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0037605"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>normal mode, mode analysis, elastic network, functional motions, functional dynamics, molecular chaperone, structural biology, acidic residues, network models, molecular dynamics, atomic resolution, protein complexes, conserved functional, hsp dynamics, correlated motions, protein folding, large biomolecular, thioredoxin system, harmonic analysis, protein motions<div style="font-size:11px;"><br><b>Abstract: </b><em>Deciphering functional mechanisms of the Hsp90 chaperone machinery is an important objective in cancer biology aiming to facilitate discovery of targeted anti-cancer therapies. Despite significant advances in understanding structure and function of molecular chaperones, organizing molecular principles that control the relationship between conformational diversity and functional mechanisms of the Hsp90 activity lack a sufficient quantitative characterization. We combined molecular dynamics simulations, principal component analysis, the energy landscape model and structure-functional analysis of Hsp90 regulatory interactions to systematically investigate functional dynamics of the molecular chaperone. This approach has identified a network of conserved regions common to the Hsp90 chaperones that could play a universal role in coordinating functional dynamics, principal collective motions and allosteric signaling of Hsp90. We have found that these functional motifs may be utilized by the molecular chaperone machinery to act collectively as central regulators of Hsp90 dynamics and activity, including the inter-domain communications, control of ATP hydrolysis, and protein client binding. These findings have provided support to a long-standing assertion that allosteric regulation and catalysis may have emerged via common evolutionary routes. The interaction networks regulating functional motions of Hsp90 may be determined by the inherent structural architecture of the molecular chaperone. At the same time, the thermodynamics-based "conformational selection" of functional states is likely to be activated based on the nature of the binding partner. This mechanistic model of Hsp90 dynamics and function is consistent with the notion that allosteric networks orchestrating cooperative protein motions can be formed by evolutionary conserved and sparsely connected residue clusters. Hence, allosteric signaling through a small network of distantly connected residue clusters may be a rather general functional requirement encoded across molecular chaperones. The obtained insights may be useful in guiding discovery of allosteric Hsp90 inhibitors targeting protein interfaces with co-chaperones and protein binding clients.</div>','<b>Title: </b>Improvements in the reconstruction of time-varying gene regulatory networks: dynamic programming and regularization by information sharing among genes<br><b>Year: </b>2011<br><b>Author: </b> M. Grzegorczyk and  D. Husmeier<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1093/bioinformatics/btq711"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>petri net, petri nets, bayesian networks, biological networks, systems biology, regulatory networks, dynamic bayesian, net model, gene regulatory, network data, biological systems, biological data, network analysis, network models, biochemical networks, time series, network structure, signal transduction, net models, dbn model<div style="font-size:11px;"><br><b>Abstract: </b><em>Method: Dynamic Bayesian networks DBNs have been applied widely to reconstruct the structure of regulatory processes from time series data, and they have established themselves as a standard modelling tool in computational systems biology. The conventional approach is based on the assumption of a homogeneous Markov chain, and many recent research efforts have focused on relaxing this restriction. An approach that enjoys particular popularity is based on a combination of a DBN with a multiple changepoint process, and the application of a Bayesian inference scheme via reversible jump Markov chain Monte Carlo RJMCMC. In the present article, we expand this approach in two ways. First, we show that a dynamic programming scheme allows the changepoints to be sampled from the correct conditional distribution, which results in improved convergence over RJMCMC. Second, we introduce a novel Bayesian clustering and information sharing scheme among nodes, which provides a mechanism for automatic model complexity tuning.Results: We evaluate the dynamic programming scheme on expression time series for Arabidopsis thaliana genes involved in circadian regulation. In a simulation study we demonstrate that the regularization scheme improves the network reconstruction accuracy over that obtained with recently proposed inhomogeneous DBNs. For gene expression profiles from a synthetically designed Saccharomyces cerevisiae strain under switching carbon metabolism we show that the combination of both: dynamic programming and regularization yields an inference procedure that outperforms two alternative established network reconstruction methods from the biology literature.</div>','<b>Title: </b>Dark proteins: Effect of inclusion body formation on quantification of protein expression<br><b>Year: </b>2008<br><b>Author: </b> Maj. Iafolla,  M. Mazumder,  V. Sardana,  T. Velauthapillai,  K. Pannu and  Dr. Mcmillen<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1002/prot.22024"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PROTEINS-STRUCTURE FUNCTION AND BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>gene expression, gene network, stochastic model, gene regulatory, systems biology, expression systems, gene regulation, network activity, expression dynamics, noise control, regulatory network, deterministic models, network components, stochastic modeling, regulation system, escherichia coli, regulatory system, synthetic gene, regulatory genes, gene<div style="font-size:11px;"><br><b>Abstract: </b><em>Plasmid-borne gene expression systems have found wide application in the emerging fields of systems biology and synthetic biology, where plasmids are used to implement simple network architectures, either to test systems biology hypotheses about issues such as gene expression noise or as a means of exerting artificial control over a cells dynamics. In both these cases, fluorescent proteins are commonly applied as a means of monitoring tire expression of genes in tire living cell, and efforts have been made to quantify protein expression levels through fluorescence intensity calibration and by monitoring the partitioning of proteins among the two daughter cells after division; such quantification is important in formulating the predictive models desired in systems and synthetic biology research. A potential pitfall of using plasmid-based gene expression systems is that the high protein levels associated with expression from plasmids can lead to the formation of inclusion bodies, insoluble aggregates of misfolded, nonfunctional proteins that will not generate fluorescence output; proteins caught in these inclusion bodies are thus "dark" to fluorescence-based detection methods. If significant numbers of proteins are incorporated into inclusion bodies rather than becoming biologically active, quantitative results obtained by fluorescent measurements will be skewed, we investigate this phenomenon here. We have created two plasmid constructs with differing average copy numbers, both incorporating air unregulated promoter PLtetO-1 in tire absence of TetR expressing the GFP derivative enhanced green fluorescent protein EGFP, and inserted them into Escherichia coli bacterial cells a common model organism for work on the dynamics of prokaryotic gene expression. We extracted tire inclusion bodies, denatured them, and refolded them to reader them active, obtaining a measurement of the average number of EGFP per cell locked into these aggregates, at tire same time, we used calibrated fluorescent intensity measurements to determine the average number of active EGFP present per cell. Both measurements were carried out as a function of cellular doubling time, over a range of 45-75 min. We found that tire ratio of inclusion body EGFP to active EGFP varied strongly as a function of the cellular growth rate, and that the number of "dark" proteins in the aggregates could in fact be substantial, reaching ratios as high as approximately five proteins locked into inclusion bodies for every active protein at the fastest growth rate, and dropping to ratios well below I for the slowest growth rate. Our results suggest that efforts to compare compulational models to protein numbers derived from fluorescence measurements should take inclusion body loss into account, especially when working with rapidly growing cells.</div>','<b>Title: </b>Novel recurrent neural network for modelling biological networks: Oscillatory p53 interaction dynamics<br><b>Year: </b>2013<br><b>Author: </b> H. Ling,  S. Samarasinghe and  D. Kulasiri<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.biosystems.2013.08.004"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOSYSTEMS<br><b>Cluster-Keywords: </b><em>dna damage, signalling networks, network dynamics, pmdm system, systems biology, systems dynamics, biological systems, single cell, kinase inhibitors, signaling network, biological networks, cellular automata, cell cycle, oscillation system, temporal dynamics, signaling pathway, individual cells, signaling processes, dynamic model, damage response<div style="font-size:11px;"><br><b>Abstract: </b><em>Understanding the control of cellular networks consisting of gene and protein interactions and their emergent properties is a central activity of Systems Biology research. For this, continuous, discrete, hybrid, and stochastic methods have been proposed. Currently, the most common approach to modelling accurate temporal dynamics of networks is ordinary differential equations ODE. However, critical limitations of ODE models are difficulty in kinetic parameter estimation and numerical solution of a large number of equations, making them more suited to smaller systems. In this article, we introduce a novel recurrent artificial neural network RNN that addresses above limitations and produces a continuous model that easily estimates parameters from data, can handle a large number of molecular interactions and quantifies temporal dynamics and emergent systems properties. This RNN is based on a system of ODEs representing molecular interactions in a signalling network. Each neuron represents concentration change of one molecule represented by an ODE. Weights of the RNN correspond to kinetic parameters in the system and can be adjusted incrementally during network training. The method is applied to the p53-Mdm2 oscillation system a crucial component of the DNA damage response pathways activated by a damage signal. Simulation results indicate that the proposed RNN can successfully represent the behaviour of the p53-Mdm2 oscillation system and solve the parameter estimation problem with high accuracy. Furthermore, we presented a modified form of the RNN that estimates parameters and captures systems dynamics from sparse data collected over relatively large time steps. We also investigate the robustness of the p53-Mdm2 system using the trained RNN under various levels of parameter perturbation to gain a greater understanding of the control of the p53-Mdm2 system. Its outcomes on robustness are consistent with the current biological knowledge of this system. As more quantitative data become available on individual proteins, the RNN would be able to refine parameter estimation and mapping of temporal dynamics of individual signalling molecules as well as signalling networks as a system. Moreover, RNN can be used to modularise large signalling networks. C 2013 Elsevier Ireland Ltd. All rights reserved.</div>','<b>Title: </b>Mathematical modeling and computational prediction of cancer drug resistance<br><b>Year: </b>2018<br><b>Author: </b> Xq. Sun and  B. Hu<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1093/bib/bbx065"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BRIEFINGS IN BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>boolean network, network models, qualitative models, systems biology, mathematical modeling, boolean models, signaling networks, dynamic models, dynamic modeling, signaling dynamics, logic models, dynamic network, mathematical models, boolean modeling, signal transduction, regulatory networks, drug resistance, biological networks, network approach, network modeling<div style="font-size:11px;"><br><b>Abstract: </b><em>Diverse forms of resistance to anticancer drugs can lead to the failure of chemotherapy. Drug resistance is one of the most intractable issues for successfully treating cancer in current clinical practice. Effective clinical approaches that could counter drug resistance by restoring the sensitivity of tumors to the targeted agents are urgently needed. As numerous experimental results on resistance mechanisms have been obtained and a mass of high-throughput data has been accumulated, mathematical modeling and computational predictions using systematic and quantitative approaches have become increasingly important, as they can potentially provide deeper insights into resistance mechanisms, generate novel hypotheses or suggest promising treatment strategies for future testing. In this review, we first briefly summarize the current progress of experimentally revealed resistance mechanisms of targeted therapy, including genetic mechanisms, epigenetic mechanisms, posttranslational mechanisms, cellular mechanisms, microenvironmental mechanisms and pharmacokinetic mechanisms. Subsequently, we list several currently available databases and Web-based tools related to drug sensitivity and resistance. Then, we focus primarily on introducing some state-of-the-art computational methods used in drug resistance studies, including mechanism-based mathematical modeling approaches e.g. molecular dynamics simulation, kinetic model of molecular networks, ordinary differential equation model of cellular dynamics, stochastic model, partial differential equation model, agent-basedmodel, pharmacokinetic-pharmacodynamic model, etc. and data-driven prediction methods e.g. omics data-based conventional screening approach for node biomarkers, static network approach for edge biomarkers and-module biomarkers, dynamic network approach for dynamic network biomarkers and dynamic module network biomarkers, etc.. Finally, we discuss several further questions and future directions for the use of computational methods for studying drug resistance, including inferring drug-induced signaling networks, multiscale modeling, drug combinations and precision medicine.</div>','<b>Title: </b>Advancing Cell Biology Through Proteomics in Space and Time PROSPECTS<br><b>Year: </b>2012<br><b>Author: </b> Ai. Lamond,  M. Uhlen,  S. Horning,  A. Makarov,  Cv. Robinson,  L. Serrano,  Fu. Hartl,  W. Baumeister,  Ak. Werenskiold,  Js. Andersen,  O. Vorm,  M. Linial,  R. Aebersold and  M. Mann<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1074/mcp.O112.017731"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MOLECULAR  CELLULAR PROTEOMICS<br><b>Cluster-Keywords: </b><em>systems biology, regulatory network, interaction networks, network model, gene regulatory, network, network biology, networks, gene network, response networks, network dynamics, dynamic model, cell networks, gene expression, biological networks, system dynamics, dynamic network, model, biological systems, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>The term "proteomics" encompasses the large-scale detection and analysis of proteins and their post-translational modifications. Driven by major improvements in mass spectrometric instrumentation, methodology, and data analysis, the proteomics field has burgeoned in recent years. It now provides a range of sensitive and quantitative approaches for measuring protein structures and dynamics that promise to revolutionize our understanding of cell biology and molecular mechanisms in both human cells and model organisms. The Proteomics Specification in Time and Space PROSPECTS Network is a unique EU-funded project that brings together leading European research groups, spanning from instrumentation to biomedicine, in a collaborative five year initiative to develop new methods and applications for the functional analysis of cellular proteins. This special issue of Molecular and Cellular Proteomics presents 16 research papers reporting major recent progress by the PROSPECTS groups, including improvements to the resolution and sensitivity of the Orbitrap family of mass spectrometers, systematic detection of proteins using highly characterized antibody collections, and new methods for absolute as well as relative quantification of protein levels. Manuscripts in this issue exemplify approaches for performing quantitative measurements of cell proteomes and for studying their dynamic responses to perturbation, both during normal cellular responses and in disease mechanisms. Here we present a perspective on how the proteomics field is moving beyond simply identifying proteins with high sensitivity toward providing a powerful and versatile set of assay systems for characterizing proteome dynamics and thereby creating a new "third generation"proteomics strategy that offers an indispensible tool for cell biology and molecular medicine. Molecular  Cellular Proteomics 11: 10.1074/mcp.O112.017731, 1-12, 2012.</div>','<b>Title: </b>Miro-dependent mitochondrial pool of CENP-F and its farnesylated C-terminal domain are dispensable for normal development in mice<br><b>Year: </b>2019<br><b>Author: </b> M. Peterka and  B. Kornmann<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pgen.1008050"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS GENETICS<br><b>Cluster-Keywords: </b><em>dna damage, signalling networks, network dynamics, pmdm system, systems biology, systems dynamics, biological systems, single cell, kinase inhibitors, signaling network, biological networks, cellular automata, cell cycle, oscillation system, temporal dynamics, signaling pathway, individual cells, signaling processes, dynamic model, damage response<div style="font-size:11px;"><br><b>Abstract: </b><em>CENP-F is a large, microtubule-binding protein that regulates multiple cellular processes including chromosome segregation and mitochondrial trafficking at cytokinesis. This multiplicity of functions is mediated through the binding of various partners, like Bub1 at the kinetochore and Miro at mitochondria. Due to the multifunctionality of CENP-F, the cellular phenotypes observed upon its depletion are difficult to interpret and there is a need to genetically separate its different functions by preventing binding to selected partners. Here we engineer a CENP-F point-mutant that is deficient in Miro binding and thus is unable to localize to mitochondria, but retains other localizations. We introduce this mutation in cultured human cells using CRISPR/Cas9 system and show it causes a defect in mitochondrial spreading similar to that observed upon Miro depletion. We further create a mouse model carrying this CENP-F variant, as well as truncated CENP-F mutants lacking the farnesylated C-terminus of the protein. Importantly, one of these truncations leads to 80 downregulation of CENP-F expression. We observe that, despite the phenotypes apparent in cultured cells, mutant mice develop normally. Taken together, these mice will serve as important models to study CENP-F biology at organismal level. In addition, because truncations of CENP-F in humans cause a lethal disease termed StrOmme syndrome, they might also be relevant disease models.Author summary CENP-F is a poorly characterized multifunctional regulator of metazoan cell division involved in chromosome segregation and mitochondrial dynamics. The physiological importance of CENP-F in both of these processes is not firmly established and numerous conflicting reports exist regarding its function in mitosis. Here we set out to investigate the function of CENP-F in vivo by CRISPR-mediated mutagenesis in mice where we specifically disrupt different domains of CENP-F important for mitosis and mitochondrial regulation. Surprisingly, contrasting to reported phenotypes in cultured cells, mutant animals display no obvious phenotypes. This argues against CENP-F being a major player in mammalian cell division and suggest context-specific roles of this protein. Importantly, in humans, truncations of CENP-F as small as 20 amino acids have recently emerged as a cause of human disease termed StrOmme syndrome. Truncations in these mice might yield information relevant to the understanding of this disease and biology of CENP-F.</div>','<b>Title: </b>Modeling discrete-variable stochastic dynamics: Ecological populations, gene networks, and a nanotube ion channel<br><b>Year: </b>2014<br><b>Author: </b> Rc. Hilborn<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1119/1.4870076"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>AMERICAN JOURNAL OF PHYSICS<br><b>Cluster-Keywords: </b><em>regulatory networks, stochastic simulation, gene networks, reaction networks, systems biology, stochastic model, network models, complex systems, networks stochastic, gene regulatory, biological systems, stochastic dynamics, differential equations, protein networks, cell cycle, continuous model, interaction networks, systems chemistry, time delays, biochemical reaction<div style="font-size:11px;"><br><b>Abstract: </b><em>Many complex systems in both the physical and life sciences can be modeled as interacting elements with discrete-value variables. In most of these systems, stochasticity plays an important role. This paper examines an ecological population dynamics model, a simple model of a gene regulatory network, and a model for the conductance of a nanotube ion channel. The interplay of discrete-value variables, stochasticity, and nonlinear dynamics produces fascinating phenomena that are important in many areas of physics and biology. C 2014 American Association of Physics Teachers.</div>','<b>Title: </b>An S-System Parameter Estimation Method SPEM for Biological Networks<br><b>Year: </b>2012<br><b>Author: </b> Xy. Yang,  Je. Dent and  C. Nardini<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1089/cmb.2011.0269"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF COMPUTATIONAL BIOLOGY<br><b>Cluster-Keywords: </b><em>ssystem model, biological systems, biological networks, parameter estimation, biochemical networks, dynamic biological, proposed algorithm, regulatory networks, structure identification, model structure, genetic algorithm, gene regulatory, gene expression, network inference, network motif, network structure, gene networks, network identification, network inferring, estimation method<div style="font-size:11px;"><br><b>Abstract: </b><em>Advances in experimental biology, coupled with advances in computational power, bring new challenges to the interdisciplinary field of computational biology. One such broad challenge lies in the reverse engineering of gene networks, and goes from determining the structure of static networks, to reconstructing the dynamics of interactions from time series data. Here, we focus our attention on the latter area, and in particular, on parameterizing a dynamic network of oriented interactions between genes. By basing the parameterizing approach on a known power-law relationship model between connected genes S-system, we are able to account for non-linearity in the network, without compromising the ability to analyze network characteristics. In this article, we introduce the S-System Parameter Estimation Method SPEM. SPEM, a freely available R software package http://www.picb.ac.cn/ClinicalGenomicNTW/temp3.html, takes gene expression data in time series and returns the network of interactions as a set of differential equations. The methods, which are presented and tested here, are shown to provide accurate results not only on synthetic data, but more importantly on real and therefore noisy by nature, biological data. In summary, SPEM shows high sensitivity and positive predicted values, as well as free availability and expansibility because based on open source software. We expect these characteristics to make it a useful and broadly applicable software in the challenging reconstruction of dynamic gene networks.</div>','<b>Title: </b>Systems Biology Approach to Model the Life Cycle of Trypanosoma cruzi<br><b>Year: </b>2016<br><b>Author: </b> A. Carrea and  L. Diambra<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0146947"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>gene regulatory, regulatory network, systems biology, cell differentiation, cell fate, model systems, potential landscape, network grn, dynamical systems, melanocyte differentiation, network models, gene expression, network dynamics, differentiation genes, dynamical models, dynamical networks, cellular differentiation, stem cells, attractor network, developmental gene<div style="font-size:11px;"><br><b>Abstract: </b><em>Due to recent advances in reprogramming cell phenotypes, many efforts have been dedicated to developing reverse engineering procedures for the identification of gene regulatory networks that emulate dynamical properties associated with the cell fates of a given biological system. In this work, we propose a systems biology approach for the reconstruction of the gene regulatory network underlying the dynamics of the Trypanosoma cruzis life cycle. By means of an optimisation procedure, we embedded the steady state maintenance, and the known phenotypic transitions between these steady states in response to environmental cues, into the dynamics of a gene network model. In the resulting network architecture we identified a small subnetwork, formed by seven interconnected nodes, that controls the parasites life cycle. The present approach could be useful for better understanding other single cell organisms with multiple developmental stages.</div>','<b>Title: </b>Information macrodynamic modelling of a random process<br><b>Year: </b>2009<br><b>Author: </b> Vs. Lerner<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1080/00207720902953169"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>INTERNATIONAL JOURNAL OF SYSTEMS SCIENCE<br><b>Cluster-Keywords: </b><em>information systems, gene expression, gene network, systems theory, system models, systems biology, systems information, network model, genetic network, mathematical models, dynamic systems, dynamic network, systemic model, network topology, information mechanisms, complex systems, systems complex, relational models, models main, information model<div style="font-size:11px;"><br><b>Abstract: </b><em>The article introduces a new approach for dynamic modelling using the variation principle, applied to a functional on trajectories of random process, and its connection to the process information path functional. The synthesised dynamic model is found by the solution of the variation problem for this path functional that allows us to build a two-level information model with a random process at the microlevel and a dynamic process at the macrolevel. The mathematical formalism has been applied for the constructive solution to the problems of the objects identification, combined with optimal controls synthesis and process consolidation. The procedure is demonstrated in this articles examples and leads to the creation of a dynamic information hierarchical network. The developed computer-based methodology and software have been successfully implemented to such areas as technology, communications, computer science, intelligent processes, biology, economy and other non-physical and physical subjects for systems modelling, identification and control of a diversity of information interactions.</div>','<b>Title: </b>Design of Adaptive Neural Tracking Controller for Pod Propulsion Unmanned Vessel Subject to Unknown Dynamics<br><b>Year: </b>2017<br><b>Author: </b> Dd. Mu,  Gf. Wang and  Ys. Fan<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.5370/JEET.2017.12.6.2365"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF ELECTRICAL ENGINEERING  TECHNOLOGY<br><b>Cluster-Keywords: </b><em>interaction network, actin filament, network model, filament network, network structures, actin network, protein interaction, actin dynamics, network, membrane protein, network dynamics, model dynamic, cell, model, network approach, protein complex, computational model, actin, adhesion dynamics, cell dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>This paper addresses two interrelated problems concerning the tracking control of pod propulsion unmanned surface vessel USV, namely, the modeling of pod propulsion USV, and tracking controller design. First, based on MMG modeling theory, the model of pod propulsion USV is derived. Furthermore, a practical adaptive neural tracking controller is proposed by backstepping technique, neural network approximation and adaptive method. Meanwhile, unlike some existing tracking methods for surface vessel whose control algorithms suffer from "explosion of complexity", a novel neural shunting model is introduced to solve the problem. Using a Lyapunov functional, it is proven that all error signals in the system are uniformly ultimately bounded. The advantages of the paper are that first, the underactuated characteristic of pod propulsion USV is proved; second, the neural shunting model is used to solve the problem of "explosion of complexity", and this is a combination of knowledge in the field of biology and engineering; third, the developed controller is able to capture the uncertainties without the exact information of hydrodynamic damping structure and the sea disturbances. Numerical examples have been given to illustrate the performance and effectiveness of the proposed scheme.</div>','<b>Title: </b>Systems Biology Analysis of Brucella Infected Peyers Patch Reveals Rapid Invasion with Modest Transient Perturbations of the Host Transcriptome<br><b>Year: </b>2013<br><b>Author: </b> Ca. Rossetti,  Kl. Drake,  P. Siddavatam,  Sd. Lawhon,  Jes. Nunes,  T. Gull,  S. Khare,  Re. Everts,  Ha. Lewin and  Lg. Adams<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0081719"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>gene expression, systems biology, metabolic network, network model, network analysis, dynamic model, signaling pathways, signaling network, regulatory network, dynamic network, biology approach, candidate genes, gene network, gene regulatory, biology analysis, mathematical model, network dynamic, biology network, metabolic pathways, pathway analysis<div style="font-size:11px;"><br><b>Abstract: </b><em>Brucella melitensis causes the most severe and acute symptoms of all Brucella species in human beings and infects hosts primarily through the oral route. The epithelium covering domed villi of jejunal-ileal Peyers patches is an important site of entry for several pathogens, including Brucella. Here, we use the calf ligated ileal loop model to study temporal in vivo Brucella-infected host molecular and morphological responses. Our results document Brucella bacteremia occurring within 30 min after intraluminal inoculation of the ileum without histopathologic traces of lesions. Based on a system biology Dynamic Bayesian Network modeling approach DBN of microarray data, a very early transient perturbation of the host enteric transcriptome was associated with the initial host response to Brucella contact that is rapidly averted allowing invasion and dissemination. A detailed analysis revealed active expression of Syndecan 2, Integrin alpha L and Integrin beta 2 genes, which may favor initial Brucella adhesion. Also, two intestinal barrier-related pathways Tight Junction and Trefoil Factors Initiated Mucosal Healing were significantly repressed in the early stage of infection, suggesting subversion of mucosal epithelial barrier function to facilitate Brucella transepithelial migration. Simultaneously, the strong activation of the innate immune response pathways would suggest that the host mounts an appropriate protective immune response; however, the expression of the two key genes that encode innate immunity anti-Brucella cytokines such as TNF-alpha and IL12p40 were not significantly changed throughout the study. Furthermore, the defective expression of Toll-Like Receptor Signaling pathways may partially explain the lack of proinflammatory cytokine production and consequently the absence of morphologically detectable inflammation at the site of infection. Cumulatively, our results indicate that the in vivo pathogenesis of the early infectious process of Brucella is primarily accomplished by compromising the mucosal immune barrier and subverting critical immune response mechanisms.</div>','<b>Title: </b>Large-scale patterns of distribution and persistence at the range margins of a butterfly<br><b>Year: </b>2002<br><b>Author: </b> Rj. Wilson,  S. Ellis,  Js. Baker,  Me. Lineham,  Rw. Whitehead and  Cd. Thomas<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.2307/3072085"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ECOLOGY<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>One major goal of conservation biology in fragmented landscapes is to identify the attributes of habitat networks that influence metapopulation persistence; and thereby to determine whether particular areas hold the key to regional persistence. In Britain, the genus Aricia is represented by A. agestis in the south and A. artaxerxes in the north. We mapped the distributions of A. agestis and A. artaxerxes at their range margins, where both species are restricted to isolated fragments of calcareous grassland containing the larval host plant Helianthemum chamaecistus. We present evidence that Aricia is likely to persist as metapopulations at its range margins. First, both individual habitat patches and semi-independent networks of patches showed colonizations and extinctions of Aricia. Second, the smallest and most isolated habitat patches were least likely to be occupied. Third, persistence was more closely related to the total area and configuration of habitat in each network consistent with a metapopulation interpretation than to the area of the largest patch predicted for mainland-island systems. In addition, the occupancy of entire networks depended on their isolation from other networks, well beyond the empirically measured dispersal range of the species. We suggest that species may persist as "metapopulations of, metapopulations" at coarse spatial scales. In some systems, all networks may be prone to extinction, but we suggest that regional persistence more often depends on the continued occupancy of extinction-resistant core areas "core/marginal metapopulations". Extinctions of small metapopulations and very long-distance colonizations may profoundly influence the distributions of species in fragmented landscapes, and set the dynamic margins of species ranges. As a result, identification and protection of extinction-resistant core areas should be priorities in conservation programs.</div>','<b>Title: </b>Mathematical modeling identifies Smad nucleocytoplasmic shuttling as a dynamic signal-interpreting system<br><b>Year: </b>2008<br><b>Author: </b> B. Schmierer,  Al. Tournier,  Pa. Bates and  Cs. Hill<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1073/pnas.0710134105"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>TGF-beta-induced Smad signal transduction from the membrane into the nucleus is not linear and unidirectional, but rather a dynamic network that couples Smad phosphorylation and dephosphorylation through continuous nucleocytoplasmic shuttling of Smads. To understand the quantitative behavior of this network, we have developed a tightly constrained computational model, exploiting the interplay between mathematical modeling and experimental strategies. The model simultaneously reproduces four distinct datasets with excellent accuracy and provides mechanistic insights into how the network operates. We use the model to make predictions about the outcome of fluorescence recovery after photobleaching experiments and the behavior of a functionally impaired Smad2 mutant, which we then verify experimentally. Successful model performance strongly supports the hypothesis of a dynamic maintenance of Smad nuclear accumulation during active signaling. The presented work establishes Smad nucleocytoplasmic shuttling as a dynamic network that flexibly transmits quantitative features of the extracellular TGF-beta signal, such as its duration and intensity, into the nucleus.</div>','<b>Title: </b>Solution of chemical master equations for nonlinear stochastic reaction networks<br><b>Year: </b>2014<br><b>Author: </b> P. Smadbeck and  Yn. Kaznessis<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.coche.2014.07.001"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>CURRENT OPINION IN CHEMICAL ENGINEERING<br><b>Cluster-Keywords: </b><em>regulatory networks, stochastic simulation, gene networks, reaction networks, systems biology, stochastic model, network models, complex systems, networks stochastic, gene regulatory, biological systems, stochastic dynamics, differential equations, protein networks, cell cycle, continuous model, interaction networks, systems chemistry, time delays, biochemical reaction<div style="font-size:11px;"><br><b>Abstract: </b><em>Stochasticity in the dynamics of small reacting systems requires discrete-probabilistic models of reaction kinetics instead of traditional continuous-deterministic ones. The master probability equation is a complete model of randomly evolving molecular populations. Because of its ambitious character, the master equation remained unsolved for all but the simplest of molecular interaction networks. With the first solution of chemical master equations CMEs, a wide range of experimental observations of small-system interactions may be mathematically conceptualized.</div>','<b>Title: </b>Functional data analysis for identifying nonlinear models of gene regulatory networks<br><b>Year: </b>2010<br><b>Author: </b> G. Summer and  Tj. Perkins<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1471-2164-11-S4-S18"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC GENOMICS<br><b>Cluster-Keywords: </b><em>regulatory networks, gene regulatory, gene expression, network inference, expression data, network structure, gene networks, networks grns, regulatory genes, boolean networks, bayesian network, network reconstruction, networks gene, network dynamics, gene regulation, inference method, systems biology, time series, infer gene, network<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: A key problem in systems biology is estimating dynamical models of gene regulatory networks. Traditionally, this has been done using regression or other ad-hoc methods when the model is linear. More detailed, realistic modeling studies usually employ nonlinear dynamical models, which lead to computationally difficult parameter estimation problems. Functional data analysis methods, however, offer a means to simplify fitting by transforming the problem from one of matching modeled and observed dynamics to one of matching modeled and observed time derivatives-a regression problem, albeit a nonlinear one.Results: We formulate a functional data analysis approach for estimating the parameters of nonlinear dynamical models and evaluate this approach on data from two real systems, the gap gene system of Drosophila melanogaster and the synthetic IRMA network, which was created expressly as a test case for genetic network inference. We also evaluate the approach on simulated data sets generated by the GeneNetWeaver program, the basis for the annual DREAM reverse engineering challenge. We assess the accuracy with which the correct regulatory relationships within the networks are extracted, and consider alternative methods of regularization for the purpose of overfitting avoidance. We also show that the computational efficiency of the functional data analysis approach, and the decomposability of the resulting regression problem, allow us to explicitly enumerate and evaluate all possible regulator combinations for every gene. This gives deeper insight into the the relevance of different regulators or regulator combinations, and lets one check for alternative regulatory explanations.Conclusions: Functional data analysis is a powerful approach for estimating detailed nonlinear models of gene expression dynamics, allowing efficient and accurate estimation of regulatory architecture.</div>','<b>Title: </b>Boolean Models of Genomic Regulatory Networks: Reduction Mappings, Inference, and External Control<br><b>Year: </b>2009<br><b>Author: </b> I. Ivanov<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.2174/138920209789177584"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>CURRENT GENOMICS<br><b>Cluster-Keywords: </b><em>boolean networks, regulatory networks, genetic regulatory, gene regulatory, probabilistic boolean, network, networks, model, network model, probabilistic model, model checking, networks pbns, network dynamics, markov chain, regulatory, models, optimal control, boolean, networks models, networks boolean<div style="font-size:11px;"><br><b>Abstract: </b><em>Computational modeling of genomic regulation has become an important focus of systems biology and genomic signal processing for the past several years. It holds the promise to uncover both the structure and dynamical properties of the complex gene, protein or metabolic networks responsible for the cell functioning in various contexts and regimes. This, in turn, will lead to the development of optimal intervention strategies for prevention and control of disease. At the same time, constructing such computational models faces several challenges. High complexity is one of the major impediments for the practical applications of the models. Thus, reducing the size/complexity of a model becomes a critical issue in problems such as model selection, construction of tractable subnetwork models, and control of its dynamical behavior. We focus on the reduction problem in the context of two specific models of genomic regulation: Boolean networks with perturbation BNP and probabilistic Boolean networks PBN. We also compare and draw a parallel between the reduction problem and two other important problems of computational modeling of genomic networks: the problem of network inference and the problem of designing external control policies for intervention/altering the dynamics of the model.</div>','<b>Title: </b>Overview of Biological Network Inference and Modeling of Dynamics<br><b>Year: </b>2016<br><b>Author: </b> <br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/B978-0-08-100095-3.00001-9"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>COMPUTATIONAL SYSTEMS BIOLOGY: INFERENCE AND MODELING<br><b>Cluster-Keywords: </b><em>interaction networks, proteinprotein interactions, protein interaction, network model, complex systems, systems biology, protein protein, protein complex, host network, dynamic interactions, network analysis, biological systems, networks network, cellular network, biological network, viral proteins, interactome networks, regulatory networks, immune system, systems network<div style="font-size:11px;"><br><b>Abstract: </b><em>In the era of high-throughput experiments, inferring and modelling the dynamics of biological systems are complex tasks. The complexity derives from the large sizes, the presence of competing interactions, stiffness, and non-linearity in the systems under investigation.Moreover, the dynamics in these systems are typically hybrid - that is, stochastic and deterministic and time irreversible - raising many technical and conceptual challenges: is it possible, at least in principle, to infer the topology and the properties of a biological network from observations of the dynamics How much and what information do we need to obtain from an experiment to accurately infer a network model Would it be possible to accurately describe the dynamics of the stochastic interactions without a full stochastic simulation, which is extremely computationally expensive for systems of huge size How is it possible to infer from partial and noisy observations the trajectory of the system for complete observationsAll these questions are born of the realistic possibility that the inference and modeling of omic-size, complex interaction networks is an underdetermined problem. In this introductory chapter, we present the challenges underdetermination modeling has to face in systems biology and its usefulness in the era of high-throughput experiments and big data collection.</div>','<b>Title: </b>A SYSTEM MODEL FOR CELL DEATH/SURVIVAL USING SPICE AND LADDER LOGIC<br><b>Year: </b>2010<br><b>Author: </b> S. Jain,  Pk. Naik and  Sv. BhooshanLink: No link<br><b>Journal: </b>DIGEST JOURNAL OF NANOMATERIALS AND BIOSTRUCTURES<br><b>Cluster-Keywords: </b><em>stem cell, stem cells, systems biology, mathematical models, cell biology, network model, signaling pathways, cell fate, computational models, cancer cells, gene expression, network dynamics, embryonic stem, signaling network, prostate cancer, mathematical modeling, psychological stress, apoptosis signaling, computational systems, molecular mechanisms<div style="font-size:11px;"><br><b>Abstract: </b><em>Systems Biology is the science of discovering, modeling, understanding and ultimately engineering at the molecular level the dynamic relationships between the biological molecules that define living organisms. Computational modeling is useful as a means to assemble and test what we know about proteins and networks. Models can help address key questions about the measurement, definition and function. Computationally intensive simulations can incorporate greater numbers of neurons to model increasingly complex and realistic properties, both electrical and chemical. Recent studies suggest that the balance between cell survival signals and pro-apoptotic stimuli controls the decision between cell repair and death. Inspired by the computational feasibility of Simulation Program For Integrated Circuit Emphasis SPICE, and Ladder logic we set out to build a systematic signaling network that would enable the predictive signal of cell death/survival given Tumor necrosis factor-alpha TNF, Epidermal growth factor EGF and Insulin as inputs. We conclude that it is possible to build self consistent prediction model that can be computationally to yield important insights into the control of cell death/survival responses.</div>','<b>Title: </b>Potential landscape of high dimensional nonlinear stochastic dynamics with large noise<br><b>Year: </b>2017<br><b>Author: </b> Y. Tang,  Rs. Yuan,  Gw. Wang,  Xm. Zhu and  P. Ao<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1038/s41598-017-15889-2"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SCIENTIFIC REPORTS<br><b>Cluster-Keywords: </b><em>gene regulatory, regulatory network, systems biology, cell differentiation, cell fate, model systems, potential landscape, network grn, dynamical systems, melanocyte differentiation, network models, gene expression, network dynamics, differentiation genes, dynamical models, dynamical networks, cellular differentiation, stem cells, attractor network, developmental gene<div style="font-size:11px;"><br><b>Abstract: </b><em>Quantifying stochastic processes is essential to understand many natural phenomena, particularly in biology, including the cell-fate decision in developmental processes as well as the genesis and progression of cancers. While various attempts have been made to construct potential landscape in high dimensional systems and to estimate transition rates, they are practically limited to the cases where either noise is small or detailed balance condition holds. A general and practical approach to investigate real-world nonequilibrium systems, which are typically high-dimensional and subject to large multiplicative noise and the breakdown of detailed balance, remains elusive. Here, we formulate a computational framework that can directly compute the relative probabilities between locally stable states of such systems based on a least action method, without the necessity of simulating the steady-state distribution. The method can be applied to systems with arbitrary noise intensities through A-type stochastic integration, which preserves the dynamical structure of the deterministic counterpart dynamics. We demonstrate our approach in a numerically accurate manner through solvable examples. We further apply the method to investigate the role of noise on tumor heterogeneity in a 38-dimensional network model for prostate cancer, and provide a new strategy on controlling cell populations by manipulating noise strength.</div>','<b>Title: </b>Approximating frustration scores in complex networks via perturbed Laplacian spectra<br><b>Year: </b>2015<br><b>Author: </b> Aj. Savol and  Cs. Chennubhotla<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1103/PhysRevE.92.062806"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PHYSICAL REVIEW E<br><b>Cluster-Keywords: </b><em>complex networks, complex systems, network theory, network structure, biological network, social network, network science, dynamic networks, network systems, network model, network topology, systems biology, network, network analysis, system dynamics, networks, realworld networks, network automata, epidemic network, model network<div style="font-size:11px;"><br><b>Abstract: </b><em>Systems of many interacting components, as found in physics, biology, infrastructure, and the social sciences, are often modeled by simple networks of nodes and edges. The real-world systems frequently confront outside intervention or internal damage whose impact must be predicted or minimized, and such perturbations are then mimicked in the models by altering nodes or edges. This leads to the broad issue of how to best quantify changes in a model network after some type of perturbation. In the case of node removal there are many centrality metrics which associate a scalar quantity with the removed node, but it can be difficult to associate the quantitieswith some intuitive aspect of physical behavior in the network. This presents a serious hurdle to the application of network theory: real-world utility networks are rarely altered according to theoretic principles unless the kinetic impact on the networks users are fully appreciated beforehand. In pursuit of a kinetically interpretable centrality score, we discuss the f-score, or frustration score. Each f-score quantifies whether a selected node accelerates or inhibits global mean first passage times to a second, independently selected target node. We show that this is a natural way of revealing the dynamical importance of a node in some networks. After discussing merits of the f-score metric, we combine spectral and Laplacian matrix theory in order to quickly approximate the exact f-score values, which can otherwise be expensive to compute. Following tests on both synthetic and real medium-sized networks, we report f-score runtime improvements over exact brute force approaches in the range of 0 to 400 with low error < 3.</div>','<b>Title: </b>Reshaping the epigenetic landscape during early flower development: induction of attractor transitions by relative differences in gene decay rates<br><b>Year: </b>2015<br><b>Author: </b> J. Davila-Velderrain,  C. Villarreal and  Er. Alvarez-Buylla<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/s12918-015-0166-y"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>gene regulatory, regulatory network, systems biology, cell differentiation, cell fate, model systems, potential landscape, network grn, dynamical systems, melanocyte differentiation, network models, gene expression, network dynamics, differentiation genes, dynamical models, dynamical networks, cellular differentiation, stem cells, attractor network, developmental gene<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Gene regulatory network GRN dynamical models are standard systems biology tools for the mechanistic understanding of developmental processes and are enabling the formalization of the epigenetic landscape EL model.Methods: In this work we propose a modeling framework which integrates standard mathematical analyses to extend the simple GRN Boolean model in order to address questions regarding the impact of gene specific perturbations in cell-fate decisions during development.Results: We systematically tested the propensity of individual genes to produce qualitative changes to the EL induced by modification of gene characteristic decay rates reflecting the temporal dynamics of differentiation stimuli. By applying this approach to the flower specification GRN FOS-GRN we uncovered differences in the functional dynamical role of their genes. The observed dynamical behavior correlates with biological observables. We found a relationship between the propensity of undergoing attractor transitions between attraction basins in the EL and the direction of differentiation during early flower development - being less likely to induce up-stream attractor transitions as the course of development progresses. Our model also uncovered a potential mechanism at play during the transition from EL basins defining inflorescence meristem to those associated to flower organs meristem. Additionally, our analysis provided a mechanistic interpretation of the homeotic property of the ABC genes, being more likely to produce both an induced inter-attractor transition and to specify a novel attractor. Finally, we found that there is a close relationship between a genes topological features and its propensity to produce attractor transitions.Conclusions: The study of how the state-space associated with a dynamical model of a GRN can be restructured by modulation of genes characteristic expression times is an important aid for understanding underlying mechanisms occurring during development. Our contribution offers a simple framework to approach such problem, as exemplified here by the case of flower development. Different GRN models and the effect of diverse inductive signals can be explored within the same framework. We speculate that the dynamical role of specific genes within a GRN, as uncovered here, might give information about which genes are more likely to link a module to other regulatory circuits and signaling transduction pathways.</div>','<b>Title: </b>Network Class Superposition Analyses<br><b>Year: </b>2013<br><b>Author: </b> Cab. Pearson,  C. Zeng and  R. Simha<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0059046"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>network models, gene network, regulatory networks, boolean networks, gene expression, systems biology, neural networks, network dynamics, gene regulatory, biological networks, network structure, gap gene, gene regulation, cell cycle, network, gene interactions, signaling network, quantum systems, system dynamics, networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Networks are often used to understand a whole system by modeling the interactions among its pieces. Examples include biomolecules in a cell interacting to provide some primary function, or species in an environment forming a stable community. However, these interactions are often unknown; instead, the pieces dynamic states are known, and network structure must be inferred. Because observed function may be explained by many different networks e. g., approximate to 1030 for the yeast cell cycle process 1, considering dynamics beyond this primary function means picking a single network or suitable sample: measuring over all networks exhibiting the primary function is computationally infeasible. We circumvent that obstacle by calculating the network class ensemble. We represent the ensemble by a stochastic matrix T, which is a transition-by-transition superposition of the system dynamics for each member of the class. We present concrete results for T derived from Boolean time series dynamics on networks obeying the Strong Inhibition rule, by applying T to several traditional questions about network dynamics. We show that the distribution of the number of point attractors can be accurately estimated with T. We show how to generate Derrida plots based on T. We show that T-based Shannon entropy outperforms other methods at selecting experiments to further narrow the network structure. We also outline an experimental test of predictions based on T. We motivate all of these results in terms of a popular molecular biology Boolean network model for the yeast cell cycle, but the methods and analyses we introduce are general. We conclude with open questions for T, for example, application to other models, computational considerations when scaling up to larger systems, and other potential analyses.</div>','<b>Title: </b>An improved kinetic model for the acetone-butanol-ethanol pathway of Clostridium acetobutylicum and model-based perturbation analysis<br><b>Year: </b>2011<br><b>Author: </b> Rd. Li,  Yy. Li,  Ly. Lu,  C. Ren,  Yx. Li and  L. Liu<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-5-S1-S12"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Comprehensive kinetic models of microbial metabolism can enhance the understanding of system dynamics and regulatory mechanisms, which is helpful in optimizing microbial production of industrial chemicals. Clostridium acetobutylicum produces solvents acetone-butanol-ethanol, ABE through the ABE pathway. To systematically assess the potential of increased production of solvents, kinetic modeling has been applied to analyze the dynamics of this pathway and make predictive simulations. Up to date, only one kinetic model for C. acetobutylicum supported by experiment has been reported as far as we know. But this model did not integrate the metabolic regulatory effects of transcriptional control and other complex factors. It also left out the information of some key intermediates e.g. butyryl-phosphate.Results: We have developed an improved kinetic model featured with the incorporation of butyryl-phosphate, inclusion of net effects of complex metabolic regulations, and quantification of endogenous enzyme activity variations caused by these regulations. The simulation results of our model are more consistent with published experimental data than the previous model, especially in terms of reflecting the kinetics of butyryl-phosphate and butyrate. Through parameter perturbation analysis, it was found that butyrate kinase has large and positive influence on butanol production while CoA transferase has negative effect on butanol production, suggesting that butyrate kinase has more efficiency in converting butyrate to butanol than CoA transferase.Conclusions: Our improved kinetic model of the ABE process has more capacity in approaching real circumstances, providing much more insight in the regulatory mechanisms and potential key points for optimization of solvent productions. Moreover, the modeling strategy can be extended to other biological processes.</div>','<b>Title: </b>The function of mechanical tension in neuronal and network development<br><b>Year: </b>2010<br><b>Author: </b> A. Ayali<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1039/b927402b"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>INTEGRATIVE BIOLOGY<br><b>Cluster-Keywords: </b><em>gene regulatory, systems biology, regulatory networks, gene network, stem cell, network model, transcription factors, gene expression, cell transcriptomes, network analysis, data analysis, stem cells, network grnsight, cell transformation, weighted network, gene ontology, networks grns, graph model, graph layout, endothelial cell<div style="font-size:11px;"><br><b>Abstract: </b><em>A bewildering series of dynamic processes participate in the proper development of the complex architecture of the nervous system. Recent years have seen a growing interest in the role of mechanical forces in neural development. This is an exciting field of multidisciplinary study that encompasses biology, physics and engineering and enjoys both conceptual and technical recent advances in all of these areas. Here I present an update of very recent work on several related questions, including the role of neurite and axonal tension in the development of single neuron morphology, the effects of mechanical cues from the substrate, the role of tension in axonal pruning and synaptogenesis, and more. Particular emphasis is placed on the very recent and exciting shift from descriptive mechanics to a possible role for tension forces in neuronal and network function.</div>','<b>Title: </b>A Logic for Checking the Probabilistic Steady- State Properties of Reaction Networks<br><b>Year: </b>2017<br><b>Author: </b> V. Picard,  A. Siegel and  J. Bourdon<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1089/cmb.2017.0099"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF COMPUTATIONAL BIOLOGY<br><b>Cluster-Keywords: </b><em>regulatory networks, stochastic simulation, gene networks, reaction networks, systems biology, stochastic model, network models, complex systems, networks stochastic, gene regulatory, biological systems, stochastic dynamics, differential equations, protein networks, cell cycle, continuous model, interaction networks, systems chemistry, time delays, biochemical reaction<div style="font-size:11px;"><br><b>Abstract: </b><em>Designing probabilistic reaction models and determining their stochastic kinetic parameters are major issues in systems biology. To assist in the construction of reaction network models, we introduce a logic that allows one to express asymptotic properties about the steady-state stochastic dynamics of a reaction network. Basically, the formulas can express properties on expectancies, variances, and covariances. If a formula encoding for experimental observations on the system is not satisfiable, then the reaction network model can be rejected. We demonstrate that deciding the satisfiability of a formula is NP-hard, but we provide a decision method based on solving systems of polynomial constraints. We illustrate our method on a toy example.</div>','<b>Title: </b>When one model is not enough: Combining epistemic tools in systems biology<br><b>Year: </b>2013<br><b>Author: </b> S. Green<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.shpsc.2013.03.012"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>STUDIES IN HISTORY AND PHILOSOPHY OF SCIENCE PART C-STUDIES IN HISTORY AND PHILOSOPHY OF BIOLOGICAL AND BIOMEDIAL SCIENCES<br><b>Cluster-Keywords: </b><em>network motifs, regulatory network, gene regulatory, systems biology, gene network, bifan motif, complex systems, model comparison, network modeling, complex gene, biological systems, dynamic systems, interaction network, feedforward loop, modeling gene, intelligent systems, modeling approach, biological pathways, motif model, model motifs<div style="font-size:11px;"><br><b>Abstract: </b><em>In recent years, the philosophical focus of the modeling literature has shifted from descriptions of general properties of models to an interest in different model functions. It has been argued that the diversity of models and their correspondingly different epistemic goals are important for developing intelligible scientific theories Leonelli, 2007; Levins, 2006. However, more knowledge is needed on how a combination of different epistemic means can generate and stabilize new entities in science. This paper will draw on Rheinbergers practice-oriented account of knowledge production. The conceptual repertoire of Rheinbergers historical epistemology offers important insights for an analysis of the modelling practice. I illustrate this with a case study on network modeling in systems biology where engineering approaches are applied to the study of biological systems. I shall argue that the use of multiple representational means is an essential part of the dynamic of knowledge generation. It is because of-rather than in spite of-the diversity of constraints of different models that the interlocking use of different epistemic means creates a potential for knowledge production. C 2013 Elsevier Ltd. All rights reserved.</div>','<b>Title: </b>Selection on Network Dynamics Drives Differential Rates of Protein Domain Evolution<br><b>Year: </b>2016<br><b>Author: </b> Bk. Mannakee and  Rn. Gutenkunst<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pgen.1006132"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS GENETICS<br><b>Cluster-Keywords: </b><em>regulatory network, systems biology, systems biological, network dynamics, gene expression, biochemical networks, raft proteins, biological networks, gene circuits, gene regulatory, signalling network, lipid raft, nuclear lamina, protein domain, networks systems, biological approaches, expression pattern, protein interaction, cell differentiation, network motifs<div style="font-size:11px;"><br><b>Abstract: </b><em>The long-held principle that functionally important proteins evolve slowly has recently been challenged by studies in mice and yeast showing that the severity of a protein knockout only weakly predicts that proteins rate of evolution. However, the relevance of these studies to evolutionary changes within proteins is unknown, because amino acid substitutions, unlike knockouts, often only slightly perturb protein activity. To quantify the phenotypic effect of small biochemical perturbations, we developed an approach to use computational systems biology models to measure the influence of individual reaction rate constants on network dynamics. We show that this dynamical influence is predictive of protein domain evolutionary rate within networks in vertebrates and yeast, even after controlling for expression level and breadth, network topology, and knockout effect. Thus, our results not only demonstrate the importance of protein domain function in determining evolutionary rate, but also the power of systems biology modeling to uncover unanticipated evolutionary forces.</div>','<b>Title: </b>Monitoring Cell-autonomous Circadian Clock Rhythms of Gene Expression Using Luciferase Bioluminescence Reporters<br><b>Year: </b>2012<br><b>Author: </b> C. Ramanathan,  Sk. Khan,  Nd. Kathale,  Hy. Xu and  Ac. Liu<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3791/4234"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOVE-JOURNAL OF VISUALIZED EXPERIMENTS<br><b>Cluster-Keywords: </b><em>circadian rhythms, systems biology, biological clock, circadian clock, biological rhythms, hpa axis, model, clock network, mathematical models, biological systems, dynamic systems, ode models, biology models, single cell, network, model parameters, experimental data, models, systems, dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>In mammals, many aspects of behavior and physiology such as sleep-wake cycles and liver metabolism are regulated by endogenous circadian clocks reviewed1,2. The circadian time-keeping system is a hierarchical multi-oscillator network, with the central clock located in the suprachiasmatic nucleus SCN synchronizing and coordinating extra-SCN and peripheral clocks elsewhere1,2. Individual cells are the functional units for generation and maintenance of circadian rhythms3,4, and these oscillators of different tissue types in the organism share a remarkably similar biochemical negative feedback mechanism. However, due to interactions at the neuronal network level in the SCN and through rhythmic, systemic cues at the organismal level, circadian rhythms at the organismal level are not necessarily cell-autonomous5-7. Compared to traditional studies of locomotor activity in vivo and SCN explants ex vivo, cell-based in vitro assays allow for discovery of cell-autonomous circadian defects5,8. Strategically, cell-based models are more experimentally tractable for phenotypic characterization and rapid discovery of basic clock mechanisms5,8-13.Because circadian rhythms are dynamic, longitudinal measurements with high temporal resolution are needed to assess clock function. In recent years, real-time bioluminescence recording using firefly luciferase as a reporter has become a common technique for studying circadian rhythms in mammals14,15, as it allows for examination of the persistence and dynamics of molecular rhythms. To monitor cell-autonomous circadian rhythms of gene expression, luciferase reporters can be introduced into cells via transient transfection13,16,17 or stable transduction5,10,18,19. Here we describe a stable transduction protocol using lentivirus-mediated gene delivery. The lentiviral vector system is superior to traditional methods such as transient transfection and germline transmission because of its efficiency and versatility: it permits efficient delivery and stable integration into the host genome of both dividing and non-dividing cells20. Once a reporter cell line is established, the dynamics of clock function can be examined through bioluminescence recording. We first describe the generation of PPer2-dLuc reporter lines, and then present data from this and other circadian reporters. In these assays, 3T3 mouse fibroblasts and U2OS human osteosarcoma cells are used as cellular models. We also discuss various ways of using these clock models in circadian studies. Methods described here can be applied to a great variety of cell types to study the cellular and molecular basis of circadian clocks, and may prove useful in tackling problems in other biological systems.</div>','<b>Title: </b>The Cell Cycle Switch Computes Approximate Majority<br><b>Year: </b>2012<br><b>Author: </b> L. Cardelli and  A. Csikasz-Nagy<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1038/srep00656"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SCIENTIFIC REPORTS<br><b>Cluster-Keywords: </b><em>blood flow, blood cells, flow rate, blood pumps, cells, cell, results, patients, stem cells, methods, blood, patients blood, flow, patients methods, patients objectives, cell viability, objectives, results cells, blood pressure, cell culture<div style="font-size:11px;"><br><b>Abstract: </b><em>Both computational and biological systems have to make decisions about switching from one state to another. The Approximate Majority computational algorithm provides the asymptotically fastest way to reach a common decision by all members of a population between two possible outcomes, where the decision approximately matches the initial relative majority. The network that regulates the mitotic entry of the cell-cycle in eukaryotes also makes a decision before it induces early mitotic processes. Here we show that the switch from inactive to active forms of the mitosis promoting Cyclin Dependent Kinases is driven by a system that is related to both the structure and the dynamics of the Approximate Majority computation. We investigate the behavior of these two switches by deterministic, stochastic and probabilistic methods and show that the steady states and temporal dynamics of the two systems are similar and they are exchangeable as components of oscillatory networks.</div>','<b>Title: </b>A systems biology analysis of brain microvascular endothelial cell lipotoxicity<br><b>Year: </b>2014<br><b>Author: </b> Hh. Aung,  A. Tsoukalas,  Jc. Rutledge and  I. Tagkopoulos<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-8-80"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>gene regulatory, systems biology, regulatory networks, gene network, stem cell, network model, transcription factors, gene expression, cell transcriptomes, network analysis, data analysis, stem cells, network grnsight, cell transformation, weighted network, gene ontology, networks grns, graph model, graph layout, endothelial cell<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Neurovascular inflammation is associated with a number of neurological diseases including vascular dementia and Alzheimers disease, which are increasingly important causes of morbidity and mortality around the world. Lipotoxicity is a metabolic disorder that results from accumulation of lipids, particularly fatty acids, in non-adipose tissue leading to cellular dysfunction, lipid droplet formation, and cell death.Results: Our studies indicate for the first time that the neurovascular circulation also can manifest lipotoxicity, which could have major effects on cognitive function. The penetration of integrative systems biology approaches is limited in this area of research, which reduces our capacity to gain an objective insight into the signal transduction and regulation dynamics at a systems level. To address this question, we treated human microvascular endothelial cells with triglyceride-rich lipoprotein TGRL lipolysis products and then we used genome-wide transcriptional profiling to obtain transcript abundances over four conditions. We then identified regulatory genes and their targets that have been differentially expressed through analysis of the datasets with various statistical methods. We created a functional gene network by exploiting co-expression observations through a guilt-by-association assumption. Concomitantly, we used various network inference algorithms to identify putative regulatory interactions and we integrated all predictions to construct a consensus gene regulatory network that is TGRL lipolysis product specific.Conclusion: System biology analysis has led to the validation of putative lipid-related targets and the discovery of several genes that may be implicated in lipotoxic-related brain microvascular endothelial cell responses. Here, we report that activating transcription factors 3 ATF3 is a principal regulator of TGRL lipolysis products-induced gene expression in human brain microvascular endothelial cell.</div>','<b>Title: </b>Visualizing regulatory interactions in metabolic networks<br><b>Year: </b>2007<br><b>Author: </b> S. Noack,  A. Wahl,  E. Qeli and  W. Wiechert<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1741-7007-5-46"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC BIOLOGY<br><b>Cluster-Keywords: </b><em>kinetic models, metabolic networks, kinetic parameters, dynamic models, systems biology, network model, dynamic metabolic, biochemical network, rate constants, enzyme kinetics, kinetic data, adjustable parameters, metabolic model, parameter estimation, dynamic optimization, model selection, mathematical model, kinetic network, model response, model<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Direct visualization of data sets in the context of biochemical network drawings is one of the most appealing approaches in the field of data evaluation within systems biology. One important type of information that is very helpful in interpreting and understanding metabolic networks has been overlooked so far. Here we focus on the representation of this type of information given by the strength of regulatory interactions between metabolite pools and reaction steps.Results: The visualization of such interactions in a given metabolic network is based on a novel concept defining the regulatory strength RS of effectors regulating certain reaction steps. It is applicable to any mechanistic reaction kinetic formula. The RS values are measures for the strength of an up- or down-regulation of a reaction step compared with the completely non-inhibited or non-activated state, respectively. One numerical RS value is associated to any effector edge contained in the network. The RS is approximately interpretable on a percentage scale where 100 means the maximal possible inhibition or activation, respectively, and 0 means the absence of a regulatory interaction. If many effectors influence a certain reaction step, the respective percentages indicate the proportion in which the different effectors contribute to the total regulation of the reaction step. The benefits of the proposed method are demonstrated with a complex example system of a dynamic E. coli network.Conclusion: The presented visualization approach is suitable for an intuitive interpretation of simulation data of metabolic networks under dynamic as well as steady-state conditions. Huge amounts of simulation data can be analyzed in a quick and comprehensive way. An extended time-resolved graphical network presentation provides a series of information about regulatory interaction within the biological system under investigation.</div>','<b>Title: </b>Ultrasensitive Negative Feedback Control: A Natural Approach for the Design of Synthetic Controllers<br><b>Year: </b>2016<br><b>Author: </b> F. Montefusco,  Oe. Akman,  Os. Soyer and  Dg. Bates<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0161605"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>systems biology, biological systems, system dynamics, molecular systems, experimental design, network model, model system, model parameters, experiment design, kinetic parameters, perfect adaptation, experimental data, molecular mechanisms, homeostatic mechanisms, switching dynamics, biological mechanisms, control systems, chemotaxis network, decentralized control, feedback control<div style="font-size:11px;"><br><b>Abstract: </b><em>Many of the most important potential applications of Synthetic Biology will require the ability to design and implement high performance feedback control systems that can accurately regulate the dynamics of multiple molecular species within the cell. Here, we argue that the use of design strategies based on combining ultrasensitive response dynamics with negative feedback represents a natural approach to this problem that fully exploits the strongly nonlinear nature of cellular information processing. We propose that such feedback mechanisms can explain the adaptive responses observed in one of the most widely studied biomolecular feedback systems-the yeast osmoregulatory response network. Based on our analysis of such system, we identify strong links with a well-known branch of mathematical systems theory from the field of Control Engineering, known as Sliding Mode Control. These insights allow us to develop design guidelines that can inform the construction of feedback controllers for synthetic biological systems.</div>','<b>Title: </b>Using in silico models to simulate dual perturbation experiments: procedure development and interpretation of outcomes<br><b>Year: </b>2009<br><b>Author: </b> N. Jamshidi and  Bo. Palsson<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-3-44"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>immune system, network models, immune response, model, biological systems, models, model reduction, dynamic model, biological networks, system, mathematical models, biochemical network, response model, systems, systems biology, network, immune, biology model, reaction networks, system systems<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: A growing number of realistic in silico models of metabolic functions are being formulated and can serve as dry lab platforms to prototype and simulate experiments before they are performed. For example, dual perturbation experiments that vary both genetic and environmental parameters can readily be simulated in silico. Genetic and environmental perturbations were applied to a cell-scale model of the human erythrocyte and subsequently investigated.Results: The resulting steady state fluxes and concentrations, as well as dynamic responses to the perturbations were analyzed, yielding two important conclusions: 1 that transporters are informative about the internal states fluxes and concentrations of a cell and, 2 that genetic variations can disrupt the natural sequence of dynamic interactions between network components. The former arises from adjustments in energy and redox states, while the latter is a result of shifting time scales in aggregate pool formation of metabolites. These two concepts are illustrated for glucose-6 phosphate dehydrogenase G6PD and pyruvate kinase PK in the human red blood cell.Conclusion: Dual perturbation experiments in silico are much more informative for the characterization of functional states than single perturbations. Predictions from an experimentally validated cellular model of metabolism indicate that the measurement of cofactor precursor transport rates can inform the internal state of the cell when the external demands are altered or a causal genetic variation is introduced. Finally, genetic mutations that alter the clinical phenotype may also disrupt the natural time scale hierarchy of interactions in the network.</div>','<b>Title: </b>Language evolution, coalescent processes, and the consensus problem on a social network<br><b>Year: </b>2014<br><b>Author: </b> H. Narayanan and  P. Niyogi<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.jmp.2014.07.002"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF MATHEMATICAL PSYCHOLOGY<br><b>Cluster-Keywords: </b><em>complex networks, complex systems, network theory, network structure, biological network, social network, network science, dynamic networks, network systems, network model, network topology, systems biology, network, network analysis, system dynamics, networks, realworld networks, network automata, epidemic network, model network<div style="font-size:11px;"><br><b>Abstract: </b><em>In recent times, there has been an increased interest in theories of language evolution that have an applicability to the study of dialect formation, linguistic change, creolization, the origin of language, and animal and robot communication systems in general. One particular question that has attracted some interest has the following general form: how might a group of linguistic agents arrive at a shared communication system purely through local patterns of interaction and without any global agency enforcing uniformity In this paper, we consider a natural model of language or more precisely, word evolution on a social network, prove several theoretical properties, and establish connections to related phenomena in biology, social sciences, and physics. C 2014 Published by Elsevier Inc.</div>','<b>Title: </b>Adaptive competitive self-organizing associative memory<br><b>Year: </b>2002<br><b>Author: </b> R. Athinarayanan,  Mr. Sayeh and  Da. Wood<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1109/TSMCA.2002.804789"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IEEE TRANSACTIONS ON SYSTEMS MAN AND CYBERNETICS PART A-SYSTEMS AND HUMANS<br><b>Cluster-Keywords: </b><em>chaotic system, control systems, neural networks, jerk chaotic, torch chaotic, mode control, sliding mode, plasma torch, dynamic neural, mitochondrial dynamics, system, systems biology, adaptive integral, model system, rodtype plasma, integral sliding, systems, jerk system, cancer cell, comparison method<div style="font-size:11px;"><br><b>Abstract: </b><em>This work presents the design of an adaptive competitive self-organizing associative memory ACSAM system for use in classification and recognition of pattern information. Volterra and Lotkas models of interacting species in biology motivated the ACSAM model; a model based on, a system of nonlinear ordinary differential equations ODEs. Self-organizing behavior is modeled for unsupervised neural networks employing the concept of interacting/competing species in biology. In this model, self-organizing properties can be implicitly coded within the systems trajectory structure using only ODES. Among the features of this continuous-time system are 1 the dynamic behavior is well-understood and characterized; 2 the desired fixed points are the only asymptotically stable states of the system; 3 the trajectories of ACSAM derived from the weight activities of the gradient system have no periodic or homoclinic orbits; and 4 the heteroclinic orbits that exist between equilibrium states are structurally unstable and can be removed by small perturbations.</div>','<b>Title: </b>Metabolic Network Modelling of Chinese Hamster Ovary CHO Culture Bioreactors Operated as Microbial Cell Factories<br><b>Year: </b>2018<br><b>Author: </b> J. Gaspersic,  M. Kastelic,  U. Novak and  B. Likozar<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.17344/acsi.2018.4591"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ACTA CHIMICA SLOVENICA<br><b>Cluster-Keywords: </b><em>metabolic network, metabolic model, dynamic flux, growth rate, genomescale metabolic, flux balance, balance analysis, metabolic flux, metabolic pathways, dynamic model, metabolic reactions, cellular growth, regulatory network, systems biology, optimal growth, metabolic, modeling metabolic, growth conditions, dynamic analysis, cell growth<div style="font-size:11px;"><br><b>Abstract: </b><em>Chinese hamster ovary CHO epithelial cells are one of the most used therapeutic medical lines for the production of different biopharmaceutical drugs. They have a high consumption rate with a fast duplication cycle that makes them an ideal biological clone. The higher accumulated amounts of toxic intracellular intermediates may lead to lower organism viability, protein productivity and manufactured biosimilar, so a careful optimal balance of medium, bioreactor operational parameters and bioprocess is needed. A precise phenomenological knowledge of metabolisms chemical transformations can predict problems that may arise during batch, semi-continuous fed batch and continuous reactor operation. For a better detailed understanding and relations, future performance optimization and scaling, mechanistic model systems have been built. In this specific work, the main metabolic pathways in mammalian structured CHO cultures are reviewed. It starts with organic biochemical background, controlling associated phenomena and kinetics, which govern the sustaining conversion routes of biology. Then, individual turnover paths are described, overviewing standard mathematical formulations that are commonly applied in engineering. These are the core of black box modeling, which relates the substrates/products in a simplified relationship manner. Moreover, metabolic flux analysis MFA/flux balance analysis FBA, that are traditionally characterizing mechanisms, are presented to a larger portion extent. Finally, similarities are discussed, illustrating the approaches for their structural design. Stated variables equations, employed for the description of the growth in the controllable environmental conditions of a vessel, the researched reaction series of proliferating dividing CHO population, joint with the values of maximal enzymatic activity, and solutions are outlined. Processes are listed in a way so that a reader can integrate the state-of-the-art. Our particular contribution is also denoted.</div>','<b>Title: </b>Can Markov chain models mimic biological regulation<br><b>Year: </b>2002<br><b>Author: </b> S. Kim,  H. Li,  Er. Dougherty,  Nw. Cao,  Yd. Chen,  M. Bittner and  Eb. Suh<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1142/S0218339002000676"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF BIOLOGICAL SYSTEMS<br><b>Cluster-Keywords: </b><em>boolean networks, regulatory networks, genetic regulatory, gene regulatory, probabilistic boolean, network, networks, model, network model, probabilistic model, model checking, networks pbns, network dynamics, markov chain, regulatory, models, optimal control, boolean, networks models, networks boolean<div style="font-size:11px;"><br><b>Abstract: </b><em>A fundamental question in biology is whether the network of interactions that regulate gene expression can be modeled by existing mathematical techniques. Studies of the ability to predict a genes state based on the states of other genes suggest that it may be possible to abstract sufficient information to build models of the system that retain steady-state behavioral characteristics of the real system. This study tests this possibility by: i constructing a finite state homogeneous Markov chain model using a small set of interesting genes; ii estimating the model parameters based on the observed experimental data; iii exploring the dynamics of this small genetic regulatory network by analyzing its steady-state long-run behavior and comparing the resulting model behavior to the observed behavior of the original system. The data used in this study are from a survey of melanoma where predictive relationships coefficient of determination, CoD between 587 genes from 31 samples were examined. Ten genes with strong interactive connectivity were chosen to formulate a finite state Markov chain on the basis of their role as drivers in the acquisition of an invasive phenotype in melanoma cells. Simulations with different perturbation probabilities and different iteration times were run. Following convergence of the chain to steady-state behavior, millions of samples of the results of further transitions were collected to estimate the steady-state distribution of network. In these samples, only a limited number of states possessed significant probability of occurrence. This behavior is nicely congruent with biological behavior, as cells appear to occupy only a negligible portion of the state space available to them. The model produced both some of the exact state vectors observed in the data, and also a number of state vectors that were near neighbors of the state vectors from the original data. By combining these similar states, a good representation of the observed states in the original data could be achieved. From this study, we End that, in this limited context, Markov chain simulation emulates well the dynamic behavior of a small regulatory network.</div>','<b>Title: </b>Sequentially firing neurons confer flexible timing in neural pattern generators<br><b>Year: </b>2011<br><b>Author: </b> A. Urban and  B. Ermentrout<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1103/PhysRevE.83.051914"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PHYSICAL REVIEW E<br><b>Cluster-Keywords: </b><em>oscillator networks, network dynamics, network, networks, dynamical systems, dynamics, node dynamics, systems, coupled oscillator, synchronization, dynamics network, complex network, coupled system, network modifications, oscillators, network connectivity, feedback vertex, network topology, synchronization properties, collective dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>Neuronal networks exhibit a variety of complex spatiotemporal patterns that include sequential activity, synchrony, and wavelike dynamics. Inhibition is the primary means through which such patterns are implemented. This behavior is dependent on both the intrinsic dynamics of the individual neurons as well as the connectivity patterns. Many neural circuits consist of networks of smaller subcircuits motifs that are coupled together to form the larger system. In this paper, we consider a particularly simple motif, comprising purely inhibitory interactions, which generates sequential periodic dynamics. We first describe the dynamics of the single motif both for general balanced coupling all cells receive the same number and strength of inputs and then for a specific class of balanced networks: circulant systems. We couple these motifs together to form larger networks. We use the theory of weak coupling to derive phase models which, themselves, have a certain structure and symmetry. We show that this structure endows the coupled system with the ability to produce arbitrary timing relationships between symmetrically coupled motifs and that the phase relationships are robust over a wide range of frequencies. The theory is applicable to many other systems in biology and physics.</div>','<b>Title: </b>Dynamic simulations on the mitochondrial fatty acid Beta-oxidation network<br><b>Year: </b>2009<br><b>Author: </b> R. Modre-Osprian,  I. Osprian,  B. Tilg,  G. Schreier,  Km. Weinberger and  A. Graber<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-3-2"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: The oxidation of fatty acids in mitochondria plays an important role in energy metabolism and genetic disorders of this pathway may cause metabolic diseases. Enzyme deficiencies can block the metabolism at defined reactions in the mitochondrion and lead to accumulation of specific substrates causing severe clinical manifestations. Ten of the disorders directly affecting mitochondrial fatty acid oxidation have been well-defined, implicating episodic hypoketotic hypoglycemia provoked by catabolic stress, multiple organ failure, muscle weakness, or hypertrophic cardiomyopathy. Additionally, syndromes of severe maternal illness HELLP syndrome and AFLP have been associated with pregnancies carrying a fetus affected by fatty acid oxidation deficiencies. However, little is known about fatty acids kinetics, especially during fasting or exercise when the demand for fatty acid oxidation is increased catabolic stress.Results: A computational kinetic network of 64 reactions with 91 compounds and 301 parameters was constructed to study dynamic properties of mitochondrial fatty acid beta-oxidation. Various deficiencies of acyl-CoA dehydrogenase were simulated and verified with measured concentrations of indicative metabolites of screened newborns in Middle Europe and South Australia. The simulated accumulation of specific acyl-CoAs according to the investigated enzyme deficiencies are in agreement with experimental data and findings in literature. Investigation of the dynamic properties of the fatty acid beta-oxidation reveals that the formation of acetyl-CoA substrate for energy production - is highly impaired within the first hours of fasting corresponding to the rapid progress to coma within 1 - 2 hours. LCAD deficiency exhibits the highest accumulation of fatty acids along with marked increase of these substrates during catabolic stress and the lowest production rate of acetyl-CoA. These findings might confirm gestational loss to be the explanation that no human cases of LCAD deficiency have been described.Conclusion: In summary, this work provides a detailed kinetic model of mitochondrial metabolism with specific focus on fatty acid beta-oxidation to simulate and predict the dynamic response of that metabolic network in the context of human disease. Our findings offer insight into the disease process e. g. rapid progress to coma and might confirm new explanations no human cases of LCAD deficiency, which can hardly be obtained from experimental data alone.</div>','<b>Title: </b>Non-homogeneous dynamic Bayesian networks with Bayesian regularization for inferring gene regulatory networks with gradually time-varying structure<br><b>Year: </b>2013<br><b>Author: </b> F. Dondelinger,  S. Lebre and  D. Husmeier<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s10994-012-5311-x"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MACHINE LEARNING<br><b>Cluster-Keywords: </b><em>petri net, petri nets, bayesian networks, biological networks, systems biology, regulatory networks, dynamic bayesian, net model, gene regulatory, network data, biological systems, biological data, network analysis, network models, biochemical networks, time series, network structure, signal transduction, net models, dbn model<div style="font-size:11px;"><br><b>Abstract: </b><em>The proper functioning of any living cell relies on complex networks of gene regulation. These regulatory interactions are not static but respond to changes in the environment and evolve during the life cycle of an organism. A challenging objective in computational systems biology is to infer these time-varying gene regulatory networks from typically short time series of transcriptional profiles. While homogeneous models, like conventional dynamic Bayesian networks, lack the flexibility to succeed in this task, fully flexible models suffer from inflated inference uncertainty due to the limited amount of available data. In the present paper we explore a semi-flexible model based on a piecewise homogeneous dynamic Bayesian network regularized by gene-specific inter-segment information sharing. We explore different choices of prior distribution and information coupling and evaluate their performance on synthetic data. We apply our method to gene expression time series obtained during the life cycle of Drosophila melanogaster, and compare the predicted segmentation with other state-of-the-art techniques. We conclude our evaluation with an application to synthetic biology, where the objective is to predict an in vivo regulatory network of five genes in Saccharomyces cerevisiae subjected to a changing environment.</div>','<b>Title: </b>A fully connected network of Bernoulli units with correlation learning<br><b>Year: </b>1996<br><b>Author: </b> Ja. Dente and  Rv. Mendes<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/0375-96019500928-0"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PHYSICS LETTERS A<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Biological evidence suggests that pattern recognition and associative memory in the mammalian nervous system operates through the establishment of spatio-temporal patterns of activity and not by the evolution towards an equilibrium point as in attractor neural networks. Information is carried by the space-time correlation of the activity intensities rather than by the details of individual neuron signals. Furthermore the fast recognition times that are achieved with relatively slow biological neurons seem to be associated to the chaotic nature of the basal nervous activity. To copy the biology hardware may not be technologically sound, but to look for inspiration in the efficient biological information processing methods is an idea that deserves consideration. Inspired by the mechanisms at work in the mammalian olfactory system we study a network where, in the absence of external inputs, the units have a dynamics of the Bernoulli shift type. When an external signal is presented, the pattern of excitation bursts depends on the learning history of the network. Association and pattern identification in the network operates by the selection, by the external stimulus, of distinct invariant measures in the chaotic system. The simplicity of the node dynamics, that is chosen, allows a reasonable analytical control of the network behavior.</div>','<b>Title: </b>In Vitro Perfused Human Capillary Networks<br><b>Year: </b>2013<br><b>Author: </b> Ml. Moya,  Yh. Hsu,  Ap. Lee,  Ccw. Hughes and  Sc. George<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1089/ten.tec.2012.0430"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>TISSUE ENGINEERING PART C-METHODS<br><b>Cluster-Keywords: </b><em>neural network, network model, network, neurogenesis model, genomic network, networks, saccade control, droplets networks, understanding networks, gene regulatory, model, tumor-immune networks, networks dynamical, network systems, artificial neurogenesis, regulatory networks, dynamic network, information flow, cancer cells, capillary networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Replicating in vitro the complex in vivo tissue microenvironment has the potential to transform our approach to medicine and also our understanding of biology. In order to accurately model the 3D arrangement and interaction of cells and extracellular matrix, new microphysiological systems must include a vascular supply. The vasculature not only provides the necessary convective transport of oxygen, nutrients, and waste in 3D culture, but also couples and integrates the responses of organ systems. Here we combine tissue engineering and microfluidic technology to create an in vitro 3D metabolically active stroma similar to 1 mm3 that, for the first time, contains a perfused,living, dynamic, interconnected human capillary network. The range of flow rate mu m/s and shear rate s-1 within the network was 0-4000 and 0-1000, respectively, and thus included the normal physiological range. Infusion of FITC dextran demonstrated microvessels 15-50 mu m to be largely impermeable to 70 kDa. Our high-throughput biology-directed platform has the potential to impact a broad range of fields that intersect with the microcirculation, including tumor metastasis, drug discovery, vascular disease, and environmental chemical toxicity.</div>','<b>Title: </b>Combinatorial complexity and dynamical restriction of network flows in signal transduction<br><b>Year: </b>2005<br><b>Author: </b> Jr. Faeder,  Ml. Blinov,  B. Goldstein and  Ws. Hlavacek<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1049/sb:20045031"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>allosteric proteins, signaling networks, protein interactions, network model, molecular dynamics, protein complexes, allosteric network, signaling proteins, cell network, biochemical networks, free energy, combinatorial complexity, molecular species, biological systems, receptor signaling, systems biology, chemical species, model, dynamics simulations, cell signaling<div style="font-size:11px;"><br><b>Abstract: </b><em>The activities and interactions of proteins that govern the cellular response to a signal generate a multitude of protein phosphorylation states and heterogeneous protein complexes. Here, using a computational model that accounts for 307 molecular species implied by specified interactions of four proteins involved in signalling by the immunoreceptor Fc epsilon RI, we determine the relative importance of molecular species that can be generated during signalling, chemical transitions among these species, and reaction paths that lead to activation of the protein tyrosine kinase PTK Syk. By all of these measures and over two- and ten-fold ranges of model parameters - rate constants and initial concentrations - only a small portion of the biochemical network is active. The spectrum of active complexes, however, can be shifted dramatically, even by a change in the concentration of a single protein, which suggests that the network can produce qualitatively different responses under different cellular conditions and in response to different inputs. Reduced models that reproduce predictions of the full model for a particular set of parameters lose their predictive capacity when parameters are varied over two-fold ranges.</div>','<b>Title: </b>Models from experiments: combinatorial drug perturbations of cancer cells<br><b>Year: </b>2008<br><b>Author: </b> S. Nelander,  Wq. Wang,  B. Nilsson,  Qb. She,  C. Pratilas,  N. Rosen,  P. Gennemark and  C. Sander<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1038/msb.2008.53"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MOLECULAR SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>network models, gene network, regulatory networks, boolean networks, gene expression, systems biology, neural networks, network dynamics, gene regulatory, biological networks, network structure, gap gene, gene regulation, cell cycle, network, gene interactions, signaling network, quantum systems, system dynamics, networks<div style="font-size:11px;"><br><b>Abstract: </b><em>We present a novel method for deriving network models from molecular profiles of perturbed cellular systems. The network models aim to predict quantitative outcomes of combinatorial perturbations, such as drug pair treatments or multiple genetic alterations. Mathematically, we represent the system by a set of nodes, representing molecular concentrations or cellular processes, a perturbation vector and an interaction matrix. After perturbation, the system evolves in time according to differential equations with built-in nonlinearity, similar to Hopfield networks, capable of representing epistasis and saturation effects. For a particular set of experiments, we derive the interaction matrix by minimizing a composite error function, aiming at accuracy of prediction and simplicity of network structure. To evaluate the predictive potential of the method, we performed 21 drug pair treatment experiments in a human breast cancer cell line MCF7 with observation of phospho-proteins and cell cycle markers. The best derived network model rediscovered known interactions and contained interesting predictions. Possible applications include the discovery of regulatory interactions, the design of targeted combination therapies and the engineering of molecular biological networks.</div>','<b>Title: </b>A dual regulation mechanism of histidine kinase CheA identified by combining network-dynamics modeling and system-level input-output data<br><b>Year: </b>2018<br><b>Author: </b> Ba. Mello,  Wl. Pan,  Gl. Hazelbauer and  Yh. Tu<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pcbi.1006305"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS COMPUTATIONAL BIOLOGY<br><b>Cluster-Keywords: </b><em>systems biology, biological systems, system dynamics, molecular systems, experimental design, network model, model system, model parameters, experiment design, kinetic parameters, perfect adaptation, experimental data, molecular mechanisms, homeostatic mechanisms, switching dynamics, biological mechanisms, control systems, chemotaxis network, decentralized control, feedback control<div style="font-size:11px;"><br><b>Abstract: </b><em>It is challenging to decipher molecular mechanisms in biological systems from system-level input-output data, especially for complex processes that involve interactions among multiple components. We addressed this general problem for the bacterial histidine kinase CheA, the activity of which is regulated in chemotaxis signaling complexes by bacterial chemoreceptors. We developed a general network model to describe the dynamics of the system, treating the receptor complex with coupling protein CheW and the P3P4P5 domains of kinase CheA as a regulated enzyme with two substrates, ATP and P1, the phosphoryl-accepting domain of CheA. Our simple network model allowed us to search hypothesis space systematically. For different and progressively more complex regulation schemes, we fit our models to a large set of input-output data with the aim of identifying the simplest possible regulation mechanisms consistent with the data. Our modeling and analysis revealed novel dual regulation mechanisms in which receptor activity regulated ATP binding plus one other process, either P1 binding or phosphoryl transfer between P1 and ATP. Strikingly, in our models receptor control affected the kinetic rate constants of substrate association and dissociation equally and thus did not alter the respective equilibrium constants. We suggest experiments that could distinguish between the two dual-regulation mechanisms. This systems-biology approach of combining modeling and a large input-output dataset should be applicable for studying other complex biological processes.</div>','<b>Title: </b>Ensemble methods for stochastic networks with special reference to the biological clock of Neurospora crassa<br><b>Year: </b>2018<br><b>Author: </b> C. Caranica,  A. Al-Omari,  Z. Deng,  J. Griffith,  R. Nilsen,  L. Mao,  J. Arnold and  Hb. Schuttler<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0196435"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>circadian rhythms, systems biology, biological clock, circadian clock, biological rhythms, hpa axis, model, clock network, mathematical models, biological systems, dynamic systems, ode models, biology models, single cell, network, model parameters, experimental data, models, systems, dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>A major challenge in systems biology is to infer the parameters of regulatory networks that operate in a noisy environment, such as in a single cell. In a stochastic regime it is hard to distinguish noise from the real signal and to infer the noise contribution to the dynamical behavior. When the genetic network displays oscillatory dynamics, it is even harder to infer the parameters that produce the oscillations. To address this issue we introduce a new estimation method built on a combination of stochastic simulations, mass action kinetics and ensemble network simulations in which we match the average periodogram and phase of the model to that of the data. The method is relatively fast compared to Metropolis-Hastings Monte Carlo Methods, easy to parallelize, applicable to large oscillatory networks and large similar to 2000 cells single cell expression data sets, and it quantifies the noise impact on the observed dynamics. Standard errors of estimated rate coefficients are typically two orders of magnitude smaller than the mean from single cell experiments with on the order of similar to 1000 cells. We also provide a method to assess the goodness of fit of the stochastic network using the Hilbert phase of single cells. An analysis of phase departures from the null model with no communication between cells is consistent with a hypothesis of Stochastic Resonance describing single cell oscillators. Stochastic Resonance provides a physical mechanism whereby intracellular noise plays a positive role in establishing oscillatory behavior, but may require model parameters, such as rate coefficients, that differ substantially from those extracted at the macroscopic level from measurements on populations of millions of communicating, synchronized cells.</div>','<b>Title: </b>Identifying latent dynamic components in biological systems<br><b>Year: </b>2015<br><b>Author: </b> I. Kondofersky,  C. Fuchs and  Fj. Theis<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1049/iet-syb.2014.0013"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IET SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>biological networks, dynamic models, biological systems, model predictions, model parameters, systems biology, mathematical models, biological models, equation models, experimental data, molecular network, interaction networks, model errors, biological interaction, molecular interactions, model, dynamic systems, biological parameters, parameter uncertainty, growth model<div style="font-size:11px;"><br><b>Abstract: </b><em>In computational systems biology, the general aim is to derive regulatory models from multivariate readouts, thereby generating predictions for novel experiments. In the past, many such models have been formulated for different biological applications. The authors consider the scenario where a given model fails to predict a set of observations with acceptable accuracy and ask the question whether this is because of the model lacking important external regulations. Real-world examples for such entities range from microRNAs to metabolic fluxes. To improve the prediction, they propose an algorithm to systematically extend the network by an additional latent dynamic variable which has an exogenous effect on the considered network. This variables time course and influence on the other species is estimated in a two-step procedure involving spline approximation, maximum-likelihood estimation and model selection. Simulation studies show that such a hidden influence can successfully be inferred. The method is also applied to a signalling pathway model where they analyse real data and obtain promising results. Furthermore, the technique can be employed to detect incomplete network structures.</div>','<b>Title: </b>Complexity and dynamics of host-fungal interactions<br><b>Year: </b>2012<br><b>Author: </b> F. Cottier and  N. Pavelka<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s12026-012-8265-y"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IMMUNOLOGIC RESEARCH<br><b>Cluster-Keywords: </b><em>sensitivity analysis, systems biology, robustness analysis, network model, network inference, regulatory network, systems analysis, gene expression, parametric uncertainty, biological networks, mathematical models, biochemical networks, biological functions, cellular networks, biological system, parametric sensitivity, dynamical models, dynamic system, networks analysis, gene regulatory<div style="font-size:11px;"><br><b>Abstract: </b><em>Pathologies attributable to fungal infections represent a growing concern in both developed and developing countries. Initially discovered as opportunistic pathogens of immunocompromised hosts, fungi such as Candida albicans are now being placed at the centre of a more complex and dynamic picture in which the outcome of an infection is the result of an intricate network of molecular interactions between the fungus, the host and the commensal microflora co-inhabiting various host niches, and especially the gastrointestinal GI tract. The complexity of the host-fungal interaction begins with the numerous pathogen-associated molecular patterns PAMPs present on the fungal cell wall that are recognized by multiple pathogen-recognition receptors PRRs, expressed by several types of host cells. PAMP-PRR interactions elicit a variety of intracellular signalling pathways leading to a wide array of immune responses, some of which promote fungal clearance while others contribute to pathogenesis. The picture is further complicated by the fact that numerous commensal bacteria normally co-inhabiting the hosts GI tract produce molecules that either directly modulate the survival and virulence of commensal fungi such as C. albicans or indirectly modulate the hosts antifungal immune responses. On top of this complexity, this host-microbiome-fungal interaction exhibits features of a dynamic system, in which the same fungi can easily switch between different morphological forms presenting different PAMPs at different moments of time. Furthermore, fungal pathogens can rapidly accumulate genomic alterations that further modify their recognition by the immune system, their virulence and their resistance to antifungal compounds. Thus, based on available molecular data alone, it is currently difficult to construct a coherent model able to explain the balance between commensalism and virulence and to predict the outcome of a fungal infection. Here, we review current advances in our understanding of this complex and dynamic system and propose new avenues of investigation to assemble a more complete picture of the host-fungal interaction, integrating microbiological and immunological data under the lens of systems biology and evolutionary genomics.</div>','<b>Title: </b>Inference of biochemical network models in S-system using multiobjective optimization approach<br><b>Year: </b>2008<br><b>Author: </b> Pk. Liu and  Fs. Wang<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1093/bioinformatics/btn075"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>ssystem model, biological systems, biological networks, parameter estimation, biochemical networks, dynamic biological, proposed algorithm, regulatory networks, structure identification, model structure, genetic algorithm, gene regulatory, gene expression, network inference, network motif, network structure, gene networks, network identification, network inferring, estimation method<div style="font-size:11px;"><br><b>Abstract: </b><em>Motivation: The inference of biochemical networks, such as gene regulatory networks, proteinprotein interaction networks, and metabolic pathway networks, from time-course data is one of the main challenges in systems biology. The ultimate goal of inferred modeling is to obtain expressions that quantitatively understand every detail and principle of biological systems. To infer a realizable S-system structure, most articles have applied sums of magnitude of kinetic orders as a penalty term in the fitness evaluation. How to tune a penalty weight to yield a realizable model structure is the main issue for the inverse problem. No guideline has been published for tuning a suitable penalty weight to infer a suitable model structure of biochemical networks.Results: We introduce an interactive inference algorithm to infer a realizable S-system structure for biochemical networks. The inference problem is formulated as a multiobjective optimization problem to minimize simultaneously the concentration error, slope error and interaction measure in order to find a suitable S-system model structure and its corresponding model parameters. The multiobjective optimization problem is solved by the -constraint method to minimize the interaction measure subject to the expectation constraints for the concentration and slope error criteria. The theorems serve to guarantee the minimum solution for the -constrained problem to achieve the minimum interaction network for the inference problem. The approach could avoid assigning a penalty weight for sums of magnitude of kinetic orders.</div>','<b>Title: </b>Competing to coordinate cell fate decisions: the MST2-Raf-1 signaling device<br><b>Year: </b>2015<br><b>Author: </b> Lk. Nguyen,  Dg. Matallanas,  D. Romano,  Bn. Kholodenko and  W. Kolch<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.4161/15384101.2014.973743"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>CELL CYCLE<br><b>Cluster-Keywords: </b><em>feedback loops, signaling network, positive feedback, network model, negative feedback, biochemical networks, signaling pathways, signal transduction, systems biology, kinase erk, protein kinase, transduction networks, dynamic model, emt network, erk pathway, cell fate, cell death, biochemical reaction, coupled feedback, impulse response<div style="font-size:11px;"><br><b>Abstract: </b><em>How do biochemical signaling pathways generate biological specificity This question is fundamental to modern biology, and its enigma has been accentuated by the discovery that most proteins in signaling networks serve multifunctional roles. An answer to this question may lie in analyzing network properties rather than individual traits of proteins in order to elucidate design principles of biochemical networks that enable biological decision-making. We discuss how this is achieved in the MST2/Hippo-Raf-1 signaling network with the help of mathematical modeling and model-based analysis, which showed that competing protein interactions with affinities controlled by dynamic protein modifications can function as Boolean computing devices that determine cell fate decisions. In addition, we discuss areas of interest for future research and highlight how systems approaches would be of benefit.</div>','<b>Title: </b>Inferring catalysis in biological systems<br><b>Year: </b>2016<br><b>Author: </b> I. Kondofersky,  Fj. Theis and  C. Fuchs<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1049/iet-syb.2015.0087"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IET SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>biological networks, dynamic models, biological systems, model predictions, model parameters, systems biology, mathematical models, biological models, equation models, experimental data, molecular network, interaction networks, model errors, biological interaction, molecular interactions, model, dynamic systems, biological parameters, parameter uncertainty, growth model<div style="font-size:11px;"><br><b>Abstract: </b><em>In systems biology, one is often interested in the communication patterns between several species, such as genes, enzymes or proteins. These patterns become more recognisable when temporal experiments are performed. This temporal communication can be structured by reaction networks such as gene regulatory networks or signalling pathways. Mathematical modelling of data arising from such networks can reveal important details, thus helping to understand the studied system. In many cases, however, corresponding models still deviate from the observed data. This may be due to unknown but present catalytic reactions. From a modelling perspective, the question of whether a certain reaction is catalysed leads to a large increase of model candidates. For large networks the calibration of all possible models becomes computationally infeasible. We propose a method which determines a substantially reduced set of appropriate model candidates and identifies the catalyst of each reaction at the same time. This is incorporated in a multiple-step procedure which first extends the network by additional latent variables and subsequently identifies catalyst candidates using similarity analysis methods. Results from synthetic data examples suggest a good performance even for non-informative data with few observations. Applied on CD95 apoptotic pathway our method provides new insights into apoptosis regulation.</div>','<b>Title: </b>Whi5 phosphorylation embedded in the G1/S network dynamically controls critical cell size and cell fate<br><b>Year: </b>2016<br><b>Author: </b> P. Palumbo,  M. Vanoni,  V. Cusimano,  S. Busti,  F. Marano,  C. Manes and  L. Alberghina<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1038/ncomms11372"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>NATURE COMMUNICATIONS<br><b>Cluster-Keywords: </b><em>cell cycle, mammalian cell, cdk network, cell proliferation, cell, cells cell, regulatory network, detailed model, cell size, molecular network, skeleton model, cycle arrest, cycle progression, cell division, cell density, kinases cdks, cycle, phase transition, cycle control, network driving<div style="font-size:11px;"><br><b>Abstract: </b><em>In budding yeast, overcoming of a critical size to enter S phase and the mitosis/mating switch-two central cell fate events-take place in the G1 phase of the cell cycle. Here we present a mathematical model of the basic molecular mechanism controlling the G1/S transition, whose major regulatory feature is multisite phosphorylation of nuclear Whi5. Cln3-Cdk1, whose nuclear amount is proportional to cell size, and then Cln1,2-Cdk1, randomly phosphorylate both decoy and functional Whi5 sites. Full phosphorylation of functional sites releases Whi5 inhibitory activity, activating G1/S transcription. Simulation analysis shows that this mechanism ensures coherent release of Whi5 inhibitory action and accounts for many experimentally observed properties of mitotically growing or conjugating G1 cells. Cell cycle progression and transcriptional analyses of a Whi5 phosphomimetic mutant verify the model prediction that coherent transcription of the G1/S regulon and ensuing G1/S transition requires full phosphorylation of Whi5 functional sites.</div>','<b>Title: </b>Comparing different ODE modelling approaches for gene regulatory networks<br><b>Year: </b>2009<br><b>Author: </b> A. Polynikis,  Sj. Hogan and  M. Di Bernardo<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.jtbi.2009.07.040"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF THEORETICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>regulatory networks, network dynamics, gene regulatory, network model, genetic network, gene networks, genetic regulatory, mathematical model, parameter space, network system, interaction networks, system dynamics, network structure, network, dynamic networks, network reduction, models models, biological networks, network motifs, biological systems<div style="font-size:11px;"><br><b>Abstract: </b><em>A fundamental step in synthetic biology and systems biology is to derive appropriate mathematical models for the purposes of analysis and design. For example, to synthesize a gene regulatory network, the derivation of a mathematical model is important in order to carry out in silico investigations of the network dynamics and to investigate parameter variations and robustness issues. Different mathematical frameworks have been proposed to derive such models. In particular, the use of sets of nonlinear ordinary differential equations ODEs has been proposed to model the dynamics of the concentrations of mRNAs and proteins. These models are usually characterized by the presence of highly nonlinear Hill function terms. A typical simplification is to reduce the number of equations by means of a quasi-steady-state assumption on the mRNA concentrations. This yields a class of simplified ODE models. A radically different approach is to replace the Hill functions by piecewise-linear approximations Casey, R., de Jong, H., Gouze, J.-L., 2006. Piecewise-linear models of genetic regulatory networks: equilibria and their stability. J. Math. Biol. 52 1, 27-56. A further modelling approach is the use of discrete-time maps Coutinho, R., Fernandez, B., Lima, R., Meyroneinc, A., 2006. Discrete time piecewise affine models of genetic regulatory networks. J. Math. Biol. 52, 524-570 where the evolution of the system is modelled in discrete, rather than continuous, time. The aim of this paper is to discuss and compare these different modelling approaches, using a representative gene regulatory network. We will show that different models often lead to conflicting conclusions concerning the existence and stability of equilibria and stable oscillatory behaviours. Moreover, we shall discuss, where possible, the viability of making certain modelling approximations e. g. quasi-steady-state mRNA dynamics or piecewise-linear approximations of Hill functions and their effects on the overall system dynamics. C 2009 Elsevier Ltd. All rights reserved.</div>','<b>Title: </b>Robustness analysis of a constraint-based metabolic model links cell growth and proteomics of Thermoanaerobacter tengcongensis under temperature perturbation<br><b>Year: </b>2013<br><b>Author: </b> W. Tong,  Z. Chen,  Z. Cao,  Qh. Wang,  Jy. Zhang,  X. Bai,  R. Wang and  Sq. Liu<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1039/c3mb25278g"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MOLECULAR BIOSYSTEMS<br><b>Cluster-Keywords: </b><em>metabolic network, metabolic model, dynamic flux, growth rate, genomescale metabolic, flux balance, balance analysis, metabolic flux, metabolic pathways, dynamic model, metabolic reactions, cellular growth, regulatory network, systems biology, optimal growth, metabolic, modeling metabolic, growth conditions, dynamic analysis, cell growth<div style="font-size:11px;"><br><b>Abstract: </b><em>The integration of omic data with metabolic networks has been demonstrated to be an effective approach to elucidate the underlying metabolic mechanisms in life. Because the metabolic pathways of Thermoanaerobacter tengcongensis T. tengcongensis are incomplete, we used a 1-C-13-glucose culture to monitor intracellular isotope-labeled metabolites by GC/MS and identified the gap gene in glucose catabolism, Re-citrate synthase. Based on genome annotation and biochemical information, we reconstructed the metabolic network of glucose metabolism and amino acid synthesis in T. tengcongensis, including 253 reactions, 227 metabolites, and 236 genes. Furthermore, we performed constraint based modeling CBM-derived robustness analysis on the model to study the dynamic changes of the metabolic network. By perturbing the culture temperature from 75 to 55 degrees C, we collected the bacterial growth rates and differential proteomes. Assuming that protein abundance changes represent metabolic flux variations, we proposed that the robustness analysis of the CBM model could decipher the effect of proteome change on the bacterial growth under perturbation. For approximately 73 of the reactions, the predicted cell growth changes due to such reaction flux variations matched the observed cell growth data. Our study, therefore, indicates that differential proteome data can be integrated with metabolic network modeling and that robustness analysis is a strong method for representing the dynamic change in cell phenotypes under perturbation.</div>','<b>Title: </b>A Method for Ovarian Follicle Encapsulation and Culture in a Proteolytically Degradable 3 Dimensional System<br><b>Year: </b>2011<br><b>Author: </b> A. Shikanov,  M. Xu,  Tk. Woodruff and  Ld. Shea<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3791/2695"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOVE-JOURNAL OF VISUALIZED EXPERIMENTS<br><b>Cluster-Keywords: </b><em>stem cell, stem cells, systems biology, mathematical models, cell biology, network model, signaling pathways, cell fate, computational models, cancer cells, gene expression, network dynamics, embryonic stem, signaling network, prostate cancer, mathematical modeling, psychological stress, apoptosis signaling, computational systems, molecular mechanisms<div style="font-size:11px;"><br><b>Abstract: </b><em>The ovarian follicle is the functional unit of the ovary that secretes sex hormones and supports oocyte maturation. In vitro follicle techniques provide a tool to model follicle development in order to investigate basic biology, and are further being developed as a technique to preserve fertility in the clinic1-4. Our in vitro culture system employs hydrogels in order to mimic the native ovarian environment by maintaining the 3D follicular architecture, cell-cell interactions and paracrine signaling that direct follicle development 5. Previously, follicles were successfully cultured in alginate, an inert algae-derived polysaccharide that undergoes gelation with calcium ions6-8. Alginate hydrogels formed at a concentration of 0.25 w/v were the most permissive for follicle culture, and retained the highest developmental competence 9. Alginate hydrogels are not degradable, thus an increase in the follicle diameter results in a compressive force on the follicle that can impact follicle growth10. We subsequently developed a culture system based on a fibrin-alginate interpenetrating network FA-IPN, in which a mixture of fibrin and alginate are gelled simultaneously. This combination provides a dynamic mechanical environment because both components contribute to matrix rigidity initially; however, proteases secreted by the growing follicle degrade fibrin in the matrix leaving only alginate to provide support. With the IPN, the alginate content can be reduced below 0.25, which is not possible with alginate alone 5. Thus, as the follicle expands, it will experience a reduced compressive force due to the reduced solids content. Herein, we describe an encapsulation method and an in vitro culture system for ovarian follicles within a FA-IPN. The dynamic mechanical environment mimics the natural ovarian environment in which small follicles reside in a rigid cortex and move to a more permissive medulla as they increase in size11. The degradable component may be particularly critical for clinical translation in order to support the greater than 106-fold increase in volume that human follicles normally undergo in vivo</div>','<b>Title: </b>A Bayesian approach to estimating hidden variables as well as missing and wrong molecular interactions in ordinary differential equation-based mathematical models<br><b>Year: </b>2017<br><b>Author: </b> B. Engelhardt,  M. Kschischo and  H. Frohlich<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1098/rsif.2017.0332"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF THE ROYAL SOCIETY INTERFACE<br><b>Cluster-Keywords: </b><em>biological networks, dynamic models, biological systems, model predictions, model parameters, systems biology, mathematical models, biological models, equation models, experimental data, molecular network, interaction networks, model errors, biological interaction, molecular interactions, model, dynamic systems, biological parameters, parameter uncertainty, growth model<div style="font-size:11px;"><br><b>Abstract: </b><em>Ordinary differential equations ODEs are a popular approach to quantitatively model molecular networks based on biological knowledge. However, such knowledge is typically restricted. Wrongly modelled biological mechanisms as well as relevant external influence factors that are not included into the model are likely to manifest in major discrepancies between model predictions and experimental data. Finding the exact reasons for such observed discrepancies can be quite challenging in practice. In order to address this issue, we suggest a Bayesian approach to estimate hidden influences in ODE-based models. The method can distinguish between exogenous and endogenous hidden influences. Thus, we can detect wrongly specified as well as missed molecular interactions in the model. We demonstrate the performance of our Bayesian dynamic elastic-net with several ordinary differential equation models from the literature, such as human JAK-STAT signalling, information processing at the erythropoietin receptor, isomerization of liquid a-Pinene, G protein cycling in yeast and UV-B triggered signalling in plants. Moreover, we investigate a set of commonly known network motifs and a gene-regulatory network. Altogether our method supports the modeller in an algorithmic manner to identify possible sources of errors in ODE-based models on the basis of experimental data.</div>','<b>Title: </b>A framework for analyzing the relationship between gene expression and morphological, topological, and dynamical patterns in neuronal networks<br><b>Year: </b>2015<br><b>Author: </b> Hf. De Arruda,  Ch. Comin,  M. Miazaki,  Mp. Viana and  Ld. Costa<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.jneumeth.2015.02.019"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF NEUROSCIENCE METHODS<br><b>Cluster-Keywords: </b><em>regulatory networks, gene regulatory, gene expression, network inference, expression data, network structure, gene networks, networks grns, regulatory genes, boolean networks, bayesian network, network reconstruction, networks gene, network dynamics, gene regulation, inference method, systems biology, time series, infer gene, network<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: A key point in developmental biology is to understand how gene expression influences the morphological and dynamical patterns that are observed in living beings.New method: In this work we propose a methodology capable of addressing this problem that is based on estimating the mutual information and Pearson correlation between the intensity of gene expression and measurements of several morphological properties of the cells. A similar approach is applied in order to identify effects of gene expression over the system dynamics. Neuronal networks were artificially grown over a lattice by considering a reference model used to generate artificial neurons. The input parameters of the artificial neurons were determined according to two distinct patterns of gene expression and the dynamical response was assessed by considering the integrate-and-fire model.Results: As far as single gene dependence is concerned, we found that the interaction between the gene expression and the network topology, as well as between the former and the dynamics response, is strongly affected by the gene expression pattern. In addition, we observed a high correlation between the gene expression and some topological measurements of the neuronal network for particular patterns of gene expression.Comparison with existing methods: To our best understanding, there are no similar analyses to compare with.Conclusions: A proper understanding of gene expression influence requires jointly studying the morphology, topology, and dynamics of neurons. The proposed framework represents a first step towards predicting gene expression patterns from morphology and connectivity. C 2015 Published by Elsevier B.V.</div>','<b>Title: </b>Computational Neurogenetics<br><b>Year: </b>2004<br><b>Author: </b> N. Kasabov and  L. Benuskova<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1166/jctn.2004.006"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF COMPUTATIONAL AND THEORETICAL NANOSCIENCE<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>The aim of the paper is to introduce the scope and the problems of a new research area called Computational Neurogenetics CNG, along with some solutions and directions for further research. CNG is concerned with the study and the development of dynamic neuronal models integrated with gene models. This area brings together knowledge from various science disciplines, such as computer and information science, neuroscience and cognitive study, genetics and molecular biology. A computational neurogenetic model is created to model a brain function or a brain disease manifestation, or to be used as a general mathematical model for solving complex scientific and engineering problems. The CNG area goes beyond modelling a simple relationship between a single gene and a single neuronal function or a neuronal parameter. It is the interaction between hundreds and thousands of genes in a neuron and their relationships with the functioning of a neuronal ensemble and the brain as a whole e.g., learning and memory, speech and vision, epilepsy, mental retardation, aging, neural stem cells, etc.. CNG models constitute a second generation of neural network models that are closer to biological neural networks in their complex symbiosis of neuronal learning dynamics and molecular processes. Concrete models are presented as examples-evolving connectionist systems ECOS with evolutionary parameter optimisation and the CNG model of a class of spiking neural network ensembles CNG-SNN.</div>','<b>Title: </b>A Detailed Modular Analysis of Heat-Shock Protein Dynamics under Acute and Chronic Stress and Its Implication in Anxiety Disorders<br><b>Year: </b>2012<br><b>Author: </b> K. Sriram,  M. Rodriguez-Fernandez and  Fj. Doyle<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0042958"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>circadian rhythms, systems biology, biological clock, circadian clock, biological rhythms, hpa axis, model, clock network, mathematical models, biological systems, dynamic systems, ode models, biology models, single cell, network, model parameters, experimental data, models, systems, dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>Physiological and psychological stresses cause anxiety disorders such as depression and post-traumatic stress disorder PTSD and induce drastic changes at a molecular level in the brain. To counteract this stress, the heat-shock protein HSP network plays a vital role in restoring the homeostasis of the system. To study the stress-induced dynamics of heat-shock network, we analyzed three modules of the HSP90 network-namely trimerization reactions, phosphorylation-dephosphorylation reactions, and the conversion of HSP90 from an open to a closed conformation-and constructed a corresponding nonlinear differential equation model based on mass action kinetics laws. The kinetic parameters of the model were obtained through global optimization, and sensitivity analyses revealed that the most sensitive parameters are the kinase and phosphatase that drive the phosphorylation-dephosphorylation reactions. Bifurcation analysis carried out with the estimated kinetic parameters of the model with stress as bifurcation parameter revealed the occurrence of "mushroom", a type of complex dynamics in which S-shaped and Z-shaped hysteretic bistable forms are present together. We mapped the molecular events responsible for generating the mushroom dynamics under stress and interpreted the occurrence of the S-shaped hysteresis to a normal level of stress, and the Z-shaped hysteresis to the HSP90 variations under acute and chronic stress in the fear conditioned system, and further, we hypothesized that this can be extended to stress-related disorders such as depression and PTSD in humans. Finally, we studied the effect of parameter variations on the mushroom dynamics to get insight about the role of phosphorylation-dephosphorylation parameters in HSP90 network in bringing about complex dynamics such as isolas, where the stable steady states in a bistable system are isolated and separated from each other and not connected by an unstable steady state.</div>','<b>Title: </b>Boolean Network Model Predicts Knockout Mutant Phenotypes of Fission Yeast<br><b>Year: </b>2013<br><b>Author: </b> Mi. Davidich and  S. Bornholdt<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0071786"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>network models, gene network, regulatory networks, boolean networks, gene expression, systems biology, neural networks, network dynamics, gene regulatory, biological networks, network structure, gap gene, gene regulation, cell cycle, network, gene interactions, signaling network, quantum systems, system dynamics, networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Boolean networks or: networks of switches are extremely simple mathematical models of biochemical signaling networks. Under certain circumstances, Boolean networks, despite their simplicity, are capable of predicting dynamical activation patterns of gene regulatory networks in living cells. For example, the temporal sequence of cell cycle activation patterns in yeasts S. pombe and S. cerevisiae are faithfully reproduced by Boolean network models. An interesting question is whether this simple model class could also predict a more complex cellular phenomenology as, for example, the cell cycle dynamics under various knockout mutants instead of the wild type dynamics, only. Here we show that a Boolean network model for the cell cycle control network of yeast S. pombe correctly predicts viability of a large number of known mutants. So far this had been left to the more detailed differential equation models of the biochemical kinetics of the yeast cell cycle network and was commonly thought to be out of reach for models as simplistic as Boolean networks. The new results support our vision that Boolean networks may complement other mathematical models in systems biology to a larger extent than expected so far, and may fill a gap where simplicity of the model and a preference for an overall dynamical blueprint of cellular regulation, instead of biochemical details, are in the focus.</div>','<b>Title: </b>Connecting Artificial Brains to Robots in a Comprehensive Simulation Framework: The Neurorobotics Platform<br><b>Year: </b>2017<br><b>Author: </b> E. Falotico,  L. Vannucci,  A. Ambrosano,  U. Albanese,  S. Ulbrich,  Jcv. Tieck,  G. Hinkel,  J. Kaiser,  I. Peric,  O. Denninger,  N. Cauli,  M. Kirtay,  A. Roennau,  G. Klinker,  A. Arnim,  L. Guyot,  D. Peppicelli,  P. Martinez-Canada,  E. Ros,  P. Maier,  S. Weber,  M. Huber,  D. Plecher,  F. Rohrbein,  S. Deser,  A. Roitberg,  P. Van Der Smagt,  R. Dillman,  P. Levi,  C. Laschi,  Ac. Knoll and  Mo. Gewaltig<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3389/fnbot.2017.00002"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>FRONTIERS IN NEUROROBOTICS<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Combined efforts in the fields of neuroscience, computer science, and biology allowed to design biologically realistic models of the brain based on spiking neural networks. For a proper validation of these models, an embodiment in a dynamic and rich sensory environment, where the model is exposed to a realistic sensory-motor task, is needed. Due to the complexity of these brain models that, at the current stage, cannot deal with real-time constraints, it is not possible to embed them into a real-world task. Rather, the embodiment has to be simulated as well. While adequate tools exist to simulate either complex neural networks or robots and their environments, there is so far no tool that allows to easily establish a communication between brain and body models. The Neurorobotics Platform is a new web-based environment that aims to fill this gap by offering scientists and technology developers a software infrastructure allowing them to connect brain models to detailed simulations of robot bodies and environments and to use the resulting neurorobotic systems for in silico experimentation. In order to simplify the workflow and reduce the level of the required programming skills, the platform provides editors for the specification of experimental sequences and conditions, environments, robots, and brain-body connectors. In addition to that, a variety of existing robots and environments are provided. This work presents the architecture of the first release of the Neurorobotics Platform developed in subproject 10 "Neurorobotics" of the Human Brain Project HBP. 1 At the current state, the Neurorobotics Platform allows researchers to design and run basic experiments in neurorobotics using simulated robots and simulated environments linked to simplified versions of brain models. We illustrate the capabilities of the platform with three example experiments: a Braitenberg task implemented on a mobile robot, a sensory-motor learning task based on a robotic controller, and a visual tracking embedding a retina model on the iCub humanoid robot. These use-cases allow to assess the applicability of the Neurorobotics Platform for robotic tasks as well as in neuroscientific experiments.</div>','<b>Title: </b>Human-Inspired Neurorobotic System for Classifying Surface Textures by Touch<br><b>Year: </b>2016<br><b>Author: </b> Ke. Friedl,  Ar. Voelker,  A. Peer and  C. Eliasmith<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1109/LRA.2016.2517213"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IEEE ROBOTICS AND AUTOMATION LETTERS<br><b>Cluster-Keywords: </b><em>network models, population model, field model, neural population, neural field, iot networks, neuronal networks, neural network, differential equations, parameter neural, lotkavolterra model, model, neuron models, model neurons, model leads, ecosystem model, iot devices, noise model, neuronal models, inhibitory synaptic<div style="font-size:11px;"><br><b>Abstract: </b><em>Giving robots the ability to classify surface textures requires appropriate sensors and algorithms. Inspired by the biology of human tactile perception, we implement a neurorobotic texture classifier with a recurrent spiking neural network, using a novel semisupervised approach for classifying dynamic stimuli. Input to the network is supplied by accelerometers mounted on a robotic arm. The sensor data are encoded by a heterogeneous population of neurons, modeled to match the spiking activity of mechanoreceptor cells. This activity is convolved by a hidden layer using bandpass filters to extract nonlinear frequency information from the spike trains. The resulting high-dimensional feature representation is then continuously classified using a neurally implemented support vector machine. We demonstrate that our system classifies 18 metal surface textures scanned in two opposite directions at a constant velocity. We also demonstrate that our approach significantly improves upon a baseline model that does not use the described feature extraction. This method can be performed in real-time using neuromorphic hardware, and can be extended to other applications that process dynamic stimuli online.</div>','<b>Title: </b>Handling variability and incompleteness of biological data by flexible nets: a case study for Wilson disease<br><b>Year: </b>2018<br><b>Author: </b> J. Julvez,  D. Dikicioglu and  Sg. Oliver<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1038/s41540-017-0044-x"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>NPJ SYSTEMS BIOLOGY AND APPLICATIONS<br><b>Cluster-Keywords: </b><em>petri net, petri nets, bayesian networks, biological networks, systems biology, regulatory networks, dynamic bayesian, net model, gene regulatory, network data, biological systems, biological data, network analysis, network models, biochemical networks, time series, network structure, signal transduction, net models, dbn model<div style="font-size:11px;"><br><b>Abstract: </b><em>Mathematical models that combine predictive accuracy with explanatory power are central to the progress of systems and synthetic biology, but the heterogeneity and incompleteness of biological data impede our ability to construct such models. Furthermore, the robustness displayed by many biological systems means that they have the flexibility to operate under a range of physiological conditions and this is difficult for many modeling formalisms to handle. Flexible nets FNs address these challenges and represent a paradigm shift in model-based analysis of biological systems. FNs can: i handle uncertainties, ranges and missing information in concentrations, stoichiometry, network topology, and transition rates without having to resort to statistical approaches; ii accommodate different types of data in a unified model that integrates various cellular mechanisms; and iii be employed for system optimization and model predictive control. We present FNs and illustrate their capabilities by modeling a well-established system, the dynamics of glucose consumption by a microbial population. We further demonstrate the ability of FNs to take control actions in response to genetic or metabolic perturbations. Having bench-marked the system, we then construct the first quantitative model for Wilson disease-a rare genetic disorder that impairs copper utilization in the liver. We used this model to investigate the feasibility of using vitamin E supplementation therapy for symptomatic improvement. Our results indicate that hepatocytic inflammation caused by copper accumulation was not aggravated by limitations on endogenous antioxidant supplies, which means that treating patients with antioxidants is unlikely to be effective.</div>','<b>Title: </b>An approach for dynamical network reconstruction of simple network motifs<br><b>Year: </b>2013<br><b>Author: </b> M. Nakatsui,  M. Araki and  A. Kondo<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-7-S6-S4"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>ssystem model, biological systems, biological networks, parameter estimation, biochemical networks, dynamic biological, proposed algorithm, regulatory networks, structure identification, model structure, genetic algorithm, gene regulatory, gene expression, network inference, network motif, network structure, gene networks, network identification, network inferring, estimation method<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: One of the most important projects in the post-genome-era is the systemic identification of biological network. The almost of studies for network identification focused on the improvement of computational efficiency in large-scale network inference of complex system with cyclic relations and few attempted have been done for answering practical problem occurred in real biological systems. In this study, we focused to evaluate inferring performance of our previously proposed method for inferring biological network on simple network motifs.Results: We evaluated the network inferring accuracy and efficiency of our previously proposed network inferring algorithm, by using 6 kinds of repeated appearance of highly significant network motifs in the regulatory network of E. coli proposed by Shen-Orr et al and Herrgard et al, and 2 kinds of network motif in S. cerevisiae proposed by Lee et. al. As a result, our method could reconstruct about 40 of interactions in network motif from time-series data set. Moreover the introduction of time-series data of one-factor disrupted model could remarkably improved the performance of network inference.Conclusions: The results of network inference examination of E. coli network motif shows that our network inferring algorithm was able to apply to typical topology of biological network. A continuous examination of inferring well established network motif in biology would strengthen the applicability of our algorithm to the realistic biological network.</div>','<b>Title: </b>Synthetic gene circuits for metabolic control: design trade-offs and constraints<br><b>Year: </b>2013<br><b>Author: </b> Da. Oyarzun and  Gbv. Stan<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1098/rsif.2012.0671"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF THE ROYAL SOCIETY INTERFACE<br><b>Cluster-Keywords: </b><em>gene circuits, synthetic biology, circuit behavior, biological parts, synthetic gene, circuit design, genetic circuits, cell types, metabolite dynamics, mathematical model, designed circuit, control circuit, biological systems, circuit models, synthetic circuit, feedback circuits, biological circuits, design models, metabolic pathways, circuit parameters<div style="font-size:11px;"><br><b>Abstract: </b><em>A grand challenge in synthetic biology is to push the design of biomolecular circuits from purely genetic constructs towards systems that interface different levels of the cellular machinery, including signalling networks and metabolic pathways. In this paper, we focus on a genetic circuit for feedback regulation of unbranched metabolic pathways. The objective of this feedback system is to dampen the effect of flux perturbations caused by changes in cellular demands or by engineered pathways consuming metabolic intermediates. We consider a mathematical model for a control circuit with an operon architecture, whereby the expression of all pathway enzymes is transcriptionally repressed by the metabolic product. We address the existence and stability of the steady state, the dynamic response of the network under perturbations, and their dependence on common tuneable knobs such as the promoter characteristic and ribosome binding site RBS strengths. Our analysis reveals trade-offs between the steady state of the enzymes and the intermediates, together with a separation principle between promoter and RBS design. We show that enzymatic saturation imposes limits on the parameter design space, which must be satisfied to prevent metabolite accumulation and guarantee the stability of the network. The use of promoters with a broad dynamic range and a small leaky expression enlarges the design space. Simulation results with realistic parameter values also suggest that the control circuit can effectively upregulate enzyme production to compensate flux perturbations.</div>','<b>Title: </b>A network model of early events in epidermal growth factor receptor signaling that accounts for combinatorial complexity<br><b>Year: </b>2006<br><b>Author: </b> Ml. Blinov,  Jr. Faeder,  B. Goldstein and  Ws. Hlavacek<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.biosystems.2005.06.014"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOSYSTEMS<br><b>Cluster-Keywords: </b><em>allosteric proteins, signaling networks, protein interactions, network model, molecular dynamics, protein complexes, allosteric network, signaling proteins, cell network, biochemical networks, free energy, combinatorial complexity, molecular species, biological systems, receptor signaling, systems biology, chemical species, model, dynamics simulations, cell signaling<div style="font-size:11px;"><br><b>Abstract: </b><em>We consider a model of early events in signaling by the epidermal growth factor EGF receptor EGFR. The model includes EGF, EGFR, the adapter proteins Grb2 and Shc, and the guanine nucleotide exchange factor Sos, which is activated through EGF-induced formation of EGFR-Grb2-Sos and EGFR-Shc-Grb2-Sos assemblies at the plasma membrane. The protein interactions involved in signaling can potentially generate a diversity of protein complexes and phosphoforms; however, this diversity has been largely ignored in models of EGFR signaling. Here, we develop a model that accounts more fully for potential molecular diversity by specifying rules for protein interactions and then using these rules to generate a reaction network that includes all chemical species and reactions implied by the protein interactions. We obtain a model that predicts the dynamics of 356 molecular species, which are connected through 3749 unidirectional reactions. This network model is compared with a previously developed model that includes only 18 chemical species but incorporates the same scope of protein interactions. The predictions of this model are reproduced by the network model, which also yields new predictions. For example, the network model predicts distinct temporal patterns of autophosphorylation for different tyrosine residues of EGFR. A comparison of the two models suggests experiments that could lead to mechanistic insights about competition among adapter proteins for EGFR binding sites and the role of EGFR monomers in signal transduction. C 2005 Elsevier Ireland Ltd. All rights reserved.</div>','<b>Title: </b>Prokaryotic regulatory systems biology: Common principles governing the functional architectures of Bacillus subtilis and Escherichia coli unveiled by the natural decomposition approach<br><b>Year: </b>2012<br><b>Author: </b> Ja. Freyre-Gonzalez,  Lg. Trevino-Quintanilla,  Ia. Valtierra-Gutierrez,  Rm. Gutierrez-Rios and  Ja. Alonso-Pavon<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.jbiotec.2012.03.028"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF BIOTECHNOLOGY<br><b>Cluster-Keywords: </b><em>interaction networks, proteinprotein interactions, protein interaction, network model, complex systems, systems biology, protein protein, protein complex, host network, dynamic interactions, network analysis, biological systems, networks network, cellular network, biological network, viral proteins, interactome networks, regulatory networks, immune system, systems network<div style="font-size:11px;"><br><b>Abstract: </b><em>Escherichia coli and Bacillus subtilis are two of the best-studied prokaryotic model organisms. Previous analyses of their transcriptional regulatory networks have shown that they exhibit high plasticity during evolution and suggested that both converge to scale-free-like structures. Nevertheless, beyond this suggestion, no analyses have been carried out to identify the common systems-level components and principles governing these organisms. Here we show that these two phylogenetically distant organisms follow a set of common novel biologically consistent systems principles revealed by the mathematically and biologically founded natural decomposition approach. The discovered common functional architecture is a diamond-shaped, matryoshka-like, three-layer coordination, processing, and integration hierarchy exhibiting feedback, which is shaped by four systems-level components: global transcription factors global TFs, locally autonomous modules, basal machinery and intermodular genes. The first mathematical criterion to identify global TFs, the kappa-value, was reassessed on B. subtilis and confirmed its high predictive power by identifying all the previously reported, plus three potential, master regulators and eight sigma factors. The functionally conserved cores of modules, basal cell machinery, and a set of non-orthologous common physiological global responses were identified via both orthologous genes and non-orthologous conserved functions. This study reveals novel common systems principles maintained between two phylogenetically distant organisms and provides a comparison of their lifestyle adaptations. Our results shed new light on the systems-level principles and the fundamental functions required by bacteria to sustain life. C 2012 Elsevier B. V. All rights reserved.</div>','<b>Title: </b>Internal signal stochastic resonance of a synthetic gene network<br><b>Year: </b>2005<br><b>Author: </b> Zw. Wang,  Zh. Hou and  Hw. Xin<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1360/042004-85"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SCIENCE IN CHINA SERIES B-CHEMISTRY<br><b>Cluster-Keywords: </b><em>gene expression, gene network, stochastic model, gene regulatory, systems biology, expression systems, gene regulation, network activity, expression dynamics, noise control, regulatory network, deterministic models, network components, stochastic modeling, regulation system, escherichia coli, regulatory system, synthetic gene, regulatory genes, gene<div style="font-size:11px;"><br><b>Abstract: </b><em>The dynamics behavior of a synthetic gene network controlled by random noise is investigated using a model proposed recently. The phenomena of noise induced oscillation NIO of the protein concentrations and internal signal stochastic resonance SR are studied by computer simulation. We also find that there exists an optimal noise intensity that can most favor the occurrence of effective oscillation EO. Finally we discuss the potential constructive roles of SR on gene expression systems.</div>','<b>Title: </b>Parkinsonian tremor and simplification in network dynamics<br><b>Year: </b>1999<br><b>Author: </b> R. Edwards,  A. Beuter and  L. Glass<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1006/bulm.1998.0086"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BULLETIN OF MATHEMATICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>regulatory networks, network dynamics, gene regulatory, network model, genetic network, gene networks, genetic regulatory, mathematical model, parameter space, network system, interaction networks, system dynamics, network structure, network, dynamic networks, network reduction, models models, biological networks, network motifs, biological systems<div style="font-size:11px;"><br><b>Abstract: </b><em>We explore the behavior of richly connected inhibitory neural networks under parameter changes that correspond to weakening of synaptic efficacies between network units, and show that transitions from irregular to periodic dynamics are common in such systems. The weakening of these connections leads to a reduction in the number of units that effectively drive the dynamics and thus to simpler behavior. We hypothesize that the multiple interconnecting loops of the brains motor circuitry, which involve many inhibitory connections, exhibit such transitions. Normal physiological tremor is irregular while other forms of tremor show more regular oscillations. Tremor in Parkinsons disease, for example, stems from weakened synaptic efficacies of dopaminergic neurons in the nigro-striatal pathway, as in our general model. The multiplicity of structures involved in the production of symptoms in Parkinsons disease and the reversibility of symptoms by pharmacological and surgical manipulation of connection parameters suggest that such a neural network model is appropriate. Furthermore, fixed points that can occur in the network models are suggestive of akinesia in Parkinsons disease. This model is consistent with the view that normal physiological systems can be regulated by robust and richly connected feedback networks with complex dynamics, and that loss of complexity in the feedback structure due to disease leads to more orderly behavior. C 1999 Society for Mathematical Biology.</div>','<b>Title: </b>Modeling the Hes1 oscillator<br><b>Year: </b>2007<br><b>Author: </b> S. Zeiser,  J. Mueller and  V. Liebscher<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1089/cmb.2007.0029"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF COMPUTATIONAL BIOLOGY<br><b>Cluster-Keywords: </b><em>gene regulatory, systems biology, synthetic biology, regulatory network, gene network, gene expression, system model, synthetic network, network dynamics, systems theory, synthetic gene, genetic network, model predicts, dynamical systems, time delays, translational time, regulation models, oscillatory expression, network structure, biological systems<div style="font-size:11px;"><br><b>Abstract: </b><em>Somitogenesis describes the segmentation of vertebrate embryonic bodies, which is thought to be induced by ultradian clocks i.e., clocks with relatively short cycles compared to circadian clocks. One candidate for such a clock is the bHLH factor Hes1, forming dimers which repress the transcription of its own encoding gene. Most models for such small autoregulative networks are based on delay equations where a Hill function represents the regulation of transcription. The aim of the present paper is to estimate the Hill coefficient in the switch of an Hes1 oscillator and to suggest a more detailed model of the autoregulative network. The promoter of Hes1 consists of three to four binding sites for Hes1 dimers. Using the sparse data from literature, we find, in contrast to other statements in literature, that there is not much evidence for synergistic binding in the regulatory region of Hes1, and that the Hill coefficient is about three. As a model for the negative feedback loop, we use a Goodwin system and find sustained oscillations for systems with a large enough number of linear differential equations. By a suitable variation of the number of equations, we provide a rational lower bound for the Hill coefficient for such a system. Our results suggest that there exist additional nonlinear processes outside of the regulatory region of Hes1.</div>','<b>Title: </b>Systems biology surveillance decrypts pathological transcriptome remodeling<br><b>Year: </b>2015<br><b>Author: </b> Rs. Faustino,  Sp. Wyles,  J. Groenendyk,  M. Michalak,  A. Terzic and  C. Perez-Terzic<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/s12918-015-0177-8"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>gene regulatory, systems biology, regulatory networks, gene network, stem cell, network model, transcription factors, gene expression, cell transcriptomes, network analysis, data analysis, stem cells, network grnsight, cell transformation, weighted network, gene ontology, networks grns, graph model, graph layout, endothelial cell<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Pathological cardiac development is precipitated by dysregulation of calreticulin, an endoplasmic reticulum ER-resident calcium binding chaperone and critical contributor to cardiogenesis and embryonic viability. However, pleiotropic phenotype derangements induced by calreticulin deficiency challenge the identification of specific downstream transcriptome elements that direct proper cardiac formation. Here, differential transcriptome navigation was used to diagnose high priority calreticulin domain-specific gene expression changes and decrypt complex cardiac-specific molecular responses elicited by discrete functional regions of calreticulin.Methods: Wild type WT, calreticulin-deficient CALR-/-, and calreticulin truncation variant CALR-/--NP and CALR-/--PC pluripotent stem cells were used to investigate molecular remodeling underlying a model of cardiopathology. Bioinformatic deconvolution of isolated transcriptomes was performed to identify predominant expression trends, gene ontology prioritizations, and molecular network features characteristic of discrete cell types.Results: Stem cell lines with wild type WT, calreticulin-deficient CALR-/- genomes, as well as calreticulin truncation variants exclusively expressing either the chaperoning CALR-/--NP or the calcium binding CALR-/--PC domain exhibited characteristic molecular signatures determined by unsupervised agglomerative clustering. Kohonen mapping of RNA expression changes identified transcriptome dynamics that segregated into 12 discrete gene expression meta-profiles which were enriched for regulation of Eukaryotic Initiation Factor 2 EIF2 signaling. Focused examination of domain-specific gene ontology remodeling revealed a general enrichment of Cardiovascular Development in the truncation variants, with unique prioritization of "Cardiovascular Disease" exclusive to the cohort of down regulated genes of the PC truncation variant. Molecular cartography of genes that comprised this cardiopathological category revealed uncharacterized and novel gene relationships, with identification of Pitx2 as a critical hub within the topology of a CALR-/- compromised network.Conclusions: Diagnostic surveillance, through an algorithm that integrates pluripotent stem cell transcriptomes with advanced high throughput assays and computational bioinformatics, revealed collective gene expression network changes that underlie differential phenotype development. Stem cell transcriptomes provide a deep collective molecular index that reflects ad hoc robustness of the pluripotent gene network. Remodeling events such as monogenic lesions provide a background by which high priority candidate disease effectors and regulators can be identified, demonstrated here by a molecular profiling algorithm that decrypts pluripotent wild type versus disrupted genomes.</div>','<b>Title: </b>Signal propagation in small-world biological networks with weak noise<br><b>Year: </b>2010<br><b>Author: </b> Dw. Hong and  Ss. Man<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.jtbi.2009.10.019"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF THEORETICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>complex networks, complex systems, network theory, network structure, biological network, social network, network science, dynamic networks, network systems, network model, network topology, systems biology, network, network analysis, system dynamics, networks, realworld networks, network automata, epidemic network, model network<div style="font-size:11px;"><br><b>Abstract: </b><em>An emerging notion in systems biology is that biological networks have evolved to function well while their components behave stochastically. Thus, the dynamics in a biological network consist of two parts, deterministic and stochastic. A fundamental question is to find a quantitative relation between the two parts. We term such a relation as a deterministic-stochastic principle DSP and propose a model for a DSP with regard to signal propagation in biological networks. In this model, i the dynamics in a biological network is supposed to be captured by a stochastic differential equation which has been a standard approach in modeling systems with internal noise; ii the internal noise of a biological network is weak as is apparent in experimental observations: and iii a biological network is organized as small-world as suggested by recent studies. We introduce the concept of a signaling sample path. Using this concept we relate the structure of a biological network to its dynamics. The network structure characterizes the deterministic part of the dynamics, which in turn ensures a probability for a signal to propagate. The weakness of the internal noise characterizes the stochastic part of the dynamics. Analysis of the proposed model yields a quantitative description as follows: in a small-world biological network with weak internal noise, the signaling pathways induced by the network structure for a signal may ensure a probability near 0 for the signal propagation. Despite such a small probability, a correct response to the signal will still occur with a probability close to I provided that this signal propagation can take a certain amount of time. Computer simulations are performed to illustrate this result. We also discuss how a recent study on the reconstruction of a transcription network in Saccharomyces cerevisiae has tested the proposed model against real data. Published by Elsevier Ltd.</div>','<b>Title: </b>Switch of Sensitivity Dynamics Revealed with DyGloSA Toolbox for Dynamical Global Sensitivity Analysis as an Early Warning for Systems Critical Transition<br><b>Year: </b>2013<br><b>Author: </b> T. Baumuratova,  S. Dobre,  T. Bastogne and  T. Sauter<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0082973"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>systems biology, dynamic network, early differentiation, complex diseases, stem cells, erythroid cells, system dynamics, network biomarkers, immune system, tcm syndrome, dnb genes, gene expression, dnb model, stem cell, expression dynamics, model system, mathematical model, model dynamics, dynamic model, sensitivity analysis<div style="font-size:11px;"><br><b>Abstract: </b><em>Systems with bifurcations may experience abrupt irreversible and often unwanted shifts in their performance, called critical transitions. For many systems like climate, economy, ecosystems it is highly desirable to identify indicators serving as early warnings of such regime shifts. Several statistical measures were recently proposed as early warnings of critical transitions including increased variance, autocorrelation and skewness of experimental or model-generated data. The lack of automatized tool for model-based prediction of critical transitions led to designing DyGloSA - a MATLAB toolbox for dynamical global parameter sensitivity analysis GPSA of ordinary differential equations models. We suggest that the switch in dynamics of parameter sensitivities revealed by our toolbox is an early warning that a system is approaching a critical transition. We illustrate the efficiency of our toolbox by analyzing several models with bifurcations and predicting the time periods when systems can still avoid going to a critical transition by manipulating certain parameter values, which is not detectable with the existing SA techniques. DyGloSA is based on the SBToolbox2 and contains functions, which compute dynamically the global sensitivity indices of the system by applying four main GPSA methods: eFAST, Sobols ANOVA, PRCC and WALS. It includes parallelized versions of the functions enabling significant reduction of the computational time up to 12 times. DyGloSA is freely available as a set of MATLAB scripts at http://bio.uni.lu/systemsbiology/software/dyglosa. It requires installation of MATLAB versions R2008b or later and the Systems Biology Toolbox2 available at www.sbtoolbox2. org.DyGloSA can be run on Windows and Linux systems, -32 and -64 bits.</div>','<b>Title: </b>Aberrant LncRNA Expression Profile in a Contusion Spinal Cord Injury Mouse Model<br><b>Year: </b>2016<br><b>Author: </b> Y. Ding,  Zw. Song and  Jb. Liu<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1155/2016/9249401"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOMED RESEARCH INTERNATIONAL<br><b>Cluster-Keywords: </b><em>gene expression, systems biology, metabolic network, network model, network analysis, dynamic model, signaling pathways, signaling network, regulatory network, dynamic network, biology approach, candidate genes, gene network, gene regulatory, biology analysis, mathematical model, network dynamic, biology network, metabolic pathways, pathway analysis<div style="font-size:11px;"><br><b>Abstract: </b><em>Long noncoding RNAs LncRNAs play a crucial role in cell growth, development, and various diseases related to the central nervous system. However, LncRNA differential expression profiles in spinal cord injury are yet to be reported. In this study, we profiled the expression pattern of LncRNAs using a microarray method in a contusion spinal cord injury SCI mouse model. Compared with a spinal cord without injury, few changes in LncRNA expression levels were noted 1 day after injury. The differential changes in LncRNA expression peaked 1 week after SCI and subsequently declined until 3 weeks after injury. Quantitative real-time polymerase chain reaction qRT-PCR was used to validate the reliability of the microarray, demonstrating that the results were reliable. Gene ontology GO analysis indicated that differentially expressed mRNAs were involved in transport, cell adhesion, ion transport, and metabolic processes, among others. Kyoto Encyclopedia of Genes and Genomes KEGG enrichment analysis showed that the neuroactive ligand-receptor interaction, the PI3K-Akt signaling pathway, and focal adhesions were potentially implicated in SCI pathology. We constructed a dynamic LncRNA-mRNA network containing 264 LncRNAs and 949 mRNAs to elucidate the interactions between the LncRNAs and mRNAs. Overall, the results from this study indicate for the first time that LncRNAs are differentially expressed in a contusion SCI mouse model.</div>','<b>Title: </b>Stochastic reaction networks with input processes: Analysis and application to gene expression inference<br><b>Year: </b>2019<br><b>Author: </b> E. Cinquemani<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.automatica.2018.11.047"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>AUTOMATICA<br><b>Cluster-Keywords: </b><em>reaction networks, network model, systems biology, biochemical reaction, stochastic reaction, model reduction, stochastic model, network dynamics, stochastic biochemical, stochastic simulation, system size, chemical reaction, gene network, elementary reactions, biochemical networks, parameter identification, parameter inference, cell populations, small system, linear noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Stochastic reaction network modelling is widely utilized to describe the probabilistic dynamics of biochemical systems in general, and gene interaction networks in particular. The statistical analysis of the response of these systems to perturbation inputs is typically dependent on specific perturbation models. Motivated by reporter gene systems, widely utilized in biology to monitor gene activity in individual cells, we address the analysis of reaction networks with state-affine rates in presence of an input process. We develop a generalization of the so-called moment equations that precisely accounts for the first- and second-order moments of arbitrary inputs without the need for a model of the input process, as well as spectral relationships between the network input and state. We then apply these results to develop a method for the reconstruction of the autocovariance function of gene activity from reporter gene population-snapshot data, a crucial step toward the investigation of gene regulation, and demonstrate its performance on a simulated case study. C 2018 Elsevier Ltd. All rights reserved.</div>','<b>Title: </b>Structural Analysis to Determine the Core of Hypoxia Response Network<br><b>Year: </b>2010<br><b>Author: </b> M. Heiner and  K. Sriram<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0008600"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>petri net, petri nets, bayesian networks, biological networks, systems biology, regulatory networks, dynamic bayesian, net model, gene regulatory, network data, biological systems, biological data, network analysis, network models, biochemical networks, time series, network structure, signal transduction, net models, dbn model<div style="font-size:11px;"><br><b>Abstract: </b><em>The advent of sophisticated molecular biology techniques allows to deduce the structure of complex biological networks. However, networks tend to be huge and impose computational challenges on traditional mathematical analysis due to their high dimension and lack of reliable kinetic data. To overcome this problem, complex biological networks are decomposed into modules that are assumed to capture essential aspects of the full networks dynamics. The question that begs for an answer is how to identify the core that is representative of a networks dynamics, its function and robustness. One of the powerful methods to probe into the structure of a network is Petri net analysis. Petri nets support network visualization and execution. They are also equipped with sound mathematical and formal reasoning based on which a network can be decomposed into modules. The structural analysis provides insight into the robustness and facilitates the identification of fragile nodes. The application of these techniques to a previously proposed hypoxia control network reveals three functional modules responsible for degrading the hypoxia-inducible factor HIF. Interestingly, the structural analysis identifies superfluous network parts and suggests that the reversibility of the reactions are not important for the essential functionality. The core network is determined to be the union of the three reduced individual modules. The structural analysis results are confirmed by numerical integration of the differential equations induced by the individual modules as well as their composition. The structural analysis leads also to a coarse network structure highlighting the structural principles inherent in the three functional modules. Importantly, our analysis identifies the fragile node in this robust network without which the switch-like behavior is shown to be completely absent.</div>','<b>Title: </b>Reorganization of the nuclear lamina and cytoskeleton in adipogenesis<br><b>Year: </b>2011<br><b>Author: </b> Vlrm. Verstraeten,  J. Renes,  Fcs. Ramaekers,  M. Kamps,  Hj. Kuijpers,  F. Verheyen,  M. Wabitsch,  Pm. Steijlen,  Mam. Van Steensel and  Jlv. Broers<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s00418-011-0792-4"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>HISTOCHEMISTRY AND CELL BIOLOGY<br><b>Cluster-Keywords: </b><em>regulatory network, systems biology, systems biological, network dynamics, gene expression, biochemical networks, raft proteins, biological networks, gene circuits, gene regulatory, signalling network, lipid raft, nuclear lamina, protein domain, networks systems, biological approaches, expression pattern, protein interaction, cell differentiation, network motifs<div style="font-size:11px;"><br><b>Abstract: </b><em>A thorough understanding of fat cell biology is necessary to counter the epidemic of obesity. Although molecular pathways governing adipogenesis are well delineated, the structure of the nuclear lamina and nuclear-cytoskeleton junction in this process are not. The identification of the linker of nucleus and cytoskeleton LINC complex made us consider a role for the nuclear lamina in adipose conversion. We herein focused on the structure of the nuclear lamina and its coupling to the vimentin network, which forms a cage-like structure surrounding individual lipid droplets in mature adipocytes. Analysis of a mouse and human model system for fat cell differentiation showed fragmentation of the nuclear lamina and subsequent loss of lamins A, C, B1 and emerin at the nuclear rim, which coincides with reorganization of the nesprin-3/plectin/vimentin complex into a network lining lipid droplets. Upon 18 days of fat cell differentiation, the fraction of adipocytes expressing lamins A, C and B1 at the nuclear rim increased, though overall lamin A/C protein levels were low. Lamin B2 remained at the nuclear rim throughout fat cell differentiation. Light and electron microscopy of a subcutaneous adipose tissue specimen showed striking indentations of the nucleus by lipid droplets, suggestive for an increased plasticity of the nucleus due to profound reorganization of the cellular infrastructure. This dynamic reorganization of the nuclear lamina in adipogenesis is an important finding that may open up new venues for research in and treatment of obesity and nuclear lamina-associated lipodystrophy.</div>','<b>Title: </b>Single-gene tuning of Caulobacter cell cycle period and noise, swarming motility, and surface adhesion<br><b>Year: </b>2010<br><b>Author: </b> Yh. Lin,  S. Crosson and  Nf. Scherer<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1038/msb.2010.95"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MOLECULAR SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>regulatory networks, stochastic simulation, gene networks, reaction networks, systems biology, stochastic model, network models, complex systems, networks stochastic, gene regulatory, biological systems, stochastic dynamics, differential equations, protein networks, cell cycle, continuous model, interaction networks, systems chemistry, time delays, biochemical reaction<div style="font-size:11px;"><br><b>Abstract: </b><em>Sensor histidine kinases underlie the regulation of a range of physiological processes in bacterial cells, from chemotaxis to cell division. In the gram-negative bacterium Caulobacter crescentus, the membrane-bound histidine kinase, DivJ, is a polar-localized regulator of cell cycle progression and development. We show that DivJ localizes to the cell pole through a dynamic diffusion and capture mechanism rather than by active localization. Analysis of single C. crescentus cells in microfluidic culture demonstrates that controlled expression of divJ permits facile tuning of both the mean and noise of the cell division period. Simulations of the cell cycle that use a simplified protein interaction network capture previously measured oscillatory protein profiles, and recapitulate the experimental observation that deletion of divJ increases the cell cycle period and noise. We further demonstrate that surface adhesion and swarming motility of C. crescentus in semi-solid media can also be tuned by divJ expression. We propose a model in which pleiotropic control of polar cell development by the DivJ-DivK-PleC signaling pathway underlies divJ-dependent tuning of cell swarming and adhesion behaviors. Molecular Systems Biology 6: 445; published online 21 December 2010; doi:10.1038/msb.2010.95</div>','<b>Title: </b>Pattern Formation on Networks: from Localised Activity to Turing Patterns<br><b>Year: </b>2016<br><b>Author: </b> N. Mccullen and  T. Wagenknecht<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1038/srep27397"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SCIENTIFIC REPORTS<br><b>Cluster-Keywords: </b><em>complex networks, complex systems, network theory, network structure, biological network, social network, network science, dynamic networks, network systems, network model, network topology, systems biology, network, network analysis, system dynamics, networks, realworld networks, network automata, epidemic network, model network<div style="font-size:11px;"><br><b>Abstract: </b><em>Networks of interactions between competing species are used to model many complex systems, such as in genetics, evolutionary biology or sociology and knowledge of the patterns of activity they can exhibit is important for understanding their behaviour. The emergence of patterns on complex networks with reaction-diffusion dynamics is studied here, where node dynamics interact via diffusion via the network edges. Through the application of a generalisation of dynamical systems analysis this work reveals a fundamental connection between small-scale modes of activity on networks and localised pattern formation seen throughout science, such as solitons, breathers and localised buckling. The connection between solutions with a single and small numbers of activated nodes and the fully developed system-scale patterns are investigated computationally using numerical continuation methods. These techniques are also used to help reveal a much larger portion of of the full number of solutions that exist in the system at different parameter values. The importance of network structure is also highlighted, with a key role being played by nodes with a certain so-called optimal degree, on which the interaction between the reaction kinetics and the network structure organise the behaviour of the system.</div>','<b>Title: </b>Emergent properties of interacting populations of spiking neurons<br><b>Year: </b>2011<br><b>Author: </b> S. Cardanobile and  S. Rotter<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3389/fncom.2011.00059"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>FRONTIERS IN COMPUTATIONAL NEUROSCIENCE<br><b>Cluster-Keywords: </b><em>network models, population model, field model, neural population, neural field, iot networks, neuronal networks, neural network, differential equations, parameter neural, lotkavolterra model, model, neuron models, model neurons, model leads, ecosystem model, iot devices, noise model, neuronal models, inhibitory synaptic<div style="font-size:11px;"><br><b>Abstract: </b><em>Dynamic neuronal networks are a key paradigm of increasing importance in brain research, concerned with the functional analysis of biological neuronal networks and, at the same time, with the synthesis of artificial brain-like systems. In this context, neuronal network models serve as mathematical tools to understand the function of brains, but they might as well develop into future tools for enhancing certain functions of our nervous system. Here, we present and discuss our recent achievements in developing multiplicative point processes into a viable mathematical framework for spiking network modeling. The perspective is that the dynamic behavior of these neuronal networks is faithfully reflected by a set of non-linear rate equations, describing all interactions on the population level. These equations are similar in structure to Lotka-Volterra equations, well known by their use in modeling predator-prey relations in population biology, but abundant applications to economic theory have also been described. We present a number of biologically relevant examples for spiking network function, which can be studied with the help of the aforementioned correspondence between spike trains and specific systems of non-linear coupled ordinary differential equations. We claim that, enabled by the use of multiplicative point processes, we can make essential contributions to a more thorough understanding of the dynamical properties of interacting neuronal populations.</div>','<b>Title: </b>Application of lumping analysis in modelling the living systems - A trade-off between simplicity and model quality<br><b>Year: </b>2006<br><b>Author: </b> G. MariaLink: No link<br><b>Journal: </b>CHEMICAL AND BIOCHEMICAL ENGINEERING QUARTERLY<br><b>Cluster-Keywords: </b><em>immune system, network models, immune response, model, biological systems, models, model reduction, dynamic model, biological networks, system, mathematical models, biochemical network, response model, systems, systems biology, network, immune, biology model, reaction networks, system systems<div style="font-size:11px;"><br><b>Abstract: </b><em>General chemical engineering modelling principles are valuable tools to represent both stationary and dynamic characteristics of complex cell processes. Elaboration of reduced lumped dynamic models uses all types of information translated from the language of molecular biology to that of mechanistical chemistry, by preserving cell structural hierarchy and component functions. A combination of non/conventional estimation methods reported significant model quality improvements by accounting for qualitative/quantitative data and global properties of the living system. Derivation of a satisfactory model is closely related to the ability of selecting the suitable lumping rules, key-parameters, and influential terms that better realize a trade-off between model simplicity and its predictive quality. Several examples, on the modular modelling of protein synthesis regulation, genetic regulatory networks, and on the successive drug-ligand release in human plasma from a complex multivalent support, illustrate the advantages but also the over-simplifications introduced by various lumping rules.</div>','<b>Title: </b>Multi-stage evolutionary algorithms for efficient identification of gene regulatory networks<br><b>Year: </b>2006<br><b>Author: </b> Ky. Kim,  Dy. Cho and  Bt. ZhangLink: No link<br><b>Journal: </b>APPLICATIONS OF EVOLUTIONARY COMPUTING, PROCEEDINGS<br><b>Cluster-Keywords: </b><em>regulatory networks, gene regulatory, gene expression, network inference, expression data, network structure, gene networks, networks grns, regulatory genes, boolean networks, bayesian network, network reconstruction, networks gene, network dynamics, gene regulation, inference method, systems biology, time series, infer gene, network<div style="font-size:11px;"><br><b>Abstract: </b><em>With the availability of the time series data from the high-throughput technologies, diverse approaches have been proposed to model gene regulatory networks. Compared with others, S-system has the advantage for these tasks in the sense that it can provide both quantitative structural and qualitative dynamical modeling in one framework. However, it is not easy to identify the structure of the true network since the number of parameters to be estimated is much larger than that of the available data. Moreover, conventional parameter estimation requires the time-consuming numerical integration to reproduce dynamic profiles for the S-system. In this paper, we propose multi-stage evolutionary algorithms to identify gene regulatory networks efficiently. With the symbolic regression by genetic programming GP, we can evade the numerical integration steps. This is because the estimation of slopes for each time-course data can be obtained from the results of GP. We also develop hybrid evolutionary algorithms and modified fitness evaluation function to identify the structure of gene regulatory networks and to estimate the corresponding parameters at the same time. By applying the proposed method to the identification of an artificial genetic network, we verify its capability of finding the true S-system.</div>','<b>Title: </b>Translational applications of evaluating physiologic variability in human endotoxemia<br><b>Year: </b>2013<br><b>Author: </b> Jd. Scheff,  Pd. Mavroudis,  Se. Calvano and  Ip. Androulakis<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s10877-012-9418-1"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF CLINICAL MONITORING AND COMPUTING<br><b>Cluster-Keywords: </b><em>inflammatory response, network model, mechanistic models, lde model, injured patients, critically injured, prediction model, tbi survivors, computational modeling, model abm, impaired astrocytes, model predicts, metabolic dynamics, significant differences, twoprocess model, agent-based model, coupled model, model, inflammatory network, computational models<div style="font-size:11px;"><br><b>Abstract: </b><em>Dysregulation of the inflammatory response is a critical component of many clinically challenging disorders such as sepsis. Inflammation is a biological process designed to lead to healing and recovery, ultimately restoring homeostasis; however, the failure to fully achieve those beneficial results can leave a patient in a dangerous persistent inflammatory state. One of the primary challenges in developing novel therapies in this area is that inflammation is comprised of a complex network of interacting pathways. Here, we discuss our approaches towards addressing this problem through computational systems biology, with a particular focus on how the presence of biological rhythms and the disruption of these rhythms in inflammation may be applied in a translational context. By leveraging the information content embedded in physiologic variability, ranging in scale from oscillations in autonomic activity driving short-term heart rate variability to circadian rhythms in immunomodulatory hormones, there is significant potential to gain insight into the underlying physiology.</div>','<b>Title: </b>UV-induced mutagenesis in Escherichia coli SOS response: A quantitative model<br><b>Year: </b>2007<br><b>Author: </b> S. Krishna,  S. Maslov and  K. Sneppen<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pcbi.0030041"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS COMPUTATIONAL BIOLOGY<br><b>Cluster-Keywords: </b><em>gene regulatory, gene expression, regulatory networks, expression data, regulatory pathway, regulatory genes, network dynamics, microarray data, genetic networks, transcription factor, models network, target gene, upstream regulatory, network topology, pathway network, gene functional, expression profile, transcriptional network, dynamic model, transcriptional regulatory<div style="font-size:11px;"><br><b>Abstract: </b><em>Escherichia coli bacteria respond to DNA damage by a highly orchestrated series of events known as the SOS response, regulated by transcription factors, protein-protein binding, and active protein degradation. We present a dynamical model of the UV-induced SOS response, incorporating mutagenesis by the error-prone polymerase, Pol V. In our model, mutagenesis depends on a combination of two key processes: damage counting by the replication forks and a long-term memory associated with the accumulation of UmuD. Together, these provide a tight regulation of mutagenesis, resulting, we show, in a "digital turn-on and turn-off of Pol V. Our model provides a compact view of the topology and design of the SOS network, pinpointing the specific functional role of each of the regulatory processes. In particular, we suggest that the recently observed second peak in the activity of promoters in the SOS regulon Friedman et al., 2005, PLoS Biology 37: e238 is the result of positive feedback from Pol V to RecA filaments.</div>','<b>Title: </b>Estimating confidence intervals in predicted responses for oscillatory biological models<br><b>Year: </b>2013<br><b>Author: </b> Pc. St John and  Fj. Doyle<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-7-71"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>circadian rhythms, systems biology, biological clock, circadian clock, biological rhythms, hpa axis, model, clock network, mathematical models, biological systems, dynamic systems, ode models, biology models, single cell, network, model parameters, experimental data, models, systems, dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: The dynamics of gene regulation play a crucial role in a cellular control: allowing the cell to express the right proteins to meet changing needs. Some needs, such as correctly anticipating the day-night cycle, require complicated oscillatory features. In the analysis of gene regulatory networks, mathematical models are frequently used to understand how a networks structure enables it to respond appropriately to external inputs. These models typically consist of a set of ordinary differential equations, describing a network of biochemical reactions, and unknown kinetic parameters, chosen such that the model best captures experimental data. However, since a models parameter values are uncertain, and since dynamic responses to inputs are highly parameter-dependent, it is difficult to assess the confidence associated with these in silico predictions. In particular, models with complex dynamics - such as oscillations - must be fit with computationally expensive global optimization routines, and cannot take advantage of existing measures of identifiability. Despite their difficulty to model mathematically, limit cycle oscillations play a key role in many biological processes, including cell cycling, metabolism, neuron firing, and circadian rhythms.Results: In this study, we employ an efficient parameter estimation technique to enable a bootstrap uncertainty analysis for limit cycle models. Since the primary role of systems biology models is the insight they provide on responses to rate perturbations, we extend our uncertainty analysis to include first order sensitivity coefficients. Using a literature model of circadian rhythms, we show how predictive precision is degraded with decreasing sample points and increasing relative error. Additionally, we show how this method can be used for model discrimination by comparing the output identifiability of two candidate model structures to published literature data.Conclusions: Our method permits modellers of oscillatory systems to confidently show that a models dynamic characteristics follow directly from experimental data and model structure, relaxing assumptions on the particular parameters chosen. Ultimately, this work highlights the importance of continued collection of high-resolution data on gene and protein activity levels, as they allow the development of predictive mathematical models.</div>','<b>Title: </b>Stochastic simulation of Boolean rxncon models: towards quantitative analysis of large signaling networks<br><b>Year: </b>2015<br><b>Author: </b> T. Mori,  M. Flottmann,  M. Krantz,  T. Akutsu and  E. Klipp<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/s12918-015-0193-8"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>boolean network, network models, qualitative models, systems biology, mathematical modeling, boolean models, signaling networks, dynamic models, dynamic modeling, signaling dynamics, logic models, dynamic network, mathematical models, boolean modeling, signal transduction, regulatory networks, drug resistance, biological networks, network approach, network modeling<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Cellular decision-making is governed by molecular networks that are highly complex. An integrative understanding of these networks on a genome wide level is essential to understand cellular health and disease. In most cases however, such an understanding is beyond human comprehension and requires computational modeling. Mathematical modeling of biological networks at the level of biochemical details has hitherto relied on state transition models. These are typically based on enumeration of all relevant model states, and hence become very complex unless severely - and often arbitrarily - reduced. Furthermore, the parameters required for genome wide networks will remain underdetermined for the conceivable future. Alternatively, networks can be simulated by Boolean models, although these typically sacrifice molecular detail as well as distinction between different levels or modes of activity. However, the modeling community still lacks methods that can simulate genome scale networks on the level of biochemical reaction detail in a quantitative or semi quantitative manner.Results: Here, we present a probabilistic bipartite Boolean modeling method that addresses these issues. The method is based on the reaction-contingency formalism, and enables fast simulation of large networks. We demonstrate its scalability by applying it to the yeast mitogen-activated protein kinase MAPK network consisting of 140 proteins and 608 nodes.Conclusion: The probabilistic Boolean model can be generated and parameterized automatically from a rxncon network description, using only two global parameters, and its qualitative behavior is robust against order of magnitude variation in these parameters. Our method can hence be used to simulate the outcome of large signal transduction network reconstruction, with little or no overhead in model creation or parameterization.</div>','<b>Title: </b>Construction of Cancer-Perturbed Protein-Protein Interaction Network of Apoptosis for Drug Target Discovery<br><b>Year: </b>2010<br><b>Author: </b> Lh. Chu and  Bs. Chen<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/978-1-4419-5797-924"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SYSTEMS BIOLOGY FOR SIGNALING NETWORKS<br><b>Cluster-Keywords: </b><em>gene networks, synthetic gene, synthetic biology, regulatory networks, design method, robust synthetic, nonlinear stochastic, molecular noises, gene regulatory, interaction networks, external disturbances, robust stability, robust gene, host cell, gene circuit, cellular network, design specifications, design procedure, signal transduction, parameter fluctuations<div style="font-size:11px;"><br><b>Abstract: </b><em>Inspecting interactive behaviors of proteins in cancer cells and comparing them with those in normal cells to obtain cancer-perturbed protein network can shed light on how a normal cell transforms into a cancer cell. Rough protein-protein interaction networks of apoptosis in cancer and normal cells are constructed according to human yeast-two-hybrid data sets and websites. The nonlinear stochastic model, maximum likelihood parameter estimation, and Akaike Information Criteria AIC are employed to reduce high false-positive rates in these large-scale interactome. By comparing protein-protein interaction networks of apoptosis between He La cancer cells and normal cells, we obtain cancer-perturbed networks and gain insight into the mechanism of apoptotic network in human cancer, which helps discovery of cancer drug targets. This proposed method could be extended to construct other perturbed protein interaction networks of cancer cells such as perturbated protein interaction network of cell cycle.</div>','<b>Title: </b>Nutrient exchange in arbuscular mycorrhizal symbiosis from a thermodynamic point of view<br><b>Year: </b>2019<br><b>Author: </b> I. Dreyer,  O. Spitz,  K. Kanonenberg,  K. Montag,  Mr. Handrich,  S. Ahmad,  S. Schott-Verdugo,  C. Navarro-Retamal,  Me. Rubio-Melendez,  Jl. Gomez-Porras,  J. Riedelsberger,  Ma. Molina-Montenegro,  A. Succurro,  A. Zuccaro,  Sb. Gould,  P. Bauer,  L. Schmitt and  H. Gohlke<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1111/nph.15646"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>NEW PHYTOLOGIST<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>To obtain insights into the dynamics of nutrient exchange in arbuscular mycorrhizal AM symbiosis, we modelled mathematically the two-membrane system at the plant-fungus interface and simulated its dynamics.In computational cell biology experiments, the full range of nutrient transport pathways was tested for their ability to exchange phosphorus P/carbon C/nitrogen N sources.As a result, we obtained a thermodynamically justified, independent and comprehensive model of the dynamics of the nutrient exchange at the plant-fungus contact zone. The predicted optimal transporter network coincides with the transporter set independently confirmed in wet-laboratory experiments previously, indicating that all essential transporter types have been discovered.The thermodynamic analyses suggest that phosphate is released from the fungus via proton-coupled phosphate transporters rather than anion channels. Optimal transport pathways, such as cation channels or proton-coupled symporters, shuttle nutrients together with a positive charge across the membranes. Only in exceptional cases does electroneutral transport via diffusion facilitators appear to be plausible. The thermodynamic models presented here can be generalized and adapted to other forms of mycorrhiza and open the door for future studies combining wet-laboratory experiments with computational simulations to obtain a deeper understanding of the investigated phenomena.</div>','<b>Title: </b>Immune signal transduction in leishmaniasis from natural to artificial systems: Role of feedback loop insertion<br><b>Year: </b>2014<br><b>Author: </b> M. Mol,  Ms. Patole and  S. Singh<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.bbagen.2013.08.018"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOCHIMICA ET BIOPHYSICA ACTA-GENERAL SUBJECTS<br><b>Cluster-Keywords: </b><em>systems biology, network models, signaling network, parameter estimation, dynamic models, biological systems, mathematical models, metabolic network, parameter sensitivities, computational model, gene circuit, circuit models, experimental data, metabolic models, model system, scalable parameter, model parameters, parameter space, biomolecular networks, model<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Modulated immune signal CD14-TLR and TNF in leishmaniasis can be linked to EGFR pathway involved in wound healing, through crosstalk points. This signaling network can be further linked to a synthetic gene circuit acting as a positive feedback loop to elicit a synchronized intercellular communication among the immune cells which may contribute to a better understanding of signaling dynamics in leishmaniasis.Methods: Network reconstruction with positive feedback loop, simulation ODE 15s solver and sensitivity analysis of CD14-TLR, TNF and EGFR was done in SimBiology MATLAB 7.11.1. Cytoscape and adjacency matrix were used to calculate network topology. PCA was extracted by using sensitivity coefficient in MATLAB. Model reduction was done using time, flux and sensitivity score.Results: Network has five crosstalk points: NIK, I kappa B-NF kappa B and MKK 4/7, 3/6, 1/2 which show high flux and sensitivity. PI3K in EGFR pathway shows high flux and sensitivity. PCA score was high for cytoplasmic ERK1/2, PI3K, Atk, STAT1/3 and nuclear JNK. Of the 125 parameters, 20 are crucial as deduced by model reduction.Conclusions: EGFR can be linked to CD14-TLR and TNF through the MAPK crosstalk points. These pathways may be controlled through Ras and Raf that lie upstream of signaling components ERK c and JNK n that have a high PCA score via a synthetic gene circuit for activating cell-cell communication to elicit an inflammatory response. Also a disease resolving effect may be achieved through P13K in the EGFR pathway.General significance: The reconstructed signaling network can be linked to a gene circuit with a positive feedback loop, for cell-cell communication resulting in synchronized response in the immune cell population, for disease resolving effect in leishmaniasis. c 2013 Elsevier B.V. All rights reserved.</div>','<b>Title: </b>Cytoskeletal turnover and Myosin contractility drive cell autonomous oscillations in a model of Drosophila Dorsal Closure<br><b>Year: </b>2014<br><b>Author: </b> Pf. Machado,  Gb. Blanchard,  J. Duque and  N. Gorfinkiel<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1140/epjst/e2014-02197-7"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>EUROPEAN PHYSICAL JOURNAL-SPECIAL TOPICS<br><b>Cluster-Keywords: </b><em>interaction network, actin filament, network model, filament network, network structures, actin network, protein interaction, actin dynamics, network, membrane protein, network dynamics, model dynamic, cell, model, network approach, protein complex, computational model, actin, adhesion dynamics, cell dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>Oscillatory behaviour in force-generating systems is a pervasive phenomenon in cell biology. In this work, we investigate how oscillations in the actomyosin cytoskeleton drive cell shape changes during the process of Dorsal Closure DC, a morphogenetic event in Drosophila embryo development whereby epidermal continuity is generated through the pulsatile apical area reduction of cells constituting the amnioserosa AS tissue. We present a theoretical model of AS cell dynamics by which the oscillatory behaviour arises due to a coupling between active myosin-driven forces, actin turnover and cell deformation. Oscillations in our model are cell-autonomous and are modulated by neighbour coupling, and our model accurately reproduces the oscillatory dynamics of AS cells and their amplitude and frequency evolution. A key prediction arising from our model is that the rate of actin turnover and Myosin contractile force must increase during DC in order to reproduce the decrease in amplitude and period of cell area oscillations observed in vivo. This prediction opens up new ways to think about the molecular underpinnings of AS cell oscillations and their link to net tissue contraction and suggests the form of future experimental measurements.</div>','<b>Title: </b>Uncovering a Predictive Molecular Signature for the Onset of NASH-Related Fibrosis in a Translational NASH Mouse Model<br><b>Year: </b>2018<br><b>Author: </b> A. Van Koppen,  L. Verschuren,  Am. Van Den Hoek,  J. Verheij,  Mc. Morrison,  K. Li,  H. Nagabukuro,  A. Costessi,  Mpm. Caspers,  Tj. Van Den Broek,  J. Sagartz,  C. Kluft,  C. Beysen,  C. Emson,  Aj. Van Gool,  R. Goldschmeding,  R. Stoop,  I. Bobeldijk-Pastorova,  Sm. Turner,  G. Hanauer and  R. Hanemaaijer<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.jcmgh.2017.10.001"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>CELLULAR AND MOLECULAR GASTROENTEROLOGY AND HEPATOLOGY<br><b>Cluster-Keywords: </b><em>systems biology, dynamic network, early differentiation, complex diseases, stem cells, erythroid cells, system dynamics, network biomarkers, immune system, tcm syndrome, dnb genes, gene expression, dnb model, stem cell, expression dynamics, model system, mathematical model, model dynamics, dynamic model, sensitivity analysis<div style="font-size:11px;"><br><b>Abstract: </b><em>BACKGROUND  AIMS: The incidence of nonalcoholic steatohepatitis NASH is increasing. The pathophysiological mechanisms of NASH and the sequence of events leading to hepatic fibrosis are incompletely understood. The aim of this study was to gain insight into the dynamics of key molecular processes involved in NASH and to rank early markers for hepatic fibrosis.METHODS: A time-course study in low-density lipoprotein-receptor knockout. Leiden mice on a high-fat diet was performed to identify the temporal dynamics of key processes contributing to NASH and fibrosis. An integrative systems biology approach was used to elucidate candidate markers linked to the active fibrosis process by combining transcriptomics, dynamic proteomics, and histopathology. The translational value of these findings were confirmed using human NASH data sets.RESULTS: High-fat-diet feeding resulted in obesity, hyperlipidemia, insulin resistance, and NASH with fibrosis in a time-dependent manner. Temporal dynamics of key molecular processes involved in the development of NASH were identified, including lipid metabolism, inflammation, oxidative stress, and fibrosis. A data-integrative approach enabled identification of the active fibrotic process preceding histopathologic detection using a novel molecular fibrosis signature. Human studies were used to identify overlap of genes and processes and to perform a network biology-based prioritization to rank top candidate markers representing the early manifestation of fibrosis.CONCLUSIONS: An early predictive molecular signature was identified that marked the active profibrotic process before histopathologic fibrosis becomes manifest. Early detection of the onset of NASH and fibrosis enables identification of novel blood-based biomarkers to stratify patients at risk, development of new therapeutics, and help shorten pre clinical experimental time frames.</div>','<b>Title: </b>Metabolic flux analysis at ultra short time scale: Isotopically non-stationary C-13 labeling experiments<br><b>Year: </b>2007<br><b>Author: </b> K. Noh,  K. Gronke,  B. Luo,  R. Takors,  M. Oldiges and  W. Wiechert<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.jbiotec.2006.11.015"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF BIOTECHNOLOGY<br><b>Cluster-Keywords: </b><em>kinetic models, metabolic networks, kinetic parameters, dynamic models, systems biology, network model, dynamic metabolic, biochemical network, rate constants, enzyme kinetics, kinetic data, adjustable parameters, metabolic model, parameter estimation, dynamic optimization, model selection, mathematical model, kinetic network, model response, model<div style="font-size:11px;"><br><b>Abstract: </b><em>A novel approach to C-13 metabolic flux analysis MFA is presented using cytosolic metabolite pool sizes and their C-13 labeling data from an isotopically non-stationary C-13 labeling experiment INST-CLE. The procedure is demonstrated with an E. coli wild type strain grown at fed batch conditions. The intra cellular labeling dynamics are excited by a sudden step increase of the C-13 portion in the substrate feed. Due to unchanged saturation of the substrate uptake system, the metabolic fluxes remain constant during the following sampling time period of only 16 s, in which 20 samples are taken by an automated rapid sampling device immediately stopping metabolism by methanol quenching. Subsequent cell disruptive sample preparation and LC-MS/MS enabled simultaneous determination of pool sizes and mass isotopomers of intra cellular metabolites requiring detection limits in the nM range. Based on this data the new computational flux analysis tool 13CFLUX/INST is used to determine the intra cellular fluxes based on a complex carbon labeling network model. The measured data is in good agreement with the model predictions, thus proving the applicability of the new isotopically non-stationary C-13 metabolic flux analysis INST-C-13-MFA concept. Moreover, it is shown that significant new information with respect to flux identifiability, non-measurable pool sizes, data consistency, or large storage pools can be taken from the novel kind of experimental data. This offers new insight into the biological operation of the metabolic network in vivo. c 2006 Elsevier B.V. All rights reserved.</div>','<b>Title: </b>Multiple Model-Informed Open-Loop Control of Uncertain Intracellular Signaling Dynamics<br><b>Year: </b>2014<br><b>Author: </b> Jp. Perley,  J. Mikolajczak,  Ml. Harrison,  Gt. Buzzard and  Ae. Rundell<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pcbi.1003546"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS COMPUTATIONAL BIOLOGY<br><b>Cluster-Keywords: </b><em>boolean network, network models, qualitative models, systems biology, mathematical modeling, boolean models, signaling networks, dynamic models, dynamic modeling, signaling dynamics, logic models, dynamic network, mathematical models, boolean modeling, signal transduction, regulatory networks, drug resistance, biological networks, network approach, network modeling<div style="font-size:11px;"><br><b>Abstract: </b><em>Computational approaches to tune the activation of intracellular signal transduction pathways both predictably and selectively will enable researchers to explore and interrogate cell biology with unprecedented precision. Techniques to control complex nonlinear systems typically involve the application of control theory to a descriptive mathematical model. For cellular processes, however, measurement assays tend to be too time consuming for real-time feedback control and models offer rough approximations of the biological reality, thus limiting their utility when considered in isolation. We overcome these problems by combining nonlinear model predictive control with a novel adaptive weighting algorithm that blends predictions from multiple models to derive a compromise open-loop control sequence. The proposed strategy uses weight maps to inform the controller of the tendency for models to differ in their ability to accurately reproduce the system dynamics under different experimental perturbations i.e. control inputs. These maps, which characterize the changing model likelihoods over the admissible control input space, are constructed using preexisting experimental data and used to produce a model-based open-loop control framework. In effect, the proposed method designs a sequence of control inputs that force the signaling dynamics along a predefined temporal response without measurement feedback while mitigating the effects of model uncertainty. We demonstrate this technique on the well-known Erk/MAPK signaling pathway in T cells. In silico assessment demonstrates that this approach successfully reduces target tracking error by 52 or better when compared with single model-based controllers and non-adaptive multiple model-based controllers. In vitro implementation of the proposed approach in Jurkat cells confirms a 63 reduction in tracking error when compared with the best of the single-model controllers. This study provides an experimentally-corroborated control methodology that utilizes the knowledge encoded within multiple mathematical models of intracellular signaling to design control inputs that effectively direct cell behavior in open-loop.Author Summary Most cell behavior arises as a response to external forces. Signals from the extracellular environment are passed to the cells nucleus through a complex network of interacting proteins. Perturbing these pathways can change the strength or outcome of the signals, which could be used to treat or prevent a pathological response. While manipulating these networks can be achieved using a variety of methods, the ability to do so predictably over time would provide an unprecedented level of control over cell behavior and could lead to new therapeutic design and research tools in medicine and systems biology. Hence, we propose a practical computational framework to aid in the design of experimental perturbations to force cell signaling dynamics to follow a predefined response. Our approach represents a novel merger of model-based control and information theory to blend the predictions from multiple mathematical models into a meaningful compromise solution. We verify through simulation and experimentation that this solution produces excellent agreement between the cell readouts and several predefined trajectories, even in the presence of significant modeling uncertainty and without measurement feedback. By combining elements of information and control theory, our approach will help advance the best practices in model-based control applications for medicine.</div>','<b>Title: </b>Oscillatory Protein Expression Dynamics Endows Stem Cells with Robust Differentiation Potential<br><b>Year: </b>2011<br><b>Author: </b> N. Suzuki,  C. Furusawa and  K. Kaneko<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0027232"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>The lack of understanding of stem cell differentiation and proliferation is a fundamental problem in developmental biology. Although gene regulatory networks GRNs for stem cell differentiation have been partially identified, the nature of differentiation dynamics and their regulation leading to robust development remain unclear. Herein, using a dynamical system modeling cell approach, we performed simulations of the developmental process using all possible GRNs with a few genes, and screened GRNs that could generate cell type diversity through cell-cell interactions. We found that model stem cells that both proliferated and differentiated always exhibited oscillatory expression dynamics, and the differentiation frequency of such stem cells was regulated, resulting in a robust number distribution. Moreover, we uncovered the common regulatory motifs for stem cell differentiation, in which a combination of regulatory motifs that generated oscillatory expression dynamics and stabilized distinct cellular states played an essential role. These findings may explain the recently observed heterogeneity and dynamic equilibrium in cellular states of stem cells, and can be used to predict regulatory networks responsible for differentiation in stem cell systems.</div>','<b>Title: </b>A spectrum of modularity in multi-functional gene circuits<br><b>Year: </b>2017<br><b>Author: </b> A. Jimenez,  J. Cotterell,  A. Munteanu and  J. Sharpe<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.15252/msb.20167347"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MOLECULAR SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>regulatory network, systems biology, systems biological, network dynamics, gene expression, biochemical networks, raft proteins, biological networks, gene circuits, gene regulatory, signalling network, lipid raft, nuclear lamina, protein domain, networks systems, biological approaches, expression pattern, protein interaction, cell differentiation, network motifs<div style="font-size:11px;"><br><b>Abstract: </b><em>A major challenge in systems biology is to understand the relationship between a circuits structure and its function, but how is this relationship affected if the circuit must perform multiple distinct functions within the same organism In particular, to what extent do multi-functional circuits contain modules which reflect the different functions Here, we computationally survey a range of bifunctional circuits which show no simple structural modularity: They can switch between two qualitatively distinct functions, while both functions depend on all genes of the circuit. Our analysis reveals two distinct classes: hybrid circuits which overlay two simpler mono-functional sub-circuits within their circuitry, and emergent circuits, which do not. In this second class, the bi-functionality emerges from more complex designs which are not fully decomposable into distinct modules and are consequently less intuitive to predict or understand. These non-intuitive emergent circuits are just as robust as their hybrid counterparts, and we therefore suggest that the common bias toward studying modular systems may hinder our understanding of real biological circuits.</div>','<b>Title: </b>From Understanding the Development Landscape of the Canonical Fate-Switch Pair to Constructing a Dynamic Landscape for Two-Step Neural Differentiation<br><b>Year: </b>2012<br><b>Author: </b> Xj. Qiu,  Ss. Ding and  Tl. Shi<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0049271"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>gene regulatory, regulatory network, systems biology, cell differentiation, cell fate, model systems, potential landscape, network grn, dynamical systems, melanocyte differentiation, network models, gene expression, network dynamics, differentiation genes, dynamical models, dynamical networks, cellular differentiation, stem cells, attractor network, developmental gene<div style="font-size:11px;"><br><b>Abstract: </b><em>Recent progress in stem cell biology, notably cell fate conversion, calls for novel theoretical understanding for cell differentiation. The existing qualitative concept of Waddingtons "epigenetic landscape" has attracted particular attention because it captures subsequent fate decision points, thus manifesting the hierarchical "tree-like" nature of cell fate diversification. Here, we generalized a recent work and explored such a developmental landscape for a two-gene fate decision circuit by integrating the underlying probability landscapes with different parameters corresponding to distinct developmental stages. The change of entropy production rate along the parameter changes indicates which parameter changes can represent a normal developmental process while other parameters change can not. The transdifferentiation paths over the landscape under certain conditions reveal the possibility of a direct and reversible phenotypic conversion. As the intensity of noise increases, we found that the landscape becomes flatter and the dominant paths more straight, implying the importance of biological noise processing mechanism in development and reprogramming. We further extended the landscape of the one-step fate decision to that for two-step decisions in central nervous system CNS differentiation. A minimal network and dynamic model for CNS differentiation was firstly constructed where two three-gene motifs are coupled. We then implemented the SDEs Stochastic Differentiation Equations simulation for the validity of the network and model. By integrating the two landscapes for the two switch gene pairs, we constructed the two-step development landscape for CNS differentiation. Our work provides new insights into cellular differentiation and important clues for better reprogramming strategies.</div>','<b>Title: </b>A systematic molecular circuit design method for gene networks under biochemical time delays and molecular noises<br><b>Year: </b>2008<br><b>Author: </b> Bs. Chen and  Yt. Chang<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-2-103"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>gene networks, synthetic gene, synthetic biology, regulatory networks, design method, robust synthetic, nonlinear stochastic, molecular noises, gene regulatory, interaction networks, external disturbances, robust stability, robust gene, host cell, gene circuit, cellular network, design specifications, design procedure, signal transduction, parameter fluctuations<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Gene networks in nanoscale are of nonlinear stochastic process. Time delays are common and substantial in these biochemical processes due to gene transcription, translation, posttranslation protein modification and diffusion. Molecular noises in gene networks come from intrinsic fluctuations, transmitted noise from upstream genes, and the global noise affecting all genes. Knowledge of molecular noise filtering and biochemical process delay compensation in gene networks is crucial to understand the signal processing in gene networks and the design of noise-tolerant and delay-robust gene circuits for synthetic biology.Results: A nonlinear stochastic dynamic model with multiple time delays is proposed for describing a gene network under process delays, intrinsic molecular fluctuations, and extrinsic molecular noises. Then, the stochastic biochemical processing scheme of gene regulatory networks for attenuating these molecular noises and compensating process delays is investigated from the nonlinear signal processing perspective. In order to improve the robust stability for delay toleration and noise filtering, a robust gene circuit for nonlinear stochastic time-delay gene networks is engineered based on the nonlinear robust H-infinity stochastic filtering scheme. Further, in order to avoid solving these complicated noise-tolerant and delay-robust design problems, based on Takagi-Sugeno T-S fuzzy time-delay model and linear matrix inequalities LMIs technique, a systematic gene circuit design method is proposed to simplify the design procedure.Conclusion: The proposed gene circuit design method has much potential for application to systems biology, synthetic biology and drug design when a gene regulatory network has to be designed for improving its robust stability and filtering ability of disease-perturbed gene network or when a synthetic gene network needs to perform robustly under process delays and molecular noises.</div>','<b>Title: </b>Comparative Genomics of Bacillus thuringiensis Reveals a Path to Specialized Exploitation of Multiple Invertebrate Hosts<br><b>Year: </b>2017<br><b>Author: </b> Js. Zheng,  Ql. Gao,  Ll. Liu,  Hl. Liu,  Yy. Wang,  Dh. Peng,  Lf. Ruan,  B. Raymond and  M. Sun<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1128/mBio.00822-17"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MBIO<br><b>Cluster-Keywords: </b><em>gene expression, systems biology, metabolic network, network model, network analysis, dynamic model, signaling pathways, signaling network, regulatory network, dynamic network, biology approach, candidate genes, gene network, gene regulatory, biology analysis, mathematical model, network dynamic, biology network, metabolic pathways, pathway analysis<div style="font-size:11px;"><br><b>Abstract: </b><em>Understanding the genetic basis of host shifts is a key genomic question for pathogen and parasite biology. The Bacillus cereus group, which encompasses Bacillus thuringiensis and Bacillus anthracis, contains pathogens that can infect insects, nematodes, and vertebrates. Since the target range of the essential virulence factors Cry toxins and many isolates is well known, this group presents a powerful system for investigating how pathogens can diversify and adapt to phylogenetically distant hosts. Specialization to exploit insects occurs at the level of the major clade and is associated with substantial changes in the core genome, and host switching between insect orders has occurred repeatedly within subclades. The transfer of plasmids with linked cry genes may account for much of the adaptation to particular insect orders, and network analysis implies that host specialization has produced strong associations between key toxin genes with similar targets. Analysis of the distribution of plasmid minireplicons shows that plasmids with orf156 and orf157, which carry genes encoding toxins against Lepidoptera or Diptera, were contained only by B. thuringiensis in the specialized insect clade clade 2, indicating that tight genome/plasmid associations have been important in adaptation to invertebrate hosts. Moreover, the accumulation of multiple virulence factors on transposable elements suggests that cotransfer of diverse virulence factors is advantageous in terms of expanding the insecticidal spectrum, overcoming insect resistance, or through gains in pathogenicity via synergistic interactions between toxins.IMPORTANCE Population genomics have provided many new insights into the formation, evolution, and dynamics of bacterial pathogens of humans and other higher animals, but these pathogens usually have very narrow host ranges. As a pathogen of insects and nematodes, Bacillus thuringiensis, which produces toxins showing toxicity to many orders of insects and other invertebrates, can be used as a model to study the evolution of pathogens with wide host ranges. Phylogenomic analysis revealed that host specialization and switching occur at the level of the major clade and subclade, respectively. A toxin gene co-occurrence network indicates that multiple toxins with similar targets were accumulated by the same cell in the whole species. This accumulation may be one of the strategies that B. thuringiensis has used to fight against host resistance. This kind of formation and evolution of pathogens represents a different path used against multiple invertebrate hosts from that used against higher animals.</div>','<b>Title: </b>Stochastic simulation and analysis of biomolecular reaction networks<br><b>Year: </b>2009<br><b>Author: </b> Jm. Frazier,  Y. Chushak and  B. Foy<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-3-64"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: In recent years, several stochastic simulation algorithms have been developed to generate Monte Carlo trajectories that describe the time evolution of the behavior of biomolecular reaction networks. However, the effects of various stochastic simulation and data analysis conditions on the observed dynamics of complex biomolecular reaction networks have not recieved much attention. In order to investigate these issues, we employed a a software package developed in out group, called Biomolecular Network Simulator BNS, to simulate and analyze the behavior of such systems. The behavior of a hypothetical two gene in vitro transcription-translation reaction network is investigated using the Gillespie exact stochastic algorithm to illustrate some of the factors that influence the analysis and interpretation of these data.Results: Specific issues affecting the analysis and interpretation of simulation data are investigated, including: 1 the effect of time interval on data presentation and time-weighted averaging of molecule numbers, 2 effect of time averaging interval on reaction rate analysis, 3 effect of number of simulations on precision of model predictions, and 4 implications of stochastic simulations on optimization procedures.Conclusion: The two main factors affecting the analysis of stochastic simulations are: 1 the selection of time intervals to compute or average state variables and 2 the number of simulations generated to evaluate the system behavior.</div>','<b>Title: </b>A cell size- and cell cycle-aware stochastic model for predicting time-dynamic gene network activity in individual cells<br><b>Year: </b>2015<br><b>Author: </b> Rj. Song,  Wl. Peng,  P. Liu and  M. Acar<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/s12918-015-0240-5"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>gene expression, gene network, stochastic model, gene regulatory, systems biology, expression systems, gene regulation, network activity, expression dynamics, noise control, regulatory network, deterministic models, network components, stochastic modeling, regulation system, escherichia coli, regulatory system, synthetic gene, regulatory genes, gene<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Despite the development of various modeling approaches to predict gene network activity, a time dynamic stochastic model taking into account real-time changes in cell volume and cell cycle stages is still missing.Results: Here we present a stochastic single-cell model that can be applied to any eukaryotic gene network with any number of components. The model tracks changes in cell volume, DNA replication, and cell division, and dynamically adjusts rates of stochastic reactions based on this information. By tracking cell division, the model can maintain cell lineage information, allowing the researcher to trace the descendants of any single cell and therefore study cell lineage effects. To test the predictive power of our model, we applied it to the canonical galactose network of the yeast Saccharomyces cerevisiae. Using a minimal set of free parameters and across several galactose induction conditions, the model effectively captured several details of the experimentally-obtained single-cell network activity levels as well as phenotypic switching rates.Conclusion: Our model can readily be customized to model any gene network in any of the commonly used cells types, offering a novel and user-friendly stochastic modeling capability to the systems biology field.</div>','<b>Title: </b>The switching dynamics of the bacterial flagellar motor<br><b>Year: </b>2009<br><b>Author: </b> Sb. Van Albada,  S. Tanase-Nicola and  Pr. Ten Wolde<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1038/msb.2009.74"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MOLECULAR SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>systems biology, biological systems, system dynamics, molecular systems, experimental design, network model, model system, model parameters, experiment design, kinetic parameters, perfect adaptation, experimental data, molecular mechanisms, homeostatic mechanisms, switching dynamics, biological mechanisms, control systems, chemotaxis network, decentralized control, feedback control<div style="font-size:11px;"><br><b>Abstract: </b><em>Many bacteria are propelled by flagellar motors that stochastically switch between the clockwise and counterclockwise rotation direction. Although the switching dynamics is one of their most important characteristics, the mechanisms that control it are poorly understood. We present a statistical-mechanical model of the bacterial flagellar motor. At its heart is the assumption that the rotor protein complex, which is connected to the flagellum, can exist in two conformational states and that switching between these states depends on the interactions with the stator proteins, which drive the rotor. This couples switching to rotation, making the switch sensitive to torque and speed. Another key element is that after a switch, it takes time for the load to build up, due to conformational transitions of the flagellum. This slow relaxation dynamics of the filament leads, in combination with the load dependence of the switching frequency, to a characteristic switching time, as recently observed. Hence, our model predicts that the switching dynamics is not only controlled by the chemotaxis-signaling network, but also by mechanical feedback of the flagellum. Molecular Systems Biology 5: 316; published online 13 October 2009; doi:10.1038/msb.2009.74</div>','<b>Title: </b>Dynamics of T-cell antagonism: enhanced viral diversity and survival<br><b>Year: </b>1998<br><b>Author: </b> Nj. Burroughs and  Da. Rand<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1098/rspb.1998.0327"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PROCEEDINGS OF THE ROYAL SOCIETY B-BIOLOGICAL SCIENCES<br><b>Cluster-Keywords: </b><em>systems biology, dynamic network, early differentiation, complex diseases, stem cells, erythroid cells, system dynamics, network biomarkers, immune system, tcm syndrome, dnb genes, gene expression, dnb model, stem cell, expression dynamics, model system, mathematical model, model dynamics, dynamic model, sensitivity analysis<div style="font-size:11px;"><br><b>Abstract: </b><em>In rapidly evolving viruses the detection of virally infected cells can possibly be subverted by the production of altered peptides. These are peptides with single amino acid changes that can dramatically change T-cell responses, e.g. a loss of cytotoxic activity. They are still recognized by the T cell, but the signals required for effector function are only partially delivered. Thus, altered peptide presenting cells can act as decoy targets for specific immune responses. The existence of altered peptides in vivo has been demonstrated in hepatitis B and HIV. Using a mathematical model we address the question of how these altered peptides can affect the virus-immune system dynamics, and demonstrate that virus survival is enhanced. If the mutation rate of the virus is sufficient, one observes complex dynamics in which the antagonism acts so as to maintain the viral diversity, possibly leading to the development of a mutually antagonistic network or a continual turnover of escape mutants. In either case the pathogen is able to outrun the immune system. Indeed, sometimes the enhancement is so great that a virus that would normally be cleared by the immune system is able to outrun it.</div>','<b>Title: </b>Use of Mathematical Modeling to Compare and Predict Hemodynamic Effects Between Hybrid and Surgical Norwood Palliations for Hypoplastic Left Heart Syndrome<br><b>Year: </b>2011<br><b>Author: </b> Ty. Hsia,  D. Cosentino,  C. Corsini,  G. Pennati,  G. Dubini and  F. Migliavacca<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1161/CIRCULATIONAHA.110.010769"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>CIRCULATION<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Background-Combining bilateral pulmonary artery banding with arterial duct stenting, the hybrid approach achieves stage 1 palliation for hypoplastic left heart syndrome with different flow characteristics than those after the surgical Norwood procedures. Accordingly, we used computational modeling to assess some of these differences, including influence on systemic and cerebral oxygen deliveries.Methods and Results-A 3-dimensional computational model of hybrid palliation was developed by the finite volume method, along with models of the Norwood operation with a modified Blalock-Tausig or right ventricle-to-pulmonary artery shunt. Hybrid circulation was modeled with a 7-mm ductal stent and bilateral pulmonary artery banding to a 2-mm diameter. A 3.5-mm conduit was used in the Blalock-Tausig shunt model, whereas a 5-mm conduit was used in the right ventricle-to-pulmonary artery shunt model. Coupled to all the models was an identical hydraulic network that described the entire circulatory system based on pre-stage 2 hemodynamics. This clinically validated multiscale approach predicts flow dynamics, as well as global cardiac output, mixed venous oxygen saturation, and systemic and cerebral oxygen delivery. Compared with either of the Norwood models, the hybrid palliation had higher pulmonary-to-systemic flow ratio and lower cardiac output. Total systemic oxygen delivery was markedly reduced in the hybrid palliation Blalock-Tausig shunt 591, right ventricle-to-pulmonary artery shunt 640, and hybrid 475 mL.min-1.m-2. Cerebral oxygen delivery was similarly lower in the hybrid palliation.Conclusions-These computational results suggest that the hybrid approach may provide inferior systemic and cerebral oxygen deliveries compared with either of the 2 surgical Norwood procedures before stage 2 palliation. Circulation. 2011; 124suppl 1: S204-S210.</div>','<b>Title: </b>Evidence for Network Evolution in an Arabidopsis Interactome Map<br><b>Year: </b>2011<br><b>Author: </b> P. Braun,  Ar. Carvunis,  B. Charloteaux,  M. Dreze,  Jr. Ecker,  De. Hill,  Fp. Roth,  M. Vidal,  M. Galli,  P. Balumuri,  V. Bautista,  Jd. Chesnut,  Rc. Kim,  C. De Los Reyes,  P. Gilles,  Cj. Kim,  U. Matrubutham,  J. Mirchandani,  E. Olivares,  S. Patnaik,  R. Quan,  G. Ramaswamy,  P. Shinn,  Gm. Swamilingiah,  S. Wu,  Jr. Ecker,  M. Dreze,  D. Byrdsong,  A. Dricot,  M. Duarte,  F. Gebreab,  Bj. Gutierrez,  A. Macwilliams,  D. Monachello,  Ms. Mukhtar,  Mm. Poulin,  P. Reichert,  V. Romero,  S. Tam,  S. Waaijers,  Em. Weiner,  M. Vidal,  De. Hill,  P. Braun,  M. Galli,  Ar. Carvunis,  Me. Cusick,  M. Dreze,  V. Romero,  Fp. Roth,  M. Tasan,  J. Yazaki,  P. Braun,  Jr. Ecker,  Ar. Carvunis,  Yy. Ahn,  Al. Barabasi,  B. Charloteaux,  Hm. Chen,  Me. Cusick,  Jl. Dangl,  M. Dreze,  Jr. Ecker,  Cy. Fan,  Lt. Gai,  M. Galli,  G. Ghoshal,  T. Hao,  De. Hill,  C. Lurin,  T. Milenkovic,  J. Moore,  Ms. Mukhtar,  Sj. Pevzner,  N. Przulj,  S. Rabello,  Ea. Rietman,  T. Rolland,  Fp. Roth,  B. Santhanam,  Rj. Schmitz,  W. Spooner,  J. Stein,  M. Tasan,  J. Vandenhaute,  D. Ware,  P. Braun,  M. Vidal,  P. Braun,  Ar. Carvunis,  B. Charloteaux,  M. Dreze,  M. Galli and  M. Vidal<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1126/science.1203877"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SCIENCE<br><b>Cluster-Keywords: </b><em>interaction networks, proteinprotein interactions, protein interaction, network model, complex systems, systems biology, protein protein, protein complex, host network, dynamic interactions, network analysis, biological systems, networks network, cellular network, biological network, viral proteins, interactome networks, regulatory networks, immune system, systems network<div style="font-size:11px;"><br><b>Abstract: </b><em>Plants have unique features that evolved in response to their environments and ecosystems. A full account of the complex cellular networks that underlie plant-specific functions is still missing. We describe a proteome-wide binary protein-protein interaction map for the interactome network of the plant Arabidopsis thaliana containing about 6200 highly reliable interactions between about 2700 proteins. A global organization of plant biological processes emerges from community analyses of the resulting network, together with large numbers of novel hypothetical functional links between proteins and pathways. We observe a dynamic rewiring of interactions following gene duplication events, providing evidence for a model of evolution acting upon interactome networks. This and future plant interactome maps should facilitate systems approaches to better understand plant biology and improve crops.</div>','<b>Title: </b>A systematic design method for robust synthetic biology to satisfy design specifications<br><b>Year: </b>2009<br><b>Author: </b> Bs. Chen and  Ch. Wu<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-3-66"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>gene networks, synthetic gene, synthetic biology, regulatory networks, design method, robust synthetic, nonlinear stochastic, molecular noises, gene regulatory, interaction networks, external disturbances, robust stability, robust gene, host cell, gene circuit, cellular network, design specifications, design procedure, signal transduction, parameter fluctuations<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Synthetic biology is foreseen to have important applications in biotechnology and medicine, and is expected to contribute significantly to a better understanding of the functioning of complex biological systems. However, the development of synthetic gene networks is still difficult and most newly created gene networks are non-functioning due to intrinsic parameter uncertainties, external disturbances and functional variations of intra- and extra-cellular environments. The design method for a robust synthetic gene network that works properly in a host cell under these intrinsic parameter uncertainties and external disturbances is the most important topic in synthetic biology.Results: In this study, we propose a stochastic model that includes parameter fluctuations and external disturbances to mimic the dynamic behaviors of a synthetic gene network in the host cell. Then, based on this stochastic model, four design specifications are introduced to guarantee that a synthetic gene network can achieve its desired steady state behavior in spite of parameter fluctuations, external disturbances and functional variations in the host cell. We propose a systematic method to select a set of appropriate design parameters for a synthetic gene network that will satisfy these design specifications so that the intrinsic parameter fluctuations can be tolerated, the external disturbances can be efficiently filtered, and most importantly, the desired steady states can be achieved. Thus the synthetic gene network can work properly in a host cell under intrinsic parameter uncertainties, external disturbances and functional variations. Finally, a design procedure for the robust synthetic gene network is developed and a design example is given in silico to confirm the performance of the proposed method.Conclusion: Based on four design specifications, a systematic design procedure is developed for designers to engineer a robust synthetic biology network that can achieve its desired steady state behavior under parameter fluctuations, external disturbances and functional variations in the host cell. Therefore, the proposed systematic design method has good potential for the robust synthetic gene network design.</div>','<b>Title: </b>To aggregate or not Capturing the spatio-temporal complexity of the thermal regime<br><b>Year: </b>2016<br><b>Author: </b> Mp. Turschwell,  Ee. Peterson,  Sr. Balcombe and  F. Sheldon<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.ecolind.2016.02.014"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ECOLOGICAL INDICATORS<br><b>Cluster-Keywords: </b><em>interaction network, actin filament, network model, filament network, network structures, actin network, protein interaction, actin dynamics, network, membrane protein, network dynamics, model dynamic, cell, model, network approach, protein complex, computational model, actin, adhesion dynamics, cell dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>Freshwater stream systems are under immense pressure from various anthropogenic impacts, including climate change. Stream systems are increasingly being altered by changes to the magnitude, timing, frequency, and duration of their thermal regimes, which will have profound impacts on the life-history dynamics of resident biota within their home range. Although temperature regimes have a significant influence on the biology of instream fauna, large spatio-temporal temperature datasets are often reduced to a single metric at discrete locations and used to describe the thermal regime of a system; potentially leading to a significant loss of information crucial to stream management. Models are often used to extrapolate these metrics to unsampled locations, but it is unclear whether predicting actual daily temperatures or an aggregated metric of the temperature regime best describes the complexity of the thermal regime. We fit spatial statistical stream-network models SSNMs, random forest and non-spatial linear models to stream temperature data from the Upper Condamine River in QLD, Australia and used them to semi-continuously predict metrics describing the magnitude, duration, and frequency of the thermal regime through space and time. We compared both daily and aggregated temperature metrics and found that SSNMs always had more predictive ability than the random forest models, but both models outperformed the non-spatial linear model. For metrics describing thermal magnitude and duration, aggregated predictions were most accurate, while metrics describing the frequency of heating events were better represented by metrics based on daily predictions generated using a SSNM. A more comprehensive representation of the spatio-temporal thermal regime allows researchers to explore new spatio-temporally explicit questions about the thermal regime. It also provides the information needed to generate a suite of ecologically meaningful metrics capturing multiple aspects of the thermal regime, which will increase our scientific understanding of how organisms respond to thermal cues and provide much-needed information for more effective management actions. C 2016 Elsevier Ltd. All rights reserved.</div>','<b>Title: </b>Adapting machine-learning algorithms to design gene circuits<br><b>Year: </b>2019<br><b>Author: </b> Tw. Hiscock<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/s12859-019-2788-3"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>regulatory network, systems biology, systems biological, network dynamics, gene expression, biochemical networks, raft proteins, biological networks, gene circuits, gene regulatory, signalling network, lipid raft, nuclear lamina, protein domain, networks systems, biological approaches, expression pattern, protein interaction, cell differentiation, network motifs<div style="font-size:11px;"><br><b>Abstract: </b><em>BackgroundGene circuits are important in many aspects of biology, and perform a wide variety of different functions. For example, some circuits oscillate e.g. the cell cycle, some are bistable e.g. as cells differentiate, some respond sharply to environmental signals e.g. ultrasensitivity, and some pattern multicellular tissues e.g. Turings model. Often, one starts from a given circuit, and using simulations, asks what functions it can perform. Here we want to do the opposite: starting from a prescribed function, can we find a circuit that executes this function Whilst simple in principle, this task is challenging from a computational perspective, since gene circuit models are complex systems with many parameters. In this work, we adapted machine-learning algorithms to significantly accelerate gene circuit discovery.ResultsWe use gradient-descent optimization algorithms from machine learning to rapidly screen and design gene circuits. With this approach, we found that we could rapidly design circuits capable of executing a range of different functions, including those that: 1 recapitulate important in vivo phenomena, such as oscillators, and 2 perform complex tasks for synthetic biology, such as counting noisy biological events.ConclusionsOur computational pipeline will facilitate the systematic study of natural circuits in a range of contexts, and allow the automatic design of circuits for synthetic biology. Our method can be readily applied to biological networks of any type and size, and is provided as an open-source and easy-to-use python module, GeneNet.</div>','<b>Title: </b>Transcriptional network dynamics in macrophage activation<br><b>Year: </b>2006<br><b>Author: </b> R. Nilsson,  Vb. Bajic,  H. Suzuki,  D. Di Bernardo,  J. Bjorkegren,  S. Katayama,  Jf. Reid,  Mj. Sweet,  M. Gariboldi,  P. Carninci,  Y. Hayashizaki,  Da. Hume,  J. Tegner and  T. Ravasi<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.ygeno.2006.03.022"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>GENOMICS<br><b>Cluster-Keywords: </b><em>gene regulatory, gene expression, regulatory networks, expression data, regulatory pathway, regulatory genes, network dynamics, microarray data, genetic networks, transcription factor, models network, target gene, upstream regulatory, network topology, pathway network, gene functional, expression profile, transcriptional network, dynamic model, transcriptional regulatory<div style="font-size:11px;"><br><b>Abstract: </b><em>Transcriptional regulatory networks govern cell differentiation and the cellular response to external stimuli. However, mammalian model systems have not yet been accessible for network analysis. Here, we present a genome-wide network analysis of the transcriptional regulation underlying the mouse macrophage response to bacterial lipopolysaccharide LPS. Key to uncovering the network structure is our combination of time-series cap analysis of gene expression with in silico prediction of transcription factor binding sites. By integrating microarray and qPCR time-series expression data with a promoter analysis, we find dynamic subnetworks that describe how signaling pathways change dynamically during the progress of the macrophage LPS response, thus defining regulatory modules characteristic of the inflammatory response. In particular, our integrative analysis enabled us to suggest novel roles for the transcription factors ATF-3 and NRF-2 during the inflammatory response. We believe that our system approach presented here is applicable to understanding cellular differentiation in higher eukaryotes. c 2006 Elsevier Inc. All rights reserved.</div>','<b>Title: </b>ENZYMES AS MOLECULAR AUTOMATA - A REFLECTION ON SOME NUMERICAL AND PHILOSOPHICAL ASPECTS OF THE HYPOTHESIS<br><b>Year: </b>1992<br><b>Author: </b> Pc. Marijuan and  J. Westley<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/0303-26479290050-9"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOSYSTEMS<br><b>Cluster-Keywords: </b><em>information systems, gene expression, gene network, systems theory, system models, systems biology, systems information, network model, genetic network, mathematical models, dynamic systems, dynamic network, systemic model, network topology, information mechanisms, complex systems, systems complex, relational models, models main, information model<div style="font-size:11px;"><br><b>Abstract: </b><em>Enzymes, by means of their properties of specific recognition and allosteric modulation, are able to integrate many separate processes into systemic units with coherent functions; in a sense, they have to be considered as the true organizers of the cytoplasmic processes. In this respect, the present article describes a simple model, based on binary variables and automata theory, which simulates the basic regulatory performance of the modulated enzyme. The model admits a variety of modifications and improvements; it also suggests some original lines of thought on which to reflect about the organization and collective phenomena of the networks of enzymes. In discussing the connection of this molecular automata hypothesis with other areas of present-day theoretical biology, a fertile panorama of initiatives appear. A special partnership between Information Science computation and Biology is developing.</div>','<b>Title: </b>The yeast PHO5 promoter: from single locus to systems biology of a paradigm for gene regulation through chromatin<br><b>Year: </b>2014<br><b>Author: </b> P. Korber and  S. Barbaric<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1093/nar/gku784"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>NUCLEIC ACIDS RESEARCH<br><b>Cluster-Keywords: </b><em>gene expression, gene network, stochastic model, gene regulatory, systems biology, expression systems, gene regulation, network activity, expression dynamics, noise control, regulatory network, deterministic models, network components, stochastic modeling, regulation system, escherichia coli, regulatory system, synthetic gene, regulatory genes, gene<div style="font-size:11px;"><br><b>Abstract: </b><em>Chromatin dynamics crucially contributes to gene regulation. Studies of the yeast PHO5 promoter were key to establish this nowadays accepted view and continuously provide mechanistic insight in chromatin remodeling and promoter regulation, both on single locus as well as on systems level. The PHO5 promoter is a context independent chromatin switch module where in the repressed state positioned nucleosomes occlude transcription factor sites such that nucleosome remodeling is a prerequisite for and not consequence of induced gene transcription. This massive chromatin transition from positioned nucleosomes to an extensive hypersensitive site, together with respective transitions at the co-regulated PHO8 and PHO84 promoters, became a prime model for dissecting how remodelers, histone modifiers and chaperones co-operate in nucleosome remodeling upon gene induction. This revealed a surprisingly complex cofactor network at the PHO5 promoter, including five remodeler ATPases SWI/SNF, RSC, INO80, Isw1, Chd1, and demonstrated for the first time histone eviction in trans as remodeling mode in vivo. Recently, the PHO5 promoter and the whole PHO regulon were harnessed for quantitative analyses and computational modeling of remodeling, transcription factor binding and promoter input-output relations such that this rewarding single-locus model becomes a paradigm also for theoretical and systems approaches to gene regulatory networks.</div>','<b>Title: </b>A systems biology approach to construct the gene regulatory network of systemic inflammation via microarray and databases mining<br><b>Year: </b>2008<br><b>Author: </b> Bs. Chen,  Sk. Yang,  Cy. Lan and  Yj. Chuang<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1755-8794-1-46"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC MEDICAL GENOMICS<br><b>Cluster-Keywords: </b><em>systems biology, regulatory network, interaction networks, network model, gene regulatory, network, network biology, networks, gene network, response networks, network dynamics, dynamic model, cell networks, gene expression, biological networks, system dynamics, dynamic network, model, biological systems, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Inflammation is a hallmark of many human diseases. Elucidating the mechanisms underlying systemic inflammation has long been an important topic in basic and clinical research. When primary pathogenetic events remains unclear due to its immense complexity, construction and analysis of the gene regulatory network of inflammation at times becomes the best way to understand the detrimental effects of disease. However, it is difficult to recognize and evaluate relevant biological processes from the huge quantities of experimental data. It is hence appealing to find an algorithm which can generate a gene regulatory network of systemic inflammation from high-throughput genomic studies of human diseases. Such network will be essential for us to extract valuable information from the complex and chaotic network under diseased conditions.Results: In this study, we construct a gene regulatory network of inflammation using data extracted from the Ensembl and JASPAR databases. We also integrate and apply a number of systematic algorithms like cross correlation threshold, maximum likelihood estimation method and Akaike Information Criterion AIC on time-lapsed microarray data to refine the genome-wide transcriptional regulatory network in response to bacterial endotoxins in the context of dynamic activated genes, which are regulated by transcription factors TFs such as NF-kappa B. This systematic approach is used to investigate the stochastic interaction represented by the dynamic leukocyte gene expression profiles of human subject exposed to an inflammatory stimulus bacterial endotoxin. Based on the kinetic parameters of the dynamic gene regulatory network, we identify important properties such as susceptibility to infection of the immune system, which may be useful for translational research. Finally, robustness of the inflammatory gene network is also inferred by analyzing the hubs and "weak ties" structures of the gene network.Conclusion: In this study, Data mining and dynamic network analyses were integrated to examine the gene regulatory network in the inflammatory response system. Compared with previous methodologies reported in the literatures, the proposed gene network perturbation method has shown a great improvement in analyzing the systemic inflammation.</div>','<b>Title: </b>Sufficient conditions for reachability in automata networks with priorities<br><b>Year: </b>2015<br><b>Author: </b> M. Folschette,  L. Pauleve,  M. Magnin and  O. Roux<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.tcs.2015.08.040"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>THEORETICAL COMPUTER SCIENCE<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>In this paper, we develop a framework for an efficient under-approximation of the dynamics of Asynchronous Automata Networks AANs. AAN is an Automata Network with synchronised transitions between automata, where each transition changes the local state of exactly one automaton but any number of synchronising local states are allowed. The work we propose here is based on static analysis by abstract interpretation, which allows to prove that reaching a state with a given property is possible, without the same computational cost of usual model checkers: the complexity is polynomial with the total number of local states and exponential with the number of local states within a single automaton. Furthermore, we address AANs with classes of priorities, and give an encoding into AANs without priorities, thus extending the application range of our under-approximation. Finally, we illustrate our method for the model checking of large-scale biological networks. C 2015 Elsevier B.V. All rights reserved.</div>','<b>Title: </b>LASSIM-A network inference toolbox for genome-wide mechanistic modeling<br><b>Year: </b>2017<br><b>Author: </b> R. Magnusson,  Gp. Mariotti,  M. Kopsen,  W. Lovfors,  Dr. Gawel,  R. Jornsten,  J. Linde,  T. Nordling,  E. Nyman,  S. Schulze,  Ce. Nestor,  H. Zhang,  G. Cedersund,  M. Benson,  A. Tjarnberg and  M. Gustafsson<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pcbi.1005608"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS COMPUTATIONAL BIOLOGY<br><b>Cluster-Keywords: </b><em>regulatory networks, gene regulatory, gene expression, network inference, expression data, network structure, gene networks, networks grns, regulatory genes, boolean networks, bayesian network, network reconstruction, networks gene, network dynamics, gene regulation, inference method, systems biology, time series, infer gene, network<div style="font-size:11px;"><br><b>Abstract: </b><em>Recent technological advancements have made time-resolved, quantitative, multi-omics data available for many model systems, which could be integrated for systems pharmacokinetic use. Here, we present large-scale simulation modeling LASSIM, which is a novel mathematical tool for performing large-scale inference using mechanistically defined ordinary differential equations ODE for gene regulatory networks GRNs. LASSIM integrates structural knowledge about regulatory interactions and non-linear equations with multiple steady state and dynamic response expression datasets. The rationale behind LASSIM is that biological GRNs can be simplified using a limited subset of core genes that are assumed to regulate all other gene transcription events in the network. The LASSIM method is implemented as a general-purpose toolbox using the PyGMO Python package to make the most of multicore computers and high performance clusters, and is available at https://gitlab.com/Gustafsson-lab/lassim. As a method, LASSIM works in two steps, where it first infers a non-linear ODE system of the pre-specified core gene expression. Second, LASSIM in parallel optimizes the parameters that model the regulation of peripheral genes by core system genes. We showed the usefulness of this method by applying LASSIM to infer a large-scale non-linear model of naive Th2 cell differentiation, made possible by integrating Th2 specific bindings, time-series together with six public and six novel siRNA-mediated knock-down experiments. ChIP-seq showed significant overlap for all tested transcription factors. Next, we performed novel time-series measurements of total T-cells during differentiation towards Th2 and verified that our LASSIM model could monitor those data significantly better than comparable models that used the same Th2 bindings. In summary, the LASSIM toolbox opens the door to a new type of model-based data analysis that combines the strengths of reliable mechanistic models with truly systems-level data. We demonstrate the power of this approach by inferring a mechanistically motivated, genome-wide model of the Th2 transcription regulatory system, which plays an important role in several immune related diseases.</div>','<b>Title: </b>Reduction of Qualitative Models of Biological Networks for Transient Dynamics Analysis<br><b>Year: </b>2018<br><b>Author: </b> L. Pauleve<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1109/TCBB.2017.2749225"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IEEE-ACM TRANSACTIONS ON COMPUTATIONAL BIOLOGY AND BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Qualitative models of dynamics of signalling pathways and gene regulatory networks allow for the capturing of temporal properties of biological networks while requiring few parameters. However, these discrete models typically suffer from the so-called state space explosion problem which makes the formal assessment of their potential behaviors very challenging. In this paper, we describe a method to reduce a qualitative model for enhancing the tractability of analysis of transient reachability properties. The reduction does not change the dimension of the model, but instead limits its degree of freedom, therefore reducing the set of states and transitions to consider. We rely on a transition-centered specification of qualitative models by the mean of automata networks. Our framework encompasses the usual asynchronous Boolean and multi-valued network, as well as 1-bounded Petri nets. Applied to different large-scale biological networks from the litterature, we show that the reduction can lead to a drastic improvement for the scalability of verification methods.</div>','<b>Title: </b>Short-term information processing, long-term responses: Insights by mathematical modeling of signal transduction<br><b>Year: </b>2012<br><b>Author: </b> A. Schneider,  U. Klingmuller and  M. Schilling<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1002/bies.201100172"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOESSAYS<br><b>Cluster-Keywords: </b><em>stem cell, stem cells, systems biology, mathematical models, cell biology, network model, signaling pathways, cell fate, computational models, cancer cells, gene expression, network dynamics, embryonic stem, signaling network, prostate cancer, mathematical modeling, psychological stress, apoptosis signaling, computational systems, molecular mechanisms<div style="font-size:11px;"><br><b>Abstract: </b><em>How do cells interpret information from their environment and translate it into specific cell fate decisions We propose that cell fate is already encoded in early signaling events and thus can be predicted from defined signal properties. Specifically, we hypothesize that the time integral of activated key signaling molecules can be correlated to cellular behavior such as proliferation or differentiation. The identification of these decisive key signal mediators and their connection to cell fate is facilitated by mathematical modeling. A possible mechanistic linkage between signaling dynamics and cellular function is the directed control of gene regulatory networks by defined signals. Targeted experiments in combination with mathematical modeling can increase our understanding of how cells process information and realize distinct cell fates.</div>','<b>Title: </b>Analysis of the zebrafish proteome during embryonic development<br><b>Year: </b>2008<br><b>Author: </b> Mb. Lucitt,  Ts. Price,  A. Pizarro,  W. Wu,  Ak. Yocum,  C. Seiler,  Ma. Pack,  Ia. Blair,  Ga. Fitzgerald and  T. Grosser<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1074/mcp.M700382-MCP200"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MOLECULAR  CELLULAR PROTEOMICS<br><b>Cluster-Keywords: </b><em>gene expression, systems biology, metabolic network, network model, network analysis, dynamic model, signaling pathways, signaling network, regulatory network, dynamic network, biology approach, candidate genes, gene network, gene regulatory, biology analysis, mathematical model, network dynamic, biology network, metabolic pathways, pathway analysis<div style="font-size:11px;"><br><b>Abstract: </b><em>The model organism zebrafish Danio rerio is particularly amenable to studies deciphering regulatory genetic networks in vertebrate development, biology, and pharmacology. Unraveling the functional dynamics of such networks requires precise quantitation of protein expression during organismal growth, which is incrementally challenging with progressive complexity of the systems. In an approach toward such quantitative studies of dynamic network behavior, we applied mass spectrometric methodology and rigorous statistical analysis to create comprehensive, high quality profiles of proteins expressed at two stages of zebrafish development. Proteins of embryos 72 and 120 h postfertilization hpf were isolated and analyzed both by two-dimensional 2D LC followed by ESI-MS/MS and by 2D PAGE followed by MALDI-TOF/TOF protein identification. We detected 1384 proteins from 327,906 peptide sequence identifications at 72 and 120 hpf with false identification rates of less than 1 using 2D LC-ESI-MS/MS. These included only similar to 30 of proteins that were identified by 2D PAGE-MALDI-TOF/TOF. Roughly 10 of all detected proteins were derived from hypothetical or predicted gene models or were entirely unannotated. Comparison of proteins expression by 2D DIGE revealed that proteins involved in energy production and transcription/translation were relatively more abundant at 72 hpf consistent with faster synthesis of cellular proteins during organismal growth at this time compared with 120 hpf. The data are accessible in a database that links protein identifications to existing resources including the Zebrafish Information Network database. This new resource should facilitate the selection of candidate proteins for targeted quantitation and refine systematic genetic network analysis in vertebrate development and biology.</div>','<b>Title: </b>Workflow for generating competing hypothesis from models with parameter uncertainty<br><b>Year: </b>2011<br><b>Author: </b> D. Gomez-Cabrero,  A. Compte and  J. Tegner<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1098/rsfs.2011.0015"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>INTERFACE FOCUS<br><b>Cluster-Keywords: </b><em>biological networks, dynamic models, biological systems, model predictions, model parameters, systems biology, mathematical models, biological models, equation models, experimental data, molecular network, interaction networks, model errors, biological interaction, molecular interactions, model, dynamic systems, biological parameters, parameter uncertainty, growth model<div style="font-size:11px;"><br><b>Abstract: </b><em>Mathematical models are increasingly used in life sciences. However, contrary to other disciplines, biological models are typically over-parametrized and loosely constrained by scarce experimental data and prior knowledge. Recent efforts on analysis of complex models have focused on isolated aspects without considering an integrated approach-ranging from model building to derivation of predictive experiments and refutation or validation of robust model behaviours. Here, we develop such an integrative workflow, a sequence of actions expanding upon current efforts with the purpose of setting the stage for a methodology facilitating an extraction of core behaviours and competing mechanistic hypothesis residing within underdetermined models. To this end, we make use of optimization search algorithms, statistical machine-learning classification techniques and cluster-based analysis of the state variables dynamics and their corresponding parameter sets. We apply the workflow to a mathematical model of fat accumulation in the arterial wall atherogenesis, a complex phenomena with limited quantitative understanding, thus leading to a model plagued with inherent uncertainty. We find that the mathematical atherogenesis model can still be understood in terms of a few key behaviours despite the large number of parameters. This result enabled us to derive distinct mechanistic predictions from the model despite the lack of confidence in the model parameters. We conclude that building integrative workflows enable investigators to embrace modelling of complex biological processes despite uncertainty in parameters.</div>','<b>Title: </b>Parallel replica dynamics method for bistable stochastic reaction networks: Simulation and sensitivity analysis<br><b>Year: </b>2017<br><b>Author: </b> T. Wang and  P. Plechac<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1063/1.5017955"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF CHEMICAL PHYSICS<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Stochastic reaction networks that exhibit bistable behavior are common in systems biology, materials science, and catalysis. Sampling of stationary distributions is crucial for understanding and characterizing the long-time dynamics of bistable stochastic dynamical systems. However, simulations are often hindered by the insufficient sampling of rare transitions between the two metastable regions. In this paper, we apply the parallel replica method for a continuous time Markov chain in order to improve sampling of the stationary distribution in bistable stochastic reaction networks. The proposed method uses parallel computing to accelerate the sampling of rare transitions. Furthermore, it can be combined with the path-space information bounds for parametric sensitivity analysis. With the proposed methodology, we study three bistable biological networks: the Schlogl model, the genetic switch network, and the enzymatic futile cycle network. We demonstrate the algorithmic speedup achieved in these numerical benchmarks. More significant acceleration is expected when multi-core or graphics processing unit computer architectures and programming tools such as CUDA are employed. Published by AIP Publishing.</div>','<b>Title: </b>Light and circadian regulation of clock components aids flexible responses to environmental signals<br><b>Year: </b>2014<br><b>Author: </b> Le. Dixon,  Sk. Hodge,  G. Van Ooijen,  C. Troein,  Oe. Akman and  Aj. Millar<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1111/nph.12853"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>NEW PHYTOLOGIST<br><b>Cluster-Keywords: </b><em>circadian clock, biological systems, network models, mathematical models, clock gene, systems biology, model selection, biological networks, network data, experimental data, model parameters, network dynamics, plant circadian, regulatory network, clock model, plant clock, complex network, clock circuit, approximate bayesian, stochastic models<div style="font-size:11px;"><br><b>Abstract: </b><em>The circadian clock measures time across a 24h period, increasing fitness by phasing biological processes to the most appropriate time of day. The interlocking feedback loop mechanism of the clock is conserved across species; however, the number of loops varies. Mathematical and computational analyses have suggested that loop complexity affects the overall flexibility of the oscillator, including its responses to entrainment signals. We used a discriminating experimental assay, at the transition between different photoperiods, in order to test this proposal in a minimal circadian network in Ostreococcus tauri and a more complex network in Arabidopsis thaliana. Transcriptional and translational reporters in O.tauri primarily tracked dawn or dusk, whereas in A.thaliana, a wider range of responses were observed, consistent with its more flexible clock. Model analysis supported the requirement for this diversity of responses among the components of the more complex network. However, these and earlier data showed that the O.tauri network retains surprising flexibility, despite its simple circuit. We found that models constructed from experimental data can show flexibility either from multiple loops and/or from multiple light inputs. Our results suggest that O.tauri has adopted the latter strategy, possibly as a consequence of genomic reduction.</div>','<b>Title: </b>Moment Fitting for Parameter Inference in Repeatedly and Partially Observed Stochastic Biological Models<br><b>Year: </b>2012<br><b>Author: </b> P. Kugler<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0043001"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>reaction networks, network model, systems biology, biochemical reaction, stochastic reaction, model reduction, stochastic model, network dynamics, stochastic biochemical, stochastic simulation, system size, chemical reaction, gene network, elementary reactions, biochemical networks, parameter identification, parameter inference, cell populations, small system, linear noise<div style="font-size:11px;"><br><b>Abstract: </b><em>The inference of reaction rate parameters in biochemical network models from time series concentration data is a central task in computational systems biology. Under the assumption of well mixed conditions the network dynamics are typically described by the chemical master equation, the Fokker Planck equation, the linear noise approximation or the macroscopic rate equation. The inverse problem of estimating the parameters of the underlying network model can be approached in deterministic and stochastic ways, and available methods often compare individual or mean concentration traces obtained from experiments with theoretical model predictions when maximizing likelihoods, minimizing regularized least squares functionals, approximating posterior distributions or sequentially processing the data. In this article we assume that the biological reaction network can be observed at least partially and repeatedly over time such that sample moments of species molecule numbers for various time points can be calculated from the data. Based on the chemical master equation we furthermore derive closed systems of parameter dependent nonlinear ordinary differential equations that predict the time evolution of the statistical moments. For inferring the reaction rate parameters we suggest to not only compare the sample mean with the theoretical mean prediction but also to take the residual of higher order moments explicitly into account. Cost functions that involve residuals of higher order moments may form landscapes in the parameter space that have more pronounced curvatures at the minimizer and hence may weaken or even overcome parameter sloppiness and uncertainty. As a consequence both deterministic and stochastic parameter inference algorithms may be improved with respect to accuracy and efficiency. We demonstrate the potential of moment fitting for parameter inference by means of illustrative stochastic biological models from the literature and address topics for future research.</div>','<b>Title: </b>On the Relationship of Steady States of Continuous and Discrete Models Arising from Biology<br><b>Year: </b>2012<br><b>Author: </b> A. Veliz-Cuba,  J. Arthur,  L. Hochstetler,  V. Klomps and  E. Korpi<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s11538-012-9778-1"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BULLETIN OF MATHEMATICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>regulatory networks, network dynamics, gene regulatory, network model, genetic network, gene networks, genetic regulatory, mathematical model, parameter space, network system, interaction networks, system dynamics, network structure, network, dynamic networks, network reduction, models models, biological networks, network motifs, biological systems<div style="font-size:11px;"><br><b>Abstract: </b><em>For many biological systems that have been modeled using continuous and discrete models, it has been shown that such models have similar dynamical properties. In this paper, we prove that this happens in more general cases. We show that under some conditions there is a bijection between the steady states of continuous and discrete models arising from biological systems. Our results also provide a novel method to analyze certain classes of nonlinear models using discrete mathematics.</div>','<b>Title: </b>Distinct tissue-specific transcriptional regulation revealed by gene regulatory networks in maize<br><b>Year: </b>2018<br><b>Author: </b> J. Huang,  Jf. Zheng,  H. Yuan and  K. Mcginnis<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/s12870-018-1329-y"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC PLANT BIOLOGY<br><b>Cluster-Keywords: </b><em>regulatory networks, gene regulatory, gene expression, network inference, expression data, network structure, gene networks, networks grns, regulatory genes, boolean networks, bayesian network, network reconstruction, networks gene, network dynamics, gene regulation, inference method, systems biology, time series, infer gene, network<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Transcription factors TFs are proteins that can bind to DNA sequences and regulate gene expression. Many TFs are master regulators in cells that contribute to tissue-specific and cell-type-specific gene expression patterns in eukaryotes. Maize has been a model organism for over one hundred years, but little is known about its tissue-specific gene regulation through TFs. In this study, we used a network approach to elucidate gene regulatory networks GRNs in four tissues leaf, root, SAM and seed in maize. We utilized GENIE3, a machine-learning algorithm combined with large quantity of RNA-Seq expression data to construct four tissue-specific GRNs. Unlike some other techniques, this approach is not limited by high-quality Position Weighed Matrix PWM, and can therefore predict GRNs for over 2000 TFs in maize.Results: Although many TFs were expressed across multiple tissues, a multi-tiered analysis predicted tissue-specific regulatory functions for many transcription factors. Some well-studied TFs emerged within the four tissue-specific GRNs, and the GRN predictions matched expectations based upon published results for many of these examples. Our GRNs were also validated by ChIP-Seq datasets KN1, FEA4 and O2. Key TFs were identified for each tissue and matched expectations for key regulators in each tissue, including GO enrichment and identity with known regulatory factors for that tissue. We also found functional modules in each network by clustering analysis with the MCL algorithm.Conclusions: By combining publicly available genome-wide expression data and network analysis, we can uncover GRNs at tissue-level resolution in maize. Since ChIP-Seq and PWMs are still limited in several model organisms, our study provides a uniform platform that can be adapted to any species with genome-wide expression data to construct GRNs. We also present a publicly available database, maize tissue-specific GRN mGRN, https://www.bio.fsu.edu/mcginnislab/mgrn/, for easy querying. All source code and data are available at Github https://github.com/timedreamer/maizetissue-specificGRN.</div>','<b>Title: </b>Scaling Laws in Spatial Network Formation<br><b>Year: </b>2016<br><b>Author: </b> N. Molkenthin and  M. Timme<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1103/PhysRevLett.117.168301"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PHYSICAL REVIEW LETTERS<br><b>Cluster-Keywords: </b><em>complex networks, complex systems, network theory, network structure, biological network, social network, network science, dynamic networks, network systems, network model, network topology, systems biology, network, network analysis, system dynamics, networks, realworld networks, network automata, epidemic network, model network<div style="font-size:11px;"><br><b>Abstract: </b><em>Geometric constraints impact the formation of a broad range of spatial networks, from amino acid chains folding to proteins structures to rearranging particle aggregates. How the network of interactions dynamically self-organizes in such systems is far from fully understood. Here, we analyze a class of spatial network formation processes by introducing a mapping from geometric to graph-theoretic constraints. Combining stochastic and mean field analyses yields an algebraic scaling law for the extent graph diameter of the resulting networks with system size, in contrast to logarithmic scaling known for networks without constraints. Intriguingly, the exponent falls between that of self-avoiding random walks and that of space filling arrangements, consistent with experimentally observed scaling of the radius of gyration of protein tertiary structures with their chain length.</div>','<b>Title: </b>A general framework for large-scale model selection<br><b>Year: </b>2006<br><b>Author: </b> Md. Haunschild,  Sa. Wahl,  B. Freisleben and  W. Wiechert<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1080/10556780600872208"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>OPTIMIZATION METHODS  SOFTWARE<br><b>Cluster-Keywords: </b><em>kinetic models, metabolic networks, kinetic parameters, dynamic models, systems biology, network model, dynamic metabolic, biochemical network, rate constants, enzyme kinetics, kinetic data, adjustable parameters, metabolic model, parameter estimation, dynamic optimization, model selection, mathematical model, kinetic network, model response, model<div style="font-size:11px;"><br><b>Abstract: </b><em>Model selection is concerned with the choice of a mathematical model from a set of candidates that best describes a given set of experimental data. Large families of models arise in the context of structured mechanistic modelling in several application fields. In this situation the model selection problem cannot be solved by brute force testing of all possible models because of the high computational costs. However, more information on the different models of a family is available by their interdependencies, given by generalization or simplification relations. Large-scale model selection algorithms should exploit these relations for navigation in the discrete space of all model candidates. This paper presents a general approach for large-scale model selection by specifying the necessary computational primitives for navigating in large model families. As a non-trivial example it is shown how families of biochemical network models arising from the evaluation of stimulus response experiments are mapped to the general formalism. Finally, a first model selection algorithm based on the mentioned computational primitives is introduced and applied to complex biochemical network experiments. It is based on a load-balancing algorithm by making use of grid computing facilities.</div>','<b>Title: </b>OKVAR-Boost: a novel boosting algorithm to infer nonlinear dynamics and interactions in gene regulatory networks<br><b>Year: </b>2013<br><b>Author: </b> N. Lim,  Y. Senbabaoglu,  G. Michailidis and  F. DAlche-Buc<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1093/bioinformatics/btt167"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>regulatory networks, gene regulatory, gene expression, network inference, expression data, network structure, gene networks, networks grns, regulatory genes, boolean networks, bayesian network, network reconstruction, networks gene, network dynamics, gene regulation, inference method, systems biology, time series, infer gene, network<div style="font-size:11px;"><br><b>Abstract: </b><em>Motivation: Reverse engineering of gene regulatory networks remains a central challenge in computational systems biology, despite recent advances facilitated by benchmark in silico challenges that have aided in calibrating their performance. A number of approaches using either perturbation knock-out or wild-type time-series data have appeared in the literature addressing this problem, with the latter using linear temporal models. Nonlinear dynamical models are particularly appropriate for this inference task, given the generation mechanism of the time-series data. In this study, we introduce a novel nonlinear autoregressive model based on operator-valued kernels that simultaneously learns the model parameters, as well as the network structure.Results: A flexible boosting algorithm OKVAR-Boost that shares features from L-2-boosting and randomization-based algorithms is developed to perform the tasks of parameter learning and network inference for the proposed model. Specifically, at each boosting iteration, a regularized Operator-valued Kernel-based Vector AutoRegressive model OKVAR is trained on a random subnetwork. The final model consists of an ensemble of such models. The empirical estimation of the ensemble models Jacobian matrix provides an estimation of the network structure. The performance of the proposed algorithm is first evaluated on a number of benchmark datasets from the DREAM3 challenge and then on real datasets related to the In vivo Reverse-Engineering and Modeling Assessment IRMA and T-cell networks. The high-quality results obtained strongly indicate that it outperforms existing approaches.</div>','<b>Title: </b>Gene Expression Complex Networks: Synthesis, Identification, and Analysis<br><b>Year: </b>2011<br><b>Author: </b> Fm. Lopes,  Rm. Cesar and  Ld. Costa<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1089/cmb.2010.0118"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF COMPUTATIONAL BIOLOGY<br><b>Cluster-Keywords: </b><em>regulatory networks, gene regulatory, gene expression, network inference, expression data, network structure, gene networks, networks grns, regulatory genes, boolean networks, bayesian network, network reconstruction, networks gene, network dynamics, gene regulation, inference method, systems biology, time series, infer gene, network<div style="font-size:11px;"><br><b>Abstract: </b><em>Thanks to recent advances in molecular biology, allied to an ever increasing amount of experimental data, the functional state of thousands of genes can now be extracted simultaneously by using methods such as cDNA microarrays and RNA-Seq. Particularly important related investigations are the modeling and identification of gene regulatory networks from expression data sets. Such a knowledge is fundamental for many applications, such as disease treatment, therapeutic intervention strategies and drugs design, as well as for planning high-throughput new experiments. Methods have been developed for gene networks modeling and identification from expression profiles. However, an important open problem regards how to validate such approaches and its results. This work presents an objective approach for validation of gene network modeling and identification which comprises the following three main aspects: 1 Artificial Gene Networks AGNs model generation through theoretical models of complex networks, which is used to simulate temporal expression data; 2 a computational method for gene network identification from the simulated data, which is founded on a feature selection approach where a target gene is fixed and the expression profile is observed for all other genes in order to identify a relevant subset of predictors; and 3 validation of the identified AGN-based network through comparison with the original network. The proposed framework allows several types of AGNs to be generated and used in order to simulate temporal expression data. The results of the network identification method can then be compared to the original network in order to estimate its properties and accuracy. Some of the most important theoretical models of complex networks have been assessed: the uniformly-random Erdos-Renyi ER, the small-world Watts-Strogatz WS, the scale-free Barabasi-Albert BA, and geographical networks GG. The experimental results indicate that the inference method was sensitive to average degree k variation, decreasing its network recovery rate with the increase of k. The signal size was important for the inference method to get better accuracy in the network identification rate, presenting very good results with small expression profiles. However, the adopted inference method was not sensible to recognize distinct structures of interaction among genes, presenting a similar behavior when applied to different network topologies. In summary, the proposed framework, though simple, was adequate for the validation of the inferred networks by identifying some properties of the evaluated method, which can be extended to other inference methods.</div>','<b>Title: </b>Reduction of chemical reaction networks through delay distributions<br><b>Year: </b>2013<br><b>Author: </b> M. Barrio,  A. Leier and  Tt. Marquez-Lago<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1063/1.4793982"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF CHEMICAL PHYSICS<br><b>Cluster-Keywords: </b><em>reaction networks, network model, systems biology, biochemical reaction, stochastic reaction, model reduction, stochastic model, network dynamics, stochastic biochemical, stochastic simulation, system size, chemical reaction, gene network, elementary reactions, biochemical networks, parameter identification, parameter inference, cell populations, small system, linear noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Accurate modelling and simulation of dynamic cellular events require two main ingredients: an adequate description of key chemical reactions and simulation of such chemical events in reasonable time spans. Quite logically, posing the right model is a crucial step for any endeavour in Computational Biology. However, more often than not, it is the associated computational costs which actually limit our capabilities of representing complex cellular behaviour. In this paper, we propose a methodology aimed at representing chains of chemical reactions by much simpler, reduced models. The abridgement is achieved by generation of model-specific delay distribution functions, consecutively fed to a delay stochastic simulation algorithm. We show how such delay distributions can be analytically described whenever the system is solely composed of consecutive first-order reactions, with or without additional "backward" bypass reactions, yielding an exact reduction. For models including other types of monomolecular reactions constitutive synthesis, degradation, or "forward" bypass reactions, we discuss why one must adopt a numerical approach for its accurate stochastic representation, and propose two alternatives for this. In these cases, the accuracy depends on the respective numerical sample size. Our model reduction methodology yields significantly lower computational costs while retaining accuracy. Quite naturally, computational costs increase alongside network size and separation of time scales. Thus, we expect our model reduction methodologies to significantly decrease computational costs in these instances. We anticipate the use of delays in model reduction will greatly alleviate some of the current restrictions in simulating large sets of chemical reactions, largely applicable in pharmaceutical and biological research. C 2013 American Institute of Physics. http://dx.doi.org/10.1063/1.4793982</div>','<b>Title: </b>Adjusting Phenotypes by Noise Control<br><b>Year: </b>2012<br><b>Author: </b> Kh. Kim and  Hm. Sauro<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pcbi.1002344"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS COMPUTATIONAL BIOLOGY<br><b>Cluster-Keywords: </b><em>gene expression, gene network, stochastic model, gene regulatory, systems biology, expression systems, gene regulation, network activity, expression dynamics, noise control, regulatory network, deterministic models, network components, stochastic modeling, regulation system, escherichia coli, regulatory system, synthetic gene, regulatory genes, gene<div style="font-size:11px;"><br><b>Abstract: </b><em>Genetically identical cells can show phenotypic variability. This is often caused by stochastic events that originate from randomness in biochemical processes involving in gene expression and other extrinsic cellular processes. From an engineering perspective, there have been efforts focused on theory and experiments to control noise levels by perturbing and replacing gene network components. However, systematic methods for noise control are lacking mainly due to the intractable mathematical structure of noise propagation through reaction networks. Here, we provide a numerical analysis method by quantifying the parametric sensitivity of noise characteristics at the level of the linear noise approximation. Our analysis is readily applicable to various types of noise control and to different types of system; for example, we can orthogonally control the mean and noise levels and can control system dynamics such as noisy oscillations. As an illustration we applied our method to HIV and yeast gene expression systems and metabolic networks. The oscillatory signal control was applied to p53 oscillations from DNA damage. Furthermore, we showed that the efficiency of orthogonal control can be enhanced by applying extrinsic noise and feedback. Our noise control analysis can be applied to any stochastic model belonging to continuous time Markovian systems such as biological and chemical reaction systems, and even computer and social networks. We anticipate the proposed analysis to be a useful tool for designing and controlling synthetic gene networks.</div>','<b>Title: </b>Development of a Chlamydomonas reinhardtii metabolic network dynamic model to describe distinct phenotypes occurring at different CO2 levels<br><b>Year: </b>2018<br><b>Author: </b> Dam. Salguero,  M. Fernandez-Nino,  Lm. Serrano-Bermudez,  Dop. Melo,  Fv. Winck,  C. Caldana and  Afg. Barrios<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.7717/peerj.5528"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PEERJ<br><b>Cluster-Keywords: </b><em>metabolic network, metabolic model, dynamic flux, growth rate, genomescale metabolic, flux balance, balance analysis, metabolic flux, metabolic pathways, dynamic model, metabolic reactions, cellular growth, regulatory network, systems biology, optimal growth, metabolic, modeling metabolic, growth conditions, dynamic analysis, cell growth<div style="font-size:11px;"><br><b>Abstract: </b><em>The increase in atmospheric CO2 due to anthropogenic activities is generating climate change, which has resulted in a subsequent rise in global temperatures with severe environmental impacts. Biological mitigation has been considered as an alternative for environmental remediation and reduction of greenhouse gases in the atmosphere. In fact, the use of easily adapted photosynthetic organisms able to fix CO2 with low-cost operation is revealing its high potential for industry. Among those organism, the algae Chlamydomonas reinhardtii have gain special attention as a model organism for studying CO2 fixation, biomass accumulation and bioenergy production upon exposure to several environmental conditions. In the present study, we studied the Chlamydomonas response to different CO2 levels by comparing metabolomics and transcriptomics data with the predicted results from our new-improved genomic-scale metabolic model. For this, we used in silico methods at steady dynamic state varying the levels of CO2. Our main goal was to improve our capacity for predicting metabolic routes involved in biomass accumulation. The improved genomic-scale metabolic model presented in this study was shown to be phenotypically accurate, predictive, and a significant improvement over previously reported models. Our model consists of 3726 reactions and 2436 metabolites, and lacks any thermodynamically infeasible cycles. It was shown to be highly sensitive to environmental changes under both steady-state and dynamic conditions. As additional constraints, our dynamic model involved kinetic parameters associated with substrate consumption at different growth conditions i.e., low CO2-heterotrophic and high CO2-mixotrophic. Our results suggest that cells growing at high CO2 i.e., photoautotrophic and mixotrophic conditions have an increased capability for biomass production. In addition, we have observed that ATP production also seems to be an important limiting factor for growth under the conditions tested. Our experimental data metabolomics and transcriptomics and the results predicted by our model clearly suggest a differential behavior between low CO2-heterotrophic and high CO2-mixotrophic growth conditions. The data presented in the current study contributes to better dissect the biological response of C. reinhardtii, as a dynamic entity, to environmental and genetic changes. These findings are of great interest given the biotechnological potential of this microalga for CO2 fixation, biomass accumulation, and bioenergy production.</div>','<b>Title: </b>Massive Exploration of Perturbed Conditions of the Blood Coagulation Cascade through GPU Parallelization<br><b>Year: </b>2014<br><b>Author: </b> P. Cazzaniga,  Ms. Nobile,  D. Besozzi,  M. Bellini and  G. Mauri<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1155/2014/863298"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOMED RESEARCH INTERNATIONAL<br><b>Cluster-Keywords: </b><em>regulatory networks, stochastic simulation, gene networks, reaction networks, systems biology, stochastic model, network models, complex systems, networks stochastic, gene regulatory, biological systems, stochastic dynamics, differential equations, protein networks, cell cycle, continuous model, interaction networks, systems chemistry, time delays, biochemical reaction<div style="font-size:11px;"><br><b>Abstract: </b><em>The introduction of general-purpose Graphics Processing Units GPUs is boosting scientific applications in Bioinformatics, Systems Biology, and Computational Biology. In these fields, the use of high-performance computing solutions is motivated by the need of performing large numbers of in silico analysis to study the behavior of biological systems in different conditions, which necessitate a computing power that usually overtakes the capability of standard desktop computers. In this work we present coagSODA, a CUDA-powered computational tool that was purposely developed for the analysis of a large mechanistic model of the blood coagulation cascade BCC, defined according to both mass-action kinetics and Hill functions. coagSODA allows the execution of parallel simulations of the dynamics of the BCC by automatically deriving the system of ordinary differential equations and then exploiting the numerical integration algorithm LSODA. We present the biological results achieved with a massive exploration of perturbed conditions of the BCC, carried out with one-dimensional and bi-dimensional parameter sweep analysis, and show that GPU-accelerated parallel simulations of this model can increase the computational performances up to a 181x speedup compared to the corresponding sequential simulations.</div>','<b>Title: </b>Polynomial algebra of discrete models in systems biology<br><b>Year: </b>2010<br><b>Author: </b> A. Veliz-Cuba,  As. Jarrah and  R. Laubenbacher<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1093/bioinformatics/btq240"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Motivation: An increasing number of discrete mathematical models are being published in Systems Biology, ranging from Boolean network models to logical models and Petri nets. They are used to model a variety of biochemical networks, such as metabolic networks, gene regulatory networks and signal transduction networks. There is increasing evidence that such models can capture key dynamic features of biological networks and can be used successfully for hypothesis generation.Results: This article provides a unified framework that can aid the mathematical analysis of Boolean network models, logical models and Petri nets. They can be represented as polynomial dynamical systems, which allows the use of a variety of mathematical tools from computer algebra for their analysis. Algorithms are presented for the translation into polynomial dynamical systems. Examples are given of how polynomial algebra can be used for the model analysis.</div>','<b>Title: </b>Dynamical genomic network applied to artificial neurogenesis<br><b>Year: </b>1997<br><b>Author: </b> O. MichelLink: No link<br><b>Journal: </b>CONTROL AND CYBERNETICS<br><b>Cluster-Keywords: </b><em>neural network, network model, network, neurogenesis model, genomic network, networks, saccade control, droplets networks, understanding networks, gene regulatory, model, tumor-immune networks, networks dynamical, network systems, artificial neurogenesis, regulatory networks, dynamic network, information flow, cancer cells, capillary networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Optimisation of the structure of artificial neural network using evolutionary techniques has been investigated by a number of authors using various approaches. In this paper, we claim that such a process requires the design of a complex neurogenesis model featuring a set of fundamental properties such as modularity and the possibility of complexity adaptation. Developmental and molecular biology might be an interesting source of inspiration for designing such powerful artificial neurogenesis systems allowing the generation of complex modular neural structures. This paper provides a description of a neurogenesis model based on a modelling of a natural genomic network and associated with an evolutionary process. Experimental results demonstrate some basic capabilities of the proposed neurogenesis model to produce multi-layered neural networks. An application to learning in the control of a mobile robot lead to unexpected results, giving hints for continuing the research towards the automatic generation of more complex adaptive neural networks.</div>','<b>Title: </b>Estimating parameters and predicting membrane voltages with conductance-based neuron models<br><b>Year: </b>2014<br><b>Author: </b> Cd. Meliza,  M. Kostuk,  H. Huang,  A. Nogaret,  D. Margoliash and  Hdi. Abarbanel<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s00422-014-0615-5"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOLOGICAL CYBERNETICS<br><b>Cluster-Keywords: </b><em>network models, population model, field model, neural population, neural field, iot networks, neuronal networks, neural network, differential equations, parameter neural, lotkavolterra model, model, neuron models, model neurons, model leads, ecosystem model, iot devices, noise model, neuronal models, inhibitory synaptic<div style="font-size:11px;"><br><b>Abstract: </b><em>Recent results demonstrate techniques for fully quantitative, statistical inference of the dynamics of individual neurons under the Hodgkin-Huxley framework of voltage-gated conductances. Using a variational approximation, this approach has been successfully applied to simulated data from model neurons. Here, we use this method to analyze a population of real neurons recorded in a slice preparation of the zebra finch forebrain nucleus HVC. Our results demonstrate that using only 1,500 ms of voltage recorded while injecting a complex current waveform, we can estimate the values of 12 state variables and 72 parameters in a dynamical model, such that the model accurately predicts the responses of the neuron to novel injected currents. A less complex model produced consistently worse predictions, indicating that the additional currents contribute significantly to the dynamics of these neurons. Preliminary results indicate some differences in the channel complement of the models for different classes of HVC neurons, which accords with expectations from the biology. Whereas the model for each cell is incomplete representing only the somatic compartment, and likely to be missing classes of channels that the real neurons possess, our approach opens the possibility to investigate in modeling the plausibility of additional classes of channels the cell might possess, thus improving the models over time. These results provide an important foundational basis for building biologically realistic network models, such as the one in HVC that contributes to the process of song production and developmental vocal learning in songbirds.</div>','<b>Title: </b>s-core network decomposition: A generalization of k-core analysis to weighted networks<br><b>Year: </b>2013<br><b>Author: </b> M. Eidsaa and  E. Almaas<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1103/PhysRevE.88.062819"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PHYSICAL REVIEW E<br><b>Cluster-Keywords: </b><em>complex networks, complex systems, network theory, network structure, biological network, social network, network science, dynamic networks, network systems, network model, network topology, systems biology, network, network analysis, system dynamics, networks, realworld networks, network automata, epidemic network, model network<div style="font-size:11px;"><br><b>Abstract: </b><em>A broad range of systems spanning biology, technology, and social phenomena may be represented and analyzed as complex networks. Recent studies of such networks using k-core decomposition have uncovered groups of nodes that play important roles. Here, we present s-core analysis, a generalization of k-core or k-shell analysis to complex networks where the links have different strengths or weights. We demonstrate the s-core decomposition approach on two random networks ER and configuration model with scale-free degree distribution where the link weights are i random, ii correlated, and iii anticorrelated with the node degrees. Finally, we apply the s-core decomposition approach to the protein-interaction network of the yeast Saccharomyces cerevisiae in the context of two gene-expression experiments: oxidative stress in response to cumene hydroperoxide CHP, and fermentation stress response FSR. We find that the innermost s-cores are i different from innermost k-cores, ii different for the two stress conditions CHP and FSR, and iii enriched with proteins whose biological functions give insight into how yeast manages these specific stresses.</div>','<b>Title: </b>Consciousness in the Universe is Scale Invariant and Implies an Event Horizon of the Human Brain<br><b>Year: </b>2017<br><b>Author: </b> Dkf. Meijer and  Hjh. Geesink<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.14704/nq.2017.15.3.1079"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>NEUROQUANTOLOGY<br><b>Cluster-Keywords: </b><em>systems biology, dynamic models, biological systems, model, dynamic systems, computational systems, network model, experimental data, model parameters, systems, systems model, models, pathway systems, data, biochemical pathway, dynamics model, biochemical networks, computational models, network, system<div style="font-size:11px;"><br><b>Abstract: </b><em>Our brain is not a "stand alone" information processing organ: it acts as a central part of our integral nervous system with recurrent information exchange with the entire organism and the cosmos. In this study, the brain is conceived to be embedded in a holographic structured field that interacts with resonant sensitive structures in the various cell types in our body. In order to explain earlier reported ultra-rapid brain responses and effective operation of the meta-stable neural system, a field-receptive mental workspace is proposed to be communicating with the brain. Our integral nervous system is seen as a dedicated neural transmission and multi-cavity network that, in a non-dual manner, interacts with the proposed supervening meta-cognitive domain. Among others, it is integrating discrete patterns of eigen-frequencies of photonic/solitonic waves, thereby continuously updating a time-symmetric global memory space of the individual. Its toroidal organization allows the coupling of gravitational, dark energy, zero-point energy field ZPE as well as earth magnetic fields energies and transmits wave information into brain tissue, that thereby is instrumental in high speed conscious and sub-conscious information processing. We propose that the supposed field-receptive workspace, in a mutual interaction with the whole nervous system, generates self-consciousness and is conceived as operating from a 4th spatial dimension hyper-sphere. Its functional structure is adequately defined by the geometry of the torus, that is envisioned as a basic unit operator of space-time. The latter is instrumental in collecting the pattern of discrete soliton frequencies that provided an algorithm for coherent life processes, as earlier identified by us. It is postulated that consciousness in the entire universe arises through, scale invariant, nested toroidal coupling of various energy fields, that may include quantum error correction. In the brain of the human species, this takes the form of the proposed holographic workspace, that collects active information in a "brain event horizon", representing an internal and fully integral model of the self. This brain-supervening workspace is equipped to convert integrated coherent wave energies into attractor type/standing waves that guide the related cortical template to a higher coordination of reflection and action as well as network synchronicity, as required for conscious states. In relation to its scale-invariant global character, we find support for a universal information matrix, that was extensively described earlier, as a supposed implicate order as well as in a spectrum of space-time theories in current physics. The presence of a field-receptive resonant workspace, associated with, but not reducible to, our brain, may provide an interpretation framework for widely reported, but poorly understood transpersonal conscious states and algorithmic origin of life. It also points out the deep connection of mankind with the cosmos and our major responsibility for the future of our planet.</div>','<b>Title: </b>Dynamic sensitivity and nonlinear interactions influence the system-level evolutionary patterns of phototransduction proteins<br><b>Year: </b>2015<br><b>Author: </b> Bm. Invergo,  L. Montanucci and  J. Bertranpetit<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1098/rspb.2015.2215"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PROCEEDINGS OF THE ROYAL SOCIETY B-BIOLOGICAL SCIENCES<br><b>Cluster-Keywords: </b><em>regulatory network, systems biology, systems biological, network dynamics, gene expression, biochemical networks, raft proteins, biological networks, gene circuits, gene regulatory, signalling network, lipid raft, nuclear lamina, protein domain, networks systems, biological approaches, expression pattern, protein interaction, cell differentiation, network motifs<div style="font-size:11px;"><br><b>Abstract: </b><em>Determining the influence of complex, molecular-system dynamics on the evolution of proteins is hindered by the significant challenge of quantifying the control exerted by the proteins on system output. We have employed a combination of systems biology and molecular evolution analyses in a first attempt to unravel this relationship. We employed a comprehensive mathematical model of mammalian phototransduction to predict the degree of influence that each protein in the system exerts on the high-level dynamic behaviour. We found that the genes encoding the most dynamically sensitive proteins exhibit relatively relaxed evolutionary constraint. We also investigated the evolutionary and epistatic influences of the many nonlinear interactions between proteins in the system and found several pairs to have coevolved, including those whose interactions are purely dynamical with respect to system output. This evidence points to a key role played by nonlinear system dynamics in influencing patterns of molecular evolution.</div>','<b>Title: </b>A system-level model for the microbial regulatory genome<br><b>Year: </b>2014<br><b>Author: </b> An. Brooks,  Dj. Reiss,  A. Allard,  Wj. Wu,  Dm. Salvanha,  Cl. Plaisier,  S. Chandrasekaran,  M. Pan,  A. Kaur and  Ns. Baliga<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.15252/msb.20145160"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MOLECULAR SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>systems biology, regulatory network, interaction networks, network model, gene regulatory, network, network biology, networks, gene network, response networks, network dynamics, dynamic model, cell networks, gene expression, biological networks, system dynamics, dynamic network, model, biological systems, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Microbes can tailor transcriptional responses to diverse environmental challenges despite having streamlined genomes and a limited number of regulators. Here, we present data-driven models that capture the dynamic interplay of the environment and genome-encoded regulatory programs of two types of prokaryotes: Escherichia coli a bacterium and Halobacterium salinarum an archaeon. The models reveal how the genome-wide distributions of cis-acting gene regulatory elements and the conditional influences of transcription factors at each of those elements encode programs for eliciting a wide array of environment-specific responses. We demonstrate how these programs partition transcriptional regulation of genes within regulons and operons to re-organize gene-gene functional associations in each environment. The models capture fitness-relevant co-regulation by different transcriptional control mechanisms acting across the entire genome, to define a generalized, system-level organizing principle for prokaryotic gene regulatory networks that goes well beyond existing paradigms of gene regulation. An online resource http://egrin2.systemsbiology.net has been developed to facilitate multiscale exploration of conditional gene regulation in the two prokaryotes.</div>','<b>Title: </b>A new time-dependent complexity reduction method for biochemical systems<br><b>Year: </b>2005<br><b>Author: </b> J. Zobeley,  D. Lebiedz,  J. Kammerer,  A. Ishmurzin and  U. KummerLink: No link<br><b>Journal: </b>TRANSACTIONS ON COMPUTATIONAL SYSTEMS BIOLOGY I<br><b>Cluster-Keywords: </b><em>immune system, network models, immune response, model, biological systems, models, model reduction, dynamic model, biological networks, system, mathematical models, biochemical network, response model, systems, systems biology, network, immune, biology model, reaction networks, system systems<div style="font-size:11px;"><br><b>Abstract: </b><em>Systems biology aims at an understanding of increasingly large and complex cellular systems making use of computational approaches, e.g. numerical simulations. The size and complexity of the underlying biochemical reaction networks call for methods to speed up simulations and/or dissect the biochemical network into smaller subsystems which can be studied independently. Both goals can be achieved by so-called complexity reduction algorithms. However, existing complexity reduction approaches for biochemical reaction networks are mostly based on studying the steady state behavior of a system and/or are based on heuristics. Given the fact that many complex biochemical systems display highly nonlinear dynamics and that this dynamics plays a crucial role in the functioning of the organism, a new methodology has to be developed. Therefore, we present a new complexity reduction method which is time-dependent and suited not only for steady states, but for all possible dynamics of a biochemical system. It makes use of the evolution of the different time-scales in the system, allowing to reduce the number of equations necessary to describe the system which is speeding up the computation time. In addition, it is possible to study the way different variables/metabolites contribute to the reduced equation system which indicates how strongly they interact and couple. In the extreme case of variables decoupling in a specific state, the method allows the complete dissection of the system resulting in subsystems that can be studied in isolation. The whole method provides a systematic tool for an automated complexity reduction of arbitrary biochemical reaction networks. With the aid of a specific example, the oscillatory peroxidase-oxidase system, we show that coupling of time-scales depends heavily on the specific dynamics of the system. Therefore, neither computational improvement nor systematic understanding can be achieved by studying these aspects solely under steady state conditions.</div>','<b>Title: </b>A road map for developing and applying object-oriented bayesian networks to "WICKED" problems<br><b>Year: </b>2017<br><b>Author: </b> N. Benjamin-Fink and  Bk. Reilly<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.ecolmodel.2017.06.028"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ECOLOGICAL MODELLING<br><b>Cluster-Keywords: </b><em>boolean networks, regulatory networks, network model, bayesian networks, model networks, systems biology, random boolean, real cells, immune system, genetic regulatory, networks, biochemical network, fluorescence images, cellular network, gene regulatory, cell differentiation, network, biological systems, ensemble approach, cell dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>Wildlife managers, conservation-based authorities and NGOS are often tasked to provide management guidelines concerning "wicked" problems and Ecosystem Services ESS with scarce data and limited resources. More often than not, this dilemma is further intensified by competing objectives and the need for economic profitability. A range of quantitative and qualitative models are often applied to model ecosystem services. However, lack of quantifiable data remains an obstacle. We provide a brief overview of Bayesian Networks BNs, Object Oriented Bayesian Networks OOBNs, Dynamic Bayesian Networks DBNs, and Dynamic Object Oriented Bayesian Networks DOOBNs construction and a flow chart highlights their utility. OOBNs are a semi-quantitative modelling approach that uses Bayes probability theorem to represent ecological complexities and conservation concerns. Its transparency allows it to act as a conceptual system and a decision support tool. And yet, Bayesian networks are not commonly applied to the ecological discipline because they are computationally expensive. We promote the notion that sustainable management is advanced by providing biology-based decision-making networks to address wicked problems. First, we provide a brief reasoning for the applicability of Bayes theorem to wicked challenges. Then, we demonstrate the efficiency of Bayesian modelling by applying it to the case study of the black wildebeest Connochaetes gnou and blue wildebeest Connochaetes taurinus hybridization concern in South Africa. Empirical data, in addition to expert explicit understanding of uncertainties was utilized to infer the probability that specific ecological, biological, and economic parameters, and their relationships, facilitate hybridization. We found the following management variables to be key to impact the probability of hybridization: i blue wildebeest male to black wildebeest male ratio, ii land cover tree savanna vs. grassland, and iii spatial connectivity i.e., fences. By quantifying ecological uncertainty, OOBN illuminates decision-making tradeoffs and serves to promote informed decision making either during the risk assessment stage or the management impact analysis stage. We offer a SWOT analysis Strength, weaknesses, opportunities, and threats and put forth a step-by-step user friendly roadmap by which OOBNs may be applied to solve similar "wicked" problems. We identify three primary phases: phases: 1 a priori data generation, 2 model development, and 3 model validation and verification. Our suggested step-by-step framework of OOBN construction and validation is of global relevance; it is designed to model similar "wicked" problems worldwide. C 2017 Elsevier B.V. All rights reserved.</div>','<b>Title: </b>Virtual interface substructure synthesis method for normal mode analysis of super-large molecular complexes at atomic resolution<br><b>Year: </b>2011<br><b>Author: </b> Xh. Chen,  Yx. Sun,  Xb. An and  Dm. Ming<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1063/1.3647314"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF CHEMICAL PHYSICS<br><b>Cluster-Keywords: </b><em>normal mode, mode analysis, elastic network, functional motions, functional dynamics, molecular chaperone, structural biology, acidic residues, network models, molecular dynamics, atomic resolution, protein complexes, conserved functional, hsp dynamics, correlated motions, protein folding, large biomolecular, thioredoxin system, harmonic analysis, protein motions<div style="font-size:11px;"><br><b>Abstract: </b><em>Normal mode analysis of large biomolecular complexes at atomic resolution remains challenging in computational structure biology due to the requirement of large amount of memory space and central processing unit time. In this paper, we present a method called virtual interface substructure synthesis method or VISSM to calculate approximate normal modes of large biomolecular complexes at atomic resolution. VISSM introduces the subunit interfaces as independent substructures that join contacting molecules so as to keep the integrity of the system. Compared with other approximate methods, VISSM delivers atomic modes with no need of a coarse-graining-then-projection procedure. The method was examined for 54 protein-complexes with the conventional all-atom normal mode analysis using CHARMM simulation program and the overlap of the first 100 low-frequency modes is greater than 0.7 for 49 complexes, indicating its accuracy and reliability. We then applied VISSM to the satellite panicum mosaic virus SPMV, 78 300 atoms and to F-actin filament structures of up to 39-mer, 228 813 atoms and found that VISSM calculations capture functionally important conformational changes accessible to these structures at atomic resolution. Our results support the idea that the dynamics of a large biomolecular complex might be understood based on the motions of its component subunits and the way in which subunits bind one another. C 2011 American Institute of Physics. doi: 10.1063/1.3647314</div>','<b>Title: </b>Parameter estimation in models combining signal transduction and metabolic pathways: the dependent input approach<br><b>Year: </b>2006<br><b>Author: </b> Naw. Van Riel and  Ed. Sontag<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1049/ip-syb:20050076"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IEE PROCEEDINGS SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>systems biology, biological systems, system dynamics, molecular systems, experimental design, network model, model system, model parameters, experiment design, kinetic parameters, perfect adaptation, experimental data, molecular mechanisms, homeostatic mechanisms, switching dynamics, biological mechanisms, control systems, chemotaxis network, decentralized control, feedback control<div style="font-size:11px;"><br><b>Abstract: </b><em>Biological complexity and limited quantitative measurements pose severe challenges to standard engineering methodologies for modelling and simulation of genes and gene products integrated in a functional network. In particular, parameter quantification is a bottleneck, and therefore parameter estimation, identifiability, and optimal experiment design are important research topics in systems biology. An approach is presented in which unmodelled dynamics are replaced by fictitious dependent inputs. The dependent input approach is particularly useful in validation experiments, because it allows one to fit model parameters to experimental data generated by a reference cell type wild-type and then test this model on data generated by a variation mutant, so long as the mutations only affect the unmodelled dynamics that produce the dependent inputs. Another novel feature of the approach is in the inclusion of a priori information in a multi-objective identification criterion, making it possible to obtain estimates of parameter values and their variances from a relatively limited experimental data set. The pathways that control the nitrogen uptake fluxes in bakers yeast Saccharomyces cerevisiae have been studied. Well-defined perturbation experiments were performed on cells growing in steady-state. Time-series data of extracellular and intracellular metabolites were obtained, as well as mRNA levels. A nonlinear model was proposed and was shown to be structurally identifiable given data of its inputs and outputs. The identified model is a reliable representation of the metabolic system, as it could correctly describe the responses of mutant cells and different perturbations.</div>','<b>Title: </b>MODULAR PARAMETER IDENTIFICATION OF BIOMOLECULAR NETWORKS<br><b>Year: </b>2016<br><b>Author: </b> M. Lang and  J. Stelling<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1137/15M103306X"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SIAM JOURNAL ON SCIENTIFIC COMPUTING<br><b>Cluster-Keywords: </b><em>systems biology, network models, signaling network, parameter estimation, dynamic models, biological systems, mathematical models, metabolic network, parameter sensitivities, computational model, gene circuit, circuit models, experimental data, metabolic models, model system, scalable parameter, model parameters, parameter space, biomolecular networks, model<div style="font-size:11px;"><br><b>Abstract: </b><em>The increasing complexity of dynamic models in systems and synthetic biology poses computational challenges especially for the identification of model parameters. While modularization of the corresponding optimization problems could help reduce the "curse of dimensionality," abundant feedback and crosstalk mechanisms prohibit a simple decomposition of most biomolecular networks into subnetworks, or modules. Drawing on ideas from network modularization and multiple-shooting optimization, we present here a modular parameter identification approach that explicitly allows for such interdependencies. Interfaces between our modules are given by the experimentally measured molecular species. This definition allows deriving good initial estimates for the inter-module communication directly from the experimental data. Given these estimates, the states and parameter sensitivities of different modules can be integrated independently. To achieve consistency between modules, we iteratively adjust the estimates for inter-module communication while optimizing the parameters. After convergence to an optimal parameter set-but not during earlier iterations-the intermodule communication as well as the individual modules state dynamics agree with the dynamics of the nonmodularized network. Our modular parameter identification approach allows for easy parallelization; it can reduce the computational complexity for larger networks and decrease the probability to converge to suboptimal local minima. We demonstrate the algorithms performance in parameter estimation for two biomolecular networks, a synthetic genetic oscillator and a mammalian signaling pathway.</div>','<b>Title: </b>Small mammals, habitat patches and PVA models: a field test of model predictive ability<br><b>Year: </b>2002<br><b>Author: </b> Db. Lindenmayer and  Rc. Lacy<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/S0006-32070100134-3"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOLOGICAL CONSERVATION<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>The results are described of comparisons between actual values for patch occupancy for two species of Australian small mammals Bush Rat Rattus fuscipes and Agile Antechinus Antechinus agilis determined from field sampling and predictions of patch occupancy made using VORTEX, a generic simulation model for Population Viability Analysis PVA. The work focussed on a fragmented forest in south-eastern Australia comprised of a network of 39 patches of native eucalypt forest surrounded by extensive stands of exotic softwood Radiata Pine Pinus radiata plantation. A range of modelling scenarios were completed in which four broad factors were varied: 1 inter-patch variation in habitat quality; 2 the Pattern of inter-patch dispersal; 3 the rate of interpatch dispersal; and 4 the population sink effects of the Radiata Pine matrix that surrounded the eucalypt patches. Model predictions were made for the total number of animals, the distribution of animal density among patches, the total number of occupied patches, and the probability of patch occupancy. Predictions were then compared with observed values for these same measures based on extensive field surveys of small mammals in the patch system. For most models for the Bush Rat, the predicted relative density of animals per patch correlated well with the values estimated from field surveys. Predictions of patch occupancy were not significantly different from the actual value for the number of occupied patches in half the models tested. The better models explained 10-16 of the log-likelihood of the probability of patch occupancy. While some of the models gave reasonable forecasts of the number of occupied patches, even in these cases, they had only moderate ability to predict which patches were occupied. Field surveys revealed there was no relationship between patch area and population density for the Agile Antechinus-an outcome correctly predicted by only a few models. Five of the 18 scenarios completed for the Agile Antechinus gave predicted numbers of occupied patches not significantly different from the observed number. In each of these five cases, large standard deviations around the mean predicted value meant uncertainty generated by the simulation model limited the predictive power of the PVA. Some of the models gave reasonable predictions for the number of occupied patches, but those models were unable to predict which ones were actually occupied. The results of our study suggest that key processes influencing which specific patches would be occupied were not modelled appropriately. High levels of variability and fecundity drive the population dynamics of the Bush Rat and Agile Antechinus, making the patch system unpredictable and difficult to model accurately. Despite the fact that both the Bush Rat and the Agile Antechinus are two of the most studied mammals in Australia, there are attributes of their biology that are presently poorly understood which were not included in the VORTEX model, but which could strongly influence patch occupancy. For example, local landscape features may be important determinants of inter-patch movement and habitat utilisation in the patch system. Further empirical studies are needed to explore this aspect of small mammal biology. C 2001 Elsevier Science Ltd. All rights reserved.</div>','<b>Title: </b>Reverse-engineering of polynomial dynamical systems<br><b>Year: </b>2007<br><b>Author: </b> As. Jarrah,  R. Laubenbacher,  B. Stigler and  M. Stillman<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.aam.2006.08.004"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ADVANCES IN APPLIED MATHEMATICS<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Multivariate polynomial dynamical systems over finite fields have been studied in several contexts, including engineering and mathematical biology. An important problem is to construct models of such systems frorn a partial specification of dynamic properties, e.g., from a collection of state transition measurements. Here, we consider static models, which are directed graphs that represent the causal relationships between system variables, so-called wiring diagrams. This paper contains an algorithm which computes all possible minimal wiring diagrams for a given set of state transition measurements. The paper also contains several statistical measures for model selection. The algorithm uses primary decomposition of monomial ideals as the principal tool. An application to the reverse-engineering of a gene regulatory network is included. The algorithm and the statistical measures are implemented in Macaulay 2, and are available from the authors. C 2006 Elsevier Inc. All rights reserved.</div>','<b>Title: </b>An Evaluation of Methods for Inferring Boolean Networks from Time-Series Data<br><b>Year: </b>2013<br><b>Author: </b> N. Berestovsky and  L. Nakhleh<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0066031"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>regulatory networks, gene regulatory, gene expression, network inference, expression data, network structure, gene networks, networks grns, regulatory genes, boolean networks, bayesian network, network reconstruction, networks gene, network dynamics, gene regulation, inference method, systems biology, time series, infer gene, network<div style="font-size:11px;"><br><b>Abstract: </b><em>Regulatory networks play a central role in cellular behavior and decision making. Learning these regulatory networks is a major task in biology, and devising computational methods and mathematical models for this task is a major endeavor in bioinformatics. Boolean networks have been used extensively for modeling regulatory networks. In this model, the state of each gene can be either on or off and that next-state of a gene is updated, synchronously or asynchronously, according to a Boolean rule that is applied to the current-state of the entire system. Inferring a Boolean network from a set of experimental data entails two main steps: first, the experimental time-series data are discretized into Boolean trajectories, and then, a Boolean network is learned from these Boolean trajectories. In this paper, we consider three methods for data discretization, including a new one we propose, and three methods for learning Boolean networks, and study the performance of all possible nine combinations on four regulatory systems of varying dynamics complexities. We find that employing the right combination of methods for data discretization and network learning results in Boolean networks that capture the dynamics well and provide predictive power. Our findings are in contrast to a recent survey that placed Boolean networks on the low end of the "faithfulness to biological reality and "ability to model dynamics spectra. Further, contrary to the common argument in favor of Boolean networks, we find that a relatively large number of time points in the time-series data is required to learn good Boolean networks for certain data sets. Last but not least, while methods have been proposed for inferring Boolean networks, as discussed above, missing still are publicly available implementations thereof. Here, we make our implementation of the methods available publicly in open source at http://bioinfo.cs.rice.edu/.</div>','<b>Title: </b>In Silico Predictions of hERG Channel Blockers in Drug Discovery: From Ligand-Based and Target-Based Approaches to Systems Chemical Biology<br><b>Year: </b>2011<br><b>Author: </b> O. Taboureau and  Fs. Jorgensen<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.2174/138620711795508322"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>COMBINATORIAL CHEMISTRY  HIGH THROUGHPUT SCREENING<br><b>Cluster-Keywords: </b><em>gene expression, systems biology, metabolic network, network model, network analysis, dynamic model, signaling pathways, signaling network, regulatory network, dynamic network, biology approach, candidate genes, gene network, gene regulatory, biology analysis, mathematical model, network dynamic, biology network, metabolic pathways, pathway analysis<div style="font-size:11px;"><br><b>Abstract: </b><em>The risk for cardiotoxic side effects represents a major problem in clinical studies of drug candidates and regulatory agencies have explicitly recommended that all new drug candidates should be tested for blockage of the human Ether-a-go-go Related-Gene hERG potassium channel. Indeed, several drugs with different therapeutic indications and recognized as hERG blockers were recently withdrawn due to the risk of QT prolongation, arrhythmia and Torsade de Pointes.In silico techniques can provide a priori knowledge of hERG blockers, thus reducing the costs associated with screening assays. Significant progress has been made in structure-based and ligand-based drug design and a number of models have been developed to predict hERG blockage.Although approaches such as homology modeling in combination with docking and molecular dynamics bring us closer to understand the drug-channel interactions whereas QSAR and classification models provide a faster assessment and detection of hERG-related drug toxicity, limitation on the applicability domain of the current models and integration of data from diverse in vitro approaches are still issues to challenge.Therefore, this review will emphasize on current methods to predict hERG blockers and the need of consistent data to improve the quality and performance of the in silico models. Finally, integration of network-based analysis on drugs inducing potentially long-QT syndrome and arrhythmia will be discussed as a new perspective for a better understanding of the drug responses in systems chemical biology.</div>','<b>Title: </b>Predicting whole genome protein interaction networks from primary sequence data in model and non-model organisms using ENTS<br><b>Year: </b>2013<br><b>Author: </b> E. Rodgers-Melnick,  M. Culp and  Sp. Difazio<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1471-2164-14-608"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC GENOMICS<br><b>Cluster-Keywords: </b><em>interaction networks, proteinprotein interactions, protein interaction, network model, complex systems, systems biology, protein protein, protein complex, host network, dynamic interactions, network analysis, biological systems, networks network, cellular network, biological network, viral proteins, interactome networks, regulatory networks, immune system, systems network<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: The large-scale identification of physical protein-protein interactions PPIs is an important step toward understanding how biological networks evolve and generate emergent phenotypes. However, experimental identification of PPIs is a laborious and error-prone process, and current methods of PPI prediction tend to be highly conservative or require large amounts of functional data that may not be available for newly-sequenced organisms.Results: In this study we demonstrate a random-forest based technique, ENTS, for the computational prediction of protein-protein interactions based only on primary sequence data. Our approach is able to efficiently predict interactions on a whole-genome scale for any eukaryotic organism, using pairwise combinations of conserved domains and predicted subcellular localization of proteins as input features. We present the first predicted interactome for the forest tree Populus trichocarpa in addition to the predicted interactomes for Saccharomyces cerevisiae, Homo sapiens, Mus musculus, and Arabidopsis thaliana. Comparing our approach to other PPI predictors, we find that ENTS performs comparably to or better than a number of existing approaches, including several that utilize a variety of functional information for their predictions. We also find that the predicted interactions are biologically meaningful, as indicated by similarity in functional annotations and enrichment of co-expressed genes in public microarray datasets. Furthermore, we demonstrate some of the biological insights that can be gained from these predicted interaction networks. We show that the predicted interactions yield informative groupings of P. trichocarpa metabolic pathways, literature-supported associations among human disease states, and theory-supported insight into the evolutionary dynamics of duplicated genes in paleopolyploid plants.Conclusion: We conclude that the ENTS classifier will be a valuable tool for the de novo annotation of genome sequences, providing initial clues about regulatory and metabolic network topology, and revealing relationships that are not immediately obvious from traditional homology-based annotations.</div>','<b>Title: </b>Autoregulation of regulatory proteins is key for dynamic operation of GAL switch in Saccharomyces cerevisiae<br><b>Year: </b>2004<br><b>Author: </b> A. Ruhela,  M. Verma,  Js. Edwards,  Pj. Bhat,  S. Bhartiya and  Kv. Venkatesh<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.febslet.2004.09.001"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>FEBS LETTERS<br><b>Cluster-Keywords: </b><em>systems biology, regulatory network, interaction networks, network model, gene regulatory, network, network biology, networks, gene network, response networks, network dynamics, dynamic model, cell networks, gene expression, biological networks, system dynamics, dynamic network, model, biological systems, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Autoregulation and nucleocytoplasmic shuttling play important roles in the operation of the GAL regulatory system. However, the significance of these mechanisms in the overall operation of the switch is unclear. In this work, we develop a dynamic model for the GAL system and further validate the same using steady-state and dynamic experimental expression data. Next, the model is used to delineate the relevance of shuttling and autoregulation in response to inducing, repressing, and non-inducing-non-repressing media. The analysis indicates that autoregulation of the repressor, Gal80p, is key in obtaining three distinct: steady states in response to the three media. In particular, the analysis rationalizes the intuitively paradoxical observation that the concentration of repressor, Gal80p, actually increases in response to an increase in the inducer concentration. On the other hand, although nucleocytoplasmic shuttling does not affect the dynamics of the system, it plays a dominant role in obtaining a sensitive response to galactose. The dynamic model was also used to obtain insights on the preculturing effect on the system behavior. C 2004 Federation of European Biochemical Societies. Published by Elsevier B.V. All rights reserved.</div>','<b>Title: </b>On the mean field approximation of a stochastic model of tumour-induced angiogenesis<br><b>Year: </b>2019<br><b>Author: </b> V. Capasso and  F. Flandoli<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1017/S0956792518000347"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>EUROPEAN JOURNAL OF APPLIED MATHEMATICS<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>In the field of Life Sciences, it is very common to deal with extremely complex systems, from both analytical and computational points of view, due to the unavoidable coupling of different interacting structures. As an example, angiogenesis has revealed to be an highly complex, and extremely interesting biomedical problem, due to the strong coupling between the kinetic parameters of the relevant branching - growth - anastomosis stochastic processes of the capillary network, at the microscale, and the family of interacting underlying biochemical fields, at the macroscale. In this paper, an original revisited conceptual stochastic model of tumour-driven angiogenesis has been proposed, for which it has been shown that it is possible to reduce complexity by taking advantage of the intrinsic multiscale structure of the system; one may keep the stochasticity of the dynamics of the vessel tips at their natural microscale, whereas the dynamics of the underlying fields is given by a deterministic mean field approximation obtained by an averaging at a suitable mesoscale. While in previous papers, only an heuristic justification of this approach had been offered; in this paper, a rigorous proof is given of the so called propagation of chaos, which leads to a mean field approximation of the stochastic relevant measures associated with the vessel dynamics, and consequently of the underlying tumour angiogenic factor TAF field. As a side, though important result, the non-extinction of the random process of tips has been proven during any finite time interval.</div>','<b>Title: </b>Abstracting the dynamics of biological pathways using information theory: a case study of apoptosis pathway<br><b>Year: </b>2017<br><b>Author: </b> Sk. Palaniappan,  F. Bertaux,  M. Pichene,  E. Fabre,  G. Batt and  B. Genest<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1093/bioinformatics/btx095"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>reaction networks, network model, systems biology, biochemical reaction, stochastic reaction, model reduction, stochastic model, network dynamics, stochastic biochemical, stochastic simulation, system size, chemical reaction, gene network, elementary reactions, biochemical networks, parameter identification, parameter inference, cell populations, small system, linear noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Motivation: Quantitative models are increasingly used in systems biology. Usually, these quantitative models involve many molecular species and their associated reactions. When simulating a tissue with thousands of cells, using these large models becomes computationally and time limiting.Results: In this paper, we propose to construct abstractions using information theory notions. Entropy is used to discretize the state space and mutual information is used to select a subset of all original variables and their mutual dependencies. We apply our method to an hybrid model of TRAIL-induced apoptosis in HeLa cell. Our abstraction, represented as a Dynamic Bayesian Network DBN, reduces the number of variables from 92 to 10, and accelerates numerical simulation by an order of magnitude, yet preserving essential features of cell death time distributions.</div>','<b>Title: </b>Graph spectral analysis of protein interaction network evolution<br><b>Year: </b>2012<br><b>Author: </b> T. Thorne and  Mph. Stumpf<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1098/rsif.2012.0220"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF THE ROYAL SOCIETY INTERFACE<br><b>Cluster-Keywords: </b><em>circadian clock, biological systems, network models, mathematical models, clock gene, systems biology, model selection, biological networks, network data, experimental data, model parameters, network dynamics, plant circadian, regulatory network, clock model, plant clock, complex network, clock circuit, approximate bayesian, stochastic models<div style="font-size:11px;"><br><b>Abstract: </b><em>We present an analysis of protein interaction network data via the comparison of models of network evolution to the observed data. We take a Bayesian approach and perform posterior density estimation using an approximate Bayesian computation with sequential Monte Carlo method. Our approach allows us to perform model selection over a selection of potential network growth models. The methodology we apply uses a distance defined in terms of graph spectra which captures the network data more naturally than previously used summary statistics such as the degree distribution. Furthermore, we include the effects of sampling into the analysis, to properly correct for the incompleteness of existing datasets, and have analysed the performance of our method under various degrees of sampling. We consider a number of models focusing not only on the biologically relevant class of duplication models, but also including models of scale-free network growth that have previously been claimed to describe such data. We find a preference for a duplication-divergence with linear preferential attachment model in the majority of the interaction datasets considered. We also illustrate how our method can be used to perform multi-model inference of network parameters to estimate properties of the full network from sampled data.</div>','<b>Title: </b>Quantifying the effects of dynamical noise on the predictability of a simple ecosystem model<br><b>Year: </b>2004<br><b>Author: </b> Ba. Bailey,  Sc. Doney and  Id. Lima<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1002/env.645"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ENVIRONMETRICS<br><b>Cluster-Keywords: </b><em>network ecology, food webs, network models, ecological systems, food web, systems biology, ecological networks, ecosystem model, signalling networks, model results, interaction networks, dynamic model, ecological interactions, network analysis, model network, network structure, transport networks, systems network, ecosystem dynamics, restoration ecology<div style="font-size:11px;"><br><b>Abstract: </b><em>The need to understand the effects of anthropogenic perturbations on ocean biology has renewed interest in ecosystem and biogeochemical models in recent years. We develop a nonlinear time series approach to quantify the effects of different types of noise on ecosystem dynamics. Different types of noise can alter the local predictability of the system, induce qualitative regime shifts in model dynamics, and destroy create internal nonlinear oscillations and chaos. The ecosystem model we use in our article is a model of plankton dynamics and nitrogen cycling. It is a compartmental model NPZD consisting of compartments for nitrogen N, phytoplankton P, zooplankton Z, and detritus D. The flows or intercompartmental exchanges are modeled as a nonlinear system of four first-order differential equations. The types of noise of interest are both independent and correlated. Because the noise is an integral part of the systems dynamics, a nonlinear time series approach is used to quantify the dynamics and predictability of the system. This involves fitting neural network models and estimating dynamical system quantities of interest such as global and local Lyapunov exponents, along with measures of uncertainty for these estimates. Copyright C 2004 John Wiley Sons, Ltd.</div>','<b>Title: </b>Understanding the apparent stator-rotor connections in the rotary ATPase family using coarse-grained computer modeling<br><b>Year: </b>2014<br><b>Author: </b> Ra. Richardson,  K. Papachristos,  Dj. Read,  Og. Harlen,  M. Harrison,  E. Paci,  Sp. Muench and  Sa. Harris<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1002/prot.24680"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PROTEINS-STRUCTURE FUNCTION AND BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>normal mode, mode analysis, elastic network, functional motions, functional dynamics, molecular chaperone, structural biology, acidic residues, network models, molecular dynamics, atomic resolution, protein complexes, conserved functional, hsp dynamics, correlated motions, protein folding, large biomolecular, thioredoxin system, harmonic analysis, protein motions<div style="font-size:11px;"><br><b>Abstract: </b><em>Advances in structural biology, such as cryo-electron microscopy cryo-EM have allowed for a number of sophisticated protein complexes to be characterized. However, often only a static snapshot of a protein complex is visualized despite the fact that conformational change is frequently inherent to biological function, as is the case for molecular motors. Computer simulations provide valuable insights into the different conformations available to a particular system that are not accessible using conventional structural techniques. For larger proteins and protein complexes, where a fully atomistic description would be computationally prohibitive, coarse-grained simulation techniques such as Elastic Network Modeling ENM are often employed, whereby each atom or group of atoms is linked by a set of springs whose properties can be customized according to the system of interest. Here we compare ENM with a recently proposed continuum model known as Fluctuating Finite Element Analysis FFEA, which represents the biomolecule as a viscoelastic solid subject to thermal fluctuations. These two complementary computational techniques are used to answer a critical question in the rotary ATPase family; implicit within these motors is the need for a rotor axle and proton pump to rotate freely of the motor domain and stator structures. However, current single particle cryo-EM reconstructions have shown an apparent connection between the stators and rotor axle or pump region, hindering rotation. Both modeling approaches show a possible role for this connection and how it would significantly constrain the mobility of the rotary ATPase family. Proteins 2014; 82:3298-3311. c 2014 The Authors. Proteins: Structure, Function, and Bioinformatics Published by Wiley Periodicals, Inc.</div>','<b>Title: </b>Descriptive vs. Mechanistic Network Models in Plant Development in the Post-Genomic Era<br><b>Year: </b>2015<br><b>Author: </b> J. Davila-Velderrain,  Jc. Martinez-Garcia and  Er. Alvarez-Buylla<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/978-1-4939-2444-823"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLANT FUNCTIONAL GENOMICS: METHODS AND PROTOCOLS, 2ND EDITION<br><b>Cluster-Keywords: </b><em>gene regulatory, regulatory network, systems biology, cell differentiation, cell fate, model systems, potential landscape, network grn, dynamical systems, melanocyte differentiation, network models, gene expression, network dynamics, differentiation genes, dynamical models, dynamical networks, cellular differentiation, stem cells, attractor network, developmental gene<div style="font-size:11px;"><br><b>Abstract: </b><em>Network modeling is now a widespread practice in systems biology, as well as in integrative genomics, and it constitutes a rich and diverse scientific research field. A conceptually clear understanding of the reasoning behind the main existing modeling approaches, and their associated technical terminologies, is required to avoid confusions and accelerate the transition towards an undeniable necessary more quantitative, multi-disciplinary approach to biology. Herein, we focus on two main network-based modeling approaches that are commonly used depending on the information available and the intended goals: inference-based methods and system dynamics approaches. As far as data-based network inference methods are concerned, they enable the discovery of potential functional influences among molecular components. On the other hand, experimentally grounded network dynamical models have been shown to be perfectly suited for the mechanistic study of developmental processes. How do these two perspectives relate to each other In this chapter, we describe and compare both approaches and then apply them to a given specific developmental module. Along with the step-by-step practical implementation of each approach, we also focus on discussing their respective goals, utility, assumptions, and associated limitations. We use the gene regulatory network GRN involved in Arabidopsis thaliana Root Stem Cell Niche patterning as our illustrative example. We show that descriptive models based on functional genomics data can provide important background information consistent with experimentally supported functional relationships integrated in mechanistic GRN models. The rationale of analysis and modeling can be applied to any other well-characterized functional developmental module in multicellular organisms, like plants and animals.</div>','<b>Title: </b>The HPA axis and ethanol: a synthesis of mathematical modelling and experimental observations<br><b>Year: </b>2017<br><b>Author: </b> Z. Cupic,  A. Stanojevic,  Vm. Markovic,  L. Kolar-Anic,  L. Terenius and  V. Vukojevic<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1111/adb.12409"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ADDICTION BIOLOGY<br><b>Cluster-Keywords: </b><em>circadian rhythms, systems biology, biological clock, circadian clock, biological rhythms, hpa axis, model, clock network, mathematical models, biological systems, dynamic systems, ode models, biology models, single cell, network, model parameters, experimental data, models, systems, dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>Stress and alcohol use are interrelatedstress contributes to the initiation and upholding of alcohol use and alcohol use alters the way we perceive and respond to stress. Intricate mechanisms through which ethanol alters the organisms response to stress remain elusive. We have developed a stoichiometric network model to succinctly describe neurochemical transformations underlying the stress response axis and use numerical simulations to model ethanol effects on complex daily changes of blood levels of cholesterol, 6 peptide and 8 steroid hormones. Modelling suggests that ethanol alters the dynamical regulation of hypothalamic-pituitary-adrenal HPA axis activity by affecting the amplitude of ultradian oscillations of HPA axis hormones, which defines the threshold with respect to which the response to stress is being set. These effects are complexlow/moderate acute ethanol challenge < 8mM may reduce, leave unaltered or increase the amplitude of ultradian cortisol CORT oscillations, giving rise to an intricate response at the organism level, offering also a potential explanation as to why apparently discordant results were observed in experimental studies. In contrast, high-dose acute ethanol challenge > 8mM increases instantaneous CORT levels and the amplitude of ultradian CORT oscillations in a dose-dependent manner, affecting the HPA axis activity also during the following days. Chronic exposure to ethanol qualitatively changes the HPA axis dynamics, whereas ethanol at intoxicating levels shuts down this dynamic regulation mechanism. Mathematical modelling gives a quantitative biology-based framework that can be used for predicting how the integral HPA axis response is perturbed by alcohol.</div>','<b>Title: </b>Computational modelling and analysis of the molecular network regulating sporulation initiation in Bacillus subtilis<br><b>Year: </b>2014<br><b>Author: </b> Aec. Ihekwaba,  I. Mura and  Gc. Barker<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/s12918-014-0119-x"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Bacterial spores are important contaminants in food, and the spore forming bacteria are often implicated in food safety and food quality considerations. Spore formation is a complex developmental process involving the expression of more than 500 genes over the course of 6 to 8 hrs. The process culminates in the formation of resting cells capable of resisting environmental extremes and remaining dormant for long periods of time, germinating when conditions promote further vegetative growth. Experimental observations of sporulation and germination are problematic and time consuming so that reliable models are an invaluable asset in terms of prediction and risk assessment. In this report we develop a model which assists in the interpretation of sporulation dynamics.Results: This paper defines and analyses a mathematical model for the network regulating Bacillus subtilis sporulation initiation, from sensing of sporulation signals down to the activation of the early genes under control of the master regulator Spo0A. Our model summarises and extends other published modelling studies, by allowing the user to execute sporulation initiation in a scenario where Isopropyl beta-D-1-thiogalactopyranoside IPTG is used as an artificial sporulation initiator as well as in modelling the induction of sporulation in wild-type cells. The analysis of the model results and the comparison with experimental data indicate that the model is good at predicting inducible responses to sporulation signals. However, the model is unable to reproduce experimentally observed accumulation of phosphorelay sporulation proteins in wild type B. subtilis. This model also highlights that the phosphorelay sub-component, which relays the signals detected by the sensor kinases to the master regulator Spo0A, is crucial in determining the response dynamics of the system.Conclusion: We show that there is a complex connectivity between the phosphorelay features and the master regulatory Spo0A. Additional we discovered that the experimentally observed regulation of the phosphotransferase Spo0B for wild-type B. subtilis may be playing an important role in the network which suggests that modelling of sporulation initiation may require additional experimental support.</div>','<b>Title: </b>Resonances and noise in a stochastic Hindmarsh-Rose model of thalamic neurons<br><b>Year: </b>2003<br><b>Author: </b> S. Reinker,  E. Puil and  Rm. Miura<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/S0092-82400300026-0"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BULLETIN OF MATHEMATICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>coupling strength, synchronization transitions, timeperiodic coupling, time delay, logic network, distribution network, network model, oscillatory networks, neuron networks, water distribution, stochastic resonance, gene networks, multiple synchronization, firing synchronization, neuronal networks, firing behaviors, logical networks, results show, regulatory networks, cell polarization<div style="font-size:11px;"><br><b>Abstract: </b><em>Thalamic neurons exhibit subthreshold resonance when stimulated with small sine wave signals of varying frequency and stochastic resonance when noise is added to these signals. We study a stochastic Hindmarsh-Rose model using Monte-Carlo simulations to investigate how noise, in conjunction with subthreshold resonance, leads to a preferred frequency in the firing pattern. The resulting stochastic resonance SR exhibits a preferred firing frequency that is approximately exponential in its dependence on the noise amplitude. In similar experiments, frequency dependent SR is found in the reliability of detection of alpha-function inputs under noise, which are more realistic inputs for neurons. A mathematical analysis of the equations reveals that the frequency preference arises from the dynamics of the slow variable. Noise can then transfer the resonance over the firing threshold because of the proximity of the fast subsystem to a Hopf bifurcation point. Our results may have implications for the behavior of thalamic neurons in a network, with noise switching the membrane potential between different resonance modes. C 2003 Society for Mathematical Biology. Published by Elsevier Science Ltd. All rights reserved.</div>','<b>Title: </b>A Statistical Approach Reveals Designs for the Most Robust Stochastic Gene Oscillators<br><b>Year: </b>2016<br><b>Author: </b> Ml. Woods,  M. Leon,  R. Perez-Carrasco and  Cp. Barnes<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1021/acssynbio.5b00179"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ACS SYNTHETIC BIOLOGY<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>The engineering of transcriptional networks presents many challenges due to the inherent uncertainty in the system structure, changing cellular context, and stochasticity in the governing dynamics. One approach to address these problems is to design and build systems that can function across a range of conditions; that is they are robust to uncertainty in their constituent components. Here we examine the parametric robustness landscape of transcriptional oscillators, which underlie many important processes such as circadian rhythms and the cell cycle, plus also serve as a model for the engineering of complex and emergent phenomena. The central questions that we address are: Can we build genetic oscillators that are more robust than those already constructed Can, we make genetic oscillators arbitrarily robust These questions are technically challenging due to the large model and parameter spaces that must be efficiently explored. Here we use a measure of robustness that coincides with the Bayesian model evidence, combined with an efficient Monte Carlo method to traverse model space and concentrate on regions of high robustness, which enables the accurate evaluation of the relative robustness of gene network models governed by stochastic dynamics. We report the most robust two and three gene oscillator systems, plus examine how the number of interactions, the presence of autoregulation, and degradation of mRNA and protein affects the frequency, amplitude, and robustness of transcriptional oscillators. We also find that there is a limit to parametric robustness, beyond which there is nothing to be gained by adding additional feedback. Importantly, we provide predictions on new oscillator systems that can be constructed to verify the theory and advance design and modeling approaches to systems and synthetic biology.</div>','<b>Title: </b>Data-Modeling Identifies Conflicting Signaling Axes Governing Myoblast Proliferation and Differentiation Responses to Diverse Ligand Stimuli<br><b>Year: </b>2017<br><b>Author: </b> Am. Loiben,  S. Soueid-Baumgarten,  Rf. Kopyto,  D. Bhattacharya,  Jc. Kim and  Bd. Cosgrove<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s12195-017-0508-5"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>CELLULAR AND MOLECULAR BIOENGINEERING<br><b>Cluster-Keywords: </b><em>stem cell, stem cells, systems biology, mathematical models, cell biology, network model, signaling pathways, cell fate, computational models, cancer cells, gene expression, network dynamics, embryonic stem, signaling network, prostate cancer, mathematical modeling, psychological stress, apoptosis signaling, computational systems, molecular mechanisms<div style="font-size:11px;"><br><b>Abstract: </b><em>Skeletal muscle tissue development and regeneration relies on the proliferation, maturation and fusion of muscle progenitor cells myoblasts, which arise transiently from muscle stem cells satellite cells. Following muscle damage, myoblasts proliferate and differentiate in response to temporally-varying inflammatory cytokines, growth factors, and extracellular matrix cues, which stimulate a shared network of intracellular signaling pathways. Here we present an integrated data-modeling approach to elucidate synergies and antagonisms among proliferation and differentiation signaling axes in myoblasts stimulated by regeneration-associated ligands.We treated mouse primary myoblasts in culture with combinations of eight regeneration-associated growth factors and cytokines in mixtures that induced additive, synergistic, and antagonistic effects on myoblast proliferation and differentiation responses. For these combinatorial stimuli, we measured the activation dynamics of seven signal transduction pathways using multiplexed phosphoprotein assays and scored proliferation and differentiation responses based on expression of myogenic commitment factors to assemble a cue-signaling-response data compendium. We interrogated the relationship between these signals and responses by partial least-squares PLS regression modeling.Partial least-squares data-modeling accurately predicted response outcomes in cross-validation on the training compendium cumulative R 2 = 0.96. The PLS model highlighted signaling axes that distinctly govern myoblast proliferation MEK-ERK, Stat3 and differentiation JNK in response to these combinatorial cues, and we confirmed these signal-response associations with small molecule perturbations. Unexpectedly, we observed that a negative feedback circuit involving the phosphatase DUSP6/MKP-3 auto-regulates MEK-ERK signaling in myoblasts.This data-modeling approach identified conflicting signaling axes that underlie muscle progenitor cell proliferation and differentiation.</div>','<b>Title: </b>Sensory prediction or motor control Application of Marr-Albus type models of cerebellar function to classical conditioning<br><b>Year: </b>2010<br><b>Author: </b> Nf. Lepora,  J. Porrill,  Ch. Yeo and  P. Dean<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3389/fncom.2010.00140"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>FRONTIERS IN COMPUTATIONAL NEUROSCIENCE<br><b>Cluster-Keywords: </b><em>systems biology, dynamic models, biological systems, model, dynamic systems, computational systems, network model, experimental data, model parameters, systems, systems model, models, pathway systems, data, biochemical pathway, dynamics model, biochemical networks, computational models, network, system<div style="font-size:11px;"><br><b>Abstract: </b><em>Marr-Albus adaptive filter models of the cerebellum have been applied successfully to a range of sensory and motor control problems. Here we analyze their properties when applied to classical conditioning of the nictitating membrane response in rabbits. We consider a system-level model of eyeblink conditioning based on the anatomy of the eyeblink circuitry, comprising an adaptive filter model of the cerebellum, a comparator model of the inferior olive and a linear dynamic model of the nictitating membrane plant. To our knowledge, this is the first model that explicitly includes all these principal components, in particular the motor plant that is vital for shaping and timing the behavioral response. Model assumptions and parameters were systematically investigated to disambiguate basic computational capacities of the model from features requiring tuning of properties and parameter values. Without such tuning, the model robustly reproduced a range of behaviors related to sensory prediction, by displaying appropriate trial-level associative learning effects for both single and multiple stimuli, including blocking and conditioned inhibition. In contrast, successful reproduction of the real-time motor behavior depended on appropriate specification of the plant, cerebellum and comparator models. Although some of these properties appear consistent with the system biology, fundamental questions remain about how the biological parameters are chosen if the cerebellar microcircuit applies a common computation to many distinct behavioral tasks. It is possible that the response profiles in classical conditioning of the eyeblink depend upon operant contingencies that have previously prevailed, for example in naturally occurring avoidance movements.</div>','<b>Title: </b>Topology analysis and visualization of Potyvirus protein-protein interaction network<br><b>Year: </b>2014<br><b>Author: </b> G. Bosque,  A. Folch-Fortuny,  J. Pico,  A. Ferrer and  Sf. Elena<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/s12918-014-0129-8"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>interaction networks, proteinprotein interactions, protein interaction, network model, complex systems, systems biology, protein protein, protein complex, host network, dynamic interactions, network analysis, biological systems, networks network, cellular network, biological network, viral proteins, interactome networks, regulatory networks, immune system, systems network<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: One of the central interests of Virology is the identification of host factors that contribute to virus infection. Despite tremendous efforts, the list of factors identified remains limited. With omics techniques, the focus has changed from identifying and thoroughly characterizing individual host factors to the simultaneous analysis of thousands of interactions, framing them on the context of protein-protein interaction networks and of transcriptional regulatory networks. This new perspective is allowing the identification of direct and indirect viral targets. Such information is available for several members of the Potyviridae family, one of the largest and more important families of plant viruses.Results: After collecting information on virus protein-protein interactions from different potyviruses, we have processed it and used it for inferring a protein-protein interaction network. All proteins are connected into a single network component. Some proteins show a high degree and are highly connected while others are much less connected, with the network showing a significant degree of dissortativeness. We have attempted to integrate this virus protein-protein interaction network into the largest protein-protein interaction network of Arabidopsis thaliana, a susceptible laboratory host. To make the interpretation of data and results easier, we have developed a new approach for visualizing and analyzing the dynamic spread on the host network of the local perturbations induced by viral proteins. We found that local perturbations can reach the entire host protein-protein interaction network, although the efficiency of this spread depends on the particular viral proteins. By comparing the spread dynamics among viral proteins, we found that some proteins spread their effects fast and efficiently by attacking hubs in the host network while other proteins exert more local effects.Conclusions: Our findings confirm that potyvirus protein-protein interaction networks are highly connected, with some proteins playing the role of hubs. Several topological parameters depend linearly on the protein degree. Some viral proteins focus their effect in only host hubs while others diversify its effect among several proteins at the first step. Future new data will help to refine our model and to improve our predictions.</div>','<b>Title: </b>A new discrete dynamic model of ABA-induced stomatal closure predicts key feedback loops<br><b>Year: </b>2017<br><b>Author: </b> R. Albert,  Br. Acharya,  Bw. Jeon,  Jgt. Zanudo,  Mm. Zhu,  K. Osman and  Sm. Assmann<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pbio.2003451"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS BIOLOGY<br><b>Cluster-Keywords: </b><em>network ecology, food webs, network models, ecological systems, food web, systems biology, ecological networks, ecosystem model, signalling networks, model results, interaction networks, dynamic model, ecological interactions, network analysis, model network, network structure, transport networks, systems network, ecosystem dynamics, restoration ecology<div style="font-size:11px;"><br><b>Abstract: </b><em>Stomata, microscopic pores in leaf surfaces through which water loss and carbon dioxide uptake occur, are closed in response to drought by the phytohormone abscisic acid ABA. This process is vital for drought tolerance and has been the topic of extensive experimental investigation in the last decades. Although a core signaling chain has been elucidated consisting of ABA binding to receptors, which alleviates negative regulation by protein phosphatases 2C PP2Cs of the protein kinase OPEN STOMATA 1 OST1 and ultimately results in activation of anion channels, osmotic water loss, and stomatal closure, over 70 additional components have been identified, yet their relationships with each other and the core components are poorly elucidated. We integrated and processed hundreds of disparate observations regarding ABA signal transduction responses underlying stomatal closure into a network of 84 nodes and 156 edges and, as a result, established those relationships, including identification of a 36-node, strongly connected feedback-rich component as well as its in-and out-components. The networks domination by a feedback-rich component may reflect a general feature of rapid signaling events. We developed a discrete dynamic model of this network and elucidated the effects of ABA plus knockout or constitutive activity of 79 nodes on both the outcome of the system closure and the status of all internal nodes. The model, with more than 1024 system states, is far from fully determined by the available data, yet model results agree with existing experiments in 82 cases and disagree in only 17 cases, a validation rate of 75. Our results reveal nodes that could be engineered to impact stomatal closure in a controlled fashion and also provide over 140 novel predictions for which experimental data are currently lacking. Noting the paucity of wet-bench data regarding combinatorial effects of ABA and internal node activation, we experimentally confirmed several predictions of the model with regard to reactive oxygen species, cytosolic Ca2 Ca2 c, and heterotrimeric G-protein signaling. We analyzed dynamics-determining positive and negative feedback loops, thereby elucidating the attractor dynamic behavior repertoire of the system and the groups of nodes that determine each attractor. Based on this analysis, we predict the likely presence of a previously unrecognized feedback mechanism dependent on Ca2 c. This mechanism would provide model agreement with 10 additional experimental observations, for a validation rate of 85. Our research underscores the importance of feedback regulation in generating robust and adaptable biological responses. The high validation rate of our model illustrates the advantages of discrete dynamic modeling for complex, nonlinear systems common in biology.</div>','<b>Title: </b>A mathematical model of microbial folate biosynthesis and utilisation: implications for antifolate development<br><b>Year: </b>2016<br><b>Author: </b> Je. Salcedo-Sora and  Mt. Mc Auley<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1039/c5mb00801h"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MOLECULAR BIOSYSTEMS<br><b>Cluster-Keywords: </b><em>gene expression, systems biology, metabolic network, network model, network analysis, dynamic model, signaling pathways, signaling network, regulatory network, dynamic network, biology approach, candidate genes, gene network, gene regulatory, biology analysis, mathematical model, network dynamic, biology network, metabolic pathways, pathway analysis<div style="font-size:11px;"><br><b>Abstract: </b><em>The metabolic biochemistry of folate biosynthesis and utilisation has evolved into a complex network of reactions. Although this complexity represents challenges to the field of folate research it has also provided a renewed source for antimetabolite targets. A range of improved folate chemotherapy continues to be developed and applied particularly to cancer and chronic inflammatory diseases. However, new or better antifolates against infectious diseases remain much more elusive. In this paper we describe the assembly of a generic deterministic mathematical model of microbial folate metabolism. Our aim is to explore how a mathematical model could be used to explore the dynamics of this inherently complex set of biochemical reactions. Using the model it was found that: 1 a particular small set of folate intermediates are overrepresented, 2 inhibitory profiles can be quantified by the level of key folate products, 3 using the model to scan for the most effective combinatorial inhibitions of folate enzymes we identified specific targets which could complement current antifolates, and 4 the model substantiates the case for a substrate cycle in the folinic acid biosynthesis reaction. Our model is coded in the systems biology markup language and has been deposited in the BioModels Database MODEL1511020000, this makes it accessible to the community as a whole.</div>','<b>Title: </b>Percolation in bipartite Boolean networks and its role in sustaining life<br><b>Year: </b>2019<br><b>Author: </b> R. Hannam,  R. Kuhn and  A. Annibale<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1088/1751-8121/ab3053"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF PHYSICS A-MATHEMATICAL AND THEORETICAL<br><b>Cluster-Keywords: </b><em>boolean networks, regulatory networks, network model, bayesian networks, model networks, systems biology, random boolean, real cells, immune system, genetic regulatory, networks, biochemical network, fluorescence images, cellular network, gene regulatory, cell differentiation, network, biological systems, ensemble approach, cell dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>Boolean networks are popular models for gene regulation, where genes are regarded as binary units, that can be either expressed or not, each updated at regular time intervals according to a random Boolean function of its neighbouring genes. Stable gene expression profiles, corresponding to cell types, are regarded as attractors of the network dynamics. However, the random character of gene updates does not allow to link explicitly the existence of attractors to the biological mechanism with which genes interact. We propose a bipartite Boolean network approach which integrates genes and regulatory proteins i.e. transcription factors TFs into a single network, where interactions incorporate two fundamental aspects of cellular biology, i.e. gene expression and gene regulation, and the resulting dynamics is highly non-linear. Since any finite stochastic system is ergodic, the emergence of an attractor structure, stable under noisy conditions, requires a giant component in the bipartite graph. By adapting graph percolation techniques to directed bipartite graphs, we are able to calculate exactly the region, in the network parameters space, where a cell can sustain steady-state gene expression profiles, in the absence of inhibitors, and we quantify numerically the effect of inhibitors. Results show that for cells to sustain a steady-state gene expression profile, TFs should typically be small protein complexes that regulate many genes. This condition is crucial for cell reprogramming and remarkably well in line with biological facts.</div>','<b>Title: </b>Dynamics of amino acid metabolism of primary human liver cells in 3D bioreactors<br><b>Year: </b>2006<br><b>Author: </b> R. Guthke,  K. Zeilinger,  S. Sickinger,  W. Schmidt-Heck,  H. Buentemeyer,  K. Iding,  J. Lehmann,  M. Pfaff,  G. Pless and  Jc. Gerlach<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s00449-005-0040-1"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOPROCESS AND BIOSYSTEMS ENGINEERING<br><b>Cluster-Keywords: </b><em>petri net, petri nets, bayesian networks, biological networks, systems biology, regulatory networks, dynamic bayesian, net model, gene regulatory, network data, biological systems, biological data, network analysis, network models, biochemical networks, time series, network structure, signal transduction, net models, dbn model<div style="font-size:11px;"><br><b>Abstract: </b><em>The kinetics of 18 amino acids, ammonia NH3 and urea UREA in 18 liver cell bioreactor runs were analyzed and simulated by a two-compartment model consisting of a system of 42 differential equations. The model parameters, most of them representing enzymatic activities, were identified and their values discussed with respect to the different liver cell bioreactor performance levels. The nitrogen balance based model was used as a tool to quantify the variability of runs and to describe different kinetic patterns of the amino acid metabolism, in particular with respect to glutamate GLU and aspartate ASP.</div>','<b>Title: </b>Exploring large macromolecular functional motions on clusters of multicore processors<br><b>Year: </b>2013<br><b>Author: </b> Jr. Lopez-Blanco,  R. Reyes,  Ji. Aliaga,  Rm. Badia,  P. Chacon and  Es. Quintana-Orti<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.jcp.2013.03.032"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF COMPUTATIONAL PHYSICS<br><b>Cluster-Keywords: </b><em>normal mode, mode analysis, elastic network, functional motions, functional dynamics, molecular chaperone, structural biology, acidic residues, network models, molecular dynamics, atomic resolution, protein complexes, conserved functional, hsp dynamics, correlated motions, protein folding, large biomolecular, thioredoxin system, harmonic analysis, protein motions<div style="font-size:11px;"><br><b>Abstract: </b><em>Normal modes in internal coordinates IC furnish an excellent way to model functional collective motions of macromolecular machines, but exhibit a high computational cost when applied to large-sized macromolecules. In this paper, we significantly extend the applicability of this approach towards much larger systems by effectively solving the computational bottleneck of these methods, the diagonalization step and associated large-scale eigenproblem, on a small cluster of nodes equipped with multicore technology. Our experiments show the superior performance of iterative Krylov-subspace methods for the solution of the dense generalized eigenproblems arising in these biological applications over more traditional direct solvers implemented on top of state-of-the-art libraries. The presented approach expedites the study of the collective conformational changes of large macromolecules opening a new window for exploring the functional motions of such relevant systems. C 2013 Elsevier Inc. All rights reserved.</div>','<b>Title: </b>Decoded Calreticulin-Deficient Embryonic Stem Cell Transcriptome Resolves Latent Cardiophenotype<br><b>Year: </b>2010<br><b>Author: </b> Rs. Faustino,  A. Chiriac,  Nj. Niederlander,  Tj. Nelson,  A. Behfar,  Pk. Mishra,  S. Macura,  M. Michalak,  A. Terzic and  C. Perez-Terzic<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1002/stem.447"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>STEM CELLS<br><b>Cluster-Keywords: </b><em>gene regulatory, systems biology, regulatory networks, gene network, stem cell, network model, transcription factors, gene expression, cell transcriptomes, network analysis, data analysis, stem cells, network grnsight, cell transformation, weighted network, gene ontology, networks grns, graph model, graph layout, endothelial cell<div style="font-size:11px;"><br><b>Abstract: </b><em>Genomic perturbations that challenge normal signaling at the pluripotent stage may trigger unforeseen ontogenic aberrancies. Anticipatory systems biology identification of transcriptome landscapes that underlie latent phenotypes would offer molecular diagnosis before the onset of symptoms. The purpose of this study was to assess the impact of calreticulin-deficient embryonic stem cell transcriptomes on molecular functions and physiological systems. Bioinformatic surveillance of calreticulin-null stem cells, a monogenic insult model, diagnosed a disruption in transcriptome dynamics, which re-prioritized essential cellular functions. Calreticulin-calibrated signaling axes were uncovered, and network-wide cartography of undifferentiated stem cell transcripts suggested cardiac manifestations. Calreticulin-deficient stem cell-derived cardiac cells verified disorganized sarcomerogenesis, mitochondrial paucity, and cytoarchitectural aberrations to validate calreticulin-dependent network forecasts. Furthermore, magnetic resonance imaging and histopathology detected a ventricular septal defect, revealing organogenic manifestation of calreticulin deletion. Thus, bioinformatic deciphering of a primordial calreticulin-deficient transcriptome decoded at the pluripotent stem cell stage a reconfigured multifunctional molecular registry to anticipate predifferentiation susceptibility toward abnormal cardiophenotype. STEM CELLS 2010:28:1281-1291</div>','<b>Title: </b>Multiscale and modular analysis of cardiac energy metabolism - Repairing the broken interfaces of isolated system components<br><b>Year: </b>2008<br><b>Author: </b> Jhgm. Van Beek<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1196/annals.1420.018"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>CONTROL AND REGULATION OF TRANSPORT PHENOMENA IN THE CARDIAC SYSTEM<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Computational models of large molecular systems can be assembled from modules representing biological function emerging from interactions among a small subset of molecules. Experimental information on isolated molecules can be integrated with the response of the network as a whole to estimate crucial missing parameters. As an example, a "skeleton" model is analyzed for the module regulating dynamic adaptation of myocardial oxidative phosphorylation OxPhos to fluctuating cardiac energy demand. The module contains adenine nucleotides, creatine, and phosphate groups. Enzyme kinetic equations for two creatine kinase CK isoforms were combined with the response time of OxPhos tmito; generalized time constant to steps in the cardiac pacing rate to identify all module parameters. To obtain the time course of O-2 uptake was measured for the whole heart. An O-2 transport model was used to deconvolute the whole-heart response to the mitochondrial level. By optimizing mitochondrial outer membrane permeability to 21 mu m/s the experimental tmito = 3.7 s was reproduced. This in vivo value is about four times larger, or smaller, respectively, than conflicting values obtained from two different in vitro studies. This demonstrates an important rule for multiscale analysis: experimental responses and modeling of the system at the larger scale allow one to estimate essential parameters for the interfaces of components which may have been altered during physical isolation. The model correctly predicts a smaller tmito when CK activity is reduced. The model further predicts a slower response if the muscle CK isoform is overexpressed and a faster response if mitochondrial CK is overexpressed. The CK system is very effective in decreasing maximum levels of ADP during systole and reducing average Pi levels over the whole cardiac cycle.</div>','<b>Title: </b>Quorum-Sensing Synchronization of Synthetic Toggle Switches: A Design Based on Monotone Dynamical Systems Theory<br><b>Year: </b>2016<br><b>Author: </b> Ev. Nikolaev and  Ed. Sontag<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pcbi.1004881"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS COMPUTATIONAL BIOLOGY<br><b>Cluster-Keywords: </b><em>gene regulatory, systems biology, synthetic biology, regulatory network, gene network, gene expression, system model, synthetic network, network dynamics, systems theory, synthetic gene, genetic network, model predicts, dynamical systems, time delays, translational time, regulation models, oscillatory expression, network structure, biological systems<div style="font-size:11px;"><br><b>Abstract: </b><em>Synthetic constructs in biotechnology, biocomputing, and modern gene therapy interventions are often based on plasmids or transfected circuits which implement some form of "on-off" switch. For example, the expression of a protein used for therapeutic purposes might be triggered by the recognition of a specific combination of inducers e.g., antigens, and memory of this event should be maintained across a cell population until a specific stimulus commands a coordinated shut-off. The robustness of such a design is hampered by molecular "intrinsic" or environmental "extrinsic" noise, which may lead to spontaneous changes of state in a subset of the population and is reflected in the bimodality of protein expression, as measured for example using flow cytometry. In this context, a "majority-vote" correction circuit, which brings deviant cells back into the required state, is highly desirable, and quorum-sensing has been suggested as a way for cells to broadcast their states to the population as a whole so as to facilitate consensus. In this paper, we propose what we believe is the first such a design that has mathematically guaranteed properties of stability and auto-correction under certain conditions. Our approach is guided by concepts and theory from the field of "monotone" dynamical systems developed by M. Hirsch, H. Smith, and others. We benchmark our design by comparing it to an existing design which has been the subject of experimental and theoretical studies, illustrating its superiority in stability and self-correction of synchronization errors. Our stability analysis, based on dynamical systems theory, guarantees global convergence to steady states, ruling out unpredictable "chaotic" behaviors and even sustained oscillations in the limit of convergence. These results are valid no matter what are the values of parameters, and are based only on the wiring diagram. The theory is complemented by extensive computational bifurcation analysis, performed for a biochemically-detailed and biologically-relevant model that we developed. Another novel feature of our approach is that our theorems on exponential stability of steady states for homogeneous or mixed populations are valid independently of the number N of cells in the population, which is usually very large N >> 1 and unknown. We prove that the exponential stability depends on relative proportions of each type of state only. While monotone systems theory has been used previously for systems biology analysis, the current work illustrates its power for synthetic biology design, and thus has wider significance well beyond the application to the important problem of coordination of toggle switches.</div>','<b>Title: </b>DREAM3: Network Inference Using Dynamic Context Likelihood of Relatedness and the Inferelator<br><b>Year: </b>2010<br><b>Author: </b> A. Madar,  A. Greenfield,  E. Vanden-Eijnden and  R. Bonneau<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0009803"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>regulatory networks, gene regulatory, gene expression, network inference, expression data, network structure, gene networks, networks grns, regulatory genes, boolean networks, bayesian network, network reconstruction, networks gene, network dynamics, gene regulation, inference method, systems biology, time series, infer gene, network<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Many current works aiming to learn regulatory networks from systems biology data must balance model complexity with respect to data availability and quality. Methods that learn regulatory associations based on unit-less metrics, such as Mutual Information, are attractive in that they scale well and reduce the number of free parameters model complexity per interaction to a minimum. In contrast, methods for learning regulatory networks based on explicit dynamical models are more complex and scale less gracefully, but are attractive as they may allow direct prediction of transcriptional dynamics and resolve the directionality of many regulatory interactions.Methodology: We aim to investigate whether scalable information based methods like the Context Likelihood of Relatedness method and more explicit dynamical models like Inferelator 1.0 prove synergistic when combined. We test a pipeline where a novel modification of the Context Likelihood of Relatedness mixed-CLR, modified to use time series data is first used to define likely regulatory interactions and then Inferelator 1.0 is used for final model selection and to build an explicit dynamical model.Conclusions/Significance: Our method ranked 2nd out of 22 in the DREAM3 100-gene in silico networks challenge. Mixed-CLR and Inferelator 1.0 are complementary, demonstrating a large performance gain relative to any single tested method, with precision being especially high at low recall values. Partitioning the provided data set into four groups knock-down, knock-out, time-series, and combined revealed that using comprehensive knock-out data alone provides optimal performance. Inferelator 1.0 proved particularly powerful at resolving the directionality of regulatory interactions, i.e. "who regulates who" approximately 93 of identified true positives were correctly resolved. Performance drops for high in-degree genes, i.e. as the number of regulators per target gene increases, but not with out-degree, i.e. performance is not affected by the presence of regulatory hubs.</div>','<b>Title: </b>Time Series Analysis of the Bacillus subtilis Sporulation Network Reveals Low Dimensional Chaotic Dynamics<br><b>Year: </b>2016<br><b>Author: </b> P. Lecca,  I. Mura,  A. Re,  Gc. Barker and  Aec. Ihekwaba<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3389/fmicb.2016.01760"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>FRONTIERS IN MICROBIOLOGY<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Chaotic behavior refers to a behavior which, albeit irregular, is generated by an underlying deterministic process. Therefore, a chaotic behavior is potentially controllable. This possibility becomes practically amenable especially when chaos is shown to be low-dimensional, i.e., to be attributable to a small fraction of the total systems components. In this case, indeed, including the major drivers of chaos in a system into the modeling approach allows us to improve predictability of the systems dynamics. Here, we analyzed the numerical simulations of an accurate ordinary differential equation model of the gene network regulating sporulation initiation in Bacillus subtilis to explore whether the non-linearity underlying time series data is due to low-dimensional chaos. Low-dimensional chaos is expectedly common in systems with few degrees of freedom, but rare in systems with many degrees of freedom such as the B. subtilis sporulation network. The estimation of a number of indices, which reflect the chaotic nature of a system, indicates that the dynamics of this network is affected by deterministic chaos. The neat separation between the indices obtained from the time series simulated from the model and those obtained from time series generated by Gaussian white and colored noise confirmed that the B. subtilis sporulation network dynamics is affected by low dimensional chaos rather than by noise. Furthermore, our analysis identifies the principal driver of the networks chaotic dynamics to be sporulation initiation phosphotransferase B Spo0B. We then analyzed the parameters and the phase space of the system to characterize the instability points of the network dynamics, and, in turn, to identify the ranges of values of Spo0B and of the other drivers of the chaotic dynamics, for which the whole system is highly sensitive to minimal perturbation. In summary, we described an unappreciated source of complexity in the B. subtilis sporulation network by gathering evidence for the chaotic behavior of the system, and by suggesting candidate molecules driving chaos in the system. The results of our chaos analysis can increase our understanding of the intricacies of the regulatory network under analysis, and suggest experimental work to refine our behavior of the mechanisms underlying B. subtilis sporulation initiation control.</div>','<b>Title: </b>Multitask Learning of Signaling and Regulatory Networks with Application to Studying Human Response to Flu<br><b>Year: </b>2014<br><b>Author: </b> S. Jain,  A. Gitter and  Z. Bar-Joseph<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pcbi.1003943"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS COMPUTATIONAL BIOLOGY<br><b>Cluster-Keywords: </b><em>systems biology, regulatory network, interaction networks, network model, gene regulatory, network, network biology, networks, gene network, response networks, network dynamics, dynamic model, cell networks, gene expression, biological networks, system dynamics, dynamic network, model, biological systems, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Reconstructing regulatory and signaling response networks is one of the major goals of systems biology. While several successful methods have been suggested for this task, some integrating large and diverse datasets, these methods have so far been applied to reconstruct a single response network at a time, even when studying and modeling related conditions. To improve network reconstruction we developed MT-SDREM, a multi-task learning method which jointly models networks for several related conditions. In MT-SDREM, parameters are jointly constrained across the networks while still allowing for condition-specific pathways and regulation. We formulate the multi-task learning problem and discuss methods for optimizing the joint target function. We applied MT-SDREM to reconstruct dynamic human response networks for three flu strains: H1N1, H5N1 and H3N2. Our multi-task learning method was able to identify known and novel factors and genes, improving upon prior methods that model each condition independently. The MT-SDREM networks were also better at identifying proteins whose removal affects viral load indicating that joint learning can still lead to accurate, condition-specific, networks. Supporting website with MT-SDREM implementation: http://sb.cs.cmu.edu/mtsdrem</div>','<b>Title: </b>Transcript Profiling Reveals Auxin and Cytokinin Signaling Pathways and Transcription Regulation during In Vitro Organogenesis of Ramie Boehmeria nivea L. Gaud<br><b>Year: </b>2014<br><b>Author: </b> X. Huang,  J. Chen,  Yn. Bao,  Lj. Liu,  H. Jiang,  X. An,  Lj. Dai,  B. Wang and  Dx. Peng<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0113768"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>gene expression, systems biology, metabolic network, network model, network analysis, dynamic model, signaling pathways, signaling network, regulatory network, dynamic network, biology approach, candidate genes, gene network, gene regulatory, biology analysis, mathematical model, network dynamic, biology network, metabolic pathways, pathway analysis<div style="font-size:11px;"><br><b>Abstract: </b><em>In vitro organogenesis, one of the most common pathways leading to in vitro plant regeneration, is widely used in biotechnology and the fundamental study of plant biology. Although previous studies have constructed a complex regulatory network model for Arabidopsis in vitro organogenesis, no related study has been reported in ramie. To generate more complete observations of transcriptome content and dynamics during ramie in vitro organogenesis, we constructed a reference transcriptome library and ten digital gene expression DGE libraries for illumina sequencing. Approximately 111.34 million clean reads were obtained for transcriptome and the DGE libraries generated between 13.5 and 18.8 million clean reads. De novo assembly produced 43,222 unigenes and a total of 5,760 differentially expressed genes DEGs were filtered. Searching against the Kyoto Encyclopedia of Genes and Genomes Pathway database, 26 auxin related and 11 cytokinin related DEGs were selected for qRT-PCR validation of two ramie cultivars, which had high Huazhu No. 5 or extremely low Dazhuhuangbaima shoot regeneration abilities. The results revealed differing regulation patterns of auxin and cytokinin in different genotypes. Here we report the first genome-wide gene expression profiling of in vitro organogenesis in ramie and provide an overview of transcription and phytohormone regulation during the process. Furthermore, the auxin and cytokinin related genes have distinct expression patterns in two ramie cultivars with high or extremely low shoot regeneration ability, which has given us a better understanding of the in vitro organogenesis mechanism. This result will provide a foundation for future phytohormone research and lead to improvements of the ramie regeneration system.</div>','<b>Title: </b>Identification of regulatory structure and kinetic parameters of biochemical networks via mixed-integer dynamic optimization<br><b>Year: </b>2013<br><b>Author: </b> G. Guillen-Gosalbez,  A. Miro,  R. Alves,  A. Sorribas and  L. Jimenez<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-7-113"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>systems biology, dynamic models, biological systems, model, dynamic systems, computational systems, network model, experimental data, model parameters, systems, systems model, models, pathway systems, data, biochemical pathway, dynamics model, biochemical networks, computational models, network, system<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Recovering the network topology and associated kinetic parameter values from time-series data are central topics in systems biology. Nevertheless, methods that simultaneously do both are few and lack generality.Results: Here, we present a rigorous approach for simultaneously estimating the parameters and regulatory topology of biochemical networks from time-series data. The parameter estimation task is formulated as a mixed-integer dynamic optimization problem with: i binary variables, used to model the existence of regulatory interactions and kinetic effects of metabolites in the network processes; and ii continuous variables, denoting metabolites concentrations and kinetic parameters values. The approach simultaneously optimizes the Akaike criterion, which captures the trade-off between complexity measured by the number of parameters, and accuracy of the fitting. This simultaneous optimization mitigates a possible overfitting that could result from addition of spurious regulatory interactions.Conclusion: The capabilities of our approach were tested in one benchmark problem. Our algorithm is able to identify a set of plausible network topologies with their associated parameters.</div>','<b>Title: </b>Understanding Networks of Computing Chemical Droplet Neurons Based on Information Flow<br><b>Year: </b>2015<br><b>Author: </b> G. Gruenert,  K. Gizynski,  G. Escuela,  B. Ibrahim,  J. Gorecki and  P. Dittrich<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1142/S0129065714500324"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>INTERNATIONAL JOURNAL OF NEURAL SYSTEMS<br><b>Cluster-Keywords: </b><em>neural network, network model, network, neurogenesis model, genomic network, networks, saccade control, droplets networks, understanding networks, gene regulatory, model, tumor-immune networks, networks dynamical, network systems, artificial neurogenesis, regulatory networks, dynamic network, information flow, cancer cells, capillary networks<div style="font-size:11px;"><br><b>Abstract: </b><em>In this paper, we present general methods that can be used to explore the information processing potential of a medium composed of oscillating self-exciting droplets. Networks of Belousov-Zhabotinsky BZ droplets seem especially interesting as chemical reaction-diffusion computers because their time evolution is qualitatively similar to neural network activity. Moreover, such networks can be self-generated in microfluidic reactors. However, it is hard to track and to understand the function performed by a medium composed of droplets due to its complex dynamics. Corresponding to recurrent neural networks, the flow of excitations in a network of droplets is not limited to a single direction and spreads throughout the whole medium. In this work, we analyze the operation performed by droplet systems by monitoring the information flow. This is achieved by measuring mutual information and time delayed mutual information of the discretized time evolution of individual droplets. To link the model with reality, we use experimental results to estimate the parameters of droplet interactions. We exemplarily investigate an evolutionary generated droplet structure that operates as a NOR gate. The presented methods can be applied to networks composed of at least hundreds of droplets.</div>','<b>Title: </b>Agent-based re-engineering of ErbB signaling: a modeling pipeline for integrative systems biology<br><b>Year: </b>2017<br><b>Author: </b> Aa. Das,  Ta. Darsana and  E. Jacob<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1093/bioinformatics/btw709"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>signaling network, systems biology, signal transduction, signaling pathways, receptor tyrosine, erbb signaling, erbb receptor, receptor signaling, intracellular signaling, mathematical models, cell signaling, computational model, factor receptor, receptor activation, growth factor, signaling models, networks signaling, receptor system, receptors systems, downstream signaling<div style="font-size:11px;"><br><b>Abstract: </b><em>Motivation: Experiments in systems biology are generally supported by a computational model which quantitatively estimates the parameters of the system by finding the best fit to the experiment. Mathematical models have proved to be successful in reverse engineering the system. The data generated is interpreted to understand the dynamics of the underlying phenomena. The question we have sought to answer is that - is it possible to use an agent-based approach to re-engineer a biological process, making use of the available knowledge from experimental and modelling efforts Can the bottom-up approach benefit from the top-down exercise so as to create an integrated modelling formalism for systems biology We propose a modelling pipeline that learns from the data given by reverse engineering, and uses it for re-engineering the system, to carry out in-silico experiments.Results: A mathematical model that quantitatively predicts co-expression of EGFR-HER2 receptors in activation and trafficking has been taken for this study. The pipeline architecture takes cues from the population model that gives the rates of biochemical reactions, to formulate knowledge-based rules for the particle model. Agent-based simulations using these rules, support the existing facts on EGFR-HER2 dynamics. We conclude that, re-engineering models, built using the results of reverse engineering, opens up the possibility of harnessing the power pack of data which now lies scattered in literature. Virtual experiments could then become more realistic when empowered with the findings of empirical cell biology and modelling studies.</div>','<b>Title: </b>Mathematical Modelling of Polyamine Metabolism in Bloodstream-Form Trypanosoma brucei: An Application to Drug Target Identification<br><b>Year: </b>2013<br><b>Author: </b> X. Gu,  D. Reid,  Dj. Higham and  D. Gilbert<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0053734"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>kinetic models, metabolic networks, kinetic parameters, dynamic models, systems biology, network model, dynamic metabolic, biochemical network, rate constants, enzyme kinetics, kinetic data, adjustable parameters, metabolic model, parameter estimation, dynamic optimization, model selection, mathematical model, kinetic network, model response, model<div style="font-size:11px;"><br><b>Abstract: </b><em>We present the first computational kinetic model of polyamine metabolism in bloodstream-form Trypanosoma brucei, the causative agent of human African trypanosomiasis. We systematically extracted the polyamine pathway from the complete metabolic network while still maintaining the predictive capability of the pathway. The kinetic model is constructed on the basis of information gleaned from the experimental biology literature and defined as a set of ordinary differential equations. We applied Michaelis-Menten kinetics featuring regulatory factors to describe enzymatic activities that are well defined. Uncharacterised enzyme kinetics were approximated and justified with available physiological properties of the system. Optimisation-based dynamic simulations were performed to train the model with experimental data and inconsistent predictions prompted an iterative procedure of model refinement. Good agreement between simulation results and measured data reported in various experimental conditions shows that the model has good applicability in spite of there being gaps in the required data. With this kinetic model, the relative importance of the individual pathway enzymes was assessed. We observed that, at low-to-moderate levels of inhibition, enzymes catalysing reactions of de novo AdoMet MAT and ornithine production OrnPt have more efficient inhibitory effect on total trypanothione content in comparison to other enzymes in the pathway. In our model, prozyme and TSHSyn the production catalyst of total trypanothione were also found to exhibit potent control on total trypanothione content but only when they were strongly inhibited. Different chemotherapeutic strategies against T. brucei were investigated using this model and interruption of polyamine synthesis via joint inhibition of MAT or OrnPt together with other polyamine enzymes was identified as an optimal therapeutic strategy.</div>','<b>Title: </b>Quality control of mitochondria during aging: Is there a good and a bad side of mitochondrial dynamics<br><b>Year: </b>2013<br><b>Author: </b> Mt. Figge,  Hd. Osiewacz and  As. Reichert<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1002/bies.201200125"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOESSAYS<br><b>Cluster-Keywords: </b><em>chaotic system, control systems, neural networks, jerk chaotic, torch chaotic, mode control, sliding mode, plasma torch, dynamic neural, mitochondrial dynamics, system, systems biology, adaptive integral, model system, rodtype plasma, integral sliding, systems, jerk system, cancer cell, comparison method<div style="font-size:11px;"><br><b>Abstract: </b><em>Maintenance of functional mitochondria is essential in order to prevent degenerative processes leading to disease and aging. Mitochondrial dynamics plays a crucial role in ensuring mitochondrial quality but may also generate and spread molecular damage through a population of mitochondria. Computational simulations suggest that this dynamics is advantageous when mitochondria are not or only marginally damaged. In contrast, at a higher degree of damage, mitochondrial dynamics may be disadvantageous. Deceleration of fusion-fission cycles could be one way to adapt to this situation and to delay a further decline in mitochondrial quality. However, this adaptive response makes the mitochondrial network more vulnerable to additional molecular damage. The mitochondrial infectious damage adaptation MIDA model explains a number of inconsistent and counterintuitive data such as the clonal expansion of mutant mitochondrial DNA. We propose that mitochondrial dynamics is a double-edged sword and suggest ways to test this experimentally.</div>','<b>Title: </b>Inferring a nonlinear biochemical network model from a heterogeneous single-cell time course data<br><b>Year: </b>2018<br><b>Author: </b> Y. Shindo,  Y. Kondo and  Y. Sako<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1038/s41598-018-25064-w"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SCIENTIFIC REPORTS<br><b>Cluster-Keywords: </b><em>boolean networks, regulatory networks, network model, bayesian networks, model networks, systems biology, random boolean, real cells, immune system, genetic regulatory, networks, biochemical network, fluorescence images, cellular network, gene regulatory, cell differentiation, network, biological systems, ensemble approach, cell dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>Mathematical modeling and analysis of biochemical reaction networks are key routines in computational systems biology and biophysics; however, it remains difficult to choose the most valid model. Here, we propose a computational framework for data-driven and systematic inference of a nonlinear biochemical network model. The framework is based on the expectation-maximization algorithm combined with particle smoother and sparse regularization techniques. In this method, a "redundant" model consisting of an excessive number of nodes and regulatory paths is iteratively updated by eliminating unnecessary paths, resulting in an inference of the most likely model. Using artificial single-cell time-course data showing heterogeneous oscillatory behaviors, we demonstrated that this algorithm successfully inferred the true network without any prior knowledge of network topology or parameter values. Furthermore, we showed that both the regulatory paths among nodes and the optimal number of nodes in the network could be systematically determined. The method presented in this study provides a general framework for inferring a nonlinear biochemical network model from heterogeneous single-cell time-course data.</div>','<b>Title: </b>Mathematical contributions to dynamics and optimization of gene-environment networks<br><b>Year: </b>2008<br><b>Author: </b> Gw. Weber,  A. Tezel,  P. Taylan,  A. Soyler and  M. Cetin<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1080/02331930701780037"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>OPTIMIZATION<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>This article contributes to a further introduction of continuous optimization in the field of computational biology which is one of the most challenging and emerging areas of science, in addition to foundations presented and the state-of-the-art displayed in C.A. Floudas and P.M. Pardalos, eds., Optimization in Computational Chemistry and Molecular Biology: Local and Global Approaches, Kluwer Academic Publishers, Boston, 2000. Based on a summary of earlier works by the coauthors and their colleagues, it refines the model on gene-environment patterns by a problem from generalized semi-infinite programming GSIP, and characterizes the condition of its structural stability. Furthermore, our paper tries to detect and understand structural frontiers of our methods applied to the recently introduced gene-environment networks and tries to overcome them. Computational biology is interdisciplinary, but it also looks for its mathematical foundations. From data got by DNA microarray experiments, non-linear ordinary differential equations are extracted by the optimization of least-squares errors; then we derive corresponding time-discretized dynamical systems. Using a combinatorial algorithm with polyhedra sequences we can detect the regions of parametric stability, contributing to a testing the goodness of data fitting of the model. To represent and interpret the dynamics, certain matrices, genetic networks and, more generally, gene-environment networks serve. Here, we consider n genes in possible dependence with m special environmental factors and a cumulative one. These networks are subject of discrete mathematical questions, but very large structures, such that we need to simplify them. This is undertaken in a careful optimization with constraints, aiming at a balanced connectedness, incorporates any type of a priori knowledge or request and should be done carefully enough to be robust against disturbation by the environment. In this way, we take into account attacks on the network, knockout phenomena and catastrophies, but also changes in lifestyle and effects of education as far as they can approximately be quantified. We characterize the structural stability of the GSIP problem against perturbations like changes between data series or due to outliers. We give explanations on the numerics and the use of splines. This study is an attempt to demonstrate some beauty and applicabilty of continuous optimization which might together one day give a support in health care, food engineering, biomedicine and -technology, including elements of bioenergy and biomaterials.</div>','<b>Title: </b>Visualizing information across multidimensional post-genomic structured and textual databases<br><b>Year: </b>2005<br><b>Author: </b> Y. Tao,  C. Friedman and  Ya. Lussier<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1093/bioinformatics/bti210"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>gene expression, systems biology, metabolic network, network model, network analysis, dynamic model, signaling pathways, signaling network, regulatory network, dynamic network, biology approach, candidate genes, gene network, gene regulatory, biology analysis, mathematical model, network dynamic, biology network, metabolic pathways, pathway analysis<div style="font-size:11px;"><br><b>Abstract: </b><em>Motivation: Visualizing relationships among biological information to facilitate understanding is crucial to biological research during the post-genomic era. Although different systems have been developed to view gene-phenotype relationships for specific databases, very few have been designed specifically as a general flexible tool for visualizing multidimensional genotypic and phenotypic information together. Our goal is to develop a method for visualizing multidimensional genotypic and phenotypic information and a model that unifies different biological databases in order to present the integrated knowledge using a uniform interface.Results: We developed a novel, flexible and generalizable visualization tool, called PhenoGenesviewer PGviewer, which in this paper was used to display gene-phenotype relationships from a human-curated database OMIM and from an automatic method using a Natural Language Processing tool called BioMedLEE. Data obtained from multiple databases were first integrated into a uniform structure and then organized by PGviewer. PGviewer provides a flexible query interface that allows dynamic selection and ordering of any desired dimension in the databases. Based on users queries, results can be visualized using hierarchical expandable trees that present views specified by users according to their research interests. We believe that this method, which allows users to dynamically organize and visualize multiple dimensions, is a potentially powerful and promising tool that should substantially facilitate biological research.</div>','<b>Title: </b>Domain-oriented reduction of rule-based network models<br><b>Year: </b>2008<br><b>Author: </b> Nm. Borisov,  As. Chistopolsky,  Jr. Faeder and  Bn. Kholodenko<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1049/iet-syb:20070081"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IET SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>allosteric proteins, signaling networks, protein interactions, network model, molecular dynamics, protein complexes, allosteric network, signaling proteins, cell network, biochemical networks, free energy, combinatorial complexity, molecular species, biological systems, receptor signaling, systems biology, chemical species, model, dynamics simulations, cell signaling<div style="font-size:11px;"><br><b>Abstract: </b><em>The coupling of membrane-bound receptors to transcriptional regulators and other effector functions is mediated by multi-domain proteins that form complex assemblies. The modularity of protein interactions lends itself to a rule-based description, in which species and reactions are generated by rules that encode the necessary context for an interaction to occur, but also can produce a combinatorial explosion in the number of chemical species that make up the signalling network. The authors have shown previously that exact network reduction can be achieved using hierarchical control relationships between sites/domains on proteins to dissect multidomain proteins into sets of non-interacting sites, allowing the replacement of each full progenitor protein with a set of derived auxiliary offspring proteins. The description of a network in terms of auxiliary proteins that have fewer sites than progenitor proteins often greatly reduces network size. The authors describe here a method for automating domain-oriented model reduction and its implementation as a module in the BioNetGen modelling package. It takes as input a standard BioNetGen model and automatically performs the following steps: 1 detecting the hierarchical control relationships between sites; 2 building up the auxiliary proteins; 3 generating a raw reduced model and 4 cleaning up the raw model to provide the correct mass balance for each chemical species in the reduced network. The authors tested the performance of this module on models representing portions of growth factor receptor and immunoreceptor-mediated signalling networks and confirmed its ability to reduce the model size and simulation cost by at least one or two orders of magnitude. Limitations of the current algorithm include the inability to reduce models based on implicit site dependencies or heterodimerisation and loss of accuracy when dynamics are computed stochastically.</div>','<b>Title: </b>Slave nodes and the controllability of metabolic networks<br><b>Year: </b>2009<br><b>Author: </b> Dh. Kim and  Ae. Motter<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1088/1367-2630/11/11/113047"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>NEW JOURNAL OF PHYSICS<br><b>Cluster-Keywords: </b><em>metabolic network, metabolic model, dynamic flux, growth rate, genomescale metabolic, flux balance, balance analysis, metabolic flux, metabolic pathways, dynamic model, metabolic reactions, cellular growth, regulatory network, systems biology, optimal growth, metabolic, modeling metabolic, growth conditions, dynamic analysis, cell growth<div style="font-size:11px;"><br><b>Abstract: </b><em>Recent work on synthetic rescues has shown that the targeted deletion of specific metabolic genes can often be used to rescue otherwise nonviable mutants. This raises a fundamental biophysical question: to what extent can the whole-cell behavior of a large metabolic network be controlled by constraining the flux of one or more reactions in the network This touches upon the issue of the number of degrees of freedom contained by one such network. Using the metabolic network of Escherichia coli as a model system, here we address this question theoretically by exploring not only reaction deletions, but also a continuum of all possible reaction expression levels. We show that the behavior of the metabolic network can be largely manipulated by the pinned expression of a single reaction. In particular, a relevant fraction of the metabolic reactions exhibits canalizing interactions, in that the specification of one reaction flux determines cellular growth as well as the fluxes of most other reactions in optimal steady states. The activity of individual reactions can thus be used as surrogates to monitor and possibly control cellular growth and other whole-cell behaviors. In addition to its implications for the study of control processes, our methodology provides a new approach to study how the integrated dynamics of the entire metabolic network emerges from the coordinated behavior of its component parts.</div>','<b>Title: </b>Inference of Biological S-System Using the Separable Estimation Method and the Genetic Algorithm<br><b>Year: </b>2012<br><b>Author: </b> Lz. Liu,  Fx. Wu and  Wj. Zhang<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1109/TCBB.2011.126"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IEEE-ACM TRANSACTIONS ON COMPUTATIONAL BIOLOGY AND BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>ssystem model, biological systems, biological networks, parameter estimation, biochemical networks, dynamic biological, proposed algorithm, regulatory networks, structure identification, model structure, genetic algorithm, gene regulatory, gene expression, network inference, network motif, network structure, gene networks, network identification, network inferring, estimation method<div style="font-size:11px;"><br><b>Abstract: </b><em>Reconstruction of a biological system from its experimental time series data is a challenging task in systems biology. The S-system which consists of a group of nonlinear ordinary differential equations ODEs is an effective model to characterize molecular biological systems and analyze the system dynamics. However, inference of S-systems without the knowledge of system structure is not a trivial task due to its nonlinearity and complexity. In this paper, a pruning separable parameter estimation algorithm PSPEA is proposed for inferring S-systems. This novel algorithm combines the separable parameter estimation method SPEM and a pruning strategy, which includes adding an l1 regularization term to the objective function and pruning the solution with a threshold value. Then, this algorithm is combined with the continuous genetic algorithm CGA to form a hybrid algorithm that owns the properties of these two combined algorithms. The performance of the pruning strategy in the proposed algorithm is evaluated from two aspects: the parameter estimation error and structure identification accuracy. The results show that the proposed algorithm with the pruning strategy has much lower estimation error and much higher identification accuracy than the existing method.</div>','<b>Title: </b>Computational modeling of the metabolic states regulated by the kinase Akt<br><b>Year: </b>2012<br><b>Author: </b> E. Mosca,  R. Alfieri,  C. Maj,  A. Bevilacqua,  G. Canti and  L. Milanesi<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3389/fphys.2012.00418"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>FRONTIERS IN PHYSIOLOGY<br><b>Cluster-Keywords: </b><em>chaotic system, control systems, neural networks, jerk chaotic, torch chaotic, mode control, sliding mode, plasma torch, dynamic neural, mitochondrial dynamics, system, systems biology, adaptive integral, model system, rodtype plasma, integral sliding, systems, jerk system, cancer cell, comparison method<div style="font-size:11px;"><br><b>Abstract: </b><em>Signal transduction and gene regulation determine a major reorganization of metabolic activities in order to support cell proliferation. Protein Kinase B PKB, also known as Akt, participates in the PI3K/Akt/mTOR pathway, a master regulator of aerobic glycolysis and cellular biosynthesis, two activities shown by both normal and cancer proliferating cells. Not surprisingly considering its relevance for cellular metabolism, Akt/PKB is often found hyperactive in cancer cells. In the last decade, many efforts have been made to improve the understanding of the control of glucose metabolism and the identification of a therapeutic window between proliferating cancer cells and proliferating normal cells. In this context, we have modeled the link between the PI3K/Akt/mTOR pathway, glycolysis, lactic acid production, and nucleotide biosynthesis. We used a computational model to compare two metabolic states generated by two different levels of signaling through the PI3K/Akt/mTOR pathway: one of the two states represents the metabolism of a growing cancer cell characterized by aerobic glycolysis and cellular biosynthesis, while the other state represents the same metabolic network with a reduced glycolytic rate and a higher mitochondrial pyruvate metabolism. Biochemical reactions that link glycolysis and pentose phosphate pathway revealed their importance for controlling the dynamics of cancer glucose metabolism.</div>','<b>Title: </b>Stromal biology and therapy in pancreatic cancer: ready for clinical translation<br><b>Year: </b>2019<br><b>Author: </b> A. Neesse,  Ca. Bauer,  D. Ohlund,  M. Lauth,  M. Buchholz,  P. Michl,  Da. Tuveson and  Tm. Gress<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1136/gutjnl-2018-316451"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>GUT<br><b>Cluster-Keywords: </b><em>neural network, network model, network, neurogenesis model, genomic network, networks, saccade control, droplets networks, understanding networks, gene regulatory, model, tumor-immune networks, networks dynamical, network systems, artificial neurogenesis, regulatory networks, dynamic network, information flow, cancer cells, capillary networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Pancreatic ductal adenocarcinoma PDA is notoriously aggressive and hard to treat. The tumour microenvironment TME in PDA is highly dynamic and has been found to promote tumour progression, metastasis niche formation and therapeutic resistance. Intensive research of recent years has revealed an incredible heterogeneity and complexity of the different components of the TME, including cancer-associated fibroblasts, immune cells, extracellular matrix components, tumour vessels and nerves. It has been hypothesised that paracrine interactions between neoplastic epithelial cells and TME compartments may result in either tumour-promoting or tumour-restraining consequences. A better preclinical understanding of such complex and dynamic network systems is required to develop more powerful treatment strategies for patients. Scientific activity and the number of compelling findings has virtually exploded during recent years. Here, we provide an update of the most recent findings in this area and discuss their translational and clinical implications for basic scientists and clinicians alike.</div>','<b>Title: </b>The role of idiotypic interactions in the adaptive immune system: a belief-propagation approach<br><b>Year: </b>2016<br><b>Author: </b> S. Bartolucci,  A. Mozeika and  A. Annibale<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1088/1742-5468/2016/08/083402"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF STATISTICAL MECHANICS-THEORY AND EXPERIMENT<br><b>Cluster-Keywords: </b><em>boolean networks, regulatory networks, network model, bayesian networks, model networks, systems biology, random boolean, real cells, immune system, genetic regulatory, networks, biochemical network, fluorescence images, cellular network, gene regulatory, cell differentiation, network, biological systems, ensemble approach, cell dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>In this work we use belief-propagation techniques to study the equilibrium behaviour of a minimal model for the immune system comprising interacting T and B clones. We investigate the effect of the so-called idiotypic interactions among complementary B clones on the systems activation. Our results show that B-B interactions increase the systems resilience to noise, making clonal activation more stable, while increasing the cross-talk between different clones. We derive analytically the noise level at which a B clone gets activated, in the absence of cross-talk, and find that this increases with the strength of idiotypic interactions and with the number of T cells sending signals to the B clones. We also derive, analytically and numerically, via population dynamics, the critical line where clonal cross-talk arises. Our approach allows us to derive the B clone size distribution, which can be experimentally measured and gives important information about the adaptive immune system response to antigens and vaccination.</div>','<b>Title: </b>A context-sensitive framework for the analysis of human signalling pathways in molecular interaction networks<br><b>Year: </b>2013<br><b>Author: </b> A. Lan,  M. Ziv-Ukelson and  E. Yeger-Lotem<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1093/bioinformatics/btt240"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>systems biology, regulatory network, interaction networks, network model, gene regulatory, network, network biology, networks, gene network, response networks, network dynamics, dynamic model, cell networks, gene expression, biological networks, system dynamics, dynamic network, model, biological systems, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Motivation: A major challenge in systems biology is to reveal the cellular pathways that give rise to specific phenotypes and behaviours. Current techniques often rely on a network representation of molecular interactions, where each node represents a protein or a gene and each interaction is assigned a single static score. However, the use of single interaction scores fails to capture the tendency of proteins to favour different partners under distinct cellular conditions.Results: Here, we propose a novel context-sensitive network model, in which genes and protein nodes are assigned multiple contexts based on their gene ontology annotations, and their interactions are associated with multiple context-sensitive scores. Using this model, we developed a new approach and a corresponding tool, ContextNet, based on a dynamic programming algorithm for identifying signalling paths linking proteins to their downstream target genes. ContextNet finds high-ranking context-sensitive paths in the interactome, thereby revealing the intermediate proteins in the path and their path-specific contexts. We validated the model using 18 348 manually curated cellular paths derived from the SPIKE database. We next applied our framework to elucidate the responses of human primary lung cells to influenza infection. Top-ranking paths were much more likely to contain infection-related proteins, and this likelihood was highly correlated with path score. Moreover, the contexts assigned by the algorithm pointed to putative, as well as previously known responses to viral infection. Thus, context sensitivity is an important extension to current network biology models and can be efficiently used to elucidate cellular response mechanisms.</div>','<b>Title: </b>Application of p-Adic Wavelets to Model Reaction-Diffusion Dynamics in Random Porous Media<br><b>Year: </b>2016<br><b>Author: </b> A. Khrennikov,  K. Oleschko and  Mdc. Lopez<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s00041-015-9433-y"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF FOURIER ANALYSIS AND APPLICATIONS<br><b>Cluster-Keywords: </b><em>boolean networks, regulatory networks, network model, bayesian networks, model networks, systems biology, random boolean, real cells, immune system, genetic regulatory, networks, biochemical network, fluorescence images, cellular network, gene regulatory, cell differentiation, network, biological systems, ensemble approach, cell dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>Fourier and more generally wavelet analysis over the fields of p-adic numbers are widely used in physics, biology and cognitive science, and recently in geophysics. In this note we present a model of the reaction-diffusion dynamics in random porous media, e.g., flow of fluid oil, water or emulsion in a a complex network of pores with known topology. Anomalous diffusion in the model is represented by the system of two equations of reaction-diffusion type, for the part of fluid not bound to solids interface e.g., free oil and for the part bound to solids interface e.g., solids-bound oil. Our model is based on the p-adic treelike representation of pore-networks. We present the system of two p-adic reaction-diffusion equations describing propagation of fluid in networks of pores in random media and find its stationary solutions by using theory of p-adic wavelets. The use of p-adic wavelets generalizing classical wavelet theory gives a possibility to find the stationary solution in the analytic form which is typically impossible for anomalous diffusion in the standard representation based on the real numbers.</div>','<b>Title: </b>Impact of environmental inputs on reverse-engineering approach to network structures<br><b>Year: </b>2009<br><b>Author: </b> Jh. Wu,  Jl. Sinfield,  V. Buchanan-Wollaston and  Jf. Feng<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-3-113"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>interaction network, actin filament, network model, filament network, network structures, actin network, protein interaction, actin dynamics, network, membrane protein, network dynamics, model dynamic, cell, model, network approach, protein complex, computational model, actin, adhesion dynamics, cell dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Uncovering complex network structures from a biological system is one of the main topic in system biology. The network structures can be inferred by the dynamical Bayesian network or Granger causality, but neither techniques have seriously taken into account the impact of environmental inputs.Results: With considerations of natural rhythmic dynamics of biological data, we propose a system biology approach to reveal the impact of environmental inputs on network structures. We first represent the environmental inputs by a harmonic oscillator and combine them with Granger causality to identify environmental inputs and then uncover the causal network structures. We also generalize it to multiple harmonic oscillators to represent various exogenous influences. This system approach is extensively tested with toy models and successfully applied to a real biological network of microarray data of the flowering genes of the model plant Arabidopsis Thaliana. The aim is to identify those genes that are directly affected by the presence of the sunlight and uncover the interactive network structures associating with flowering metabolism.Conclusion: We demonstrate that environmental inputs are crucial for correctly inferring network structures. Harmonic causal method is proved to be a powerful technique to detect environment inputs and uncover network structures, especially when the biological data exhibit periodic oscillations.</div>','<b>Title: </b>A Visual Metaphor Describing Neural Dynamics in Schizophrenia<br><b>Year: </b>2008<br><b>Author: </b> Njm. Van Beveren and  L. De Haan<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0002577"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>network models, gene network, regulatory networks, boolean networks, gene expression, systems biology, neural networks, network dynamics, gene regulatory, biological networks, network structure, gap gene, gene regulation, cell cycle, network, gene interactions, signaling network, quantum systems, system dynamics, networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: In many scientific disciplines the use of a metaphor as an heuristic aid is not uncommon. A well known example in somatic medicine is the defense army metaphor used to characterize the immune system. In fact, probably a large part of the everyday work of doctors consists of translating scientific and clinical information i.e. causes of disease, percentage of succes versus risk of side-effects into information tailored to the needs and capacities of the individual patient. The ability to do so in an effective way is at least partly what makes a clinician a good communicator. Schizophrenia is a severe psychiatric disorder which affects approximately 1 of the population. Over the last two decades a large amount of molecular-biological, imaging and genetic data have been accumulated regarding the biological underpinnings of schizophrenia. However, it remains difficult to understand how the characteristic symptoms of schizophrenia such as hallucinations and delusions are related to disturbances on the molecular-biological level. In general, psychiatry seems to lack a conceptual framework with sufficient explanatory power to link the mental-and molecular-biological domains.Methodology/ Principal Findings: Here, we present an essay-like study in which we propose to use visualized concepts stemming from the theory on dynamical complex systems as a visual metaphor to bridge the mental- and molecular-biological domains in schizophrenia. We first describe a computer model of neural information processing; we show how the information processing in this model can be visualized, using concepts from the theory on complex systems. We then describe two computer models which have been used to investigate the primary theory on schizophrenia, the neurodevelopmental model, and show how disturbed information processing in these two computer models can be presented in terms of the visual metaphor previously described. Finally, we describe the effects of dopamine neuromodulation, of which disturbances have been frequently described in schizophrenia, in terms of the same visualized metaphor.Conclusions/Significance: The conceptual framework and metaphor described offers a heuristic tool to understand the relationship between the mental- and molecular-biological domains in an intuitive way. The concepts we present may serve to facilitate communication between researchers, clinicians and patients.</div>','<b>Title: </b>Lifes Attractors Understanding Developmental Systems Through Reverse Engineering and In Silico Evolution<br><b>Year: </b>2012<br><b>Author: </b> J. Jaeger and  A. Crombach<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/978-1-4614-3567-95"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>EVOLUTIONARY SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>network models, gene network, regulatory networks, boolean networks, gene expression, systems biology, neural networks, network dynamics, gene regulatory, biological networks, network structure, gap gene, gene regulation, cell cycle, network, gene interactions, signaling network, quantum systems, system dynamics, networks<div style="font-size:11px;"><br><b>Abstract: </b><em>We propose an approach to evolutionary systems biology which is based on reverse engineering of gene regulatory networks and in silico evolutionary simulations. We infer regulatory parameters for gene networks by fitting computational models to quantitative expression data. This allows us to characterize the regulatory structure and dynamical repertoire of evolving gene regulatory networks with a reasonable amount of experimental and computational effort. We use the resulting network models to identify those regulatory interactions that are conserved, and those that have diverged between different species. Moreover, we use the models obtained by data fitting as starting points for simulations of evolutionary transitions between species. These simulations enable us to investigate whether such transitions are random, or whether they show stereotypical series of regulatory changes which depend on the structure and dynamical repertoire of an evolving network. Finally, we present a case study-the gap gene network in dipterans flies, midges, and mosquitoes-to illustrate the practical application of the proposed methodology, and to highlight the kind of biological insights that can be gained by this approach.</div>','<b>Title: </b>A physical sciences network characterization of circulating tumor cell aggregate transport<br><b>Year: </b>2015<br><b>Author: </b> Mr. King,  Kg. Phillips,  A. Mitrugno,  Tr. Lee,  Ame. De Guillebon,  S. Chandrasekaran,  Mj. Mcguire,  Rt. Carr,  Sm. Baker-Groberg,  Ra. Rigg,  A. Kolatkar,  M. Luttgen,  K. Bethel,  P. Kuhn,  P. Decuzzi and  Ojt. Mccarty<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1152/ajpcell.00346.2014"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>AMERICAN JOURNAL OF PHYSIOLOGY-CELL PHYSIOLOGY<br><b>Cluster-Keywords: </b><em>neural network, network model, network, neurogenesis model, genomic network, networks, saccade control, droplets networks, understanding networks, gene regulatory, model, tumor-immune networks, networks dynamical, network systems, artificial neurogenesis, regulatory networks, dynamic network, information flow, cancer cells, capillary networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Circulating tumor cells CTC have been implicated in the hematogenous spread of cancer. To investigate the fluid phase of cancer from a physical sciences perspective, the multi-institutional Physical Sciences-Oncology Center PS-OC Network performed multidisciplinary biophysical studies of single CTC and CTC aggregates from a patient with breast cancer. CTCs, ranging from single cells to aggregates comprised of 2-5 cells, were isolated using the high-definition CTC assay and biophysically profiled using quantitative phase microscopy. Single CTCs and aggregates were then modeled in an in vitro system comprised of multiple breast cancer cell lines and microfluidic devices used to model E-selectin mediated rolling in the vasculature. Using a numerical model coupling elastic collisions between red blood cells and CTCs, the dependence of CTC vascular margination on single CTCs and CTC aggregate morphology and stiffness was interrogated. These results provide a multifaceted characterization of single CTC and CTC aggregate dynamics in the vasculature and illustrate a framework to integrate clinical, biophysical, and mathematical approaches to enhance our understanding of the fluid phase of cancer.</div>','<b>Title: </b>Interaction networks, ecological stability, and collective antibiotic tolerance in polymicrobial infections<br><b>Year: </b>2017<br><b>Author: </b> Mgj. De Vos,  M. Zagorski,  A. Mcnally and  T. Bollenbach<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1073/pnas.1713372114"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA<br><b>Cluster-Keywords: </b><em>network ecology, food webs, network models, ecological systems, food web, systems biology, ecological networks, ecosystem model, signalling networks, model results, interaction networks, dynamic model, ecological interactions, network analysis, model network, network structure, transport networks, systems network, ecosystem dynamics, restoration ecology<div style="font-size:11px;"><br><b>Abstract: </b><em>Polymicrobial infections constitute small ecosystems that accommodate several bacterial species. Commonly, these bacteria are investigated in isolation. However, it is unknown to what extent the isolates interact and whether their interactions alter bacterial growth and ecosystem resilience in the presence and absence of antibiotics. We quantified the complete ecological interaction network for 72 bacterial isolates collected from 23 individuals diagnosed with polymicrobial urinary tract infections and found that most interactions cluster based on evolutionary relatedness. Statistical network analysis revealed that competitive and cooperative reciprocal interactions are enriched in the global network, while cooperative interactions are depleted in the individual host community networks. A population dynamics model parameterized by our measurements suggests that interactions restrict community stability, explaining the observed species diversity of these communities. We further show that the clinical isolates frequently protect each other from clinically relevant antibiotics. Together, these results highlight that ecological interactions are crucial for the growth and survival of bacteria in polymicrobial infection communities and affect their assembly and resilience.</div>','<b>Title: </b>Modeling the Evolution of Complex Genetic Systems: The Gene Network Family Tree<br><b>Year: </b>2015<br><b>Author: </b> Jl. Fierst and  Pc. Phillips<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1002/jez.b.22597"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF EXPERIMENTAL ZOOLOGY PART B-MOLECULAR AND DEVELOPMENTAL EVOLUTION<br><b>Cluster-Keywords: </b><em>network models, gene network, regulatory networks, boolean networks, gene expression, systems biology, neural networks, network dynamics, gene regulatory, biological networks, network structure, gap gene, gene regulation, cell cycle, network, gene interactions, signaling network, quantum systems, system dynamics, networks<div style="font-size:11px;"><br><b>Abstract: </b><em>In 1994 and 1996, Andreas Wagner introduced a novel model in two papers addressing the evolution of genetic regulatory networks. This work, and a suite of papers that followed using similar models, helped integrate network thinking into biology and motivate research focused on the evolution of genetic networks. The Wagner network has its mathematical roots in the Ising model, a statistical physics model describing the activity of atoms on a lattice, and in neural networks. These models have given rise to two branches of applications, one in physics and biology and one in artificial intelligence and machine learning. Here, we review development along these branches, outline similarities and differences between biological models of genetic regulatory circuits and neural circuits models used in machine learning, and identify ways in which these models can provide novel insights into biological systems. J. Exp. Zool. Mol. Dev. Evol. 324B: 1-12, 2015. c 2014 Wiley Periodicals, Inc.</div>','<b>Title: </b>Efficient modelling of yeast cell cycles based on multisite phosphorylation using coloured hybrid Petri nets with marking-dependent arc weights<br><b>Year: </b>2018<br><b>Author: </b> M. Herajy,  F. Liu and  M. Heiner<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.nahs.2017.09.002"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>NONLINEAR ANALYSIS-HYBRID SYSTEMS<br><b>Cluster-Keywords: </b><em>petri net, petri nets, bayesian networks, biological networks, systems biology, regulatory networks, dynamic bayesian, net model, gene regulatory, network data, biological systems, biological data, network analysis, network models, biochemical networks, time series, network structure, signal transduction, net models, dbn model<div style="font-size:11px;"><br><b>Abstract: </b><em>With the increasing interest in systems biology to investigate the dynamics and behaviour of biological reaction networks, the scales as well as the complexities of the models under study grew rapidly and continue to grow at even faster pace. Traditional single-scale simulation methods become more and more impractical and inefficient to study these complex reaction networks. A daunting example of biological systems that falls into this category is the cell cycle regulation. In order to accurately model repeated cell growth and division, the corresponding reaction network should exhibit some sort of nonlinearity. One of the techniques able to reproduce this nonlinear behaviour is to include a series of phosphorylation and dephosphorylation reactions of the regulating proteins. However, this modelling approach results in two main challenges: the existence of components with different abundance of molecules and substantially larger biochemical networks in terms of number of reactions and species, with many of them exposing equivalent structure and behaviour. In this paper, we address these two issues by exploiting the modelling power of coloured hybrid Petri nets HPNC. HPNC are a hybrid Petri net class that combines stochastic and deterministic events over a continuous time scale at the coloured level. Moreover, motivated by this case study we extend HPNC to include marking-dependent arc weights instead of just having constant values to define such weights. C 2017 Elsevier Ltd. All rights reserved.</div>','<b>Title: </b>Constructing backbone network by using tinker algorithm<br><b>Year: </b>2017<br><b>Author: </b> Zw. He,  M. Zhan,  Jx. Wang and  Cg. Yao<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s11467-016-0645-7"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>FRONTIERS OF PHYSICS<br><b>Cluster-Keywords: </b><em>network models, gene network, regulatory networks, boolean networks, gene expression, systems biology, neural networks, network dynamics, gene regulatory, biological networks, network structure, gap gene, gene regulation, cell cycle, network, gene interactions, signaling network, quantum systems, system dynamics, networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Revealing how a biological network is organized to realize its function is one of the main topics in systems biology. The functional backbone network, defined as the primary structure of the biological network, is of great importance in maintaining the main function of the biological network. We propose a new algorithm, the tinker algorithm, to determine this core structure and apply it in the cell-cycle system. With this algorithm, the backbone network of the cell-cycle network can be determined accurately and efficiently in various models such as the Boolean model, stochastic model, and ordinary differential equation model. Results show that our algorithm is more efficient than that used in the previous research. We hope this method can be put into practical use in relevant future studies.</div>','<b>Title: </b>Strategies to discover regulatory circuits of the mammalian immune system<br><b>Year: </b>2011<br><b>Author: </b> I. Amit,  A. Regev and  N. Hacohen<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1038/nri3109"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>NATURE REVIEWS IMMUNOLOGY<br><b>Cluster-Keywords: </b><em>systems biology, regulatory network, interaction networks, network model, gene regulatory, network, network biology, networks, gene network, response networks, network dynamics, dynamic model, cell networks, gene expression, biological networks, system dynamics, dynamic network, model, biological systems, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Recent advances in technologies for genome- and proteome-scale measurements and perturbations promise to accelerate discovery in every aspect of biology and medicine. Although such rapid technological progress provides a tremendous opportunity, it also demands that we learn how to use these tools effectively. One application with great potential to enhance our understanding of biological systems is the unbiased reconstruction of genetic and molecular networks. Cells of the immune system provide a particularly useful model for developing and applying such approaches. Here, we review approaches for the reconstruction of signalling and transcriptional networks, with a focus on applications in the mammalian innate immune system.</div>','<b>Title: </b>Computational Modeling of Mitochondrial Function from a Systems Biology Perspective<br><b>Year: </b>2018<br><b>Author: </b> S. Cortassa,  Sj. Sollott and  Ma. Aon<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/978-1-4939-7831-114"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MITOCHONDRIAL BIOENERGETICS: METHODS AND PROTOCOLS, 2ND EDITION<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>The advent of "big data" in biology e.g., genomics, proteomics, metabolomics, holding the promise to reveal the nature of the formidable complexity in cellular and organ makeup and function, has highlighted the compelling need for analytical and integrative computational methods to interpret and make sense of the patterns and changes in those complex networks. Computational models need to be built on sound physicochemical mechanistic principles in order to integrate, interpret, and simulate high-throughput experimental data. Energy transduction processes have been traditionally studied with thermodynamic, kinetic, or thermo-kinetic models, with the latter proving superior to understand the control and regulation of mitochondrial energy metabolism and its interactions with cytoplasmic and other cellular compartments. In this work, we survey the methods to be followed to build a computational model of mitochondrial energetics in isolation or integrated into a network of cellular processes. We describe the use of analytical tools such as elementary flux modes, linear optimization of metabolic models, and control analysis, to help refine our grasp of biologically meaningful behaviors and model reliability. The use of these tools should improve the design, building, and interpretation of steady-state behaviors of computational models while assessing validation criteria and paving the way to prediction.</div>','<b>Title: </b>A Survey on Methods for Modeling and Analyzing Integrated Biological Networks<br><b>Year: </b>2011<br><b>Author: </b> N. Tenazinha and  S. Vinga<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1109/TCBB.2010.117"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IEEE-ACM TRANSACTIONS ON COMPUTATIONAL BIOLOGY AND BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>petri net, petri nets, bayesian networks, biological networks, systems biology, regulatory networks, dynamic bayesian, net model, gene regulatory, network data, biological systems, biological data, network analysis, network models, biochemical networks, time series, network structure, signal transduction, net models, dbn model<div style="font-size:11px;"><br><b>Abstract: </b><em>Understanding how cellular systems build up integrated responses to their dynamically changing environment is one of the open questions in Systems Biology. Despite their intertwinement, signaling networks, gene regulation and metabolism have been frequently modeled independently in the context of well-defined subsystems. For this purpose, several mathematical formalisms have been developed according to the features of each particular network under study. Nonetheless, a deeper understanding of cellular behavior requires the integration of these various systems into a model capable of capturing how they operate as an ensemble. With the recent advances in the "omics" technologies, more data is becoming available and, thus, recent efforts have been driven toward this integrated modeling approach. We herein review and discuss methodological frameworks currently available for modeling and analyzing integrated biological networks, in particular metabolic, gene regulatory and signaling networks. These include network-based methods and Chemical Organization Theory, Flux-Balance Analysis and its extensions, logical discrete modeling, Petri Nets, traditional kinetic modeling, Hybrid Systems and stochastic models. Comparisons are also established regarding data requirements, scalability with network size and computational burden. The methods are illustrated with successful case studies in large-scale genome models and in particular subsystems of various organisms.</div>','<b>Title: </b>Quantitative stem cell biology: computational studies in the hematopoietic system<br><b>Year: </b>2006<br><b>Author: </b> I. Roeder<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1097/01.moh.0000231418.08031.48"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>CURRENT OPINION IN HEMATOLOGY<br><b>Cluster-Keywords: </b><em>stem cell, stem cells, systems biology, mathematical models, cell biology, network model, signaling pathways, cell fate, computational models, cancer cells, gene expression, network dynamics, embryonic stem, signaling network, prostate cancer, mathematical modeling, psychological stress, apoptosis signaling, computational systems, molecular mechanisms<div style="font-size:11px;"><br><b>Abstract: </b><em>Purpose of review This review is intended to provide an overview of recently published computational methods, including bioinformatic algorithms, mathematical models and simulation studies, applied to stem cell biology, with particular reference to the hematopoietic system.Recent findings The analysis of molecular data is making an increased contribution to identify dynamic system responses. Specifically, there are promising computational approaches to characterizing the functional interrelation of network components regulating the process of differentiation and lineage specification of hematopoietic stem cells. Furthermore, evidence is accumulating that stem cell organization should be regarded as a flexible, self-organizing process rather than as a predetermined sequence of events. A number of mathematical models relevant to the hematopoietic stem cell system are applied successfully to clinical situations, demonstrating the predictive power of theoretical methods.Summary Based on the advances in measurement technology, an increasing amount of cellular and molecular data is being generated within the field of stem cell biology. The complexity of the underlying systems, however, most often limits a direct interpretation of the data and makes computational methods indispensable. Mathematical models and simulation techniques are contributing considerably to the discovery of general regulatory principles of stem cell organization and are providing clinically relevant predictions.</div>','<b>Title: </b>Augmented sparse reconstruction of protein signaling networks<br><b>Year: </b>2008<br><b>Author: </b> D. Napoletani,  T. Sauer,  Dc. Struppa,  E. Petricoin and  L. Liotta<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.jtbi.2008.07.026"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF THEORETICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>regulatory networks, gene regulatory, gene expression, network inference, expression data, network structure, gene networks, networks grns, regulatory genes, boolean networks, bayesian network, network reconstruction, networks gene, network dynamics, gene regulation, inference method, systems biology, time series, infer gene, network<div style="font-size:11px;"><br><b>Abstract: </b><em>The problem of reconstructing and identifying intracellular protein signaling and biochemical networks is of critical importance in biology. We propose a mathematical approach called augmented sparse reconstruction for the identification of links among nodes of ordinary differential equation ODE networks, given a small set of observed trajectories with various initial conditions. As a test case, the method is applied to the epidermal growth factor receptor EGFR driven signaling cascade, a well-studied and clinically important signaling network. Our method builds a system of representation from a collection of trajectory integrals, selectively attenuating blocks of terms in the representation. The system of representation is then augmented with random vectors, and l1 minimization is used to find sparse representations for the dynamical interactions of each node. After showing the performance of our method on a model of the EGFR protein network, we sketch briefly the potential future therapeutic applications of this approach. C 2008 Elsevier Ltd. All rights reserved.</div>','<b>Title: </b>Dynamical properties of a two-gene network with hysteresis<br><b>Year: </b>2014<br><b>Author: </b> Q. Shu and  Rg. Sanfelice<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.ic.2014.01.015"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>INFORMATION AND COMPUTATION<br><b>Cluster-Keywords: </b><em>regulatory networks, network dynamics, gene regulatory, network model, genetic network, gene networks, genetic regulatory, mathematical model, parameter space, network system, interaction networks, system dynamics, network structure, network, dynamic networks, network reduction, models models, biological networks, network motifs, biological systems<div style="font-size:11px;"><br><b>Abstract: </b><em>Motivated by the recent development of advanced experimental techniques in molecular biology, this paper focuses on the study of the dynamical properties of a two-gene regulatory network. A mathematical model for a two-gene regulatory network is derived and several of their properties are analyzed. Due to the presence of mixed continuous/discrete dynamics and hysteresis, we employ hybrid systems models to capture the dynamics of the system. The proposed model incorporates binary hysteresis with different thresholds capturing the interaction between the genes. We analyze properties of the solutions and asymptotic stability of equilibria in the system as a function of their parameters. As a difference to previous efforts employing piecewise-linear models, the analysis of our hybrid system model reveals the presence of limit cycles for a certain range of parameters, a behavior that is associated with the presence of hysteresis. The set of points defining the limit cycle is characterized and its asymptotic stability properties are studied. Furthermore, we determine conditions under which the stability properties of the limit cycle are robust to changes of parameters. Numerical simulations are presented to illustrate the findings. C 2014 Elsevier Inc. All rights reserved.</div>','<b>Title: </b>Natural Human Mobility Patterns and Spatial Spread of Infectious Diseases<br><b>Year: </b>2011<br><b>Author: </b> V. Belik,  T. Geisel and  D. Brockmann<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1103/PhysRevX.1.011001"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PHYSICAL REVIEW X<br><b>Cluster-Keywords: </b><em>complex networks, complex systems, network theory, network structure, biological network, social network, network science, dynamic networks, network systems, network model, network topology, systems biology, network, network analysis, system dynamics, networks, realworld networks, network automata, epidemic network, model network<div style="font-size:11px;"><br><b>Abstract: </b><em>We investigate a model for spatial epidemics explicitly taking into account bidirectional movements between base and destination locations on individual mobility networks. We provide a systematic analysis of generic dynamical features of the model on regular and complex metapopulation network topologies and show that significant dynamical differences exist to ordinary reaction-diffusion and effective force of infection models. On a lattice we calculate an expression for the velocity of the propagating epidemic front and find that, in contrast to the diffusive systems, our model predicts a saturation of the velocity with an increasing traveling rate. Furthermore, we show that a fully stochastic system exhibits a novel threshold for the attack ratio of an outbreak that is absent in diffusion and force of infection models. These insights not only capture natural features of human mobility relevant for the geographical epidemic spread, they may serve as a starting point for modeling important dynamical processes in human and animal epidemiology, population ecology, biology, and evolution.</div>','<b>Title: </b>Mathematical Modeling of Biomolecular Network Dynamics<br><b>Year: </b>2011<br><b>Author: </b> Av. Ratushny,  Sa. Ramsey and  Jd. Aitchison<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/978-1-61779-276-221"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>NETWORK BIOLOGY: METHODS AND APPLICATIONS<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Mathematical and computational models have become indispensable tools for integrating and interpreting heterogeneous biological data, understanding fundamental principles of biological system functions, generating reliable testable hypotheses, and identifying potential diagnostic markers and therapeutic targets. Thus, such tools are now routinely used in the theoretical and experimental systematic investigation of biological system dynamics. Here, we discuss model building as an essential part of the theoretical and experimental analysis of biomolecular network dynamics. Specifically, we describe a procedure for defining kinetic equations and parameters of biomolecular processes, and we illustrate the use of fractional activity functions for modeling gene expression regulation by single and multiple regulators. We further discuss the evaluation of model complexity and the selection of an optimal model based on information criteria. Finally, we discuss the critical roles of sensitivity, robustness analysis, and optimal experiment design in the model building cycle.</div>','<b>Title: </b>DBSolve Optimum: a software package for kinetic modeling which allows dynamic visualization of simulation results<br><b>Year: </b>2010<br><b>Author: </b> Nm. Gizzatkulov,  Ii. Goryanin,  Ea. Metelkin,  Ea. Mogilevskaya,  Kv. Peskov and  Ov. Demin<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-4-109"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Systems biology research and applications require creation, validation, extensive usage of mathematical models and visualization of simulation results by end-users. Our goal is to develop novel method for visualization of simulation results and implement it in simulation software package equipped with the sophisticated mathematical and computational techniques for model development, verification and parameter fitting.Results: We present mathematical simulation workbench DBSolve Optimum which is significantly improved and extended successor of well known simulation software DBSolve5. Concept of "dynamic visualization" of simulation results has been developed and implemented in DBSolve Optimum. In framework of the concept graphical objects representing metabolite concentrations and reactions change their volume and shape in accordance to simulation results. This technique is applied to visualize both kinetic response of the model and dependence of its steady state on parameter. The use of the dynamic visualization is illustrated with kinetic model of the Krebs cycle.Conclusion: DBSolve Optimum is a user friendly simulation software package that enables to simplify the construction, verification, analysis and visualization of kinetic models. Dynamic visualization tool implemented in the software allows user to animate simulation results and, thereby, present them in more comprehensible mode. DBSolve Optimum and built-in dynamic visualization module is free for both academic and commercial use. It can be downloaded directly from http://www.insysbio.ru.</div>','<b>Title: </b>Simulated Interventions to Ameliorate Age-Related Bone Loss Indicate the Importance of Timing<br><b>Year: </b>2016<br><b>Author: </b> Cj. Proctor and  A. Gartland<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3389/fendo.2016.00061"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>FRONTIERS IN ENDOCRINOLOGY<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Bone remodeling is the continuous process of bone resorption by osteoclasts and bone formation by osteoblasts, in order to maintain homeostasis. The activity of osteoclasts and osteoblasts is regulated by a network of signaling pathways, including Wnt, parathyroid hormone PTH, RANK ligand/osteoprotegrin, and TGF-beta, in response to stimuli, such as mechanical loading. During aging there is a gradual loss of bone mass due to dysregulation of signaling pathways. This may be due to a decline in physical activity with age and/or changes in hormones and other signaling molecules. In particular, hormones, such as PTH, have a circadian rhythm, which may be disrupted in aging. Due to the complexity of the molecular and cellular networks involved in bone remodeling, several mathematical models have been proposed to aid understanding of the processes involved. However, to date, there are no models, which explicitly consider the effects of mechanical loading, the circadian rhythm of PTH, and the dynamics of signaling molecules on bone remodeling. Therefore, we have constructed a network model of the system using a modular approach, which will allow further modifications as required in future research. The model was used to simulate the effects of mechanical loading and also the effects of different interventions, such as continuous or intermittent administration of PTH. Our model predicts that the absence of regular mechanical loading and/or an impaired PTH circadian rhythm leads to a gradual decrease in bone mass over time, which can be restored by simulated interventions and that the effectiveness of some interventions may depend on their timing.</div>','<b>Title: </b>Reconstructing gene networks: what are the limits<br><b>Year: </b>2003<br><b>Author: </b> J. Stark,  D. Brewer,  M. Barenco,  D. Tomescu,  R. Callard and  M. HubankLink: No link<br><b>Journal: </b>BIOCHEMICAL SOCIETY TRANSACTIONS<br><b>Cluster-Keywords: </b><em>systems biology, regulatory network, interaction networks, network model, gene regulatory, network, network biology, networks, gene network, response networks, network dynamics, dynamic model, cell networks, gene expression, biological networks, system dynamics, dynamic network, model, biological systems, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>To fully realize the benefits of high-throughput post-genomic technologies it is necessary to reconstruct and analyse the complicated network of interactions through which most genes operate. We briefly summarize the mathematical frameworks that can be used to model such networks, and the types of algorithms available for their reconstruction. We then focus on dynamic models, typically described using differential equations, and explain the two main reconstruction approaches in current use. We discuss the data requirements of these algorithms and ask how well they correspond to current microarray data.</div>','<b>Title: </b>Heat shock response in CHO mammalian cells is controlled by a Nonlinear Stochastic process<br><b>Year: </b>2007<br><b>Author: </b> O. Lipan,  Jm. Navenot,  Zx. Wang,  L. Huang and  Sc. Peiper<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pcbi.0030187"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS COMPUTATIONAL BIOLOGY<br><b>Cluster-Keywords: </b><em>systems biology, regulatory network, interaction networks, network model, gene regulatory, network, network biology, networks, gene network, response networks, network dynamics, dynamic model, cell networks, gene expression, biological networks, system dynamics, dynamic network, model, biological systems, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>In many biological systems, the interactions that describe the coupling between different units in a genetic network are nonlinear and stochastic. We study the interplay between stochasticity and nonlinearity using the responses of Chinese hamster ovary CHO mammalian cells to different temperature shocks. The experimental data show that the mean value response of a cell population can be described by a mathematical expression empirical law which is valid for a large range of heat shock conditions. A nonlinear stochastic theoretical model was developed that explains the empirical law for the mean response. Moreover, the theoretical model predicts a specific biological probability distribution of responses for a cell population. The prediction was experimentally confirmed by measurements at the single-cell level. The computational approach can be used to study other nonlinear stochastic biological phenomena.</div>','<b>Title: </b>Synchronization of Coupled Limit Cycles<br><b>Year: </b>2011<br><b>Author: </b> Gs. Medvedev<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s00332-010-9088-4"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF NONLINEAR SCIENCE<br><b>Cluster-Keywords: </b><em>oscillator networks, network dynamics, network, networks, dynamical systems, dynamics, node dynamics, systems, coupled oscillator, synchronization, dynamics network, complex network, coupled system, network modifications, oscillators, network connectivity, feedback vertex, network topology, synchronization properties, collective dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>A unified approach to the analysis of synchronization in coupled systems of autonomous differential equations is presented in this work. Through a careful analysis of the variational equation of the coupled system we establish a sufficient condition for synchronization in terms of the geometric properties of the local limit cycles and the coupling operator. This result applies to a large class of differential equation models in physics and biology. The stability analysis is complemented by a discussion of numerical simulations of a compartmental model of a neuron.</div>','<b>Title: </b>The Impact of Stochasticity and Its Control on a Model of the Inflammatory Response<br><b>Year: </b>2018<br><b>Author: </b> Pd. Mavroudis,  Jd. Scheff,  Jc. Doyle,  Y. Vodovotz and  Ip. Androulakis<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3390/computation7010003"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>COMPUTATION<br><b>Cluster-Keywords: </b><em>inflammatory response, network model, mechanistic models, lde model, injured patients, critically injured, prediction model, tbi survivors, computational modeling, model abm, impaired astrocytes, model predicts, metabolic dynamics, significant differences, twoprocess model, agent-based model, coupled model, model, inflammatory network, computational models<div style="font-size:11px;"><br><b>Abstract: </b><em>The dysregulation of inflammation, normally a self-limited response that initiates healing, is a critical component of many diseases. Treatment of inflammatory disease is hampered by an incomplete understanding of the complexities underlying the inflammatory response, motivating the application of systems and computational biology techniques in an effort to decipher this complexity and ultimately improve therapy. Many mathematical models of inflammation are based on systems of deterministic equations that do not account for the biological noise inherent at multiple scales, and consequently the effect of such noise in regulating inflammatory responses has not been studied widely. In this work, noise was added to a deterministic system of the inflammatory response in order to account for biological stochasticity. Our results demonstrate that the inflammatory response is highly dependent on the balance between the concentration of the pathogen and the level of biological noise introduced to the inflammatory network. In cases where the pro- and anti-inflammatory arms of the response do not mount the appropriate defense to the inflammatory stimulus, inflammation transitions to a different state compared to cases in which pro- and anti-inflammatory agents are elaborated adequately and in a timely manner. In this regard, our results show that noise can be both beneficial and detrimental for the inflammatory endpoint. By evaluating the parametric sensitivity of noise characteristics, we suggest that efficiency of inflammatory responses can be controlled. Interestingly, the time period on which parametric intervention can be introduced efficiently in the inflammatory system can be also adjusted by controlling noise. These findings represent a novel understanding of inflammatory systems dynamics and the potential role of stochasticity thereon.</div>','<b>Title: </b>The systems view in addiction research<br><b>Year: </b>2008<br><b>Author: </b> Pj. Gebicke-Haerter and  F. Tretter<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1111/j.1369-1600.2008.00137.x"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ADDICTION BIOLOGY<br><b>Cluster-Keywords: </b><em>circadian rhythms, systems biology, biological clock, circadian clock, biological rhythms, hpa axis, model, clock network, mathematical models, biological systems, dynamic systems, ode models, biology models, single cell, network, model parameters, experimental data, models, systems, dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>The wealth of information accessible on the molecular level after completion of sequencing of the human and other genomes and in conjunction with new high-throughput technologies like microarrays has paved the way for research on whole systems rather than on single components. Here we describe exemplarily the construction of a rather complex molecular network involved in alcohol addiction by using information from DNA-microarray studies in alcohol-dependent animals. In this network, haemoglobin downregulation in different parts of the brain reward system plays a central role in affecting synaptic plasticity, circadian rhythmicity and opioid receptors. Furthermore, we discuss the dynamic aspect of biological systems with respect to repeated and intermittent drug intake. This aspect can best be captured by the allostatic model on the molecular level. Using a molecular oscillator model where levels of oscillations are changed by repetitive drug administration, changes in set point adjustment are described that underlay allostatic shifts in drug reinforcement processes.</div>','<b>Title: </b>Macroscopic models for networks of coupled biological oscillators<br><b>Year: </b>2018<br><b>Author: </b> Km. Hannay,  Db. Forger and  V. Booth<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1126/sciadv.1701047"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SCIENCE ADVANCES<br><b>Cluster-Keywords: </b><em>oscillator networks, network dynamics, network, networks, dynamical systems, dynamics, node dynamics, systems, coupled oscillator, synchronization, dynamics network, complex network, coupled system, network modifications, oscillators, network connectivity, feedback vertex, network topology, synchronization properties, collective dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>The study of synchronization of coupled biological oscillators is fundamental to many areas of biology including neuroscience, cardiac dynamics, and circadian rhythms. Mathematical models of these systems may involve hundreds of variables in thousands of individual cells resulting in an extremely high-dimensional description of the system. This often contrasts with the low-dimensional dynamics exhibited on the collective or macroscopic scale for these systems. We introduce a macroscopic reduction for networks of coupled oscillators motivated by an elegant structure we find in experimental measurements of circadian protein expression and several mathematical models for coupled biological oscillators. The observed structure in the collective amplitude of the oscillator population differs from the well-known Ott-Antonsen ansatz, but its emergence can be characterized through a simple argument depending only on general phase-locking behavior in coupled oscillator systems. We further demonstrate its emergence in networks of noisy heterogeneous oscillators with complex network connectivity. Applying this structure, we derive low-dimensional macroscopic models for oscillator population activity. This approach allows for the incorporation of cellular-level experimental data into the macroscopic model whose parameters and variables can then be directly associated with tissue- or organism-level properties, thereby elucidating the core properties driving the collective behavior of the system.</div>','<b>Title: </b>Model-driven experimental analysis of the function of SHP-2 in IL-6-induced Jak/STAT signaling<br><b>Year: </b>2012<br><b>Author: </b> A. Dittrich,  T. Quaiser,  C. Khouri,  D. Gortz,  M. Monnigmann and  F. Schaper<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1039/c2mb05488d"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MOLECULAR BIOSYSTEMS<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Misregulated interleukin-6 IL-6-induced Jak/STAT signaling contributes to many diseases. Under non-pathological conditions Jak/STAT signaling is tightly regulated by a complex network of regulators. One of these regulators is the protein tyrosine phosphatase SH2-containing phosphatase 2 SHP2. Although SHP2 is known to be a negative regulator of IL-6-induced Jak/STAT signaling, its exact molecular function is not entirely understood. To elucidate the function of SHP2 in IL-6 signaling we followed a systems biology approach, in which modeling, stepwise model refinement, and experimental analysis are closely linked. We come up with an identifiable model of early Jak/STAT signaling that describes the data and proves to be predictive. The model-based analysis implies that 1 the stepwise association of IL-6 with gp80 and gp130 and 2 STAT3 dimerization at the receptor are essential for the dynamics of early pathway activation, and 3 phosphorylation of SHP2 is nonlinear. Furthermore, the modeling results indicate that SHP2 does not act as a feedback inhibitor in an early phase of IL-6-induced Jak/STAT signaling. However, experimental data reveal that SHP2 exhibits a basal repressory function.</div>','<b>Title: </b>Probabilistic polynomial dynamical systems for reverse engineering of gene regulatory networks<br><b>Year: </b>2011<br><b>Author: </b> Es. Dimitrova,  I. Mitra and  As. Jarrah<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1687-4153-2011-1"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>EURASIP JOURNAL ON BIOINFORMATICS AND SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Elucidating the structure and/or dynamics of gene regulatory networks from experimental data is a major goal of systems biology. Stochastic models have the potential to absorb noise, account for un-certainty, and help avoid data overfitting. Within the frame work of probabilistic polynomial dynamical systems, we present an algorithm for the reverse engineering of any gene regulatory network as a discrete, probabilistic polynomial dynamical system. The resulting stochastic model is assembled from all minimal models in the model space and the probability assignment is based on partitioning the model space according to the likeliness with which a minimal model explains the observed data. We used this method to identify stochastic models for two published synthetic network models. In both cases, the generated model retains the key features of the original model and compares favorably to the resulting models from other algorithms.</div>','<b>Title: </b>Integration of Network Biology and Imaging to Study Cancer Phenotypes and Responses<br><b>Year: </b>2014<br><b>Author: </b> Y. Tian,  Ss. Wang,  Z. Zhang,  Oc. Rodriguez,  E. Petricoin,  Im. Shih,  D. Chan,  M. Avantaggiati,  Gq. Yu,  Sz. Ye,  R. Clarke,  C. Wang,  B. Zhang,  Y. Wang and  C. Albanese<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1109/TCBB.2014.2338304"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IEEE-ACM TRANSACTIONS ON COMPUTATIONAL BIOLOGY AND BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>systems biology, regulatory network, interaction networks, network model, gene regulatory, network, network biology, networks, gene network, response networks, network dynamics, dynamic model, cell networks, gene expression, biological networks, system dynamics, dynamic network, model, biological systems, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Ever growing "omics" data and continuously accumulated biological knowledge provide an unprecedented opportunity to identify molecular biomarkers and their interactions that are responsible for cancer phenotypes that can be accurately defined by clinical measurements such as in vivo imaging. Since signaling or regulatory networks are dynamic and context-specific, systematic efforts to characterize such structural alterations must effectively distinguish significant network rewiring from random background fluctuations. Here we introduced a novel integration of network biology and imaging to study cancer phenotypes and responses to treatments at the molecular systems level. Specifically, Differential Dependence Network DDN analysis was used to detect statistically significant topological rewiring in molecular networks between two phenotypic conditions, and in vivo Magnetic Resonance Imaging MRI was used to more accurately define phenotypic sample groups for such differential analysis. We applied DDN to analyze two distinct phenotypic groups of breast cancer and study how genomic instability affects the molecular network topologies in high-grade ovarian cancer. Further, FDA-approved arsenic trioxide ATO and the ND2-SmoA1 mouse model of Medulloblastoma MB were used to extend our analyses of combined MRI and Reverse Phase Protein Microarray RPMA data to assess tumor responses to ATO and to uncover the complexity of therapeutic molecular biology.</div>','<b>Title: </b>Paradoxical signaling regulates structural plasticity in dendritic spines<br><b>Year: </b>2016<br><b>Author: </b> P. Rangamani,  Mg. Levy,  S. Khan and  G. Oster<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1073/pnas.1610391113"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Transient spine enlargement 3-to 5-min timescale is an important event associated with the structural plasticity of dendritic spines. Many of the molecular mechanisms associated with transient spine enlargement have been identified experimentally. Here, we use a systems biology approach to construct a mathematical model of biochemical signaling and actin-mediated transient spine expansion in response to calcium influx caused by NMDA receptor activation. We have identified that a key feature of this signaling network is the paradoxical signaling loop. Paradoxical components act bifunctionally in signaling networks, and their role is to control both the activation and the inhibition of a desired response function protein activity or spine volume. Using ordinary differential equation ODE-based modeling, we show that the dynamics of different regulators of transient spine expansion, including calmodulin-dependent protein kinase II CaMKII, RhoA, and Cdc42, and the spine volume can be described using paradoxical signaling loops. Our model is able to capture the experimentally observed dynamics of transient spine volume. Furthermore, we show that actin remodeling events provide a robustness to spine volume dynamics. We also generate experimentally testable predictions about the role of different components and parameters of the network on spine dynamics.</div>','<b>Title: </b>Exploring Host-Microbiome Interactions using an in Silico Model of Biomimetic Robots and Engineered Living Cells<br><b>Year: </b>2015<br><b>Author: </b> Kc. Heyde and  Wc. Ruder<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1038/srep11988"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SCIENTIFIC REPORTS<br><b>Cluster-Keywords: </b><em>gene regulatory, systems biology, synthetic biology, regulatory network, gene network, gene expression, system model, synthetic network, network dynamics, systems theory, synthetic gene, genetic network, model predicts, dynamical systems, time delays, translational time, regulation models, oscillatory expression, network structure, biological systems<div style="font-size:11px;"><br><b>Abstract: </b><em>The microbiomes underlying dynamics play an important role in regulating the behavior and health of its host. In order to explore the details of these interactions, we created an in silico model of a living microbiome, engineered with synthetic biology, that interfaces with a biomimetic, robotic host. By analytically modeling and computationally simulating engineered gene networks in these commensal communities, we reproduced complex behaviors in the host. We observed that robot movements depended upon programmed biochemical network dynamics within the microbiome. These results illustrate the models potential utility as a tool for exploring inter-kingdom ecological relationships. These systems could impact fields ranging from synthetic biology and ecology to biophysics and medicine.</div>','<b>Title: </b>Ensembles of signal transduction models using Pareto Optimal Ensemble Techniques POETs<br><b>Year: </b>2010<br><b>Author: </b> So. Song,  A. Chakrabarti and  Jd. Varner<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1002/biot.201000059"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOTECHNOLOGY JOURNAL<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Mathematical modeling of complex gene expression programs is an emerging tool for understanding disease mechanisms. However, identification of large models sometimes requires training using qualitative, conflicting or even contradictory data sets. One strategy to address this challenge is to estimate experimentally constrained model ensembles using multiobjective optimization. In this study, we used Pareto Optimal Ensemble Techniques POETs to identify a family of proof-of-concept signal transduction models. POETs integrate Simulated Annealing SA with Pareto optimality to identify models near the optimal tradeoff surface between competing training objectives. We modeled a prototypical-signaling network using mass-action kinetics within an ordinary differential equation ODE framework 64 ODEs in total. The true model was used to generate synthetic immunoblots from which the POET algorithm identified the 117 unknown model parameters. POET generated an ensemble of signaling models, which collectively exhibited population-like behavior. For example, scaled gene expression levels were approximately normally distributed over the ensemble following the addition of extracellular ligand. Also, the ensemble recovered robust and fragile features of the true model, despite significant parameter uncertainty. Taken together, these results suggest that experimentally constrained model ensembles could capture qualitatively important network features without exact parameter information.</div>','<b>Title: </b>Stochastic Simulation of the Cell Cycle Model for Budding Yeast<br><b>Year: </b>2011<br><b>Author: </b> D. Liu<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.4208/cicp.311009.100310a"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>COMMUNICATIONS IN COMPUTATIONAL PHYSICS<br><b>Cluster-Keywords: </b><em>regulatory networks, stochastic simulation, gene networks, reaction networks, systems biology, stochastic model, network models, complex systems, networks stochastic, gene regulatory, biological systems, stochastic dynamics, differential equations, protein networks, cell cycle, continuous model, interaction networks, systems chemistry, time delays, biochemical reaction<div style="font-size:11px;"><br><b>Abstract: </b><em>We use the recently proposed Nested Stochastic Simulation Algorithm Nested SSA to simulate the cell cycle model for budding yeast. The results show that Nested SSA is able to significantly reduce the computational cost while capturing the essential dynamical features of the system.</div>','<b>Title: </b>Modularization of biochemical networks based on classification of Petri net t-invariants<br><b>Year: </b>2008<br><b>Author: </b> E. Grafahrend-Belau,  F. Schreiber,  M. Heiner,  A. Sackmann,  Bh. Junker,  S. Grunwald,  A. Speer,  K. Winder and  I. Koch<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1471-2105-9-90"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>petri net, petri nets, bayesian networks, biological networks, systems biology, regulatory networks, dynamic bayesian, net model, gene regulatory, network data, biological systems, biological data, network analysis, network models, biochemical networks, time series, network structure, signal transduction, net models, dbn model<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Structural analysis of biochemical networks is a growing field in bioinformatics and systems biology. The availability of an increasing amount of biological data from molecular biological networks promises a deeper understanding but confronts researchers with the problem of combinatorial explosion. The amount of qualitative network data is growing much faster than the amount of quantitative data, such as enzyme kinetics. In many cases it is even impossible to measure quantitative data because of limitations of experimental methods, or for ethical reasons. Thus, a huge amount of qualitative data, such as interaction data, is available, but it was not sufficiently used for modeling purposes, until now. New approaches have been developed, but the complexity of data often limits the application of many of the methods. Biochemical Petri nets make it possible to explore static and dynamic qualitative system properties. One Petri net approach is model validation based on the computation of the systems invariant properties, focusing on t-invariants. T-invariants correspond to subnetworks, which describe the basic system behavior.With increasing system complexity, the basic behavior can only be expressed by a huge number of t-invariants. According to our validation criteria for biochemical Petri nets, the necessary verification of the biological meaning, by interpreting each subnetwork t-invariant manually, is not possible anymore. Thus, an automated, biologically meaningful classification would be helpful in analyzing t-invariants, and supporting the understanding of the basic behavior of the considered biological system.Methods: Here, we introduce a new approach to automatically classify t-invariants to cope with network complexity. We apply clustering techniques such as UPGMA, Complete Linkage, Single Linkage, and Neighbor Joining in combination with different distance measures to get biologically meaningful clusters t-clusters, which can be interpreted as modules. To find the optimal number of t-clusters to consider for interpretation, the cluster validity measure, Silhouette Width, is applied.Results: We considered two different case studies as examples: a small signal transduction pathway  pheromone response pathway in Saccharomyces cerevisiae and a medium-sized gene regulatory network  gene regulation of Duchenne muscular dystrophy. We automatically classified the t-invariants into functionally distinct t-clusters, which could be interpreted biologically as functional modules in the network. We found differences in the suitability of the various distance measures as well as the clustering methods. In terms of a biologically meaningful classification of t-invariants, the best results are obtained using the Tanimoto distance measure. Considering clustering methods, the obtained results suggest that UPGMA and Complete Linkage are suitable for clustering t-invariants with respect to the biological interpretability.Conclusion: We propose a new approach for the biological classification of Petri net t-invariants based on cluster analysis. Due to the biologically meaningful data reduction and structuring of network processes, large sets of t-invariants can be evaluated, allowing for model validation of qualitative biochemical Petri nets. This approach can also be applied to elementary mode analysis.</div>','<b>Title: </b>Combinatorial regulation of lipoprotein lipase by microRNAs during mouse adipogenesis<br><b>Year: </b>2014<br><b>Author: </b> M. Bouvy-Liivrand,  M. Heinaniemi,  E. John,  Jg. Schneider,  T. Sauter and  L. Sinkkonen<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.4161/rna.27655"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>RNA BIOLOGY<br><b>Cluster-Keywords: </b><em>regulatory network, systems biology, systems biological, network dynamics, gene expression, biochemical networks, raft proteins, biological networks, gene circuits, gene regulatory, signalling network, lipid raft, nuclear lamina, protein domain, networks systems, biological approaches, expression pattern, protein interaction, cell differentiation, network motifs<div style="font-size:11px;"><br><b>Abstract: </b><em>MicroRNAs miRNAs regulate gene expression directly through base pairing to their targets or indirectly through participating in multi-scale regulatory networks. Often miRNAs take part in feed-forward motifs where a miRNA and a transcription factor act on shared targets to achieve accurate regulation of processes such as cell differentiation. Here we show that the expression levels of miR-27a and miR-29a inversely correlate with the mRNA levels of lipoprotein lipase Lpl, their predicted combinatorial target, and its key transcriptional regulator peroxisome proliferator-activated receptor gamma Pparg during 3T3-L1 adipocyte differentiation. More importantly, we show that Lpl, a key lipogenic enzyme, can be negatively regulated by the two miRNA families in a combinatorial fashion on the mRNA and functional level in maturing adipocytes. This regulation is mediated through the Lpl 3UTR as confirmed by reporter gene assays. In addition, a small mathematical model captures the dynamics of this feed-forward motif and predicts the changes in Lpl mRNA levels upon network perturbations. The obtained results might offer an explanation to the dysregulation of LPL in diabetic conditions and could be extended to quantitative modeling of regulation of other metabolic genes under similar regulatory network motifs.</div>','<b>Title: </b>Mathematical modeling of atopic dermatitis reveals "double-switch" mechanisms underlying 4 common disease phenotypes<br><b>Year: </b>2017<br><b>Author: </b> E. Dominguez-Huttinger,  P. Christodoulides,  K. Miyauchi,  Ad. Irvine,  M. Okada-Hatakeyama,  M. Kubo and  Rj. Tanaka<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.jaci.2016.10.026"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF ALLERGY AND CLINICAL IMMUNOLOGY<br><b>Cluster-Keywords: </b><em>systems biology, dynamic network, early differentiation, complex diseases, stem cells, erythroid cells, system dynamics, network biomarkers, immune system, tcm syndrome, dnb genes, gene expression, dnb model, stem cell, expression dynamics, model system, mathematical model, model dynamics, dynamic model, sensitivity analysis<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: The skin barrier acts as the first line of defense against constant exposure to biological, microbial, physical, and chemical environmental stressors. Dynamic interplay between defects in the skin barrier, dysfunctional immune responses, and environmental stressors are major factors in the development of atopic dermatitis AD. A systems biology modeling approach can yield significant insights into these complex and dynamic processes through integration of prior biological data.Objective: We sought to develop a multiscale mathematical model of AD pathogenesis that describes the dynamic interplay between the skin barrier, environmental stress, and immune dysregulation and use it to achieve a coherent mechanistic understanding of the onset, progression, and prevention of AD.Methods: We mathematically investigated synergistic effects of known genetic and environmental risk factors on the dynamic onset and progression of the AD phenotype, froma mostly asymptomatic mild phenotype to a severe treatment-resistant form.Results: Our model analysis identified a "double switch," with 2 concatenated bistable switches, as a key network motif that dictates AD pathogenesis: the first switch is responsible for the reversible onset of inflammation, and the second switch is triggered by long-lasting or frequent activation of the first switch, causing irreversible onset of systemic TH2 sensitization and worsening of AD symptoms.Conclusions: Our mathematical analysis of the bistable switch predicts that genetic risk factors decrease the threshold of environmental stressors to trigger systemic TH2 sensitization. This analysis predicts and explains 4 common clinical AD phenotypes from a mild and reversible phenotype through to severe and recalcitrant disease and provides a mechanistic explanation for clinically demonstrated preventive effects of emollient treatments against development of AD.</div>','<b>Title: </b>Analysis and Characterization of Asynchronous State Transition Graphs Using Extremal States<br><b>Year: </b>2013<br><b>Author: </b> T. Lorenz,  H. Siebert and  A. Bockmayr<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s11538-012-9782-5"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BULLETIN OF MATHEMATICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>regulatory networks, network dynamics, gene regulatory, network model, genetic network, gene networks, genetic regulatory, mathematical model, parameter space, network system, interaction networks, system dynamics, network structure, network, dynamic networks, network reduction, models models, biological networks, network motifs, biological systems<div style="font-size:11px;"><br><b>Abstract: </b><em>Logical modeling of biological regulatory networks gives rise to a representation of the systems dynamics as a so-called state transition graph. Analysis of such a graph in its entirety allows for a comprehensive understanding of the functionalities and behavior of the modeled system. However, the size of the vertex set of the graph is exponential in the number of the network components making analysis costly, motivating development of reduction methods. In this paper, we present results allowing for a complete description of an asynchronous state transition graph of a Thomas network solely based on the analysis of the subgraph induced by certain extremal states. Utilizing this notion, we compare the behavior of a simple multivalued network and a corresponding Boolean network and analyze the conservation of dynamical properties between them. Understanding the relation between such coarser and finer models is a necessary step toward meaningful network reduction as well as model refinement methods.</div>','<b>Title: </b>Bistability, Oscillations, and Traveling Waves in Frog Egg Extracts<br><b>Year: </b>2015<br><b>Author: </b> Jj. Tyson and  B. Novak<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s11538-014-0009-9"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BULLETIN OF MATHEMATICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>cell cycle, mammalian cell, cdk network, cell proliferation, cell, cells cell, regulatory network, detailed model, cell size, molecular network, skeleton model, cycle arrest, cycle progression, cell division, cell density, kinases cdks, cycle, phase transition, cycle control, network driving<div style="font-size:11px;"><br><b>Abstract: </b><em>Mathematical modeling is a powerful tool for unraveling the complexities of the molecular regulatory networks underlying all aspects of cell physiology. To support this claim, we review our experiences modeling the cyclin-dependent kinase CDK network that controls events of the eukaryotic cell cycle. The model was derived from classic experiments on the biochemistry and molecular genetics of CDKs and their partner proteins. Because the dynamical properties of CDK activity depend in large part on positive and negative feedback loops in the regulatory network, it is difficult to predict its behavior by intuitive reasoning alone. Mathematical modeling is the correct tool for reliably determining the properties of the network in comparison with observed properties of dividing cells and for predicting the behavior of the control system under novel conditions. In this review, we describe six unexpected predictions of our 1993 model of the CDK control system in frog egg extracts and the remarkable experiments, performed much later, that verified all six predictions. The dynamical properties of the CDK network are consequences of feedback signals and ultrasensitive responses of regulatory proteins to CDK activity, and we describe the experimental evidence for the predicted ultrasensitivity. This case study illustrates the novel insights that mathematical modeling, analysis, and simulation can provide cell physiologists, and it points the way to a new "dynamical perspective" on molecular cell biology.</div>','<b>Title: </b>From intracellular signaling to population oscillations: bridging size- and time-scales in collective behavior<br><b>Year: </b>2015<br><b>Author: </b> Ae. Sgro,  Dj. Schwab,  J. Noorbakhsh,  T. Mestler,  P. Mehta and  T. Gregor<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.15252/msb.20145352"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MOLECULAR SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>oscillator networks, network dynamics, network, networks, dynamical systems, dynamics, node dynamics, systems, coupled oscillator, synchronization, dynamics network, complex network, coupled system, network modifications, oscillators, network connectivity, feedback vertex, network topology, synchronization properties, collective dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>Collective behavior in cellular populations is coordinated by biochemical signaling networks within individual cells. Connecting the dynamics of these intracellular networks to the population phenomena they control poses a considerable challenge because of network complexity and our limited knowledge of kinetic parameters. However, from physical systems, we know that behavioral changes in the individual constituents of a collectively behaving system occur in a limited number of well-defined classes, and these can be described using simple models. Here, we apply such an approach to the emergence of collective oscillations in cellular populations of the social amoeba Dictyostelium discoideum. Through direct tests of our model with quantitative in vivo measurements of single-cell and population signaling dynamics, we show how a simple model can effectively describe a complex molecular signaling network at multiple size and temporal scales. The model predicts novel noise-driven single-cell and population-level signaling phenomena that we then experimentally observe. Our results suggest that like physical systems, collective behavior in biology may be universal and described using simple mathematical models.</div>','<b>Title: </b>CellNOptR: a flexible toolkit to train protein signaling networks to data using multiple logic formalisms<br><b>Year: </b>2012<br><b>Author: </b> C. Terfve,  T. Cokelaer,  D. Henriques,  A. Macnamara,  E. Goncalves,  Mk. Morris,  M. Van Iersel,  Da. Lauffenburger and  J. Saez-Rodriguez<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-6-133"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>boolean network, network models, qualitative models, systems biology, mathematical modeling, boolean models, signaling networks, dynamic models, dynamic modeling, signaling dynamics, logic models, dynamic network, mathematical models, boolean modeling, signal transduction, regulatory networks, drug resistance, biological networks, network approach, network modeling<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Cells process signals using complex and dynamic networks. Studying how this is performed in a context and cell type specific way is essential to understand signaling both in physiological and diseased situations. Context-specific medium/high throughput proteomic data measured upon perturbation is now relatively easy to obtain but formalisms that can take advantage of these features to build models of signaling are still comparatively scarce.Results: Here we present CellNOptR, an open-source R software package for building predictive logic models of signaling networks by training networks derived from prior knowledge to signaling typically phosphoproteomic data. CellNOptR features different logic formalisms, from Boolean models to differential equations, in a common framework. These different logic model representations accommodate state and time values with increasing levels of detail. We provide in addition an interface via Cytoscape CytoCopteR to facilitate use and integration with Cytoscape network-based capabilities.Conclusions: Models generated with this pipeline have two key features. First, they are constrained by prior knowledge about the network but trained to data. They are therefore context and cell line specific, which results in enhanced predictive and mechanistic insights. Second, they can be built using different logic formalisms depending on the richness of the available data. Models built with CellNOptR are useful tools to understand how signals are processed by cells and how this is altered in disease. They can be used to predict the effect of perturbations individual or in combinations, and potentially to engineer therapies that have differential effects/side effects depending on the cell type or context.</div>','<b>Title: </b>Maintenance of Mitochondrial Morphology in Cryptococcus neoformans Is Critical for Stress Resistance and Virulence<br><b>Year: </b>2018<br><b>Author: </b> Al. Chang and  Tl. Doering<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1128/mBio.01375-18"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MBIO<br><b>Cluster-Keywords: </b><em>gene regulatory, regulatory network, systems biology, cell differentiation, cell fate, model systems, potential landscape, network grn, dynamical systems, melanocyte differentiation, network models, gene expression, network dynamics, differentiation genes, dynamical models, dynamical networks, cellular differentiation, stem cells, attractor network, developmental gene<div style="font-size:11px;"><br><b>Abstract: </b><em>Mitochondria are essential organelles that act in pathways including ATP production, beta-oxidation, and clearance of reactive oxygen species. They occur as a complex reticular network that constantly undergoes fusion and fission, mediated by dynamin-related proteins DRPs. DRPs include Fzo1, which mediates fusion, and Dnm1, Mdv1, and Fis1, which mediate fission. Mitochondrial morphology has been implicated in virulence in multiple fungi, as with the association between virulence and increased mitochondrial fusion in Cryptococcus gattii. This relationship, however, has not been studied in Cryptococcus neoformans, a related opportunistic pathogen. C. neoformans is an environmental yeast that can adapt to the human host environment, overcome the innate immune system, and eventually disseminate and cause lethal meningoencephalitis. We used gene deletion of key DRPs to study their role in mitochondrial morphology and pathogenesis of this yeast. Interestingly, increasing mitochondrial fusion did not increase resistance to oxidative stress, unlike in model yeast. Blocking mitochondrial fusion, however, yielded increased susceptibility to oxidative and nitrosative stresses as well as complete avirulence. This lack of virulence was not mediated by any effects of altered mitochondrial function on two major virulence factors, capsule and melanin. Instead, it was due to decreased survival within macrophages, which in turn was a consequence of increased susceptibility to oxidative and nitrosative stress. Supporting this conclusion, reactive oxygen species ROS scavengers rescued the ability of fusion mutants to survive intracellularly. These findings increase our understanding of cryptococcal biology and virulence and shed light on an important group of proteins and cellular processes in this pathogen.IMPORTANCE C. neoformans is a yeast that causes fatal brain infection in close to 200,000 people worldwide every year, mainly afflicting individuals with AIDS or others who are severely immunocompromised. One feature of this microbe that helps it cause disease is that it is able to withstand toxic molecules it encounters when host cells engulf it in their efforts to control the infection. Mitochondria are important organelles responsible for energy production and other key cellular processes. They typically exist in a complex network that changes morphology by fusing and dividing; these alterations also influence mitochondrial function. Using genetic approaches, we found that changes in mitochondrial morphology dramatically influence cryptococcal virulence. We showed that this occurs because the altered mitochondria are less able to eliminate the harmful molecules that host cells produce to kill invading microbes. These findings are important because they elucidate fundamental biology and virulence and may suggest avenues for therapy.</div>','<b>Title: </b>Multiset of Agents in a Network for Simulation of Complex Systems<br><b>Year: </b>2009<br><b>Author: </b> Vk. Murthy and  Ev. KrishnamurthyLink: No link<br><b>Journal: </b>RECENT ADVANCES IN NONLINEAR DYNAMICS AND SYNCHRONIZATION: THEORY AND APPLICATIONS<br><b>Cluster-Keywords: </b><em>systems biology, regulatory network, interaction networks, network model, gene regulatory, network, network biology, networks, gene network, response networks, network dynamics, dynamic model, cell networks, gene expression, biological networks, system dynamics, dynamic network, model, biological systems, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>A Complex System CS exhibits the four salient properties: i Collective, coordinated and efficient interaction among its components ii Self-organization and emergence iii Power law scaling under emergence iv Adaptation, fault tolerance and resilience against damage of its components. We describe briefly, three interrelated mathematical models that enable us to understand these properties: Fractal and percolation model, Stochastic / Chaotic nonlinear dynamical model and Topological network or graph model. These models have been very well studied in recent years and are closely related to the properties such as: self-similarity, scale-free, resilience, self-organization and emergence. We explain how these properties of CS can be simulated using the multi-set of agents-based paradigm MAP through random enabling, inhibiting, preferential attachment and growth of the multiagent network. We discuss these aspects from the point of view of geometric parameters-Lyapunov exponents, strange attractors, metric entropy, and topological indices-Cluster coefficient, Average degree distribution and the correlation length of the interacting network.We also describe the advantages of agent-based modelling, simulation and animation. These are illustrated by a few examples in swarm dynamics- ant colony, bacterial colonies, human-animal trails, and graph-growth. We briefly consider the engineering of CS, the role of scales, and the limitations arising from quantum mechanics. A brief summary of currently available agent-tool kits is provided. Further developments of agent technology will be of great value to model, simulate and animate, many phenomena in Systems biology-cellular dynamics, cell motility, growth and development biology Morphogenesis, and can provide for improved capability in complex systems modelling.</div>','<b>Title: </b>Pattern formation by dynamically interacting network motifs<br><b>Year: </b>2009<br><b>Author: </b> J. Lembong,  N. Yakoby and  Sy. Shvartsman<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1073/pnas.0810728106"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA<br><b>Cluster-Keywords: </b><em>regulatory networks, network dynamics, gene regulatory, network model, genetic network, gene networks, genetic regulatory, mathematical model, parameter space, network system, interaction networks, system dynamics, network structure, network, dynamic networks, network reduction, models models, biological networks, network motifs, biological systems<div style="font-size:11px;"><br><b>Abstract: </b><em>Systematic validation of pattern formation mechanisms revealed by molecular studies of development is essentially impossible without mathematical models. Models can provide a compact summary of a large number of experiments that led to mechanism formulation and guide future studies of pattern formation. Here, we realize this program by analyzing a mathematical model of epithelial patterning by the highly conserved EGFR and BMP signaling pathways in Drosophila oogenesis. The model accounts for the dynamic interaction of the feedforward and feedback network motifs that control the expression of Broad, a zinc finger transcription factor expressed in the cells that form the upper part of the respiratory eggshell appendages. Based on the combination of computational analysis and genetic experiments, we show that the model accounts for the key features of wild-type pattern formation, correctly predicts patterning defects in multiple mutants, and guides the identification of additional regulatory links in a complex pattern formation mechanism.</div>','<b>Title: </b>Networks and circuits in cell regulation<br><b>Year: </b>2010<br><b>Author: </b> P. Palumbo,  G. Mavelli,  L. Farina and  L. Alberghina<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.bbrc.2010.05.015"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOCHEMICAL AND BIOPHYSICAL RESEARCH COMMUNICATIONS<br><b>Cluster-Keywords: </b><em>blood flow, blood cells, flow rate, blood pumps, cells, cell, results, patients, stem cells, methods, blood, patients blood, flow, patients methods, patients objectives, cell viability, objectives, results cells, blood pressure, cell culture<div style="font-size:11px;"><br><b>Abstract: </b><em>Large-scale "omics" data are often represented as networks of interacting components, but such representation is inherently static and, as such, cannot provide a realistic picture of the temporal dynamics of complex cellular functions. These difficulties suggest moving to a modeling strategy that explicitly takes into account both the wiring of the components and the task they perform. From an engineering perspective, this problem resembles that of "circuit analysis". In this paper, we focus on a limited but relevant biological circuit, the Cl to S transition in yeast cell cycle, and investigate both the network representation and the corresponding circuit described by a mathematical model, by means of a wide range of numerical simulation analysis. Reliable predictions of system-level properties are achieved and the parameters that mostly affect these properties are found out. c 2010 Elsevier Inc. All rights reserved.</div>','<b>Title: </b>Network Modeling of TGF beta Signaling in Hepatocellular Carcinoma Epithelial-to-Mesenchymal Transition Reveals Joint Sonic Hedgehog and Wnt Pathway Activation<br><b>Year: </b>2014<br><b>Author: </b> Sn. Steinway,  Jgt. Zanudo,  W. Ding,  Cb. Rountree,  Dj. Feith,  Tp. Loughran and  R. Albert<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1158/0008-5472.CAN-14-0225"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>CANCER RESEARCH<br><b>Cluster-Keywords: </b><em>feedback loops, signaling network, positive feedback, network model, negative feedback, biochemical networks, signaling pathways, signal transduction, systems biology, kinase erk, protein kinase, transduction networks, dynamic model, emt network, erk pathway, cell fate, cell death, biochemical reaction, coupled feedback, impulse response<div style="font-size:11px;"><br><b>Abstract: </b><em>Epithelial-to-mesenchymal transition EMT is a developmental process hijacked by cancer cells to leave the primary tumor site, invade surrounding tissue, and establish distant metastases. A hallmark of EMT is the loss of E-cadherin expression, and one major signal for the induction of EMT is TGF beta, which is dysregulated in up to 40 of hepatocellular carcinoma HCC. We have constructed an EMT network of 70 nodes and 135 edges by integrating the signaling pathways involved in developmental EMT and known dysregulations in invasive HCC. We then used discrete dynamic modeling to understand the dynamics of the EMT network driven by TGF beta. Our network model recapitulates known dysregulations during the induction of EMT and predicts the activation of the Wnt and Sonic hedgehog SHH signaling pathways during this process. We show, across multiple murine P2E and P2M and human HCC cell lines Huh7, PLC/PRF/5, HLE, and HLF, that the TGF beta signaling axis is a conserved driver of mesenchymal phenotype HCC and confirm that Wnt and SHH signaling are induced in these cell lines. Furthermore, we identify by network analysis eight regulatory feedback motifs that stabilize the EMT process and show that these motifs involve cross-talk among multiple major pathways. Our model will be useful in identifying potential therapeutic targets for the suppression of EMT, invasion, and metastasis in HCC. C 2014 AACR.</div>','<b>Title: </b>Adaptive intervention in probabilistic boolean networks<br><b>Year: </b>2009<br><b>Author: </b> R. Layek,  A. Datta,  R. Pal and  Er. Dougherty<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1093/bioinformatics/btp349"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>boolean networks, regulatory networks, genetic regulatory, gene regulatory, probabilistic boolean, network, networks, model, network model, probabilistic model, model checking, networks pbns, network dynamics, markov chain, regulatory, models, optimal control, boolean, networks models, networks boolean<div style="font-size:11px;"><br><b>Abstract: </b><em>Motivation: A basic problem of translational systems biology is to utilize gene regulatory networks as a vehicle to design therapeutic intervention strategies to beneficially alter network and, therefore, cellular dynamics. One strain of research has this problem from the perspective of control theory via the design of optimal Markov chain decision processes, mainly in the framework of probabilistic Boolean networks PBNs. Full optimization assumes that the network is accurately modeled and, to the extent that model inference is inaccurate, which can be expected for gene regulatory networks owing to the combination of model complexity and a paucity of time-course data, the designed intervention strategy may perform poorly. We desire intervention strategies that do not assume accurate full-model inference.Results: This article demonstrates the feasibility of applying online adaptive control to improve intervention performance in genetic regulatory networks modeled by PBNs. It shows via simulations that when the network is modeled by a member of a known family of PBNs, an adaptive design can yield improved performance in terms of the average cost. Two algorithms are presented, one better suited for instantaneously random PBNs and the other better suited for context-sensitive PBNs with low switching probability between the constituent BNs.</div>','<b>Title: </b>Derivation, identification and validation of a computational model of a novel synthetic regulatory network in yeast<br><b>Year: </b>2011<br><b>Author: </b> L. Marucci,  S. Santini,  M. Di Bernardo and  D. Di Bernardo<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s00285-010-0350-z"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF MATHEMATICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>regulatory network, systems biology, systems biological, network dynamics, gene expression, biochemical networks, raft proteins, biological networks, gene circuits, gene regulatory, signalling network, lipid raft, nuclear lamina, protein domain, networks systems, biological approaches, expression pattern, protein interaction, cell differentiation, network motifs<div style="font-size:11px;"><br><b>Abstract: </b><em>Systems biology aims at building computational models of biological pathways in order to study in silico their behaviour and to verify biological hypotheses. Modelling can become a new powerful method in molecular biology, if correctly used. Here we present step-by-step the derivation and identification of the dynamical model of a biological pathway using a novel synthetic network recently constructed in the yeast Saccharomyces cerevisiae for In-vivo Reverse-Engineering and Modelling Assessment. This network consists of five genes regulating each other transcription. Moreover, it includes one protein-protein interaction, and its genes can be switched on by addition of galactose to the medium. In order to describe the network dynamics, we adopted a deterministic modelling approach based on non-linear differential equations. We show how, through iteration between experiments and modelling, it is possible to derive a semi-quantitative prediction of network behaviour and to better understand the biology of the pathway of interest.</div>','<b>Title: </b>The auxin signalling network translates dynamic input into robust patterning at the shoot apex<br><b>Year: </b>2011<br><b>Author: </b> T. Vernoux,  G. Brunoud,  E. Farcot,  V. Morin,  H. Van Den Daele,  J. Legrand,  M. Oliva,  P. Das,  A. Larrieu,  D. Wells,  Y. Guedon,  L. Armitage,  F. Picard,  S. GuyomarcH,  C. Cellier,  G. Parry,  R. Koumproglou,  Jh. Doonan,  M. Estelle,  C. Godin,  S. Kepinski,  M. Bennett,  L. De Veylder and  J. Traas<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1038/msb.2011.39"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MOLECULAR SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>network ecology, food webs, network models, ecological systems, food web, systems biology, ecological networks, ecosystem model, signalling networks, model results, interaction networks, dynamic model, ecological interactions, network analysis, model network, network structure, transport networks, systems network, ecosystem dynamics, restoration ecology<div style="font-size:11px;"><br><b>Abstract: </b><em>The plant hormone auxin is thought to provide positional information for patterning during development. It is still unclear, however, precisely how auxin is distributed across tissues and how the hormone is sensed in space and time. The control of gene expression in response to auxin involves a complex network of over 50 potentially interacting transcriptional activators and repressors, the auxin response factors ARFs and Aux/IAAs. Here, we perform a large-scale analysis of the Aux/IAA-ARF pathway in the shoot apex of Arabidopsis, where dynamic auxin-based patterning controls organogenesis. A comprehensive expression map and full interactome uncovered an unexpectedly simple distribution and structure of this pathway in the shoot apex. A mathematical model of the Aux/IAA-ARF network predicted a strong buffering capacity along with spatial differences in auxin sensitivity. We then tested and confirmed these predictions using a novel auxin signalling sensor that reports input into the signalling pathway, in conjunction with the published DR5 transcriptional output reporter. Our results provide evidence that the auxin signalling network is essential to create robust patterns at the shoot apex. Molecular Systems Biology 7: 508; published online 5 July 2011; doi:10.1038/msb.2011.39</div>','<b>Title: </b>Statistical inference of the time-varying structure of gene-regulation networks<br><b>Year: </b>2010<br><b>Author: </b> S. Lebre,  J. Becq,  F. Devaux,  Mph. Stumpf and  G. Lelandais<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-4-130"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>circadian clock, biological systems, network models, mathematical models, clock gene, systems biology, model selection, biological networks, network data, experimental data, model parameters, network dynamics, plant circadian, regulatory network, clock model, plant clock, complex network, clock circuit, approximate bayesian, stochastic models<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Biological networks are highly dynamic in response to environmental and physiological cues. This variability is in contrast to conventional analyses of biological networks, which have overwhelmingly employed static graph models which stay constant over time to describe biological systems and their underlying molecular interactions.Methods: To overcome these limitations, we propose here a new statistical modelling framework, the ARTIVA formalism Auto Regressive TIme VArying models, and an associated inferential procedure that allows us to learn temporally varying gene-regulation networks from biological time-course expression data. ARTIVA simultaneously infers the topology of a regulatory network and how it changes over time. It allows us to recover the chronology of regulatory associations for individual genes involved in a specific biological process development, stress response, etc..Results: We demonstrate that the ARTIVA approach generates detailed insights into the function and dynamics of complex biological systems and exploits efficiently time-course data in systems biology. In particular, two biological scenarios are analyzed: the developmental stages of Drosophila melanogaster and the response of Saccharomyces cerevisiae to benomyl poisoning.Conclusions: ARTIVA does recover essential temporal dependencies in biological systems from transcriptional data, and provide a natural starting point to learn and investigate their dynamics in greater detail.</div>','<b>Title: </b>Systems biology network-based discovery of a small molecule activator BL-AD008 targeting AMPK/ZIPK and inducing apoptosis in cervical cancer<br><b>Year: </b>2015<br><b>Author: </b> Ll. Fu,  Sy. Zhang,  L. Zhang,  Xp. Tong,  J. Zhang,  Yh. Zhang,  L. Ouyang,  B. Liu and  J. Huang<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.18632/oncotarget.3513"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ONCOTARGET<br><b>Cluster-Keywords: </b><em>stem cell, stem cells, systems biology, mathematical models, cell biology, network model, signaling pathways, cell fate, computational models, cancer cells, gene expression, network dynamics, embryonic stem, signaling network, prostate cancer, mathematical modeling, psychological stress, apoptosis signaling, computational systems, molecular mechanisms<div style="font-size:11px;"><br><b>Abstract: </b><em>The aim of this study was to discover a small molecule activator BL-AD008 targeting AMPK/ZIPK and inducing apoptosis in cervical cancer. In this study, we systematically constructed the global protein-protein interaction PPI network and predicted apoptosis-related protein connections by the Naive Bayesian model. Then, we identified some classical apoptotic PPIs and other previously unrecognized PPIs between apoptotic kinases, such as AMPK and ZIPK. Subsequently, we screened a series of candidate compounds targeting AMPK/ZIPK, synthesized some compounds and eventually discovered a novel dual-target activator BL-AD008. Moreover, we found BL-AD008 bear remarkable anti-proliferative activities toward cervical cancer cells and could induce apoptosis by death-receptor and mitochondrial pathways. Additionally, we found that BL-AD008-induced apoptosis was affected by the combination of AMPK and ZIPK. Then, we found that BL-AD008 bear its anti-tumor activities and induced apoptosis by targeting AMPK/ZIPK in vivo. In conclusion, these results demonstrate the ability of systems biology network to identify some key apoptotic kinase targets AMPK and ZIPK; thus providing a dual-target small molecule activator BL-AD008 as a potential new apoptosis-modulating drug in future cervical cancer therapy.</div>','<b>Title: </b>Adaptive Integral Sliding Mode Controller Design for the Control and Synchronization of a Novel Jerk Chaotic System<br><b>Year: </b>2017<br><b>Author: </b> S. Vaidyanathan<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/978-3-319-55598-017"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>APPLICATIONS OF SLIDING MODE CONTROL IN SCIENCE AND ENGINEERING<br><b>Cluster-Keywords: </b><em>chaotic system, control systems, neural networks, jerk chaotic, torch chaotic, mode control, sliding mode, plasma torch, dynamic neural, mitochondrial dynamics, system, systems biology, adaptive integral, model system, rodtype plasma, integral sliding, systems, jerk system, cancer cell, comparison method<div style="font-size:11px;"><br><b>Abstract: </b><em>Chaos has important applications in physics, chemistry, biology, ecology, secure communications, cryptosystems and many scientific branches. Control and synchronization of chaotic systems are important research problems in chaos theory. Sliding mode control is an important method used to solve various problems in control systems engineering. In robust control systems, the sliding mode control is often adopted due to its inherent advantages of easy realization, fast response and good transient performance as well as insensitivity to parameter uncertainties and disturbance. In this work, we first describe the Sprott jerk chaotic system 1997, which is an important model of a jerk chaotic system with two cubic nonlinearities. Next, we derive a novel jerk chaotic system by adding a linear term to Sprott jerk chaotic system. The phase portraits of the novel jerk chaotic system are displayed and the qualitative properties of the novel jerk chaotic system are discussed. We demonstrate that the novel jerk chaotic system has a unique equilibrium point at the origin, which is a saddle-focus. The Lyapunov exponents of the novel jerk chaotic system are obtained as L-1 = 0.2062, L-2 = 0 and L-3 = -3.8062. The Kaplan-Yorke dimension of the novel jerk system is derived as D-KY = 2.0542, which shows the complexity of the system. Next, an adaptive integral sliding mode control scheme is proposed to globally stabilize all the trajectories of the novel jerk chaotic system. Furthermore, an adaptive integral sliding mode control scheme is proposed for the global chaos synchronization of identical novel jerk chaotic systems. The adaptive control mechanism helps the control design by estimating the unknown parameters. Numerical simulations using MATLAB are shown to illustrate all the main results derived in this work.</div>','<b>Title: </b>Efficient weighting strategy for enhancing synchronizability of complex networks<br><b>Year: </b>2018<br><b>Author: </b> Yq. Wang,  F. Yu,  Sc. Huang,  Jj. Tu and  Y. Chen<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1142/S0217984918501282"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MODERN PHYSICS LETTERS B<br><b>Cluster-Keywords: </b><em>complex networks, complex systems, network theory, network structure, biological network, social network, network science, dynamic networks, network systems, network model, network topology, systems biology, network, network analysis, system dynamics, networks, realworld networks, network automata, epidemic network, model network<div style="font-size:11px;"><br><b>Abstract: </b><em>Networks with high propensity to synchronization are desired in many applications ranging from biology to engineering. In general, there are two ways to enhance the synchronizability of a network: link rewiring and/or link weighting. In this paper, we propose a new link weighting strategy based on the concept of the neighborhood subgroup. The neighborhood subgroup of a node i through node j in a network, i.e. Gi -> j means that node u belongs to Gi -> j if node u belongs to the first-order neighbors of j not include i. Our proposed weighting schema used the local and global structural properties of the networks such as the node degree, betweenness centrality and closeness centrality measures. We applied the method on scale-free and Watts Strogatz networks of different structural properties and show the good performance of the proposed weighting scheme. Furthermore, as model networks cannot capture all essential features of real-world complex networks, we considered a number of undirected and unweighted real-world networks. To the best of our knowledge, the proposed weighting strategy outperformed the previously published weighting methods by enhancing the synchronizability of these real-world networks.</div>','<b>Title: </b>Modelling cell-free RNA and protein synthesis with minimal systems<br><b>Year: </b>2019<br><b>Author: </b> A. Doerr,  E. De Reus,  P. Van Nies,  M. Van Der Haar,  K. Wei,  J. Kattan,  A. Wahl and  C. Danelon<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1088/1478-3975/aaf33d"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PHYSICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>systems biology, network models, signaling network, parameter estimation, dynamic models, biological systems, mathematical models, metabolic network, parameter sensitivities, computational model, gene circuit, circuit models, experimental data, metabolic models, model system, scalable parameter, model parameters, parameter space, biomolecular networks, model<div style="font-size:11px;"><br><b>Abstract: </b><em>DNA-guided cell-free protein synthesis using a minimal set of purified components has emerged as a versatile platform in constructive biology. The E. coli-based PURE protein synthesis using recombinant elements system offers the basic protein synthesis factory in a prospective minimal cell relying on extant molecules. However, there is an urgent need to improve the systems performance and to build a mechanistic computational model that can help interpret and predict gene expression dynamics. Herein, we utilized all three commercially available PURE system variants: PURExpress, PUREfrex and PUREfrex2.0. We monitored apparent kinetics of mRNA and protein synthesis by fluorescence spectroscopy at different concentrations of DNA template. Analysis of polysome distributions by atomic force microscopy, combined with a stochastic model of translation, revealed inefficient usage of ribosomes, consistent with the idea that translation initiation is a limiting step. This preliminary dataset was used to formulate hypotheses regarding possible mechanisms impeding robust gene expression. Next, we challenged these hypotheses by devising targeted experiments aimed to alleviate the current limitations of PUREfrex. We identified depletion of key initiation factors IFs by translationally inactive mRNA as a possible inhibitory mechanism. This adverse process could partly be remedied by targeted mRNA degradation, whereas addition of more IFs and of the hrpA RNA helicase had no substantial effects. Moreover, the depletion of tRNAs as peptidyl-tRNAs can become limiting in PUREfrex but not in PURExpress, which can be alleviated by addition of peptidyl-tRNA-hydrolase PTH. We attempted to build a new model for PURE system dynamics integrating all experimental observations. Although a satisfying global fit can be obtained in specific conditions with PTH, a unifying systems level model is still missing.</div>','<b>Title: </b>Search for a Minimal Set of Parameters by Assessing the Total Optimization Potential for a Dynamic Model of a Biochemical Network<br><b>Year: </b>2017<br><b>Author: </b> E. Stalidzans,  I. Mozga,  J. Sulins and  P. Zikmanis<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1109/TCBB.2016.2550451"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IEEE-ACM TRANSACTIONS ON COMPUTATIONAL BIOLOGY AND BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>kinetic models, metabolic networks, kinetic parameters, dynamic models, systems biology, network model, dynamic metabolic, biochemical network, rate constants, enzyme kinetics, kinetic data, adjustable parameters, metabolic model, parameter estimation, dynamic optimization, model selection, mathematical model, kinetic network, model response, model<div style="font-size:11px;"><br><b>Abstract: </b><em>Selecting an efficient small set of adjustable parameters to improve metabolic features of an organism is important for a reduction of implementation costs and risks of unpredicted side effects. In practice, to avoid the analysis of a huge combinatorial space for the possible sets of adjustable parameters, experience-, and intuition-based subsets of parameters are often chosen, possibly leaving some interesting counter-intuitive combinations of parameters unrevealed. The combinatorial scan of possible adjustable parameter combinations at the model optimization level is possible; however, the number of analyzed combinations is still limited. The total optimization potential TOP approach is proposed to assess the full potential for increasing the value of the objective function by optimizing all possible adjustable parameters. This seemingly unpractical combination of adjustable parameters allows assessing the maximum attainable value of the objective function and stopping the combinatorial space scanning when the desired fraction of TOP is reached and any further increase in the number of adjustable parameters cannot bring any reasonable improvement. The relation between the number of adjustable parameters and the reachable fraction of TOP is a valuable guideline in choosing a rational solution for industrial implementation. The TOP approach is demonstrated on the basis of two case studies.</div>','<b>Title: </b>DOTcvpSB, a software toolbox for dynamic optimization in systems biology<br><b>Year: </b>2009<br><b>Author: </b> T. Hirmajer,  E. Balsa-Canto and  Jr. Banga<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1471-2105-10-199"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>kinetic models, metabolic networks, kinetic parameters, dynamic models, systems biology, network model, dynamic metabolic, biochemical network, rate constants, enzyme kinetics, kinetic data, adjustable parameters, metabolic model, parameter estimation, dynamic optimization, model selection, mathematical model, kinetic network, model response, model<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Mathematical optimization aims to make a system or design as effective or functional as possible, computing the quality of the different alternatives using a mathematical model. Most models in systems biology have a dynamic nature, usually described by sets of differential equations. Dynamic optimization addresses this class of systems, seeking the computation of the optimal time-varying conditions control variables to minimize or maximize a certain performance index. Dynamic optimization can solve many important problems in systems biology, including optimal control for obtaining a desired biological performance, the analysis of network designs and computer aided design of biological units.Results: Here, we present a software toolbox, DOTcvpSB, which uses a rich ensemble of state-of-the-art numerical methods for solving continuous and mixed-integer dynamic optimization MIDO problems. The toolbox has been written in MATLAB and provides an easy and user friendly environment, including a graphical user interface, while ensuring a good numerical performance. Problems are easily stated thanks to the compact input definition. The toolbox also offers the possibility of importing SBML models, thus enabling it as a powerful optimization companion to modelling packages in systems biology. It serves as a means of handling generic black-box models as well.Conclusion: Here we illustrate the capabilities and performance of DOTcvpSB by solving several challenging optimization problems related with bioreactor optimization, optimal drug infusion to a patient and the minimization of intracellular oscillations. The results illustrate how the suite of solvers available allows the efficient solution of a wide class of dynamic optimization problems, including challenging multimodal ones. The toolbox is freely available for academic use.</div>','<b>Title: </b>Network Analyses Reveal Novel Aspects of ALS Pathogenesis<br><b>Year: </b>2015<br><b>Author: </b> M. Sanhueza,  A. Chai,  C. Smith,  Ba. Mccray,  Ti. Simpson,  Jp. Taylor and  G. Pennetta<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pgen.1005107"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS GENETICS<br><b>Cluster-Keywords: </b><em>gene regulatory, systems biology, regulatory networks, gene network, stem cell, network model, transcription factors, gene expression, cell transcriptomes, network analysis, data analysis, stem cells, network grnsight, cell transformation, weighted network, gene ontology, networks grns, graph model, graph layout, endothelial cell<div style="font-size:11px;"><br><b>Abstract: </b><em>Amyotrophic Lateral Sclerosis ALS is a fatal neurodegenerative disease characterized by selective loss of motor neurons, muscle atrophy and paralysis. Mutations in the human VAMP-associated protein B hVAPB cause a heterogeneous group of motor neuron diseases including ALS8. Despite extensive research, the molecular mechanisms underlying ALS pathogenesis remain largely unknown. Genetic screens for key interactors of hVAPB activity in the intact nervous system, however, represent a fundamental approach towards understanding the in vivo function of hVAPB and its role in ALS pathogenesis. Targeted expression of the disease-causing allele leads to neurodegeneration and progressive decline in motor performance when expressed in the adult Drosophila, eye or in its entire nervous system, respectively. By using these two phenotypic readouts, we carried out a systematic survey of the Drosophila genome to identify modifiers of hVAPB-induced neurotoxicity. Modifiers cluster in a diverse array of biological functions including processes and genes that have been previously linked to hVAPB function, such as proteolysis and vesicular trafficking. In addition to established mechanisms, the screen identified endocytic trafficking and genes controlling proliferation and apoptosis as potent modifiers of ALS8-mediated defects. Surprisingly, the list of modifiers was mostly enriched for proteins linked to lipid droplet biogenesis and dynamics. Computational analysis reveals that most modifiers can be linked into a complex network of interacting genes, and that the human genes homologous to the Drosophila modifiers can be assembled into an interacting network largely overlapping with that in flies. Identity markers of the endocytic process were also found to abnormally accumulate in ALS patients, further supporting the relevance of the fly data for human biology. Collectively, these results not only lead to a better understanding of hVAPB function but also point to potentially relevant targets for therapeutic intervention.</div>','<b>Title: </b>Critical perspective on the consequences of the limited availability of kinetic data in metabolic dynamic modelling<br><b>Year: </b>2011<br><b>Author: </b> Rs. Costa,  D. Machado,  I. Rocha and  Ec. Ferreira<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1049/iet-syb.2009.0058"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IET SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>kinetic models, metabolic networks, kinetic parameters, dynamic models, systems biology, network model, dynamic metabolic, biochemical network, rate constants, enzyme kinetics, kinetic data, adjustable parameters, metabolic model, parameter estimation, dynamic optimization, model selection, mathematical model, kinetic network, model response, model<div style="font-size:11px;"><br><b>Abstract: </b><em>Detailed kinetic models at the network reaction level are usually constructed using enzymatic mechanistic rate equations and the associated kinetic parameters. However, during the cellular life cycle thousands of different reactions occur, which makes it very difficult to build a detailed large-scale ldnetic model. In this work, we provide a critical overview of specific limitations found during the reconstruction of the central carbon metabolism dynamic model from E. coli based on kinetic data available. In addition, we provide clues that will hopefully allow the systems biology community to more accurately construct metabolic dynamic models in the future. The difficulties faced during the construction of dynamic models are due not only to the lack of kinetic information but also to the fact that some data are still not curated. We hope that in the future, with the standardization of the in vitro enzyme protocols the approximation of in vitro conditions to the in vivo ones, it will be possible to integrate the available kinetic data into a complete large scale model. We also expect that collaborative projects between modellers and biologists will provide valuable kinetic data and permit the exchange of important information to solve most of these issues.</div>','<b>Title: </b>Amplification without instability: applying fluid dynamical insights in chemistry and biology<br><b>Year: </b>2013<br><b>Author: </b> Jh. Mccoy<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1088/1367-2630/15/11/113036"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>NEW JOURNAL OF PHYSICS<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>While amplification of small perturbations often arises from instability, transient amplification is possible locally even in asymptotically stable systems. That is, knowledge of a systems stability properties can mislead ones intuition for its transient behaviors. This insight, which has an interesting history in fluid dynamics, has more recently been rediscovered in ecology. Surprisingly, many nonlinear fluid dynamical and ecological systems share linear features associated with transient amplification of noise. This paper aims to establish that these features are widespread in many other disciplines concerned with noisy systems, especially chemistry, cell biology and molecular biology. Here, using classic nonlinear systems and the graphical language of network science, we explore how the noise amplification problem can be reframed in terms of activatory and inhibitory interactions between dynamical variables. The interaction patterns considered here are found in a great variety of systems, ranging from autocatalytic reactions and activator-inhibitor systems to influential models of nerve conduction, glycolysis, cell signaling and circadian rhythms.</div>','<b>Title: </b>An Analytical Tool that Quantifies Cellular Morphology Changes from Three-dimensional Fluorescence Images<br><b>Year: </b>2012<br><b>Author: </b> Cl. Haass-Koffler,  M. Naeemuddin and  Se. Bartlett<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3791/4233"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOVE-JOURNAL OF VISUALIZED EXPERIMENTS<br><b>Cluster-Keywords: </b><em>boolean networks, regulatory networks, network model, bayesian networks, model networks, systems biology, random boolean, real cells, immune system, genetic regulatory, networks, biochemical network, fluorescence images, cellular network, gene regulatory, cell differentiation, network, biological systems, ensemble approach, cell dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>The most common software analysis tools available for measuring fluorescence images are for two-dimensional 2D data that rely on manual settings for inclusion and exclusion of data points, and computer-aided pattern recognition to support the interpretation and findings of the analysis. It has become increasingly important to be able to measure fluorescence images constructed from three-dimensional 3D datasets in order to be able to capture the complexity of cellular dynamics and understand the basis of cellular plasticity within biological systems. Sophisticated microscopy instruments have permitted the visualization of 3D fluorescence images through the acquisition of multispectral fluorescence images and powerful analytical software that reconstructs the images from confocal stacks that then provide a 3D representation of the collected 2D images. Advanced design-based stereology methods have progressed from the approximation and assumptions of the original model-based stereology1 even in complex tissue sections2. Despite these scientific advances in microscopy, a need remains for an automated analytic method that fully exploits the intrinsic 3D data to allow for the analysis and quantification of the complex changes in cell morphology, protein localization and receptor trafficking.Current techniques available to quantify fluorescence images include Meta-Morph Molecular Devices, Sunnyvale, CA and Image J NIH which provide manual analysis. Imaris Andor Technology, Belfast, Northern Ireland software provides the feature MeasurementPro, which allows the manual creation of measurement points that can be placed in a volume image or drawn on a series of 2D slices to create a 3D object. This method is useful for single-click point measurements to measure a line distance between two objects or to create a polygon that encloses a region of interest, but it is difficult to apply to complex cellular network structures. Filament Tracer Andor allows automatic detection of the 3D neuronal filament-like however, this module has been developed to measure defined structures such as neurons, which are comprised of dendrites, axons and spines tree-like structure. This module has been ingeniously utilized to make morphological measurements to non-neuronal cells3, however, the output data provide information of an extended cellular network by using a software that depends on a defined cell shape rather than being an amorphous-shaped cellular model. To overcome the issue of analyzing amorphous-shaped cells and making the software more suitable to a biological application, Imaris developed Imaris Cell. This was a scientific project with the Eidgenossische Technische Hochschule, which has been developed to calculate the relationship between cells and organelles. While the software enables the detection of biological constraints, by forcing one nucleus per cell and using cell membranes to segment cells, it cannot be utilized to analyze fluorescence data that are not continuous because ideally it builds cell surface without void spaces. To our knowledge, at present no user-modifiable automated approach that provides morphometric information from 3D fluorescence images has been developed that achieves cellular spatial information of an undefined shape Figure 1.We have developed an analytical platform using the Imaris core software module and Imaris XT interfaced to MATLAB Mat Works, Inc.. These tools allow the 3D measurement of cells without a pre-defined shape and with inconsistent fluorescence network components. Furthermore, this method will allow researchers who have extended expertise in biological systems, but not familiarity to computer applications, to perform quantification of morphological changes in cell dynamics.</div>','<b>Title: </b>The number and probability of canalizing functions<br><b>Year: </b>2004<br><b>Author: </b> W. Just,  I. Shmulevich and  J. Konvalina<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.physd.2004.07.002"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PHYSICA D-NONLINEAR PHENOMENA<br><b>Cluster-Keywords: </b><em>boolean network, boolean functions, regulatory networks, canalizing functions, network models, gene regulatory, boolean control, control networks, biological networks, boolean models, update functions, interaction networks, large network, regulatory functions, canalizing boolean, genetic regulatory, systems biology, driver nodes, networks bcns, biological systems<div style="font-size:11px;"><br><b>Abstract: </b><em>Canalizing functions have important applications in physics and biology. For example, they represent a mechanism capable of stabilizing chaotic behavior in Boolean network models of discrete dynamical systems. When comparing the class of canalizing functions to other classes of functions with respect to their evolutionary plausibility as emergent control rules in genetic regulatory systems, it is informative to know the number of canalizing functions with a given number of input variables. This is also important in the context of using the class of canalizing functions as a constraint during the inference of genetic networks from gene expression data. To this end, we derive an exact formula for the number of canalizing Boolean functions of n variables. We also derive a formula for the probability that a random Boolean function is canalizing for any given bias p of taking the value 1. In addition, we consider the number and probability of Boolean functions that are canalizing for exactly k variables. Finally, we provide an algorithm for randomly generating canalizing functions with a given bias p and any number of variables, which is needed for Monte Carlo simulations of Boolean networks. C 2004 Elsevier B.V. All rights reserved.</div>','<b>Title: </b>STOCHASTIC-ANALYSIS OF SYNCHRONOUS NEURAL NETWORKS WITH ASYMMETRIC WEIGHTS<br><b>Year: </b>1991<br><b>Author: </b> O. Bernier<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1209/0295-5075/16/6/004"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>EUROPHYSICS LETTERS<br><b>Cluster-Keywords: </b><em>network models, gene network, regulatory networks, boolean networks, gene expression, systems biology, neural networks, network dynamics, gene regulatory, biological networks, network structure, gap gene, gene regulation, cell cycle, network, gene interactions, signaling network, quantum systems, system dynamics, networks<div style="font-size:11px;"><br><b>Abstract: </b><em>We present a method to analyse the properties of a synchronous stochastic neural network with asymmetric connections.  We obtain an equivalence between the dynamics of the network and the iteration defined by a deterministic continuous mapping.  This approach enables us to predict the existence of a bifurcation from a stable fixed point to an attracting cycle in a simple case.</div>','<b>Title: </b>Scalable Rule-Based Modelling of Allosteric Proteins and Biochemical Networks<br><b>Year: </b>2010<br><b>Author: </b> Jf. Ollivier,  V. Shahrezaei and  Ps. Swain<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pcbi.1000975"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS COMPUTATIONAL BIOLOGY<br><b>Cluster-Keywords: </b><em>allosteric proteins, signaling networks, protein interactions, network model, molecular dynamics, protein complexes, allosteric network, signaling proteins, cell network, biochemical networks, free energy, combinatorial complexity, molecular species, biological systems, receptor signaling, systems biology, chemical species, model, dynamics simulations, cell signaling<div style="font-size:11px;"><br><b>Abstract: </b><em>Much of the complexity of biochemical networks comes from the information-processing abilities of allosteric proteins, be they receptors, ion-channels, signalling molecules or transcription factors. An allosteric protein can be uniquely regulated by each combination of input molecules that it binds. This "regulatory complexity" causes a combinatorial increase in the number of parameters required to fit experimental data as the number of protein interactions increases. It therefore challenges the creation, updating, and re-use of biochemical models. Here, we propose a rule-based modelling framework that exploits the intrinsic modularity of protein structure to address regulatory complexity. Rather than treating proteins as "black boxes", we model their hierarchical structure and, as conformational changes, internal dynamics. By modelling the regulation of allosteric proteins through these conformational changes, we often decrease the number of parameters required to fit data, and so reduce over-fitting and improve the predictive power of a model. Our method is thermodynamically grounded, imposes detailed balance, and also includes molecular cross-talk and the background activity of enzymes. We use our Allosteric Network Compiler to examine how allostery can facilitate macromolecular assembly and how competitive ligands can change the observed cooperativity of an allosteric protein. We also develop a parsimonious model of G protein-coupled receptors that explains functional selectivity and can predict the rank order of potency of agonists acting through a receptor. Our methodology should provide a basis for scalable, modular and executable modelling of biochemical networks in systems and synthetic biology.</div>','<b>Title: </b>Robustness of complex feedback systems: application to oncological biochemical networks<br><b>Year: </b>2013<br><b>Author: </b> F. Bianconi,  E. Baldelli,  V. Ludovini,  L. Crino,  K. Perruccio and  P. Valigi<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1080/00207179.2013.800646"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>INTERNATIONAL JOURNAL OF CONTROL<br><b>Cluster-Keywords: </b><em>feedback loops, signaling network, positive feedback, network model, negative feedback, biochemical networks, signaling pathways, signal transduction, systems biology, kinase erk, protein kinase, transduction networks, dynamic model, emt network, erk pathway, cell fate, cell death, biochemical reaction, coupled feedback, impulse response<div style="font-size:11px;"><br><b>Abstract: </b><em>Biochemical transduction networks can be modelled through a proper set of differential equations, and at the same time they can be experimentally analysed only by measuring a few signals at the end of the cascades. The study of those networks is of special importance in oncology.The approach proposed in this paper is aimed at characterising the network robustness with respect to both parameters and initial condition perturbations. The key idea is to introduce a robustness index, the proliferation index, and to study its behaviour over the parameter space. The index relates the measurable signals, and the shape of their time behaviour, to the model parameters and initial conditions. In addition, the paper will also provide specific results for a dynamic model of the EGFR-IGFR receptor pathway, which turns out to be relevant to the study of some specific pathologies, such as lung cancer. The connection between the proposed robustness index and the pathology will also be addressed.</div>','<b>Title: </b>Systems biology approaches to abscisic acid signaling<br><b>Year: </b>2011<br><b>Author: </b> T. Umezawa<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s10265-011-0418-x"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF PLANT RESEARCH<br><b>Cluster-Keywords: </b><em>network ecology, food webs, network models, ecological systems, food web, systems biology, ecological networks, ecosystem model, signalling networks, model results, interaction networks, dynamic model, ecological interactions, network analysis, model network, network structure, transport networks, systems network, ecosystem dynamics, restoration ecology<div style="font-size:11px;"><br><b>Abstract: </b><em>Recent advances in our understanding of abscisic acid ABA signaling have identified a core pathway consisting of receptors PYR/PYL/RCAR, protein phosphatases PP2C, protein kinases SnRK2, and several downstream factors that will lead to the next stage of ABA research. Systems biology will be an important concept for further understanding ABA responses in plants. In this review, two practical approaches of systems biology to ABA signaling are presented: the one is transcriptome analysis, which covers coding genes as well as unannotated transcripts, and the other is phosphoproteomics. The latter technology will offer an unprecedented overview of the regulatory networks involved in ABA signaling because protein phosphorylation/dephosphorylation is a major center of such regulation. Systematic studies will contribute to our understanding of the network structure and dynamics of ABA signaling; moreover, systems biology will facilitate ABA signaling studies as well as future biotechnological applications in crops or trees.</div>','<b>Title: </b>Hybrid dual-complementary metal-oxide-semiconductor/memristor synapse-based neural network with its applications in image super-resolution<br><b>Year: </b>2019<br><b>Author: </b> Zk. Dong,  Cs. Lai,  Yf. He,  Dl. Qi and  Sk. Duan<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1049/iet-cds.2018.5062"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IET CIRCUITS DEVICES  SYSTEMS<br><b>Cluster-Keywords: </b><em>chaotic system, control systems, neural networks, jerk chaotic, torch chaotic, mode control, sliding mode, plasma torch, dynamic neural, mitochondrial dynamics, system, systems biology, adaptive integral, model system, rodtype plasma, integral sliding, systems, jerk system, cancer cell, comparison method<div style="font-size:11px;"><br><b>Abstract: </b><em>Biology-inspired neural computing is a potential candidate for the implementation of next-generation intelligent systems. Memristor is a passive electrical element with resistance-switching dynamics. Owing to its natural advantages of non-volatility, nanoscale geometries, and variable conductance, memristor can effectively simulate the synaptic connecting strength between the neurones in the multilayer neural networks. This study presents a kind of memristor synapse-based multilayer neural network hardware architecture with a suitable training methodology. Specifically, a novel dual-complementary metal-oxide-semiconductor/memristor synaptic circuit is presented, which is capable of performing the negative, zero, and positive synaptic weights via controlling the direction of current passing through the memristors. Then, the neurone circuit synthesised with multiple synaptic circuits and an activation unit is further designed, which can be utilised to constitute a compact multilayer neural network with fully connected configuration. Also, a hardware-friendly chip-in-the-loop training method is provided during the network training phase. For the verification purpose, the presented neural network is applied for the realisation of single image super-resolution reconstruction.</div>','<b>Title: </b>An Automated Biomodel Selection System BMSS for Gene Circuit Designs<br><b>Year: </b>2019<br><b>Author: </b> Jw. Yeoh,  Kbi. Ng,  Ay. Teh,  Jy. Zhang,  Wkd. Chee and  Cl. Poh<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1021/acssynbio.8b00523"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ACS SYNTHETIC BIOLOGY<br><b>Cluster-Keywords: </b><em>gene circuits, synthetic biology, circuit behavior, biological parts, synthetic gene, circuit design, genetic circuits, cell types, metabolite dynamics, mathematical model, designed circuit, control circuit, biological systems, circuit models, synthetic circuit, feedback circuits, biological circuits, design models, metabolic pathways, circuit parameters<div style="font-size:11px;"><br><b>Abstract: </b><em>Constructing a complex functional gene circuit composed of different modular biological parts to achieve the desired performance remains challenging without a proper understanding of how the individual module behaves. To address this, mathematical models serve as an important tool toward better interpretation by quantifying the performance of the overall gene circuit, providing insights, and guiding the experimental designs. As different gene circuits might require exclusively different mathematical representations in the form of ordinary differential equations to capture their transient dynamic behaviors, a recurring challenge in model development is the selection of the appropriate model. Here, we developed an automated biomodel selection system BMSS which includes a library of pre-established models with intuitive or unintuitive features derived from a vast array of expression profiles. Selection of models is built upon the Akaike information criteria AIC. We tested the automated platform using characterization data of routinely used inducible systems, constitutive expression systems, and several different logic gate systems NOT, AND, and OR gates. The BMSS achieved a good agreement for all the different characterization data sets and managed to select the most appropriate model accordingly. To enable exchange and reproducibility of gene circuit design models, the BMSS platform also generates Synthetic Biology Open Language SBOLcompliant gene circuit diagrams and Systems Biology Markup Language SBML output files. All aspects of the algorithm were programmed in a modular manner to ease the efforts on model extensions or customizations by users. Taken together, the BMSS which is implemented in Python supports users in deriving the best mathematical model candidate in a fast, efficient, and automated way using part/circuit characterization data.</div>','<b>Title: </b>Neural stem cells deriving from chick embryonic hindbrain recapitulate hindbrain development in culture<br><b>Year: </b>2018<br><b>Author: </b> Y. Peretz,  A. Kohl,  N. Slutsky,  M. Komlos,  S. Varshavsky and  D. Sela-Donenfeld<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1038/s41598-018-32203-w"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SCIENTIFIC REPORTS<br><b>Cluster-Keywords: </b><em>gene regulatory, regulatory network, systems biology, cell differentiation, cell fate, model systems, potential landscape, network grn, dynamical systems, melanocyte differentiation, network models, gene expression, network dynamics, differentiation genes, dynamical models, dynamical networks, cellular differentiation, stem cells, attractor network, developmental gene<div style="font-size:11px;"><br><b>Abstract: </b><em>Neural stem cells NSCs are self-renewing multipotent cells that line the neural-tube and generate all the nervous system. Understanding NSC biology is fundamental for neurodevelopmental research and therapy. Many studies emphasized the need to culture NSCs, which are typically purified from mammalian embryonic/adult brains. These sources are somewhat limited in terms of quantity, availability and animal ethical guidelines. Therefore, new sources are needed. The chick is a powerful system for experimental embryology which contributed enormously to neurodevelopmental concepts. Its accessibility, genetic/molecular manipulations, and homology to other vertebrates, makes it valuable for developmental biology research. Recently, we identified a population of NSCs in the chick hindbrain. It resides in rhombomere-boundaries, expresses Sox2 and generates progenitors and neurons. Here, we investigated whether these cells can recapitulate hindbrain development in culture. By developing approaches to propagate and image cells, manipulate their growth-conditions and separate them into subpopulations, we demonstrate the ordered formation of multipotent and self-renewing neurospheres that maintain regional identity and display differential stem/differentiation/proliferation properties. Live imaging revealed new cellular dynamics in the culture. Collectively, these NSC cultures reproduce major aspects of hindbrain development in-vitro, proposing the chick as a model for culturing hindbrain-NSCs that can be directly applied to other neural-tube domains and species.</div>','<b>Title: </b>Sparse Regression Based Structure Learning of Stochastic Reaction Networks from Single Cell Snapshot Time Series<br><b>Year: </b>2016<br><b>Author: </b> A. Klimovskaia,  S. Ganscha and  M. Claassen<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pcbi.1005234"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS COMPUTATIONAL BIOLOGY<br><b>Cluster-Keywords: </b><em>regulatory networks, gene regulatory, gene expression, network inference, expression data, network structure, gene networks, networks grns, regulatory genes, boolean networks, bayesian network, network reconstruction, networks gene, network dynamics, gene regulation, inference method, systems biology, time series, infer gene, network<div style="font-size:11px;"><br><b>Abstract: </b><em>Stochastic chemical reaction networks constitute a model class to quantitatively describe dynamics and cell-to-cell variability in biological systems. The topology of these networks typically is only partially characterized due to experimental limitations. Current approaches for refining network topology are based on the explicit enumeration of alternative topologies and are therefore restricted to small problem instances with almost complete knowledge. We propose the reactionet lasso, a computational procedure that derives a stepwise sparse regression approach on the basis of the Chemical Master Equation, enabling large-scale structure learning for reaction networks by implicitly accounting for billions of topology variants. We have assessed the structure learning capabilities of the reactionet lasso on synthetic data for the complete TRAIL induced apoptosis signaling cascade comprising 70 reactions. We find that the reactionet lasso is able to efficiently recover the structure of these reaction systems, ab initio, with high sensitivity and specificity. With only <1 false discoveries, the reactionet lasso is able to recover 45 of all true reactions ab initio among >6000 possible reactions and over 10 2000 network topologies. In conjunction with information rich single cell technologies such as single cell RNA sequencing or mass cytometry, the reactionet lasso will enable large-scale structure learning, particularly in areas with partial network structure knowledge, such as cancer biology, and thereby enable the detection of pathological alterations of reaction networks. We provide software to allow for wide applicability of the reactionet lasso.</div>','<b>Title: </b>Scaling and Integration of Kinetic Models of Photosynthesis: Towards Comprehensive E-Photosynthesis<br><b>Year: </b>2009<br><b>Author: </b> L. Nedbal,  J. Cerveny and  H. SchmidtLink: No link<br><b>Journal: </b>PHOTOSYNTHESIS IN SILICO: UNDERSTANDING COMPLEXITY FROM MOLECULES TO ECOSYSTEMS<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Mathematical models are essential to understand dynamic behavior of complex biological systems. Photosynthesis as it occurs in a natural environment reflects not only the primary biophysical and biochemical reactions but also a network of regulatory interactions that act across timescales and spatial boundaries. Modeling such a tightly regulated biosystem is feasible when the model is reduced to describe only a rather particular experimental situation such as fluorescence response to a single turnover light flash or the dynamics around the steady-state of Calvin-Benson cycle. Then, the external regulatory interactions can be considered negligible or not changing so that the investigated dynamics can be predicted by modeling the system with only few key components that are relevant for the given time and complexity scale. Such an empiric dimensionality reduction has been successfully applied in photosynthesis research, leading to a mosaic of partial models that map along the Z-scheme of light reactions as well as covering parts of carbon metabolism. The validity ranges of the partial models are frequently not overlapping, leaving gaps in the photosynthesis modeling space. Filling the gaps and, even more important, modeling of regulatory interactions between modeled entities are hampered by incompatibility of the partial models that focus on different time scales or that are restricted to particular experimental situations. This led us to propose the Comprehensive Modeling Space, CMS where the partial photosynthesis models would be shared by means of the Systems Biology Mark-Up Language, SBML, which is the de-facto standard for the formal representation of biochemical models. The model validity is defined by a customized extension of the biology-wide standard of Minimum Information Requested in the Annotation of Biochemical Models, MIRIAM. The hierarchy and connectivity of the partial models within the Comprehensive Modeling Space is determined by rigorous dimensionality reduction techniques. Here, we exemplify the principles of the comprehensive modeling approach based on partial models of the Photosystem II.</div>','<b>Title: </b>Dynamic modelling and analysis of biochemical networks: mechanism-based models and model-based experiments<br><b>Year: </b>2006<br><b>Author: </b> Naw. Van Riel<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1093/bib/bbl040"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BRIEFINGS IN BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Systems biology applies quantitative, mechanistic modelling to study genetic networks, signal transduction pathways and metabolic networks. Mathematical models of biochemical networks can look very different. An important reason is that the purpose and application of a model are essential for the selection of the best mathematical framework. Fundamental aspects of selecting an appropriate modelling framework and a strategy for model building are discussed.Concepts and methods from system and control theory provide a sound basis for the further development of improved and dedicated computational tools for systems biology. Identification of the network components and rate constants that are most critical to the output behaviour of the system is one of the major problems raised in systems biology. Current approaches and methods of parameter sensitivity analysis and parameter estimation are reviewed. It is shown how these methods can be applied in the design of model-based experiments which iteratively yield models that are decreasingly wrong and increasingly gain predictive power.</div>','<b>Title: </b>Hydrology: the changing paradigm<br><b>Year: </b>2002<br><b>Author: </b> Nj. Clifford<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1191/0309133302pp337pr"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PROGRESS IN PHYSICAL GEOGRAPHY<br><b>Cluster-Keywords: </b><em>interaction network, actin filament, network model, filament network, network structures, actin network, protein interaction, actin dynamics, network, membrane protein, network dynamics, model dynamic, cell, model, network approach, protein complex, computational model, actin, adhesion dynamics, cell dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>In common with the earth sciences, hydrology has an interesting classical and medieval parentage, motivated by practical as well as philosophical concerns with the origins and agency of water Biswas, 1970. More formally, hydrology is an established twentieth-century environmental science, whose content is necessarily interdisciplinary spanning civil engineering, biology, chemistry, physics, geomorphology and, increasingly, the social environmental sciences such as planning, economics and political conflict resolution, yet whose identity is maintained internationally by a strong institutional structure. In 1922, the International Association of Scientific Hydrology renamed the International Association of Hydrological Sciences IAHS in 1974 was established as the hydrological section of the International Union of Geodesy and Geophysics, concerned with the study of the circulation and distribution of water in the natural environment Gurnell, 1994. By the 1940s, hydrology was firmly established as a professional discipline with a burgeoning supply of textbooks to support teaching within the universities as well as practice in government agencies Shaw, 1994. Four-yearly General Assemblies within the International Union of Geodesy and Geophysics IUGG, together with Scientific Assemblies in between, have maintained both academic and applied hydrological discourse at an international level, assisted by other international agencies such as the United Nations UN and World Meteorological Organization WMO. The International Hydrological Decade became the International Hydrological Programme of UNESCO in 1975, and is now in its fifth phase; and the WMOs Hydrology and Water Resources Programme is 40 years old.</div>','<b>Title: </b>From Immune Cells to Self-Organizing Ultra-Dense Small Cell Networks<br><b>Year: </b>2016<br><b>Author: </b> H. Klessig,  D. Ohmann,  Ai. Reppas,  H. Hatzikirou,  M. Abedi,  M. Simsek and  Gp. Fettweis<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1109/JSAC.2016.2544638"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IEEE JOURNAL ON SELECTED AREAS IN COMMUNICATIONS<br><b>Cluster-Keywords: </b><em>systems biology, regulatory network, interaction networks, network model, gene regulatory, network, network biology, networks, gene network, response networks, network dynamics, dynamic model, cell networks, gene expression, biological networks, system dynamics, dynamic network, model, biological systems, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>In order to cope with the wireless traffic demand explosion within the next decade, operators are underlying their macrocellular networks with low power base stations in a more dense manner. Such networks are typically referred to as heterogeneous or ultra-dense small cell networks, and their deployment entails a number of challenges in terms of backhauling, capacity provision, and dynamics in spatio-temporally fluctuating traffic load. Self-organizing network SON solutions have been defined to overcome these challenges. Since self-organization occurs in a plethora of biological systems, we identify the design principles of immune system self-regulation and draw analogies with respect to ultra-dense small cell networks. In particular, we develop a mathematical model of an artificial immune system AIS that autonomously activates or deactivates small cells in response to the local traffic demand. The main goal of the proposed AIS-based SON approach is the enhancement of energy efficiency and improvement of cell-edge throughput. As a proof of principle, system level simulations are carried out in which the bio-inspired algorithm is evaluated for various parameter settings, such as the speed of small cell activation and the delay of deactivation. Analysis using spatio-temporally varying traffic exhibiting uncertainty through geo-location demonstrates the robustness of the AIS-based SON approach proposed.</div>','<b>Title: </b>Food web structure and vulnerability of a deep-sea ecosystem in the NW Mediterranean Sea<br><b>Year: </b>2013<br><b>Author: </b> S. Tecchio,  M. Coll,  V. Christensen,  Jb. Company,  E. Ramirez-Llodra and  F. Sarda<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.dsr.2013.01.003"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>DEEP-SEA RESEARCH PART I-OCEANOGRAPHIC RESEARCH PAPERS<br><b>Cluster-Keywords: </b><em>network ecology, food webs, network models, ecological systems, food web, systems biology, ecological networks, ecosystem model, signalling networks, model results, interaction networks, dynamic model, ecological interactions, network analysis, model network, network structure, transport networks, systems network, ecosystem dynamics, restoration ecology<div style="font-size:11px;"><br><b>Abstract: </b><em>There is increasing fishing pressure on the continental margins of the oceans, and this raises concerns about the vulnerability of the ecosystems thriving there. The current knowledge of the biology of deep-water fish species identifies potential reduced resilience to anthropogenic disturbance. However, there are extreme difficulties in sampling the deep sea, resulting in poorly resolved and indirectly obtained food-web relationships. Here, we modelled the flows and biomasses of a Mediterranean deep-sea ecosystem, the Catalan Sea continental slope at depths of 1000-1400 m. This is the first model of a deep-water ecosystem in the Mediterranean Sea. The objectives were to a quantitatively describe the food web structure of the ecosystem, b examine the role of key species in the ecosystem, and c explore the vulnerability of this deep-sea ecosystem to potential future fishing exploitation. We used the Ecopath with Ecosim EwE modelling approach and software to model the ecosystem. The trophic model included 18 consumers, a marine snow group, and a sediment detritus group. Trophic network analysis identified low levels of consumer biomass cycling and low system omnivory index when compared with expected values of marine ecosystems, and higher cycling and omnivory when compared with available EwE models of shallower areas of the Mediterranean Sea. The majority of flows in the ecosystem were concentrated at the trophic level of first-order consumers TL 2. Benthic invertebrates and demersal sharks were identified to have key ecological roles in the ecosystem. We used the dynamic temporal model Ecosim to simulate expansion of the red-shrimp benthic trawl fishery that currently operates at shallower depths, down to 800 m depth. The simulations showed reductions in fish biomass and that the state of the deep continental slope ecosystem in the western Mediterranean seems to be the result of a long-term succession process, which has reached ecological stability, and is particularly vulnerable to human impact and, specifically, to fisheries exploitation. C 2013 Elsevier Ltd. All rights reserved.</div>','<b>Title: </b>Drought and Recovery: Independently Regulated Processes Highlighting the Importance of Protein Turnover Dynamics and Translational Regulation in Medicago truncatula<br><b>Year: </b>2016<br><b>Author: </b> D. Lyon,  Ma. Castillejo,  V. Mehmeti-Tershani,  C. Staudinger,  C. Kleemaier and  S. Wienkoop<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1074/mcp.M115.049205"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MOLECULAR  CELLULAR PROTEOMICS<br><b>Cluster-Keywords: </b><em>gene expression, systems biology, metabolic network, network model, network analysis, dynamic model, signaling pathways, signaling network, regulatory network, dynamic network, biology approach, candidate genes, gene network, gene regulatory, biology analysis, mathematical model, network dynamic, biology network, metabolic pathways, pathway analysis<div style="font-size:11px;"><br><b>Abstract: </b><em>Climate change in conjunction with population growth necessitates a systems biology approach to characterize plant drought acclimation as well as a more thorough understanding of the molecular mechanisms of stress recovery. Plants are exposed to a continuously changing environment. Extremes such as several weeks of drought are followed by rain. This requires a molecular plasticity of the plant enabling drought acclimation and the necessity of deacclimation processes for recovery and continuous growth. During drought stress and subsequent recovery, the metabolome and proteome are regulated through a sequence of molecular processes including synthesis and degradation and molecular interaction networks are part of this regulatory process. In order to study this complex regulatory network, a comprehensive analysis is presented for the first time, investigating protein turnover and regulatory classes of proteins and metabolites during a stress recovery scenario in the model legume Medicago truncatula. The data give novel insights into the molecular capacity and differential processes required for acclimation and deacclimation of severe drought stressed plants. Functional cluster and network analyses unraveled independent regulatory mechanisms for stress and recovery with different dynamic phases that during the course of recovery define the plants deacclimation from stress. The combination of relative abundance levels and turnover analysis revealed an early transition phase that seems key for recovery initiation through water resupply and is independent from renutrition. Thus, a first indication for a metabolite and protein-based load capacity was observed necessary for the recovery from drought, an important but thus far ignored possible feature toward tolerance. The data indicate that apart from the plants molecular stress response mechanisms, plasticity may be related to the nutritional status of the plant prior to stress initiation. A new perspective and possible new targets as well as metabolic mechanisms for future plant-bioengineering toward enhanced drought stress tolerance are presented.</div>','<b>Title: </b>Inference of Boolean Networks Using Sensitivity Regularization<br><b>Year: </b>2008<br><b>Author: </b> Wb. Liu,  H. Lahadesmaki,  Er. Dougherty and  I. Shmulevich<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1155/2008/780541"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>EURASIP JOURNAL ON BIOINFORMATICS AND SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>boolean networks, regulatory networks, genetic regulatory, gene regulatory, probabilistic boolean, network, networks, model, network model, probabilistic model, model checking, networks pbns, network dynamics, markov chain, regulatory, models, optimal control, boolean, networks models, networks boolean<div style="font-size:11px;"><br><b>Abstract: </b><em>The inference of genetic regulatory networks from global measurements of gene expressions is an important problem in computational biology. Recent studies suggest that such dynamical molecular systems are poised at a critical phase transition between an ordered and a disordered phase, affording the ability to balance stability and adaptability while coordinating complex macroscopic behavior. We investigate whether incorporating this dynamical system-wide property as an assumption in the inference process is beneficial in terms of reducing the inference error of the designed network. Using Boolean networks, for which there are well-defined notions of ordered, critical, and chaotic dynamical regimes as well as well-studied inference procedures, we analyze the expected inference error relative to deviations in the networks dynamical regimes fromthe assumption of criticality. We demonstrate that taking criticality into account via a penalty term in the inference procedure improves the accuracy of prediction both in terms of state transitions and network wiring, particularly for small sample sizes. Copyright C 2008 Wenbin Liu et al.</div>','<b>Title: </b>Piecewise affine approximations of fluxes and enzyme kinetics from in vivo 13C labeling experiments<br><b>Year: </b>2012<br><b>Author: </b> A. Abate,  Rc. Hillen and  Sa. Wahl<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1002/rnc.2798"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>INTERNATIONAL JOURNAL OF ROBUST AND NONLINEAR CONTROL<br><b>Cluster-Keywords: </b><em>kinetic models, metabolic networks, kinetic parameters, dynamic models, systems biology, network model, dynamic metabolic, biochemical network, rate constants, enzyme kinetics, kinetic data, adjustable parameters, metabolic model, parameter estimation, dynamic optimization, model selection, mathematical model, kinetic network, model response, model<div style="font-size:11px;"><br><b>Abstract: </b><em>Owing to the ever increasing amount of available information on metabolic networks and, in particular, to the increase in information content from in vivo 13C dynamic labeling experiments, this work investigates the problem of reconstructing dynamic fluxes and enzyme kinetics. The model structure is based on the use of piecewise affine approximations. The optimization procedure at the basis of the model identification is improved by separating the parameter estimation procedure into two different phases. As a first step, a dynamic flux profile in time is reconstructed using functions that are piecewise affine in time. To achieve scalability for this step, several approaches have been developed and compared. Afterwards, the time-dependent profiles are embedded in the concentration space and the enzyme kinetic functions for single reactions are identified independently. This is an advantage compared with standard complete kinetic network approaches, which are typically characterized by hundreds of parameters, because now only a few need to be optimized simultaneously. Additionally, different kinetic formats can be rapidly compared. The overall approach is demonstrated using an informative in silico experiment. Copyright c 2012 John Wiley  Sons, Ltd.</div>','<b>Title: </b>A glucose-sensitive block glycopolymer hydrogel based on dynamic boronic ester bonds for insulin delivery<br><b>Year: </b>2017<br><b>Author: </b> Bq. Cai,  Yp. Luo,  Qq. Guo,  Xg. Zhang and  Zm. Wu<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.carres.2017.04.006"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>CARBOHYDRATE RESEARCH<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Hydrogels are good candidates to satisfy many needs for functional and tunable biomaterials. How to precisely control the gel structure and functions is crucial for the construction of sophisticated soft biomaterials comprising the hydrogels, which facilitates the impact of the surrounding environment on a unique biological function occurring. Here, glucose-responsive hydrogels comprised of 3-acrylamidophenyl boronic acid copolymerized with 2-lactobionamidoethyl methacrylate pAPBA-b-LAMA were synthesized, and further evaluated as carriers for insulin delivery. The formation of pAPBA-b-LAMA hydrogel was based on dynamic covalent bond using the association of boronic acid with diols. PAPBA-b-LAMA hydrogel with the typical porous structure showed a rapid increase in equilibrium of swelling, which was up to 1856 after incubation with aqueous solution. Using insulin as a model protein therapeutic, pAPBA-b-LAMA hydrogel exhibited high drug loading capability up to 15.6, and also displayed glucose-dependent insulin release under physiological conditions. Additionally, the viability of NIH3T3 cells was more than 90 after treated with pAPBA-b-LAMA hydrogel, indicating that the hydrogel had no cytotoxicity. Consequently, the novel pAPBA-b-LAMA hydrogel has a practical application for diabetes treatment. C 2017 Published by Elsevier Ltd.</div>','<b>Title: </b>Finding gene regulatory network candidates using the gene expression knowledge base<br><b>Year: </b>2014<br><b>Author: </b> A. Venkatesan,  S. Tripathi,  As. De Galdeano,  W. Blond,  A. Lgreid,  V. Mironov and  M. Kuiper<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/s12859-014-0386-y"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>gene expression, systems biology, metabolic network, network model, network analysis, dynamic model, signaling pathways, signaling network, regulatory network, dynamic network, biology approach, candidate genes, gene network, gene regulatory, biology analysis, mathematical model, network dynamic, biology network, metabolic pathways, pathway analysis<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Network-based approaches for the analysis of large-scale genomics data have become well established. Biological networks provide a knowledge scaffold against which the patterns and dynamics of omics data can be interpreted. The background information required for the construction of such networks is often dispersed across a multitude of knowledge bases in a variety of formats. The seamless integration of this information is one of the main challenges in bioinformatics. The Semantic Web offers powerful technologies for the assembly of integrated knowledge bases that are computationally comprehensible, thereby providing a potentially powerful resource for constructing biological networks and network-based analysis.Results: We have developed the Gene eXpression Knowledge Base GeXKB, a semantic web technology based resource that contains integrated knowledge about gene expression regulation. To affirm the utility of GeXKB we demonstrate how this resource can be exploited for the identification of candidate regulatory network proteins. We present four use cases that were designed from a biological perspective in order to find candidate members relevant for the gastrin hormone signaling network model. We show how a combination of specific query definitions and additional selection criteria derived from gene expression data and prior knowledge concerning candidate proteins can be used to retrieve a set of proteins that constitute valid candidates for regulatory network extensions.Conclusions: Semantic web technologies provide the means for processing and integrating various heterogeneous information sources. The GeXKB offers biologists such an integrated knowledge resource, allowing them to address complex biological questions pertaining to gene expression. This work illustrates how GeXKB can be used in combination with gene expression results and literature information to identify new potential candidates that may be considered for extending a gene regulatory network.</div>','<b>Title: </b>The rise of Network Ecology: Maps of the topic diversity and scientific collaboration<br><b>Year: </b>2014<br><b>Author: </b> Sr. Borrett,  J. Moody and  A. Edelmann<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.ecolmodel.2014.02.019"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ECOLOGICAL MODELLING<br><b>Cluster-Keywords: </b><em>network ecology, food webs, network models, ecological systems, food web, systems biology, ecological networks, ecosystem model, signalling networks, model results, interaction networks, dynamic model, ecological interactions, network analysis, model network, network structure, transport networks, systems network, ecosystem dynamics, restoration ecology<div style="font-size:11px;"><br><b>Abstract: </b><em>Network ecologists investigate the structure, function, and evolution of ecological systems using network models and analyses. For example, network techniques have been used to study community interactions i.e., food-webs, mutualisms, gene flow across landscapes, and the sociality of individuals in populations. The work presented here uses a bibliographic and network approach to 1 document the rise of Network Ecology, 2 identify the diversity of topics addressed in the field, and 3 map the structure of scientific collaboration among contributing scientists. Our aim is to provide a broad overview of this emergent field that highlights its diversity and to provide a foundation for future advances. To do this, we searched the ISI Web of Science database for ecology publications between 1900 and 2012 using the search terms for research areas of Environmental Sciences  Ecology and Evolutionary Biology and the topic ecology. From these records we identified the Network Ecology publications using the topic terms network, graph theory, and web, while controlling for the usage of misleading phrases. The resulting corpus entailed 29,513 publications between 1936 and 2012. We found that Network Ecology spans across more than 1500 sources with core ecological journals being among the top 20 most frequent outlets. We document the rapid rise in Network Ecology publications per year reaching a magnitude of over 5 of the ecological publications in 2012. Drawing topical information from the publication record content titles, abstracts, keywords and collaboration information from author listing, our analysis highlights the diversity and clustering of topics addressed within Network Ecology. The largest connected component of the topic network contained 73 of the corpus, and exhibited strong clustering clustering coefficient 0.93. The coauthorship network revealed that while network ecologists are generally collaborative, the field is deeply fragmented into topic and co-author cliques. The largest component of the co-author network comprised 46 of the authors and contained 149 distinct clusters. We suggest ways to build on the collaborative spirit and reduce the field fragmentation so as to improve the development and spread of ideas. We conclude that Network Ecology will likely continue to grow because the forces driving its increase are likely to persist. C 2014 Elsevier B.V. All rights reserved.</div>','<b>Title: </b>Bacterial Molecular Networks: Bridging the Gap Between Functional Genomics and Dynamical Modelling<br><b>Year: </b>2012<br><b>Author: </b> J. Van Helden,  A. Toussaint and  D. Thieffry<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/978-1-61779-361-51"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BACTERIAL MOLECULAR NETWORKS: METHODS AND PROTOCOLS<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>This introductory review synthesizes the contents of the volume Bacterial Molecular Networks of the series Methods in Molecular Biology. This volume gathers 9 reviews and 16 method chapters describing computational protocols for the analysis of metabolic pathways, protein interaction networks, and regulatory networks. Each protocol is documented by concrete case studies dedicated to model bacteria or interacting populations. Altogether, the chapters provide a representative overview of state-of-the-art methods for data integration and retrieval, network visualization, graph analysis, and dynamical modelling.</div>','<b>Title: </b>Quantitative inference of dynamic regulatory pathways via microarray data<br><b>Year: </b>2005<br><b>Author: </b> Wc. Chang,  Cw. Li and  Bs. Chen<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1471-2105-6-44"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>gene regulatory, gene expression, regulatory networks, expression data, regulatory pathway, regulatory genes, network dynamics, microarray data, genetic networks, transcription factor, models network, target gene, upstream regulatory, network topology, pathway network, gene functional, expression profile, transcriptional network, dynamic model, transcriptional regulatory<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: The cellular signaling pathway  network is one of the main topics of organismic investigations. The intracellular interactions between genes in a signaling pathway are considered as the foundation of functional genomics. Thus, what genes and how much they influence each other through transcriptional binding or physical interactions are essential problems. Under the synchronous measures of gene expression via a microarray chip, an amount of dynamic information is embedded and remains to be discovered. Using a systematically dynamic modeling approach, we explore the causal relationship among genes in cellular signaling pathways from the system biology approach.Results: In this study, a second- order dynamic model is developed to describe the regulatory mechanism of a target gene from the upstream causality point of view. From the expression profile and dynamic model of a target gene, we can estimate its upstream regulatory function. According to this upstream regulatory function, we would deduce the upstream regulatory genes with their regulatory abilities and activation delays, and then link up a regulatory pathway. Iteratively, these regulatory genes are considered as target genes to trace back their upstream regulatory genes. Then we could construct the regulatory pathway  or network to the genome wide. In short, we can infer the genetic regulatory pathways from gene-expression profiles quantitatively, which can confirm some doubted paths or seek some unknown paths in a regulatory pathway  network. Finally, the proposed approach is validated by randomly reshuffling the time order of microarray data.Conclusion: We focus our algorithm on the inference of regulatory abilities of the identified causal genes, and how much delay before they regulate the downstream genes. With this information, a regulatory pathway would be built up using microarray data. In the present study, two signaling pathways, i. e. circadian regulatory pathway in Arabidopsis thaliana and metabolic shift pathway from fermentation to respiration in yeast Saccharomyces cerevisiae, are reconstructed using microarray data to evaluate the performance of our proposed method. In the circadian regulatory pathway, we identified mainly the interactions between the biological clock and the photoperiodic genes consistent with the known regulatory mechanisms. We also discovered the now less-known regulations between crytochrome and phytochrome. In the metabolic shift pathway, the casual relationship of enzymatic genes could be detected properly.</div>','<b>Title: </b>ON MODELING OF LIVING ORGANISMS USING HIERARCHICAL COARSE-GRAINING ABSTRACTIONS OF KNOWLEDGE<br><b>Year: </b>2013<br><b>Author: </b> Y. Derbal<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1142/S0218339013500083"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF BIOLOGICAL SYSTEMS<br><b>Cluster-Keywords: </b><em>systems biology, regulatory network, interaction networks, network model, gene regulatory, network, network biology, networks, gene network, response networks, network dynamics, dynamic model, cell networks, gene expression, biological networks, system dynamics, dynamic network, model, biological systems, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>High throughput technologies such as gene expression microarray, ChIP-chips, siRNA and protein arrays and high throughput mass spectrometry are enabling an ever increasing amount of data becoming available about DNA, RNA, proteins, metabolites as well as biological pathways and networks. The knowledge embedded in this data deluge needs to be recast in forms that lend themselves to analysis with the expectation of developing analytical instruments to gain insight and answer questions about life and living organisms. The powers of abstraction and model building are fundamental to the quest of making sense of the biological complexity embedded in these biological and clinical datasets. The modeling of living organisms is explored with a proposed framework for model representation of biological complexity. The principal foundational assumption of the proposed modeling philosophy recognizes the symbiotic relationship between information and energy flows, required for the transformation of matter, as a fundamental organizing force underlying the observable nature of living organisms. The use of the concept of regularities to refer to complexity of structure, function and dynamics alike provides a unified approach to the reasoning about the integration of knowledge representations of varying natures and scales of granularities. The application of the proposed modeling approach is illustrated in broad qualitative terms for the human organism.</div>','<b>Title: </b>Compensatory effects in the PI3K/PTEN/AKT signaling network following receptor tyrosine kinase inhibition<br><b>Year: </b>2011<br><b>Author: </b> A. Goltsov,  D. Faratian,  Sp. Langdon,  J. Bown,  I. Goryanin and  Dj. Harrison<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.cellsig.2010.10.011"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>CELLULAR SIGNALLING<br><b>Cluster-Keywords: </b><em>signaling network, systems biology, signal transduction, signaling pathways, receptor tyrosine, erbb signaling, erbb receptor, receptor signaling, intracellular signaling, mathematical models, cell signaling, computational model, factor receptor, receptor activation, growth factor, signaling models, networks signaling, receptor system, receptors systems, downstream signaling<div style="font-size:11px;"><br><b>Abstract: </b><em>Overcoming de novo and acquired resistance to anticancer drugs that target signaling networks is a formidable challenge for drug design and effective cancer therapy. Understanding the mechanisms by which this resistance arises may offer a route to addressing the insensitivity of signaling networks to drug intervention and restore the efficacy of anticancer therapy. Extending our recent work identifying PTEN as a key regulator of Herceptin sensitivity, we present an integrated theoretical and experimental approach to study the compensatory mechanisms within the PI3K/PTEN/AKT signaling network that afford resistance to receptor tyrosine kinase RTK inhibition by anti-HER2 monoclonal antibodies. In a computational model representing the dynamics of the signaling network, we define a single control parameter that encapsulates the balance of activities of the enzymes involved in the PI3K/PTEN/AKT cycle. By varying this control parameter we are able to demonstrate both distinct dynamic regimes of behavior of the signaling network and the transitions between those regimes. We demonstrate resistance, sensitivity, and suppression of RTK signals by the signaling network. Through model analysis we link the sensitivity-to-resistance transition to specific compensatory mechanisms within the signaling network. We study this transition in detail theoretically by variation of activities of PTEN, PI3K, AKT enzymes, and use the results to inform experiments that perturb the signaling network using combinatorial inhibition of RTK, PTEN, and PI3K enzymes in human ovarian carcinoma cell lines. We find good alignment between theoretical predictions and experimental results. We discuss the application of the results to the challenges of hypersensitivity of the signaling network to RTK signals, suppression of drug resistance, and efficacy of drug combinations in anticancer therapy. C 2010 Elsevier Inc. All rights reserved.</div>','<b>Title: </b>Stochastic filtering of a random Fibonacci sequence: Theory and applications<br><b>Year: </b>2014<br><b>Author: </b> A. Farina,  C. Fantacci and  M. Frasca<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.sigpro.2014.03.052"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SIGNAL PROCESSING<br><b>Cluster-Keywords: </b><em>circadian rhythms, systems biology, biological clock, circadian clock, biological rhythms, hpa axis, model, clock network, mathematical models, biological systems, dynamic systems, ode models, biology models, single cell, network, model parameters, experimental data, models, systems, dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>This paper conceives a stochastic filtering problem to estimate, from noisy measurements, the numbers of the random Fibonacci sequence. The dynamical system is amenable to an exact solution, being the convolution of Bernoullian and Gaussian variables, yielding a closed form for the equations of the filter. The derived optimal filter has exponential computational complexity, thus a suboptimal filter with affordable computational load is conceived. The stochastic filter performance is then evaluated with two applications: one of theoretical value and one for a more practical application. More precisely the first case study estimates the Viswanath constant from noisy measurements of the random Fibonacci sequence. It is shown how the filter performs well in estimating the Viswanath constant even if the noise is significantly increased. The second case study refers to a model of malware propagation in a computer network. In this case, it is assumed that there is a random rate for the infection, assuming that a finite time is needed before a computer is infected. A random generalized Fibonacci sequence fits well in this case. Additional applications are possible in view of the fact that several systems both in biology and economy are well represented by Fibonacci binary random trees. C 2014 Elsevier B.V. All rights reserved.</div>','<b>Title: </b>Predicting the Lifetime of Dynamic Networks Experiencing Persistent Random Attacks<br><b>Year: </b>2015<br><b>Author: </b> B. Podobnik,  T. Lipic,  D. Horvatic,  A. Majdandzic,  Sr. Bishop and  He. Stanley<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1038/srep14286"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SCIENTIFIC REPORTS<br><b>Cluster-Keywords: </b><em>complex networks, complex systems, network theory, network structure, biological network, social network, network science, dynamic networks, network systems, network model, network topology, systems biology, network, network analysis, system dynamics, networks, realworld networks, network automata, epidemic network, model network<div style="font-size:11px;"><br><b>Abstract: </b><em>Estimating the critical points at which complex systems abruptly flip from one state to another is one of the remaining challenges in network science. Due to lack of knowledge about the underlying stochastic processes controlling critical transitions, it is widely considered difficult to determine the location of critical points for real-world networks, and it is even more difficult to predict the time at which these potentially catastrophic failures occur. We analyse a class of decaying dynamic networks experiencing persistent failures in which the magnitude of the overall failure is quantified by the probability that a potentially permanent internal failure will occur. When the fraction of active neighbours is reduced to a critical threshold, cascading failures can trigger a total network failure. For this class of network we find that the time to network failure, which is equivalent to network lifetime, is inversely dependent upon the magnitude of the failure and logarithmically dependent on the threshold. We analyse how permanent failures affect network robustness using network lifetime as a measure. These findings provide new methodological insight into system dynamics and, in particular, of the dynamic processes of networks. We illustrate the network model by selected examples from biology, and social science.</div>','<b>Title: </b>Transient responses and adaptation to steady state in a eukaryotic gene regulation system<br><b>Year: </b>2004<br><b>Author: </b> E. Braun and  N. Brenner<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1088/1478-3967/1/2/003"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PHYSICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>systems biology, regulatory network, interaction networks, network model, gene regulatory, network, network biology, networks, gene network, response networks, network dynamics, dynamic model, cell networks, gene expression, biological networks, system dynamics, dynamic network, model, biological systems, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Understanding the structure and functionality of eukaryotic gene regulation systems is of fundamental importance in many areas of biology. While most recent studies focus on static or short-term properties, measuring the long-term dynamics of these networks under controlled conditions is necessary for their complete characterization. We demonstrate adaptive dynamics in a well-known system of metabolic regulation, the GAL system in the yeast S. cerevisiae. This is a classic model for a eukaryotic genetic switch, induced by galactose and repressed by glucose. We followed the expression of a reporter gfp under a GAL promoter at single-cell resolution in large population of yeast cells. Experiments were conducted for long time scales, several generations, while controlling the environment in continuous culture. This combination enabled us, for the first time, to distinguish between transient responses and steady state. We find that both galactose induction and glucose repression are only transient responses. Over several generations, the system converges to a single robust steady state, independent of external conditions. Thus, at steady state the GAL network loses its hallmark functionality as a sensitive carbon source rheostat. This result suggests that, while short-term dynamics are determined by specific modular responses, over long time scales inter-modular interactions take over and shape a robust steady state response of the regulatory system.</div>','<b>Title: </b>An extended gene protein/products boolean network model including post-transcriptional regulation<br><b>Year: </b>2014<br><b>Author: </b> A. Benso,  S. Di Carlo,  G. Politano,  A. Savino and  A. Vasciaveo<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1742-4682-11-S1-S5"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>THEORETICAL BIOLOGY AND MEDICAL MODELLING<br><b>Cluster-Keywords: </b><em>boolean networks, regulatory networks, network model, bayesian networks, model networks, systems biology, random boolean, real cells, immune system, genetic regulatory, networks, biochemical network, fluorescence images, cellular network, gene regulatory, cell differentiation, network, biological systems, ensemble approach, cell dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Networks Biology allows the study of complex interactions between biological systems using formal, well structured, and computationally friendly models. Several different network models can be created, depending on the type of interactions that need to be investigated. Gene Regulatory Networks GRN are an effective model commonly used to study the complex regulatory mechanisms of a cell. Unfortunately, given their intrinsic complexity and non discrete nature, the computational study of realistic-sized complex GRNs requires some abstractions. Boolean Networks BNs, for example, are a reliable model that can be used to represent networks where the possible state of a node is a boolean value 0 or 1. Despite this strong simplification, BNs have been used to study both structural and dynamic properties of real as well as randomly generated GRNs.Results: In this paper we show how it is possible to include the post-transcriptional regulation mechanism a key process mediated by small non-coding RNA molecules like the miRNAs into the BN model of a GRN. The enhanced BN model is implemented in a software toolkit EBNT that allows to analyze boolean GRNs from both a structural and a dynamic point of view. The open-source toolkit is compatible with available visualization tools like Cytoscape and allows to run detailed analysis of the network topology as well as of its attractors, trajectories, and state-space. In the paper, a small GRN built around the mTOR gene is used to demonstrate the main capabilities of the toolkit.Conclusions: The extended model proposed in this paper opens new opportunities in the study of gene regulation. Several of the successful researches done with the support of BN to understand high-level characteristics of regulatory networks, can now be improved to better understand the role of post-transcriptional regulation for example as a network-wide noise-reduction or stabilization mechanisms.</div>','<b>Title: </b>Reducing complexity: An iterative strategy for parameter determination in biological networks<br><b>Year: </b>2015<br><b>Author: </b> Sc. Binder,  Ea. Hernandez-Vargas and  M. Meyer-Hermann<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.cpc.2015.01.001"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>COMPUTER PHYSICS COMMUNICATIONS<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>The dynamics of biological networks are fundamental to a variety of processes in many areas of biology and medicine. Understanding of such networks on a systemic level is facilitated by mathematical models describing these networks. However, since mathematical models of signalling networks commonly aim to describe several highly connected biological quantities and many model parameters cannot be measured directly, quantitative dynamic models often present challenges with respect to model calibration. Here, we propose an iterative fitting routine to decompose the problem of fitting a system of coupled ordinary differential equations describing a signalling network into smaller subproblems. Parameters for each differential equation are estimated separately using a Differential Evolution algorithm while all other dynamic quantities in the model are treated as input to the system. The performance of this algorithm is evaluated on artificial networks with known structure and known model parameters and compared to a conventional optimisation procedure for the same problem. Our analysis indicates that the procedure results in a significantly higher quality of fit and more efficient reconstruction of the true parameters than the conventional algorithm. C 2015 The Authors. Published by Elsevier B.V.</div>','<b>Title: </b>Signal processing at the Ras circuit: what shapes Ras activation patterns<br><b>Year: </b>2004<br><b>Author: </b>AnonymousLink: No link<br><b>Journal: </b>SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>blood flow, blood cells, flow rate, blood pumps, cells, cell, results, patients, stem cells, methods, blood, patients blood, flow, patients methods, patients objectives, cell viability, objectives, results cells, blood pressure, cell culture<div style="font-size:11px;"><br><b>Abstract: </b><em>A systems biology approach is applied to gain it quantitative understanding ofthe integration of signalling by the small GTPase Ras. The Ras protein acts its a critical switch in response to signals that determine the cells fate. Inunstimulated cells, Ras switching between till inactive GDP-binding and active GTP-binding state is controlled by the intrinsic catalytic activities of Ras. Tile calculated high sensitivity of the basal Ras-GTP fraction to changes inthe rate constant of GTP-hydrolysis by Ras can account for the carcinogenic potential of Ras Mutants with decreased GTPase activities. Extracelluar stimuliinitiate Ras interactions with GDP/GTP exchange factors such as SOS, and GTP-hydrolysis activating proteins such as RasGAP. Our data oil freshly isolated hepatocytes stimulated with epidermal growth factor EGF show transient SOS activation and sustained Ras-GTP patterns, We demonstrate that these dose-response data Call Only be explained by transient RasGAP activitation, and not by merely switching off the SOS signal. e.g. by inhibitory phosphorylation of SOS. A transient RasGAP activity can be brought about by it number of mechanisms. Acomprehensive kinetic model of the EGF receptor EGFR network was developed to explore feasible molecular scenarios, including the receptor-mediated recruitment of SOS and RasGAP to the plasma membrane, phosphorylation of RasGAP andp190 RhoGAP by soluble tyrosine kinases. and RasGAP interactions with phosphoinositides and p190 RhoGAP. We show that a transient RasGAP association with EGFR followed by the capture of RasGAP through file formation of complexes withp 190 RhoGAP can account for data oil hepatocytes. In summary, our results demonstrate that a combination of experimental monitoring and integrated dynamicanalysis is capable of dissecting regulatory mechanisms that govern cellular signal transduction.</div>','<b>Title: </b>A guide to sensitivity analysis of quantitative models of gene expression dynamics<br><b>Year: </b>2013<br><b>Author: </b> B. Taylor,  Tj. Lee and  Js. Weitz<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.ymeth.2013.03.007"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>METHODS<br><b>Cluster-Keywords: </b><em>systems biology, dynamic network, early differentiation, complex diseases, stem cells, erythroid cells, system dynamics, network biomarkers, immune system, tcm syndrome, dnb genes, gene expression, dnb model, stem cell, expression dynamics, model system, mathematical model, model dynamics, dynamic model, sensitivity analysis<div style="font-size:11px;"><br><b>Abstract: </b><em>We provide a guide to performing a sensitivity analysis SA of quantitative models of gene expression dynamics appropriate to the levels of uncertainty in the model: spanning cases where parameters are relatively well-constrained to cases where they are poorly constrained. In the well-constrained case, we present methods to perform "local" SA LSA, which considers small perturbations for a single set of model parameter values. In the poorly-constrained case, we present methods to perform "global" SA GSA as a means to evaluate the sensitivity of a model over large regions of parameter space. We apply these methods to quantitative models of increasing complexity. The models we consider are simple logistic growth, negative feedback in a mRNA-protein model, and two models of decision making within bacteriophage lambda. We discuss the best practices for how SA can be utilized in an iterative fashion to advance biological understanding. C 2013 Elsevier Inc. All rights reserved.</div>','<b>Title: </b>A Quantitative Approach to Analyzing Genome Reductive Evolution Using Protein-Protein Interaction Networks: A Case Study of Mycobacterium leprae<br><b>Year: </b>2016<br><b>Author: </b> Ro. Akinola,  Gk. Mazandu and  Nj. Mulder<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.5589/fgene.2016.00039"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>FRONTIERS IN GENETICS<br><b>Cluster-Keywords: </b><em>interaction networks, proteinprotein interactions, protein interaction, network model, complex systems, systems biology, protein protein, protein complex, host network, dynamic interactions, network analysis, biological systems, networks network, cellular network, biological network, viral proteins, interactome networks, regulatory networks, immune system, systems network<div style="font-size:11px;"><br><b>Abstract: </b><em>The advance in high-throughput sequencing technologies has yielded complete genome sequences of several organisms, including complete bacterial genomes. The growing number of these available sequenced genomes has enabled analyses of their dynamics, as well as the molecular and evolutionary processes which these organisms are under. Comparative genomics of different bacterial genomes have highlighted their genome size and gene content in association with lifestyles and adaptation to various environments and have contributed to enhancing our understanding of the mechanisms of their evolution. Protein-protein functional interactions mediate many essential processes for maintaining the stability of the biological systems under changing environmental conditions. Thus, these interactions play crucial roles in the evolutionary processes of different organisms, especially for obligate intracellular bacteria, proven to generally have reduced genome sizes compared to their nearest free-living relatives. In this study, we used the approach based on the Renormalization Group RG analysis technique and the Maximum-Excluded-Mass-Burning MEMB model to investigate the evolutionary process of genome reduction in relation to the organization of functional networks of two organisms. Using a Mycobacteriurn leprae MLP network in comparison with a Mycobacteriurn tuberculosis MTB network as a case study, we show that reductive evolution in MLP was as a result of removal of important proteins from neighbors of corresponding orthologous MTB proteins. While each orthologous MTB protein had an increase in number of interacting partners in most instances, the corresponding MLP protein had lost some of them. This work provides a quantitative model for mapping reductive evolution and protein-protein functional interaction network organization in terms of roles played by different proteins in the network structure.</div>','<b>Title: </b>GPU Code Generation for ODE-Based Applications with Phased Shared-Data Access Patterns<br><b>Year: </b>2013<br><b>Author: </b> A. Hagiescu,  B. Liu,  R. Ramanathan,  Sk. Palaniappan,  Z. Cui,  B. Chattopadhyay,  Ps. Thiagarajan and  Wf. Wong<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1145/2555289.2555311"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ACM TRANSACTIONS ON ARCHITECTURE AND CODE OPTIMIZATION<br><b>Cluster-Keywords: </b><em>circadian rhythms, systems biology, biological clock, circadian clock, biological rhythms, hpa axis, model, clock network, mathematical models, biological systems, dynamic systems, ode models, biology models, single cell, network, model parameters, experimental data, models, systems, dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>We present a novel code generation scheme for GPUs. Its key feature is the platform-aware generation of a heterogeneous pool of threads. This exposes more data-sharing opportunities among the concurrent threads and reduces the memory requirements that would otherwise exceed the capacity of the on-chip memory. Instead of the conventional strategy of focusing on exposing as much parallelism as possible, our scheme leverages on the phased nature of memory access patterns found in many applications that exhibit massive parallelism. We demonstrate the effectiveness of our code generation strategy on a computational systems biology application. This application consists of computing a Dynamic Bayesian Network DBN approximation of the dynamics of signalling pathways described as a system of Ordinary Differential Equations ODEs. The approximation algorithm involves i sampling many of the order of a few million times from the set of initial states, ii generating trajectories through numerical integration, and iii storing the statistical properties of this set of trajectories in Conditional Probability Tables CPTs of a DBN via a prespecified discretization of the time and value domains. The trajectories can be computed in parallel. However, the intermediate data needed for computing them, as well as the entries for the CPTs, are too large to be stored locally. Our experiments show that the proposed code generation scheme scales well, achieving significant performance improvements on three realistic signalling pathways models. These results suggest how our scheme could be extended to deal with other applications involving systems of ODEs.</div>','<b>Title: </b>Calcium Ion Fluctuations Alter Channel Gating in a Stochastic Luminal Calcium Release Site Model<br><b>Year: </b>2017<br><b>Author: </b> H. Ji,  Yh. Li and  Sh. Weinberg<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1109/TCBB.2015.2498552"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IEEE-ACM TRANSACTIONS ON COMPUTATIONAL BIOLOGY AND BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>reaction networks, network model, systems biology, biochemical reaction, stochastic reaction, model reduction, stochastic model, network dynamics, stochastic biochemical, stochastic simulation, system size, chemical reaction, gene network, elementary reactions, biochemical networks, parameter identification, parameter inference, cell populations, small system, linear noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Stochasticity and small system size effects in complex biochemical reaction networks can greatly alter transient and steady-state system properties. A common approach to modeling reaction networks, which accounts for system size, is the chemical master equation that governs the dynamics of the joint probability distribution for molecular copy number. However, calculation of the stationary distribution is often prohibitive, due to the large state-space associated with most biochemical reaction networks. Here, we analyze a network representing a luminal calcium release site model and investigate to what extent small system size effects and calcium fluctuations, driven by ion channel gating,influx and diffusion, alter steady-state ion channel properties including open probability. For a physiological ion channel gating model and number of channels, the state-space may be between approximately 106 - 108 elements, and a novel modified block power method is used to solve the associated dominant eigenvector problem required to calculate the stationary distribution. We demonstrate that both small local cytosolic domain volume and a small number of ion channels drive calcium fluctuations that result in deviation from the corresponding model that neglects small system size effects.</div>','<b>Title: </b>Faithful modeling of transient expression and its application to elucidating negative feedback regulation<br><b>Year: </b>2007<br><b>Author: </b> A. Rubinstein,  V. Gurevich,  Z. Kasulin-Boneh,  L. Pnueli,  Y. Kassir and  Ry. Pinter<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1073/pnas.0611168104"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Modeling and analysis of genetic regulatory networks is essential both for better understanding their dynamic behavior and for elucidating and refining open issues. We hereby present a discrete computational model that effectively describes the transient and sequential expression of a network of genes in a representative developmental pathway. Our model system is a transcriptional cascade that includes positive and negative feedback loops directing the initiation and progression through meiosis in budding yeast. The computational model allows qualitative analysis of the transcription of early meiosis-specific genes, specifically, Ime2 and their master activator, Ime1. The simulations demonstrate a robust transcriptional behavior with respect to the initial levels of Ime1 and Ime2. The computational results were verified experimentally by deleting various genes and by changing initial conditions. The model has a strong predictive aspect, and it provides insights into how to distinguish among and reason about alternative hypotheses concerning the mode by which negative regulation through Ime1 and Ime2 is accomplished. Some predictions were validated experimentally, for instance, showing that the decline in the transcription of IME1 depends on Rpd3, which,is recruited by Ime1 to its promoter. Finally, this general model promotes the analysis of systems that are devoid of consistent quantitative data, as is often the case, and it can be easily adapted to other developmental pathways.</div>','<b>Title: </b>A probabilistic approach to identify putative drug targets in biochemical networks<br><b>Year: </b>2011<br><b>Author: </b> E. Murabito,  K. Smallbone,  J. Swinton,  Hv. Westerhoff and  R. Steuer<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1098/rsif.2010.0540"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF THE ROYAL SOCIETY INTERFACE<br><b>Cluster-Keywords: </b><em>kinetic models, metabolic networks, kinetic parameters, dynamic models, systems biology, network model, dynamic metabolic, biochemical network, rate constants, enzyme kinetics, kinetic data, adjustable parameters, metabolic model, parameter estimation, dynamic optimization, model selection, mathematical model, kinetic network, model response, model<div style="font-size:11px;"><br><b>Abstract: </b><em>Network-based drug design holds great promise in clinical research as a way to overcome the limitations of traditional approaches in the development of drugs with high efficacy and low toxicity. This novel strategy aims to study how a biochemical network as a whole, rather than its individual components, responds to specific perturbations in different physiological conditions. Proteins exerting little control over normal cells and larger control over altered cells may be considered as good candidates for drug targets. The application of network-based drug design would greatly benefit from using an explicit computational model describing the dynamics of the system under investigation. However, creating a fully characterized kinetic model is not an easy task, even for relatively small networks, as it is still significantly hampered by the lack of data about kinetic mechanisms and parameters values. Here, we propose a Monte Carlo approach to identify the differences between flux control profiles of a metabolic network in different physiological states, when information about the kinetics of the system is partially or totally missing. Based on experimentally accessible information on metabolic phenotypes, we develop a novel method to determine probabilistic differences in the flux control coefficients between the two observable phenotypes. Knowledge of how differences in flux control are distributed among the different enzymatic steps is exploited to identify points of fragility in one of the phenotypes. Using a prototypical cancerous phenotype as an example, we demonstrate how our approach can assist researchers in developing compounds with high efficacy and low toxicity.</div>','<b>Title: </b>Chemical reaction systems with a homoclinic bifurcation: an inverse problem<br><b>Year: </b>2016<br><b>Author: </b> T. Plesa,  T. Vejchodsky and  R. Erban<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s10910-016-0656-1"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF MATHEMATICAL CHEMISTRY<br><b>Cluster-Keywords: </b><em>reaction networks, reaction systems, metabolic networks, chemical reaction, network structure, systems biology, biological systems, biological networks, biochemical reaction, reaction rates, differential equations, reaction graph, complex networks, ssi modules, networks, life systems, ordinary differential, cellular networks, reaction, network flows<div style="font-size:11px;"><br><b>Abstract: </b><em>An inverse problem framework for constructing reaction systems with prescribed properties is presented. Kinetic transformations are defined and analysed as a part of the framework, allowing an arbitrary polynomial ordinary differential equation to be mapped to the one that can be represented as a reaction network. The framework is used for construction of specific two- and three-dimensional bistable reaction systems undergoing a supercritical homoclinic bifurcation, and the topology of their phase spaces is discussed.</div>','<b>Title: </b>A signaling visualization toolkit to support rational design of combination therapies and biomarker discovery: SiViT<br><b>Year: </b>2017<br><b>Author: </b> Jl. Bown,  M. Shovman,  P. Robertson,  A. Boiko,  A. Goltsov,  P. Mullen and  Dj. Harrison<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.18632/oncotarget.8747"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ONCOTARGET<br><b>Cluster-Keywords: </b><em>boolean network, network models, qualitative models, systems biology, mathematical modeling, boolean models, signaling networks, dynamic models, dynamic modeling, signaling dynamics, logic models, dynamic network, mathematical models, boolean modeling, signal transduction, regulatory networks, drug resistance, biological networks, network approach, network modeling<div style="font-size:11px;"><br><b>Abstract: </b><em>Targeted cancer therapy aims to disrupt aberrant cellular signalling pathways. Biomarkers are surrogates of pathway state, but there is limited success in translating candidate biomarkers to clinical practice due to the intrinsic complexity of pathway networks. Systems biology approaches afford better understanding of complex, dynamical interactions in signalling pathways targeted by anticancer drugs. However, adoption of dynamical modelling by clinicians and biologists is impeded by model inaccessibility. Drawing on computer games technology, we present a novel visualization toolkit, SiViT, that converts systems biology models of cancer cell signalling into interactive simulations that can be used without specialist computational expertise. SiViT allows clinicians and biologists to directly introduce for example loss of function mutations and specific inhibitors. SiViT animates the effects of these introductions on pathway dynamics, suggesting further experiments and assessing candidate biomarker effectiveness. In a systems biology model of Her2 signalling we experimentally validated predictions using SiViT, revealing the dynamics of biomarkers of drug resistance and highlighting the role of pathway crosstalk. No model is ever complete: the iteration of real data and simulation facilitates continued evolution of more accurate, useful models. SiViT will make accessible libraries of models to support preclinical research, combinatorial strategy design and biomarker discovery.</div>','<b>Title: </b>Fluctuation-induced distributed resonances in oscillatory networks<br><b>Year: </b>2019<br><b>Author: </b> Xz. Zhang,  S. Hallerberg,  M. Matthiae,  D. Witthaut and  M. Timme<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1126/sciadv.aav1027"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SCIENCE ADVANCES<br><b>Cluster-Keywords: </b><em>coupling strength, synchronization transitions, timeperiodic coupling, time delay, logic network, distribution network, network model, oscillatory networks, neuron networks, water distribution, stochastic resonance, gene networks, multiple synchronization, firing synchronization, neuronal networks, firing behaviors, logical networks, results show, regulatory networks, cell polarization<div style="font-size:11px;"><br><b>Abstract: </b><em>Across physics, biology, and engineering, the collective dynamics of oscillatory networks often evolve into self-organized operating states. How such networks respond to external fluctuating signals fundamentally underlies their function, yet is not well understood. Here, we present a theory of dynamic network response patterns and reveal how distributed resonance patterns emerge in oscillatory networks once the dynamics of the oscillatory units become more than one-dimensional. The network resonances are topology specific and emerge at an intermediate frequency content of the input signals, between global yet homogeneous responses at low frequencies and localized responses at high frequencies. Our analysis reveals why these patterns arise and where in the network they are most prominent. These results may thus provide general theoretical insights into how fluctuating signals induce response patterns in networked systems and simultaneously help to develop practical guiding principles for real-world network design and control.</div>','<b>Title: </b>The Transcriptional Regulatory Network of Mycobacterium tuberculosis<br><b>Year: </b>2011<br><b>Author: </b> J. Sanz,  J. Navarro,  A. Arbues,  C. Martin,  Pc. Marijuan and  Y. Moreno<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0022178"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>network motifs, regulatory network, gene regulatory, systems biology, gene network, bifan motif, complex systems, model comparison, network modeling, complex gene, biological systems, dynamic systems, interaction network, feedforward loop, modeling gene, intelligent systems, modeling approach, biological pathways, motif model, model motifs<div style="font-size:11px;"><br><b>Abstract: </b><em>Under the perspectives of network science and systems biology, the characterization of transcriptional regulatory TR networks beyond the context of model organisms offers a versatile tool whose potential remains yet mainly unexplored. In this work, we present an updated version of the TR network of Mycobacterium tuberculosis M.tb, which incorporates newly characterized transcriptional regulations coming from 31 recent, different experimental works available in the literature. As a result of the incorporation of these data, the new network doubles the size of previous data collections, incorporating more than a third of the entire genome of the bacterium. We also present an exhaustive topological analysis of the new assembled network, focusing on the statistical characterization of motifs significances and the comparison with other model organisms. The expanded M. tb transcriptional regulatory network, considering its volume and completeness, constitutes an important resource for diverse tasks such as dynamic modeling of gene expression and signaling processes, computational reliability determination or protein function prediction, being the latter of particular relevance, given that the function of only a small percent of the proteins of M. tb is known.</div>','<b>Title: </b>The dynamics of developmental system drift in the gene network underlying wing polyphenism in ants: a mathematical model<br><b>Year: </b>2008<br><b>Author: </b> M. Nahmad,  L. Glass and  E. Abouheif<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1111/j.1525-142X.2008.00244.x"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>EVOLUTION  DEVELOPMENT<br><b>Cluster-Keywords: </b><em>network models, gene network, regulatory networks, boolean networks, gene expression, systems biology, neural networks, network dynamics, gene regulatory, biological networks, network structure, gap gene, gene regulation, cell cycle, network, gene interactions, signaling network, quantum systems, system dynamics, networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Understanding the complex interaction between genotype and phenotype is a major challenge of Evolutionary Developmental Biology. One important facet of this complex interaction has been called "Developmental System Drift" DSD. DSD occurs when a similar phenotype, which is homologous across a group of related species, is produced by different genes or gene expression patterns in each of these related species. We constructed a mathematical model to explore the developmental and evolutionary dynamics of DSD in the gene network underlying wing polyphenism in ants. Wing polyphenism in ants is the ability of an embryo to develop into a winged queen or a wingless worker in response to an environmental cue. Although wing polyphenism is homologous across all ants, the gene network that underlies wing polyphenism has evolved. In winged ant castes, our simulations reproduced the conserved gene expression patterns observed in the network that controls wing development in holometabolous insects. In wingless ant castes, we simulated the suppression of wings by interrupting up- or downregulating the expression of genes in the network. Our simulations uncovered the existence of four groups of genes that have similar effects on target gene expression and growth. Although each group is comprised of genes occupying different positions in the network, their interruption produces vestigial discs that are similar in size and shape. The implications of our results for understanding the origin, evolution, and dissociation of the gene network underlying wing polyphenism in ants are discussed.</div>','<b>Title: </b>An investigation of spatial signal transduction in cellular networks<br><b>Year: </b>2012<br><b>Author: </b> A. Alam-Nazki and  J. Krishnan<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-6-83"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>blood flow, blood cells, flow rate, blood pumps, cells, cell, results, patients, stem cells, methods, blood, patients blood, flow, patients methods, patients objectives, cell viability, objectives, results cells, blood pressure, cell culture<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Spatial signal transduction plays a vital role in many intracellular processes such as eukaryotic chemotaxis, polarity generation and cell division. Furthermore it is being increasingly realized that the spatial dimension to signalling may play an important role in other apparently purely temporal signal transduction processes. It is increasingly being recognized that a conceptual basis for studying spatial signal transduction in signalling networks is necessary.Results: In this work we examine spatial signal transduction in a series of standard motifs/networks. These networks include coherent and incoherent feedforward, positive and negative feedback, cyclic motifs, monostable switches, bistable switches and negative feedback oscillators. In all these cases, the driving signal has spatial variation. For each network we consider two cases, one where all elements are essentially non-diffusible, and the other where one of the network elements may be highly diffusible. A careful analysis of steady state signal transduction provides many insights into the behaviour of all these modules. While in the non-diffusible case for the most part, spatial signalling reflects the temporal signalling behaviour, in the diffusible cases, we see significant differences between spatial and temporal signalling characteristics. Our results demonstrate that the presence of diffusible elements in the networks provides important constraints and capabilities for signalling.Conclusions: Our results provide a systematic basis for understanding spatial signalling in networks and the role of diffusible elements therein. This provides many insights into the signal transduction capabilities and constraints in such networks and suggests ways in which cellular signalling and information processing is organized to conform to or bypass those constraints. It also provides a framework for starting to understand the organization and regulation of spatial signal transduction in individual processes.</div>','<b>Title: </b>CADLIVE toolbox for MATLAB: automatic dynamic modeling of biochemical networks with comprehensive system analysis<br><b>Year: </b>2014<br><b>Author: </b> K. Inoue,  K. Maeda,  T. Miyabe,  Y. Matsuoka and  H. Kurata<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s00449-014-1167-8"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOPROCESS AND BIOSYSTEMS ENGINEERING<br><b>Cluster-Keywords: </b><em>kinetic models, metabolic networks, kinetic parameters, dynamic models, systems biology, network model, dynamic metabolic, biochemical network, rate constants, enzyme kinetics, kinetic data, adjustable parameters, metabolic model, parameter estimation, dynamic optimization, model selection, mathematical model, kinetic network, model response, model<div style="font-size:11px;"><br><b>Abstract: </b><em>Mathematical modeling has become a standard technique to understand the dynamics of complex biochemical systems. To promote the modeling, we had developed the CADLIVE dynamic simulator that automatically converted a biochemical map into its associated mathematical model, simulated its dynamic behaviors and analyzed its robustness. To enhance the feasibility by CADLIVE and extend its functions, we propose the CADLIVE toolbox available for MATLAB, which implements not only the existing functions of the CADLIVE dynamic simulator, but also the latest tools including global parameter search methods with robustness analysis. The seamless, bottom-up processes consisting of biochemical network construction, automatic construction of its dynamic model, simulation, optimization, and S-system analysis greatly facilitate dynamic modeling, contributing to the research of systems biology and synthetic biology. This application can be freely downloaded from together with an instruction.</div>','<b>Title: </b>Comparative study of circadian oscillatory network models of Drosophila<br><b>Year: </b>2008<br><b>Author: </b> Y. Ogawa,  K. Arakawa,  K. Kaizu,  F. Miyoshi,  Y. Nakayama and  M. Tomita<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1162/artl.2008.14.1.29"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ARTIFICIAL LIFE<br><b>Cluster-Keywords: </b><em>circadian rhythms, systems biology, biological clock, circadian clock, biological rhythms, hpa axis, model, clock network, mathematical models, biological systems, dynamic systems, ode models, biology models, single cell, network, model parameters, experimental data, models, systems, dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>The circadian clock of Drosophila is a model pathway for research in biological clock mechanisms, both with traditional experimental approaches and with emerging systems biology approaches utilizing mathematical modeling and in silico computer simulation. Dynamic diurnal oscillations are achieved by the complex interaction of components as a system, and mathematical reconstruction has proven to be an invaluable means of understanding such systematic behavior. In this study, we implemented eight published models of the Drosophila circadian clock in Systems Biology Markup Language SBML for comparative systems biology studies using E-Cell Simulation Environment version 3, to examine the system-level requirements for the clock mechanism to be robust, by calculating the period and amplitude sensitivity coefficients with simulation experiments. While all models were generally robust as determined by the network topology of the oscillatory feedback loop structure, existing models place relatively strong emphasis on transcription regulation, although this is a Imitation on robustness. We suggest that more comprehensive modeling including protein phosphorylation, polymerization, and nuclear transport with regard to amplitude sensitivity will be necessary for understanding the fight entrainment and temperature compensation of circadian clocks.</div>','<b>Title: </b>Model aggregation: a building-block approach to creating large macromolecular regulatory networks<br><b>Year: </b>2009<br><b>Author: </b> R. Randhawa,  Ca. Shaffer and  Jj. Tyson<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1093/bioinformatics/btp581"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Motivation: Models of regulatory networks become more difficult to construct and understand as they grow in size and complexity. Modelers naturally build large models from smaller components that each represent subsets of reactions within the larger network. To assist modelers in this process, we present model aggregation, which defines models in terms of components that are designed for the purpose of being combined.Results: We have implemented a model editor that incorporates model aggregation, and we suggest supporting extensions to the Systems Biology Markup Language SBML Level 3. We illustrate aggregation with a model of the eukaryotic cell cycle engine created from smaller pieces.</div>','<b>Title: </b>The Control of the Controller: Molecular Mechanisms for Robust Perfect Adaptation and Temperature Compensation<br><b>Year: </b>2009<br><b>Author: </b> Xy. Ni,  T. Drengstig and  P. Ruoff<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.bpj.2009.06.030"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOPHYSICAL JOURNAL<br><b>Cluster-Keywords: </b><em>systems biology, biological systems, system dynamics, molecular systems, experimental design, network model, model system, model parameters, experiment design, kinetic parameters, perfect adaptation, experimental data, molecular mechanisms, homeostatic mechanisms, switching dynamics, biological mechanisms, control systems, chemotaxis network, decentralized control, feedback control<div style="font-size:11px;"><br><b>Abstract: </b><em>Organisms have the property to adapt to a changing environment and keep certain components within a cell regulated at the same level homeostasis. "Perfect adaptation" describes an organisms response to an external stepwise perturbation by regulating some of its variables/components precisely to their original preperturbation values. Numerous examples of perfect adaptation/homeostasis have been found, as for example, in bacterial chemotaxis, photoreceptor responses, MAP kinase activities, or in metal-ion homeostasis. Two concepts have evolved to explain how perfect adaptation may be understood: In one approach robust perfect adaptation, the adaptation is a network property, which is mostly, but not entirely, independent of rate constant values; in the other approach nonrobust perfect adaptation, a fine-tuning of rate constant values is needed. Here we identify two classes of robust molecular homeostatic mechanisms, which compensate for environmental variations in a controlled variables inflow or outflow fluxes, and allow for the presence of robust temperature compensation. These two classes of homeostatic mechanisms arise due to the fact that concentrations must have positive values. We show that the concept of integral control or integral feedback, which leads to robust homeostasis, is associated with a control species that has to work under zero-order flux conditions and does not necessarily require the presence of a physico-chemical feedback structure. There are interesting links between the two identified classes of homeostatic mechanisms and molecular mechanisms found in mammalian iron and calcium homeostasis, indicating that homeostatic mechanisms may underlie similar molecular control structures.</div>','<b>Title: </b>Predictive model identifies strategies to enhance TSP1-mediated apoptosis signaling<br><b>Year: </b>2017<br><b>Author: </b> Qh. Wu and  Sd. Finley<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/s12964-017-0207-9"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>CELL COMMUNICATION AND SIGNALING<br><b>Cluster-Keywords: </b><em>stem cell, stem cells, systems biology, mathematical models, cell biology, network model, signaling pathways, cell fate, computational models, cancer cells, gene expression, network dynamics, embryonic stem, signaling network, prostate cancer, mathematical modeling, psychological stress, apoptosis signaling, computational systems, molecular mechanisms<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Thrombospondin-1 TSP1 is a matricellular protein that functions to inhibit angiogenesis. An important pathway that contributes to this inhibitory effect is triggered by TSP1 binding to the CD36 receptor, inducing endothelial cell apoptosis. However, therapies that mimic this function have not demonstrated clear clinical efficacy. This study explores strategies to enhance TSP1-induced apoptosis in endothelial cells. In particular, we focus on establishing a computational model to describe the signaling pathway, and using this model to investigate the effects of several approaches to perturb the TSP1-CD36 signaling network.Methods: We constructed a molecularly-detailed mathematical model of TSP1-mediated intracellular signaling via the CD36 receptor based on literature evidence. We employed systems biology tools to train and validate the model and further expanded the model by accounting for the heterogeneity within the cell population. The initial concentrations of signaling species or kinetic rates were altered to simulate the effects of perturbations to the signaling network.Results: Model simulations predict the population-based response to strategies to enhance TSP1-mediated apoptosis, such as downregulating the apoptosis inhibitor XIAP and inhibiting phosphatase activity. The model also postulates a new mechanism of low dosage doxorubicin treatment in combination with TSP1 stimulation. Using computational analysis, we predict which cells will undergo apoptosis, based on the initial intracellular concentrations of particular signaling species.Conclusions: This new mathematical model recapitulates the intracellular dynamics of the TSP1-induced apoptosis signaling pathway. Overall, the modeling framework predicts molecular strategies that increase TSP1-mediated apoptosis, which is useful in many disease settings.</div>','<b>Title: </b>Expectation propagation for large scale Bayesian inference of non-linear molecular networks from perturbation data<br><b>Year: </b>2017<br><b>Author: </b> Z. Narimani,  H. Beigy,  A. Ahmad,  A. Masoudi-Nejad and  H. Frohlich<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0171240"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>regulatory networks, gene regulatory, gene expression, network inference, expression data, network structure, gene networks, networks grns, regulatory genes, boolean networks, bayesian network, network reconstruction, networks gene, network dynamics, gene regulation, inference method, systems biology, time series, infer gene, network<div style="font-size:11px;"><br><b>Abstract: </b><em>Inferring the structure of molecular networks from time series protein or gene expression data provides valuable information about the complex biological processes of the cell. Causal network structure inference has been approached using different methods in the past. Most causal network inference techniques, such as Dynamic Bayesian Networks and ordinary differential equations, are limited by their computational complexity and thus make large scale inference infeasible. This is specifically true if a Bayesian framework is applied in order to deal with the unavoidable uncertainty about the correct model. We devise a novel Bayesian network reverse engineering approach using ordinary differential equations with the ability to include non-linearity. Besides modeling arbitrary, possibly combinatorial and time dependent perturbations with unknown targets, one of our main contributions is the use of Expectation Propagation, an algorithm for approximate Bayesian inference over large scale network structures in short computation time. We further explore the possibility of integrating prior knowledge into network inference. We evaluate the proposed model on DREAM4 and DREAM8 data and find it competitive against several state-of-the-art existing network inference methods.</div>','<b>Title: </b>Toward robust network based complex systems: from evolutionary cellular automata to biological models<br><b>Year: </b>2011<br><b>Author: </b> C. Darabos,  M. Tomassini,  F. Di Cunto,  P. Provero,  Jh. Moore and  M. Giacobini<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3233/IA-2011-0003"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>INTELLIGENZA ARTIFICIALE<br><b>Cluster-Keywords: </b><em>complex networks, complex systems, network theory, network structure, biological network, social network, network science, dynamic networks, network systems, network model, network topology, systems biology, network, network analysis, system dynamics, networks, realworld networks, network automata, epidemic network, model network<div style="font-size:11px;"><br><b>Abstract: </b><em>In this work we demonstrate in two different contexts how we can introduce recent discoveries and technological advances into existing computational models. In the first case, we worked on improving the performance of a simple paradigm for distributed computation: cellular automata. This was achieved by applying principles inspired by Darwinian evolution to alter the connections between the cells of the system, hence changing its topological structure. We have studied the performance of these evolved structures on prototypical problems, and analyzed their response to probabilistic transient faults and permanent failures. In the second case, we consider the context of biological genetic regulatory networks and in particular a model thereof proposed by Kaufmann in the late 60s: random Boolean networks. Since the model was developed, biology has made tremendous progress and these new discoveries can be used to improve the original model. From the structure of the network, to timing of the event taking place on it, to the specifics of the genes activation, we have added a great deal of modern knowledge into the original model, studying, analyzing, and validating it on biological case studies.</div>','<b>Title: </b>Temporal self-organization of the cyclin/Cdk network driving the mammalian cell cycle<br><b>Year: </b>2009<br><b>Author: </b> C. Gerard and  A. Goldbeter<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1073/pnas.0903827106"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA<br><b>Cluster-Keywords: </b><em>cell cycle, mammalian cell, cdk network, cell proliferation, cell, cells cell, regulatory network, detailed model, cell size, molecular network, skeleton model, cycle arrest, cycle progression, cell division, cell density, kinases cdks, cycle, phase transition, cycle control, network driving<div style="font-size:11px;"><br><b>Abstract: </b><em>We propose an integrated computational model for the network of cyclin-dependent kinases Cdks that controls the dynamics of the mammalian cell cycle. The model contains four Cdk modules regulated by reversible phosphorylation, Cdk inhibitors, and protein synthesis or degradation. Growth factors GFs trigger the transition from a quiescent, stable steady state to self-sustained oscillations in the Cdk network. These oscillations correspond to the repetitive, transient activation of cyclin D/Cdk4-6 in G1, cyclin E/Cdk2 at the G1/S transition, cyclin A/Cdk2 in S and at the S/G2 transition, and cyclin B/Cdk1 at the G2/M transition. The model accounts for the following major properties of the mammalian cell cycle: i repetitive cell cycling in the presence of suprathreshold amounts of GF; ii control of cell-cycle progression by the balance between antagonistic effects of the tumor suppressor retinoblastoma protein pRB and the transcription factor E2F; and iii existence of a restriction point in G1, beyond which completion of the cell cycle becomes independent of GF. The model also accounts for endoreplication. Incorporating the DNA replication checkpoint mediated by kinases ATR and Chk1 slows down the dynamics of the cell cycle without altering its oscillatory nature and leads to better separation of the S and M phases. The model for the mammalian cell cycle shows how the regulatory structure of the Cdk network results in its temporal self-organization, leading to the repetitive, sequential activation of the four Cdk modules that brings about the orderly progression along cell-cycle phases.</div>','<b>Title: </b>An Introduction to Gaussian Bayesian Networks<br><b>Year: </b>2010<br><b>Author: </b> M. Grzegorczyk<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/978-1-60761-800-36"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SYSTEMS BIOLOGY IN DRUG DISCOVERY AND DEVELOPMENT: METHODS AND PROTOCOLS<br><b>Cluster-Keywords: </b><em>petri net, petri nets, bayesian networks, biological networks, systems biology, regulatory networks, dynamic bayesian, net model, gene regulatory, network data, biological systems, biological data, network analysis, network models, biochemical networks, time series, network structure, signal transduction, net models, dbn model<div style="font-size:11px;"><br><b>Abstract: </b><em>The extraction of regulatory networks and pathways from postgenomic data is important for drug discovery and development, as the extracted pathways reveal how genes or proteins regulate each other. Following up on the seminal paper of Friedman et al. J Comput Biol 7:601-620, 2000, Bayesian networks have been widely applied as a popular tool to this end in systems biology research. Their popularity stems from the tractability of the marginal likelihood of the network structure, which is a consistent scoring scheme in the Bayesian context. This score is based on an integration over the entire parameter space, for which highly expensive computational procedures have to be applied when using more complex models based on differential equations; for example, see Bioinformatics 24:833-839, 2008. This chapter gives an introduction to reverse engineering regulatory networks and pathways with Gaussian Bayesian networks, that is Bayesian networks with the probabilistic BGe scoring metric see Geiger and Heckerman 235-243, 1995. In the BGe model, the data are assumed to stem from a Gaussian distribution and a normal-Wishart prior is assigned to the unknown parameters. Gaussian Bayesian network methodology for analysing static observational, static interventional as well as dynamic observational time series data will be described in detail in this chapter. Finally, we apply these Bayesian network inference methods 1 to observational and interventional flow cytometry protein data from the well-known RAF pathway to evaluate the global network reconstruction accuracy of Bayesian network inference and 2 to dynamic gene expression time series data of nine circadian genes in Arabidopsis thaliana to reverse engineer the unknown regulatory network topology for this domain.</div>','<b>Title: </b>Stochastic simulation and modelling of metabolic networks in a machine learning framework<br><b>Year: </b>2011<br><b>Author: </b> M. Biba,  F. Xhafa,  F. Esposito and  S. Ferilli<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.simpat.2011.05.002"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SIMULATION MODELLING PRACTICE AND THEORY<br><b>Cluster-Keywords: </b><em>kinetic models, metabolic networks, kinetic parameters, dynamic models, systems biology, network model, dynamic metabolic, biochemical network, rate constants, enzyme kinetics, kinetic data, adjustable parameters, metabolic model, parameter estimation, dynamic optimization, model selection, mathematical model, kinetic network, model response, model<div style="font-size:11px;"><br><b>Abstract: </b><em>Metabolomics is increasingly becoming an important field. The fundamental task in this area is to measure and interpret complex time and condition dependent parameters such as the activity or flux of metabolites in cells, their concentration, tissues elements and other biosamples. The careful study of all these elements has led to important insights in the functioning of metabolism. Recently, however, there is a growing interest towards an integrated approach to studying biological systems. This is the main goal in Systems Biology where a combined investigation of several components of a biological system is thought to produce a thorough understanding of such systems. Biological circuits are complex to model and simulate and many efforts are being made to develop models that can handle their intrinsic complexity. A significant part of biological networks still remains unknown even though recent technological developments allow simultaneous acquisition of many metabolite measurements. Metabolic networks are not only structurally complex but behave also in a stochastic fashion. Therefore, it is necessary to express structure and handle uncertainty to construct complete dynamics of these networks. In this paper we describe how stochastic modeling and simulation can be performed in a symbolic-statistical machine learning ML framework. We show that symbolic ML deal with structural and relational complexity while statistical ML provides principled approaches to uncertainty modeling. Learning is used to analyze traces of biochemical reactions and model the dynamicity through parameter learning, while inference is used to produce stochastic simulation of the network. C 2011 Elsevier B. V. All rights reserved.</div>','<b>Title: </b>Dynamic Hebbian learning in adaptive frequency oscillators<br><b>Year: </b>2006<br><b>Author: </b> L. Righetti,  J. Buchli and  Aj. Ijspeert<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.physd.2006.02.009"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PHYSICA D-NONLINEAR PHENOMENA<br><b>Cluster-Keywords: </b><em>oscillator networks, network dynamics, network, networks, dynamical systems, dynamics, node dynamics, systems, coupled oscillator, synchronization, dynamics network, complex network, coupled system, network modifications, oscillators, network connectivity, feedback vertex, network topology, synchronization properties, collective dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>Nonlinear oscillators are widely used in biology, physics and engineering for modeling and control. They are interesting because of their synchronization properties when coupled to other dynamical systems. In this paper, we propose a learning rule for oscillators which adapts their frequency to the frequency of any periodic or pseudo-periodic input signal. Learning is done in a dynamic way: it is part of the dynamical system and not an offline process. An interesting property of our model is that it is easily generalizable to a large class of oscillators, from phase oscillators to relaxation oscillators and strange attractors with a generic learning rule. One major feature of our learning rule is that the oscillators constructed can adapt their frequency without any signal processing or the need to specify a time window or similar free parameters. All the processing is embedded in the dynamics of the adaptive oscillator. The convergence of the learning is proved for the Hopf oscillator, then numerical experiments are carried out to explore the learning capabilities of the system. Finally. we generalize the learning rule to non-harmonic oscillators like relaxation oscillators and strange attractors. c 2006 Elsevier B.V. All rights reserved.</div>','<b>Title: </b>Generating Stochastic Gene Regulatory Networks Consistent With Pathway Information and Steady-State Behavior<br><b>Year: </b>2012<br><b>Author: </b> Jm. Knight,  A. Datta and  Er. Dougherty<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1109/TBME.2012.2192117"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IEEE TRANSACTIONS ON BIOMEDICAL ENGINEERING<br><b>Cluster-Keywords: </b><em>boolean networks, regulatory networks, genetic regulatory, gene regulatory, probabilistic boolean, network, networks, model, network model, probabilistic model, model checking, networks pbns, network dynamics, markov chain, regulatory, models, optimal control, boolean, networks models, networks boolean<div style="font-size:11px;"><br><b>Abstract: </b><em>We present a procedure to generate a stochastic genetic regulatory network model consistent with pathway information. Using the stochastic dynamics of Markov chains, we produce a model constrained by the prior knowledge despite the sometimes incomplete, time independent, and often conflicting nature of these pathways. We apply the Markov theory to study the models long run behavior and introduce a biologically important transformation to aid in comparison with real biological outcome prediction in the steady-state domain. Our technique produces biologically faithful models without the need for rate kinetics, detailed timing information, or complex inference procedures. To demonstrate the method, we produce a model using 28 pathways from the biological literature pertaining to the transcription factor family nuclear factor-kappa B. Predictions from this model in the steady-state domain are then validated against nine mice knockout experiments.</div>','<b>Title: </b>Coupled feedback regulation of nuclear factor of activated T-cells NFAT modulates activation-induced cell death of T cells<br><b>Year: </b>2019<br><b>Author: </b> Sy. Shin,  Mw. Kim,  Kh. Cho and  Lk. Nguyen<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1038/s41598-019-46592-z"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SCIENTIFIC REPORTS<br><b>Cluster-Keywords: </b><em>feedback loops, signaling network, positive feedback, network model, negative feedback, biochemical networks, signaling pathways, signal transduction, systems biology, kinase erk, protein kinase, transduction networks, dynamic model, emt network, erk pathway, cell fate, cell death, biochemical reaction, coupled feedback, impulse response<div style="font-size:11px;"><br><b>Abstract: </b><em>A properly functioning immune system is vital for an organisms wellbeing. Immune tolerance is a critical feature of the immune system that allows immune cells to mount effective responses against exogenous pathogens such as viruses and bacteria, while preventing attack to self-tissues. Activationinduced cell death AICD in T lymphocytes, in which repeated stimulations of the T-cell receptor TCR lead to activation and then apoptosis of T cells, is a major mechanism for T cell homeostasis and helps maintain peripheral immune tolerance. Defects in AICD can lead to development of autoimmune diseases. Despite its importance, the regulatory mechanisms that underlie AICD remain poorly understood, particularly at an integrative network level. Here, we develop a dynamic multi-pathway model of the integrated TCR signalling network and perform model-based analysis to characterize the network-level properties of AICD. Model simulation and analysis show that amplified activation of the transcriptional factor NFAT in response to repeated TCR stimulations, a phenomenon central to AICD, is tightly modulated by a coupled positive-negative feedback mechanism. NFAT amplification is predominantly enabled by a positive feedback self-regulated by NFAT, while opposed by a NFAT-induced negative feedback via Carabin. Furthermore, model analysis predicts an optimal therapeutic window for drugs that help minimize proliferation while maximize AICD of T cells. Overall, our study provides a comprehensive mathematical model of TCR signalling and model-based analysis offers new network-level insights into the regulation of activation-induced cell death in T cells.</div>','<b>Title: </b>Reconstructing gene regulatory dynamics from high-dimensional single-cell snapshot data<br><b>Year: </b>2015<br><b>Author: </b> A. Ocone,  L. Haghverdi,  Ns. Mueller and  Fj. Theis<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1093/bioinformatics/btv257"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>systems biology, dynamic network, early differentiation, complex diseases, stem cells, erythroid cells, system dynamics, network biomarkers, immune system, tcm syndrome, dnb genes, gene expression, dnb model, stem cell, expression dynamics, model system, mathematical model, model dynamics, dynamic model, sensitivity analysis<div style="font-size:11px;"><br><b>Abstract: </b><em>Motivation: High-dimensional single-cell snapshot data are becoming widespread in the systems biology community, as a mean to understand biological processes at the cellular level. However, as temporal information is lost with such data, mathematical models have been limited to capture only static features of the underlying cellular mechanisms.Results: Here, we present a modular framework which allows to recover the temporal behaviour from single-cell snapshot data and reverse engineer the dynamics of gene expression. The framework combines a dimensionality reduction method with a cell time-ordering algorithm to generate pseudo time-series observations. These are in turn used to learn transcriptional ODE models and do model selection on structural network features. We apply it on synthetic data and then on real hematopoietic stem cells data, to reconstruct gene expression dynamics during differentiation pathways and infer the structure of a key gene regulatory network.</div>','<b>Title: </b>BSim: An Agent-Based Tool for Modeling Bacterial Populations in Systems and Synthetic Biology<br><b>Year: </b>2012<br><b>Author: </b> Te. Gorochowski,  A. Matyjaszkiewicz,  T. Todd,  N. Oak,  K. Kowalska,  S. Reid,  Kt. Tsaneva-Atanasova,  Nj. Savery,  Cs. Grierson and  M. Di Bernardo<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0042790"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>gene regulatory, systems biology, synthetic biology, regulatory network, gene network, gene expression, system model, synthetic network, network dynamics, systems theory, synthetic gene, genetic network, model predicts, dynamical systems, time delays, translational time, regulation models, oscillatory expression, network structure, biological systems<div style="font-size:11px;"><br><b>Abstract: </b><em>Large-scale collective behaviors such as synchronization and coordination spontaneously arise in many bacterial populations. With systems biology attempting to understand these phenomena, and synthetic biology opening up the possibility of engineering them for our own benefit, there is growing interest in how bacterial populations are best modeled. Here we introduce BSim, a highly flexible agent-based computational tool for analyzing the relationships between single-cell dynamics and population level features. BSim includes reference implementations of many bacterial traits to enable the quick development of new models partially built from existing ones. Unlike existing modeling tools, BSim fully considers spatial aspects of a model allowing for the description of intricate micro-scale structures, enabling the modeling of bacterial behavior in more realistic three-dimensional, complex environments. The new opportunities that BSim opens are illustrated through several diverse examples covering: spatial multicellular computing, modeling complex environments, population dynamics of the lac operon, and the synchronization of genetic oscillators. BSim is open source software that is freely available from http://bsim-bccs.sf.net and distributed under the Open Source Initiative OSI recognized MIT license. Developer documentation and a wide range of example simulations are also available from the website. BSim requires Java version 1.6 or higher.</div>','<b>Title: </b>Role of SPI-1 Secreted Effectors in Acute Bovine Response to Salmonella enterica Serovar Typhimurium: A Systems Biology Analysis Approach<br><b>Year: </b>2011<br><b>Author: </b> Sd. Lawhon,  S. Khare,  Ca. Rossetti,  Re. Everts,  Cl. Galindo,  Sa. Luciano,  Jf. Figueiredo,  Jes. Nunes,  T. Gull,  Gs. Davidson,  Kl. Drake,  Hr. Garner,  Ha. Lewin,  Aj. Baumler and  Lg. Adams<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0026869"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>gene expression, systems biology, metabolic network, network model, network analysis, dynamic model, signaling pathways, signaling network, regulatory network, dynamic network, biology approach, candidate genes, gene network, gene regulatory, biology analysis, mathematical model, network dynamic, biology network, metabolic pathways, pathway analysis<div style="font-size:11px;"><br><b>Abstract: </b><em>Salmonella enterica Serovar Typhimurium S. Typhimurium causes enterocolitis with diarrhea and polymorphonuclear cell PMN influx into the intestinal mucosa in humans and calves. The Salmonella Type III Secretion System T3SS encoded at Pathogenicity Island I translocates Salmonella effector proteins SipA, SopA, SopB, SopD, and SopE2 into epithelial cells and is required for induction of diarrhea. These effector proteins act together to induce intestinal fluid secretion and transcription of C-X-C chemokines, recruiting PMNs to the infection site. While individual molecular interactions of the effectors with cultured host cells have been characterized, their combined role in intestinal fluid secretion and inflammation is less understood. We hypothesized that comparison of the bovine intestinal mucosal response to wild type Salmonella and a SipA, SopABDE2 effector mutant relative to uninfected bovine ileum would reveal heretofore unidentified diarrhea-associated host cellular pathways. To determine the coordinated effects of these virulence factors, a bovine ligated ileal loop model was used to measure responses to wild type S. Typhimurium WT and a Delta sipA, sopABDE2 mutant MUT across 12 hours of infection using a bovine microarray. Data were analyzed using standard microarray analysis and a dynamic Bayesian network modeling approach DBN. Both analytical methods confirmed increased expression of immune response genes to Salmonella infection and novel gene expression. Gene expression changes mapped to 219 molecular interaction pathways and 1620 gene ontology groups. Bayesian network modeling identified effects of infection on several interrelated signaling pathways including MAPK, Phosphatidylinositol, mTOR, Calcium, Toll-like Receptor, CCR3, Wnt, TGF-beta, and Regulation of Actin Cytoskeleton and Apoptosis that were used to model of host-pathogen interactions. Comparison of WT and MUT demonstrated significantly different patterns of host response at early time points of infection 15 minutes, 30 minutes and one hour within phosphatidylinositol, CCR3, Wnt, and TGF-b signaling pathways and the regulation of actin cytoskeleton pathway.</div>','<b>Title: </b>Stabilizing patterning in the Drosophila segment polarity network by selecting models in silico<br><b>Year: </b>2010<br><b>Author: </b> G. Stoll,  M. Bischofberger,  J. Rougemont and  F. Naef<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.biosystems.2010.07.014"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOSYSTEMS<br><b>Cluster-Keywords: </b><em>network models, gene network, regulatory networks, boolean networks, gene expression, systems biology, neural networks, network dynamics, gene regulatory, biological networks, network structure, gap gene, gene regulation, cell cycle, network, gene interactions, signaling network, quantum systems, system dynamics, networks<div style="font-size:11px;"><br><b>Abstract: </b><em>The segmentation of Drosophila is a prime model to study spatial patterning during embryogenesis. The spatial expression of segment polarity genes results from a complex network of interacting proteins whose expression products are maintained after successful segmentation. This prompted us to investigate the stability and robustness of this process using a dynamical model for the segmentation network based on Boolean states. The model consists of intra-cellular as well as inter-cellular interactions between adjacent cells in one spatial dimension. We quantify the robustness of the dynamical segmentation process by a systematic analysis of mutations. Our starting point consists in a previous Boolean model for Drosophila segmentation. We define mathematically the notion of dynamical robustness and show that the proposed model exhibits limited robustness in gene expression under perturbations. We applied in silico evolution mutation and selection and discover two classes of modified gene networks that have a more robust spatial expression pattern. We verified that the enhanced robustness of the two new models is maintained in differential equations models. By comparing the predicted model with experiments on mutated flies, we then discuss the two types of enhanced models. Drosophila patterning can be explained by modelling the underlying network of interacting genes. Here we demonstrate that simple dynamical considerations and in silico evolution can enhance the model to robustly express the expected pattern, helping to elucidate the role of further interactions. C 2010 Elsevier Ireland Ltd. All rights reserved.</div>','<b>Title: </b>Cellular Potts Model: Applications to Vasculogenesis and Angiogenesis<br><b>Year: </b>2018<br><b>Author: </b> Sem. Boas,  Y. Jiang,  Rmh. Merks,  Sa. Prokopiou and  Eg. Rens<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/978-3-319-65558-118"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PROBABILISTIC CELLULAR AUTOMATA: THEORY, APPLICATIONS AND FUTURE PERSPECTIVES<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>The cellular Potts model CPM, a.k.a. Glazier-Graner-Hogeweg or GGH model is a somewhat liberal extension of probabilistic cellular automata. The model is derived from the Ising and Potts models and represents biological cells as domains of CA-sites of the same state. A Hamiltonian energy is used to describe the balance of forces that the biological cells apply onto one another and their local environment. A Metropolis algorithm iteratively copies the state from one site into one of the adjacent sites, thus shifting the domain interfaces and moving the biological cells along the lattice. The approach is commonly used in applications of developmental biology, where the CPM often interacts with systems of ordinary-differential equations that model the intracellular chemical kinetics and partial-differential equations that model the extracellular chemical signal dynamics to constitute a hybrid and multiscale description of the biological system. In this chapter we will introduce the cellular Potts model and discuss its use in developmental biology, focusing on the development of blood vessels, a process called vascular morphogenesis. We will start by introducing a range of models focusing on uncovering the basic mechanisms of vascular morphogenesis: network formation and sprouting and then show how these models are extended with models of intracellular regulation and with interactions with the extracellular micro-environment. We then briefly review the integration of models of vascular morphogenesis in several examples of organ development in health and disease, including development, cancer, and age-related macular degeneration. We end by discussing the computational efficiency of the CPM and the available strategies for the validation of CPM-based simulation models.</div>','<b>Title: </b>Molecular systems biology of ErbB1 signaling: bridging the gap through multiscale modeling and high-performance computing<br><b>Year: </b>2008<br><b>Author: </b> Aj. Shih,  J. Purvis and  R. Radhakrishnan<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1039/b803806f"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MOLECULAR BIOSYSTEMS<br><b>Cluster-Keywords: </b><em>signaling network, systems biology, signal transduction, signaling pathways, receptor tyrosine, erbb signaling, erbb receptor, receptor signaling, intracellular signaling, mathematical models, cell signaling, computational model, factor receptor, receptor activation, growth factor, signaling models, networks signaling, receptor system, receptors systems, downstream signaling<div style="font-size:11px;"><br><b>Abstract: </b><em>The complexity in intracellular signaling mechanisms relevant for the conquest of many diseases resides at different levels of organization with scales ranging from the subatomic realm relevant to catalytic functions of enzymes to the mesoscopic realm relevant to the cooperative association of molecular assemblies and membrane processes. Consequently, the challenge of representing and quantifying functional or dysfunctional modules within the networks remains due to the current limitations in our understanding of mesoscopic biology, i.e., how the components assemble into functional molecular ensembles. A multiscale approach is necessary to treat a hierarchy of interactions ranging from molecular nm, ns to signaling mm, ms length and time scales, which necessitates the development and application of specialized modeling tools. Complementary to multiscale experimentation encompassing structural biology, mechanistic enzymology, cell biology, and single molecule studies multiscale modeling offers a powerful and quantitative alternative for the study of functional intracellular signaling modules. Here, we describe the application of a multiscale approach to signaling mediated by the ErbB1 receptor which constitutes a network hub for the cells proliferative, migratory, and survival programs. Through our multiscale model, we mechanistically describe how point-mutations in the ErbB1 receptor can profoundly alter signaling characteristics leading to the onset of oncogenic transformations. Specifically, we describe how the point mutations induce cascading fragility mechanisms at the molecular scale as well as at the scale of the signaling network to preferentially activate the survival factor Akt. We provide a quantitative explanation for how the hallmark of preferential Akt activation in cell-lines harboring the constitutively active mutant ErbB1 receptors causes these cell-lines to be addicted to ErbB1-mediated generation of survival signals. Consequently, inhibition of ErbB1 activity leads to a remarkable therapeutic response in the addicted cell lines.</div>','<b>Title: </b>Application of Constraint-based Methods in Staphylococcus aureus Metabolic Network<br><b>Year: </b>2009<br><b>Author: </b> Dw. Ding,  Xq. He,  Kz. Lu,  P. Wu and  Hs. HuangLink: No link<br><b>Journal: </b>RIVISTA DI BIOLOGIA-BIOLOGY FORUM<br><b>Cluster-Keywords: </b><em>metabolic network, metabolic model, dynamic flux, growth rate, genomescale metabolic, flux balance, balance analysis, metabolic flux, metabolic pathways, dynamic model, metabolic reactions, cellular growth, regulatory network, systems biology, optimal growth, metabolic, modeling metabolic, growth conditions, dynamic analysis, cell growth<div style="font-size:11px;"><br><b>Abstract: </b><em>With the remarkable advances in reconstruction of genome-scale metabolic networks, computational methods are increasingly important in investigation of these networks. Since they only need a few available kinetic parameters, constraint-based modeling methods have attracted so much attention in recent years With the important aid by COBRA Toolbox, Staphylococcus aureus metabolic network S. aureustSB619 model which contains 619 genes, 571 metabolites and 640 metabolic reactions is studied in the present paper. We investigated mainly under glucose minimal media condition the optimal flux distributions, optimal growth rates, dynamic growth, robustness analysis to PGK and MDH reactions, gene deletion and uniform random sampling all allowed flux distributions of the model. The study provided a constraint-based modeling framework of Staphylococcus aureus, and the results could be used in its metabolic engineering and industry microbial research</div>','<b>Title: </b>Bicoid gradient formation mechanism and dynamics revealed by protein lifetime analysis<br><b>Year: </b>2018<br><b>Author: </b> L. Durrieu,  D. Kirrmaier,  T. Schneidt,  I. Kats,  S. Raghavan,  L. Hufnagel,  Te. Saunders and  M. Knop<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.15252/msb.20188355"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MOLECULAR SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>biological system, network, pattern formation, system, network analysis, systems, turing patterns, patterns, reaction networks, networks, turing system, network topologies, stoichiometric network, networks pattern, network topology, biological, model, gradient formation, pattern, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Embryogenesis relies on instructions provided by spatially organized signaling molecules known as morphogens. Understanding the principles behind morphogen distribution and how cells interpret locally this information remains a major challenge in developmental biology. Here, we introduce morphogen-age measurements as a novel approach to test models of morphogen gradient formation. Using a tandem fluorescent timer as a protein age sensor, we find a gradient of increasing age of Bicoid along the anterior-posterior axis in the early Drosophila embryo. Quantitative analysis of the protein age distribution across the embryo reveals that the synthesis-diffusion-degradation model is the most likely model underlying Bicoid gradient formation, and rules out other hypotheses for gradient formation. Moreover, we show that the timer can detect transitions in the dynamics associated with syncytial cellularization. Our results provide new insight into Bicoid gradient formation and demonstrate how morphogen-age information can complement knowledge about movement, abundance, and distribution, which should be widely applicable to other systems.</div>','<b>Title: </b>An incoherent regulatory network architecture that orchestrates B cell diversification in response to antigen signaling<br><b>Year: </b>2011<br><b>Author: </b> R. Sciammas,  Y. Li,  A. Warmflash,  Yq. Song,  Ar. Dinner and  H. Singh<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1038/msb.2011.25"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MOLECULAR SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>gene expression, gene network, stochastic model, gene regulatory, systems biology, expression systems, gene regulation, network activity, expression dynamics, noise control, regulatory network, deterministic models, network components, stochastic modeling, regulation system, escherichia coli, regulatory system, synthetic gene, regulatory genes, gene<div style="font-size:11px;"><br><b>Abstract: </b><em>The B-lymphocyte lineage is a leading system for analyzing gene regulatory networks GRNs that orchestrate distinct cell fate transitions. Upon antigen recognition, B cells can diversify their immunoglobulin Ig repertoire via somatic hypermutation SHM and/or class switch DNA recombination CSR before differentiating into antibody-secreting plasma cells. We construct a mathematical model for a GRN underlying this developmental dynamic. The intensity of signaling through the Ig receptor is shown to control the bimodal expression of a pivotal transcription factor, IRF-4, which dictates B cell fate outcomes. Computational modeling coupled with experimental analysis supports a model of kinetic control, in which B cell developmental trajectories pass through an obligate transient state of variable duration that promotes diversification of the antibody repertoire by SHM/CSR in direct response to antigens. More generally, this network motif could be used to translate a morphogen gradient into developmental inductive events of varying time, thereby enabling the specification of distinct cell fates. Molecular Systems Biology 7: 495; published online 24 May 2011; doi: 10.1038/msb.2011.25</div>','<b>Title: </b>Chaos in a ring circuit<br><b>Year: </b>2019<br><b>Author: </b> E. Farcot,  S. Best,  R. Edwards,  I. Belgacem,  X. Xu and  P. Gill<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1063/1.5079941"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>CHAOS<br><b>Cluster-Keywords: </b><em>regulatory networks, network dynamics, gene regulatory, network model, genetic network, gene networks, genetic regulatory, mathematical model, parameter space, network system, interaction networks, system dynamics, network structure, network, dynamic networks, network reduction, models models, biological networks, network motifs, biological systems<div style="font-size:11px;"><br><b>Abstract: </b><em>A ring-shaped logic circuit is proposed here as a robust design for a True Random Number Generator TRNG. Most existing TRNGs rely on physical noise as a source of randomness, where the underlying idealized deterministic system is simply oscillatory. The design proposed here is based on chaotic dynamics and therefore intrinsically displays random behavior, even in the ideal noise-free situation. The paper presents several mathematical models for the circuit having different levels of detail. They take the form of differential equations using steep sigmoid terms for the transfer functions of logic gates. A large part of the analysis is concerned with the hard step-function limit, leading to a model known in mathematical biology as a Glass network. In this framework, an underlying discrete structure a state space diagram is used to describe the likely structure of the global attractor for this system. The latter takes the form of intertwined periodic paths, along which trajectories alternate unpredictably. It is also invariant under the action of the cyclic group. A combination of analytical results and numerical investigations confirms the occurrence of symmetric chaos in this system, which when implemented in noisy hardware, should therefore serve as a robust TRNG. Published under license by AIP Publishing.</div>','<b>Title: </b>Sensitivity analysis of oscillatory biochemical systems<br><b>Year: </b>2005<br><b>Author: </b> De. Zak,  J. Stelling and  Fj. Doyle<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.compchemeng.2004.08.021"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>COMPUTERS  CHEMICAL ENGINEERING<br><b>Cluster-Keywords: </b><em>circadian rhythms, systems biology, biological clock, circadian clock, biological rhythms, hpa axis, model, clock network, mathematical models, biological systems, dynamic systems, ode models, biology models, single cell, network, model parameters, experimental data, models, systems, dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>From the cell cycle to circadian rhythms, oscillatory processes are fundamental to biology. Emerging from nonlinear dynamical interactions, oscillatory mechanisms are best understood through mathematical modeling. Ordinary differential equations ODEs are one framework in which the complex interactions giving rise to biological oscillations may be modeled. Key to ODE models are the model parameters that determine whether or not oscillations will occur, and the period and amplitude of the oscillations when they do. Sensitivity analysis is a means to acquire insight about the importance of the model parameters. Sensitivity analysis of oscillatory systems provides unique challenges and must be addressed carefully. In the present study, we describe a method for determining the sensitivity of the period to the model parameters that is straightforward to implement and interpret. We apply this method to a model for circadian rhythms, and obtain results suggesting a link between network structure and parameter sensitivity. c 2004 Elsevier Ltd. All rights reserved.</div>','<b>Title: </b>The role of computational models of the immune system in designing vaccination strategies<br><b>Year: </b>2005<br><b>Author: </b> F. Castiglione and  A. Liso<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1080/08923970500241030"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IMMUNOPHARMACOLOGY AND IMMUNOTOXICOLOGY<br><b>Cluster-Keywords: </b><em>immune system, network models, immune response, model, biological systems, models, model reduction, dynamic model, biological networks, system, mathematical models, biochemical network, response model, systems, systems biology, network, immune, biology model, reaction networks, system systems<div style="font-size:11px;"><br><b>Abstract: </b><em>Mathematical and computational models are designed to improve our understanding of biological phenomena, to confirm/reject hypotheses, and to find points of intervention by altering the behavior of the studied systems, Here we describe the role of mathematical/computational models of the immune system. In particular, we analyze some examples of how mathematical modeling can contribute to finding optimal vaccination strategies. Indeed, computational modeling offers an intriguing opportunity from the theoretical point of view, and it will be of interest for clinically oriented investigators who wish to find optimal therapeutic strategies and for pharmaceutical industries that want to produce effective and successful drugs.</div>','<b>Title: </b>WASABI: a dynamic iterative framework for gene regulatory network inference<br><b>Year: </b>2019<br><b>Author: </b> A. Bonnaffoux,  U. Herbach,  A. Richard,  A. Guillemin,  S. Gonin-Giraud,  Pa. Gros and  O. Gandrillon<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/s12859-019-2798-1"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>BackgroundInference of gene regulatory networks from gene expression data has been a long-standing and notoriously difficult task in systems biology. Recently, single-cell transcriptomic data have been massively used for gene regulatory network inference, with both successes and limitations.ResultsIn the present work we propose an iterative algorithm called WASABI, dedicated to inferring a causal dynamical network from time-stamped single-cell data, which tackles some of the limitations associated with current approaches. We first introduce the concept of waves, which posits that the information provided by an external stimulus will affect genes one-by-one through a cascade, like waves spreading through a network. This concept allows us to infer the network one gene at a time, after genes have been ordered regarding their time of regulation. We then demonstrate the ability of WASABI to correctly infer small networks, which have been simulated in silico using a mechanistic model consisting of coupled piecewise-deterministic Markov processes for the proper description of gene expression at the single-cell level. We finally apply WASABI on in vitro generated data on an avian model of erythroid differentiation. The structure of the resulting gene regulatory network sheds a new light on the molecular mechanisms controlling this process. In particular, we find no evidence for hub genes and a much more distributed network structure than expected. Interestingly, we find that a majority of genes are under the direct control of the differentiation-inducing stimulus.ConclusionsTogether, these results demonstrate WASABI versatility and ability to tackle some general gene regulatory networks inference issues. It is our hope that WASABI will prove useful in helping biologists to fully exploit the power of time-stamped single-cell data.</div>','<b>Title: </b>Cascades of Failure and Extinction in Evolving Complex Systems<br><b>Year: </b>2006<br><b>Author: </b> P. Ormerod and  R. ColbaughLink: No link<br><b>Journal: </b>JASSS-THE JOURNAL OF ARTIFICIAL SOCIETIES AND SOCIAL SIMULATION<br><b>Cluster-Keywords: </b><em>complex networks, complex systems, network theory, network structure, biological network, social network, network science, dynamic networks, network systems, network model, network topology, systems biology, network, network analysis, system dynamics, networks, realworld networks, network automata, epidemic network, model network<div style="font-size:11px;"><br><b>Abstract: </b><em>There is empirical evidence from a range of disciplines that as the connectivity of a network increases, we observe an increase in the average fitness of the system. But at the same time, there is an increase in the proportion of failure/extinction events which are extremely large. The probability of observing an extreme event remains very low, but it is markedly higher than in the system with lower degrees of connectivity. We are therefore concerned with systems whose properties are not static but which evolve dynamically over time. The focus in this paper, motivated by the empirical examples, is on networks in which the robustness or fragility of the vertices is not given, but which themselves evolve over time We give examples from complex systems such as outages in the US power grid, the robustness properties of cell biology networks, and trade links and the propagation of both currency crises and disease. We consider systems which are populated by agents which are heterogeneous in terms of their fitness for survival. The agents are connected on a network, which evolves over time. In each period agents take self-interested decisions to increase their fitness for survival to form alliances which increase the connectivity of the network. The network is subjected to external negative shocks both with respect to the size of the shock and the spatial impact of the shock. We examine the size/frequency distribution of extinctions and how this distribution evolves as the connectivity of the network grows. The results are robust with respect to the choice of statistical distribution of the shocks. The model is deliberately kept as parsimonious and simple as possible, and refrains from incorporating features such as increasing returns and externalities arising from preferential attachment which might bias the model in the direction of having the empirically observed features of many real world networks. The model still generates results consistent with the empirical evidence: increasing the number of connections causes an increase in the average fitness of agents, yet at the same time makes the system as whole more vulnerable to catastrophic failure/extinction events on an near-global scale.</div>','<b>Title: </b>Partially non-homogeneous dynamic Bayesian networks based on Bayesian regression models with partitioned design matrices<br><b>Year: </b>2019<br><b>Author: </b> Ms. Kamalabad,  Am. Heberle,  K. Thedieck and  M. Grzegorczyk<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1093/bioinformatics/bty917"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>petri net, petri nets, bayesian networks, biological networks, systems biology, regulatory networks, dynamic bayesian, net model, gene regulatory, network data, biological systems, biological data, network analysis, network models, biochemical networks, time series, network structure, signal transduction, net models, dbn model<div style="font-size:11px;"><br><b>Abstract: </b><em>Motivation Non-homogeneous dynamic Bayesian networks NH-DBNs are a popular modelling tool for learning cellular networks from time series data. In systems biology, time series are often measured under different experimental conditions, and not rarely only some network interaction parameters depend on the condition while the other parameters stay constant across conditions. For this situation, we propose a new partially NH-DBN, based on Bayesian hierarchical regression models with partitioned design matrices. With regard to our main application to semi-quantitative immunoblot timecourse data from mammalian target of rapamycin complex 1 mTORC1 signalling, we also propose a Gaussian process-based method to solve the problem of non-equidistant time series measurements.Results On synthetic network data and on yeast gene expression data the new model leads to improved network reconstruction accuracies. We then use the new model to reconstruct the topologies of the circadian clock network in Arabidopsis thaliana and the mTORC1 signalling pathway. The inferred network topologies show features that are consistent with the biological literature.Availability and implementation All datasets have been made available with earlier publications. Our Matlab code is available upon request.Supplementary informationSupplementary data are available at Bioinformatics online.</div>','<b>Title: </b>Multi-component gene network design as a survival strategy in diverse environments<br><b>Year: </b>2018<br><b>Author: </b> Xy. Luo,  Rj. Song and  M. Acar<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/s12918-018-0609-3"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>gene expression, gene network, stochastic model, gene regulatory, systems biology, expression systems, gene regulation, network activity, expression dynamics, noise control, regulatory network, deterministic models, network components, stochastic modeling, regulation system, escherichia coli, regulatory system, synthetic gene, regulatory genes, gene<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Gene-environment interactions are often mediated though gene networks in which gene expression products interact with other network components to dictate network activity levels, which in turn determines the fitness of the host cell in specific environments. Even though a gene network is the right context for studying gene-environment interactions, we have little understanding on how systematic genetic perturbations affects fitness in the context of a gene network.Results: Here we examine the effect of combinatorial gene dosage alterations on gene network activity and cellular fitness. Using the galactose utilization pathway as a model network in diploid yeast, we reduce the copy number of four regulatory genes GAL2, GAL3, GAL4, GAL80 from two to one, and measure the activity of the perturbed networks. We integrate these results with competitive fitness measurements made in six different rationally-designed environments containing different galactose concentrations representing the natural induction spectrum of the galactose network. In the lowest galactose environment, we find a nonlinear relationship between gene expression and fitness while high galactose environments lead to a linear relationship between the two with a saturation regime reached at a sufficiently high galactose concentration. We further uncover environment-specific relevance of the different network components for dictating the relationship between the network activity and organismal fitness, indicating that none of the network components are redundant.Conclusions: These results provide experimental support to the hypothesis that dynamic changes in the environment throughout natural evolution is key to structuring natural gene networks in a multi-component fashion, which robustly provides protection against population extinction in different environments.</div>','<b>Title: </b>Probabilistic approximations of ODEs based bio-pathway dynamics<br><b>Year: </b>2011<br><b>Author: </b> B. Liu,  D. Hsu and  Ps. Thiagarajan<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.tcs.2011.01.021"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>THEORETICAL COMPUTER SCIENCE<br><b>Cluster-Keywords: </b><em>circadian rhythms, systems biology, biological clock, circadian clock, biological rhythms, hpa axis, model, clock network, mathematical models, biological systems, dynamic systems, ode models, biology models, single cell, network, model parameters, experimental data, models, systems, dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>Bio-chemical networks are often modeled as systems of ordinary differential equations ODEs. Such systems will not admit closed form solutions and hence numerical simulations will have to be used to perform analyses. However, the number of simulations required to carry out tasks such as parameter estimation can become very large. To get around this, we propose a discrete probabilistic approximation of the ODEs dynamics. We do so by discretizing the value and the time domain and assuming a distribution of initial states w.r.t. the discretization. Then we sample a representative set of initial states according to the assumed initial distribution and generate a corresponding set of trajectories through numerical simulations. Finally, using the structure of the signaling pathway we encode these trajectories compactly as a dynamic Bayesian network.This approximation of the signaling pathway dynamics has several advantages. First, the discretized nature of the approximation helps to bridge the gap between the accuracy of the results obtained by ODE simulation and the limited precision of experimental data used for model construction and verification. Second and more importantly, many interesting pathway properties can be analyzed efficiently through standard Bayesian inference techniques instead of resorting to a large number of ODE simulations. We have tested our method on ODE models of the EGF-NGF signaling pathway 1 and the segmentation clock pathway 2. The results are very promising in terms of accuracy and efficiency. C 2011 Elsevier B.V. All rights reserved.</div>','<b>Title: </b>A simplistic dynamic circuit analogue of adaptive transport networks in true slime mold<br><b>Year: </b>2016<br><b>Author: </b> Ha. Tanaka,  K. Nakada,  Y. Kondo,  T. Morikawa and  I. Nishikawa<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1587/nolta.7.86"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IEICE NONLINEAR THEORY AND ITS APPLICATIONS<br><b>Cluster-Keywords: </b><em>network ecology, food webs, network models, ecological systems, food web, systems biology, ecological networks, ecosystem model, signalling networks, model results, interaction networks, dynamic model, ecological interactions, network analysis, model network, network structure, transport networks, systems network, ecosystem dynamics, restoration ecology<div style="font-size:11px;"><br><b>Abstract: </b><em>This paper presents a simplistic dynamic circuit analogue for an adaptive transport network model in true slime mold by Tero et al. This circuit analogue model is derived from Teros model through nontrivial simplification under certain assumptions, and it realizes less computational complexity through a reduction of the number of variables. Despite of its simplicity, systematic simulations confirm that the shortest path search task is efficiently accomplished with this model; i the shortest path is always identified, for various random networks; ii if there are multiple, competing shortest paths in the network, they are simultaneously identified; and iii for random deletions of a link in the shortest path, a new shortest path is quickly identified accordingly. The model proposed here is easily implemented on the circuit simulator SPICE for instance, and hence the path search time will be further reduced with certain numerical devices including automatic adaptive numerical integration schemes as well as an acceleration method proposed in the end of the paper.</div>','<b>Title: </b>Hierarchy and polysynchrony in an adaptive network<br><b>Year: </b>2014<br><b>Author: </b> V. Botella-Soler and  P. Glendinning<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1103/PhysRevE.89.062809"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PHYSICAL REVIEW E<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>We describe a simple adaptive network of coupled chaotic maps. The network reaches a stationary state frozen topology for all values of the coupling parameter, although the dynamics of the maps at the nodes of the network can be nontrivial. The structure of the network shows interesting hierarchical properties and in certain parameter regions the dynamics is polysynchronous: Nodes can be divided in differently synchronized classes but, contrary to cluster synchronization, nodes in the same class need not be connected to each other. These complicated synchrony patterns have been conjectured to play roles in systems biology and circuits. The adaptive system we study describes ways whereby this behavior can evolve from undifferentiated nodes.</div>','<b>Title: </b>Limitations and tradeoffs in synchronization of large-scale networks with uncertain links<br><b>Year: </b>2016<br><b>Author: </b> A. Diwadkar and  U. Vaidya<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1038/srep21157"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SCIENTIFIC REPORTS<br><b>Cluster-Keywords: </b><em>oscillator networks, network dynamics, network, networks, dynamical systems, dynamics, node dynamics, systems, coupled oscillator, synchronization, dynamics network, complex network, coupled system, network modifications, oscillators, network connectivity, feedback vertex, network topology, synchronization properties, collective dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>The synchronization of nonlinear systems connected over large-scale networks has gained popularity in a variety of applications, such as power grids, sensor networks, and biology. Stochastic uncertainty in the interconnections is a ubiquitous phenomenon observed in these physical and biological networks. We provide a size-independent network sufficient condition for the synchronization of scalar nonlinear systems with stochastic linear interactions over large-scale networks. This sufficient condition, expressed in terms of nonlinear dynamics, the Laplacian eigenvalues of the nominal interconnections, and the variance and location of the stochastic uncertainty, allows us to define a synchronization margin. We provide an analytical characterization of important trade-offs between the internal nonlinear dynamics, network topology, and uncertainty in synchronization. For nearest neighbour networks, the existence of an optimal number of neighbours with a maximum synchronization margin is demonstrated. An analytical formula for the optimal gain that produces the maximum synchronization margin allows us to compare the synchronization properties of various complex network topologies.</div>','<b>Title: </b>Subnetwork analysis reveals dynamic features of complex biochemical networks<br><b>Year: </b>2007<br><b>Author: </b> C. Conradi,  D. Flockerzi,  J. Raisch and  J. Stelling<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1073/pnas.0705731104"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA<br><b>Cluster-Keywords: </b><em>reaction networks, reaction systems, metabolic networks, chemical reaction, network structure, systems biology, biological systems, biological networks, biochemical reaction, reaction rates, differential equations, reaction graph, complex networks, ssi modules, networks, life systems, ordinary differential, cellular networks, reaction, network flows<div style="font-size:11px;"><br><b>Abstract: </b><em>In analyzing and mathematical modeling of complex biochemical reaction networks, formal methods that connect network structure and dynamic behavior are needed because often, quantitative knowledge of the networks is very limited. This applies to many important processes in cell biology. Chemical reaction network theory allows for the classification of the potential network behavior-for instance, with respect to the existence of multiple steady states-but is computationally limited to small systems. Here, we show that by analyzing subnetworks termed elementary flux modes, the applicability of the theory can be extended to more complex networks. For an example network inspired by cell cycle control in budding yeast, the approach allows for model discrimination, identification of key mechanisms for multistationarity, and robustness analysis. The presented methods will be helpful in modeling and analyzing other complex reaction networks.</div>','<b>Title: </b>A micro-perfusion bioreactor for on line investigation of ECM remodeling under hydrodynamic and biochemical stimulation<br><b>Year: </b>2016<br><b>Author: </b> A. Garziano,  F. Urciuolo,  G. Imparato,  F. Martorina,  B. Corrado and  P. Netti<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1039/c5lc01481f"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>LAB ON A CHIP<br><b>Cluster-Keywords: </b><em>neural network, network model, network, neurogenesis model, genomic network, networks, saccade control, droplets networks, understanding networks, gene regulatory, model, tumor-immune networks, networks dynamical, network systems, artificial neurogenesis, regulatory networks, dynamic network, information flow, cancer cells, capillary networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Tissue-on-chip TOC systems aim at replicating complex biological dynamics in vitro with the potential either to improve the understanding of human biology or to develop more accurate therapeutic strategies. To replicate faithfully the intricate interrelationships between cells and their surrounding microenvironment, the three-dimensional 3D tissue model must possess a responsive extracellular matrix ECM. ECM remodeling plays a pivotal role in guiding cells and tissues functions and such aspect is somewhat denied during in vitro studies. For this purpose, we fabricated a micro-perfusion bioreactor capable to sustain the viability of 3D engineered tissue models recapitulating the process of the native ECM deposition and assembly. Engineered human dermis micro-tissue precursors HD-mu TP were used as building blocks to generate a final tissue. HD-mu TP were loaded in the perfusion space of the micro-perfusion bioreactor and, under the superimposition of different fluid dynamic regimes and biochemical stimulation, they synthesized new collagen proteins that were, then, assembled in the perfusion space forming a continuum of cells embedded in their own ECM. The micro-perfusion bioreactor was fabricated to allow the on-line monitoring of the oxygen consumption and the assembly of the newly formed collagen network via real time acquisition of the second harmonic generation SHG signal. The possibility to detect the collagen reorganization due to both fluid dynamic and biochemical stimulation, let us to define the optimal perfusion configuration in order to obtain a TOC system based on an endogenous and responsive ECM.</div>','<b>Title: </b>An enzyme-centric approach for modelling non-linear biological complexity<br><b>Year: </b>2008<br><b>Author: </b> Cr. Yang<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-2-70"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>biological system, network, pattern formation, system, network analysis, systems, turing patterns, patterns, reaction networks, networks, turing system, network topologies, stoichiometric network, networks pattern, network topology, biological, model, gradient formation, pattern, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: The current challenge of Systems Biology is to integrate high throughput data sets for simulating the complexity of biological networks, exploit the evolution of nature-designed networks that maintain the robustness of a biological system, and thereby generate novel, experimentally testable hypotheses. In order to simulate non-linear biological complexities, we have previously developed an Enzyme-Centric mechanistic modeling approach and validated it using metabolic network in E. coli. The idea is to use prior knowledge of catalytic and regulatory mechanisms of each enzyme within the metabolic network to build a dynamic model for investigating the network level regulation and thus understand the nature design principle behind the network.Results: In this paper, we further demonstrate the application of complex enzyme catalytic and regulatory modules to simulate nonlinear network regulatory patterns vs. simple linear conversion model. We learned and validated that it is essential to incorporate prior knowledge from the literature to simulate non-linear biological complexities. The network expandability is demonstrated and validated with the complex amino acid biosynthetic network with multi-regulations. Also, we demonstrated the compatibility of mechanistic models within close species. Furthermore, the eukaryotic protein factory model for insuring steady mRNA production is simulated and the coupling of RNA transcription and splicing is validated by both mathematical simulation and experimental analysis.Conclusion: We demonstrated the importance of modeling complex enzyme catalytic and regulatory mechanisms to further understand nonlinear network regulatory patterns. The simulations presented in this paper reveal how a living system maintains homeostasis and its robustness to continue functioning while facing environmental stresses or genetic mutations.</div>','<b>Title: </b>Systemic Cold Stress Adaptation of Chlamydomonas reinhardtii<br><b>Year: </b>2013<br><b>Author: </b> L. Valledor,  T. Furuhashi,  Am. Hanak and  W. Weckwerth<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1074/mcp.M112.026765"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MOLECULAR  CELLULAR PROTEOMICS<br><b>Cluster-Keywords: </b><em>gene expression, systems biology, metabolic network, network model, network analysis, dynamic model, signaling pathways, signaling network, regulatory network, dynamic network, biology approach, candidate genes, gene network, gene regulatory, biology analysis, mathematical model, network dynamic, biology network, metabolic pathways, pathway analysis<div style="font-size:11px;"><br><b>Abstract: </b><em>Chlamydomonas reinhardtii is one of the most important model organisms nowadays phylogenetically situated between higher plants and animals Merchant et al. 2007. Stress adaptation of this unicellular model algae is in the focus because of its relevance to biomass and biofuel production. Here, we have studied cold stress adaptation of C. reinhardtii hitherto not described for this algae whereas intensively studied in higher plants. Toward this goal, high throughput mass spectrometry was employed to integrate proteome, metabolome, physiological and cell-morphological changes during a time-course from 0 to 120 h. These data were complemented with RT-qPCR for target genes involved in central metabolism, signaling, and lipid biosynthesis. Using this approach dynamics in central metabolism were linked to cold-stress dependent sugar and autophagy pathways as well as novel genes in C. reinhardtii such as CKIN1, CKIN2 and a hitherto functionally not annotated protein named CKIN3. Cold stress affected extensively the physiology and the organization of the cell. Gluconeogenesis and starch biosynthesis pathways are activated leading to a pronounced starch and sugar accumulation. Quantitative lipid profiles indicate a sharp decrease in the lipophilic fraction and an increase in polyunsaturated fatty acids suggesting this as a mechanism of maintaining membrane fluidity. The proteome is completely remodeled during cold stress: specific candidates of the ribosome and the spliceosome indicate altered biosynthesis and degradation of proteins important for adaptation to low temperatures. Specific proteasome degradation may be mediated by the observed cold-specific changes in the ubiquitinylation system. Sparse partial least squares regression analysis was applied for protein correlation network analysis using proteins as predictors and Fv/Fm, FW, total lipids, and starch as responses. We applied also Granger causality analysis and revealed correlations between proteins and metabolites otherwise not detectable. Twenty percent of the proteins responsive to cold are uncharacterized proteins. This presents a considerable resource for new discoveries in cold stress biology in alga and plants.</div>','<b>Title: </b>Network motifs: structure does not determine function<br><b>Year: </b>2006<br><b>Author: </b> Pj. Ingram,  Mph. Stumpf and  J. Stark<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1471-2164-7-108"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC GENOMICS<br><b>Cluster-Keywords: </b><em>network motifs, regulatory network, gene regulatory, systems biology, gene network, bifan motif, complex systems, model comparison, network modeling, complex gene, biological systems, dynamic systems, interaction network, feedforward loop, modeling gene, intelligent systems, modeling approach, biological pathways, motif model, model motifs<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: A number of publications have recently examined the occurrence and properties of the feed-forward motif in a variety of networks, including those that are of interest in genome biology, such as gene networks. The present work looks in some detail at the dynamics of the bi-fan motif, using systems of ordinary differential equations to model the populations of transcription factors, mRNA and protein, with the aim of extending our understanding of what appear to be important building blocks of gene network structure.Results: We develop an ordinary differential equation model of the bi-fan motif and analyse variants of the motif corresponding to its behaviour under various conditions. In particular, we examine the effects of different steady and pulsed inputs to five variants of the bifan motif, based on evidence in the literature of bifan motifs found in Saccharomyces cerevisiae commonly known as bakers yeast. Using this model, we characterize the dynamical behaviour of the bi-fan motif for a wide range of biologically plausible parameters and configurations. We find that there is no characteristic behaviour for the motif, and with the correct choice of parameters and of internal structure, very different, indeed even opposite behaviours may be obtained.Conclusion: Even with this relatively simple model, the bi-fan motif can exhibit a wide range of dynamical responses. This suggests that it is difficult to gain significant insights into biological function simply by considering the connection architecture of a gene network, or its decomposition into simple structural motifs. It is necessary to supplement such structural information by kinetic parameters, or dynamic time series experimental data, both of which are currently difficult to obtain.</div>','<b>Title: </b>Kinase inhibitors can produce off-target effects and activate linked pathways by retroactivity<br><b>Year: </b>2011<br><b>Author: </b> Ml. Wynn,  Ac. Ventura,  Ja. Sepulchre,  Hj. Garcia and  Sd. Merajver<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-5-156"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>dna damage, signalling networks, network dynamics, pmdm system, systems biology, systems dynamics, biological systems, single cell, kinase inhibitors, signaling network, biological networks, cellular automata, cell cycle, oscillation system, temporal dynamics, signaling pathway, individual cells, signaling processes, dynamic model, damage response<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: It has been shown in experimental and theoretical work that covalently modified signaling cascades naturally exhibit bidirectional signal propagation via a phenomenon known as retroactivity. An important consequence of retroactivity, which arises due to enzyme sequestration in covalently modified signaling cascades, is that a downstream perturbation can produce a response in a component upstream of the perturbation without the need for explicit feedback connections. Retroactivity may, therefore, play an important role in the cellular response to a targeted therapy. Kinase inhibitors are a class of targeted therapies designed to interfere with a specific kinase molecule in a dysregulated signaling pathway. While extremely promising as anti-cancer agents, kinase inhibitors may produce undesirable off-target effects by non-specific interactions or pathway cross-talk. We hypothesize that targeted therapies such as kinase inhibitors can produce off-target effects as a consequence of retroactivity alone.Results: We used a computational model and a series of simple signaling motifs to test the hypothesis. Our results indicate that within physiologically and therapeutically relevant ranges for all parameters, a targeted inhibitor can naturally induce an off-target effect via retroactivity. The kinetics governing covalent modification cycles in a signaling network were more important for propagating an upstream off-target effect in our models than the kinetics governing the targeted therapy itself. Our results also reveal the surprising and crucial result that kinase inhibitors have the capacity to turn "on" an otherwise "off" parallel cascade when two cascades share an upstream activator.Conclusions: A proper and detailed characterization of a pathways structure is important for identifying the optimal protein to target as well as what concentration of the targeted therapy is required to modulate the pathway in a safe and effective manner. We believe our results support the position that such characterizations should consider retroactivity as a robust potential source of off-target effects induced by kinase inhibitors and other targeted therapies.</div>','<b>Title: </b>Bridging the gap between modules in isolation and as part of networks: A systems framework for elucidating interaction and regulation of signalling modules<br><b>Year: </b>2016<br><b>Author: </b> G. Menon and  J. Krishnan<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1063/1.4953914"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF CHEMICAL PHYSICS<br><b>Cluster-Keywords: </b><em>blood flow, blood cells, flow rate, blood pumps, cells, cell, results, patients, stem cells, methods, blood, patients blood, flow, patients methods, patients objectives, cell viability, objectives, results cells, blood pressure, cell culture<div style="font-size:11px;"><br><b>Abstract: </b><em>While signalling and biochemical modules have been the focus of numerous studies, they are typically studied in isolation, with no examination of the effects of the ambient network. In this paper we formulate and develop a systems framework, rooted in dynamical systems, to understand such effects, by studying the interaction of signalling modules. The modules we consider are i basic covalent modification, ii monostable switches, iii bistable switches, iv adaptive modules, and v oscillatory modules. We systematically examine the interaction of these modules by analyzing a sequential interaction without shared components, b sequential interaction with shared components, and c oblique interactions. Our studies reveal that the behaviour of a module in isolation may be substantially different from that in a network, and explicitly demonstrate how the behaviour of a given module, the characteristics of the ambient network, and the possibility of shared components can result in new effects. Our global approach illuminates different aspects of the structure and functioning of modules, revealing the importance of dynamical characteristics as well as biochemical features; this provides a methodological platform for investigating the complexity of natural modules shaped by evolution, elucidating the effects of ambient networks on a module in multiple cellular contexts, and highlighting the capabilities and constraints for engineering robust synthetic modules. Overall, such a systems framework provides a platform for bridging the gap between non-linear information processing modules, in isolation and as parts of networks, and a basis for understanding new aspects of natural and engineered cellular networks. Published by AIP Publishing.</div>','<b>Title: </b>Systems Biology and Metabolic Engineering in Bacteria<br><b>Year: </b>2014<br><b>Author: </b> J. Geiselmann<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/978-3-642-38505-613"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SYSTEMS BIOLOGY OF METABOLIC AND SIGNALING NETWORKS: ENERGY, MASS AND INFORMATION TRANSFER<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Complete metabolic maps are currently available for a number of important bacteria. Even when these maps are not experimentally confirmed, the topology of the metabolic network can be reconstructed from the genome sequence. Despite this extensive information, we still lack a good understanding of metabolic adaptations, the interactions of metabolism with gene regulation and tools for predicting the metabolic consequences of modifying the metabolic or genetic regulatory network of a bacterium. This chapter will briefly review current methods for analyzing bacterial metabolism from topological models and steady state techniques, such as flux balance analysis, to dynamical models using ordinary differential equations. Even though still incomplete, these models can predict the metabolic behavior of modified organisms. Using these tools, we can create novel metabolic pathways or optimize the yield of a desired metabolite. Focusing on Escherichia coli, we present examples of successful metabolic engineering using such systems-wide, rational approaches, integrating modeling and experiments. The conjunction of systems biology to metabolic engineering yields new insights into the fundamental functioning of the cell and opens the path to the biological production of a large variety of commodity chemicals.</div>','<b>Title: </b>Dissipative structures and biological rhythms<br><b>Year: </b>2017<br><b>Author: </b> A. Goldbeter<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1063/1.4990783"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>CHAOS<br><b>Cluster-Keywords: </b><em>circadian rhythms, systems biology, biological clock, circadian clock, biological rhythms, hpa axis, model, clock network, mathematical models, biological systems, dynamic systems, ode models, biology models, single cell, network, model parameters, experimental data, models, systems, dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>Sustained oscillations abound in biological systems. They occur at all levels of biological organization over a wide range of periods, from a fraction of a second to years, and with a variety of underlying mechanisms. They control major physiological functions, and their dysfunction is associated with a variety of physiological disorders. The goal of this review is i to give an overview of the main rhythms observed at the cellular and supracellular levels, ii to briefly describe how the study of biological rhythms unfolded in the course of time, in parallel with studies on chemical oscillations, iii to present the major roles of biological rhythms in the control of physiological functions, and iv the pathologies associated with the alteration, disappearance, or spurious occurrence of biological rhythms. Two tables present the main examples of cellular and supracellular rhythms ordered according to their period, and their role in physiology and pathophysiology. Among the rhythms discussed are neural and cardiac rhythms, metabolic oscillations such as those occurring in glycolysis in yeast, intracellular Ca oscillations, cyclic AMP oscillations in Dictyostelium amoebae, the segmentation clock that controls somitogenesis, pulsatile hormone secretion, circadian rhythms which occur in all eukaryotes and some bacteria with a period close to 24 h, the oscillatory dynamics of the enzymatic network driving the cell cycle, and oscillations in transcription factors such as NF-KB and tumor suppressors such as p53. Ilya Prigogines concept of dissipative structures applies to temporal oscillations and allows us to unify within a common framework the various rhythms observed at different levels of biological organization, regardless of their period and underlying mechanism. Published by AIP Publishing.</div>','<b>Title: </b>Methods of Information Geometry in Computational System Biology Consistency between Chemical and Biological Evolution<br><b>Year: </b>2009<br><b>Author: </b> V. Astakhov<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/978-1-59745-524-46"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOMEDICAL INFORMATICS<br><b>Cluster-Keywords: </b><em>immune system, network models, immune response, model, biological systems, models, model reduction, dynamic model, biological networks, system, mathematical models, biochemical network, response model, systems, systems biology, network, immune, biology model, reaction networks, system systems<div style="font-size:11px;"><br><b>Abstract: </b><em>Interest in simulation of large-scale metabolic networks, species development, and genesis of various diseases requires new simulation techniques to accommodate the high complexity of realistic biological networks. Information geometry and topological formalisms are proposed to analyze information processes. We analyze the complexity of large-scale biological networks as well as transition of the system functionality, due to modification in the system architecture, system environment, and system components.The dynamic core model is developed. The term dynamic core is used to define a set of causally related network functions. Delocalization of dynamic core model provides a mathematical formalism to analyze migration of specific functions in biosystems which undergo structure transition induced by the environment. The term delocalization is used to describe these processes of migration. We constructed a holographic model with self-poetic dynamic cores which preserves functional properties under those transitions. Topological constraints Such as Ricci flow and Pfaff dimension Were found for statistical manifolds which represent biological networks. These constraints can provide insight on processes of degeneration and recovery which take place in large-scale networks. We would like to Suggest that therapies Which arc able to effectively implement estimated constraints, Will Successfully adjust biological systems and recover altered functionality Also, we mathematically formulate the hypothesis that there is a direct consistency between biological and chemical evolution. Any set of causal relations within a biological network has its dual reimplementation in the chemistry of the system environment.</div>','<b>Title: </b>A consensus approach for estimating the predictive accuracy of dynamic models in biology<br><b>Year: </b>2015<br><b>Author: </b> Af. Villaverde,  S. Bongard,  K. Mauch,  D. Muller,  E. Balsa-Canto,  J. Schmid and  Jr. Banga<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.cmpb.2015.02.001"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>COMPUTER METHODS AND PROGRAMS IN BIOMEDICINE<br><b>Cluster-Keywords: </b><em>biological networks, dynamic models, biological systems, model predictions, model parameters, systems biology, mathematical models, biological models, equation models, experimental data, molecular network, interaction networks, model errors, biological interaction, molecular interactions, model, dynamic systems, biological parameters, parameter uncertainty, growth model<div style="font-size:11px;"><br><b>Abstract: </b><em>Mathematical models that predict the complex dynamic behaviour of cellular networks are fundamental in systems biology, and provide an important basis for biomedical and biotechnological applications. However, obtaining reliable predictions from large-scale dynamic models is commonly a challenging task due to lack of identifiability. The present work addresses this challenge by presenting a methodology for obtaining high-confidence predictions from dynamic models using time-series data. First, to preserve the complex behaviour of the network while reducing the number of estimated parameters, model parameters are combined in sets of meta-parameters, which are obtained from correlations between biochemical reaction rates and between concentrations of the chemical species. Next, an ensemble of models with different parameterizations is constructed and calibrated. Finally, the ensemble is used for assessing the reliability of model predictions by defining a measure of convergence of model outputs consensus that is used as an indicator of confidence. We report results of computational tests carried out on a metabolic model of Chinese Hamster Ovary CHO cells, which are used for recombinant protein production. Using noisy simulated data, we find that the aggregated ensemble predictions are on average more accurate than the predictions of individual ensemble models. Furthermore, ensemble predictions with high consensus are statistically more accurate than ensemble predictions with large variance. The procedure provides quantitative estimates of the confidence in model predictions and enables the analysis of sufficiently complex networks as required for practical applications. C 2015 Elsevier Ireland Ltd. All rights reserved.</div>','<b>Title: </b>Maximin and Bayesian robust experimental design for measurement set selection in modelling biochemical regulatory systems<br><b>Year: </b>2010<br><b>Author: </b> F. He,  M. Brown and  H. Yue<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1002/rnc.1558"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>INTERNATIONAL JOURNAL OF ROBUST AND NONLINEAR CONTROL<br><b>Cluster-Keywords: </b><em>systems biology, biological systems, system dynamics, molecular systems, experimental design, network model, model system, model parameters, experiment design, kinetic parameters, perfect adaptation, experimental data, molecular mechanisms, homeostatic mechanisms, switching dynamics, biological mechanisms, control systems, chemotaxis network, decentralized control, feedback control<div style="font-size:11px;"><br><b>Abstract: </b><em>Experimental design is important in system identification, especially when the models are complex and the measurement data are sparse and noisy, as often occurs in modelling of biochemical regulatory networks. The quality of conventional optimal experimental design largely depends on the accuracy of model parameter estimation, which is often either unavailable or poorly estimated at the stage of design. Robust experimental design RED algorithms have thus been proposed when model parametric uncertainties need to be addressed during the design process. In this paper, two robust design strategies are investigated and the comparative study has been made on signal pathway models. The first method is a maximin experimental design approach which is a worst-case design strategy, and the second method is the Bayesian experimental design that takes an average of the parametric uncertainty effects. The limitations of the maximin design which describes the structural uncertainty using a local Taylor representation are quantitatively evaluated. To better quantitatively assess the differences between the maximin and the Bayesian REDs, a concept of effective design parameters is proposed, from which the advantages of the Bayesian design is demonstrated especially in the case of large model uncertainties. Copyright C 2010 John Wiley  Sons, Ltd.</div>','<b>Title: </b>Systems Approaches to Understanding and Designing Allosteric Proteins<br><b>Year: </b>2018<br><b>Author: </b> S. Raman<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1021/acs.biochem.7b01094"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOCHEMISTRY<br><b>Cluster-Keywords: </b><em>allosteric proteins, signaling networks, protein interactions, network model, molecular dynamics, protein complexes, allosteric network, signaling proteins, cell network, biochemical networks, free energy, combinatorial complexity, molecular species, biological systems, receptor signaling, systems biology, chemical species, model, dynamics simulations, cell signaling<div style="font-size:11px;"><br><b>Abstract: </b><em>The study of allostery has a central place in biology because of the myriad roles of allosteric proteins in cellular function. As technologies for probing the spatiotemporal resolution of biomolecules have become increasingly sophisticated, so has our understanding of the diverse structural and molecular mechanisms of allosteric proteins. Studies have shown that the allosteric signal is transmitted a through a network of residue residue interactions connecting distal sites on a protein. Linking structural and dynamical changes to the functional role of individual residues will give a more complete molecular view of allostery. In this work, we highlight new mutational technologies that enable a systems-level, quantitative description of allostery that dissect the role of individual residues through large-scale functional screens. A molecular model for predicting allosteric hot spots can be developed by applying statistical tools on the resulting large sequence structure function data sets. Design of allosteric proteins with new function is essential for engineering biological systems. Previous design efforts demonstrate that the allosteric network is a powerful functional constraint in the design of novel or enhanced allosteric proteins. We discuss how a priori knowledge of an allosteric network could improve rational design by facilitating better navigation of the design space. Understanding the molecular "rules" governing allostery would elucidate the molecular basis of dysfunction in disease-associated allosteric proteins, provide a means for designing tailored therapeutics, and enable the design of new sensors and enzymes for synthetic biology.</div>','<b>Title: </b>Combinatorial Representation of Parameter Space for Switching Networks<br><b>Year: </b>2016<br><b>Author: </b> B. Cummins,  T. Gedeon,  S. Harker,  K. Mischaikow and  K. Mok<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1137/15M1052743"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SIAM JOURNAL ON APPLIED DYNAMICAL SYSTEMS<br><b>Cluster-Keywords: </b><em>regulatory networks, network dynamics, gene regulatory, network model, genetic network, gene networks, genetic regulatory, mathematical model, parameter space, network system, interaction networks, system dynamics, network structure, network, dynamic networks, network reduction, models models, biological networks, network motifs, biological systems<div style="font-size:11px;"><br><b>Abstract: </b><em>We describe the theoretical and computational framework for the Dynamic Signatures Generated by Regulatory Networks DSGRN database. The motivation stems from an urgent need to understand the global dynamics of biologically relevant signal transduction/gene regulatory networks that have at least 5 to 10 nodes, involve multiple interactions, and have decades of parameters. The input to the database computations is a regulatory network, i.e., a directed graph with edges indicating up or down regulation. A computational model based on switching networks is generated from the regulatory network. The phase space dimension of this model equals the number of nodes and the associated parameter space consists of one parameter for each node a decay rate and three parameters for each edge low level of expression, high level of expression, and threshold at which expression levels change. Since the nonlinearities of switching systems are piecewise constant, there is a natural decomposition of phase space into cells from which the dynamics can be described combinatorially in terms of a state transition graph. This in turn leads to a compact representation of the global dynamics called an annotated Morse graph that identifies recurrent and nonrecurrent dynamics. The focus of this paper is on the construction of a natural computable finite decomposition of parameter space into domains where the annotated Morse graph description of dynamics is constant. We use this decomposition to construct an SQL database that can be effectively searched for dynamical signatures such as bistability, stable or unstable oscillations, and stable equilibria. We include two simple 3-node networks to provide small explicit examples of the type of information stored in the DSGRN database. To demonstrate the computational capabilities of this system we consider a simple network associated with p53 that involves 5 nodes and a 29-dimensional parameter space.</div>','<b>Title: </b>Dynamical regimes in non-ergodic random Boolean networks<br><b>Year: </b>2017<br><b>Author: </b> M. Villani,  D. Campioli,  C. Damiani,  A. Roli,  A. Filisetti and  R. Serra<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s11047-016-9552-7"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>NATURAL COMPUTING<br><b>Cluster-Keywords: </b><em>boolean networks, regulatory networks, network model, bayesian networks, model networks, systems biology, random boolean, real cells, immune system, genetic regulatory, networks, biochemical network, fluorescence images, cellular network, gene regulatory, cell differentiation, network, biological systems, ensemble approach, cell dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>Random boolean networks are a model of genetic regulatory networks that has proven able to describe experimental data in biology. Random boolean networks not only reproduce important phenomena in cell dynamics, but they are also extremely interesting from a theoretical viewpoint, since it is possible to tune their asymptotic behaviour from order to disorder. The usual approach characterizes network families as a whole, either by means of static or dynamic measures. We show here that a more detailed study, based on the properties of systems attractors, can provide information that makes it possible to predict with higher precision important properties, such as systems response to gene knock-out. A new set of principled measures is introduced, that explains some puzzling behaviours of these networks. These results are not limited to random Boolean network models, but they are general and hold for any discrete model exhibiting similar dynamical characteristics.</div>','<b>Title: </b>Reconstruction and Analysis of a Genome-Scale Metabolic Network for Eriocheir Sinensis Hepatopancreas<br><b>Year: </b>2018<br><b>Author: </b> T. Hao,  B. Wang,  Lx. Zhao,  X. Feng and  Js. Sun<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1109/ACCESS.2018.2885005"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IEEE ACCESS<br><b>Cluster-Keywords: </b><em>gene expression, systems biology, metabolic network, network model, network analysis, dynamic model, signaling pathways, signaling network, regulatory network, dynamic network, biology approach, candidate genes, gene network, gene regulatory, biology analysis, mathematical model, network dynamic, biology network, metabolic pathways, pathway analysis<div style="font-size:11px;"><br><b>Abstract: </b><em>The application of big data in biology has been a hot trend in the research of bioinformatics. High throughput sequencing and large-scale biology network contribute massive valuable data for the complex biology analysis, in which genome-scale metabolic network GSMN has become an important tool for the systematic analysis of metabolism in various organisms. We reconstructed a GSMN model based on the high-throughput transcriptome sequencing of Eriocheir sinensis hepatopancreas. The model contains 1471 reactions, 1882 unigenes, and 1400 metabolites distributed in 101 pathways and 11 subsystems. Functional module analysis indicates that the metabolic modules in the network is in consistence with the functions of hepatopancreas for lipid metabolism, amino acid metabolism, and other related metabolic processes. Differential analysis of the hepatopancreas reveals that the eyestalk-ablation has the greatest effect on the lipid metabolism pathway, with a total of 66 related unigenes differentially expressed, which agrees with the functions of the hepatopancreas for nutrient absorption, storage, and utilization. The GSMN supplies a novel dataset of the metabolic information in E.sinensis to facilitate the further analysis of the metabolic mechanism of E.sinensis and other aquatic crustaceans. Besides, it also provides a valuable reference for the studies on regulation mechanism of eyestalk on hepatopancreas in E.sinensis.</div>','<b>Title: </b>Hybrid dynamic modeling of Escherichia coli central metabolic network combining Michaelis-Menten and approximate kinetic equations<br><b>Year: </b>2010<br><b>Author: </b> Rs. Costa,  D. Machado,  I. Rocha and  Ec. Ferreira<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.biosystems.2010.03.001"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOSYSTEMS<br><b>Cluster-Keywords: </b><em>kinetic models, metabolic networks, kinetic parameters, dynamic models, systems biology, network model, dynamic metabolic, biochemical network, rate constants, enzyme kinetics, kinetic data, adjustable parameters, metabolic model, parameter estimation, dynamic optimization, model selection, mathematical model, kinetic network, model response, model<div style="font-size:11px;"><br><b>Abstract: </b><em>The construction of dynamic metabolic models at reaction network level requires the use of mechanistic enzymatic rate equations that comprise a large number of parameters. The lack of knowledge on these equations and the difficulty in the experimental identification of their associated parameters, represent nowadays the limiting factor in the construction of such models. In this study, we compare four alternative modeling approaches based on Michaelis-Menten kinetics for the bi-molecular reactions and different types of simplified rate equations for the remaining reactions generalized mass action, convenience kinetics, lin-log and power-law. Using the mechanistic model for Escherichia coli central carbon metabolism as a benchmark, we investigate the alternative modeling approaches through comparative simulations analyses. The good dynamic behavior and the powerful predictive capabilities obtained using the hybrid model composed of Michaelis-Menten and the approximate lin-log kinetics indicate that this is a possible suitable approach to model complex large-scale networks where the exact rate laws are unknown. C 2010 Elsevier Ireland Ltd. All rights reserved.</div>','<b>Title: </b>Modelling and analysis of gene regulatory network using feedback control theory<br><b>Year: </b>2010<br><b>Author: </b> H. El-Samad and  M. Khammash<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1080/00207720903144545"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>INTERNATIONAL JOURNAL OF SYSTEMS SCIENCE<br><b>Cluster-Keywords: </b><em>regulatory networks, stochastic simulation, gene networks, reaction networks, systems biology, stochastic model, network models, complex systems, networks stochastic, gene regulatory, biological systems, stochastic dynamics, differential equations, protein networks, cell cycle, continuous model, interaction networks, systems chemistry, time delays, biochemical reaction<div style="font-size:11px;"><br><b>Abstract: </b><em>Molecular pathways are a part of a remarkable hierarchy of regulatory networks that operate at all levels of organisation. These regulatory networks are responsible for much of the biological complexity within the cell. The dynamic character of these pathways and the prevalence of feedback regulation strategies in their operation make them amenable to systematic mathematical analysis using the same tools that have been used with success in analysing and designing engineering control systems. In this article, we aim at establishing this strong connection through various examples where the behaviour exhibited by gene networks is explained in terms of their underlying control strategies. We complement our analysis by a survey of mathematical techniques commonly used to model gene regulatory networks and analyse their dynamic behaviour.</div>','<b>Title: </b>Allele-Specific Behavior of Molecular Networks: Understanding Small-Molecule Drug Response in Yeast<br><b>Year: </b>2013<br><b>Author: </b> F. Zhang,  B. Gao,  Ld. Xu,  Cq. Li,  Dp. Hao,  Sj. Zhang,  M. Zhou,  F. Su,  X. Chen,  H. Zhi and  X. Li<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0053581"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>systems biology, regulatory network, interaction networks, network model, gene regulatory, network, network biology, networks, gene network, response networks, network dynamics, dynamic model, cell networks, gene expression, biological networks, system dynamics, dynamic network, model, biological systems, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>The study of systems genetics is changing the way the genetic and molecular basis of phenotypic variation, such as disease susceptibility and drug response, is being analyzed. Moreover, systems genetics aids in the translation of insights from systems biology into genetics. The use of systems genetics enables greater attention to be focused on the potential impact of genetic perturbations on the molecular states of networks that in turn affects complex traits. In this study, we developed models to detect allele-specific perturbations on interactions, in which a genetic locus with alternative alleles exerted a differing influence on an interaction. We utilized the models to investigate the dynamic behavior of an integrated molecular network undergoing genetic perturbations in yeast. Our results revealed the complexity of regulatory relationships between genetic loci and networks, in which different genetic loci perturb specific network modules. In addition, significant within-module functional coherence was found. We then used the network perturbation model to elucidate the underlying molecular mechanisms of individual differences in response to 100 diverse small molecule drugs. As a result, we identified sub-networks in the integrated network that responded to variations in DNA associated with response to diverse compounds and were significantly enriched for known drug targets. Literature mining results provided strong independent evidence for the effectiveness of these genetic perturbing networks in the elucidation of small-molecule responses in yeast.</div>','<b>Title: </b>Assigning probabilities to qualitative dynamics of gene regulatory networks<br><b>Year: </b>2014<br><b>Author: </b> L. Ironi and  E. Lanzarone<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s00285-014-0765-z"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF MATHEMATICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>regulatory networks, network dynamics, gene regulatory, network model, genetic network, gene networks, genetic regulatory, mathematical model, parameter space, network system, interaction networks, system dynamics, network structure, network, dynamic networks, network reduction, models models, biological networks, network motifs, biological systems<div style="font-size:11px;"><br><b>Abstract: </b><em>Mathematical and computational modeling frameworks play the leading role in the analysis and prediction of the dynamics of gene regulatory networks. The literature abounds in various approaches, all of which characterized by strengths and weaknesses. Among the others, Ordinary Differential Equations ODE models give a more general and detailed description of the network structure. But, analytical computations might be prohibitive as soon as the network dimension increases, and numerical simulation could be nontrivial, time-consuming and very often impracticable as precise and quantitative information on model parameters are frequently unknown and hard to estimate from experimental data. Last but not least, they do not account for the intrinsic stochasticity of regulation. In the present paper we consider a class of ODE models with stochastic parameters. The proposed method separates the deterministic aspects from the stochastic ones. Under specific assumptions and conditions, all possible trajectories of an ODE model, where incomplete knowledge of parameter values is symbolically and qualitatively expressed by initial inequalities only, are simulated in a single run from an initial state. Then, the occurrence probability of each trajectory, characterized by a sequence of parameter inequalities, is computed by associating probability density functions with network parameters. As demonstrated by its application to the gene repressilator system, the method seems particularly promising in the design of synthetic networks.</div>','<b>Title: </b>Networking Brains: Modeling Spatial Relationships of the Cerebral Cortex<br><b>Year: </b>2018<br><b>Author: </b> E. Bruner,  B. Esteve-Altava and  D. Rasskin-Gutman<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/978-4-431-56582-613"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>DIGITAL ENDOCASTS: FROM SKULLS TO BRAINS<br><b>Cluster-Keywords: </b><em>complex networks, complex systems, network theory, network structure, biological network, social network, network science, dynamic networks, network systems, network model, network topology, systems biology, network, network analysis, system dynamics, networks, realworld networks, network automata, epidemic network, model network<div style="font-size:11px;"><br><b>Abstract: </b><em>Brain mapping has always been a priority in neurobiology and evolutionary neuroanatomy. In the last century, methodological issues and technical advances have generated a vivid debate on the parcellation and functions of the cortical territories. Brain structure is generally analyzed by considering the network of connections associated with neural pathways. Nonetheless, there is still a major debate on the recognition of the spatial and geometrical components of the cerebral cortex. The maps produced by Korbinian Brodmann in the early twentieth century on the basis of histological patterns represented a pioneering and decisive step in this sense, being a reference until the present day. Network models allow a numerical analysis of the spatial relationships among anatomical elements, supplying a quantitative tool to evaluate their reciprocal geometrical organization. This approach is able to analyze the spatial parameters associated with an anatomical system, characterized by the relationships of its elements. The network analysis of the spatial contiguity of Brodmanns areas approximately describes the major cerebral lobes. A frontal cluster includes only the prefrontal areas. There is a large parieto-occipital block including also the precentral and paracentral cortex. The cortical areas identified by the model match different areas of craniocerebral relationships, namely, the anterior fossa influenced by the upper face prefrontal cortex, the middle fossa influenced by cranial base and mandibular integration temporal cortex, and the vault which is characterized by more linear brain-bone dynamics parieto-occipital cortex. The maps of Brodmann, after one century of contributions, are now replaced by finer parcellations obtained with new technical approaches based on histology, biochemistry, and metabolism, enhanced by advances in brain imaging and digital biology. Besides issues associated with cognitive processing, structural factors can influence geometrical and mechanical properties of the cerebral morphology. Network theory, applied to alternative parcellation schemes or to specific brain districts, can provide essential information on evolutionary factors channeling or constraining the evolution of the brain spatial organization.</div>','<b>Title: </b>Laplacian Dynamics with Synthesis and Degradation<br><b>Year: </b>2015<br><b>Author: </b> I. Mirzaev and  Dm. Bortz<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s11538-015-0075-7"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BULLETIN OF MATHEMATICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>reaction networks, reaction systems, metabolic networks, chemical reaction, network structure, systems biology, biological systems, biological networks, biochemical reaction, reaction rates, differential equations, reaction graph, complex networks, ssi modules, networks, life systems, ordinary differential, cellular networks, reaction, network flows<div style="font-size:11px;"><br><b>Abstract: </b><em>Analyzing qualitative behaviors of biochemical reactions using its associated network structure has proven useful in diverse branches of biology. As an extension of our previous work, we introduce a graph-based framework to calculate steady state solutions of biochemical reaction networks with synthesis and degradation. Our approach is based on a labeled directed graph and the associated system of linear non-homogeneous differential equations with first-order degradation and zeroth-order synthesis. We also present a theorem which provides necessary and sufficient conditions for the dynamics to engender a unique stable steady state. Although the dynamics are linear, one can apply this framework to nonlinear systems by encoding nonlinearity into the edge labels. We answer an open question from our previous work concerning the non-positiveness of the elements in the inverse of a perturbed Laplacian matrix. Moreover, we provide a graph theoretical framework for the computation of the inverse of such a matrix. This also completes our previous framework and makes it purely graph theoretical. Lastly, we demonstrate the utility of this framework by applying it to a mathematical model of insulin secretion through ion channels in pancreatic -cells.</div>','<b>Title: </b>BioPreDyn-bench: a suite of benchmark problems for dynamic modelling in systems biology<br><b>Year: </b>2015<br><b>Author: </b> Af. Villaverde,  D. Henriques,  K. Smallbone,  S. Bongard,  J. Schmid,  D. Cicin-Sain,  A. Crombach,  J. Saez-Rodriguez,  K. Mauch,  E. Balsa-Canto,  P. Mendes,  J. Jaeger and  Jr. Banga<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/s12918-015-0144-4"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>kinetic models, metabolic networks, kinetic parameters, dynamic models, systems biology, network model, dynamic metabolic, biochemical network, rate constants, enzyme kinetics, kinetic data, adjustable parameters, metabolic model, parameter estimation, dynamic optimization, model selection, mathematical model, kinetic network, model response, model<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Dynamic modelling is one of the cornerstones of systems biology. Many research efforts are currently being invested in the development and exploitation of large-scale kinetic models. The associated problems of parameter estimation model calibration and optimal experimental design are particularly challenging. The community has already developed many methods and software packages which aim to facilitate these tasks. However, there is a lack of suitable benchmark problems which allow a fair and systematic evaluation and comparison of these contributions.Results: Here we present BioPreDyn-bench, a set of challenging parameter estimation problems which aspire to serve as reference test cases in this area. This set comprises six problems including medium and large-scale kinetic models of the bacterium E. coli, bakers yeast S. cerevisiae, the vinegar fly D. melanogaster, Chinese Hamster Ovary cells, and a generic signal transduction network. The level of description includes metabolism, transcription, signal transduction, and development. For each problem we provide i a basic description and formulation, ii implementations ready-to-run in several formats, iii computational results obtained with specific solvers, iv a basic analysis and interpretation.Conclusions: This suite of benchmark problems can be readily used to evaluate and compare parameter estimation methods. Further, it can also be used to build test problems for sensitivity and identifiability analysis, model reduction and optimal experimental design methods. The suite, including codes and documentation, can be freely downloaded from the BioPreDyn-bench website, https://sites.google.com/site/biopredynbenchmarks/.</div>','<b>Title: </b>Algebraic statistical model for biochemical network dynamics inference<br><b>Year: </b>2013<br><b>Author: </b> Df. Linder and  Ga. Rempala<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1166/jcsmd.2013.1032"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF COUPLED SYSTEMS AND MULTISCALE DYNAMICS<br><b>Cluster-Keywords: </b><em>reaction networks, reaction systems, metabolic networks, chemical reaction, network structure, systems biology, biological systems, biological networks, biochemical reaction, reaction rates, differential equations, reaction graph, complex networks, ssi modules, networks, life systems, ordinary differential, cellular networks, reaction, network flows<div style="font-size:11px;"><br><b>Abstract: </b><em>With modern molecular quantification methods, like, for instance, high throughput sequencing, biologists may perform multiple complex experiments and collect longitudinal data on RNA and DNA concentrations. Such data may be then used to infer cellular level interactions between the molecular entities of interest. One method which formalizes such inference is the stoichiometric algebraic statistical model SASM of Ref. 2 which allows to analyze the so-called conic or single source networks. Despite its intuitive appeal, up until now the SASM has been only heuristically studied on few simple examples. The current paper provides a more formal mathematical treatment of the SASM, expanding the original model to a wider class of reaction systems decomposable into multiple conic subnetworks. In particular, it is proved here that on such networks the SASM enjoys the so-called sparsistency property, that is, it asymptotically with the number of observed network trajectories discards the false interactions by setting their reaction rates to zero. For illustration, we apply the extended SASM to in silico data from a generic decomposable network as well as to biological data from an experimental search for a possible transcription factor for the heat shock protein 70 Hsp70 in the zebrafish retina.</div>','<b>Title: </b>The next step in systems biology: simulating the temporospatial dynamics of molecular network<br><b>Year: </b>2004<br><b>Author: </b> H. Zhu,  S. Huang and  P. Dhar<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1002/bies.10383"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOESSAYS<br><b>Cluster-Keywords: </b><em>biological networks, dynamic models, biological systems, model predictions, model parameters, systems biology, mathematical models, biological models, equation models, experimental data, molecular network, interaction networks, model errors, biological interaction, molecular interactions, model, dynamic systems, biological parameters, parameter uncertainty, growth model<div style="font-size:11px;"><br><b>Abstract: </b><em>As a result of the time- and context-dependency of gene expression, gene regulatory and signaling pathways undergo dynamic changes during development. Creating a model of the dynamics of molecular interaction networks offers enormous potential for understanding how a genome orchestrates the developmental processes of an organism. The dynamic nature of pathway topology calls for new modeling strategies that can capture transient molecular links at the runtime. The aim of this paper is to present a brief and informative, but not all-inclusive, viewpoint on the computational aspects of modeling and simulation of a non-static molecular network. C 2003 Wiley Periodicals, Inc.</div>','<b>Title: </b>From digital genetics to knowledge discovery: Perspectives in genetic network understanding<br><b>Year: </b>2010<br><b>Author: </b> G. Beslon,  Dp. Parsons,  Jm. Pena,  C. Rigotti and  Y. Sanchez-Dehesa<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3233/IDA-2010-0415"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>INTELLIGENT DATA ANALYSIS<br><b>Cluster-Keywords: </b><em>systems biology, regulatory network, interaction networks, network model, gene regulatory, network, network biology, networks, gene network, response networks, network dynamics, dynamic model, cell networks, gene expression, biological networks, system dynamics, dynamic network, model, biological systems, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>In this paper, we propose an original computational approach to assist knowledge discovery in complex biological networks. First, we present an integrated model of the evolution of regulation networks that can be used to uncover organization principles of such networks. Then, we propose to use the results of our model as a benchmark for knowledge discovery algorithms. We describe a first experiment of such benchmarking by using gene knock-out data generated from the modeled organisms.</div>','<b>Title: </b>Parameter optimization by using differential elimination: a general approach for introducing constraints into objective functions<br><b>Year: </b>2010<br><b>Author: </b> M. Nakatsui,  K. Horimoto,  M. Okamoto,  Y. Tokumoto and  J. Miyake<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-4-S2-S9"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>systems biology, regulatory network, interaction networks, network model, gene regulatory, network, network biology, networks, gene network, response networks, network dynamics, dynamic model, cell networks, gene expression, biological networks, system dynamics, dynamic network, model, biological systems, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: The investigation of network dynamics is a major issue in systems and synthetic biology. One of the essential steps in a dynamics investigation is the parameter estimation in the model that expresses biological phenomena. Indeed, various techniques for parameter optimization have been devised and implemented in both free and commercial software. While the computational time for parameter estimation has been greatly reduced, due to improvements in calculation algorithms and the advent of high performance computers, the accuracy of parameter estimation has not been addressed.Results: We propose a new approach for parameter optimization by using differential elimination, to estimate kinetic parameter values with a high degree of accuracy. First, we utilize differential elimination, which is an algebraic approach for rewriting a system of differential equations into another equivalent system, to derive the constraints between kinetic parameters from differential equations. Second, we estimate the kinetic parameters introducing these constraints into an objective function, in addition to the error function of the square difference between the measured and estimated data, in the standard parameter optimization method. To evaluate the ability of our method, we performed a simulation study by using the objective function with and without the newly developed constraints: the parameters in two models of linear and non-linear equations, under the assumption that only one molecule in each model can be measured, were estimated by using a genetic algorithm GA and particle swarm optimization PSO. As a result, the introduction of new constraints was dramatically effective: the GA and PSO with new constraints could successfully estimate the kinetic parameters in the simulated models, with a high degree of accuracy, while the conventional GA and PSO methods without them frequently failed.Conclusions: The introduction of new constraints in an objective function by using differential elimination resulted in the drastic improvement of the estimation accuracy in parameter optimization methods. The performance of our approach was illustrated by simulations of the parameter optimization for two models of linear and non-linear equations, which included unmeasured molecules, by two types of optimization techniques. As a result, our method is a promising development in parameter optimization.</div>','<b>Title: </b>Multiscale Coupling of an Agent-Based Model of Tissue Fibrosis and a Logic-Based Model of Intracellular Signaling<br><b>Year: </b>2019<br><b>Author: </b> Sm. Rikard,  Tl. Athey,  Ar. Nelson,  Slm. Christiansen,  Jj. Lee,  Jw. Holmes,  Sm. Peirce and  Jj. Saucerman<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3389/fphys.2019.01481"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>FRONTIERS IN PHYSIOLOGY<br><b>Cluster-Keywords: </b><em>inflammatory response, network model, mechanistic models, lde model, injured patients, critically injured, prediction model, tbi survivors, computational modeling, model abm, impaired astrocytes, model predicts, metabolic dynamics, significant differences, twoprocess model, agent-based model, coupled model, model, inflammatory network, computational models<div style="font-size:11px;"><br><b>Abstract: </b><em>Wound healing and fibrosis following myocardial infarction MI is a dynamic process involving many cell types, extracellular matrix ECM, and inflammatory cues. As both incidence and survival rates for MI increase, management of post-MI recovery and associated complications are an increasingly important focus. Complexity of the wound healing process and the need for improved therapeutics necessitate a better understanding of the biochemical cues that drive fibrosis. To study the progression of cardiac fibrosis across spatial and temporal scales, we developed a novel hybrid multiscale model that couples a logic-based differential equation LDE model of the fibroblast intracellular signaling network with an agent-based model ABM of multi-cellular tissue remodeling. The ABM computes information about cytokine and growth factor levels in the environment including TGF beta, TNF alpha, IL-1 beta, and IL-6, which are passed as inputs to the LDE model. The LDE model then computes the network signaling state of individual cardiac fibroblasts within the ABM. Based on the current network state, fibroblasts make decisions regarding cytokine secretion and deposition and degradation of collagen. Simulated fibroblasts respond dynamically to rapidly changing extracellular environments and contribute to spatial heterogeneity in model predicted fibrosis, which is governed by many parameters including cell density, cell migration speeds, and cytokine levels. Verification tests confirmed that predictions of the coupled model and network model alone were consistent in response to constant cytokine inputs and furthermore, a subset of coupled model predictions were validated with in vitro experiments with human cardiac fibroblasts. This multiscale framework for cardiac fibrosis will allow for systematic screening of the effects of molecular perturbations in fibroblast signaling on tissue-scale extracellular matrix composition and organization.</div>','<b>Title: </b>A non-homogeneous dynamic Bayesian network with a hidden Markov model dependency structure among the temporal data points<br><b>Year: </b>2016<br><b>Author: </b> M. Grzegorczyk<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s10994-015-5503-2"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MACHINE LEARNING<br><b>Cluster-Keywords: </b><em>petri net, petri nets, bayesian networks, biological networks, systems biology, regulatory networks, dynamic bayesian, net model, gene regulatory, network data, biological systems, biological data, network analysis, network models, biochemical networks, time series, network structure, signal transduction, net models, dbn model<div style="font-size:11px;"><br><b>Abstract: </b><em>In the topical field of systems biology there is considerable interest in learning regulatory networks, and various probabilistic machine learning methods have been proposed to this end. Popular approaches include non-homogeneous dynamicBayesian networks DBNs, which can be employed to model time-varying regulatory processes. Almost all non-homogeneous DBNs that have been proposed in the literature follow the same paradigm and relax the homogeneity assumption by complementing the standard homogeneous DBN with a multiple changepoint process. Each time series segment defined by two demarcating changepoints is associated with separate interactions, and in this way the regulatory relationships are allowed to vary over time. However, the configuration space of the data segmentations allocations that can be obtained by changepoints is restricted. A complementary paradigm is to combine DBNs with mixture models, which allow for free allocations of the data points to mixture components. But this extension of the configuration space comes with the disadvantage that the temporal order of the data points can no longer be taken into account. In this paper I present a novel non-homogeneous DBN model, which can be seen as a consensus between the free allocation mixture DBN model and the changepoint-segmented DBN model. The key idea is to assume that the underlying allocation of the temporal data points follows a Hidden Markov model HMM. The novel HMM-DBN model takes the temporal structure of the time series into account without putting a restriction onto the configuration space of the data point allocations. I define the novel HMM-DBN model and the competing models such that the regulatory network structure is kept fixed among components, while the network interaction parameters are allowed to vary, and I show how the novel HMM-DBN model can be inferred with Markov Chain Monte Carlo MCMC simulations. For the new HMM-DBNmodel I also present two new pairs of MCMC moves, which can be incorporated into the recently proposed allocation sampler for mixture models to improve convergence of the MCMC simulations. In an extensive comparative evaluation study I systematically compare the performance of the proposed HMM-DBN model with the performances of the competing DBN models in a reverse engineering context, where the objective is to learn the structure of a network from temporal network data.</div>','<b>Title: </b>A reliable simulator for dynamic flux balance analysis<br><b>Year: </b>2013<br><b>Author: </b> K. Hoffner,  Sm. Harwood and  Pi. Barton<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1002/bit.24748"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOTECHNOLOGY AND BIOENGINEERING<br><b>Cluster-Keywords: </b><em>metabolic network, metabolic model, dynamic flux, growth rate, genomescale metabolic, flux balance, balance analysis, metabolic flux, metabolic pathways, dynamic model, metabolic reactions, cellular growth, regulatory network, systems biology, optimal growth, metabolic, modeling metabolic, growth conditions, dynamic analysis, cell growth<div style="font-size:11px;"><br><b>Abstract: </b><em>Dynamic flux balance analysis DFBA provides a platform for detailed design, control and optimization of biochemical process technologies. It is a promising modeling framework that combines genome-scale metabolic network analysis with dynamic simulation of the extracellular environment. Dynamic flux balance analysis assumes that the intracellular species concentrations are in equilibrium with the extracellular environment. The resulting underdetermined stoichiometric model is solved under the assumption of a biochemical objective such as growth rate maximization. The model of the metabolism is coupled with the dynamic mass balance equations of the extracellular environment via expressions for the rates of substrate uptake and product excretion, which imposes additional constraints on the linear program LP defined by growth rate maximization of the metabolism. The linear program is embedded into the dynamic model of the bioreactor, and together with the additional constraints this provides an accurate model of the substrate consumption, product secretion, and biomass production during operation. A DFBA model consists of a system of ordinary differential equations for which the evaluation of the right-hand side requires not only function evaluations, but also the solution of one or more linear programs. The numerical tool presented here accurately and efficiently simulates large-scale dynamic flux balance models. The main advantages that this approach has over existing implementation are that the integration scheme has a variable step size, that the linear program only has to be solved when qualitative changes in the optimal flux distribution of the metabolic network occur, and that it can reliably simulate behavior near the boundary of the domain where the model is defined. This is illustrated through large-scale examples taken from the literature. Biotechnol. Bioeng. 2013; 110: 792802. c 2012 Wiley Periodicals, Inc.</div>','<b>Title: </b>Inferring causal metabolic signals that regulate the dynamic TORC1-dependent transcriptome<br><b>Year: </b>2015<br><b>Author: </b> Ap. Oliveira,  S. Dimopoulos,  Ag. Busetto,  S. Christen,  R. Dechant,  L. Falter,  Mh. Chehreghani,  S. Jozefczuk,  C. Ludwig,  F. Rudroff,  Jc. Schulz,  A. Gonzalez,  A. Soulard,  D. Stracka,  R. Aebersold,  Jm. Buhmann,  Mn. Hall,  M. Peter,  U. Sauer and  J. Stelling<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.15252/msb.20145475"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MOLECULAR SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>systems biology, regulatory network, interaction networks, network model, gene regulatory, network, network biology, networks, gene network, response networks, network dynamics, dynamic model, cell networks, gene expression, biological networks, system dynamics, dynamic network, model, biological systems, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Cells react to nutritional cues in changing environments via the integrated action of signaling, transcriptional, and metabolic networks. Mechanistic insight into signaling processes is often complicated because ubiquitous feedback loops obscure causal relationships. Consequently, the endogenous inputs of many nutrient signaling pathways remain unknown. Recent advances for system-wide experimental data generation have facilitated the quantification of signaling systems, but the integration of multi-level dynamic data remains challenging. Here, we co-designed dynamic experiments and a probabilistic, model-based method to infer causal relationships between metabolism, signaling, and gene regulation. We analyzed the dynamic regulation of nitrogen metabolism by the target of rapamycin complex 1 TORC1 pathway in budding yeast. Dynamic transcriptomic, proteomic, and metabolomic measurements along shifts in nitrogen quality yielded a consistent dataset that demonstrated extensive re-wiring of cellular networks during adaptation. Our inference method identified putative downstream targets of TORC1 and putative metabolic inputs of TORC1, including the hypothesized glutamine signal. The work provides a basis for further mechanistic studies of nitrogen metabolism and a general computational framework to study cellular processes.</div>','<b>Title: </b>MUFINS: multi-formalism interaction network simulator<br><b>Year: </b>2016<br><b>Author: </b> Hh. Wu,  A. Von Kamp,  V. Leoncikas,  W. Mori,  N. Sahin,  A. Gevorgyan,  C. Linley,  M. Grabowski,  Aa. Mannan,  N. Stoy,  Gr. Stewart,  Lt. Ward,  Djm. Lewis,  J. Sroka,  H. Matsuno,  S. Klamt,  Hv. Westerhoff,  J. Mcfadden,  Nj. Plant and  Am. Kierzek<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1038/npjsba.2016.32"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>NPJ SYSTEMS BIOLOGY AND APPLICATIONS<br><b>Cluster-Keywords: </b><em>systems biology, network models, signaling network, parameter estimation, dynamic models, biological systems, mathematical models, metabolic network, parameter sensitivities, computational model, gene circuit, circuit models, experimental data, metabolic models, model system, scalable parameter, model parameters, parameter space, biomolecular networks, model<div style="font-size:11px;"><br><b>Abstract: </b><em>Systems Biology has established numerous approaches for mechanistic modeling of molecular networks in the cell and a legacy of models. The current frontier is the integration of models expressed in different formalisms to address the multi-scale biological system organization challenge. We present MUFINS MUlti-Formalism Interaction Network Simulator software, implementing a unique set of approaches for multi-formalism simulation of interaction networks. We extend the constraint-based modeling CBM framework by incorporation of linear inhibition constraints, enabling for the first time linear modeling of networks simultaneously describing gene regulation, signaling and whole-cell metabolism at steady state. We present a use case where a logical hypergraph model of a regulatory network is expressed by linear constraints and integrated with a Genome-Scale Metabolic Network GSMN of mouse macrophage. We experimentally validate predictions, demonstrating application of our software in an iterative cycle of hypothesis generation, validation and model refinement. MUFINS incorporates an extended version of our Quasi-Steady State Petri Net approach to integrate dynamic models with CBM, which we demonstrate through a dynamic model of cortisol signaling integrated with the human Recon2 GSMN and a model of nutrient dynamics in physiological compartments. Finally, we implement a number of methods for deriving metabolic states from similar to omics data, including our new variant of the iMAT congruency approach. We compare our approach with iMAT through the analysis of 262 individual tumor transcriptomes, recovering features of metabolic reprogramming in cancer. The software provides graphics user interface with network visualization, which facilitates use by researchers who are not experienced in coding and mathematical modeling environments.</div>','<b>Title: </b>Control of multilayer biological networks and applied to target identification of complex diseases<br><b>Year: </b>2019<br><b>Author: </b> W. Zheng,  Dj. Wang and  Xf. Zou<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/s12859-019-2841-2"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>boolean network, boolean functions, regulatory networks, canalizing functions, network models, gene regulatory, boolean control, control networks, biological networks, boolean models, update functions, interaction networks, large network, regulatory functions, canalizing boolean, genetic regulatory, systems biology, driver nodes, networks bcns, biological systems<div style="font-size:11px;"><br><b>Abstract: </b><em>BackgroundNetworks have been widely used to model the structures of various biological systems. The ultimate aim of research on biological networks is to steer biological system structures to desired states by manipulating signals. Despite great advances in the linear control of single-layer networks, it has been observed that many complex biological systems have a multilayer networked structure and extremely complicated nonlinear processes.ResultIn this study, we propose a general framework for controlling nonlinear dynamical systems with multilayer networked structures by formulating the problem as a minimum union optimization problem. In particular, we offer a novel approach for identifying the minimal driver nodes that can steer a multilayered nonlinear dynamical system toward any desired dynamical attractor. Three disease-related biology multilayer networks are used to demonstrate the effectiveness of our approaches. Moreover, in the set of minimum driver nodes identified by the algorithm we proposed, we confirmed that some nodes can act as drug targets in the biological experiments. Other nodes have not been reported as drug targets; however, they are also involved in important biological processes from existing literature.ConclusionsThe proposed method could be a promising tool for determining higher drug target enrichment or more meaningful steering nodes for studying complex diseases.</div>','<b>Title: </b>The National Cancer Institute Investment in Biomechanics in Oncology Research<br><b>Year: </b>2018<br><b>Author: </b> A. Dickherber,  Sk. Hughes and  N. Zahir<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/978-3-319-95294-91"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOMECHANICS IN ONCOLOGY<br><b>Cluster-Keywords: </b><em>neural network, network model, network, neurogenesis model, genomic network, networks, saccade control, droplets networks, understanding networks, gene regulatory, model, tumor-immune networks, networks dynamical, network systems, artificial neurogenesis, regulatory networks, dynamic network, information flow, cancer cells, capillary networks<div style="font-size:11px;"><br><b>Abstract: </b><em>The qualitative description of tumors feeling stiffer than surrounding normal tissue has been long appreciated in the clinical setting. These empirical observations have been corroborated by the precise measurement and characterization of mechanical properties of cancerous tissues. Much of the advancement in our understanding of mechanics in oncology has been enabled by the development of innovative technologies designed to probe cells and tissues as well as integrative software analysis tools that facilitate biological interpretation and generation of testable hypotheses. While some mechanics in oncology research has been investigatorinitiated and supported by the National Cancer Institute NCI, several NCI programs described herein have helped to foster the growth of the burgeoning field. Programs highlighted in this chapter include Innovative Molecular Analysis Technologies IMAT, Physical Sciences-Oncology Network PSON, Tumor Microenvironment Network TMEN, Integrative Cancer Biology Program ICBP, and the Cancer Systems Biology Consortium CSBC. This chapter showcases the scientific contributions of these programs to the field of biomechanics in oncology.</div>','<b>Title: </b>Dynamics and processing in finite self-similar networks<br><b>Year: </b>2012<br><b>Author: </b> S. Dedeo and  Dc. Krakauer<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1098/rsif.2011.0840"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF THE ROYAL SOCIETY INTERFACE<br><b>Cluster-Keywords: </b><em>complex networks, complex systems, network theory, network structure, biological network, social network, network science, dynamic networks, network systems, network model, network topology, systems biology, network, network analysis, system dynamics, networks, realworld networks, network automata, epidemic network, model network<div style="font-size:11px;"><br><b>Abstract: </b><em>A common feature of biological networks is the geometrical property of self-similarity. Molecular regulatory networks through to circulatory systems, nervous systems, social systems and ecological trophic networks show self-similar connectivity at multiple scales. We analyse the relationship between topology and signalling in contrasting classes of such topologies. We find that networks differ in their ability to contain or propagate signals between arbitrary nodes in a network depending on whether they possess branching or loop-like features. Networks also differ in how they respond to noise, such that one allows for greater integration at high noise, and this performance is reversed at low noise. Surprisingly, small-world topologies, with diameters logarithmic in system size, have slower dynamical time scales, and may be less integrated more modular than networks with longer path lengths. All of these phenomena are essentially mesoscopic, vanishing in the infinite limit but producing strong effects at sizes and time scales relevant to biology.</div>','<b>Title: </b>Challenges in Building an End-to-End System for Acquisition, Management, and Integration of Diverse Data From Sensor Networks in Watersheds: Lessons From a Mountainous Community Observatory in East River, Colorado<br><b>Year: </b>2019<br><b>Author: </b> C. Varadharajan,  Da. Agarwal,  W. Brown,  M. Burrus,  Rwh. Carroll,  Ds. Christianson,  B. Dafflon,  D. Dwivedi,  Bj. Enquist,  B. Faybishenko,  A. Henderson,  M. Henderson,  Vc. Hendrix,  Ss. Hubbard,  Z. Kakalia,  A. Newman,  B. Potter,  H. Steltzer,  R. Versteeg,  Kh. Williams,  C. Wilmer and  Yx. Wu<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1109/ACCESS.2019.2957793"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IEEE ACCESS<br><b>Cluster-Keywords: </b><em>interaction network, actin filament, network model, filament network, network structures, actin network, protein interaction, actin dynamics, network, membrane protein, network dynamics, model dynamic, cell, model, network approach, protein complex, computational model, actin, adhesion dynamics, cell dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>The U.S. Department of Energys Watershed Function Scientific Focus Area SFA, centered in the East River, Colorado, generates diverse datasets including hydrological, geological, geochemical, geophysical, ecological, microbiological and remote sensing data. The project has deployed extensive field infrastructure involving hundreds of sensors that measure highly diverse phenomena e.g. stream and groundwater hydrology, water quality, soil moisture, weather across the watershed. Data from the sensor network are telemetered and automatically ingested into a queryable database. The data are subsequently quality checked, integrated with the United States Geological Surveys stream monitoring network using a custom data integration broker, and published to a portal with interactive visualizations. The resulting data products are used in a variety of scientific modeling and analytical efforts. This paper describes the SFAs end-to-end infrastructure and services that support the generation of integrated datasets from a watershed sensor network. The development and maintenance of this infrastructure, presents a suite of challenges from practical field logistics to complex data processing, which are addressed through various solutions. In particular, the SFA adopts a holistic view for data collection, assessment and integration, which dramatically improves the products generated, and enables a co-design approach wherein data collection is informed by model results and vice-versa.</div>','<b>Title: </b>Using aggregation methods to assess toxicant effects on population dynamics in spatial systems<br><b>Year: </b>2002<br><b>Author: </b> A. Chaumot,  S. Charles,  P. Flammarion,  J. Garric and  P. Auger<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1890/1051-076120020121771:UAMTAT2.0.CO;2"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ECOLOGICAL APPLICATIONS<br><b>Cluster-Keywords: </b><em>metabolic network, metabolic model, dynamic flux, growth rate, genomescale metabolic, flux balance, balance analysis, metabolic flux, metabolic pathways, dynamic model, metabolic reactions, cellular growth, regulatory network, systems biology, optimal growth, metabolic, modeling metabolic, growth conditions, dynamic analysis, cell growth<div style="font-size:11px;"><br><b>Abstract: </b><em>Risk assessment in ecotoxicology currently requires extrapolating laboratory results e.g., bioassays to populations living. in fragmented heterogeneous environments. To date, metapopulation modeling has been able to contribute little to pollutant management for environment preservation and conservation biology. This situation results from the more general issue of the spatial dimension of ecological systems. We have used aggregation methods to build an ecotoxicological population model for a river network. Our model described a brown trout population exposed to chronic cadmium pollution. Age structure, spatial distribution, and demographic and migration processes were taken into account to estimate population response to different scenarios of pollutant discharge. For this purpose a multi-region Leslie matrix model was coupled with dose-response curves. Endpoints were features of population dynamics: the asymptotic population growth rate, the stable age structure, and the reproductive values. Thus, different pollution scenarios were compared in terms of subsequent reduction in the population growth rate or in terms of stable age-structure changes.</div>','<b>Title: </b>The Effect of Insertions and Deletions on Wirings in Protein-Protein Interaction Networks: A Large-Scale Study<br><b>Year: </b>2009<br><b>Author: </b> F. Hormozdiari,  R. Salari,  M. Hsing,  A. Schonhuth,  Sk. Chan,  Sc. Sahinalp and  A. Cherkasov<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1089/cmb.2008.03TT"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF COMPUTATIONAL BIOLOGY<br><b>Cluster-Keywords: </b><em>interaction networks, proteinprotein interactions, protein interaction, network model, complex systems, systems biology, protein protein, protein complex, host network, dynamic interactions, network analysis, biological systems, networks network, cellular network, biological network, viral proteins, interactome networks, regulatory networks, immune system, systems network<div style="font-size:11px;"><br><b>Abstract: </b><em>Although insertions and deletions indels are a common type of sequence variation, their origin and their functional consequences have not yet been fully understood. It has been known that indels preferably occur in the loop regions of the affected proteins. Moreover, it has recently been demonstrated that indels are significantly more strongly correlated with functional changes than substitutions. In sum, there is substantial evidence that indels, not substitutions, are the predominant evolutionary factor when it comes to structural changes in proteins. As a consequence it comes natural to hypothesize that sizable indels can modify protein interaction interfaces, causing a gain or loss of protein-protein interactions, thereby significantly rewiring the interaction networks. In this paper, we have analyzed this relationship in a large-scale study. We have computed all paralogous protein pairs in Saccharomyces cerevisiae Yeast and Drosophila melanogaster Fruit Fly, and sorted the respective alignments according to whether they contained indels of significant lengths as per a pair Hidden Markov Model HMM-based framework of a recent study. We subsequently computed well known centrality measures for proteins that participated in indel alignments indel proteins and those that did not. We found that indel proteins indeed showed greater variation in terms of these measures. This demonstrates that indels have a significant influence when it comes to rewiring of the interaction networks due to evolution, which confirms our hypothesis. In general, this study may yield relevant insights into the functional interplay of proteins and the evolutionary dynamics behind it.</div>','<b>Title: </b>Random and systematic dilutions of synaptic connections in a neural network with a nonmonotonic response function<br><b>Year: </b>1998<br><b>Author: </b> M. Okada,  T. Fukai and  M. Shiino<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1103/PhysRevE.57.2095"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PHYSICAL REVIEW E<br><b>Cluster-Keywords: </b><em>network models, gene network, regulatory networks, boolean networks, gene expression, systems biology, neural networks, network dynamics, gene regulatory, biological networks, network structure, gap gene, gene regulation, cell cycle, network, gene interactions, signaling network, quantum systems, system dynamics, networks<div style="font-size:11px;"><br><b>Abstract: </b><em>It has been observed that the dilution of synaptic connections in neural networks has relevance to biology and applicability to engineering. From this viewpoint, the effects of synaptic dilution on the retrieval performance of an associative memory model with a nonmonotonic response function are investigated through the self-consistent signal-to-noise analysis. Compared with a fully connected neural network, for which a nonmonotonic response function is known to achieve a large enhancement of storage capacity and the occurrence of the superretrieval phase leads to an errorless memory retrieval, the nonmonotonic neural network with a random synaptic dilution undergoes a considerable decrease in storage capacity. It is shown, however, that by employing a systematic dilution technique characterized by a nonlinear learning rule, in which larger connections are retained, it is possible to significantly reverse the undesirable rapid reduction in storage capacity. It is also proved that the superretrieval phase is structurally unstable against the dilution of synapses.</div>','<b>Title: </b>Reverse engineering of logic-based differential equation models using a mixed-integer dynamic optimization approach<br><b>Year: </b>2015<br><b>Author: </b> D. Henriques,  M. Rocha,  J. Saez-Rodriguez and  Jr. Banga<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1093/bioinformatics/btv314"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>boolean network, network models, qualitative models, systems biology, mathematical modeling, boolean models, signaling networks, dynamic models, dynamic modeling, signaling dynamics, logic models, dynamic network, mathematical models, boolean modeling, signal transduction, regulatory networks, drug resistance, biological networks, network approach, network modeling<div style="font-size:11px;"><br><b>Abstract: </b><em>Motivation: Systems biology models can be used to test new hypotheses formulated on the basis of previous knowledge or new experimental data, contradictory with a previously existing model. New hypotheses often come in the shape of a set of possible regulatory mechanisms. This search is usually not limited to finding a single regulation link, but rather a combination of links subject to great uncertainty or no information about the kinetic parameters.Results: In this work, we combine a logic-based formalism, to describe all the possible regulatory structures for a given dynamic model of a pathway, with mixed-integer dynamic optimization MIDO. This framework aims to simultaneously identify the regulatory structure represented by binary parameters and the real-valued parameters that are consistent with the available experimental data, resulting in a logic-based differential equation model. The alternative to this would be to perform real-valued parameter estimation for each possible model structure, which is not tractable for models of the size presented in this work. The performance of the method presented here is illustrated with several case studies: a synthetic pathway problem of signaling regulation, a twocomponent signal transduction pathway in bacterial homeostasis, and a signaling network in liver cancer cells.</div>','<b>Title: </b>Oscillatory expression in Escherichia coli mediated by microRNAs with transcriptional and translational time delays<br><b>Year: </b>2016<br><b>Author: </b> Y. Zhang,  Hh. Liu and  J. Zhou<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1049/iet-syb.2016.0017"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IET SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>gene regulatory, systems biology, synthetic biology, regulatory network, gene network, gene expression, system model, synthetic network, network dynamics, systems theory, synthetic gene, genetic network, model predicts, dynamical systems, time delays, translational time, regulation models, oscillatory expression, network structure, biological systems<div style="font-size:11px;"><br><b>Abstract: </b><em>The primary objective of this study is to study oscillatory expression of gene regulatory network in Escherichia coli mediated by microRNAs sRNAs with transcriptional and translational time delays. Motivated by the regulation of gene expression proposed by Shimoni et al. Molecular Systems Biology, 2007, a general model of delayed gene regulatory network by sRNAs is formulated. This model can well describe many practical architectures of gene regulatory network by sRNAs, particularly when both transcriptional and translational time delays are introduced. Two functional issues on oscillatory expression of the gene regulatory network, i.e. stability and bifurcation, are investigated. Furthermore, an explicit algorithm determining the direction of Hopf bifurcation and stability of bifurcating periodic solutions is presented. It turns out that both transcriptional and translational time delays can induce gene expression in E. coli to be oscillatory even when its deterministic counterpart exhibits no oscillations. Moreover, the obtained results are in consistence with the experimental observations found in the biological literatures Elowitz MB et al.: Nature, 2000, 403, pp. 335-338; Lennart Hilbert et al.: Mol. BioSyst, 2011, 7, pp. 2599-2607, which demonstrates that both transcriptional and translational time delays play an important role in the gene regulatory process of E. coli.</div>','<b>Title: </b>Model Discrimination in Dynamic Molecular Systems: Application to Parotid De-differentiation Network<br><b>Year: </b>2013<br><b>Author: </b> J. Kim,  Jx. Li,  Sg. Venkatesh,  Ds. Darling and  Ga. Rempala<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1089/cmb.2011.0222"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF COMPUTATIONAL BIOLOGY<br><b>Cluster-Keywords: </b><em>network models, gene network, regulatory networks, boolean networks, gene expression, systems biology, neural networks, network dynamics, gene regulatory, biological networks, network structure, gap gene, gene regulation, cell cycle, network, gene interactions, signaling network, quantum systems, system dynamics, networks<div style="font-size:11px;"><br><b>Abstract: </b><em>In modern systems biology the modeling of longitudinal data, such as changes in mRNA concentrations, is often of interest. Fully parametric, ordinary differential equations ODE-based models are typically developed for the purpose, but their lack of fit in some examples indicates that more flexible Bayesian models may be beneficial, particularly when there are relatively few data points available. However, under such sparse data scenarios it is often difficult to identify the most suitable model. The process of falsifying inappropriate candidate models is called model discrimination. We propose here a formal method of discrimination between competing Bayesian mixture-type longitudinal models that is both sensitive and sufficiently flexible to account for the complex variability of the longitudinal molecular data. The ideas from the field of Bayesian analysis of computer model validation are applied, along with modern Markov Chain Monte Carlo MCMC algorithms, in order to derive an appropriate Bayes discriminant rule. We restrict attention to the two-model comparison problem and present the application of the proposed rule to the mRNA data in the de-differentiation network of three mRNA concentrations in mammalian salivary glands as well as to a large synthetic dataset derived from the model used in the recent DREAM6 competition.</div>','<b>Title: </b>Systematic component selection for gene-network refinement<br><b>Year: </b>2006<br><b>Author: </b> N. Radde,  J. Gebert and  Cv. Forst<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1093/bioinformatics/btl440"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Motivation: A quantitative description of interactions between cell components is a major challenge in Computational Biology. As a method of choice, differential equations are used for this purpose, because they provide a detailed insight into the dynamic behavior of the system. In most cases, the number of time points of experimental time series is usually too small to estimate the parameters of a model of a whole gene regulatory network based on differential equations, such that one needs to focus on subnetworks consisting of only a few components. For most approaches, the set of components of the subsystem is given in advance and only the structure has to be estimated. However, the set of components that influence the system significantly are not always known in advance, making a method desirable that determines both, the components that are included into the model and the parameters.Results: We have developed a method that uses gene expression data as well as interaction data between cell components to define a set of genes that we use for our modeling. In a subsequent step, we estimate the parameters of our model of piecewise linear differential equations and evaluate the results simulating the behavior of the system with our model.We have applied our method to the DNA repair system of Mycobacterium tuberculosis. Our analysis predicts that the gene Rv2719c plays an important role in this system.</div>','<b>Title: </b>Interacting Agents in a Network for in silico Modeling of Nature-Inspired Smart Systems<br><b>Year: </b>2007<br><b>Author: </b> Vk. Murthy and  Ev. KrishnamurthyLink: No link<br><b>Journal: </b>COMPUTATIONAL INTELLIGENCE FOR AGENT-BASED SYSTEMS<br><b>Cluster-Keywords: </b><em>systems biology, regulatory network, interaction networks, network model, gene regulatory, network, network biology, networks, gene network, response networks, network dynamics, dynamic model, cell networks, gene expression, biological networks, system dynamics, dynamic network, model, biological systems, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>An interacting multi-agent system in a network can model the evolution of a Nature-Inspired Smart System NISS exhibiting the four salient properties: i Collective, coordinated and efficient ii Self-organization and emergence iii Power law scaling or scale invariance under emergence iv Adaptive, fault tolerant and resilient against damage. We explain how these basic properties can arise among agents through random enabling, inhibiting, preferential attachment and growth of a multiagent system. The quantitative understanding of a Smart system with an arbitrary interactive topology is extremely difficult. However, for specific applications and a pre-defined static interactive topology among the agents, the quantitative parameters can be obtained through simulation to build a specific NISS. Further developments of agent technology will be of great value to model, simulate and animate, many phenomena in Systems biology - pattern formation, cellular dynamics, cell motility, growth and development biology, and can provide for improved capability in complex systems modelling. Also agents will serve as useful tools to model, design and develop biomorphic robots and neuromorphic chips.</div>','<b>Title: </b>Cyclin-CDK Complexes are Key Controllers of Capacitation-Dependent Actin Dynamics in Mammalian Spermatozoa<br><b>Year: </b>2019<br><b>Author: </b> N. Bernabo,  M. Ramal-Sanchez,  L. Valbonetti,  J. Machado-Simoes,  A. Ordinelli,  G. Capacchietti,  A. Taraschi and  B. Barboni<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3390/ijms20174236"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>INTERNATIONAL JOURNAL OF MOLECULAR SCIENCES<br><b>Cluster-Keywords: </b><em>gene regulatory, systems biology, regulatory networks, gene network, stem cell, network model, transcription factors, gene expression, cell transcriptomes, network analysis, data analysis, stem cells, network grnsight, cell transformation, weighted network, gene ontology, networks grns, graph model, graph layout, endothelial cell<div style="font-size:11px;"><br><b>Abstract: </b><em>Mammalian spermatozoa are infertile immediately after ejaculation and need to undergo a functional maturation process to acquire the competence to fertilize the female egg. During this process, called capacitation, the actin cytoskeleton dramatically changes its organization. First, actin fibers polymerize, forming a network over the anterior part of the sperm cells head, and then it rapidly depolymerizes and disappears during the exocytosis of the acrosome content the acrosome reaction AR. Here, we developed a computational model representing the actin dynamics AD process on mature spermatozoa. In particular, we represented all the molecular events known to be involved in AD as a network of nodes linked by edges the interactions. After the network enrichment, using an online resource STRING, we carried out the statistical analysis on its topology, identifying the controllers of the system and validating them in an experiment of targeted versus random attack to the network. Interestingly, among them, we found that cyclin-dependent kinase cyclin-CDK complexes are acting as stronger controllers. This finding is of great interest since it suggests the key role that cyclin-CDK complexes could play in controlling AD during sperm capacitation, leading us to propose a new and interesting non-genomic role for these molecules.</div>','<b>Title: </b>Addressing Selective Polypharmacology of Antipsychotic Drugs Targeting the Bioaminergic Receptors through Receptor Dynamic Conformational Ensembles<br><b>Year: </b>2013<br><b>Author: </b> B. Selvam,  Sl. Porter and  Ig. Tikhonova<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1021/ci400282q"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF CHEMICAL INFORMATION AND MODELING<br><b>Cluster-Keywords: </b><em>signaling network, systems biology, signal transduction, signaling pathways, receptor tyrosine, erbb signaling, erbb receptor, receptor signaling, intracellular signaling, mathematical models, cell signaling, computational model, factor receptor, receptor activation, growth factor, signaling models, networks signaling, receptor system, receptors systems, downstream signaling<div style="font-size:11px;"><br><b>Abstract: </b><em>Selective polypharmacology, where a drug acts on multiple rather than a single molecular target involved in a disease, emerges to develop a structure-based system biology approach to design drugs selectively targeting a disease-active protein network. We focus on the bioaminergic receptors that belong to the group of G-protein-coupled receptors GPCRs and represent targets for therapeutic agents against schizophrenia and depression. Among them, it has been shown that the serotonin 5-HT2A and 5-HT6 and dopamine D-2 and D-3 receptors induce a cognition-enhancing effect group 1, while the histamine H-1 and serotonin 5-HT2C receptors lead to metabolic side effects and the 5-HT2B serotonin receptor causes pulmonary hypertension group 2. Thus, the problem arises to develop an approach that allows identifying drugs targeting only the disease-active receptors, i.e. group 1. The recent release of several crystal structures of the bioaminergic receptors, involving the D-3 and H-1 receptors, provides the possibility to model the structures of all receptors and initiate a study of the structural and dynamic context of selective polypharmacology. In this work, we use molecular dynamics simulations to generate a conformational space of the receptors and subsequently characterize its binding properties applying molecular probe mapping. All-against-all comparison of the generated probe maps of the selected diverse conformations of all receptors with the Tanimoto similarity coefficient Tc enable the separation of the receptors of group 1 from group 2. The pharmacophore built based on the Tc-selected receptor conformations, using the multiple probe maps discovers structural features that can be used to design molecules selective toward the receptors of group 1. The importance of several predicted residues to ligand selectivity is supported by the available mutagenesis and ligand structure activity relationship studies. In addition, the Tc-selected conformations of the receptors for group 1 show good performance in isolation of known ligands from a random decoy. Our computational structure-based protocol to tackle selective polypharmacology of antipsychotic drugs could be applied for other diseases involving multiple drug targets, such as oncologic and infectious disorders.</div>','<b>Title: </b>State-time spectrum of signal transduction logic models<br><b>Year: </b>2012<br><b>Author: </b> A. Macnamara,  C. Terfve,  D. Henriques,  Bp. Bernabe and  J. Saez-Rodriguez<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1088/1478-3975/9/4/045003"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PHYSICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>boolean network, network models, qualitative models, systems biology, mathematical modeling, boolean models, signaling networks, dynamic models, dynamic modeling, signaling dynamics, logic models, dynamic network, mathematical models, boolean modeling, signal transduction, regulatory networks, drug resistance, biological networks, network approach, network modeling<div style="font-size:11px;"><br><b>Abstract: </b><em>Despite the current wealth of high-throughput data, our understanding of signal transduction is still incomplete. Mathematical modeling can be a tool to gain an insight into such processes. Detailed biochemical modeling provides deep understanding, but does not scale well above relatively a few proteins. In contrast, logic modeling can be used where the biochemical knowledge of the system is sparse and, because it is parameter free or, at most, uses relatively a few parameters, it scales well to large networks that can be derived by manual curation or retrieved from public databases. Here, we present an overview of logic modeling formalisms in the context of training logic models to data, and specifically the different approaches to modeling qualitative to quantitative data state and dynamics time of signal transduction. We use a toy model of signal transduction to illustrate how different logic formalisms Boolean, fuzzy logic and differential equations treat state and time. Different formalisms allow for different features of the data to be captured, at the cost of extra requirements in terms of computational power and data quality and quantity. Through this demonstration, the assumptions behind each formalism are discussed, as well as their advantages and disadvantages and possible future developments.</div>','<b>Title: </b>Inferring reaction systems from ordinary differential equations<br><b>Year: </b>2015<br><b>Author: </b> F. Fages,  S. Gay and  S. Soliman<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.tcs.2014.07.032"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>THEORETICAL COMPUTER SCIENCE<br><b>Cluster-Keywords: </b><em>reaction networks, reaction systems, metabolic networks, chemical reaction, network structure, systems biology, biological systems, biological networks, biochemical reaction, reaction rates, differential equations, reaction graph, complex networks, ssi modules, networks, life systems, ordinary differential, cellular networks, reaction, network flows<div style="font-size:11px;"><br><b>Abstract: </b><em>In Mathematical Biology, many dynamical models of biochemical reaction systems are presented with Ordinary Differential Equations ODE. Once kinetic parameter values are fixed, this simple mathematical formalism completely defines the dynamical behavior of a system of biochemical reactions and provides powerful tools for deterministic simulations, parameter sensitivity analysis, bifurcation analysis, etc. However, without requiring any information on the reaction kinetics and parameter values, various qualitative analyses can be performed using the structure of the reactions, provided the reactants, products and modifiers of each reaction are precisely defined. In order to apply these structural methods to parametric ODE models, we study a mathematical condition for expressing the consistency between the structure and the kinetics of a reaction, without restricting to Mass Action law kinetics. This condition, satisfied in particular by standard kinetic laws, entails a remarkable property of independence of the influence graph from the kinetics of the reactions. We derive from this study a heuristic algorithm which, given a system of ODEs as input, computes a system of reactions with the same ODE semantics, by inferring well-formed reactions whenever possible. We show how this strategy is capable of automatically curating the writing of ODE models in SBIVIL, and present some statistics obtained on the model repository biomodels.net. C 2014 Elsevier B.V. All rights reserved.</div>','<b>Title: </b>Novel application of multi-stimuli network inference to synovial fibroblasts of rheumatoid arthritis patients<br><b>Year: </b>2014<br><b>Author: </b> P. Kupfer,  R. Huber,  M. Weber,  S. Vlaic,  T. Haupl,  D. Koczan,  R. Guthke and  Rw. Kinne<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1755-8794-7-40"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC MEDICAL GENOMICS<br><b>Cluster-Keywords: </b><em>regulatory networks, gene regulatory, gene expression, network inference, expression data, network structure, gene networks, networks grns, regulatory genes, boolean networks, bayesian network, network reconstruction, networks gene, network dynamics, gene regulation, inference method, systems biology, time series, infer gene, network<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Network inference of gene expression data is an important challenge in systems biology. Novel algorithms may provide more detailed gene regulatory networks GRN for complex, chronic inflammatory diseases such as rheumatoid arthritis RA, in which activated synovial fibroblasts SFBs play a major role. Since the detailed mechanisms underlying this activation are still unclear, simultaneous investigation of multi-stimuli activation of SFBs offers the possibility to elucidate the regulatory effects of multiple mediators and to gain new insights into disease pathogenesis.Methods: A GRN was therefore inferred from RA-SFBs treated with 4 different stimuli IL-1 beta, TNF-alpha, TGF-beta, and PDGF-D. Data from time series microarray experiments 0, 1, 2, 4, 12 h; Affymetrix HG-U133 Plus 2.0 were batch-corrected applying ComBat, analyzed for differentially expressed genes over time with Limma, and used for the inference of a robust GRN with NetGenerator V2.0, a heuristic ordinary differential equation-based method with soft integration of prior knowledge.Results: Using all genes differentially expressed over time in RA-SFBs for any stimulus, and selecting the genes belonging to the most significant gene ontology GO term, i.e., cartilage development, a dynamic, robust, moderately complex multi-stimuli GRN was generated with 24 genes and 57 edges in total, 31 of which were gene-to-gene edges. Prior literature-based knowledge derived from Pathway Studio or manual searches was reflected in the final network by 25/57 confirmed edges 44. The model contained known network motifs crucial for dynamic cellular behavior, e.g., cross-talk among pathways, positive feed-back loops, and positive feed-forward motifs including suppression of the transcriptional repressor OSR2 by all 4 stimuli.Conclusion: A multi-stimuli GRN highly concordant with literature data was successfully generated by network inference from the gene expression of stimulated RA-SFBs. The GRN showed high reliability, since 10 predicted edges were independently validated by literature findings post network inference. The selected GO term cartilage development contained a number of differentiation markers, growth factors, and transcription factors with potential relevance for RA. Finally, the model provided new insight into the response of RA-SFBs to multiple stimuli implicated in the pathogenesis of RA, in particular to the novel potent growth factor PDGF-D.</div>','<b>Title: </b>Application of a dynamic metabolic flux algorithm during a temperature-induced lag phase<br><b>Year: </b>2017<br><b>Author: </b> D. Vercammen,  D. Telen,  P. Nimmegeers,  A. Janssens,  S. Akkermans,  En. Fernandez,  F. Logist and  J. Van Impe<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.fbp.2016.10.003"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>FOOD AND BIOPRODUCTS PROCESSING<br><b>Cluster-Keywords: </b><em>kinetic models, metabolic networks, kinetic parameters, dynamic models, systems biology, network model, dynamic metabolic, biochemical network, rate constants, enzyme kinetics, kinetic data, adjustable parameters, metabolic model, parameter estimation, dynamic optimization, model selection, mathematical model, kinetic network, model response, model<div style="font-size:11px;"><br><b>Abstract: </b><em>In predictive microbiology, the induced lag-phase is a phenomenon of specific interest, as it has a large impact on the assessment of safety and quality of food products. This lag phase has been studied mostly on a macroscopic level. However, a quest for more mechanistically based predictive models has started, for example, through the integration of a metabolic reaction network into widely used macroscopic model structures. This multi-scale modeling approach is called dynamic metabolic flux analysis dMFA. In this contribution, a recently developed algorithm for dMFA is used to estimate the metabolic fluxes in Escherichia coli K12 during an experimentally induced lag phase through a sudden shift in temperature. To study this phenomenon, controlled bioreactor experiments were performed: on the one hand at a fixed and optimal temperature for growth 37 degrees C, and on the other hand starting at 20 degrees C, with a sudden temperature shift to 37 degrees C during the exponential growth, inducing an intermediate lag phase. The evolution of biomass and metabolite concentrations was monitored during these experiments. After dMFA analysis of the gathered measurements, some interesting patterns in metabolic activity during the different growth phases are revealed. The described case study is a first practical test case to assess the capabilities of the recently developed dMFA methodology in an experimental predictive microbiology setting. C 2016 Published by Elsevier B.V. on behalf of Institution of Chemical Engineers.</div>','<b>Title: </b>Finding Common Modules in a Time-Varying Network with Application to the Drosophila Melanogaster Gene Regulation Network<br><b>Year: </b>2017<br><b>Author: </b> Jf. Zhang and  Jg. Cao<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1080/01621459.2016.1260465"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF THE AMERICAN STATISTICAL ASSOCIATION<br><b>Cluster-Keywords: </b><em>complex networks, complex systems, network theory, network structure, biological network, social network, network science, dynamic networks, network systems, network model, network topology, systems biology, network, network analysis, system dynamics, networks, realworld networks, network automata, epidemic network, model network<div style="font-size:11px;"><br><b>Abstract: </b><em>Finding functional modules in gene regulation networks is an important task in systems biology. Many methods have been proposed for finding communities in static networks; however, the application of such methods is limited due to the dynamic nature of gene regulation networks. In this article, we first propose a statistical framework for detecting common modules in the Drosophila melanogaster time-varying gene regulation network. We then develop both a significance test and a robustness test for the identified modular structure. We apply an enrichment analysis to our community findings, which reveals interesting results. Moreover, we investigate the consistency property of our proposed method under a time-varying stochastic block model framework with a temporal correlation structure. Although we focus on gene regulation networks in our work, our method is general and can be applied to other time-varying networks. Supplementary materials for this article are available online.</div>','<b>Title: </b>DSGRN: Examining the Dynamics of Families of Logical Models<br><b>Year: </b>2018<br><b>Author: </b> B. Cummins,  T. Gedeon,  S. Harker and  K. Mischaikow<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3389/fphys.2018.00549"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>FRONTIERS IN PHYSIOLOGY<br><b>Cluster-Keywords: </b><em>regulatory networks, network dynamics, gene regulatory, network model, genetic network, gene networks, genetic regulatory, mathematical model, parameter space, network system, interaction networks, system dynamics, network structure, network, dynamic networks, network reduction, models models, biological networks, network motifs, biological systems<div style="font-size:11px;"><br><b>Abstract: </b><em>We present a computational tool DSGRN for exploring the dynamics of a network by computing summaries of the dynamics of switching models compatible with the network across all parameters. The network can arise directly from a biological problem, or indirectly as the interaction graph of a Boolean model. This tool computes a finite decomposition of parameter space such that for each region, the state transition graph that describes the coarse dynamical behavior of a network is the same. Each of these parameter regions corresponds to a different logical description of the network dynamics. The comparison of dynamics across parameters with experimental data allows the rejection of parameter regimes or entire networks as viable models for representing the underlying regulatory mechanisms. This in turn allows a search through the space of perturbations of a given network for networks that robustly fit the data. These are the first steps toward discovering a network that optimally matches the observed dynamics by searching through the space of networks.</div>','<b>Title: </b>Genome Scale Modeling in Systems Biology: Algorithms and Resources<br><b>Year: </b>2014<br><b>Author: </b> A. Najafi,  G. Bidkhori,  Jh. Bozorgmehr,  I. Koch and  A. Masoudi-Nejad<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.2174/1389202915666140319002221"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>CURRENT GENOMICS<br><b>Cluster-Keywords: </b><em>petri net, petri nets, bayesian networks, biological networks, systems biology, regulatory networks, dynamic bayesian, net model, gene regulatory, network data, biological systems, biological data, network analysis, network models, biochemical networks, time series, network structure, signal transduction, net models, dbn model<div style="font-size:11px;"><br><b>Abstract: </b><em>In recent years, in silico studies and trial simulations have complemented experimental procedures. A model is a description of a system, and a system is any collection of interrelated objects; an object, moreover, is some elemental unit upon which observations can be made but whose internal structure either does not exist or is ignored. Therefore, any network analysis approach is critical for successful quantitative modeling of biological systems. This review highlights some of most popular and important modeling algorithms, tools, and emerging standards for representing, simulating and analyzing cellular networks in five sections. Also, we try to show these concepts by means of simple example and proper images and graphs. Overall, systems biology aims for a holistic description and understanding of biological processes by an integration of analytical experimental approaches along with synthetic computational models. In fact, biological networks have been developed as a platform for integrating information from high to low-throughput experiments for the analysis of biological systems. We provide an overview of all processes used in modeling and simulating biological networks in such a way that they can become easily understandable for researchers with both biological and mathematical backgrounds. Consequently, given the complexity of generated experimental data and cellular networks, it is no surprise that researchers have turned to computer simulation and the development of more theory-based approaches to augment and assist in the development of a fully quantitative understanding of cellular dynamics.</div>','<b>Title: </b>Kriging-Based Parameter Estimation Algorithm for Metabolic Networks Combined with Single-Dimensional Optimization and Dynamic Coordinate Perturbation<br><b>Year: </b>2016<br><b>Author: </b> H. Wang,  Xc. Wang,  Z. Li and  Kq. Li<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1109/TCBB.2015.2505291"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IEEE-ACM TRANSACTIONS ON COMPUTATIONAL BIOLOGY AND BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>systems biology, network models, signaling network, parameter estimation, dynamic models, biological systems, mathematical models, metabolic network, parameter sensitivities, computational model, gene circuit, circuit models, experimental data, metabolic models, model system, scalable parameter, model parameters, parameter space, biomolecular networks, model<div style="font-size:11px;"><br><b>Abstract: </b><em>The metabolic network model allows for an in-depth insight into the molecular mechanism of a particular organism. Because most parameters of the metabolic network cannot be directly measured, they must be estimated by using optimization algorithms. However, three characteristics of the metabolic network model, i.e., high nonlinearity, large amount parameters, and huge variation scopes of parameters, restrict the application of many traditional optimization algorithms. As a result, there is a growing demand to develop efficient optimization approaches to address this complex problem. In this paper, a Kriging-based algorithm aiming at parameter estimation is presented for constructing the metabolic networks. In the algorithm, a new infill sampling criterion, named expected improvement and mutual information EIMI, is adopted to improve the modeling accuracy by selecting multiple new sample points at each cycle, and the domain decomposition strategy based on the principal component analysis is introduced to save computing time. Meanwhile, the convergence speed is accelerated by combining a single-dimensional optimization method with the dynamic coordinate perturbation strategy when determining the new sample points. Finally, the algorithm is applied to the arachidonic acid metabolic network to estimate its parameters. The obtained results demonstrate the effectiveness of the proposed algorithm in getting precise parameter values under a limited number of iterations.</div>','<b>Title: </b>An improved constraint filtering technique for inferring hidden states and parameters of a biological model<br><b>Year: </b>2013<br><b>Author: </b> Sm. Baker,  Ch. Poskar,  F. Schreiber and  Bh. Junker<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1093/bioinformatics/btt097"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Motivation: In systems biology, kinetic models represent the biological system using a set of ordinary differential equations ODEs. The correct values of the parameters within these ODEs are critical for a reliable study of the dynamic behaviour of such systems. Typically, it is only possible to experimentally measure a fraction of these parameter values. The rest must be indirectly determined from measurements of other quantities. In this article, we propose a novel statistical inference technique to computationally estimate these unknown parameter values. By characterizing the ODEs with non-linear state-space equations, this inference technique models the unknown parameters as hidden states, which can then be estimated from noisy measurement data.Results: Here we extended the square-root unscented Kalman filter SR-UKF proposed by Merwe and Wan to include constraints with the state estimation process. We developed the constrained square-root unscented Kalman filter CSUKF to estimate parameters of non-linear state-space models. This probabilistic inference technique was successfully used to estimate parameters of a glycolysis model in yeast and a gene regulatory network. We showed that our method is numerically stable and can reliably estimate parameters within a biologically meaningful parameter space from noisy observations. When compared with the two common non-linear extensions of Kalman filter in addition to four widely used global optimization algorithms, CSUKF is shown to be both accurate and computationally efficient. With CSUKF, statistical analysis is straightforward, as it directly provides the uncertainty on the estimation result.</div>','<b>Title: </b>Integrated analysis of metabolic phenotypes in Saccharomyces cerevisiae<br><b>Year: </b>2004<br><b>Author: </b> Nc. Duarte,  Bo. Palsson and  Pc. Fu<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1471-2164-5-63"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC GENOMICS<br><b>Cluster-Keywords: </b><em>metabolic network, metabolic model, dynamic flux, growth rate, genomescale metabolic, flux balance, balance analysis, metabolic flux, metabolic pathways, dynamic model, metabolic reactions, cellular growth, regulatory network, systems biology, optimal growth, metabolic, modeling metabolic, growth conditions, dynamic analysis, cell growth<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: The yeast Saccharomyces cerevisiae is an important microorganism for both industrial processes and scientific research. Consequently, there have been extensive efforts to characterize its cellular processes. In order to fully understand the relationship between yeasts genome and its physiology, the stockpiles of diverse biological data sets that describe its cellular components and phenotypic behavior must be integrated at the genome-scale. Genome-scale metabolic networks have been reconstructed for several microorganisms, including S. cerevisiae, and the properties of these networks have been successfully analyzed using a variety of constraint-based methods. Phenotypic phase plane analysis is a constraint-based method which provides a global view of how optimal growth rates are affected by changes in two environmental variables such as a carbon and an oxygen uptake rate. Some applications of phenotypic phase plane analysis include the study of optimal growth rates and of network capacity and function.Results: In this study, the Saccharomyces cerevisiae genome-scale metabolic network was used to formulate a phenotypic phase plane that displays the maximum allowable growth rate and distinct patterns of metabolic pathway utilization for all combinations of glucose and oxygen uptake rates. In silico predictions of growth rate and secretion rates and in vivo data for three separate growth conditions aerobic glucose-limited, oxidative-fermentative, and microaerobic were concordant.Conclusions: Taken together, this study examines the function and capacity of yeasts metabolic machinery and shows that the phenotypic phase plane can be used to accurately predict metabolic phenotypes and to interpret experimental data in the context of a genome-scale model.</div>','<b>Title: </b>A Dynamic Interface for Capsaicinoid Systems Biology<br><b>Year: </b>2009<br><b>Author: </b> M. Mazourek,  A. Pujar,  Y. Borovsky,  I. Paran,  L. Mueller and  Mm. Jahn<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1104/pp.109.136549"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLANT PHYSIOLOGY<br><b>Cluster-Keywords: </b><em>gene expression, systems biology, metabolic network, network model, network analysis, dynamic model, signaling pathways, signaling network, regulatory network, dynamic network, biology approach, candidate genes, gene network, gene regulatory, biology analysis, mathematical model, network dynamic, biology network, metabolic pathways, pathway analysis<div style="font-size:11px;"><br><b>Abstract: </b><em>Capsaicinoids are the pungent alkaloids that give hot peppers Capsicum spp. their spiciness. While capsaicinoids are relatively simple molecules, much is unknown about their biosynthesis, which spans diverse metabolisms of essential amino acids, phenylpropanoids, benzenoids, and fatty acids. Pepper is not a model organism, but it has access to the resources developed in model plants through comparative approaches. To aid research in this system, we have implemented a comprehensive model of capsaicinoid biosynthesis and made it publicly available within the SolCyc database at the SOL Genomics Network http://www.sgn.cornell.edu. As a preliminary test of this model, and to build its value as a resource, targeted transcripts were cloned as candidates for nearly all of the structural genes for capsaicinoid biosynthesis. In support of the role of these transcripts in capsaicinoid biosynthesis beyond correct spatial and temporal expression, their predicted subcellular localizations were compared against the biosynthetic model and experimentally determined compartmentalization in Arabidopsis Arabidopsis thaliana. To enable their use in a positional candidate gene approach in the Solanaceae, these genes were genetically mapped in pepper. These data were integrated into the SOL Genomics Network, a clade-oriented database that incorporates community annotation of genes, enzymes, phenotypes, mutants, and genomic loci. Here, we describe the creation and integration of these resources as a holistic and dynamic model of the characteristic specialized metabolism of pepper.</div>','<b>Title: </b>Topological sensitivity analysis for systems biology<br><b>Year: </b>2014<br><b>Author: </b> Ac. Babtie,  P. Kirk and  Mph. Stumpf<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1073/pnas.1414026112"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA<br><b>Cluster-Keywords: </b><em>biological networks, dynamic models, biological systems, model predictions, model parameters, systems biology, mathematical models, biological models, equation models, experimental data, molecular network, interaction networks, model errors, biological interaction, molecular interactions, model, dynamic systems, biological parameters, parameter uncertainty, growth model<div style="font-size:11px;"><br><b>Abstract: </b><em>Mathematical models of natural systems are abstractions of much more complicated processes. Developing informative and realistic models of such systems typically involves suitable statistical inference methods, domain expertise, and a modicum of luck. Except for cases where physical principles provide sufficient guidance, it will also be generally possible to come up with a large number of potential models that are compatible with a given natural system and any finite amount of data generated from experiments on that system. Here we develop a computational framework to systematically evaluate potentially vast sets of candidate differential equation models in light of experimental and prior knowledge about biological systems. This topological sensitivity analysis enables us to evaluate quantitatively the dependence of model inferences and predictions on the assumed model structures. Failure to consider the impact of structural uncertainty introduces biases into the analysis and potentially gives rise to misleading conclusions.</div>','<b>Title: </b>From systems biology to P4 medicine: applications in respiratory medicine<br><b>Year: </b>2018<br><b>Author: </b> G. Noell,  R. Faner and  A. Agusti<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1183/16000617.0110-2017"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>EUROPEAN RESPIRATORY REVIEW<br><b>Cluster-Keywords: </b><em>systems biology, regulatory network, interaction networks, network model, gene regulatory, network, network biology, networks, gene network, response networks, network dynamics, dynamic model, cell networks, gene expression, biological networks, system dynamics, dynamic network, model, biological systems, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Human health and disease are emergent properties of a complex, nonlinear, dynamic multilevel biological system: the human body. Systems biology is a comprehensive research strategy that has the potential to understand these emergent properties holistically. It stems from advancements in medical diagnostics, "omics" data and bioinformatic computing power. It paves the way forward towards "P4 medicine" predictive, preventive, personalised and participatory, which seeks to better intervene preventively to preserve health or therapeutically to cure diseases. In this review, we: 1 discuss the principles of systems biology; 2 elaborate on how P4 medicine has the potential to shift healthcare from reactive medicine treatment of illness to predict and prevent illness, in a revolution that will be personalised in nature, probabilistic in essence and participatory driven; 3 review the current state of the art of network systems medicine in three prevalent respiratory diseases chronic obstructive pulmonary disease, asthma and lung cancer; and 4 outline current challenges and future goals in the field.</div>','<b>Title: </b>Analysis for gene networks based on logic relationships<br><b>Year: </b>2010<br><b>Author: </b> Sd. Wang,  Y. Chen,  Qy. Wang,  Ey. Li,  Ys. Su and  Dz. Meng<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s11424-010-0205-0"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF SYSTEMS SCIENCE  COMPLEXITY<br><b>Cluster-Keywords: </b><em>coupling strength, synchronization transitions, timeperiodic coupling, time delay, logic network, distribution network, network model, oscillatory networks, neuron networks, water distribution, stochastic resonance, gene networks, multiple synchronization, firing synchronization, neuronal networks, firing behaviors, logical networks, results show, regulatory networks, cell polarization<div style="font-size:11px;"><br><b>Abstract: </b><em>The reverse construction and analysis of the networks of molecular interactions are essential for understanding their functions within cells. In this paper, a logic network model is constructed to investigate the complicated regulation mechanism of shoot genes of Arabidopsis Thaliana in response to stimuli. The dynamics of the complicated logic network is analyzed, discussed, and simulated. The simulation results show that the logic network of the active genes of shoot eventually evolves into eleven attractors under the stimuli, including five 1-periodic and six 2-periodic attractors. Our work provides valuable reference and guidance for biologists to understand and explain Arabidopsis response to external stimuli by experiments.</div>','<b>Title: </b>On the predictability of infectious disease outbreaks<br><b>Year: </b>2019<br><b>Author: </b> Sv. Scarpino and  G. Petri<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1038/s41467-019-08616-0"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>NATURE COMMUNICATIONS<br><b>Cluster-Keywords: </b><em>complex networks, complex systems, network theory, network structure, biological network, social network, network science, dynamic networks, network systems, network model, network topology, systems biology, network, network analysis, system dynamics, networks, realworld networks, network automata, epidemic network, model network<div style="font-size:11px;"><br><b>Abstract: </b><em>Infectious disease outbreaks recapitulate biology: they emerge from the multi-level interaction of hosts, pathogens, and environment. Therefore, outbreak forecasting requires an integrative approach to modeling. While specific components of outbreaks are predictable, it remains unclear whether fundamental limits to outbreak prediction exist. Here, adopting permutation entropy as a model independent measure of predictability, we study the predictability of a diverse collection of outbreaks and identify a fundamental entropy barrier for disease time series forecasting. However, this barrier is often beyond the time scale of single outbreaks, implying prediction is likely to succeed. We show that forecast horizons vary by disease and that both shifting model structures and social network heterogeneity are likely mechanisms for differences in predictability. Our results highlight the importance of embracing dynamic modeling approaches, suggest challenges for performing model selection across long time series, and may relate more broadly to the predictability of complex adaptive systems.</div>','<b>Title: </b>PetriScape A plugin for discrete Petri net simulations in Cytoscape<br><b>Year: </b>2016<br><b>Author: </b> D. Almeida,  V. Azevedo,  A. Silva and  J. Baumbach<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.2390/biecoll-jib-2016-284"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF INTEGRATIVE BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>petri net, petri nets, bayesian networks, biological networks, systems biology, regulatory networks, dynamic bayesian, net model, gene regulatory, network data, biological systems, biological data, network analysis, network models, biochemical networks, time series, network structure, signal transduction, net models, dbn model<div style="font-size:11px;"><br><b>Abstract: </b><em>Systems biology plays a central role for biological network analysis in the post-genomic era. Cytoscape is the standard bioinformatics tool offering the community an extensible platform for computational analysis of the emerging cellular network together with experimental omics data sets. However, only few apps/plugins/tools are available for simulating network dynamics in Cytoscape 3. Many approaches of varying complexity exist but none of them have been integrated into Cytoscape as app/plugin yet. Here, we introduce PetriScape, the first Petri net simulator for Cytoscape. Although discrete Petri nets are quite simplistic models, they are capable of modeling global network properties and simulating their behaviour. In addition, they are easily understood and well visualizable. PetriScape comes with the following main functionalities: 1 import of biological networks in SBML format, 2 conversion into a Petri net, 3 visualization as Petri net, and 4 simulation and visualization of the token flow in Cytoscape. PetriScape is the first Cytoscape plugin for Petri nets. It allows a straightforward Petri net model creation, simulation and visualization with Cytoscape, providing clues about the activity of key components in biological networks.</div>','<b>Title: </b>Spatio-temporal filtering properties of a dendritic cable with active spines: A modeling study in the spike-diffuse-spike framework<br><b>Year: </b>2006<br><b>Author: </b> Y. Timofeeva,  Gj. Lord and  S. Coombes<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s10827-006-8776-4"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF COMPUTATIONAL NEUROSCIENCE<br><b>Cluster-Keywords: </b><em>oscillator networks, network dynamics, network, networks, dynamical systems, dynamics, node dynamics, systems, coupled oscillator, synchronization, dynamics network, complex network, coupled system, network modifications, oscillators, network connectivity, feedback vertex, network topology, synchronization properties, collective dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>The spike-diffuse-spike SDS model describes a passive dendritic tree with active dendritic spines. Spine-head dynamics is modeled with a simple integrate-and-fire process, whilst communication between spines is mediated by the cable equation. In this paper we develop a computational framework that allows the study of multiple spiking events in a network of such spines embedded on a simple one-dimensional cable. In the first instance this system is shown to support saltatory waves with the same qualitative features as those observed in a model with Hodgkin-Huxley kinetics in the spine-head. Moreover, there is excellent agreement with the analytically calculated speed for a solitary saltatory pulse. Upon driving the system with time-varying external input we find that the distribution of spines can play a crucial role in determining spatio-temporal filtering properties. In particular, the SDS model in response to periodic pulse train shows a positive correlation between spine density and low-pass temporal filtering that is consistent with the experimental results of Rose and Fortune 1999, Mechanisms for generating temporal filters in the electrosensory system, The Journal of Experimental Biology 202: 1281-1289. Further, we demonstrate the robustness of observed wave properties to natural sources of noise that arise both in the cable and the spine-head, and highlight the possibility of purely noise induced waves and coherent oscillations.</div>','<b>Title: </b>Modeling genetic regulatory networks: Continuous or discrete<br><b>Year: </b>2006<br><b>Author: </b> I. Ivanov and  Er. Dougherty<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1142/S0218339006001763"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF BIOLOGICAL SYSTEMS<br><b>Cluster-Keywords: </b><em>boolean networks, regulatory networks, genetic regulatory, gene regulatory, probabilistic boolean, network, networks, model, network model, probabilistic model, model checking, networks pbns, network dynamics, markov chain, regulatory, models, optimal control, boolean, networks models, networks boolean<div style="font-size:11px;"><br><b>Abstract: </b><em>Selecting an appropriate mathematical model to describe the dynamical behavior of a genetic regulatory network plays an important part in discovering gene regulatory mechanisms. Whereas fine-scale models can in principle provide a very accurate description of the real genetic regulatory system, one must be aware of the availability and quality of the data used to infer such models. Consequently, pragmatic considerations motivate the selection of a model possessing minimal complexity among those capable of capturing the level of real gene regulation being studied, particularly in relation to the prediction capability of the model. This paper compares fine-scale stochastic-differential-equation models with coarse-scale discrete models in the context of currently available data and with respect to their description of switch-like behavior among specific groups of genes.</div>','<b>Title: </b>Mathematical Modeling of the Circadian Rhythm of Key Neuroendocrine-Immune System Players in Rheumatoid Arthritis A Systems Biology Approach<br><b>Year: </b>2009<br><b>Author: </b> M. Meyer-Hermann,  Mt. Figge and  Rh. Straub<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1002/art.24797"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ARTHRITIS AND RHEUMATISM<br><b>Cluster-Keywords: </b><em>circadian rhythms, systems biology, biological clock, circadian clock, biological rhythms, hpa axis, model, clock network, mathematical models, biological systems, dynamic systems, ode models, biology models, single cell, network, model parameters, experimental data, models, systems, dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>Objective. Healthy subjects and patients with rheumatoid arthritis RA exhibit circadian rhythms of the neuroendocrine-immune system. Understanding circadian dynamics is complex due to the nonlinear behavior of the neuroendocrine-immune network. This study was undertaken to seek and test a mathematical model for studying this network.Methods. We established a quantitative computational model to simulate nonlinear interactions between key factors in the neuroendocrine-immune system, such as plasma tumor necrosis factor TNF, plasma cortisol and adrenal cholesterol store, and plasma noradrenaline NA and presynaptic NA store.Results. The model was nicely fitted with measured reference data on healthy subjects and RA patients. Although the individual circadian pacemakers of cortisol, NA, and TNF were installed without a phase shift, the relative phase shift between these factors evolved as a consequence of the modeled network interactions. Combined long-term and short-term TNF increase the "RA model" increased cortisol plasma levels for only a few days, and cholesterol stores started to become markedly depleted. This nicely demonstrated the phenomenon of inadequate cortisol secretion relative to plasma TNF levels, as a consequence of adrenal deficiency. Using the RA model, treatment with glucocorticoids between midnight and 2:00 AM was found to have the strongest inhibitory effect on TNF secretion, which supports recent studies on RA therapy. Longterm reduction of TNF levels by simulation of anti-TNF therapy normalized cholesterol stores under "RA" conditions.Conclusion. These first in silico studies of the neuroendocrine-immune system in rheumatology demonstrate that computational biology in medicine, making use of large collections of experimental data, supports understanding of the pathophysiology of complex nonlinear systems.</div>','<b>Title: </b>Block-tridiagonal state-space realization of Chemical Master Equations: A tool to compute explicit solutions<br><b>Year: </b>2016<br><b>Author: </b> A. Borri,  F. Carravetta,  G. Mavelli and  P. Palumbo<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.cam.2015.10.008"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF COMPUTATIONAL AND APPLIED MATHEMATICS<br><b>Cluster-Keywords: </b><em>regulatory networks, stochastic simulation, gene networks, reaction networks, systems biology, stochastic model, network models, complex systems, networks stochastic, gene regulatory, biological systems, stochastic dynamics, differential equations, protein networks, cell cycle, continuous model, interaction networks, systems chemistry, time delays, biochemical reaction<div style="font-size:11px;"><br><b>Abstract: </b><em>Chemical Master Equations CMEs provide a comprehensive way to model the probabilistic behavior in biochemical networks. Despite their widespread diffusion in systems biology, the explicit computation of their solution is often avoided in favor of purely statistic Monte Carlo methods, due to the dramatically high dimension of the CME system.In this work, we investigate some structural properties of CMEs and their solutions, focusing on the efficient computation of the stationary distribution. We introduce a generalized notion of one-step process, which results in a sparse dynamic matrix describing the collection of the scalar CMEs, showing a recursive block-tridiagonal structure as well. Further properties are inferred by means of a graph-theoretical interpretation of the reaction network. We exploit this structure by proposing different methods, including a dedicated LU decomposition, to compute the explicit solution.Examples are included to illustrate the introduced concepts and to show the effectiveness of the proposed approach. C 2015 Elsevier B.V. All rights reserved.</div>','<b>Title: </b>Cellular polarization: Interaction between extrinsic bounded noises and the wave-pinning mechanism<br><b>Year: </b>2013<br><b>Author: </b> S. De Franciscis and  A. DOnofrio<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1103/PhysRevE.88.032709"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PHYSICAL REVIEW E<br><b>Cluster-Keywords: </b><em>coupling strength, synchronization transitions, timeperiodic coupling, time delay, logic network, distribution network, network model, oscillatory networks, neuron networks, water distribution, stochastic resonance, gene networks, multiple synchronization, firing synchronization, neuronal networks, firing behaviors, logical networks, results show, regulatory networks, cell polarization<div style="font-size:11px;"><br><b>Abstract: </b><em>Cell polarization cued or uncued is a fundamental mechanism in cell biology. As an alternative to the classical Turing bifurcation, it has been proposed that the onset of cell polarity might arise by means of the well-known phenomenon of wave-pinning Gamba et al., Proc. Natl. Acad. Sci. USA 102, 16927 2005. A particularly simple and elegant deterministic model of cell polarization based on the wave-pinning mechanism has been proposed by Edelstein-Keshet and coworkers Biophys. J. 94, 3684 2008. This model consists of a small biomolecular network where an active membrane-bound factor interconverts into its inactive form that freely diffuses in the cell cytosol. However, biomolecular networks do communicate with other networks as well as with the external world. Thus, their dynamics must be considered as perturbed by extrinsic noises. These noises may have both a spatial and a temporal correlation, and in any case they must be bounded to preserve the biological meaningfulness of the perturbed parameters. Here we numerically show that the inclusion of external spatiotemporal bounded parametric perturbations in the above wave-pinning-based model of cellular polarization may sometimes destroy the polarized state. The polarization loss depends on both the extent of temporal and spatial correlations and on the kind of noise employed. For example, an increase of the spatial correlation of the noise induces an increase of the probability of cell polarization. However, if the noise is spatially homogeneous then the polarization is lost in the majority of cases. These phenomena are independent of the type of noise. Conversely, an increase of the temporal autocorrelation of the noise induces an effect that depends on the model of noise.</div>','<b>Title: </b>Top-Down Network Analysis to Drive Bottom-Up Modeling of Physiological Processes<br><b>Year: </b>2013<br><b>Author: </b> Cl. Poirel,  Rr. Rodrigues,  Kc. Chen,  Jj. Tyson and  Tm. Murali<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1089/cmb.2012.0274"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF COMPUTATIONAL BIOLOGY<br><b>Cluster-Keywords: </b><em>systems biology, regulatory network, interaction networks, network model, gene regulatory, network, network biology, networks, gene network, response networks, network dynamics, dynamic model, cell networks, gene expression, biological networks, system dynamics, dynamic network, model, biological systems, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Top-down analyses in systems biology can automatically find correlations among genes and proteins in large-scale datasets. However, it is often difficult to design experiments from these results. In contrast, bottom-up approaches painstakingly craft detailed models that can be simulated computationally to suggest wet lab experiments. However, developing the models is a manual process that can take many years. These approaches have largely been developed independently.We present LINKER, an efficient and automated data-driven method that can analyze molecular interactomes to propose extensions to models that can be simulated. LINKER combines teleporting random walks and k-shortest path computations to discover connections from a source protein to a set of proteins collectively involved in a particular cellular process.We evaluate the efficacy of LINKER by applying it to a well-known dynamic model of the cell division cycle in Saccharomyces cerevisiae. Compared to other state-of-the-art methods, sub-networks computed by LINKER are heavily enriched in Gene Ontology GO terms relevant to the cell cycle. Finally, we highlight how networks computed by LINKER elucidate the role of a protein kinase Cdc5 in the mitotic exit network of a dynamic model of the cell cycle.</div>','<b>Title: </b>Synchronization and Redundancy: Implications for Robustness of Neural Learning and Decision Making<br><b>Year: </b>2011<br><b>Author: </b> J. Bouvrie and  Jj. Slotine<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1162/NECOa00183"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>NEURAL COMPUTATION<br><b>Cluster-Keywords: </b><em>oscillator networks, network dynamics, network, networks, dynamical systems, dynamics, node dynamics, systems, coupled oscillator, synchronization, dynamics network, complex network, coupled system, network modifications, oscillators, network connectivity, feedback vertex, network topology, synchronization properties, collective dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>Learning and decision making in the brain are key processes critical to survival, and yet are processes implemented by nonideal biological building blocks that can impose significant error. We explore quantitatively how the brain might cope with this inherent source of error by taking advantage of two ubiquitous mechanisms, redundancy and synchronization. In particular we consider a neural process whose goal is to learn a decision function by implementing a nonlinear gradient dynamics. The dynamics, however, are assumed to be corrupted by perturbations modeling the error, which might be incurred due to limitations of the biology, intrinsic neuronal noise, and imperfect measurements. We show that error, and the associated uncertainty surrounding a learned solution, can be controlled in large part by trading off synchronization strength among multiple redundant neural systems against the noise amplitude. The impact of the coupling between such redundant systems is quantified by the spectrum of the network Laplacian, and we discuss the role of network topology in synchronization and in reducing the effect of noise. We discuss range of situations in which the mechanisms we model arise in brain science and draw attention to experimental evidence suggesting that cortical circuits capable of implementing the computations of interest here can be found on several scales. Finally, simulations comparing theoretical bounds to the relevant empirical quantities show that the theoretical estimates we derive can be tight.</div>','<b>Title: </b>Model your genes the mathematical way-a mathematical biology workshop for secondary school teachers<br><b>Year: </b>2008<br><b>Author: </b> Am. Martins,  P. Vera-Licona and  R. Laubenbacher<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1093/teamat/hrn003"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>TEACHING MATHEMATICS AND ITS APPLICATIONS<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>This article describes a mathematical biology workshop given to secondary school teachers of the Danville area in Virginia, USA. The goal of the workshop was to enable teams of teachers with biology and mathematics expertise to incorporate lesson plans in mathematical modelling into the curriculum. The biological focus of the activities is the lactose operon in Escherichia coli, one of the first known intracellular regulatory networks. The modelling approach utilizes Boolean networks and tools from discrete mathematics for model simulation and analysis. The workshop structure simulated the team science approach common in todays practice in computational molecular biology and thus represents a social case study in collaborative research. The workshop provided all the necessary background in molecular biology and discrete mathematics required to complete the project. The activities developed in the workshop show students the value of mathematical modelling in understanding biochemical network mechanisms and dynamics. The use of Boolean networks, rather than the more common systems of differential equations, makes the material accessible to students with a minimal mathematical background.High school students can be exposed to the excitement of mathematical biology from both the biological and mathematical point of view. Through the development of instructional modules, high school biology and mathematics courses can be joined without having to restructure the curriculum for either subject. The relevance of an early introduction to mathematical biology allows students not only to learn curriculum material in a innovative setting, but also creates an awareness of new educational and career opportunities that are arising from the interconnections between biological and mathematical sciences.</div>','<b>Title: </b>An applied mathematicians perspective on Rosennean Complexity<br><b>Year: </b>2018<br><b>Author: </b> I. Siekmann<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.ecocom.2017.07.007"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ECOLOGICAL COMPLEXITY<br><b>Cluster-Keywords: </b><em>information systems, gene expression, gene network, systems theory, system models, systems biology, systems information, network model, genetic network, mathematical models, dynamic systems, dynamic network, systemic model, network topology, information mechanisms, complex systems, systems complex, relational models, models main, information model<div style="font-size:11px;"><br><b>Abstract: </b><em>The theoretical biologist Robert Rosen developed a highly original approach for investigating the question "What is life", the most fundamental problem of biology. Considering that Rosen made extensive use of mathematics it might seem surprising that his ideas have only rarely been implemented in mathematical models. On the one hand, Rosen propagates relational models that neglect underlying structural details of the components and focus on relationships between the elements of a biological system, according to the motto "throw away the physics, keep the organisation". Rosens strong rejection of mechanistic models that he implicitly associates with a strong form of reductionism might have deterred mathematical modellers from adopting his ideas for their own work. On the other hand Rosens presentation of his modelling framework, M, R systems, is highly abstract which makes it hard to appreciate how this approach could be applied to concrete biological problems. In this article, both the mathematics as well as those aspects of Rosens work are analysed that relate to his philosophical ideas. It is shown that Rosens relational models are a particular type of mechanistic model with specific underlying assumptions rather than a fundamentally different approach that excludes mechanistic models. The strengths and weaknesses of relational models are investigated by comparison with current network biology literature. Finally, it is argued that Rosens definition of life, "organisms are closed to efficient causation", should be considered as a hypothesis to be tested and ideas how this postulate could be implemented in mathematical models are presented. C 2017 Elsevier B.V. All rights reserved.</div>','<b>Title: </b>Hidden Markov induced Dynamic Bayesian Network for recovering time evolving gene regulatory networks<br><b>Year: </b>2015<br><b>Author: </b> Sj. Zhu and  Yd. Wang<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1038/srep17841"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SCIENTIFIC REPORTS<br><b>Cluster-Keywords: </b><em>petri net, petri nets, bayesian networks, biological networks, systems biology, regulatory networks, dynamic bayesian, net model, gene regulatory, network data, biological systems, biological data, network analysis, network models, biochemical networks, time series, network structure, signal transduction, net models, dbn model<div style="font-size:11px;"><br><b>Abstract: </b><em>Dynamic Bayesian Networks DBN have been widely used to recover gene regulatory relationships from time-series data in computational systems biology. Its standard assumption is stationarity, and therefore, several research efforts have been recently proposed to relax this restriction. However, those methods suffer from three challenges: long running time, low accuracy and reliance on parameter settings. To address these problems, we propose a novel non-stationary DBN model by extending each hidden node of Hidden Markov Model into a DBN called HMDBN, which properly handles the underlying time-evolving networks. Correspondingly, an improved structural EM algorithm is proposed to learn the HMDBN. It dramatically reduces searching space, thereby substantially improving computational efficiency. Additionally, we derived a novel generalized Bayesian Information Criterion under the non-stationary assumption called BWBIC, which can help significantly improve the reconstruction accuracy and largely reduce over-fitting. Moreover, the re-estimation formulas for all parameters of our model are derived, enabling us to avoid reliance on parameter settings. Compared to the state-of-the-art methods, the experimental evaluation of our proposed method on both synthetic and real biological data demonstrates more stably high prediction accuracy and significantly improved computation efficiency, even with no prior knowledge and parameter settings.</div>','<b>Title: </b>In vivo and systems biology studies implicate IL-18 as a central mediator in chronic pain<br><b>Year: </b>2015<br><b>Author: </b> K. Vasudeva,  Y. Vodovotz,  N. Azhar,  D. Barclay,  Jm. Janjic and  Ja. Pollock<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.jneuroim.2015.04.012"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF NEUROIMMUNOLOGY<br><b>Cluster-Keywords: </b><em>inflammatory response, network model, mechanistic models, lde model, injured patients, critically injured, prediction model, tbi survivors, computational modeling, model abm, impaired astrocytes, model predicts, metabolic dynamics, significant differences, twoprocess model, agent-based model, coupled model, model, inflammatory network, computational models<div style="font-size:11px;"><br><b>Abstract: </b><em>Inflammation is associated with peripheral neuropathy, however the interplay among cytokines, chemokines, and neurons is still unclear. We hypothesized that this neuroinflammatory interaction can be defined by computational modeling based on the dynamics of protein expression in the sciatic nerve of rats subjected to chronic constriction injury. Using Dynamic Bayesian Network inference, we identified interleukin IL-18 as a central node associated with neuropathic pain in this animal model. Immunofluorescence supported a role for inflammasome activation and induction of IL-18 at the site of injury. Combined in vivo and in silico approaches may thus highlight novel targets in peripheral neuropathy. C 2015 Elsevier B.V. All rights reserved.</div>','<b>Title: </b>Coordinated photomorphogenic UV-B signaling network captured by mathematical modeling<br><b>Year: </b>2014<br><b>Author: </b> Xh. Ouyang,  X. Huang,  X. Jin,  Z. Chen,  Py. Yang,  H. Ge,  Sg. Li and  Xw. Deng<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1073/pnas.1412050111"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA<br><b>Cluster-Keywords: </b><em>dna damage, signalling networks, network dynamics, pmdm system, systems biology, systems dynamics, biological systems, single cell, kinase inhibitors, signaling network, biological networks, cellular automata, cell cycle, oscillation system, temporal dynamics, signaling pathway, individual cells, signaling processes, dynamic model, damage response<div style="font-size:11px;"><br><b>Abstract: </b><em>Long-wavelength and low-fluence UV-B light is an informational signal known to induce photomorphogenic development in plants. Using the model plant Arabidopsis thaliana, a variety of factors involved in UV-B-specific signaling have been experimentally characterized over the past decade, including the UV-B light receptor UV resistance locus 8; the positive regulators constitutive photomorphogenesis 1 and elongated hypocotyl 5; and the negative regulators cullin4, repressor of UV-B photomorphogenesis 1 RUP1, and RUP2. Individual genetic and molecular studies have revealed that these proteins function in either positive or negative regulatory capacities for the sufficient and balanced transduction of photomorphogenic UV-B signal. Less is known, however, regarding how these signaling events are systematically linked. In our study, we use a systems biology approach to investigate the dynamic behaviors and correlations of multiple signaling components involved in Arabidopsis UV-B-induced photomorphogenesis. We define a mathematical representation of photomorphogenic UV-B signaling at a temporal scale. Supplemented with experimental validation, our computational modeling demonstrates the functional interaction that occurs among different protein complexes in early and prolonged response to photomorphogenic UV-B.</div>','<b>Title: </b>Prediction and measurement of an autoregulatory genetic module<br><b>Year: </b>2003<br><b>Author: </b> Fj. Isaacs,  J. Hasty,  Cr. Cantor and  Jj. Collins<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1073/pnas.1332628100"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA<br><b>Cluster-Keywords: </b><em>gene regulatory, systems biology, synthetic biology, regulatory network, gene network, gene expression, system model, synthetic network, network dynamics, systems theory, synthetic gene, genetic network, model predicts, dynamical systems, time delays, translational time, regulation models, oscillatory expression, network structure, biological systems<div style="font-size:11px;"><br><b>Abstract: </b><em>The deduction of phenotypic cellular responses from the structure and behavior of complex gene regulatory networks is one of the defining challenges of systems biology. This goal will require a quantitative understanding of the modular components that constitute such networks. We pursued an integrated approach, combining theory and experiment, to analyze and describe the dynamics of an isolated genetic module, an in vivo autoregulatory gene network. As predicted by the model, temperature-induced protein destabilization led to the existence of two expression states, thus elucidating the trademark bistability of the positive feedback-network architecture. After sweeping the temperature, observed population distributions and coefficients of variation were in quantitative agreement with those predicted by a stochastic version of the model. Because model fluctuations originated from small molecule-number effects, the experimental validation underscores the importance of internal noise in gene expression. This work demonstrates that isolated gene networks, coupled with proper quantitative descriptions, can elucidate key properties of functional genetic modules. Such an approach could lead to the modular dissection of naturally occurring gene regulatory networks, the deduction of cellular processes such as differentiation, and the development of engineered cellular control.</div>','<b>Title: </b>On the relationship between sloppiness and identifiability<br><b>Year: </b>2016<br><b>Author: </b> Ot. Chis,  Af. Villaverde,  Jr. Banga and  E. Balsa-Canto<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.mbs.2016.10.009"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MATHEMATICAL BIOSCIENCES<br><b>Cluster-Keywords: </b><em>biological networks, dynamic models, biological systems, model predictions, model parameters, systems biology, mathematical models, biological models, equation models, experimental data, molecular network, interaction networks, model errors, biological interaction, molecular interactions, model, dynamic systems, biological parameters, parameter uncertainty, growth model<div style="font-size:11px;"><br><b>Abstract: </b><em>Dynamic models of biochemical networks are often formulated as sets of non-linear ordinary differential equations, whose states are the concentrations or abundances of the network components. They typically have a large number of kinetic parameters, which must be determined by calibrating the model with experimental data. In recent years it has been suggested that dynamic systems biology models are universally sloppy, meaning that the values of some parameters can be perturbed by several orders of magnitude without causing significant changes in the model output. This observation has prompted calls for focusing on model predictions rather than on parameters. In this work we examine the concept of sloppiness, investigating its links with the long-established notions of structural and practical identifiability. By analysing a set of case studies we show that sloppiness is not equivalent to lack of identifiability, and that sloppy models can be identifiable. Thus, using sloppiness to draw conclusions about the possibility of estimating parameter values can be misleading. Instead, structural and practical identifiability analyses are better tools for assessing the confidence in parameter estimates. Furthermore, we show that, when designing new experiments to decrease parametric uncertainty, designs that optimize practical identifiability criteria are more informative than those that minimize sloppiness. C 2016 Elsevier Inc. All rights reserved.</div>','<b>Title: </b>Bistability in feedback circuits as a byproduct of evolution of evolvability<br><b>Year: </b>2012<br><b>Author: </b> H. Kuwahara and  Os. Soyer<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1038/msb.2011.98"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MOLECULAR SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>gene expression, gene network, stochastic model, gene regulatory, systems biology, expression systems, gene regulation, network activity, expression dynamics, noise control, regulatory network, deterministic models, network components, stochastic modeling, regulation system, escherichia coli, regulatory system, synthetic gene, regulatory genes, gene<div style="font-size:11px;"><br><b>Abstract: </b><em>Noisy bistable dynamics in gene regulation can underlie stochastic switching and is demonstrated to be beneficial under fluctuating environments. It is not known, however, if fluctuating selection alone can result in bistable dynamics. Using a stochastic model of simple feedback networks, we apply fluctuating selection on gene expression and run in silico evolutionary simulations. We find that independent of the specific nature of the environment-fitness relationship, the main outcome of fluctuating selection is the evolution of increased evolvability in the network; system parameters evolve toward a nonlinear regime where phenotypic diversity is increased and small changes in genotype cause large changes in expression level. In the presence of noise, the evolution of increased nonlinearity results in the emergence and maintenance of bistability. Our results provide the first direct evidence that bistability and stochastic switching in a gene regulatory network can emerge as a mechanism to cope with fluctuating environments. They strongly suggest that such emergence occurs as a byproduct of evolution of evolvability and exploitation of noise by evolution. Molecular Systems Biology 8: 564; published online 17 January 2012; doi:10.1038/msb.2011.98</div>','<b>Title: </b>Mutation-induced protein interaction kinetics changes affect apoptotic network dynamic properties and facilitate oncogenesis<br><b>Year: </b>2015<br><b>Author: </b> Lj. Zhao,  Tl. Sun,  Jf. Pei and  Q. Ouyang<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1073/pnas.1502126112"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA<br><b>Cluster-Keywords: </b><em>chaotic system, control systems, neural networks, jerk chaotic, torch chaotic, mode control, sliding mode, plasma torch, dynamic neural, mitochondrial dynamics, system, systems biology, adaptive integral, model system, rodtype plasma, integral sliding, systems, jerk system, cancer cell, comparison method<div style="font-size:11px;"><br><b>Abstract: </b><em>It has been a consensus in cancer research that cancer is a disease caused primarily by genomic alterations, especially somatic mutations. However, the mechanism of mutation-induced oncogenesis is not fully understood. Here, we used the mitochondrial apoptotic pathway as a case study and performed a systematic analysis of integrating pathway dynamics with protein interaction kinetics to quantitatively investigate the causal molecular mechanism of mutation-induced oncogenesis. A mathematical model of the regulatory network was constructed to establish the functional role of dynamic bifurcation in the apoptotic process. The oncogenic mutation enrichment of each of the protein functional domains involved was found strongly correlated with the parameter sensitivity of the bifurcation point. We further dissected the causal mechanism underlying this correlation by evaluating the mutational influence on protein interaction kinetics using molecular dynamics simulation. We analyzed 29 matched mutant-wild-type and 16 matched SNP-wild-type protein systems. We found that the binding kinetics changes reflected by the changes of free energy changes induced by protein interaction mutations, which induce variations in the sensitive parameters of the bifurcation point, were a major cause of apoptosis pathway dysfunction, and mutations involved in sensitive interaction domains show high oncogenic potential. Our analysis provided a molecular basis for connecting protein mutations, protein interaction kinetics, network dynamics properties, and physiological function of a regulatory network. These insights provide a framework for coupling mutation genotype to tumorigenesis phenotype and help elucidate the logic of cancer initiation.</div>','<b>Title: </b>A Digitally Programmable Cytomorphic Chip for Simulation of Arbitrary Biochemical Reaction Networks<br><b>Year: </b>2018<br><b>Author: </b> Ss. Woo,  J. Kim and  R. Sarpeshkar<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1109/TBCAS.2017.2781253"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IEEE TRANSACTIONS ON BIOMEDICAL CIRCUITS AND SYSTEMS<br><b>Cluster-Keywords: </b><em>regulatory networks, stochastic simulation, gene networks, reaction networks, systems biology, stochastic model, network models, complex systems, networks stochastic, gene regulatory, biological systems, stochastic dynamics, differential equations, protein networks, cell cycle, continuous model, interaction networks, systems chemistry, time delays, biochemical reaction<div style="font-size:11px;"><br><b>Abstract: </b><em>Prior work has shown that compact analog circuits can faithfully represent and model fundamental biomolecular circuits via efficient log-domain cytomorphic transistor equivalents. Such circuits have emphasized basis functions that are dominant in genetic transcription and translation networks and deoxyribonucleic acid DNA-protein binding. Here, we report a system featuring digitally programmable 0.35 mu m BiCMOS analog cytomorphic chips that enable arbitrary biochemical reaction networks to be exactly represented thus enabling compact and easy composition of protein networks as well. Since all biomolecular networks can be represented as chemical reaction networks, our protein networks also include the former genetic network circuits as a special case. The cytomorphic analog protein circuits use one fundamental association-dissociation-degradation building-block circuit that can be configured digitally to exactly represent any zeroth-, first-, and second-order reaction including loading, dynamics, nonlinearity, and interactions with other building-block circuits. To address a divergence issue caused by random variations in chip fabrication processes, we propose a unique way of performing computation based on total variables and conservation laws, which we instantiate at both the circuit and network levels. Thus, scalable systems that operate with finite error over infinite time can be built. We show how the building-block circuits can be composed to form various network topologies, such as cascade, fan-out, fan-in, loop, dimerization, or arbitrary networks using total variables. We demonstrate results from a system that combines interacting cytomorphic chips to simulate a cancer pathway and a glycolysis pathway. Both simulations are consistent with conventional software simulations. Our highly parallel digitally programmable analog cytomorphic systems can lead to a useful design, analysis, and simulation tool for studying arbitrary large-scale biological networks in systems and synthetic biology.</div>','<b>Title: </b>Social and landscape effects on food webs: a multi-level network simulation model<br><b>Year: </b>2013<br><b>Author: </b> M. Scotti,  F. Ciocchetta and  F. Jordan<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1093/comnet/cnt013"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF COMPLEX NETWORKS<br><b>Cluster-Keywords: </b><em>network ecology, food webs, network models, ecological systems, food web, systems biology, ecological networks, ecosystem model, signalling networks, model results, interaction networks, dynamic model, ecological interactions, network analysis, model network, network structure, transport networks, systems network, ecosystem dynamics, restoration ecology<div style="font-size:11px;"><br><b>Abstract: </b><em>One source of complexity in ecological systems is the hierarchical organization of parallel biological processes. Our horizontal knowledge describing different levels is quite massive, but the understanding of their vertical interactions is very poor. We present a toy model linking social networks, food webs and a landscape graph. Horizontal processes refer to population, community and metacommunity dynamics, while vertical processes connect the three organizational levels. The model is stochastic and individual-based. We parametrized it by using reasonable empirical values found in the literature. Sensitivity analysis shows how the parameters describing the dynamics of a particular species e.g., probability of social tie formation with conspecific individuals, or migration rate can affect metapopulation size and spatial heterogeneity of all food web species. Changing the values of various parameters at any of the three levels have commensurable effects on the population size of all species. In contrast to the general intuition, community dynamics do not dominate population biology; social and landscape processes can trigger greater effects than food web interactions. More rapidly changing social relationships lead to a decrease in social network cohesion, thus impairing the feeding efficiency of consumers. In food webs, trophic specialization provides an advantage when it contributes to avoid competition, being detrimental otherwise. Highest migration rates result in a more heterogeneous metapopulation distribution of the generalist consumer, indirectly supporting its specialist competitor. We discuss conceptual and methodological aspects of the model, demonstrating the importance of an integrative view. We also emphasize the relevance of vertical connections, suggesting how such a modelling framework could support conservation biology. Further studies should focus on methods to approximate external pressures with changes in model parameters, thus allowing to characterize possible impacts on ecological systems.</div>','<b>Title: </b>Fathead minnow steroidogenesis: in silico analyses reveals tradeoffs between nominal target efficacy and robustness to cross-talk<br><b>Year: </b>2010<br><b>Author: </b> Je. Shoemaker,  K. Gayen,  N. Garcia-Reyero,  Ej. Perkins,  Dl. Villeneuve,  L. Liu and  Fj. Doyle<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-4-89"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>sensitivity analysis, systems biology, robustness analysis, network model, network inference, regulatory network, systems analysis, gene expression, parametric uncertainty, biological networks, mathematical models, biochemical networks, biological functions, cellular networks, biological system, parametric sensitivity, dynamical models, dynamic system, networks analysis, gene regulatory<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Interpreting proteomic and genomic data is a major challenge in predictive ecotoxicology that can be addressed by a systems biology approach. Mathematical modeling provides an organizational platform to consolidate protein dynamics with possible genomic regulation. Here, a model of ovarian steroidogenesis in the fathead minnow, Pimephales promelas, FHM is developed to evaluate possible transcriptional regulation of steroid production observed in microarray studies.Results: The model was developed from literature sources, integrating key signaling components G-protein and PKA activation with their ensuing effect on steroid production. The model properly predicted trajectory behavior of estradiol and testosterone when fish were exposed to fadrozole, a specific aromatase inhibitor, but failed to predict the steroid hormone behavior occurring one week post-exposure as well as the increase in steroid levels when the stressor was removed. In vivo microarray data implicated three modes of regulation which may account for over-production of steroids during a depuration phase when the stressor is removed: P450 enzyme up-regulation, inhibin down-regulation, and luteinizing hormone receptor up-regulation. Simulation studies and sensitivity analysis were used to evaluate each case as possible source of compensation to endocrine stress.Conclusions: Simulation studies of the testosterone and estradiol response to regulation observed in microarray data supported the hypothesis that the FHM steroidogenesis network compensated for endocrine stress by modulating the sensitivity of the ovarian network to global cues coming from the hypothalamus and pituitary. Model predictions of luteinizing hormone receptor regulation were consistent with depuration and in vitro data. These results challenge the traditional approach to network elucidation in systems biology. Generally, the most sensitive interactions in a network are targeted for further elucidation but microarray evidence shows that homeostatic regulation of the steroidogenic network is likely maintained by a mildly sensitive interaction. We hypothesize that effective network elucidation must consider both the sensitivity of the target as well as the targets robustness to biological noise in this case, to cross-talk when identifying possible points of regulation.</div>','<b>Title: </b>Integrated signaling pathway and gene expression regulatory model to dissect dynamics of Escherichia coli challenged mammary epithelial cells<br><b>Year: </b>2014<br><b>Author: </b> Ny. Den Breems,  Lk. Nguyen and  D. Kulasiri<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.biosystems.2014.09.011"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOSYSTEMS<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Cells transform external stimuli, through the activation of signaling pathways, which in turn activate gene regulatory networks, in gene expression. As more omics data are generated from experiments, eliciting the integrated relationship between the external stimuli, the signaling process in the cell and the subsequent gene expression is a major challenge in systems biology. The complex system of non-linear dynamic protein interactions in signaling pathways and gene networks regulates gene expression.The complexity and non-linear aspects have resulted in the study of the signaling pathway or the gene network regulation in isolation. However, this limits the analysis of the interaction between the two components and the identification of the source of the mechanism differentiating the gene expression profiles. Here, we present a study of a model of the combined signaling pathway and gene network to highlight the importance of integrated modeling.Based on the experimental findings we developed a compartmental model and conducted several simulation experiments. The model simulates the mRNA expression of three different cytokines RANTES, IL8 and TNF alpha regulated by the transcription factor NF kappa B in mammary epithelial cells challenged with E. coli. The analysis of the gene network regulation identifies a lack of robustness and therefore sensitivity for the transcription factor regulation. However, analysis of the integrated signaling and gene network regulation model reveals distinctly different underlying mechanisms in the signaling pathway responsible for the variation between the three cytokines mRNA expression levels. Our key findings reveal the importance of integrating the signaling pathway and gene expression dynamics in modeling. Modeling infers valid research questions which need to be verified experimentally and can assist in the design of future biological experiments. C 2014 The Authors. Published by Elsevier Ireland Ltd.</div>','<b>Title: </b>Steady-State Differential Dose Response in Biological Systems<br><b>Year: </b>2018<br><b>Author: </b> P. Yordanov and  J. Stelling<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.bpj.2017.11.3780"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOPHYSICAL JOURNAL<br><b>Cluster-Keywords: </b><em>reaction networks, reaction systems, metabolic networks, chemical reaction, network structure, systems biology, biological systems, biological networks, biochemical reaction, reaction rates, differential equations, reaction graph, complex networks, ssi modules, networks, life systems, ordinary differential, cellular networks, reaction, network flows<div style="font-size:11px;"><br><b>Abstract: </b><em>In pharmacology and systems biology, it is a fundamental problem to determine how biological systems change their dose-response behavior upon perturbations. In particular, it is unclear how topologies, reactions, and parameters differentially affect the dose response. Because parameters are often unknown, systematic approaches should directly relate network structure and function. Here, we outline a procedure to compare general non-monotone dose-response curves and subsequently develop a comprehensive theory for differential dose responses of biochemical networks captured by non-equilibrium steady-state linear framework models. Although these models are amenable to analytical derivations of non-equilibrium steady states in principle, their size frequently increases super exponentially with model size. We extract general principles of differential responses based on a models graph structure and thereby alleviate the combinatorial explosion. This allows us, for example, to determine reactions that affect differential responses, to identify classes of networks with equivalent differential, and to reject hypothetical models reliably without needing to know parameter values. We exemplify such applications for models of insulin signaling.</div>','<b>Title: </b>A method of speed coefficients for biochemical model reduction applied to the NF-kappa B system<br><b>Year: </b>2015<br><b>Author: </b> S. West,  Lj. Bridge,  Mrh. White,  P. Paszek and  Vn. Biktashev<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s00285-014-0775-x"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF MATHEMATICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>immune system, network models, immune response, model, biological systems, models, model reduction, dynamic model, biological networks, system, mathematical models, biochemical network, response model, systems, systems biology, network, immune, biology model, reaction networks, system systems<div style="font-size:11px;"><br><b>Abstract: </b><em>The relationship between components of biochemical network and the resulting dynamics of the overall system is a key focus of computational biology. However, as these networks and resulting mathematical models are inherently complex and non-linear, the understanding of this relationship becomes challenging. Among many approaches, model reduction methods provide an avenue to extract components responsible for the key dynamical features of the system. Unfortunately, these approaches often require intuition to apply. In this manuscript we propose a practical algorithm for the reduction of biochemical reaction systems using fast-slow asymptotics. This method allows the ranking of system variables according to how quickly they approach their momentary steady state, thus selecting the fastest for a steady state approximation. We applied this method to derive models of the Nuclear Factor kappa B network, a key regulator of the immune response that exhibits oscillatory dynamics. Analyses with respect to two specific solutions, which corresponded to different experimental conditions identified different components of the system that were responsible for the respective dynamics. This is an important demonstration of how reduction methods that provide approximations around a specific steady state, could be utilised in order to gain a better understanding of network topology in a broader context.</div>','<b>Title: </b>Prospect and Limits of Explaining Biological Systems in Engineering Terms<br><b>Year: </b>2015<br><b>Author: </b> Pa. Braillard<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/978-94-017-9822-814"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>EXPLANATION IN BIOLOGY: AN ENQUIRY INTO THE DIVERSITY OF EXPLANATORY PATTERNS IN THE LIFE SCIENCES<br><b>Cluster-Keywords: </b><em>systems biology, biological systems, system dynamics, molecular systems, experimental design, network model, model system, model parameters, experiment design, kinetic parameters, perfect adaptation, experimental data, molecular mechanisms, homeostatic mechanisms, switching dynamics, biological mechanisms, control systems, chemotaxis network, decentralized control, feedback control<div style="font-size:11px;"><br><b>Abstract: </b><em>Systems biology represents an effort to develop new modelling approaches in molecular and cell biology, drawing inspiration from disciplines like physics, computer sciences and engineering. In particular, many scientists have called for a transfer of methods, models and concepts from engineering in order to analyze and explain biological systems in all their complexity. In this paper, I examine how such transfer can contribute to systems biology explanatory project. Model building in the context of post-genomic biology raises a number of difficult challenges, mainly due to the complexity of the processes studied and their intricate dynamical features. Engineering methods can be used to efficiently analyze quantitative data about systems behaviour and use them to build mathematical dynamical models, in a way that goes beyond classical mechanistic approaches. More generally, engineering has suggested adopting a modular framework, as a general approach of decomposition and explanation based on analogies between biological and engineered systems, which promises to identify intelligible principles in the complex organization of molecular networks. I discuss the nature of this explanatory framework and on what assumptions it rests.</div>','<b>Title: </b>MACROECONOMIC ANALYSIS OF THE INFORMATION DYNAMIC-MODEL<br><b>Year: </b>1993<br><b>Author: </b> V. Lerner<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1080/01969729308961728"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>CYBERNETICS AND SYSTEMS<br><b>Cluster-Keywords: </b><em>blood flow, blood cells, flow rate, blood pumps, cells, cell, results, patients, stem cells, methods, blood, patients blood, flow, patients methods, patients objectives, cell viability, objectives, results cells, blood pressure, cell culture<div style="font-size:11px;"><br><b>Abstract: </b><em>Macroeconomic analysis is considered the information dynamic model of macroeconomics, taking into account the interaction of the material, social labor, and biology processes on the base of the united information mathematical formalism. Aggregation of that model is based on the systematic structurization in the information decomposed macro structures. The macroeconomics analogies of the informational description are used for definition of the optimal ratio of distribution of incomes from property and social labor, maximizing the information criteria of usefulness.The solution of the variation problem defines the optimal dynamic macro model, describing the informational regularities, which coordinates the interests of the set of interacting elements all collective with interests of each element.The natures of the macro model characteristics are complexity, information quality, value of the adaptation potential, the proper time, and the specific geometry that are discrete and dependable from complexity. The macro model binds and structurizes the system of decomposed macro units, connected in some hierarchical network.The evolution of the macro model includes the mechanisms: of renovation of dynamic, geometrical properties with the states duplication and memorization, and their transformation to new macro model, the selection of the macrosystems with maximum of adaptation potential; the united mechanism of structure changing, consolidation, and self organization; development of order with forming of the hierarchical cooperative macro structures. The methodology of the spatial time synthesis was realized on computers by the system package of applied programs. It has constructed the family of standardized macro models of different dimensions, comparing the dynamics. geometry, and adaptation potential to the index of the information complexity.The obtained results determine the natural limit of increasing of dimension and complexity for macrosystems, created by humans. Discovery of the information regularities of the optimal evolution opens new opportunities of the progressive development of society.</div>','<b>Title: </b>Systems level analysis of the Chlamydomonas reinhardtii metabolic network reveals variability in evolutionary co-conservation<br><b>Year: </b>2016<br><b>Author: </b> A. Chaiboonchoe,  L. Ghamsari,  B. Dohai,  P. Ng,  B. Khraiwesh,  A. Jaiswal,  K. Jijakli,  J. Koussa,  Dr. Nelson,  H. Cai,  Xp. Yang,  Rl. Chang,  J. Papin,  Hy. Yu,  S. Balaji and  K. Salehi-Ashtiani<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1039/c6mb00237d"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MOLECULAR BIOSYSTEMS<br><b>Cluster-Keywords: </b><em>gene expression, systems biology, metabolic network, network model, network analysis, dynamic model, signaling pathways, signaling network, regulatory network, dynamic network, biology approach, candidate genes, gene network, gene regulatory, biology analysis, mathematical model, network dynamic, biology network, metabolic pathways, pathway analysis<div style="font-size:11px;"><br><b>Abstract: </b><em>Metabolic networks, which are mathematical representations of organismal metabolism, are reconstructed to provide computational platforms to guide metabolic engineering experiments and explore fundamental questions on metabolism. Systems level analyses, such as interrogation of phylogenetic relationships within the network, can provide further guidance on the modification of metabolic circuitries. Chlamydomonas reinhardtii, a biofuel relevant green alga that has retained key genes with plant, animal, and protist affinities, serves as an ideal model organism to investigate the interplay between gene function and phylogenetic affinities at multiple organizational levels. Here, using detailed topological and functional analyses, coupled with transcriptomics studies on a metabolic network that we have reconstructed for C. reinhardtii, we show that network connectivity has a significant concordance with the co-conservation of genes; however, a distinction between topological and functional relationships is observable within the network. Dynamic and static modes of co-conservation were defined and observed in a subset of gene-pairs across the network topologically. In contrast, genes with predicted synthetic interactions, or genes involved in coupled reactions, show significant enrichment for both shorter and longer phylogenetic distances. Based on our results, we propose that the metabolic network of C. reinhardtii is assembled with an architecture to minimize phylogenetic profile distances topologically, while it includes an expansion of such distances for functionally interacting genes. This arrangement may increase the robustness of C. reinhardtiis network in dealing with varied environmental challenges that the species may face. The defined evolutionary constraints within the network, which identify important pairings of genes in metabolism, may offer guidance on synthetic biology approaches to optimize the production of desirable metabolites.</div>','<b>Title: </b>Markov State Models of gene regulatory networks<br><b>Year: </b>2017<br><b>Author: </b> Bk. Chu,  Mj. Tse,  Rr. Sato and  El. Read<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/s12918-017-0394-4"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>gene regulatory, regulatory network, systems biology, cell differentiation, cell fate, model systems, potential landscape, network grn, dynamical systems, melanocyte differentiation, network models, gene expression, network dynamics, differentiation genes, dynamical models, dynamical networks, cellular differentiation, stem cells, attractor network, developmental gene<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Gene regulatory networks with dynamics characterized by multiple stable states underlie cell fate-decisions. Quantitative models that can link molecular-level knowledge of gene regulation to a global understanding of network dynamics have the potential to guide cell-reprogramming strategies. Networks are often modeled by the stochastic Chemical Master Equation, but methods for systematic identification of key properties of the global dynamics are currently lacking.Results: The method identifies the number, phenotypes, and lifetimes of long-lived states for a set of common gene regulatory network models. Application of transition path theory to the constructed Markov State Model decomposes global dynamics into a set of dominant transition paths and associated relative probabilities for stochastic state-switching.Conclusions: In this proof-of-concept study, we found that the Markov State Model provides a general framework for analyzing and visualizing stochastic multistability and state-transitions in gene networks. Our results suggest that this framework-adopted from the field of atomistic Molecular Dynamics-can be a useful tool for quantitative Systems Biology at the network scale.</div>','<b>Title: </b>Transient dynamics around unstable periodic orbits in the generalized repressilator model<br><b>Year: </b>2011<br><b>Author: </b> N. Strelkowa and  M. Barahona<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1063/1.3574387"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>CHAOS<br><b>Cluster-Keywords: </b><em>oscillator networks, network dynamics, network, networks, dynamical systems, dynamics, node dynamics, systems, coupled oscillator, synchronization, dynamics network, complex network, coupled system, network modifications, oscillators, network connectivity, feedback vertex, network topology, synchronization properties, collective dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>We study the temporal dynamics of the generalized repressilator, a network of coupled repressing genes arranged in a directed ring topology, and give analytical conditions for the emergence of a finite sequence of unstable periodic orbits that lead to reachable long-lived oscillating transients. Such transients dominate the finite time horizon dynamics that is relevant in confined, noisy environments such as bacterial cells see our previous work Strelkowa and Barahona, J. R. Soc. Interface 7, 1071 2010, and are therefore of interest for bioengineering and synthetic biology. We show that the family of unstable orbits possesses spatial symmetries and can also be understood in terms of traveling wave solutions of kink-like topological defects. The long-lived oscillatory transients correspond to the propagation of quasistable two-kink configurations that unravel over a long time. We also assess the similarities between the generalized repressilator model and other unidirectionally coupled electronic systems, such as magnetic flux gates, which have been implemented experimentally. C 2011 American Institute of Physics. doi:10.1063/1.3574387</div>','<b>Title: </b>Two-category model of task allocation with application to ant societies<br><b>Year: </b>2001<br><b>Author: </b> Wam. Brandts,  A. Longtin and  Leh. Trainor<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1006/bulm.2001.0262"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BULLETIN OF MATHEMATICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>network models, gene network, regulatory networks, boolean networks, gene expression, systems biology, neural networks, network dynamics, gene regulatory, biological networks, network structure, gap gene, gene regulation, cell cycle, network, gene interactions, signaling network, quantum systems, system dynamics, networks<div style="font-size:11px;"><br><b>Abstract: </b><em>In many network models of interacting units such as cells or insects, the coupling coefficients between units are independent of the state of the units. Here we analyze the temporal behavior of units that can switch between two category states according to rules that involve category-dependent coupling coefficients. The behaviors of the category populations resulting from the asynchronous random updating of units are first classified according to the signs of the coupling coefficients using numerical simulations. They range from isolated fixed points to lines of fixed points and stochastic attractors. These behaviors are then explained analytically using iterated function systems and birth-death jump processes. The main inspiration for our work comes from studies of non-hierarchical task allocation in, e.g., harvester ant colonies where temporal fluctuations in the numbers of ants engaged in various tasks occur as circumstances require and depend on interactions between ants. We identify interaction types that produce quick recovery from perturbations to an asymptotic behavior whose characteristics are function of the coupling coefficients between ants as well as between ants and their environment. We also compute analytically the probability density of the population numbers, and show that perturbations in our model decay twice as fast as in a model with random switching dynamics. A subset of the interaction types between ants yields intrinsic stochastic asymptotic behaviors which could account for some of the experimentally observed fluctuations. Such noisy trajectories are shown to be random walks with state-dependent biases in the category population phase space. With an external stimulus, the parameters of the category-switching rules become time-dependent. Depending on the growth rate of the stimulus in comparison to its population-dependent decay rate, the dynamics may qualitatively differ from the case without stimulus. Our simple two-category model provides a framework for understanding the rich variety of behaviors in network dynamics with state-dependent coupling coefficients, and especially in task allocation processes with many tasks. C 2001 Society for Mathematical Biology.</div>','<b>Title: </b>Understanding TB latency using computational and dynamic modelling procedures<br><b>Year: </b>2013<br><b>Author: </b> G. Magombedze and  N. Mulder<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.meegid.2012.09.017"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>INFECTION GENETICS AND EVOLUTION<br><b>Cluster-Keywords: </b><em>sensitivity analysis, systems biology, robustness analysis, network model, network inference, regulatory network, systems analysis, gene expression, parametric uncertainty, biological networks, mathematical models, biochemical networks, biological functions, cellular networks, biological system, parametric sensitivity, dynamical models, dynamic system, networks analysis, gene regulatory<div style="font-size:11px;"><br><b>Abstract: </b><em>The Mycobacterium tuberculosis bacillis potency to cause persistent latent infection that is unresponsive to the current cocktail of TB drugs is strongly associated with its ability to adapt to changing intracellular environments, and tolerating, evading and subverting host defence mechanisms. We applied a combination of bioinformatics and mathematical modelling methods to enhance the understanding of TB latency dynamics. Analysis of time course microarray gene expression data was carried out and gene profiles for bacilli adaptation and survival in latency, simulated by hypoxia were determined. Reverse network engineering techniques were used to predict gene dependencies and regulatory interactions. Biochemical systems theory was applied to mathematically model the inferred gene regulatory networks. Significant regulatory genes involved in latency were determined by a combination of systems biology procedures and mathematical modelling of the inferred regulatory networks. Analysis of gene clusters of the inferred networks in the stationary and non-replicating phases of the bacilli predicted probable functions of some of the latency genes to be associated with latency genes of known functions. The systems biology approach and mathematical computational deletion experiments predicted key genes in the TB latency/dormancy program that may be possible TB drug targets. However, these gene candidates require experimental testing and validation. Published by Elsevier B.V.</div>','<b>Title: </b>Transient oscillatory dynamics of interferon beta signaling in macrophages<br><b>Year: </b>2013<br><b>Author: </b> I. Pertsovskaya,  E. Abad,  N. Domedel-Puig,  J. Garcia-Ojalvo and  P. Villoslada<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-7-59"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>regulatory network, systems biology, systems biological, network dynamics, gene expression, biochemical networks, raft proteins, biological networks, gene circuits, gene regulatory, signalling network, lipid raft, nuclear lamina, protein domain, networks systems, biological approaches, expression pattern, protein interaction, cell differentiation, network motifs<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Interferon-beta IFN-beta activates the immune response through the type I IFN signaling pathway. IFN-beta is important in the response to pathogen infections and is used as a therapy for Multiple Sclerosis. The mechanisms of self-regulation and control of this pathway allow precise and environment-dependent response of the cells in different conditions. Here we analyzed type I IFN signaling in response to IFN-beta in the macrophage cell line RAW 264.7 by RT-PCR, ELISA and xMAP assays. The experimental results were interpreted by means of a theoretical model of the pathway.Results: Phosphorylation of the STAT1 protein pSTAT1 and mRNA levels of the pSTAT1 inhibitor SOCS1 displayed an attenuated oscillatory behavior after IFN-beta activation. In turn, mRNA levels of the interferon regulatory factor IRF1 grew rapidly in the first 50-90 minutes after stimulation until a maximum value, and started to decrease slowly around 200-250 min. The analysis of our kinetic model identified a significant role of the negative feedback from SOCS1 in driving the observed damped oscillatory dynamics, and of the positive feedback from IRF1 in increasing STAT1 basal levels. Our study shows that the system works as a biological damped relaxation oscillator based on a phosphorylation-dephosphorylation network centered on STAT1. Moreover, a bifurcation analysis identified translocation of pSTAT1 dimers to the nucleus as a critical step for regulating the dynamics of type I IFN pathway in the first steps, which may be important in defining the response to IFN-beta therapy.Conclusions: The immunomodulatory effect of IFN-beta signaling in macrophages takes the form of transient oscillatory dynamics of the JAK-STAT pathway, whose specific relaxation properties determine the lifetime of the cellular response to the cytokine.</div>','<b>Title: </b>Modeling of Gene Regulatory Network Dynamics Using Threshold Logic<br><b>Year: </b>2009<br><b>Author: </b> T. Gowda,  S. Vrudhula and  S. Kim<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1111/j.1749-6632.2008.03754.x"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>CHALLENGES OF SYSTEMS BIOLOGY: COMMUNITY EFFORTS TO HARNESS BIOLOGICAL COMPLEXITY<br><b>Cluster-Keywords: </b><em>boolean networks, regulatory networks, genetic regulatory, gene regulatory, probabilistic boolean, network, networks, model, network model, probabilistic model, model checking, networks pbns, network dynamics, markov chain, regulatory, models, optimal control, boolean, networks models, networks boolean<div style="font-size:11px;"><br><b>Abstract: </b><em>Gene regulation modeling is one of the most active research topics in systems biology. The aim of modeling gene regulation is to understand how, individual genes function and interact with each other to create complex biological phenomena. In this paper we propose a novel gene regulatory model based on threshold logic. The approach is developed by a combination of threshold logic properties and perceptron learning techniques. This work does not focus on determination of the pair-wise interactions among genes. Instead, the objective of this work is to generate a model that will describe and predict phenomena associated with it biological system. The utility of the approach is demonstrated by modeling a cellular system of 50 genes. The model could effectively replicate both the steady state and the transient behavior of genes.</div>','<b>Title: </b>A minimal and self-consistent in silico cell model based on macromolecular interactions<br><b>Year: </b>2007<br><b>Author: </b> C. Flamm,  L. Endler,  S. Muller,  S. Widder and  P. Schuster<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1098/rstb.2007.2075"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PHILOSOPHICAL TRANSACTIONS OF THE ROYAL SOCIETY B-BIOLOGICAL SCIENCES<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>A self-consistent minimal cell model with a physically motivated schema for molecular interaction is introduced and described. The genetic and metabolic reaction network of the cell is modelled by multidimensional nonlinear ordinary differential equations, which are derived from biochemical kinetics. The strategy behind this modelling approach is to keep the model sufficiently simple in order to be able to perform studies on evolutionary optimization in populations of cells. At the same time, the model should be complex enough to handle the basic features of genetic control of metabolism and coupling to environmental factors. Thereby, the model system will provide insight into the mechanisms leading to important biological phenomena, such as homeostasis, circadian rhythms, robustness and adaptation to a changing environment. One example of modelling a molecular regulatory mechanism, cooperative binding of transcription factors, is discussed in detail.</div>','<b>Title: </b>A Directed Protein Interaction Network for Investigating Intracellular Signal Transduction<br><b>Year: </b>2011<br><b>Author: </b> A. Vinayagam,  U. Stelzl,  R. Foulle,  S. Plassmann,  M. Zenkner,  J. Timm,  He. Assmus,  Ma. Andrade-Navarro and  Ee. Wanker<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1126/scisignal.2001699"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SCIENCE SIGNALING<br><b>Cluster-Keywords: </b><em>systems biology, regulatory network, interaction networks, network model, gene regulatory, network, network biology, networks, gene network, response networks, network dynamics, dynamic model, cell networks, gene expression, biological networks, system dynamics, dynamic network, model, biological systems, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Cellular signal transduction is a complex process involving protein-protein interactions PPIs that transmit information. For example, signals from the plasma membrane may be transduced to transcription factors to regulate gene expression. To obtain a global view of cellular signaling and to predict potential signal modulators, we searched for protein interaction partners of more than 450 signaling-related proteins by means of automated yeast two-hybrid interaction mating. The resulting PPI network connected 1126 proteins through 2626 PPIs. After expansion of this interaction map with publicly available PPI data, we generated a directed network resembling the signal transduction flow between proteins with a naive Bayesian classifier. We exploited information on the shortest PPI paths from membrane receptors to transcription factors to predict input and output relationships between interacting proteins. Integration of directed PPI with time-resolved protein phosphorylation data revealed network structures that dynamically conveyed information from the activated epidermal growth factor and extracellular signal-regulated kinase EGF/ERK signaling cascade to directly associated proteins and more distant proteins in the network. From the model network, we predicted 18 previously unknown modulators of EGF/ERK signaling, which we validated in mammalian cell-based assays. This generic experimental and computational approach provides a framework for elucidating causal connections between signaling proteins and facilitates the identification of proteins that modulate the flow of information in signaling networks.</div>','<b>Title: </b>Modeling of the Bacterial Mechanism of Methicillin-Resistance by a Systems Biology Approach<br><b>Year: </b>2009<br><b>Author: </b> I. Autiero,  S. Costantini and  G. Colonna<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0006226"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: A microorganism is a complex biological system able to preserve its functional features against external perturbations and the ability of the living systems to oppose to these external perturbations is defined "robustness. The antibiotic resistance, developed by different bacteria strains, is a clear example of robustness and of ability of the bacterial system to acquire a particular functional behaviour in response to environmental changes. In this work we have modeled the whole mechanism essential to the methicillin-resistance through a systems biology approach. The methicillin is a beta-lactamic antibiotic that act by inhibiting the penicillin-binding proteins PBPs. These PBPs are involved in the synthesis of peptidoglycans, essential mesh-like polymers that surround cellular enzymes and are crucial for the bacterium survival.Methodology: The network of genes, mRNA, proteins and metabolites was created using CellDesigner program and the data of molecular interactions are stored in Systems Biology Markup Language SBML. To simulate the dynamic behaviour of this biochemical network, the kinetic equations were associated with each reaction.Conclusions: Our model simulates the mechanism of the inactivation of the PBP by methicillin, as well as the expression of PBP2a isoform, the regulation of the SCCmec elements SCC: staphylococcal cassette chromosome and the synthesis of peptidoglycan by PBP2a. The obtained results by our integrated approach show that the model describes correctly the whole phenomenon of the methicillin resistance and is able to respond to the external perturbations in the same way of the real cell. Therefore, this model can be useful to develop new therapeutic approaches for the methicillin control and to understand the general mechanism regarding the cellular resistance to some antibiotics.</div>','<b>Title: </b>Inferring species interaction networks from species abundance data: A comparative evaluation of various statistical and machine learning methods<br><b>Year: </b>2010<br><b>Author: </b> A. Faisal,  F. Dondelinger,  D. Husmeier and  Cm. Beale<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.ecoinf.2010.06.005"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ECOLOGICAL INFORMATICS<br><b>Cluster-Keywords: </b><em>boolean networks, regulatory networks, network model, bayesian networks, model networks, systems biology, random boolean, real cells, immune system, genetic regulatory, networks, biochemical network, fluorescence images, cellular network, gene regulatory, cell differentiation, network, biological systems, ensemble approach, cell dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>The complexity of ecosystems is staggering, with hundreds or thousands of species interacting in a number of ways from competition and predation to facilitation and mutualism. Understanding the networks that form the systems is of growing importance, e.g. to understand how species will respond to climate change, or to predict potential knock-on effects of a biological control agent. In recent years, a variety of summary statistics for characterising the global and local properties of such networks have been derived, which provide a measure for gauging the accuracy of a mathematical model for network formation processes. However, the critical underlying assumption is that the true network is known. This is not a straightforward task to accomplish, and typically requires minute observations and detailed field work. More importantly, knowledge about species interactions is restricted to specific kinds of interactions. For instance, while the interactions between pollinators and their host plants are amenable to direct observation, other types of species interactions, like those mentioned above, are not, and might not even be clearly defined from the outset. To discover information about complex ecological systems efficiently, new tools for inferring the structure of networks from field data are needed. In the present study, we investigate the viability of various statistical and machine learning methods recently applied in molecular systems biology: graphical Gaussian models, L1-regularised regression with least absolute shrinkage and selection operator LASSO, sparse Bayesian regression and Bayesian networks. We have assessed the performance of these methods on data simulated from food webs of known structure, where we combined a niche model with a stochastic population model in a 2-dimensional lattice. We assessed the network reconstruction accuracy in terms of the area under the receiver operating characteristic ROC curve, which was typically in the range between 0.75 and 0.9, corresponding to the recovery of about 60 of the true species interactions at a false prediction rate of 5. We also applied the models to presence/absence data for 39 European warblers, and found that the inferred species interactions showed a weak yet significant correlation with phylogenetic similarity scores, which tended to weakly increase when including bio-climate covariates and allowing for spatial autocorrelation. Our findings demonstrate that relevant patterns in ecological networks can be identified from large-scale spatial data sets with machine learning methods, and that these methods have the potential to contribute novel important tools for gaining deeper insight into the structure and stability of ecosystems. C 2010 Elsevier B.V. All rights reserved.</div>','<b>Title: </b>Hierarchical Cluster-based Partial Least Squares Regression HC-PLSR is an efficient tool for metamodelling of nonlinear dynamic models<br><b>Year: </b>2011<br><b>Author: </b> K. Tondel,  Ug. Indahl,  Ab. Gjuvsland,  Jo. Vik,  P. Hunter,  Sw. Omholt and  H. Martens<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-5-90"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>systems biology, network models, signaling network, parameter estimation, dynamic models, biological systems, mathematical models, metabolic network, parameter sensitivities, computational model, gene circuit, circuit models, experimental data, metabolic models, model system, scalable parameter, model parameters, parameter space, biomolecular networks, model<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Deterministic dynamic models of complex biological systems contain a large number of parameters and state variables, related through nonlinear differential equations with various types of feedback. A metamodel of such a dynamic model is a statistical approximation model that maps variation in parameters and initial conditions inputs to variation in features of the trajectories of the state variables outputs throughout the entire biologically relevant input space. A sufficiently accurate mapping can be exploited both instrumentally and epistemically. Multivariate regression methodology is a commonly used approach for emulating dynamic models. However, when the input-output relations are highly nonlinear or non-monotone, a standard linear regression approach is prone to give suboptimal results. We therefore hypothesised that a more accurate mapping can be obtained by locally linear or locally polynomial regression. We present here a new method for local regression modelling, Hierarchical Cluster-based PLS regression HC-PLSR, where fuzzy C-means clustering is used to separate the data set into parts according to the structure of the response surface. We compare the metamodelling performance of HC-PLSR with polynomial partial least squares regression PLSR and ordinary least squares OLS regression on various systems: six different gene regulatory network models with various types of feedback, a deterministic mathematical model of the mammalian circadian clock and a model of the mouse ventricular myocyte function.Results: Our results indicate that multivariate regression is well suited for emulating dynamic models in systems biology. The hierarchical approach turned out to be superior to both polynomial PLSR and OLS regression in all three test cases. The advantage, in terms of explained variance and prediction accuracy, was largest in systems with highly nonlinear functional relationships and in systems with positive feedback loops.Conclusions: HC-PLSR is a promising approach for metamodelling in systems biology, especially for highly nonlinear or non-monotone parameter to phenotype maps. The algorithm can be flexibly adjusted to suit the complexity of the dynamic model behaviour, inviting automation in the metamodelling of complex systems.</div>','<b>Title: </b>Impact of Dimensionality and Network Disruption on Microrheology of Cancer Cells in 3D Environments<br><b>Year: </b>2014<br><b>Author: </b> M. Mak,  Rd. Kamm and  Mh. Zaman<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pcbi.1003959"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS COMPUTATIONAL BIOLOGY<br><b>Cluster-Keywords: </b><em>interaction network, actin filament, network model, filament network, network structures, actin network, protein interaction, actin dynamics, network, membrane protein, network dynamics, model dynamic, cell, model, network approach, protein complex, computational model, actin, adhesion dynamics, cell dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>Dimensionality is a fundamental component that can have profound implications on the characteristics of physical systems. In cell biology, however, the majority of studies on cell physical properties, from rheology to force generation to migration, have been performed on 2D substrates, and it is not clear how a more realistic 3D environment influences cell properties. Here, we develop an integrated approach and demonstrate the combination of mitochondria-tracking microrheology, microfluidics, and Brownian dynamics simulations to explore the impact of dimensionality on intracellular mechanics and on the effects of intracellular disruption. Additionally, we consider both passive thermal and active motor-driven processes within the cell and demonstrate through modeling how active internal fluctuations are modulated via dimensionality. Our results demonstrate that metastatic breast cancer cells MDA-MB-231 exhibit more solid-like internal motions in 3D compared to 2D, and actin network disruption via Cytochalasin D has a more pronounced effect on internal cell fluctuations in 2D. Our computational results and modeling show that motor-induced active stress fluctuations are enhanced in 2D, leading to increased local intracellular particle fluctuations and apparent fluid-like behavior.</div>','<b>Title: </b>Discrete Dynamic Modeling with Asynchronous Update, or How to Model Complex Systems in the Absence of Quantitative Information<br><b>Year: </b>2009<br><b>Author: </b> Sm. Assmann and  R. Albert<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/978-1-60327-563-710"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLANT SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>network models, gene network, regulatory networks, boolean networks, gene expression, systems biology, neural networks, network dynamics, gene regulatory, biological networks, network structure, gap gene, gene regulation, cell cycle, network, gene interactions, signaling network, quantum systems, system dynamics, networks<div style="font-size:11px;"><br><b>Abstract: </b><em>A major aim of systems biology is the Study of the inter-relationships found within and between large biological data sets. Here we describe one systems biology method, in which the tools of network analysis and discrete dynamic Boolean modeling are used to develop predictive models of cellular signaling in cases where detailed temporal and kinetic information regarding the propagation of the signal through the system is lacking. This approach is also applicable to data sets derived from some other types of biological systems, Such as transcription factor-mediated regulation of gene expression during the control of developmental fare, or host defense responses following pathogen attack, and is equally applicable to plant and non-plant systems. The method also allows prediction of how elimination of one or more individual signaling components will affect the ultimate outcome, thus allowing the researcher to model the effects of genetic knockout or pharmacological block. The method also serves as a starting point from which more quantitative models can be developed as additional information becomes available.</div>','<b>Title: </b>COMPLEXITY OF THE SEARCH SPACE IN A MODEL OF ARTIFICIAL EVOLUTION OF GENE REGULATORY NETWORKS CONTROLLING 3D MULTICELLULAR MORPHOGENESIS<br><b>Year: </b>2009<br><b>Author: </b> M. Joachimczak and  B. Wrobel<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1142/S0219525909002246"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ADVANCES IN COMPLEX SYSTEMS<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>The question of what properties of biological systems allow for efficient evolutionary search in complex fitness landscapes evolvability is one of the central interests both for the research in the field of evolutionary biology and artificial life. Here, we attempt to address this issue by using a model of 3D multicellular development in which cell fate is determined by differential gene expression in each cell. In our model, cells can vary in size and can move freely in 3D space, affected by forces of adhesion and repulsion. The development relies on an indirect mapping between the genotype and the morphology the phenotype. Cell differentiation is allowed by positional information provided by diffusible factors. The state of the gene regulatory network GRN coded by the genome determines the cell fate such as division, death, growth. The genetic elements in our systems de. ne points in N-dimensional space. The connectivity in the GRN is determined by the proximity of these points; one can imagine the evolutionary process as their movement in space. Changing the number of dimensions of this space allows to ask directly the questions about the effect of the complexity of the search space on the efficiency of the evolutionary search. Higher dimensionality results in a larger search space, but in our model this search space can still be explored thanks to the action of genetic operators that allow for duplications of genetic elements, a mutational mechanism that allows for regulatory innovations in the network.</div>','<b>Title: </b>Linking gene dynamics to vascular hyperplasia - Toward a predictive model of vein graft adaptation<br><b>Year: </b>2017<br><b>Author: </b> S. Casarin,  Sa. Berceli and  M. Garbey<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0187606"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>systems biology, regulatory network, interaction networks, network model, gene regulatory, network, network biology, networks, gene network, response networks, network dynamics, dynamic model, cell networks, gene expression, biological networks, system dynamics, dynamic network, model, biological systems, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Reductionist approaches, where individual pieces of a process are examined in isolation, have been the mainstay of biomedical research. While these methods are effective in highly compartmentalized systems, they fail to account for the inherent plasticity and non-linearity within the signaling structure. In the current manuscript, we present the computational architecture for tracking an acute perturbation in a biologic system through a multiscale model that links gene dynamics to cell kinetics, with the overall goal of predicting tissue adaptation. Given the complexity of the genome, the problem is made tractable by clustering temporal changes in gene expression into unique patterns. These cluster elements form the core of an integrated network that serves as the driving force for the response of the biologic system. This modeling approach is illustrated using the clinical scenario of vein bypass graft adaptation. Vein segments placed in the arterial circulation for treatment of advanced occlusive disease can develop an aggressive hyperplastic response that narrows the lumen, reduces blood flow, and induces in situ thrombosis. Reducing this hyperplastic response has been a long-standing but unrealized goal of biologic researchers in the field. With repeated failures of single target therapies, the redundant response pathways are thought to be a fundamental issue preventing progress towards a solution. Using the current framework, we demonstrate how theoretical genomic manipulations can be introduced into the system to shift the adaptation to a more beneficial phenotype, where the hyperplastic response is mitigated and the risk of thrombosis reduced. Utilizing our previously published rabbit vein graft genomic data, where grafts were harvested at time points ranging from 2 hours to 28 days and under differential flow conditions, and a customized clustering algorithm, five gene clusters that differentiated the low flow i.e., pro-hyperplastic from high flow i.e., anti-hyperplastic response were identified. The current analysis advances these general associations to create a model that identifies those genes sets most likely to be of therapeutic benefit. Using this approach, we examine the range of potential opportunities for intervention via gene cluster over-expression or inhibition, delivered in isolation or combination, at the time of vein graft implantation.</div>','<b>Title: </b>Establishing positional information through gradient dynamics A lesson from the Hedgehog signaling pathway<br><b>Year: </b>2010<br><b>Author: </b> M. Nahmad and  A. Stathopoulos<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.4161/fly.4.4.12677"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>FLY<br><b>Cluster-Keywords: </b><em>biological system, network, pattern formation, system, network analysis, systems, turing patterns, patterns, reaction networks, networks, turing system, network topologies, stoichiometric network, networks pattern, network topology, biological, model, gradient formation, pattern, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Along standing question in developmental biology is how morphogen gradients establish positional information during development. Although the existence of gradients and their role in developmental patterning is no longer in doubt, the ability of cells to respond to different morphogen concentrations has been controversial. In the Drosophila wing disc, Hedgehog Hh forms a concentration gradient along the anterior-posterior axis and establishes at least three different gene expression patterns. In a recent study, we challenged the prevailing idea that Hh establishes positional information in a dose-dependent manner and proposed a model in which dynamics of the gradient, resulting from the Hh gene network architecture, determines pattern formation in the wing disc. In this Extra View, we discuss further the methodology used in this study, highlight differences between this and other models of developmental patterning, and also present some questions that remain to be answered in this system.</div>','<b>Title: </b>Differential RNAi screening provides insights into the rewiring of signalling networks during oxidative stress<br><b>Year: </b>2012<br><b>Author: </b> Ma. Garcia,  Ms. Alvarez,  H. Sailem,  V. Bousgouni,  J. Sero and  C. Bakal<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1039/c2mb25092f"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MOLECULAR BIOSYSTEMS<br><b>Cluster-Keywords: </b><em>systems biology, regulatory network, interaction networks, network model, gene regulatory, network, network biology, networks, gene network, response networks, network dynamics, dynamic model, cell networks, gene expression, biological networks, system dynamics, dynamic network, model, biological systems, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Reactive Oxygen Species ROS are a natural by-product of cellular growth and proliferation, and are required for fundamental processes such as protein-folding and signal transduction. However, ROS accumulation, and the onset of oxidative stress, can negatively impact cellular and genomic integrity. Signalling networks have evolved to respond to oxidative stress by engaging diverse enzymatic and non-enzymatic antioxidant mechanisms to restore redox homeostasis. The architecture of oxidative stress response networks during periods of normal growth, and how increased ROS levels dynamically reconfigure these networks are largely unknown. In order to gain insight into the structure of signalling networks that promote redox homeostasis we first performed genome-scale RNAi screens to identify novel suppressors of superoxide accumulation. We then infer relationships between redox regulators by hierarchical clustering of phenotypic signatures describing how gene inhibition affects superoxide levels, cellular viability, and morphology across different genetic backgrounds. Genes that cluster together are likely to act in the same signalling pathway/complex and thus make "functional interactions". Moreover we also calculate differential phenotypic signatures describing the difference in cellular phenotypes following RNAi between untreated cells and cells submitted to oxidative stress. Using both phenotypic signatures and differential signatures we construct a network model of functional interactions that occur between components of the redox homeostasis network, and how such interactions become rewired in the presence of oxidative stress. This network model predicts a functional interaction between the transcription factor Jun and the IRE1 kinase, which we validate in an orthogonal assay. We thus demonstrate the ability of systems-biology approaches to identify novel signalling events.</div>','<b>Title: </b>Stochastic simulation of multiscale complex systems with PISKaS: A rule-based approach<br><b>Year: </b>2018<br><b>Author: </b> T. Perez-Acle,  I. Fuenzalida,  Ajm. Martin,  R. Santibanez,  R. Avaria,  A. Bernardin,  Am. Bustos,  D. Garrido,  J. Dushoff and  Jh. Liu<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.bbrc.2017.11.138"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOCHEMICAL AND BIOPHYSICAL RESEARCH COMMUNICATIONS<br><b>Cluster-Keywords: </b><em>regulatory networks, stochastic simulation, gene networks, reaction networks, systems biology, stochastic model, network models, complex systems, networks stochastic, gene regulatory, biological systems, stochastic dynamics, differential equations, protein networks, cell cycle, continuous model, interaction networks, systems chemistry, time delays, biochemical reaction<div style="font-size:11px;"><br><b>Abstract: </b><em>Computational simulation is a widely employed methodology to study the dynamic behavior of complex systems. Although common approaches are based either on ordinary differential equations or stochastic differential equations, these techniques make several assumptions which, when it comes to biological processes, could often lead to unrealistic models. Among others, model approaches based on differential equations entangle kinetics and causality, failing when complexity increases, separating knowledge from models, and assuming that the average behavior of the population encompasses any individual deviation. To overcome these limitations, simulations based on the Stochastic Simulation Algorithm SSA appear as a suitable approach to model complex biological systems. In this work, we review three different models executed in PISKaS: a rule-based framework to produce multiscale stochastic simulations of complex systems. These models span multiple time and spatial scales ranging from gene regulation up to Game Theory. In the first example, we describe a model of the core regulatory network of gene expression in Escherichia coli highlighting the continuous model improvement capacities of PISKaS. The second example describes a hypothetical outbreak of the Ebola virus occurring in a compartmentalized environment resembling cities and highways. Finally, in the last example, we illustrate a stochastic model for the prisoners dilemma; a common approach from social sciences describing complex interactions involving trust within human populations. As whole, these models demonstrate the capabilities of PISKaS providing fertile scenarios where to explore the dynamics of complex systems. C 2017 The Authors. Published by Elsevier Inc.</div>','<b>Title: </b>Coupling actin flow, adhesion, and morphology in a computational cell motility model<br><b>Year: </b>2012<br><b>Author: </b> Dy. Shao,  H. Levine and  Wj. Rappel<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1073/pnas.1203252109"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA<br><b>Cluster-Keywords: </b><em>interaction network, actin filament, network model, filament network, network structures, actin network, protein interaction, actin dynamics, network, membrane protein, network dynamics, model dynamic, cell, model, network approach, protein complex, computational model, actin, adhesion dynamics, cell dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>Cell migration is a pervasive process in many biology systems and involves protrusive forces generated by actin polymerization, myosin dependent contractile forces, and force transmission between the cell and the substrate through adhesion sites. Here we develop a computational model for cell motion that uses the phase-field method to solve for the moving boundary with physical membrane properties. It includes a reaction-diffusion model for the actin-myosin machinery and discrete adhesion sites which can be in a "gripping" or "slipping" mode and integrates the adhesion dynamics with the dynamics of the actin filaments, modeled as a viscous network. To test this model, we apply it to fish keratocytes, fast moving cells that maintain their morphology, and show that we are able to reproduce recent experimental results on actin flow and stress patterns. Furthermore, we explore the phase diagram of cell motility by varying myosin II activity and adhesion strength. Our model suggests that the pattern of the actin flow inside the cell, the cell velocity, and the cell morphology are determined by the integration of actin polymerization, myosin contraction, adhesion forces, and membrane forces.</div>','<b>Title: </b>Prototype of integrated pseudo-dynamic crosstalk network for cancer molecular mechanism<br><b>Year: </b>2013<br><b>Author: </b> Sj. Wu,  Wy. Chen,  Ch. Chou and  Ct. Wu<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.mbs.2013.02.008"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MATHEMATICAL BIOSCIENCES<br><b>Cluster-Keywords: </b><em>signaling network, systems biology, signal transduction, signaling pathways, receptor tyrosine, erbb signaling, erbb receptor, receptor signaling, intracellular signaling, mathematical models, cell signaling, computational model, factor receptor, receptor activation, growth factor, signaling models, networks signaling, receptor system, receptors systems, downstream signaling<div style="font-size:11px;"><br><b>Abstract: </b><em>In this study, we attempted to solve two important challenges in systems biology. First, although the Michaelis-Menten MM model provides local kinetic information, it is hard to generalize MM models to model a large system because increasingly large amounts of experimental data are necessary for the parameter identification. In addition, it is not possible to develop an MM model that provides information about the strength of the interactions in the system. Second, although the dynamic simulation of various signal transduction pathways is important in cancer research, it is impossible to theoretically derive a mathematical model to describe the cancer molecular mechanism. Predictive computational approaches can be used to analyze the dynamics of a system and to determine the dysfunction of a regulatory process. In this report, we first propose a pseudo-dynamic pathway to describe protein interactions in an MM system. We then discuss the dynamic behavior of two large-scale systems antigrowth-signal-induced cell cycle and apoptotic-signal-transduction mechanism. These two systems were constructed through the in-series and organic integration, respectively, of MM modules with Petri net modules; moreover, more than 30 additional reactions were added during this integration step. We then described an extremely large multi-stream system growth signal transduction; however, the analysis of this system to obtain dynamic predictions is critical but appears impossible. Thus, we introduced a fuzzy concept that can be used to develop a physically realizable model prototype. In the future, through step-by-step in vivo modifications, researchers will be able to develop a complete model of cancer metabolism to achieve accurate predictions. C 2013 Elsevier Inc. All rights reserved.</div>','<b>Title: </b>A Feedback Quenched Oscillator Produces Turing Patterning with One Diffuser<br><b>Year: </b>2012<br><b>Author: </b> J. Hsia,  Wj. Holtz,  Dc. Huang,  M. Arcak and  Mm. Maharbiz<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pcbi.1002331"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS COMPUTATIONAL BIOLOGY<br><b>Cluster-Keywords: </b><em>biological system, network, pattern formation, system, network analysis, systems, turing patterns, patterns, reaction networks, networks, turing system, network topologies, stoichiometric network, networks pattern, network topology, biological, model, gradient formation, pattern, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Efforts to engineer synthetic gene networks that spontaneously produce patterning in multicellular ensembles have focused on Turings original model and the "activator-inhibitor" models of Meinhardt and Gierer. Systems based on this model are notoriously difficult to engineer. We present the first demonstration that Turing pattern formation can arise in a new family of oscillator-driven gene network topologies, specifically when a second feedback loop is introduced which quenches oscillations and incorporates a diffusible molecule. We provide an analysis of the system that predicts the range of kinetic parameters over which patterning should emerge and demonstrate the systems viability using stochastic simulations of a field of cells using realistic parameters. The primary goal of this paper is to provide a circuit architecture which can be implemented with relative ease by practitioners and which could serve as a model system for pattern generation in synthetic multicellular systems. Given the wide range of oscillatory circuits in natural systems, our system supports the tantalizing possibility that Turing pattern formation in natural multicellular systems can arise from oscillator-driven mechanisms.</div>','<b>Title: </b>Inverse Eigenvalue Problems for Exploring the Dynamics of Systems Biology Models<br><b>Year: </b>2009<br><b>Author: </b> J. Lu<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.4208/aamm.09-m09S04"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ADVANCES IN APPLIED MATHEMATICS AND MECHANICS<br><b>Cluster-Keywords: </b><em>circadian rhythms, systems biology, biological clock, circadian clock, biological rhythms, hpa axis, model, clock network, mathematical models, biological systems, dynamic systems, ode models, biology models, single cell, network, model parameters, experimental data, models, systems, dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>This paper describes inverse eigenvalue problems that arise in studying qualitative dynamics in systems biology models. An algorithm based on lift-and-project iterations is proposed, where the lifting step entails solving a constrained matrix inverse eigenvalue problem. In particular, prior to carrying out the iterative steps, a-priori bounds on the entries of the Jacobian matrix are computed by relying on the reaction network structure as well as the form of the rate law expressions for the model under consideration. Numerical results on a number of models show that the proposed algorithm can be used to computationally explore the possible dynamical scenarios while identifying the important mechanisms via the use of sparsity-promoting regularization.</div>','<b>Title: </b>Computational approaches to the topology, stability and dynamics of metabolic networks<br><b>Year: </b>2007<br><b>Author: </b> R. Steuer<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.phytochem.2007.04.041"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PHYTOCHEMISTRY<br><b>Cluster-Keywords: </b><em>kinetic models, metabolic networks, kinetic parameters, dynamic models, systems biology, network model, dynamic metabolic, biochemical network, rate constants, enzyme kinetics, kinetic data, adjustable parameters, metabolic model, parameter estimation, dynamic optimization, model selection, mathematical model, kinetic network, model response, model<div style="font-size:11px;"><br><b>Abstract: </b><em>Cellular metabolism is characterized by an intricate network of interactions between biochemical fluxes, metabolic compounds and regulatory interactions. To investigate and eventually understand the emergent global behavior arising from such networks of interaction is not possible by intuitive reasoning alone. This contribution seeks to describe recent computational approaches that aim to asses the topological and functional properties of metabolic networks. In particular, based on a recently proposed method, it is shown that it is possible to acquire a quantitative picture of the possible dynamics of metabolic systems, without assuming detailed knowledge of the underlying enzyme-kinetic rate equations and parameters. Rather, the method builds upon a statistical exploration of the comprehensive parameter space to evaluate the dynamic capabilities of a metabolic system, thus providing a first step towards the transition from topology to function of metabolic pathways. Utilizing this approach, the role of feedback mechanisms in the maintenance of stability is discussed using minimal models of cellular pathways. C 2007 Elsevier Ltd. All rights reserved.</div>','<b>Title: </b>Development of Petri Net-Based Dynamic Model for Improved Production of Farnesyl Pyrophosphate by Integrating Mevalonate and Methylerythritol Phosphate Pathways in Yeast<br><b>Year: </b>2012<br><b>Author: </b> Rr. Baadhe,  Nk. Mekala,  Sr. Palagiri and  Sr. Parcha<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s12010-012-9583-1"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>APPLIED BIOCHEMISTRY AND BIOTECHNOLOGY<br><b>Cluster-Keywords: </b><em>gene expression, systems biology, metabolic network, network model, network analysis, dynamic model, signaling pathways, signaling network, regulatory network, dynamic network, biology approach, candidate genes, gene network, gene regulatory, biology analysis, mathematical model, network dynamic, biology network, metabolic pathways, pathway analysis<div style="font-size:11px;"><br><b>Abstract: </b><em>In this case study, we designed a farnesyl pyrophosphate FPP biosynthetic network using hybrid functional Petri net with extension HFPNe which is derived from traditional Petri net theory and allows easy modeling with graphical approach of various types of entities in the networks together. Our main objective is to improve the production of FPP in yeast, which is further converted to amorphadiene AD, a precursor of artemisinin antimalarial drug. Natively, mevalonate MEV pathway is present in yeast. Methyl erythritol phosphate pathways MEP are present only in higher plant plastids and eubacteria, but not present in yeast. IPP and DAMP are common isomeric intermediate in these two pathways, which immediately yields FPP. By integrating these two pathways in yeast, we augmented the FPP synthesis approximately two folds higher 431.16 U/pt than in MEV pathway alone 259.91 U/pt by using HFPNe technique. Further enhanced FPP levels converted to AD by amorphadiene synthase gene yielding 436.5 U/pt of AD which approximately two folds higher compared to the AD 258.5 U/pt synthesized by MEV pathway exclusively. Simulation and validation processes performed using these models are reliable with identified biological information and data.</div>','<b>Title: </b>Analysis and extension of a biochemical network model using robust control theory<br><b>Year: </b>2010<br><b>Author: </b> Js. Kim,  Nv. Valeyev,  I. Postlethwaite,  P. Heslop-Harrison,  Kh. Cho and  Dg. Bates<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1002/rnc.1528"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>INTERNATIONAL JOURNAL OF ROBUST AND NONLINEAR CONTROL<br><b>Cluster-Keywords: </b><em>sensitivity analysis, systems biology, robustness analysis, network model, network inference, regulatory network, systems analysis, gene expression, parametric uncertainty, biological networks, mathematical models, biochemical networks, biological functions, cellular networks, biological system, parametric sensitivity, dynamical models, dynamic system, networks analysis, gene regulatory<div style="font-size:11px;"><br><b>Abstract: </b><em>Mathematical models of biological processes which have been observed in vivo to be highly robust to intracellular and environmental variations should themselves display appropriate levels of robustness when analysed in silico. This paper uses techniques from robust control theory to analyse and extend a mathematical model of the interacting proteins underlying adenosine 3, 5-cyclic monophosphate cAMP oscillations in aggregating Dictyostelium cells. Starting with a previously proposed minimal model, we show how robustness analysis using the structured singular value can identify points of structural fragility in the network. By combining these results with insights from recent results from the experimental literature, we show how the original model can be augmented with some important additional modules, comprising networks involving IP3 and Ca2. By analysing the robustness of our new extended model, we are able to show that dynamic interactions between the different modules play a pivotal role in generating robust cAMP oscillations; thus, significantly improving our understanding of the design principles underlying this complex biological system. Copyright C 2009 John Wiley  Sons, Ltd.</div>','<b>Title: </b>Dynamics of Influenza Virus and Human Host Interactions During Infection and Replication Cycle<br><b>Year: </b>2013<br><b>Author: </b> A. Madrahimov,  T. Helikar,  B. Kowal,  Gq. Lu and  J. Rogers<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s11538-012-9777-2"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BULLETIN OF MATHEMATICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>network motifs, regulatory network, gene regulatory, systems biology, gene network, bifan motif, complex systems, model comparison, network modeling, complex gene, biological systems, dynamic systems, interaction network, feedforward loop, modeling gene, intelligent systems, modeling approach, biological pathways, motif model, model motifs<div style="font-size:11px;"><br><b>Abstract: </b><em>The replication and life cycle of the influenza virus is governed by an intricate network of intracellular regulatory events during infection, including interactions with an even more complex system of biochemical interactions of the host cell. Computational modeling and systems biology have been successfully employed to further the understanding of various biological systems, however, computational studies of the complexity of intracellular interactions during influenza infection is lacking. In this work, we present the first large-scale dynamical model of the infection and replication cycle of influenza, as well as some of its interactions with the hosts signaling machinery. Specifically, we focus on and visualize the dynamics of the internalization and endocytosis of the virus, replication and translation of its genomic components, as well as the assembly of progeny virions. Simulations and analyses of the models dynamics qualitatively reproduced numerous biological phenomena discovered in the laboratory. Finally, comparisons of the dynamics of existing and proposed drugs, our results suggest that a drug targeting PB1:PA would be more efficient than existing Amantadin/Rimantaine or Zanamivir/Oseltamivir.</div>','<b>Title: </b>Circle maps with gaps: Understanding the dynamics of the two-process model for sleep-wake regulation<br><b>Year: </b>2018<br><b>Author: </b> Mp. Bailey,  G. Derks and  Ac. Skeldon<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1017/S0956792518000190"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>EUROPEAN JOURNAL OF APPLIED MATHEMATICS<br><b>Cluster-Keywords: </b><em>inflammatory response, network model, mechanistic models, lde model, injured patients, critically injured, prediction model, tbi survivors, computational modeling, model abm, impaired astrocytes, model predicts, metabolic dynamics, significant differences, twoprocess model, agent-based model, coupled model, model, inflammatory network, computational models<div style="font-size:11px;"><br><b>Abstract: </b><em>For more than 30 years the two-process model has played a central role in the understanding of sleep/wake regulation. This ostensibly simple model is an interesting example of a nonsmooth dynamical system, whose rich dynamical structure has been relatively unexplored. The two-process model can be framed as a one-dimensional map of the circle, which, for some parameter regimes, has gaps. We show how border collision bifurcations that arise naturally in maps with gaps extend and supplement the Arnold tongue saddle-node bifurcation set that is a feature of continuous circle maps. The novel picture that results shows how the periodic solutions that are created by saddle-node bifurcations in continuous maps transition to periodic solutions created by period-adding bifurcations as seen in maps with gaps.</div>','<b>Title: </b>Inference for reaction networks using the linear noise approximation<br><b>Year: </b>2014<br><b>Author: </b> P. Fearnhead,  V. Giagos and  C. Sherlock<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1111/biom.12152"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOMETRICS<br><b>Cluster-Keywords: </b><em>reaction networks, network model, systems biology, biochemical reaction, stochastic reaction, model reduction, stochastic model, network dynamics, stochastic biochemical, stochastic simulation, system size, chemical reaction, gene network, elementary reactions, biochemical networks, parameter identification, parameter inference, cell populations, small system, linear noise<div style="font-size:11px;"><br><b>Abstract: </b><em>We consider inference for the reaction rates in discretely observed networks such as those found in models for systems biology, population ecology, and epidemics. Most such networks are neither slow enough nor small enough for inference via the true state-dependent Markov jump process to be feasible. Typically, inference is conducted by approximating the dynamics through an ordinary differential equation ODE or a stochastic differential equation SDE. The former ignores the stochasticity in the true model and can lead to inaccurate inferences. The latter is more accurate but is harder to implement as the transition density of the SDE model is generally unknown. The linear noise approximation LNA arises from a first-order Taylor expansion of the approximating SDE about a deterministic solution and can be viewed as a compromise between the ODE and SDE models. It is a stochastic model, but discrete time transition probabilities for the LNA are available through the solution of a series of ordinary differential equations. We describe how a restarting LNA can be efficiently used to perform inference for a general class of reaction networks; evaluate the accuracy of such an approach; and show how and when this approach is either statistically or computationally more efficient than ODE or SDE methods. We apply the LNA to analyze Google Flu Trends data from the North and South Islands of New Zealand, and are able to obtain more accurate short-term forecasts of new flu cases than another recently proposed method, although at a greater computational cost.</div>','<b>Title: </b>AutoBioCAD: Full Biodesign Automation of Genetic Circuits<br><b>Year: </b>2013<br><b>Author: </b> G. Rodrigo and  A. Jaramillo<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1021/sb300084h"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ACS SYNTHETIC BIOLOGY<br><b>Cluster-Keywords: </b><em>gene circuits, synthetic biology, circuit behavior, biological parts, synthetic gene, circuit design, genetic circuits, cell types, metabolite dynamics, mathematical model, designed circuit, control circuit, biological systems, circuit models, synthetic circuit, feedback circuits, biological circuits, design models, metabolic pathways, circuit parameters<div style="font-size:11px;"><br><b>Abstract: </b><em>Synthetic regulatory networks with prescribed functions are engineered by assembling a reduced set of functional elements. We could also assemble them computationally if the mathematical models of those functional elements were predictive enough in different genetic contexts. Only after achieving this will we have libraries of models of biological parts able to provide predictive dynamical behaviors for most circuits constructed with them. We thus need tools that can automatically explore different genetic contexts, in addition to being able to use such libraries to design novel circuits with targeted dynamics. We have implemented a new tool, AutoBioCAD, aimed at the automated design of gene regulatory circuits. AutoBioCAD loads a library of models of genetic elements and implements evolutionary design strategies to produce i nucleotide sequences encoding circuits with targeted dynamics that can then be tested experimentally and ii circuit models for testing regulation principles in natural systems, providing a new tool for synthetic biology. AutoBioCAD can be used to model and design genetic circuits with dynamic behavior, thanks to the incorporation of stochastic effects, robustness, qualitative dynamics, multiobjective optimization, or degenerate nucleotide sequences, all facilitating the link with biological part/circuit engineering.</div>','<b>Title: </b>Applying data mining to learn system dynamics in a biological model<br><b>Year: </b>2006<br><b>Author: </b> Bc. Jeng,  Jx. Chen and  Tp. Liang<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.eswa.2005.09.068"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>EXPERT SYSTEMS WITH APPLICATIONS<br><b>Cluster-Keywords: </b><em>systems biology, regulatory network, interaction networks, network model, gene regulatory, network, network biology, networks, gene network, response networks, network dynamics, dynamic model, cell networks, gene expression, biological networks, system dynamics, dynamic network, model, biological systems, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Data mining consists of a set of powerful methods that have been successfully applied to many different application domains, including business, engineering, and bioinformatics. In this paper, we propose an innovative approach that uses genetic algorithms to mine a set of temporal behavior data output by a biological system in order to determine the kinetic parameters of the system. Analyzing the behavior of a biological network is a complicated task. In our approach, the machine leaming method is integrated with the framework of system dynamics so that its findings are expressed in a form of system dynamics model. An application of the method to the cell division cycle model has shown that the method can discover approximate parametric values of the system and reproduce the input behavior. c 2005 Elsevier Ltd. All rights reserved.</div>','<b>Title: </b>Bistable switches control memory and plasticity in cellular differentiation<br><b>Year: </b>2009<br><b>Author: </b> L. Wang,  Bl. Walker,  S. Iannaccone,  D. Bhatt,  Pj. Kennedy and  Wt. Tse<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1073/pnas.0806137106"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA<br><b>Cluster-Keywords: </b><em>gene regulatory, regulatory network, systems biology, cell differentiation, cell fate, model systems, potential landscape, network grn, dynamical systems, melanocyte differentiation, network models, gene expression, network dynamics, differentiation genes, dynamical models, dynamical networks, cellular differentiation, stem cells, attractor network, developmental gene<div style="font-size:11px;"><br><b>Abstract: </b><em>Development of stem and progenitor cells into specialized tissues in multicellular organisms involves a series of cell fate decisions. Cellular differentiation in higher organisms is generally considered irreversible, and the idea of developmental plasticity in postnatal tissues is controversial. Here, we show that inhibition of mitogen-activated protein kinase MAPK in a human bone marrow stromal cell-derived myogenic subclone suppresses their myogenic ability and converts them into satellite cell-like precursors that respond to osteogenic stimulation. Clonal analysis of the induced osteogenic response reveals ultrasensitivity and an "all-or-none" behavior, hallmarks of a bistable switch mechanism with stochastic noise. The response demonstrates cellular memory, which is contingent on the accumulation of an intracellular factor and can be erased by factor dilution through cell divisions or inhibition of protein synthesis. The effect of MAPK inhibition also exhibits memory and appears to be controlled by another bistable switch further upstream that determines cell fate. Once the memory associated with osteogenic differentiation is erased, the cells regain their myogenic ability. These results support a model of cell fate decision in which a network of bistable switches controls inducible production of lineage-specific differentiation factors. A competitive balance between these factors determines cell fate. Our work underscores the dynamic nature of cellular differentiation and explains mechanistically the dual properties of stability and plasticity associated with the process.</div>','<b>Title: </b>The Max-Min High-Order Dynamic Bayesian Network for Learning Gene Regulatory Networks with Time-Delayed Regulations<br><b>Year: </b>2016<br><b>Author: </b> Yf. Li,  Hf. Chen,  J. Zheng and  A. Ngom<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1109/TCBB.2015.2474409"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IEEE-ACM TRANSACTIONS ON COMPUTATIONAL BIOLOGY AND BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>regulatory networks, gene regulatory, gene expression, network inference, expression data, network structure, gene networks, networks grns, regulatory genes, boolean networks, bayesian network, network reconstruction, networks gene, network dynamics, gene regulation, inference method, systems biology, time series, infer gene, network<div style="font-size:11px;"><br><b>Abstract: </b><em>Accurately reconstructing gene regulatory network GRN from gene expression data is a challenging task in systems biology. Although some progresses have been made, the performance of GRN reconstruction still has much room for improvement. Because many regulatory events are asynchronous, learning gene interactions with multiple time delays is an effective way to improve the accuracy of GRN reconstruction. Here, we propose a new approach, called Max-Min high-order dynamic Bayesian network MMHO-DBN by extending the Max-Min hill-climbing Bayesian network technique originally devised for learning a Bayesian networks structure from static data. Our MMHO-DBN can explicitly model the time lags between regulators and targets in an efficient manner. It first uses constraint-based ideas to limit the space of potential structures, and then applies search-and-score ideas to search for an optimal HO-DBN structure. The performance of MMHO-DBN to GRN reconstruction was evaluated using both synthetic and real gene expression time-series data. Results show that MMHO-DBN is more accurate than current time-delayed GRN learning methods, and has an intermediate computing performance. Furthermore, it is able to learn long time-delayed relationships between genes. We applied sensitivity analysis on our model to study the performance variation along different parameter settings. The result provides hints on the setting of parameters of MMHO-DBN.</div>','<b>Title: </b>From biological pathways to regulatory networks<br><b>Year: </b>2011<br><b>Author: </b> Rk. Layek,  A. Datta and  Er. Dougherty<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1039/c0mb00263a"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MOLECULAR BIOSYSTEMS<br><b>Cluster-Keywords: </b><em>regulatory networks, network dynamics, gene regulatory, network model, genetic network, gene networks, genetic regulatory, mathematical model, parameter space, network system, interaction networks, system dynamics, network structure, network, dynamic networks, network reduction, models models, biological networks, network motifs, biological systems<div style="font-size:11px;"><br><b>Abstract: </b><em>This paper presents a general theoretical framework for generating Boolean networks whose state transitions realize a set of given biological pathways or minor variations thereof. This ill-posed inverse problem, which is of crucial importance across practically all areas of biology, is solved by using Karnaugh maps which are classical tools for digital system design. It is shown that the incorporation of prior knowledge, presented in the form of biological pathways, can bring about a dramatic reduction in the cardinality of the network search space. Constraining the connectivity of the network, the number and relative importance of the attractors, and concordance with observed time-course data are additional factors that can be used to further reduce the cardinality of the search space. The networks produced by the approaches developed here should facilitate the understanding of multivariate biological phenomena and the subsequent design of intervention approaches that are more likely to be successful in practice. As an example, the results of this paper are applied to the widely studied p53 pathway and it is shown that the resulting network exhibits dynamic behavior consistent with experimental observations from the published literature.</div>','<b>Title: </b>Agent-Based Model of Therapeutic Adipose-Derived Stromal Cell Trafficking during Ischemia Predicts Ability To Roll on P-Selectin<br><b>Year: </b>2009<br><b>Author: </b> Am. Bailey,  Mb. Lawrence,  H. Shang,  Aj. Katz and  Sm. Peirce<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pcbi.1000294"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS COMPUTATIONAL BIOLOGY<br><b>Cluster-Keywords: </b><em>inflammatory response, network model, mechanistic models, lde model, injured patients, critically injured, prediction model, tbi survivors, computational modeling, model abm, impaired astrocytes, model predicts, metabolic dynamics, significant differences, twoprocess model, agent-based model, coupled model, model, inflammatory network, computational models<div style="font-size:11px;"><br><b>Abstract: </b><em>Intravenous delivery of human adipose-derived stromal cells hASCs is a promising option for the treatment of ischemia. After delivery, hASCs that reside and persist in the injured extravascular space have been shown to aid recovery of tissue perfusion and function, although low rates of incorporation currently limit the safety and efficacy of these therapies. We submit that a better understanding of the trafficking of therapeutic hASCs through the microcirculation is needed to address this and that selective control over their homing organ- and injury-specific may be possible by targeting bottlenecks in the homing process. This process, however, is incredibly complex, which merited the use of computational techniques to speed the rate of discovery. We developed a multicell agent-based model ABM of hASC trafficking during acute skeletal muscle ischemia, based on over 150 literature-based rules instituted in Netlogo and MatLab software programs. In silico, trafficking phenomena within cell populations emerged as a result of the dynamic interactions between adhesion molecule expression, chemokine secretion, integrin affinity states, hemodynamics and microvascular network architectures. As verification, the model reasonably reproduced key aspects of ischemia and trafficking behavior including increases in wall shear stress, upregulation of key cellular adhesion molecules expressed on injured endothelium, increased secretion of inflammatory chemokines and cytokines, quantified levels of monocyte extravasation in selectin knockouts, and circulating monocyte rolling distances. Successful ABM verification prompted us to conduct a series of systematic knockouts in silico aimed at identifying the most critical parameters mediating hASC trafficking. Simulations predicted the necessity of an unknown selectin-binding molecule to achieve hASC extravasation, in addition to any rolling behavior mediated by hASC surface expression of CD15s, CD34, CD62e, CD62p, or CD65. In vitro experiments confirmed this prediction; a subpopulation of hASCs slowly rolled on immobilized P-selectin at speeds as low as 2 mu m/s. Thus, our work led to a fundamentally new understanding of hASC biology, which may have important therapeutic implications.</div>','<b>Title: </b>Evolution-development congruence in pattern formation dynamics: Bifurcations in gene expression and regulation of networks structures<br><b>Year: </b>2016<br><b>Author: </b> T. Kohsokabe and  K. Kaneko<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1002/jez.b.22666"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF EXPERIMENTAL ZOOLOGY PART B-MOLECULAR AND DEVELOPMENTAL EVOLUTION<br><b>Cluster-Keywords: </b><em>network models, gene network, regulatory networks, boolean networks, gene expression, systems biology, neural networks, network dynamics, gene regulatory, biological networks, network structure, gap gene, gene regulation, cell cycle, network, gene interactions, signaling network, quantum systems, system dynamics, networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Search for possible relationships between phylogeny and ontogeny is important in evolutionary-developmental biology. Here we uncover such relationships by numerical evolution and unveil their origin in terms of dynamical systems theory. By representing developmental dynamics of spatially located cells with gene expression dynamics with cell-to-cell interaction under external morphogen gradient, gene regulation networks are evolved under mutation and selection with the fitness to approach a prescribed spatial pattern of expressed genes. For most numerical evolution experiments, evolution of pattern over generations and development of pattern by an evolved network exhibit remarkable congruence. Both in the evolution and development pattern changes consist of several epochs where stripes are formed in a short time, while for other temporal regimes, pattern hardly changes. In evolution, these quasi-stationary regimes are generations needed to hit relevant mutations, while in development, they are due to some gene expression that varies slowly and controls the pattern change. The morphogenesis is regulated by combinations of feedback or feedforward regulations, where the upstream feedforward network reads the external morphogen gradient, and generates a pattern used as a boundary condition for the later patterns. The ordering from up to downstream is common in evolution and development, while the successive epochal changes in development and evolution are represented as common bifurcations in dynamical-systems theory, which lead to the evolution-development congruence. Mechanism of exceptional violation of the congruence is also unveiled. Our results provide a new look on developmental stages, punctuated equilibrium, developmental bottlenecks, and evolutionary acquisition of novelty in morphogenesis. J. Exp. Zool. Mol. Dev. Evol. 326B:61-84, 2016. c 2015 The Authors. Journal of Experimental Zoology Part B: Molecular and Developmental Evolution Published by Wiley Periodicals, Inc.</div>','<b>Title: </b>Network structure, predator-prey modules, and stability in large food webs<br><b>Year: </b>2008<br><b>Author: </b> S. Allesina and  M. Pascual<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s12080-007-0007-8"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>THEORETICAL ECOLOGY<br><b>Cluster-Keywords: </b><em>network ecology, food webs, network models, ecological systems, food web, systems biology, ecological networks, ecosystem model, signalling networks, model results, interaction networks, dynamic model, ecological interactions, network analysis, model network, network structure, transport networks, systems network, ecosystem dynamics, restoration ecology<div style="font-size:11px;"><br><b>Abstract: </b><em>Large, complex networks of ecological interactions with random structure tend invariably to instability. This mathematical relationship between complexity and local stability ignited a debate that has populated ecological literature for more than three decades. Here we show that, when species interact as predators and prey, systems as complex as the ones observed in nature can still be stable. Moreover, stability is highly robust to perturbations of interaction strength, and is largely a property of structure driven by predator-prey loops with the stability of these small modules cascading into that of the whole network. These results apply to empirical food webs and models that mimic the structure of natural systems as well. These findings are also robust to the inclusion of other types of ecological links, such as mutualism and interference competition, as long as consumer-resource interactions predominate. These considerations underscore the influence of food web structure on ecological dynamics and challenge the current view of interaction strength and long cycles as main drivers of stability in natural communities.</div>','<b>Title: </b>A systems biology analysis of the Drosophila phagosome<br><b>Year: </b>2007<br><b>Author: </b> Lm. Stuart,  J. Boulais,  Gm. Charriere,  Ej. Hennessy,  S. Brunet,  I. Jutras,  G. Goyette,  C. Rondeau,  S. Letarte,  H. Huang,  P. Ye,  F. Morales,  C. Kocks,  Js. Bader,  M. Desjardins and  Rab. Ezekowitz<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1038/nature05380"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>NATURE<br><b>Cluster-Keywords: </b><em>interaction networks, proteinprotein interactions, protein interaction, network model, complex systems, systems biology, protein protein, protein complex, host network, dynamic interactions, network analysis, biological systems, networks network, cellular network, biological network, viral proteins, interactome networks, regulatory networks, immune system, systems network<div style="font-size:11px;"><br><b>Abstract: </b><em>Phagocytes have a critical function in remodelling tissues during embryogenesis and thereafter are central effectors of immune defence1,2. During phagocytosis, particles are internalized into phagosomes, organelles from which immune processes such as microbial destruction and antigen presentation are initiated3. Certain pathogens have evolved mechanisms to evade the immune system and persist undetected within phagocytes, and it is therefore evident that a detailed knowledge of this process is essential to an understanding of many aspects of innate and adaptive immunity. However, despite the crucial role of phagosomes in immunity, their components and organization are not fully defined. Here we present a systems biology analysis of phagosomes isolated from cells derived from the genetically tractable model organism Drosophila melanogaster and address the complex dynamic interactions between proteins within this organelle and their involvement in particle engulfment. Proteomic analysis identified 617 proteins potentially associated with Drosophila phagosomes; these were organized by protein - protein interactions to generate the phagosome interactome, a detailed protein - protein interaction network of this subcellular compartment. These networks predicted both the architecture of the phagosome and putative biomodules. The contribution of each protein and complex to bacterial internalization was tested by RNA-mediated interference and identified known components of the phagocytic machinery. In addition, the prediction and validation of regulators of phagocytosis such as the exocyst4, a macromolecular complex required for exocytosis but not previously implicated in phagocytosis, validates this strategy. In generating this systems-based model, we show the power of applying this approach to the study of complex cellular processes and organelles and expect that this detailed model of the phagosome will provide a new framework for studying host pathogen interactions and innate immunity.</div>','<b>Title: </b>Inference of gene regulatory networks with the strong-inhibition Boolean model<br><b>Year: </b>2011<br><b>Author: </b> Qz. Xia,  Ll. Liu,  Wm. Ye and  G. Hu<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1088/1367-2630/13/8/083002"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>NEW JOURNAL OF PHYSICS<br><b>Cluster-Keywords: </b><em>regulatory networks, gene regulatory, gene expression, network inference, expression data, network structure, gene networks, networks grns, regulatory genes, boolean networks, bayesian network, network reconstruction, networks gene, network dynamics, gene regulation, inference method, systems biology, time series, infer gene, network<div style="font-size:11px;"><br><b>Abstract: </b><em>The inference of gene regulatory networks GRNs is an important topic in biology. In this paper, a logic-based algorithm that infers the strong-inhibition Boolean genetic regulatory networks where regulation by any single repressor can definitely suppress the expression of the gene regulated from time series is discussed. By properly ordering various computation steps, we derive for the first time explicit formulae for the probabilities at which different interactions can be inferred given a certain number of data. With the formulae, we can predict the precision of reconstructions of regulation networks when the data are insufficient. Numerical simulations coincide well with the analytical results. The method and results are expected to be applicable to a wide range of general dynamic networks, where logic algorithms play essential roles in the network dynamics and the probabilities of various logics can be estimated well.</div>','<b>Title: </b>An intelligent two-stage evolutionary algorithm for dynamic pathway identification from gene expression profiles<br><b>Year: </b>2007<br><b>Author: </b> Sy. Ho,  Ch. Hsieh,  Fc. Yu and  Hl. Huang<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1109/TCBB.2007.1051"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IEEE-ACM TRANSACTIONS ON COMPUTATIONAL BIOLOGY AND BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>ssystem model, biological systems, biological networks, parameter estimation, biochemical networks, dynamic biological, proposed algorithm, regulatory networks, structure identification, model structure, genetic algorithm, gene regulatory, gene expression, network inference, network motif, network structure, gene networks, network identification, network inferring, estimation method<div style="font-size:11px;"><br><b>Abstract: </b><em>From gene expression profiles, it is desirable to rebuild cellular dynamic regulation networks to discover more delicate and substantial functions in molecular biology, biochemistry, bioengineering, and pharmaceutics. The S-system model is suitable to characterize biochemical network systems and capable of analyzing the regulatory system dynamics. However, the inference of an S-system model of N-gene genetic networks has 2NN1 parameters in a set of nonlinear differential equations to be optimized. This paper proposes an intelligent two-stage evolutionary algorithm iTEA to efficiently infer the S-system models of genetic networks from time-series data of gene expression. To cope with the curse of dimensionality, the proposed algorithm consists of two stages, where each uses a divide-and-conquer strategy. The optimization problem is first decomposed into N subproblems having 2N1 parameters each. At the first stage, each subproblem is solved using a novel intelligent genetic algorithm IGA with intelligent crossover based on an orthogonal experimental design OED. At the second stage, the obtained N solutions to the N subproblems are combined and refined using an OED-based simulated annealing algorithm for handling noisy gene expression profiles. The effectiveness of iTEA is evaluated using simulated expression patterns with and without noise running on a single-processor PC. It is shown that 1 IGA is efficient enough to solve subproblems, 2 IGA is significantly superior to the existing method GA with simplex crossover SPXGA, and 3 iTEA performs well in inferring S-system models for dynamic pathway identification.</div>','<b>Title: </b>Molecular systems biology and control<br><b>Year: </b>2005<br><b>Author: </b> Ed. Sontag<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3166/ejc.11.396-435"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>EUROPEAN JOURNAL OF CONTROL<br><b>Cluster-Keywords: </b><em>systems biology, biological systems, system dynamics, molecular systems, experimental design, network model, model system, model parameters, experiment design, kinetic parameters, perfect adaptation, experimental data, molecular mechanisms, homeostatic mechanisms, switching dynamics, biological mechanisms, control systems, chemotaxis network, decentralized control, feedback control<div style="font-size:11px;"><br><b>Abstract: </b><em>This paper, prepared for a tutorial at the 2005 IEEE Conference on Decision and Control, presents an introduction to molecular systems biology and some associated problems in control theory. It provides an introduction to basic biological concepts, describes several questions in dynamics and control that arise in the field, and argues that new theoretical problems arise naturally in this context. A final section focuses on the combined use of graph-theoretic, qualitative knowledge about monotone building-blocks and steady-state step responses for components.</div>','<b>Title: </b>Ensembles, dynamics, and cell types: Revisiting the statistical mechanics perspective on cellular regulation<br><b>Year: </b>2019<br><b>Author: </b> S. Bornholdt and  S. Kauffman<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.jtbi.2019.01.036"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF THEORETICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>network models, gene network, regulatory networks, boolean networks, gene expression, systems biology, neural networks, network dynamics, gene regulatory, biological networks, network structure, gap gene, gene regulation, cell cycle, network, gene interactions, signaling network, quantum systems, system dynamics, networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Genetic regulatory networks control ontogeny. For fifty years Boolean networks have served as models of such systems, ranging from ensembles of random Boolean networks as models for generic properties of gene regulation to working dynamical models of a growing number of sub-networks of real cells. At the same time, their statistical mechanics has been thoroughly studied. Here we recapitulate their original motivation in the context of current theoretical and empirical research.We discuss ensembles of random Boolean networks whose dynamical attractors model cell types. A sub-ensemble is the critical ensemble. There is now strong evidence that genetic regulatory networks are dynamically critical, and that evolution is exploring the critical sub-ensemble. The generic properties of this sub-ensemble predict essential features of cell differentiation. In particular, the number of attractors in such networks scales as the DNA content raised to the 0.63 power. Data on the number of cell types as a function of the DNA content per cell shows a scaling relationship of 0.88. Thus, the theory correctly predicts a power law relationship between the number of cell types and the DNA contents per cell, and a comparable slope. We discuss these new scaling values and show prospects for new research lines for Boolean networks as a base model for systems biology. C 2019 Published by Elsevier Ltd.</div>','<b>Title: </b>The Use of Kernel Density Estimation With a Bio-Physical Model Provides a Method to Quantify Connectivity Among Salmon Farms: Spatial Planning and Management With Epidemiological Relevance<br><b>Year: </b>2018<br><b>Author: </b> Dl. Cantrell,  Ee. Rees,  R. Vanderstichel,  J. Grant,  R. Filgueira and  Cw. Revie<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3389/fvets.2018.00269"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>FRONTIERS IN VETERINARY SCIENCE<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Connectivity in an aquatic setting is determined by a combination of hydrodynamic circulation and the biology of the organisms driving linkages. These complex processes can be simulated in coupled biological-physical models. The physical model refers to an underlying circulation model defined by spatially-explicit nodes, often incorporating a particle-tracking model. The particles can then be given biological parameters or behaviors such as maturity and/or survivability rates, diel vertical migrations, avoidance, or seeking behaviors. The output of the bio-physical models can then be used to quantify connectivity among the nodes emitting and/or receiving the particles. Here we propose a method that makes use of kernel density estimation KDE on the output of a particle-tracking model, to quantify the infection or infestation pressure IP that each node causes on the surrounding area. Because IP is the product of both exposure time and the concentration of infectious agent particles, using KDE which also combine elements of time and space, more accurately captures IP. This method is especially useful for those interested in infectious agent networks, a situation where IP is a superior measure of connectivity than the probability of particles from each node reaching other nodes. Here we illustrate the method by modeling the connectivity of salmon farms via sea lice larvae in the Broughton Archipelago, British Columbia, Canada. Analysis revealed evidence of two sub-networks of farms connected via a single farm, and evidence that the highest IP from a given emitting farm was often tens of kilometers or more away from that farm. We also classified farms as net emitters, receivers, or balanced, based on their structural role within the network. By better understanding how these salmon farms are connected to each other via their sea lice larvae, we can effectively focus management efforts to minimize the spread of sea lice between farms, advise on future site locations and coordinated treatment efforts, and minimize any impact of farms on juvenile wild salmon. The method has wide applicability for any system where capturing infectious agent networks can provide useful guidance for management or preventative planning decisions.</div>','<b>Title: </b>Isologous diversification: A theory of cell differentiation<br><b>Year: </b>1997<br><b>Author: </b> K. Kaneko and  T. Yomo<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/BF02459474"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BULLETIN OF MATHEMATICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>An isologous diversification theory for cell differentiation is proposed, based on simulations of interacting cells with biochemical networks and the cell division process following consumption of some chemicals. According to the simulations of the interaction-based dynamical systems model, the following scenario of the cell differentiation is proposed. 1 Up to some threshold number, divisions bring about almost identical cells with synchronized biochemical oscillations. 2 As the number is increased, the oscillations lose synchrony, leading to groups of cells with different phases of oscillations. 3 Amplitudes of oscillation and averaged chemical compositions start to differ by groups of cells. The differentiated behavior of states is transmitted to daughter cells. 4 Recursivity is formed so that the daughter cells keep the identical chemical character. This memory is made possible through the transfer of initial conditions. 5 Successive differentiation proceeds.The mechanism of tumor cell formation, origin of stem cells, anomalous differentiation by transplantations, apoptosis and other features of cell differentiation process are also discussed, with some novel predictions. C 1997 Society for Mathematical Biology</div>','<b>Title: </b>Mesoscopic model of neuronal system deficits in Multiple Sclerosis<br><b>Year: </b>2016<br><b>Author: </b> B. Safarbali,  F. Hadaeghi and  S. Gharibzadeh<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.jtbi.2016.07.013"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF THEORETICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>coupling strength, synchronization transitions, timeperiodic coupling, time delay, logic network, distribution network, network model, oscillatory networks, neuron networks, water distribution, stochastic resonance, gene networks, multiple synchronization, firing synchronization, neuronal networks, firing behaviors, logical networks, results show, regulatory networks, cell polarization<div style="font-size:11px;"><br><b>Abstract: </b><em>Multiple Sclerosis MS is a devastating autoimmune disease which deteriorates the connections in central nervous system CNS through the attacks to oligodendrocytes. Studying its origin and progression, in addition to clinical developments such as MRI brain images, cerebrospinal fluid CSF variation and quantitative measures of disability EDSS, which sought to early diagnosis and efficient therapy, there is an increasing interest in developing computational models using the experimental data obtained from MS patients. From the perspective of mathematical modelling, although the origin of systemic symptoms might be attributed to cellular phenomena in microscopic level such as axonal demyelination, symptoms mainly are observed in macroscopic levels. How to fill the gap between these two levels of system modelling, however, remains as a challenge in systems biology studies. Trying to provide a conceptual framework to bridge between these two levels of modelling in systems biology, we have suggested a mesoscopic model composed of interacting neuronal population, which successfully replicates the changes in neuronal population synchrony due to MS progression. C 2016 Published by Elsevier Ltd.</div>','<b>Title: </b>Exploring the rate-limiting steps in visual phototransduction recovery by bottom-up kinetic modeling<br><b>Year: </b>2013<br><b>Author: </b> Bm. Invergo,  L. Montanucci,  Kw. Koch,  J. Bertranpetit and  D. DellOrco<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1478-811X-11-36"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>CELL COMMUNICATION AND SIGNALING<br><b>Cluster-Keywords: </b><em>systems biology, dynamic network, early differentiation, complex diseases, stem cells, erythroid cells, system dynamics, network biomarkers, immune system, tcm syndrome, dnb genes, gene expression, dnb model, stem cell, expression dynamics, model system, mathematical model, model dynamics, dynamic model, sensitivity analysis<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Phototransduction in vertebrate photoreceptor cells represents a paradigm of signaling pathways mediated by G-protein-coupled receptors GPCRs, which share common modules linking the initiation of the cascade to the final response of the cell. In this work, we focused on the recovery phase of the visual photoresponse, which is comprised of several interacting mechanisms.Results: We employed current biochemical knowledge to investigate the response mechanisms of a comprehensive model of the visual phototransduction pathway. In particular, we have improved the model by implementing a more detailed representation of the recoverin Rec-mediated calcium feedback on rhodopsin kinase and including a dynamic arrestin Arr oligomerization mechanism. The model was successfully employed to investigate the rate limiting steps in the recovery of the rod photoreceptor cell after illumination. Simulation of experimental conditions in which the expression levels of rhodospin kinase RK, of the regulator of the G-protein signaling RGS, of Arr and of Rec were altered individually or in combination revealed severe kinetic constraints to the dynamics of the overall network.Conclusions: Our simulations confirm that RGS-mediated effector shutdown is the rate-limiting step in the recovery of the photoreceptor and show that the dynamic formation and dissociation of Arr homodimers and homotetramers at different light intensities significantly affect the timing of rhodopsin shutdown. The transition of Arr from its oligomeric storage forms to its monomeric form serves to temper its availability in the functional state. Our results may explain the puzzling evidence that overexpressing RK does not influence the saturation time of rod cells at bright light stimuli. The approach presented here could be extended to the study of other GPCR signaling pathways.</div>','<b>Title: </b>Determination of the six rate constants of a three-state enzymatic network and a noninvasive test of detailed balance<br><b>Year: </b>2009<br><b>Author: </b> H. Berthoumieux,  C. Antoine and  A. Lemarchand<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1063/1.3206736"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF CHEMICAL PHYSICS<br><b>Cluster-Keywords: </b><em>kinetic models, metabolic networks, kinetic parameters, dynamic models, systems biology, network model, dynamic metabolic, biochemical network, rate constants, enzyme kinetics, kinetic data, adjustable parameters, metabolic model, parameter estimation, dynamic optimization, model selection, mathematical model, kinetic network, model response, model<div style="font-size:11px;"><br><b>Abstract: </b><em>The Michaelis-Menten mechanism is unanimously recognized by experimentalists and theoreticians as the reference model for the description of enzymatic catalysis. The recent explosion in the diversity of fluorescent probes solves the problem of in situ observation of proteins and the experimental investigation of enzymatic dynamics, which determines the Michaelis constant or a small number of relaxation times, is becoming more and more common. We propose a protocol for the full characterization of enzyme kinetics in the framework of the Michaelis-Menten mechanism. The method relies on the measurement of the oscillation amplitude of the enzymatic concentrations, when the biological medium is submitted to a temperature modulation of a few degrees. Analytical expressions of all the rate constants as functions of the concentration amplitudes are derived. The noninvasive character of the perturbation and the assessable uncertainty on the rate constant values make an in situ test of detailed balance possible.</div>','<b>Title: </b>The slow-scale linear noise approximation: an accurate, reduced stochastic description of biochemical networks under timescale separation conditions<br><b>Year: </b>2012<br><b>Author: </b> P. Thomas,  Av. Straube and  R. Grima<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-6-39"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>reaction networks, network model, systems biology, biochemical reaction, stochastic reaction, model reduction, stochastic model, network dynamics, stochastic biochemical, stochastic simulation, system size, chemical reaction, gene network, elementary reactions, biochemical networks, parameter identification, parameter inference, cell populations, small system, linear noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: It is well known that the deterministic dynamics of biochemical reaction networks can be more easily studied if timescale separation conditions are invoked the quasi-steady-state assumption. In this case the deterministic dynamics of a large network of elementary reactions are well described by the dynamics of a smaller network of effective reactions. Each of the latter represents a group of elementary reactions in the large network and has associated with it an effective macroscopic rate law. A popular method to achieve model reduction in the presence of intrinsic noise consists of using the effective macroscopic rate laws to heuristically deduce effective probabilities for the effective reactions which then enables simulation via the stochastic simulation algorithm SSA. The validity of this heuristic SSA method is a priori doubtful because the reaction probabilities for the SSA have only been rigorously derived from microscopic physics arguments for elementary reactions.Results: We here obtain, by rigorous means and in closed-form, a reduced linear Langevin equation description of the stochastic dynamics of monostable biochemical networks in conditions characterized by small intrinsic noise and timescale separation. The slow-scale linear noise approximation ssLNA, as the new method is called, is used to calculate the intrinsic noise statistics of enzyme and gene networks. The results agree very well with SSA simulations of the non-reduced network of elementary reactions. In contrast the conventional heuristic SSA is shown to overestimate the size of noise for Michaelis-Menten kinetics, considerably under-estimate the size of noise for Hill-type kinetics and in some cases even miss the prediction of noise-induced oscillations.Conclusions: A new general method, the ssLNA, is derived and shown to correctly describe the statistics of intrinsic noise about the macroscopic concentrations under timescale separation conditions. The ssLNA provides a simple and accurate means of performing stochastic model reduction and hence it is expected to be of widespread utility in studying the dynamics of large noisy reaction networks, as is common in computational and systems biology.</div>','<b>Title: </b>Parasite spreading in spatial ecological multiplex networks<br><b>Year: </b>2017<br><b>Author: </b> M. Stella,  Cs. Andreazzi,  S. Selakovic,  A. Goudarzi and  A. Antonioni<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1093/comnet/cnw028"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF COMPLEX NETWORKS<br><b>Cluster-Keywords: </b><em>network ecology, food webs, network models, ecological systems, food web, systems biology, ecological networks, ecosystem model, signalling networks, model results, interaction networks, dynamic model, ecological interactions, network analysis, model network, network structure, transport networks, systems network, ecosystem dynamics, restoration ecology<div style="font-size:11px;"><br><b>Abstract: </b><em>Network ecology is a rising field of quantitative biology representing ecosystems as complex networks. A suitable example is parasite spreading: several parasites may be transmitted among their hosts through different mechanisms, each one giving rise to a network of interactions. Modelling these networked, ecological interactions at the same time is still an open challenge. We present a novel spatially embedded multiplex network framework for modelling multi-host infection spreading through multiple routes of transmission. Our model is inspired by Trypanosoma cruzi, a parasite transmitted by trophic and vectorial mechanisms. Our ecological network model is represented by a multiplex in which nodes represent species populations interacting through a food web and a parasite contaminative layer at the same time. We modelled Susceptible-Infected dynamics in two different scenarios: a simple theoretical food web and an empirical one. Our simulations in both scenarios showthat the infection is more widespread when both the trophic and the contaminative interactions are considered with equal rates. This indicates that trophic and contaminative transmission may have additive effects in real ecosystems. We also find that the ratio of vectors-to-host in the community i crucially influences the infection spread, ii regulates a percolating phase transition in the rate of parasite transmission and iii increases the infection rate in hosts. By immunizing the same fractions of predator and prey populations, we show that the multiplex topology is fundamental in outlining the role that each host species plays in parasite transmission in a given ecosystem. We also show that the multiplex models provide a richer phenomenology in terms of parasite spreading dynamics compared to more limited mono-layer models. Our work opens new challenges and provides new quantitative tools for modelling multi-channel spreading in networked systems.</div>','<b>Title: </b>Immune network for viral hepatitis B: Topological representation<br><b>Year: </b>2019<br><b>Author: </b> E. Asin-Prieto,  Zp. Parra-Guillen,  Jdg. Mantilla,  J. Vandenbossche,  K. Stuyckens,  Xw. De Trixhe,  Jj. Perez-Ruixo and  If. Troconiz<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.ejps.2019.05.017"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>EUROPEAN JOURNAL OF PHARMACEUTICAL SCIENCES<br><b>Cluster-Keywords: </b><em>systems biology, dynamic network, early differentiation, complex diseases, stem cells, erythroid cells, system dynamics, network biomarkers, immune system, tcm syndrome, dnb genes, gene expression, dnb model, stem cell, expression dynamics, model system, mathematical model, model dynamics, dynamic model, sensitivity analysis<div style="font-size:11px;"><br><b>Abstract: </b><em>The liver is a well-known immunotolerogenic environment, which provides the adequate setting for liver infectious pathogens persistence such as the hepatitis B virus HBV. Consequently, HBV infection can derive in the development of chronic disease in a proportion of the patients. If this situation persists in time, chronic hepatitis B CHB would end in cirrhosis, hepatocellular carcinoma and eventually, the death of the patient. It is thought that this immunotolerogenic environment is the result of complex interactions between different elements of the immune system and the viral biology. Therefore, the purpose of this work is to unravel the mechanisms implied in the development of CHB and to design a tool able to help in the study of adequate therapies. Firstly, a conceptual framework with the main components of the immune system and viral dynamics was constructed providing an overall insight on the pathways and interactions implied in this disease. Secondly, a review of the literature was performed in a modular fashion: i viral dynamics, ii innate immune response, iii humoral and iv cellular adaptive immune responses and v tolerogenic aspects. Finally, the information collected was integrated into a single topological representation that could serve as the plan for the systems pharmacology model architecture. This representation can be considered as the previous unavoidable step to the construction of a quantitative model that could assist in biomarker and target identification, drug design and development, dosing optimization and disease progression analysis.</div>','<b>Title: </b>Prospects of a computational origin of life endeavor<br><b>Year: </b>2004<br><b>Author: </b> B. Shenhav and  D. Lancet<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1023/B:ORIG.0000009839.53483.42"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ORIGINS OF LIFE AND EVOLUTION OF BIOSPHERES<br><b>Cluster-Keywords: </b><em>regulatory networks, stochastic simulation, gene networks, reaction networks, systems biology, stochastic model, network models, complex systems, networks stochastic, gene regulatory, biological systems, stochastic dynamics, differential equations, protein networks, cell cycle, continuous model, interaction networks, systems chemistry, time delays, biochemical reaction<div style="font-size:11px;"><br><b>Abstract: </b><em>While the last century brought an exquisite understanding of the molecular basis of life, very little is known about the detailed chemical mechanisms that afforded the emergence of life on early earth. There is a broad agreement that the problem lies in the realm of chemistry, and likely resides in the formation and mutual interactions of carbon-based molecules in aqueous medium. Yet, present-day experimental approaches can only capture the synthesis and behavior of a few molecule types at a time. On the other hand, experimental simulations of prebiotic syntheses, as well as chemical analyses of carbonaceous meteorites, suggest that the early prebiotic hydrosphere contained many thousands of different compounds. The present paper explores the idea that given the limitations of test-tube approaches with regards to such a random chemistry scenario, an alternative mode of analysis should be pursued. It is argued that as computational tools for the reconstruction of molecular interactions improve rapidly, it may soon become possible to perform adequate computer-based simulations of prebiotic evolution. We thus propose to launch a computational origin of life endeavor http://ool.weizmann.ac.il/CORE, involving computer simulations of realistic complex prebiotic chemical networks. In the present paper we provide specific examples, based on a novel algorithmic approach, which constitutes a hybrid of molecular dynamics and stochastic chemistry. As one potential solution for the immense hardware requirements dictated by this approach, we have begun to implement an idle CPU harvesting scheme, under the title oolhome.</div>','<b>Title: </b>Decoding complex biological networks - tracing essential and modulatory parameters in complex and simplified models of the cell cycle<br><b>Year: </b>2011<br><b>Author: </b> O. Eriksson,  T. Andersson,  Ys. Zhou and  J. Tegner<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-5-123"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>network models, gene network, regulatory networks, boolean networks, gene expression, systems biology, neural networks, network dynamics, gene regulatory, biological networks, network structure, gap gene, gene regulation, cell cycle, network, gene interactions, signaling network, quantum systems, system dynamics, networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: One of the most well described cellular processes is the cell cycle, governing cell division. Mathematical models of this gene-protein network are therefore a good test case for assessing to what extent we can dissect the relationship between model parameters and system dynamics. Here we combine two strategies to enable an exploration of parameter space in relation to model output. A simplified, piecewise linear approximation of the original model is combined with a sensitivity analysis of the same system, to obtain and validate analytical expressions describing the dynamical role of different model parameters.Results: We considered two different output responses to parameter perturbations. One was qualitative and described whether the system was still working, i.e. whether there were oscillations. We call parameters that correspond to such qualitative change in system response essential. The other response pattern was quantitative and measured changes in cell size, corresponding to perturbations of modulatory parameters. Analytical predictions from the simplified model concerning the impact of different parameters were compared to a sensitivity analysis of the original model, thus evaluating the predictions from the simplified model. The comparison showed that the predictions on essential and modulatory parameters were satisfactory for small perturbations, but more discrepancies were seen for larger perturbations. Furthermore, for this particular cell cycle model, we found that most parameters were either essential or modulatory. Essential parameters required large perturbations for identification, whereas modulatory parameters were more easily identified with small perturbations. Finally, we used the simplified model to make predictions on critical combinations of parameter perturbations.Conclusions: The parameter characterizations of the simplified model are in large consistent with the original model and the simplified model can give predictions on critical combinations of parameter perturbations. We believe that the distinction between essential and modulatory perturbation responses will be of use for sensitivity analysis, and in discussions of robustness and during the model simplification process.</div>','<b>Title: </b>Reconstructing the genotype-to-fitness map for the bacterial chemotaxis network and its emergent behavioural phenotypes<br><b>Year: </b>2017<br><b>Author: </b> A. Nakauma and  Gs. Van Doorn<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.jtbi.2017.03.016"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF THEORETICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>systems biology, biological systems, system dynamics, molecular systems, experimental design, network model, model system, model parameters, experiment design, kinetic parameters, perfect adaptation, experimental data, molecular mechanisms, homeostatic mechanisms, switching dynamics, biological mechanisms, control systems, chemotaxis network, decentralized control, feedback control<div style="font-size:11px;"><br><b>Abstract: </b><em>The signal-transduction network responsible for chemotaxis in Escherichia coli has been characterised in extraordinary detail. Yet, relatively little is known about eco-evolutionary aspects of chemotaxis, such as how the network has been shaped by selection and to what extent natural populations may fine-tune their chemotactic behaviour to the ecological conditions. To address these questions, we here develop an evolutionary-systems biology model of the chemotaxis network of E. coli, which we apply to estimate the resource accumulation rate here used as a proxy for fitness of wildtype and a large number of potential mutant genotypes. Mutant genotypes differ from the wildtype in the concentrations of one or more constituent proteins of the chemotaxis signalling network or in one or more of its kinetic parameters. To guarantee model consistency across the genotype space, we explicitly incorporated biochemical constraints that underly observed phenotypic trade-offs. The model was validated by reconstructing the phenotypic properties of several known mutant genotypes. We also characterised differences in the fitness distribution between genotypes, and reconstructed adaptive walks in genotype space for populations exposed to different environmental conditions. We found that the local fitness landscape is rugged, due to non-additive interactions between mutations. When selection has a consistent direction, just a few adaptive mutations are required to reach a local peak, and different local peaks can be reached by adaptive walks starting from the same initial genotype. However, when the direction of selection is fluctuating, evolutionary paths are much longer and genotype space is explored further. Longer adaptive walks were also observed when evolution was started from a low-fitness genotype such as a CheZ knockout mutant. In line with empirical observations, the initial Delta cheZ mutant did not respond to a step-down stimulus, but a dynamic response similar to the wildtype was recovered following the fixation of compensatory mutations.</div>','<b>Title: </b>Identification of cell-to-cell heterogeneity through systems engineering approaches<br><b>Year: </b><br><b>Author: </b> D. Lee,  A. Jayaraman and  Jsi. Kwon<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1002/aic.16925"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>AICHE JOURNAL<br><b>Cluster-Keywords: </b><em>reaction networks, network model, systems biology, biochemical reaction, stochastic reaction, model reduction, stochastic model, network dynamics, stochastic biochemical, stochastic simulation, system size, chemical reaction, gene network, elementary reactions, biochemical networks, parameter identification, parameter inference, cell populations, small system, linear noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Cells in a genetically homogeneous cell-population exhibit a significant degree of heterogeneity in their responses to an external stimulus. To understand origins and importance of this heterogeneity, individual-based population model IBPM, where parameters follow probability density functions PDFs instead of being constants, has been previously developed. However, parameter identification for an IBPM is challenging as estimating PDFs is computationally expensive. Also, because of experimental limitations and nonlinearity of models, not all parameters PDFs are identifiable. Motivated by the above considerations, a new methodology is proposed in this study. First, a subset of parameters whose PDFs is identifiable are determined through sensitivity analysis, and only these PDFs are estimated. Second, an artificial neural network model is developed to find an empirical relation between these parameter and output PDFs to reduce computational costs of the parameter identification. The proposed approach is validated by estimating PDFs of parameters of a tumor necrosis factor-alpha signaling model.</div>','<b>Title: </b>Hybrid Dynamic Optimization Methods for Systems Biology with Efficient Sensitivities<br><b>Year: </b>2015<br><b>Author: </b> Nr. Lewis,  Jd. Hedengren and  El. Haseltine<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3390/pr3030701"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PROCESSES<br><b>Cluster-Keywords: </b><em>systems biology, dynamic network, early differentiation, complex diseases, stem cells, erythroid cells, system dynamics, network biomarkers, immune system, tcm syndrome, dnb genes, gene expression, dnb model, stem cell, expression dynamics, model system, mathematical model, model dynamics, dynamic model, sensitivity analysis<div style="font-size:11px;"><br><b>Abstract: </b><em>In recent years, model optimization in the field of computational biology has become a prominent area for development of pharmaceutical drugs. The increased amount of experimental data leads to the increase in complexity of proposed models. With increased complexity comes a necessity for computational algorithms that are able to handle the large datasets that are used to fit model parameters. In this study the ability of simultaneous, hybrid simultaneous, and sequential algorithms are tested on two models representative of computational systems biology. The first case models the cells affected by a virus in a population and serves as a benchmark model for the proposed hybrid algorithm. The second model is the ErbB model and shows the ability of the hybrid sequential and simultaneous method to solve large-scale biological models. Post-processing analysis reveals insights into the model formulation that are important for understanding the specific parameter optimization. A parameter sensitivity analysis reveals shortcomings and difficulties in the ErbB model parameter optimization due to the model formulation rather than the solver capacity. Suggested methods are model reformulation to improve input-to-output model linearity, sensitivity ranking, and choice of solver.</div>','<b>Title: </b>A Spatial Filtering Approach to Biological Patterning<br><b>Year: </b>2019<br><b>Author: </b> Ml. Perkins and  M. Arcak<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1137/18M1216092"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SIAM JOURNAL ON APPLIED DYNAMICAL SYSTEMS<br><b>Cluster-Keywords: </b><em>biological system, network, pattern formation, system, network analysis, systems, turing patterns, patterns, reaction networks, networks, turing system, network topologies, stoichiometric network, networks pattern, network topology, biological, model, gradient formation, pattern, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Interactions between neighboring cells are essential for generating or refining patterns in a number of biological systems. We propose a discrete filtering approach to predict how networks of cells modulate spatially varying input signals to produce more complicated or precise output signals. The interconnections between cells determine the set of spatial modes that are amplified or suppressed based on the coupling and internal dynamics of each cell, analogously to the way a traditional digital filter modifies the frequency components of a discrete signal. We apply the framework to two systems in developmental biology, the Notch-Delta interaction that shapes Drosophila wing veins and the Sox9/Bmp/Wnt network responsible for digit formation in vertebrate limbs, as well as to one synthetic biological system that acts as an edge detector. In the course of our analysis, we identify experiments to differentiate between similar models and find that Turing-like patterns may occur even in the absence of instabilities. Results also indicate patterning systems may be inherently robust to both correlated and uncorrelated noise sources. Our work shows that a spatial frequency-based interpretation simplifies the process of predicting patterning in living organisms when both environmental influences and intercellular interactions are present, and may facilitate designs for synthetic multicellular systems that execute image processing tasks in vivo.</div>','<b>Title: </b>Modern temporal network theory: a colloquium<br><b>Year: </b>2015<br><b>Author: </b> P. Holme<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1140/epjb/e2015-60657-4"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>EUROPEAN PHYSICAL JOURNAL B<br><b>Cluster-Keywords: </b><em>complex networks, complex systems, network theory, network structure, biological network, social network, network science, dynamic networks, network systems, network model, network topology, systems biology, network, network analysis, system dynamics, networks, realworld networks, network automata, epidemic network, model network<div style="font-size:11px;"><br><b>Abstract: </b><em>The power of any kind of network approach lies in the ability to simplify a complex system so that one can better understand its function as a whole. Sometimes it is beneficial, however, to include more information than in a simple graph of only nodes and links. Adding information about times of interactions can make predictions and mechanistic understanding more accurate. The drawback, however, is that there are not so many methods available, partly because temporal networks is a relatively young field, partly because it is more difficult to develop such methods compared to for static networks. In this colloquium, we review the methods to analyze and model temporal networks and processes taking place on them, focusing mainly on the last three years. This includes the spreading of infectious disease, opinions, rumors, in social networks; information packets in computer networks; various types of signaling in biology, and more. We also discuss future directions.</div>','<b>Title: </b>Self-organization, scale and stability in a spatial predator-prey interaction<br><b>Year: </b>2000<br><b>Author: </b> Wsc. Gurney and  Ar. Veitch<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1006/bulm.1999.0130"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BULLETIN OF MATHEMATICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>chaotic system, control systems, neural networks, jerk chaotic, torch chaotic, mode control, sliding mode, plasma torch, dynamic neural, mitochondrial dynamics, system, systems biology, adaptive integral, model system, rodtype plasma, integral sliding, systems, jerk system, cancer cell, comparison method<div style="font-size:11px;"><br><b>Abstract: </b><em>Simple predator-prey models often predict extreme instability in interactions where the prey are depressed well below their carrying capacity. Although the behaviour of some laboratory systems conforms to this pattern, field and mesocosm studies generally show prolonged co-existence of prey and predator. Prominent among the possible causes of this discrepancy are the effects of spatial heterogeneity. In this paper we show that both discrete and continuous representations of the spatial Rosenzweig-McArthur model with immobile prey can be stabilized by self-organized prey heterogeneity. This concordance of behaviour closely parallels that which we have previously established in the context of invasion waves. We use the continuous model variant to calculate the characteristic spatial scales of the self-organized structures. The discrete variant forms the basis of a simulation study demonstrating the variety of stable structures and elucidating their relation to the history of the system. We note that all stable prey distributions take the form of a network of occupied patches separated by prey-free regions, and liken the process which generates such assemblages to the formation of a landscape mozaic. C 2000 Society for Mathematical Biology.</div>','<b>Title: </b>Expanding the Logic of Bacterial Promoters Using Engineered Overlapping Operators for Global Regulators<br><b>Year: </b>2014<br><b>Author: </b> Me. Guazzaroni and  R. Silva-Rocha<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1021/sb500084f"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ACS SYNTHETIC BIOLOGY<br><b>Cluster-Keywords: </b><em>blood flow, blood cells, flow rate, blood pumps, cells, cell, results, patients, stem cells, methods, blood, patients blood, flow, patients methods, patients objectives, cell viability, objectives, results cells, blood pressure, cell culture<div style="font-size:11px;"><br><b>Abstract: </b><em>The understanding of how the architecture of cis-regulatory elements at bacterial promoters determines their final output is of central interest in modern biology. In this work, we attempt to gain insight into this process by analyzing complex promoter architectures in the model organism Escherichia coli. By focusing on the relationship between different TFs at the genomic scale in terms of their binding site arrangement and their effect on the target promoters, we found no strong constraint limiting the combinatorial assembly of TF pairs in E. coli. More strikingly, overlapping binding sites were found equally associated with both equivalent both TFs have the same this information on hand, we set an in salt approach to design overlapping sites for three global regulators GRs of E. coli, specifically CRP, Fis, and IHF. Using random sequence assembly and an evolutionary algorithm, we were able to identify potential overlapping operators for all TF pairs. In order to validate our prediction, we constructed two lac promoter variants containing overlapping sites for CRP and IHF designed in silk. By assaying the synthetic promoters using a GFP reporter system, we demonstrated that these variants were functional and activated by CRP and IHF in vivo. Taken together, presented results add new information on the mechanisms of signal integration in bacterial promoters and provide new strategies for the engineering of synthetic regulatory circuits in bacteria.</div>','<b>Title: </b>Approach of Complex-Systems Biology to Reproduction and Evolution<br><b>Year: </b>2011<br><b>Author: </b> K. Kaneko<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/978-3-642-18137-510"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PRINCIPLES OF EVOLUTION: FROM THE PLANCK EPOCH TO COMPLEX MULTICELLULAR LIFE<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Two basic issues in biology the origin of life and evolution of phenotypes are discussed on the basis of statistical physics and dynamical systems. In section "A Bridge Between Catalytic Reaction Networks and Reproducing Cells", we survey recent developments in the origin of reproducing cells from an ensemble of catalytic reactions. After surveying several models of catalytic reaction networks briefly, we provide possible answers to the following three questions: 1 How are nonequilibrium states sustained in catalytic reaction dynamics 2 How is recursive production of a cell maintaining composition of a variety of chemicals possible 3 How does a specific molecule species carry information for heredity In section "Evolution", general relationships between plasticity, robustness, and evolvability are presented in terms of phenotypic fluctuations. First, proportionality between evolution speed, phenotypic plasticity, and isogenic phenotypic fluctuation is proposed by extending the fluctuation response relationship in physics. We then derive a general proportionality relationship between the phenotypic fluctuations of epigenetic and genetic origin: the former is the variance of phenotype due to noise in the developmental process, and the latter due to genetic mutation. The relationship also suggests a link between robustness to noise and to mutation. These relationships are confirmed in models of gene expression dynamics, as well as in laboratory experiments, and then are explained by a theory based on an evolutionary stability hypothesis For both sections "A Bridge Between Catalytic Reaction Networks and Reproducing Cells" and "Evolution", consistency between two levels of hierarchy i.e., molecular and cellular, or genetic and phenotypic levels is stressed as a principle for complex-systems biology.</div>','<b>Title: </b>Robustification as a tool in modeling biochemical reaction networks<br><b>Year: </b>2011<br><b>Author: </b> Ew. Jacobsen and  V. Nenchev<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.jprocont.2011.05.009"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF PROCESS CONTROL<br><b>Cluster-Keywords: </b><em>sensitivity analysis, systems biology, robustness analysis, network model, network inference, regulatory network, systems analysis, gene expression, parametric uncertainty, biological networks, mathematical models, biochemical networks, biological functions, cellular networks, biological system, parametric sensitivity, dynamical models, dynamic system, networks analysis, gene regulatory<div style="font-size:11px;"><br><b>Abstract: </b><em>Biological functions have evolved to become robust against a multitude of perturbations such as gene mutations, intracellular noise and changes in the physical and chemical environment. This robustness should be reflected in models of the underlying biochemical networks, and robustness analysis has consequently been established as an important tool for model validation in systems biology. However, while robustness analysis can be used to invalidate a given model, it does not support the postulation of model modifications that can serve to improve the robustness. Herein we propose a method for this purpose, based on computing the sensitivity of the robustness with respect to generic dynamic perturbations applied to the individual network edges. To quantify robustness we compute the smallest simultaneous change in the activity of the network nodes that induces a bifurcation in the network, resulting in a qualitative change in the network behavior. The proposed method can be used to identify network interactions with the most significant impact on the overall robustness of a given function. By considering the impact of adding new nodes and edges, the method can also be used to postulate the presence of important unmodeled components and interactions. The focus here is on biological functions related to bistable switches and sustained oscillations, and the proposed methodology is demonstrated through application to metabolic oscillations in white blood cells and bistable switching in MAPK signal transduction. The classical Goodwin model of oscillations in a simple gene regulatory network is used for illustration throughout the paper. C 2011 Elsevier Ltd. All rights reserved.</div>','<b>Title: </b>Modeling nonlinear dynamic biological systems with human-readable fuzzy rules optimized by convergent heterogeneous particle swarm<br><b>Year: </b>2015<br><b>Author: </b> Nj. Cheung,  Zk. Xu,  Xm. Ding and  Hb. Shen<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.ejor.2015.03.047"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>EUROPEAN JOURNAL OF OPERATIONAL RESEARCH<br><b>Cluster-Keywords: </b><em>ssystem model, biological systems, biological networks, parameter estimation, biochemical networks, dynamic biological, proposed algorithm, regulatory networks, structure identification, model structure, genetic algorithm, gene regulatory, gene expression, network inference, network motif, network structure, gene networks, network identification, network inferring, estimation method<div style="font-size:11px;"><br><b>Abstract: </b><em>It is an important issue to model the dynamic biological networks from their time-course response datasets, which is critical to understand the system behaviors. This task includes two sub-tasks: network structure identification and associated parameters estimation. In most existing methods, the two sub-tasks are dealt with step by step, which may result in inconsistence between them and hence inaccuracy of the final model. Another challenge is how to transparently understand the derived model, which cannot be achieved by the traditional black-box methods. A human readable fuzzy rule-based model, denoted as MoPath, is developed for the identification of both structure topology and associated parameters, simultaneously, of a biological network within an optimization framework. MoPath encodes the fuzzy rules into particles of heterogeneous particle swarm optimization CHPSO algorithm to generate the optimal model. Theoretically, we demonstrate that the cooperation in CHPSO can maintain a balance between exploration and exploitation to guarantee the particles converge to stable points, which is greatly helpful for finding the optimal model consisting of both the network topology and parameters. We demonstrate MoPath on two dynamic biological networks, and successfully generate a few human readable rules that can well represent the network with high accuracy and good robustness. C 2015 Elsevier B.V. and Association of European Operational Research Societies EURO within the International Federation of Operational Research Societies IFORS. All rights reserved.</div>','<b>Title: </b>Stochastic modeling of human papillomavirus early promoter gene regulation<br><b>Year: </b>2020<br><b>Author: </b> A. Giaretta,  Gm. Toffolo and  Tc. Elston<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.jtbi.2019.110057"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF THEORETICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>gene expression, gene network, stochastic model, gene regulatory, systems biology, expression systems, gene regulation, network activity, expression dynamics, noise control, regulatory network, deterministic models, network components, stochastic modeling, regulation system, escherichia coli, regulatory system, synthetic gene, regulatory genes, gene<div style="font-size:11px;"><br><b>Abstract: </b><em>High risk forms of human papillomaviruses HPVs promote cancerous lesions and are implicated in almost all cervical cancer. Of particular relevance to cancer progression is regulation of the early promoter that controls gene expression in the initial phases of infection and can eventually lead to pre-cancer progression. Our goal was to develop a stochastic model to investigate the control mechanisms that regulate gene expression from the HPV early promoter. Our model integrates modules that account for transcriptional, post-transcriptional, translational and post-translational regulation of El and E2 early genes to form a functioning gene regulatory network. Each module consists of a set of biochemical steps whose stochastic evolution is governed by a chemical Master Equation and can be simulated using the Gillespie algorithm. To investigate the role of noise in gene expression, we compared our stochastic simulations with solutions to ordinary differential equations for the mean behavior of the system that are valid under the conditions of large molecular abundances and quasi-equilibrium for fast reactions.The model produced results consistent with known HPV biology. Our simulation results suggest that stochasticity plays a pivotal role in determining the dynamics of HPV gene expression. In particular, the combination of positive and negative feedback regulation generates stochastic bursts of gene expression. Analysis of the model reveals that regulation at the promoter affects burst amplitude and frequency, whereas splicing is more specialized to regulate burst frequency. Our results also suggest that splicing enhancers are a significant source of stochasticity in pre-mRNA abundance and that the number of viruses infecting the host cell represents a third important source of stochasticity in gene expression. C 2019 Elsevier Ltd. All rights reserved.</div>','<b>Title: </b>MICROELECTRONICS FOR DYNAMIC NEURAL NETWORKS<br><b>Year: </b>1993<br><b>Author: </b> Sj. Prange,  A. Jahnke and  H. KlarLink: No link<br><b>Journal: </b>ANNALES DES TELECOMMUNICATIONS-ANNALS OF TELECOMMUNICATIONS<br><b>Cluster-Keywords: </b><em>chaotic system, control systems, neural networks, jerk chaotic, torch chaotic, mode control, sliding mode, plasma torch, dynamic neural, mitochondrial dynamics, system, systems biology, adaptive integral, model system, rodtype plasma, integral sliding, systems, jerk system, cancer cell, comparison method<div style="font-size:11px;"><br><b>Abstract: </b><em>Artificial Neural Networks are the massively parallel interconnection of simple processing elements. Computing times for the simulation of these parallel systems on todays von-Neumann-computers increase with the squared number of processing elements. There is a need for application specific hardware. This paper describes various investigations of analog as well as digital hardware for neural networks. Possible solutions for the connection problem and different circuit designs will be explained. Then our cascadable digital circuit for the emulation of a biology-oriented, dynamic neural network will be presented.</div>','<b>Title: </b>Identification of a Topological Characteristic Responsible for the Biological Robustness of Regulatory Networks<br><b>Year: </b>2009<br><b>Author: </b> Yl. Wu,  Xm. Zhang,  Jl. Yu and  Q. Ouyang<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pcbi.1000442"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS COMPUTATIONAL BIOLOGY<br><b>Cluster-Keywords: </b><em>network models, gene network, regulatory networks, boolean networks, gene expression, systems biology, neural networks, network dynamics, gene regulatory, biological networks, network structure, gap gene, gene regulation, cell cycle, network, gene interactions, signaling network, quantum systems, system dynamics, networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Attribution of biological robustness to the specific structural properties of a regulatory network is an important yet unsolved problem in systems biology. It is widely believed that the topological characteristics of a biological control network largely determine its dynamic behavior, yet the actual mechanism is still poorly understood. Here, we define a novel structural feature of biological networks, termed regulation entropy, to quantitatively assess the influence of network topology on the robustness of the systems. Using the cell-cycle control networks of the budding yeast Saccharomyces cerevisiae and the fission yeast Schizosaccharomyces pombe as examples, we first demonstrate the correlation of this quantity with the dynamic stability of biological control networks, and then we establish a significant association between this quantity and the structural stability of the networks. And we further substantiate the generality of this approach with a broad spectrum of biological and random networks. We conclude that the regulation entropy is an effective order parameter in evaluating the robustness of biological control networks. Our work suggests a novel connection between the topological feature and the dynamic property of biological regulatory networks.</div>','<b>Title: </b>The JigCell Model Builder: A spreadsheet interface for creating biochemical reaction network models<br><b>Year: </b>2006<br><b>Author: </b> Mt. Vass,  Ca. Shaffer,  N. Ramakrishnan,  Lt. Watson and  Jj. Tyson<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1109/TCBB.2006.27"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IEEE-ACM TRANSACTIONS ON COMPUTATIONAL BIOLOGY AND BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Converting a biochemical reaction network to a set of kinetic rate equations is tedious and error prone. We describe known interface paradigms for inputing models of intracellular regulatory networks: graphical layout diagrams, wizards, scripting languages, and direct entry of chemical equations. We present the JigCell Model Builder, which allows users to define models as a set of reaction equations using a spreadsheet an example of direct entry of equations and outputs model definitions in the Systems Biology Markup Language, Level 2. We present the results of two usability studies. The spreadsheet paradigm demonstrated its effectiveness in reducing the number of errors made by modelers when compared to hand conversion of a wiring diagram to differential equations. A comparison of representatives of the four interface paradigms for a simple model of the cell cycle was conducted which measured time, mouse clicks, and keystrokes to enter the model, and the number of screens needed to view the contents of the model. All four paradigms had similar data entry times. The spreadsheet and scripting language approaches require significantly fewer screens to view the models than do the wizard or graphical layout approaches.</div>','<b>Title: </b>Optimal Objective-Based Experimental Design for Uncertain Dynamical Gene Networks with Experimental Error<br><b>Year: </b>2018<br><b>Author: </b> Dn. Mohsenizadeh,  R. Dehghannasiri and  Er. Dougherty<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1109/TCBB.2016.2602873"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IEEE-ACM TRANSACTIONS ON COMPUTATIONAL BIOLOGY AND BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>boolean networks, regulatory networks, genetic regulatory, gene regulatory, probabilistic boolean, network, networks, model, network model, probabilistic model, model checking, networks pbns, network dynamics, markov chain, regulatory, models, optimal control, boolean, networks models, networks boolean<div style="font-size:11px;"><br><b>Abstract: </b><em>In systems biology, network models are often used to study interactions among cellular components, a salient aim being to develop drugs and therapeutic mechanisms to change the dynamical behavior of the network to avoid undesirable phenotypes. Owing to limited knowledge, model uncertainty is commonplace and network dynamics can be updated in different ways, thereby giving multiple dynamic trajectories, that is, dynamics uncertainty. In this manuscript, we propose an experimental design method that can effectively reduce the dynamics uncertainty and improve performance in an interaction-based network. Both dynamics uncertainty and experimental error are quantified with respect to the modeling objective, herein, therapeutic intervention. The aim of experimental design is to select among a set of candidate experiments the experiment whose outcome, when applied to the network model, maximally reduces the dynamics uncertainty pertinent to the intervention objective.</div>','<b>Title: </b>Analysis of a synthetic gene switching motif: Systems and control approaches<br><b>Year: </b>2014<br><b>Author: </b> Kkk. Kim,  Ks. Cheong,  Kj. Chen and  Rd. Braatz<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.jprocont.2013.12.004"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF PROCESS CONTROL<br><b>Cluster-Keywords: </b><em>gene regulatory, systems biology, synthetic biology, regulatory network, gene network, gene expression, system model, synthetic network, network dynamics, systems theory, synthetic gene, genetic network, model predicts, dynamical systems, time delays, translational time, regulation models, oscillatory expression, network structure, biological systems<div style="font-size:11px;"><br><b>Abstract: </b><em>Developing predictive mathematical models for regulatory networks in biological systems would be useful for their analysis and design. This paper studies the parameter-dependent characteristics of a gene switching model that consists of dual positive feedback loops. Deterministic and stochastic stability are studied for this model, as well as other important system behaviors such as convergence rate to a stable equilibrium point, hysteresis induced by two time scales of the system model, and noise sensitivity with respect to the system parameters. Sensitivity of system performance indices with respect to the system parameters are analyzed in terms of H-infinity- and H-2-norms of the linearized system model with their closed-form solutions. The presented qualitative and quantitative studies of the system characteristics enable the synthesis of a robust gene regulatory network that achieves desired static and dynamic responses. C 2014 Elsevier Ltd. All rights reserved.</div>','<b>Title: </b>NEW PHYSICAL CONCEPTS FOR CELL AMEBOID MOTION<br><b>Year: </b>1993<br><b>Author: </b> E. Evans<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/S0006-34959381497-8"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOPHYSICAL JOURNAL<br><b>Cluster-Keywords: </b><em>interaction network, actin filament, network model, filament network, network structures, actin network, protein interaction, actin dynamics, network, membrane protein, network dynamics, model dynamic, cell, model, network approach, protein complex, computational model, actin, adhesion dynamics, cell dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>Amoeboid motion of cells is an essential mechanism in the function of many biological organisms e.g., the regiment of scavenger cells in the immune defense system of animals. This process involves rapid chemical polymerization with numerous protein constituents to create a musclelike contractile network that advances the cell over the surface. Significant progress has been made in the biology and biochemistry of motile cells, but the physical dynamics of cell spreading and contraction are not well understood. The reason is that general approaches are formulated from complex mass, momentum, and chemical reaction equations for multiphase-multicomponent flow with the nontrivial difficulty of moving boundaries. However, there are strong clues to the dynamics that allow bold steps to be taken in simplifying the physics of motion. First, amoeboid cells often exhibit exceptional kinematics, i.e., steady advance and retraction of local fixed-shape patterns. Second, recent evidence has shown that cell projections grow by polymerization along the advancing boundary of the cell. Together, these characteristics represent a local growth process pinned to the interfacial contour of a contractile network. As such, the moving boundary becomes tractable, but subtle features of the motion lead to specific requirements for the chemical nature of the boundary polymerization process. To demonstrate these features, simple examples for limiting conditions of substrate interaction i.e., strong and weak adhesion are compared with data from experimental studies of yeast particle engulfment by blood granulocytes and actin network dynamics in fishscale keratocytes.</div>','<b>Title: </b>Planning Interventions in Biological Networks<br><b>Year: </b>2010<br><b>Author: </b> D. Bryce,  M. Verdicchio and  S. Kim<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1145/1869397.1869400"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ACM TRANSACTIONS ON INTELLIGENT SYSTEMS AND TECHNOLOGY<br><b>Cluster-Keywords: </b><em>boolean networks, regulatory networks, genetic regulatory, gene regulatory, probabilistic boolean, network, networks, model, network model, probabilistic model, model checking, networks pbns, network dynamics, markov chain, regulatory, models, optimal control, boolean, networks models, networks boolean<div style="font-size:11px;"><br><b>Abstract: </b><em>Modeling the dynamics of biological processes has recently become an important research topic in computational biology and systems engineering. One of the most important reasons to model a biological process is to enable high-throughput in-silico experiments that attempt to predict or intervene in the process. These experiments can help accelerate the design of therapies through their rapid and inexpensive replication and alteration. While some techniques exist for reasoning with biological processes, few take advantage of the flexible and scalable algorithms popular in AI research. In reasoning about interventions in biological processes, where scalability is crucial for feasible application, we apply AT planning-based search techniques and demonstrate their advantage over existing enumerative methods. We also present a novel formulation of intervention planning that relies on models that characterize and attempt to change the phenotype of a system. We study three biological systems: the yeast cell cycle, a model of the human aging process, and the Wnt5a network governing the metastasis of melanoma in humans. The contribution of our investigation is in demonstrating that: i prior approaches, based on dynamic programming, cannot scale as well as heuristic search, and ii the newly found scalability enables us to plan previously unknown sequences of interventions that reveal novel and biologically significant responses in the systems which are consistent with biological knowledge in the literature.</div>','<b>Title: </b>Inferring Network Connectivity from Event Timing Patterns<br><b>Year: </b>2018<br><b>Author: </b> J. Casadiego,  D. Maoutsa and  M. Timme<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1103/PhysRevLett.121.054101"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PHYSICAL REVIEW LETTERS<br><b>Cluster-Keywords: </b><em>oscillator networks, network dynamics, network, networks, dynamical systems, dynamics, node dynamics, systems, coupled oscillator, synchronization, dynamics network, complex network, coupled system, network modifications, oscillators, network connectivity, feedback vertex, network topology, synchronization properties, collective dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>Reconstructing network connectivity from the collective dynamics of a system typically requires access to its complete continuous-time evolution, although these are often experimentally inaccessible. Here we propose a theory for revealing physical connectivity of networked systems only from the event time series their intrinsic collective dynamics generate. Representing the patterns of event timings in an event space spanned by interevent and cross-event intervals, we reveal which other units directly influence the interevent times of any given unit. For illustration, we linearize an event-space mapping constructed from the spiking patterns in model neural circuits to reveal the presence or absence of synapses between any pair of neurons, as well as whether the coupling acts in an inhibiting or activating excitatory manner. The proposed model-independent reconstruction theory is scalable to larger networks and may thus play an important role in the reconstruction of networks from biology to social science and engineering.</div>','<b>Title: </b>Synchrony in networks of coupled non-smooth dynamical systems: Extending the master stability function<br><b>Year: </b>2016<br><b>Author: </b> S. Coombes and  R. Thul<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1017/S0956792516000115"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>EUROPEAN JOURNAL OF APPLIED MATHEMATICS<br><b>Cluster-Keywords: </b><em>oscillator networks, network dynamics, network, networks, dynamical systems, dynamics, node dynamics, systems, coupled oscillator, synchronization, dynamics network, complex network, coupled system, network modifications, oscillators, network connectivity, feedback vertex, network topology, synchronization properties, collective dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>The master stability function is a powerful tool for determining synchrony in high-dimensional networks of coupled limit cycle oscillators. In part, this approach relies on the analysis of a low-dimensional variational equation around a periodic orbit. For smooth dynamical systems, this orbit is not generically available in closed form. However, many models in physics, engineering and biology admit to non-smooth piece-wise linear caricatures, for which it is possible to construct periodic orbits without recourse to numerical evolution of trajectories. A classic example is the McKean model of an excitable system that has been extensively studied in the mathematical neuroscience community. Understandably, the master stability function cannot be immediately applied to networks of such non-smooth elements. Here, we show how to extend the master stability function to non-smooth planar piece-wise linear systems, and in the process demonstrate that considerable insight into network dynamics can be obtained. In illustration, we highlight an inverse period-doubling route to synchrony, under variation in coupling strength, in globally linearly coupled networks for which the node dynamics is poised near a homoclinic bifurcation. Moreover, for a star graph, we establish a mechanism for achieving so-called remote synchronisation where the hub oscillator does not synchronise with the rest of the network, even when all the oscillators are identical. We contrast this with node dynamics close to a non-smooth Andronov-Hopf bifurcation and also a saddle node bifurcation of limit cycles, for which no such bifurcation of synchrony occurs.</div>','<b>Title: </b>Confidence from uncertainty - A multi-target drug screening method from robust control theory<br><b>Year: </b>2010<br><b>Author: </b> C. Luni,  Je. Shoemaker,  Kr. Sanft,  Lr. Petzold and  Fj. Doyle<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-4-161"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>sensitivity analysis, systems biology, robustness analysis, network model, network inference, regulatory network, systems analysis, gene expression, parametric uncertainty, biological networks, mathematical models, biochemical networks, biological functions, cellular networks, biological system, parametric sensitivity, dynamical models, dynamic system, networks analysis, gene regulatory<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Robustness is a recognized feature of biological systems that evolved as a defence to environmental variability. Complex diseases such as diabetes, cancer, bacterial and viral infections, exploit the same mechanisms that allow for robust behaviour in healthy conditions to ensure their own continuance. Single drug therapies, while generally potent regulators of their specific protein/gene targets, often fail to counter the robustness of the disease in question. Multi-drug therapies offer a powerful means to restore disrupted biological networks, by targeting the subsystem of interest while preventing the diseased network from reconciling through available, redundant mechanisms. Modelling techniques are needed to manage the high number of combinatorial possibilities arising in multi-drug therapeutic design, and identify synergistic targets that are robust to system uncertainty.Results: We present the application of a method from robust control theory, Structured Singular Value or mu-analysis, to identify highly effective multi-drug therapies by using robustness in the face of uncertainty as a new means of target discrimination. We illustrate the method by means of a case study of a negative feedback network motif subject to parametric uncertainty.Conclusions: The paper contributes to the development of effective methods for drug screening in the context of network modelling affected by parametric uncertainty. The results have wide applicability for the analysis of different sources of uncertainty like noise experienced in the data, neglected dynamics, or intrinsic biological variability.</div>','<b>Title: </b>A specialized ODE integrator for the efficient computation of parameter sensitivities<br><b>Year: </b>2012<br><b>Author: </b> P. Gonnet,  S. Dimopoulos,  L. Widmer and  J. Stelling<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-6-46"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>systems biology, network models, signaling network, parameter estimation, dynamic models, biological systems, mathematical models, metabolic network, parameter sensitivities, computational model, gene circuit, circuit models, experimental data, metabolic models, model system, scalable parameter, model parameters, parameter space, biomolecular networks, model<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Dynamic mathematical models in the form of systems of ordinary differential equations ODEs play an important role in systems biology. For any sufficiently complex model, the speed and accuracy of solving the ODEs by numerical integration is critical. This applies especially to systems identification problems where the parameter sensitivities must be integrated alongside the system variables. Although several very good general purpose ODE solvers exist, few of them compute the parameter sensitivities automatically.Results: We present a novel integration algorithm that is based on second derivatives and contains other unique features such as improved error estimates. These features allow the integrator to take larger time steps than other methods. In practical applications, i.e. systems biology models of different sizes and behaviors, the method competes well with established integrators in solving the system equations, and it outperforms them significantly when local parameter sensitivities are evaluated. For ease-of-use, the solver is embedded in a framework that automatically generates the integrator input from an SBML description of the system of interest.Conclusions: For future applications, comparatively cheap parameter sensitivities will enable advances in solving large, otherwise computationally expensive parameter estimation and optimization problems. More generally, we argue that substantially better computational performance can be achieved by exploiting characteristics specific to the problem domain; elements of our methods such as the error estimation could find broader use in other, more general numerical algorithms.</div>','<b>Title: </b>Modeling the Dynamics of Acute Phase Protein Expression in Human Hepatoma Cells Stimulated by IL-6<br><b>Year: </b>2015<br><b>Author: </b> Zb. Xu,  Jom. Karlsson and  Zy. Huang<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3390/pr3010050"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PROCESSES<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Interleukin-6 IL-6 is a systemic inflammatory mediator that triggers the human bodys acute phase response to trauma or inflammation. Although mathematical models for IL-6 signaling pathways have previously been developed, reactions that describe the expression of acute phase proteins were not included. To address this deficiency, a recent model of IL-6 signaling was extended to predict the dynamics of acute phase protein expression in IL-6-stimulated HepG2 cells a human hepatoma cell line. This included reactions that describe the regulation of haptoglobin, fibrinogen, and albumin secretion by nuclear transcription factors STAT3 dimer and C/EBP beta. This new extended model was validated against two different sets of experimental data. Using the validated model, a sensitivity analysis was performed to identify seven potential drug targets to regulate the secretion of haptoglobin, fibrinogen, and albumin. The drug-target binding kinetics for these seven targets was then integrated with the IL-6 kinetic model to rank them based upon the influence of their pairing with drugs on acute phase protein dynamics. It was found that gp80, JAK, and gp130 were the three most promising drug targets and that it was possible to reduce the therapeutic dosage by combining drugs aimed at the top three targets in a cocktail. These findings suggest hypotheses for further experimental investigation.</div>','<b>Title: </b>Qualitative dynamics semantics for SBGN process description<br><b>Year: </b>2016<br><b>Author: </b> A. Rougny,  C. Froidevaux,  L. Calzone and  L. Pauleve<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/s12918-016-0285-0"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Qualitative dynamics semantics provide a coarse-grain modeling of networks dynamics by abstracting away kinetic parameters. They allow to capture general features of systems dynamics, such as attractors or reachability properties, for which scalable analyses exist. The Systems Biology Graphical Notation Process Description language SBGN-PD has become a standard to represent reaction networks. However, no qualitative dynamics semantics taking into account all the main features available in SBGN-PD had been proposed so far.Results: We propose two qualitative dynamics semantics for SBGN-PD reaction networks, namely the general semantics and the stories semantics, that we formalize using asynchronous automata networks. While the general semantics extends standard Boolean semantics of reaction networks by taking into account all the main features of SBGN-PD, the stories semantics allows to model several molecules of a network by a unique variable. The obtained qualitative models can be checked against dynamical properties and therefore validated with respect to biological knowledge. We apply our framework to reason on the qualitative dynamics of a large network more than 200 nodes modeling the regulation of the cell cycle by RB/E2F.Conclusion: The proposed semantics provide a direct formalization of SBGN-PD networks in dynamical qualitative models that can be further analyzed using standard tools for discrete models. The dynamics in stories semantics have a lower dimension than the general one and prune multiple behaviors which can be considered as spurious by enforcing the mutual exclusiveness between the activity of different nodes of a same story. Overall, the qualitative semantics for SBGN-PD allow to capture efficiently important dynamical features of reaction network models and can be exploited to further refine them.</div>','<b>Title: </b>Mathematical Models of Pluripotent Stem Cells: At the Dawn of Predictive Regenerative Medicine<br><b>Year: </b>2016<br><b>Author: </b> P. Pir and  N. Le Novere<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/978-1-4939-3283-215"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SYSTEMS MEDICINE<br><b>Cluster-Keywords: </b><em>stem cell, stem cells, systems biology, mathematical models, cell biology, network model, signaling pathways, cell fate, computational models, cancer cells, gene expression, network dynamics, embryonic stem, signaling network, prostate cancer, mathematical modeling, psychological stress, apoptosis signaling, computational systems, molecular mechanisms<div style="font-size:11px;"><br><b>Abstract: </b><em>Regenerative medicine, ranging from stem cell therapy to organ regeneration, is promising to revolutionize treatments of diseases and aging. These approaches require a perfect understanding of cell reprogramming and differentiation. Predictive modeling of cellular systems has the potential to provide insights about the dynamics of cellular processes, and guide their control. Moreover in many cases, it provides alternative to experimental tests, difficult to perform for practical or ethical reasons. The variety and accuracy of biological processes represented in mathematical models grew in-line with the discovery of underlying molecular mechanisms. High-throughput data generation led to the development of models based on data analysis, as an alternative to more established modeling based on prior mechanistic knowledge. In this chapter, we give an overview of existing mathematical models of pluripotency and cell fate, to illustrate the variety of methods and questions. We conclude that current approaches are yet to overcome a number of limitations: Most of the computational models have so far focused solely on understanding the regulation of pluripotency, and the differentiation of selected cell lineages. In addition, models generally interrogate only a few biological processes. However, a better understanding of the reprogramming process leading to ESCs and iPSCs is required to improve stem-cell therapies. One also needs to understand the links between signaling, metabolism, regulation of gene expression, and the epigenetics machinery.</div>','<b>Title: </b>CellDesigner 3.5: A versatile modeling tool for biochemical networks<br><b>Year: </b>2008<br><b>Author: </b> A. Funahashi,  Y. Matsuoka,  A. Jouraku,  M. Morohashi,  N. Kikuchi and  H. Kitano<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1109/JPROC.2008.925458"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PROCEEDINGS OF THE IEEE<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Understanding of the logic and dynamics of gene-regulatory and biochemical networks is a major challenge of systems biology. To facilitate this research topic, we have developed a modeling/simulating tool called CellDesigner. CellDesigner primarily has capabilities to visualize, model, and simulate gene-regulatory and biochemical networks. Two major characteristics embedded in CellDesigner boost its usability to create/import/export models: 1 solidly defined and comprehensive graphical representation systems biology graphical notation of network models and 2 systems biology markup language SBML as a model-describing basis, which function as intertool media to import/export SBML-based models. in addition, since its initial release in 2004, we have extended various capabilities of CellDesigner. For example, we integrated other Systems Biology Workbench enabled simulation/analysis software packages. CellDesigner also supports simulation and parameter search, supported by integration with SBML ODE Solver, enabling users to simulate through our sophisticated graphical user interface. Users can also browse and modify existing models by referring to existing databases directly through CellDesigner. Those extended functions empower CellDesigner as not only a modeling/simulating tool but also an integrated analysis suite. CellDesigner is implemented in Java and thus supports various platforms i.e., windows, Linux, and MacOS X. CellDesigner is freely available via our Web site.</div>','<b>Title: </b>Reconstructing the regulatory network controlling commitment and sporulation in Physarum polycephalum based on hierarchical Petri Net modelling and simulation<br><b>Year: </b>2005<br><b>Author: </b> W. Marwan,  A. Sujatha and  C. Starostzik<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.jtbi.2005.03.018"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF THEORETICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>petri net, petri nets, bayesian networks, biological networks, systems biology, regulatory networks, dynamic bayesian, net model, gene regulatory, network data, biological systems, biological data, network analysis, network models, biochemical networks, time series, network structure, signal transduction, net models, dbn model<div style="font-size:11px;"><br><b>Abstract: </b><em>We reconstruct the regulatory network controlling commitment and sporulation of Physarum polycephalum from experimental results using a hierarchical Petri Net-based modelling and simulation framework. The stochastic Petri Net consistently describes the structure and simulates the dynamics of the molecular network as analysed by genetic, biochemical and physiological experiments within a single coherent model. The Petri Net then is extended to simulate time-resolved somatic complementation experiments performed by mixing the cytoplasms of mutants altered in the sporulation response, to systematically explore the network structure and to probe its dynamics. This reverse engineering approach presumably can be employed to explore other molecular or genetic signalling systems where the activity of genes or their products can be experimentally controled in a time-resolved manner. c 2005 Elsevier Ltd. All rights reserved.</div>','<b>Title: </b>EQUILIBRIUM AND NONEQUILIBRIUM MODELING OF YINYANG WUXING FOR DIAGNOSTIC DECISION SUPPORT IN TRADITIONAL CHINESE MEDICINE<br><b>Year: </b>2009<br><b>Author: </b> Wr. Zhang and  Ss. Chen<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1142/S0219622009003521"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>INTERNATIONAL JOURNAL OF INFORMATION TECHNOLOGY  DECISION MAKING<br><b>Cluster-Keywords: </b><em>chaotic system, control systems, neural networks, jerk chaotic, torch chaotic, mode control, sliding mode, plasma torch, dynamic neural, mitochondrial dynamics, system, systems biology, adaptive integral, model system, rodtype plasma, integral sliding, systems, jerk system, cancer cell, comparison method<div style="font-size:11px;"><br><b>Abstract: </b><em>Although the theory of YinYang WuXing YYWX or YinYang-5-Elements subsystems of human body has been a major analytical basis for Traditional Chinese Medicine TCM for thousands of years, the theory has been primarily empirical in nature without a formal mathematical or scientific foundation. Based on bipolar set theory, YinYang bipolar linear algebra is introduced and an equilibrium-or nonequilibrium-based computational model of YYWX is proposed. Nourishing and regulating relations are defined; YYWX dynamic equations are formulated; global and local equilibrium and nonequilibrium conditions are established and proved. Computer simulations of equilibrium and nonequilibrium processes show that the new approach provides a unique scientific basis for future research and development in herbal medicine, Qi vital energy, QiGong, JingLuo acupuncture channels, and acupuncture. A decision support system is prototyped and presented for diagnostic decision analysis in TCM. Furthermore, this work is expected to provide a basic YinYang cellular network architecture for modern scientific research in socioeconomics as well as in genomics, bioeconomics, metabolism, computational biology, aging, artificial intelligence, and biomedical engineering.</div>','<b>Title: </b>Integrating protein localization with automated signaling pathway reconstruction<br><b>Year: </b>2019<br><b>Author: </b> I. Youssef,  J. Law and  A. Ritz<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/s12859-019-3077-x"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>systems biology, regulatory network, interaction networks, network model, gene regulatory, network, network biology, networks, gene network, response networks, network dynamics, dynamic model, cell networks, gene expression, biological networks, system dynamics, dynamic network, model, biological systems, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Background Understanding cellular responses via signal transduction is a core focus in systems biology. Tools to automatically reconstruct signaling pathways from protein-protein interactions PPIs can help biologists generate testable hypotheses about signaling. However, automatic reconstruction of signaling pathways suffers from many interactions with the same confidence score leading to many equally good candidates. Further, some reconstructions are biologically misleading due to ignoring protein localization information. Results We propose LocPL, a method to improve the automatic reconstruction of signaling pathways from PPIs by incorporating information about protein localization in the reconstructions. The method relies on a dynamic program to ensure that the proteins in a reconstruction are localized in cellular compartments that are consistent with signal transduction from the membrane to the nucleus. LocPL and existing reconstruction algorithms are applied to two PPI networks and assessed using both global and local definitions of accuracy. LocPL produces more accurate and biologically meaningful reconstructions on a versatile set of signaling pathways. Conclusion LocPL is a powerful tool to automatically reconstruct signaling pathways from PPIs that leverages cellular localization information about proteins. The underlying dynamic program and signaling model are flexible enough to study cellular signaling under different settings of signaling flow across the cellular compartments.</div>','<b>Title: </b>Inference of gene regulatory networks with multi-objective cellular genetic algorithm<br><b>Year: </b>2019<br><b>Author: </b> J. Garcia-Nieto,  Aj. Nebro and  Jf. Aldana-Montes<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.compbiolchem.2019.05.003"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>COMPUTATIONAL BIOLOGY AND CHEMISTRY<br><b>Cluster-Keywords: </b><em>ssystem model, biological systems, biological networks, parameter estimation, biochemical networks, dynamic biological, proposed algorithm, regulatory networks, structure identification, model structure, genetic algorithm, gene regulatory, gene expression, network inference, network motif, network structure, gene networks, network identification, network inferring, estimation method<div style="font-size:11px;"><br><b>Abstract: </b><em>Reverse engineering of biochemical networks remains an important open challenge in computational systems biology. The goal of model inference is to, based on time-series gene expression data, obtain the sparse topological structure and parameters that quantitatively understand and reproduce the dynamics of biological systems. In this paper, we propose a multi-objective approach for the inference of S-System structures for Gene Regulatory Networks GRNs based on Pareto dominance and Pareto optimality theoretical concepts instead of the conventional single-objective evaluation of Mean Squared Error MSE. Our motivation is that, using a multi-objective formulation for the GRN, it is possible to optimize the sparse topology of a given GRN as well as the kinetic order and rate constant parameters in a decoupled S-System, yet avoiding the use of additional penalty weights. A flexible and robust Multi-Objective Cellular Evolutionary Algorithm is adapted to perform the tasks of parameter learning and network topology inference for the proposed approach. The resulting software, called MONET, is evaluated on real-based academic and synthetic time-series of gene expression taken from the DREAM3 challenge and the IRMA in vivo datasets. The ability to reproduce biological behavior and robustness to noise is assessed and compared. The results obtained are competitive and indicate that the proposed approach offers advantages over previously used methods. In addition, MONET is able to provide experts with a set of trade-off solutions involving GRNs with different typologies and MSEs.</div>','<b>Title: </b>GeStoDifferent: a Cytoscape plugin for the generation and the identification of gene regulatory networks describing a stochastic cell differentiation process<br><b>Year: </b>2013<br><b>Author: </b> M. Antoniotti,  Gd. Bader,  G. Caravagna,  S. Crippa,  A. Graudenzi and  G. Mauri<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1093/bioinformatics/bts726"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>boolean networks, regulatory networks, network model, bayesian networks, model networks, systems biology, random boolean, real cells, immune system, genetic regulatory, networks, biochemical network, fluorescence images, cellular network, gene regulatory, cell differentiation, network, biological systems, ensemble approach, cell dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>The characterization of the complex phenomenon of cell differentiation is a key goal of both systems and computational biology. GESTODIFFERENT is a Cytoscape plugin aimed at the generation and the identification of gene regulatory networks GRNs describing an arbitrary stochastic cell differentiation process. The dynamical model adopted to describe general GRNs is that of noisy random Boolean networks NRBNs, with a specific focus on their emergent dynamical behavior. GESTODIFFERENT explores the space of GRNs by filtering the NRBN instances inconsistent with a stochastic lineage differentiation tree representing the cell lineages that can be obtained by following the fate of a stem cell descendant. Matched networks can then be analyzed by Cytoscape network analysis algorithms or, for instance, used to define multiscale models of cellular dynamics.</div>','<b>Title: </b>A framework for scalable parameter estimation of gene circuit models using structural information<br><b>Year: </b>2013<br><b>Author: </b> H. Kuwahara,  M. Fan,  Sj. Wang and  X. Gao<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1093/bioinformatics/btt232"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>systems biology, network models, signaling network, parameter estimation, dynamic models, biological systems, mathematical models, metabolic network, parameter sensitivities, computational model, gene circuit, circuit models, experimental data, metabolic models, model system, scalable parameter, model parameters, parameter space, biomolecular networks, model<div style="font-size:11px;"><br><b>Abstract: </b><em>Motivation: Systematic and scalable parameter estimation is a key to construct complex gene regulatory models and to ultimately facilitate an integrative systems biology approach to quantitatively understand the molecular mechanisms underpinning gene regulation.Results: Here, we report a novel framework for efficient and scalable parameter estimation that focuses specifically on modeling of gene circuits. Exploiting the structure commonly found in gene circuit models, this framework decomposes a system of coupled rate equations into individual ones and efficiently integrates them separately to reconstruct the mean time evolution of the gene products. The accuracy of the parameter estimates is refined by iteratively increasing the accuracy of numerical integration using the model structure. As a case study, we applied our framework to four gene circuit models with complex dynamics based on three synthetic datasets and one time series microarray data set. We compared our framework to three state-of-the-art parameter estimation methods and found that our approach consistently generated higher quality parameter solutions efficiently. Although many general-purpose parameter estimation methods have been applied for modeling of gene circuits, our results suggest that the use of more tailored approaches to use domain-specific information may be a key to reverse engineering of complex biological systems.</div>','<b>Title: </b>Metrics for regulated biochemical pathway systems<br><b>Year: </b>2019<br><b>Author: </b> Jd. Davis and  Eo. Voit<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1093/bioinformatics/bty942"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>systems biology, dynamic models, biological systems, model, dynamic systems, computational systems, network model, experimental data, model parameters, systems, systems model, models, pathway systems, data, biochemical pathway, dynamics model, biochemical networks, computational models, network, system<div style="font-size:11px;"><br><b>Abstract: </b><em>Motivation The assessment of graphs through crisp numerical metrics has long been a hallmark of biological network analysis. However, typical graph metrics ignore regulatory signals that are crucially important for optimal pathway operation, for instance, in biochemical or metabolic studies. Here we introduce adjusted metrics that are applicable to both static networks and dynamic systems.Results The metrics permit quantitative characterizations of the importance of regulation in biochemical pathway systems, including systems designed for applications in synthetic biology or metabolic engineering. They may also become criteria for effective model reduction.Availability and implementation The source code is available at https://gitlab.com/tienbien44/metrics-bsa</div>','<b>Title: </b>Discrete Dynamic Modeling: A Network Approach for Systems Pharmacology<br><b>Year: </b>2016<br><b>Author: </b> Sn. Steinway,  Rs. Wang and  R. Albert<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/978-3-319-44534-25"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SYSTEMS PHARMACOLOGY AND PHARMACODYNAMICS<br><b>Cluster-Keywords: </b><em>boolean network, network models, qualitative models, systems biology, mathematical modeling, boolean models, signaling networks, dynamic models, dynamic modeling, signaling dynamics, logic models, dynamic network, mathematical models, boolean modeling, signal transduction, regulatory networks, drug resistance, biological networks, network approach, network modeling<div style="font-size:11px;"><br><b>Abstract: </b><em>Systems pharmacology is an interdisciplinary field that aims to apply the theoretical and experimental tools of systems biology to drug development. The goal is to go beyond the interaction between a drug and the target to which it binds to explore drug effects on the cellular networks affected by disease. Over the years, vast amounts of information about the regulatory relationships among genes, proteins, and small molecules have been acquired. Similarly, there is much known about the deregulation of these systems during disease. However, many knowledge gaps still exist. There is an abundance of qualitative or relative information related to the activation of signaling pathways, but a paucity of kinetic and temporal information. Discrete dynamic modeling provides a means to create predictive models of signal transduction pathways by integrating fragmentary and qualitative interaction information. Using discrete dynamic modeling, a structural static network of biological regulatory relationships can be translated into a mathematical model without the use of kinetic parameters. This model can describe the dynamics of a biological system over time, both in normal and in perturbation scenarios. In this chapter, we discuss the fundamentals of discrete dynamic modeling as it pertains to systems pharmacology. As an example, we apply this methodology to a previously constructed pharmacodynamic model of epidermal derived growth factor receptor EGFR signaling. We 1 translate this model into two types of discrete models, a Boolean model and a three-state model, 2 show how the effects of an EGFR inhibitor such as gefitinib can suppress tumor growth, and 3 model how genomic variants can augment the effect of EGFR inhibition in tumor growth. We argue that discrete dynamic models can be used to facilitate many of the goals of systems pharmacology. These include understanding how individual differences contribute to variability in drug response and determining which drugs would be best depending on individual genetic differences.</div>','<b>Title: </b>Modeling the Metabolism of Arabidopsis thaliana: Application of Network Decomposition and Network Reduction in the Context of Petri Nets<br><b>Year: </b>2017<br><b>Author: </b> I. Koch,  J. Nothen and  E. Schleiff<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3389/fgene.2017.00085"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>FRONTIERS IN GENETICS<br><b>Cluster-Keywords: </b><em>petri net, petri nets, bayesian networks, biological networks, systems biology, regulatory networks, dynamic bayesian, net model, gene regulatory, network data, biological systems, biological data, network analysis, network models, biochemical networks, time series, network structure, signal transduction, net models, dbn model<div style="font-size:11px;"><br><b>Abstract: </b><em>Motivation: Arabidopsis thaliana is a well-established model system for the analysis of the basic physiological and metabolic pathways of plants. Nevertheless, the system is not yet fully understood, although many mechanisms are described, and information for many processes exists. However, the combination and interpretation of the large amount of biological data remain a big challenge, not only because data sets for metabolic paths are still incomplete. Moreover, they are often inconsistent, because they are coming from different experiments of various scales, regarding, for example, accuracy and/or significance. Here, theoretical modeling is powerful to formulate hypotheses for pathways and the dynamics of the metabolism, even if the biological data are incomplete. To develop reliable mathematical models they have to be proven for consistency. This is still a challenging task because many verification techniques fail already for middle-sized models. Consequently, new methods, like decomposition methods or reduction approaches, are developed to circumvent this problem.Methods: We present a new semi-quantitative mathematical model of the metabolism of Arabidopsis thaliana. We used the Petri net formalism to express the complex reaction system in a mathematically unique manner. To verify the model for correctness and consistency we applied concepts of network decomposition and network reduction such as transition invariants, common transition pairs, and invariant transition pairs.Results: We formulated the core metabolism of Arabidopsis thaliana based on recent knowledge from literature, including the Calvin cycle, glycolysis and citric acid cycle, glyoxylate cycle, urea cycle, sucrose synthesis, and the starch metabolism. By applying network decomposition and reduction techniques at steady-state conditions, we suggest a straightforward mathematical modeling process. We demonstrate that potential steady-state pathways exist, which provide the fixed carbon to nearly all parts of the network, especially to the citric acid cycle. There is a close cooperation of important metabolic pathways, e.g., the de novo synthesis of uridine-5-monophosphate, the gamma-aminobutyric acid shunt, and the urea cycle. The presented approach extends the established methods for a feasible interpretation of biological network models, in particular of large and complex models.</div>','<b>Title: </b>A Gene Regulatory Network Model for Floral Transition of the Shoot Apex in Maize and Its Dynamic Modeling<br><b>Year: </b>2012<br><b>Author: </b> Zs. Dong,  O. Danilevskaya,  T. Abadie,  C. Messina,  N. Coles and  M. Cooper<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0043450"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>The transition from the vegetative to reproductive development is a critical event in the plant life cycle. The accurate prediction of flowering time in elite germplasm is important for decisions in maize breeding programs and best agronomic practices. The understanding of the genetic control of flowering time in maize has significantly advanced in the past decade. Through comparative genomics, mutant analysis, genetic analysis and QTL cloning, and transgenic approaches, more than 30 flowering time candidate genes in maize have been revealed and the relationships among these genes have been partially uncovered. Based on the knowledge of the flowering time candidate genes, a conceptual gene regulatory network model for the genetic control of flowering time in maize is proposed. To demonstrate the potential of the proposed gene regulatory network model, a first attempt was made to develop a dynamic gene network model to predict flowering time of maize genotypes varying for specific genes. The dynamic gene network model is composed of four genes and was built on the basis of gene expression dynamics of the two late flowering id1 and dlf1 mutants, the early flowering landrace Gaspe Flint and the temperate inbred B73. The model was evaluated against the phenotypic data of the id1 dlf1 double mutant and the ZMM4 overexpressed transgenic lines. The model provides a working example that leverages knowledge from model organisms for the utilization of maize genomic information to predict a whole plant trait phenotype, flowering time, of maize genotypes.</div>','<b>Title: </b>How to Synchronize Biological Clocks<br><b>Year: </b>2009<br><b>Author: </b> G. Russo and  M. Di Bernardo<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1089/cmb.2008.21TT"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF COMPUTATIONAL BIOLOGY<br><b>Cluster-Keywords: </b><em>gene regulatory, systems biology, synthetic biology, regulatory network, gene network, gene expression, system model, synthetic network, network dynamics, systems theory, synthetic gene, genetic network, model predicts, dynamical systems, time delays, translational time, regulation models, oscillatory expression, network structure, biological systems<div style="font-size:11px;"><br><b>Abstract: </b><em>This paper is concerned with a novel algorithm to study networks of biological clocks. A new set of conditions is established that can be used to verify whether an existing network synchronizes or to give guidelines to construct a new synthetic network of biological oscillators that synchronize. The methodology uses the so-called contraction theory from dynamical system theory and Gershgorin disk theorem. The strategy is validated on two examples: a model of glycolisis in yeast cells and a synthetic network of Repressilators that synchronizes.</div>','<b>Title: </b>Inference of complex biological networks: distinguishability issues and optimization-based solutions<br><b>Year: </b>2011<br><b>Author: </b> G. Szederkenyi,  Jr. Banga and  Aa. Alonso<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-5-177"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>reaction networks, reaction systems, metabolic networks, chemical reaction, network structure, systems biology, biological systems, biological networks, biochemical reaction, reaction rates, differential equations, reaction graph, complex networks, ssi modules, networks, life systems, ordinary differential, cellular networks, reaction, network flows<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: The inference of biological networks from high-throughput data has received huge attention during the last decade and can be considered an important problem class in systems biology. However, it has been recognized that reliable network inference remains an unsolved problem. Most authors have identified lack of data and deficiencies in the inference algorithms as the main reasons for this situation.Results: We claim that another major difficulty for solving these inference problems is the frequent lack of uniqueness of many of these networks, especially when prior assumptions have not been taken properly into account. Our contributions aid the distinguishability analysis of chemical reaction network CRN models with mass action dynamics. The novel methods are based on linear programming LP, therefore they allow the efficient analysis of CRNs containing several hundred complexes and reactions. Using these new tools and also previously published ones to obtain the network structure of biological systems from the literature, we find that, often, a unique topology cannot be determined, even if the structure of the corresponding mathematical model is assumed to be known and all dynamical variables are measurable. In other words, certain mechanisms may remain undetected or they are falsely detected while the inferred model is fully consistent with the measured data. It is also shown that sparsity enforcing approaches for determining true reaction structures are generally not enough without additional prior information.Conclusions: The inference of biological networks can be an extremely challenging problem even in the utopian case of perfect experimental information. Unfortunately, the practical situation is often more complex than that, since the measurements are typically incomplete, noisy and sometimes dynamically not rich enough, introducing further obstacles to the structure/parameter estimation process. In this paper, we show how the structural uniqueness and identifiability of the models can be guaranteed by carefully adding extra constraints, and that these important properties can be checked through appropriate computation methods.</div>','<b>Title: </b>Dynamic analysis of optimality in myocardial energy metabolism under normal and ischemic conditions<br><b>Year: </b>2006<br><b>Author: </b> Ry. Luo,  S. Liao,  Gy. Tao,  Yy. Li,  Sq. Zeng,  Yx. Li and  Qm. Luo<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1038/msb4100071"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MOLECULAR SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>metabolic network, metabolic model, dynamic flux, growth rate, genomescale metabolic, flux balance, balance analysis, metabolic flux, metabolic pathways, dynamic model, metabolic reactions, cellular growth, regulatory network, systems biology, optimal growth, metabolic, modeling metabolic, growth conditions, dynamic analysis, cell growth<div style="font-size:11px;"><br><b>Abstract: </b><em>To better understand the dynamic regulation of optimality in metabolic networks under perturbed conditions, we reconstruct the energetic-metabolic network in mammalian myocardia using dynamic flux balance analysis DFBA. Additionally, we modified the optimal objective from the maximization of ATP production to the minimal fluctuation of the profile of metabolite concentration under ischemic conditions, extending the hypothesis of original minimization of metabolic adjustment to create a composite modeling approach called M-DFBA. The simulation results are more consistent with experimental data than are those of the DFBA model, particularly the retentive predominant contribution of fatty acid to oxidative ATP synthesis, the exact mechanism of which has not been elucidated and seems to be unpredictable by the DFBA model. These results suggest that the systemic states of metabolic networks do not always remain optimal, but may become suboptimal when a transient perturbation occurs. This finding supports the relevance of our hypothesis and could contribute to the further exploration of the underlying mechanism of dynamic regulation in metabolic networks.</div>','<b>Title: </b>Facing the challenges of multiscale modelling of bacterial and fungal pathogen-host interactions<br><b>Year: </b>2017<br><b>Author: </b> J. Schleicher,  T. Conrad,  M. Gustafsson,  G. Cedersund,  R. Guthke and  J. Linde<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1093/bfgp/elv064"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BRIEFINGS IN FUNCTIONAL GENOMICS<br><b>Cluster-Keywords: </b><em>systems biology, network models, signaling network, parameter estimation, dynamic models, biological systems, mathematical models, metabolic network, parameter sensitivities, computational model, gene circuit, circuit models, experimental data, metabolic models, model system, scalable parameter, model parameters, parameter space, biomolecular networks, model<div style="font-size:11px;"><br><b>Abstract: </b><em>Recent and rapidly evolving progress on high-throughput measurement techniques and computational performance has led to the emergence of new disciplines, such as systems medicine and translational systems biology. At the core of these disciplines lies the desire to produce multiscale models: mathematical models that integrate multiple scales of biological organization, ranging from molecular, cellular and tissue models to organ, whole-organism and population scale models. Using such models, hypotheses can systematically be tested. In this review, we present state-of-the-art multiscale modelling of bacterial and fungal infections, considering both the pathogen and host as well as their interaction. Multiscale modelling of the interactions of bacteria, especially Mycobacterium tuberculosis, with the human host is quite advanced. In contrast, models for fungal infections are still in their infancy, in particular regarding infections with the most important human pathogenic fungi, Candida albicans and Aspergillus fumigatus. We reflect on the current availability of computational approaches for multiscale modelling of host-pathogen interactions and point out current challenges. Finally, we provide an outlook for future requirements of multiscale modelling.</div>','<b>Title: </b>Predictive Modeling of the Hypothalamic-Pituitary-Adrenal HPA Function. Dynamic Systems Theory Approach by Stoichiometric Network Analysis and Quenching Small Amplitude Oscillations<br><b>Year: </b>2009<br><b>Author: </b> S. Jelic,  Z. Cupic,  L. Kolar-Anic and  V. VukojevicLink: No link<br><b>Journal: </b>INTERNATIONAL JOURNAL OF NONLINEAR SCIENCES AND NUMERICAL SIMULATION<br><b>Cluster-Keywords: </b><em>circadian rhythms, systems biology, biological clock, circadian clock, biological rhythms, hpa axis, model, clock network, mathematical models, biological systems, dynamic systems, ode models, biology models, single cell, network, model parameters, experimental data, models, systems, dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>Two methods for dynamic systems analysis, Stoichiometric Network Analysis SNA and Quenching of Small Amplitude Oscillations QA, are used to study the behaviour of a vital biological system. Both methods use geometric approaches for the study of complex reaction systems. In SNA, methods based on convex polytopes geometry are applied for stability analysis and optimization of reaction networks. QA relies on a geometric representation of the concentration phase space, introduces the concept of manifolds and the singular perturbation theory to study the dynamics of complex processes. The analyzed system, the Hypothalamic-Pituitary-Adrenal HPA axis, as a major constituent of the neuroendocrine system has a critical role in integrating biological responses in basal conditions and during stress. Self-regulation in the HPA system was modeled through a positive and negative feedback effect of cortisol. A systematically reduced low-dimensional model of HPA activity in humans was fine-tuned by SNA, until quantitative agreement with experimental findings was achieved. By QA, we revealed an important dynamic regulatory mechanism that is a natural consequence of the intrinsic rhythmicity of the considered system.</div>','<b>Title: </b>Quantitative prediction of cellular metabolism with constraint-based models: the COBRA Toolbox<br><b>Year: </b>2007<br><b>Author: </b> Sa. Becker,  Am. Feist,  Ml. Mo,  G. Hannum,  Bo. Palsson and  Mj. Herrgard<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1038/nprot.2007.99"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>NATURE PROTOCOLS<br><b>Cluster-Keywords: </b><em>metabolic network, metabolic model, dynamic flux, growth rate, genomescale metabolic, flux balance, balance analysis, metabolic flux, metabolic pathways, dynamic model, metabolic reactions, cellular growth, regulatory network, systems biology, optimal growth, metabolic, modeling metabolic, growth conditions, dynamic analysis, cell growth<div style="font-size:11px;"><br><b>Abstract: </b><em>The manner in which microorganisms utilize their metabolic processes can be predicted using constraint-based analysis of genome-scale metabolic networks. Herein, we present the constraint-based reconstruction and analysis toolbox, a software package running in the Matlab environment, which allows for quantitative prediction of cellular behavior using a constraint-based approach. Specifically, this software allows predictive computations of both steady-state and dynamic optimal growth behavior, the effects of gene deletions, comprehensive robustness analyses, sampling the range of possible cellular metabolic states and the determination of network modules. Functions enabling these calculations are included in the toolbox, allowing a user to input a genome-scale metabolic model distributed in Systems Biology Markup Language format and perform these calculations with just a few lines of code. The results are predictions of cellular behavior that have been verified as accurate in a growing body of research. After software installation, calculation time is minimal, allowing the user to focus on the interpretation of the computational results.</div>','<b>Title: </b>Modelling the widespread effects of TOC1 signalling on the plant circadian clock and its outputs<br><b>Year: </b>2013<br><b>Author: </b> A. Pokhilko,  P. Mas and  Aj. Millar<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-7-23"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>circadian clock, biological systems, network models, mathematical models, clock gene, systems biology, model selection, biological networks, network data, experimental data, model parameters, network dynamics, plant circadian, regulatory network, clock model, plant clock, complex network, clock circuit, approximate bayesian, stochastic models<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: 24-hour biological clocks are intimately connected to the cellular signalling network, which complicates the analysis of clock mechanisms. The transcriptional regulator TOC1 TIMING OF CAB EXPRESSION 1 is a founding component of the gene circuit in the plant circadian clock. Recent results show that TOC1 suppresses transcription of multiple target genes within the clock circuit, far beyond its previously-described regulation of the morning transcription factors LHY LATE ELONGATED HYPOCOTYL and CCA1 CIRCADIAN CLOCK ASSOCIATED 1. It is unclear how this pervasive effect of TOC1 affects the dynamics of the clock and its outputs. TOC1 also appears to function in a nested feedback loop that includes signalling by the plant hormone Abscisic Acid ABA, which is upregulated by abiotic stresses, such as drought. ABA treatments both alter TOC1 levels and affect the clocks timing behaviour. Conversely, the clock rhythmically modulates physiological processes induced by ABA, such as the closing of stomata in the leaf epidermis. In order to understand the dynamics of the clock and its outputs under changing environmental conditions, the reciprocal interactions between the clock and other signalling pathways must be integrated.Results: We extended the mathematical model of the plant clock gene circuit by incorporating the repression of multiple clock genes by TOC1, observed experimentally. The revised model more accurately matches the data on the clocks molecular profiles and timing behaviour, explaining the clocks responses in TOC1 over-expression and toc1 mutant plants. A simplified representation of ABA signalling allowed us to investigate the interactions of ABA and circadian pathways. Increased ABA levels lengthen the free-running period of the clock, consistent with the experimental data. Adding stomatal closure to the model, as a key ABA- and clock-regulated downstream process allowed to describe TOC1 effects on the rhythmic gating of stomatal closure.Conclusions: The integrated model of the circadian clock circuit and ABA-regulated environmental sensing allowed us to explain multiple experimental observations on the timing and stomatal responses to genetic and environmental perturbations. These results crystallise a new role of TOC1 as an environmental sensor, which both affects the pace of the central oscillator and modulates the kinetics of downstream processes.</div>','<b>Title: </b>JOINT ESTIMATION OF MULTIPLE RELATED BIOLOGICAL NETWORKS<br><b>Year: </b>2014<br><b>Author: </b> Cj. Oates,  J. Korkola,  Jw. Gray and  S. Mukherjee<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1214/14-AOAS761"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ANNALS OF APPLIED STATISTICS<br><b>Cluster-Keywords: </b><em>systems biology, regulatory network, interaction networks, network model, gene regulatory, network, network biology, networks, gene network, response networks, network dynamics, dynamic model, cell networks, gene expression, biological networks, system dynamics, dynamic network, model, biological systems, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Graphical models are widely used to make inferences concerning interplay in multivariate systems. In many applications, data are collected from multiple related but nonidentical units whose underlying networks may differ but are likely to share features. Here we present a hierarchical Bayesian formulation for joint estimation of multiple networks in this nonidentically distributed setting. The approach is general: given a suitable class of graphical models, it uses an exchangeability assumption on networks to provide a corresponding joint formulation. Motivated by emerging experimental designs in molecular biology, we focus on time-course data with interventions, using dynamic Bayesian networks as the graphical models. We introduce a computationally efficient, deterministic algorithm for exact joint inference in this setting. We provide an upper bound on the gains that joint estimation offers relative to separate estimation for each network and empirical results that support and extend the theory, including an extensive simulation study and an application to proteomic data from human cancer cell lines. Finally, we describe approximations that are still more computationally efficient than the exact algorithm and that also demonstrate good empirical performance.</div>','<b>Title: </b>Dynamic change in promoter activation during lysine biosynthesis in Escherichia coli cells<br><b>Year: </b>2008<br><b>Author: </b> Jh. Ou,  T. Yamada,  K. Nagahisa,  T. Hirasawa,  C. Furusawa,  T. Yomo and  H. Shimizu<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1039/b711035a"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MOLECULAR BIOSYSTEMS<br><b>Cluster-Keywords: </b><em>gene expression, gene network, stochastic model, gene regulatory, systems biology, expression systems, gene regulation, network activity, expression dynamics, noise control, regulatory network, deterministic models, network components, stochastic modeling, regulation system, escherichia coli, regulatory system, synthetic gene, regulatory genes, gene<div style="font-size:11px;"><br><b>Abstract: </b><em>We investigated the expression dynamics of genes involved in lysine biosynthesis in Escherichia coli cells to obtain a quantitative understanding of the gene regulatory system. By constructing reporter strains expressing the green fluorescence protein gfp gene under the control of the promoter regions of those genes associated with lysine biosynthesis, time-dependent changes in gene expression in response to changes in lysine concentration in the medium were monitored by flow cytometry. Five promoters involved in lysine biosynthesis respond to the changes in lysine concentration in the medium. For these five promoters, time-dependent gene expression data were fitted to a simple dynamical model of gene expression to estimate the parameters of the gene regulatory system. According to the fitting parameters, dapD shows a significantly larger coefficient of repression than the other genes in the lysine synthesis pathway, which indicates the weak binding activity of the repressor to the dapD promoter region. Moreover, there is a trend that the closer an enzyme is to the start of the lysine biosynthesis pathway, the smaller its maximal promoter activity is. The results provide a better quantitative understanding of the expression dynamics in the lysine biosynthesis pathway.</div>','<b>Title: </b>Inference of Developmental Gene Regulatory Networks Beyond Classical Model Systems: New Approaches in the Post-genomic Era<br><b>Year: </b>2018<br><b>Author: </b> Sl. Fernandez-Valverde,  F. Aguilera and  Ra. Ramos-Diaz<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1093/icb/icy061"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>INTEGRATIVE AND COMPARATIVE BIOLOGY<br><b>Cluster-Keywords: </b><em>gene regulatory, regulatory network, systems biology, cell differentiation, cell fate, model systems, potential landscape, network grn, dynamical systems, melanocyte differentiation, network models, gene expression, network dynamics, differentiation genes, dynamical models, dynamical networks, cellular differentiation, stem cells, attractor network, developmental gene<div style="font-size:11px;"><br><b>Abstract: </b><em>The advent of high-throughput sequencing HTS technologies has revolutionized the way we understand the transformation of genetic information into morphological traits. Elucidating the network of interactions between genes that govern cell differentiation through development is one of the core challenges in genome research. These networks are known as developmental gene regulatory networks dGRNs and consist largely of the functional linkage between developmental control genes, cis-regulatory modules, and differentiation genes, which generate spatially and temporally refined patterns of gene expression. Over the last 20 years, great advances have been made in determining these gene interactions mainly in classical model systems, including human, mouse, sea urchin, fruit fly, and worm. This has brought about a radical transformation in the fields of developmental biology and evolutionary biology, allowing the generation of high-resolution gene regulatory maps to analyze cell differentiation during animal development. Such maps have enabled the identification of gene regulatory circuits and have led to the development of network inference methods that can recapitulate the differentiation of specific cell-types or developmental stages. In contrast, dGRN research in non-classical model systems has been limited to the identification of developmental control genes via the candidate gene approach and the characterization of their spatiotemporal expression patterns, as well as to the discovery of cis-regulatory modules via patterns of sequence conservation and/or predicted transcription-factor binding sites. However, thanks to the continuous advances in HTS technologies, this scenario is rapidly changing. Here, we give a historical overview on the architecture and elucidation of the dGRNs. Subsequently, we summarize the approaches available to unravel these regulatory networks, highlighting the vast range of possibilities of integrating multiple technical advances and theoretical approaches to expand our understanding on the global gene regulation during animal development in non-classical model systems. Such new knowledge will not only lead to greater insights into the evolution of molecular mechanisms underlying cell identity and animal body plans, but also into the evolution of morphological key innovations in animals.</div>','<b>Title: </b>Predicting stochastic gene expression dynamics in single cells<br><b>Year: </b>2006<br><b>Author: </b> Jt. Mettetal,  D. Muzzey,  Jm. Pedraza,  Em. Ozbudak and  A. Van Oudenaarden<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1073/pnas.0509874103"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA<br><b>Cluster-Keywords: </b><em>gene expression, gene network, stochastic model, gene regulatory, systems biology, expression systems, gene regulation, network activity, expression dynamics, noise control, regulatory network, deterministic models, network components, stochastic modeling, regulation system, escherichia coli, regulatory system, synthetic gene, regulatory genes, gene<div style="font-size:11px;"><br><b>Abstract: </b><em>Fluctuations in protein numbers noise due to inherent stochastic effects in single cells can have large effects on the dynamic behavior of gene regulatory networks. Although deterministic models can predict the average network behavior, they fail to incorporate the stochasticity characteristic of gene expression, thereby limiting their relevance when single cell behaviors deviate from the population average. Recently, stochastic models have been used to predict distributions of steady-state protein levels within a population but not to predict the dynamic, presteadystate distributions. In the present work, we experimentally examine a system whose dynamics are heavily influenced by stochastic effects. We measure population distributions of protein numbers as a function of time in the Escherichia coli lactose uptake network lac operon. We then introduce a dynamic stochastic model and show that prediction of dynamic distributions requires only a few noise parameters in addition to the rates that characterize a deterministic model. Whereas the deterministic model cannot fully capture the observed behavior, our stochastic model correctly predicts the experimental dynamics without any fit parameters. Our results provide a proof of principle for the possibility of faithfully predicting dynamic population distributions from deterministic models supplemented by a stochastic component that captures the major noise sources.</div>','<b>Title: </b>Multicellular Computing Using Conjugation for Wiring<br><b>Year: </b>2013<br><b>Author: </b> A. Goni-Moreno,  M. Amos and  F. De La Cruz<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0065986"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>gene circuits, synthetic biology, circuit behavior, biological parts, synthetic gene, circuit design, genetic circuits, cell types, metabolite dynamics, mathematical model, designed circuit, control circuit, biological systems, circuit models, synthetic circuit, feedback circuits, biological circuits, design models, metabolic pathways, circuit parameters<div style="font-size:11px;"><br><b>Abstract: </b><em>Recent efforts in synthetic biology have focussed on the implementation of logical functions within living cells. One aim is to facilitate both internal "re-programming" and external control of cells, with potential applications in a wide range of domains. However, fundamental limitations on the degree to which single cells may be re-engineered have led to a growth of interest in multicellular systems, in which a "computation" is distributed over a number of different cell types, in a manner analogous to modern computer networks. Within this model, individual cell type perform specific sub-tasks, the results of which are then communicated to other cell types for further processing. The manner in which outputs are communicated is therefore of great significance to the overall success of such a scheme. Previous experiments in distributed cellular computation have used global communication schemes, such as quorum sensing QS, to implement the "wiring" between cell types. While useful, this method lacks specificity, and limits the amount of information that may be transferred at any one time. We propose an alternative scheme, based on specific cell-cell conjugation. This mechanism allows for the direct transfer of genetic information between bacteria, via circular DNA strands known as plasmids. We design a multicellular population that is able to compute, in a distributed fashion, a Boolean XOR function. Through this, we describe a general scheme for distributed logic that works by mixing different strains in a single population; this constitutes an important advantage of our novel approach. Importantly, the amount of genetic information exchanged through conjugation is significantly higher than the amount possible through QS-based communication. We provide full computational modelling and simulation results, using deterministic, stochastic and spatially-explicit methods. These simulations explore the behaviour of one possible conjugation-wired cellular computing system under different conditions, and provide baseline information for future laboratory implementations.</div>','<b>Title: </b>Rapidly exploring structural and dynamic properties of signaling networks using PathwayOracle<br><b>Year: </b>2008<br><b>Author: </b> D. Ruths,  L. Nakhleh and  Pt. Ram<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-2-76"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>systems biology, network models, signaling network, parameter estimation, dynamic models, biological systems, mathematical models, metabolic network, parameter sensitivities, computational model, gene circuit, circuit models, experimental data, metabolic models, model system, scalable parameter, model parameters, parameter space, biomolecular networks, model<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: In systems biology the experimentalist is presented with a selection of software for analyzing dynamic properties of signaling networks. These tools either assume that the network is in steady-state or require highly parameterized models of the network of interest. For biologists interested in assessing how signal propagates through a network under specific conditions, the first class of methods does not provide sufficiently detailed results and the second class requires models which may not be easily and accurately constructed. A tool that is able to characterize the dynamics of a signaling network using an unparameterized model of the network would allow biologists to quickly obtain insights into a signaling networks behavior.Results: We introduce PathwayOracle, an integrated suite of software tools for computationally inferring and analyzing structural and dynamic properties of a signaling network. The feature which differentiates PathwayOracle from other tools is a method that can predict the response of a signaling network to various experimental conditions and stimuli using only the connectivity of the signaling network. Thus signaling models are relatively easy to build. The method allows for tracking signal flow in a network and comparison of signal flows under different experimental conditions. In addition, PathwayOracle includes tools for the enumeration and visualization of coherent and incoherent signaling paths between proteins, and for experimental analysis-loading and superimposing experimental data, such as microarray intensities, on the network model.Conclusion: PathwayOracle provides an integrated environment in which both structural and dynamic analysis of a signaling network can be quickly conducted and visualized along side experimental results. By using the signaling network connectivity, analyses and predictions can be performed quickly using relatively easily constructed signaling network models. The application has been developed in Python and is designed to be easily extensible by groups interested in adding new or extending existing features. PathwayOracle is freely available for download and use.</div>','<b>Title: </b>Next-generation neural field model: The evolution of synchrony within patterns and waves<br><b>Year: </b>2019<br><b>Author: </b> A. Byrne,  D. Avitabile and  S. Coombes<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1103/PhysRevE.99.012313"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PHYSICAL REVIEW E<br><b>Cluster-Keywords: </b><em>network models, population model, field model, neural population, neural field, iot networks, neuronal networks, neural network, differential equations, parameter neural, lotkavolterra model, model, neuron models, model neurons, model leads, ecosystem model, iot devices, noise model, neuronal models, inhibitory synaptic<div style="font-size:11px;"><br><b>Abstract: </b><em>Neural field models are commonly used to describe wave propagation and bump attractors at a tissue level in the brain. Although motivated by biology, these models are phenomenological in nature. They are built on the assumption that the neural tissue operates in a near synchronous regime, and hence, cannot account for changes in the underlying synchrony of patterns. It is customary to use spiking neural network models when examining within population synchronization. Unfortunately, these high-dimensional models are notoriously hard to obtain insight from. In this paper, we consider a network of theta-neurons, which has recently been shown to admit an exact mean-field description in the absence of a spatial component. We show that the inclusion of space and a realistic synapse model leads to a reduced model that has many of the features of a standard neural field model coupled to a further dynamical equation that describes the evolution of network synchrony. Both Turing instability analysis and numerical continuation software are used to explore the existence and stability of spatiotemporal patterns in the system. In particular, we show that this new model can support states above and beyond those seen in a standard neural field model. These states are typified by structures within bumps and waves showing the dynamic evolution of population synchrony.</div>','<b>Title: </b>Modeling genetic regulatory networks: a delay discrete dynamical model approach<br><b>Year: </b>2012<br><b>Author: </b> H. Jiang,  Wk. Ching,  Kf. Aoki-Kinoshita and  Dj. Guo<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s11424-012-0283-2"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF SYSTEMS SCIENCE  COMPLEXITY<br><b>Cluster-Keywords: </b><em>boolean networks, regulatory networks, genetic regulatory, gene regulatory, probabilistic boolean, network, networks, model, network model, probabilistic model, model checking, networks pbns, network dynamics, markov chain, regulatory, models, optimal control, boolean, networks models, networks boolean<div style="font-size:11px;"><br><b>Abstract: </b><em>Modeling genetic regulatory networks is an important research topic in genomic research and computational systems biology. This paper considers the problem of constructing a genetic regulatory network GRN using the discrete dynamic system DDS model approach. Although considerable research has been devoted to building GRNs, many of the works did not consider the time-delay effect. Here, the authors propose a time-delay DDS model composed of linear difference equations to represent temporal interactions among significantly expressed genes. The authors also introduce interpolation scheme and re-sampling method for equalizing the non-uniformity of sampling time points. Statistical significance plays an active role in obtaining the optimal interaction matrix of GRNs. The constructed genetic network using linear multiple regression matches with the original data very well. Simulation results are given to demonstrate the effectiveness of the proposed method and model.</div>','<b>Title: </b>Modeling Gene Regulation Networks Using Ordinary Differential Equations<br><b>Year: </b>2012<br><b>Author: </b> Jg. Cao,  X. Qi and  Hy. Zhao<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/978-1-61779-400-112"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>NEXT GENERATION MICROARRAY BIOINFORMATICS: METHODS AND PROTOCOLS<br><b>Cluster-Keywords: </b><em>regulatory networks, gene regulatory, gene expression, network inference, expression data, network structure, gene networks, networks grns, regulatory genes, boolean networks, bayesian network, network reconstruction, networks gene, network dynamics, gene regulation, inference method, systems biology, time series, infer gene, network<div style="font-size:11px;"><br><b>Abstract: </b><em>Gene regulation networks arc composed of transcription factors, their interactions, and targets. It is of great interest to reconstruct and study these regulatory networks from genomics data. Ordinary differential equations ODEs are popular tools to model the dynamic system of gene regulation networks. Although the form of ODEs is often provided based on expert knowledge, the values for ODE parameters arc seldom known. It is a challenging problem to infer ODE parameters from gene expression data, because the ODEs do not have analytic solutions and the time-course gene expression data are usually sparse and associated with large noise. In this chapter, we review how the generalized profiling method can be applied to obtain estimates for ODE parameters from the time-course gene expression data. We also summarize the consistency and asymptotic normality results for the generalized profiling estimates.</div>','<b>Title: </b>Predicting dynamic signaling network response under unseen perturbations<br><b>Year: </b>2014<br><b>Author: </b> F. Zhu and  Yf. Guan<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1093/bioinformatics/btu382"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>regulatory networks, gene regulatory, gene expression, network inference, expression data, network structure, gene networks, networks grns, regulatory genes, boolean networks, bayesian network, network reconstruction, networks gene, network dynamics, gene regulation, inference method, systems biology, time series, infer gene, network<div style="font-size:11px;"><br><b>Abstract: </b><em>Motivation: Predicting trajectories of signaling networks under complex perturbations is one of the most valuable, but challenging, tasks in systems biology. Signaling networks are involved in most of the biological pathways, and modeling their dynamics has wide applications including drug design and treatment outcome prediction.Results: In this paper, we report a novel model for predicting the cell type-specific time course response of signaling proteins under unseen perturbations. This algorithm achieved the top performance in the 2013 8th Dialogue for Reverse Engineering Assessments and Methods DREAM 8 subchallenge: time course prediction in breast cancer cell lines. We formulate the trajectory prediction problem into a standard regularization problem; the solution becomes solving this discrete ill-posed problem. This algorithm includes three steps: denoising, estimating regression coefficients and modeling trajectories under unseen perturbations. We further validated the accuracy of this method against simulation and experimental data. Furthermore, this method reduces computational time by magnitudes compared to state-of-the-art methods, allowing genome-wide modeling of signaling pathways and time course trajectories to be carried out in a practical time.</div>','<b>Title: </b>Dynamics and robustness of the cardiac progenitor cell induced pluripotent stem cell network during cell phenotypes transition<br><b>Year: </b>2017<br><b>Author: </b> Yg. Yao,  Cz. Ma,  Hy. Deng,  Q. Liu,  W. Cao,  R. Gui,  Tq. Feng and  M. Yi<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1049/iet-syb.2015.0051"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IET SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>allosteric proteins, signaling networks, protein interactions, network model, molecular dynamics, protein complexes, allosteric network, signaling proteins, cell network, biochemical networks, free energy, combinatorial complexity, molecular species, biological systems, receptor signaling, systems biology, chemical species, model, dynamics simulations, cell signaling<div style="font-size:11px;"><br><b>Abstract: </b><em>Robustness is a fundamental characteristic of biological systems since all living systems need to adapt to internal or external perturbations, unpredictable environments, stochastic events and unreliable components, and so on. A long-term challenge in systems biology is to reveal the origin of robustness underlying molecular regulator network. In this study, a simple Boolean model is used to investigate the global dynamic properties and robustness of cardiac progenitor cell CPC induced pluripotent stem cell network that governs reprogramming and directed differentiation process. It is demonstrated that two major attractors correspond to source and target cell phenotypes, respectively, and two dominating attracting trajectories characterise the biological pathways between two major cell phenotypes. In particular, the experimentally observed transition between different cell phenotypes can be reproduced and explained theoretically. Furthermore, the robustness of major attractors and trajectories is largely maintained with respect to small perturbations to the network. Taken together, the CPC-induced pluripotent stem cell network is extremely robustly designed for their functions.</div>','<b>Title: </b>Simulation of the dynamics of primary immunodeficiencies in CD4T-cells<br><b>Year: </b>2017<br><b>Author: </b> Gn. Teku and  M. Vihinen<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0176500"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>boolean network, network models, qualitative models, systems biology, mathematical modeling, boolean models, signaling networks, dynamic models, dynamic modeling, signaling dynamics, logic models, dynamic network, mathematical models, boolean modeling, signal transduction, regulatory networks, drug resistance, biological networks, network approach, network modeling<div style="font-size:11px;"><br><b>Abstract: </b><em>Primary immunodeficiencies PIDs form a large and heterogeneous group of mainly rare disorders that affect the immune system. T-cell deficiencies account for about one-tenth of PIDs, most of them being monogenic. Apart from genetic and clinical information, lots of other data are available for PID proteins and genes, including functions and interactions. Thus, it is possible to perform systems biology studies on the effects of PIDs on T-cell physiology and response. To achieve this, we reconstructed a T-cell network model based on literature mining and TPPIN, a previously published core T-cell network, and performed semi-quantitative dynamic network simulations on both normal and T-cell PID failure modes. The results for several loss-of-function PID simulations correspond to results of previously reported molecular studies. The simulations for TCR PTPRC, LCK, ZAP70 and ITK indicate profound changes to numerous proteins in the network. Significant effects were observed also in the BCL10, CARD11, MALT1, NEMO, IKKB and MAP3K14 simulations. No major effects were observed for PIDs that are caused by constitutively active proteins. The T-cell model facilitates the understanding of the underlying dynamics of PID disease processes. The approach confirms previous knowledge about T-cell signaling network and indicates several new important proteins that may be of interest when developing novel diagnosis and therapies to treat immunological defects.</div>','<b>Title: </b>Information networks for disease: commonalities in human management networks and within-host signalling networks<br><b>Year: </b>2012<br><b>Author: </b> Ka. Garrett<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s10658-012-9963-y"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>EUROPEAN JOURNAL OF PLANT PATHOLOGY<br><b>Cluster-Keywords: </b><em>network ecology, food webs, network models, ecological systems, food web, systems biology, ecological networks, ecosystem model, signalling networks, model results, interaction networks, dynamic model, ecological interactions, network analysis, model network, network structure, transport networks, systems network, ecosystem dynamics, restoration ecology<div style="font-size:11px;"><br><b>Abstract: </b><em>Network models of human epidemics can often be improved by including the effects of behaviour modification in response to information about the approach of epidemics. Similarly, there are opportunities to incorporate the flow of information and its effects in plant disease epidemics in network models at multiple scales. 1 In the case of human management networks for plant disease, each node of a network has four main components: plant communities, microbial communities, human information among researchers, extension agents, farmers, and other stakeholders, and environmental conditions, along with their interactions. The links between nodes, representing the rate of movement between them, have three parts: the rates for plant materials, the rates for microbes, and the rates for information. Network resilience for information flow is an important goal for such systems. Game theory can provide insights into how human agents decide how to invest their efforts in strengthening information networks, and how policies can support more resilient networks. 2 For the case of within-plant signalling networks, each node has a comparable set of four main components: plant signals often in the form of phytohormones and development status, microbial communities and plant disease status, microbial signals often in the form of quorum sensing molecules, and micro-environmental conditions, along with their interactions. In effect, human information is replaced by plant signals and microbial signals in this second model. The links between nodes have three parts: the rates for microbes, the rates for microbial signals which may move separately from the microbes, themselves, and the rates for plant signals. Understanding how to enhance adaptive plant signalling networks and microbial signalling networks that support plant productivity, and disrupt microbial signalling networks that contribute to pathogenicity, will be an important step for improved disease management.</div>','<b>Title: </b>Predicting perturbation patterns from the topology of biological networks<br><b>Year: </b>2018<br><b>Author: </b> M. Santolini and  Al. Barabasi<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1073/pnas.1720589115"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA<br><b>Cluster-Keywords: </b><em>interaction networks, proteinprotein interactions, protein interaction, network model, complex systems, systems biology, protein protein, protein complex, host network, dynamic interactions, network analysis, biological systems, networks network, cellular network, biological network, viral proteins, interactome networks, regulatory networks, immune system, systems network<div style="font-size:11px;"><br><b>Abstract: </b><em>High-throughput technologies, offering an unprecedented wealth of quantitative data underlying the makeup of living systems, are changing biology. Notably, the systematic mapping of the relationships between biochemical entities has fueled the rapid development of network biology, offering a suitable framework to describe disease phenotypes and predict potential drug targets. However, our ability to develop accurate dynamical models remains limited, due in part to the limited knowledge of the kinetic parameters underlying these interactions. Here, we explore the degree to which we can make reasonably accurate predictions in the absence of the kinetic parameters. We find that simple dynamically agnostic models are sufficient to recover the strength and sign of the biochemical perturbation patterns observed in 87 biological models for which the underlying kinetics are known. Surprisingly, a simple distance-based model achieves 65 accuracy. We show that this predictive power is robust to topological and kinetic parameter perturbations, and we identify key network properties that can increase up to 80 the recovery rate of the true perturbation patterns. We validate our approach using experimental data on the chemotactic pathway in bacteria, finding that a network model of perturbation spreading predicts with similar to 80 accuracy the directionality of gene expression and phenotype changes in knock-out and overproduction experiments. These findings show that the steady advances in mapping out the topology of biochemical interaction networks opens avenues for accurate perturbation spread modeling, with direct implications for medicine and drug development.</div>','<b>Title: </b>Multiscale systems biology of trauma-induced coagulopathy<br><b>Year: </b>2018<br><b>Author: </b> E. Tsiklidis,  C. Sims,  T. Sinno and  Sl. Diamond<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1002/wsbm.1418"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>WILEY INTERDISCIPLINARY REVIEWS-SYSTEMS BIOLOGY AND MEDICINE<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Trauma with hypovolemic shock is an extreme pathological state that challenges the body to maintain blood pressure and oxygenation in the face of hemorrhagic blood loss. In conjunction with surgical actions and transfusion therapy, survival requires the patients blood to maintain hemostasis to stop bleeding. The physics of the problem are multiscale: a the systemic circulation sets the global blood pressure in response to blood loss and resuscitation therapy, b local tissue perfusion is altered by localized vasoregulatory mechanisms and bleeding, and c altered blood and vessel biology resulting from the trauma as well as local hemodynamics control the assembly of clotting components at the site of injury. Building upon ongoing modeling efforts to simulate arterial or venous thrombosis in a diseased vasculature, computer simulation of trauma-induced coagulopathy is an emerging approach to understand patient risk and predict response. Despite uncertainties in quantifying the patients dynamic injury burden, multiscale systems biology may help link blood biochemistry at the molecular level to multiorgan responses in the bleeding patient. As an important goal of systems modeling, establishing early metrics of a patients high-dimensional trajectory may help guide transfusion therapy or warn of subsequent later stage bleeding or thrombotic risks.This article is categorized under:Analytical and Computational Methods > Computational Methods Biological Mechanisms > Regulatory Biology Models of Systems Properties and Processes > Mechanistic Models</div>','<b>Title: </b>Adaptive information processing of network modules to dynamic and spatial stimuli<br><b>Year: </b>2019<br><b>Author: </b> J. Krishnan and  I. Floros<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/s12918-019-0703-1"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>blood flow, blood cells, flow rate, blood pumps, cells, cell, results, patients, stem cells, methods, blood, patients blood, flow, patients methods, patients objectives, cell viability, objectives, results cells, blood pressure, cell culture<div style="font-size:11px;"><br><b>Abstract: </b><em>BackgroundAdaptation and homeostasis are basic features of information processing in cells and seen in a broad range of contexts. Much of the current understanding of adaptation in network modules/motifs is based on their response to simple stimuli. Recently, there have also been studies of adaptation in dynamic stimuli. However a broader synthesis of how different circuits of adaptation function, and which circuits enable a broader adaptive behaviour in classes of more complex and spatial stimuli is largely missing.ResultsWe study the response of a variety of adaptive circuits to time-varying stimuli such as ramps, periodic stimuli and static and dynamic spatial stimuli. We find that a variety of responses can be seen in ramp stimuli, making this a basis for discriminating between even similar circuits. We also find that a number of circuits adapt exactly to ramp stimuli, and dissect these circuits to pinpoint what characteristics architecture, feedback, biochemical aspects, information processing ingredients allow for this. These circuits include incoherent feedforward motifs, inflow-outflow motifs and transcritical circuits. We find that changes in location in such circuits where a signal acts can result in non-adaptive behaviour in ramps, even though the location was associated with exact adaptation in step stimuli. We also demonstrate that certain augmentations of basic inflow-outflow motifs can alter the behaviour of the circuit from exact adaptation to non-adaptive behaviour. When subject to periodic stimuli, some circuits inflow-outflow motifs and transcritical circuits are able to maintain an average output independent of the characteristics of the input. We build on this to examine the response of adaptive circuits to static and dynamic spatial stimuli. We demonstrate how certain circuits can exhibit a graded response in spatial static stimuli with an exact maintenance of the spatial mean-value. Distinct features which emerge from the consideration of dynamic spatial stimuli are also discussed. Finally, we also build on these results to show how different circuits which show any combination of presence or absence of exact adaptation in ramps, exact mainenance of time average output in periodic stimuli and exact maintenance of spatial average of output in static spatial stimuli may be realized.ConclusionsBy studying a range of network circuits/motifs on one hand and a range of stimuli on the other, we isolate characteristics of these circuits structural which enable different degrees of exact adaptive and homeostatic behaviour in such stimuli, how they may be combined, and also identify cases associated with non-homeostatic behaviour. We also reveal constraints associated with locations where signals may act to enable homeostatic behaviour and constraints associated with augmentations of circuits. This consideration of multiple experimentally/naturally relevant stimuli along with circuits of adaptation of relevance in natural and engineered biology, provides a platform for deepening our understanding of adaptive and homeostatic behaviour in natural systems, bridging the gap between models of adaptation and experiments and in engineering homeostatic synthetic circuits.</div>','<b>Title: </b>Modelling and analysing neural networks using a hybrid process algebra<br><b>Year: </b>2016<br><b>Author: </b> Rj. Colvin<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.tcs.2015.08.019"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>THEORETICAL COMPUTER SCIENCE<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Research involving artificial neural networks has tended to be driven towards efficient computation, especially in the domain of pattern recognition, or towards elucidating biological processes in the brain. Models have become more detailed as our understanding of the biology of the brain has increased, incorporating real-time behaviour of individual neurons interacting within complex system structures and dynamics. There are few examples of abstract and fully formal models of biologically plausible neural networks: in the neural networks literature models are often presented as a mixture of mathematical equations and natural language, supported by simulation code and associated experimental results. The informality often hides or obscures important aspects of a particular model, and leaves a large conceptual gap between the model descriptions and the usually low-level programming code used to simulate them.The main contribution of this paper is formally modelling and analysing a biologically plausible neural network model from the literature that exhibits complex neuron-level behaviour and network-level structure. To achieve this a modelling language Pann is developed, based on the process algebras CSP and Hybrid chi. It is designed to be convenient for mixing the behaviour of discrete events such as a neuron spike with mutable continuous and discrete variables representing chemical properties of a neuron, for instance. Its behaviour is defined using an operational semantics, from which a set of general properties of the language is proved.The groundwork for the biological model is laid by first formalising some well-known concepts from the artificial neural networks domain, such as feedforward behaviour, backpropagation, and recurrent neural networks. The Pann model of a feedforward network, comprising a set of communicating processes representing individual neurons, is proved equivalent to the standard one-line calculation of feedforward behaviour. C 2015 Elsevier B.V. All rights reserved.</div>','<b>Title: </b>Linking the Transcriptional Profiles and the Physiological States of Mycobacterium tuberculosis during an Extended Intracellular Infection<br><b>Year: </b>2012<br><b>Author: </b> Kh. Rohde,  Dft. Veiga,  S. Caldwell,  G. Balazsi and  Dg. Russell<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.ppat.1002769"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS PATHOGENS<br><b>Cluster-Keywords: </b><em>sensitivity analysis, systems biology, robustness analysis, network model, network inference, regulatory network, systems analysis, gene expression, parametric uncertainty, biological networks, mathematical models, biochemical networks, biological functions, cellular networks, biological system, parametric sensitivity, dynamical models, dynamic system, networks analysis, gene regulatory<div style="font-size:11px;"><br><b>Abstract: </b><em>Intracellular pathogens such as Mycobacterium tuberculosis have evolved strategies for coping with the pressures encountered inside host cells. The ability to coordinate global gene expression in response to environmental and internal cues is one key to their success. Prolonged survival and replication within macrophages, a key virulence trait of M. tuberculosis, requires dynamic adaptation to diverse and changing conditions within its phagosomal niche. However, the physiological adaptations during the different phases of this infection process remain poorly understood. To address this knowledge gap, we have developed a multi-tiered approach to define the temporal patterns of gene expression in M. tuberculosis in a macrophage infection model that extends from infection, through intracellular adaptation, to the establishment of a productive infection. Using a clock plasmid to measure intracellular replication and death rates over a 14-day infection and electron microscopy to define bacterial integrity, we observed an initial period of rapid replication coupled with a high death rate. This was followed by period of slowed growth and enhanced intracellular survival, leading finally to an extended period of net growth. The transcriptional profiles of M. tuberculosis reflect these physiological transitions as the bacterium adapts to conditions within its host cell. Finally, analysis with a Transcriptional Regulatory Network model revealed linked genetic networks whereby M. tuberculosis coordinates global gene expression during intracellular survival. The integration of molecular and cellular biology together with transcriptional profiling and systems analysis offers unique insights into the host-driven responses of intracellular pathogens such as M. tuberculosis.</div>','<b>Title: </b>Systems biology of Ewing sarcoma: a network model of EWS-FLI1 effect on proliferation and apoptosis<br><b>Year: </b>2013<br><b>Author: </b> G. Stoll,  D. Surdez,  F. Tirode,  K. Laud,  E. Barillot,  A. Zinovyev and  O. Delattre<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1093/nar/gkt678"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>NUCLEIC ACIDS RESEARCH<br><b>Cluster-Keywords: </b><em>gene regulatory, systems biology, regulatory networks, gene network, stem cell, network model, transcription factors, gene expression, cell transcriptomes, network analysis, data analysis, stem cells, network grnsight, cell transformation, weighted network, gene ontology, networks grns, graph model, graph layout, endothelial cell<div style="font-size:11px;"><br><b>Abstract: </b><em>Ewing sarcoma is the second most frequent pediatric bone tumor. In most of the patients, a chromosomal translocation leads to the expression of the EWS-FLI1 chimeric transcription factor that is the major oncogene in this pathology. Relative genetic simplicity of Ewing sarcoma makes it particularly attractive for studying cancer in a systemic manner. Silencing EWS-FLI1 induces cell cycle alteration and ultimately leads to apoptosis, but the exact molecular mechanisms underlying this phenotype are unclear. In this study, a network linking EWS-FLI1 to cell cycle and apoptosis phenotypes was constructed through an original method of network reconstruction. Transcriptome time-series after EWS-FLI1 silencing were used to identify core modulated genes by an original scoring method based on fitting expression profile dynamics curves. Literature data mining was then used to connect these modulated genes into a network. The validity of a subpart of this network was assessed by siRNA/RT-QPCR experiments on four additional Ewing cell lines and confirmed most of the links. Based on the network and the transcriptome data, CUL1 was identified as a new potential target of EWS-FLI1. Altogether, using an original methodology of data integration, we provide the first version of EWS-FLI1 network model of cell cycle and apoptosis regulation.</div>','<b>Title: </b>Evolutionary ecology in silico: evolving food webs, migrating population and speciation<br><b>Year: </b>2005<br><b>Author: </b> D. Stauffer,  A. Kunwar and  D. Chowdhury<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.physa.2004.12.036"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PHYSICA A-STATISTICAL MECHANICS AND ITS APPLICATIONS<br><b>Cluster-Keywords: </b><em>network ecology, food webs, network models, ecological systems, food web, systems biology, ecological networks, ecosystem model, signalling networks, model results, interaction networks, dynamic model, ecological interactions, network analysis, model network, network structure, transport networks, systems network, ecosystem dynamics, restoration ecology<div style="font-size:11px;"><br><b>Abstract: </b><em>After a brief review of our recent works on "unified" models of evolutionary ecology, we have generalized our "unified" model by taking into account spatial variations from one "patch" to another. We model the spatial extension of the ecosystem i.e., the geography by a square lattice where each site corresponds to a distinct "patch". A distinct self-organizing hierarchical food web describes the prey predator relations at each patch in the ecosystem. By carrying out computer simulations up to 107 time steps, we found that, depending on the values of the set of parameters, the distribution of the lifetimes of the species can be fitted to power laws, but only over a very restricted regime of lifetimes. We also interpret our model in terms of taxonomy and present results to elucidate some evolutionary trends in genus, family, order, class, phylum, etc. c 2005 Elsevier B.V. All rights reserved.</div>','<b>Title: </b>Metabolic Adaptation after Whole Genome Duplication<br><b>Year: </b>2009<br><b>Author: </b> Mja. Van Hoek and  P. Hogeweg<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1093/molbev/msp160"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MOLECULAR BIOLOGY AND EVOLUTION<br><b>Cluster-Keywords: </b><em>metabolic network, metabolic model, dynamic flux, growth rate, genomescale metabolic, flux balance, balance analysis, metabolic flux, metabolic pathways, dynamic model, metabolic reactions, cellular growth, regulatory network, systems biology, optimal growth, metabolic, modeling metabolic, growth conditions, dynamic analysis, cell growth<div style="font-size:11px;"><br><b>Abstract: </b><em>Whole genome duplications WGDs have been hypothesized to be responsible for major transitions in evolution. However, the effects of WGD and subsequent gene loss oil cellular behavior and metabolism are still poorly understood. Here we develop a genome scale evolutionary model to study the dynamics of gene loss and metabolic adaptation after WGD. Using the metabolic network of Saccharomyces cerevisiae as all example, we primarily study the Outcome of WGD on yeast as it currently is. However, similar results were obtained using a recontructed hypothetical metabolic network of the pre-WGD ancestor. We show that the retention of genes in duplicate in the model, corresponds nicely with those retained in duplicate after the ancestral WGD in S. cerevisiae. Also, we observe that transporter and glycolytic genes have a higher probability to be retained in duplicate after WGD and subsequent gene loss, both in the model as in S. cerevisiae, which leads to all increase in glycolytic flux after WGD. Furthermore, the model shows that WGD leads to better adaptation than small-scale duplications, in environments for which duplication of a whole pathway instead of single reactions is needed to increase fitness. This is indeed the case for adaptation to high glucose levels. Thus, Our model confirms the hypothesis that WGD has been important in the adaptation of yeast to the new, glucose-rich environment that arose after the appearance of angiosperms. Moreover, the model shows that WGD is almost always detrimental oil the short terml in environments to which the lineage is preadapted, but can have immediate fitness benefits in "new" environments. This explains why WGD, while pivotal in the evolution of many lineages and an apparent "easy" genetic operator, Occurs relatively rarely.</div>','<b>Title: </b>Systems biology of the human red blood cell<br><b>Year: </b>2006<br><b>Author: </b> N. Jamshidi and  Bo. Palsson<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.bcmd.2006.01.006"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BLOOD CELLS MOLECULES AND DISEASES<br><b>Cluster-Keywords: </b><em>immune system, network models, immune response, model, biological systems, models, model reduction, dynamic model, biological networks, system, mathematical models, biochemical network, response model, systems, systems biology, network, immune, biology model, reaction networks, system systems<div style="font-size:11px;"><br><b>Abstract: </b><em>While the potential possibilities of systems biology appear to hold great promise, there are still many challenges to overcome, notably lack of complete characterization of biological components and their interactions and ways to measure these changes experimentally. The comparative simplicity of the human red cell sidesteps much of these challenges. Detailed models of human red cell metabolism have been developed for the last quarter century and continued development of these models has resulted in the ability to make functional phenotype predictions, in silico. Additionally, existing experimental proteomic and metabolomic technologies now provide the capability to perform individualized characterization of erythrocytes. While an experimentally testable systems biology model of most human cells is presently out of reach, it may be achievable with the red cell. c 2006 Elsevier Inc. All rights reserved.</div>','<b>Title: </b>Mammalian Cell Culture Process for Monoclonal Antibody Production: Nonlinear Modelling and Parameter Estimation<br><b>Year: </b>2015<br><b>Author: </b> D. Selisteanu,  D. Sendrescu,  V. Georgeanu and  M. Roman<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1155/2015/598721"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOMED RESEARCH INTERNATIONAL<br><b>Cluster-Keywords: </b><em>systems biology, regulatory network, interaction networks, network model, gene regulatory, network, network biology, networks, gene network, response networks, network dynamics, dynamic model, cell networks, gene expression, biological networks, system dynamics, dynamic network, model, biological systems, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Monoclonal antibodies mAbs are at present one of the fastest growing products of pharmaceutical industry, with widespread applications in biochemistry, biology, and medicine. The operation of mAbs production processes is predominantly based on empirical knowledge, the improvements being achieved by using trial-and-error experiments and precedent practices. The nonlinearity of these processes and the absence of suitable instrumentation require an enhanced modelling effort and modern kinetic parameter estimation strategies. The present work is dedicated to nonlinear dynamic modelling and parameter estimation for a mammalian cell culture process used for mAb production. By using a dynamical model of such kind of processes, an optimization-based technique for estimation of kinetic parameters in the model of mammalian cell culture process is developed. The estimation is achieved as a result of minimizing an error function by a particle swarm optimization PSO algorithm. The proposed estimation approach is analyzed in this work by using a particular model of mammalian cell culture, as a case study, but is generic for this class of bioprocesses. The presented case study shows that the proposed parameter estimation technique provides a more accurate simulation of the experimentally observed process behaviour than reported in previous studies.</div>','<b>Title: </b>Dynamic Modelling of Pathways to Cellular Senescence Reveals Strategies for Targeted Interventions<br><b>Year: </b>2014<br><b>Author: </b> Pd. Pezze,  G. Nelson,  Eg. Otten,  Vi. Korolchuk,  Tbl. Kirkwood,  T. Von Zglinicki and  Dp. Shanley<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pcbi.1003728"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS COMPUTATIONAL BIOLOGY<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Cellular senescence, a state of irreversible cell cycle arrest, is thought to help protect an organism from cancer, yet also contributes to ageing. The changes which occur in senescence are controlled by networks of multiple signalling and feedback pathways at the cellular level, and the interplay between these is difficult to predict and understand. To unravel the intrinsic challenges of understanding such a highly networked system, we have taken a systems biology approach to cellular senescence. We report a detailed analysis of senescence signalling via DNA damage, insulin-TOR, FoxO3a transcription factors, oxidative stress response, mitochondrial regulation and mitophagy. We show in silico and in vitro that inhibition of reactive oxygen species can prevent loss of mitochondrial membrane potential, whilst inhibition of mTOR shows a partial rescue of mitochondrial mass changes during establishment of senescence. Dual inhibition of ROS and mTOR in vitro confirmed computational model predictions that it was possible to further reduce senescence-induced mitochondrial dysfunction and DNA double-strand breaks. However, these interventions were unable to abrogate the senescence-induced mitochondrial dysfunction completely, and we identified decreased mitochondrial fission as the potential driving force for increased mitochondrial mass via prevention of mitophagy. Dynamic sensitivity analysis of the model showed the network stabilised at a new late state of cellular senescence. This was characterised by poor network sensitivity, high signalling noise, low cellular energy, high inflammation and permanent cell cycle arrest suggesting an unsatisfactory outcome for treatments aiming to delay or reverse cellular senescence at late time points. Combinatorial targeted interventions are therefore possible for intervening in the cellular pathway to senescence, but in the cases identified here, are only capable of delaying senescence onset.</div>','<b>Title: </b>Mathematical modeling of monolignol biosynthesis in Populus xylem<br><b>Year: </b>2010<br><b>Author: </b> Y. Lee and  Eo. Voit<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.mbs.2010.08.009"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MATHEMATICAL BIOSCIENCES<br><b>Cluster-Keywords: </b><em>systems biology, dynamic models, biological systems, model, dynamic systems, computational systems, network model, experimental data, model parameters, systems, systems model, models, pathway systems, data, biochemical pathway, dynamics model, biochemical networks, computational models, network, system<div style="font-size:11px;"><br><b>Abstract: </b><em>Recalcitrance of lignocellulosic biomass to sugar release is a central issue in the production of biofuel as an economically viable energy source. Among all contributing factors, variations in lignin content and its syringyl-guaiacyl monomer composition have been directly linked with the yield of fermentable sugars. While recent advances in genomics and metabolite profiling have significantly broadened our understanding of lignin biosynthesis, its regulation at the pathway level is yet poorly understood. During the past decade, computational and mathematical methods of systems biology have become effective tools for deciphering the structure and regulation of complex metabolic networks. As increasing amounts of data from various organizational levels are being published, the application of these methods to studying lignin biosynthesis appears to be very beneficial for the future development of genetically engineered crops with reduced recalcitrance. Here, we use techniques from flux balance analysis and nonlinear dynamic modeling to construct a mathematical model of monolignol biosynthesis in Populus xylem. Various types of experimental data from the literature are used to identify the statistically most significant parameters and to estimate their values through an ensemble approach. The thus generated ensemble of models yields results that are quantitatively consistent with several transgenic experiments, including two experiments not used in the model construction. Additional model results not only reveal probable substrate saturation at steps leading to the synthesis of sinapyl alcohol, but also suggest that the ratio of syringyl to guaiacyl monomers might not be affected by genetic modulations prior to the reactions involving coniferaldehyde. This latter model prediction is directly supported by data from transgenic experiments. Finally, we demonstrate the applicability of the model in metabolic engineering, where the pathway is to be optimized toward a higher yield of xylose through modification of the relative amounts of the two major monolignols. The results generated by our preliminary model of in vivo lignin biosynthesis are encouraging and demonstrate that mathematical modeling is poised to become an effective and predictive complement to traditional biotechnological and transgenic approaches, not just in microorganisms but also in plants. C 2010 Elsevier Inc. All rights reserved.</div>','<b>Title: </b>Segment Particle Swarm Optimization Adoption for Large-Scale Kinetic Parameter Identification of Escherichia Coli Metabolic Network Model<br><b>Year: </b>2018<br><b>Author: </b> Mak. Azrag,  Taa. Kadir and  As. Jaber<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1109/ACCESS.2018.2885118"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IEEE ACCESS<br><b>Cluster-Keywords: </b><em>kinetic models, metabolic networks, kinetic parameters, dynamic models, systems biology, network model, dynamic metabolic, biochemical network, rate constants, enzyme kinetics, kinetic data, adjustable parameters, metabolic model, parameter estimation, dynamic optimization, model selection, mathematical model, kinetic network, model response, model<div style="font-size:11px;"><br><b>Abstract: </b><em>Kinetic parameter identification in the dynamic metabolic model of Escherichia coli E. coli has become important and is needed to obtain appropriate metabolite and enzyme data that are valid under in vilv conditions. The dynamic metabolic model under study represents five metabolic pathways with more than 170 kinetic parameters at steady state with a 0.1 dilution rate. In this paper, identification is declared in two steps. The first step is to identify which kinetic parameters have a higher impact on the model response using local sensitivity analysis results upon increasing each kinetic parameter up to 2.0 by steps of 0.5, while the second step uses highly sensitive kinetic results to be identified and minimized the model simulation metabolite errors using real experimental data by adopting. However, this paper focuses on adopting segment particle swarm optimization PSO and PSO algorithms for large-scale kinetic parameters identification. Among the 170 kinetic parameters investigated, seven kinetic parameters were found to be the most effective kinetic parameters in the model response after finalizing the sensitivity. The seven sensitive kinetic parameters were used in both the algorithms to minimize the model response errors. The validation results proved the effectiveness of both the proposed methods, which identified the kinetics and minimized the model response errors perfectly.</div>','<b>Title: </b>Approximating individual interactions in compartmental system dynamics models<br><b>Year: </b>2018<br><b>Author: </b> Pj. Lamberson<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1002/sdr.1599"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SYSTEM DYNAMICS REVIEW<br><b>Cluster-Keywords: </b><em>complex networks, complex systems, network theory, network structure, biological network, social network, network science, dynamic networks, network systems, network model, network topology, systems biology, network, network analysis, system dynamics, networks, realworld networks, network automata, epidemic network, model network<div style="font-size:11px;"><br><b>Abstract: </b><em>This paper introduces an approach known as a pair approximation developed in physics and mathematical biology that captures some of the effects of network structure in a compartmental system dynamics CSD model without substantially increasing model complexity or computational efficiency. We describe the approach using a familiar example: the standard SIS model from epidemiology that underpins many models from the marketing and innovation diffusion literature. We show that the pair approximation better approximates the results of an agent-based simulation than the standard CSD. The model provides insights regarding the epidemic threshold, with implications for halting the spread of a disease or encouraging the spread of a product, that cannot be obtained from the standard CSD. We further illustrate the technique through the development of two new CSD models: a model of social contagion, SIS2, that allows for social recovery, and a model of technology adoption with local network effects. Copyright c 2018 System Dynamics Society</div>','<b>Title: </b>Control of Boolean networks: Hardness results and algorithms for tree structured networks<br><b>Year: </b>2007<br><b>Author: </b> T. Akutsu,  M. Hayashida,  Wk. Ching and  Mk. Ng<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.jtbi.2006.09.023"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF THEORETICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>boolean networks, regulatory networks, genetic regulatory, gene regulatory, probabilistic boolean, network, networks, model, network model, probabilistic model, model checking, networks pbns, network dynamics, markov chain, regulatory, models, optimal control, boolean, networks models, networks boolean<div style="font-size:11px;"><br><b>Abstract: </b><em>Finding control strategies of cells is a challenging and important problem in the post-genomic era. This paper considers theoretical aspects of the control problem using the Boolean network BN, which is a simplified model of genetic networks. It is shown that finding a control strategy leading to the desired global state is computationally intractable NP-hard in general. Furthermore, this hardness result is extended for BNs with considerably restricted network structures. These results justify existing exponential time algorithms for finding control strategies for probabilistic Boolean networks PBNs. On the other hand, this paper shows that the control problem can be solved in polynomial time if the network has a tree structure. Then, this algorithm is extended for the case where the network has a few loops and the number of time steps is small. Though this paper focuses on theoretical aspects, biological implications of the theoretical results are also discussed. c 2006 Elsevier Ltd. All rights reserved.</div>','<b>Title: </b>Evolutionary optimization with data collocation for reverse engineering of biological networks<br><b>Year: </b>2005<br><b>Author: </b> Ky. Tsai and  Fs. Wang<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1093/bioinformatics/bti099"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>ssystem model, biological systems, biological networks, parameter estimation, biochemical networks, dynamic biological, proposed algorithm, regulatory networks, structure identification, model structure, genetic algorithm, gene regulatory, gene expression, network inference, network motif, network structure, gene networks, network identification, network inferring, estimation method<div style="font-size:11px;"><br><b>Abstract: </b><em>Motivation: Modern experimental biology is moving away from analyses of single elements to whole-organism measurements. Such measured time-course data contain a wealth of information about the structure and dynamic of the pathway or network. The dynamic modeling of the whole systems is formulated as a reverse problem that requires a well-suited mathematical model and a very efficient computational method to identify the model structure and parameters. Numerical integration for differential equations and finding global parameter values are still two major challenges in this field of the parameter estimation of nonlinear dynamic biological systems.Results: We compare three techniques of parameter estimation for nonlinear dynamic biological systems. In the proposed scheme, the modified collocation method is applied to convert the differential equations to the system of algebraic equations. The observed time-course data are then substituted into the algebraic system equations to decouple system interactions in order to obtain the approximate model profiles. Hybrid differential evolution HDE with population size of five is able to find a global solution. The method is not only suited for parameter estimation but also can be applied for structure identification. The solution obtained by HDE is then used as the starting point for a local search method to yield the refined estimates.</div>','<b>Title: </b>A dynamical model with adaptive pixel moving for microarray images segmentation<br><b>Year: </b>2004<br><b>Author: </b> Apg. Damiance,  L. Zhao and  Acplf. Carvalho<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.rti.2004.05.008"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>REAL-TIME IMAGING<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Gene expression analysis is one of the main research issues in computational biology. Such analysis can provide very relevant information related to cell activity. Several techniques have been employed for this analysis. One of them is the analysis of microarray images. This paper proposes a new data clustering method based on dynamical system modelling for the segmentation of microarray images.The proposed approach employs a network consisting of interacting elements, where each element represents an input data as an attribute vector. Each element of the network receives attractions from other elements within a certain region. Those attractions, determined by a predefined similarity measure, drive the elements to converge to their corresponding cluster centre. With this model, neither the number of pixel clusters nor the initial guessing of cluster centres is required. Moreover, the proposed model allows the omission of the gridding process. The results obtained so far have been very promising. C 2004 Elsevier Ltd. All rights reserved.</div>','<b>Title: </b>Mathematical model of biofilm-mediated pathogen persistence in a water distribution network with time-constant flows<br><b>Year: </b>2018<br><b>Author: </b> S. Al Marzooq,  A. Ortiz-Lugo and  Bl. Vaughan<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1017/S0956792518000281"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>EUROPEAN JOURNAL OF APPLIED MATHEMATICS<br><b>Cluster-Keywords: </b><em>coupling strength, synchronization transitions, timeperiodic coupling, time delay, logic network, distribution network, network model, oscillatory networks, neuron networks, water distribution, stochastic resonance, gene networks, multiple synchronization, firing synchronization, neuronal networks, firing behaviors, logical networks, results show, regulatory networks, cell polarization<div style="font-size:11px;"><br><b>Abstract: </b><em>In industrialized nations, potable water is often provided through sophisticated water distribution systems. If pathogenic bacteria are introduced into the water distribution network, the presence of a biofilm can lead to biofilm-assisted retention of the pathogens, affecting the potability of the water. To study the dynamics of planktonic and biofilm-bound pathogens within the large network of pipes in a water distribution system, we develop a network model governing the concentration of introduced pathogens within the bulk fluid and the biofilms lining the pipes. Under time-constant flow regimes within the network, we prove that the long-time behaviour of the entire network is dependent on the Lyapunov exponents for each connection in the network when viewed in isolation and the network connectivity. An efficient algorithm is developed for predicting the long-time behaviour of the pathogen population within large networks using the networks topological ordering. The algorithms predictions are validated using numerical simulations of the full non-linear system on a range of water distribution network sizes.</div>','<b>Title: </b>Turing patterns and apparent competition in predator-prey food webs on networks<br><b>Year: </b>2012<br><b>Author: </b> Ld. Fernandes and  Mam. De Aguiar<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1103/PhysRevE.86.056203"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PHYSICAL REVIEW E<br><b>Cluster-Keywords: </b><em>biological system, network, pattern formation, system, network analysis, systems, turing patterns, patterns, reaction networks, networks, turing system, network topologies, stoichiometric network, networks pattern, network topology, biological, model, gradient formation, pattern, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Reaction-diffusion systems may lead to the formation of steady-state heterogeneous spatial patterns, known as Turing patterns. Their mathematical formulation is important for the study of pattern formation in general and plays central roles in many fields of biology, such as ecology and morphogenesis. Here we show that Turing patterns may have a decisive role in shaping the abundance distribution of predators and prey living in patchy landscapes. We extend the original model proposed by Nakao and Mikhailov Nat. Phys. 6, 544 2010 by considering food chains with several interacting pairs of prey and predators distributed on a scale-free network of patches. We identify patterns of species distribution displaying high degrees of apparent competition driven by Turing instabilities. Our results provide further indication that differences in abundance distribution among patches can be generated dynamically by self organized Turing patterns and not only by intrinsic environmental heterogeneity.</div>','<b>Title: </b>Reconstructing nonlinear dynamic models of gene regulation using stochastic sampling<br><b>Year: </b>2009<br><b>Author: </b> J. Mazur,  D. Ritter,  G. Reinelt and  L. Kaderali<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1471-2105-10-448"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>systems biology, dynamic models, biological systems, model, dynamic systems, computational systems, network model, experimental data, model parameters, systems, systems model, models, pathway systems, data, biochemical pathway, dynamics model, biochemical networks, computational models, network, system<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: The reconstruction of gene regulatory networks from time series gene expression data is one of the most difficult problems in systems biology. This is due to several reasons, among them the combinatorial explosion of possible network topologies, limited information content of the experimental data with high levels of noise, and the complexity of gene regulation at the transcriptional, translational and post-translational levels. At the same time, quantitative, dynamic models, ideally with probability distributions over model topologies and parameters, are highly desirable.Results: We present a novel approach to infer such models from data, based on nonlinear differential equations, which we embed into a stochastic Bayesian framework. We thus address both the stochasticity of experimental data and the need for quantitative dynamic models. Furthermore, the Bayesian framework allows it to easily integrate prior knowledge into the inference process. Using stochastic sampling from the Bayes posterior distribution, our approach can infer different likely network topologies and model parameters along with their respective probabilities from given data. We evaluate our approach on simulated data and the challenge 3 data from the DREAM 2 initiative. On the simulated data, we study effects of different levels of noise and dataset sizes. Results on real data show that the dynamics and main regulatory interactions are correctly reconstructed.Conclusions: Our approach combines dynamic modeling using differential equations with a stochastic learning framework, thus bridging the gap between biophysical modeling and stochastic inference approaches. Results show that the method can reap the advantages of both worlds, and allows the reconstruction of biophysically accurate dynamic models from noisy data. In addition, the stochastic learning framework used permits the computation of probability distributions over models and model parameters, which holds interesting prospects for experimental design purposes.</div>','<b>Title: </b>Crosstalk and signaling switches in mitogen-activated protein kinase cascades<br><b>Year: </b>2012<br><b>Author: </b> D. Fey,  Dr. Croucher,  W. Kolch and  Bn. Kholodenko<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3389/fphys.2012.00355"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>FRONTIERS IN PHYSIOLOGY<br><b>Cluster-Keywords: </b><em>feedback loops, signaling network, positive feedback, network model, negative feedback, biochemical networks, signaling pathways, signal transduction, systems biology, kinase erk, protein kinase, transduction networks, dynamic model, emt network, erk pathway, cell fate, cell death, biochemical reaction, coupled feedback, impulse response<div style="font-size:11px;"><br><b>Abstract: </b><em>Mitogen-activated protein kinase MARK cascades control cell fate decisions, such as proliferation, differentiation, and apoptosis by integrating and processing intra- and extracellular cues. However, similar MARK kinetic profiles can be associated with opposing cellular decisions depending on cell type, signal strength, and dynamics. This implies that signaling by each individual MARK cascade has to be considered in the context of the entire MARK network. Here, we develop a dynamic model of feedback and crosstalk for the three major MARK cascades; extracellular signal-regulated kinase ERK, p38 mitogen-activated protein kinase p38, c-Jun N-terminal kinase JNK, and also include input from protein kinase B AKT signaling. Focusing on the bistable activation characteristics of the JNK pathway, this model explains how pathway crosstalk harmonizes different MARK responses resulting in pivotal cell fate decisions. We show that JNK can switch from a transient to sustained activity due to multiple positive feedback loops. Once activated, positive feedback locks JNK in a highly active state and promotes cell death. The switch is modulated by the ERK, p38, and AKT pathways. ERK activation enhances the dual specificity phosphatase DUSP mediated dephosphorylation of JNK and shifts the threshold of the apoptotic switch to higher inputs. Activation of p38 restores the threshold by inhibiting ERK activity via the PP1 or PP2A phosphatases. Finally, AKT activation inhibits the JNK positive feedback, thus abrogating the apoptotic switch and allowing only proliferative signaling. Our model facilitates understanding of how cancerous deregulations disturb MARK signal processing and provides explanations for certain drug resistances. We highlight a critical role of DUSP1 and DUSP2 expression patterns in facilitating the switching of JNK activity and show how oncogene induced ERK hyperactivity prevents the normal apoptotic switch explaining the failure of certain drugs to induce apoptosis.</div>','<b>Title: </b>Spatiotemporal Analysis of Cytokinetic Events in Fission Yeast<br><b>Year: </b>2017<br><b>Author: </b> B. Wei,  Bs. Hercyk,  J. Habiyaremye and  M. Das<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3791/55109"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOVE-JOURNAL OF VISUALIZED EXPERIMENTS<br><b>Cluster-Keywords: </b><em>cell cycle, mammalian cell, cdk network, cell proliferation, cell, cells cell, regulatory network, detailed model, cell size, molecular network, skeleton model, cycle arrest, cycle progression, cell division, cell density, kinases cdks, cycle, phase transition, cycle control, network driving<div style="font-size:11px;"><br><b>Abstract: </b><em>Cytokinesis, the final step in cell division is critical for maintaining genome integrity. Proper cytokinesis is important for cell differentiation and development. Cytokinesis involves a series of events that are well coordinated in time and space. Cytokinesis involves the formation of an actomyosin ring at the division site, followed by ring constriction, membrane furrow formation and extra cellular matrix remodeling. The fission yeast, Schizosaccharomyces pombe S. pombe is a well-studied model system that has revealed with substantial clarity the initial events in cytokinesis. However, we do not understand clearly how different cytokinetic events are coordinated spatiotemporally. To determine this, one needs to analyze the different cytokinetic events in great details in both time and in space. Here we describe a microscopy approach to examine different cytokinetic events in live cells. With this approach it is possible to time different cytokinetic events and determine the time of recruitment of different proteins during cytokinesis. In addition, we describe protocols to compare protein localization, and distribution at the site of cell division. This is a basic protocol to study cytokinesis in fission yeast and can also be used for other yeasts and fungal systems.</div>','<b>Title: </b>Bayesian design strategies for synthetic biology<br><b>Year: </b>2011<br><b>Author: </b> Cp. Barnes,  D. Silk and  Mph. Stumpf<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1098/rsfs.2011.0056"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>INTERFACE FOCUS<br><b>Cluster-Keywords: </b><em>circadian clock, biological systems, network models, mathematical models, clock gene, systems biology, model selection, biological networks, network data, experimental data, model parameters, network dynamics, plant circadian, regulatory network, clock model, plant clock, complex network, clock circuit, approximate bayesian, stochastic models<div style="font-size:11px;"><br><b>Abstract: </b><em>We novel biological systems. Bayesian techniques have found widespread application and acceptance in the systems biology community, where they are used for both parameter estimation and model selection. Here we show that the same approaches can also be used in order to engineer synthetic biological systems by inferring the structure and parameters that are most likely to give rise to the dynamics that we require a system to exhibit. Problems that are shared between applications in systems and synthetic biology include the vast potential spaces that need to be searched for suitable models and model parameters; the complex forms of likelihood functions; and the interplay between noise at the molecular level and non-linearity in the dynamics owing to often complex feedback structures. In order to meet these challenges, we have to develop suitable inferential tools and here, in particular, we illustrate the use of approximate Bayesian computation and unscented Kalman filtering-based approaches. These partly complementary methods allow us to tackle a number of recurring problems in the design of biological systems. After a brief exposition of these two methodologies, we focus on their application to oscillatory systems.</div>','<b>Title: </b>Functional Roles of Multiple Feedback Loops in Extracellular Signal-Regulated Kinase and Wnt Signaling Pathways That Regulate Epithelial-Mesenchymal Transition<br><b>Year: </b>2010<br><b>Author: </b> Sy. Shin,  O. Rath,  A. Zebisch,  Sm. Choo,  W. Kolch and  Kh. Cho<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1158/0008-5472.CAN-10-1377"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>CANCER RESEARCH<br><b>Cluster-Keywords: </b><em>feedback loops, signaling network, positive feedback, network model, negative feedback, biochemical networks, signaling pathways, signal transduction, systems biology, kinase erk, protein kinase, transduction networks, dynamic model, emt network, erk pathway, cell fate, cell death, biochemical reaction, coupled feedback, impulse response<div style="font-size:11px;"><br><b>Abstract: </b><em>Epithelial-mesenchymal transition EMT is a key event in the generation of invasive tumor cells. A hallmark of EMT is the repression of E-cadherin expression, which is regulated by various signal transduction pathways including extracellular signal-regulated kinase ERK and Wnt. These pathways are highly interconnected via multiple coupled feedback loops CFL. As the function of such coupled feedback regulations is difficult to analyze experimentally, we used a systems biology approach where computational models were designed to predict biological effects that result from the complex interplay of CFLs. Using epidermal growth factor EGF and Wnt as input and E-cadherin transcriptional regulation as output, we established an ordinary differential equation model of the ERK and Wnt signaling network containing six feedback links and used extensive computer simulations to analyze the effects of these feedback links in isolation and different combinations. The results show that the feedbacks can generate a rich dynamic behavior leading to various dose-response patterns and have a decisive role in determining network responses to EGF and Wnt. In particular, we made two important findings: first, that coupled positive feedback loops composed of phosphorylation of Raf kinase inhibitor RKIP by ERK and transcriptional repression of RKIP by Snail have an essential role in causing a switch-like behavior of E-cadherin expression; and second, that RKIP expression inhibits EMT progression by preventing E-cadherin suppression. Taken together, our findings provide us with a system-level understanding of how RKIP can regulate EMT progression and may explain why RKIP is downregulated in so many metastatic cancer cells. Cancer Res; 7017; 6715-24. C2010 AACR.</div>','<b>Title: </b>Competition-Based Benchmarking of Influence Ranking Methods in Social Networks<br><b>Year: </b>2018<br><b>Author: </b> A. Topirceanu<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1155/2018/4562609"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>COMPLEXITY<br><b>Cluster-Keywords: </b><em>complex networks, complex systems, network theory, network structure, biological network, social network, network science, dynamic networks, network systems, network model, network topology, systems biology, network, network analysis, system dynamics, networks, realworld networks, network automata, epidemic network, model network<div style="font-size:11px;"><br><b>Abstract: </b><em>The development of new methods to identify influential spreaders in complex networks has been a significant challenge in network science over the last decade. Practical significance spans from graph theory to interdisciplinary fields like biology, sociology, economics, and marketing. Despite rich literature in this direction, we find small notable effort to consistently compare and rank existing centralities considering both the topology and the opinion diffusion model, as well as considering the context of simultaneous spreading. To this end, our study introduces a new benchmarking framework targeting the scenario of competitive opinion diffusion; our method differs from classic SIR epidemic diffusion, by employing competition-based spreading supported by the realistic tolerance-based diffusion model. We review a wide range of state-of-the-art node ranking methods and apply our novel method on large synthetic and real-world datasets. Simulations show that our methodology offers much higher quantitative differentiation between ranking methods on the same dataset and notably high granularity for a ranking method over different datasets. We are able to pinpoint-with consistency-which influence the ranking method performs better against the other one, on a given complex network topology. We consider that our framework can offer a forward leap when analysing diffusion characterized by real-time competition between agents. These results can greatly benefit the tackling of social unrest, rumour spreading, political manipulation, and other vital and challenging applications in social network analysis.</div>','<b>Title: </b>Hybrid stochastic simplifications for multiscale gene networks<br><b>Year: </b>2009<br><b>Author: </b> A. Crudu,  A. Debussche and  O. Radulescu<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-3-89"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>regulatory networks, stochastic simulation, gene networks, reaction networks, systems biology, stochastic model, network models, complex systems, networks stochastic, gene regulatory, biological systems, stochastic dynamics, differential equations, protein networks, cell cycle, continuous model, interaction networks, systems chemistry, time delays, biochemical reaction<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Stochastic simulation of gene networks by Markov processes has important applications in molecular biology. The complexity of exact simulation algorithms scales with the number of discrete jumps to be performed. Approximate schemes reduce the computational time by reducing the number of simulated discrete events. Also, answering important questions about the relation between network topology and intrinsic noise generation and propagation should be based on general mathematical results. These general results are difficult to obtain for exact models.Results: We propose a unified framework for hybrid simplifications of Markov models of multiscale stochastic gene networks dynamics. We discuss several possible hybrid simplifications, and provide algorithms to obtain them from pure jump processes. In hybrid simplifications, some components are discrete and evolve by jumps, while other components are continuous. Hybrid simplifications are obtained by partial Kramers-Moyal expansion 1-3 which is equivalent to the application of the central limit theorem to a sub-model. By averaging and variable aggregation we drastically reduce simulation time and eliminate non-critical reactions. Hybrid and averaged simplifications can be used for more effective simulation algorithms and for obtaining general design principles relating noise to topology and time scales. The simplified models reproduce with good accuracy the stochastic properties of the gene networks, including waiting times in intermittence phenomena, fluctuation amplitudes and stationary distributions. The methods are illustrated on several gene network examples.Conclusion: Hybrid simplifications can be used for onion-like multi-layered approaches to multiscale biochemical systems, in which various descriptions are used at various scales. Sets of discrete and continuous variables are treated with different methods and are coupled together in a physically justified approach.</div>','<b>Title: </b>Approximating the stabilization of cellular metabolism by compartmentalization<br><b>Year: </b>2016<br><b>Author: </b> L. Furtauer and  T. Nagele<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s12064-016-0225-y"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>THEORY IN BIOSCIENCES<br><b>Cluster-Keywords: </b><em>kinetic models, metabolic networks, kinetic parameters, dynamic models, systems biology, network model, dynamic metabolic, biochemical network, rate constants, enzyme kinetics, kinetic data, adjustable parameters, metabolic model, parameter estimation, dynamic optimization, model selection, mathematical model, kinetic network, model response, model<div style="font-size:11px;"><br><b>Abstract: </b><em>Biochemical regulation in compartmentalized metabolic networks is highly complex and non-intuitive. This is particularly true for cells of higher plants showing one of the most compartmentalized cellular structures across all kingdoms of life. The interpretation and testable hypothesis generation from experimental data on such complex systems is a challenging step in biological research and biotechnological applications. While it is known that subcellular compartments provide defined reaction spaces within a cell allowing for the tight coordination of complex biochemical reaction sequences, its role in the coordination of metabolic signals during metabolic reprogramming due to environmental fluctuations is less clear. In the present study, we numerically analysed the effects of environmental fluctuations in a subcellular metabolic network with regard to the stability of an experimentally observed steady state in the genetic model plant Arabidopsis thaliana. Applying a method for kinetic parameter normalization, several millions of probable enzyme kinetic parameter constellations were simulated and evaluated with regard to the stability information of the metabolic homeostasis. Information about the stability of the metabolic steady state was derived from real parts of eigenvalues of Jacobian matrices. Our results provide evidence for a differential stabilizing contribution of different subcellular compartments. We could identify stabilizing and destabilizing network components which we could classify according to their subcellular localization. The findings prove that a highly dynamic interplay between intracellular compartments is preliminary for an efficient stabilization of a metabolic homeostasis after environmental perturbation. Further, our results provide evidence that feedback-inhibition originating from the cytosol and plastid seem to stabilize the sucrose homeostasis more efficiently than vacuolar control. In summary, our results indicate stabilizing and destabilizing network components in context of their subcellular organization.</div>','<b>Title: </b>Simulating Multilevel Dynamics of Antimicrobial Resistance in a Membrane Computing Model<br><b>Year: </b>2019<br><b>Author: </b> M. Campos,  R. Capilla,  F. Naya,  R. Futami,  T. Coque,  A. Moya,  V. Fernandez-Lanza,  R. Canton,  Jm. Sempere,  C. Llorens and  F. Baquero<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1128/mBio.02460-18"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MBIO<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Membrane computing is a bio-inspired computing paradigm whose devices are the so-called membrane systems or P systems. The P system designed in this work reproduces complex biological landscapes in the computer world. It uses nested "membrane-surrounded entities" able to divide, propagate, and die; to be transferred into other membranes; to exchange informative material according to flexible rules; and to mutate and be selected by external agents. This allows the exploration of hierarchical interactive dynamics resulting from the probabilistic interaction of genes phenotypes, clones, species, hosts, environments, and antibiotic challenges. Our model facilitates analysis of several aspects of the rules that govern the multilevel evolutionary biology of antibiotic resistance. We examined a number of selected landscapes where we predict the effects of different rates of patient flow from hospital to the community and vice versa, the cross-transmission rates between patients with bacterial propagules of different sizes, the proportion of patients treated with antibiotics, and the antibiotics and dosing found in the opening spaces in the microbiota where resistant phenotypes multiply. We also evaluated the selective strengths of some drugs and the influence of the time 0 resistance composition of the species and bacterial clones in the evolution of resistance phenotypes. In summary, we provide case studies analyzing the hierarchical dynamics of antibiotic resistance using a novel computing model with reciprocity within and between levels of biological organization, a type of approach that may be expanded in the multilevel analysis of complex microbial landscapes.IMPORTANCE The work that we present here represents the culmination of many years of investigation in looking for a suitable methodology to simulate the multi-hierarchical processes involved in antibiotic resistance. Everything started with our early appreciation of the different independent but embedded biological units that shape the biology, ecology, and evolution of antibiotic-resistant microorganisms. Genes, plasmids carrying these genes, cells hosting plasmids, populations of cells, microbial communities, and hosts populations constitute a complex system where changes in one component might influence the other ones. How would it be possible to simulate such a complexity of antibiotic resistance as it occurs in the real world Can the process be predicted, at least at the local level A few years ago, and because of their structural resemblance to biological systems, we realized that membrane computing procedures could provide a suitable frame to approach these questions. Our manuscript describes the first application of this modeling methodology to the field of antibiotic resistance and offers a bunch of examples-just a limited number of them in comparison with the possible ones to illustrate its unprecedented explanatory power.</div>','<b>Title: </b>Reverse-engineering the genetic circuitry of a cancer cell with predicted intervention in chronic lymphocytic leukemia<br><b>Year: </b>2013<br><b>Author: </b> L. Vallat,  Ca. Kemper,  N. Jung,  M. Maumy-Bertrand,  F. Bertrand,  N. Meyer,  A. Pocheville,  Jw. Fisher,  Jg. Gribben and  S. Bahram<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1073/pnas.1211130110"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Cellular behavior is sustained by genetic programs that are progressively disrupted in pathological conditions-notably, cancer. High-throughput gene expression profiling has been used to infer statistical models describing these cellular programs, and development is now needed to guide orientated modulation of these systems. Here we develop a regression-based model to reverse-engineer a temporal genetic program, based on relevant patterns of gene expression after cell stimulation. This method integrates the temporal dimension of biological rewiring of genetic programs and enables the prediction of the effect of targeted gene disruption at the system level. We tested the performance accuracy of this model on synthetic data before reverse-engineering the response of primary cancer cells to a proliferative protumorigenic stimulation in a multistate leukemia biological model i.e., chronic lymphocytic leukemia. To validate the ability of our method to predict the effects of gene modulation on the global program, we performed an intervention experiment on a targeted gene. Comparison of the predicted and observed gene expression changes demonstrates the possibility of predicting the effects of a perturbation in a gene regulatory network, a first step toward an orientated intervention in a cancer cell genetic program.</div>','<b>Title: </b>A modular gradient-sensing network for chemotaxis in Escherichia coli revealed by responses to time-varying stimuli<br><b>Year: </b>2010<br><b>Author: </b> Ts. Shimizu,  Yh. Tu and  Hc. Berg<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1038/msb.2010.37"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MOLECULAR SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>systems biology, biological systems, system dynamics, molecular systems, experimental design, network model, model system, model parameters, experiment design, kinetic parameters, perfect adaptation, experimental data, molecular mechanisms, homeostatic mechanisms, switching dynamics, biological mechanisms, control systems, chemotaxis network, decentralized control, feedback control<div style="font-size:11px;"><br><b>Abstract: </b><em>The Escherichia coli chemotaxis-signaling pathway computes time derivatives of chemoeffector concentrations. This network features modules for signal reception/amplification and robust adaptation, with sensing of chemoeffector gradients determined by the way in which these modules are coupled in vivo. We characterized these modules and their coupling by using fluorescence resonance energy transfer to measure intracellular responses to time-varying stimuli. Receptor sensitivity was characterized by step stimuli, the gradient sensitivity by exponential ramp stimuli, and the frequency response by exponential sine-wave stimuli. Analysis of these data revealed the structure of the feedback transfer function linking the amplification and adaptation modules. Feedback near steady state was found to be weak, consistent with strong fluctuations and slow recovery from small perturbations. Gradient sensitivity and frequency response both depended strongly on temperature. We found that time derivatives can be computed by the chemotaxis system for input frequencies below 0.006 Hz at 22 degrees C and below 0.018 Hz at 32 degrees C. Our results show how dynamic input-output measurements, time honored in physiology, can serve as powerful tools in deciphering cell-signaling mechanisms. Molecular Systems Biology 6: 382; published online 22 June 2010; doi:10.1038/msb.2010.37</div>','<b>Title: </b>PlantSimLab-a modeling and simulation web tool for plant biologists<br><b>Year: </b>2019<br><b>Author: </b> S. Ha,  E. Dimitrova,  S. Hoops,  D. Altarawy,  M. Ansariola,  D. Deb,  J. Glazebrook,  R. Hillmer,  H. Shahin,  F. Katagiri,  J. Mcdowell,  M. Megraw,  J. Setubal,  Bm. Tyler and  R. Laubenbacher<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/s12859-019-3094-9"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>boolean network, network models, qualitative models, systems biology, mathematical modeling, boolean models, signaling networks, dynamic models, dynamic modeling, signaling dynamics, logic models, dynamic network, mathematical models, boolean modeling, signal transduction, regulatory networks, drug resistance, biological networks, network approach, network modeling<div style="font-size:11px;"><br><b>Abstract: </b><em>Background At the molecular level, nonlinear networks of heterogeneous molecules control many biological processes, so that systems biology provides a valuable approach in this field, building on the integration of experimental biology with mathematical modeling. One of the biggest challenges to making this integration a reality is that many life scientists do not possess the mathematical expertise needed to build and manipulate mathematical models well enough to use them as tools for hypothesis generation. Available modeling software packages often assume some modeling expertise. There is a need for software tools that are easy to use and intuitive for experimentalists. Results This paper introduces PlantSimLab, a web-based application developed to allow plant biologists to construct dynamic mathematical models of molecular networks, interrogate them in a manner similar to what is done in the laboratory, and use them as a tool for biological hypothesis generation. It is designed to be used by experimentalists, without direct assistance from mathematical modelers. Conclusions Mathematical modeling techniques are a useful tool for analyzing complex biological systems, and there is a need for accessible, efficient analysis tools within the biological community. PlantSimLab enables users to build, validate, and use intuitive qualitative dynamic computer models, with a graphical user interface that does not require mathematical modeling expertise. It makes analysis of complex models accessible to a larger community, as it is platform-independent and does not require extensive mathematical expertise.</div>','<b>Title: </b>From Pharmacogenomics and Systems Biology to Personalized Care: A Framework of Systems and Dynamical Medicine<br><b>Year: </b>2014<br><b>Author: </b> Q. Yan<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/978-1-4939-0956-81"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PHARMACOGENOMICS IN DRUG DISCOVERY AND DEVELOPMENT, 2ND EDITION<br><b>Cluster-Keywords: </b><em>systems biology, dynamic network, early differentiation, complex diseases, stem cells, erythroid cells, system dynamics, network biomarkers, immune system, tcm syndrome, dnb genes, gene expression, dnb model, stem cell, expression dynamics, model system, mathematical model, model dynamics, dynamic model, sensitivity analysis<div style="font-size:11px;"><br><b>Abstract: </b><em>With the integration of pharmacogenomics and systems biology, personalized medicine would be possible by switching the gear from the reductionism-based and disease-focused medical system toward a dynamical systems-based and human-centric health care. Comprehensive models are needed to represent the properties of complex adaptive systems CASs to elucidate the complexity in health and diseases, including the features of emergence, nonlinearity, self-organization, and adaptation. As all diseases have the dynamical elements, nonlinear time-series analyses are necessary to characterize the system dynamics at various levels to elucidate the physiological and pathological rhythms, oscillations, and feedback loops. Such analyses can help detect patterns across multiple scales in both the spatial e.g., from molecules to cells, from organisms to psychosocial environments and the temporal e.g., from nanoseconds to hours, from years to decades dimensions. Based on such understanding, systems and dynamical medicine can be developed with the emphasis on the whole systems that change over time to address the nonlinearity and interconnectivity toward a holistic and proactive care. Accurate and robust biomarkers with predictive values can be discovered to reflect the systemic conditions and disease stages. Network and dynamical models may support individualized risk analysis, presymptomatic diagnosis, precise prognosis, and integrative interventions. Systems and dynamical medicine may provide the root for the achievement of predictive, preventive, personalized, and participatory P4 medicine.</div>','<b>Title: </b>Communicating oscillatory networks: frequency domain analysis<br><b>Year: </b>2011<br><b>Author: </b> Aec. Ihekwaba and  S. Sedwards<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-5-203"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>dna damage, signalling networks, network dynamics, pmdm system, systems biology, systems dynamics, biological systems, single cell, kinase inhibitors, signaling network, biological networks, cellular automata, cell cycle, oscillation system, temporal dynamics, signaling pathway, individual cells, signaling processes, dynamic model, damage response<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Constructing predictive dynamic models of interacting signalling networks remains one of the great challenges facing systems biology. While detailed dynamical data exists about individual pathways, the task of combining such data without further lengthy experimentation is highly nontrivial. The communicating links between pathways, implicitly assumed to be unimportant and thus excluded, are precisely what become important in the larger system and must be reinstated. To maintain the delicate phase relationships between signals, signalling networks demand accurate dynamical parameters, but parameters optimised in isolation and under varying conditions are unlikely to remain optimal when combined. The computational burden of estimating parameters increases exponentially with increasing system size, so it is crucial to find precise and efficient ways of measuring the behaviour of systems, in order to re-use existing work.Results: Motivated by the above, we present a new frequency domain-based systematic analysis technique that attempts to address the challenge of network assembly by defining a rigorous means to quantify the behaviour of stochastic systems. As our focus we construct a novel coupled oscillatory model of p53, NF-kB and the mammalian cell cycle, based on recent experimentally verified mathematical models. Informed by online databases of protein networks and interactions, we distilled their key elements into simplified models containing the most significant parts. Having coupled these systems, we constructed stochastic models for use in our frequency domain analysis. We used our new technique to investigate the crosstalk between the components of our model and measure the efficacy of certain network-based heuristic measures.Conclusions: We find that the interactions between the networks we study are highly complex and not intuitive: i points of maximum perturbation do not necessarily correspond to points of maximum proximity to influence; ii increased coupling strength does not necessarily increase perturbation; iii different perturbations do not necessarily sum and iv overall, susceptibility to perturbation is amplitude and frequency dependent and cannot easily be predicted by heuristic measures. Our methodology is particularly relevant for oscillatory systems, though not limited to these, and is most revealing when applied to the results of stochastic simulation. The technique is able to characterise precisely the distance in behaviour between different models, different systems and different parts within the same system. It can also measure the difference between different simulation algorithms used on the same system and can be used to inform the choice of dynamic parameters. By measuring crosstalk between subsystems it can also indicate mechanisms by which such systems may be controlled in experiments and therapeutics. We have thus found our technique of frequency domain analysis to be a valuable benchmark systems-biological tool.</div>','<b>Title: </b>A dynamic Bayesian network to represent discrete duration models<br><b>Year: </b>2010<br><b>Author: </b> R. Donat,  P. Leray,  L. Bouillaut and  P. Aknin<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.neucom.2009.09.012"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>NEUROCOMPUTING<br><b>Cluster-Keywords: </b><em>circadian rhythms, systems biology, biological clock, circadian clock, biological rhythms, hpa axis, model, clock network, mathematical models, biological systems, dynamic systems, ode models, biology models, single cell, network, model parameters, experimental data, models, systems, dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>Originally devoted to specific applications such as biology, medicine and demography, duration models are now widely used in economy, finance or reliability. Recent works in various fields of application have shown the relevancy of using Bayesian networks to model complex systems, namely stochastic systems with an underlying distribution that does not fit to a well-known parametric form. In this paper, the description of a specific dynamic Bayesian network, referred to as Graphical Duration Model GDM, is given. A GDM aims to represent a wide range of duration models. Its structure allows especially to fit multi-state systems featuring complex sojourn-time distributions and contextual dependencies. To that end. a duration variable is explicitly introduced to the state transition model which is classically represented by a Markov chain. A recursive algorithm efficiently perform inference in this model is derived along with its proof of correctness and space and time complexity studies. Finally, this approach is illustrated with an application in survival analysis in which the proposed model is compared with the commonly used Markov chain modelling. C 2009 Elsevier B.V. All rights reserved.</div>','<b>Title: </b>Regulatory networks, genes and glycerophospholipid biosynthesis pathway in schistosomiasis: A systems biology view for pharmacological intervention<br><b>Year: </b>2014<br><b>Author: </b> S. Shinde,  M. Mol and  S. Singh<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.gene.2014.08.031"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>GENE<br><b>Cluster-Keywords: </b><em>gene expression, systems biology, metabolic network, network model, network analysis, dynamic model, signaling pathways, signaling network, regulatory network, dynamic network, biology approach, candidate genes, gene network, gene regulatory, biology analysis, mathematical model, network dynamic, biology network, metabolic pathways, pathway analysis<div style="font-size:11px;"><br><b>Abstract: </b><em>Understanding network topology through embracing the global dynamical regulation of genes in an active state space rather than traditional one-gene-one trait approach facilitates the rational drug development process. Schistosomiasis, a neglected tropical disease, has glycerophospholipids as abundant molecules present on its surface. Lack of effective clinical solutions to treat pathogens encourages us to carry out systems-level studies that could contribute to the development of an effective therapy. Development of a strategy for identifying drug targets by combined genome-scale metabolic network and essentiality analyses through in silico approaches provides tantalizing opportunity to investigate the role of protein/substrate metabolism. A genome-scale metabolic network model reconstruction represents choline-phosphate cytidyltransferase as the rate limiting enzyme and regulates the rate of phosphatidylcholine PC biosynthesis. The uptake of choline was regulated by choline concentration, promoting the regulation of phosphocholine synthesis. In Schistosoma, the change in developmental stage could result from the availability of choline, hampering its developmental cycle. There are no structural reports for this protein. In order to inhibit the activity of choline-phosphate cytidyltransferase CCT, it was modeled by homology modeling using 1COZ as the template from Bacillus subtilis. The transition-state stabilization and catalytic residues were mapped as HXGH and RTEGISTT motif. CCT catalyzes the formation of CDP-choline from phosphocholine in which nucleotidyltransferase adds CTP to phosphocholine. The presence of phosphocholine permits the parasite to survive in an immunologically hostile environment. This feature endeavors development of an inhibitor specific for cytidyltransferase in Schistosoma. Flavonolignans were used to inhibit this activity in which hydnowightin showed the highest affinity as compared to miltefosine. C 2014 Elsevier B.V. All rights reserved.</div>','<b>Title: </b>Designing evolvable systems in a framework of robust, resilient and sustainable engineering analysis<br><b>Year: </b>2012<br><b>Author: </b> Ab. Urken,  A. Nimz and  Tm. Schuck<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.aei.2012.05.006"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ADVANCED ENGINEERING INFORMATICS<br><b>Cluster-Keywords: </b><em>network models, gene network, regulatory networks, boolean networks, gene expression, systems biology, neural networks, network dynamics, gene regulatory, biological networks, network structure, gap gene, gene regulation, cell cycle, network, gene interactions, signaling network, quantum systems, system dynamics, networks<div style="font-size:11px;"><br><b>Abstract: </b><em>"Evolvability" is a concept normally associated with biology or ecology, but recent work on control of interdependent critical infrastructures reveals that network informatics systems can be designed to enable artificial, human systems to "evolve". To explicate this finding, we draw on an analogy between disruptive behavior and stable variation in the history of science and the adaptive patterns of robustness and resilience in engineered systems. We present a definition of an evolvable system in the context of a model of robust, resilient and sustainable systems. Our review of this context and standard definitions indicates that many analysts in engineering as well as in biology and ecology do not differentiate Resilience from Robustness. Neither do they differentiate overall dependable system adaptability from a multi-phase process that includes graceful degradation and time-constrained recovery, restabilization, and prevention of catastrophic failure.We analyze how systemic Robustness, Resilience, and Sustainability are related to Evolvability. Our analysis emphasizes the importance of Resilience as an adaptive capability that integrates Sustainability and Robustness to achieve Evolvability.This conceptual framework is used to discuss nine engineering principles that should frame systems thinking about developing evolvable systems. These principles are derived from Kevin Kellys book: Out of Control, which describes living and artificial self-sustaining systems. Kellys last chapter, "The Nine Laws of God," distills nine principles that govern all life-like systems. We discuss how these principles could be applied to engineering evolvability in artificial systems. This discussion is motivated by a wide range of practical problems in engineered artificial systems. Our goal is to analyze a few examples of system designs across engineering disciplines to explicate a common framework for designing and testing artificial systems. This framework highlights managing increasing complexity, intentional evolution, and resistance to disruptive events. From this perspective, we envision a more imaginative and time-sensitive appreciation of the evolution and operation of "reliable" artificial systems. We conclude with a short discussion of two hypothetical examples of engineering evolvable systems in network-centric communications using Error Resilient Data Fusion ERDF and cognitive radio. C 2012 Elsevier Ltd. All rights reserved.</div>','<b>Title: </b>Lumped dynamic model for a bistable genetic regulatory circuit within a variable-volume whole-cell modelling framework<br><b>Year: </b>2009<br><b>Author: </b> G. Maria<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1002/apj.297"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ASIA-PACIFIC JOURNAL OF CHEMICAL ENGINEERING<br><b>Cluster-Keywords: </b><em>immune system, network models, immune response, model, biological systems, models, model reduction, dynamic model, biological networks, system, mathematical models, biochemical network, response model, systems, systems biology, network, immune, biology model, reaction networks, system systems<div style="font-size:11px;"><br><b>Abstract: </b><em>Genetic regulatory circuits GRCs including switches. oscillators, signal amplifiers or filters, and suanalling circuits, are responsible for the control of cell inetabolisin Modelling such complex GRCs is a clifticult task due to high complexity of the the process partly known and the structural. functional and temporal hierarchical organisation of the cell system. Modular lumped representation, grouping some reactions/cornponents and including different types of variables. is a promising alternative allowing individual module characterisatiun and elaboration of extended Simulation platforms for representing the GRC dynamic properties and designing new cell functions. Such models allow to in-silico design rnodified micro-orgamstris with desirable properties for practical applications in bioprocess enginczring and biotechnolovy In the present work, the analysis of a designed bistable switch fornied by two gene expres,.ion modules is performed in a variable-volurne and whole-cell modelling frarnework. by Iminicking the Esheru:hia coli cell growth. The advantages but also limitations of;;uch a new approach are investigated, by using it Hill-type kinetics combined with few elementary steps, with the aim ofbetter representing the adjUstable levels if key intermediates tuning the GRC reaUlatory properties in term,, of stability strenL11h. species connectivity, responsiveness. and regulatory efficiency under stationar and dynarnic perturbations C 2009 Curtin University of Technology and y John Wiley  Sons. Ltd.</div>','<b>Title: </b>Evaluating Uncertainty in Signaling Networks Using Logical Modeling<br><b>Year: </b>2018<br><b>Author: </b> K. Thobe,  C. Kuznia,  C. Sers and  H. Siebert<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3389/fphys.2018.01335"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>FRONTIERS IN PHYSIOLOGY<br><b>Cluster-Keywords: </b><em>boolean network, network models, qualitative models, systems biology, mathematical modeling, boolean models, signaling networks, dynamic models, dynamic modeling, signaling dynamics, logic models, dynamic network, mathematical models, boolean modeling, signal transduction, regulatory networks, drug resistance, biological networks, network approach, network modeling<div style="font-size:11px;"><br><b>Abstract: </b><em>Systems biology studies the structure and dynamics of biological systems using mathematical approaches. Bottom-up approaches create models from prior knowledge but usually cannot cope with uncertainty, whereas top-down approaches infer models directly from data using statistical methods but mostly neglect valuable known information from former studies. Here, we want to present a workflow that includes prior knowledge while allowing for uncertainty in the modeling process. We build not one but all possible models that arise from the uncertainty using logical modeling and subsequently filter for those models in agreement with data in a top-down manner. This approach enables us to investigate new and more complex biological research questions, however, the encoding in such a framework is often not obvious and thus not easily accessible for researcher from life sciences. To mitigate this problem, we formulate a pipeline with specific templates to address some research questions common in signaling network analysis. To illustrate the potential of this approach, we applied the pipeline to growth factor signaling processes in two renal cancer cell lines. These two cell lines originate from similar tissue, but surprisingly showed a very different behavior toward the cancer drug Sorafenib. Thus our aim was to explore differences between these cell lines regarding three sources of uncertainty in one analysis: possible targets of Sorafenib, crosstalk between involved pathways, and the effect of a mutation in mammalian target of Rapamycin mTOR in one of the cell lines. We were able to show that the model pools from the cell lines are disjoint, thus the discrepancies in behavior originate from differences in the cellular wiring. Also the mutation in mTOR is not affecting its activity in the pathway. The results on Sorafenib, while not fully clarifying the mechanisms involved, illustrate the potential of this analysis for generating new hypotheses.</div>','<b>Title: </b>Cerebellarlike Corrective Model Inference Engine for Manipulation Tasks<br><b>Year: </b>2011<br><b>Author: </b> Nr. Luque,  Ja. Garrido,  Rr. Carrillo,  Ojmd. Coenen and  E. Ros<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1109/TSMCB.2011.2138693"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IEEE TRANSACTIONS ON SYSTEMS MAN AND CYBERNETICS PART B-CYBERNETICS<br><b>Cluster-Keywords: </b><em>systems biology, dynamic models, biological systems, model, dynamic systems, computational systems, network model, experimental data, model parameters, systems, systems model, models, pathway systems, data, biochemical pathway, dynamics model, biochemical networks, computational models, network, system<div style="font-size:11px;"><br><b>Abstract: </b><em>This paper presents how a simple cerebellumlike architecture can infer corrective models in the framework of a control task when manipulating objects that significantly affect the dynamics model of the system. The main motivation of this paper is to evaluate a simplified bio-mimetic approach in the framework of a manipulation task. More concretely, the paper focuses on how the model inference process takes place within a feedforward control loop based on the cerebellar structure and on how these internal models are built up by means of biologically plausible synaptic adaptation mechanisms. This kind of investigation may provide clues on how biology achieves accurate control of non-stiff-joint robot with low-power actuators which involve controlling systems with high inertial components. This paper studies how a basic temporal-correlation kernel including long-term depression LTD and a constant long-term potentiation LTP at parallel fiber-Purkinje cell synapses can effectively infer corrective models. We evaluate how this spike-timing-dependent plasticity correlates sensorimotor activity arriving through the parallel fibers with teaching signals dependent on error estimates arriving through the climbing fibers from the inferior olive. This paper addresses the study of how these LTD and LTP components need to be well balanced with each other to achieve accurate learning. This is of interest to evaluate the relevant role of homeostatic mechanisms in biological systems where adaptation occurs in a distributed manner. Furthermore, we illustrate how the temporal-correlation kernel can also work in the presence of transmission delays in sensorimotor pathways. We use a cerebellumlike spiking neural network which stores the corrective models as well-structured weight patterns distributed among the parallel fibers to Purkinje cell connections.</div>','<b>Title: </b>A Dynamic Analysis of IRS-PKR Signaling in Liver Cells: A Discrete Modeling Approach<br><b>Year: </b>2009<br><b>Author: </b> M. Wu,  Xr. Yang and  C. Chan<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0008040"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>network models, gene network, regulatory networks, boolean networks, gene expression, systems biology, neural networks, network dynamics, gene regulatory, biological networks, network structure, gap gene, gene regulation, cell cycle, network, gene interactions, signaling network, quantum systems, system dynamics, networks<div style="font-size:11px;"><br><b>Abstract: </b><em>A major challenge in systems biology is to develop a detailed dynamic understanding of the functions and behaviors in a particular cellular system, which depends on the elements and their inter-relationships in a specific network. Computational modeling plays an integral part in the study of network dynamics and uncovering the underlying mechanisms. Here we proposed a systematic approach that incorporates discrete dynamic modeling and experimental data to reconstruct a phenotype-specific network of cell signaling. A dynamic analysis of the insulin signaling system in liver cells provides a proof-of-concept application of the proposed methodology. Our group recently identified that double-stranded RNA-dependent protein kinase PKR plays an important role in the insulin signaling network. The dynamic behavior of the insulin signaling network is tuned by a variety of feedback pathways, many of which have the potential to cross talk with PKR. Given the complexity of insulin signaling, it is inefficient to experimentally test all possible interactions in the network to determine which pathways are functioning in our cell system. Our discrete dynamic model provides an in silico model framework that integrates potential interactions and assesses the contributions of the various interactions on the dynamic behavior of the signaling network. Simulations with the model generated testable hypothesis on the response of the network upon perturbation, which were experimentally evaluated to identify the pathways that function in our particular liver cell system. The modeling in combination with the experimental results enhanced our understanding of the insulin signaling dynamics and aided in generating a context-specific signaling network.</div>','<b>Title: </b>Static network structure can be used to model the phenotypic effects of perturbations in regulatory networks<br><b>Year: </b>2012<br><b>Author: </b> A. Feiglin,  A. Hacohen,  A. Sarusi,  J. Fisher,  R. Unger and  Y. Ofran<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1093/bioinformatics/bts517"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>regulatory network, systems biology, systems biological, network dynamics, gene expression, biochemical networks, raft proteins, biological networks, gene circuits, gene regulatory, signalling network, lipid raft, nuclear lamina, protein domain, networks systems, biological approaches, expression pattern, protein interaction, cell differentiation, network motifs<div style="font-size:11px;"><br><b>Abstract: </b><em>Motivation: Biological processes are dynamic, whereas the networks that depict them are typically static. Quantitative modeling using differential equations or logic-based functions can offer quantitative predictions of the behavior of biological systems, but they require detailed experimental characterization of interaction kinetics, which is typically unavailable. To determine to what extent complex biological processes can be modeled and analyzed using only the static structure of the network i.e. the direction and sign of the edges, we attempt to predict the phenotypic effect of perturbations in biological networks from the static networkResults: We analyzed three networks from different sources: The EGFR/MAPK and PI3K/AKT network from a detailed experimental study, the TNF regulatory network from the STRING database and a large network of all NCI-curated pathways from the Protein Interaction Database. Altogether, we predicted the effect of 39 perturbations e.g. by one or two drugs on 433 target proteins/genes. In up to 82 of the cases, an algorithm that used only the static structure of the network correctly predicted whether any given protein/gene is upregulated or downregulated as a result of perturbations of other proteins/genes.Conclusion: While quantitative modeling requires detailed experimental data and heavy computations, which limit its scalability for large networks, a wiring-based approach can use available data from pathway and interaction databases and may be scalable. These results lay the foundations for a large-scale approach of predicting phenotypes based on the schematic structure of networks.</div>','<b>Title: </b>Systems Biology Approaches in Aging Research<br><b>Year: </b>2015<br><b>Author: </b> A. Chauhan,  Uw. Liebal,  J. Vera,  S. Baltrusch,  C. Junghanss,  M. Tiedge,  G. Fuellen,  O. Wolkenhauer and  R. Kohling<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1159/000364981"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>AGING AND HEALTH - A SYSTEMS BIOLOGY PERSPECTIVE<br><b>Cluster-Keywords: </b><em>regulatory network, systems biology, systems biological, network dynamics, gene expression, biochemical networks, raft proteins, biological networks, gene circuits, gene regulatory, signalling network, lipid raft, nuclear lamina, protein domain, networks systems, biological approaches, expression pattern, protein interaction, cell differentiation, network motifs<div style="font-size:11px;"><br><b>Abstract: </b><em>Aging is a systemic process which progressively manifests itself at multiple levels of structural and functional organization from molecular reactions and cell-cell interactions in tissues to the physiology of an entire organ. There is ever increasing data on biomedical relevant network interactions for the aging process at different scales of time and space. To connect the aging process at different structural, temporal and spatial scales, extensive systems biological approaches need to be deployed. Systems biological approaches can not only systematically handle the large-scale datasets like high-throughput data and the complexity of interactions feedback loops, cross talk, but also can delve into nonlinear behaviors exhibited by several biological processes which are beyond intuitive reasoning. Several public-funded agencies have identified the synergistic role of systems biology in aging research. Using one of the notable public-funded programs GERONTOSYS, we discuss how systems biological approaches are helping the scientists to find new frontiers in aging research. We elaborate on some systems biological approaches deployed in one of the projects of the consortium ROSage. The systems biology field in aging research is at its infancy. It is open to adapt existing systems biological methodologies from other research fields and devise new aging-specific systems biological methodologies. C 2015 S. Karger AG, Basel</div>','<b>Title: </b>Geometric analysis of pathways dynamics: Application to versatility of TGF-beta receptors<br><b>Year: </b>2016<br><b>Author: </b> Ss. Samal,  A. Naldi,  D. Grigoriev,  A. Weber,  N. Theret and  O. Radulescu<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.biosystems.2016.07.004"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOSYSTEMS<br><b>Cluster-Keywords: </b><em>immune system, network models, immune response, model, biological systems, models, model reduction, dynamic model, biological networks, system, mathematical models, biochemical network, response model, systems, systems biology, network, immune, biology model, reaction networks, system systems<div style="font-size:11px;"><br><b>Abstract: </b><em>We propose a new geometric approach to describe the qualitative dynamics of chemical reactions networks. By this method we identify metastable regimes, defined as low dimensional regions of the phase space close to which the dynamics is much slower compared to the rest of the phase space. These metastable regimes depend on the network topology and on the orders of magnitude of the kinetic parameters. Benchmarking of the method on a computational biology model repository suggests that the number of metastable regimes is sub-exponential in the number of variables and equations. The dynamics of the network can be described as a sequence of jumps from one metastable regime to another. We show that a geometrically computed connectivity graph restricts the set of possible jumps. We also provide finite state machine Markov chain models for such dynamic changes. Applied to signal transduction models, our approach unravels dynamical and functional capacities of signalling pathways, as well as parameters responsible for specificity of the pathway response. In particular, for a model of TGF beta signalling, we find that the ratio of TGFBR2 to TGFBR1 receptors concentrations can be used to discriminate between metastable regimes. Using expression data from the NCI60 panel of human tumor cell lines, we show that aggressive and non-aggressive tumour cell lines function in different metastable regimes and can be distinguished by measuring the relative concentrations of receptors of the two types. C 2016 Elsevier Ireland Ltd. All rights reserved.</div>','<b>Title: </b>GRNsight: a web application and service for visualizing models of small- to medium-scale gene regulatory networks<br><b>Year: </b>2016<br><b>Author: </b> Kd. Dahlquist,  Jdn. Dionisio,  B. Fitzpatrick,  Na. Anguiano,  A. Varshneya,  Bj. Southwick and  M. Samdarshi<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.7717/peerj-cs.85"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PEERJ COMPUTER SCIENCE<br><b>Cluster-Keywords: </b><em>gene regulatory, systems biology, regulatory networks, gene network, stem cell, network model, transcription factors, gene expression, cell transcriptomes, network analysis, data analysis, stem cells, network grnsight, cell transformation, weighted network, gene ontology, networks grns, graph model, graph layout, endothelial cell<div style="font-size:11px;"><br><b>Abstract: </b><em>GRNsight is a web application and service for visualizing models of gene regulatory networks GRNs. A gene regulatory network GRN consists of genes, transcription factors, and the regulatory connections between them which govern the level of expression of mRNA and protein from genes. The original motivation came from our efforts to perform parameter estimation and forward simulation of the dynamics of a differential equations model of a small GRN with 21 nodes and 31 edges. We wanted a quick and easy way to visualize the weight parameters from the model which represent the direction and magnitude of the influence of a transcription factor on its target gene, so we created GRNsight. GRNsight automatically lays out either an unweighted or weighted network graph based on an Excel spreadsheet containing an adjacency matrix where regulators are named in the columns and target genes in the rows, a Simple Interaction Format SIF text file, or a GraphML XML file. When a user uploads an input file specifying an unweighted network, GRNsight automatically lays out the graph using black lines and pointed arrowheads. For a weighted network, GRNsight uses pointed and blunt arrowheads, and colors the edges and adjusts their thicknesses based on the sign positive for activation or negative for repression and magnitude of the weight parameter. GRNsight is written in JavaScript, with diagrams facilitated by D3.js, a data visualization library. Node.js and the Express framework handle server-side functions. GRNsights diagrams are based on D3.jss force graph layout algorithm, which was then extensively customized to support the specific needs of GRNs. Nodes are rectangular and support gene labels of up to 12 characters. The edges are arcs, which become straight lines when the nodes are close together. Self-regulatory edges are indicated by a loop. When a user mouses over an edge, the numerical value of the weight parameter is displayed. Visualizations can be modified by sliders that adjust the force graph layout parameters and through manual node dragging. GRNsight is best-suited for visualizing networks of fewer than 35 nodes and 70 edges, although it accepts networks of up to 75 nodes or 150 edges. GRNsight has general applicability for displaying any small, unweighted or weighted network with directed edges for systems biology or other application domains. GRNsight serves as an example of following and teaching best practices for scientific computing and complying with FAIR principles, using an open and test-driven development model with rigorous documentation of requirements and issues on GitHub. An exhaustive unit testing framework using Mocha and the Chai assertion library consists of around 160 automated unit tests that examine nearly 530 test files to ensure that the program is running as expected. The GRNsight application http://dondi.github.io/GRNsight/ and code https://github.com/dondi/GRNsight are available under the open source BSD license.</div>','<b>Title: </b>Synchronization in complex oscillator networks and smart grids<br><b>Year: </b>2013<br><b>Author: </b> F. Dorfler,  M. Chertkov and  F. Bullo<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1073/pnas.1212134110"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA<br><b>Cluster-Keywords: </b><em>oscillator networks, network dynamics, network, networks, dynamical systems, dynamics, node dynamics, systems, coupled oscillator, synchronization, dynamics network, complex network, coupled system, network modifications, oscillators, network connectivity, feedback vertex, network topology, synchronization properties, collective dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>The emergence of synchronization in a network of coupled oscillators is a fascinating topic in various scientific disciplines. A widely adopted model of a coupled oscillator network is characterized by a population of heterogeneous phase oscillators, a graph describing the interaction among them, and diffusive and sinusoidal coupling. It is known that a strongly coupled and sufficiently homogeneous network synchronizes, but the exact threshold from incoherence to synchrony is unknown. Here, we present a unique, concise, and closed-form condition for synchronization of the fully nonlinear, nonequilibrium, and dynamic network. Our synchronization condition can be stated elegantly in terms of the network topology and parameters or equivalently in terms of an intuitive, linear, and static auxiliary system. Our results significantly improve upon the existing conditions advocated thus far, they are provably exact for various interesting network topologies and parameters; they are statistically correct for almost all networks; and they can be applied equally to synchronization phenomena arising in physics and biology as well as in engineered oscillator networks, such as electrical power networks. We illustrate the validity, the accuracy, and the practical applicability of our results in complex network scenarios and in smart grid applications.</div>','<b>Title: </b>Adaptive neural-based fuzzy modeling for biological systems<br><b>Year: </b>2013<br><b>Author: </b> Sj. Wu,  Ct. Wu and  Jy. Chang<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.mbs.2013.01.004"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MATHEMATICAL BIOSCIENCES<br><b>Cluster-Keywords: </b><em>ssystem model, biological systems, biological networks, parameter estimation, biochemical networks, dynamic biological, proposed algorithm, regulatory networks, structure identification, model structure, genetic algorithm, gene regulatory, gene expression, network inference, network motif, network structure, gene networks, network identification, network inferring, estimation method<div style="font-size:11px;"><br><b>Abstract: </b><em>The inverse problem of identifying dynamic biological networks from their time-course response data set is a cornerstone of systems biology. Hill and Michaelis-Menten model, which is a forward approach, provides local kinetic information. However, repeated modifications and a large amount of experimental data are necessary for the parameter identification. S-system model, which is composed of highly nonlinear differential equations, provides the direct identification of an interactive network. However, the identification of skeletal-network structure is challenging. Moreover, biological systems are always subject to uncertainty and noise. Are there suitable candidates with the potential to deal with noise-contaminated data sets Fuzzy set theory is developed for handing uncertainty, imprecision and complexity in the real world; for example, we say "driving speed is high" wherein speed is a fuzzy variable and high is a fuzzy set, which uses the membership function to indicate the degree of a element belonging to the set words in Italics to denote fuzzy variables or fuzzy sets. Neural network possesses good robustness and learning capability. In this study we hybrid these two together into a neural-fuzzy modeling technique. A biological system is formulated to a multi-input-multi-output MIMO Takagi-Sugeno T-S fuzzy system, which is composed of rule-based linear subsystems. Two kinds of smooth membership functions MFs, Gaussian and Bell-shaped MFs, are used. The performance of the proposed method is tested with three biological systems. C 2013 Elsevier Inc. All rights reserved.</div>','<b>Title: </b>Convergence in parameters and predictions using computational experimental design<br><b>Year: </b>2013<br><b>Author: </b> Dr. Hagen,  Jk. White and  B. Tidor<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1098/rsfs.2013.0008"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>INTERFACE FOCUS<br><b>Cluster-Keywords: </b><em>biological networks, dynamic models, biological systems, model predictions, model parameters, systems biology, mathematical models, biological models, equation models, experimental data, molecular network, interaction networks, model errors, biological interaction, molecular interactions, model, dynamic systems, biological parameters, parameter uncertainty, growth model<div style="font-size:11px;"><br><b>Abstract: </b><em>Typically, biological models fitted to experimental data suffer from significant parameter uncertainty, which can lead to inaccurate or uncertain predictions. One school of thought holds that accurate estimation of the true parameters of a biological system is inherently problematic. Recent work, however, suggests that optimal experimental design techniques can select sets of experiments whose members probe complementary aspects of a biochemical network that together can account for its full behaviour. Here, we implemented an experimental design approach for selecting sets of experiments that constrain parameter uncertainty. We demonstrated with a model of the epidermal growth factor-nerve growth factor pathway that, after synthetically performing a handful of optimal experiments, the uncertainty in all 48 parameters converged below 10 per cent. Furthermore, the fitted parameters converged to their true values with a small error consistent with the residual uncertainty. When untested experimental conditions were simulated with the fitted models, the predicted species concentrations converged to their true values with errors that were consistent with the residual uncertainty. This paper suggests that accurate parameter estimation is achievable with complementary experiments specifically designed for the task, and that the resulting parametrized models are capable of accurate predictions.</div>','<b>Title: </b>Robust Exponential Memory in Hopfield Networks<br><b>Year: </b>2018<br><b>Author: </b> Cj. Hillar and  Nm. Tran<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/s13408-017-0056-2"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF MATHEMATICAL NEUROSCIENCE<br><b>Cluster-Keywords: </b><em>network models, gene network, regulatory networks, boolean networks, gene expression, systems biology, neural networks, network dynamics, gene regulatory, biological networks, network structure, gap gene, gene regulation, cell cycle, network, gene interactions, signaling network, quantum systems, system dynamics, networks<div style="font-size:11px;"><br><b>Abstract: </b><em>The Hopfield recurrent neural network is a classical auto-associative model of memory, in which collections of symmetrically coupled McCulloch-Pitts binary neurons interact to perform emergent computation. Although previous researchers have explored the potential of this network to solve combinatorial optimization problems or store reoccurring activity patterns as attractors of its deterministic dynamics, a basic open problem is to design a family of Hopfield networks with a number of noise-tolerant memories that grows exponentially with neural population size. Here, we discover such networks by minimizing probability flow, a recently proposed objective for estimating parameters in discrete maximum entropy models. By descending the gradient of the convex probability flow, our networks adapt synaptic weights to achieve robust exponential storage, even when presented with vanishingly small numbers of training patterns. In addition to providing a new set of low-density error-correcting codes that achieve Shannons noisy channel bound, these networks also efficiently solve a variant of the hidden clique problem in computer science, opening new avenues for real-world applications of computational models originating from biology.</div>','<b>Title: </b>Systems biochemistry approaches to vertebrate phototransduction: towards a molecular understanding of disease<br><b>Year: </b>2010<br><b>Author: </b> D. DellOrco and  Kw. Koch<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1042/BST0381275"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOCHEMICAL SOCIETY TRANSACTIONS<br><b>Cluster-Keywords: </b><em>systems biology, dynamic network, early differentiation, complex diseases, stem cells, erythroid cells, system dynamics, network biomarkers, immune system, tcm syndrome, dnb genes, gene expression, dnb model, stem cell, expression dynamics, model system, mathematical model, model dynamics, dynamic model, sensitivity analysis<div style="font-size:11px;"><br><b>Abstract: </b><em>Phototransduction in vertebrates represents a paradigm of signalling pathways, in particular those mediated by G-protein-coupled receptors. The variety of protein-protein, protein-ion and protein-nucleotide interactions makes up an intricate network which is finely regulated by activating-deactivating molecules and chemical modifications. The holistic systems properties of the network allow for typical adaptation mechanisms, which ultimately result in fine adjustments of sensitivity and electrical response of the photoreceptor cells to the broad range of light stimuli. In the present article, we discuss a novel bottom-up strategy to study the phototransduction cascade in rod cells starting from the underlying biochemistry. The resulting network model can be simulated and the predicted dynamic behaviour directly compared with data from electrophysiological experiments performed on a wide range of illumination conditions. The advantage of applying procedures typical of systems theory to a well-studied signalling pathway is also discussed. Finally, the potential application to the study of the molecular basis of retinal diseases is highlighted through a practical example, namely the simulation of conditions related to Leber congenital amaurosis.</div>','<b>Title: </b>Synchronization transitions on complex thermo-sensitive neuron networks with time delays<br><b>Year: </b>2010<br><b>Author: </b> Yh. Xie,  Yb. Gong,  Yh. Hao and  Xg. Ma<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.bpc.2009.11.004"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOPHYSICAL CHEMISTRY<br><b>Cluster-Keywords: </b><em>coupling strength, synchronization transitions, timeperiodic coupling, time delay, logic network, distribution network, network model, oscillatory networks, neuron networks, water distribution, stochastic resonance, gene networks, multiple synchronization, firing synchronization, neuronal networks, firing behaviors, logical networks, results show, regulatory networks, cell polarization<div style="font-size:11px;"><br><b>Abstract: </b><em>We have numerically studied the firing synchronization transitions on random thermo-sensitive neuron networks in dependence on information transmission delay tau, network randomness p, and coupling strength g. It is found that as tau is increased the neurons can exhibit transitions from burst synchronization BS to clustering anti-phase synchronization APS, and further to spike synchronization SS. It is also found that, with increasing p or g, there are transitions from spatiotemporal chaos to BS, then to APS, and finally to SS. However, the APS state with p or g exists only for intermediate tau values within a narrow range. For tau values outside this range, the APS state does not appear and the firings change directly from spatiotemporal chaos to BS or SS. These results show that, as time delay can do, network topology and coupling strength can also cause complex synchronization transitions in the neurons. in particular, the novel phenomenon of APS state with p or g shows that, with the help of appropriate random connections or coupling strength, the neurons may exhibit the APS behavior at a certain time delay for which the APS does not appear originally. These findings imply that time delay, network randomness, and coupling strength may have subtle effects on the firing behaviors on neuronal networks, and thus could play important roles in the information processing in neural systems. C 2009 Elsevier B.V. All rights reserved.</div>','<b>Title: </b>Local elastic constants of LacI and implications for allostery<br><b>Year: </b>2015<br><b>Author: </b> Aast. Ribeiro and  V. Ortiz<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.jmgm.2015.01.013"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF MOLECULAR GRAPHICS  MODELLING<br><b>Cluster-Keywords: </b><em>normal mode, mode analysis, elastic network, functional motions, functional dynamics, molecular chaperone, structural biology, acidic residues, network models, molecular dynamics, atomic resolution, protein complexes, conserved functional, hsp dynamics, correlated motions, protein folding, large biomolecular, thioredoxin system, harmonic analysis, protein motions<div style="font-size:11px;"><br><b>Abstract: </b><em>Allostery connects subtle changes in a proteins potential energy surface, induced by perturbations like ligand-binding, to significant changes in its function. Understanding this phenomenon and predicting its occurrence are major goals of current research in biophysics and molecular biology. In this paper we introduce a novel approach for studying complex structural transformations such as those typical for allostery. We show that the calculation and analysis of atomic elastic constants of a known allosterically regulated protein, lac repressor, highlights regions that are particularly prone to suffer structural deformation and are experimentally linked to allosteric function. The calculations are based on a high resolution, all-atom description of the protein. We also show that, for the present system, modifying the description of the system from an all-atom forcefield to an elastic network model yields qualitatively different results, indicating the importance of adequately describing the local environment surrounding the different parts of the protein. C 2015 Elsevier Inc. All rights reserved.</div>','<b>Title: </b>Noise-induced dynamics in the mixed-feedback-loop network motif<br><b>Year: </b>2008<br><b>Author: </b> Df. Li and  Cg. Li<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1103/PhysRevE.77.011903"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PHYSICAL REVIEW E<br><b>Cluster-Keywords: </b><em>circadian rhythms, systems biology, biological clock, circadian clock, biological rhythms, hpa axis, model, clock network, mathematical models, biological systems, dynamic systems, ode models, biology models, single cell, network, model parameters, experimental data, models, systems, dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>In this paper, we present a stochastic model for the mixed-feedback loop MFL, a motif found in integrated cellular networks of transcription regulation and protein-protein interaction. Previous bifurcation analysis indicates that this motif can serve as a bistable switch or a clock. We investigate how extrinsic and intrinsic noise affects its dynamic behaviors systematically. We find that this motif can exploit noise to enrich its dynamical performance. When the MFL is in the bistable region, under fluctuation of extrinsic noise, the MFL system can switch from one steady state to the other and meanwhile one proteins production is amplified for more than three orders of magnitude. Further, from an engineering perspective, this noise-based switch and amplifier for gene expression is very easy to control. Without extrinsic noise, spontaneous transition between states occurs as the consequence of intrinsic noise. Such a switch is controlled by the parameters and system size. On the other hand, intrinsic noise can induce sustained stochastic oscillation when the corresponding deterministic system does not oscillate. Such stochastic oscillation shows the best performance at an optimal noise level, indicating the occurrence of intrinsic noise stochastic resonance which can contribute to the robustness of this oscillator. When considering the effects of extrinsic noise near bifurcation points, a similar phenomenon of extrinsic noise stochastic resonance is unveiled.</div>','<b>Title: </b>Application of Petri net based analysis techniques to signal transduction pathways<br><b>Year: </b>2006<br><b>Author: </b> A. Sackmann,  M. Heiner and  I. Koch<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1471-2105-7-482"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>petri net, petri nets, bayesian networks, biological networks, systems biology, regulatory networks, dynamic bayesian, net model, gene regulatory, network data, biological systems, biological data, network analysis, network models, biochemical networks, time series, network structure, signal transduction, net models, dbn model<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Signal transduction pathways are usually modelled using classical quantitative methods, which are based on ordinary differential equations ODEs. However, some difficulties are inherent in this approach. On the one hand, the kinetic parameters involved are often unknown and have to be estimated. With increasing size and complexity of signal transduction pathways, the estimation of missing kinetic data is not possible. On the other hand, ODEs based models do not support any explicit insights into possible signal- flows within the network. Moreover, a huge amount of qualitative data is available due to high-throughput techniques. In order to get information on the systems behaviour, qualitative analysis techniques have been developed. Applications of the known qualitative analysis methods concern mainly metabolic networks. Petri net theory provides a variety of established analysis techniques, which are also applicable to signal transduction models. In this context special properties have to be considered and new dedicated techniques have to be designed.Methods: We apply Petri net theory to model and analyse signal transduction pathways first qualitatively before continuing with quantitative analyses. This paper demonstrates how to build systematically a discrete model, which reflects provably the qualitative biological behaviour without any knowledge of kinetic parameters. The mating pheromone response pathway in Saccharomyces cerevisiae serves as case study.Results: We propose an approach for model validation of signal transduction pathways based on the network structure only. For this purpose, we introduce the new notion of feasible t-invariants, which represent minimal self-contained subnets being active under a given input situation. Each of these subnets stands for a signal flow in the system. We define maximal common transition sets MCT-sets, which can be used for t-invariant examination and net decomposition into smallest biologically meaningful functional units.Conclusion: The paper demonstrates how Petri net analysis techniques can promote a deeper understanding of signal transduction pathways. The new concepts of feasible t-invariants and MCT-sets have been proven to be useful for model validation and the interpretation of the biological system behaviour. Whereas MCT-sets provide a decomposition of the net into disjunctive subnets, feasible t-invariants describe subnets, which generally overlap. This work contributes to qualitative modelling and to the analysis of large biological networks by their fully automatic decomposition into biologically meaningful modules.</div>','<b>Title: </b>Continuous time boolean modeling for biological signaling: application of Gillespie algorithm<br><b>Year: </b>2012<br><b>Author: </b> G. Stoll,  E. Viara,  E. Barillot and  L. Calzone<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-6-116"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>boolean network, network models, qualitative models, systems biology, mathematical modeling, boolean models, signaling networks, dynamic models, dynamic modeling, signaling dynamics, logic models, dynamic network, mathematical models, boolean modeling, signal transduction, regulatory networks, drug resistance, biological networks, network approach, network modeling<div style="font-size:11px;"><br><b>Abstract: </b><em>Mathematical modeling is used as a Systems Biology tool to answer biological questions, and more precisely, to validate a network that describes biological observations and predict the effect of perturbations. This article presents an algorithm for modeling biological networks in a discrete framework with continuous time.Background: There exist two major types of mathematical modeling approaches: 1 quantitative modeling, representing various chemical species concentrations by real numbers, mainly based on differential equations and chemical kinetics formalism; 2 and qualitative modeling, representing chemical species concentrations or activities by a finite set of discrete values. Both approaches answer particular and often different biological questions. Qualitative modeling approach permits a simple and less detailed description of the biological systems, efficiently describes stable state identification but remains inconvenient in describing the transient kinetics leading to these states. In this context, time is represented by discrete steps. Quantitative modeling, on the other hand, can describe more accurately the dynamical behavior of biological processes as it follows the evolution of concentration or activities of chemical species as a function of time, but requires an important amount of information on the parameters difficult to find in the literature.Results: Here, we propose a modeling framework based on a qualitative approach that is intrinsically continuous in time. The algorithm presented in this article fills the gap between qualitative and quantitative modeling. It is based on continuous time Markov process applied on a Boolean state space. In order to describe the temporal evolution of the biological process we wish to model, we explicitly specify the transition rates for each node. For that purpose, we built a language that can be seen as a generalization of Boolean equations. Mathematically, this approach can be translated in a set of ordinary differential equations on probability distributions. We developed a C software, MaBoSS, that is able to simulate such a system by applying Kinetic Monte-Carlo or Gillespie algorithm on the Boolean state space. This software, parallelized and optimized, computes the temporal evolution of probability distributions and estimates stationary distributions.Conclusions: Applications of the Boolean Kinetic Monte-Carlo are demonstrated for three qualitative models: a toy model, a published model of p53/Mdm2 interaction and a published model of the mammalian cell cycle. Our approach allows to describe kinetic phenomena which were difficult to handle in the original models. In particular, transient effects are represented by time dependent probability distributions, interpretable in terms of cell populations.</div>','<b>Title: </b>Kinetic Modelling of GlmU Reactions - Prioritization of Reaction for Therapeutic Application<br><b>Year: </b>2012<br><b>Author: </b> Vk. Singh,  K. Das and  K. Seshadri<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0043969"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>gene expression, systems biology, metabolic network, network model, network analysis, dynamic model, signaling pathways, signaling network, regulatory network, dynamic network, biology approach, candidate genes, gene network, gene regulatory, biology analysis, mathematical model, network dynamic, biology network, metabolic pathways, pathway analysis<div style="font-size:11px;"><br><b>Abstract: </b><em>Mycobacterium tuberculosisMtu, a successful pathogen, has developed resistance against the existing anti-tubercular drugs necessitating discovery of drugs with novel action. Enzymes involved in peptidoglycan biosynthesis are attractive targets for antibacterial drug discovery. The bifunctional enzyme mycobacterial GlmU Glucosamine 1-phosphate N-acetyltransferase/N-acetylglucosamine-1-phosphate uridyltransferase has been a target enzyme for drug discovery. Its C-and N-terminal domains catalyze acetyltransferase rxn-1 and uridyltransferase rxn-2 activities respectively and the final product is involved in peptidoglycan synthesis. However, the bifunctional nature of GlmU poses difficulty in deciding which function to be intervened for therapeutic advantage. Genetic analysis showed this as an essential gene but it is still unclear whether any one or both of the activities are critical for cell survival. Often enzymatic activity with suitable high-throughput assay is chosen for random screening, which may not be the appropriate biological function inhibited for maximal effect. Prediction of rate-limiting function by dynamic network analysis of reactions could be an option to identify the appropriate function. With a view to provide insights into biochemical assays with appropriate activity for inhibitor screening, kinetic modelling studies on GlmU were undertaken. Kinetic model of Mtu GlmU-catalyzed reactions was built based on the available kinetic data on Mtu and deduction from Escherichia coli data. Several model variants were constructed including coupled/decoupled, varying metabolite concentrations and presence/absence of product inhibitions. This study demonstrates that in coupled model at low metabolite concentrations, inhibition of either of the GlmU reactions cause significant decrement in the overall GlmU rate. However at higher metabolite concentrations, rxn-2 showed higher decrement. Moreover, with available intracellular concentration of the metabolites and in vivo variant of model, uncompetitive inhibition of rxn-2 caused highest decrement. Thus, at physiologically relevant metabolite concentrations, targeting uridyltranferase activity of Mtu GlmU would be a better choice for therapeutic intervention.</div>','<b>Title: </b>Systems analysis of regulatory processes underlying eukaryotic gradient perception<br><b>Year: </b>2008<br><b>Author: </b> J. Krishnan and  Pa. Iglesias<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1109/TAC.2007.911358"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IEEE TRANSACTIONS ON AUTOMATIC CONTROL<br><b>Cluster-Keywords: </b><em>blood flow, blood cells, flow rate, blood pumps, cells, cell, results, patients, stem cells, methods, blood, patients blood, flow, patients methods, patients objectives, cell viability, objectives, results cells, blood pressure, cell culture<div style="font-size:11px;"><br><b>Abstract: </b><em>This paper analyzes regulatory processes involved in the biological process of directed cell locomotion-known as chemotaxis. We focus on the nature of regulation involved in a subprocess of chemotaxis called gradient perception. We examine two different models from a dynamics/control perspective to gain insight into the working mechanisms of these models. One model is a minimal model which reconciles gradient perception to the property of adaptation. The second model is a biochemical model of a lipid network and its regulation by enzymes. In both cases, we focus on the extent of regulation of the modules/networks by inputs, and examine various limiting cases. Both the models and the resulting insights have broader applicability than the context in which they were developed.</div>','<b>Title: </b>A Systems Biology Road Map for the Discovery of Drugs Targeting Cancer Cell Metabolism<br><b>Year: </b>2014<br><b>Author: </b> L. Alberghina,  D. Gaglio,  Rm. Moresco,  Mc. Gilardi,  C. Messa and  M. Vanoni<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.2174/13816128113199990490"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>CURRENT PHARMACEUTICAL DESIGN<br><b>Cluster-Keywords: </b><em>chaotic system, control systems, neural networks, jerk chaotic, torch chaotic, mode control, sliding mode, plasma torch, dynamic neural, mitochondrial dynamics, system, systems biology, adaptive integral, model system, rodtype plasma, integral sliding, systems, jerk system, cancer cell, comparison method<div style="font-size:11px;"><br><b>Abstract: </b><em>Despite their different histological and molecular properties, different types of cancers share few essential functional alterations. Some of these cancer hallmarks may easily be studied in in vitro cultures, while others are related to the way in which tumors grow in vivo.According to the systems biology paradigm, complex cellular functions arise as system-level properties from the dynamic interaction of a large number of biomolecules. We previously newly defined four basic cancer cell properties derived from known cancer hallmarks amenable to system-level investigation in cell cultures: enhanced growth, altered response to apoptotic cues, genomic instability and inability to enter senescence following oncogenic signaling.Here we summarize the major properties of enhanced growth that is dependent on metabolism rewiring - in which glucose is mostly used by fermentation while glutamine provides nitrogen and carbon atoms for biosyntheses - and controlled by oncogene signaling. We then briefly review the major drugs used to target signaling pathways in preclinical and clinical studies, whose clinical efficacy is unfortunately severely limited by tumor resistance, substantially due to signaling cross-talk.We present a systems biology roadmap that integrates different types of mathematical models with conventional and post-genomic bio-molecular analyses that will provide a deeper mechanistic understanding of the links between metabolism and uncontrolled cancer cell growth. This approach is taken to be instrumental both in unraveling cancers first principles and in designing novel drugs able to target one or more control or execution steps of the cancer rewired metabolism, in order to achieve permanent arrest of tumor development.</div>','<b>Title: </b>DFBA-LQR: An optimal control approach to flux balance analysis<br><b>Year: </b>2006<br><b>Author: </b> K. Uygun,  Hwt. Matthew and  Yl. Huang<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1021/ie060218f"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>INDUSTRIAL  ENGINEERING CHEMISTRY RESEARCH<br><b>Cluster-Keywords: </b><em>metabolic network, metabolic model, dynamic flux, growth rate, genomescale metabolic, flux balance, balance analysis, metabolic flux, metabolic pathways, dynamic model, metabolic reactions, cellular growth, regulatory network, systems biology, optimal growth, metabolic, modeling metabolic, growth conditions, dynamic analysis, cell growth<div style="font-size:11px;"><br><b>Abstract: </b><em>Systems biology is becoming an emerging field, as systems science is demonstrated as an essential tool in untangling biocomplexity. While various steady-state-based methods have been developed for study of metabolic networks, the study of the network dynamics is a less explored area, as it is inherently related to the regulatory network. The difficulty of regulatory network inference opens up the possibility of employing control techniques and methods for analysis of metabolic networks. In this work, the potential for application of optimal control theory to metabolic control problems is investigated. Considering that the regulatory network in essence is a cascade control structure, the proposed modeling approach relies on the postulate that evolution is a genetic optimization scheme leading to "optimal" regulatory networks. Thus the approach utilizes a modified linear quadratic regulator formulation to evaluate the optimal regulatory network for a given flux model. Utilizing a homeostasis-type objective, the proposed dynamic flux balance analysis with linear quadratic regulator DFBA-LQR approach enables constructing dynamic flux balance analysis models and produces analytical descriptions of the responses of the metabolic fluxes to disturbances. The analytical description can also be used to calculate the flux control coefficients and linearized reaction kinetics for the system. The application potential of the approach is demonstrated by a sample problem of modeling of lipid accumulation in rat hepatocytes cultures.</div>','<b>Title: </b>A Spiking Neural Network Model of the Lateral Geniculate Nucleus on the SpiNNaker Machine<br><b>Year: </b>2017<br><b>Author: </b> B. Sen-Bhattacharya,  T. Serrano-Gotarredona,  L. Balassa,  A. Bhattacharya,  Ab. Stokes,  A. Rowley,  I. Sugiarto and  S. Furber<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3389/fnins.2017.00454"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>FRONTIERS IN NEUROSCIENCE<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>We present a spiking neural network model of the thalamic Lateral Geniculate Nucleus LGN developed on SpiNNaker, which is a state-of-the-art digital neuromorphic hardware built with very-low-power ARM processors. The parallel, event-based data processing in SpiNNaker makes it viable for building massively parallel neuro-computational frameworks. The LGN model has 140 neurons representing a "basic building block" for larger modular architectures. The motivation of this work is to simulate biologically plausible LGN dynamics on SpiNNaker. Synaptic layout of the model is consistent with biology. The model response is validated with existing literature reporting entrainment in steady state visually evoked potentials SSVEP-brain oscillations corresponding to periodic visual stimuli recorded via electroencephalography EEG. Periodic stimulus to the model is provided by: a synthetic spike-train with inter-spike-intervals in the range 10-50 Hz at a resolution of 1 Hz; and spike-train output from a state-of-the-art electronic retina subjected to a light emitting diode flashing at 10, 20, and 40 Hz, simulating real-world visual stimulus to the model. The resolution of simulation is 0.1 ms to ensure solution accuracy for the underlying differential equations defining Izhikevichs neuron model. Under this constraint, 1 s of model simulation time is executed in 10 s real time on SpiNNaker; this is because simulations on SpiNNaker work in real time for time-steps dt >= 1 ms. The model output shows entrainment with both sets of input and contains harmonic components of the fundamental frequency. However, suppressing the feed-forward inhibition in the circuit produces subharmonics within the gamma band > 30 Hz implying a reduced information transmission fidelity. These model predictions agree with recent lumped-parameter computational model-based predictions, using conventional computers. Scalability of the framework is demonstrated by a multi-node architecture consisting of three "nodes," where each node is the " basic building block" LGN model. This 420 neuron model is tested with synthetic periodic stimulus at 10 Hz to all the nodes. The model output is the average of the outputs from all nodes, and conforms to the above-mentioned predictions of each node. Power consumption for model simulation on SpiNNaker is << 1 W.</div>','<b>Title: </b>Toward a multiscale description of microvascular flow regulation: O-2-dependent release of ATP from human erythrocytes and the distribution of ATP in capillary networks<br><b>Year: </b>2012<br><b>Author: </b> D. Goldman,  Gm. Fraser,  Cg. Ellis,  Rs. Sprague,  Ml. Ellsworth and  Ah. Stephenson<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3389/fphys.2012.00246"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>FRONTIERS IN PHYSIOLOGY<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Integration of the numerous mechanisms that have been suggested to contribute to optimization of O-2 supply to meet O-2 need in skeletal muscle requires a systems biology approach which permits quantification of these physiological processes over a wide range of length scales. Here we describe two individual computational models based on in vivo and in vitro studies which, when incorporated into a single robust multiscale model, will provide information on the role of erythrocyte-released ATP in perfusion distribution in skeletal muscle under both physiological and pathophysiological conditions. Healthy human erythrocytes exposed to low O-2 tension release ATP via a well characterized signaling pathway requiring activation of the G-protein, Gi, and adenylyl cyclase leading to increases in cAMP. This cAMP then activates PKA and subsequently CFTR culminating in ATP release via pannexin 1. A critical control point in this pathway is the level of cAMP which is regulated by pathway-specific phosphodiesterases. Using time constants similar to 100 ms that are consistent with measured erythrocyte ATP release, we have constructed a dynamic model of this pathway. The model predicts levels of ATP release consistent with measurements obtained over a wide range of hemoglobin O-2 saturations sO2. The model further predicts how insulin, at concentrations found in pre-diabetes, enhances the activity of PDE3 and reduces intracellular cAMP levels leading to decreased low O-2-induced ATP release from erythrocytes. The second model, which couples O-2 and ATP transport in capillary networks, shows how intravascular ATP and the resulting conducted vasodilation are affected by local sO2, convection and ATP degradation. This model also predicts network-level effects of decreased ATP release resulting from elevated insulin levels. Taken together, these models lay the groundwork for investigating the systems biology of the regulation of microvascular perfusion distribution by erythrocyte-derived ATP.</div>','<b>Title: </b>Perturbation Detection Through Modeling of Gene Expression on a Latent Biological Pathway Network: A Bayesian Hierarchical Approach<br><b>Year: </b>2016<br><b>Author: </b> Lm. Pham,  L. Carvalho,  S. Schaus and  Ed. Kolaczyk<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1080/01621459.2015.1110523"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF THE AMERICAN STATISTICAL ASSOCIATION<br><b>Cluster-Keywords: </b><em>gene expression, systems biology, metabolic network, network model, network analysis, dynamic model, signaling pathways, signaling network, regulatory network, dynamic network, biology approach, candidate genes, gene network, gene regulatory, biology analysis, mathematical model, network dynamic, biology network, metabolic pathways, pathway analysis<div style="font-size:11px;"><br><b>Abstract: </b><em>Cellular response to a perturbation is the result of a dynamic system of biological variables linked in a complex network. A major challenge in drug and disease studies is identifying the key factors of a biological network that are essential in determining the cells fate. Here, our goal is the identification of perturbed pathways from high-throughput gene expression data. We develop a three-level hierarchical model, where i the first level captures the relationship between gene expression and biological pathways using confirmatory factor analysis, ii the second level models the behavior within an underlying network of pathways induced by an unknown perturbation using a conditional autoregressive model, and iii the third level is a spike-and-slab prior on the perturbations. We then identify perturbations through posterior-based variable selection. We illustrate our approach using gene transcription drug perturbation profiles from the DREAM7 drug sensitivity predication challenge dataset. Our proposed method identified regulatory pathways that are known to play a causative role and that were not readily resolved using gene set enrichment analysis or exploratory factor models. Simulation results are presented assessing the performance of this model relative to a network-free variant and its robustness to inaccuracies in biological databases. Supplementary materials for this article are available online.</div>','<b>Title: </b>The Evolution of Synapse Models - from Numbers to Networks to Spaces<br><b>Year: </b>2010<br><b>Author: </b> B. Dulam-Banawa,  A. Marin-Sanguino and  E. Mendoza<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1055/s-0030-1249025"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PHARMACOPSYCHIATRY<br><b>Cluster-Keywords: </b><em>systems biology, dynamic models, biological systems, model, dynamic systems, computational systems, network model, experimental data, model parameters, systems, systems model, models, pathway systems, data, biochemical pathway, dynamics model, biochemical networks, computational models, network, system<div style="font-size:11px;"><br><b>Abstract: </b><em>We review the evolution of synapse models over the last sixty-five years in terms of the changing paradigms: initially, the synapse was modelled only as part of a neuronal system, both as a number weight of an edge in a connectionist network and as a channel in a conductance-based model. With the availability of more structural and kinetic data, it came to be seen as a full-fledged biochemical network, accompanied by a shift from the previous "top-down" approach to a more "bottom-up" network reconstruction. Most recently, the synapse is seen as a geometric 3-dimensional space with various processes driving the dynamics. A particular focus is placed on models of the dopamine synapse and their connections to schizophrenia. The advances of detailed modelling on the synaptic level have highlighted the challenges of integrating the various functional levels, which are tightly coupled with processes on different scales in time and space. On the other hand, this effort will contribute to bridging the currently perceived gap between computational neuroscience and computational systems biology.</div>','<b>Title: </b>On Macroscopic Quantum Phenomena in Biomolecules and Cells: From Levinthal to Hopfield<br><b>Year: </b>2014<br><b>Author: </b> D. Rakovic,  M. Dugic,  J. Jeknic-Dugic,  M. Plavsic,  S. Jacimovski and  J. Setrajcic<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1155/2014/580491"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOMED RESEARCH INTERNATIONAL<br><b>Cluster-Keywords: </b><em>systems biology, dynamic models, biological systems, model, dynamic systems, computational systems, network model, experimental data, model parameters, systems, systems model, models, pathway systems, data, biochemical pathway, dynamics model, biochemical networks, computational models, network, system<div style="font-size:11px;"><br><b>Abstract: </b><em>In the context of the macroscopic quantum phenomena of the second kind, we hereby seek for a solution-in-principle of the long standing problem of the polymer folding, which was considered by Levinthal as semi classically intractable. To illuminate it, we applied quantum-chemical and quantum decoherence approaches to conformational transitions. Our analyses imply the existence of novel macroscopic quantum biomolecular phenomena, with biomolecular chain folding in an open environment considered as a subtle interplay between energy and conformation eigenstates of this biomolecule, governed by quantum-chemical and quantum decoherence laws. On the other hand, within an open biological cell, a system of all identical noninteracting and dynamically noncoupled biomolecular proteins might be considered as corresponding spatial quantum ensemble of these identical biomolecular processors, providing spatially distributed quantum solution to a single corresponding biomolecular chain folding, whose density of conformational states might be represented as Hopfield-like quantum-holographic associative neural network too providing an equivalent global quantum-informational alternative to standard molecular-biology local biochemical approach in biomolecules and cells and higher hierarchical levels of organism, as well.</div>','<b>Title: </b>Comparing Transcription Rate and mRNA Abundance as Parameters for Biochemical Pathway and Network Analysis<br><b>Year: </b>2010<br><b>Author: </b> B. Hayles,  S. Yellaboina and  Dg. Wang<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0009908"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>systems biology, regulatory network, interaction networks, network model, gene regulatory, network, network biology, networks, gene network, response networks, network dynamics, dynamic model, cell networks, gene expression, biological networks, system dynamics, dynamic network, model, biological systems, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>The cells adapt to extra- and intra-cellular signals by dynamic orchestration of activities of pathways in the biochemical networks. Dynamic control of the gene expression process represents a major mechanism for pathway activity regulation. Gene expression has thus been routinely measured, most frequently at steady-state mRNA abundance level using microarray technology. The results are widely used in statistical inference of the structures of underlying biochemical networks, with the assumption that functionally related genes exhibit similar dynamic profiles. Steady-state mRNA abundance, however, is a composite of two factors: transcription rate and mRNA degradation rate. The question being asked here is therefore whether steady-state mRNA abundance or any of two factors is a more informative measurement target for studying network dynamics. The yeast S. cerevisiae was used as model organism and transcription rate was chosen out of the two factors in this study, because genome-wide determination of transcription rates has been reported for several physiological processes in this species. Our strategy is to test which one is a better measurement of functional relatedness between genes. The analysis was performed on those S. cerevisiae genes that have bacterial orthologs as identified by reciprocal BLAST analysis, so that functional relatedness of a gene pair can be measured by the frequency at which their bacterial orthologs co-occur in the same operon in the collection of bacterial genomes. It is found that transcription rate data is generally a better parameter for functional relatedness than steady state mRNA abundance, suggesting transcription rate data is more informative to use in deciphering the logics used by the cells in dynamic regulation of biochemical network behaviors. The significance of this finding for network and systems biology, as well as biomedical research in general, is discussed.</div>','<b>Title: </b>Iron acquisition and oxidative stress response in aspergillus fumigatus<br><b>Year: </b>2015<br><b>Author: </b> M. Brandon,  B. Howard,  C. Lawrence and  R. Laubenbacher<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/s12918-015-0163-1"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Aspergillus fumigatus is a ubiquitous airborne fungal pathogen that presents a life-threatening health risk to individuals with weakened immune systems. A. fumigatus pathogenicity depends on its ability to acquire iron from the host and to resist host-generated oxidative stress. Gaining a deeper understanding of the molecular mechanisms governing A. fumigatus iron acquisition and oxidative stress response may ultimately help to improve the diagnosis and treatment of invasive aspergillus infections.Results: This study follows a systems biology approach to investigate how adaptive behaviors emerge from molecular interactions underlying A. fumigatus iron regulation and oxidative stress response. We construct a Boolean network model from known interactions and simulate how changes in environmental iron and superoxide levels affect network dynamics. We propose rules for linking long term model behavior to qualitative estimates of cell growth and cell death. These rules are used to predict phenotypes of gene deletion strains. The model is validated on the basis of its ability to reproduce literature data not used in model generation.Conclusions: The model reproduces gene expression patterns in experimental time course data when A. fumigatus is switched from a low iron to a high iron environment. In addition, the model is able to accurately represent the phenotypes of many knockout strains under varying iron and superoxide conditions. Model simulations support the hypothesis that intracellular iron regulates A. fumigatus transcription factors, SreA and HapX, by a post-translational, rather than transcriptional, mechanism. Finally, the model predicts that blocking siderophore-mediated iron uptake reduces resistance to oxidative stress. This indicates that combined targeting of siderophore-mediated iron uptake and the oxidative stress response network may act synergistically to increase fungal cell killing.</div>','<b>Title: </b>Statecharts for Gene Network Modeling<br><b>Year: </b>2010<br><b>Author: </b> Yj. Shin and  M. Nourani<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0009376"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>network motifs, regulatory network, gene regulatory, systems biology, gene network, bifan motif, complex systems, model comparison, network modeling, complex gene, biological systems, dynamic systems, interaction network, feedforward loop, modeling gene, intelligent systems, modeling approach, biological pathways, motif model, model motifs<div style="font-size:11px;"><br><b>Abstract: </b><em>State diagrams stategraphs are suitable for describing the behavior of dynamic systems. However, when they are used to model large and complex systems, determining the states and transitions among them can be overwhelming, due to their flat, unstratified structure. In this article, we present the use of statecharts as a novel way of modeling complex gene networks. Statecharts extend conventional state diagrams with features such as nested hierarchy, recursion, and concurrency. These features are commonly utilized in engineering for designing complex systems and can enable us to model complex gene networks in an efficient and systematic way. We modeled five key gene network motifs, simple regulation, autoregulation, feed-forward loop, single-input module, and dense overlapping regulon, using statecharts. Specifically, utilizing nested hierarchy and recursion, we were able to model a complex interlocked feed-forward loop network in a highly structured way, demonstrating the potential of our approach for modeling large and complex gene networks.</div>','<b>Title: </b>Combined Model of Intrinsic and Extrinsic Variability for Computational Network Design with Application to Synthetic Biology<br><b>Year: </b>2013<br><b>Author: </b> T. Toni and  B. Tidor<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pcbi.1002960"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS COMPUTATIONAL BIOLOGY<br><b>Cluster-Keywords: </b><em>reaction networks, network model, systems biology, biochemical reaction, stochastic reaction, model reduction, stochastic model, network dynamics, stochastic biochemical, stochastic simulation, system size, chemical reaction, gene network, elementary reactions, biochemical networks, parameter identification, parameter inference, cell populations, small system, linear noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Biological systems are inherently variable, with their dynamics influenced by intrinsic and extrinsic sources. These systems are often only partially characterized, with large uncertainties about specific sources of extrinsic variability and biochemical properties. Moreover, it is not yet well understood how different sources of variability combine and affect biological systems in concert. To successfully design biomedical therapies or synthetic circuits with robust performance, it is crucial to account for uncertainty and effects of variability. Here we introduce an efficient modeling and simulation framework to study systems that are simultaneously subject to multiple sources of variability, and apply it to make design decisions on small genetic networks that play a role of basic design elements of synthetic circuits. Specifically, the framework was used to explore the effect of transcriptional and post-transcriptional autoregulation on fluctuations in protein expression in simple genetic networks. We found that autoregulation could either suppress or increase the output variability, depending on specific noise sources and network parameters. We showed that transcriptional autoregulation was more successful than post-transcriptional in suppressing variability across a wide range of intrinsic and extrinsic magnitudes and sources. We derived the following design principles to guide the design of circuits that best suppress variability: i high protein cooperativity and low miRNA cooperativity, ii imperfect complementarity between miRNA and mRNA was preferred to perfect complementarity, and iii correlated expression of mRNA and miRNA - for example, on the same transcript - was best for suppression of protein variability. Results further showed that correlations in kinetic parameters between cells affected the ability to suppress variability, and that variability in transient states did not necessarily follow the same principles as variability in the steady state. Our model and findings provide a general framework to guide design principles in synthetic biology.</div>','<b>Title: </b>Toward First Principle Medical Diagnostics: On the Importance of Disease-Disease and Sign-Sign Interactions<br><b>Year: </b>2017<br><b>Author: </b> A. Ramezanpour and  A. Mashaghi<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3389/fphy.2017.00032"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>FRONTIERS IN PHYSICS<br><b>Cluster-Keywords: </b><em>boolean networks, regulatory networks, network model, bayesian networks, model networks, systems biology, random boolean, real cells, immune system, genetic regulatory, networks, biochemical network, fluorescence images, cellular network, gene regulatory, cell differentiation, network, biological systems, ensemble approach, cell dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>A fundamental problem in medicine and biology is to assign states, e.g., healthy or diseased, to cells, organs or individuals. State assignment or making a diagnosis is often a nontrivial and challenging process and, with the advent of omics technologies, the diagnostic challenge is becoming more and more serious. The challenge lies not only in the increasing number of measured properties and dynamics of the system e.g., cell or human body but also in the co-evolution of multiple states and overlapping properties, and degeneracy of states. We develop, from first principles, a generic rational framework for state assignment in cell biology and medicine, and demonstrate its applicability with a few simple theoretical case studies from medical diagnostics. We show how disease-related statistical information can be used to build a comprehensive model that includes the relevant dependencies between clinical and laboratory findings signs and diseases. In particular, we include disease-disease and sign-sign interactions and study how one can infer the probability of a disease in a patient with given signs. We perform comparative analysis with simple benchmark models to check the performances of our models. We find that including interactions can significantly change the statistical importance of the signs and diseases. This first principles approach, as we show, facilitates the early diagnosis of disease by taking interactions into accounts, and enables the construction of consensus diagnostic flow charts. Additionally, we envision that our approach will find applications in systems biology, and in particular, in characterizing the phenome via the metabolome, the proteome, the transcriptome, and the genome.</div>','<b>Title: </b>On the estimation of robustness and filtering ability of dynamic biochemical networks under process delays, internal parametric perturbations and external disturbances<br><b>Year: </b>2009<br><b>Author: </b> Bs. Chen and  Pw. Chen<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.mbs.2009.09.004"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MATHEMATICAL BIOSCIENCES<br><b>Cluster-Keywords: </b><em>gene networks, synthetic gene, synthetic biology, regulatory networks, design method, robust synthetic, nonlinear stochastic, molecular noises, gene regulatory, interaction networks, external disturbances, robust stability, robust gene, host cell, gene circuit, cellular network, design specifications, design procedure, signal transduction, parameter fluctuations<div style="font-size:11px;"><br><b>Abstract: </b><em>Inherently, biochemical regulatory networks suffer from process delays, internal parametrical perturbations as well as external disturbances. Robustness is the property to maintain the functions of intracellular biochemical regulatory networks despite these perturbations. In this study, system and signal processing theories are employed for measurement of robust stability and filtering ability of linear and nonlinear time-delay biochemical regulatory networks. First, based on Lyapunov stability theory. the robust stability of biochemical network is measured for the tolerance of additional process delays and additive internal parameter fluctuations. Then the filtering ability of attenuating additive external disturbances is estimated for time-delay biochemical regulatory networks. In order to overcome the difficulty of solving the Hamilton Jacobi inequality HJI, the global linearization technique is employed to simplify the measurement procedure by a simple linear matrix inequality LMI method. Finally, an example is given in silico to illustrate how to measure the robust stability and filtering ability of a nonlinear time-delay perturbative biochemical network. This robust stability and filtering ability measurement for biochemical network has potential application to synthetic biology. gene therapy and drug design. C 2009 Elsevier Inc. All rights reserved.</div>','<b>Title: </b>Emergence in stigmergic and complex adaptive systems: A formal discrete event systems perspective<br><b>Year: </b>2013<br><b>Author: </b> S. Mittal<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.cogsys.2012.06.003"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>COGNITIVE SYSTEMS RESEARCH<br><b>Cluster-Keywords: </b><em>complex networks, complex systems, network theory, network structure, biological network, social network, network science, dynamic networks, network systems, network model, network topology, systems biology, network, network analysis, system dynamics, networks, realworld networks, network automata, epidemic network, model network<div style="font-size:11px;"><br><b>Abstract: </b><em>Complex systems have been studied by researchers from every discipline: biology, chemistry, physics, sociology, mathematics and economics and more. Depending upon the discipline, complex systems theory has accrued many flavors. We are after a formal representation, a model that can predict the outcome of a complex adaptive system CAS. In this article, we look at the nature of complexity, then provide a perspective based on discrete event systems DEVS theory. We pin down many of the shared features between CAS and artificial systems. We begin with an overview of network science showing how adaptive behavior in these scale-free networks can lead to emergence through stigmergy in CAS. We also address how both self-organization and emergence interplay in a CAS. We then build a case for the view that stigmergic systems are a special case of CAS. We then discuss DEVS levels of systems specifications and present the dynamic structure extensions of DEVS formalism that lends itself to a study of CAS and in turn, stigmergy. Finally, we address the shortcomings and the limitation of current DEVS extensions and propose the required augmentation to model stigmergy and CAS. C 2012 Elsevier B.V. All rights reserved.</div>','<b>Title: </b>An algebra-based method for inferring gene regulatory networks<br><b>Year: </b>2014<br><b>Author: </b> P. Vera-Licona,  A. Jarrah,  Ld. Garcia-Puente,  J. Mcgee and  R. Laubenbacher<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-8-37"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>regulatory networks, gene regulatory, gene expression, network inference, expression data, network structure, gene networks, networks grns, regulatory genes, boolean networks, bayesian network, network reconstruction, networks gene, network dynamics, gene regulation, inference method, systems biology, time series, infer gene, network<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: The inference of gene regulatory networks GRNs from experimental observations is at the heart of systems biology. This includes the inference of both the network topology and its dynamics. While there are many algorithms available to infer the network topology from experimental data, less emphasis has been placed on methods that infer network dynamics. Furthermore, since the network inference problem is typically underdetermined, it is essential to have the option of incorporating into the inference process, prior knowledge about the network, along with an effective description of the search space of dynamic models. Finally, it is also important to have an understanding of how a given inference method is affected by experimental and other noise in the data used.Results: This paper contains a novel inference algorithm using the algebraic framework of Boolean polynomial dynamical systems BPDS, meeting all these requirements. The algorithm takes as input time series data, including those from network perturbations, such as knock-out mutant strains and RNAi experiments. It allows for the incorporation of prior biological knowledge while being robust to significant levels of noise in the data used for inference. It uses an evolutionary algorithm for local optimization with an encoding of the mathematical models as BPDS. The BPDS framework allows an effective representation of the search space for algebraic dynamic models that improves computational performance. The algorithm is validated with both simulated and experimental microarray expression profile data. Robustness to noise is tested using a published mathematical model of the segment polarity gene network in Drosophila melanogaster. Benchmarking of the algorithm is done by comparison with a spectrum of state-of-the-art network inference methods on data from the synthetic IRMA network to demonstrate that our method has good precision and recall for the network reconstruction task, while also predicting several of the dynamic patterns present in the network.Conclusions: Boolean polynomial dynamical systems provide a powerful modeling framework for the reverse engineering of gene regulatory networks, that enables a rich mathematical structure on the model search space. A C implementation of the method, distributed under LPGL license, is available, together with the source code, at http://www.paola-vera-licona.net/Software/EARevEng/REACT.html.</div>','<b>Title: </b>Emergent decision-making in biological signal transduction networks<br><b>Year: </b>2008<br><b>Author: </b> T. Helikar,  J. Konvalina,  J. Heidel and  Ja. Rogers<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1073/pnas.0705088105"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>The complexity of biochemical intracellular signal transduction networks has led to speculation that the high degree of interconnectivity that exists in these networks transforms them into an information processing network. To test this hypothesis directly, a large scale model was created with the logical mechanism of each node described completely to allow simulation and dynamical analysis. Exposing the network to tens of thousands of random combinations of inputs and analyzing the combined dynamics of multiple outputs revealed a robust system capable of clustering widely varying input combinations into equivalence classes of biologically relevant cellular responses. This capability was nontrivial in that the network performed sharp, nonfuzzy classifications even in the face of added noise, a hallmark of real-world decision-making.</div>','<b>Title: </b>Dynamic optimization of biological networks under parametric uncertainty<br><b>Year: </b>2016<br><b>Author: </b> P. Nimmegeers,  D. Telen,  F. Logist and  J. Van Impe<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/s12918-016-0328-6"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Micro-organisms play an important role in various industrial sectors including biochemical, food and pharmaceutical industries. A profound insight in the biochemical reactions inside micro-organisms enables an improved biochemical process control. Biological networks are an important tool in systems biology for incorporating microscopic level knowledge. Biochemical processes are typically dynamic and the cells have often more than one objective which are typically conflicting, e.g., minimizing the energy consumption while maximizing the production of a specific metabolite. Therefore multi-objective optimization is needed to compute trade-offs between those conflicting objectives. In model-based optimization, one of the inherent problems is the presence of uncertainty. In biological processes, this uncertainty can be present due to, e.g., inherent biological variability. Not taking this uncertainty into account, possibly leads to the violation of constraints and erroneous estimates of the actual objective functions. To account for the variance in model predictions and compute a prediction interval, this uncertainty should be taken into account during process optimization. This leads to a challenging optimization problem under uncertainty, which requires a robustified solution.Results: Three techniques for uncertainty propagation: linearization, sigma points and polynomial chaos expansion, are compared for the dynamic optimization of biological networks under parametric uncertainty. These approaches are compared in two case studies: i a three-step linear pathway model in which the accumulation of intermediate metabolites has to be minimized and ii a glycolysis inspired network model in which a multi-objective optimization problem is considered, being the minimization of the enzymatic cost and the minimization of the end time before reaching a minimum extracellular metabolite concentration. A Monte Carlo simulation procedure has been applied for the assessment of the constraint violations. For the multi-objective case study one Pareto point has been considered for the assessment of the constraint violations. However, this analysis can be performed for any Pareto point.Conclusions: The different uncertainty propagation strategies each offer a robustified solution under parametric uncertainty. When making the trade-off between computation time and the robustness of the obtained profiles, the sigma points and polynomial chaos expansion strategies score better in reducing the percentage of constraint violations. This has been investigated for a normal and a uniform parametric uncertainty distribution. The polynomial chaos expansion approach allows to directly take prior knowledge of the parametric uncertainty distribution into account.</div>','<b>Title: </b>Hit-and-run transcriptional control by bZIP1 mediates rapid nutrient signaling in Arabidopsis<br><b>Year: </b>2014<br><b>Author: </b> A. Para,  Y. Li,  A. Marshall-Colon,  K. Varala,  Nj. Francoeur,  Tm. Moran,  Mb. Edwards,  C. Hackley,  Bor. Bargmann,  Kd. Birnbaum,  Wr. Mccombie,  G. Krouk and  Gm. Coruzzi<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1073/pnas.1404657111"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA<br><b>Cluster-Keywords: </b><em>gene expression, systems biology, metabolic network, network model, network analysis, dynamic model, signaling pathways, signaling network, regulatory network, dynamic network, biology approach, candidate genes, gene network, gene regulatory, biology analysis, mathematical model, network dynamic, biology network, metabolic pathways, pathway analysis<div style="font-size:11px;"><br><b>Abstract: </b><em>The dynamic nature of gene regulatory networks allows cells to rapidly respond to environmental change. However, the underlying temporal connections are missed, even in kinetic studies, as transcription factor TF binding within at least one time point is required to identify primary targets. The TF-regulated but un-bound genes are dismissed as secondary targets. Instead, we report that these genes comprise transient TF-target interactions most relevant to rapid signal transduction. We temporally perturbed a master TF Basic Leucine Zipper 1, bZIP1 and the nitrogen N signal it transduces and integrated TF regulation and binding data from the same cell samples. Our enabling approach could identify primary TF targets based solely on gene regulation, in the absence of TF binding. We uncovered three classes of primary TF targets: i poised TF-bound but not TF-regulated, ii stable TF-bound and TF-regulated, and iii transient TF-regulated but not TF-bound, the largest class. Unexpectedly, the transient bZIP1 targets are uniquely relevant to rapid N signaling in planta, enriched in dynamic N-responsive genes, and regulated by TF and N signal interactions. These transient targets include early N responders nitrate transporter 2.1 and NIN-like protein 3, bound by bZIP1 at 1-5 min, but not at later time points following TF perturbation. Moreover, promoters of these transient targets are uniquely enriched with cis-regulatory motifs coinherited with bZIP1 binding sites, suggesting a recruitment role for bZIP1. This transient mode of TF action supports a classic, but forgotten, "hit-and-run" transcription model, which enables a "catalyst TF" to activate a large set of targets within minutes of signal perturbation.</div>','<b>Title: </b>Bayesian Learning from Marginal Data in Bionetwork Models<br><b>Year: </b>2011<br><b>Author: </b> Fv. Bonassi,  Lc. You and  M. West<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.2202/1544-6115.1684"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>STATISTICAL APPLICATIONS IN GENETICS AND MOLECULAR BIOLOGY<br><b>Cluster-Keywords: </b><em>circadian clock, biological systems, network models, mathematical models, clock gene, systems biology, model selection, biological networks, network data, experimental data, model parameters, network dynamics, plant circadian, regulatory network, clock model, plant clock, complex network, clock circuit, approximate bayesian, stochastic models<div style="font-size:11px;"><br><b>Abstract: </b><em>In studies of dynamic molecular networks in systems biology, experiments are increasingly exploiting technologies such as flow cytometry to generate data on marginal distributions of a few network nodes at snapshots in time. For example, levels of intracellular expression of a few genes, or cell surface protein markers, can be assayed at a series of interim time points and assumed steady-states under experimentally stimulated growth conditions in small cellular systems. Such marginal data on a small number of cellular markers will typically carry very limited information on the parameters and structure of dynamic network models, though experiments will typically be designed to expose variation in cellular phenotypes that are inherently related to some aspects of model parametrization and structure. Our work addresses statistical questions of how to integrate such data with dynamic stochastic models in order to properly quantify the information-or lack of information-it carries relative to models assumed. We present a Bayesian computational strategy coupled with a novel approach to summarizing and numerically characterizing biological phenotypes that are represented in terms of the resulting sample distributions of cellular markers. We build on Bayesian simulation methods and mixture modeling to define the approach to linking mechanistic mathematical models of network dynamics to snapshot data, using a toggle switch example integrating simulated and real data as context.</div>','<b>Title: </b>Molecular Network Dynamics of Cell Cycle Control: Transitions to Start and Finish<br><b>Year: </b>2011<br><b>Author: </b> A. Csikasz-Nagy,  A. Palmisano and  J. Zamborszky<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/978-1-61779-182-619"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>CELL CYCLE SYNCHRONIZATION: METHODS AND PROTOCOLS<br><b>Cluster-Keywords: </b><em>cell cycle, mammalian cell, cdk network, cell proliferation, cell, cells cell, regulatory network, detailed model, cell size, molecular network, skeleton model, cycle arrest, cycle progression, cell division, cell density, kinases cdks, cycle, phase transition, cycle control, network driving<div style="font-size:11px;"><br><b>Abstract: </b><em>The cell cycle is controlled by complex regulatory network to ensure that the phases of the cell cycle happen in the right order and transitions between phases happen only if the earlier phase is properly finished. This regulatory network receives signals from the environment, monitors the state of the DNA, and decides when the cell can proceed in its cycle. The transcriptional and post-translational regulatory interactions in this network can lead to complex dynamical responses. The cell cycle dependent oscillations in protein activities are driven by these interactions as the regulatory system moves between steady states that correspond to different phases of the cell cycle. The analysis of such complex molecular network behavior can be investigated with the tools of computational systems biology. Here we review the basic physiological and molecular transitions in the cell cycle and present how the system-level emergent properties were found by the help of mathematical/computational modeling.</div>','<b>Title: </b>Modeling and analysis of retinoic acid induced differentiation of uncommitted precursor cells<br><b>Year: </b>2011<br><b>Author: </b> R. Tasseff,  S. Nayak,  So. Song,  A. Yen and  Jd. Varner<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1039/c0ib00141d"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>INTEGRATIVE BIOLOGY<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Manipulation of differentiation programs has therapeutic potential in a spectrum of human cancers and neurodegenerative disorders. In this study, we integrated computational and experimental methods to unravel the response of a lineage uncommitted precursor cell-line, HL-60, to Retinoic Acid RA. HL-60 is a human myeloblastic leukemia cell-line used extensively to study human differentiation programs. Initially, we focused on the role of the BLR1 receptor in RA-induced differentiation and G1/0-arrest in HL-60. BLR1, a putative G protein-coupled receptor expressed following RA exposure, is required for RA-induced cell-cycle arrest and differentiation and causes persistent MAPK signaling. A mathematical model of RA-induced cell-cycle arrest and differentiation was formulated and tested against BLR1 wild-type wt knock-out and knock-in HL-60 cell-lines with and without RA. The current model described the dynamics of 729 proteins and protein complexes interconnected by 1356 interactions. An ensemble strategy was used to compensate for uncertain model parameters. The ensemble of HL-60 models recapitulated the positive feedback between BLR1 and MAPK signaling. The ensemble of models also correctly predicted Rb and p47phox regulation and the correlation between p21-CDK4-cyclin D formation and G1/0-arrest following exposure to RA. Finally, we investigated the robustness of the HL-60 network architecture to structural perturbations and generated experimentally testable hypotheses for future study. Taken together, the model presented here was a first step toward a systematic framework for analysis of programmed differentiation. These studies also demonstrated that mechanistic network modeling can help prioritize experimental directions by generating falsifiable hypotheses despite uncertainty.</div>','<b>Title: </b>Systems biology from micro-organisms to human metabolic diseases: the role of detailed kinetic models<br><b>Year: </b>2010<br><b>Author: </b> Bm. Bakker,  K. Van Eunen,  Jal. Jeneson,  Naw. Van Riel,  Fj. Bruggeman and  B. Teusink<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1042/BST0381294"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOCHEMICAL SOCIETY TRANSACTIONS<br><b>Cluster-Keywords: </b><em>kinetic models, metabolic networks, kinetic parameters, dynamic models, systems biology, network model, dynamic metabolic, biochemical network, rate constants, enzyme kinetics, kinetic data, adjustable parameters, metabolic model, parameter estimation, dynamic optimization, model selection, mathematical model, kinetic network, model response, model<div style="font-size:11px;"><br><b>Abstract: </b><em>Human metabolic diseases are typically network diseases. This holds not only for multifactorial diseases, such as metabolic syndrome or Type 2 diabetes, but even when a single gene defect is the primary cause, where the adaptive response of the entire network determines the severity of disease. The latter may differ between individuals carrying the same mutation. Understanding the adaptive responses of human metabolism naturally requires a systems biology approach. Modelling of metabolic pathways in microorganisms and some mammalian tissues has yielded many insights, qualitative as well as quantitative, into their control and regulation. Yet, even for a well-known pathway such as glycolysis, precise predictions of metabolite dynamics from experimentally determined enzyme kinetics have been only moderately successful. In the present review, we compare kinetic models of glycolysis in three cell types African trypanosomes, yeast and skeletal muscle, evaluate their predictive power and identify limitations in our understanding. Although each of these models has its own merits and shortcomings, they also share common features. For example, in each case independently measured enzyme kinetic parameters were used as input. Based on these lessons from glycolysis, we will discuss how to make best use of kinetic computer models to advance our understanding of human metabolic diseases.</div>','<b>Title: </b>Estimation and Discrimination of Stochastic Biochemical Circuits from Time-Lapse Microscopy Data<br><b>Year: </b>2012<br><b>Author: </b> D. Thorsley and  E. Klavins<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0047151"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>reaction networks, network model, systems biology, biochemical reaction, stochastic reaction, model reduction, stochastic model, network dynamics, stochastic biochemical, stochastic simulation, system size, chemical reaction, gene network, elementary reactions, biochemical networks, parameter identification, parameter inference, cell populations, small system, linear noise<div style="font-size:11px;"><br><b>Abstract: </b><em>The ability of systems and synthetic biologists to observe the dynamics of cellular behavior is hampered by the limitations of the sensors, such as fluorescent proteins, available for use in time-lapse microscopy. In this paper, we propose a generalized solution to the problem of estimating the state of a stochastic chemical reaction network from limited sensor information generated by microscopy. We mathematically derive an observer structure for cells growing under time-lapse microscopy and incorporates the effects of cell division in order to estimate the dynamically-changing state of each cell in the colony. Furthermore, the observer can be used to discrimate between models by treating model indices as states whose values do not change with time. We derive necessary and sufficient conditions that specify when stochastic chemical reaction network models, interpreted as continuous-time Markov chains, can be distinguished from each other under both continual and periodic observation. We validate the performance of the observer on the Thattai-van Oudenaarden model of transcription and translation. The observer structure is most effective when the system model is well-parameterized, suggesting potential applications in synthetic biology where standardized biological parts are available. However, further research is necessary to develop computationally tractable approximations to the exact generalized solution presented here.</div>','<b>Title: </b>Mosaic gene network modelling identified new regulatory mechanisms in HCV infection<br><b>Year: </b>2016<br><b>Author: </b> Ov. Popik,  Ed. Petrovskiy,  El. Mishchenko,  In. Lavrik and  Va. Ivanisenko<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.virusres.2015.10.004"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>VIRUS RESEARCH<br><b>Cluster-Keywords: </b><em>gene expression, systems biology, metabolic network, network model, network analysis, dynamic model, signaling pathways, signaling network, regulatory network, dynamic network, biology approach, candidate genes, gene network, gene regulatory, biology analysis, mathematical model, network dynamic, biology network, metabolic pathways, pathway analysis<div style="font-size:11px;"><br><b>Abstract: </b><em>Modelling of gene networks is widely used in systems biology to study the functioning of complex biological systems. Most of the existing mathematical modelling techniques are useful for analysis of well-studied biological processes, for which information on rates of reactions is available. However, complex biological processes such as those determining the phenotypic traits of organisms or pathological disease processes, including pathogen-host interactions, involve complicated cross-talk between interacting networks. Furthermore, the intrinsic details of the interactions between these networks are often missing. In this study, we developed an approach, which we call mosaic network modelling, that allows the combination of independent mathematical models of gene regulatory networks and, thereby, description of complex biological systems. The advantage of this approach is that it allows us to generate the integrated model despite the fact that information on molecular interactions between parts of the model so-called mosaic fragments might be missing. To generate a mosaic mathematical model, we used control theory and mathematical models, written in the form of a system of ordinary differential equations ODES. In the present study, we investigated the efficiency of this method in modelling the dynamics of more than 10,000 simulated mosaic regulatory networks consisting of two pieces. Analysis revealed that this approach was highly efficient, as the mean deviation of the dynamics of mosaic network elements from the behaviour of the initial parts of the model was less than 10. It turned out that for construction of the control functional, data on perturbation of one or two vertices of the mosaic piece are sufficient. Further, we used the developed method to construct a mosaic gene regulatory network including hepatitis C virus HCV as the first piece and the tumour necrosis factor TNF-induced apoptosis and NF-kappa B induction pathways as the second piece. Thus, the mosaic model integrates the model of HCV subgenomic replicon replication with the model of TNF-induced apoptosis and NF-kappa B induction. Analysis of the mosaic model revealed that the regulation of TNF-induced signaling by the HCV network is crucially dependent on the RIP1, TRADD, TRAF2, FADD, IKK, I kappa B alpha, c-FLIP, and BAR genes. Overall, the developed mosaic gene network modelling approach demonstrated good predictive power and allowed the prediction of new regulatory nodes in HCV action on apoptosis and the NF-kappa B pathway. Those theoretical predictions could be a basis for further experimental verification. C 2015 The Authors. Published by Elsevier B.V.</div>','<b>Title: </b>Interpreting correlated motions using normal mode analysis<br><b>Year: </b>2006<br><b>Author: </b> Aw. Van Wynsberghe and  Q. Cui<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.str.2006.09.003"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>STRUCTURE<br><b>Cluster-Keywords: </b><em>normal mode, mode analysis, elastic network, functional motions, functional dynamics, molecular chaperone, structural biology, acidic residues, network models, molecular dynamics, atomic resolution, protein complexes, conserved functional, hsp dynamics, correlated motions, protein folding, large biomolecular, thioredoxin system, harmonic analysis, protein motions<div style="font-size:11px;"><br><b>Abstract: </b><em>With the increased popularity of normal mode analyses in structural biology, it is important to carefully consider how to best utilize the results for gaining biological insights without over interpretation. The discussion in this article argues that for the purpose of identifying correlated motions in biomolecules, a case separate from concomitant conformational changes of structural motifs, it is generally important to use a large number of normal modes. This is illustrated through three increasingly complex examples. The simplest case includes two bilinearly coupled harmonic oscillators and serves as a straightforward problem where the important considerations are explicit and transparent. The argument is then generalized to include a system of N-coupled harmonic oscillators and finally to a realistic biomolecule. Although a small number of normal modes are useful for probing structural flexibility, it is clear that a much larger number of modes are required for properly investigating correlated motions in biomolecules.</div>','<b>Title: </b>Sharpening of expression domains induced by transcription and microRNA regulation within a spatio-temporal model of mid-hindbrain boundary formation<br><b>Year: </b>2013<br><b>Author: </b> S. Hock,  Yk. Ng,  J. Hasenauer,  D. Wittmann,  D. Lutter,  D. Trumbach,  W. Wurst,  N. Prakash and  Fj. Theis<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-7-48"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>regulatory network, systems biology, systems biological, network dynamics, gene expression, biochemical networks, raft proteins, biological networks, gene circuits, gene regulatory, signalling network, lipid raft, nuclear lamina, protein domain, networks systems, biological approaches, expression pattern, protein interaction, cell differentiation, network motifs<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: The establishment of the mid-hindbrain region in vertebrates is mediated by the isthmic organizer, an embryonic secondary organizer characterized by a well-defined pattern of locally restricted gene expression domains with sharply delimited boundaries. While the function of the isthmic organizer at the mid-hindbrain boundary has been subject to extensive experimental studies, it remains unclear how this well-defined spatial gene expression pattern, which is essential for proper isthmic organizer function, is established during vertebrate development. Because the secreted Wnt1 protein plays a prominent role in isthmic organizer function, we focused in particular on the refinement of Wnt1 gene expression in this context.Results: We analyzed the dynamics of the corresponding murine gene regulatory network and the related, diffusive signaling proteins using a macroscopic model for the biological two-scale signaling process. Despite the discontinuity arising from the sharp gene expression domain boundaries, we proved the existence of unique, positive solutions for the partial differential equation system. This enabled the numerically and analytically analysis of the formation and stability of the expression pattern. Notably, the calculated expression domain of Wnt1 has no sharp boundary in contrast to experimental evidence. We subsequently propose a post-transcriptional regulatory mechanism for Wnt1 miRNAs which yields the observed sharp expression domain boundaries. We established a list of candidate miRNAs and confirmed their expression pattern by radioactive in situ hybridization. The miRNA miR-709 was identified as a potential regulator of Wnt1 mRNA, which was validated by luciferase sensor assays.Conclusion: In summary, our theoretical analysis of the gene expression pattern induction at the mid-hindbrain boundary revealed the need to extend the model by an additional Wnt1 regulation. The developed macroscopic model of a two-scale process facilitate the stringent analysis of other morphogen-based patterning processes.</div>','<b>Title: </b>A transcriptomics-based kinetic model for ethylene biosynthesis in tomato Solanum lycopersicum fruit: development, validation and exploration of novel regulatory mechanisms<br><b>Year: </b>2014<br><b>Author: </b> B. Van De Poel,  I. Bulens,  Mlatm. Hertog,  Bm. Nicolai and  Ah. Geeraerd<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1111/nph.12685"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>NEW PHYTOLOGIST<br><b>Cluster-Keywords: </b><em>kinetic models, metabolic networks, kinetic parameters, dynamic models, systems biology, network model, dynamic metabolic, biochemical network, rate constants, enzyme kinetics, kinetic data, adjustable parameters, metabolic model, parameter estimation, dynamic optimization, model selection, mathematical model, kinetic network, model response, model<div style="font-size:11px;"><br><b>Abstract: </b><em>The gaseous plant hormone ethylene is involved in many physiological processes including climacteric fruit ripening, in which it is a key determinant of fruit quality. A detailed model that describes ethylene biochemistry dynamics is missing. Often, kinetic modeling is used to describe metabolic networks or signaling cascades, mostly ignoring the link with transcriptomic data. We have constructed an elegant kinetic model that describes the transfer of genetic information into abundance and metabolic activity of proteins for the entire ethylene biosynthesis pathway during fruit development and ripening of tomato Solanum lycopersicum. Our model was calibrated against a vast amount of transcriptomic, proteomic and metabolic data and showed good descriptive qualities. Subsequently it was validated successfully against several ripening mutants previously described in the literature. The model was used as a predictive tool to evaluate novel and existing hypotheses regarding the regulation of ethylene biosynthesis. This bottom-up kinetic network model was used to indicate that a side-branch of the ethylene pathway, the formation of the dead-end product 1-malonylamino-1-aminocyclopropane-1-carboxylic acid MACC, might have a strong effect on eventual ethylene production. Furthermore, our in silico analyses indicated potential post- translational regulation of the ethylene-forming enzyme ACC oxidase.</div>','<b>Title: </b>Facile: a command-line network compiler for systems biology<br><b>Year: </b>2007<br><b>Author: </b> F. Siso-Nadal,  Jf. Ollivier and  Ps. Swain<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-1-36"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: A goal of systems biology is the quantitative modelling of biochemical networks. Yet for many biochemical systems, parameter values and even the existence of interactions between some chemical species are unknown. It is therefore important to be able to easily investigate the effects of adding or removing reactions and to easily perform a bifurcation analysis, which shows the qualitative dynamics of a model for a range of parameter values.Results: We present Facile, a Perl command-line tool for analysing the dynamics of a systems biology model. Facile implements the law of mass action to automatically compile a biochemical network written as, for example, E  S <-> C into scripts for analytical analysis Mathematica and Maple, for simulation XPP and Matlab, and for bifurcation analysis AUTO. Facile automatically identifies mass conservations and generates the reduced form of a model with the minimum number of independent variables. This form is essential for bifurcation analysis, and Facile produces a C version of the reduced model for AUTO.Conclusion: Facile is a simple, yet powerful, tool that greatly accelerates analysis of the dynamics of a biochemical network. By acting at the command-line and because of its intuitive, text-based input, Facile is quick to learn and can be incorporated into larger programs or into automated tasks.</div>','<b>Title: </b>Towards understanding pressure ulcer formation: Coupling an inflammation regulatory network to a tissue scale finite element model<br><b>Year: </b>2019<br><b>Author: </b> Vd. Sree,  Mk. Rausch and  Ab. Tepole<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.mechrescom.2019.05.003"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MECHANICS RESEARCH COMMUNICATIONS<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Pressure ulcers PUs are a major clinical concern affecting more than 2.5 million adults annually in the US. lschemia is recognized as one of the most important contributors to ulcer formation, triggering a sterile inflammatory cascade that culminates in necrosis of native skin cells and ulceration. While there is a general understanding of the biological elements involved in this process and their interdependence within the biological PU signaling network, this systems spatio-temporal dynamics have not yet been studied in detail. Here we first present a 0D mathematical description of the PU regulatory network consisting of two cell types - keratinocytes and neutrophils- and six chemical species - TNF alpha, KC, ROS, DAMPs, O-2 and XO. We extend this regulatory network from a set of ordinary differential equations to realistic spatial domains by coupling each species dynamic equations to reaction diffusion partial differential equations. Furthermore, we couple this model to mechanical deformation of the spatial domain by including a pressure-sensitive oxygen perfusion term. The total model provides solutions to the regulatory network dynamics at the tissue scale with spatio-temporal detail on the evolution of each species. Among other features, the model correctly predicts patterns of PU formation in response to moderate loads, as seen clinically and experimentally. In conclusion, this work extends our current PU modeling capabilities and improves our understanding of PUs by carefully analyzing the motifs of the regulatory network and by exploring the implications of coupling this system to a tissue scale model of skin mechanics and oxygen perfusion. C 2019 Elsevier Ltd. All rights reserved.</div>','<b>Title: </b>Inference of dynamical gene-regulatory networks based on time-resolved multi-stimuli multi-experiment data applying NetGenerator V2.0<br><b>Year: </b>2013<br><b>Author: </b> M. Weber,  Sg. Henkel,  S. Vlaic,  R. Guthke,  Ej. Van Zoelen and  D. Driesch<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-7-1"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>regulatory networks, gene regulatory, gene expression, network inference, expression data, network structure, gene networks, networks grns, regulatory genes, boolean networks, bayesian network, network reconstruction, networks gene, network dynamics, gene regulation, inference method, systems biology, time series, infer gene, network<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Inference of gene-regulatory networks GRNs is important for understanding behaviour and potential treatment of biological systems. Knowledge about GRNs gained from transcriptome analysis can be increased by multiple experiments and/or multiple stimuli. Since GRNs are complex and dynamical, appropriate methods and algorithms are needed for constructing models describing these dynamics. Algorithms based on heuristic approaches reduce the effort in parameter identification and computation time.Results: The NetGenerator V2.0 algorithm, a heuristic for network inference, is proposed and described. It automatically generates a system of differential equations modelling structure and dynamics of the network based on time-resolved gene expression data. In contrast to a previous version, the inference considers multi-stimuli multi-experiment data and contains different methods for integrating prior knowledge. The resulting significant changes in the algorithmic procedures are explained in detail. NetGenerator is applied to relevant benchmark examples evaluating the inference for data from experiments with different stimuli. Also, the underlying GRN of chondrogenic differentiation, a real-world multi-stimulus problem, is inferred and analysed.Conclusions: NetGenerator is able to determine the structure and parameters of GRNs and their dynamics. The new features of the algorithm extend the range of possible experimental set-ups, results and biological interpretations. Based upon benchmarks, the algorithm provides good results in terms of specificity, sensitivity, efficiency and model fit.</div>','<b>Title: </b>Splitting the dynamics of large biochemical interaction networks<br><b>Year: </b>2005<br><b>Author: </b> E. Pecou<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.jtbi.2004.08.018"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF THEORETICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>regulatory networks, stochastic simulation, gene networks, reaction networks, systems biology, stochastic model, network models, complex systems, networks stochastic, gene regulatory, biological systems, stochastic dynamics, differential equations, protein networks, cell cycle, continuous model, interaction networks, systems chemistry, time delays, biochemical reaction<div style="font-size:11px;"><br><b>Abstract: </b><em>This article is inscribed in the general motivation of understanding the dynamics on biochemical networks including metabolic and genetic interactions. Our approach is continuous modeling by differential equations. We address the problem of the huge size of those systems. We present a mathematical tool for reducing the size of the model, master-slave synchronization, and fit it to the biochemical context. C 2004 Elsevier Ltd. All rights reserved.</div>','<b>Title: </b>Discriminating between rival biochemical network models: three approaches to optimal experiment design<br><b>Year: </b>2010<br><b>Author: </b> B. Melykuti,  E. August,  A. Papachristodoulou and  H. El-Samad<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-4-38"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>systems biology, biological systems, system dynamics, molecular systems, experimental design, network model, model system, model parameters, experiment design, kinetic parameters, perfect adaptation, experimental data, molecular mechanisms, homeostatic mechanisms, switching dynamics, biological mechanisms, control systems, chemotaxis network, decentralized control, feedback control<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: The success of molecular systems biology hinges on the ability to use computational models to design predictive experiments, and ultimately unravel underlying biological mechanisms. A problem commonly encountered in the computational modelling of biological networks is that alternative, structurally different models of similar complexity fit a set of experimental data equally well. In this case, more than one molecular mechanism can explain available data. In order to rule out the incorrect mechanisms, one needs to invalidate incorrect models. At this point, new experiments maximizing the difference between the measured values of alternative models should be proposed and conducted. Such experiments should be optimally designed to produce data that are most likely to invalidate incorrect model structures.Results: In this paper we develop methodologies for the optimal design of experiments with the aim of discriminating between different mathematical models of the same biological system. The first approach determines the best initial condition that maximizes the L-2 energy distance between the outputs of the rival models. In the second approach, we maximize the L-2-distance of the outputs by designing the optimal external stimulus input profile of unit L-2-norm. Our third method uses optimized structural changes corresponding, for example, to parameter value changes reflecting gene knock-outs to achieve the same goal. The numerical implementation of each method is considered in an example, signal processing in starving Dictyostelium amoebae.Conclusions: Model-based design of experiments improves both the reliability and the efficiency of biochemical network model discrimination. This opens the way to model invalidation, which can be used to perfect our understanding of biochemical networks. Our general problem formulation together with the three proposed experiment design methods give the practitioner new tools for a systems biology approach to experiment design.</div>','<b>Title: </b>Inferring Epidemic Network Topology from Surveillance Data<br><b>Year: </b>2014<br><b>Author: </b> X. Wan,  Jm. Liu,  Wk. Cheung and  Tj. Tong<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0100661"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>complex networks, complex systems, network theory, network structure, biological network, social network, network science, dynamic networks, network systems, network model, network topology, systems biology, network, network analysis, system dynamics, networks, realworld networks, network automata, epidemic network, model network<div style="font-size:11px;"><br><b>Abstract: </b><em>The transmission of infectious diseases can be affected by many or even hidden factors, making it difficult to accurately predict when and where outbreaks may emerge. One approach at the moment is to develop and deploy surveillance systems in an effort to detect outbreaks as timely as possible. This enables policy makers to modify and implement strategies for the control of the transmission. The accumulated surveillance data including temporal, spatial, clinical, and demographic information, can provide valuable information with which to infer the underlying epidemic networks. Such networks can be quite informative and insightful as they characterize how infectious diseases transmit from one location to another. The aim of this work is to develop a computational model that allows inferences to be made regarding epidemic network topology in heterogeneous populations. We apply our model on the surveillance data from the 2009 H1N1 pandemic in Hong Kong. The inferred epidemic network displays significant effect on the propagation of infectious diseases.</div>','<b>Title: </b>Conditional robustness analysis for fragility discovery and target identification in biochemical networks and in cancer systems biology<br><b>Year: </b>2015<br><b>Author: </b> F. Bianconi,  E. Baldelli,  V. Luovini,  Ef. Petricoin,  L. Crino and  P. Valigi<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/s12918-015-0216-5"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>feedback loops, signaling network, positive feedback, network model, negative feedback, biochemical networks, signaling pathways, signal transduction, systems biology, kinase erk, protein kinase, transduction networks, dynamic model, emt network, erk pathway, cell fate, cell death, biochemical reaction, coupled feedback, impulse response<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: The study of cancer therapy is a key issue in the field of oncology research and the development of target therapies is one of the main problems currently under investigation. This is particularly relevant in different types of tumor where traditional chemotherapy approaches often fail, such as lung cancer.Results: We started from the general definition of robustness introduced by Kitano and applied it to the analysis of dynamical biochemical networks, proposing a new algorithm based on moment independent analysis of input/output uncertainty. The framework utilizes novel computational methods which enable evaluating the model fragility with respect to quantitative performance measures and parameters such as reaction rate constants and initial conditions. The algorithm generates a small subset of parameters that can be used to act on complex networks and to obtain the desired behaviors. We have applied the proposed framework to the EGFR-IGF1R signal transduction network, a crucial pathway in lung cancer, as an example of Cancer Systems Biology application in drug discovery. Furthermore, we have tested our framework on a pulse generator network as an example of Synthetic Biology application, thus proving the suitability of our methodology to the characterization of the input/output synthetic circuits.Conclusions: The achieved results are of immediate practical application in computational biology, and while we demonstrate their use in two specific examples, they can in fact be used to study a wider class of biological systems.</div>','<b>Title: </b>Attractor stability in nonuniform Boolean networks<br><b>Year: </b>2014<br><b>Author: </b> Cj. Kuhlman and  Hs. Mortveit<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.tcs.2014.08.010"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>THEORETICAL COMPUTER SCIENCE<br><b>Cluster-Keywords: </b><em>boolean networks, regulatory networks, network model, bayesian networks, model networks, systems biology, random boolean, real cells, immune system, genetic regulatory, networks, biochemical network, fluorescence images, cellular network, gene regulatory, cell differentiation, network, biological systems, ensemble approach, cell dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>The attractor graph of a Boolean network F has the attractors of F as vertices. There is a directed edge between attractors Ai and Aj if there is a state x is an element of Ai and an index k such that x when perturbed in the kth coordinate yields a state x in the basin of attractor Aj. Kauffman 2007 proposed to use the strongly connected components of the attractor graph called ergodic sets to model possible targets for cell differentiation. However, computational follow-up work by him and others has revealed that most Boolean networks have a single strongly connected component in an overwhelming number of cases, and two strongly connected components in rare cases.As a proof of concept, we explicitly construct classes of hierarchical, nonuniform Boolean networks where the attractor graph may have an arbitrary number of ergodic sets. The construction uses threshold functions and networks that are combinations of connected cliques. Earlier work does not address the structure of ergodic sets. Building on the previous construction we show how to generate Boolean networks where ergodic sets possess binary hyper-cubes Q2m as sub-graphs.Although the focus of this paper is to demonstrate that there are Boolean networks with a large number of ergodic sets and that ergodic sets can be arbitrarily large; we remark that networks we construct have community structure and the results may therefore be relevant to application areas ranging from biology to social science. The results also provide a starting point for additional studies into the attractor structure of Boolean networks, their basins of attractions, and conditions for stability under state noise. C 2014 Elsevier B.V. All rights reserved.</div>','<b>Title: </b>Queueing up for enzymatic processing: correlated signaling through coupled degradation<br><b>Year: </b>2011<br><b>Author: </b> Na. Cookson,  Wh. Mather,  T. Danino,  O. Mondragon-Palomino,  Rj. Williams,  Ls. Tsimring and  J. Hasty<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1038/msb.2011.94"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MOLECULAR SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>gene regulatory, systems biology, synthetic biology, regulatory network, gene network, gene expression, system model, synthetic network, network dynamics, systems theory, synthetic gene, genetic network, model predicts, dynamical systems, time delays, translational time, regulation models, oscillatory expression, network structure, biological systems<div style="font-size:11px;"><br><b>Abstract: </b><em>High-throughput technologies have led to the generation of complex wiring diagrams as a post-sequencing paradigm for depicting the interactions between vast and diverse cellular species. While these diagrams are useful for analyzing biological systems on a large scale, a detailed understanding of the molecular mechanisms that underlie the observed network connections is critical for the further development of systems and synthetic biology. Here, we use queueing theory to investigate how waiting lines can lead to correlations between protein customers that are coupled solely through a downstream set of enzymatic servers. Using the E. coli ClpXP degradation machine as a model processing system, we observe significant cross-talk between two networks that are indirectly coupled through a common set of processors. We further illustrate the implications of enzymatic queueing using a synthetic biology application, in which two independent synthetic networks demonstrate synchronized behavior when common ClpXP machinery is overburdened. Our results demonstrate that such post-translational processes can lead to dynamic connections in cellular networks and may provide a mechanistic understanding of existing but currently inexplicable links. Molecular Systems Biology 7: 561; published online 20 December 2011; doi: 10.1038/msb.2011.94</div>','<b>Title: </b>Bridging the gap between in silico and cell-based analysis of the nuclear factor-kappa B signaling pathway by in vitro studies of IKK2<br><b>Year: </b>2007<br><b>Author: </b> Aec. Ihekwaba,  Sj. Wilkinson,  D. Waithe,  Ds. Broomhead,  P. Li,  Rl. Grimley and  N. Benson<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1111/j.1742-4658.2007.05713.x"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>FEBS JOURNAL<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Previously, we have shown by sensitivity analysis, that the oscillatory behavior of nuclear factor NF-kappa B is coupled to free IkappaB kinase-2 IKK2 and IkappaBalphaI kappa B alpha, and that the phosphorylation of I kappa B alpha by IKK influences the amplitude of NF-kappa B oscillations. We have performed further analyses of the behavior of NF-kappa B and its signal transduction network to understand the dynamics of this system. A time lapse study of NF-kappa B translocation in 10 000 cells showed discernible oscillations in levels of nuclear NF-kappa B amongst cells when stimulated with interleukin IL-1 alpha, which suggests a small degree of synchronization amongst the cell population. When the kinetics for the phosphorylation of I kappa B alpha by IKK were measured, we found that the values for the affinity and catalytic efficiency of IKK2 for I kappa B alpha were dependent on assay conditions. The application of these kinetic parameters in our computational model of the NF-kappa B pathway resulted in significant differences in the oscillatory patterns of NF-kappa B depending on the rate constant value used. Hence, interpretation of in silico models should be made in the context of this uncertainty.</div>','<b>Title: </b>Molecular mechanisms of system responses to novel stimuli are predictable from public data<br><b>Year: </b>2014<br><b>Author: </b> Sa. Danziger,  Av. Ratushny,  Jj. Smith,  Ra. Saleem,  Yk. Wan,  Ce. Arens,  Am. Armstrong,  K. Sitko,  Wm. Chen,  Jh. Chiang,  Dj. Reiss,  Ns. Baliga and  Jd. Aitchison<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1093/nar/gkt938"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>NUCLEIC ACIDS RESEARCH<br><b>Cluster-Keywords: </b><em>systems biology, regulatory network, interaction networks, network model, gene regulatory, network, network biology, networks, gene network, response networks, network dynamics, dynamic model, cell networks, gene expression, biological networks, system dynamics, dynamic network, model, biological systems, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Systems scale models provide the foundation for an effective iterative cycle between hypothesis generation, experiment and model refinement. Such models also enable predictions facilitating the understanding of biological complexity and the control of biological systems. Here, we demonstrate the reconstruction of a globally predictive gene regulatory model from public data: a model that can drive rational experiment design and reveal new regulatory mechanisms underlying responses to novel environments. Specifically, using similar to 1500 publically available genome-wide transcriptome data sets from Saccharomyces cerevisiae, we have reconstructed an environment and gene regulatory influence network that accurately predicts regulatory mechanisms and gene expression changes on exposure of cells to completely novel environments. Focusing on transcriptional networks that induce peroxisomes biogenesis, the model-guided experiments allow us to expand a core regulatory network to include novel transcriptional influences and linkage across signaling and transcription. Thus, the approach and model provides a multi-scalar picture of gene dynamics and are powerful resources for exploiting extant data to rationally guide experimentation. The techniques outlined here are generally applicable to any biological system, which is especially important when experimental systems are challenging and samples are difficult and expensive to obtain-a common problem in laboratory animal and human studies.</div>','<b>Title: </b>A systems biology view of blood vessel growth and remodelling<br><b>Year: </b>2014<br><b>Author: </b> Ea. Logsdon,  Sd. Finley,  As. Popel and  F. Mac Gabhann<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1111/jcmm.12164"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF CELLULAR AND MOLECULAR MEDICINE<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Blood travels throughout the body in an extensive network of vessels - arteries, veins and capillaries. This vascular network is not static, but instead dynamically remodels in response to stimuli from cells in the nearby tissue. In particular, the smallest vessels - arterioles, venules and capillaries - can be extended, expanded or pruned, in response to exercise, ischaemic events, pharmacological interventions, or other physiological and pathophysiological events. In this review, we describe the multi-step morphogenic process of angiogenesis - the sprouting of new blood vessels - and the stability of vascular networks in vivo. In particular, we review the known interactions between endothelial cells and the various blood cells and plasma components they convey. We describe progress that has been made in applying computational modelling, quantitative biology and high-throughput experimentation to the angiogenesis process.</div>','<b>Title: </b>Conformational flexibility of Mycobacterium tuberculosis thioredoxin reductase: crystal structure and normal-mode analysis<br><b>Year: </b>2005<br><b>Author: </b> M. Akif,  K. Suhre,  C. Verma and  Sc. Mande<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1107/S0907444905030519"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ACTA CRYSTALLOGRAPHICA SECTION D-STRUCTURAL BIOLOGY<br><b>Cluster-Keywords: </b><em>normal mode, mode analysis, elastic network, functional motions, functional dynamics, molecular chaperone, structural biology, acidic residues, network models, molecular dynamics, atomic resolution, protein complexes, conserved functional, hsp dynamics, correlated motions, protein folding, large biomolecular, thioredoxin system, harmonic analysis, protein motions<div style="font-size:11px;"><br><b>Abstract: </b><em>The thioredoxin system exists ubiquitously and participates in essential antioxidant and redox-regulation processes via a pair of conserved cysteine residues. In Mycobacterium tuberculosis, which lacks a genuine glutathione system, the thioredoxin system provides reducing equivalents inside the cell. The three-dimensional structure of thioredoxin reductase from M. tuberculosis has been determined at 3 A resolution. TLS refinement reveals a large libration axis, showing that NADPH-binding domain has large anisotropic disorder. The relative rotation of the NADPH domain with respect to the FAD domain is necessary for the thioredoxin reduction cycle, as it brings the spatially distant reacting sites close together. Normal-mode analysis carried out based on the elastic network model shows that the motion required to bring about the functional conformational change can be accounted for by motion along one single mode. TLS refinement and normal-mode analysis thus enhance our understanding of the associated conformational changes.</div>','<b>Title: </b>Dynamic Changes in the Splenic Transcriptome of Chickens during the Early Infection and Progress of Mareks Disease<br><b>Year: </b>2017<br><b>Author: </b> L. Dang,  M. Teng,  Hw. Li,  Hz. Li,  Sm. Ma,  P. Zhao,  Xj. Li,  Rg. Deng,  Gp. Zhang and  J. Luo<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1038/s41598-017-11304-y"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SCIENTIFIC REPORTS<br><b>Cluster-Keywords: </b><em>gene regulatory, systems biology, regulatory networks, gene network, stem cell, network model, transcription factors, gene expression, cell transcriptomes, network analysis, data analysis, stem cells, network grnsight, cell transformation, weighted network, gene ontology, networks grns, graph model, graph layout, endothelial cell<div style="font-size:11px;"><br><b>Abstract: </b><em>Gallid alphaherpesvirus 2 GaHV2 is an oncogenic avian herpesvirus inducing Mareks disease MD and rapid-onset T-cell lymphomas. To reveal molecular events in MD pathogenesis and tumorigenesis, the dynamic splenic transcriptome of GaHV2-infected chickens during early infection and pathogenic phases has been determined utilizing RNA-seq. Based on the significant differentially expressed genes DEGs, analysis of gene ontology, KEGG pathway and protein-protein interaction network has demonstrated that the molecular events happening during GaHV2 infection are highly relevant to the disease course. In the Cornell Model description of MD, innate immune responses and inflammatory responses were established at early cytolytic phase but persisted until lymphoma formation. Humoral immunity in contrast began to play a role firstly in the intestinal system and started at late cytolytic phase. Neurological damage caused by GaHV2 is first seen in early cytolytic phase and is then sustained throughout the following phases over a long time period. During the proliferative phase many pathways associated with transcription and/or translation were significantly enriched, reflecting the cell transformation and lymphoma formation. Our work provides an overall view of host responses to GaHV2 infection and offers a meaningful basis for further studies of MD biology.</div>','<b>Title: </b>Prediction of Growth Factor-Dependent Cleft Formation During Branching Morphogenesis Using A Dynamic Graph-Based Growth Model<br><b>Year: </b>2016<br><b>Author: </b> N. Dhulekar,  S. Ray,  D. Yuan,  A. Baskaran,  B. Oztan,  M. Larsen and  B. Yener<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1109/TCBB.2015.2452916"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IEEE-ACM TRANSACTIONS ON COMPUTATIONAL BIOLOGY AND BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>biological networks, dynamic models, biological systems, model predictions, model parameters, systems biology, mathematical models, biological models, equation models, experimental data, molecular network, interaction networks, model errors, biological interaction, molecular interactions, model, dynamic systems, biological parameters, parameter uncertainty, growth model<div style="font-size:11px;"><br><b>Abstract: </b><em>This study considers the problem of describing and predicting cleft formation during the early stages of branching morphogenesis in mouse submandibular salivary glands SMG under the influence of varied concentrations of epidermal growth factors EGF. Given a time-lapse video of a growing SMG, first we build a descriptive model that captures the underlying biological process and quantifies the ground truth. Tissue-scale global and morphological features related to regions of interest local features are used to characterize the biological ground truth. Second, we devise a predictive growth model that simulates EGF-modulated branching morphogenesis using a dynamic graph algorithm, which is driven by biological parameters such as EGF concentration, mitosis rate, and cleft progression rate. Given the initial configuration of the SMG, the evolution of the dynamic graph predicts the cleft formation, while maintaining the local structural characteristics of the SMG. We determined that higher EGF concentrations cause the formation of higher number of buds and comparatively shallow cleft depths. Third, we compared the prediction accuracy of our model to the Glazier-Graner-Hogeweg GGH model, an on-lattice Monte-Carlo simulation model, under a specific energy function parameter set that allows new rounds of de novo cleft formation. The results demonstrate that the dynamic graph model yields comparable simulations of gland growth to that of the GGH model with a significantly lower computational complexity. Fourth, we enhanced this model to predict the SMG morphology for an EGF concentration without the assistance of a ground truth time-lapse biological video data; this is a substantial benefit of our model over other similar models that are guided and terminated by information regarding the final SMG morphology. Hence, our model is suitable for testing the impact of different biological parameters involved with the process of branching morphogenesis in silico, while reducing the requirement of in vivo experiments.</div>','<b>Title: </b>Fundamental limitation of the instantaneous approximation in fold-change detection models<br><b>Year: </b>2015<br><b>Author: </b> M. Skataric,  Ev. Nikolaev and  Ed. Sontag<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1049/iet-syb.2014.0006"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IET SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>systems biology, biological systems, system dynamics, molecular systems, experimental design, network model, model system, model parameters, experiment design, kinetic parameters, perfect adaptation, experimental data, molecular mechanisms, homeostatic mechanisms, switching dynamics, biological mechanisms, control systems, chemotaxis network, decentralized control, feedback control<div style="font-size:11px;"><br><b>Abstract: </b><em>The phenomenon of fold-change detection, or scale-invariance, is exhibited by a variety of sensory systems, in both bacterial and eukaryotic signalling pathways. It has been often remarked in the systems biology literature that certain systems whose output variables respond at a faster time scale than internal components give rise to an approximate scale-invariant behaviour, allowing approximate fold-change detection in stimuli. This study establishes a fundamental limitation of such a mechanism, showing that there is a minimal fold-change detection error that cannot be overcome, no matter how large the separation of time scales is. To illustrate this theoretically predicted limitation, the authors discuss two common biomolecular network motifs, an incoherent feedforward loop and a feedback system, as well as a published model of the chemotaxis signalling pathway of Dictyostelium discoideum.</div>','<b>Title: </b>Birhythmicity and Hard Excitation from Coupled Synthetic Feedback Loops<br><b>Year: </b>2014<br><b>Author: </b> Am. Chen,  Cx. Liu and  Jw. Wang<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1155/2014/694854"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF APPLIED MATHEMATICS<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Synthetic biology opens up the possibility of creating circuits that would not survive in the natural world and studying their behaviors in living cells, expanding our notion of biology. Based on this, we analyze on a synthetic biological system the effect of coupling between two instability-generating mechanisms. The systems considered are two topologically equivalent synthetic networks. In addition to simple periodic oscillations and stable steady state, the system can exhibit a variety of new modes of dynamic behavior: coexistence between two stable periodic regimes birhythmicity and coexistence of a stable periodic regime with a stable steady state hard excitation. Birhythmicity and hard excitation have been proved to exist in biochemical networks. Through bifurcation analysis on these two synthetic cellular networks, we analyze the function of network structure for the collapse and revival of birhythmicity and hard excitation with the variation of parameters. The results have illustrated that the bifurcation space can be divided into four subspaces for which the dynamical behaviors of the system are generically distinct. Our analysis corroborates the results obtained by numerical simulation of the dynamics.</div>','<b>Title: </b>Autophagy capacity and sub-mitochondrial heterogeneity shape Bnip3-induced mitophagy regulation of apoptosis<br><b>Year: </b>2015<br><b>Author: </b> Sc. Choe,  A. Hamacher-Brady and  Nr. Brady<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/s12964-015-0115-9"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>CELL COMMUNICATION AND SIGNALING<br><b>Cluster-Keywords: </b><em>chaotic system, control systems, neural networks, jerk chaotic, torch chaotic, mode control, sliding mode, plasma torch, dynamic neural, mitochondrial dynamics, system, systems biology, adaptive integral, model system, rodtype plasma, integral sliding, systems, jerk system, cancer cell, comparison method<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Mitochondria are key regulators of apoptosis. In response to stress, BH3-only proteins activate pro-apoptotic Bcl2 family proteins Bax and Bak, which induce mitochondrial outer membrane permeabilization MOMP. While the large-scale mitochondrial release of pro-apoptotic proteins activates caspase-dependent cell death, a limited release results in sub-lethal caspase activation which promotes tumorigenesis. Mitochondrial autophagy mitophagy targets dysfunctional mitochondria for degradation by lysosomes, and undergoes extensive crosstalk with apoptosis signaling, but its influence on apoptosis remains undetermined. The BH3-only protein Bnip3 integrates apoptosis and mitophagy signaling at different signaling domains. Bnip3 inhibits pro-survival Bcl2 members via its BH3 domain and activates mitophagy through its LC3 Interacting Region LIR, which is responsible for binding to autophagosomes. Previously, we have shown that Bnip3-activated mitophagy prior to apoptosis induction can reduce mitochondrial activation of caspases, suggesting that a reduction to mitochondrial levels may be pro-survival. An outstanding question is whether organelle dynamics and/or recently discovered subcellular variations of protein levels responsible for both MOMP sensitivity and crosstalk between apoptosis and mitophagy can influence the cellular apoptosis decision event. To that end, here we undertook a systems biology analysis of mitophagy-apoptosis crosstalk at the level of cellular mitochondrial populations.Results: Based on experimental findings, we developed a multi-scale, hybrid model with an individually adaptive mitochondrial population, whose actions are determined by protein levels, embedded in an agent-based model ABM for simulating subcellular dynamics and local feedback via reactive oxygen species signaling. Our model, supported by experimental evidence, identified an emergent regulatory structure within canonical apoptosis signaling. We show that the extent of mitophagy is determined by levels and spatial localization of autophagy capacity, and subcellular mitochondrial protein heterogeneities. Our model identifies mechanisms and conditions that alter the mitophagy decision within mitochondrial subpopulations to an extent sufficient to shape cellular outcome to apoptotic stimuli.Conclusion: Overall, our modeling approach provides means to suggest new experiments and implement findings at multiple scales in order to understand how network topologies and subcellular heterogeneities can influence signaling events at individual organelle level, and hence, determine the emergence of heterogeneity in cellular decisions due the actions of the collective intra-cellular population.</div>','<b>Title: </b>Agreeing to disagree: synergies between particle swarm optimisation and complex networks<br><b>Year: </b>2018<br><b>Author: </b> M. Udrescu and  A. LihuLink: No link<br><b>Journal: </b>SWARM INTELLIGENCE, VOL 1: PRINCIPLES, CURRENT ALGORITHMS AND METHODS<br><b>Cluster-Keywords: </b><em>complex networks, complex systems, network theory, network structure, biological network, social network, network science, dynamic networks, network systems, network model, network topology, systems biology, network, network analysis, system dynamics, networks, realworld networks, network automata, epidemic network, model network<div style="font-size:11px;"><br><b>Abstract: </b><em>Due to its numerous applications in fields like systems biology, medicine, technology, engineering, or social sciences, the new science of complex networks CNs has become extremely popular over the last couple of decades. Particle swarm optimisation PSO and CN science have significant common ground, as they both deal with a set of agents particles for PSOs, vertices for CNs which interact according to an underlying topology. Moreover, one of the most important branches of CN science is represented by social networks, while PSO was originally conceived for optimisation through the simulation of social behaviour, thus further emphasising the strong tie between the two scientific fields. A prominent problem in CNs is the opinion dynamics in social networks. To this end, opinion interaction models are tested in order to verify if they can recreate a realistic social behaviour; a ubiquitous feature of such real-life social behaviours is persistent opinion disagreement. One of the most important characteristics of a social network which fosters disagreement is that it contains stubborn agents, namely vertices which never change their opinion. We discuss models of disagreement, such as that with stubborn agents and tolerance- based models, then offer a new perspective in exploring the frontiers between network science and PSO.</div>','<b>Title: </b>Reaction network realizations of rational biochemical systems and their structural properties<br><b>Year: </b>2015<br><b>Author: </b> A. Gabor,  Km. Hangos,  Jr. Banga and  G. Szederkenyi<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s10910-015-0511-9"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF MATHEMATICAL CHEMISTRY<br><b>Cluster-Keywords: </b><em>reaction networks, reaction systems, metabolic networks, chemical reaction, network structure, systems biology, biological systems, biological networks, biochemical reaction, reaction rates, differential equations, reaction graph, complex networks, ssi modules, networks, life systems, ordinary differential, cellular networks, reaction, network flows<div style="font-size:11px;"><br><b>Abstract: </b><em>In this paper, a frequently used representation of mass-action type reaction networks is extended to a more general system class where the reaction rates are in rational function form. An algorithm is given to compute a possible reaction graph from the kinetic differential equations. However, this structure is generally non-unique, as it is illustrated through the phenomenon of dynamical equivalence, when different reaction network structures correspond to exactly the same dynamics. It is shown that under some technical assumptions, the so-called dense realization containing the maximal number of reactions, forms a super-structure in the sense that the reaction graph of any dynamically equivalent reaction network is the sub-graph of the dense realization. Additionally, optimization based methods are given to find dynamically equivalent realizations with preferred properties, such as dense realizations or sparse realizations. The introduced concepts are illustrated by examples.</div>','<b>Title: </b>Gene Coexpression as Hebbian Learning in Prokaryotic Genomes<br><b>Year: </b>2013<br><b>Author: </b> G. Vey<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s11538-013-9900-z"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BULLETIN OF MATHEMATICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>biological networks, dynamic models, biological systems, model predictions, model parameters, systems biology, mathematical models, biological models, equation models, experimental data, molecular network, interaction networks, model errors, biological interaction, molecular interactions, model, dynamic systems, biological parameters, parameter uncertainty, growth model<div style="font-size:11px;"><br><b>Abstract: </b><em>Biological interaction networks represent a powerful tool for characterizing intracellular functional relationships, such as transcriptional regulation and protein interactions. Although artificial neural networks are routinely employed for a broad range of applications across computational biology, their underlying connectionist basis has not been extensively applied to modeling biological interaction networks. In particular, the Hopfield network offers nonlinear dynamics that represent the minimization of a system energy function through temporally distinct rewiring events. Here, a scaled energy minimization model is presented to test the feasibility of deriving a composite biological interaction network from multiple constituent data sets using the Hebbian learning principle. The performance of the scaled energy minimization model is compared against the standard Hopfield model using simulated data. Several networks are also derived from real data, compared to one another, and then combined to produce an aggregate network. The utility and limitations of the proposed model are discussed, along with possible implications for a genomic learning analogy where the fundamental Hebbian postulate is rendered into its genomic equivalent: Genes that function together junction together.</div>','<b>Title: </b>Strategies for dealing with incomplete information in the modeling of molecular interaction networks<br><b>Year: </b>2006<br><b>Author: </b> H. De Jong and  D. Ropers<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1093/bib/bbl034"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BRIEFINGS IN BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>regulatory networks, network dynamics, gene regulatory, network model, genetic network, gene networks, genetic regulatory, mathematical model, parameter space, network system, interaction networks, system dynamics, network structure, network, dynamic networks, network reduction, models models, biological networks, network motifs, biological systems<div style="font-size:11px;"><br><b>Abstract: </b><em>Modelers of molecular interaction networks encounter the paradoxical situation that while large amounts of data are available, these are often insufficient for the formulation and analysis of mathematical models describing the network dynamics. In particular, information on the reaction mechanisms and numerical values of kinetic parameters are usually not available for all but a few well-studied model systems. In this article we review two strategies that have been proposed for dealing with incomplete information in the study of molecular interaction networks: parameter sensitivity analysis and model simplification. These strategies are based on the biologically justified intuition that essential properties of the system dynamics are robust against moderate changes in the value of kinetic parameters or even in the rate laws describing the interactions. Although advanced measurement techniques can be expected to relieve the problem of incomplete information to some extent, the strategies discussed in this article will retain their interest as tools providing an initial characterization of essential properties of the network dynamics.</div>','<b>Title: </b>Analytical reduction of combinatorial complexity arising from multiple protein modification sites<br><b>Year: </b>2015<br><b>Author: </b> Mr. Birtwistle<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1098/rsif.2014.1215"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF THE ROYAL SOCIETY INTERFACE<br><b>Cluster-Keywords: </b><em>allosteric proteins, signaling networks, protein interactions, network model, molecular dynamics, protein complexes, allosteric network, signaling proteins, cell network, biochemical networks, free energy, combinatorial complexity, molecular species, biological systems, receptor signaling, systems biology, chemical species, model, dynamics simulations, cell signaling<div style="font-size:11px;"><br><b>Abstract: </b><em>Combinatorial complexity is a major obstacle to ordinary differential equation ODE modelling of biochemical networks. For example, a protein with 10 sites that can each be unphosphorylated, phosphorylated or bound to adaptor protein requires 310 ODEs. This problem is often dealt with by making ad hoc assumptions which have unclear validity and disallow modelling of site-specific dynamics. Such site-specific dynamics, however, are important in many biological systems. We show here that for a common biological situation where adaptors bind modified sites, binding is slow relative to modification/demodification, and binding to one modified site hinders binding to other sites, for a protein with n modification sites and m adaptor proteins the number of ODEs needed to simulate the site-specific dynamics of biologically relevant, lumped bound adaptor states is independent of the number of modification sites and equal to m  1, giving a significant reduction in system size. These considerations can be relaxed considerably while retaining reasonably accurate descriptions of the true system dynamics. We apply the theory to model, using only 11 ODEs, the dynamics of ligand-induced phosphorylation of nine tyrosines on epidermal growth factor receptor EGFR and primary recruitment of six signalling proteins Grb2, PI3K, PLC gamma 1, SHP2, RasA1 and Shc1. The model quantitatively accounts for experimentally determined site-specific phosphorylation and dephosphorylation rates, differential affinities of binding proteins for the phosphorylated sites and binding protein expression levels. Analysis suggests that local concentration of site-specific phosphatases such as SHP2 in membrane subdomains by a factor of approximately 107 is critical for effective site-specific regulation. We further show how our framework can be extended with minimal effort to consider binding cooperativity between Grb2 and c-Cbl, which is important for receptor trafficking. Our theory has potentially broad application to reduce combinatorial complexity and allow practical simulation of a variety ODE models relevant to systems biology and pharmacology applications to allow exploration of key aspects of complexity that control signal flux.</div>','<b>Title: </b>Network Representation of T-Cell Repertoire - A Novel Tool to Analyze Immune Response to Cancer Formation<br><b>Year: </b>2018<br><b>Author: </b> A. Priel,  M. Gordin,  H. Philip,  A. Zilberberg and  S. Efroni<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3389/fimmu.2018.02913"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>FRONTIERS IN IMMUNOLOGY<br><b>Cluster-Keywords: </b><em>interaction networks, proteinprotein interactions, protein interaction, network model, complex systems, systems biology, protein protein, protein complex, host network, dynamic interactions, network analysis, biological systems, networks network, cellular network, biological network, viral proteins, interactome networks, regulatory networks, immune system, systems network<div style="font-size:11px;"><br><b>Abstract: </b><em>The T cell repertoire potentially presents complexity compatible, or greater than, that of the human brain. T cell based immune response is involved with practically every part of human physiology, and high-throughput biology needed to follow the T-cell repertoire has made great leaps with the advent of massive parallel sequencing 1. Nevertheless, tools to handle and observe the dynamics of this complexity have only recently started to emerge e.g., 2-4 in parallel with sequencing technologies. Here, we present a network-based view of the dynamics of the T cell repertoire, during the course of mammary tumors development in a mouse model. The transition from the T cell receptor as a feature, to network-based clustering, followed by network-based temporal analyses, provides novel insights to the workings of the system and provides novel tools to observe cancer progression via the perspective of the immune system. The crux of the approach here is at the network-motivated clustering. The purpose of the clustering step is not merely data reduction and exposing structures, but rather to detect hubs, or attractors, within the T cell receptor repertoire that might shed light on the behavior of the immune system as a dynamic network. The Clone-Attractor is in fact an extension of the clone concept, i.e., instead of looking at particular clones we observe the extended clonal network by assigning clusters to graph nodes and edges to adjacent clusters editing distance metric. Viewing the system as dynamical brings to the fore the notion of an attractors landscape, hence the possibility to chart this space and map the sample state at a given time to a vector in this large space. Based on this representation we applied two different methods to demonstrate its effectiveness in identifying changes in the repertoire that correlate with changes in the phenotype: 1 network analysis of the TCR repertoire in which two measures were calculated and demonstrated the ability to differentiate control from transgenic samples, and, 2 machine learning classifier capable of both stratifying control and trangenic samples, as well as to stratify pre-cancer and cancer samples.</div>','<b>Title: </b>Population structure of spotted salamanders Ambystoma maculatum in a fragmented landscape<br><b>Year: </b>2009<br><b>Author: </b> Jl. Purrenhage,  Ph. Niewiarowski and  Fbg. Moore<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1111/j.1365-294X.2008.04024.x"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MOLECULAR ECOLOGY<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Understanding the impacts of landscape-level processes on the population biology of amphibians is critical, especially for species inhabiting anthropogenically modified landscapes. Many pond-breeding amphibians are presumed to exist as metapopulations, but few studies demonstrate the extent and consequences of this metapopulation structure. Gene flow measures may facilitate the construction of more realistic models of population structure than direct measures of migration. This is especially true for species that are cryptic, such as many amphibians. We used eight polymorphic microsatellite loci to determine the genetic population structure of spotted salamanders Ambystoma maculatum breeding at 17 ponds in northeastern Ohio, a landscape fragmented by roads, agriculture, urban areas and the Cuyahoga River. Using a variety of analyses Bayesian clustering, F-statistics, AMOVA we generated a model of salamander population genetic structure. Our data revealed patterns of genetic connectivity that were not predicted by geographical distances between ponds no isolation by distance. We also tested for a relationship between population structure and several indices of landscape resistance, but found no effect of potential barriers to dispersal on genetic connectivity. Strong overall connectivity among ponds, despite the hostile habitat matrix, may be facilitated by a network of riparian corridors associated with the Cuyahoga River; however, high gene flow in this system may indicate a general ability to disperse and colonize beyond particular corridors.</div>','<b>Title: </b>On the holistic approach in cancer biology: Tumor necrosis factor, colon cancer cells, chaos theory and complexity<br><b>Year: </b>2004<br><b>Author: </b> M. Muc-Wierzgon,  E. Nowakowska-Zajdel,  T. Kokot,  K. Sosada,  B. Zubelewicz,  J. Wierzgon,  M. Cichocka,  E. Fatyga and  A. BrodziakLink: No link<br><b>Journal: </b>JOURNAL OF BIOLOGICAL REGULATORS AND HOMEOSTATIC AGENTS<br><b>Cluster-Keywords: </b><em>regulatory networks, network dynamics, gene regulatory, network model, genetic network, gene networks, genetic regulatory, mathematical model, parameter space, network system, interaction networks, system dynamics, network structure, network, dynamic networks, network reduction, models models, biological networks, network motifs, biological systems<div style="font-size:11px;"><br><b>Abstract: </b><em>TNF alpha plays a role in the pathogenesis of septic shock, inflammatory diseases, autoimmune diseases, graft rejection reaction, acute, and chronic respiratory inefficiency among others. Its activity depends on the type of target cells and different regulating factors, but the effect of biological activity is conditioned by specific receptors such as p55 type I, TNF R55 and p75 type II, TNF R75. The aim of the study was to answer the following questions: 1 Is it possible to apply elements of non-linear dynamics to assess the level of expression of TNF, TNFRI, TNFRII genes in tumor cells, pathologically unchanged tissue and metastatically changed lymph nodes 2 Is theoretically anticipated variability of cytokine and its receptors in colorectal carcinoma cells and the immediate vicinity justified in the developed mathematical model The research material - specimens taken from tumor, unchanged tissue and metastatic lymph nodes - were histopathologically and molecularly analysed. Results of the molecular research were used to develop a mathematical model using the basic studies on the theory of chaos and biological system modelling.</div>','<b>Title: </b>46th ESAO Congress 3-7 September 2019 Hannover, Germany Abstracts<br><b>Year: </b>2019<br><b>Author: </b>Anonymous<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1177/0391398819860985"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>INTERNATIONAL JOURNAL OF ARTIFICIAL ORGANS<br><b>Cluster-Keywords: </b><em>blood flow, blood cells, flow rate, blood pumps, cells, cell, results, patients, stem cells, methods, blood, patients blood, flow, patients methods, patients objectives, cell viability, objectives, results cells, blood pressure, cell culture<div style="font-size:11px;"><br><b>Abstract: </b><em>Objectives:ADVOS multi is a recirculating albumin-based dialysis device that supports kidney, liver and lung function by eliminating CO2, water-soluble and protein-bound substances. In the present work in vitro data on the removal of these substances and CO2 are presented. Methods: An ex vivo model using porcine blood was established and applied in detoxification tests for water soluble and protein bound retention solutes. 3 x 3.3L of blood with high bilirubin 30 mg/dl and lactate levels >10 mmol/l were treated with ADVOS multi for 4 hours each. This design, with 3 phases changing blood every 4 hours, led to high concentrations of both markers in blood during the 12 h test period. For CO2 removal tests, 5 liters of blood were used instead. In both cases, a continuous CO2 supply through an additional dialyzer was applied. Results: Bilirubin and lactate were efficiently removed during 12hrs of in vitro detoxification. Lactate removal rates were 90, 86 and 84 for phase 1, 2 and 3, respectively. Bilirubin elimination rates were 66, 62, and 57, resulting in a total elimination of 1150 mg in 12 hrs. Albumin binding capacity was determined to be >76 at the end of the treatment indicating that albumin was not denatured in the dialysate of the ADVOS system. CO2 removal with ADVOS multi depends on three variables: 1 The amount of supplied CO2 depends on concentrate flow affecting both, blood pCO2 and bicarbonate levels; 2 blood flow, and 3 dialysate pH and composition i.e., carbonate concentration. A maximum CO2 removal of 142 ml/min was achieved with a carbonate-free dialysate at pH 10, a blood flow of 400 ml/min and a concentrate flow of 160 ml/min. Given that blood gases are maintained within physiological conditions, a CO2 removal rate of 61 ml/min can be achieved. During all the experiments blood pH was set to 7.35-7.45. Discussion:ADVOS multi is a device using albumin recirculation in its secondary circuit. Here, albumin binding capacity remains stable through a systematic modification of its tertiary structure through temperature and pH changes in the ADVOS multi circuit. This facilitates the release of toxins from albumin and allows for further binding. Moreover, presence of albumin, variable dialysate composition and the flexible dialysate pH might facilitate the treatment of patients with multiple organ failure.Objectives:Given the extent of bypass graft failure, the motivation behind this multidisciplinary project is to improve the patency of the current bypass grafts by developing a novel and optimised blood flow augmentation technique. Methods: One of the most significant contributions to the improvement of haemodynamics in grafts was based on a research which showed that the spiral flow is a natural phenomenon in the whole arterial system and is induced by the twisting of the left ventricle during contraction and then accentuated upon entering the aortic arch. The benefit of this flow pattern lies in removing unfavourable haemodynamic environment such as turbulence, stagnation and oscillatory shear stress, which are believed to be the main causes of intimal hyperplasia at anastomotic configurations. Results:This multi-disciplinary engineering venture has resulted in a unique product which makes use of both non-planar helicity and an optimised internal ridge within the graft to achieve a significantly improved haemodynamic condition within the anastomosis an anastomosis is a surgical connection between autologous/prosthetic grafts and veins/arteries inside the human body Discussion:This truly multidisciplinary project has integrated fluid mechanics, biomechanics and biology with cardiovascular surgery to develop a novel biomedical device, inspired by the nuclear engineering sector. The novel spiral-inducing bypass graft, nominated for this award, is the best example of how engineering techniques, tools and designs can lead to life-saving innovations that could potentially save the lives of thousands of people and save millions of pounds for the healthcare systems across the world. Such successful engineering stories are what would encourage the next generation of engineers to go beyond the traditional boundaries of engineering disciplines to make a difference.Objectives:The goal of this work was to study methods that will allow increase in spatial freedom of couplers orientation in inductive powering unit for implantable medical devices. An inductive energy transfer system is considered, therefore the freedom of relative orientation of the transmitting and receiving coils must be optimized. Methods:The design procedure of inductive powering unit IPU consists of two main steps: design of the power amplifier and optimization of an inductive link. To account for the patient specifics, such as tissue thickness near the implantation site, a number of different couplers were simulated using finite element modeling. Effect of coils geometrical parameters on mutual inductance and inductive link efficiency was examined. Class E power amplifier was chosen as a driver for the transmitting coil. Capacitors in the amplifier loading network were tuned to achieve zero-voltage switching ZVS. A patients everyday activity, including walking and even breathing, leads to misalignment of the transmitting and receiving coils, and, as a result, detuning of the amplifier from ZVS and increased losses. Feedback schemes were studied to compensate the misalignments effect on the amplifier. Results:A self-oscillating IPU with class E amplifier was designed that provides stable output power about 0.5 W for the distance between the couplers in range 10-20 mm and the lateral distance up to 20 mm. The use of self-oscillating circuit in the transmitter increases spatial freedom of the transmitting and receiving coils corresponding to ZVS. Discussion: Inductive link geometrical parameters optimization and implementation of self-oscillating class E driver the IPU contributes to stable output power and efficiency of IPU. As a result, less intervention from the patient and physician is required, and patient quality of life is increased Precision medicine emerges from integration of a number of emerging technologies and the data they produce with modern data analytics. For precision diagnostics and for predicting drug responses new computational and in-vitro models are needed from gene regulation to cellular and organ functions. Human induced pluripotent stem cells hIPSC derived e.g., from patient blood cells provide means to produce most cell types and thus provide means to get patient specific in-vitro models. New technologies are needed to produce tissues from these cells and to assess the cell functions in-vitro. In addition, computational in-silico models can be used to augment our understanding of the diseases or drug effects. They also provide tools to translate the in-vitro findings to clinical settings and patient populations. We are in transition to turn our in-vitro cell culture models to body-on-chip platforms including environmental control and biophysical functional sensing. We have developed methods to assess the cellular functions based on electrophysiological sensing as well as 2D and 3D bioimaging. For example, we have developed imaging methods to assess functions of hIPSC cardiac cell with simultaneous assessment of electrophysiology such as Calcium and voltage transients as well as mechanobiology in vitro. Further, we have developed in-silico models of various cellular function including multi-cell-type neuronal networks and in-silico population models of the hIPSC cardiomyocytes. The later ones are providing us ionic machinery of hIPSC derived cardiomyocyte electrophysiology in various populations of patiets.We have shown that these computational models can represent pathological patient phenotype cells and populations of patients with specific mutations, e.g., long QT syndrome. We have also demonstrated the power of in-silico as possible pre-screening method for drug effects prior to in-vitro examinations. Moreover, our in-silico results highlight the need of careful consideration of use of HiPSC models before they can be turned from immature cell models to mature tissues in vitro. With integration of novel engineering expertise from multimodal sensing, imaging and computational modelling, we have shown their power on studying diseases and for pre-screening of compounds. Our results demonstrate the power of combined in-vitro and in-silco methods for future precision medicine.Objectives: We have developed in vivo tissue engineered autologous vascular grafts constructed in the subcutaneous of the recipient body. However, since the formation of the vascular grafts depends on the conditions of recipients including high risk or immature patients, immaturity in the fabricated tissues might be problematic for the severely diseased patients because of their suppressed regenerative activity. Therefore, possibility of the xenogeneic or the allogeneic implantation of the grafts should be evaluated. The objective of this study is to fabricate cardiovascular grafts using xenogeneic or allogeneic animals. Methods: Silicone rod molds were placed into subcutaneous pouches of beagle dogs, and after 4 weeks the implants with their surrounded connective tissues were harvested. Those were decellularized with detergents and stored at -20 degrees C for 1 week. Decellularized tubular connective tissues internal diameter: 2 mm were xeno-transplanted to abdominal aorta of the rats. Decellularized tubular connective tissues internal diameter: 5 mm were cut open and trimmed to elliptical sheets of 15 x 8 mm, they were allo-transplanted to carotid arteries of other beagle dogs as vascular patches. Results: Both xenogeneic vascular grafts and allogeneic patch grafts performed well after transplantation, and the luminal surfaces after resection were very smooth. Histological evaluation also showed host cells infiltration into the grafts. Discussion: Decellularized xenogeneic and allogeneic connective tissue membranes could be ideal vascular grafts.Objectives: This work aimed to print high-resolution, collagen-based, constructs via suspended 3D printing with load-bearing and compositions closer to native bone; for potential use as implant materials. Methods: Collagen type I Col and gelatin methacroylate GelMA blends were systematically investigated as bio-inks, probing their rheological properties and crosslinking efficiency for printing. An adapted 3D bioprinter 3DDiscovery, regenHU, Switzerland based on an extrusion principle was used to print constructs. Calcification was investigated, in vitro, using a polymer-induced liquid precursor for the mineralization process. Results: Careful control over the formulation and processing resulted in refined construct properties such as: wall width 500um, lattice length 2cm and shape bone trabeculae. Once printed, the ability to cure the GelMA/Col blends was dependent on photo-polymerisation methodology, with enhanced curing and lower remaining soluble fractions 10 vs 40 for visible light  Riboflavin/SPS in comparison to UV  Irgacure. Control over the construct structure allowed defined mineralisation, and subsequent material responses. Discussion: In recent years the development of 3D printing technologies has attempted to combat the growing need for bone repair solutions, although is limited by the number of bio-inks, and printable resolutions available. Suspended manufacture has sought to address this issue, using a fluid gel to support a secondary biologically relevant bio-ink whilst it undergoes a curing step, during or post-printing. To date, printing techniques have not been shown to provide fully resorbable and/or mechanically satisfactory bone implants. This research has shown promise as the first steps towards printing high resolution constructs with chemical compositions more closely matching that of natural bone. Further works involve deeper investigation of calcification and impact on implants mechanical properties and microstructure.Shifting from treating symptoms to curing chronic diseases by making the transformative promise of Advanced Therapies a reality for the benefit of patients and society and by making Europe a spearhead of Advanced Therapies in Science, Clinics and Biomedical Industry, that is the vision of the large-scale research initiative RESTORE - Health by Advanced Therapies. The increasing prevalence of chronic diseases and multi-morbidity due to demographic factors represents a high socio-economic burden for Europe. The direct health costs increased by 50 during the last decade and reached euro1.526bn in 2017, a staggering 9.6  of Europes GDP. As current therapies rarely cure, but merely fight symptoms, never-ending treatment is required, which means diminished quality-of-life, adverse effects and soaring cost for society. There is a high need to reach sustainable improvement for patients or even to cure them of chronic diseases - in other words, to disrupt the paradigm of "treating symptoms" with "restoring health". Advanced Therapies are the game changers that open up transforming therapeutic opportunities. For genetic diseases, immune diseases, cancer and tissue injury potential cures through Advanced Therapies exist - it is reality, not fiction. Some products are already on the market, mostly for rare diseases which means only a few thousand patients worldwide have benefitted from Advanced Therapies until now. At the advent of such a trailblazing change, obstacles and roadblocks abound. To make the disruptive promise of Advanced Therapies to cure chronic diseases a reality and to make Advanced Therapies accessible as standard-of-care for every European patient in need, RESTORE envisage to establish a sustainable pan-European ecosystem integrating transdisciplinary research, clinics, patients, and industry. Please join us for more details. RESTORE is funded by the EU for the preparatory phase of the large-scale research initiative h2020, No 820292.Our main objective is to develop safe medical products for a minimally-invasive surgical intervention that fulfill all requirements by the new Medical Device Regulation MDR. The secondary objective is to minimize the time to market by implementing standard conforming processes in a minimalistic way so they can be operated by only a handful of people. The twist in the story is that one of our products of our system will be individualized in the operating theater. While basically everybody is forced to operate in a "learning by doing" mode regarding MDR compliance, we do rely on external trainings provided by notified bodies and other sources like websites, webinars, and dedicated conferences to learn more about how to fulfil regulatory requirements. Additionally, we are partnering with our strategic investor a manufacturer of class-III implantable active devices to receive trainings and advice regarding regulatory affairs. However, it has to be stressed that one has to consider all parts of the MDR, especially the annexes and the applicable norms. Here we present our technical approach and our strategy on how we aim full MDR compliance. We established a completely digital workflow for all our documents, including cryptographic signatures, version history, reviews, mainly based on open source tools. This allows us to handle the document management very swiftly. We have one person dedicated to our quality management system. The risk management is integrated as a cross-cutting-concern into all the product development processes. The main challenge in the planning and founding phase of OtoJig GmbH was and still is to estimate what has to be done, which norms do apply, and how much effort time and money it is to perform the steps and create all the required documents. A difficulty is that contradicting statements between the MDR and other applicable norms are still under interpretation and public discussion.Objectives: The aim of this work is to study the influence of person physiology and anatomy, which include postoperative edema, movement and breathing of a patient, on the results of designing of inductive powering unit IPU for VAD and to propose design recommendations. Methods: A characteristic feature of the IPU is the misalignment of the coils. The stability of the VAD power supply especially with continuous powering is very important. Therefore, it is necessary to design an IPU tolerant to coil misalignment. The procedure of geometrical optimization for increasing the stability of energy transfer of IPU was performed for 3 axial distances between coils d 8, 10, 12 mm, which is a typical misalignment with a postoperative edema. The lateral misalignment of the coils reached the value of the outer radius of the receiving coil 35 mm, which characterizes the patient movement or breathing. The operating frequency was taken 1 MHz, and the output power of the system was 10 W. The power drop was within 10. The results were verified by numerical simulation in MATLAB and PSpice. Results: It was found that with a decrease of d from 12 to 10 and 8 mm, the optimal outer radius of the transmitting coil increases from 53.9 to 54.6 and 55.7 mm. Turn pitch in the transmitting coil increases from 4.9 to 5.2 and 5.3 mm. For a receiving coil, turns pitch is increasing from 3 to 3.2 and 3.6, respectively. For all cases of d in each coil there are 11 turns except for d = 8 mm, where there are 10 turns in the receiving coil. It is recommended to design coils with a slightly larger coils turns pitch and the outer radius of the transmitting coil because of disappearance of postoperative edema. Discussion: In this work, the influence of person physiology and anatomy on the design of IPU for VAD was investigated, and recommendations for designing were given.Objectives: Membrane oxygenators are an indispensable part of critical care medicine. Though necessary to supply sufficient gas exchange, the high intrinsic surface introduced by the hollow fiber packing has serious side effects on blood platelet parameters. To reduce these side effects the membrane surface must be minimized and gas exchange improved. Methods: Computational fluid dynamics CFD can support oxygenator optimization and supplement experimental data by delivering a spatial and temporal resolution of the gas exchange. While current research mostly focusses on the gas transport in the blood flow, this work presents a fully resolved CFD approach including transmembrane transport as well as convective and diffusive blood gas transport on shell- and lumen-side of the hollow fibers. Results: CO2 transport in a packing segment of a prototype hollow fiber module was fully resolved and simulated utilizing an inhouse solver membraneFoam based on the open source CFD code OpenFOAM R. Simulation results show a CO2 partial pressure decline from 50 to 15 mmHg in the laminar boundary layer and an additional drop of 12 mmHg at the selective membrane surface. Boundary conditions for the gas transport simulations were computed by blood flow simulations of the whole module. Simulation results were compared to in vitro tests comprising measurements of CO2 exchange performance and blood side pressure drop of the prototype module. Discussion: Flow simulations predict the experimentally determined pressure drop of 68 mmHg at blood flow rates of 1280 mL/min accurately. The specific CO2 exchange rate of 220 mL STP/min/m2 is overpredicted due to the reduction of the whole packing to an idealized packing segment. Nevertheless, CFD allows for a structured optimization of membrane oxygenators as design changes can be efficiently investigated.Objectives: Anticoagulation therapy in LVAD patients is essential to reduce hemocompatibility related adverse events HRAE. Phenprocoumon dose must be adapted and monitored by INR point-of-care-testing POCT in outpatients. The study aims to determine if the frequency of INR POCT in LVAD outpatients has an influence on the quality of anticoagulation therapy, HRAE and clinical outcomes. Methods: This retrospective, pseudo-randomized study included n=48 patients who received an LVAD implantation HMII, HM3 and HVAD between Jan. 2012 and Oct. 2016. Based on the frequency of weekly INR POCT, we compared a daily n=36 and a 3x/week n=12 group, specifically the 1-year anticoagulation quality  of INR Tests in Range as well as clinical outcomes, readmissions and HRAE using Kaplan-Meier curves. Readmission profiles and outcomes in three groups, based on the achieved quality of anticoagulation  of INR Tests in Range ranging from 0-60 poor, 60-70 acceptable, 70-100 well controlled were compared. Results: Daily and 3x/week groups were similar in demographic and pre-operative risk factors, INR target 2.0-3.0, p=0.27 and Aspirin daily doses p=0.29. Freedom from any HRAE 38.9 vs. 25.0, p=0.44, any readmission 72.2 vs. 75.0, p=0.97 and 1-year survival 91.7 vs. 91.7, p=0.98 were comparable in both groups. The  of INR Tests in Range was significantly higher with the daily self-assessments 73.5 vs. 68.4, p=0.006. Freedom from any neurological event 91.7 vs. 75.0, p=0.14 was n.s. higher in the daily POCT group. Well vs. poorly controlled INR POCT patients had a significant higher freedom from any neurological event 96.0 vs 69.2, p=0.024 as well as hemorrhagic strokes 100 vs. 76.9, p=0.011. Discussion: Well controlled anticoagulation of LVAD outpatients results in less neurological events including hemorrhagic stroke. Daily INR POCT and subsequent dose adjustment of vitamin-K antagonists result in a better quality of anticoagulation than 3x/week checks.Objectives: Spectroscopic method is widely used for non-invasive blood glucose BG measurement. Despite the progress in implementation of transmission NIR-spectroscopic method, applicable mostly for earlap measurements, research of non-transmission methods allows for the exapnsion of spectroscopy range of use. The aim of research is to estimate the penetration depth for 1600 nm radiation using reflection NIR-spectroscopy. Sufficiency of penetration depth on this wavelength would allow for the use of a mathematical model implemented in the transmission method. Methods: The developed experimental setup includes a semiconductor laser with wavelength of 1600 nm, two photodiodes, reflecting surface, control unit and power supply. For determining the position of optical elements a MATLAB program was developed, which calculated efficient detected radiation intensity depending on distances between the photodiode and the laser and between the laser and reflective surface. Scheme for measuring BG by reflection NIR-spectroscopy partially repeats the scheme used for transmission method. The main differences are in location of photodiodes on the same plane with the radiation source, while their optical axes are co-aligned and parallel to each other, and the reflecting surface tightly abuts the back wall of analytical cell with test solution. Results: The optimal distance between the photodiode and the laser is 5 mm, and between the laser and the reflective surface is 20 mm. This configuration allows intensity of the reflected radiation at about 20 of incident radiation to be obtained. Taking into account the permissible radiation density for skin, this value is enough for skin probing to a depth of 1.5 mm. Discussion: Reflection NIR-spectroscopy is promising method for non-invasive BG measurement. Research of transmission method has shown that the penetration depth of 1.5 mm is enough for receiving information about BG. Thereby mathematical apparatus applied for transmission NIR-spectroscopy can be used and similar error less than 20 can be expected.Objectives: Proper anatomical fitting of implants is crucial for a successful clinical outcome. However, every patients anatomy is unique and there is a wide variety in the anatomical and morphological characteristics among individuals. Virtual fitting based on imaging data of a high number of patients has crucial benefits compared to conventional approaches during the design process. Methods: Population based engineering is a method that enables a virtual implantation combined with iterative design optimization based on 3D anatomical models created from imaging data of a high cohort of patients. This approach was successfully used during the design process of a novel inflow cannula for a Ventricular Assist Device and a Total Artificial Heart for maximizing the number of treatable patients. The objective was to create a design that contains all components, but which at the same time works for a wide variety of different body types and sizes. Results: The virtual studies have proven to give results that may not have been possible with conventional approaches. Compared with cadaver studies, this approach was a more accurate and economical way for determining the device fit and identifying areas for improvement. Clinical trials and in vivo studies of the devices have shown positive outcomes. Virtual fitting was able to reduce the risk of inflow obstruction, device-vessel misalignment, unexpected variabilities in the patients anatomy and improper patient selection due to anatomical constraints. Discussion: Population based engineering is a cost-effective solution for including a large number of patients and anatomy variations in the design process. Additionally, this approach can be used for regulatory submission, e.g. to determine and justify anatomical and morphological eligibility or exclusion criteria for proper patient selection and/or the correct implant size. This is especially important in consideration of the future requirements of the Medical Device Regulation MDR for patient-specific implants.Objectives: None Alcoholic Fatty Liver disorders NAFLDis a complex systemic disorder becauseit is associated with clinical states such as obesity, insulin resistance, and type 2 diabetes thus involving both liver and pancreas. In particular, pathological pancreas such as in diabetic patients, in non-alcoholic fatty pancreas disorders patients led to mis control of insulin secretion the insulin modulates the lipid accumulation in liver. Methods: Organ on chip approaches is one way to mimic human physiology. In this paper, we will present the development of a liver, pancreas and liver pancreas co-culture model to simulate the interaction between both organs. Results: The morphological analysis confirmed the rat hepatocytes and the rat Langehrans islets were cultivated successfully after the extraction for 7 days. The tissues functionality was confirmed by the production of albumin in the liver on chip models and by the insulin secretion in the pancreas biochips. The RTqPCR analysis confirmed that the pancreas on chip culture contribute to maintain high level mRNA of genes related to glucose insulin homeostasis when compared to Petri control. Then, the GLP1 drug contribute to increase the insulin metabolism in pancreas on chip. In liver pancreas co-culture, we found that the presence of pancreas islet contributed to modify the mRNA levels of glucose-insulin homeostasis related genes in the hepatocytes. It also contributed to increase the insulin production when compared to pancreas biochip control. Discussion: Those results demonstrated the potential of our liver pancreas model to be upgraded to a complex disease model.Objectives: We are developing a novel autologous tissue-engineered heart valve with a unique in-body tissue engineering. This is expected to be a viable bioprosthesis with better biocompatibility. In this study, we developed a conduit-type valve without any foreign materials and tested the feasibility and long-term availability in large animal experiments. Methods: We created plastic molds for Biovalves with a 3D printer easily and quickly considering the recipient character. We embedded them in the subcutaneous spaces of adult goats for about 2 months. After extracting the molds with the tissue en-block and removing the plastic molds only, Biovalves with tri-leaflets similar to those of the native valves were constituted from completely autologous connective tissues and fibroblasts. Total 21 conduit-type Biovalves were implanted in the apico-aortic bypass or the pulmonary artery of goats, 8 and 13, respectively. No anticoagurants were used after implantation. Results: The valves were successfully implanted and showed smooth movement of the leaflets with a little regurgitation in angiogram, and the maximum duration reached to 3 years 7 months. Histological examination of the Biovalves showed the autologous cells covering the laminar surface of the valve leaflets as the endothelium and also migrating into the leaflet body to construct characteristic tissues like native leaflets. Discussion: The valves have a potential to be used for viable bioprosthetic valves and to keep better function and biocompatibility longer than current ones.Objectives: The main purpose of this study is to analyze the correlation between different types of dnDSA and AMR after renal transplantation. Methods: We retrospectively analyzed the patients after renal transplantation from January 2002 to March 2017 in our Center. A total of 47 patients with positive PRA and confirmed as dnDSA were included, which were grouped according to the DSA binding to C1q, C3d and subtypes of IgG. Patients were divided into AMR and non-AMR groups according to the pathology of graft biopsy. Results: The pre-transplantation dialysis time of the non-AMR group was longer than that of the AMR group in 47 patients with dnDSA positive 35.4 33.2 vs 9.8 10.5, p=0.014. C1q-binding dnDSA had no significant effect on the graft survival after operation and biopsy. Among 47 patients with dnDSA positive, C3d-binding DSA group had lower graft survival time p=0.009, higher HLA-DP mismatch 0.1 /- 0.3, p=0.043 and higher percentage of pericapillary C4d deposition p=0.042, with statistical significance. The graft survival rate of IgG3 subtype negative patients was higher than that of IgG3 subtype positive patients p=0.003. Cox analysis found that the risk factors for graft survival included IgG3 OR = 46.877, 95 CI = 4.211-521.830, P = 0.002, HLA-DR mismatch OR = 0.103, 95 CI = 0.021-0.496, P = 0.005, proteinuriuria at biopsy OR = 2.097, 95 CI = 1.184-3.713, p=0.024 and creatinine at biopsy OR = 1.004, 95 CI = 1.001-1.007, P = 1.007 Discussion: Single-center study showed that there was no significant correlation between the incidence of AMR and different types of DSA. The accurate HLA-DR typing should be emphasized during transplantation. The monitoring of specific types of dnDSA will help us to take interventions and thus contribute to the survival of transplanted kidneys.Objectives: Rotary blood pumps RBPs are successfully used in high-risk treatments, but clot formation still threatens their long-term application. To reduce the risk of clot formation, in-vitro thrombogenicity testing could help to improve RBP design, as several studies have shown. Those studies were able to simulate in-vitro clot formation in RBPs, but they had limitations regarding a proper quantification of their thrombotic impact. In this study, we assessed if thromboelastometric analyses TEM are feasible to quantify the thrombotic impact of RBPs in-vitro. Methods: Five RBPs n = 5 were placed into simple pump circuits that were built of silicone tubes and reservoirs. Each circuit was filled with 150 ml of slightly heparinized porcine blood one donor pig per circuit and the pumps were brought into operation. The pumps operated until a drastic drop in volume flow indicated thrombus formation. We carefully cleaned the RBPs from blood and documented any found thrombus. Prior to this, blood samples were taken at certain time points during pump operation. The blood samples were then analyzed by TEM. Results: TEM measurements showed a decrease in clotting time CT over the duration of the test, which indicates an ongoing increase in the activation of the coagulation system caused by the pump. Correspondingly, RBPs revealed visible blood clots at high-risk thrombus formation spots. Discussion: The decrease in CT over time corresponding to clot formation in the RBPs shows that TEM is able to detect and quantify the thrombotic impact of RBPs in-vitro. Thus, TEM could be used in future studies to compare the thrombogenicity of different RBPs by performing comparison tests similar to hemolysis testing. This could lead towards a standardization of in-vitro thrombogenicity testing of RBPs.Objectives: Cryopreservation of tissue-engineered constructs TECs is very important to provide such ready-to-use products for regenerative medicine and clinical application upon demand. Although cryopreservation of isolated cells seems to be well established, there are still a number of challenges associated with the cryopreservation of native and artificial tissues due to adherent cell state, limited heat and mass transfer as well as inadequate cryopreservation protocols. Here, we aim at developing an approach for efficient cryopreservation of electrospun TECs based on multipotent stromal cells MSCs. Methods: Blend electrospun fibre mats fibre diameter 0.8 /- 0.2 mu m, thickness 100 /- 10 mu m were produced from polycaprolactone and polylactic acid PCL-PLA, ratio 100:50 using electrospinning. The fibre mats diameter 16 mm were UV sterilised and seeded with MSCs 5x104 cells/cm2. The cells were cultivated on fibre mats for 7 days under static conditions and then frozen using 1 K/min cooling rate in a controlled rate freezer with different formulations of cryoprotective agents CPAs, such as dimethyl sulfoxide DMSO and its combination with sucrose with and without pre-culture with sucrose for 24 h. The viability of cells growing on fibre mats was monitored for 2 weeks after seeding and 24 h after thawing. Results: The results indicate that PCL-PLA fibre mats are biocompatible with MSCs viability higher than 82. Pre-culture with sucrose before freezing as well as its addition to DMSO-containing freezing medium significantly improved cell viability after thawing. Moreover, duration of equilibration of cell-seeded fibre mats with the CPAs before freezing affected cell viability post-thaw. Discussion: We showed that it is feasible to effectively cryopreserve electrospun TECs using controlled technological steps. This work could serve as a solid background for further development of efficient cryopreservation methods for biobanking of electrospun constructs for vascular or corneal tissue engineering.Objectives: There have been discussions on effects of pleural effusion on the breathing muscles dynamics. In particular, hemidiaphragm inversion influence on the pleural pressure PPL and ventilation parameters has not been precisely determined. The aim of this study was to analyze changes in PPL and ventilation parameters in patients undergoing therapeutic thoracentesis TT. Particular attention has been paid to inversion of the hemidiaphragm caused by large one-sided pleural effusion. The analysis was based on virtual experiments performed on an artificial cardio-respiratory patient AP. Methods: TT was simulated on AP, which consists of several cooperating models of the respiratory system mechanics, gas transport and exchange, and circulation. Three scenarios were considered: a proper work of the diaphragm, b flattening and fixation of the hemidiaphragm due to the large amount of fluid, c paradoxical excursion of the inverted hemidiaphragm. Results: Simulations showed that during progressive pleural fluid withdrawal significant changes in the course of PPL were observed, particularly in scenarios b and c. Paradoxical excursion of the inverted hemidiaphragm significantly influenced the alveolar oxygen partial pressure PAO2 due to a kind of pendelluft: e.g., air flows out from the corresponding lung during inspiration and thus it flows to the lung in the hemithorax without pleural effusion. Discussion: Flattening and inversion of the hemidiaphragm have an influence on several physiological factors of which PPL and PAO2 seem to be the most important. Hence, TT may improve pulmonary system function particularly in patients with inverted hemidiaphragm.Objectives: Bone loss at implantation sites on oral cavities is a major problem for dental surgeons; in order to combat this issue, we developed 3 types of colagen biomaterial blends: chondrohitin sulfate, carbon nanotubes, and electric stimulated. Methods: Bovine colagen type I was dissoluted at 4 in formic acid 0,1M Synth - Brazil and divided in 4 groups: A- with Chondroitin Sulphate Sigma Aldrich; B - carbon nanotubes suspention Sigma Aldrich; C- eletric field ; D - control group. All of the samples were crosslinked with NHS N-hydroxysuccinimide esters Thermo Scientific-USA and freeze dried at a LH2000 equipment Terroni - Brazil. The samples were analized by: SEM; EDS; XPS; Bartha respirometry and FET. Results: All of the samples have the same macroscopic morphology. The SEM of the group submitted to electric field shows organization of the colagen fibers. The EDS shows atomic content of carbon, oxygen, and nitrogen with other substances Discussion: The absence of contaminants within the samples and the increase in stiffness exhibit the compatibility of this material for use in bone augmentation in implantology. The material is shown to be not toxic, however more tests should be conducted prior to human use.Objectives: The optimization of the long term storage of cells and tissues is a challenging process with many variables but one factor is often overlooked: the freezing device itself. There are freezing containers that have to be placed in a -80 degrees C freezer and the manufacturer promises a cooling rate of 1 K/min. On the other hand there are controlled rate freezers where cooling rates from 0.1 K/min up to 50 K/min are promised. In this study we compared two commercially available freezing containers and four controlled rate freezers with respect to their functioning principle. Furthermore, we investigated the accuracy of the adjusted cooling rate and the nucleation temperature of the samples. Methods: Seven 1.5 ml cryovials filled with 1 ml 0.9 w/v sodium chloride solution were dispersed evenly over the rack of each freezing device. Constantan thermocouples type T connected to a RedLab device were placed in the middle of the solution of each cryovial to record temperatures every second with the respective software. The cooling rates were calculated from the melting point -0.6 degrees C of the solution to -30 degrees C with n=3. For all freezing devices a cooling rate of 1 K/min was set according to the manufacturers guidelines. Results: The freezing containers had sample cooling rates between 0.5 and 0.8 K/min. All controlled rate freezers showed cooling rates similar to the programmed cooling rate of 1 K/min. Higher cooling rates resulted in increasing deviations between programmed and measured cooling rates. The nucleation temperatures of the samples in the freezing containers were mainly between 0 degrees C and -6 degrees C. However, the controlled rate freezers showed nucleation temperatures mainly in the range of -6 degrees C to -12 degrees C. Discussion: Varying definitions from each manufacturer resulted in the observed cooling rate differences of the freezing containers. Limited heat transfer accounted for the increased cooling rate deviation recorded in higher programmed cooling rates.Objectives: Patients using ventricular assist devices VADs still suffer from adverse events such as pump malfunctions or thromboembolic events. This can be caused by thrombi that have formed inside the pump pump thrombus. Therefore, there is a great need to prevent such adverse events through engineering measures in the early development stage of blood pumps. Currently, a numerical model to predict thrombus sites inside VADs is still missing and the risk can only be assessed with in vitro experiments in the late development stage. A recently found model for thrombus formation at high shear rates that was derived from simple stenosis experiments promises great potential in the application in computational simulations CFD of VADs. Methods: In this study, advanced high resolution URANS simulations of rotational blood pumps were conducted with the flow solver of StarCCM Siemens at an operating point of 5L/min at 75 mmHg. A k- omega SST turbulence model and the sliding mesh method was applied. The existing model for predicting high shear thrombus formation was applied and compared with observations from explanted pumps. Based on these results a modification of that model is proposed that suggests a wall normal transport due to the change of shear rate in flow direction. This modification was realized in the CFD by applying a correlation between the computed direction of the pressure gradient and the flow direction. Results: The application of the model shows that thrombus sites are overpredicted in rotational blood pumps when compared with experimental results or observations from explanted pumps. However, with a modification of the model which proposes that the influence is a wall normal transport due to a change in shear rate, a good agreement was found. Discussion: Since there is little data available in the literature that shows the position of pump thrombi the validity of these models remains unclear and has to be experimentally evaluated further.Objectives: Cryptogenic stroke is the cause of 40 of ischaemic acute cerebrovascular events. Study aim was to evaluate the recurrence of ischaemic cerebrovascular events in patients successfully treated by percutaneous closure of patent foramen ovale PFO. Methods: From February 2004 to January 2019, 314 symptomatic 243 stroke, 71 TIA patients, 153 M 161 F; mean age 41 yrs, range 10-69 underwent percutaneous closure of PFO. 151 patients/314 48 had concomitant migraine, 90 60 with aura. 7 different occluder devices were implanted by transesophageal echocardiography, for a total of 317 implants. During follow-up all patients underwent clinical Rankin modified scale and quality of life SF36 evaluations, transcranial Doppler TCD, trans-thoracic echocardiography, and MRI. Cerebral and angio-MR assessed the degree of lesions by quantitative and qualitative comparative analysis performed before and after treatment. Sizes of lesions were measured by manual segmentation on the axial, coronal and sagittal images acquired. Results: Successful device deployment was achieved in 99 of pateints; patients were discharged home within 3 days. Follow-up was 100 complete median 55.4, range 1-178 months. At 6 months, Rankin scale was 0 p<0.0001 in 230 patients 95 affected by stroke and 10 patients reached score 1. Quality of life improved significantly P<0.0001. In 101/151 patients 67 with migraine, intensity and frequency of attacks significantly decreased P<0.0001. TCD showed residual microembolic signals in 10 patients, 3 patients required secondary successful treatment for an associate defect. TTE after 1, 3, 6, 12 months and once a year for 5 years showed optimal sealing of all devices without signs of erosion, incomplete closure and thrombus. In 265 patients cerebral MRI showed no new lesions at 2 years. Discussion: Our 15 year experience suggests that percutaneous treatment of PFO is safe and beneficial for secondary prevention of recurrence of acute cerebrovascular events irrespective of the device used.Objectives: Endovascular stent grafting is the standard treatment for patients with acute traumatic aortic rupture with extensive associated lesions. Very little long term information is available in large series. Methods: From March 1999 to September 2018, 83 patients 72 M and 11 F; mean age: 37.25 /- 13.46;range 16 to 69 admitted with acute or chronic traumatic aortic lesions underwent endovascular repair. 60 cases had acute traumatic aortic rupture, due to road accidents in 69 patients and accidental falls in 3 patients. All procedures were carried out in the angiography suite. Left subclavian artery was always identified. Patients were followed-up in the out-patient clinic and by yearly angio CT-scan with regard to survival and complications. The follow-up was 100 complete. Results: Endovascular stent-graft treatment was successful in all cases of acute or chronic aortic injury. No post-operative paraplegia occurred. Control angiography showed optimal sealing and complete exclusion of the pseudo-aneurysm from blood flow with no primary endoleak. Patients underwent treatment of all associated lesions later on during hospital stay. Two patients died in the hospital: 1 patient of cerebral haemorrhage and 1 patient of sepsis. During the follow-up 5 patients died survival: 91.4 for causes unrelated to the aortic procedure. no cases of perigraft leakage or aortic disruption were detected. During follow-up 1 patient had a steal syndrome and 1 patient paraplegia due to the covering of the left subclavian artery by endovascular graft. 4 years after treatment 1 patient had inner thrombosis of the graft developing a gradient; a new endovascular stent graft was deployed successfully. Freedom from complications was 92.3. Discussion: The outcomes over 20 years of follow-up proves that endovascular stent graft repair is the first choice treatment in patients with traumatic aortic injuries. Our experience demonstrates the feasibility and safety of endovascular treatment including patients with extensive associated injuries.Objectives: In vitro simulation of an in vivo environment for human stem cell research is crucial for any kind of biomedical purpose. While important factors like 3D-cultivation and physiological oxygen concentrations gain more attention in the scientific community, we still lack reliable methods to visualize the hypoxic response of cells in 2D and 3D in vitro systems. In this study we present human adipose-derived MSCs, modified with a genetically encoded hypoxia-sensor. Methods: We used a lentiviral system to stably integrate the genetic construct into the chromosomal DNA of mesenchymal stem cells. To investigate the features of this novel biosensor we cultivated our cells in a 2D- and 3D-environment under various oxygen levels and evaluated the outset of biosensor-fluorescence via microscopy and flow cytometry. Results: The hypoxia sensor was successfully integrated in mesenchymal stem cells and could easily be induced by cultivating the cells in a hypoxic condition. MSCs, modified with hypoxia biosensors could be cultivated up to passage 20. Trilineage differentiation of MSCs adipogenous, osteogenous and chondrogenouse differentiation was also preserved by the cells after transfection. Using these cells we could monitor which 3D-cultivation conditions lead to hypoxic response of MSCs. Discussion: Our findings can help to improve our understanding of the influence of cultivation conditions on in situ oxygen concentrations. Moreover, by choosing the right 3D cultivation system, MSC can be cultivated in vitro under physiological hypoxic conditions.Objectives: Heart failure with reduced Ejection Fraction HFrEF is a progressive disease with a low 5-year survival of <50, which affects 23 million people worldwide. It is characterized by adverse remodeling of the left ventricle dilated cardiomyopathy due to an increase in filling pressures and myocardial wall stress. Pharmacological treatment and cardiac resynchronisation therapy have proven beneficial for survival. For patients with end-stage heart failure, a heart transplant or Left Ventricular Assist Device can be considered. A shortage of donors, patient selection and major downsides such as invasiveness and driveline infections limit the use of these treatments. Research has shown a 13 decrease in mortality for every 5 increase in left ventricular ejection fraction. Therefore, we developed a smart memory alloy configuration in order to increase the ejection fraction and obtain an increase of 3,5 in a bench model. To cope with ongoing left ventricular dilatation and rise in wall stress, this should be combined with adjustable and measurable ventricular restraint therapy. Our first aim is to measure local wall stress during a full cardiac cycle. Next, we aim to develop a mathematical model of the left ventricle to characterize the left ventricle in HFrEF patients. Methods: We will characterize in vivo wall stresses during the full cardiac cycle using Transesophageal Echocardiography and a left ventricular pressure catheter in 10 patients undergoing cardiac surgery for heart failure. With these parameters, we will develop a simplified mathematical model of the left ventricle and we will improve our bench model for experimental testing. Results: This research will provide a characterization of the weakened left ventricular wall and the determination of optimal smart material properties and configuration of the cardiac assist device. Discussion: With this information, a patient-specific HFrEF treatment device will be developed combining active cardiac support and restraint therapy.Objectives: Clinical data on the hemocompatibility of membranes used in double filtration lipoprotein apheresis LP is virtually unavailable. The present trial compared the hemocompatibility of a recently introduced polyethersulfone PES based plasma fractionator membrane, FractioPES R 200, to an ethylene-vinyl alcohol copolymer EVAL membrane during LP. Methods: In a prospective, randomized, controlled, crossover trial, eight patients on routine LP were subjected to one treatment with PES plasma 0.6 m2, 3M PlasCure R 0.6 and fractionation 1.9 m2, 3M SelectiCure R H19 membranes and one control treatment using a set of EVAL membranes 0.5 m2, Asahi Plasmaflo OP-05W; 2.0 m2, Asahi Cascadeflo EC-50W. Intraindividual treatment conditions were kept identical. At defined times, samples were drawn at different sites of the extracorporeal blood and plasma circuit to measure white blood cell WBC and platelet PC counts, complement factor C5a and thrombin-anti-thrombin III ATIII. Results: With a nadir at 25 min, WBC in EVAL decreased to 34  of baseline vs. 64  at 20 min in PES P<0.001. PC only marginally decreased over time with both membrane types. Maximum C5a in venous blood was 30.0 /- 11.2 mu g/L at 30 min with EVAL and 14.0 /- 12.8 mu g/L at 25 min with PES P<0.001. Compared to PES 23.3 /- 15.2 at 5 min and 16.9 /- 12.3 at 20 min, resp., highest C5a concentrations were found in plasma after the EVAL plasma 56.1 /- 22.0 mu g/L at 10 min; P<0.001 and fractionation filters 50.6 /- 19.4 mu g/L at 30 min; P<0.001. ATIII levels did not rise until the end of the treatment without differences between membranes. Regarding Lpa, LDL and HDL removal, both membrane sets performed equally PES, 69.8 /- 5.7, 64.9 /- 8.8, and 17.4 /- 13.6 , resp., vs. EVAL, 69.5 /- 6.0, 65.2 /- 6.9, and 18.2 /- 7.3 , resp.. Discussion: Compared to EVAL, PES membranes are more beneficial with respect to the classical hemocompatibility of extracorporeal treatment procedures, namely leukocyte and complement system activation.Objectives: Mechanical circulatory support MCS has become a standard therapy for adult end-stage heart failure patients. For pediatric patients, technological development lags behind with no currently approved implantable rotary blood pump. As an alternative, the HeartWare HVAD, originally designed for adults, is increasingly used in pediatric patients. The aim of this multicenter study was to assess in-silico, in-vitro and in-vivo the blood trauma potential of this pump in pediatric application. Methods: Blood trauma potential of the HVAD was investigated in-silico and in-vitro at an adult and pediatric operating point 5L/min and 2.5L/min at 2800rpm and 2200rpm, respectively. The flow was simulated by computational fluid dynamics and analyzed regarding flow structures, shear stresses and washout. Hemolysis was assessed with pumps circulating bovine blood in a temperate flow circuit. Clinical outcome and indicators for in-vivo blood trauma were investigated retrospectively in 14 pediatric HVAD patients age 11.3 /- 4.8years. Results: In the pediatric conditions, simulations predicted elevated mechanical stress profile below 50mPa, more stagnant flow field, with longer washout times within the pump. In-vitro measurements revealed an increased normalized index of hemolysis NIH = 17.5 mg/100L vs. 8.2 mg/100L, p=0.0021. In the retrospective in vivo analysis, LDH and D-Dimer values were 1.5 and 3-fold elevated, respectively, compared to adult HVAD patients. Major bleedings were observed in 42.9, suspected pump thrombosis and neurologic dysfunction in 14.3 of all patients. Discussion: The HVAD, operated at lower speeds and flows, induces elevated blood trauma. These results highlight the need for specifically adapted ventricular assist devices, optimized for the pediatric population. Further studies are required to assess the clinical implications of these findings.Objectives: Native arteriovenous fistula AVF is the preferred vascular access for hemodialysis, but it still has high rate of failure due to stenosis formation. Convincing evidence supports a key role of local hemodynamics in vascular remodeling, suggesting that disturbed flow conditions may be related to stenosis development. The purpose of our investigation was to explore the feasibility of coupling non-contrast enhanced MRI and high-resolution computational fluid dynamics HR-CFD to relate morphological vessel changes to local hemodynamics in AVF over time. Methods: We acquired non-contrast enhanced 3D fast spin echo MRI CUBE T1 at 1 and 6 weeks, 6 months and 1 year after radio-cephalic AVF creation in one patient. We generated 3D models and evaluated lumen cross-sectional area changes over time. We perfomed CFD simulations using pimpleFoam solver of OpenFoam, prescribing blood flow waveforms derived by Ultrasound examination. We computed the 2 components of the wall shear stress vector over time, namely WSSdir, the component in the mean direction of the WSS vector and WSStr, the transversal component. Results: We observed a dilatation of the vein until 6 months, with a more pronounced increase in the venous outflow as compared to the juxta-anastomotic vein JAV. The increase in veins diameter was then followed by a narrowing of JAV at 1 year after AVF surgery. We found high-frequency fluctuations both for WSSdir and WSStr components, in different locations of the vein, at 6 weeks and 6 months after AVF creation. Oscillations of both components damped at 1 year after AVF creation, as a result of vessel remodeling. Discussion: Optimized CUBE T1, coupled with HR-CFD, allowed a characterization of morphological and hemodynamic changes over time. Our MRI-to-CFD pipeline represents a promising approach to elucidate mechanisms of local vascular remodeling and can be used for clinical investigations aimed at identifying critical hemodynamic factors responsible for AVF failure.Objectives: For 30 years, the mortality rate of patients hospitalized in intensive care unit has been drastically reduced. But an increase in muscle dysfunctions at the end of intensive care stay, leading to long term functional disability was observed at the same time. The physiological mechanism remains poorly understood due to a lack of study tools. The objective of this work is therefore to create a new tool for the tissue construction of an in vitro skeletal muscle. This tool should allow a muscle construction which mimics physiological reality, in order to model the disease more accurately. Also, it should allow mechanical and electrical stimulation in order to simulate the resumption of muscle contraction of patients. Methods: Using sol-gel process, we synthesized a new biomaterial, based on an epoxy organic-inorganic hybrid precursor g-glycidyloxypropyltrimethoxysilane. This biomaterial was deposited as a thin layer spin-coating process of 7 mu m thickness on a silicone membrane suitable to undergo mechanical stretching. The biomaterial was microstructured using the UV laser writing lithography to create a line network. This line network was revealed with a 2-minute isopropanol bath and we obtained lines of 8 mu m thickness spaced of 175 mu m. To ensure a biological environment and a strong adhesion of cells on microstructured silicone support during mechanical stretching, we grafted silylated bioactive peptides using dip-coating process. Results: Muscular stem cells which were isolated from patients quadriceps biopsy were seemed and, by immunofluorescence staining, we observed a growth of muscle fibers along the lines, mimicking the physiological organization of a muscle. Discussion: We were able to model the first stages of a complex muscle organoid in vitro using a new tool manufactured by a fast, simple and reproducible process. With the mechanical and electrical stimulation of this muscle-on-a-chip, this work should allow us to better understand these muscle dysfunctions and find new treatments.Objectives: Cryopreservation of ready-to-use tissue-engineered constructs TECs is a promising strategy which may facilitate their future clinical application. This is very challenging and ambitious task and therefore recent efforts have been focused on developing new cryopreservation strategies for long-term storage of TECs. This work covers some practical considerations for cryopreservation of cell-free and cell-seeded scaffolds vastly differing by structure and composition. Methods: The first test system includes 3D porous collagen-hydroxyapatite HAP scaffolds prepared by freeze-drying and coaxial alginate macrospheres prepared by electrospraying. Samples were frozen at 1 K/min either in a bulk DMSO solution with and without sucrose or after removal of residual solution. After thawing, we evaluated compression collagen-HAP scaffolds and rheological properties coaxial alginate macrospheres of cell-free systems. Viability of mesenchymal stromal cells MSCs within both types of scaffolds was evaluated 24-h post-thaw using live-dead assay. The second test system comprises flat fiber mats produced from polycaprolactone/polylactic acid using electrospinning seeded with CHO cells. This system intends to develop plate electrodes for electroporation of attached cells with non-permeable cryoprotective agents CPAs such as sugars for future cryopreservation applications. Results: All scaffolds were cytocompatible with corresponding cell types. Freezing after removal of residual solution was superior to conventional freezing. Addition of sucrose increased cell viability both scaffold types and improved viscoelastic properties of coaxial macrospheres. Constructed plate electrodes provided good compromise between high cell permeabilisation and viability after electroporation with sucrose at 1.7 kV/cm electric field. Discussion: The findings suggest that it is feasible to cryopreserve cell-free and cell-seeded scaffolds using DMSO and sucrose. As a step further, there are high expectations associated with using electroporation as a mean for intracellular delivery of non-toxic CPAs towards DMSO-free cryopreservation of TECs.Objectives: Tailored Forming is a new manufacturing technology to manufacture solid components out of two or more different metals. The components are joined to a hybrid semi-finished workpiece. Afterwards, a forming process is performed to improve the materials properties. This allows creating hybrid metallic parts that are adjusted to their specific loads and their field of application in comparison to parts made out of monomaterials. The potential use case of Tailored Forming-parts is still being researched. Biomedical implants are a potential field of application. In the presented study, the contact mechanics of two potential concepts for Tailored Forming hip implants were analysed. Methods: Both concepts consist of a magnesium component that should be resorbed in the human body and leads to a better bone growth. A second component in the implants is used to absorb loads. In a numerical analysis the two implants were compared to a conventional implant. For the potential evaluation two load cases "walking" and "walking upstairs" were considered. Results: While one concept leads to higher stresses in the implant, the other concept shows almost similar stress distributions as the conventional implant and has the additional advantage of the better bone growth due to the magnesium component. Discussion: Based on this work further research on different implant concepts has to be made to give a clear statement about the potential of Tailored Forming-Implants.Objectives: In consideration of the recently published implant files, testing of medical devices has gained signifigant importance. Many medical implants are subject to normative testing during their regulatory approval process, such as stents, heart valves and blood pumps. No testing norm or standard, however, exists for regulatory approval of left atrial appendage occlusion devices. Therefore, this study aimed to establish in-vitro bench tests for LAA occlusion LAAo devices and compares the clinically most widely used devices. Methods: Seven different LAA occlusion systems with device diameter ranging between 22 and 34 mm were tested regarding tug force and radial force resulting in a total of 24 devices. Radial force was assessed in a commercially available tester whereas tug force was evaluated in a novel in-vitro test setup consisting of bovine tissue. Results: Significant differences in the mechanical properties of the different devices were observed. Radial force ranged between 8.6 N at maximum compression for the LAmbre 2228 device and 0.1 N for the Occlutech 27 mm implant at minimum compression. A similar variability of mechanical properties was seen in the tug test results. Values ranged from 4.6 N to 0.4 N for the Wavecrest 22 mm and the Occlutech 24 mm device, respectively, at maximum and minimum compression. Discussion: Large variations in mechanical properties were seen between the different devices. The study showed that device stability is more dependent on anchoring structures, such as hooks and barbs, than on radial force. A strong positive correlation between the number of anchoring structures per millimeter circumference of an occluder and its tug force was found r=0.87, p < 0.01. The large variations in mechanical properties aggravate comparison of current LAA occlusion devices which underlines the need for standardized preclinical testing to prompt clinical compatibility.Objectives: Pump thrombosis is a severe adverse event in ventricular assist devices. Current therapy often involves an exchange of the entire pump. Alternative solutions must be explored to reduce surgery numbers. A protocol for in vitro lysis of LVAD specific thrombus with alteplase was established in this study. Methods: A fluid chamber of silicone tubing was filled with isotonic sodium chloride solution. An artificial human thrombus was suspended within. Alteplase was added to the system according to the maximum dose for clinical use, 0.028 mg/ml. Five experiments each were conducted for stasis and fibrin thrombi respectively. Both types were lysis tested for four and 24 hours each. Fluid samples and photographs were taken for dissolution evaluation. Results: Visual inspection of the fibrin thrombus showed clear dissolution. For the stasis thrombus, the 24 h experiment produced no discernible dissolution of the thrombi, while the thrombi investigated over the 4 h period fragmented into small pieces which did not dissolve entirely. D-dimer levels in the fibrin thrombus tests rose steadily through the alteplase treatment for both test durations short test 12800-25600 ng/ml, long test 25600-51200 ng/ml. For the short term stasis thrombus testing, D-dimer values rose similarly to the fibrin thrombus, but levels remained much lower short test 1600-3200 ng/ml, long test >200 ng/ml. Long term stasis thrombus tests showed no significant rise in D-dimer levels. Discussion: Alteplase was successful in dissolving fibrin thrombi in our experimental setup. D-dimer analysis supported the visual impression. The fragmentation of stasis thrombi and D-dimer levels measured may be due to the drugs fibrinolytic effect. The amount of erythrocytes in a stasis thrombus may resist lysis and produce challenges in the clinical application of thrombus lysis treatment. With this setup we were able to examine the reaction of the LVAD thrombi to Alteplase. This information can be used to further optimise clinical lysis therapy.Objectives: Methylprednisolon pulse therapy was used in the initial phase of induction therapy in some patients. This study was conducted in order to examine the short and long term effect of methylprednisolone pulse therapy for the lupus nephritis treated with muti-target therapy. Methods: The retrospective study included 43 patients diagnosed with lupus nephritis receiving muti-target therapy in our center, including prednisone combined with MMF and tacrolimus. 19 patients received methylprednisolon pulse therapy in the initial phase of induction therapy, 24 patients only received prednisone combined with MMF and tacrolimus. The dose of methylprednisolonpulse therapy was intravenous injection 500mgx3 days and the following prednisone dose was 0.8-1.0mg/kg. We compared the primary and secondary outcomes of the two groups. In the patients combined with acute renal injury AKI subgroup, we also compared the primary and secondary outcomes between the two groups. Results: In the primary outcome, the CR rates are similar between the two groups. We did not find significant difference in the total remission rate either complete or partial between the two groups p=0.93. The same is true for the remission time p=0.87. There was no significant difference in the time and number of patients entering hemodialysis between the two groups p=0.65. Although, in the AKI subgroup, there was a significant difference in the serum creatinine between the two groups in 1 month P=0.045. However, there was no significant difference in the following up, and the probability of achieving CR and replase calculated by using the Kaplan - Meier method also had no significant difference. Discussion: Methylprednisolon pulse therapy seemed to have no significant effect on the long-term prognosis of lupus nephritis treated with muti-target therapy and it could not delay the patients entry into hemodialysis. However, it may have had a better effect on the recovery of AKI patients and rapid relief of inflammatory reactions.Objectives: The aim of this study was to evaluate intradialytic on-line multicomponent total removed solutes TRS monitoring in the spent dialysate, as potentially more objective parameter for removal of uremic solute markers urea, indoxyl sulfate IS, and beta 2-Microglobulin B2M compared to relative indices as removal ratio, using a novel miniaturized optical sensor during hemodialysis HD and hemodiafiltration HDF with different settings. Methods: Ten ESKD patients 6 M and 4 F, 60.2 /- 16.8 yrs on chronic HDF were followed during 5 midweek dialysis sessions each length 240min, HD: N=1, Qb=200ml/min, Qd=300ml/min, FX60; HDF: N=4, Qb > 300ml/min, Qd > 500ml/min, Vsubst > 15l, FX800 and FX1000. Spent dialysate from the drain was monitored on-line by a miniaturized sensor prototype Optofluid Technologies Ou, Estonia. For total dialysate collection method as the reference, a spent dialysate sample from the tank at the end of each dialysis session was taken. The concentrations of urea and B2M were determined in the clinical laboratory. Concentration of IS was determined utilizing the HPLC. Based on differences between laboratory and optical sensor TRS values BIAS and standard error SE were calculated. A t-test was used to determine significant differences P <= 0.05. Results: The average TRS values of optical sensor and laboratory were 510 /- 86mM/L and 485 /- 106mM/L for urea, 224 /- 41mg/L and 228 /- 47mg/L for B2M, and 732 /- 371 mu M/L and 719 /- 392 mu M/L for IS, respectively. No statistical differences were found between optical and laboratory TRS values for any uremic solute. The relative BIAS /- SE values of the TRS were -7.99 /- 19.43 for urea, 0.07 /- 15.60 for B2M, and -4.72 /- 13.25 for IS, respectively. Discussion: Novel miniaturized optical sensor successfully carried out intradialytic on-line multicomponent TRS monitoring in the spent dialysate.Objectives: Perioperative tight glycemic control TGC and management is of paramount importance during perioperative treatment of patients with multiorgan failure. Recent results have shown that such interoperative TGC led to an improvement of morbidity and mortality and to a prevention of Surgical site infection SSI in ICU patients. Consequently, a user-friendly device combining glucose analysis and delivery of glucose and insulin should allow for better therapies. Methods: Based on an already existing artificial pancreas device STG-22, a new artificial endocrine pancreas system STG-55 was developed. With a battery incorporated, it uses again a closed loop analytical system. Due to its in-built sensors, the closed-loop system allows for a quick and easy feed-back analysis of patient data through already established algorithms for glucose and insulin infusion. With the help of a disposable modular tubing circuit blood sampling from the patients vein is realized and assessed by an automatically calibrated sensor. The device exhibits a maneuverable weight of only 36 kg and can be set-up within a single hour. Results: Clinical trials with STG-55, in Japan Kochi University Hospital have proven its re not sign liable practicability. Glucose monitoring was stable and results comparable to the analyses of an independent glucose analyzer with a high correlation coefficient. Due to appropriate insulin administration, no hypoglycemia was observed in patients under investigation. Discussion: The advent of new technological tools, such as miniaturized analytical devices in combination with the application of patient-specific precise algorithms allows for a strict and successful therapy without hypoglycemia in perioperative patients. With the new design and concept of STG-55, the set-up time for the artificial pancreas can be reduced down to 30 of preexisting devices, which finally makes a glucose and insulin management in both the interoperative and the ICU user friendly.The generation of complex cellular structures for human application often requires a matrix, defining the three-dimensional shape and mechanical properties of the artificial tissue. The matrix as such is not viable. If this matrix is of human origin, Directive 2004/23/EC in its national version shall apply. In that case only "non-substantial" processing methods may be utilized, which are specified in Regulation 1294/2007/EC. For example, non-cellular substances can be extracted from the tissue, so-called derivatives. These derivatives, if of human origin, are also tissues within the meaning of Directive 2004/23/EC. If these derivatives are further processed, or if the tissue originates from animals, the matrix is classified as a medical device. The Human Tissue Authority describes the delimitation as follows: "Derivatives of non-viable tissues and cells such collagen fillers i.e. collagen extracted from tissues and cells fall under the Medical Device Regulation MDR. Non-viable tissue and cell products such as demineralized bone matrix, or acellular human tissues or tissue matrices, will not be covered by the MDR. They will continue to fall under the Directive 2004/23/EC." The paper deals in more detail with the problem of demarcation and describes the consequences of this in practice. We will discuss the problem of demarcation in more detail as well as describe the consequences in practical implementation.Objectives: In this study, two attempts of new biomaterial development based on the latest results of cell biology and tissue engineering are reported. One is specific cell separation using DNA as linker, and another is novel hybrid materials constructed by decellularized tissues complexed with synthetic polymer. Methods: Antibody was immobilized via desthiobiotin-avidin interaction. Single strand DNA 20mer was chemically immobilized on the surface and then antibody anti-mouse CD45, mCD45 modified with the complementary single strand DNA was immobilized on the surface through DNA duplex formation. Decellularized porcine dermis was prepared and lyophilized. After that, methyl methacrylate MMA monomer were immersed in stages, then polymerization was proceeded. The obtained hybrid material was analyzed by tensile strength and SEM observation. Results: Cells were also adhered on the surface at 37 degrees C and detached by 4 degrees C incubation with remaining cells that interacted with the surface via antibody-antigen interaction. The adhered cells were released by DNase treatment. These results suggest that cells can be selectively captured and collected by using the surface that immobilizes an antibody via dissociation molecules. The monomer absorption depends strongly on the tissue structure. Gradient-type decellularized dermis-polymethyl methacrylate complex was prepared by controlling the permeation time of the methyl methacrylate monomer. The mechanical strength of this complex gradually increased from the dermis side to the polymer side. Discussion: Developing new biomaterials combining new knowledge among different science discipline is an indispensable methodology for the next generation of medical development. The materials presented here are example of these biomaterials, and show the direction of future biomaterial development.Objectives: This investigation discusses the method of electrical stimulation of human umbilical cord-derived mesenchymal stem cells MSC and skin fibroblasts FB growing on substrates composed by carbon nanotubes and organic compounds. Methods: Substrates were produced from a water dispersion of single-walled carbon nanotubes and organic compounds serum albumin, collagen, chitosan and formed by evaporation of the liquid component using infrared laser radiation. Laser exposure stimulated the formation of conductive nanotube clusters responsible for electrical signal transmission to the cells, and organic compounds improved cell adhesion due to the presence of cell membrane receptors sensitive to matrix proteins and aminosugar molecules. An aseptic system for electrical cell stimulation included an electric pulse generator, culture plate with cells with a breadboard and electrodes made of surgical steel with gold plating. Cells were stimulated for 48 hours with pulsed electric signal 60-200 mV amplitude, 1 ms pulse duration, 1 s pause directly inside the CO2-incubator. Results: The effects of electrical stimulation on the cell proliferation were estimated by MTT-test. It was found that both MSC and FB are sensitive to the stimulus, but the most prominent increase of cell proliferation was detected at 80 mV pulse amplitude. In these conditions, a 13 and 16 percent augmentation of cell growth rate was obtained for MSC and FB, respectively. Similar results the formation of denser cell monolayer were achieved by fluorescent microscopy of cell nuclei stained by ethidium bromide and fluorescein diacetate. No pathological changes in cell morphology were observed. Discussion: The obtained results suggest that MSC and FB growing on conductive substrates are sensitive to electrical stimulation. This observation can be explained by the activation of potential-controlled ion channels and the stimulation of metabolic processes that accelerate cell growth.Objectives: The Modular Extracorporeal Lung Assist System ModELAS is a highly compact, modular, fully integrated pump-lung. The device can be configured to provide adult low-flow extracorporeal CO2 removal ECCO2R removal of 30-50 of metabolic CO2, full adult respiratory support or full pediatric respiratory support. The modular design allows for the exchange of the 0.65 m2 adult bundle and the 0.3 m2 pediatric bundle, while all other device components are identical. These studies characterized the in vivo performance of the ModELAS for each application. Methods: In vivo device performance was evaluated in healthy sheep n=4 7-day ECCO2R, n=6 30-day adult ECMO, n=3 7-day pediatric ECMO. Pediatric studies utilized a right atrium to pulmonary artery cannulation strategy while adult studies utilized a dual lumen cannula placed in the jugular vein. Animals were recovered and tethered within a pen. The target blood flow rate was 0.5 L/min for ECCO2R and 2-3 L/min for the ECMO applications. Anticoagulation was achieved via continuous heparin infusion. Animal hemodynamics were measured hourly, and blood chemistry, gas transfer and plasma free hemoglobin PfHb were measured throughout each study. Results: All animals were recovered and 9 animals completed the full study duration. Average CO2 removal during ECCO2R was 74 /- 6 mL/min. Average adult and pediatric oxygenation was 134 /- 10 mL/min and 94 /- 11 mL/min, respectively. Average PfHb was less than 25 mg/dL, with the exception of one study following an acute kidney injury. Early terminations were due to a pulmonary embolism, a hemothorax, an intra-bundle thrombus and a fractured cannula. Typical explanted devices exhibited minimal thrombus. When present, thrombi were primarily located at the bundle inlet surface and impeller pivot. Discussion: These studies are on-going, but demonstrate positive performance of the ModELAS across a clinically relevant respiratory support spectrum.Objectives: Various modes of extracorporeal membrane oxygenation with or without a pump have been used as a bridge to lung transplantation in clinical settings. We have started to develop a large animal model for testing newly developed artificial lungs that can be used as a long-term bridging device. Methods: A total of eight goats weighing 40-55 kg were used. A membrane lung Capiox EBS R, Terumo, Tokyo, Japan accompanied by a centrifugal pump Emersave R, Terumo, Tokyo, Japan was connected to the pulmonary artery PA trunk for outflow, and the left atrial LA appendage for inflow. The PA pressure PAP, systemic arterial pressure AP, pre- and post-oxygenator pressures were monitored. The blood flow of the main PA and the membrane lung were monitored continuously by using an ultrasonic perivascular probe Transonic Systems Inc., Ithaca, NY. The blood flow to the membrane lung was maintained at around 2.0 L/min during the trials. Results: Four out of eight goats did not survive because of initial surgical failures N=2, and introducing air bubbles in the membrane lungs N=2. Other four goats were successfully kept for device testing. Durations of the circuit run were 7, 16, 24, and 48 days with or without heparin infusions according to the experimental protocols. Mean AP and mean PAP at the recovery from the surgery were 81 /- 16 mmHg and 19 /- 2 mmHg, respectively. The PA flow rate was 4.4 /- 0.5 L/min. Discussion: The animal model for testing long-term membrane lung application as PA-to-LA configuration has been established. The model makes it possible for us to assess artificial lungs in vivo setting for long-term with stable blood flow through the device.Objectives: The effectiveness of the inhaled drugs is strictly related to the areas which are reachable by the drug particles. Unless the particles reach the desired part of the bronchial tree, their influence might not meet the required expectations and consequently the disease progress might not be stopped. Therefore, the primary objective of this research was to analyze the airflow patterns and particles deposition of a standard inhaled drug, commonly used during the treatment of asthma, with the use of computational fluid dynamics CFD. The study was devoted to the analysis of the particles diameter influence on their deposition areas within the entire respiratory tract. Methods: Two patient-specific respiratory tract models, for 6 and 12-year-old patients, were reconstructed basing on the DICOM image sets obtained during the Computed Tomography Examinations. The reconstruction of the digital models was performed within the Mimics Research 20.0 program, whereas the meshes were prepared with the use of the Ansys ICEM package. Numerical analyses were carried out as stationary ones with the constant inflow of the particles of various diameters within the range of 1.0-50.0 mu m. Results: It was proven that depending on the particles size, their deposition within the respiratory tract varies significantly. The vast majority of the particles with diameters over 20 mu m are gathered on the walls of the throat, whereas particles of diameters 5-15 mu m are accumulating mainly on the trachea walls, leaving the alveoli insufficiently supplied with the drug particles. Similar phenomena could be observed for both investigated cases of the reconstructed geometries. Discussion: The significant changes of the drug particle distribution resulting from the various diameters of the particles indicate that the inhaled drug size cannot be treated as negligible factor during the drug spraying. Improper distribution of the particles, i.e. their aggregation on the wall of the throat and trachea, might not inhibit the symptoms of the asthma.Objectives: The improvement of immunosuppressive strategies have signifigantly reduced the risk and mortality of cellular rejection after Heart Transplantation HTx. Recently, Humoral Rejection and antibodies have been recognized as a major problem in the management of recipients during the follow-up. Methods: 23 patients out of more than 700 recipients undergoing HTx and 6 undergoing retransplantation required techniques of apheretic medicine to manage Humoral Rejection episodes or to lead to transplant patients with Donor Specific Antibodies DSA. Results: In our experience 4 patients were treated with photopheresis with excellent tolerability, no adverse event and no rejection episodes. Photopheresis was added on top of standard immunosuppressive therapy in patients requiring minimization of calcineurins due to other clinical conditions. 19 patients experiencing humoral rejection were treated alternatively with Plasma Exchange 8 patients or with immunoadsorption. Plasma Exchange was preferred in acute patients requiring removal of inflammatory factors other than antibodies while Immunoabsorption was better tolerated and adopted to reduce the metabolic impact and coagulative disorders when the patient was more stable. Discussion: Apheretic Medicine has opened a new door and represents an efficient weapon in the armamentarium of an HTx center. The complexity of the management of this new diseased in frail patients highlights the need for a multidisciplinary group establishing the right treatment for every patient.Realheart TAH is designed to imitate the construction of the human heart in order to create a blood flow as natural as possible. Realheart TAH consists of two pumps, corresponding to the right and left side of the natural heart. It pumps blood from the left side and right side simultaneously in a synchronized manner. Each pump of the Realheart TAH consists of an atrium and a ventricle, with a valve cylinder in between. When the valve cylinder moves towards ventricle, the blood pumps out from the ventricle creating the systolic phase. The ventricle fills with blood when the valve cylinder moves towards the atrium creating a diastolic phase. The cardiac output of the Realheart TAH depends on the stroke length of the valve cylinder and the frequency of the movement. The left and right pump regulate cardiac output independently in order to coordinate the left/right flow balance of the circulatory system. Similar to natural heart, Realheart TAH is integrated into the circulatory system, which facilitates the uninterrupted blood flow to the atrium, when it pumps the blood out of the ventricle in pulses. The atrium is constructed with a compliance membrane to avoid blood suction from the veins during the systolic phase.Objectives: Pre-vascularized, fibrin-based implants that can be attached to the blood vessels in vivo are a promising solution to faster and more efficient vascularization of implants, to facilitate implant healing and survival. Here, low concentrated up to 10mg/mL fibrin hydrogels have been shown to promote tube formation of endothelial cells in co-culture with ASC. However, these fibrin hydrogels are fairly unstable and higher fibrin concentrations are necessary for implant fabrication. We investigated whether higher concentrated 20-40 mg/mL fibrin hydrogels were suitable for tube formation and if perfusion of the hydrogels could support capillarization of a fibrin-based implant. Methods: Capillary formation of endothelial cells was tested in tube assays using fibrin concentrations of 1-30 mg/ml. Hydrogels were fabricated with cryoprecipitated fibrinogen from blood plasma that had been activated with thrombin. Gels were seeded with EC and ASC co-cultures, supplied with feeding medium containing pro-angiogenic factors, and cultivated for 7 days. Hydrogels with 40 mg/mL fibrin and pre-formed channels were cellularized with EC and ASC and perfused with feeding medium for 10 days under continuous flow. Results: In tube assays, only concentrations of 1 mg/mL and 5 mg/mL fibrin were suitable for endothelial tube formation, which started after 2 days and peaked after 5 days in culture. Tubes were stable for the total assay duration of 7 days. However, under perfusion, fibrin gels of 40 mg/mL showed an endothelialized pore formation throughout the gel. Discussion: High-concentration fibrin hydrogels can successfully be capillarized by EC-ASC co-cultures and perfusion. In static conditions, only low-concentration fibrin hydrogels are suitable for tube formation. As the next step hydrogels of different concentrations are to be connected to fibrin vascular grafts and perfused as a whole implant construct using a custom-made flow chamber as mechanical support.Objectives: The aim of the study was to analyze the effect of the passage number and glucose concentration on viability of the human umbilical vein endothelial cells HUVEC cultured in the culture flasks. Methods: The HUVEC were isolated from umbilical veins obtained by Caesarean section. Cells after the 2nd and 7th passage were seeded with density of 4,600 cells/cm2. Glucose concentration in the culture medium was kept constant at the normal 5 mM or high 20 mM level or was switched every 24 hours from normal to high level or vice versa for 7 or 14 days. The cell viability, mitochondrial membrane potential Delta psi m and activity of the reactive oxygen species ROS were analyzed using the flow cytometry with propidium iodide, JC1 and DCFDA test, respectively. The glucose uptake and lactate production was also monitored. Microscopic visualization of cells was made after fixation in formalin using hematoxylin and eosin staining. Results: After 7 days of culturing in medium with normal, high and variable glucose concentration the cell viability was higher respectively by 5, 4 and 4 in the HUVEC passaged 7 times than in those passaged 2 times. After 14 days the differences were higher and equal to 21, 21 and 31, respectively. After 7 days Delta psi m was higher 1.3, 3.2 and 2.6 times in HUVEC after 7th passage than those after 2nd one, which were cultured in medium with the normal, high and variable glucose level, respectively. The glucose uptake and the lactate production were higher after 14 days and similar for all passages. The ROS activity was lower after 14 days of culturing for all cultures. Microscopic visualization of cells did not show any morphological differences. Discussion: The obtained results show that the properties and phenotype of HUVEC after the second passage are closer to the properties and phenotype of native cells. The HUVEC after the 7th passage are better adapted to in vitro conditions deviating from physiological conditions.One of the pillars for a more patient oriented healthcare is found in patient individualised medicine. Custom-made implants tailored to the individuals specific requirements is one of the key elements herein to obtain better outcomes. However, the production of patient individualised implants is nowadays still often related to a high rate of manual processes, causing both high cost and long delivery periods. Technical innovations play a crucial role in introducing these patient individualised therapy options into clinical prac-tice. Advances in medical imaging and the evolution of new production technologies e.g. additive manufacturing build the basis for a time and cost effective production of implants of lot size one at industrial-scale level. Combining these aspects with digitalisation leads to the potential of a new era of patient individualised implants and appropriate business models. It is crucial to systematically review and adapt the data management and data processing throughout the entire value chain in order to achieve the goal of a more patient individualised care. Different tools are used to de-fine interfaces, actors, data types throughout the entire process and methods need to be established for the data pro-cessing itself. A digital process chain from diagnosis to the actual delivery of a custom-made implant to the hospital including the manufacturing process is established. The underlying principles and assumptions used are exemplarily shown. The digital process chain is used to pave the way from science into a successful business model and company, in order to introduce these patient individualised implants to achieve a better patient care. The use of technical and medical progresses in combination with digitalisation is used to obtain a new generation of patient individualised implants. The way from science to start-up within this field is presented.Objectives: There are currently no available noninvasive methods that allow for the simultaneous and reliable measure of cardiac output CO and pulse wave velocity PWV in the arterial tree. Both of those indices are important biomarkers of cardiovascular status. Methods: Radial pressure profiles were recorded using applanation tonometry in a group of healthy subjects N=18 and hemodialysis patients N=30. Recorded pulse waves were used to estimate patient-specific parameters of the mathematical model describing the blood flow in the system of 55 major arteries. Calibration of the model with patient data allowed to calculate patient-specific CO and PWV between any two points in the arterial tree. In all of the patients and healthy subjects we performed additional measurements of PWV using ECG gated applanation tonometry at two arterial sites SphygmoCor, AtCor Medical and CO using bioimpedance cardiography PhysioFlow, Manatec Biomedical. Results: The model was able to reproduce all of the recorded pressure profiles with high accuracy average relative error < 10. Model-estimated PWV highly correlated with the one measured using SphygmoCor device R = 0.75, p-value < 0.001. Model-predicted CO also correlated with the one measured by the bioimpedance cardiography, but at a lower level R = 0.52, p-value <0.001. Discussion: Our study demonstrates that coupling a mathematical model with a single peripheral pressure profile has the potential to provide information about cardiovascular state that was previously only available when using more complicated methods.Objectives: The purpose of this work is the study of biocompatibility for samples of three-dimensional cellular and tissue engineering constructions for layer-by-layer repair of tissues of heart and blood vessels. Samples components are albumin, collagen and chitosan as matrices and carbon nanotubes as reinforcement. Three-dimensional structures are formed in layers by laser printing. Methods: The biocompatibility of the samples and layers of their components was evaluated after their incubation with endothelial cells for 72 hours. Cell morphology was assessed by fluorescence microscopy and quantitative analysis was performed using MTT-test and real-time cell analysis by means of electrical impedance monitoring. For in vivo studies, samples were implanted in breasts of laboratory birds. Results: It has been established that the layer consisting of albumin and nanotubes supports proliferation of cells the most. Moreover, all layers have surface properties that provide cell adhesion. Real-time cell analysis showed improved cell proliferation compared to control during the entire incubation time, however the greatest differences were observed after 25 hours of incubation. The morphology of the cells on the samples corresponds to the morphology of the control cells. With implantation of samples in experimental animals, the absence of their toxicity was observed. Discussion: Samples of three-dimensional cellular and tissue-engineering structures are biocompatible because of characteristics of their components and suitable surface structure. The obtained samples can be used to repair damages in cardiovascular system. Due to the method of laser printing, samples can be made in any form, taking into account the features of the damaged area of tissues.Objectives: The blood flow rate is the only parameter that can be controlled during the operation of an intracorporeal membrane oxygenator that significantly influences the gas exchange rate. Due to anatomical size restrictions, the catheter is of limited size and the tight packaging does not allow for placement of a dedicated flow rate sensor. Therefore, for a flow rate control system, the correct estimation of the blood flow rate is essential. In this work, a system specific flow rate estimation method is presented based on motor current and motor speed measurements. Methods: A training data set of motor current - I and rotational speed - omega, as well as volumetric flow rate - Q were recorded in an in-vitro model of the intracorporeal membrane oxygenator. A 40/60 water/glycerol mixture was used to approximate the dynamic properties of blood. The rotational speed of the motor was varied in a range of 10000 to 30000 RPM. For each motor speed, the hydraulic resistance of the system was varied with a throttle resulting in different flow rate values. Additionally, a test data set was acquired consisting of different rotational speed, hydraulic resistance, and flow rate triplets. The I, omega and Q training data was used to construct a fitting surface of the type QI, omega by a regularized cubic spline least-squares approximation method. The surface was defined in a grid of 591x591 points with Imin=-70 mA, Imax=520 mA, omega min=500 RPM, omega max=30000 RPM. The test set of I - omega pairs was then used to predict Q by means of a nearest neighbor search in the modeled surface. Results: A correlation coefficient of r=0.96 was achieved between the estimated and measured flow rates of the water/glycerol mixture. Discussion: The strong correlation between estimated and measured flow rate suggests that sensorless flow rate control is possible.Objectives: Developing new bioinks that are able to meet the crucial requirements for 3D bioprinting of different tissues has emerged recently as an important topic in the field of biomaterials. In this work, linear chains of hyaluronic acid HA were hydrophobically modified using hexadecyl short chains. The rheological properties as well as the 3D printability were investigated. Methods: HA of Mw = 76.6 kDa was hydrophobically modified by conjugation of parts of its primary hydroxyl groups with hexadecyl chains in two steps. The ratio of substitution was determined using 1HNMR spectroscopy. Two different hydrogels were prepared at polymer concentration of 0.4 and 1.6 wt. The rheological properties of both hydrogels were investigated using continuous flow experiment and cyclic strain time sweep experiment. Furthermore, the 3D printability of the hydrogels was studied using an extrusion-based 3D printer. Results: 1H NMR spectrum of the new product revealed that the substitution ratio was 40 mol. Furthermore, the modified polymer was able to form sable hydrogels at both polymer concentrations whereas unmodified HA was unable to form hydrogel at the same concentrations. The rheological experiments showed decreasing of hydrogels viscosity by increasing the shearing rate which points toward a shear thinning property. Both hydrogels showed sharp decreases in the storage moduli upon applying high strains. The hydrogel prepared using 2.6 wt of the modified polymer showed, however, higher shape-fidelity for 3D printing compared to hydrogel at polymer concentration of 0.4 wt. Discussion: Hydrophobic modification of HA using relatively short alkyl chains enables physical crosslinking between polymer chains to produce biocompatible and 3D-printable hydrogel using low concentration of modified polymer. The rheological properties suggest their suitability as ink for extrusion-based 3D bioprintingObjectives: High resolution imaging of living cells is impaired by artefacts introduced by chemical fixation and by the rapid movement of molecules during live cell imaging. We have therefore developed tools to cryo-arrest living cells on a microscope and image them by high- and super-resolution microscopy. However, the developed equipment is also applicable to investigate the process of cryopreservation. Methods: We have developed two different microscope cryo-stages that allow for continuous observation on the fluorescence microscope. One stage allows for relatively slow controlled cooling <50 degrees C/min and warming <150 degrees C/min of cells. It allows for medium exchange during cooling and warming and thus for change of concentration of cryoprotective agents. Due to the design of this microscope stage, oil immersion objectives can be used down to at least -45 degrees C. The second stage is designed to ultra-rapidly cool cells with >10000 degrees C/s to temperatures Results: We established a protocol to reversibly cryo-arrest living cells on a microscope stage. This permits to image physiological processes in the same living cells with practically unlimited acquisition time at consecutive points in time. This allowed for super-resolution imaging and microspectroscopy measurements, which were impeded at physiological temperatures, because of the dynamics of cellular molecules. The development of a device to ultra-rapidly cryo-arrest living cells on a microscope under continuous observation allowed for the investigation of cells, which have been cryo-arrested/vitrified within Discussion: The presented cryo-fixations methods enable a more precise measurement of molecular patterns and thus a better understanding of molecular organization in cells under physiological conditions. The developed microscopy stages should however also enable high- and super-resolution fluorescence imaging of molecular processes in cells during the process of cryopreservation.Objectives: The aim of this work is to provide a general overview of the design challenges in the wireless powering of artificial organs, to develop a general design strategy for inductive powering units IPU, and to design a prototype IPU for mid-power application with high-stable output characteristic. Methods: Artificial organs can benefit greatly from inductive wireless powering, but there are several major design challenges which must be overcome. The high energy transfer efficiency must be ensured. The IPU must tolerate coils misalignments. Tissue overheating must be avoided. Biocompatible materials must be used. Size and weight restrictions must be taken into account. Finally, patient body specifics should be considered. We have developed a new general design strategy in which all mentioned problems are addressed. The main feature is that we simultaneously optimize the inductive link and the schematics of the transmitter and the receiver and take the body specific elements the thickness of the skin and the fat layer into account. We have developed strictly formal procedures which can be implemented using software in computer-aided design procedures. Results: In order to test and verify our approach we have designed a prototype IPU. The design goal was set as follows: IPU must ensure output power in the range 0.45 horizontal ellipsis 0.55 W for the axial distance between the coils in the range 10 horizontal ellipsis 20 mm and the lateral misalignments up to 20 mm. The design goal was achieved. Several coil couples were used to adjust optimal energy transfer distance to the expected implantation depth. Measurements were performed in the air as well as with the samples of "skin-fat" biological objects were performed. Discussion: We have developed and tested design strategy for biomedical IPU which can provide reliable design adjusted to the various requirements and the specific of the patients body.Objectives: The HeartMate 3 HM3, Abbott Laboratories, USA and HeartWare HVAD Medtronic Inc., USA clinically revealed a significantly different thrombogenic profile. In detail, while the HM3 has abated incidence of pump thrombosis, it was not associated with reduced stroke incidence. We numerically characterized and compared prothrombotic flow characteristics of the two pumps to provide mechanistic insights into observed clinical outcomes. Methods: Computational fluid dynamics CFD simulations were performed to evaluate flow fields associated with periodic speed modulation in the two pumps the artificial pulse for the HM3 and the Lavare cycle for the HVAD. One-way coupling to a lumped parameter model of the cardiovascular system was employed to account for physiologic boundary conditions. Results were compared to baseline simulations with constant pressure head and rotor speed to comprehensively evaluate geometrical and speed modulation effects. Analysis of the thrombogenic potential was based on calculation of: i a specific platelet activation marker, the Platelet Activity State PAS, using time course of viscous and Reynolds stress computed on platelet trajectories probed using Lagrangian particle tracking, ii scalar washout, iii volumes of flow stagnation, and iv wall shear stress patterns. Results: The HM3 and HVAD showed different PAS distribution. Speed modulation was associated with increased sum of viscous and Reynolds stresses and PAS magnitude. Analysis of temporal and spatial gradients of the shear stress histories calculated on platelet trajectories, volumes of stagnation, and wall shear stresses allowed identification of geometrical and operating parameters that might contribute to pump thrombogenicity and pump washout Discussion: We compared prothrombotic fluid dynamic features of the HM3 and HVAD under realistic dynamic operating conditions. Our results provide a "numerical rationale" for the observed clinical outcomes.Objectives: Over time left ventricular assist devices LVAD have become an alternative to heart transplantation because of enormous technical development and miniaturization. Implantation of these systems has been proven to lead to considerably improved survival, with an increase in quality of life. Following implantation, most patients display a considerable improvement in physical capacity, measured according to the New York Heart Association NYHA classification, even though exercise tolerance levels following the implantation of an LVAD are still considerably restricted. Methods: Current state of scientific knowledge with respect to the physical capacity of patients with terminal heart failure after LVAD implantation is described at rest and during exercise. The significance of new diagnostic tools, such as the non-invasive inert gas rebreathing method for measurement of cardiac output and arteriovenous oxygen difference AVDO2 in assessment of the performance of LVAD patients is discussed. Results: Pump flow across the LVAD can be increased to a certain degree during exercise and normal everyday activities via two mechanisms: exercise-dependent tachycardia and increase of enddiastolic pressure in the left ventricle in conjunction with increased preloading. The share of the native left ventricle in systemic perfusion can also increase during exercise. Here, left-ventricular pressure exceeds systemic blood pressure and the aortic valve opens. The contractile reserve of the myocardium does, of course, play a crucial role. Discussion: Ultimately, the cardiopulmonary capacity of LVAD patients is restricted to a higher degree during exercise, even after implantation. Larger studies are required to clarify the extent to which longer-term physical exercise training after discharge from rehabilitation and/or an increase in the number of pump rotations can lead to increased physical capacity in LVAD patients.Objectives: The epidemiology of heart failure HF is a rising medical burden with an estimated growing rate of 46 from 2012 to 2030. Left Ventricular Assist Devices LVAD and minimally invasive techniques are nowadays key therapies to guarantee optimal treatments for each HF patient. So far, clinical experience with minimally-invasive LVAD surgery showed promising short-term results. The aim of this study is to evaluate the long-term results of the first series of patients who received minimally-invasive LVAD surgery at our institution. Methods: We reviewed the short and long term outcomes of 111 end-stage HF patients who received an LVAD implantation HVAD, HeartWare, Medtronic, USA at our institution between 2011 and 2014. All patients underwent an upper hemisternotomy and left-sided anterolateral thoracotomy with cardiopulmonary bypass. Results: Between 2011 and 2014, 111 minimally-invasive LVAD implantations were performed at our institution 78 male, 22 female; mean age 52 /- 4; dilated cardiomyopathy 42.6; ischaemic cardiomyopathy 44.4, other aetiologies 13. Postoperative bleeding incidence was 9.6 leading to a low amount of applied packed red blood cells mean: 4.9 rbc, short ICU stay mean 9.3 days, and a low incidence of right heart failure 4.6. Thirty-day mortality was 5.2, 90-day mortality 9.0 and 1-year survival 88. Long-term follow-up median 6.4 years demonstrated a mortality of 51,4, a transplant rate of 16.4, and an explant rate of 1,7. 15.8 of patients underwent pump exchange and 30,3 of patients had still an ongoing LVAD therapy at the time of follow-up. Discussion: Minimally-invasive LVAD implantations are proven to be safe and associated with a lower perioperative complication and mortality rate. By using an upper hemisternotomy with an anterolateral thoracotomy, the intra-hospital outcomes showed promising results impacting also the long-term follow-up which presented an excellent percentage of ongoing LVAD therapies.Objectives: Quantitative phase microscopy QPM provides fast and label-free analysis of living cells by minimized interaction with the sample. The analysis of QPM images enables the determination of morphological cell parameters like cell thickness and elongation and facilitates single cell tracking for migration analysis. Methods: To demonstrate the potential of QPM in determining the metastatic potential of cells, we have developed a fully automated QPM system utilizing digital holographic microscopy DHM in an off-axis Mach-Zehnder configuration. Different pancreatic tumor cell lines, beating heart cells, white blood cells and in vitro toxicology models were analyzed for refraction index, cell volume, cell area, dry mass and cell morphology dependent processes. Results: The morphological analysis of the different cell lines via QPM in combination with image segmentation-based evaluation of the retrieved quantitative phase images showed that QPI could identify and quantify cell types and cellular processes by cell specific properties as cell proliferation, cell death, 3D cell migration or cell height. E. g. cells with a high metastatic potential had a lower cell thickness and a higher elongation than cells with a low metastatic potential. Moreover, computer assisted tracking of single cells showed that highly metastatic cells covered a longer distance and had a higher motility compared to cells with a low metastatic potential. Also white blood cell types and disease or drug induced morphological blood cell alterations could be separated. Discussion: In summary, the multi-functional potential of QPM in various cell biological applications as cancer cell or stem cell research, toxicological and pharmacological screenings and cell quality control represents a promising new approach for a fast and label free phenotyping of different cell types and cellular processes. QPM meets the growing interest in label-free, optical techniques minimizing samples interaction.Objectives: Severe life-threatening thrombocytopenia may be caused by exceptionally wide range of clinical conditions. Currently, donor platelet PLT transfusion is the only efficient clinical approach to successfully treat thrombocytopenia. However, PLT transfusion is challenged by adverse immune responses and is limited the short shelf life of donated material. Previously, we showed the feasibility to generate iPSC-derived MKs in a large scale, which can be used as an alternative to conventional PLT transfusion. However, success of this perspective therapeutic approach requires efficient biobanking technologies that would allow long-term storage and accumulation of MKs in clinically sufficient amounts. In this study, we aimed at the development of an optimal biobanking strategies for the preservation of iPSC-derived MKs. Methods: Dimethyl sulfoxide Me2SO and propane-1,2-diol PD were evaluated for their cryoprotective efficiency in iPSC-derived MKs. Comprehensive phenotypic characterization of MKs before and after cryopreservation was performed by fluorescence microscopy and flow cytometry. Capability of cryopreserved MKs to form proPLTs and release functional PLTs has been tested in vitro. Transfusion into animal model has been performed to evaluate capability of MKs to release PLTs in vivo. Results and Discussion: Application of developed cryopreservation procedures allowed recovery of over 80 of iPSC-derived MKs cryopreserved with Me2SO and PD. Recovered cell populations expressed typical MK markers, including CD41, CD61, and CD42a and showed high polyploidy. Moreover, cryopreserved MKs showed the capability to form proPLTs and release PLTs in vitro as well as in vivo after transfusion in animal model. Present findings propose efficient clinically relevant biobanking strategies necessary for progress in MK-based regenerative and replacement approaches.Objectives: Adverse events due to flow-induced blood damage remain a serious problem for blood pumps as cardiac support systems. The numerical prediction of blood damage via computational fluid dynamics CFD is a helpful tool for the design and optimization of reliable pumps. Blood damage prediction models primarily are based on the acting shear stresses, which are calculated by solving the Navier-Stokes equations on computational grids. The purpose of this paper is to analyze the influence of the spatial discretization and the associated discretization error on the shear stress calculation in a blood pump in comparison to other important flow quantities like the pressure head of the pump. Methods: CFD analysis using seven Unsteady Reynolds-Averaged Navier-Stokes URANS simulations were performed. Two simple stress calculation indicators were applied to estimate the influence of the discretization on the results using an approach to calculate numerical uncertainties, which indicates discretization errors. Results: For the finest grid with 19 million elements, numerical uncertainties up to 20  for shear stresses were determined, while the pressure heads show smaller uncertainties with a maximum of 4.8 . Discussion: No grid-independent solution for velocity gradient-dependent variables could be obtained on a grid size that is comparable to mesh sizes in state-of-the-art blood pump studies. It can be concluded that the grid size has a major influence on the shear stress calculation, and therefore the potential blood damage prediction, and that the quantification of this error should always be taken into account.Objectives: The piezoelectric effect of polyvinylidene fluoride PVDF is a promising property to be considered when controlling the diffusion of ions through membranes. This study examines the impact of different solvents and their ratios on the piezoelectric beta-phase of PVDF and on the diffusion rates of fiber membranes. Methods: Fiber mats were polarized via electrospinning using 20w PVDF solutions with solvent mixtures of dimethylformamide DMF or dimethylacetamide DMAc as first solvent and acetone or methyl acetate MeOAc as second solvent in different ratios. Conversion of non-polar alpha-phase into polar beta-phase was evaluated via Fourier-transform infrared spectroscopy FTIR. Diffusion experiments were performed by measuring changes in conductivity of deionized water 1.5 mu S/cm for 150min under static and cyclic loading conditions. Results: Polymer solutions showed a non-Newtonian behavior. An increasing acetone ratio led to decreasing densities in PVDF solutions. Densities remained unchanged for all MeOAc ratios. FTIR results for MeOAc-solubilized PVDF fiber mats showed lower intensity of alpha-phase peaks at 762cm-1 as compared to acetone solubilized ones. The intensity of the characteristic 840cm-1 polar beta-phase peak rose with a higher ratio of the second solvent. Diffusion experiments with piezoelectric PVDF membranes n=5 inhibited sodium chloride diffusion under static conditions 67 mu S/cm while induction of the direct piezoelectric effect allowed for increased diffusion 533 mu S/cm. Discussion: Solvent properties and ratios influence the intrinsic properties of PVDF solutions for electrospinning and consequently affect beta-phase conversion. Especially, MeOAc in combination with DMF or DMAc has a positive effect on piezoelectric beta-phase conversion. With suitable process parameters, piezoelectric fiber mats can be produced for active filter membranes. Ongoing experiments aim to investigate their molecular weight cut-off properties.Objectives: A cochlear implant CI is an auditory neuroprosthesis used to restore hearing in deaf and profoundly deaf patients. It features an electrode array EA which needs to be inserted into the inner ear cochlea to enable electrical stimulation of the auditory nerve. Spatial proximity of the EA and the nerve fibers is considered beneficial. This requires a shape change of the initially straight EA to enclose the inner wall of the spiral-shaped inner ear after implantation. Thin wires made of Nitinol-a shape memory alloy SMA-are investigated regarding their applicability as embedded actuators to implement a shape change. Methods: Nitinol wires  null 100 mu m have been trained to a spiral shape which was derived from cross-sectional images of a human specimen as an exemplary adaption to a specific cochlea geometry. Ten different variations of thermomechanical processing were applied by the supplier G.RAU GmbH  Co. KG to tailor the shape memory effect to the application specific requirements. In order to determine the resulting transformation temperatures AS, AF a bend and free recovery BFR test suitable for curved wires was developed and utilized. Results: It is possible to design and fabricate spiral-shaped SMA wires which fit to a specific cochlea shape. The developed BFR test setup allows for measuring the thermomechanical properties of these actuators in an application oriented manner. This provides slightly higher but more realistic measures for AS and AF compared to differential scanning calorimetry. Discussion: A complete design, manufacturing and validation process for thin spiral shape memory wires could be established. The next step is to integrate these wires into an EA and perform BFR measurements again as it is known that external forces due to stiffness of the EA alter the transformation temperatures.Objectives: The purpose of the work is to develop a theoretical basis for determining the thickness of the laminar boundary layer in the nasal cavity during respiration. Methods: Methods for obtaining basic scientific and practical results are based on the general principles of theoretical physics and aerodynamics in order to determine the laminar boundary layer of air flow in the upper respiratory tract. Results: Using the aerodynamic approach to the study of respiratory and olfactory disorders revealed that the thickness of the laminar boundary layer decreases with decreasing equivalent diameter of the nasal cavity and with increasing Reynolds number, which characterizes the degree of turbulence in the air flow. Typical values of the thickness of the laminar boundary layer are in the range of 0.2-0.05 mm, depending on the mode of nasal breathing and the configuration of the nasal cavity. The data obtained characterizes the effect of airflow on the mucous membrane of the nasal cavity: if the heterogeneity of the mucosal surface extends beyond the boundary of the laminar boundary layer, then such portions of the nasal cavity are exposed to turbulent flow, which leads to overdrying. The size of the heterogeneities of the mucosa is determined by tomographic data during their processing at the sub-pixel level, which allows for the investigation of the effect of the intensity of the boundary elements of the walls of the nasal passages. Discussion: A model of the distribution of airflow velocities in the nasal cavity in the turbulent mode is proposed; it has a logarithmic or power profile cross section, and within the laminar boundary layer the air velocity increases linearly to almost the max value. The current air velocity in the cross sections amounted to 5-7 m/s. The feature of the developed method is transitioned to hydraulic diameter, which takes into account complex configuration of the nasal cavity, as well as an anatomically narrow area - the olfactory slit.Objectives: Bioengineering of elements of the musculo-skeletal system is rich in studies concerning bone reconstruction. Among them, the culture of cells on a osteogenic electrospun scaffold under dynamic conditions is an interesting option. However, one should also consider the reconstruction of tendon and muscle finally mimicking a continuum from bone to muscle. Methods: Poly-caprolactone PCL 10 was electrospun on a rotating collector whose velocity varied to get random or aligned nanofibers. Bone marrow stem cells BMSCs and C2C12 cells were respectively seeded on the materials. Mechanical or electrical stresses were applied to the biohybrid scaffolds to differentiate cells towards tendons or muscles phenotypes, in the absence any specific biochemical factors. Results: After 2 weeks of culture under dynamic stretching 1Hz, 4 strain for an hour followed by 11h of rest, BMSCs were found to align in the stretching direction, presented increased levels of tenomodulin and neosynthetized collagen, compared to static conditions. C2C12 differentiation until aligned myotubes stage was more difficult to achieve. We proposed to add a PEG microstructure to guide cell growth. Combined to electrical stimulation, it led to the cells fusion into very long myotubes hosting many nuclei, as expected. Discussion: In this study, we managed to grow two different types of cells and generate their differentiation towards tendon and muscle lineage, in the absence of any differentiation factors. The next step is to implement the co-culture on different areas of the same electrospun scaffold, to let the cells proliferate and study the events occurring at the junction.Objectives: Alarm fatigue is a safety and quality problem where exposure to high rates of clinical alarms results in desensitization leading to cognitive bias or slowed response to alarms. One reason for alarm fatigue is false pressure alarms during extracorporeal blood therapies. It is clear that current machine algorithms based on simple threshold monitoring must be replaced by intelligent algorithms that avoid any false positive alarm. Methods: Detailed analysis of arterial pressure, venous pressure, transmembrane pressure, ultrafiltration rate and relative blood volume during hemodialysis was performed using a patient data base. Main causes of pressure alarms were identified, analysed and replicated during simulated hemodialysis therapies with a FMC 4008 hemodialysis device and a patient simulator. Results: Data analysis shows clear evidence that even slowly drifting adaptive baseline alarms which are currently in use lead to false positive alarms with a high probability during nearly every treatment. Pressure variations which are induced by local flow and blood viscosity changes can be identified reliably by use of algorithms based on pattern recognition. Pressure variations from patient movements and changes in blood pressure show a different pattern that can be used to distinguish between true alarm events and false positive events. Discussion: For 25 years, it has been known that alarm systems which monitor whether a threshold value is exceeded have a rate of false alarms of 77-99,5 depending on the physiologic parameter monitored Wiklund et al., 1994. New adaptive algorithms based on pattern recognition principles help to combine both maximum patient safety and avoidance of false positive pressure alarms.Objectives: In vivo tissue engineered vascular tissues "Biotubes" constructed in the subcutaneous spaces of the recipients exhibited superior functions. However, since the formation of the vascular tissues depends on the recipient conditions, chemical pretreatments such as dehydration by ethanol ET or crosslinking by glutaraldehyde GA were attempted to improve their initial mechanical durability. Last year, we reported that ET and GA increased the burst pressure of the tissues and did not affect their suture retention strength. This year, we measured 3 parameters of mechanical properties: ultimate tensile strength UTS, ultimate strain  and Youngs modulus. Methods: Tubular tissues ID: 5 mm constructed in the subcutaneous tissues of beagle dogs 4 weeks, n=3 were separated to the 3 groups, raw tissue group with no treatments RAW, that under dehydration with 70 ET ET for 10 min and crosslinking with 0.6 GA GA for 10 min. We measured 3 parameters shown above, by tissue ringlet pull test. Results: There were no significant differences in UTS RAW: 3.765 /- 0.662 MPa, ET: 4.330 /- 0.619 MPa, GA: 3.763 /- 0.438 MPa, p=0.409 or ultimate strain RAW: 30.15 /- 1.60 , ET: 41.22 /- 6.11 , GA: 37.73 /- 4.36 , p=0.209 between the 3 groups. Youngs modulus of ET group was significantly higher among the 3 groups RAW: 5.41 /- 1.16 MPa, ET: 12.28 /- 2.55 MPa, GA: 7.65 /- 1.18 MPa, p= 0.0289. Discussion: Short period of chemical treatment using ET and GA might make it possible to control the mechanical properties of in vivo tissue engineered vascular tissues to produce the ideal grafts in mechanical aspects.Objectives: Stent graft development is one of the most important topics for countries with a large number of elderly people. However, very little research has been carried out using model circulation of the aortic dissection. Quantitative evaluation and development have been difficult, because there was no model. Methods: In this study, a fluid dynamical circuit model circulation simulating the left heart circulation of the dissecting aortic aneurysm was developed with a pulsatile blood pump. Various shapes of the inflow side and outflow side for the pseudo lumen were used in the experimental series. Furthermore, an animal model of the dissecting aneurysm was developed using a mechanical dissecting method and flow patterns of the animal aortas with the model dissecting aneurysm, after the ethical committees approval. Flow and pressure pattern time series data was recorded in the digital recorder and a quantitative evaluation was carried out. Results: Flow of the true lumen and pseudo lumen was divided with a thin membrane. Various amounts of the flow and pressure was observed and recorded. Membrane of the dissecting aortic aneurysm was simulated with rubber sheet. After the insertion of the aortic stent, the shapes of the flow and pressure pattern were altered. Animal experiments to develop the dissecting aneurysm were difficult. Firstly the aorta of the goats was removed, and then the dissecting aneurysm was to be constructed by the detachment of the tunica media of the aortic wall. Flow pattern of the experimental Aorta was recorded. Discussion: Quantitative evaluation of the Aortic Stent expanding performance and pressure and flow pattern in the true and pseudo lumen had been observed in the mock circulation loop. Furthermore, animal model of the dissecting aorta will be useful material for the development of the stent graft in the near future.Objectives: The aim of this work is to investigate the performance of the RealHeart TMpsi total artificial heart TAH with a hybrid cardiac simulator HCS. Methods: The HCS, developed at the Nalecz Institute of Biocybernetics and Biomedical Engineering, is a computational-hydraulic model of the cardiovascular system. Four hydro-numerical interfaces reproduced the pressure/flow profiles of atria, pulmonary artery, and aorta hydraulically, while the rest of the circulation was numerical. The TAH was connected accordingly. The TAH is a pulsatile 4-chamber device whose left and right sides can operate independently, by changing pulse rate PR and stroke length SL. The TAH was tested on the HCS at low/medium operating points: PR 60/80/100/120 bpm; SL 20/22/24/26 mm; systole duration ratio 0.4 of the TAH cycle. Systemic resistance was changed between 0.8 and 1.2 mmHg/ml/s, pulmonary resistance was changed between 0.1 and 0.2 mmHg/ml/s. Systemic and pulmonary compliances were set to 1.4 ml/mmHg. Results: The independent tuning of PR and SL permits fine regulation the TAH flow on the left and right sides. As an example, for a peripheral resistance of 1.0 mmHg/ml/s the flow ranged from 2.7 l/min PR=60,SL=20mm to 7.0 l/min PS=120,SL=26mm. For the investigated PRs and SLs, power consumption on the left pump ranged between 1.3 and 10.0 Watts and on the right pump between 0.9 and 6.0 Watts. The TAH provides a pulsatile flow, whose waveform depends on the SL and systole duration. This induced a pressure pulsatility ranging from 10mmHg PR=60,SL=20mm to 38 mmHg PR=120,SL=26mm in both the aorta and pulmonary artery. Discussion: The HCS is a flexible test bench to evaluate the TAH performance in different hemodynamic conditions. The tuning of PR and SL on the left and right side of the TAH, independently, offers flexibility in regulating cardiac output and in managing the left/right flow balance. Power consumption is in the range suitable for clinical applications.Objectives: In case of chronic mechanical circulatory support, it is highly desirable to fully understand the interaction between the beating left ventricle and the VAD in order to potentially reduce adverse events. Thus, we took the VADs perspective and investigated the flow inside a VAD in an advanced development status during the time course of a heartbeat. Methods: The study is conducted with computational fluid dynamics CFD applying realistic pressure boundary conditions for left ventricular and arterial pressure and the results are compared to results produced with an established, active mock circulation loop. Due to the high resolution in time and space and the use of sliding mesh interfaces, local flow phenomena during different phases of systole and diastole are unmasked. The effects of inertia in the pump and cannulae become evident, which are not modeled in more traditional VAD flow simulations with steady-state boundary conditions. Results: The results show that hysteresis effects due to inertia are quantitatively of high importance. The hysteresis curves of the VAD operational point obtained experimentally and numerically correlate well. Moreover, the results show that a VAD design for a highly pulsed flow can lead to a situation, where almost no remaining stagnation zones, as quantified with a numerical stagnation index, exist during a heartbeat. Discussion: Despite increased computational costs, highly resolved CFD simulations in time and space with realistic time-varying boundary conditions are a powerful tool to increase understanding of the interaction of the mechanically supported circulation and the device itself. This enabled a verification of a VAD design with low to almost non-existing flow stagnation at constant rotational speed. This finding correlates well with the fact that the VAD does not require any anticoagulation in chronic animal trial testing lasting weeks to months.Objectives: The friction properties for the contact between contact lenses and the eyelid are the focus of current research in order to optimize the wearing comfort. The tribological system consisting of contact lense, eyelid and tear fluid should be considered. Suitable measurement conditions must be used for the examinations. Methods: In this research the nano- and microtribological friction properties of hard and soft hydrogel contact lenses were investigated. The friction conditions were investigated by nano scratch test; moreover, a nanotribometer was used to evaluate the frictional properties during sliding on the lenses. Contact lens fluid was applied as lubricant for both cases. Results: The friction coefficients of hard contact lenses and hydrogel lenses were detected for a range of speeds and loads. It was possible to examine the differences of the both contact lenses in detail. For the nano scratch and friction tests, there is a dependency of speed and normal force. Discussion: In case of nano-scratch tests, the frictional properties were investigated wile ploughing. This case shows the elastic- and plastic material properties besides the friction. In the case of the frictional tests, the measuring tip slides without ploughing. Due to the lubrication conditions, the soft hydrogel lenses feature a much lower friction coefficient. In case of the hard contact lenses, the coefficient of friction is comparable to the nano scratch test, as there is no lubrication film between both measuring tips and the contact lens. For further work it is mandatory to work under standazdized measurement conditions.Objectives: Patients suffering from chronic kidney disease CKD show an increased cardiovascular mortality and morbidity mainly due to the accumulation of protein-bound uremic retention solutes. Dialysis therapies have been optimized for the removal of low-molecular weight water-soluble and unbound uremic retention solutes, but not for protein-bound uremic retention solutes. Therefore, there is a strong need to increase the portion of unbound fractions in order to increase the clearance for these uremic toxins. Methods: We developed dialysis techniques where the protein-bound uremic retention solutes are removed more efficiently under high ionic strength or high-frequent electric fields. The protein integrity of proteins and enzymatic activities were analysed. Results: The protein-bound fraction of phenylacetic acid, indoxyl sulfate and p-cresyl sulfate was significantly decreased by using high ionic strength or high-frequent electric fields Discussion: Although protein-bound uremic retention solutes have a major impact on the morbidity and mortality of CKD patients, the clearance of these solutes by conventional extracorporeal therapies is low until now. These uremic toxins bind to plasma proteins with a molecular weight greater than the cut-off of dialysis membranes and therefore conventional dialysis therapies do not sufficiently remove protein-bound uremic toxins. Disruption of association with protein and uremic solute is one possibility to significantly improve the removal of protein-bound uremic toxins. Therefore, we developed approaches for an increased release protein-bound uremic retention solutes from their protein-binding resulting in an increased clearance. Conclusion: Improvements of therapies for CKD patients are essential. Especial the consideration of removing hydrophobic protein-bound uremic toxins via new methods like application of high-electric fields or increased ionic strength is a further progressObjectives: Tissue decellularization is one of the novel techniques in tissue engineering and regenerative medicine for obtaining extracellular matrix, which doesnt preserve mechanical properties of native tissue. Modified by the extracellular matrix, silk fibroin scaffolds can combine high biocompatibility and suitable mechanical properties. Methods: Wistar rats were used as liver donors. Decellularization of liver tissue was performed by sequential incubation with three solutions of 0,1 SDS containing Triton X-100 in the following concentrations: 1, 2 and 3. Livers were ground in liquid nitrogen and a certain fraction of microparticles was separated by precipitation and centrifugation. Silk fibroin scaffolds were prepared by a casting method using water as a solvent. Microparticle suspension was blended with fibroin solution before film preparation. Scaffold structure was investigated by scanning probe nanotomography SPM and scanning electron microscopy SEM. The cytotoxicity of scaffolds, cell adhesion and proliferation were evaluated. Modified scaffolds were used as wound coatings in a full-thickness skin wound model. Results: Microparticles with a size less than 5 mu m were prepared from decellularized liver tissue. Modified silk fibroin scaffolds were obtained. Sinuous and rough topography of the obtained matrix was shown. The produced scaffolds did not show cytotoxic effect. The cells adhesion was significantly higher on scaffolds modified with decellularized liver tissue microparticles in comparison with non-modified scaffolds. Modified silk fibroin scaffolds promoted skin wound regeneration by 30 in comparison with non-modified scaffolds. Discussion: Regenerative potential of extracellular matrix as microparticles was shown. Decellularization of organs and tissues may be used in tissue engineering and regenerative medicine for increasing the biocompatibility of the artificial grafts.Hemodialysis is an important therapy for End Stage Renal Disease ESRD patients if a donor kidney is not available. During four hours of conventional therapy, three times a week, mainly small water-soluble toxins and a limited number of middle molecules are effectively removed from the patients blood 1. Besides, the therapy is non-continuous, causing large fluctuations in water balance and uremic waste, potassium and phosphate. Therefore, in recent years, many developments are focussed on longer treatment times, such as, nocturnal dialysis or application of a portable or wearable artificial kidney PAK or WAK respectively. The continuous therapy outside the hospital requires application of membranes with long-term selectivity, fouling resistance and blood compatibility as well as demands reduced amount of water. In fact, for WAK, a small volume of spent dialysate should be continuously regenerated and reused. In this lecture, we will discuss all these important issues, including:Strategies for achieving endotoxin-free dialysate since the continuous recirculation of the dialysate for prolonged time demands additional efforts to avoid microbial contamination, without extra water consumption and without affecting the overall weight of the system. Strategies for achieving dialysate regeneration and especially urea removal since its daily molar production is higher than that of other waste solutes.Objectives: Ventricular assist devices are a commonly used therapy for end stage heart failure patients. Pump thrombosis is one of the most common problems in patients supported by an LVAD. Two types of thrombus can be distinguished: white thrombi and red thrombi. The only available option to combat a PT is a lysis therapy or a pump exchange. The objective of the presented study is to develop a relevant white pump thrombus model to promote development of these concepts. Methods: We developed an in vitro mock circulation system. The artificial blood circulation is provided by a HVAD. The HVAD pumps 80ml of human blood through the silicone loop. Parameters for inducing the white thrombus are high shear stress 4000 revolutions per minute, a rougher surface of the rotor, and an added activator of the intrinsic coagulation cascade. After two hours, the HVAD-system was checked for a white thrombus. Generated thrombi were stored for subsequent analysis scanning electron microscopy and mechanical stability. Results: Thrombus generation was reproducible in number and geometry. After ten test runs, 23 thrombi were detected. All these generated thrombi had a minimal size of 12 mm2. The developed thrombi are very similar in structure compared to white patient thrombi explanted in clinic. The scanning electron microscope showed a fibrin net on the surface of all tested thrombi n=5. All tested thrombi samples n=5 resist high strength max. 1.029 MPa in compression tests, which is typical for white PT. Discussion: The size of the thrombus should be increased for special investigational needs. A protocol was developed for reproducible and reliable production of white, shear induced thrombi for lysis therapy investigations. In addition, the similarity biological structure, mechanical stability to explanted patient thrombi was shown. However, additional analyses should be performed for further characterization of the artificial thrombi.Patient-specific anatomies can be reconstructed with a precision of approximately 0.5mm, using state of the art imaging modalities. This refers both to hard and soft tissues, hence with applications in clinical domains such as orthopedics, dentistry, cardiovascular surgery. Datamining techniques such as statistical shape modeling support the generation of biofidelic computer models. However, these models have to be fed with accurate data to reflect not only the geometry but also the material properties, the loading conditions and the interface between implants and bone in case of prosthetic reconstruction. The challenge is to optimize the interface between host tissue and implant in order to achieve maximum long-term functionality, but this optimization is also dependent on the optimization of the interfacing between the implant, the surgeon and the host tissue. Thus also the surgical technique will benefit from engineering support. Biomedical engineering is a key player to achieve this. A good geometric fit between implant and host tissue will ensure optimal initial stability. Patient-specific implants can ensure this stability, incorporating solid and porous structures where needed. Pre-operative computer assisted planning allows to optimize the surgical intervention, incorporating patient-specific biomechanical models to predict the functional outcome of the surgical intervention. And finally, in order to achieve that the pre-operative plan is accurately transferred into the surgical practice, computer assisted surgery enabled by e.g. navigation, robotics of patient-specific surgical instrumentation guides and implants can come into play. Key to all state of the art developments in computer assisted surgery is the engineering on anatomy, enabled by multifunctional softwares and 3D printing technologies. Biomechanical modeling allows for patient-specific outcome prediction after surgical interventions and should become a routine practice in patient treatment. Patient specific surgical instrumentation is able to assist in realising the plan with sub-millimeter precision. Finally, quantification of the uncertainties that are inherent to biomechanical modeling is an essential contribution to reliable patient-specific outcome prediction.Objectives: Hemodialysis HD is a lifesaving therapy in patients with end stage renal disease. Little is known about why life expectancy for these patients is reduced in Europe and USA as compared to Japan. The aim of this study was to evaluate the importance of the interdialytic weight gain fluid retention in relation to estimated body weight in , IDWG in relation to the dialysis length hours. Methods: 20 hemodialysis patients performed besides their regular dialyses 1 hemodialysis/week within the frame of the study during a 3 week period. The baseline values and the change during HD of NT-pro-BNP ProBNP and Troponin T TnT at 180 minutes and pentraxin 3 PTX at 30 and 180min were compared with the change in IDWG divided by the hours the patient performed the HD as a marker of speed of ultrafiltration. The PROOF formula used was: 100 x Weight gain between dialysis kg/estimated body dry weight kg x time of HD session hours. Spearmans correlation analysis was used to adjust for eventual outliers. Results: There was a correlation between extent of removal of the IDWG/hours of HD: 1 the baseline values of ProBNP rho=0.54, p<0.001, TnT rho=0.37, p=0.004, PTX rho=0.26, p=0.043 and 2 the change during HD at 30 min of PTX rho=0.27, p=0.038 and at 180 minutes of ProBNP rho=0.68, p<0.001 and TnT rho=0.40, p=0.002. 3 a PROOF breakpoint < 0.63 caused less release of ProBNP p<0.001. Discussion: The increase in weight between dialyses should be related to the body weight. A larger IDWG gives worse conditions. An increased speed of fluid in relation to body weight removal causes a higher release of cardiac markers indicating a strain and damage to the heart by each HD in contrast to what would be expected. Patients should be advised to limit fluid intake in relation to their body weight and adjust the HD length.Objectives: Fish roe is a highly nutritional product with diverse beneficial effects for the human health. This work used sardine Sardina pilchardus roe to produce novel delivery systems presenting antioxidant and anti-inflammatory properties. Methods: Methyl-tert-butyl ether extraction method was used to obtain lipid- and water-soluble compound rich phases. The fatty acids profile was analyzed by GC/MS. The identification of the hydrophilic compounds was performed by LC-HRMS. Lipids were used to produce large unilamellar liposomes LUVs. The hydration of the lipid film was performed with PBS or sardine roe-derived aqueous phase. LUVs were characterized by their size, surface potential and morphology. The cytotoxicity of the extracts was evaluated using L929 cells. Their antioxidant activity was evaluated against different radicals, and their anti-inflammatory activity was assessed in the presence of LPS-stimulated macrophages. Results: GC/MS analyses demonstrated the presence of several fatty acids, such as omega 3 fatty acids. LC-HRMS analyses revealed the presence of numerous bioactive compounds e.g. essential amino acids and gadusol. LUVs prepared with the aqueous phase presented a higher heterogeneity in terms of size than when they were prepared with buffer. Both LUVs formulations have a negative surface charge and a spherical shape. The extracts presented antioxidant activity against peroxyl, hydroxyl and nitric oxide radicals. Biological assays indicated that the extracts are cytocompatible for the lowest tested concentrations and that they can inhibit the production of proinflammatory cytokines, which reveals their anti-inflammatory effect. Discussion: Our sardine roe-derived delivery systems and compounds have both antioxidant and anti-inflammatory activities and thus they may be considered as a valid alternative for the treatment of inflammatory conditions.Objectives: Engineered skeletal muscle tissues in three-dimensional 3D cell culture platforms that resemble the muscles complex native structure and organization can be used as in vitro models to study muscle physiology and metabolism. This project allows us to develop a new platform to model muscle diseases such as myotonic dystrophy 1 in vitro in order to study its response to candidate therapeutics and to better understand disease mechanisms of pathogenesis. To this end, we monitor the secretion of disease-associated biomarker proteins and metabolites. Methods: Here, we present 3D skeletal muscle constructs, fabricated by encapsulating C2C12 cells and patient-derived transdifferentiated skin fibroblasts in a photocrosslinkable Gelatin Methacrylate and Carbolymethylcellulose Methacrylate GelMA:CMCMA hydrogel and cryogel scaffold. These scaffolds present a microgrooved topography that promotes cell alignment and differentiation. Electrical stimulation ES was then applied to the engineered tissues during cell culture to induce spontaneous contraction and maturation of the sarcomeres. Cell alignment differentiation, and the effect of ES were assessed by calculating the orientation angle and fusion index of immunostained myotubes expressing Myosin Heavy Chain MHC. Results: We have obtained a new platform to study the evolution of congenital muscle diseases, specifically myotonic dystrophy 1 and evaluate the functional tissues by metabolic and gene expression analysis. Monitor the secretion of biomarkers proteins, metabolites, and the glycolysis pathway of muscle tissues for different drug candidates. Discussion: This platform has been tested with different drugs assays and represent a step toward the goal of producing in vitro drug testing systems for medical and pharmaceutical industry applications. Finally, such "muscle tissue-on-a-chip" devices can be fabricated using patients own cells as a major step toward personalized medicine.Objectives: The aim of this work is the creation and research of properties of tissue-engineering structures by the method of laser structuring a nanocarbon framework in bioorganic matrices. Methods: For the formation of the framework fibers from single-walled carbon nanotubes were used. Aqueous dispersions of nanocarbon fibers and proteins of albumin, collagen and amino sugar of chitosan were used as the initial medium for printing tissue-engineering structures. Printing of structures was carried out using the laser device generating pulsed laser radiation. The radiation wavelength was 1064 nm 100 ns pulse duration, 100 kHz frequency. Laser radiation using the scanner system of the device was moved on the dispersion layer. A computer model regulated the trajectory of the movement of radiation. Results: We have demonstrated the effect of welding nanocarbon fibers into a framework under the action of laser irradiation. The functionalization of nanocarbon fibers by protein and amino sugar molecules were proven. It was found the diameter of nanocarbon fibers was increased by several tens of nanometers due to their wrapping with a bio-organic matrix. Tissue-engineering structures had a meshy frame. The mesh size was 100x100 mu m. The electrical conductivity of the structures increased several times and amounted to 1 S/m with laser irradiation. The hardness of tissue-engineering structures was 150-370 MPa. The cytological compatibility of tissue-engineering structures with fibroblast cells and endothelial cells was tested. The density of proliferating cells on tissue-engineering structures was 33 more compared to the control type. In vivo investigations indicated the rate of biodegradation of tissue-engineering structures when implanted in laboratory animals for 60-90 days. Discussion: The investigated structures are promising for use in medical practice. Such structures can act as cardiac tissue-engineering implants since they have physicomechanical and biological properties similar to the parameters of cardiac tissue.Objectives: A major part of developing rotary blood pumps requires the optimization of hemolytic properties of the entire pump. The aim of this project was to compare blood flow simulation in the first and second generations of the ventricular assist device Sputnik and to evaluate its effect. Methods: Studies of the flow field were conducted with commercial CFD software ANSYS FLUENT 16.0, which uses a finite-volume approach. Scalar shear stresses were calculated for both pumps. The pumps were compared in terms of volumes subjected to certain viscous shear stress thresholds, below which no trauma was assumed von Willebrand factor cleavage: 9 Pa, platelet activation: 50 Pa, and hemolysis: 150 Pa, associated residence times and recirculation zones. Results: Volume fractions of shear stress above 9 Pa, 50 Pa, 150 Pa were received: Sputnik-1 had 7.69 ml, 2,58ml, 0.74 ml, Sputnik-2 - 3.41 ml, 1.07 ml, 0.22 ml. The sum of the cell residence times at these shear stresses is 0.5095 s for the Sputnik-1 and 0.2019 s for the Sputnik-2. The volumes of recirculation zones were obtained, the value is 4.36 ml for the Sputnik-1 and 1.72 ml for the Sputnik-2. Discussion: The second generation of the VAD Sputnik have better flow characteristics than first. In the operating point, the volume fraction of the shear stresses is less, the residence time is less and the volume of recirculation zones is less.Objectives: End-stage renal disease is increasing dramatically in the western world and represents a major challenge for health care. Due to organ shortage transplantation can help only a few patients who otherwise require lifelong hemodialysis. Access for hemodialysis is an essential point and carries a high morbidity mainly due to occlusion of arterio-venous fistulas AVF and/or grafts AVG. Therefore, alternative graft material has to be considered such as tissue engineered vascular grafts for access in hemodialysis. Methods: Besides patient characteristics and technical issues of surgical procedures the choice of the vascular graft is essential. Despite optimization over the last years, the clinical results are still sub-optimal due to early thrombosis and late intimal hyperplasia of the non-degradable synthetic vascular grafts such as Expanded-polytetrafluoroethylene ePTFE, or, polyethylene terephthalate Dacron. Results: New material developments and tissue engineering techniques have shown good pre-clinical and initial clinical results for AVG implantation in hemodialysis patients. These range from acellular and cell-based, stable or bio-degradable, synthetic scaffolds to biological mandril-induced or decellularized grafts as well as self-assembly technologies. Discussion: This symposium will review the clinical need and especially address new developments of vascular grafts based on tissue engineering.Modern heart failure therapy makes use of left ventricular assist devices for severe cases as well as other active implants. While these devices have come far from their beginnings with improved hemocompatibility, smaller size and less invasive implantation techniques, their advancement takes close cooperation of engineers and clinicians to face the challenges of further advancements. It is vital to ensure interdisciplinary communication and exchange to define the future of Mechanical Circulatory Support. On the engineering side, greater biocompatibility, patient-specificity and options for smart devices and telemedicine offer great potential. These developments go hand in hand with improved surgical technique and clinical expertise. This cooperation will allow us to work towards reduced mortality, less adverse clinical events and ultimately improved quality of life for the cardiac assist device patient of the future.Objectives: To elucidate the role of the solvent and pH in the conformational changes of human serum albumin HSA and its binding to fatty acids. Albumin is a key biomolecule, found in most fluids of the body. HSA functions as depot and carrier for many compounds like fatty acids and affects the pharmacokinetics of drugs. Importantly, HSA acts as a toxic waste handler, displays pseudo enzymatic properties and it is a valuable biomarker in many diseases. Structurally, HSA is a very soluble, extremely robust and usually monomeric protein, which features three alpha-helical homologous domains that fold into a characteristic heart shape. Despite its importance, the reversible pH-dependent conformational transitions of HSA in solution are not yet fully understood. Methods: To compute the stability differences between the different conformers of HSA, we combine molecular dynamics simulations MD with free energy calculations. We determine the conformational free-energy change between conformers of HSA obtained upon introducing perturbations in the surroundings pH, solvent mixtures and its physiologically native state. Extended constant-pH MDs followed by Gaussian accelerated molecular dynamics simulations allow us modelling these systems at different pH values. We perform free energy perturbation calculations to characterize the binding sites of fatty acids to HSA under different conditions. Results and Discussion: We elucidate, at the molecular level, the effects of the solvent and pH on the conformational transitions and functionality of HSA. Our work allows establishing ways to modulate such transitions by tuning key biomolecular interactions, with implications for biomedical research.Objectives: The metal ions can affect differentiation of stem cells, which makes them interesting for applications in tissue engineering and regenerative in medicine. The layer-by-layer LbL technique is based on the alternating adsorption of oppositely charged polyelectrolytes to form polyelectrolyte multilayers PEM. Multilayer systems combined with metal ions of different concentration might be promising for making coatings on implants and scaffolds to regenerate tissues like bone, cartilage, and others. The purpose of this study is to investigate the influence of the concentration of metal ions on the physical properties of Polyelectrolyte multilayers PEM, and evaluate the effect of metal ions on stem cell adhesion. Methods: Polyelectrolyte multilayers PEM prepared by layer-by-layer technique made of polysaccharides hyaluronan HA as polyanion and chitosan Chi as polycation were additionally cross-linked by metal ions Cu2, Co2, Ca2 and Fe3. to modulate physical properties and bioactivity of multilayers to control adhesion and function of mesenchymal murine C3H10T1/2 embryonic fibroblasts. Characterization of multilayer formation and surface properties was performed by different analytical methods. Results: Changes of wetting, thickness and mechanical properties of multilayers depending on concentration and type of metal ion. Most interesting, however, was the finding that metal ions like Fe3 promoted adhesion and spreading of C3H10T1/2 cell greatly on the less adhesive HA/Chi multilayer system Discussion: Multilayer systems combined with metal ions can be used to regenerate tissues like bone, cartilage and othersObjectives: The aim of the study is to analyse and modify the gas transfer properties of a novel PMP fiber blood gas exchanger with a cylindrical shaped blood flow channel by improving the sweep gas flow profile through the fiber bundle while considering the fiber length in blood contact. Methods: A test set-up has been developed to visualize the gas flow through the fiber bundle by means of steam. In an in-vitro setting with porcine blood the gas transfer rate has been measured subsequently. The cylindrical blood flow geometry and the fact that the fibers are stacked with an offset of 90 degrees onto each other results in fibers with various effective lengths. For the study, various prototypes of gas inlets with different inlet geometries have been designed, manufactured and evaluated regarding their gas transfer rates. Results: The different designs resulted in the aimed flow profiles during the flow visualization. Selected prototypes have been tested with blood and the gas transfer rates have been determined. However, the modified designs with improved gas flow properties do not significantly impact the gas transfer rates measured with blood. Discussion: Two possible explanations can be derived from the results which may be tested in the course of further studies. Firstly, it is possible that the current gas inlet is already sufficiently adapted to the cylindrical blood flow geometry and fibers of different effective length. Secondly, it could be considered that side parameters such as the diffusive transfer of water vapour from the blood side to the gas side have a superseding influence on the gas flow profile. In conclusion, the gas transfer rates cannot be significantly increased by manipulating the gas flow profile through the fiber bundle. Nevertheless, the blood gas exchanger achieves already comparable transfer rates to products on the market but has reduced PMP membrane surface.Objectives: Throughout the history of medical devices, highly invasive systems underwent a series of generations prior a successful establishment as a widely accepted therapy. The only available THA today SynCardia TAH is successfully saving lives but comes with a significant restriction of the quality of life QoL and a stigmatization of the patients. According to surgeons this is an important factor contributing to the low number of implantations today. Methods: Next generation devices using an implanted drive promise a new level of QoL and are currently developed by several institutions worldwide. Reasons for the long time to market of the next TAH generation >15 years might be found in the enormous challenges of technological aspects together with a difficult marked situation from an investors point of view. However, a higher QoL and an increasing trust in the reliability of the next TAH generations will inherently lead to a higher number of implantations, and further the acceptance of these devices towards an established therapy option. Results: The approach of the ReinHeart TAH is to adopt the well-proven hydraulic features of the SynCardia TAH while improving its current limitations. The ReinHeart TAH features a passive filling during diastole, a principal that is also successfully applied by the SynCardia TAH. The simple and robust linear driving mechanism allows for a compact design, achieving fit in the vast majority of patients. Beyond that, the drive principle makes additional sensors unnecessary. The compact, smart and usability-driven design of the external components will lead to a broad acceptance by the patients. Discussion: The unique features of the ReinHeart TAH elevate TAH-therapy to the next evolutionary step with regards to robust long-term application, size, ease of use and quality of life.Objectives: Circulating endothelial cells can be used to diagnose cardiovascular diseases. However, their concentration in the blood is very low <100 per ml. A large volume of blood is needed, which should not be drawn from the patient. Instead a probe is inserted into the bloodstream of the patient to isolate as many cells as possible. The geometry of the cell probe is one of the key factors affecting cell deposition because its shape influences the flow conditions along the cell probe. The aim of this combined in vitro and in silico study is to find out the optimal flow conditions for cell deposition aiming to develop in future novel optimal cell probe shape. Methods: In vitro experiments in a closed loop with endothelial cells from HUVEC cell line were carried out and the cell deposition as a function of the probe shape 4 probe types and five different flow rates were investigated. The knowledge of the flow conditions at the cell probe was obtained by flow simulations by using CFD software STAR CCM. By combining the experimental and simulation data with an iterative procedure, wall shear stresses, which seems to promote cell deposition, can be identified. Results: It could be shown that the geometry of the cell probe has a much greater influence on cell deposition than the flow rates in the blood vessel. Wall shear stress around 0.4 Pa seems to be responsible for the optimal deposition of endothelial cells. However, this flow condition does not correlate with all flow rate experiments. Discussion: The results of the combined in vitro - in silico study on deposition of endothelial cells on the existing cell probe design are promising. However, the used in vitro set up does not allow to extract maximal information from each test and should be redesigned.Objectives: The left ventricular assist device LVAD consists of the pump, driveline and peripherals, all of which are potentially subject to failure. According to the recent research only 13 of failures were due to the pump itself, while over 60 were due to batteries, controllers and peripheral cables. Taking into account usability factors we present a novel control unit for the rotary blood pumps aimed to decrease LVAD malfunction. Methods: The pump motor windings are leaded out by doubled wires six total via driveline to the extracorporeal splitter where the driveline is divided by two ways. Each way consists of three pump motor phase signals and three current sense signals from the other way. The two identical control units with integrated rechargeable batteries are connected to each way of the splitter simultaneously. This symmetrical peripherals architecture allows each control unit to drive the pump in the active mode or sense the parameters from the parallel way in the passive mode. The control is based on the field oriented control method that allows the passive unit to continuously monitor the direction and magnitude of the magnetic field vector in the pump, which is created by the active unit and, upon loss of control, immediately begins to generate control signals with the necessary amplitude and phase. In case of mismatch of the parameters in any control units, alarms are to occur. Results: Novel Sputnik LVAD was developed according to the described philosophy. This LVAD system has a backup feature which covers almost all peripherals including driveline, control system and battery. In case of the single fault condition in the peripherals the system continues to drive the pump with pause in the control less than 4 ms. Discussion: Developed peripherals integrated in the LVAD allow reducing the risk of the device malfunction and improving usability.Objectives: Polysaccharides belong to the most abundant biomaterials on earth. They form structural elements of plant and animal tissues, but many of them have also important regulatory functions towards cells, tissues and organs. Glycosaminoglycans GAG and other carboxylated and sulfated polysaccharides possess a bioactivity that is due to their high affinity to a plethora of proteins forming insoluble and soluble components of cell environments, but also direct interaction with cell receptors. Methods: Making surface coatings for implants from natural or semisynthetic GAG that guide cell behavior requires different strategies of covalent or physical attachment. Simple covalent binding mechanisms can be based on oxidation of pendant hydroxyl groups of monosaccharide subunits or introduction of reactive side groups like thiols that permit direct coupling to surfaces. Results: Physical can exploit the inherent charge of these molecules that permits formation of multilayers kept together by ion pairing, but also intrinsic cross-linking of activated GAG. Both mechanism can be also used to form 3D structures as biocompatible in situ gelling hydrogels that permit embedding of growth factors and cells. Discussion: These 2D and 3D systems are instructive controlling cell spreading, growth and differentiation, which will be shown with examples on chondrogenic and osteogenic differentiation of mesenchymal stem and other cells.Objectives: A successful creation and adequate maturation of arteriovenous fistula AVF provides efficient treatment and long term patient survival. The aim of the study was to determine the predictors for primary failure of AVF, such as gender, age, AVF site of creation, and primary renal disease, in patients with chronic kidney disease CKD stage 4/5. Methods: The medical records of 178 created arteriovenous fistulae in patients with CKD stage 4/5, in a single center for the year 2018, were retrospectively studied. Primary failure of AVF was defined as thrombosis or inability for cannulation of the fistula within 3 months of creation. Early thrombosis of AVF was defined as an immediate failure due to thrombosis of the fistula within 24 hours of creation. Adequate maturation of AVF was defined as successful cannulation of AVF for efficient HD. Results: The mean age of the patients was 59.75 /- 14.65 years, and 65.16 116/178 were men. Adequate maturation of fistulae was achieved in 83.71 149/178. Primary failure of AVF occurred in 16.29 29/178 of the created fistulae, while 10.11 18/178 had early thrombosis. Distal AVF radio-cephalic was the dominant site with 38.76 69/178 of the created fistulae, followed by middle-arm AVF radio-cephalic site in 32.02 57/178 and proximal AVF brachio-basilic site in 29.22 52/178 of the created fistulae. The distal fistulae were significantly more frequently created in male patients 51 vs 18; p=0,015. The female patients were significantly older than the male patients 63.27 vs 57.86 years; p=0,018. The patients with diabetes mellitus DM were significantly older than the non-DM patients 63.65 vs 58.05 years; p=0,018. Discussion: Hospital strategy for creation of a permanent vascular access for hemodialysis is an "Arteriovenous fistula first". The female gender was more frequently associated with primary failure of AVF, while age, AVF creation site, and primary renal disease were not.Objectives: Polyvinylidene fluoride PVDF and its co-polymer with trifluoroethylene PVDF-TrFE demonstrating biocompatibility and piezoelectric property are among the most promising biomaterials for nerve regeneration. Their blends with polydiphenylene phthalide PDP can improve nerve tissue regeneration due to combined electrical switching effect of PDP. Methods: Electrospinning and film casting were used to produce porous electrospun fibre mats and thin films, respectively. SEM, DSC, RAMAN and FTIR spectroscopy, as well as piezoelectric measurements were performed to analyse morphology and physical-chemical properties of initial and final materials. A special measuring cell was designed to conduct dielectric spectroscopy measurements on samples with both deposited and pressed contacts. Results: Electrospun PVDF-TrFE and PVDF-TrFEPDP fibre mats and casted PVDF-TrFE, PVDF, PVDF-TrFEPDP film samples were produced. The range of fibre diameter of 0.5-1.2 mu m was obtained from SEM images. RAMAN 500-3500 cm-1 and FTIR 600-2000 cm-1 spectra were obtained for initial and final materials. The dependence of the samples capacity on the uniaxial pressure was measured. The frequency dependence of the capacitance and tangent of the dielectric loss angle tgdelta at a constant uniaxial pressure has also been analysed. Discussion: RAMAN spectrum of PDPPVDF-TrFE fibre mats contains peaks from PDP and PVDF-TrFE, indicating that electrospinning method is suitable to produce PDP-containing PVDF-TrFE materials. Results of dielectric measurements showed that the presence of TrFE in co-polymer significantly reduces the value of the samples capacitance. The capacitance of fibrous samples is much lower than that of films due to high porosity of fibre mats >90. TrFE-containing samples demonstrate high dielectric losses at low frequencies. The combined analysis of dielectric and piezoelectric properties is ongoing.Objectives: The objective of this study is the development of a magnetic levitation system of a new type of blood pump. A fully magnetically levitated plate is oscillating in a trapezoidal shaped pump housing to create a relatively blood gentle flow. The oscillating movement of the plate is realized with a combination of a driven harmonic oscillator and a 1-degree-of-freedom magnetic levitation system. Methods: Multiple design concepts utilizing different electromagnetic principles were 3D modelled with Solid Works and imported into Ansys Mawell 3D to execute electromagnetic FEM simulations. Based on the results, the most suitable concept was selected by means of electromagnetic performance, stability, use of space and general feasibility. A PID-controller was implemented using Matlab/Simulink. The feasibility and stability of the concept was tested on a full-scale prototype. Results: The prototype demonstrates the feasibility and stability of the concept. The stabilization of the plate can be realized with an electric current that is about 7 times lower than bearable for the electromagnetic coils of the first prototype, allowing further miniaturization of future prototypes. Discussion: Though the general feasibility of the concept could be demonstrated, tests of the pumping performances are still pending to evaluate the suitability of the pump as an implantable blood pump.Objectives: Long term survival rate after the resection for the esophageal carcinoma has not been satisfactory. One of the most important topics in the area of the esophageal cancer is the reconstruction of the esophageal tract. If the artificial esophagus can be constructed, this technology will be beneficial for patients with esophageal cancer in the advanced stage. Methods: In the previous reports, artificial esophagus with shape memory SMA alloy actuator had been reported. However, total occlusion of the esophageal tract had been difficult. In this study, Origami engineering technology had been used for the peristaltic motion. Triangle shape of the Origami structure had been designed in this study for the total occlusion of the esophageal tracts to prevent the gastroesophageal reflux. The corner of the triangle had been pulled by SMA actuators following the diagonal lines for the folding power. Results: In the design concept of this artificial esophagus, the artificial esophageal tracts will be sutured to the humans esophagus. In this stage, the artificial esophageal tracts were consisted by the artificial vessels material, and in the next stage regenerative esophagus will be the candidates. Esophageal tracts will be covered with triangle shape tracts with SMA actuator to embody the peristalsis. In the results in this study, total occlusion of the artificial esophageal tracts had been embodied by the use of the Origami structure. Through the arrangement of the triangle shape origami structures, co-driving will be embodied for the peristalsis function. Discussion: Until now, there had been no success in achieving peristalsis motion of the artificial esophagus. For the application of the human esophageal tracts, more study is necessary. This study was the first step for the application of the new artificial esophagus in the world for the achievement of the totally occluded peristalsis.Objectives: The transplantation of human induced pluripotent stem cell hiPSC-derived grafts represents an alternative treatment option for post-myocardial infarction patients. So far, myocardial tissue engineering mainly focused on the generation of ventricular-like tissues providing sufficient contractile function. Atrial- and pacemaker-like cardiomyocytes CMs are hypothesized to have a negative effect on tissue function, as they could lead to a lower force development and to arrhythmias after transplantation, respectively. Therefore, we aim to characterize subtype composition in the differentiated CM cultures, as well as in generated tissues thereof and analyze tissue functionality. Methods: The hiPSC-CMs are then mixed with human fibroblasts and a matrix to generate bioartifical cardiac tissues BCTs using our established bioreactor technology. In order to assess the ratio of individual iPSC-derived CM subtypes, flow cytometry and immunofluorescence IF stainings with antibodies targeting subtype-specific markers MLC2v, COUPTF-2, and MLC2a were performed. Results: First results revealed a high percentage 80-90 of MLC2v positive cells in differentiated CM populations. The remaining cells stained positive for atrial markers COUPTF-2, MLC2a. Functional measurements mainly focused on BCT contractility and showed forces similar to the native myocardium 6.5 mN/mm2. In addition, we observed that a higher proportion of ventricular-like CMs leads to higher contractile BCT function. Discussion: Taken together, these molecular and functional results reveal that our differentiation protocol led to the formation of mainly ventricular-like CMs and that BCTs generated thereof show typical characteristics of myocardial tissues. For future ventricular-like tissue replacement therapy, we aim to perform a risk assessment for "contaminating" cardiac cell types with respect to both BCT function and arrhythmias.Objectives: Standardized hemolysis tests according to ASTM F1841-97 2017 are the current gold standard for hemolysis evaluation. As part of the standard flow loop, a screw clamp is used to regulate the flow resistance in the circulations. Especially with extracorporeal pumps used for lung support, cardiopulmonary bypass application, or C02-Removal, resistance has to maintain a pressure gradient of more than 500 mmHg. As a consequence, the screw clamp itself turns into a critical hemolysis hotspot, possibly overlying and biasing the hemolysis caused by the tested pump. Therefore, we present an alternative resistance causing only low hemolysis and at the same time maintaining ASTM testing specifications. Methods: We developed one novel resistance and evaluated the hemolysis potential of the novel and one alternative in flow loops. The flow loops were set up according to the ASTM standard with three identical pumps Deltastream R DP3, Medos AG, two with the novel and alternative resistances each and one with the state-of-the art resistance, the screw clamp. The loops were set to high pressure conditions present in cardiopulmonary bypass applications, ECMO or C02-Removal. The loops operated for 6 hours with porcine blood n = 6. Blood samples were drawn hourly and free plasma hemoglobin was evaluated by means of photometric measurement. Results: A new resistance for ASTM pump evaluation at high pressure regimes with little impact on hemolysis was developed. The resistance is easy to integrate into the recirculation loop and can be adjusted continuously. In-vitro blood trials revealed lower hemolysis for the new resistance compared to the current gold standard. Discussion: Extracorporeal continuous flow blood pumps with transfemoral cannulation work against high pressure at low flow. Up to now, hemolysis evaluation of the pumps is biased by hemolysis caused by the flow loop resistance. Our newly designed resistance causes less hemolysis, thus allowing for more meaningful and reliable hemolysis test results.Objectives: To characterize the biological background of prothrombotic platelet function in the setting of durable Mechanical Circulatory Support MCS with Left Ventricular Assist Devices LVADs and to evaluate influence of different antithrombotic regimens dictated by specific clinical needs. Methods: Platelet thrombin generation was quantified in 75 LVAD patients implanted with the HeartMate II n=10, 13, HeartMate 3 n=27, 36, or HeartWare HVAD n=38, 51 using the Platelet Activity State PAS Assay and the Thrombin Generation Test TGT. Data was compared in patients managed with i oral anticoagulation plus aspirin n=46, 61 or ii anticoagulation alone n=29, 39 due to significant bleeding risk. Coagulation parameters platelet count, International Normalized Ratio INR, activated Partial Thromboplastin Time aPTT, Fibrinogen and D-Dimers levels and hemolysis index lactate dehydrogenase levels, LDH were also recorded to comprehensively characterize hemostatic profiles and actual prothrombotic risk in the two groups. Results: The PAS assay revealed low-intensity increase in platelet prothrombinase activity in patients without aspirin 1.13-fold higher; p=0.04. Similarly, the TGT revealed moderate higher platelet reactivity in patients not on aspirin, consistent with reduction of lag time 0.87-fold lower; p<0.001, increase in peak of thrombin generation 1.5-fold higher; p=0.002 and thrombin generation rate 2-fold higher; p=0.01, but comparable endogenous thrombin potential p>0.05. Coagulation parameters and LDH levels were not different in these two groups p>0.05. Discussion: In patients with a durable MCS, aspirin minimally modulates the biochemical pathway of platelet thrombin generation. Re-evaluation of antithrombotic management criteria in selected patients stratified according to bleeding risk might significantly reduce LVAD hemcompatibility-related adverse events.Objectives: To determine the value of post-reperfusion kidney biopsyin predicting renal function and the incidence rate of DGF in deceased donor renal transplant recipients within 1 year after the surgery. Methods: We retrospectively included 461 patients who accepted the donation after the death of citizen at our center. According to the Remuzzi criteria, we compared the pathological biopsy and graft outcome between right and left donor kidneys from the same donor.Remuzzi score  pound 3 was divided into low group, 4-6 middle group, > 6 high group. The incidence of delayed graft function DGF, acute rejection AR and renal function within 1 year after surgery was observed. A multivariate analysis was conducted on the risk factors of kidney function and DGF incidence. Results: Of the 461 recipients,458 recipients have good graft function and 3 grafts failed. The reperfusion kidney biopsy, the incidence of graft DGF and renal function at each time point after transplantation between the left and right kidneys from the same donor were same p>0.05. 88.1 was in low group, 10.4 in middle group, and 1.5 in high group. The incidence of DGF and the eGFR in different time points after transplantation were different p<0.05. Multivariate analysis revealed that last creatinine before acquisition and Remuzzi score were independent predictors of DGF after transplantation p<0.05. Vascular scores were independent risk factors affecting renal function at 1 year after transplantationp<0.05 Discussion: The reperfusion renal allograft biopsy can reflect the donors baseline data and predict early renal function after transplantation. Renal vascular disease may be an independent predictor of poor prognosis of the graft.The ongoing development of modern manufacturing processes, as well as the automation and cross-linking of production facilities opens up new freedoms for the individualized manufacturing of medical devices. The patient-specific design of these highly loaded devices is associated with a high degree of experience on the part of the designer. The assessment of the safety and performance of these " lot size 1 products" cannot be carried out economically by mechanical testing. This results in a verification gap in the product life cycle of patient-specific medical devices. A verification of the design and thus the evaluation of the function and safety of the products can only be carried out economically with the help of automated numerical simulations. A software to perform virtual strength assessment of custom implants was developed. The lecture shows how the verification process of patient-individual designs with numerical methods, can contribute to the increase of patient safety. In addition, the integration and documentation of numerical simulations within the product life cycle will be presented using industrial examples.Objectives: One of the most pressing issues of regenerative medicine is the fabrication of constructs with defined parameters in order to create conditions for proper cell adhesion and proliferation that will be close to native tissue conditions. An application utilizing the electrospinning method may potentially solve this problem. The main goal of this study was to fabricate electrospun scaffolds based on silk fibroin and to characterize their properties. Methods: Silk fibroin scaffolds were obtained by electrospinning method. There were two types of scaffolds: 1 silk fibroin scaffolds, 2 silk fibroin scaffolds with addition of 30 per cent of gelatin by mass. Scaffold structure was investigated by scanning electron microscopy and scanning probe nanotomography. Cell adhesion and proliferation were investigated in mice fibroblasts 3T3 model. The regenerative potential of scaffolds was estimated in a model of Wistar rat full-thickness skin wound regeneration. Results: The electrospun scaffolds have fibrous porous structure with an average fiber thickness 300-600 nm. Both types of scaffolds are biocompatible in vitro. Presence of gelatin in scaffold composition improves cell proliferation. Scaffolds promote skin wound regeneration and restore native skin structure. Histological analysis did not evolve inflammatory process. Discussion: Electrospinning is a promising method to obtain fibrous biomimetic constructs with defined properties. The biocompatibility of produced constructs was shown in vitro and in vivo. Both types of scaffolds have a significant regenerative potential and are prospects for different fields of regenerative medicine.Objectives: Biodegradable PCL-based electrospun scaffolds are often used in TE. However, such scaffolds lack biocompatibility, wettability and chemical accessibility for tendon replacement applications. Here, we propose a new method to overcome those shortcomings by using PCL:Chitosan-graft-PCL PCL:CS-g-PCL blends instead of post-process coatings. Methods: Blended solutions of PCL 175 mg/ml in 2,2,2-TFE and CS-g-PCL 1200 mg/ml in chloroform were electrospun into scaffolds with CS-g-PCL from 0 to 90 wt 10 ratios. SEM based measurements were carried out to investigate fiber diameter and morphology. Chitosan amino groups within the fibers were quantified by nuclear magnetic resonance spectroscopy H-1-NMR and x-ray photoelectron spectroscopy XPS. DSC measurements were conducted to assess thermal properties and crystallinity. Subsequently, metabolic activity of human mesenchymal stromal cells hMSC seeded on the scaffolds, was determined over 7 days. Results: Blends with CS-g-PCL contents < 90 wt led to fiber diameter of 1 /- 0.35 mu m and to 2 /- 0.38 mu m for 90 wt. XPS measurements showed an enrichment of chitosan amino groups at the surface of the fibers. The enthalpy of fusion increased by up to 40 J/g for CS-g-PCL contents from 0 to 90 wt, indicating a significant increase in crystallinity. hMSCs had maximum metabolic activity on samples with CS-g-PCL contents between 10 and 30 wt. Discussion: Electrospinning of blended PCL:CS-g-PCL solutions resulted in higher crystallinity while preserving mechanical properties. Furthermore, increased wettability improved biocompatibility and chemical accessibility, which ultimately influenced the metabolic activity of the hMSCs positively. This promising novel approach will increase the biocompatibility of PCL-based scaffolds with simultaneous reduction in production complexity.Objectives: Biventricular assist device BVAD is reported to be used in 3-5 of continuous-flow VAD cf-VAD. Our recent strategy is to stabilize INTERMACS profile 1 or unstable profile 2 patients by paracorporeal VAD, central ECMO or IABP before proceeding to cf-VAD. Patients with RV failure are also aggressively treated by PDE-5 inhibitors or endothelin receptor antagonist at the outpatient clinic after implant. We examined validity of this strategy by analyzing BVAD necessity and patient survival. All cf-VAD was used for bridge to transplantation. Average waiting time for heart transplantation is nearly 3.5 years in our country. Methods: 166 consecutive patients were implanted with cf-VAD. DCM was seen in 113, ICM in 14, dHCM in 12, post-myocarditis CM in 6, ARVC in 3, RCM in 3 and others in 15. 130 patients underwent primary cf-LVAD implantation. Pre-implant profiles of those patients were 2 in 62, 3 in 62 and 4 in 6. 36 patients were converted from another type of mechanical circulatory support MCS. Actuarial survival was calculated by Kaplan-Meier analysis and comparisons were made using Log-rank method. Results: No patient required RVAD at the time of implantation nor late after implantation. Average support duration was 787 days, and 53 patients were on the support longer than 3 years. During follow-up 56 patients reached transplantation, 9 weaned from VAD due to functional recovery and 80 on-going support. 1-, 2-, 3-year survival of 166 patients was 92.8, 89.5 and 85.9. There were no statistical differences between subgroups, such as sex, pre-implant profile, body size and primary/redo sternotomy. Discussion: We successfully implanted cf-LVAD and managed in the outpatient clinic without RVAD support in 166 consecutive patients, including those who were severely ill in the pre-implant stage. Our strategy for stabilizing profile 1 and unstable profile 2 patients by other types of MCS and aggressive use of PDE-5 inhibitors or endothelin receptor antagonist seemed to be safe and effective.Objectives: Recently, composite bionanomaterials containing biological materials and carbon nanotubes CNTs have been actively studied. CNTs may contain catalytic ferromagnetic nanoparticles and they can affect the properties of the composite bionanomaterial. To the study of this issue, we paid attention. The object of the study was a composite nanomaterial consisting of bovine serum albumin BSA - matrix and single-walled CNT SWCNT - filler. Methods: Aqueous dispersion in the composition of 20 wt. BSA and 0.01 wt.. SWCNTs were prepared using traditional nanotechnology methods thorough mixing, ultrasonic homogenization, decanting, etc.. The optical density of dispersion was measured after a magnetic field gradient  1 T/m was applied to it for 100 hours. The obtained values were compared with the optical dispersion density in the absence of a magnetic field. Recent samples were considered control. Results: It was found that under the action of a magnetic field, the optical density of the water dispersion of BSA/SWCNT in the region of 300 divided by 800 nm decreases by > 10 relative to control samples. Additionally, it was found that under the action of a magnetic field, the optical density of the aqueous dispersion of BSA/SWCNT decreases by > 10 relative to control samples. Discussion: The decrease in the optical dispersion density of BSA/SWCNT is caused by the deposition of nanotubes on the cell surface under the action of a magnetic field gradient. It is assumed that the single walled carbon nanotubes contain catalytic ferromagnetic nanoparticles. Estimates showed that in bionanomaterial at a level with concentrations > 0.0001 wt. CNT nanotubes can be fixed with modern sensors of a weak magnetic field with a threshold sensitivity of <= 1 pT. It was concluded that changes in implants in the BSA/SWCNT composite bionanomaterials can be observed non-invasively with the help of weak magnetic field sensors for example, SQUID or CMFS.Objectives: Recent developments in imaging technology have enabled cryobiologists to reveal some mechanisms employed by nature to cope with freezing stress. The systematic study of these mechanisms may prove useful for developing ultimately novel and efficient cryopreservation protocols. In this work, we examine the potential advantages of using infrared video thermography IVT for monitoring latent heat release and freeze-thaw events in 3D porous collagen-hydroxyapatite scaffolds used as a model system. In this context, we present ongoing results on the identification of warm ice-nucleating agents INA from Hippophae rhamnoides with potential use in cryopreservation of tissue-engineered constructs TECs. Methods: The scaffolds were prepared by freeze-drying of mineralized collagen suspensions and characterized by Raman microscopy, X-ray microcomputed tomography and differential scanning calorimetry. Visualization of freezing/thawing events in scaffolds frozen with and without cryoprotective solutions based on leaf homogenate from Hippophae rhamnoides was performed using the thermography-based platform developed at the University of Thessaly. Results: Scaffolds presented 3D porous architecture and characteristic RAMAN peaks of collagen type I and hydroxyapatite HAP. Presence of HAP decreased the specific heat capacity of collagen scaffolds. The probability of ice nucleation was higher in cryoptotective solutions with scaffolds than in solutions without. The ice nucleation temperature within scaffolds using INA was determined to be around -1.5 degrees C. Discussion: IVT proved to be a very effective approach for visualizing latent heat release and freeze-thaw events within scaffolds frozen in cell culture plates. INA from Hippophae rhamnoides may potentially be used for prevention of supercoiling in cryopreservation. The results obtained could serve as a basis for further development of efficient cryopreservation protocols for 3D TECs.Objectives: In the development process of heart assist devices, numerical simulations become more and more essential. The variety of applications comprises different types of cannulas or flow geometries of ventricular assist devices VADs. The analysis of the flow field inside VADs by means of Computational Fluid Dynamics CFD is extremely useful. Numerical parameter studies allow the treatment of several device related problems in conjunction with a well-balanced cost-benefit ratio. Methods: Results obtained from properly performed CFD calculations permit the determination of flow induced blood damage of new flow designs in VADs and enable the prediction of the risk of harmful hemolysis for the patient. The literature provides different models for blood cell damages. In application, a proper definition of acceptable thresholds is required, in order to optimize blood guiding geometries as rotor blades or diffusor shapes in VADs. Another application of CFD can be found in the field of flow path optimization of new cannula designs. Corresponding results can lead to various design changes and can significantly contribute to a comprehensive risk analysis for regulatory assessment. Results: Based on the numerical results, geometrical design iterations can considerably improve the flow design of blood pumps and cannulas. Thoroughly optimized designs exert reduced shear stresses to the cells and create less damage to the patients blood. Furthermore, CFD studies can provide valuable information in the process of risk analysis during the regulatory assessment of heart assist devices and support the interpretation of experimental in-vitro investigations. Discussion: Although numerical simulations represent a commonly used tool during research and development in the field of heart assist devices, the introduction of strict requirements and well-defined guidelines would considerably simplify the implementation of numerical studies for regulatory approval and enhance the comparability of independently gained results.Objectives: Current standards in cryopreservation for biomedical technology are still based on slow freezing processes inducing crystallization within the sample. In preparation for this cryopreservation regime, cells and tissues are detached from growth surfaces and dissociated into single cell suspensions. Existing infrastructures like freezing devices, lab ware, and liquid nitrogen tanks for long-term storage, are designed for the handling of suspended cells and tissues and for stock keeping purposes, these procedures usually are efficient enough. Through the addition of cryoprotective agents, the sample can cope with the emerging damaging mechanism resulting from ice crystal formation. However, diagnostically and therapeutically relevant cell systems, e.g. human induced pluripotent stem cells hiPSCs or their derivates like neuronal cells, depend on cell-cell and cell-surface interactions mediated by cytoskeletal proteins. Their structures are disordered, either due to these preparatory dissociating steps or due to crystallization, resulting in loss of functionality or cell loss in general. Methods: A second cryopreservation regime, the vitrification, avoids crystallization within the sample and thus avoids the occurring damaging mechanisms e.g. shear forces, osmotic shock. Thereby, the structure of cells and tissues is preserved, leading to viable and functional cells even in their adherent state with cell-cell and cell-surface contacts after thawing. However, vitrification requires a skilled handling and so far, the infrastructure has not been established to enable routine vitrification workflows for large cell numbers. Results and Discussion: Our work shows the potential and capacity of vitrification on different cell types e.g. hiPSCs, dopaminergic neurons. We demonstrate that vitrification is superior to slow freezing regarding cell loss and functionality and holds the possibility to cryopreserve scalable and ready-to-use cell products for a broad range of biomedical research.Objectives: The aim of this work is to characterize the decellularized human chorion membrane dHCM and to study its suitability as a substrate to model biological barriers. Methods: HCMs were submitted to a chemical and physical decellularization process. The decellularization protocol was optimized by quantifying and analyzing the presence and distribution of cell nuclei and dsDNA by DAPI staining, PicoGreen and electrophoresis. The dHCM was characterized by scanning electron microscopy SEM, immunohistochemistry, Dot-Blot, SDS-PAGE, and GAGs and Collagen quantification. Moreover, dHCMs mechanical properties were assessed and biocompatibility tests metabolic and proliferation assays with an endothelial human cell line EA.hy.926 were performed. Results: The decellularization process of the HCM was successful, since the presence of nuclei was not observed, the amount of dsDNA was around 10 ng/mg of dry tissue, and the DNA fragments had less than 200 bp. A compact membrane that preserved the HCMs reticular layer and basement membrane was obtained. SEM images revealed that dHCM is a substrate with two different surfaces: one composed by nanofibers and the other in trophoblast layers side covered by a thin compact layer basement membrane. In tissue sections it was possible to verify the presence of collagen type I and IV, fibronectin and laminin in both dHCM and native tissue. Corroborating these results higher molecular weight species were preserved in dHCM SDS-PAGE. Additionally, dHCMs mechanical properties were determined: youngs modulus of 5.9 MPa and ultimate tensile strength of 5.3 MPa. Differences in metabolic activity and cell proliferation were observed between both sides of the dHCM. Discussion: dHCM retained the main proteins of the extracellular matrix. Cell proliferation and viability were higher in the side of dHCM that preserved the basement membrane. dHCMs physical and mechanical properties make it a suitable substrate to mimic biological barriers.Objectives: Cryogels with improved non-fouling and cell adhesive surface properties were investigated as potential cell scaffold for BAL purpose. Cryogel monolith columns were used by other groups to fill the bioreactor chamber. However, device performance was not maintained for more than a few hours. Fluid dynamic measurements using a purpose-built mu PIV setup with video post-processing were conducted in order to extrapolate the effect of flow inside the cryogel matrix to improve cell viability and avoid blood cell activation. Starting from PIV results, a multi-layered bioreactor composed of spaced cryogel discs was developed to maximise blood/hepatocyte mass-exchange. This study aimed to investigate whether the multi-layered design results in improved performance compared to the column version in terms of hepatocytes viability and functionality. Methods: pHEMA-co-MBA cryogels were synthesised by cryogelation technique. Cryogels were then functionalised with alginate and RGD peptide and synthesised by solid phase method. Porous structure was analysed with SEM, confocal imaging and mu CT. Non-fouling properties were investigated by protein absorption studies. Cell viability was assessed by MTT/ATP activity and live/dead imaging. Hepatocyte functionality in bioreactors was investigated by quantification of albumin and urea production using ELISA and colorimetric assay, respectively, over time. Results: Synthesised cryogels possessed an open porosity with pore sizes of up to 100 mu m and an interconnected network of pores. Alginate helped preventing protein absorption from plasma. RGD peptide enhanced hepatocyte functionality. Cryogel perfused bioreactors maintained hepatocytes viability and functionality for up to 1 week. Discussion: Multi-layered bioreactor design showed a significantly higher production of albumin and urea compared to the column version, suggesting an improved mass exchange between medium and cells. Also cell colonization and proliferation through the device were improved.Objectives: Placental complex, including placental tissues, foetal membranes, and umbilical cord, is one of the most promising sources of a range of tissues and cells for clinical and experimental application in regenerative medicine and tissue engineering. Certain components of placental material are already applied in medicine since decades, while others undergo intensive research and pre-clinical trials. Since placenta is a temporary organ, it provides the unique opportunity to obtain large amounts of tissues, including autologous, without the adverse effects to the donor. Moreover, great potential of placental multipotent stromal cells is recognized in generation of tissue engineered multicellular constructs. At the same time, clinical use and research proceedings are significantly restricted due to imperfections in storage technologies. Application of low-temperature preservation methods is especially challenging for 3-D multicellular structures, such as organs, tissues, and tissue engineered constructs. Methods: Here we aim to analyze the variety of cryoinjuries in a range of placental tissues and tissue engineered multicellular placental constructs, as well as support the cellular viability after low temperature storage. A range of cell and tissue culture as well as tissue engineering methods, cryomicroscopic analysis, evaluation of phenotypic and metabolic characteristics have been applied in this study. Results: Results have shown that the mechanisms of damage caused by cryopreservation procedures depend on particular structure of the studied object, peculiarities of intercellular bonds, as well as interaction with cryoprotective agents. At the same time, developed protocols allowed us to efficiently isolate viable cells after thawing of the studied tissues. Discussion: Conclusions of this study may lead to progress in improvement of biobanking technologies for tissues and tissue engineered multicellular constructs.Extracorporeal therapies, such as hemodialysis, apheresis or blood oxygenation, are only possible when recurrent blood access is feasible. Both, artificial grafts, e.g. made of polyethylene-terephthalate Dacron R or a fistula are not ideal models for that purpose and are commonly considered to represent the Achilles Heel of extracorporeal blood circuits. A stable, implantable biologic vascular graft would be an ideal solution for the benefit of patients and many bioengineers are working on this dream to come true. When considering such a vascular graft to be used clinically and as a product to be sold in the medical device market, a series of problems with regard to approval have to be solved and some fundamental questions be answered. Regarding approval, current EU regulations refer to either the Medical Device Regulation MDR from 2017 or to Advanced Therapy Medicinal Products Regulation ATMPs from 2007. Under the premise "Better safe than sorry" regulations, such as MDR and ATMP, represent a European law. A bioengineered vascular graft is taken as a "combination product", as it is commonly made of biological cells and a possibly biodegradable scaffold material. When implanted, it would represent a high-risk class III medical device under the MDR regulation and would require a CE-mark. The ATMP regulation, however, says that a combination product has to be approved following the "principal mode of action". For products containing viable cells or tissues, a specific approach has been taken by the EU. For these products it holds true that "whatever the role of the medical device, the pharmacological, immunological or metabolic action of these cells are considered to be the principal mode of action". Under these premises, a combination product, such as a bioengineered vascular graft, is regulated under the ATMP rules, which renders approval rather complex. Consequences refer to the need of Phase I-III clinical trials including post-market surveillanceNeuroengineering is a research area where engineering skills are used to solve basic and clinical problems in neuroscience. The aim of the presentation is to give an insight into neuroengineering applied in neurosurgery. Deep brain stimulation DBS is a method for reducing symptoms in movement disorders as Parkinsons disease and essential tremor. A thin electrode is surgically implanted with stereotactic technique at a well-defined brain structure. Our research focus on support systems for improving the surgical implantation procedure, and to increase the understanding of the DBS mechanism. A method for performing patient-specific simulations of the electric field around a DBS lead and map the data to the anatomy has been developed. The goal is to create improvement and side-disorder maps for the most common symptoms and brain targets used in DBS. Optics is another core area we apply in neurosurgery. Several optical probe systems for intraoperative guidance have been developed. Forward-looking probes can act as "vessel alarm" by recording the tissues microcirculation with laser Doppler flowmetry during creation of the brain trajectories in relation to stereotactic neurosurgery. In DBS this method has been evaluated in the clinical setting in more than 100 implantations. For brain tumor biopsies the method is presently introduced in combination with 5-ALA fluorescence measurements at the tumor border. 5-ALA induced fluorescence spectroscopy- and microscopy can also be used together in brain tumor surgery. Under the neurosurgical "blue-light" microscope, a hand-held probe supports the surgeon by enhancing malignant tissue. Another challenge is optical monitoring of cerebral microcirculation in the neuro-intensive care setting. A prerequisite for successful project outcome is a close collaboration between biomedical engineers and neurosurgeons. difference.Objectives: The main goal of this study was to synthesize polysaccharide-based hydrogel systems that favor bone regeneration. The hydrogel formation is driven by Schiff base crosslinking reaction between the amino functionalities of carboxymethyl chitosan CMChi and the aldehyde groups of the different oxidized polysaccharides resulting in an imine bond formation. Methods: Different amounts of NaIO4 were used in order to oxidize Alginate ALG and Hyaluronic acid HA, forming reactive dialdehyde derivatives with different oxidation degrees. The amount of aldehyde groups was determined by: A using Schiffs reagent and detected by a UV-Vis spectrometer 550nm and B acid base titration after reacting the aldehyde groups with Hydroxylammoniumchloride. Gel Permeation Chromatography GPC was used for molecular weight determination of the different polysaccharides. Biocompatibility of the products was tested using Qblue R assay and CFDA staining. Results: Data obtained from both UV-VIS spectroscopy and titration show that the different polysaccharides have more aldehyde groups when high concentrations of NaIO4 were used and the reaction times were extended. The weight average molecular weights of all the products were lower after the oxidation step. Products with high oxidation degrees showed low biocompatibility. Discussion: ALG and HA with high oxidation degrees are required to prepare stable hydrogel systems and the dual utility of using CMChi and oxidized polysaccharides as both structural and bioactive component is promising for the formation of biocompatible hydrogel systems that can be used for bone tissue regeneration.Objectives: Xenotransplantation of transgenic porcine pancreatic islets offers a promising alternative source to circumvent current limitations posed by the scarcity of allogeneic donors. However, either immune rejection or oxygen supply in immune protected encapsulated islets remains a major concern. To decrease xenogeneic immune responses, we have investigated the feasibility to generate tissue engineered SLA silenced islet cell clusters ICC from alpha-Gal knock out, CD46, CD55 and CD59 transgenic minipigs. Methods: Pancreatic islets single cell suspensions were generated by enzymatic digestion of porcine ICCs. The single cells were silenced for SLA class I and II by lentiviral vectors encoding for Nanoluciferase as reporter gene and for short hairpin RNAs targeting beta2-microglobulin shb2m or class II transactivator shCIITA, respectively. SLA transcripts were evaluated by real-time PCR and protein levels by flow cytometry and fluorescence microscopy analyses. Cell death was evaluated by Propidium Iodide staining. The effect of SLA class I silencing was evaluated in human T and NK cell cytotoxicity assays. SLA-silenced pancreatic beta-cells were then used to form new ICCs in stirred bioreactors. Results: SLA class I silencing was designed to reach a level of up to 86 and class II by up to 64 on pancreatic islet cell monolayers. Silencing SLA expression did not affect cell viability and the insulin-producing beta-cell phenotype as indicated by Dithizone staining and levels of insulin production. Xenogeneic T-cell immune responses p<0.05 as well as antibody-mediated cellular-dependent immune responses p<0.01 were significantly decreased. Silencing SLA class I expression did not increase susceptibility to NK-cell cytotoxicity. In stirred bioreactors, tissue engineered islets showing the typical 3D-structure and morphology of ICC were assembled from SLA-silenced pancreatic cell suspensions to be used for transplantation in humanized mice as a first model. Discussion: These data shows the feasibility to generate low immunogenic porcine ICC from transgenic pigs after single cell engineering and post- transduction islet reassembling that might serve as a robust alternative to allogeneic pancreatic islet cell transplantation.Objectives: In patients with circulatory support, preload is a valuable clinical monitoring variable. While it is difficult to measure directly, it can be indirectly measured by PCWP or left atrial pressure LAP. Typically, assessments of LAP require invasive methods. However, non-invasive estimation of LAP can be a useful tool for monitoring patients and adjusting pump speed to reduce the likelihood of hazardous events, such as ventricular suction and pulmonary congestion. This study presents an artificial neural network model estimating the LAP noninvasively across different patient conditions. Methods: LAP was estimated based on the HeartWare HVAD pump characteristics current, flow and speed derived from 186 experiments performed in a mock circulatory system with HVAD speed range from 1800 to 3600 rpm. The LAP estimation was evaluated by measuring the hemodynamic variables in response to a range of changes to the mock cardiovascular system, including changes in PVR, SVR and HR. A neural network was trained with 70 of the data randomly chosen, and was validated and tested with the rest of data. The model performance was quantified by the mean squared error MSE, correlation coefficient r2 and Bland-Altman plot between measured and estimated LAP in different scenarios. Results: The neural network model was able to estimate LAP accurately using data collected in-vitro for training, validation and test sets. The MSE for all data is 1.36 mmHg, and correlation coefficients are 0.99, 0.91 and 0.94 for training, validation and test sets, respectively. The Bland-Altman plot showed that error has a bias of 0.28 mmHg with a reproducibility coefficient of 2.2 mmHg. Discussion: The developed model would be useful for a real-time estimation of preload noninvasively which can be employed to adjust pump speed in heart-failure patients without inserting a pulmonary catheter into the pulmonary artery. Implementation of the developed estimator would also create a new physiological control system that can work noninvasively.Objectives: Blood vessels are integral to the maintenance of all tissues. They deliver oxygen and nutrients and remove waste. Blood vessels not only contain endothelial cells ECs that are in direct contact with flowing blood cells and fluids but they may also have pericytes and smooth muscle cells on the outside that stabilize and regulate their size. Defects in EC and pericyte/smooth muscle cell interaction are often implicated in many pathological conditions. Patients with a genetic disease called hereditary hemorrhagic telangiectasia HHT suffer from heavy and recurrent nosebleeds, due to unstable blood vessels as a result of defective interaction between ECs and pericytes. Methods: Here we used human induced pluripotent stem cells hiPSCs as a source of patient-specific cells, as they can be derived from all individuals, including children and patients with genetic disease and subsequently differentiated towards all cells of the body. Besides, we can now use hiPSCs to derive all the component of blood vessels in large numbers and use them to re-create blood vessels on a chip to model inflammation and disease. Results: HHT-hiPSC-ECs displayed no apparent functional differences using a set of standard two-dimensional 2D assays. The ability of hiPSC-ECs to form a perfused microvascular network was next examined using three-dimensional 3D cultures in microfluidic chips. Using these 3D vessels on a chip, we found that in contrast to 2D microvascular cultures, the ability to form 3D microvessels in microfluidic chips was strikingly compromised when HHT-hiPSC-ECs were used compared to isogenic control ECs. Discussion: This patient-based hiPSC model thus serves as the first proof of principle that vascular diseases could be modeled using patient-specific hiPSCs in 3D microfluidic chips and to identify new target cells and possible pathways for therapy.Objectives: This study was aimed at fabricating various layer-by-layer LbL systems using glycosaminoglycans GAGs with the capability to bind BMP-2 specifically in order to control osteogenic differentiation of cells by biocompatible release systems. Methods: Heparin, chondroitin sulfate and their oxidized forms as polyanions were combined with chitosan and collagen I as polycations to form various multilayer coatings on model materials with the advantage of the intrinsic cross-linking formed between oxidized glycosaminoglycans GAGs and polycations, improving multilayer stability and affecting the release of BMP-2. The myoblast cell line C2C12, which can differentiate into osteoblasts was seeded on 5 mu g/mL BMP-2 loaded multilayers. Cell viability, adhesion, osteogenic differentiation and BMP-2 release were investigated. Results: C2C12 cells cultured directly on the top of multilayers showed that particularly BMP-2 loaded multilayers made of oxidized GAGs promoted an osteogenic differentiation that was nearly comparable to the positive control, when 5 mu g/mL BMP-2 was added directly to the medium. Interestingly, the BMP-2 had synergistic effect on cell adhesion and spreading. BMP-2 in oxidized chondroitin sulfate multilayers was successfully loaded to the layers, sustainably released over time and affected cell differentiation more than the soluble BMP-2. Discussion: The results show that oxidized GAGs forming intrinsically cross-linked multilayers are useful as reservoirs for sustained release of BMP-2 in which the intrinsic cross-linking affected BMP-2 release, improved multilayers stability due to the resulting stiff surface compared to the native ones, supported cell adhesion, proliferation and subsequent differentiation. This can pave the way for coating implants and scaffolds for repair and regeneration of bone fractures.Objectives: Functional myocardial recovery during support with left ventricular assist devices LVADs is highly desirable but only observed in approx. 1 of all patients. Previously, the more frequently implanted pulsatile blood pumps PBPs showed higher recovery rates than the currently used rotary blood pumps RBPs. The aim of this study was to comparatively assess the capability of PBPs and RPBs to unload the left ventricle and restore physiological cardiac energetics in isolated hearts. Methods: An RBP Medtronic HVAD and a heartbeat synchronized PBP BerlinHeart Excor were alternately connected to eight isolated porcine hearts. To modulate the mode of LVAD support, RBP rotational speed was set to different support levels and ejection delay of the PBP was phased from 0 to 90. Pressures, flows, left ventricular volumes, and myocardial oxygen consumption were recorded and cardiac efficiency quantified by the ratio of external work EW over myocardial oxygen consumption. Results: With increasing RBP support, lower left atrial pressures LAP were found to coincide with a decreased cardiac efficiency r=0.91 /- 0.12 with a median inter quartile range slope of 0.28 0.91 /mmHg. In contrast, depending on the phase delay of ejection of a PBP, LAP and cardiac efficiency follow a sinusoidal course with the LAP minimum at 90 and efficiency maximum at 60. Discussion: In RBP support, aggressive ventricular unloading was found to lead to lower cardiac efficiencies. On the other side, phasing of a PBP offers the possibility to restore cardiac efficiency and simultaneously unload the ventricle. These results may justify future studies linking optimized cardiac mechanics and efficiency to functional recovery and reverse myocardial remodeling with LVAD therapy.Objectives: We focused on the interventricular dyssynchrony caused by the shortened systole of left ventricle LV compared to right ventricle RV during left ventricular assist device LVAD support. The purpose of this study was to assess and quantify the mechanical interventricular dyssynchorny according to the support condition using conductance catheter. Methods: We studied five goats with normal heart condition. A centrifugal LVAD was implanted under general anesthesia. We inserted the conductance catheter into the LV and RV and obtained the pressure-volume relationship. We defined the dyssynchronous status by the sign plus or minus of the LV volume LVV change is opposite to that of RV volume RVV. i.e., dLVV/dtdRVV/dt<0. Interventicular dyssynchrony DYS was quantified by calculating the percentage of time within the cardiac cycle. Bypass rate was set dividing the LVAD pump flow by the main pulmonary artery flow. We calculated the DYS under LVAD support with various bypass rate. Results: The mean DYS of normal heart, LVAD clamp, LVAD support with bypass rate of 50, 75  and 100 were 6.0 /- 1.3, 8.6 /- 2.6, 9.6 /- 1.8, 15.5 /- 5.6, and 24.6 /- 8.6, respectively. Also, there was a higher linear negative correlation between LV stroke volume and DYS. Discussion: Interventricular dyssynchrony during LVAD support was assessed and quantified using conductance catheters. Dyssynchrony became significant when the LV was unloaded with high rotational speed under LVAD support. Assessment of influences for the right ventricular function and heart failure models will be studied in the further study.Objectives: Biochemical modification of implants with drug delivery systems can support the bodys natural healing capabilities by imitation of smooth tissue transitions along bone-tendon conjunctions. Modified interconnected 3D porous polyepsilon-caprolactone could act as a scaffold material for these implants. Methods: Cocontinuous blends of polyepsilon-caprolactone PCL and polyethylene oxide PEO were prepared by batch blending in an internal mixer. Annealing and leaching of the blends with water yielded the porous PCL scaffolds, which were then modified with chitosan CS grafted PCL CS-g-PCL via dip coating. Nanoparticle systems containing CS and tripolyphosphate TPP were obtained via ionotropic gelation. These CS-TPP nanoparticles CSNPs can be attached to the modified scaffolds via a Layer-by-Layer LbL deposition approach involving alginate ALG. Results: Interconnected 3D porous PCL scaffolds with different morphologies have been prepared. Modification with CS-g-PCL, alginate, alginate fluorescein amine ALG-FA, CS-TPP and fluorescein isothiocyanate labelled nanoparticles CS-FITC-TPP in a LbL approach was carried out. The blank scaffolds were analyzed in various ways, including via scanning electron microscopy and mercury intrusion porosimetry. Modified versions of the scaffolds were analyzed accordingly and the fluorescently labeled species were investigated via confocal laser scanning microscopy. CSNPs were loaded with bovine serum albumin BSA and layered on the modified scaffolds. Release of BSA from the scaffolds was investigated by ultraviolet-visible spectroscopy at 37 degrees C in phosphate-buffered saline PBS. Discussion: Porous PCL scaffolds with different pore sizes and morphologies can be obtained via annealing and leaching of cocontinuous PCL/PEO blends. Surface modification with CS-g-PCL, alginate and CSNPs open up possibilities regarding graded implant fabrication.Objectives: Vascular calcification is accelerated and worsened in chronic kidney disease CKD and dialysis patients compared to the general population. Misbalanced phosphate and calcium metabolism plays an important role in this health burden in CKD. In this study the role of phosphate and calcium in inducing medial vascular calcification is researched. Methods: Rat aortic rings were incubated in the normal and calcifying mediums for seven days. Three independent experiments, three rings per condition in each, were performed N=9. Normal medium consisted of DMEM with 0.9 mM phosphate and 1.8 mM calcium. Calcium medium Ca consisted of 4.0 mM calcium and 0.9 mM phosphate; phosphate medium P of 2.8 mM phosphate and 1.8 mM calcium; and calciumphosphate medium CaP of 4.0 mM calcium and 2.8 mM phosphate. Histochemical von Kossa staining was performed, and calcification area of a medial and adventitial section of the aorta was calculated. The results were compared using a two-way ANOVA test. Results: Von Kossa stainings and evaluations of the calcified area demonstrated increased calcification in aortic rings incubated in different calcification mediums, whereas the calcification takes part in different sections of the aorta. Calcium alone causes calcification in the adventitia, phosphate alone in the medial section of the aorta. Combination of both leads to extensive calcification in the adventitia and media. Discussion: In dialysis patients, serum calcium levels are maintained quite well, but serum phosphate levels may exceed suggested levels multiple times. The results of the study demonstrate that high phosphate level is the trigger of excessive vascular calcification and the control of levels of phosphate in CKD patients is crucially important.Objectives: Heterogeneously structured electrospun composite scaffolds have performed well as ECM-mimetic tissue replacements in recent years. In light of this, we present an assessment of amnion derived multipotent stromal cell aMSC growth on hybrid polycaprolactonePCL-gelatinG scaffolds in terms of viability, infiltration depth and metabolic activity. Methods: Four scaffold types were electrospun 150 mu m thick with different PCL:G ratios and fibre diameter distributions using vertical V and horizontal H orientations - PCL175V 175 mg/ml, homogenous, PCL125G50H 125:50 mg/ml, homogeneous, PCL125G50V 125:50 mg/ml, heterogeneous, PCL100G75V mg/ml 100:75 mg/ml, heterogeneous. A 7 day live/dead assay and a 15 day infiltration study depths measured using confocal laser scanning microscope with aMSCs from Callithrix jacchus were performed. Metabolic activity was measured by MTT assay 7 days. Additionally, gelatin loss during scaffold degradation 30 days was measured by Raman spectroscopy. Results: In general, PCL-gelatin blends perform better than unblended PCL. PCL125G50V shows the deepest cell infiltration on all days 76.3 /- 7.1 mu m by day 15. It is also the only sample that shows an increasing trend in metabolic activity from day 1 0.0436 /- 0.002 to day 7 0.0996 /- 0.009. Raman spectroscopy revealed a significant gelatin loss of 53.44 /- 6.48  in PCL100G75V while PCL125G50V showed no significant loss by day 15. Discussion: A combination of heterogeneous fiber morphology and composite polymer content in electrospun scaffolds positively affects cell growth and infiltration. Considerable gelatin swelling during degradation can physically inhibit infiltrating cells. Therefore, choosing the right ratio of PCL:gelatin when designing such scaffolds is key.A total artificial heart is used for orthotopic replacement of the native ventricles in cases of irreversible biventricular heart failure resulting either from chronic dilated cardiomyopathy or fulminant acute myocardial infraction. It is essential to find the right indication. There are several surgical options to perform a total artificial heart operation. To prevent therapy-failure it essential to set the right indication for implanting a total artificial heart. In general therapy with a TAH is feasible with good postoperative results. Setting the right clinical frame is crucial for the establishment of a therapy concept.Objectives: Although diabetes is the leading cause of end-stage renal disease in many countries, data about its impact on patient-specific reaction to haemodialysis HD is scarce. Models that allow identifying and predicting patient-specific response to HD are in development and their improvement is encouraged by clinicians to obtain suitable decision support systems DSS. Here, possible correlations between diabetes and alterations in the patient-specific parameters identified by a multi-pool, multi-solute model, developed in our laboratory, are investigated. Methods: Data acquired during the DialysIS Interreg Project 141 patients, 4 centres, 1276 HD have been used. For each patient, a set of specific parameters was identified: one modulates the permeability at the capillary wall rho, the others account for altered transport of catabolites and electrolytes at the cellular wall ki and at the dialyser membrane etai, where i indicates different solutes. Patients have been classified as diabetic D or not ND and among Ds, as insulin-dependent DID or not DNID. Initial glucose concentrations and identified values of rho, ki, eta i in the different groups have been compared using proper statistical tests p<0.01. Results: There were were 37 26,4 D patients and 16 of them were DID 43,2. Initial plasma glucose concentration statistically differs both between D and ND and between DID and DNID patients. Among the patient-specific parameters rho, and K-Na, K-Mg, K-Creatinine and K-Bicarbonate results were different between D and ND. The values of etaNa, etaMg, etaCa, etaUrea and etaCreatinine, instead, significantly differ between DID and DNID. Discussion: Diabetes-induced autonomic and peripheral neuropathies can justify the finding of altered capillary wall permeability in D; moreover the use of insulin, protein based, seems to impact on the mass transport through the dialyzer membrane. The proposed approach allows adding a new tile to the studies on how the diabetic pathology affects the response to the HD treatment.Objectives: Left Ventricular Assist Device LVAD therapy has become an accepted treatment for end-stage heart-failure. While patient management has improved over the past decades, limitations in caregiver resources necessitate efficient allocation. Decisions on the optimal setting of LVAD support currently occurs in intervals ranging from a few weeks to a few months. However, the optimal setting for LVAD support may change multiple times per minute. Thus, automation is the only viable solution to this problem. Physiological control promises to integrate the cardiac assist device into patient physiology even more closely, reacting to varying demands or specifying the load for training of the heart muscle. One important design consideration is the choice of feedback variable. Methods: Literature of the past two decades was collected and the presented physiological control concepts were clustered into categories corresponding to functional similarity of feedback variables. Results: It was found that most literature could be summarized into five categories. Control algorithms either used preload, heart rate, ventricular contraction, or afterload as feedback variables or modified the flowrate/headpressure relation. Discussion: Each of these categories contains information about the cardiac state. While preload variables, most closely resemble the native functioning of the left ventricle, they are inherently difficult to measure without using additional sensors. Chronotropic incompetence might be a limiting factor for heart rate based control. Ventricular functioning might fluctuate over time due to changes in inotropy while afterload is heavily regulated by other autoregulatory mechanisms. The headpressure/flowrate relation suffers from uncertainty regarding the origin of pressure changes. Combining elements of these five categories into one control concept might prove beneficial, as different information could be extracted for the controller, which could then be tuned by the physician.Objectives: Current understanding of membrane performance in hemodialysis implies that their permeability in terms of molecular weight cut-off must be restricted to a figure below the molecular weight of albumin, i.e. < 60.000. Nephrologists commonly request a maximal allowable albumin loss during a 4h treatment to be below 4g per session. Methods: Recent investigations have shown that the 3D-conformation of albumin is unstable and changes in e.g., uremia or liver failure. Thus, detoxification efficiency of albumin may decrease by about 70 and can be explained by a considerable reduction of albumins binding constants for both pharmaceutical drugs and uremic retention solutes, as well as for exogenous toxins. As a result, free concentrations of such solutes increase in patients blood and a higher toxicity is observed. Binding constants for defined uremic retention solutes, such as p-Cresylsulfate, are further affected by the simultaneous presence of other solutes, such as Indoxylsulfate. Results: Oxidative stress, as manifested in many dialysis patients, further promotes changes in albumin conformation. Its thus no surprise, that hemodialysis with protein-permeable membranes improves patient performance, as shown recently in a clinical trial by Nagai et al., Ther Apher Dial, 2017; 21:378. They reported, that in a 7-year observation period, albumin leakage of 3g or more per HD session provided a better prognosis than albumin leakage less than 3g. Obviously, a clinically acceptable large albumin leakage provides beneficial effects on maintenance HD patients. Discussion: The performance of albumin as an adsorbing protein is inefficient during uremia and liver failure. Due to lower binding constants for pharma-drugs and uremic toxins following conformational changes in the sick state, albumin is unable to perform like in the healthy state. It should, thus, be removed. Identification of 3D-changes of albumin and thus its removal should become a priority.Cataract surgery followed by implantation of an intraocular lens IOL is one of the most frequently performed surgical procedures in medicine. In 2020 alone, there are expected to be 32 million interventions. Here, ophthalmologists can look back on 70 years of experience. Since there is no blood perfusion at the lens, however, the IOL has a comparatively simple immunological special position. Furthermore, the ophthalmologists also have extensive experience with implants in the cornea, in the chamber angle of the eye and on the retina. In the presentation an overview will be given of the current implants and the necessary regulatory processes. In addition, own experiences with medical devices with the new requirements of the MDR will be described.Recent evidence from randomised controlled trials RCTs suggests that post-dilutional haemodiafiltration HDF is associated with a benefit in survival. This benefit has been mainly observed in re-analyses of the primary data looking at the patients treated with the highest convection volumes HCV-HDF. To obtain HCV-HDF, significant pressure has to be applied to the dialyser membrane and the dialysis systems are submitted to significantly stronger transmembrane pressure TMP constraints, frequently surpassing the limits proposed by the guidelines. Several automatized systems are proposed to avoid or overcome the alarms induced by the increases in convection and still maintain the volumes prescribed by the physician. The present overview willi Critically analyse the evidence on the effect of convection volumes on survival. ii Review the automatic systems presently proposed to achieve a given level of convection. iii Analyse the possible pitfalls in determining the limits established by a dialysis monitor when performing HCV-HDF. iv Explore a harmonised definition of the parameters that may drive the convection volumes during a dialysis procedure with the aim to help in better defining the Best Practice Guidelines for HCV-HDF.Objectives: Human Parvovirus B19 HPV B19 infection is known to cause pure red cell aplasia PRCA in kidney transplant recipients. We investigated the diagnosis and prognosis of HPV B19 infection by urine detection in kidney transplant patients with anemia and graft dysfunction. Methods: 1195 recipients living donor/donation after citizens death: 492/703 were performed kidney transplant in our center from January 2015 to January 2018. Thirty-six patients diagnosed with HPV B19 infection during follow-up 3.01, 36/1195. Among those, fourteen HPV B19 infected patients underwent transplanted kidney biopsy, including 12 persons with PRCA and graft dysfunction and 2 recipients with donor-derived HPV B19 infection. We collected the urine samples for detection of HPV B19 DNA copies at the day of kidney biopsy. Meanwhile, the biopsy specimen were studied for immunohistochemical detection by using polyclonal mouse Anti-Parvovirus B19 antibody. Results: The mean diagnosis time of fourteen HPV B19 infected patients was 44 days 7-300days after transplantation. The reticulocyte percentage was 0.1-0.2 and the mean hemoglobin level was 67 g/L 51-87 g/L. Among 12 biopsies with PRCA and graft dysfunction, 6 were diagnosed with T cell-mediated rejection IA-IB-IIA, 3 borderline rejection, 2 without rejection signs and 1 was diagnosed with thrombotic microangiopathy TMA. The patient, who was diagnosed with TMA, had performed ABO-incompatible kidney transplantation. It displayed glomerular microthrombus and diffuse necrosis of renal tubules on renal biopsy. Despite the treatment of IVIG, the graft was ultimately lost. The immunohistochemical result of the TMA biopsy specimen was positive and the others were negative. The result of urine detection for HPV B19 virus DNA copies was >10 boolean AND 5/ml with TMA and the others were negative. Discussion: If the urine test is positive for HPV B19 DNA, parvovirus -associated kidney injury should be suspected.Hemodynamic simulations are increasingly used to assess the flow phenomena in intracranial aneurysms IAs. Based on detailed simulation results existing therapies such as coiling or stenting can be improved or novel approaches are tested. However, the accuracy of patient-specific blood flow acquisitions strongly depends on preliminary and highly interdisciplinary steps. These include for instance medical imaging, image reconstruction, image segmentation and acquisition of appropriate boundary conditions. To assess capabilities, but also limitations of state-of-art IA modeling techniques, international simulation challenges are frequently organized. These comparisons address single or combined steps along the simulation process. Within this talk, the most relevant competitions are presented and the main findings are summarized. In this regard, sources of error in the context of hemodynamic modeling are revealed and the awareness for standardized simulations is sharpened.Objectives: Heart failure HF is a critical healthcare issue and a primary source of cardiovascular mortality worldwide. Globally, it is estimated that up to 100,000 people are candidates for heart transplantation, but only about 5800 transplants were performed in 2016. At Cleveland Clinic, we are developing a continuous-flow total artificial heart CFTAH in an attempt to provide a viable alternative to transplantation for the growing population of patients with end-stage HF. Methods: The Cleveland Clinic CFTAH is a double-ended centrifugal pump with only one moving part that is designed to passively self-balance left and right circulations without active intervention. The device has no mechanical bearings to wear out and is driven by an extremely reliable 3-phase brushless, sensorless motor. The nominal external dimensions of the latest version of our adult CFTAH design to be tested in-vivo were 98.4 mm in length, 62 mm in diameter, 160 cc in volume displacement and 486 g in weight without the cable. Results: Biocompatibility of the CFTAH has been demonstrated in two full-term 90-day in vivo tests, completed without the use of any anticoagulation. Since the conclusion of these tests, our team has been working on design improvements to further reduce susceptibility to thrombosis, enhance autoregulation, improve motor controller hardware, add clinically useful features to the control interface and enhance manufacturability. Discussion: Our results to date suggest that a small, mechanically and electronically simple, auto-regulated total artificial heart has significant potential to provide an important clinical option for HF patients.We discuss a novel method for hemolysis prediction in computational fluid dynamics, which avoids potential problems inherent to the traditionally used Eulerian and Lagrangian methods. The traditionally used Eulerian method computes a hemolysis index HI at the flow outlet by solving a transport partial differential equation PDE where the source term equals shear stress raised to some power which is determined empirically, and then looks at the HI transported to the flow outlet. The largest contribution to HI typically occurs at the flow boundaries walls, which is precisely the location where numerical noise is most likely to be inadvertently introduced, e.g., from sharp edges in the wall or from low quality mesh elements. This can cause errors in the Eulerian method because its PDE lacks a diffusion term, and consequently any numerical noise is transported to, and accumulates at, the flow outlet surface. We have not seen this potential problem occurring when using a Lagrangian method, although theoretically it could still occur. Regardless, a major disadvantage of the Lagrangian method is a large computational cost required to achieve reasonable accuracy. Instead of using one of the two traditional methods, we use a volume integral where the source term includes both shear stress and flow speed. In situations with sufficient mixing, such as in an LVAD, this approach predicts blood damage at least as well as either of the other methods, for a fraction of the computational cost, and avoids the numerical problems. The method also lends itself better to frozen rotor simulations, while the traditional methods prefer more costly time-dependent simulations. We present an example where the method is used to evaluate potential changes to a HM3-like rotor design.Objectives: A great and long pursued goal in cancer research is the development of a suitable culture system for expanding haematopoietic stem cells HSCs, which can thereafter be successfully used in transplantations. Since HSCs only reproduce in their natural environment, novel systems try to mimic the in vivo niche of HSCs, the bone marrow, in many factors such as material rigidity, chemical and cellular compositions. Mesenchymal stem/stromal cells MSCs are known to support HSC expansion. This effect is even greater when the cells are cocultured in a 3D environment. The aim of this project is to study the interactions of HSCs and MSCs in a conventional 2D culture compared with a 3D porous hydrogel, which was previously developed in our group. Methods: The influence of HSCs on MSCs and vice versa will be studied on a transcriptomic level by next generation sequencing and verified by qPCR, cytokine arrays and western blots. To regain the cells from the system we want to make our hydrogel degradable by introducing disulfide bonds, which can be cleaved by reducing agents. Afterwards the RNA will be fixed within the cells and these will be separated by magnetic cell sorting, previous to RNA isolation. Results: Previous experiments indicate that the 3D system has a positive effect on HSCs only in coculture with MSCs. With a deep insight into the transcriptomes of both cell types in coculture, we hope to identify key molecules playing a role in this phenomenon and in HSC expansion. Such key molecules shall further be tested for their influence on the coculture by inhibition or enhancement. Discussion: The HSC expansion in vitro while maintaining their stem cell character has been a major and challenging goal in research for more than 60 years. By finding and influencing key molecules of the HSC-MSC interaction, we hope to further develop our 3D hydrogel system and increase HSC expansion in vitro.Objectives: Small molecule drugs can be bound to plasma proteins. Examples are Apixaban 459 Da, 87 protein bound, Carvedilol 406 Da, 98 protein bound or Calcitriol 416 Da, 99.9  protein bound. In plasma, these drugs have a free and a protein bound fraction. Protein binding impacts the removal behavior of the drug during dialysis. The purpose of this work was to develop a quantitative model to describe the removal of these solutes, and validate the model experimentally. Methods: A mathematical model was developed by coupling the equations of transport in dialyzers reported in one of our previous studies, and mass balance equations reported in literature to include the effect of binding. Simulated HD treatments with human plasma were performed with 3 different types of commercial dialyzers having different membrane permeability to study the removal of Apixaban, Carvedilol or Calcitriol, expressed in terms of solute clearance. Additionally, the loss of albumin was quantified. Results: Experimental clearance results ml/min are given as mean /- MAD from 3 independent experiments in the order of Apixaban / Carvedilol / Calcitriol. Calculated values are reported in brackets. Dialyzer 1 was tested in HDF mode, yielding clearance of 60 /- 1.0 61 / 41 /- 1.8 10/ < 6 0.5. Dialyzer 2 tested in HD mode yielded clearance of 60 /- 1.6 60 / 31 /- 1.6 10 / Discussion: Clearance of tested drugs was lower than expected for their nominal molecular weight. The levels scaled Apixaban > Carvedilol >> Calcitriol, which is inverse to their degree of protein binding Apixaban < Carvedilol << Calcitriol. The degree of protein binding was revealed to be the actual critical determinant, rather than dialyzer design and operational mode. Drug clearance seemed to be driven by the proportion of the free fractions, as is reflected in the model by the increase of clearance for decreasing bound fraction.Objectives: In the development of biocompatible materials for biomedical applications, the foreign body response is an important issue. The healing of surrounding tissue often interferes with the function of an implanted biomaterial. Events like protein deposition, hemostasis, inflammation, tissue repair, infections and the encapsulation of the functional part of the implant are the main cause of failure of the implanted device. Methods: In this study biocompatible nanofilms are produced by means of a plasma polymerization process using a low-pressure magnetron-enhanced 15 kHz glow discharge. This process allows the precise control of the film nature and behaviour. The resulting hydrocarbon film has a thickness of a few nanometer and keeps therefore the inherent properties of the substrate material. Measurements on protein adsorption gave the possibility to tailor the thin films in the needed direction. This means to tailor a native secondary structure of in situ adsorbed proteins. Results: The nanofilms were investigated using different surface analytical methods. Also the interaction in contact with different biological sample materials was tested in-vitro. The precise measurement of the adsorbed proteins indicated a native secondary structure of proteins on these surfaces. Different in-vivo sensor dummies which are in contact with blood and soft-tissue were coated by these nanofilms. The explanted sensors were kept free of any encapsulation by this coating. This stands in good correlation with the measurement of adsorbed proteins. Discussion: The first in-vitro results of the adsorption of blood proteins indicated already a very biocompatible character of these nanofilms. The explanted sensors were kept free of any encapsulation by this coating. These coatings can open the door for many new applications in the field of new implants but also other biomedical products.Objectives: Human placenta tissue has great advantages to be used as tissue source due to its molecular composition and its availability as clinical waste product. In our previous studies, human placenta arteries were decellularized and showed low immunogenicity and a supportive environment for cell migration and proliferation. Decellularized extracellular matrix ECM hydrogels are a widely-used tool in the field of tissue engineering and regenerative medicine. To create placental ECM hydrogel pECM-HG decellularized arterial grafts were further processed by an enzymatic pepsin digestion. Methods: pECM-HG were biochemically analyzed. Scanning electron microscopy was used to visualize fiber structures. Cell adherence and migration assays were carried out on pECM-HG coated culture plates to confirm cytocompatibility of the material. As next step pECM-HG was used for coating of vascular grafts in a reseeding experiment using a perfused 3D bioreactor system. Furthermore, biomechanical properties of 3D hydrogels were tested by rheological measurements. Results: pECM-HG showed low amount of DNA residuals and preservation of cell interacting ECM proteins. SEM analysis revealed the formation of specific fibrous structures, dependent on respective hydrogel concentration. Cell binding and viability assays showed significantly enhanced cell adherence and increased proliferation rates when seeded on surfaces coated with pECM-HG p<0.001 compared to controls. pECM-HG showed beneficial properties for cell reseeding on acellular vascular grafts in a flow bioreactor system. Discussion: We developed a hydrogel from human vascular tissue that is cytocompatible and supports cell adherence and proliferation. We hypothesize that this hydrogel has great potential due to its origin and composition. Further investigations on the hydrogel composition are ongoing. It is planned use the pECM-HG for 3D cell printing applications or as injectable hydrogel to support ischemic injury regeneration.Objectives: Intra-aortic Balloon Pump IABP is undoubtedly the most widely used mechanical circulatory support system for a range of severe heart failure indications. Average duration of IABP use is 1-3 days, however its longer-term use has not been adequately studied. The objective of this work is to establish the extent to which the IABP duration has been used, and also to examine the clinical outcome pertaining to prolonged use of IABP, using meta-analysis of literature data. Methods: We defined IABP prolonged use as 7 days or longer; a choice based on a duration that is greater than double the current perceived average use. We systematically reviewed the literature Pubmed, Scopus, Web of Science and identified 14 studies that have prolonged IABP use between 1968 - 2018. Patients indications included heart failure refractory, congestive, end stage, ventricular arrhythmia, myocardial infarction and cardiogenic shock. Results: A total number of 1707 patients of both genders received prolonged IABP use with an overall average of 22.6 days. The studied cohort is split into: Group 1: n= 1683 adults, mean age 57.9 years, range 19-84 years old, average use of 21.2 days. Survival rate in this group was 64.4. Group 2: n=24 infants and children, mean age 5.5 years, range 7 days-17.5 years old, average use 24 days. Overall mean survival rate to discharge was 62.5. At a mean follow-up of 85 months, all 15 survivors were alive and well. Discussion: Due to its ease of insertion and relative low cost, IABP presents the first port of call in a wide range of heart failure scenarios. This analysis indicates that, age does not seem to be a significant parameter that affect outcomes of IABP long-term use. Although, IABP is perceived to be only a short-term support, its prolonged use is evident with successful outcomes. Further trials of IABP long-term use for children and adults is warranted to further establish the range of indications IABP can be utilised as a long-term support.Objectives: Human serum albumin HSA, one of the dominating proteins in human plasma with its exceptional binding capacity plays an important role in storing and transporting different substances throughout the human circulatory system. Clinically used to treat a variety of diseases, particularly in liver failure therapy, pharmaceutical-grade HSA contains high amounts of stabilizers, such as Na-caprylate and N-acetyltryptophanate. These compounds represent potential risks to patients with restricted liver function. The aim of this study was to test if the elimination of stabilizers by adsorbent technique affects HSAs secondary structure. Methods: Albumin infusion solutions BIOMED, Germany were filtered using a commercial system Hepalbin Albutec, Germany. Raman- and FTIR spectroscopy as well as DSC analysis were applied to analyze conformational changes in treated and untreated HSA samples. For Raman based studies different parameters such as object slides type, integration time, accumulations number and laser capacity were investigated. Results: Preliminarily results of the physical-chemical analyses showed some differences for filtered samples in comparison to the unfiltered ones. Raman and FTIR spectra exhibited both, peak shifts in characteristic HSA amide bands and an alteration in the peaks shape. DSC thermograms revealed significant differences in denaturation temperatures of the compared samples. Discussion: A Raman shift in amide I band and changes in the alpha-helix/beta-sheet ratio detected via FTIR spectroscopy indicated a possible alteration in the HSA conformation after filtration. These findings were corroborated by DCS showing a tendency towards lower denaturation temperatures in samples after stabilizers removal. The results may contribute to finding a compromise between stability and high functional performance of HSA for the improvement of liver failure therapy.Objectives: The protein fibrinogen is important in blood coagulation and wound healing. Hence, nanofibrous fibrinogen scaffolds are highly attractive as biomaterials. One currently used technique to prepare fibrinogen nanofibers in vitro is electrospinning. Since this technique uses organic solvents and high electric fields, which can impede the biological protein function, we have developed a novel technique to prepare fibrinogen scaffolds with physiological buffers. Methods: We introduced the novel method of salt-induced self assembly under controlled drying conditions to prepare nanofibrous fibrinogen scaffolds. Scanning electron microscopy SEM analysis was used to characterize the fiber morphology. We studied the secondary structure with Fourier-transform infrared FTIR spectroscopy. To assess the biocompatibility of our novel scaffolds we analyzed the proliferation of 3T3 mouse fibroblasts with WST1 assays and studied the cell morphology with SEM. Results: Salt-induced self assembly reproducibly yielded fibrinogen nanofibers with overall scaffold dimensions in the centimeter range. SEM analysis revealed fiber diameters between 200 and 300 nm. By adding a customized fixation step after the fiber formation we fabricated either immobilized or free-standing fibrinogen networks depending on the underlying substrate material. With FTIR spectroscopy we found a higher content of beta-sheet structures for fibrinogen nanofibers in comparison to planar fibrinogen. 3T3 fibroblasts were found to proliferate well on fibrinogen nanofibers with cell numbers comparable to planar fibrinogen substrates. SEM analysis showed that 3T3 fibroblasts grew in close contact with fibrinogen nanofibers. Discussion: Our new self assembly process offers a well controllable and physiological biofabrication method for large-scale fibrinogen scaffolds. Based on the good biocompatibility results our fibrinogen networks are very promising candidates for a new class of wound dressings.Total artificial hearts TAH are used to restore quality of life to biventricular heart failure patients but can also introduce significant risks. Several features were included in to the BiVACOR design to mitigate some of the major risks, such as 1 device malfunction, 2 insufficient hemocompatibility, 3 ineffective physiological interaction, and 4 inappropriate device geometry. To achieve a durable design the BiVACOR TAH reduces mechanical wear by levitating a single rotating impeller in a magnetic field. To reduce the incidence of single fault failures, the levitation and drive systems incorporate backup features. Large clearance gaps in the blood flow path and cyclic speed variations assist hemocompatibility. Physiological interaction is improved via autonomous left/right outflow balancing, the ability to inherently adapt total outflow based on physiological conditions, and to respond to variations in filling volumes. Initial durability testing is ongoing with no device failures after 180 days. The redundant motor and hydrodynamic backup bearing maintain hemodynamic function when failure modes are induced. Preliminary NIH, platelet activation and vWF degradation during in-vitro tests show lower levels than implantable VADs. In-vivo results 1x90day, 3x30-day confirm benchtop observations. When operated with either pulsatile or non-pulsatile outflow up to 15 l/min, or with the hydrodynamic backup bearing, pfHb ranged between 0.5-2mg/dL. The system can balance outflows under a wide variety of physiological conditions, while inherently adjusting flows by more than 4 l/min. Finally, device weight and size allowed a good anatomical fit in human cadavers with BSA >1.4m2. By implementing the risk mitigating design features, the BiVACOR device aims to provide patients with a durable, hemocompatible, and physiological interactive TAH with suitable anatomical fit. Successful development of this device may ultimately lead to a long-term alternative to heart transplants.Objectives: Visualization of hemodynamics often represents a simplification of blood as a monophasic fluid. In this study, blood is used as a multiphase fluid. The aim is to fabricate artificial blood to visualize hemodynamics with Particle Image Velocimetry PIV. More specific, micro beads ArtErys, which mimic the erythrocytes, are fabricated with a membrane-emulsion-technology system MET. This system is developed and compared with the results e.g. ArtEry size of three other methods: microfluidic, electro-spraying and air flow. Methods: The ArtErys are made of hydrogels Agarose 0.3-0.7 wt. to match refractive index and biomechanics. For PIV tracking, particles Sphericals c, 20 mu m are encapsulated directly into the ArtErys and are not added to the plasma where they might affect the hemodynamics. For the MET system, different oils e.g. paraffin are used as disperse phase. The Agarose/PIV mixture is pumped 50 to 100 ml/h into a constant oil flow 5 to 40 ml/h through a PTFE membrane Reichelt Chemietchnik GmbH, pore size 60 mu m. The ArtErys are created during this process and transported away by the oil. The MET process is carried out in a heated water bath with a temperature of 55 degrees C. Results: The ArtEry size depends mainly on two parameters: the flow rate of the agarose/PIV solution and the flow rate of the oil. A flow rate of 80 ml/h Agarose/PIV solution leads to smallest bead sizes with a mean diameter of 150 mu m. High rates lead to merging of the ArtErys, which is reflected in a larger diameter. Furthermore, the bead size depends on the parameter oil flow rate. Here, a rate of 30 ml/h leads to the smallest beads. Discussion: The MET method has been successfully implemented to fabricate ArtErys with encapsulated PIV particles. Compared to the other methods, the MET system fabricates the biggest ArtEry sizes MET: 150 mu m, microfluidic: 15 mu m, electro-spraying: 80 mu m, air flow: 100 mu m.Objectives: Since the latest revision of the Medical Devices MDs Directive, the European regulatory framework considers stand-alone software as a MD per se, as confirmed by the MD Regulation. The recent surge of AI tools, which promise to be ubiquitous soon, poses problems relative to their certification as MDs. In view of the already granted market approval in Europe and the USA for some AI-based MDs, we aimed to investigate the main relevant issues of such MDs regulation. Methods: Unlike traditional SW as MDs, for which established standards already exist e.g., IEC 62304, "Software life-cycle processes", for AI tools there are very few specific standards. The definition of AI, or intelligence itself, is by no means universally accepted. Thus, we limited the scope of AI to Machine/Deep Learning ML/DL. Results: 1 ML/DL tools are data intensive, so that a critical issue is how to construct a representative set of input data. E.g., IEC 62304 recommends that the manufacturer include software system inputs and outputs, in the SW requirements: this is clearly too generic for a DL tool. 2 Guidelines for acceptable input data size should be drawn, for a given system/software architecture. Moreover, quality check of training data should always be executed, before SW training. Quality of ML/DL tools and of physical architecture is also relevant. 3 Generally, DL tools lack an explicit declarative knowledge representation, hence it is difficult or impossible to assess to "reasoning" behind the algorithmic decisions. "Explainable" AI should be the standard, in order to rule out fuzziness in machine clinical decision making and impossibility to pinpoint the problems, in view of the legal responsibility for MD use. It has been found that DL performance conflicts with explainability, thus priority setting will be crucial. Discussion: The regulatory treatment of AI-based MDs needs improvements, in order to have safe and effective products, as well as wider societal acceptance.Objectives: The aim of this work is to design a reliable and efficient sheath for an implantable part of a device for the wireless inductive powering of artificial organs. Methods: Wireless inductive power transfer can help get rid of the problems of recharge and wired constant power supply to artificial organs. However, the design of the implantable part of the transcutaneous powering device, or so-called assembly, is complicated by the lack of a comprehensive solution of technical and medical problems. The assembly must be encapsulated in the sheath which requires the choice of a biocompatible and non-magnetic material. This eliminates the formation of eddy currents on the sheath, whereby negative influence on the energy transfer is minimized. At the same time, the material should have a low thermal conductivity about 0.5 W / m . K to prevent overheating of the tissue from the circuitry. The sheath of the assembly must have sufficient durability to protect the circuitry. The form-factor of the sheath should be minimized for the convenience of the patient dimensions in the limit of 20 x 20 x 3 mm. Results: The sheath was developed for the assembly prototype. The assembly provides an output power in the range 0.45 horizontal ellipsis 0.55 W for the axial distance between the coils in the range 10 horizontal ellipsis 20 mm and the lateral misalignment up to 20 mm. The calculation and measurement were performed in the air and with the samples of "skin-fat" biological objects, in order to determine and verify the sheath characteristics. Discussion: We have developed and tested the assemblys sheath for inductive powering of artificial organs that meets technical and medical requirements.Objectives: Magnetic nanoparticles MNPs are used in medicine for targeted drug delivery to the area of cancer. There is the problem of remotely determining in vivo concentrations of MNPs. The aim of our work was to experimentally test acoustomagnetic method AMM for detecting MNPs in a model sample of a biological substance BS. Methods: A colloidal solution of Fe3O4 nanoparticles in a mixture of oleic acid and kerosene was used as a model sample. The weight concentration of the MNPs in the solution was 0.15, and the viscosity of the solution was close to the blood viscosity. To detect MNPs in solution, a special experimental setup was developed. The installation consists of an ultrasound generator, a permanent magnet, a glass container with a solution, an induction multi-turn coil near the glass tube, and a voltmeter as a measure of the voltage across the coil. Ultrasound causes periodic movement of the particles of the solution together with the MNPs along the container axis. The magnet orients the MNPs in the direction perpendicular to the axis of the container. As a result, the summation of MNPs creates an alternating magnetic field with an ultrasonic frequency in the coil area. An alternating voltage U arises on the coil, as measured by a voltmeter. Results: As measurements have shown, the magnitude of the voltage is proportional to the concentration of MNPs in the field of action of the ultrasound and the magnet field, as well as the intensity of the ultrasound. In particular, when the intensity of ultrasound was at the level of 0.02 W/ cm2, U = 1mV was obtained. Discussion: The experimental results are consistent with the calculated estimates and suggest that AMM can be used to detect MNPs in a real BS. Considering that the magnitude of the constant magnetic field in these experiments was 0.1T, we can also conclude that this method is safe, more accurate, and easier to implement than traditional X-ray and MRT methods.Objectives: For a single hemodialysis session several hundred liters of water are consumed. Water scarcity and inadequate water purification facilities worsen contamination risk especially in developing countries. Here, we investigate the application of a mixed matrix membrane MMM, which combines filtration and adsorption, for achieving endotoxin-free dialysate combined to high removal of uremic toxins from human plasma. Methods: We investigate the adsorption of lipo-polysacharide LPS by a MMM in both static and dynamic conditions. Dynamic adsorption of LPS is also investigated in presence of uremic toxins in human plasma. Diffusion experiments using dialysate contaminated with bacterial culture filtrates are also performed to assess the ability of the MMM to act as a safety-barrier to avoid transfer of pyrogens to the plasma. A membrane without sorbents is used as control and the obtained results are compared with literature studies using current dialysis membranes. Results: The MMM can remove approximately 10 times more endotoxins from dialysate compared to commercial dialysis membranes. No transfer of pyrogens was detected in the blood compartment, revealing safety-barrier properties of the MMM. Importantly, endotoxins from dialysate and protein-bound toxins from human plasma can be removed simultaneously without compromising AC adsorption capacity. Discussion: We estimate that 0.15 m2of MMM is needed to remove the daily production of the protein-bound toxins, indoxyl sulfate and hippuric acid and to completely remove endotoxins in a wearable artificial kidney WAK device. Future studies would focus on the applicability of the MMM in wearable/portable artificial kidneys and in general for home-hemodialysis treatments.Objectives: Nearly half of all heart failure patients suffer from heart failure with preserved ejection fraction HFpEF. With no effective treatment, a major clinical need is presented. The aim of this study was the hemo - and fluid dynamic investigation of a valveless pulsatile pump for HFpEF using in-vitro, ex-vivo, computational fluid dynamic CFD, and 4D flow MRI imaging studies. Methods: The pump is a pneumatically driven pulsatile pump with a single valveless cannula that is to be implanted through the LV apex and is operated synchronously with the left ventricle. A prototype of the proposed pump system was fabricated. The pump was tested in an in-vitro hybrid mock loop system to evaluate its hemodynamic effect in four HFpEF phenotypes. Further hemodynamic evaluation was conducted by implanting the pump in an ex-vivo isolated heart model. CFD was employed for flow field analysis of two different pump designs-a symmetric and an asymmetric configuration. The results were validated by 4D flow MRI imaging of the respective prototypes. Results: In the in-vitro studies, the pump augmented cardiac output CO by 14 - 30 and reduced left atrial pressure LAP by 14 - 30. Ex-vivo studies showed the similar hemodynamic trends: augmented CO by 20.0 /- 9.7, and reduced LAP by 57.5 /- 14.4. Fluid dynamic simulations revealed good correlation with the respective 4D flow measurements. The asymmetric configuration disclosed superior fluid dynamic performance with 94.5 of the initial blood being exchanged after 3 beats. Discussion: It was demonstrated that the valveless pulsatile device can improve hemodynamics in HFpEF patients by augmenting CO and reducing LAP. Appropriate washout of the pump chamber is achieved by a persisting vortex evolving within the asymmetric design throughout each pump cycle. By demonstrating the hemodynamic effects and promising fluid dynamic performance of the pump, feasibility of the system as a potential treatment option for HFpEF patients is supported.Objectives: In this work, the denaturation behaviour of bovine serum albumin BSA in single wall carbon nanotubes SWCNTs aqueous nanodispersions during heating was analyzed using differential scanning calorimetry. Thermodynamic effects of BSA interaction with SWCNTs have been discussed. The calculation procedure for estimating the parameters of laser treatment on the basis of differential scanning calorimetry data is presented. Accurate calculation of the optimal parameters of laser treatment, under which there is complete denaturation of albumin and the solid nanocomposite material is forming, will minimize the thermal damage of the connected tissues, while not reducing the tensile strength of the seam. Methods: Laser solders based on water dispersion of BSA 25  w/w and SWCNTs 0, 0.1, 0.01 and 0.001  w/w were used for differential scanning calorimetry study. Studies were conducted in hermetically sealed crucibles with heating rates of 3, 5, 10 and 20 degrees C/min. Results: The specific energies and denaturation temperatures of the laser solders were measured. The use of SWCNTs led to a decrease in the specific energy of denaturation from 4.11 to 3.27 J/g and a denaturation temperature from 87.2 to 86.9 degrees C. The activation energy of albumin denaturation in laser solder has doubled during use SWCNTs Discussion: Thermodynamic effects of BSA interaction with SWCNTs have been discussed, namely the changes in temperature characteristics of denaturation processes and the partial thermal destabilization of albumin. The energetic parameters of laser treatment have been estimated on the basis of differential scanning calorimetry data. The values obtained are optimal for complete BSA denaturation resulting in formation of solid nanocomposite material with minimal thermal damage of the connected tissues and sufficient tensile strength of the seam.Objectives: Clot formation in the membrane oxygenator is a serious complication in long-term extracorporeal membrane oxygenation ECMO. In the blood vessels, nitric oxide NO generated by nitric oxide synthase on the endothelial cells suppresses both aggregation/activation of platelets and adhesion/activation of leukocytes. Therefore, biocompatibility of the membrane oxygenator may be improved by the addition of NO gas to the sweep gas. The objective of the present study was to determine, using a rat ECMO model, whether addition of NO gas to the blood can suppress blood coagulation. Methods: Male Sprague Dawley rats weighing 350 to 400 g were used for the experiments. After the rats were anesthetized, extracorporeal circulation was established by guiding the blood from the right jugular vein into the left femoral vein. Heparin was administered by bolus injection 0.7 unit/g-rat only at the start of the ECMO. Polymethyl pentene membranes were used for the oxygenator. The sweep gas NO: 0, 100 or 200 ppm, O2: 30, CO2: 5, N2: balance was added to the blood via the oxygenator. During the experiment, the arterial pressure, oxygenator inlet pressure, venous pressure, activated whole blood clotting time ACT, and methemoglobin Met-Hb concentration were measured. After circulation, the hemoglobin Hb content and lactate dehydrogenase LDH activity in the eluate from the residual blood in the oxygenator were measured. Results: Extracorporeal circulation was stably performed for 4 hours. There were no differences in the ACT during the experiment among the three groups. The amount of Met-Hb was below 10 in the NO group. The Hb content and LDH activity in the eluate were significantly lower in the 200-ppm NO group than those in the 0 ppm NO. Discussion: The amount of Met-Hb in the NO group indicates that there is no adverse effect related to Met-Hb. The Hb content and LDH activity indicate that the NO contained in the sweep gas inhibited blood coagulation in this rat extracorporeal circulation model.Objectives: Data from the WAA apheresis registry was investigated regarding patients treated with extracorporeal photopheresis ECP as a tool to modulate immune response in various diseases. Methods: 381 patients 34 women had been treated with ECP for a total of 10668 procedures. The mean age was 48 years /- 18, range 3-81 years. Estimation of quality of life was made using grade 0 suicidal up to 10 best ever and health quality grade 1 Bed ridden, ICU condition up to 10 athletic. Adverse events were analyzed. ANOVA comparisons were used. Results: Most patients were treated due to graft versus host disease GVHD, n=284 and other hematological diseases n=34. Adverse events were registered in 5.8 of the first treatments and in 2.1 of the subsequent procedures. Severe adverse events were present in 0.4 of all procedures. No patient died due to the procedure. Tingling and stitching were the most common side effects. For those with GVHD the QoL was initially at a mean 6.3 /- 1.5, range 2-10 and improved significantly within 10 procedures and the Health Condition estimate improved significantly within 9 procedures and improved further with added procedures. Discussion: Photopheresis is an established therapy with few side effects. The present data indicate that approximately 10 procedures are necessary to note significant effect to GVHD.Objectives: The most fundamental goal in improving quality of care in hospitals is to eliminate unnecessary deaths. AIM: Three-year mortality and quality of care between 2015-2017 were evaluated in a tertiary nephrology clinic. Methods: Audit of 392 deaths was carried out to reveal prevalence of suboptimal clinical observations. Admission pressure, direct admissions avoiding public healthcare system, severity of clinical conditions, elderly persons dying in the hospital, time of admissions and death were evaluated. Results: In 2015 and 2016, 5.5 of every 100 inpatients died in the hospital, and this proportion decreased to 4.9 in 2017, with high percentage of patients who died in the first 24 or 48 hours from admission. The percentage of hospital deaths occurring in combination with AKI rose from 17 up to 37 over the three-year study period. Many patients who died with worsening CKD, were not ever referred to a nephrologist. More than half of patients were admitted on duty and died over weekends. About 5 of the hospital deaths were among patients aged 85 and over, and in 2017 that number rose up to 10. Patients who died were seriously ill, at least 17 had malignancy. Most of the cardiovascular deaths were directly caused by ischemic stroke. Discussion: In this study we did not find patients with preventable deaths. The results indicated that patients were seriously ill, elderly, some being admitted to hospital to die. Some of the patients were kept long period prior to death because of shortage on certain facilities. There is admission pressure and other external factors contributing to high burden in treatment of the dying patients. Still, further effors are necessary for further providing and auditing health quality of care.Objectives: Human immune cells have been produced using Advanced Therapy Medicinal Products ATMP guidelines and have been tested in clinical studies for a number of diseases, for which still no or only inadequate alternative therapies are available. Examples include several cancer entities, stroke, myocardial infarction, severe autoimmune disorders and chronic infections. Despite many in vitro or in animal studies report positive outcomes after immune cell therapy, convincing results showing the benefits of ATMP in clinical studies are rare. Part of this discrepancy in the outcome of the clinical studies in comparison to the in vitro and in animal studies may be caused by the manufacturing of immune cells for clinical trials. Cells prepared for clinical trials were mostly manufactured using archaic, scarcely controlled and incomparable production processes and stringent analytical methods for assessing proliferation and differentiation of the immune cells during the manufacturing process were not applied. Methods: The production of ATMPs is first of all dependent on the donor specific biology, which significantly varies from one donor or patient, respectively, to another biological heterogeneity. In case of immune cell therapy, the manufacturing process is key for cytotoxic function and is the basis for the biological activity of the cells after injection into the patient. Results: The challenge for future clinical trials therefore is to reproducibly provide a sufficient number of biologically active cells of appropriate quality preferably in accordance with quality manufacturing guidelines e. g. Good Manufacturing Practice, ICH etc.. Only using cells which have been produced under these rigorous quality control regimes will allow the conduction of clinically meaningful trials. Discussion: The talk presents a survey and discusses consequences on future requirements for manufacturing processes of products for immune cell therapies.Objectives: Cryopreservation plays a major role in the storage of tissues that are kept for subsequent transplantation. A wide variety of tissues intended for transplantation for a patient are stored in this way. In addition to the assignment to a suitable recipient, it must be ensured in particular that a sufficiently long period of time is available for infection diagnostic analyses. Cryopreservation of tissue is therefore routine in everyday tissue medicine. Methods: As is well known, the success of low temperature storage depends in particular on coordinated freezing and storage processes, in the optimization of which many research approaches have already been investigated. In contrast to these well-known principles of important parameter adjustment in this context, little attention is paid to optimizing the freezing process in the daily routine of tissue medicine. In fact, for some applications the suboptimal storage of tissue is not critical, as no vital cells are necessary or desired. Results: This reduces the process to a minimum and eliminates the need for additives such as cryoprotective agent CPA or animal/human serum. This is particularly important for tissue preparations which, as in Germany, require a medicinal product approval by a competent authority. Any change to the process must be first approved by the competent authority, the Paul-Ehrlich-Institut PEI. Optimizing the freezing process would therefore entail a new approval procedure. If the changed process was not previously known in the EU, a clinical study is also required. Discussion: For this reason, the conditions are not improved in case of doubt, although it is known that this could lead to a higher quality of the tissue after thawing. Less toxic CPA could simplify the approval procedure and thus contribute to improved tissue storage for transplantation. The same applies to the cryopreservation of tissue engineered products.Vascular access is a key issue for survival for uremic patients that need acute or chronic dialysis. In acute settings the use of central venous catheters, holding two luminae, are most commonly used in the intensive care for continuous or intermittent hemodialysis. Such catheters may be of acute type or a chronic type containing a separate cuff to limit bacterial invasion into the vessels. Another option is to place a peritoneal dialysis catheter into the abdominal cavity. If using the right technique, this catheter can be surgically inserted in local anesthesia and could be used directly postoperatively for acute but also chronic peritoneal dialysis. For chronic hemodialysis HD the preferable option is to place an arteriovenous fistula AVF at the lower arm. This results in enlargement of the vein, and partly fibrosis, that allows for frequent punctures leaving no material externalized after the end of a HD procedure. A problem is the development of vascular stenosis and thrombosis that need interventions in at least 50 of patients within one year. When the AVF fails an option is to insert a graft AVG. This may be either a biologic or synthetic graft interpositioned between the feeding artery and a more proximal vein below the elbow instead of the local vein that usually is occluded. If this option even fails, the patient has to rely on a sufficient permanent central dialysis catheter where complication such as bacterial contamination and occlusions are problems. To avoid such catheters a dream for future would be a graft as AV access that is made of an autologous biodegradable material that has physiological characteristics that prevent stenosis and clotting and also is possible to easy correct in its diameter to adjust for the volume of blood returned/minute to the heart to avoid congestive heart failure.Objectives: It has been observed that the in-vivo dialyser clearance for small solutes is not constant during a hemodialysis HD session. This may influence the choice of the value to be measured and reported. We investigated the presence of trends in the clearances of urea and creatinine, and how it impacts their values. Methods: Diffusive clearances were calculated for urea and creatinine from dialysate concentration data measured in 25 patients who underwent 3 consecutive HD sessions each. Clearance values were obtained for each hour at 0, 60, 120, 180, 240 minutes. The significance of the observed trend in the data was statistically tested, as was any difference between the values at 2 and 3 hours, and the average clearance during the session. Results: Urea clearance showed an overall increase from the start of HD by 10.1 /- 19.3 mL/min, p = 0.0002 while creatinine clearance decreased by -30.7 /- 17.1 mL/min, p < 10-4. For urea, there was, however, a significant increase during the first hour by 26.5 /- 16.3 mL/min, p < 10-4 which was not observed for creatinine. Starting from 2 hours, both clearances decreased significantly, but the overall drop for creatinine was twice that for urea by -34.1 /- 15.7 vs. -16.3 /- 19.8 mL/min, p < 10-4. As a consequence of these different trends, clearance at 2 h, 3 h, and the time-average clearance were not significantly different for urea 199.2 /- 25.9 mL/min, p = 0.18, whereas they were so for creatinine 2 h: 115.9 /- 26.3 mL/min vs. 3 h: 106.2 /- 29.2 mL/min vs. average: 109.7 /- 26.1 mL/min, p < 10-4. Discussion: A decrease in clearance during HD can be expected because of the progressive clogging of the dialyzer membranes pores. It is not clear however the reason for the higher decrease in creatinine compared to urea. Our results suggest anyway that the time at which creatinine clearance is measured in-vivo should be clearly reported, as the difference during the session might be as high as 30.Objectives: Currently, there are no reliable internal treatment options for large segmental bone defects. The objective of this study is to develop an alternative treatment for a young male patient with a huge 180mm diaphyseal tibial bone defect, who was already recommended amputation. Methods: A modular add-on system is developed for a commercially available motorized bone lengthening nail Ellipse Technolo-gies, USA. The system allows bifocal segmental transport of osteotomized bone segments. By this, the bone defect is filled from two sides. A wire rope hoist shall ensure that the bone can overcome the distance from both sides, which re-duces the time of healing and the amount of implanted parts. A 3D-printed model of the patients bone segments helps to design the implant system itself and the handling which includes fixation, transport and explantation. Results: The results show in accordance to previous studies that the developed implant system effectively transports both osteot-omized transport bone segments. The wire rope hoist mechanism and the modular construction works as planned. The implant system can be explanted successfully. Discussion: In this study, we present an individualized modular add-on implant solution. The advantage of the modular setup is that many parts can be manufactured in universal sizes. This method of bone reconstruction could help many patients, who suffer from large critical size defects.Among the various possible embodiments of Advanced Therapies and in particular of Tissue Engineering the use of temporary scaffolds to regenerate tissue defects is one of the key issues. The scaffolds should be specifically designed to create environments that promote tissue development and not merely to support the maintenance of communities of cells. To achieve that goal, highly functional scaffolds may combine specific morphologies and surface chemistry with the local release of bioactive agents. Many biomaterials have been proposed to produce scaffolds aiming the regeneration of a wealth of human tissues. We have a particular interest in developing systems based in biodegradable polymers. Those demanding applications require a combination of mechanical properties, processability, cell-friendly surfaces and tunable biodegradability that need to be tailored for the specific application envisioned. Those biomaterials are usually processed by different routes into devices with wide range of morphologies such as biodegradable fibers and meshes, films or particles and adaptable to different biomedical applications. In our approach, we combine the temporary scaffolds populated with therapeutically relevant communities of cells to generate a hybrid implant. For that we have explored different sources of adult and also embryonic stem cells. We are exploring the use of adult MSCs, namely obtained from the bone marrow for the development autologous-based therapies. We also develop strategies based in extra-embryonic tissues, such as the perivascular region of the umbilical cord Whartons Jelly. We are currently involved in a European consortium aiming at developing films with antimicrobial properties to be used in hospitals to cover surfaces that are prone to facilitate the transmission of infections. Those films use surface topographies and natural-derived oils to ensure the needed antibacterial properties to the films. This talk will review our latest developments of functionalized biomaterials and scaffolds in combination with stem cells for advanced biomedical devices and therapies.Objectives: Corwave LVAD is being developed to employ gentle oscillation of a membrane to propel blood, based on the wave motion of swimming fish. The pump output can be readily tuned by adjusting membrane oscillation frequency and magnitude. The purpose of this project was to increase the hydraulic efficiency, implement a physiologic pulsatility control algorithm, and confirm performance in animal implants. Methods: The fluid path of the pump was simulated by Fluid-Structure-Interaction FSI computational fluid dynamic analysis in COMSOL. Membrane size, oscillation frequency, oscillation magnitude, and the blood flow path were modeled and refined to improve hydraulic performance and eliminate areas of flow stagnation. Pumps were then tested in blood analogs and blood in mock circulation loops and in vitro hemolysis testing. Non-hermetic prototype pumps were implanted in a total of 25 sheep for acute and chronic implants. Results: Design simulations and individual component testing resulted in a pump which can generate 6 LPM of blood flow against physiologic pressures with maximum shear rates orders of magnitude lower than those of rotary blood pumps. Mock loop testing demonstrated the pumps have "flat" HQ pressure vs. flow curves. Three control methods were developed: fixed, asynchronous pulsatile, and synchronous pulsatile. Pulsatile modes generated dP/dt>400 mmHg/s using sensorless detection of native ventricle systole. Animal implants demonstrated low hemolysis and an absence of renal infarcts, but were limited to about a week in duration due to the non-hermetic sealing of the pumps. Discussion: Application of robust computational simulation and bench top testing have produced a unique new kind of VAD, offering the flow capacity and reliability of rotary pumps, but adding physiologically relevant pulsatility without excessive shear rates. Future efforts will implement full hermeticity to extend animal study durations and confirm von Willebrand Factor compatibility.Objectives: To investigate the colonization potential of Ovine Endothelial Progenitor Cells OEPCs on thermoplastic silicone-based polyurethane TSPU polymer under static condition. Methods: Functionalised TSPU F-TSPU was prepared by first subjecting the TSPU nonwovens to plasma treatment, followed by sterilization with 70 ethanol for seven minutes and then coupling with VEGF under sterile conditions. TSPU without any additional treatment or modification, identified as NF-TSPU, were used as control. OEPCs were seeded at the density of 5x104 cell/cm2 onto square 4 cm2 sized polymers under static conditions. Samples were taken at three time points: 1, 3 and 7 days. Samples were examined through live/dead staining and scanning electron microscopy SEM to check the progression of cellular colonization on the polymer. MTS assay was used to quantify the metabolic activity of the cells on the polymer. Results: Cells seeded on the polymer surface were clearly visible on the SEM images. On day 3, the cells could be observed as patches on surface but did not cover all the surface. By day 7, the cells had almost filled the polymer surface. In case of F-TSPU, at Day 7, the underlying fiber pattern of the polymer could be observed on the cell sheet covering the polymer. Whereas this feature was not seen for NF-TSPU, indicating that the cells attached firmly to F-TSPU compared to NF-TSPU. Bright green fluorescence was observed on samples that underwent live/dead staining, indicating presence of live cells. Only a few red spots, representing dead cells, were seen. The metabolic activity on NF-TSPU was quantified to be higher than in the F-TSPU, though the difference was not statistically significant. Discussion: Static cell seeding experiments showed that OEPCs were able to attach and grow on the surface of both types of TSPU. These are promising results suggesting that TSPU holds potential for engineering of a new generation of heart valve prostheses, which will be assessed in pre-clinical animal models in the near future.Objectives: To investigate the impact of rotational speed modulation on VAD hemodynamics by comparing the effects induced by the artificial pulse in the HeartMate 3 HM3, Abbott Laboratories, USA to those induced by the Lavare cycle in the HeartWare HVAD Medtronic Inc., USA. Methods: We compared the effect of speed modulation to a baseline case with constant speed and pressure head in both HM3 and HVAD using computational fluid dynamics CFD. CFD simulations employed high spatial 107 elements and temporal 2 degrees per time step resolution and one-way coupling to a lumped parameter model LPM of the cardiovascular system for physiologic boundary conditions. Lagrangian particle tracking was implemented to probe viscous and total stresses along cell paths. Results were validated against experimental pressure-flow HQ curves. Results: LPM results showed that HM3 and HVAD follow drastically different HQ loops during speed modulation, reflecting differences in pump characteristics, modulation rate and magnitude and associated fluid inertial effects. In the HM3, pump flow varied between -1.2 and 8.8 L/min during the artificial pulse vs. 3.7 to 7.1 L/min in the HVAD during the Lavare cycle. Induced aortic pressure pulses were comparable 11.3 vs 13.2 mmHg. CFD results revealed higher baseline shear stresses in the HVAD than in the HM3, coherent with previous studies. In both pumps, stresses were highest during flow acceleration. Sudden flow deceleration during the artificial pulse destabilized the flow and thereby also increased turbulence and total stresses on particles. Discussion: While speed modulation may contribute to pump wash out and pulsatility, it may also increase flow disturbances and shear. By relating shear stress to the rate and magnitude of the speed changes, this study sheds light on the impact of the imposed accelerations on the stress fields. This is of relevance for the establishment of both built-in and physiologically-controlled speed modulation sequences.Objectives: Thrombotic deposition is a major consideration in the development of implantable cardiovascular devices. Recently, it has been demonstrated that fluid mechanical shear micro-gradients play a critical role in thrombogenesis. The goal of the present work is to develop a predictive computational model of platelet activation and deposition that can be used to assess the thrombotic burden of cardiovascular devices. We have developed a comprehensive model of platelet-mediated thrombogenesis which includes platelet transport in the blood flow, platelet activation induced by both agonists generated at sites of vascular injury and shear micro-gradients, kinetics and mechanics of platelet adhesion, and changes in the local fluid dynamics due to the growth of a thrombus. Methods: A 2D computational model was developed using the multi-physics finite element solver COMSOL 5.3a. The model can be described by a coupled set of convection-diffusion-reaction equations, and it comprises 7 species: resting and activated platelets, agonists that induce thrombosis, and an anticoagulant agent. Platelet adhesion at the surface was modeled via flux boundary conditions. Using a moving mesh for the surface, thrombus growth and consequent alterations in blood flow were modeled. In the case of a stenosis, the notions of shear stress-induced platelet activation in the acceleration zone and platelet deposition in the expansion zone downstream of the stenosis were studied. Results: The model provides the spatial and temporal evolution of thrombosis in the flow field. The computed density of platelets adherent to the surface was validated against experimental data. The results confirm the importance of considering both mechanical and chemical activation of platelets. Discussion: The developed model represents a potentially useful tool for the optimization of the design of the cardiovascular device flow path.Objectives: The aim of the study was to analyze the effect of glucose concentration in the culture medium on metabolism and viability of the human umbilical vein endothelial cells HUVEC co-cultured with the human umbilical artery smooth muscle cells HUASMC on polysulfone semipermeable flat membranes. Methods: Cells were isolated from umbilical veins and arteries obtained by Caesarean section. Membranes with a surface area of 2 cm2 were covered with fibronectin to promote cells attachment and then were placed in specially designed inserts. Cultures were carried out in 12-well plates filled with culture medium containing glucose in normal 5 mM or high 20 mM concentration. The HUVEC and HUASMC were seeded on separate membranes or on both sides of the same membrane with density of 4,600 cells/cm2. Cultures have been conducted in standard conditions 37oC, 5 CO2 for 7 days. Activity of the reactive oxygen species ROS and cell viability were analyzed using the flow cytometry with DCFDA test and Annexin V / 7AAD test, respectively. Microscopic visualization of cells was made after fixation in formalin using hematoxylin and eosin staining. Results: In the HUVEC co-cultured with HUASMC the ROS activity was lower by 17 in the normal than in high glucose concentration. In normal glucose concentration the ROS activity of the HUVEC co-cultured with HUASMC was lower by 8 than in HUVEC cultured separately, whereas in high glucose concentration it was higher by 19. Viability of HUVEC cultured separately in normal glucose concentration was higher by 4 than in high glucose concentration. Microscopic visualization of cells did not reveal any morphological differences. Discussion: In terms of the ROS activity the obtained preliminary results indicate a beneficial effect of HUASMC on HUVEC co-cultured in normal glucose concentration. The opposite tendency was observed in high glucose concentration.Objectives: Continuous rhythm monitoring is useful for predicting, determining and treating a large class of diseases. An implantable heart monitor is useful for this purpose. it has small dimensions and records events over a timeframe measured in years. The objective of the study is to develop methods and algorithms for determining QRS-complexes and disorders in the work of the heart. Methods: Several sources were used to obtain the ECG: large collections of recorded physiologic signals PhysioBank and BioModule BioHarness 3. Records of database have annotations with tags of QRS-complexes and conclusions of doctors. BioModule BioHarness 3 record lead V4 or V5 dependent on position electrodes. Sampling frequency is 1000 Hz. The records were made on the basis of I.M.Sechenov University Clinical Hospital No.1, Russia. The monitor of physiological parameters and the device for daily monitoring of ECG was installed by the patient simultaneously. Recording was carried out for 24 hours simultaneously on both instruments. The device for daily monitoring of ECG is used to determine the sensitivity and specificity of the developed methods. MATLAB was used to implement algorithms. Results: Testing the algorithm with PhisioNet database resulted in a mean sensitivity of 99.21 and a mean specificity of 99.49. Testing the algorithm with Biomodule 3 resulted in a mean sensitivity of 99.02 and a mean specificity of 93.69. 16 records were analyzed. The analyzer gave an incorrect result in 2 cases. Discussion: A careful study of records revealed that algorithm sets false tags in the absence of contact between the electrodes and the surface of the human body. The device for daily monitoring of ECG continues to operate normally at the same time. A careful study of the annotations PhysioNet showed that they contain errors. Some records have been shortened or excluded for this reason.Objectives: The aim of this paper is to improve the design of the current commercial cannula being used by clinicians in order to minimise the blood flow stasis and thus avoid thrombosis in ECMO drainage cannula Methods: A CFD simulation is used in order to determine the optimum desing for the drainage cannula. The improved cannula design was fabricated and validated in-vitro using particle image velocimetry. Results: 16 different cannula designs were simulated and results were analysed in terms of velocity and mass flow rate at side holes and different sections of the cannula. Different parameters including side holes diameter, number, spacing, distance to the cannula tip, and angle relative to flow direction were investigated. Simulation results were in agreement with PIV. Discussion: This study demonstrated that the improved drainage cannula can significantly reduce blood flow stasis, and potentially thrombosis, in patients receiving ECMO.Objectives: The aim of this work was to study the influence of the constructive and personal characteristics of the wireless energy transfer system on the temperature distribution in biological tissues. The basic parameters of the transcutaneous energy transfer system, such as transmitted power, efficiency and displacements, were taken into account. The patients personal characteristics, such as the thickness of biological tissue layers, were also taken into account. To obtain data on the level of thermal damage, Henriques and Moritz study was used. Methods: The temperature data was obtained using the numerical solution of a bioheat equation by the finite element method. This article mathematically evaluates thermal injury based on Henriques and Moritz study, in which damage can be represented as the speed of a chemical process. Results: The temperature distributions in the tissues corresponding to the following types of displacements were obtained: axial, lateral, angular and also for the case where there are no displacements. Also the temperature distributions in tissues with different structures were obtained. Further, the temperature values at the maximum heating points were used to estimate thermal damage to the tissues. Discussion: It was established that when a wireless energy transfer system is operated without displacements, the severity of thermal injury does not exceed the threshold corresponding to a first-degree burn. It was also found that the patients personal characteristics are influenced by the temperature distribution in the case when a change in the thickness of a layer of biological tissue occurs in the region of the nearest heat source.Objectives: The objective of this work is the development of a two-phase blood model fluid for the use in upscaled flow models. In large scaled models of different flow geometries, frequently used water-glycerol mixtures are not sufficient because in critical regions with small gaps, e.g. between the rotor and the wall in cardiac support systems, the two-phases of blood have a great influence on the flow. Therefore, it is necessary that the model fluid represents both phases of blood: the plasma and the cells. Methods: Transparent alginate beads were used as a model of red blood cells. A water-glycerol mixture was used as a model fluid for plasma. The alginate beads made up about 40  of the volume to model the proper hematocrit. Furthermore, the beads have a slightly higher density than the surrounding medium and keep floating once in motion. This was achieved by embedding hollow glass particles in the spheres that have a density <1.0 kg/l, also enabling particle tracking and flow analysis using the Particle Image Velocimetry method. A camera tracks the flow path of the beads through the required geometry. Reynolds similarity was maintained. Results: Flow analysis has been performed in an upscaled model of blade and gap of a ventricular assist device. Compared to a water-glycerol mixture, the proposed blood model fluid enabled the visualization of the strong influence of the two phases on the flow. Because of their transparency, it was possible to visualize the flow field at several layers in the fluid by choosing the appropriate light sections. Discussion: The proposed blood model fluid showed promising results in an initial case. Moreover, the results of the experiment lead to a possible optimization of the geometry of future rotor blades in blood pumps in order to reduce hemolysis. Further channel geometries will be explored. In the long term, the model can be used for experiments on flow analysis and optimization in blood pumps or as a validation model for CFD-codes to calculate multiphase flows.Objectives: FTR, the death rate among patients with complications, is an emerging quality metric, that has received an increasing focus in the last years. Concerning the management and outcome in life-threatening complications due to ECMO-insertion there are only few reports available. We report the successful rescue therapy in two patients of our hospitals large ECMO-patient cohort with severe ARDS who experienced right ventricular RV-perforation after v-v-ECMO-cannula insertion by the Seldingers wire. Methods: RV perforation was assumed by deteriorating hemodynamics and immediate echo-cardiography showing cardiac tamponade during ECMO-cannula insertion. In cases of two patients an immediate bedside sternotomy was performed and the diagnosis verified. The source of bleeding was controlled and the tamponade was cleared. Either patient was transferred to the OR after ROSC for a definitive hemostasis and subsequent optimally controlled cannula placement. Results: Both patients survived this life-threatening event without neurological complications or further bleeding as well as no infections of the wounds. In Patient 1 the ECMO-therapy was not instituted immediately but later. This patient was weaned from ECMO after one week, survived six months but finally died due to an underlying disease. In Patient 2 switching the v-v- to an a-v-ECMO in the acute setting lead to a sufficient circulation during CPR. This patient was completely weaned from ECMO after 2 weeks and was finally discharged to rehabilitation after two months. Discussion: Successful management of seriously complicated ECMO- insertion is possible. The provision of an immediate bedside operative therapy and every kind of circulatory support to achieve ROSC seem to be essential in this setting. By reason of this concept a No-FTR situation could be achieved, which depicts a positive marker for quality of care for ECMO patients.Objectives: Blood trauma is an unintended consequence of mechanical circulatory support. Evidence indicates that exposure to high shears may induce sublethal changes to blood cells and also overt cell destruction. To determine whether such changes in blood properties have a functional impact on microcirculatory function, we examined the effect of surgery requiring cardiopulmonary bypass on the rheological properties of blood and also direct assessments of cell flux in the microcirculation. Methods: Thirty individuals requiring elective coronary artery bypass graft n = 15 or valve surgery n = 15 volunteered to participate. Dependent variables were collected prior to use of cardiopulmonary bypass, after 60 min of being on pump, and in the post-surgical period. Blood analyses included measurements of blood and plasma viscosity, red cell deformability, red cell aggregability, mechanical sensitivity of red cells, and levels of haemolysis, among others. Microcirculatory function was assessed using incident dark field visualisation of the sublingual vessels. Results and Discussion: Initial findings of this project indicate that blood viscosity and red cell aggregability are significantly affected by 60 min of cardiopulmonary bypass; these measures do not restore following the early post-surgical period. Microcirculatory assessments reveal altered transit and velocities of red cells, potentially indicating a link between biophysical assessments of blood and microcirculatory function. Extended presentation of this ongoing study will be presented, and discussed in the context of the secondary complications that still plague circulatory support patients.Objectives: Granulocyte and monocyte apheresis GMA, in which granulocytes and monocytes are removed from the patients whole blood, is used in the treatment of inflammatory bowel disease IBD. The objective of the present study was to clarify the effects of the anticoagulation methods used on blood clot formation in the column during GMA. Methods: Extracorporeal circulation was established for 60 minutes in male Sprague Dawley rats. A small column filled with 0.7 g of cellulose diacetate beads was used. We administered heparin at a total dose of 0.5 U/g by 2 different administration methods: the bolus administration method single bolus administration prior to extracorporeal circulation and the half bolus and half continuous administration method. During extracorporeal circulation the arterial pressure, column inlet pressure, venous pressure, and activated clotting time ACT were measured. The hemoglobin Hb concentration of the eluate of the residual clots was measured after the extracorporeal circulation. Results: Extracorporeal circulation was performed stably for 60 minutes without events like in-circuit coagulation, in both groups. The ACT reached its peak value within 30 minutes in the bolus administration group, to decline gradually thereafter; on the other hand, in the half bolus and half continuous administration group, the ACT remained nearly constant during the extracorporeal circulation. The Hb concentration in the eluate was significantly lower in the bolus administration group than in the half bolus and half continuous administration group. Discussion: The results indicated that even for the same total dose of heparin, clot formation in the column was influenced by the method of administration. We concluded that bolus administration of heparin, which led to early peaking of the ACT value was more effective to reduce residual blood clotting in the column as compared to half bolus and half continuous administration.Objectives: Suction at the inlet of a Left Ventricular Assist Device LVAD is common and known to cause adverse events. It occurs when tissue is drawn towards the cannula, thus leading to a reduced or even ceased flow. Moreover, the sudden occlusion can cause flow situations inside the device to be dramatically different from typical operation. The objectives were to quantify the effect of suction on the rotor bearing of an LVAD in an advanced development status and to mitigate unfavorable effects on the ongoing therapy and pump performance. Methods: A custom-made suction module was integrated into an established, active mock circulation loop upstream of an LVAD. This module comprises a membrane that can occlude the inflow cannula depending on the pressure in its fluid chamber relative to an adjustable air pressure chamber. The LVAD is equipped with rotor position sensors for online position tracking. Suction was initiated by reducing venous return and thus pump inlet pressure or by actively increasing air pressure in the respective chamber. The tests were performed with active and inactive suction speed control algorithms. Results: Spontaneous suction by reducing venous return was unlikely for the LVAD under test independently of the stationary air pressure applied. By actively provoking suction, increased radial force on the rotor was observed with uncharacteristic movement of the rotor. The radial gap remained satisfactory for all operational points and patient scenarios investigated. Suction was accurately detected by the controller and the speed reduced accordingly, thus leading to a relieve of the suction event. Discussion: Ventricular suction is a critical event in two ways as it negatively affects the LVAD therapy and secondly provokes uncharacteristic forces that might be critical for device performance. Future research is needed to ensure that experimental models of suction represent the variety of supported ventricles accurately.Objectives: Human amniotic membrane AM is employed in a broad field of applications as a wound dressing, corneal treatment, and scaffolding material. However, mechanical properties of native membranes vary depending on the zone of the placenta where the AM is obtained. For this reason, mimicking the AM through tissue engineering is a way to obtain a membrane with standardized properties. One technique to mimic the morphology of the extracellular matrix of AM is electrospinning. With electrospinning, a nanometric arranged similarly to the AM extracellular matrix made of different polymeric blends could be obtained. In addition, the correct storage is a key point in order to guarantee the availability of such tissue-engineered constructs for clinical applications. Methods: The AM extract is first prepared from native material and then freeze-dried. Afterwards, the AM-based scaffolds are obtained using blend electrospinning of polyethylene oxide and freeze-dried AM. Structural and compositional properties are assessed by scanning electron microscopy and confocal Raman spectroscopy. The mechanical properties are evaluated by static mechanical testing before and after cryopreservation. The scaffold is then seeded with multipotent stromal cells. The biocompatibility of the tissue-engineered AM is analyzed using cytotoxicity assay. Results: The tissue-engineered membrane presents a nanostructured morphology similar to the extracellular matrix of AM. The compositional results can detect the presence of collagen, fibronectin and hyaluronic acid in the final material. Preliminary results present changes in the mechanical properties related to the cryopreservation. The fabrication technique for the tissue-engineered AM seemed not to alter the biocompatibility with stem cells. Discussion: The AM is ideal for treatment of different tissues, and in combination with polymeric materials by electrospinning, it could be a suitable candidate for a tissue-engineered membrane with standardized properties.Objectives: A mixed flow blood pump with an outer casing diameter of 9 mm is proposed as part of an intracorporeal membrane oxygenator. The usage of a blood pump deals with major challenges in the design of an intravenous device, specifically overcoming the pressure drop and enabling a controllable blood flow. This study presents the design, manufacturing, experimental and numerical investigation of the pump prototype. Methods: The pump prototype was manufactured using a Kudo3D Titan 1 Kudo3D Inc. Dublin, USA 3D printer, utilizing liquid photo polymeric resin coupled with a digital light processing DLP curing process. A static test loop consisting of the pump, ultrasonic flow sensor and pressure sensors was used to obtain the pump characteristics at different rotational speeds ranging from 23000 to 32000 rpm. A water glycerol mixture with a dynamic viscosity of 3.4 mPa.s was selected as working fluid. Computational fluid dynamics CFD simulations were carried out using the open source CFD code OpenFOAM R. Results: Over the investigated range from 30 to 500 mmHg, stable characteristic curves at different rotational speeds were measured. At 27700 rpm, a flow rate of 1.4 L/min against a pressure difference of 174 mmHg was observed. CFD simulations show a good agreement of the numerically predicted and measured pump characteristics, with an average deviation of about 8 . Discussion: The applied manufacturing process was capable to produce thin structured, complex parts down to 300 mu m wall thickness, durable enough to be used in experimental investigations up to 32000 rpm. Experimental pump characteristics show adequate performance of the proposed mixed flow pump for the intended purpose. CFD simulations accurately match the experimental pressure difference.Objectives: Ex vivo lung perfusion EVLP is a technique to evaluate lungs before transplantation. As a tool for experimental research, this method allows organ modification and assessment. However, the costs are high. The aim of this study was to establish a low-cost EVLP model on the one hand and to investigate the feasibility of surgical graft downsizing during EVLP on the other hand. We compared a lobectomy, a stapled wedge and a sutured wedge resection during EVLP. Methods: Pigs of 60 kg weight were used for organ harvesting. After cardiac death lungs were retrieved, flushed with Perfadex and perfused with autologous blood for 4 hours of EVLP. The circuit was assembled from an ECMO system consisting of a Deltastream DP2 pump Medos, an Affinity Fusion oxygenator Medtronic and a hardshell reservoir. After 90 minutes of perfusion a lobectomy was performed. At intervals of 30 minutes, a stapled wedge resection and a sutured wedge resection were performed. Air leak and the loss of blood were measured. The function of the lung was monitored including pulmonary artery pressure and blood gas analysis. Results: 20 porcine lungs were used to establish a stable model. Another 7 lungs underwent surgical interventions. The blood loss after lobectomy 3,14 /- 4,14 ml/min and after stapled wedge resection 5,29 /- 8,1 ml/min was lower than after sutured wedge resection 33 /- 17,36 ml/min. There was no major difference in air leak after the surgical interventions lobectomy: 0,06 /- 0,16 l/min vs. stapler: 0,03 /- 0,09 l/min vs. suture: 0,04 /- 0,1 l/min. The oxygenation performance was satisfactory. The pulmonary artery pressure was in a physiological range. Bleeding into the parenchyma after suture was higher compared to the other interventions. Discussion: The low-cost custom-made EVLP model presented here is feasible and stable. Surgical interventions during EVLP are possible. Performing a lobectomy and a wedge resection is superior to a wedge resection with suture regarding to the blood loss.Objectives: Human induced pluripotent stem cells hiPSCs possess an exceptional differentiation potential, as they are able to differentiate towards cells from all three germ layers. Owing to this capability, they are an attractive cell source for regenerative medicine applications and in vitro disease models. To date, coated substrates with Matrigel R and Geltrex TM have been providing promising platforms for hiPSC culture and differentiation. Besides many advantages, both substrates provide some drawbacks such as batch-to-batch variation and xenogeneic origin, which pose problems in potential clinical applications. To tackle these issues, a matrigel-mimetic synthetic biomaterial able to maintain hiPSCs in culture is required. Cell behavior is controlled by a complex set of biophysical e.g. stiffness as well as biological e.g. growth factors parameters. In this study, we want to develop a fully synthetic hiPSC-supporting hydrogel that allows investigating the influence of biophysical and biological parameters on hiPSCs. Methods: The substrates were prepared via crosslinking starPEG and heparin molecules with different molar ratios. Substrates were biofunctionalized with two methods based on NHS ester binding to amine groups of laminins in different ratios. hiPSCs were seeded on substrates and the medium was changed every day. Cells were stained after 72 hours for pluripotency markers OCT4-SSEA4. Results: Functionalized substrates with storage modulus of 4kPa maintained hiPSCs culture. On the other hand, no adherent cells were observed on the substrates with a storage modulus of 1kPa and 10kPa. Discussion: StarPEG-heparin hydrogels functionalized with laminin mixtures clearly represented an appropriate matrigel-mimetic growth substrate for the hiPSC culture and maintenance of their pluripotency. 3D cultures have already been shown to be superior to 2D and it is an outlook of this work.Objectives: Radiological procedures utilizing intravascular iodinated contrast media are being widely applied for both diagnostic and therapeutic purposes and represent one of the main causes of contrast-induced nephropathy CIN.In this hospital-based study we tried to assess predictors for the development of CIN in patients undergoing cardiac catheterization. Methods: A total of 5604 patients undergoing coronary angiogram/PCI from 2007-2017 were enrolled in the study. Multivariate predictors of CIN were identified by logistic regression using stepwise selection with entry and exit criteria of p < 0.1. A two-sided 95 confidence interval CI was constructed around the point estimate of the odds OR. A p value Results: CIN occurred in 6 1 parts per thousand patients. The mean age of patients suffering from CIN was higher than in the whole population 66.5 /- 31.15 vs. 58.66 /- 28.57, p=0.03. Characteristics of patients who developed CIN were: older age, diabetes, higher creatinine and lower EF. The incidence of CIN in patients with diabetes was higher and statistically significant 84 vs. 16, p=0.01. Emergency cases were at higher risk of developing CIN than elective patients 85 vs.15, p=0.001, respectively. Diabetes, CKD and EF < 50 were independent predictors of CIN RR 2.4, 95 CI: 1.88 - 7.132, p=0.008; RR 3.1, 95 CI: 2.17 - 6.682, p=0.003; RR 1.6, 95 CI: 2.88 - 7.132, p=0.01, respectively. Discussion: The development of contrast-induced nephropathy in patients who underwent angiography and PCI was mainly related to older age, diabetes, lower GFR and heart failure, but not contrast material exposure.Objectives: To examine the efficacy of the short-time total liquid ventilation TLV with oxygen micro/nano bubble dispersed saline for severe and lethal respiratory failure experiment 1 and 2, respectively induced by intratracheal administration of lipopolysaccharide LPS. Methods: In experiment 1, fifteen rats were divided into three groups: LPSTLV group treatment group received 5-min TLV 20 min after LPS 5 mg/kg administration, LPSMGV group received 5-min mechanical gas ventilation MGV after the LPS administration, and PBSMGV group received 5-min MGV 20 min after administration of phosphate buffered saline PBS instead of LPS. Ventilation weaning was performed after respiratory management for three hours. At two days after each treatment, hemodynamics and blood gas parameters were measured and the rats were subsequently sacrificed. The excised lungs were used for histological analysis and bronchoalveolar lavage BAL. In experiment 2, ten rats were divided into two groups: LPSTLV group and LPSMGV group. Ten mg/kg LPS was used for inducing lethal acute lung injury and the survival rate was used for the primary outcome. Results: In experiment 1, significant differences were confirmed between LPSTLV group and LPSMGV group in the value of blood oxygen, weight loss ratio, and inflammatory cytokines levels in BAL-fluid p < 0.05. On the other hand, there was a significant difference only in arterial oxygen pressure between LPSTLV group and PBSMGV group. Severe inflammation in LPSMGV group was also confirmed in the histological analysis. In experiment 2, all rats in LPSMGV group died within two days, whereas 80 of the rats in LPSTLV group survived for one week. Moreover, the conditions of the survival rats showed normal in the pulmonary function and the alveolar structure after one week. Discussion: It is indicated that 5-min TLV with oxygen micro/nano bubble dispersed saline after LPS administration prevented severe respiratory failure and improved the survival rate dramatically without any disorders.Objectives: Natural biological tissues have electrical conductivities and it is required that artificial tissues also possessed such a property. We investigated the specific conductivity s of the bionanomaterial in the composition of bovine serum albumin BSA and single-walled carbon nanotubes SWCNT. Methods: We investigated the specific conductivity s of the bionanomaterial in the composition of bovine serum albumin BSA and single-walled carbon nanotubes SWCNT. The aqueous dispersion contained 25 wt. BSA, 0.45 wt. SWCNT. Dispersion on silicon substrates was applied by screen printing. After drying, the formed layers had approximate sizes: area - 520 mm2, thickness - 5 15mm. For all samples the dependence of s on temperature t was recorded when they were in a thermostat. The st curves were taken with a numerous cycle n = 1 5 with increasing and decreasing temperatures in the region of t 20 45 degrees C. Results: The following data was obtained. With increasing n, the values of s and the hysteresis on st decreased: at n = 1 - s 0.11 S/m, temperature resistance coefficient a -0.004 K-1, hysteresis on st, i.e. R/R0 0.90, where R0 is the initial resistance, R is the final resistance; at n = 5 - s 36 S/m, a -0.002 K-1, R/R0 0.95. Spectral studies in the optical range did not reveal any changes in the sample before and after a cyclic thermal action, which underlines the stability of BSA. Thus, in the layers of the BSA/SWCNT the s increases and the hysteresis on st decreases with n = 1 5. Apparently, this behavior is associated with an increase in the number of contacts between nanotubes in the albumin matrix. Discussion: The studied layers of the composite bionanomaterial BSA/SWCNT has an acceptable value of specific conductivity 1,10 S/m and mechanical properties hardness 300 MPa and is promising for biomedical applications.Objectives: Determine the outcomes and survival of patients who receive a temporary total artificial heart TAH as bridge to transplant or as bridge to decision by evaluating data from the Interagency Registry for Mechanically Assisted Circulatory Support INTERMACS database in the U.S. Methods: Review of the INTERMACS database between 2006 and April 2017 revealed a total of 450 patients had been implanted. The data was reviewed to determine survival rates, adverse events, and competing outcomes. Risk factors for mortality were determined using hazard function analysis. Results: Data from 450 patients 87 men; mean age, 50 years were available in the INTERMACS database. The 2 most common diagnoses were dilated cardiomyopathy 50 and ischemic cardiomyopathy 20. Risk factors for right heart failure were present in 82 of patients. Most patients were INTERMACS Profile 1 43 or 2 37 at implantation. There were 266 patients who eventually underwent transplantation, and 162 died. Overall 3-, 6-, and 12-month actuarial survival rates were 73, 62, and 53, respectively. Risk factors for death included older age p = 0.001, need for pre-implantation dialysis p = 0.006, higher creatinine p = 0.008 and lower albumin p < 0.001 levels, and implantation at a low-volume center <= 10 TAHs; p < 0.001. Competing-outcomes analysis showed 71 of patients in high-volume centers were alive on the device or had undergone transplantation at 12 months after TAH implantation vs 57 in low-volume centers p = 0.003. Discussion: Patients receiving TAHs have rapidly declining cardiac function and require prompt intervention. Experienced centers have better outcomes, likely related to patient selection, timing of implantation, patient care, and device management. Organized transfer of knowledge to low-volume centers could improve outcomesObjectives: To analyze the hemodynamic effect of ventricular assist devices using a multi-scale mathematical model of the human extracellular fluid system. Methods: A Ventricular Assist Device VAD model has been incorporated into the Digital Avatar, a multi-scale, closed-loop mathematical model of the human fluid system is composed of a network of major arteries, major veins, and lumped parameter models for the heart, pulmonary circulation, arterioles, capillaries, venules, cerebrospinal fluid and brain interstitial fluid. The mathematical model relies on one-dimensional PDEs for blood vessels and on ODEs for lumped parameter models. Results: Based on the model, we have quantified the hemodynamics of the systemic circulation with and without the medical device in healthy and unhealthy scenarios. Results will be shown at the conference. Discussion: Traditionally, bench testing, animal studies and clinical trials are used as the main source of evidence for getting medical devices on the market in the US. The associated costs and time are enormous and represent a burden for MedTech companies. In addition, the FDA wants to minimize animal and human study testing. In recent years, Computational Modelling and Simulation CMS has made huge steps forward. CMS are a valuable regulatory tool for its significant cost-savings in evaluating medical devices, simulating performance under scenarios that may not be possible with human use and quantifying the safety and efficacy of medical devices. However, the existing CMS tools are highly specialized for each field and do not provide a platform that integrates all human body mechanisms. Computational Lifes vision is to replicate all body mechanisms into a single, integrated software called Digital Avatar platform. The Digital Avatar platform will enable MedTech companies to perform feasibility testing and quantify the safety and the efficacy of ideally any type of medical device. Here we will show an application with the VAD system.Objectives: Three-dimensional 3D in vitro cultivation systems have gained increased attention, since using these systems allows the precise study of mammalian cell physiology, intercellular interaction, as well as cell-matrix interactions. Another important cell culture parameter, often neglected in traditional cultivation, is the concentration of dissolved oxygen. Generally, cell culture is performed under ambient oxygen saturation 21, while the oxygen amount available in vivo in human tissue is much lower 1 to 15. The stiffness of in vivo tissues and in vitro constructs also plays a crucial role in cell fate and physiology. Methods: Semi-synthetic hydrogels with tunable properties allow creating a wide range of in vitro material stiffness. The development of hydrogel-based 3D cell culture gradient systems enables to obtain complex, but precisely defined in vitro microenvironments, which in turn allow a systematic investigation of the influence of microenvironmental conditions. Oxygen gradients in 3D and 2D cell culture can be created with the help of incubation chambers. Results: Encapsulation of mesenchymal stem cells containing fluorescent hypoxia sensors reveals specific conditions where the hypoxic response is activated and determines the influence of hydrogel stiffness on the hypoxic response. In the presented study, exact correlations between in situ O2 concentrations in 3D cell cultures hydrogels and microtissues and hypoxia inducible factor HIF activation are disclosed for the first time. Discussion: Creating stable in vitro oxygen gradients in 3D hydrogel-based cultures with stiffness variety helps to better understand and predict cell behavior survival, cytokine expression, spontaneous differentiation of stem cells or migration capacity in physiological and pathological in vivo conditions, as well as to find optimal conditions to simulate the in vivo cell niche.Objectives: Most breast and prostate cancers metastasize to the bone. To establish macrometastases, the disseminated tumor cells DTC need to have stem cell-like properties such as self-renewal and differentiation into other tumor cells. They compete with hematopoietic stem and progenitor cells HSPC and mesenchymal stem/stromal cells MSC for their specific microenvironment or niches. Apart from biochemical factors in niches, biophysical cues also influence such interactions. The aim of this project is to investigate the intercellular crosstalk that occurs among these cells and the signalling pathways in a 3D porous hydrogel that was previously established in our group. Methods: The hydrogel mimics in structure the trabecular region of the bone marrow. Cell-cell and cell-matrix interactions will be investigated using fluorescence imaging techniques. The pathways will be analyzed with the help of Western blots and specific inhibitors. Cytokine array and ELISA will be used for assessing auto- and paracrine communication via soluble factors. The biophysical parameters such as the 3D architecture and mechanical properties will also be tuned and their effect studied. Results: The coculture of the breast cancer cell line MDA-MB-231 with bone marrow-derived MSCs and the leukemic cell line KG-1a reveal formation of spheroid-like structures in the gels. Experiments are being performed to study the cell-cell interactions within these structures. Parallel experiments are being carried out regarding the effect of the 3D architecture, which will be introduced by varying the porosity of the gel. Discussion: The results of these studies will help us to develop in vitro models of different tumor cell niches that will allow us in gaining a fundamental insight into the early events of bone metastasis. Tuning these parameters in the 3D bone marrow analog, we can have an understanding about how the biophysical cues affect the cellular activities.Objectives: Bioartificial liver devices aim to replace the detoxification and metabolic functions of the liver in people with liver failure. Shear stress and local velocities created by flow inside porous scaffolds need to be optimised to improve cell viability and avoid inflammation, however there are no relevant studies on the topic to date. Here, fluid dynamics inside porous cryogels were characterised using a purpose-built micro-PIV system. The aims were to develop a micro-PIV setup to record the flow inside cryogel structure and to characterise the nature of flow inside the cryogel channels by video post-processing in order to determine inlet design. Methods: HEMA- based cryogels were synthesised by cryogelation technique and the microstructure was analysed by SEM, confocal microscope and mu CT. Open porosity, overall porosity and permeability were extrapolated from mu CT. A mu PIV set up was developed to visualise flow inside cryogels which was composed of a digital camera with a long-distance microscope objective, a pulsating LED light source, a pump and an optical chamber to host the cryogel. Glass beads of 10 mu m size were used as tracers to mimic red blood cells. Videos were analysed with PIVlab in Matlab which allowed the interpolation of local velocities. Results: Cryogels possessed an open porosity, with pore size up to 100 mu m, as well as an interconnected network of pores, making it suitable for a perfused system. Fluid dynamics were optimized using a mu PIV setup which allows for the visualisation of flow inside the cryogel channels. Discussion: Videos showed that internal flow did not reorganize itself through the channels and was found to be laminar with Re < 1 with low vorticity. This suggests that molecules and toxins of the blood in the centre of a channel would not interact with hepatocytes seeded on the cryogel pore walls. Hence, flow reorganization could be promoted through a layered bioreactor. A layered cryogel structure was introduced to improve flow characteristics and improve toxin/hepatocyte interaction.Objectives: Generation of a large-scale stem cell derived bioartificial cardiac tissue BCT as a potential therapeutic option to replace damaged myocardium after myocardial infarction. Methods: Large-scale BCT was generated from cardiomyocytes CMs derived from human induced pluripotent stem cells hiPSC mixed with human fibroblasts and a hydrogel solution in a newly designed tissue chamber with a diameter of 43 mm. The BCT development was microscopically recorded and evaluated. Maturation of CMs within the tissue was confirmed by immunofluorescence staining for cardiac markers alpha-SA, cTnT as well as cell junction proteins Con. 43, N-Cad. The integration of the tissue chamber in the novel custom-made bioreactor allowed investigating the initial parameters for electromechanical stimulation such as pressure, stress amplitude, frequency necessary for proper tissue maturation in a more physiological manner. To investigate the calcium handling through the tissue a new transgenic hiPSC line containing a genetically encoded calcium indicator GCaMP6f has been developed. Results: Microscopically evaluation at different time points reveals a progressive reduction of nearly 27 of the initial volume after 15 days, ccorresponding with an increased intercellular connectivity. After 48h, the tissue exhibited spontaneous contractions and coordinated and rhythmic contraction within the whole tissue after 6 days. As preliminary analysis, the beating of the tissue was video-optically recorded showing an estimated beating rate of 102 beats per minute. Immunofluorescence staining performed at day 15 showed highly organised cross-striations of sarcomere proteins. Initial integration of the tissue chamber in the bioreactor and implementation of the new GCaMP6f reporter cell line is still on-going. Discussion: The initial results confirmed that is possible to generate a large-scale BCT. However, the properties of the tissue have to be further investigated.Objectives: The Fontan procedure is performed for univentricular correction of the heart in pediatric patients with tricuspid atresia. In the Fontan circulation, the veins connect directly to the pulmonary artery. However, as a result of the procedure the load on the single functional ventricle of the heart and veins increases. Eventually, patients need a heart transplantation. A possible solution to the circulation deficiencies of the Fontan patients could be the ventricular assist device VAD. The VAD is implanted between veins and the pulmonary artery. The aim of this study is to develop a model of the Fontan circulation with VAD support. Methods: We use mathematical models to reduce the time and resources spent on in vivo research. Nonlinear mathematical models represent sections of the cardiovascular system CVS in the form of blocks. Pressure variations in the CVS sections are described by a system of differential equations. Results: Appling VAD to the Fontan patients exhibits the potential to increase the venous return of blood. According to the Frank-Starling law, the stroke volume increases with increasing venous return. VAD creates a necessary pressure differential between the venous and pulmonary sections of CVS. Discussion: The interaction model of the Fontan circulation with VAD support demonstrates the normalization of the pressure and volume distribution in the CVS. This indicates that the method studied in this work may be a way to optimize the health condition of patients after Fontan procedure.Objectives: Autologous pericardium is one of the most ideal materials for cardiovascular reconstruction including pulmonary arteries and valvular cusps especially in pediatric patients. To improve the surgical handling, chemical pretreatments such as dehydration by ethanol ET or crosslinking by glutaraldehyde GA have been commonly attempted. In this study, we compared the effects of such chemical treatments with short period 10 min on the mechanical properties of the porcine pericardium. Methods: Porcine pericardium n=3 was separated into 3 groups, raw group with no treatments RAW, that under dehydration with 70 ET for10 min ET and crosslinking with 0.6 GA for 10 min GA. We measured 5 parameters of mechanical properties: the burst pressure, suture retention strength, ultimate tensile strength UTS, ultimate strain  and stimulus modulus. Results: Minimal burst pressure of RAW group, ET group and GA group were 12160 mmHg, 10640 mmHg, and 8360 mmHg, respectively, all of which were high enough for cardiovascular substitutes. There were no significant differences in suture retention strength RAW: 3.242 /- 0.261 N, ET: 3.298 /- 0.214 N, GA: 2.985 /- 0.165 N, p=0.529, UTS RAW: 18.238 /- 1.987 MPa, ET: 19.104 /- 2.234 MPa, GA: 16.059 /- 1.233 MPa, p=0.486 or Youngs modulus RAW: 4.964 /- 0.551 MPa, ET: 4.544 /- 0.719 MPa, GA: 3.921 /- 0.288 MPa, p=0.408. Only the ultimate strain of GA group was significantly higher between the 3 groups RAW: 35.64 /- 2.82 , ET: 37.99 /- 2.38 , GA: 46.39 /- 3.96 , p=0.046. Discussion: Short period of chemical treatment using ET and GA might make it possible to improve the surgical handling without deteriorating the mechanical properties of pericardium.Engineering modelling and simulation have been being used in different industries for better product and process development for already many decades. Together with the developments in medical imaging, this engineering approach can now be applied to understand how different human organs function and eventually develop devices to replace them or support their correct functioning. A digital environment for development of artificial organs and medical devices is possible with correct application of engineering modelling and simulation. These concepts enable researchers and engineers to perform design iterations rapidly and with ease. This is why the application of engineering modelling and simulation is considered as one of the very important bio-technological advancements in the recent years and is projected to be so in the close future. In this respect; cardiology is one of the areas where this potential is being converted to valuable outputs. Virtual modelling and analysis of a representative beating human heart, patient specific heart modelling or digital prototypes of heart devices are main research interests of an interdisciplinary community of cardiologists, surgeons, engineers, and physicians. In the scope of cardiovascular simulations; patient specific or normalized among a group of patients medical imaging based CFD simulations and coupled multiphysics FSI simulations are the main approaches for generating the simulation frameworks moving forward in the road to ultimate digital twins of the human heart. This presentation is intended to give insights about various processes that lead to higher fidelity digital representations in the scope of cardiovascular simulations, including below:- CFD model generation based on CT scans - Non-invasive digital angiography  FFR CT Fractional Flow Reserve - Post-TAVI coronary flow - Blood flow in SIMULIA LHHM Living Heart Human Model with mechanical valves - Transcatheter tissue engineered aortic valve FSI simulation - Evolut R stent deployment in LHHM - Mitral Valve and Mitral Clip FSI simulation The presentation intends to give a brief introduction on the simulation concepts/approaches which can be applied to cardiovascular problems. While it intends to encourage the community to take the advantage of these concepts in cardiovascular research, it highlights the importance of correct modelling and accurate numerical simulations.Objectives: The Berlin Heart EXCOR is the most commonly used ventricular assist device in children. Pump thrombosis is a major complication of the system and is most likely to occur in smaller pumps used for children under the age of 2. Pump exchange is seen as the therapeutic gold standard, but is associated with high mortality if in- or outflow cannulas are affected and therefor have to be changed surgically. Methods: Herein we report our experience in the use of tissue-type plasminogen activator Alteplase, Actilyse R for the therapy of an acute pump thrombosis subtotally occluding the inflow cannula of a 7 month old girl supported by a Berlin Heart EXCOR LVAD. The infant was supported by LVAD therapy used for acute left ventricular failure due to fulminant myocarditis, and the system was implanted 20 days prior to when the thrombotic event was reported. A treatment protocol of 1.2mg/kg - initial bolus application of 0.12mg/kg/2mins - of Alteplase infusion given continuously for 120mins was used, and the assist device was connected to a Maquet Quadrox-I neonatal and pediatric oxygenator to obviate thrombembolic complications. Results: The infant supported by Berlin Heart EXCOR LVAD was haemodynamically stable prior to the thrombotic event, and anticoagulation was performed with a continuous heparin infusion. Within a couple of hours, she became tachycardic and developed signs of low cardiac output, and a subtotal occlusion of the inflow cannula of the Berlin Heart System was diagnosed. As surgery was felt to be highly problematic, we decided to infuse Alteplase and achieved short-term success with resolution of the pump thrombosis within 2 hours. There was no adverse event. Discussion: Tissue-type plasminogen activator thrombolysis might be an effective treatment for Berlin Heart EXCOR thrombosis in haemodynamically compromised children. Additionally, the use of a Maquet Quadrox-I oxygenator might minimize the risk of major complications i.e. thromboembolism that have been reported in the literature.Drug discovery and development to date has relied on animal models, which are useful, but fail to resemble human physiology. The discovery of human induced pluripotent stem cells hiPSC has led to the emergence of a new paradigm of drug screening using human patient- and disease-specific organ/tissue-models. One promising approach to generate these models is by combining the hiPSC technology with microfluidic devices tailored to create microphysiological environments. Such organ-on-a-chip platforms OoCs or microphysiological systems combine human genetic background, in vivo-like tissue structure, physiological functionality, and "vasculature-like" perfusion. Using microfabrication techniques, we have developed a variety of OoCs that incorporate complex human 3D tissues, such as retinal, choroidal, cardiac, pancreatic, and adipose tissue. By generating microphysiological environments the platform is capable of keeping the tissues viable and functional over multiple weeks. The OoCs generally consist of three functional components: organ-specific tissue chambers mimicking in vivo structure and microenvironment of the respective tissues; "vasculature-like" media channels enabling a precise and computationally predictable delivery of soluble compounds nutrients, drugs, hormones; "endothelial-like" barriers protecting the tissues from shear forces while allowing diffusive transport. The small scale and accessibility for in situ analysis makes our OoCs amenable for both massive parallelization and integration into a high-content-screening approach. To facilitate and enable the adoption of OoCs in industrial and non-specialized laboratories, we have developed technologies for automated 3D tissue generation and for the flexible plugplay connection of individual modules to multi-organ-chips. These technologies paired with the versatility of our OoCs pave the way for applications in drug development, personalized medicine, toxicity screening, and mechanistic research.Objectives: The HeartMate 3 HM3; Abbott, USA is a new compact LVAD which is associated with several new technical features e.g. fully magnetically levitated pump, artificial pulse, large pump gaps, modular driveline, etc.. These benefits are supposed to lead to superior outcomes and reduced adverse events as compared to its predecessor HeartMate II HMII; Abbott, USA and other comparable assist devices. With this study, we investigated the effect of HM3 implantation on the occurrence of gastrointestinal GI bleedings compared to other LVADs. Methods: We retrospectively studied a patient cohort of 595 patients who were supported with left ventricular assist devices. 170 Patients were supported by HMII, 81 Patients were supported by HM3 and 344 Patients were supported by HVAD HeartWare, Medtronic, USA. Data was determined through retrospective examination of medical records. Exclusion criteria were biventricular assist devices and other types of assist devices as well as LVAD exchange and re-operative procedures. Results: Out of 595 patients 146 patients 24.4 presented with gastrointestinal bleeding after LVAD implantation. In the multivariate analysis GI bleeding was significantly dependent on age but not on the BMI or gender of the patient. Statistically we were able to show that the occurrence of GI bleedings is significantly connected to general bleedings but not to cerebral hemorrhages. The subgroup analysis revealed that the occurrence of GI bleedings was higher in the HVAD 26 and HMII 26 groups compared to the HM3 patients 14. However, statistical significance could not be achieved. Discussion: The novel HeartMate 3 shows a promising adverse event profile with a reduced rate of gastrointestinal bleedings compared to its competitors. However statistical significance could not be achieved in this study. Thus, larger multi-center cohort analyses are needed.Objectives: In medical practice, it is necessary to control the movements of various parts of the body: limbs, joints, chest, tumors, etc. For such purposes, the most suitable are the composite bionanomaterials in the composition of carbon nanotubes. We investigated the strain sensor tensoresistor based on the layers of the bionanomaterial contained biological material bovine serum albumin, BSA - matrix and multi-walled carbon nanotubes MWCNT - filler. Methods: The aqueous dispersion of 25 wt. BSA/0.3 wt. MWCNT was applied by screen printing on flexible polyethylene terephthalate substrates. After drying layers by laser irradiation  970 nm various parameters of layers were controlled, i.e. resistance R, bending angle q, number of cycles n, measurement time, etc. One measurement cycle corresponded to a change within the range q >>/- 150 degrees. Results: The slopes of Sq = 1/R0dR/dq of the Rq curves were considered to be strain-sensitivities, where R0 is the resistance of the sensor with q=0. It was found that with increasing the number n, R and Sq increase and the hysteresis on Rq decreases. For the tensoresistor obtained: specific resistances 0.1 divided by 1 Wxm, Sq1.0 divided by 1.5/grad. These results are high. Discussion: The physical picture of the change in the Rq is as follows. When q>0, compression occurs, the amount of contacts between the nanotubes increases and the resistance drops relative to the state q=0. When q<0, the nanotubes move away from each other, the amount of contacts between the nanotubes decreases and the resistance increases relative to the state q=0. The examined layers of the bionanomaterial BSA/MWCNT as a strain sensor is of a particular interest for medical practice. In particular, for monitoring: movements arms, blinking and detection of signs of pathology respiratory diseases, angina, et. al..The new European Medical Device Regulation MDR is applicable on 26th May 2020. Then the current Medical Device DirectiveMDD and its national transpositions like the current German Medical Device Act Medizinproduktegesetz will be replaced by the new rules. Further transitions periods for existing certificates will not help for devices of the lowest risk class and will not free parties from taking into consideration the new rules for handling of device problems or registration. The new certification of existing devices in time will be challenging, because the accreditation and notification of notified bodies in accordance with the new requirements, which are needed for certification of higher risk class medical devices, is ongoing very slowly. The new European database is still not gone live, but the device and economic operator registration is dependent on the functioning of that database. A strong focus of the new regulation is on clinical evaluation of medical devices, which will leave manufacturers and developers probably with open questions. For example, what extend of clinical data is sufficient for keeping an existing device on the market The presentation will highlight several obvious or less obvious changes due to the new regulation. It will provide guidance where to find additional answers for specific questions. And it will give an outlook on upcoming additional legislation and guidance.Objectives: Exercise capacity in patients with a continuous-flow left ventricular assist device LVAD is 50 compared to healthy subjects. Some patients show a better performance than others. Aim of this work is to investigate the factors responsible of this inter-subject variability. Methods: A cardiorespiratory simulator CRS was developed and combined with a model of the HVAD running at 2700 rpm to reproduce an "average" LVAD patients hemodynamic at rest and at exercise. The CRS was validated using the average hemodynamic data of >200 LVAD patients from the literature. Starting from this "average" LVAD profile, a sensitivity analysis was conducted by simulating a left/right ventricular contractility Elmax/Ermax and heat rate HR 20 higher than value usually observed at exercise. Results: The simulated "average" LVAD profile at rest is in full support with a total cardiac output CO of 4.8 l/min and a wedge pressure Pw of 13 mmHg. At exercise Pw increases to 26 mmHg and CO increases to 6.8 l/min, of which 5.3 l/min pumped by the LVAD QLVAD and 1.5 l/min by the ventricle QLV. An increased Elmax at exercise assures lower Pw 22 mmHg and higher CO 7.4 l/min due to a better contribution of the ventricle QVAD=4.7 l/min, QLV=2.7 l/min. An improved HR also helps to sustain a better CO 7.1 l/min as a combined effect of both pumps QVAD= 5.4 l/min, QLV=1.7 l/min. A better Emaxr induces a smaller improvement in CO 7.0 l/min due to the increased LVAD support QVAD=5.5 l/min, QLV=1.5 l/min, however Pw reaches 31 mmHg. Discussion: A better left ventricular inotropic response is mostly effective in accommodating a higher CO at exercise. A better right ventricular function has mild effects on CO and can increase Pw if not sustained by a good ejection on the left side. These results should be taken into account for the design of a physiological LVAD speed controller, tailored on specific patients needs.The electro-hydraulically actuated Carmat total artificial heart C-TAH is designed for biventricular replacement in patients suffering from end-stage heart failure, either as bridge to transplant or destination therapy. It generates pulsatile flow which is automatically adapted to the patients physiological needs. The device contains bio-prosthetic blood contacting materials, allowing for low dose anti-coagulation. A multi-centre European study is currently underway to assess the safety and performance of the C-TAH, with 6-months survival as primary objective. Preliminary results will be presented from a cohort of 10 patients included in the study.Objectives: The mechano-biological behavior of mesenchymal stem cells MSCs in 2D and 3D cultures is determined by formation of actin filaments which occur as branched stress fibers. For clinical-scale expansion of functional MSCs, the use of xeno-based serum products is prohibited, but human platelet lysate HPL can potentially replace fetal bovine serum. Here we investigate actin filament formation and mitochondrial morphology in the 2D and 3D MSC culture supplemented with HPL and analyzed immunomodulatory molecules GARP, Foxp3 and IDO following pro-inflammatory stimulation. Methods: Human platelet lysate composition was determined with a proteome profiler assay. Amnion derived MSCs were cultured in HPL supplemented media and MSC morphology was analyzed by flow cytometry, confocal microscopy and scanning electron microscopy. Mechano-biological properties of cultured MSCs were performed by atomic force microscopy. Results: The occurrence of actin filaments in 2D MSCs cultivated in HPL supplemented media was decreased as compared to conventional growth media, where the number of ventral stress fibers, anchored at each end by focal adhesions, was affected and influenced membrane elasticity. Dorsal stress fibers or transverse arcs were not affected. Mitochondrial network dynamics in HPL cultivated MSCs showed a perinuclear accumulation. 3D cultivation of MSCs was less sensitive to media supplementation. Functional activity of GARP was found non-stimulated MSCs while expression of Foxp3 and IDO relied on pro-inflammatory stimuli. Discussion: In conclusion we can say that mechano-biological capabilities of MSCs cultivated in 2D and 3D rely on stress fibers, mitochondrial distribution and certain bioactive molecules such as GARP, Foxp3 and IDO. HPL supplementation leads to downregulation of f-actin filament formation as well as conserved membrane elasticity.Objectives: Modern research in cryobiology requires a deeper understanding of the influence of different factors on the cryopreservation of cells, tissues, and organs. One of these factors is ice crystallization which has a tremendous impact on the surveillance and quality of live objects during freezing, long term storage, and thawing. Analysis of this process requires software which should be able to obtain quantitative parameters of crystals in a human-like manner with acceptable processing speed. The purpose of this work is to consider the possibility of using the multiscale image representation for the quantitative analysis of ice crystals. Methods: In our research, we used microscopic images of ice crystals during crystals formation and thawing. In previous studies for the segmentation of ice crystals on the image, we used different approaches such as active contour. At the same time, it should be noticed that the speed of active contour expansion is low and thus time-consuming to process large time sequence. Thus, we suggest the application of a Gaussian Pyramid. This multiscale representation allows analysis at a low scale and improves at a high scale. Results: We have analyzed multiple images using the proposed approach. The results in the first approximation show a 2-fold increase in speed when using our implementation of active contours. At the same time, the segmented areas of crystals correspond to the approach without the use of multi-scale image representation. Discussion: The results of this work show that multiscale image representation can be applied to improve the speed and applicability of modern software for automated image analysis. The next steps will include applicability evaluation of multiscale representation for different cases as well as the development of software realizing vector processor architecture.Objectives: One of the objectives of hemodialysis HD is preserving optimal acid-base homeostasis. Bicarbonate is an obligatory component of dialysis fluids in order to titrate the acidic products of protein metabolism that accumulate between treatments. Even though bicarbonate can easily transfer across HD membranes, its plasma concentration does not equilibrate with that in dialysis fluid by the end of the treatment, for reasons unclear. We used a mathematical model to test whether an intradialytic increase in total non-bicarbonate buffer concentration in plasma tNBBp could explain this lack of equilibrium. Methods: We modified the model of CO2 and bicarbonate whole body transport proposed in 1995 by Rees and Andreassen, adding bicarbonate transfer from dialysis fluid to plasma via a dialyzer connected to an arteriovenous fistula. The model described the transport of O2, CO2, and base excess between different compartments: arterial and venous circulations, interstitial fluid and lung capillaries. In each compartment, equations describing the biochemistry of blood were solved to estimate the components of bicarbonate and non-bicarbonate buffer systems. Results: The model qualitatively described the typical intradialytic pattern of bicarbonate concentration, pH, base excess, CO2 and O2 partial pressures observed in patients under routine thrice weekly HD. Model predictions were compared assuming a constant or increasing tNBBp during HD. When tNBBp was allowed to increase, the intradialytic increase in plasma bicarbonate concentration was blunted and the model predictions better fit published time-dependent intradialytic plasma bicarbonate concentrations. Discussion: It was suggested in literature that HD treatment may stimulate an increased concentration of non-bicarbonate buffers and release of hydrogen ions from these organic acids. Our preliminary modeling results are consistent with this hypothesis and show that the model is capable of predicting the effect of the HD treatment on the buffer systems of the body.Objectives: To develop an implant for the bone-tendon transition, it is our approach to optimize polycaprolactone PCL scaffolds by coating for a functionalization with drug release systems. The procedure during the functionalization should enable a release of growth factors in a spatially and temporally graded way. Methods: The PCL fiber mats were prepared by electrospinning. Polydopamine pDA-based coatings were used to modify the surface of the fibers. To get thin polymeric, hydrophilic and biocompatible coatings we applied the pure pDA system and a co-deposition with amines ethylenediamine ED, hexamethylenediamine HMD and polyethyleneimine PEI. To functionalize our system with growth factors we used chitosan-based nanoparticles which can be loaded with the therapeutic proteins BMP-2 and TGF-beta. By a defined dipping procedure we generate fiber mats with a graded functionalization. Results: The measurement of the contact angle and the surface zeta potential showed a change from a hydrophobic PCL surface to a highly hydrophilic and amphoteric character for the coated systems. The charged surface enables the further functionalization with loaded nanoparticles. The release of BMP2 and TGF-beta from the functionalized scaffolds was established. With a dipping procedure we are able to functionalize our scaffolds in a graded way. Confocal laser scanning microscopy with fluorescence-labeled NP showed that we could generate up to three differently functionalized regions. Discussion: The results show the methodology is suited to prepare implants, which can release growth factors in a spatially and temporally controlled way. These graded implants can be used for in-situ tissue engineering.Objectives: Solute movement from blood to dialysis solution is largely determined by transmembrane pressure TMP. Accordingly, it was imperative to develop a TMP control system that allows for the maintanence of constant pressure. TMP in dialyzer is one of the tasks in wearable artificial kidney development. In most of the known devices, transmembrane pressure is regulated only by adjusting the flow-pressure characteristics of the pump or mechanical uncontrolled clamping line. Methods: A system was proposed which consists of a pinch valve, a DC motor and a control circuit. This device allows you to automatically adjust the pressure. At first, the device reads data from the differential sensor, which measures the TMP. After that, the position of the moving part of the pinch valve changes and the degree of clamping of the line changes, which in turn affects the value of TMP. The case of the device was made using a 3D printer. A test bench was created in order to determine the effectiveness of maintaining a constant TMP. It consists of five parts: two circuits with two peristaltic pumps, a dialyzer, a differential pressure sensor and a power source. The pressure from the primary circuit was varied by manually clamping the line and the degree of pressure differential compensation by the pinch valve was determined. Results: The experiment showed that the TMP changes are fully compensated by the pinch valve. Accuracy of the developed device depends largely on accuracy of pressure measurement sensor. There was no change in the flow rate of the fluid, and this is important, because the rate of dialysate regeneration depends on the flow rate of the fluid. Discussion: The experiment showed that this device can be used to control TMP. Accuracy of the system allows you to use it in the wearable artificial kidney.Objectives: This paper describes the testing methods used in the development of software for an implantable heart monitor. It is a complex medical system. To identify errors made while writing software, as well as to detect deviations in the operation of the device in a timely manner, it is necessary to constantly carry out testing of all elements of the system. Methods: Testing methods used in the development of an implantable heart monitor: - debugging using JTAG, - self-test system, - testing via COM port, - recording test information in the devices internal memory data logging, - interprocessor testing. Consider some of them in more detail. Debugging and testing using JTAG: Due to the fact that the development was carried out in Code Composer environment, this allowed the use of a special script language GEL General Extension Language. This is an expression language used by the CCS debugger. Launch GEL files are used to automate software development. GEL is an interpreted language, and its syntax is the same as C. Thereby it is possible to carry out tests, and change them, or add new ones in the process of the main software. Logging: One of the testing methods is to track changes in the state of the system over time. One way is to write data to the devices internal memory. But in such a way that you can track at what point in time the change occurred. For this you can use time stamps. If the data is recorded strictly with a certain frequency, you can do without time stamps. Results: The presented test methods differ in that some of them can be used only in the early stages of development and some even when the device is at the end user. Discussion: Cardiovascular diseases are very dangerous and difficult to diagnose. Early diagnosis and prediction of these diseases can lead to an improvement in the quality of life of patients.Objectives: To clarify the reasons, beneficial effects and duration of AVF patency after radiological interventions in AVF stenosis. Methods: In 174 patients, 522 radiological investigations and endovascular treatments such as percutaneous trans luminal angioplasty PTA were analyzed, retrospectively. All investigations were performed due to clinical suspicion of impaired AVF function. Results: Arterial stenosis was significantly more frequent among patients with diabetes mellitus p < 0.001 and interstitial nephritis p < 0.001. According to the venous stenosis, the diagnosis did not affect the frequency p=0.22, neither the degree of the stenosis p=0.39. The degree of stenosis prior to PTA correlated significantly with the degree of remaining stenosis after intervention p<0.001. Of the 174 patients, 123 71 performed a total of 318 PTA investigations. Repeated PTA was performed significantly more often in patients with diabetic nephropathy. The median times to first PTA and to the subsequent PTAs were 9.5 months and 5 months, respectively. Discussion: Patients with diabetic nephropathy are at a greater risk of getting recurrent stenosis; this motivates a closer follow up for these patients. Clinically significant stenosis should be dilated as meticulously and as soon as possible. Occlusions of the AVF in most instances can be successfully thrombolyzed or dilated upon early diagnosis.Objectives: Designing new ventricular assist devices to coupe two major challenges in the field: less invasive intervention and reducing side effects was our objective. Methods: Computer-aided design and computational fluid dynamics were performed with Solidworks 2017 to design and test several new bearingless centrifugal ventricular assist devices. Fused deposition modeling was used to build the models and subcutaneous implantation was simulated and transcutaneous power transfer was tested. Results: Two bearingless, fully implantable ventricular assisted devices were developed. The first is a radial flux motor fully magnetically levitated ventricular assist device. The second is an axial flux hydro-magnetically suspended rotor ventricular assist device. Both devices contain an internal rechargeable battery, power transfer coil, electronic circuit and have a cylindrical shape with 15-20 mm height and 100 mm diameter. The flow is 3 l/min at 135 mmHg is achieved and efficiency of power transfer of 80 with a 5 mm gap. The devices are implantable in a pectoral subcutaneous pocket with the inflow connected to left atrium and the outflow to subclavian artery in a fashion pioneered in this field by CircuLite Synergy. Discussion: Both devices came with innovations in motor and pump design that set the foundation to build the entire system in a single body low height cylindrical shape, optimal for subcutaneous implantation and by this with good transcutaneous power transfer efficiency.Objectives: For most in vitro cultivations of mammalian cells, culture medium is supplemented with serum, which contains important proteins and factors for the cell growth. Previous experiments showed already a positive effect of human platelet lysate hPL as a medium supplement for the two-dimensional 2D cultivation of mesenchymal stem cells MSC. Since 3D culture systems gain big interest for in vitro applications and regenerative medicine, the influence of hPL on 3D cell culture models must also be investigated. Semi synthetic protein not sign based hydrogels represent a very promising 3D cell culture platform, which is biocompatible and biodegradable. Methods: In our work, gelatine-methacryloyl GelMA hydrogels with a various degree of functionalization were dissolved in different hPL concentrations 0 , 2.5 , 50 , 100  hPL. The Influence of hPL on proliferation and differentiation of in GelMa encapsulated MSCs were monitored and measured via viability assays, immunocytochemical stainings and microscopy. Furthermore, the influence of hPL addition on mechanical characteristics of GelMA hydrogels was examined by swelling studies and rheometric analysis. Results: The addition of hPL to GelMA hydrogels enhances the cell-spreading, the degree of differentiation as well as cell viability in concentration-depended manner. Besides, the results of the rheometric tests showed an increase of hydrogel stiffness by using higher concentrations of hPL. Discussion: Obtained results demonstrate that supplementation of hydrogels with hPL has a positive effect on cell growth and differentiation. Moreover, hPL can be a promising hydrogel stiffness enhancer for application in 3D bioprinting.Objectives: Generally, LVAD thrombosis is diagnosed only by reviewing online logfiles that show an increase of power consumption and flow of the device. In order to increase diagnostic power and improve treatment management, trans-thoracic echocardiography was carried out to assess any blood flow changes. Methods: Eight patients M 7; F 1 experienced a total of 17 Heart Ware VAD thromboses. At echocardiogram, a 4-chamber view was obtained and the echo probe beam was aligned to blood flow of the inflow cannula. Continous-wave Doppler was applied for the assessment of: 1 the increase of flow maximal velocity Vmax above basal ranges over several cardiac cycles and 2 the irregular discontinous sound of the device. Online logfiles were reviewed for abnormal increase in power consumption and flow compared to basal data. All patients underwent thrombolysis with rTPA. Echocardiography results before and after treatment were compared. Results: Thrombolysis was successful in all patients; one patient died due to cerebral haemorrhage. Echocardiograms before thrombolysis showed an increased mean Vmax of 6.4 /- 0.5 m/sec, whereas after treatment Vmax was 360 /- 0.2 m/sec p < 0.001. In all cases irregular, abnormal and discontinuous sounds were heard in thrombosed inflow cannulae. After thrombolysis only washing jet sounds were heard. Before thrombolysis logfiles showed a mean increase of power consumption of 5.6 /- 0.3 watts, returning to basal values of 3.8 /- 0.3 watts after thrombolysis p<0.001. Before thrombolysis logfiles showed mean flow of 6.2 /- 0.6 L/min, after treatment mean flow was back to normal ranges 4.2 /- 0.5 L/sec p<0.001. Discussion: Transthoracic echocardiography showed a significant increase of blood flows in the inflow cannula of thrombosed LVAD devices, confirming findings of log files review. A combined approach by logfiles review and echocardiography is recommended in patients with LVAD thrombosis.Objectives: Leukocyte activation is correlated to a reduced immune response in ventricular assist device VAD patients. Research has displayed leukocyte microparticle MP; 200-1000nm diameter increases following VAD shear stress. However, questions still arise regarding MP role, impact and biogenesis. MPs are plasma membrane derived, but there are smaller 50-100nm diameter, cytoplasmic vesicles exosomes which are often overlooked, or mistaken for MPs. As such, unless these vesicles are successfully separated, they should be categorised as extracellular vesicles EVs. EVs are known to have roles in cellular communications and implications in diseases, and it is desirable to assess leukocyte sub-sets and any associated roles in VAD complications. Methods: Bovine blood was diluted to a haematocrit of 30 /- 2 according to ASTM standards. Blood was pumped for 6 hours in vitro with the CentriMag R centrifugal pump Thoratec R Corp., USA. Flow cytometry determined a leukocyte EV population, having eliminated cellular debris. Platelet-poor plasma was tested for vesicle size and quantity using Nanoparticle Tracking Analysis NTA, and immunoblotting conducted for exosome markers. Results: Flow cytometry data displayed an increase in leukocyte death in the CentriMag, correlating to an increase in leukocyte derived EV production. Further analysis determined that these vesicles were mainly monocyte derived, suggesting a potential implication in inflammation. Preliminary NTA and immunoblotting displayed increased EVs following shear, namely exosomes. Discussion: The presence of exosome marker staining and NTA size analysis displays evidence in favour of vesicle increase not being MP specific. As EVs have known implications in inflammation, infection and disease, these results suggest that exosomes may be associated with adverse events in clinical settings. Thus, exosomes should be further examined to elucidate any role as biomarkers of VAD related complications.Objectives: To investigate the influence of bone turnover markers in end-stage renal disease patients for severe secondary hyperparathyroidism SHPT. Methods: 221 ESRD patients with SHPT were split into two groups according to the iPTH level: Mild SHPT group M-group iPTH130 -600 pg/ml, n=86 and severe SHPT group S-group iPTH > 1500pg/ml, n=53. Serum BTM such as beta- CrossLaps ss-CTX procollagen type 1 N-terminal propeptide P1NP, osteocalcin OC were detected. Results: Compared to the M-group, the patients in S-group had higher levels of the ss-CTX 5877.40 /- 423.01 pg/ml, vs 3206.90 /- 1438.27, P=0.000, the P1NP 1145.50 /- 171.36 ug /ml, vs 440.00 /- 308.12, P=0.000, the OC 258.52 /- 48.61 ng/ml, vs 221.74 /- 75.99 ng/ml, P=0.000. ROC Curve analysis for the main risk factors of the parathyroid hyperplasia were ss-CTX, P1NP, sALP, iPTH. Discussion: Severe SHPT had higher levels of bone turnover markers, which may have aggravated CKD-MBD.Objectives: One of the main functions of renal replacement therapy is removal of excess products of metabolism from blood. Uremic toxins, waste products found in the blood of patients in need of renal replacement therapy RRT, have very high blood concentrations and are harmful to the organism. Uremic toxins have been found over 100 and their classification is based on their molecular mass and linkage to proteins. EUTox, the European uremic toxin work group, has divided uremic toxins into 3 groups: water soluble toxins with small molecular mass MM < 500 Da e.g. urea; middle size molecules, MM > 500 Da, e.g. beta 2-microglobulin; protein-bound uremic toxins e.g. indoxyl sulphate and rho-cresyl sulfate. Methods: TalTech and a spin-off company grown out of the university, OFT, have developed a new, real-time multicomponent MCM-sensor to monitor all uremic toxin groups in RRT. MCM-sensor contains miniature LED-technology to measure UV-absorbance and fluorescence. During online measurements, the sensor is connected to dialysis machine outlet with easy-to-handle connectors. Results: The results from a pilot clinical study at Nephrology Department of North Estonia Medical Centre and Nephrology Department of Linkoping University Hospital will be presented. The outputs of the sensor are on-line concentration patterns of small, middle and protein-bound uremic toxins in the waste dialysis and the clinical parameters of the uremic toxin removal removal rate, total solute removal. A telemetry application enables to follow the dialytic removal even on remote distances. Discussion: A concept for smart real-time monitoring of intermittent RRT has been realized as a novel miniaturized optical sensor performing intradialytic on-line multicomponent uremic toxins monitoring in the spent dialysate.Objectives: Since Carbon Nanotubes CNT were shown to enhance neural outgrowth towards them, an approach with a CNT-Silicone rubber compound that immerses the CNTs and thus anchors them to the implant is investigated in an animal model Methods: To compare the CNT-silicone rubber to state-of-the-art neuronal implant materials, 6 variations of the same implant with varying exposed materials were fabricated. The thin-film implants consisted of a polyimide substrate and metal pathways encapsulated in silicone rubber. Depending on the batch, either metal or CNT-silicone rubber were exposed. Female mice were implanted on the brain and under the skin flank area for 60 or 180 days. After necropsy, the head was fixed in 4 neutral buffered formaldehyde. After 24h, it was stored in 70 ethanol for another 24h and then decalcified. It was then trimmed, dehydrated in ascending alcohol series and xylene. After embedding in Technovit 8100, sections with nominal thickness of 2 mu m were cut. The same procedure was performed on the implants and skin from the flank, except they were embedded in paraffin and cut into 3 mu m slices. Light microscopic examination of Hematoxylin and eosin stained slices was performed. Results: During histopathological examination, both findings associated with surgery and implant-related microscopic changes were observed in sections from the head and flank. The tissue around the implant showed similar reactions in all groups. Macrophages and inflammatory cells as well as slight fibrosis were found around the implant in all animals Discussion: Preparation of tissue sections including thin film implants was shown to be possible, yet the differences in material properties led to problems during cutting. However, usage of different embedding materials for the tissues worked. These first results suggest that the used CNT-Silicone rubber materials could be used in short term applications.Objectives: The properties of nanocomposite NC poly N-isopropylacrylamide NIPAM hydrogels as well as 3D printing techniques for hydrogels have gained a lot of significance in the tissue engineering community. This study aims at the development of a stable water based ink formulation for nanocomposite hydrogels usable by DLP/SLA 3D-printers in the range of 365 - 405 nm to use them e.g. for investigations in water flows. Methods: Diphenyl 2,4,6-trimethylbenzoylphosphine oxide as photo-initiator, Laponite XLG as nanocomposite crosslinker, sodium dodecyl sulfate as surfactant and polyvinylpyrrolidone as crystallization inhibitor were dissolved in a N-butyl acetat and isopropanol solution at room temperature. This organic phase was then mixed with bi-distilled water and stirred for 20 minutes in a closed container to form a micro emulsion. The micro emulsion was flash freezed in liquid nitrogen and freeze dried in a Virtis Advantage Plus freeze drying unit for at least 24 h at -50 degrees C and 0.4 mbar. The obtained powder could be dispersed in a NIPAM/water solution to form a stable, UV-curable ink. To show the applicability of hydrogels for investigating flows around bodies in principle, experiments in a water channel will be conducted. Results: The obtained ink is suitable for the most commonly used SLA and DLP printers in the range of 365 - 405 nm. Whereas the expected curing time is much higher than the curing time of commercially available standard polymeric printing inks due to the inhibiting effect of the Laponite XLG. The applicability experiments are in preparation. Discussion: NC-hydrogels obtain through this photo-polymerization method show similar mechanical properties to the standard NC-hydrogels obtained through radical polymerization with potassium peroxodisulfate.Objectives: Laser interference lithography LIL and the layer-by-layer LbL technique are combined here for the first time to design a system with variable nanotopographies and surface viscoelasticity to regulate cell behavior. Methods: LIL is used to generate hexagonally arranged nanostructures of gold with different periodicity. In contrast, LBL is used to assemble a multilayer system of poly-L-lysine PLL and hyaluronic acid HA on top of the nanostructures. Moreover, the viscoelastic properties of that system are controlled by chemical cross-linking. Results: We show that the topography designed with LIL is still present after multilayer deposition and that the formation of the multilayer system renders the surfaces hydrophilic, which is opposite to the hydrophobic nature of pristine nanostructures. The heterogenic system is applied to study the effect on adhesion and differentiation of human adipose-derived stem cells hADSC. We show that hADSC spreading is increasing with cross-linking degree on flat multilayers, while it is decreasing on nanostructures modified with multilayers. In addition, early effects on signal transduction processes are seen. Finally, hADSC differentiation into chondrogenic and osteogenic lineages is superior to adipogenic lineages on nanostructures modified with multilayers. Discussion: Hence, the presented system offers great potential to guide stem cell differentiation on surfaces of implants and tissue engineering scaffolds.Objectives: The Gibbs-Donnan theory describes the equilibrium of ionic solutions separated by a semipermeable membrane, when only one solution contains charged macromolecules that cannot pass the membrane. The problem was originally formulated for simple solutions, such as dissociated NaCl, which allowed to calculate the ratio between ions concentrations at the sides of the membrane Gibbs-Donnan factor. Solutions like plasma and interstitial fluid, and dialysate, contain however ions with different valence, and proteins in each compartment have a different concentration and therefore charge; the classic theory needs an extension to describe these real-world scenarios. In this study we propose a new method to calculate Gibbs-Donnan factors for such more complex cases. Methods: From electroneutrality considerations and the definition of Nernst potential it is possible to derive a new expression of Gibbs-Donnan factor which depends on the total concentrations of ions of the same valence. This allows, for two solutions A and B, to calculate the concentration of all ions in solution B, given: 1 macromolecule concentrations in A and B, 2 solutes concentrations in A. The transitivity of Donnan factors was proved: for 3 solutions at equilibrium A, B, C the factor of A to C is equal to the ratio of the factors from A to B and B to C. Results: As a demonstration, the theory was applied to the calculation of the concentrations in interstitial fluid Na, K, Ca2, Mg2, Cl-, HCO3-, HPO42-, SO42-, C3H5O3- starting from values reported in literature. The calculated interstitial concentrations were close to the reported ones, with a global RMSE equal to 6. Discussion: The Donnan factors for small ions in biological fluids have relatively small deviation from unity, but their precise value may be of importance for ions whose concentrations are similar at both sides of the membrane, as for example sodium and calcium transport in hemodialyzers and peritoneal dialysis.Objectives: Biobanking of in vitro engineered tissues presents obvious advantages associated with flexible preparation and shipping scheduling. To provide sufficient cryoprotection, alternative research models, freezing approaches and cryoprotective agents CPAs have to be established in view of complexity and diversity of composite tissues. Here, we evaluated the effects of conventional freezing CF and directional freezing DF for cryopreservation of cell-seeded electrospun fiber mats. Methods: Fiber mats were produced from polycaprolactone and polylactic acid PCL-PLA, ratio 200:100 using electrospinning. Structure of electrospun mats was characterized using SEM and FTIR, whereas their heat capacity was determined using DSC. The square-shape mats 1x1 cm, UV-sterilized were seeded with HeLa cells 5x104/scaffold and cultivated at static conditions. Samples were frozen using 1 K/min to -80 degrees C CF or with the directional ice growth at the speed of 30 mu m/sec over a 2.6 mm gradient gap from -4 to -10 degrees C i.e. 4 K/min followed by further cooling to -20 degrees C at 1 K/min translational cryostage and to -80 degrees C with 1 K/min LN flow cooling stage. The viability of cells was evaluated 24 h after thawing using fluorescein diacetate and propidium iodide staining. Results: PCL-PLA mats exhibited no cytotoxicity on HeLa cells in vitro viability higher than 90. Only single cells survived both CF and DF when frozen in culture medium negative control. In contrast, the majority of cells on fiber mats were alive after freezing under protection of DMSO. Discussion: We showed that although not yet optimized, DF was as effective as CF for cryopreservation of cell-seeded PCL-PLA fiber mats. Preliminary studies on using antifreeze protein Type III AFPIII 10 and 100 mu M as a sole CPA indicates that extracellular protection might not be enough to prevent freezing injury. Our recent work on electroporation of cells within electrospun fiber mats yielded encouraging results and the further initiative will be to deliver AFPs into cells using electroporation to provide DMSO-free cryopreservation of tissue-engineered constructs.Objectives: Computational tools for accurate evaluation of healthcare products are increasingly being recognized by the European Parliament, US Congress and FDA and represent a highly valuable tool for speeding up and reducing investments associated with the demanding certification process of medical devices. InSilicoTrials Technologies is developing a collaborative, automatized cloud-based platform to assist medical device and pharma companies in performing modelling and simulation activities to accelerate medical innovation while responding to recommendations of European and US parliaments and regulatory bodies. Methods: Simulations provided by scientific partners are integrated within a secured and privacy-preserved environment, ultimately providing the users companies, research and clinical centers with in silico solutions for different steps of the RD process on medical devices and drugs. Simulation results as well as an automatically created simulation report can be formatted in conformity with guidelines provided by EMA and FDA while taking in consideration the new European Medical Device Regulation. Results: Tools currently hosted on the platform are: NuMRIs, an automatic tool to assess magnetic resonance imaging radio-frequency safety of medical devices, implemented in collaboration with the US FDA Center for Devices and Radiological Health and ANSYS. QT/TdP Risk Screen, an automated tool to assess drug-induced pro-arrhythmic and Torsade de Pointes risk of small molecules, developed in collaboration with Universitat Politecnica de Valencia and Fundacio Institut Mar dInvestigations Mediques. Discussion: The use of computational technologies to support med registration is limited in scope or absent while the agencies are promoting the use of MS in RD and registration. This causes a gap in know-how between reality of the industry and regulatory expectations. InSilicoTrials bridges these gaps and helps industries to conform to regulatory recommendations in a timely and cost-effective manner.Objectives: Dialyzer design enhancing internal filtration may be beneficial to remove middle molecules from the patients blood for the treatment of renal pathologies. Mathematical models have been proposed to predict the rate of internal filtration, at varying dialyzer geometry and operating conditions. However, such models generally consist of difficult differential equations, that may not be easily used and implemented in clinical practice. In this work, an analytical model to predict internal filtration, at varying dialyzer geometry and operating conditions, is presented. Methods: To identify an analytical formula allowing for the prediction of internal filtration, the Poiseuille equation was used to describe axial momentum transport in the blood and dialysate compartment, assuming constant fluid viscosities, whereas radial momentum transport across the membrane was described with the Darcy equation, assuming linear ultrafiltration flux. The model was validated by comparing the pressure profiles in blood and dialysate compartments, and the rate of internal filtration predicted by the model with experimental data reported in literature for in-house Theranova 400 dialyzer. Results: An analytical formula to predict the rate of internal filtration at varying dialyzer geometry and operating conditions was obtained. Very good agreement i.e. percentage error lower than 2 was found between model predictions and experimental data for blood flow rate increasing from 300 to 400 ml/min, at constant dialyzer flow rate and zero net ultrafiltration flow rate. Good agreement was also found between internal filtration predicted by the analytical model presented in this work, and that predicted by using other one-dimensional mathematical models reported in literature. Discussion: The analytical model presented in this work agrees well with experimental data, and can be used predict the rate of internal filtration at varying dialyzer geometry and operating conditions.Objectives: Despite the growing acceptance of LVAD therapy, concerns relating to adverse events have been raised. Fields such as implantation techniques, patient management and patient selection pose opportunities to advance outcomes, however, new and enhanced device technology, such as the ReinVAD LVAD, will have to contribute to improve and grow the therapy. Methods: The ReinVAD GmbH was founded as spin-off company of the Helmholtz-Institute of the RWTH Aachen University, Germany, and leads the development project to regulatory approval and clinical application. The core technology is a miniaturized, 3rd generation, centrifugal blood pump. It is the result of a multidisciplinary research and development program equipping the system with features that will improve outcome for bridge to transplantation and especially destination therapy patients. Results: Chronic trials typically up to 90 days in a sheep confirm the capabilities of the system with excellent resistance to thrombogenicity since no design related thrombus could be found despite ceased anti-coagulation after 6 weeks. Implemented features are a pump for partial and full support with flow up to 10 l/min, a fully integrated flow sensor, an implantable driveline connector, a highly pressure-sensitive pump characteristic, an automated speed controller and a user-friendly one-piece extracorporeal system. The sensor provides real flow measurements. The driveline connector ensures easy exchange in case of infections and renders the system ready-for-TET. The pump pressure sensitivity, among other effects, leads to increased exercise capacity and arterial pulsation. The controller ensures that the pump always operates at its optimal speed. The one-piece extracorporeal system reduces use errors and improves satisfaction with this component of everyday life. Discussion: The results indicate that the ReinVAD LVAD will be an effective system to enhance long-term circulatory support therapy.Objectives: Cardiovascular diseases shorten life span of hemodialysis HD patients. Recently pentraxin 3 PTX was considered as a biomarker of cardiac function, as well as an inflammatory biomarker. The aim of this study was to investigate if PTX was a useful biomarker to estimate patients cardiac condition more than as inflammatory biomarker by analysing plasma in PTX, CRP, NT-pro-BNP Pro-BNP and troponin T before and after HD. Methods: Twenty patients on chronic HD were studied in a prospective cross-over study with three modes of HD with polysulfone dialyzers: a dry-stored F8HPS, Fresenius with a low blood level in the venous chamber DL, b dry-stored with a blood level kept high DH, and c a wet-stored dialyzer Rexeed18L, Asahi Kasei Medical with high level WH. PTX, CRP, Pro-BNP and troponin T were assessed before and during 180 min of HD. Results: The mean value of PTX at baseline was correlated to Pro-BNP rho=0.52, p<0.001 and troponin T rho=0.363, p= 0.005, but there was no correlation to CRP. The mean value of PTX was significantly increased at 180 min by 57 compared to baseline data p<0.001. The baseline values of PTX correlated with the extent of rise of PTX rho=0.89, p<0.001, such as a higher baseline was related with more extensive increase. The mean difference value of PTX between baseline to 180 min of HD was correlated to that of Pro-BNPrho=0.31, p=0.025, while there was no correlation to that of troponin T nor CRP. PTX rose less while on WH than on DL mean 2.9 /- 1.4 vs 3.8 /- 2.8, p=0.025. There was no difference in PTX at baseline among these three settings. Discussion: These data indicate that PTX is a more evident marker for the influence of the cardiac condition than an inflammatory marker during HD. Dialysis with WH causes least air contamination to the patient what may be a plausible reason for the difference.Objectives: Amniotic membrane AM is applied in variety of clinical settings, mainly in wound healing and as a biological dressing. Currently, AM is used in ophthalmology for the treatment of cornea pathologies. In this study, the regenerative potential of human amniotic membrane after cryopreservation was evaluated. Methods: AM was prepared under sterile conditions and frozen without cryoprotectants at -80 degrees C. Native non-cryopreserved AM served as a control. The strategy for AM cryopreservation included one or two freezing and thawing cycles. Cytokine and growth factor profiles as well as morphology and mechanical properties of AM were compared before and after one and two cycles of cryopreservation. Levels of EGF, HGF, TGF-beta 1, bFGF, Laminin and Hyaluronic acid were measured by ELISA. Surface structure of the membrane was analyzed by scanning electron microscopy SEM. Morphology was evaluated by histological analyses after Hematoxylin Eosin HE staining. Additionally, biomechanical characteristics, such as the modulus of elasticity Youngs modulus and the tensile strength, were also investigated. Results: EGF, HGF, TGF-beta 1, bFGF, Laminin and Hyaluronic acid were detected in all studied samples. No significant differences were detected in the levels of those cytokines before and after cryopreservation. SEM analysis showed minor structural alterations of double cryopreserved samples. Analysis of HE stained AM showed minor changes between native and frozen membranes, such as few areas with desquamated epithelium which was caused by repeating freezing and thawing cycles. Mechanical tests did not reveal relevant changes between frozen and non-frozen samples. Discussion: Multiple cryopreservation steps are an essential part of manufacturing of human amniotic membranes to ensure a ready-to-use product availability. This study demonstrated that the cryopreservation does not impair the regenerative potential of AM required to support its therapeutic efficacy.Objectives: Cells are used in bioartificial organs BAOs to temporarily replace the complex metabolic functions that neither failing organs, nor their artificial substitutes may provide. This makes cell bioreactors a critical technology for BAOs development. Bioreactors are used for expanding stem SCs, precursor PCs, or line cells to clinical mass, for promoting SCs and PCs differentiation to the needed phenotype, and for providing patients with the organ regulatory and synthetic functions needed to foster organ healing or maintain patients state until transplantation Tx. In some cases, they are also used to regenerate or preserve tissue until Tx. The structural and functional complexity of tissues and organs, the lack of information on their pathophysiology and on disease/damage evolution, the treatment time broadly varying with patients disease and state, all make bioreactor design and operation for BAOs very challenging. Methods: In spite of its complexity, cell bioreactors design for BAOs is generally based on the use of one cell type only often line cells, scaffolds with suitable immune protective properties, and bioreactor configurations suitable for hosting clinical cell mass and intellectual property protection. Bioreactor performance has been enhanced with actuators to provide cells with mechanical, electrical and/or magnetic cues, often irrespective of tissue or organ specificity. Thus far, this approach has resulted in poor therapeutic success of the proposed BAOs. Results: Cell bioreactor design for BAOs may benefit of the approaches developed over the years to bioreactor design for engineering or regenerating 3D tissue or large solid organs in vitro. Discussion: Prospectively, future approaches to cell bioreactor design for BAOs should be patterned after those typical of tissue engineering and should aim to the optimization of the BAO treatment as a whole.Objectives: To find the most appropriate lock solution for central venous catheters to prevent catheter-related complications. Methods: Medline, Embase and Cochrane Central Register of Controlled Trials were systematically searched up to August 2018. We included randomized controlled trials comparing different lock solutions for adult dialysis patients > 18 years old. The trials should report at least one primary or secondary outcome. Network meta-analyses of outcomes were performed using the netmeta 0.9-6 package in R version 3.5.1. Other analyses were carried out by means of the network and network graphs packages in Stata version 15.0. Results: 50 trials 7142 patients were included for this study. Compared with heparin 5000U/ml, antibiotic locks antibiotics with TSC, EDTA, heparin 5000U/ml, low-dose heparin, or urokinase and ethanol locks were more effective in preventing catheter-related bloodstream infections. Antimicrobial agents plus low-dose heparin 500-2500 U/ml, TSC and low-dose heparin locks had lower risk of bleeding events than heparin 5000 U/ml. No lock solution reduced rates of catheter malfunction and all-cause mortality compared with heparin 5000 U/ml. In summary, antibiotics plus low-dose heparin was ranked as the best lock solution. Considering the toxicity and drug resistance of antibiotics, ethanol could be the alternative antimicrobial lock solution. The overall results were not materially changed in sensitivity analyses. Discussion: In total, according to the two-dimensional graph, antimicrobial agents plus low-dose heparin 500-2500 U/ml was the highest ranking lock solution, as it reduced the rates of CRBSI and the risk of bleeding significantly. Ethanol was the promising alternative antimicrobial lock solution of antibiotics to combine with anticoagulant agents. TSC alone did not prevent CRBSI, but it is safe.Objectives: The Fontan procedure is the current standard and the best palliative treatment for most patients with a functional univentricular heart. During the Fontan procedure the caval veins are anastomosed to the pulmonary arteries. The resulting total cavopulmonary connection TCPC allows for the support of an acceptable level of pulmonary circulation. At the same time there is the high risk of cardiovascular system failure due to the increased load on a single ventricle. In order to increase patient survival and quality of life, the different TCPC connections aimed at improving blood hemodynamic seems to be a logical treatment option. Methods: A CFD study was carried out for four types of TCPC TCPC-1, TCPC-2, TCPC-3, TCPC-4 to determine the optimal connection from the point of view of hydraulic and power losses with and without the rotary blood pump RBP connected. The distribution of scalar shear stresses SSS in the volume was calculated, allowing for evaluation of the effect of the RBP on the blood at the operating point of 2.2 l/min and 11 mmHg of head pressure. Results: The results showed that the TCPC-3 connection has the optimal geometry, as there is a decrease in hydraulic losses corresponding to 5 mmHg, with power losses of 0.055 W. Hydraulic losses and power losses for other types of TCPC are equal to 6 mmHg and 0.07 W for TCPC-2, 9 mmHg and 0.08 W for TCPC-4, 10 mmHg and 0.09 W for TCPC-1 respectively. The geometry optimization of the TCPC connection can significantly reduce the system losses, but only the integration of the RBP prevents these losses even for non-optimal TCPC. Discussion: Although the use of a pump has significantly reduced losses, the presence of increased SSS can have a negative effect on blood. The obtained values of SSS will allow in the future the estimation of the degree of blood damage and, as a consequence, hemolysis level.Objectives: The complex tree-like architecture of the lungs that ends with tiny alveolar sacs is difficult to mimic in-vitro. The delicate and ultra-thin alveolar barrier with its air-liquid interface is constantly exposed to the rhythmic respiratory movements. We report here about two organs-on-chip models that uniquely reproduce this environment and in which patients cells are cultured. A first model recapitulates an array of alveoli with in-vivo dimensions using a biological stretchable membrane. The second model mimics a functional lung capillary network. Methods: The key part of the lung alveolar model is a biological membrane made of collagen and elastin CE that is pipetted on a gold mesh, whose pores correspond to the alveolar size about 200um. Once the dried CE-membrane is rehydrated, primary human cells alveolar epithelial and endothelial can be cultured on both sides. The second model is based on the self-assembly of endothelial cells and pericytes that are confined in fibrin gel in microengineered compartments. Results: A functional alveolar barrier made of primary human alveolar epithelial cells and lung endothelial cells is reported. The CE-membrane, on which the cells are cultured, is thin a few micrometers, porous enables the culture of cells at the air-liquid interface during several days, and stretchable. The reported lung microvasculature made of self-assembled endothelial cells and pericytes is perfusable, vasoactive contracts in presence of phenylephrine, a vasoconstrictor and is permeable. Discussion: These advanced in-vitro models enable mimicking the lung parenchymal environment in an unprecedented way. As a result, the cultured tissues made of primary human lung cells are able to maintain organ-specific functions, such as air-blood barrier tightness and microvascular perfusability and contractility. Organs-on-chip solutions, such as those presented here, may open new possibilities for specific tissue engineering applications.Objectives: Development of simple and effective technologies for cell storage at positive temperatures facilitates to extend their use and ensures safe transport between research and clinical centers. The aim is to study the viability and metabolic activity of human mesenchymal stromal cells MSCs encapsulated in alginate microspheres AMS during storage at positive temperatures. Methods: The experiments were performed on human dermal MSCs derived from adult donors after their informed consent. MSCs in suspension and in AMS were stored in sealed cryovials at 4, 22 and 37 degrees C. Cell viability and metabolic activity were evaluated by MTT- and AlamarBlue-tests and the ability to adhere to the culture plastic. Mitochondrial membrane potential was assessed with fluorescent dye JC-1. Differentiation potential was evaluated after cells induction to osteogenic and adipogenic lineages. Results: Encapsulation in AMS supported viability and metabolic activity of MSCs during storage in culture medium at ambient temperature at list for 3 days. Storage of MSCs under hypothermic conditions 4 degrees C using special media contributed to a longer sustainability of cell viability. During storage alginate encapsulated cells retained the ability to multilineage differentiation, but had lower metabolic activity and mitochondrial membrane potential than cells in monolayer. On release from alginate microspheres, the cells were shown to attach, reverse metabolic activity and proliferate in a similar manner to that seen before encapsulation. Discussion: Encapsulation in alginate microspheres may be considered as a cheap and robust alternative to cryopreservation for the shirt-term storage and transport of cells for clinical and research projects.Objectives: In case of organ failure, several options can theoretically be considered, thanks to the progresses in medicine, materials science and biotechnology: i transplant the same organ from a donor, ii replace the functions of the failing organ by an artificial one, iii repair its structure thanks to cell therapy or iv engineer a new one in vitro before its implantation. Methods: As far as abdominal organs kidney, liver, pancreas, the history of organ replacement follows different ways that will be considered. In addition to medical and scientific hurdles, economical or ethical issues can drive the choices for the "best" solution for the patient. Results: The case of pancreas is very specific since its transplantation is not the first option chosen to treat the failing organs, whereas it represents the gold standard for the liver and the kidney. In the case of liver, we still have to face the lack of purely artificial organ to treat acute liver failure or even chronic disease. Discussion: This is due to the complexity of the organs and the variety of its functions, some of them remaining relatively unknown. In this case, the option of engineering new liver in vitro combining hepatic cells of different phenotypes with an adequate matrix allowing high blood perfusion appears as the most promising option. Several approaches have been proposed, starting from the "basic" use of a semi-permeable hollow fiber membrane to host the cells to the bioprinting of the whole organ. We will discuss here the pros and cons of these different strategies.SensorStim Neurotechnology GmbH is a start-up of the TU Berlin. The business area is medical engineering based on neurostimulation and eHealth for rehabilitation and homecare. Our vision is to help people with limited mobility due to paralysis, spasticity or pain and to mobilize them with technical assistance. Our novel medical device enables the use of noninvasive neurostimulation for rehabilitation and therapy of people with disabilities after damage to the central nervous system - brain, spinal cord, pyramidal tract. It is mobile, easy to use and it can be used in water, too. Our device consists of a neurostimulator combined with body sensors and can be operated via a smart App. Furthermore, it can be used for clinical and home care. After a successful market entry, we further want to support walking, cycling and aqua gymnastics with electrical stimulation. We started in 2018 and are currently funded by grants of the Federal Ministry of Education and Research BMBF, Investitionsbank Berlin IBB and private equity and development cooperations. By the end of 2019, we plan to achieve CE marking for our first product and ISO 13485 certification for our quality management system.Objectives: Intraventricular flow patterns during mechanical circulatory support MCS cannot be accessed by clinical imaging; therefore, either computational or in-vitro models are used. However, the complex anatomy of the heart cannot be replicated and simulations inherently rely on assumptions and simplifications. In an isolated porcine heart setup the feasibility of flow measurements by Echocardiographic Particle Image Velocimetry E-PIV was evaluated. Methods: Similar to cardiac transplantation, porcine hearts n=8, animal weight: 80-106 kg were excised and connected to the isolated heart setup. After resuscitation using blood as perfusate, a rotary blood pump was implanted, microbubbles were injected via the left atrium at different support situations and echocardiographic 3-chamber-view B-mode images were recorded with the highest possible frame rate of up to 141 Hz Philips iE33, X5-1 xMatrix probe. By iterative PIV algorithms using correlation domain averaging and beam sweep correction, flow fields were evaluated for the different hemodynamic situations. Results: All hearts were successfully resuscitated in the isolated heart setup and different hemodynamic situations were adjusted. In the unsupported heart physiologic flow patterns with a large clockwise vortex structure that warrants washout of the whole cardiac chamber were found. With increasing MCS 2200-2700 rpm the formation of this flow feature is diminished caused by the additional flow sink at the apex. In full support, without aortic valve opening in the left ventricular outflow tract, a stagnant structure was identified, that might be connected to thromboembolic events. Discussion: For the first time, the contribution of the mitral valve apparatus to blood flow patterns especially in the LVOT, which may be linked to energy loss, thrombus formation and valve deterioration during MCS was investigated under realistic conditions.Objectives: In order to determine the most effective ratio of liver cells LC and multipotent mesenchymal stromal cells of bone marrow MSC BM into implantable cell-engineering constructs CECs used for a correcting of chronic liver failure CLF. Methods: In order to create liver CECs we employed a biopolymer implant - a composition of a heterogeneous collagen-containing gel, which concluded viable LC and MSC BM in ratios:1:1, 5:1 and 10:1 respectively. CECs with different ratios of LC and MSC BM were implanted into the livers of rats n=40 in which CLF was modeled by using CCl4. The efficiency of the regulatory effects of CECs with different cell ratios on regenerative processes in livers were assessed by using biochemical, morphological and morphometric methods at different periods after their implantation. Results: During studying of the liver CECs with various ratios of LC and MSC BM 1:1, 5:1, 10:1, it was found that the most optimal ratio of cells into the CECs was 5:1, because at such ratio of cells the most distinct normalization of morphological and functional liver parameters as well as maintenance of the structural homeostasis within the CECs took place within 365 days after modeling CLF. Discussion: The effective correction of CLF can be carried out by using the implanted liver CECs, in which donor liver cells and MSC BM are presented in ratios - 1:1, 5:1 and 10:1. But analysis of prolonged correction of liver morphological and functional parameters at CECs using allows to recommend the preference using of CECs using with ratio 5:1, because prolonged preservation of structural homeostasis into themselves CECs makes possible to prognosticate their prolonged regulatory action on the liver tissue at CLF, especially for recipients on a waiting list for liver transplantation.Conventional hemolysis researches were performed under mainly constant shear stress, however, real flow in vivo would be under fluctuating shear stress. In addition, several conventional numerical simulations predicted abrupt shear stress fluctuation in the blood pumps in short time period. Furthermore, it was also reported that erythrocyte showed its unique viscoelastic shape change response to sinusoidal changing shear stress. Based on such information, we hypothesized that abrupt shear change would be additional hemolysis factor in addition to conventional shear stress and its exposure time. Therefore, the purpose of this study was to examine the feasibility of such our hypothesis. We developed special cylindrical shear chamber which can generate varying shear stress by controlling the rotational speed. This study used the shear device to generate several trapezoidal waveforms of shear stresses. In such shear fluctuation, the maximal shear stress of approximate 22.5 Pa were loaded to porcine whole blood for 5.0 seconds. And the time series integrated shear stress was set to equal between abrupt and slow shear change conditions. Then, generated hemolysis levels were compared between them. Each comparative studies were performed for twelve times. Furthermore, such studies were repeated under several different shear change speeds. The result of our all comparison showed greater hemolysis amount at abruptly fluctuating shear condition. Such results successfully validated our hypothesis.Objectives: Ex vivo platelet production in microfluidic bioreactors is a promising alternative to platelet donation for transfusion therapy. Platelets are formed by fragmentation of large bone marrow cells called megakaryocytes MK. Hydrodynamic forces play a major role in the formation of platelets. In this work, we design microfluidic chips where isolated MK are exposed to hydrodynamic forces and we characterize their elongation and fragmentation. Methods: Polydimethylsiloxane chips are fabricated using standard soft lithography protocols and sealed to a glass slide. The 50 mu m deep rectangular chambers contain rows of 30 mu m-wide adhesive pillars. Human CD34 cells isolated from umbilical cord blood are differentiated in vitro for 12 days to yield mature MK and infused with a concentration of 2x105 /mL into the chips at a wall shear rate of 1800s-1. Their elongation and fragmentation is monitored by videomicroscopy. Images were analysed using ImageJ. Results: We perform a spatio-temporal analysis of MK elongation and show that platelet release is always preceded by a remodeling of the cell that spans over about 20 minutes, followed by a local, sudden increase in elongation velocity 5-fold increase in the 10 seconds prior to fragmentation. The amplitude of these variations is much larger than the spatial and temporal variations in the surrounding flow field. Discussion: Earlier studies have shown that dynamic conditions for in vitro platelet production not only accelerates the process but also enhances the quality of released platelets. Here, we show that MK elongation and fragmentation, though accelerated by the flow, are not solely governed by fluid-structure interactions. It seems that other mechanisms are involved in the regulation of platelet release, preventing it to occur before the platelet is fully functional.Objectives: The Eulerian version of the strain-based hemolysis model has already shown to predict different critical regions in blood pumps compared to the commonly used stress-based models. Where the stress-based model assumes an instantaneous deformation of red blood cells RBCs due to the action of fluid forces, the strain-based model is able to estimate the viscoelastic deformation of RBCs. Thus, the effect of exposure time is considered in a natural way. However, both models are based on the empirical power law assumption for hemoglobin transport and turned out to overpredict hemolysis compared to experimental data. In order to improve the prediction of free-plasma hemoglobin, the strain-based model is combined with a multiscale model for hemoglobin transport. The model is able to account for pore formation in the RBC membrane and the resulting hemoglobin mass transfer into the blood plasma. Thus, the model does not rely on the power law assumption. Methods: The formation of pores is computed based on an energy balance of the stretched RBC membrane, where an area strain can be computed by the tensor equation of the strain-based model. The hemoglobin release is modeled by a transport equation with a mass transfer coefficient as a function of shear, which can be fitted to the data of blood shearing experiments. As an application, the model is used for a benchmark blood pump provided by the FDA. Results: The estimated total hemolytic performance for the benchmark blood pump is significantly reduced compared to the power law version of the strain-based model. The new results are in good agreement with the measured data by the FDA. Discussion: The multiscale model for hemoglobin transport turns out to be a useful extension of the strain-based hemolysis model, allowing for an overall more physical model description of the hemolysis process. Application to different devices and blood shearing experiments should be investigated in the future, in order to study the parameter sensitivity.Objectives: Extracellular vesicles EVs are membrane enclosed structures, which are released from all types of cells upon activation. Two examples for the role of EVs in extracorporeal therapies will be discussed in detail, i EVs as markers for cellular activation and ii EVs as platform for the binding of plasma proteins. We assessed the impact of regional citrate anticoagulation 2.8 vs. 5.6 mM on the cellular activation in lipoprotein apheresis, using EVs as marker for cellular activation upon blood-adsorbent interaction. Moreover, we studied the ability of a commercial adsorbent PentraSorb for C-reactive protein CRP to remove both, soluble and EV-associated CRP from septic plasma. Methods: EVs were characterized in human blood or plasma via flow cytometry after calibration with fluorescent silica beads 1 mu m, 0.5 mu m, 0.3 mu m; Kisker Biotech. Phosphatidylserine-exposing EVs were identified as lactadherin-positive events in the EV gate. Antibodies against cell specific surface markers CD41 for platelets, CD235a for red blood cells were used to identify the cellular origin of EVs. An anti-CRP antibody was used to identify CRP carrying EVs. Results: During lipoprotein apheresis, platelet adhesion to the adsorbent polymer and EV release were associated with elevated markers for platelet activation and EV release was dependent on the citrate concentration in the extracorporeal circuit. Septic plasma contained significantly elevated levels of EVs, CRP EVs, as well as soluble CRP as compared to healthy donors. Incubation of plasma with the CRP adsorbent resulted in almost complete depletion of soluble CRP, and remaining EVs did not show any association with CRP, indicating detachment of CRP from the EVs. Discussion: In the context of apheresis, EVs can be considered as markers for cellular activation. Moreover, our data show that EVs can serve as a platform for the binding of CRP, which has been suggested to induce conformational changes of CRP and a shift towards pro-inflammatory characteristics.Objectives: In developing endovascular treatment devices, understanding their physical properties is one of the most important points. We have been developed film-type sensor for force measurement, applied to the field of biomechanics. In this study, the blood vessel model embedded pressure film sensor was fabricated and the contact pressure distribution in the circumferential direction was measured at balloon dilatation. Methods: The film sensor was constructed by a couple of patterned electrodes and pressure-sensitive layer. The film sensor was rolled and embedded to straight-shape silicone vessel model. The balloon catheter was expanded in a mock vessel. Results: The measured pressure at the mock vessel increased according to balloon dilatation pressure. At expanding of general balloon, the measured pressure at each point was almost the same. When expanding a scoring balloon, a balloon with wedge elements, the measured pressure at wedge contact points was higher than that of the other parts. Discussion: It is considerable that stress concentration state changes depending on the wedge shape and/or stiffness. The mock vessel is useful to understand dilatation property of scoring balloon.Objectives: Extra Corporeal Membrane Oxygenation ECMO is used to treat acute respiratory and circulatory failure. Recent advancements have led to impressive improvements in treatment results. Now clinicians urge for an ever-earlier ECMO application and a broadening of the indications: Mobility demand -> Every ECMO patient needs to be transported under severely restricted space conditions. Cost demand -> With rising ECMO numbers the cost ratio plays an increasing role for healthcare economics. Methods: A novel system uses a recombination of existing technologies to create a straightforward solution with a size and complexity reduction. The key principle is the sourcing of pump energy from the oxygen pressure, which is required to ventilate the oxygenator anyway. This modification allows a purely pneumatic operation and the usage of pulsatile pumps. Results: A pneumatically powered dual chamber pulsatile pump has been developed which avoids suction and high shear rates with improved wash out behaviour for the entire ECC system. The innovative pump is driven completely pneumatically without a complex and expensive driving console. Furthermore, a novel oxygenator has been developed. Initial prototypes of pumps and oxygenators have been optimized during multiple design iterations and tested in terms hydraulic characteristics and gas exchange properties on the bench using water glycerol and porcine blood. The entire system has a reduced surface area and priming volume and shows low internal pressure loss, excellent wash out and gas exchange parameters. The bench test results have been confirmed within a chronic GLP in vivo study in large animals. The new devices size and complexity have been significantly reduced compared to the state of the art systems. Discussion: The novel approach brings many advantages for the next generation ECMO/ECLS systems particularly in terms of function, mobility and cost-effectiveness.Objectives: The American Society for Testing and Materials ASTM standards recommend bovine blood for in vitro testing of medical devices. Haemodilution is used to standardise haematocrit HCT to 30 /- 2 with phosphate buffered saline PBS. Research has shown that PBS increases red blood cell mechanical fragility and significantly increases haemolysis. Thus, the objective of this study was to investigate the impact of diluent and dilution for in vitro testing, which may result in future considerations for ASTM standards. Methods: Bovine blood was diluted with either PBS or PBS4g BSA bovine serum albumin to a HCT of 30 /- 2. The blood was either pumped with the CentriMag R Thoratec,USA device under haemodynamic conditions or exposed to VAD-like shear for 6 hours. Blood was diluted to a range of 70-94 n=40. Plasma free haemoglobin levels were measured to calculate the normalised index of haemolysis NIH. Protein concentrations were measured. Results: Haemodilution altered mechanical fragility depending on diluent and concentration. At a 70 blood dilution, PBS alone caused significantly higher haemolysis than PBS4g BSA. However, at a 90 blood dilution, PBS4g BSA caused significantly higher haemolysis than PBS alone. As such, a positive correlation was observed between dilution with PBS alone and NIH, whereas a negative correlation was observed between dilution with PBS4g BSA and NIH. Total protein concentration was significantly reduced at a 70 blood dilution with PBS alone but not at 90. Discussion: To reach a HCT of 30/- 2, this study recommends that bovine blood with a HCT of > 37 should be diluted with PBS  4g BSA >80 blood dilution, whereas blood with a HCT of <37 should be diluted with PBS alone <80 blood dilution. Haemodilution with BSA maintains physiological protein concentrations and may reduce bias during haemolysis testing with high dilutions. Thus, ASTM standards could consider including BSA as a diluent, when needed.Objectives: Obtaining normal serum potassium level is an important goal in maintenance hemodialysis patients. Hyperkalemia is known to be associated with mortality. In this study we aimed to access the relationship between pre-dialysis potassium level, nutritional status and survival in dialysis patients. Methods: This study used annual cohort of hemodialysis patients with 36 months of follow up. To determine the impact of potassium level on mortality, patients were followed from first potassium measurement until death or a censoring event; hypokalemia was defined by potassium levels below 5.5 mmol/l and albumin levels below 35g/l was considered an index for undernourished. Time-dependent Cox proportional hazard modeling was used to estimate the association between potassium level and mortality. Results: 199 patients were included in the study. Mean age was approximately 56 years, about 59 were men and 23 had end-stage renal disease caused by diabetes. Albumin below 35g/l was observed in 26 13 of the patients. In the follow up period 40 20 patients died, comprising 24 32 of the 74 hypokalemic and 16 19 of 82 hyperkalemic patients. The Kaplan-Meier survival rate was significantly longer in the hyperkalemic population 34.30 /- 0.71 vs 31.06 /- 1.16, p=0.055. Hypokalemia, when defined as serum potassium 5.5 pound mmol/l, was associated with all-cause mortality hazards ratio HR 1.857, 95 CI 0.986-3.496, p = 0.051. The significance was lost in the model after adjustment for albumin level. Only albumin level determined mortality p=0.03. Discussion: Lower potassium level was associated with all-cause mortality, but only as a confounding effect of malnutrition in dialysis patientsObjectives: A new method to adhere biological tissue to another or to metal such as stainless steel using integrated low-level energy sources, heat, pressure, and vibration is proposed. Methods: Heat, pressure, and vibration are simultaneously delivered by pressing the porcine aorta tissues on the vibration plate with the heated tip. Tensile tests on adhered specimens were performed to determine adhesion characteristics to adhesion temperature, time, pressure, and vibration. The effect of metal surface coating, fluorine-doped DLC coating, on the adhesion strength was examined. In one application, the inlet port of a ventricular assist device adhered to a porcine ventricle muscle by applying the novel method with a temperature of 80 degrees Centigrade for 120 seconds. Sufficient adhesion was observed without additional vibration. In a second application, the vessel adhesion apparatus was developed to connect a bypass graft to a coronary artery. The device has both a heating and pressurizing part on the tip, which was used to attach the bypass graft to the coronary artery by applying heat with a temperature of 120 degrees Centigrade for 120 seconds. Results and Discussion: The maximal shear tensile strength using the proposed novel adhesion method was 200 kPa, which is half the strength of the porcine aorta. Adhesion strength increased in proportion to temperature, time, and pressure. The adhesion strength can be controlled by surface treatment of metals. In the first application, the inlet port adhered to the ventricle muscle with a shear tensile strength of 91 kPa, which is enough to keep the pump attached to the ventricle. In the second application, the bypass graft adhered to the coronary artery with a shear tensile strength of 50 kPa, which is sufficient to maintain attachment when exposed to normal blood pressure. The proposed adhesion method indicates satisfactory performance.Objectives: The requirements for safety and efficacy of medical devices are constantly increasing leading to higher costs and time to market. Simulations are proving to be particularly promising in accelerating the approval process and at the same time further increasing patient safety. The FDA related Medical Device Innovation Consortium predicts that in some years about 40 of the evidences for regulatory approval are coming from virtual patients and simulations. What is the status today and how do we get there Methods: Simulations can improve the regulatory assessment in various phases of the product life cycle, e.g. to justify design adjustments, worst-case estimates of sizes and variants, definition of test loads for experiments, to perform virtual tests on a large number of patients/conditions or to identify the root cause of failure. This may especially be important in consideration of the future requirements of the MDR for patient-specific implants. Results: So far, the use of simulations was hindered by a lack of guidance and expectations within the medical device community. However, today the FDA and ASME are strongly supporting the use of this approach with new guidelines guidance 1807 and a VV standard VV40. "FDAs Office of Science and Engineering Laboratories has committed significant resources for transforming computational modeling from a valuable scientific tool to a valuable regulatory tool because of its potential for significant cost-savings in evaluating medical devices, simulating performance under scenarios that may not be possible with human use or that could more effectively be evaluated with simulation." Discussion: Computer-aided simulations have the potential to revolutionize the field of regulatory approval. However, it is crucial that the simulations are performed with care and its limitations are understood. To foster confidence and wider acceptance of in silico methods, a proper methodology is needed to ensure appropriate credibility.Objectives: TAVI is the treatment of severe aortic valve stenosis in elderly patients or in patients with too high risk for conventional surgery. CT-scan is mandatory for valve sizing. It requires injection of contrast medium that may cause or increase kidney failure. The study aim was to establish the accuracy of intravascular ultrasound IVUS, a no contrast medium technique, for measurements of native aortic valve in patients undergoing TAVI. Values by IVUS were compared to CT-scan ones in the same patients. Methods: From June 2014 to January 2019 137 consecutive patients 74 M, mean age 82.7 /- 5.6 years Logistic EuroSCORE 22.7 /- 14.9; STS score mortality 21.3 /- 13.7 undergoing TAVI through femoral access were enrolled. Each patient had high resolution angio-CT for these measurements: diameter, perimeter, and area of the aortic annulus. In all patients, during procedure and before prosthetic valve implantation, a manual IVUS pullback, from left ventricular outflow tract to ascending aorta, was performed by 7F IVUS probe. On the recorded IVUS pullback, a second operator blind to CT measures sized minimum and maximum diameter, perimeter and area of the aortic annulus. In order to assess inter-rater concordance and agreement between CT and IVUS data, Bland-Altman analysis and Pearsons correlation were applied. A p value Results: A very strong correlation was found between the measures. Maximum diameter: CT 2.5 /- 0.3 vs IVUS 2.6 /- 0.3 cm; r=0.985; pDiscussion: These results prove the accuracy and reliability of aortic annulus measurements by IVUS compared to those by CT-scan. Sizing of the aortic annulus by IVUS only may be considered in patients likely to develop kidney failure due to contrast medium injection.Objectives: The long-term usage of rotary blood pumps is limited by the high amount of blood damage, which leads to a high number of adverse events. Most of the blood damage is generated in the gap between the rotor blades and the housing wall, the area with the highest stress load and very short exposure times. The damage is caused by both shear and normal forces on the blood components. The objective is to investigate the blood damage in this region by examining different gap geometries. The focus is the damage caused by high dynamic normal stresses and the comparison with the blood damage models. Methods: A section of the blade gap of an axial rotary blood pump was reconstructed on a microchannel chip, which is sealed with siliconized cover glasses. In additional channels the inlet and outlet rounding are varied between 0.05 and 0.6 mm. The gap length and width are kept the same for all channels. A channel without a blade gap is integrated to determine the damage caused by the test stand. Flow simulations are carried out to determine the occurring stresses. The test stand consists of a syringe pump that transports 20 mL of human blood back and forth through the microchannels for up to 6 hours. Hemolysis is measured every half hour. Results: With the test stand it is possible to investigate the influence of different gap geometries on the blood damage under controlled conditions. The experiments demonstrated the influence of different inlet and outlet geometries on the hemolysis. The measured hemolysis was compared with the damage models. Discussion: With the test stand it is possible to simulate the stress loads in blade gaps on human blood. The influence of the high dynamic normal stresses on hemolysis can also be determined through variations of the inlet and outlet. It is therefore a well-suited tool for improving the corresponding damage models in the area of high dynamic stress loads. The findings allow the optimization of the region to further reduce blood damage caused by rotary blood pumps.Objectives: Cardiac grafts produced by combining human pluripotent stem cell hPSC-derivatives and tissue engineering are a promising therapeutic option for the replacement of cardiomyocytes CM lost due to myocardial infarction. Currently, the vascularization and maturation of such grafts are the most critical aspects. In addition to endothelial cells EC, pericytes PC are known to play a crucial role in vessel development and stabilization. Methods: Here, an hPSC-derived PC differentiation protocol is shown as well as an in vitro model for the investigation of the role of PCs in hPSC-derived bioartificial cardiac tissues BCT. Results: In the established differentiation protocol, the population giving rise to PCs CD31-/PDGFR beta represented up to 94 of all cells and could be further purified via cell sorting. The obtained hPSC-PCs exhibited functional characteristics of PCs in co-culture assays, and showed similar gene expression profile to primary PCs. Differentiated hPSC-PCs were used w/ or w/o purified hPSC-ECs to generate BCTs and to address their effect on tissue morphology, metabolism, and electrophysiological parameters compared to control tissues containing primary fibroblasts Fb. Interestingly, tissues with hPSC-PCs exhibited equivalent contraction forces compared to controls, but more organized sarcomere structures as well as improved longitudinal cell- and extracellular matrix orientation. Furthermore, the beating frequency and passive forces showed native-like values. Addition of hPSC-ECs resulted in spontaneously formed and maintained endothelial network structures, which were distributed throughout the BCTs and the involvement of PCs in such structures was demonstrated. Discussion: The characterized and functional hPSC-PCs together with hPSC-derived CMs and ECs represent a promising cell source for myocardial tissue replacement therapy. Their dual ability in supporting EC function and matrix remodeling during tissue maturation makes them a favorable candidate to replace primary Fbs.Objectives: High prevalence of thrombi around the Left Ventricular Assist Device LVAD cannula has been proven by pathological studies. Thrombogenic potential TP inside of the left ventricle LV is evaluated based on shear stress history SSH and residence time RT of particles; however, the effect of the atrial inflow - including rotation and uneven flow distribution of the pulmonary veins - is ignored. In this study, the influence of the atrial inflow on the TP was investigated via Computational Fluid Dynamics CFD simulations. Methods: In a patient-specific left ventricle LV model under full LVAD support, the flow fields were simulated with three different inflow conditions: with perpendicular velocity to the inflow LAper, flow rate: 3.5l/min, with an additional rotational component at the inflow LArot: 35rpm and with asymmetric inflow conditions LAasym: 60/40 left/right. Platelet motion was simulated with a combination of laminar and the Lagrangian methods for 7s. The TP value was calculated based on the RT and SSH of particles 0<TP<1, with higher values corresponding to higher risk. Results: The ventricular flow patterns were comparable for perpendicular and rotational inflows and different for asymmetric conditions. TP values were different for each of the atrial inflow conditions LAasym: 0.37, LArot: 0.35, LAper: 0.4. The number of the particles with a SSH value more than 0.6 Pa center dot s non-outlier range increased for LAasym simulation LAasym: 11, LArot: 5, LAper: 3;p<0.05; however the particles that remain for more than 7s inside the LV were higher for LAper simulation LAasym: 13, LArot: 18, LAper: 19;p<0.05. Discussion: CFD is an advanced tool for the evaluation of ventricular flow during LVAD support; however, neglecting the atrial inflow conditions could lead to an inaccurate prediction of flow parameters linked to ventricular and pump thrombosis. Reliable evaluation of ventricular blood flow, therefore, requires the consideration of realistic atrial inflow conditions.Objectives: Thrombus formation is a major risk of implants with artificial materials in blood contact. This adhesion of blood platelets plays an important role in thrombus formation and is influenced by the electrochemical conditions of the surface. The objective of this research is to change the electrochemical conditions by applying a low voltage, high frequency alternating electrical current to the surface, in order to avoid the adhesion of the platelets involved in thrombus formation. Methods: Thin sheet samples with a thickness of 0.5 mm from different materials titanium, stainless steel and copper are attached to a high frequency generator and exposed to human whole blood under controlled flow conditions in an in vitro setting. An electrical current ranging from 50 mu A until 400 mu A in a frequency range from 10 kHz to 100 kHz was applied. The deposition of the blood platelets on the surface is identified and quantified by using fluorescent microscopy and digital image processing. The thrombus formation and therefore the anti-thrombogenic effect of the method is investigated in respect to different generator settings for each material. Results: The characterization of all materials revealed changes of the surfaces in comparison to the zero samples. A differently pronounced reduction of thrombus formation with the proposed anti-thrombogenic method could be demonstrated on all three materials. Discussion: Though the method could improve the anti-thrombogenicity under certain conditions, the mechanism is not fully understood yet and the parameters are not optimized. Further investigations will also focus on possible side effects caused by potential structural changes of blood proteins.Objectives: Cationic liposomes OO4 N-6-amino-1-N-9Z -octadec9-enylamino -1-oxohexan-2S -2-yl -N0- 2- N, N-bis2-aminoethyl amino ethyl -2-hexadecylpropandiamide and DOPE contain a higher amount of amino groups than classical lipids. A polyelectrolyte multilayer system PEM made of chondroitin sulphate CS and collagen type I COL I was prepared by Layer-by-Layer technique with liposomes embedded in the terminal layers for the controlled release of components to promote osteogenic differentiation for bone regeneration. Methods: Characterization of the PEM was performed using ellipsometry. This technique was used to investigate the thickness on substrate coated with PEM of CS, liposomes, COL I as terminal layer. The layer growth behavior of PEM was studied using Surface Plasmon Resonance SPR. The cell adhesion of C2C12 cells and the cellular uptake efficiency of the liposomes embedded onto PEM was evaluated using fluorescence staining and flow cytometry. Results: The SPR results showed a linear growth with an increase in the angle shift corresponding to the adsorbed mass of PEM. The increase in thickness after liposome adsorption was significant. The cell studies indicated that cells seeded on terminal liposome layer showed a higher amount of cells on the surface. The population of cells that took up liposomes are higher in the absence of serum than in medium with FBS. Discussion: The quantification of liposome uptake indicated that the population of cells that took up liposomes were higher in the absence of serum. The results showed the uptake efficiency of cationic liposomes embedded onto a PEM system for controlled release of hydrophobic and hydrophilic models compounds into the cells.Objectives: Many different tissue sealants are available on the market. BioGlue tissue sealants are produced by a bovine serum albumin treated intra-operatively with glutaraldehyde. Here we report a sterile peri-graft abscess formation, possibly caused as a reaction to the liberal use of BioGlue. Methods: A-54 years-old male patient underwent ascending aortic and hemiarch replacement for acute type A aortic dissection. Exceptionally, the anastomotic suture line was reinforced by application of BioGlue tissue sealant. The initial post-operative course was uneventful. After transfer to the general ward on day four the patient developed fever and an inflammatory syndrome. A broad spectrum antibiotherapy was started for a suspicion of pneumonia. In the following chest CT scan a suspicious peri-graft fluid collection was present. The patient underwent a surgical revision for acute mediastinitis rule out. During the procedure a whitish-milky liquid was found that differed macroscopically from a classical abscess. The chest was left open with a negative-pressure wound dressing. The patient became apyretic and the inflammatory markers normalized. After a one week follow-up with negative cultures and 3 consecutive dressing changes, the chest was closed and recovery uneventful. The patient was discharged to an outside hospital at six post-operative days. Results: All the samples which were analysed histologically and for different pathologic stains as well as and by broad spectrum culture analysis, remained negative throughout. At one month follow-up the patient remained well and without fever or inflammatory response. Discussion: Although several published case reports about tissue necrosis and anastomotic rupture following BioGlue use, the formation of a sterile abscess is a rare complication. In this patient, the peri-graft effusion might have been an allergy like inflammatory response to the bovine albumin within BioGlue. A prolonged postoperative mediastinal drainage when using BioGlue might prevent this rare complication.Objectives: The use of loop diuretics like furosemide is discussed in intensive care medicine regarding their clinical efficacy and safety. The hypothesis arose whether both parameters depend on the capability of serum albumin to transfer the drug to the kidneys. Methods: Serial blood  urine samples were taken from intensive care patients receiving furosemide infusions for volume overload. Free furosemide serum fractions  urine concentrations were determined by HPLC/LCMS. Albumin binding capacity ABiC was quantified by adding a binding site II diazepam binding site specific fluorescence marker Dansylsarcosine, DS. The amount of unbound DS was detected by fluorescence analysis after ultrafiltration, corresponding with ABiC. Results: 62 patients were assessed. 50 patients 30male, 67,5 /- 10,3 years were included into statistical analysis. Main reasons for intensive care admission were sepsis n=18 and perioperative care. Indications for furosemide were volume overload in acute or acute on chronic kidney injury n=42, volume management in sepsis, ARDS or lung edema n=8. ABiC was reduced to 65,2/-12,2, The free furosemide fraction normal <5 correlated negatively with ABiC r -0,638, p<0,001 with 15,76/-11,52 in those with ABIC<60 and 5,36/-5,29 in ABiC 60 p<0,001. The urinary fraction of furosemide normal >65 was significantly lower in those with ABiC <60 as compared to 60 7,47/-5,83 vs. 23,98/-17,91, p<0,01. Furosemide induced increase in urine output was strongly correlated with ABiC r=0,908, p<0,017. Discussion: Our results indicate a role of ABiC in the pharmacokinetics of furosemide, impacting its free serum fraction and urinary fraction. Data are suggestive for a link between status of the albumin-dependend drug carrier mechanism and the safety and efficacy of the drug. Clinicians will profit from our results regarding safe use of loop diuretics  furosemide responsiveness in patients with volume overload and kidney injury.Objectives: Albumin is a blood protein with various physiological functions as there are: maintenance of osmotic pressure, antioxidant and transport function for endogenous and exogenous ligands. Binding of these ligands can affect conformational changes of human serum albumin. Methods: The "Albumin-functionality-test" is based on EPR spectroscopy and detects modified binding and functional characteristics. The used radical supporting spin probe binds analogous ligands variably strong at different albumin binding sites and thus, serves as an indirect marker for serum albumin functionality with regard to its ability for binding, transport and detoxification. A comparison of three different mixtures of albumin and a spin-labeled fatty acid allows for the in vitro simulation of binding-, transport- and release-conditions by the Albumin-functionality-test. Thus, an "effective" albumin concentration could be determined. It quantifies the amount of functional albumin in the patient in comparison to healthy controls. Results: We applied Albumin-functionality-test to albumin preparations differing in redox state of cysteine-34, namely human mercaptalbumin HMA, human non-mercaptalbumin1 HNA1 and human non-mercaptalbumin2 HNA2 and found differences in their functionality. The analysis of samples from patients with different states of liver failure showed a strong correlation of binding efficiency and detoxification efficiency, depending on the severity of the disease. Discussion: Due to the increasing numbers of studies using commercial albumin preparations for the treatment of patients with liver disease, we investigated different available albumin samples regarding their albumin functionality. We found dramatically reduced binding and detoxification efficiencies. Our findings might be used for patient selection and prognoses for personalized therapies. Taken together, these results might affect the importance of albumin from the patient view and refer to its stability as a drug.Objectives: The analysis of Left Ventricular Assist Device LVAD power is a commonly used method to identify complications such as pump thrombosis. However, the LVAD power is also affected by different influences like the circadian rhythm. To optimize our thrombosis detection algorithm, we analyzed LVAD log files. Methods: For the investigation of power behavior in LVAD log files, we retrospectively analyzed 840 log files from 189 patients. Each patient received an HVAD HeartWare. The log files contain on average 3000 power values collected over a period of appx. 31 days. None of the analyzed log files contained serious complications, such as a pump thrombosis. Results: To show a significant difference between the power values of the daytime 8 am-10 pm and the night-time, a statistical test significance level: 5 was carried out. 813 of 840 log files showed a significant difference average t = 16.94. In addition, the average difference between the maximum and the minimum power value of a day was calculated for each day average: 1071.9mW, SD: 158.2mW, range: 497.1mW - 2299.5mW. Furthermore, the average power was calculated for each day and log file. To get a measure for the fluctuation in daily average of power development within a log file, the SD has been calculated SD average: 65.3mW, range 10.4mW - 435.9mW. Discussion: The retrospective analysis of LVAD log files shows that power behavior is individual. Although most patients show a significant power difference between daytime and night-time hours, the maximum difference of a day and the average power changes over several days varies from patient to patient. A patient-specific power model, considering the circadian rhythm, was developed and applied to our thrombosis detection algorithms, which achieved a better thrombosis detection time of up to 93 hours compared to the LVAD controller alarming algorithm.Objectives: Suppression of membrane fouling is an important key technology for the development of portable and implantable artificial kidneys. Among several factors known to affect membrane fouling, we focused on the blood flow condition. In the present study, to clarify the effect of pulsatile and non-pulsatile blood flow on membrane fouling, an ex vivo hemofiltration experiment using porcine blood was carried out. Methods: Hemofiltration with filtrate recycling was performed under three conditions; pulsatile flow using a roller pump, non-pulsatile flow using a roller pump with three chambers to stabilize the flow, and non-pulsatile flow using a centrifugal pump. The blood flow rate QB was set at 50 or 100 mL/min and the maximum filtration rate at which irreversible membrane fouling would not occur was determined as follows; the first filtration rate QF was set at 2.5 of the QB for one hour baseline, increased by 2.5 to 5 of the QB for one hour, and then returned to the baseline value for 20 min, and the transmembrane pressure TMP was measured to check if it had returned to the baseline value. This step was repeated until the QF reached 25 of the QB. Results: For both blood flow rates 50 and 100 mL/min, the maximum filtration flow rate at which the TMP did not return to the baseline, implying that irreversible membrane fouling had occurred, was higher under the non-pulsatile flow conditions than under the pulsatile flow condition. Discussion: The flow rate and the TMP changed periodically under the pulsatile flow condition and the peak TMP value was higher than that under the non-pulsatile flow condition, which increases the burden on the membrane. Thus, there is an increased likelihood of membrane fouling, as compared to the constant the pressure under the non-pulsatile flow condition.Objectives: The objective is to develop a simulator of the anatomy and hemodynamics of the radial artery in order to design a robust controller for a noninvasive, continuous blood pressure measurement method. Currently, blood pressure, in intensive care units, is measured invasively, which can lead to complications e.g. infections. The novel approach facilitates a continuous noninvasive flow measurement, in addition to the instantaneous blood pressure measurement. Methods: During measurements, a controller pneumatically adjusts the pressure in a balloon, positioned on top of the radial artery, so that the Doppler derived flow in the radial artery is reduced to a constant target value. The resulting pressure in the balloon correlates with the blood pressure. For the development of this method, a simulator, modelling the anatomical conditions of the lower arm, was constructed. The simulator was designed to create arbitrary physiological pressure curves. For the validation of the simulator, invasively measured blood pressure curves and data found in literature related to the mass flow rates were compared to the curves created by the simulator. Results: The pressure curve, created with the simulator, shows a deviation of less than 5  to the invasively measured blood pressure. The mass flow rate shows a good agreement with the literature data. The simulator is capable of simulating blood pressure curves with a mean arterial pressure of 40 mmHg to 120 mmHg. First tests with the controller have shown that it is possible to control the pulsatile flow to the constant target flow rate. Discussion: Regarding the simulator, the next step is to adjust it, so that it can simulate flows above a mean pressure of 120 mmHg. Regarding the controller, stress tests must be conducted. It must be evaluated whether it is robust enough to work with pressure curves that fluctuate in mean pressure and frequency.Objectives: Relative blood volume RBV changes during haemodialysis are typically estimated from variations in haematocrit, haemoglobin or total blood protein measured by optical, acoustic or other sensors integrated in the dialysis circuit. Due to dynamic changes in the circulation during the initial phase of dialysis following filling of the extracorporeal circuit with the patients blood and starting dialyzer ultrafiltration, the indications of RBV monitors depend on the exact moment of starting RBV measurements. The aim of this study was to assess this issue quantitatively using a mathematical modelling approach. Methods: The developed compartmental model describes the flow of blood across the cardiovascular system and the extracorporeal circuit combined with the whole-body transport of water and the most important osmotically active solutes and the baroreflex mechanisms. The model was validated using clinical data from patients on maintenance HD. A standard 4-hour HD session with 3 L ultrafiltration was studied for a virtual reference patient with all model parameters based on the literature data. RBV changes were derived from the simulated haematocrit HCT changes. Two cases were analysed: the priming saline being infused to the patient or discarded. Results: For the case when the priming saline was infused to the patient, the total RBV reduction at the end of the simulated dialysis session varied between 8  when HCT changes were monitored from the very beginning of dialysis and 13  when HCT monitoring was started circa 4 minutes later. For the case when the priming saline was discarded, the analogous difference in RBV reduction was less than 1 percentage point. Discussion: When the priming saline is infused to the patient, a few minute difference in the moment of starting RBV measurements may significantly affect the indicated RBV changes, which is due to the dilution of blood by the infused saline and the time it takes for the uniform mixing of blood across the circulatory system.We have developed autologous in vivo tissue engineered vascular grafts in vivo TEVG, i.e. BIOTUBE mainly composed of fibroblasts and collagen constructed in the patients subcutaneous tissues. After long term animal experiments over 5 years, we clinically applied them to the congenital heart surgery of a 2-year-old child as a pulmonary artery patch plasty material in 2015. Three years after implantation, 3D-CT exhibited neither aneurysmal dilatation, significant calcification, nor shrinkage of the patch graft interfering the growth of the pulmonary artery. The clinical course was good without any significant complications over 4 years presently. However, since the tissue formation largely depends on the regenerative ability of the host, individual differences exist, and it is an issue to secure the mechanical reliability of the grafts formed in high-risk patients. We are currently evaluating the changes in mechanical properties of TEVGs by chemical treatment, and are examining whether it is possible to improve mechanical durability. In addition, in order to fabricate the TEVGs in healthy individuals as allogeneic parents or xenogeneic tissue substitutes, we are conducting transplantation experiments on decellularized tissues. The in vivo TEVGs could also be applied to AV-shunts. However, in the animal pilot experiments, significant stenosis was observed at the anastomosis with host vein resulting in early occlusion probably because of high sheer stresses. The attempts to overcome the problems by several investigators are also reviewed.Objectives: To evaluate different designs or insertion techniques of cochlear implant electrode carriers ECs insertion forces are measured by a force sensor, which is mounted directly underneath an artificial cochlea model aCM leading to a summed force profile. One of the next steps in CI research leads from post-experimental evaluation of measured insertion force profiles to pre-experimental predictions of these profiles using analytical models based on an improved knowledge about factors impacting the insertion forces. Three likely factors were chosen for further investigation: speed, EC stiffness and curvature of the aCM. Methods: Three aCM were fabricated out of PTFE blocks, each model having one constant curvature r = 6.4 / 8.5 / 12.7 mm. Additionally EC substitutes were fabricated using a two-component silicone, all with a constant diameter 0.7 mm, a total length of 20.5 mm and embedded bare copper wires. In order to vary the stiffness of the EC substitutes, one type had four and the other six wires embedded. They were inserted into the aCM with three different insertion speeds  v = 0.11 / 0.4 / 1.6 mm/s. In order to increase reproducibility, insertions were conducted using an automated insertion test bench, comprising a linear actuator to clamp the EC substitutes and move them into the aCM and the force sensor underneath the model. Results: In accordance with theoretical considerations all varied factors showed effects on the insertion force profiles. Increased insertion speed and sample stiffness increased the insertion forces, whereas an increased model radius decreased the insertion forces. Discussion: The variance of measured forces within one set of parameters is due to the manual fabrication of samples and may be decreased by usage of commercial ECs. The influence of lubricants has to be analysed within future experiments.Objectives: The purpose of this work is to design and create a high efficiency inductive powering unit IPU for a wide range of implantable devices and artificial organs AO. Methods: A possible way to realize wireless powering is to use inductive energy transfer systems. To maximize overall system efficiency class E power amplifier is widely used due to its 100 efficiency in the ideal case. But maximum efficiency can be achieved only if input and output system parameters will remain constant during IPU operation. Inductive coils have nonzero distance between them because of patients tissue thickness. It is worth noting that tissue thickness is different for every patient: the difference may be several millimeters. Moreover, this value can change during IPU operation as a result of patients motor activity or tissue edema which will lead to PA detuning and decreasing efficiency. To avoid PAs detuning, frequency tuning systems can be used. There are two approaches for tuning systems that can be found in the literature. The first uses voltage and/or current levels of important circuit nodes to estimate system status. This method is good and quite simple, but requires external control signals that means additional power losses and decreased efficiency. The other method uses the phase of the voltage signals from important nodes and consumes less power. The second method was chosen by authors to design a tuning system. Results: During this work, a calculation method for PA and the frequency tuning system were designed. The experimental prototype was created. The authors of this article obtained an overall system efficiency of more than 60  for three different coil sets and output power from 0.7 W to 1 W for same coil sets and range of axial misalignments from 10 to 20 mm. Discussion: The high efficiency inductive powering system for AO has been designed and tested during this work.Objectives: Native arteriovenous fistula AVF is the preferred vascular access for hemodialysis HD, but it still fails in more than 40 cases within the first year. Stenosis is the main cause of AVF failure, but its detection and prediction are still open clinical challenges. The purpose of our study was to explore the potential of using sound analysis to reveal unique characteristics in AVF sounds and to detect potentially relevant changes over time. Methods: We acquired the sounds of 10 AVFs, 7 referred by the nephrologists as well-functioning and 3 as stenotic AVFs. We also acquired the sound of 1 AVF characterized by severe stenosis and we repeated the recording after surgical revision. Sounds were recorded using the Littmann Electronic Stethoscope 3200 using wide mode. Sounds were transmitted via Bluetooth to a laptop and frequency spectra were obtained using an in-house Matlab code embedding the Fast Fourier Transform and a high-pass filter with a cutoff frequency of 50 Hz, aimed at attenuating stethoscopes noise and heart sounds frequencies. Results: In well-functioning AVFs we consistently found a low-frequency peak, located in the bandwidth 100-200 Hz, while no relevant spectral features were present at higher frequencies. On the other hand, stenotic AVFs showed high-frequency peaks, located in the bandwidth of 500-600 Hz. In the AVF with severe stenosis we found high-frequency peaks in the bandwidth 700-750 Hz, which were replaced by low-frequency peaks after AVF revision. Discussion: Sound analysis revealed unique characteristics in the frequency spectra of AVFs, allowing an objective discrimination between well-functioning and stenotic AVFs. This technique may bring the advantage of limiting clinicians skill-dependency and personal interpretation. Despite being preliminary, our results suggest that sound analysis may be used to detect stenosis development and allow patients self-monitoring, resulting in early prediction of AVF failure.Objectives: Biofilms on percutaneous leads of implanted blood pumps can cause severe infections. By treating infections with antibiotics over a long period of time, bacteria develop resistance. Today, in a clinical setting, antibiotic resistance is determined in-vitro for all available drugs. The aim of the EvolChip project is to predict the probabilities of resistances that might evolve and suggest combinations of antibiotics that slow down or even prevent future resistance. Methods: The EvolChip reactor consists of an agar plate that serves as a diffusion and growth medium. Three membranes permit diffusion of active substances from microfluidic channels into the agar medium. Two of these channels serve as sources for different antibiotics, the third one as the sink. As a result, two overlapping drug gradients establish, with a low concentration near the sink and high concentration close to the sources. Antibiotics were mixed with a nutrient solution to promote bacteria migration towards higher concentrated areas. The bacteria contain a green fluorescent protein gene, which enables real-time tracking via polarisation filters and a camera. Results: In preliminary experiments, the gradient course could be characterized by making use of fluorescent markers. Only resistant bacteria migrated towards the sources with high antibiotic concentration. The spatial distribution of bacteria facilitates the identification of optimal concentration ratios and mixtures of antibiotics. In addition, the speed of resistance evolution with respect to different antibiotic mixtures was determined. Discussion: The proposed EvolChip concept showed promising results in preliminary experiments. Further geometries, growth media, and bacteria will be tested. In order to validate the method, patient strains with known treatment and resistance history will be compared to the predictions of the presented method.Objectives: Due to the growing demand for organs in liver transplantation, techniques for optimization of suboptimal livers are needed such as normotermic liver perfusion PNT. In this way, hepatic repercussion of ischaemic preconditioning PI still remains unclear. This study aims to analyze the haemodynamic and biochemical alterations produced after intraoperative PI on livers subjected to 6 hours of PNT. As a secondary objective, we aim to determine the effect of PI on liver function. Methods: Eight minipig livers, randomized to control n=4 and PI group n=4, were perfused during six hours in a PNT circuit 34oC. The PI group was subjected to 10 minutes of ischaemia by means of a Pringle manoeuvrein the hepatic hilum, followed by 10 minutes of reperfusion. Biochemical and gasometric parameters were measured at baseline and hourly in perfusate. Pressures and flows in hepatic artery HA and portal vein PV were registered. For statistical analysis Spearmans rho test and Mann-Whitney U test was performed, considering p < 0,05 as significant. Results: There were significant differences in HA flow, adjusted to weight, doubling the control group PI 0,42 /- 0,17 ml/min/g Vs GC 0,17 /- 0,05 ml/min/g; p=0,000; no differences were found in PV flow. Total proteins PI 1,9 1,7-2g/dL Vs GC 1,3 0,95-1,5g/dL; p=0,000 and O-2 consumption rate PI 0,086 0,055-0,113 ml/min/g Vs GC 0,040 0,017-0,060ml/min/g; p=0,000were significantly increased in the PI group. There were no significant differences in AST, ALT, or GGT. Discussion: The PI, as a previous maneuver to PNT, increases hepatic artery flow and O2 consumption without altering citolytic parameters in liver. This may improve hepatic functionality as a previous step in transplantation. Further molecular and histological study should be made to clarify our finding in PI on PNT.</div>','<b>Title: </b>Takeover times for a simple model of network infection<br><b>Year: </b>2017<br><b>Author: </b> B. Ottino-Loffler,  Jg. Scott and  Sh. Strogatz<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1103/PhysRevE.96.012313"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PHYSICAL REVIEW E<br><b>Cluster-Keywords: </b><em>complex networks, complex systems, network theory, network structure, biological network, social network, network science, dynamic networks, network systems, network model, network topology, systems biology, network, network analysis, system dynamics, networks, realworld networks, network automata, epidemic network, model network<div style="font-size:11px;"><br><b>Abstract: </b><em>We study a stochastic model of infection spreading on a network. At each time step a node is chosen at random, along with one of its neighbors. If the node is infected and the neighbor is susceptible, the neighbor becomes infected. How many time steps T does it take to completely infect a network of N nodes, starting from a single infected node An analogy to the classic "coupon collector" problem of probability theory reveals that the takeover time T is dominated by extremal behavior, either when there are only a few infected nodes near the start of the process or a few susceptible nodes near the end. We show that for N >> 1, the takeover time T is distributed as a Gumbel distribution for the star graph, as the convolution of two Gumbel distributions for a complete graph and an Erd" os-Renyi random graph, as a normal for a one-dimensional ring and a two-dimensional lattice, and as a family of intermediate skewed distributions for d-dimensional lattices with d >= 3 these distributions approach the convolution of two Gumbel distributions as d approaches infinity. Connections to evolutionary dynamics, cancer, incubation periods of infectious diseases, first-passage percolation, and other spreading phenomena in biology and physics are discussed.</div>','<b>Title: </b>Quantum Cybernetics and Complex Quantum Systems Science: A Quantum Connectionist Exploration<br><b>Year: </b>2015<br><b>Author: </b> Cp. GoncalvesLink: No link<br><b>Journal: </b>NEUROQUANTOLOGY<br><b>Cluster-Keywords: </b><em>network models, gene network, regulatory networks, boolean networks, gene expression, systems biology, neural networks, network dynamics, gene regulatory, biological networks, network structure, gap gene, gene regulation, cell cycle, network, gene interactions, signaling network, quantum systems, system dynamics, networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Quantum cybernetics and its connections to complex quantum systems science is addressed from the perspective of quantum artificial neural networks as complex quantum computing systems. In this way, the notion of an autonomous quantum computing system is introduced in regards to quantum artificial intelligence, and applied to quantum artificial neural networks, considered as autonomous quantum computing systems, which leads to a quantum connectionist framework within quantum cybernetics for complex quantum computing systems. Several examples of quantum feedforward neural networks are addressed in regards to Boolean functions computation, multilayer quantum computation dynamics, entanglement and quantum complementarity. The examples provide a basis for a reflection on the role of quantum artificial neural networks as a general framework for addressing complex quantum systems that perform network-based quantum computation, possible consequences are drawn regarding quantum technologies, as well as fundamental research in complex quantum systems science, neuroscience and quantum biology.</div>','<b>Title: </b>Clusters in nonsmooth oscillator networks<br><b>Year: </b>2018<br><b>Author: </b> R. Nicks,  L. Chambon and  S. Coombes<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1103/PhysRevE.97.032213"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PHYSICAL REVIEW E<br><b>Cluster-Keywords: </b><em>oscillator networks, network dynamics, network, networks, dynamical systems, dynamics, node dynamics, systems, coupled oscillator, synchronization, dynamics network, complex network, coupled system, network modifications, oscillators, network connectivity, feedback vertex, network topology, synchronization properties, collective dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>For coupled oscillator networks with Laplacian coupling, the master stability function MSF has proven a particularly powerful tool for assessing the stability of the synchronous state. Using tools from group theory, this approach has recently been extended to treat more general cluster states. However, theMSF and its generalizations require the determination of a set of Floquet multipliers from variational equations obtained by linearization around a periodic orbit. Since closed form solutions for periodic orbits are invariably hard to come by, the framework is often explored using numerical techniques. Here, we show that further insight into network dynamics can be obtained by focusing on piecewise linear PWL oscillator models. Not only do these allow for the explicit construction of periodic orbits, their variational analysis can also be explicitly performed. The price for adopting such nonsmooth systems is that many of the notions from smooth dynamical systems, and in particular linear stability, need to be modified to take into account possible jumps in the components of Jacobians. This is naturally accommodated with the use of saltation matrices. By augmenting the variational approach for studying smooth dynamical systems with such matrices we show that, for a wide variety of networks that have been used as models of biological systems, cluster states can be explicitly investigated. By way of illustration, we analyze an integrate-and-fire network model with event-driven synaptic coupling as well as a diffusively coupled network built from planar PWL nodes, including a reduction of the popular Morris-Lecar neuron model. We use these examples to emphasize that the stability of network cluster states can depend as much on the choice of single node dynamics as it does on the form of network structural connectivity. Importantly, the procedure that we present here, for understanding cluster synchronization in networks, is valid for a wide variety of systems in biology, physics, and engineering that can be described by PWL oscillators.</div>','<b>Title: </b>Quantitative proteomics profiling of the polyADP-ribose-related response to genotoxic stress<br><b>Year: </b>2012<br><b>Author: </b> Jp. Gagne,  E. Pic,  M. Isabelle,  J. Krietsch,  C. Ethier,  E. Paquet,  I. Kelly,  M. Boutin,  Km. Moon,  Lj. Foster and  Gg. Poirier<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1093/nar/gks486"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>NUCLEIC ACIDS RESEARCH<br><b>Cluster-Keywords: </b><em>gene regulatory, systems biology, regulatory networks, gene network, stem cell, network model, transcription factors, gene expression, cell transcriptomes, network analysis, data analysis, stem cells, network grnsight, cell transformation, weighted network, gene ontology, networks grns, graph model, graph layout, endothelial cell<div style="font-size:11px;"><br><b>Abstract: </b><em>Upon DNA damage induction, DNA-dependent polyADP-ribose polymerases PARPs synthesize an anionic polyADP-ribose pADPr scaffold to which several proteins bind with the subsequent formation of pADPr-associated multiprotein complexes. We have used a combination of affinity-purification methods and proteomics approaches to isolate these complexes and assess protein dynamics with respect to pADPr metabolism. As a first approach, we developed a substrate trapping strategy by which we demonstrate that a catalytically inactive PolyADP-ribose glycohydrolase PARG mutant can act as a physiologically selective bait for the isolation of specific pADPr-binding proteins through its macrodomain-like domain. In addition to antibody-mediated affinity-purification methods, we used a pADPr macrodomain affinity resin to recover pADPr-binding proteins and their complexes. Second, we designed a time course experiment to explore the changes in the composition of pADPr-containing multiprotein complexes in response to alkylating DNA damage-mediated PARP activation. Spectral count clustering based on GeLC-MS/MS analysis was complemented with further analyses using high precision quantitative proteomics through isobaric tag for relative and absolute quantitation iTRAQ- and Stable isotope labeling by amino acids in cell culture SILAC-based proteomics. Here, we present a valuable resource in the interpretation of systems biology of the DNA damage response network in the context of polyADP-ribosylation and provide a basis for subsequent investigations of pADPr-binding protein candidates.</div>','<b>Title: </b>Sensitivity and Robustness of Spatially Dependent Thrombin Generation and Fibrin Clot Propagation<br><b>Year: </b>2018<br><b>Author: </b> Ad. Kuprash,  Am. Shibeko,  R. Vijay,  Sc. Nair,  A. Srivastava,  Fi. Ataullakhanov,  Ma. Panteleev and  An. Balandina<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.bpj.2018.11.009"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOPHYSICAL JOURNAL<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Blood coagulation is a delicately regulated space- and time-dependent process that leads to the formation of fibrin clots preventing blood loss upon vascular injury. The sensitivity of the coagulation network was previously investigated without accounting for transport processes. To investigate its sensitivity to coagulation factor deficiencies in a spatial reaction-diffusion system, we combined an in vitro experimental design with a computational systems biology model. Clot formation in platelet-free plasma supplemented with phospholipids was activated with identical amounts of tissue factor TF either homogeneously distributed concentration 5 pM, homogeneous model or immobilized on the surface surface density 100 pmole/m2, spatially heterogeneous model. Fibrin clot growth and thrombin concentration dynamic in space were observed using video microscopy in plasma of healthy donors or patients with deficiencies in factors F II, FV, FVII, FVIII, FIX, FX, or FXI. In the spatially heterogeneous model, near-activator thrombin generation was decreased in FV-, FVII-, and FX-deficient plasma. In the homogeneous model, clotting was not registered in these samples. The simulation and experiment data showed that the coagulation threshold depended on the TF concentration. Our data indicate that the velocity of spatial clot propagation correlates linearly with the concentration of thrombin at the clot wave front but not with the overall thrombin wave amplitude. Spatial clot growth in normal plasma at early stages was neither reaction nor diffusion limited but became diffusion limited later. In contrast, clot growth was always diffusion limited in FV-, FVII-, and FX-deficient plasma and reaction limited in FVIII-, FIX-, and FXI-deficient plasma. We conclude that robustness of the spatially heterogeneous coagulation system was achieved because of the combination of 1 a local high TF surface density that overcomes activation thresholds, 2 diffusion control being shared between different active factors, and 3 an early saturated stimulus-response dependence of fibrin clot formation by thrombin.</div>','<b>Title: </b>Advances in restoration ecology: rising to the challenges of the coming decades<br><b>Year: </b>2015<br><b>Author: </b> Mp. Perring,  Rj. Standish,  Jn. Price,  Md. Craig,  Te. Erickson,  Kx. Ruthrof,  As. Whiteley,  Le. Valentine and  Rj. Hobbs<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1890/ES15-00121.1"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ECOSPHERE<br><b>Cluster-Keywords: </b><em>network ecology, food webs, network models, ecological systems, food web, systems biology, ecological networks, ecosystem model, signalling networks, model results, interaction networks, dynamic model, ecological interactions, network analysis, model network, network structure, transport networks, systems network, ecosystem dynamics, restoration ecology<div style="font-size:11px;"><br><b>Abstract: </b><em>Simultaneous environmental changes challenge biodiversity persistence and human wellbeing. The science and practice of restoration ecology, in collaboration with other disciplines, can contribute to overcoming these challenges. This endeavor requires a solid conceptual foundation based in empirical research which confronts, tests and influences theoretical developments. We review conceptual developments in restoration ecology over the last 30 years. We frame our review in the context of changing restoration goals which reflect increased societal awareness of the scale of environmental degradation and the recognition that inter-disciplinary approaches are needed to tackle environmental problems. Restoration ecology now encompasses facilitative interactions and network dynamics, trophic cascades, and above- and below ground linkages. It operates in a non-equilibrium, alternative states framework, at the landscape scale, and in response to changing environmental, economic and social conditions. Progress has been marked by conceptual advances in the fields of trait-environment relationships, community assembly, and understanding the links between biodiversity and ecosystem functioning. Conceptual and practical advances have been enhanced by applying evolving technologies, including treatments to increase seed germination and overcome recruitment bottlenecks, high throughput DNA sequencing to elucidate soil community structure and function, and advances in satellite technology and GPS tracking to monitor habitat use. The synthesis of these technologies with systematic reviews of context dependencies in restoration success, model based analyses and consideration of complex socio-ecological systems will allow generalizations to inform evidence based interventions. Ongoing challenges include setting realistic, socially acceptable goals for restoration under changing environmental conditions, and prioritizing actions in an increasingly space-competitive world. Ethical questions also surround the use of genetically modified material, translocations, taxon substitutions, and de-extinction, in restoration ecology. Addressing these issues, as the Ecological Society of America looks to its next century, will require current and future generations of researchers and practitioners, including economists, engineers, philosophers, landscape architects, social scientists and restoration ecologists, to work together with communities and governments to rise to the environmental challenges of the coming decades.</div>','<b>Title: </b>STOCHASTIC TIME DELAYS IN REGULATORY NETWORKS AND MODEL REDUCTION<br><b>Year: </b>2011<br><b>Author: </b> Jgd. Ochoa<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1142/S0129183111016312"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>INTERNATIONAL JOURNAL OF MODERN PHYSICS C<br><b>Cluster-Keywords: </b><em>regulatory networks, stochastic simulation, gene networks, reaction networks, systems biology, stochastic model, network models, complex systems, networks stochastic, gene regulatory, biological systems, stochastic dynamics, differential equations, protein networks, cell cycle, continuous model, interaction networks, systems chemistry, time delays, biochemical reaction<div style="font-size:11px;"><br><b>Abstract: </b><em>Stochastic time delays and asynchronism in regulatory networks is an ubiquitous phenomena in biology, in particular in regulatory networks. However, this phenomena is poorly understood. In particular, variable-stochastic time intervals in the realization of chemical reactions have a crucial effect in the dynamics of the system. I analyze the effect that different time delays in feedback loops can have in the transient behavior of a regulatory network. Using a simple toy model for a regulatory network with forcing I show that such time delays introduce an oscillatory behavior that can alternatively be explained if additional loops are defined. The results suggest that the introduction of time delays is a mechanism that is helpful to theoretically reduce the number of elements, in particular the number of feedback loops, in competing network models.</div>','<b>Title: </b>Identification of feedback loops embedded in cellular circuits by investigating non-causal impulse response components<br><b>Year: </b>2010<br><b>Author: </b> Cy. Dong,  Tw. Yoon,  Dg. Bates and  Kh. Cho<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s00285-009-0263-x"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF MATHEMATICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>feedback loops, signaling network, positive feedback, network model, negative feedback, biochemical networks, signaling pathways, signal transduction, systems biology, kinase erk, protein kinase, transduction networks, dynamic model, emt network, erk pathway, cell fate, cell death, biochemical reaction, coupled feedback, impulse response<div style="font-size:11px;"><br><b>Abstract: </b><em>Feedback circuits are crucial dynamic motifs which occur in many biomolecular regulatory networks. They play a pivotal role in the regulation and control of many important cellular processes such as gene transcription, signal transduction, and metabolism. In this study, we develop a novel computationally efficient method to identify feedback loops embedded in intracellular networks, which uses only time-series experimental data and requires no knowledge of the network structure. In the proposed approach, a non-parametric system identification technique, as well as a spectral factor analysis, is applied to derive a graphical criterion based on non-causal components of the systems impulse response. The appearance of non-causal components in the impulse response sequences arising from stochastic output perturbations is shown to imply the presence of underlying feedback connections within a linear network. In order to extend the approach to nonlinear networks, we linearize the intracellular networks about an equilibrium point, and then choose the magnitude of the output perturbations sufficiently small so that the resulting time-series responses remain close to the chosen equilibrium point. In this way, the impulse response sequences of the linearized system can be used to determine the presence or absence of feedback loops in the corresponding nonlinear network. The proposed method utilizes the time profile data from intracellular perturbation experiments and only requires the perturbability of output nodes. Most importantly, the method does not require any a priori knowledge of the system structure. For these reasons, the proposed approach is very well suited to identifying feedback loops in large-scale bio-molecular networks. The effectiveness of the proposed method is illustrated via two examples: a synthetic network model with a negative feedback loop and a nonlinear caspase function model of apoptosis with a positive feedback loop.</div>','<b>Title: </b>Using normal mode analysis in teaching mathematical modeling to biology students<br><b>Year: </b>2011<br><b>Author: </b> Da. Kondrashov<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1051/mmnp/20116615"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MATHEMATICAL MODELLING OF NATURAL PHENOMENA<br><b>Cluster-Keywords: </b><em>normal mode, mode analysis, elastic network, functional motions, functional dynamics, molecular chaperone, structural biology, acidic residues, network models, molecular dynamics, atomic resolution, protein complexes, conserved functional, hsp dynamics, correlated motions, protein folding, large biomolecular, thioredoxin system, harmonic analysis, protein motions<div style="font-size:11px;"><br><b>Abstract: </b><em>Linear oscillators are used for modeling a diverse array of natural systems, for instance acoustics, materials science, and chemical spectroscopy. In this paper I describe simple models of structural interactions in biological molecules, known as elastic network models, as a useful topic for undergraduate biology instruction in mathematical modeling. These models use coupled linear oscillators to model the fluctuations of molecular structures around the equilibrium state. I present many learning activities associated with building and understanding these models, ranging from analytical to computational. I provide a number of web resources where students can obtain structural data, perform calculations, and suggest research directions for independent projects.</div>','<b>Title: </b>Non-linear interactions between candidate genes of myocardial infarction revealed in mRNA expression profiles<br><b>Year: </b>2016<br><b>Author: </b> K. Hartmann,  M. Seweryn,  Sk. Handleman,  Ga. Rempala and  W. Sadee<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/s12864-016-3075-6"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC GENOMICS<br><b>Cluster-Keywords: </b><em>gene expression, systems biology, metabolic network, network model, network analysis, dynamic model, signaling pathways, signaling network, regulatory network, dynamic network, biology approach, candidate genes, gene network, gene regulatory, biology analysis, mathematical model, network dynamic, biology network, metabolic pathways, pathway analysis<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Alterations in gene expression are key events in disease etiology and risk. Poor reproducibility in detecting differentially expressed genes across studies suggests individual genes may not be sufficiently informative for complex diseases, such as myocardial infarction MI. Rather, dysregulation of the molecular network may be critical for pathogenic processes. Such a dynamic network can be built from pairwise non-linear interactions.Results: We investigate non-linear interactions represented in mRNA expression profiles that integrate genetic background and environmental factors. Using logistic regression, we test the association of individual GWAS-based candidate genes and non-linear interaction terms between these mRNA expression levels with MI. Based on microarray data in CATHGEN CATHeterization in GENetics and FHS Framingham Heart Study, we find individual genes and pairs of mRNAs, encoded by 41 MI candidate genes, with significant interaction terms in the logistic regression model. Two pairs replicate between CATHGEN and FHS CNNM2 vertical bar GUCY1A3 and CNNM2 vertical bar ZEB2.Analysis of RNAseq data from GTEx Genotype-Tissue Expression shows that 20  of these disease-associated RNA pairs are co-expressed, further prioritizing significant interactions. Because edges in sparse co-expression networks formed solely by the 41 candidate genes are unlikely to represent direct physical interactions, we identify additional RNAs as links between network pairs of candidate genes. This approach reveals additional mRNAs and interaction terms significant in the context of MI, for example, the path CNNM2 vertical bar ACSL5 vertical bar SCARF1 vertical bar GUCY1A3, characterized by the common themes of magnesium and lipid processing.Conclusions: The results of this study support a role for non-linear interactions between genes in MI and provide a basis for further study of MI systems biology. mRNA expression profiles encoded by a limited number of candidate genes yield sparse networks of MI-relevant interactions that can be expanded to include additional candidates by co-expression analysis. The non-linear interactions observed here inform our understanding of the clinical relevance of gene-gene interactions in the pathophysiology of MI, while providing a new strategy in developing clinical biomarker panels.</div>','<b>Title: </b>From Petri nets to differential equations - An integrative approach for biochemical network analysis<br><b>Year: </b>2006<br><b>Author: </b> D. Gilbert and  M. HeinerLink: No link<br><b>Journal: </b>PETRI NETS AND OTHER MODELS OF CONCURRENCY - ICATPN 2006<br><b>Cluster-Keywords: </b><em>petri net, petri nets, bayesian networks, biological networks, systems biology, regulatory networks, dynamic bayesian, net model, gene regulatory, network data, biological systems, biological data, network analysis, network models, biochemical networks, time series, network structure, signal transduction, net models, dbn model<div style="font-size:11px;"><br><b>Abstract: </b><em>We report on the results of an investigation into the integration of Petri nets and ordinary differential equations ODES for the modelling and analysis of biochemical networks, and the application of our approach to the model of the influence of the Raf Kinase Inhibitor Protein RKIP on the Extracellular signal Regulated Kinase ERK signalling pathway. We show that analysis based on a discrete Petri net model of the system can be used to derive the sets of initial concentrations required by the corresponding continuous ordinary differential equation model, and no other initial concentrations produce meaningful steady states. Altogether, this paper represents a tutorial in step-wise modelling and analysis of larger models as well as in structured design of ODES.</div>','<b>Title: </b>HOW FAR AND HOW FAST CAN ONE MOVE ON NEUTRAL NETWORK<br><b>Year: </b>2013<br><b>Author: </b> Ch. Du,  H. Chen,  Yj. Zhao and  C. Zeng<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1142/S0219633613410101"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF THEORETICAL  COMPUTATIONAL CHEMISTRY<br><b>Cluster-Keywords: </b><em>network models, gene network, regulatory networks, boolean networks, gene expression, systems biology, neural networks, network dynamics, gene regulatory, biological networks, network structure, gap gene, gene regulation, cell cycle, network, gene interactions, signaling network, quantum systems, system dynamics, networks<div style="font-size:11px;"><br><b>Abstract: </b><em>A central theme in systems biology is to reveal the intricate relationship between structure and dynamics of many complex biological networks. Using Boolean models that describe yeast cell cycle process, we developed a unique logic-based theoretical framework to quantitatively determine the structure-dynamics mapping, also known as genotype-phenotype mapping. Moreover, under the dominant inhibition condition, we used a superposition property to show rigorously that the neutral network - the network of all possible structures that encode the same dynamics and are connected via single interaction mutations - forms one giant connected and conductive component. This may help shed light on the evolution landscape of biological networks based on the distance and speed a network can evolve on this neutral network.</div>','<b>Title: </b>Cell cycle and protein complex dynamics in discovering signaling pathways<br><b>Year: </b>2019<br><b>Author: </b> D. Inostroza,  C. Hernandez,  D. Seco,  G. Navarro and  A. Olivera-Nappa<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1142/S0219720019500112"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF BIOINFORMATICS AND COMPUTATIONAL BIOLOGY<br><b>Cluster-Keywords: </b><em>interaction networks, proteinprotein interactions, protein interaction, network model, complex systems, systems biology, protein protein, protein complex, host network, dynamic interactions, network analysis, biological systems, networks network, cellular network, biological network, viral proteins, interactome networks, regulatory networks, immune system, systems network<div style="font-size:11px;"><br><b>Abstract: </b><em>Signaling pathways are responsible for the regulation of cell processes, such as monitoring the external environment, transmitting information across membranes, and making cell fate decisions. Given the increasing amount of biological data available and the recent discoveries showing that many diseases are related to the disruption of cellular signal transduction cascades, in silico discovery of signaling pathways in cell biology has become an active research topic in past years. However, reconstruction of signaling pathways remains a challenge mainly because of the need for systematic approaches for predicting causal relationships, like edge direction and activation/inhibition among interacting proteins in the signal flow. We propose an approach for predicting signaling pathways that integrates protein interactions, gene expression, phenotypes, and protein complex information. Our method first finds candidate pathways using a directed-edge-based algorithm and then defines a graph model to include causal activation relationships among proteins, in candidate pathways using cell cycle gene expression and phenotypes to infer consistent pathways in yeast. Then, we incorporate protein complex coverage information for deciding on the final predicted signaling pathways. We show that our approach improves the predictive results of the state of the art using different ranking metrics.</div>','<b>Title: </b>A U-system approach for predicting metabolic behaviors and responses based on an alleged metabolic reaction network<br><b>Year: </b>2014<br><b>Author: </b> K. Sriyudthsak,  Y. Sawada,  Y. Chiba,  Y. Yamashita,  S. Kanaya,  H. Onouchi,  T. Fujiwara,  S. Naito,  Eo. Voit,  F. Shiraishi and  My. Hirai<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-8-S5-S4"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>systems biology, dynamic models, biological systems, model, dynamic systems, computational systems, network model, experimental data, model parameters, systems, systems model, models, pathway systems, data, biochemical pathway, dynamics model, biochemical networks, computational models, network, system<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Progress in systems biology offers sophisticated approaches toward a comprehensive understanding of biological systems. Yet, computational analyses are held back due to difficulties in determining suitable model parameter values from experimental data which naturally are subject to biological fluctuations. The data may also be corrupted by experimental uncertainties and sometimes do not contain all information regarding variables that cannot be measured for technical reasons.Results: We show here a streamlined approach for the construction of a coarse model that allows us to set up dynamic models with minimal input information. The approach uses a hybrid between a pure mass action system and a generalized mass action GMA system in the framework of biochemical systems theory BST with rate constants of 1, normal kinetic orders of 1, and -0.5 and 0.5 for inhibitory and activating effects, named Unity U-system. The U-system model does not necessarily fit all data well but is often sufficient for predicting metabolic behavior of metabolites which cannot be simultaneously measured, identifying inconsistencies between experimental data and the assumed underlying pathway structure, as well as predicting system responses to a modification of gene or enzyme. The U-system approach was validated with small, generic systems and implemented to model a large-scale metabolic reaction network of a higher plant, Arabidopsis. The dynamic behaviors obtained by predictive simulations agreed with actually available metabolomic time-series data, identified probable errors in the experimental datasets, and estimated probable behavior of unmeasurable metabolites in a qualitative manner. The model could also predict metabolic responses of Arabidopsis with altered network structures due to genetic modification.Conclusions: The U-system approach can effectively predict metabolic behaviors and responses based on structures of an alleged metabolic reaction network. Thus, it can be a useful first-line tool of data analysis, model diagnostics and aid the design of next-step experiments.</div>','<b>Title: </b>Cellular automata modelling of biomolecular networks dynamics<br><b>Year: </b>2010<br><b>Author: </b> D. Bonchev,  S. Thomas,  A. Apte and  Lb. Kier<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1080/10629360903568580"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SAR AND QSAR IN ENVIRONMENTAL RESEARCH<br><b>Cluster-Keywords: </b><em>dna damage, signalling networks, network dynamics, pmdm system, systems biology, systems dynamics, biological systems, single cell, kinase inhibitors, signaling network, biological networks, cellular automata, cell cycle, oscillation system, temporal dynamics, signaling pathway, individual cells, signaling processes, dynamic model, damage response<div style="font-size:11px;"><br><b>Abstract: </b><em>The modelling of biological systems dynamics is traditionally performed by ordinary differential equations ODEs. When dealing with intracellular networks of genes, proteins and metabolites, however, this approach is hindered by network complexity and the lack of experimental kinetic parameters. This opened the field for other modelling techniques, such as cellular automata CA and agent-based modelling ABM. This article reviews this emerging field of studies on network dynamics in molecular biology. The basics of the CA technique are discussed along with an extensive list of related software and websites. The application of CA to networks of biochemical reactions is exemplified in detail by the case studies of the mitogen-activated protein kinase MAPK signalling pathway, the FAS-ligand FASL-induced and Bcl-2-related apoptosis. The potential of the CA method to model basic pathways patterns, to identify ways to control pathway dynamics and to help in generating strategies to fight with cancer is demonstrated. The different line of CA applications presented includes the search for the best-performing network motifs, an analysis of importance for effective intracellular signalling and pathway cross-talk.</div>','<b>Title: </b>A conceptual and computational framework for modelling and understanding the non-equilibrium gene regulatory networks of mouse embryonic stem cells<br><b>Year: </b>2017<br><b>Author: </b> Rb. Greaves,  S. Dietmann,  A. Smith,  S. Stepney and  Jd. Halley<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pcbi.1005713"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS COMPUTATIONAL BIOLOGY<br><b>Cluster-Keywords: </b><em>stem cell, stem cells, systems biology, mathematical models, cell biology, network model, signaling pathways, cell fate, computational models, cancer cells, gene expression, network dynamics, embryonic stem, signaling network, prostate cancer, mathematical modeling, psychological stress, apoptosis signaling, computational systems, molecular mechanisms<div style="font-size:11px;"><br><b>Abstract: </b><em>The capacity of pluripotent embryonic stem cells to differentiate into any cell type in the body makes them invaluable in the field of regenerative medicine. However, because of the complexity of both the core pluripotency network and the process of cell fate computation it is not yet possible to control the fate of stem cells. We present a theoretical model of stem cell fate computation that is based on Halley and Winklers Branching Process Theory BPT and on Greaves et al.s agent-based computer simulation derived from that theoretical model. BPT abstracts the complex production and action of a Transcription Factor TF into a single critical branching process that may dissipate, maintain, or become supercritical. Here we take the single TF model and extend it to multiple interacting TFs, and build an agent-based simulation of multiple TFs to investigate the dynamics of such coupled systems. We have developed the simulation and the theoretical model together, in an iterative manner, with the aim of obtaining a deeper understanding of stem cell fate computation, in order to influence experimental efforts, which may in turn influence the outcome of cellular differentiation. The model used is an example of self-organization and could be more widely applicable to the modelling of other complex systems. The simulation based on this model, though currently limited in scope in terms of the biology it represents, supports the utility of the Halley and Winkler branching process model in describing the behaviour of stem cell gene regulatory networks. Our simulation demonstrates three key features: i the existence of a critical value of the branching process parameter, dependent on the details of the cistrome in question; ii the ability of an active cistrome to "ignite" an otherwise fully dissipated cistrome, and drive it to criticality; iii how coupling cistromes together can reduce their critical branching parameter values needed to drive them to criticality.</div>','<b>Title: </b>A framework for parameter estimation and model selection from experimental data in systems biology using approximate Bayesian computation<br><b>Year: </b>2014<br><b>Author: </b> J. Liepe,  P. Kirk,  S. Filippi,  T. Toni,  Cp. Barnes and  Mph. Stumpf<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1038/nprot.2014.025"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>NATURE PROTOCOLS<br><b>Cluster-Keywords: </b><em>circadian clock, biological systems, network models, mathematical models, clock gene, systems biology, model selection, biological networks, network data, experimental data, model parameters, network dynamics, plant circadian, regulatory network, clock model, plant clock, complex network, clock circuit, approximate bayesian, stochastic models<div style="font-size:11px;"><br><b>Abstract: </b><em>As modeling becomes a more widespread practice in the life sciences and biomedical sciences, researchers need reliable tools to calibrate models against ever more complex and detailed data. Here we present an approximate Bayesian computation ABC framework and software environment, ABC-SysBio, which is a Python package that runs on Linux and Mac OS X systems and that enables parameter estimation and model selection in the Bayesian formalism by using sequential Monte Carlo SMC approaches. We outline the underlying rationale, discuss the computational and practical issues and provide detailed guidance as to how the important tasks of parameter inference and model selection can be performed in practice. Unlike other available packages, ABC-SysBio is highly suited for investigating, in particular, the challenging problem of fitting stochastic models to data. In order to demonstrate the use of ABC-SysBio, in this protocol we postulate the existence of an imaginary reaction network composed of seven interrelated biological reactions involving a specific mRNA, the protein it encodes and a post-translationally modified version of the protein, a network that is defined by two files containing observed data that we provide as supplementary information. In the first part of the PROCEDURE, ABC-SysBio is used to infer the parameters of this system, whereas in the second part we use ABC-SysBios relevant functionality to discriminate between two different reaction network models, one of them being the true one. Although computationally expensive, the additional insights gained in the Bayesian formalism more than make up for this cost, especially in complex problems.</div>','<b>Title: </b>Temporal transcriptional response to latency reversing agents identifies specific factors regulating HIV-1 viral transcriptional switch<br><b>Year: </b>2015<br><b>Author: </b> Nj. Venkatachari,  Jm. Zerbato,  S. Jain,  Ae. Mancini,  A. Chattopadhyay,  N. Sluis-Cremer,  Z. Bar-Joseph and  V. Ayyavoo<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/s12977-015-0211-3"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>RETROVIROLOGY<br><b>Cluster-Keywords: </b><em>systems biology, regulatory network, interaction networks, network model, gene regulatory, network, network biology, networks, gene network, response networks, network dynamics, dynamic model, cell networks, gene expression, biological networks, system dynamics, dynamic network, model, biological systems, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Latent HIV-1 reservoirs are identified as one of the major challenges to achieve HIV-1 cure. Currently available strategies are associated with wide variability in outcomes both in patients and CD4 T cell models. This underlines the critical need to develop innovative strategies to predict and recognize ways that could result in better reactivation and eventual elimination of latent HIV-1 reservoirs.Results and discussion: In this study, we combined genome wide transcriptome datasets post activation with Systems Biology approach Signaling and Dynamic Regulatory Events Miner, SDREM analyses to reconstruct a dynamic signaling and regulatory network involved in reactivation mediated by specific activators using a latent cell line. This approach identified several critical regulators for each treatment, which were confirmed in follow-up validation studies using small molecule inhibitors. Results indicate that signaling pathways involving JNK and related factors as predicted by SDREM are essential for virus reactivation by suberoylanilide hydroxamic acid. ERK1/2 and NF-kappa B pathways have the foremost role in reactivation with prostratin and TNF-alpha, respectively. JAK-STAT pathway has a central role in HIV-1 transcription. Additional evaluation, using other latent J-Lat cell clones and primary T cell model, also confirmed that many of the cellular factors associated with latency reversing agents are similar, though minor differences are identified. JAK-STAT and NF-kappa B related pathways are critical for reversal of HIV-1 latency in primary resting T cells.Conclusion: These results validate our combinatorial approach to predict the regulatory cellular factors and pathways responsible for HIV-1 reactivation in latent HIV-1 harboring cell line models. JAK-STAT have a role in reversal of latency in all the HIV-1 latency models tested, including primary CD4 T cells, with additional cellular pathways such as NF-kappa B, JNK and ERK 1/2 that may have complementary role in reversal of HIV-1 latency.</div>','<b>Title: </b>It is not the parts, but how they interact that determines the behaviour of circadian rhythms across scales and organisms<br><b>Year: </b>2014<br><b>Author: </b> D. Dewoskin,  Wh. Geng,  Ar. Stinchcombe and  Db. Forger<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1098/rsfs.2013.0076"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>INTERFACE FOCUS<br><b>Cluster-Keywords: </b><em>circadian rhythms, systems biology, biological clock, circadian clock, biological rhythms, hpa axis, model, clock network, mathematical models, biological systems, dynamic systems, ode models, biology models, single cell, network, model parameters, experimental data, models, systems, dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>Biological rhythms, generated by feedback loops containing interacting genes, proteins and/or cells, time physiological processes in many organisms. While many of the components of the systems that generate biological rhythms have been identified, much less is known about the details of their interactions. Using examples from the circadian daily clock in three organisms, Neurospora, Drosophila and mouse, we show, with mathematical models of varying complexity, how interactions among i promoter sites, ii proteins forming complexes, and iii cells can have a drastic effect on timekeeping. Inspired by the identification of many transcription factors, for example as involved in the Neurospora circadian clock, that can both activate and repress, we show how these multiple actions can cause complex oscillatory patterns in a transcription-translation feedback loop TTFL. Inspired by the timekeeping complex formed by the NMO-PER-TIM-SGG complex that regulates the negative TTFL in the Drosophila circadian clock, we show how the mechanism of complex formation can determine the prevalence of oscillations in a TTFL. Finally, we note that most mathematical models of intracellular clocks model a single cell, but compare with experimental data from collections of cells. We find that refitting the most detailed model of the mammalian circadian clock, so that the coupling between cells matches experimental data, yields different dynamics and makes an interesting prediction that also matches experimental data: individual cells are bistable, and network coupling removes this bistability and causes the network to be more robust to external perturbations. Taken together, we propose that the interactions between components in biological timekeeping systems are carefully tuned towards proper function. We also show how timekeeping can be controlled by novel mechanisms at different levels of organization.</div>','<b>Title: </b>An extended fractional Kalman filter for inferring gene regulatory networks using time-series data<br><b>Year: </b>2014<br><b>Author: </b> Yq. Zhang,  Yf. Pu,  Hs. Zhang,  Y. Cong and  Jl. Zhou<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.chemolab.2014.07.007"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>CHEMOMETRICS AND INTELLIGENT LABORATORY SYSTEMS<br><b>Cluster-Keywords: </b><em>regulatory networks, gene regulatory, gene expression, network inference, expression data, network structure, gene networks, networks grns, regulatory genes, boolean networks, bayesian network, network reconstruction, networks gene, network dynamics, gene regulation, inference method, systems biology, time series, infer gene, network<div style="font-size:11px;"><br><b>Abstract: </b><em>In recent years, inference of gene regulatory networks has received ever increasing attention in the systems biology field. In this paper, for the first time, a fractional gene regulatory algorithm by extended fractional Kalman filter EFKF is proposed to estimate the hidden states as well as the unknown static parameters of the model, which can provide insight into the underlying regulatory relations among genes in the biological system. In the proposed method, gene regulatory networks are inferred via evolutionary modeling based on time-series microarray measurements. The gene regulatory network is considered as a fractional order discrete stochastic dynamic model that consists of the gene measurement equation and the gene regulation equation. After specifying the model structure, we apply the EFKF algorithm for identifying both the model parameters and the actual value of gene expression levels. In this paper, the main advantages of using fractional order systems, increasing the flexibility and improving the accuracy of the system state equation in EFKF are highlighted. The performance of the EFKF algorithm is compared with EKF and other nonlinear algorithms in predicting the parameters of gene regulatory networks from synthetic data and real biological data. Extensive computer simulations illustrate that the proposed algorithm outperforms EKF and other methods, and therefore, it can serve as a natural framework for inference gene regulatory networks with a nonlinear structure. C 2014 Elsevier B.V. All rights reserved.</div>','<b>Title: </b>An analysis of the class of gene regulatory functions implied by a biochemical model<br><b>Year: </b>2006<br><b>Author: </b> J. Grefenstette,  S. Kim and  S. Kauffman<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.biosystems.2005.09.009"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOSYSTEMS<br><b>Cluster-Keywords: </b><em>boolean network, boolean functions, regulatory networks, canalizing functions, network models, gene regulatory, boolean control, control networks, biological networks, boolean models, update functions, interaction networks, large network, regulatory functions, canalizing boolean, genetic regulatory, systems biology, driver nodes, networks bcns, biological systems<div style="font-size:11px;"><br><b>Abstract: </b><em>Understanding the integrated behavior of genetic regulatory networks, in which genes regulate one anothers activities via RNA and protein products, is emerging as a dominant problem in systems biology. One widely studied class of models of such networks includes genes whose expression values assume Boolean values i.e., on or off. Design decisions in the development of Boolean network models of gene regulatory systems include the topology of the network including the distribution of input- and output-connectivity and the class of Boolean functions used by each gene e.g., canalizing functions, post functions, etc.. For example, evidence from simulations suggests that biologically realistic dynamics can be produced by scale-free network topologies with canalizing Boolean functions. This work seeks further insights into the design of Boolean network models through the construction and analysis of a class of models that include more concrete biochemical mechanisms than the usual abstract model, including genes and gene products, dimerization, cis-binding sites, promoters and repressors. In this model, it is assumed that the system consists of N genes, with each gene producing one protein product. Proteins may forth complexes such as dinners, trimers. etc. The model also includes cis-binding sites to which proteins may bind to form activators or repressors. Binding affinities are based on structural complementarity between proteins and binding sites, with molecular binding sites modeled by bit-strings. Biochemically plausible gene expression rules are used to derive a Boolean regulatory function for each gene in the system. The result is a network model in which both topological features and Boolean functions arise as emergent properties of the interactions of components at the biochemical level. A highly biased set of Boolean functions is observed in simulations of networks of various sizes, suggesting a new characterization of the subset of Boolean functions that are likely to appear in gene regulatory networks. c 2005 Elsevier Ireland Ltd. All rights reserved.</div>','<b>Title: </b>Information and material flows in complex networks<br><b>Year: </b>2006<br><b>Author: </b> D. Helbing,  D. Armbruster,  As. Mikhallov and  E. Lefeber<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.physa.2006.01.042"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PHYSICA A-STATISTICAL MECHANICS AND ITS APPLICATIONS<br><b>Cluster-Keywords: </b><em>interaction networks, proteinprotein interactions, protein interaction, network model, complex systems, systems biology, protein protein, protein complex, host network, dynamic interactions, network analysis, biological systems, networks network, cellular network, biological network, viral proteins, interactome networks, regulatory networks, immune system, systems network<div style="font-size:11px;"><br><b>Abstract: </b><em>In this special issue, an overview of the Thematic Institute TI on Information and Material Flows in Complex Systems is given. The TI was carried out within EXYSTENCE, the first EU Network of Excellence in the area of complex systems. Its motivation, research approach and subjects are presented here. Among the various methods used are many-particle and statistical physics, nonlinear dynamics, as well as complex systems, network and control theory. The contributions are relevant for complex systems as diverse as vehicle and data traffic in networks, logistics, production, and material flows in biological systems. The key disciplines involved are socio-, econo-, traffic- and bio-physics, and a new research area that could be called "biologistics". c 2006 Elsevier B.V. All rights reserved.</div>','<b>Title: </b>Multiple synchronization transitions due to periodic coupling strength in delayed Newman-Watts networks of chaotic bursting neurons<br><b>Year: </b>2013<br><b>Author: </b> B. Xu,  Yb. Gong,  L. Wang and  Yn. Wu<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s11071-012-0691-9"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>NONLINEAR DYNAMICS<br><b>Cluster-Keywords: </b><em>coupling strength, synchronization transitions, timeperiodic coupling, time delay, logic network, distribution network, network model, oscillatory networks, neuron networks, water distribution, stochastic resonance, gene networks, multiple synchronization, firing synchronization, neuronal networks, firing behaviors, logical networks, results show, regulatory networks, cell polarization<div style="font-size:11px;"><br><b>Abstract: </b><em>In this paper, we numerically study the effect of time-periodic coupling strength on the synchronization of firing activity in delayed Newman-Watts networks of chaotic bursting neurons. We first examine how the firing synchronization transitions induced by time delay under fixed coupling strength changes in the presence of time-periodic coupling strength, and then focus on how time-periodic coupling strength induces synchronization transitions in the networks. It is found that time delay can induce more synchronization transitions in the presence of time-periodic coupling strength compared to fixed coupling strength. As the frequency of time-periodic coupling strength is varied, the firing exhibits multiple synchronization transitions between spiking antiphase synchronization and in-phase synchronization of various firing behaviors including bursting, spiking, and both bursting and spiking, depending on the values of time delay. These results show that time-periodic coupling strength can increase the synchronization transitions by time delay and can induce multiple synchronization transitions of various firing behaviors in the neuronal networks. This means that time-periodic coupling strength plays an important role in the information processing and transmission in neural systems.</div>','<b>Title: </b>Kinetics from Replica Exchange Molecular Dynamics Simulations<br><b>Year: </b>2017<br><b>Author: </b> Ls. Stelzl and  G. Hummer<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1021/acs.jctc.7b00372"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF CHEMICAL THEORY AND COMPUTATION<br><b>Cluster-Keywords: </b><em>allosteric proteins, signaling networks, protein interactions, network model, molecular dynamics, protein complexes, allosteric network, signaling proteins, cell network, biochemical networks, free energy, combinatorial complexity, molecular species, biological systems, receptor signaling, systems biology, chemical species, model, dynamics simulations, cell signaling<div style="font-size:11px;"><br><b>Abstract: </b><em>Transitions between metastable states govern many fundamental processes in physics, chemistry and biology, from nucleation events in phase transitions to the folding of proteins. The free energy surfaces underlying these processes can be obtained from simulations using enhanced sampling methods. However, their altered dynamics makes kinetic and mechanistic information difficult or impossible to extract. Here, we show that, with replica exchange molecular dynamics REMD, one can not only sample equilibrium "properties but also extract kinetic information. For systems that strictly obey first-order kinetics, the procedure to extract rates is rigorous. For actual molecular systems whose long-time dynamics are captured by kinetic rate models, accurate rate, coefficients can be determined from the statistics of the transitions between the metastable states at each replica temperature. We demonstrate the practical applicability of the procedure by constructing master equation Markov state models of peptide and RNA folding from REMD simulations.</div>','<b>Title: </b>Parametric uncertainty analysis of pulse wave propagation in a model of a human arterial network<br><b>Year: </b>2007<br><b>Author: </b> Db. Xiu and  Sj. Sherwin<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.jcp.2007.05.020"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF COMPUTATIONAL PHYSICS<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Reduced models of human arterial networks are an efficient approach to analyze quantitative macroscopic features of human arterial flows. The justification for such models typically arise due to the significantly long wavelength associated with the system in comparison to the lengths of arteries in the networks. Although these types of models have been employed extensively and many issues associated with their implementations have been widely researched, the issue of data uncertainty has received comparatively little attention. Similar to many biological systems, a large amount of uncertainty exists in the value of the parameters associated with the models. Clearly reliable assessment of the system behaviour cannot be made unless the effect of such data uncertainty is quantified.In this paper we present a study of parametric data uncertainty in reduced modelling of human arterial networks which is governed by a hyperbolic system. The uncertain parameters are modelled as random variables and the governing equations for the arterial network therefore become stochastic. This type stochastic hyperbolic systems have not been previously systematically studied due to the difficulties introduced by the uncertainty such as a potential change in the mathematical character of the system and imposing boundary conditions. We demonstrate how the application of a high-order stochastic collocation method based on the generalized polynomial chaos expansion, combined with a discontinuous Galerkin spectral/hp element discretization in physical space, can successfully simulate this type of hyperbolic system subject to uncertain inputs with bounds. Building upon a numerical study of propagation of uncertainty and sensitivity in a simplified model with a single bifurcation, a systematical parameter sensitivity analysis is conducted on the wave dynamics in a multiple bifurcating human arterial network. Using the physical understanding of the dynamics of pulse waves in these types of networks we are able to provide an insight into the results of the stochastic simulations, thereby demonstrating the effects of uncertainty in physiologically accurate human arterial networks. c 2007 Elsevier Inc. All rights reserved.</div>','<b>Title: </b>Identification of biochemical networks by S-tree based genetic programming<br><b>Year: </b>2006<br><b>Author: </b> Dy. Cho,  Kh. Cho and  Bt. Zhang<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1093/bioinformatics/btl122"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>ssystem model, biological systems, biological networks, parameter estimation, biochemical networks, dynamic biological, proposed algorithm, regulatory networks, structure identification, model structure, genetic algorithm, gene regulatory, gene expression, network inference, network motif, network structure, gene networks, network identification, network inferring, estimation method<div style="font-size:11px;"><br><b>Abstract: </b><em>Motivation: Most previous approaches to model biochemical networks have focused either on the characterization of a network structure with a number of components or on the estimation of kinetic parameters of a network with a relatively small number of components. For system-level understanding, however, we should examine both the interactions among the components and the dynamic behaviors of the components. A key obstacle to this simultaneous identification of the structure and parameters is the lack of data compared with the relatively large number of parameters to be estimated. Hence, there are many plausible networks for the given data, but most of them are not likely to exist in the real system.Results: We propose a new representation named S-trees for both the structural and dynamical modeling of a biochemical network within a unified scheme. We further present S-tree based genetic programming to identify the structure of a biochemical network and to estimate the corresponding parameter values at the same time. While other evolutionary algorithms require additional techniques for sparse structure identification, our approach can automatically assemble the sparse primitives of a biochemical network in an efficient way. We evaluate our algorithm on the dynamic profiles of an artificial genetic network. In 20 trials for four settings, we obtain the true structure and their relative squared errors are < 5 regardless of releasing constraints about structural sparseness. In addition, we confirm that the proposed algorithm is robust within /- 10 noise ratio. Furthermore, the proposed approach ensures a reasonable estimate of a real yeast fermentation pathway. The comparatively less important connections with non-zero parameters can be detected even though their orders are below 10-2. To demonstrate the usefulness of the proposed algorithm for real experimental biological data, we provide an additional example on the transcriptional network of SOS response to DNA damage in Escherichia coli. We confirm that the proposed algorithm can successfully identify the true structure except only one relation.</div>','<b>Title: </b>Martini Coarse-Grained Force Field: Extension to RNA<br><b>Year: </b>2017<br><b>Author: </b> Jj. Uusitalo,  Hi. Ingolfsson,  Sj. Marrink and  I. Faustino<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.bpj.2017.05.043"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOPHYSICAL JOURNAL<br><b>Cluster-Keywords: </b><em>allosteric proteins, signaling networks, protein interactions, network model, molecular dynamics, protein complexes, allosteric network, signaling proteins, cell network, biochemical networks, free energy, combinatorial complexity, molecular species, biological systems, receptor signaling, systems biology, chemical species, model, dynamics simulations, cell signaling<div style="font-size:11px;"><br><b>Abstract: </b><em>RNA has an important role not only as the messenger of genetic information but also as a regulator of gene expression. Given its central role in cell biology, there is significant interest in studying the structural and dynamic behavior of RNA in relation to other biomolecules. Coarse-grain molecular dynamics simulations are a key tool to that end. Here, we have extended the coarse-grain Martini force field to include RNA after our recent extension to DNA. In the same way DNA was modeled, the tertiary structure of RNA is constrained using an elastic network. This model, therefore, is not designed for applications involving RNA folding but rather offers a stable RNA structure for studying RNA interactions with other bio molecules. The RNA model is compatible with all other Martini models and opens the way to large-scale explicit-solvent molecular dynamics simulations of complex systems involving RNA.</div>','<b>Title: </b>In silico Derivation of a Reduced Kinetic Model for Stationary or Oscillating Glycolysis in Escherichia coli Bacterium<br><b>Year: </b>2014<br><b>Author: </b> G. MariaLink: No link<br><b>Journal: </b>CHEMICAL AND BIOCHEMICAL ENGINEERING QUARTERLY<br><b>Cluster-Keywords: </b><em>kinetic models, metabolic networks, kinetic parameters, dynamic models, systems biology, network model, dynamic metabolic, biochemical network, rate constants, enzyme kinetics, kinetic data, adjustable parameters, metabolic model, parameter estimation, dynamic optimization, model selection, mathematical model, kinetic network, model response, model<div style="font-size:11px;"><br><b>Abstract: </b><em>Modelling bacteria glycolysis is a classical subject but still of high interest. Glycolysis, together with the phosphotransferase PTS-system for glucose transport into the cell, the pentose-phosphate pathway PPP, and tricarboxylic acid cycle TCA characterize the central carbon metabolism. Such a model usually serves as the foundation for developing modular simulation platforms used for consistent analysis of the control / regulation of target metabolite synthesis. The present study is focused on analyzing the advantage and limitations of using a simplified but versatile core model of mTRM of glycolysis when incomplete experimental information is available. Exemplification is made for a reduced glycolysis model from literature for Escherichia coli cells, by performing a few modifications 17 identifiable parameters to increase its agreement with simulated data generated by using an extended model 127 parameters over a large operating domain of an experimental bioreactor. With the expense of ca. 8-13  increase in the relative model error vs. extended simulation models, derivation of reduced kinetic structures to describe some parts of the core metabolism is worth the associated identification effort, due to the considerable reduction in model parameterization e.g. 17 parameters in mTRM vs. 127 in the extendedChassM model of Chassagnole et al., while preserving a fair adequacy over a wide experimental domain generated in-silico by using the valuable extended ChassM. The reduced model flexibility is tested by reproducing stationary or oscillatory glycolysis conditions. The reduced mTRM model includes enough information to reproduce not only the cell energy-related potential through the total AMDTP level, but also the role played by ATP/ADP ratio as a glycolysis driving force. The model can also reproduce the oscillatory behaviour occurrence conditions, as well as situations when homeostatic conditions are not fulfilled.</div>','<b>Title: </b>A Dynamic Network Model of mTOR Signaling Reveals TSC-Independent mTORC2 Regulation<br><b>Year: </b>2012<br><b>Author: </b> Pd. Pezze,  Ag. Sonntag,  A. Thien,  Mt. Prentzell,  M. Godel,  S. Fischer,  E. Neumann-Haefelin,  Tb. Huber,  R. Baumeister,  Dp. Shanley and  K. Thedieck<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1126/scisignal.2002469"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SCIENCE SIGNALING<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>The kinase mammalian target of rapamycin mTOR exists in two multiprotein complexes mTORC1 and mTORC2 and is a central regulator of growth and metabolism. Insulin activation of mTORC1, mediated by phosphoinositide 3-kinase PI3K, Akt, and the inhibitory tuberous sclerosis complex 1/2 TSC1-TSC2, initiates a negative feedback loop that ultimately inhibits PI3K. We present a data-driven dynamic insulin-mTOR network model that integrates the entire core network and used this model to investigate the less well understood mechanisms by which insulin regulates mTORC2. By analyzing the effects of perturbations targeting several levels within the network in silico and experimentally, we found that, in contrast to current hypotheses, the TSC1-TSC2 complex was not a direct or indirect acting through the negative feedback loop regulator of mTORC2. Although mTORC2 activation required active PI3K, this was not affected by the negative feedback loop. Therefore, we propose an mTORC2 activation pathway through a PI3K variant that is insensitive to the negative feedback loop that regulates mTORC1. This putative pathway predicts that mTORC2 would be refractory to Akt, which inhibits TSC1-TSC2, and, indeed, we found that mTORC2 was insensitive to constitutive Akt activation in several cell types. Our results suggest that a previously unknown network structure connects mTORC2 to its upstream cues and clarifies which molecular connectors contribute to mTORC2 activation.</div>','<b>Title: </b>Inference of gene regulatory networks from time series by Tsallis entropy<br><b>Year: </b>2011<br><b>Author: </b> Fm. Lopes,  Ea. De Oliveira and  Rm. Cesar<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-5-61"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: The inference of gene regulatory networks GRNs from large-scale expression profiles is one of the most challenging problems of Systems Biology nowadays. Many techniques and models have been proposed for this task. However, it is not generally possible to recover the original topology with great accuracy, mainly due to the short time series data in face of the high complexity of the networks and the intrinsic noise of the expression measurements. In order to improve the accuracy of GRNs inference methods based on entropy mutual information, a new criterion function is here proposed.Results: In this paper we introduce the use of generalized entropy proposed by Tsallis, for the inference of GRNs from time series expression profiles. The inference process is based on a feature selection approach and the conditional entropy is applied as criterion function. In order to assess the proposed methodology, the algorithm is applied to recover the network topology from temporal expressions generated by an artificial gene network AGN model as well as from the DREAM challenge. The adopted AGN is based on theoretical models of complex networks and its gene transference function is obtained from random drawing on the set of possible Boolean functions, thus creating its dynamics. On the other hand, DREAM time series data presents variation of network size and its topologies are based on real networks. The dynamics are generated by continuous differential equations with noise and perturbation. By adopting both data sources, it is possible to estimate the average quality of the inference with respect to different network topologies, transfer functions and network sizes.Conclusions: A remarkable improvement of accuracy was observed in the experimental results by reducing the number of false connections in the inferred topology by the non-Shannon entropy. The obtained best free parameter of the Tsallis entropy was on average in the range 2.5 <= q <= 3.5 hence, subextensive entropy, which opens new perspectives for GRNs inference methods based on information theory and for investigation of the nonextensivity of such networks. The inference algorithm and criterion function proposed here were implemented and included in the DimReduction software, which is freely available at http://sourceforge.net/projects/dimreduction and http://code.google.com/p/dimreduction/.</div>','<b>Title: </b>Biological network design strategies: discovery through dynamic optimization<br><b>Year: </b>2006<br><b>Author: </b> Bs. Adiwijaya,  Pi. Barton and  B. Tidor<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1039/b610090b"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MOLECULAR BIOSYSTEMS<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>An important challenge in systems biology is the inherent complexity of biological network models, which complicates the task of relating network structure to function and of understanding the conceptual design principles by which a given network operates. Here we investigate an approach to analyze the relationship between a network structure and its function using the framework of optimization. A common feature found in a variety of biochemical networks involves the opposition of a pair of enzymatic chemical modification reactions such as phosphorylation-dephosphorylation or methylation-demethylation. The modification pair frequently adjusts biochemical properties of its target, such as activating and deactivating function. We applied optimization methodology to study a reversible modification network unit commonly found in signal transduction systems, and we explored the use of this methodology to discover design principles. The results demonstrate that different sets of rate constants used to parameterize the same network topology represent different compromises made in the resulting network operating characteristics. Moreover, the same topology can be used to encode different strategies for achieving performance goals. The ability to adopt multiple strategies may lead to significantly improved performance across a range of conditions through rate modulation or evolutionary processes. The optimization framework explored here is a practical approach to support the discovery of design principles in biological networks.</div>','<b>Title: </b>Alzheimers as a Systems-Level Disease Involving the Interplay of Multiple Cellular Networks<br><b>Year: </b>2016<br><b>Author: </b> Ji. Castrillo and  Sg. Oliver<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/978-1-4939-2627-51"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SYSTEMS BIOLOGY OF ALZHEIMERS DISEASE<br><b>Cluster-Keywords: </b><em>systems biology, regulatory network, interaction networks, network model, gene regulatory, network, network biology, networks, gene network, response networks, network dynamics, dynamic model, cell networks, gene expression, biological networks, system dynamics, dynamic network, model, biological systems, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Alzheimers disease AD, and many neurodegenerative disorders, are multifactorial in nature. They involve a combination of genomic, epigenomic, interactomic and environmental factors. Progress is being made, and these complex diseases are beginning to be understood as having their origin in altered states of biological networks at the cellular level. In the case of AD, genomic susceptibility and mechanisms leading to or accompanying the impairment of the central Amyloid Precursor Protein APP processing and tau networks are widely accepted as major contributors to the diseased state. The derangement of these networks may result in both the gain and loss of functions, increased generation of toxic species e.g., toxic soluble oligomers and aggregates and imbalances, whose effects can propagate to supra -cellular levels. Although well sustained by empirical data and widely accepted, this global perspective often overlooks the essential roles played by the main counteracting homeostatic networks e.g., protein quality control/ proteostasis, unfolded protein response, protein folding chaperone networks, disaggregases, ER-associated degradation/ubiquitin proteasome system, endolysosomal network, autophagy, and other stress-protective and clearance networks, whose relevance to AD is just beginning to be fully realized. In this chapter, an integrative perspective is presented. Alzheimers disease is characterized to be a result of: a intrinsic genomic/epigenomic susceptibility and, b a continued dynamic interplay between the deranged networks and the central homeostatic networks of nerve cells. This interplay of networks will underlie both the onset and rate of progression of the disease in each individual. Integrative Systems Biology approaches are required to effect its elucidation. Comprehensive Systems Biology experiments at different omics levels in simple model organisms, engineered to recapitulate the basic features of AD may illuminate the onset and sequence of events underlying AD. Indeed, studies of models of AD in simple organisms, differentiated cells in culture and rodents are beginning to offer hope that the onset and progression of AD, if detected at an early stage, may be stopped, delayed, or even reversed, by activating or modulating networks involved in proteostasis and the clearance of toxic species. In practice, the incorporation of next generation neuroimaging, high-throughput and computational approaches are opening the way towards early diagnosis well before irreversible cell death. Thus, the presence or co-occurrence of: a accumulation of toxic AP oligomers and tau species; b altered splicing and transcriptome patterns; c impaired redox, proteostatic, and metabolic networks together with, d compromised homeostatic capacities may constitute relevant AD hallmarks at the cellular level towards reliable and early diagnosis. From here, preventive lifestyle changes and tailored therapies may be investigated, such as combined strategies aimed at both lowering the production of toxic species and potentiating homeostatic responses, in order to prevent or delay the onset, and arrest, alleviate, or even reverse the progression of the disease.</div>','<b>Title: </b>Semi-quantitative Modelling of Gene Regulatory Processes with Unknown Parameter Values Using Fuzzy Logic and Petri Nets<br><b>Year: </b>2018<br><b>Author: </b> J. Bordon,  M. Moskon,  N. Zimic and  M. Mraz<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3233/FI-2018-1675"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>FUNDAMENTA INFORMATICAE<br><b>Cluster-Keywords: </b><em>petri net, petri nets, bayesian networks, biological networks, systems biology, regulatory networks, dynamic bayesian, net model, gene regulatory, network data, biological systems, biological data, network analysis, network models, biochemical networks, time series, network structure, signal transduction, net models, dbn model<div style="font-size:11px;"><br><b>Abstract: </b><em>Petri nets are a well-established modelling framework in life sciences and have been widely applied to systems and synthetic biology in recent years. With the various extensions they serve as graphical and simulation interface for both qualitative and quantitative modelling approaches. In terms of quantitative approaches, Stochastic and Continuous Petri nets are extensively used for modelling biological systems dynamics if underlying kinetic data are known. However, these are often only vaguely defined or even missing. In this paper we present a fuzzy approach, which can be used to model biological processes with unknown kinetic data in order to still obtain quantitatively relevant simulation results. We define fuzzy firing rate functions, which can be used in Continuous Petri nets and are able to describe different processes that govern the dynamics of gene expression networks. They can be used in combination with the conventional firing rate functions and applied only in the parts of the system for which the kinetic data are missing. The case study of the proposed approach is performed on models of a hypothetical repressilator and Neurospora circadian rhythm.</div>','<b>Title: </b>Single molecules can operate as primitive biological sensors, switches and oscillators<br><b>Year: </b>2018<br><b>Author: </b> Rd. Hernansaiz-Ballesteros,  L. Cardelli and  A. Csikasz-Nagy<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/s12918-018-0596-4"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Switch-like and oscillatory dynamical systems are widely observed in biology. We investigate the simplest biological switch that is composed of a single molecule that can be autocatalytically converted between two opposing activity forms. We test how this simple network can keep its switching behaviour under perturbations in the system.Results: We show that this molecule can work as a robust bistable system, even for alterations in the reactions that drive the switching between various conformations. We propose that this single molecule system could work as a primitive biological sensor and show by steady state analysis of a mathematical model of the system that it could switch between possible states for changes in environmental signals. Particularly, we show that a single molecule phosphorylation-dephosphorylation switch could work as a nucleotide or energy sensor. We also notice that a given set of reductions in the reaction network can lead to the emergence of oscillatory behaviour.Conclusions: We propose that evolution could have converted this switch into a single molecule oscillator, which could have been used as a primitive timekeeper. We discuss how the structure of the simplest known circadian clock regulatory system, found in cyanobacteria, resembles the proposed single molecule oscillator. Besides, we speculate if such minimal systems could have existed in an RNA world.</div>','<b>Title: </b>Exploiting locational and topological overlap model to identify modules in protein interaction networks<br><b>Year: </b>2019<br><b>Author: </b> Lx. Cheng,  Pf. Liu,  D. Wang and  Ks. Leung<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/s12859-019-2598-7"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>systems biology, regulatory network, interaction networks, network model, gene regulatory, network, network biology, networks, gene network, response networks, network dynamics, dynamic model, cell networks, gene expression, biological networks, system dynamics, dynamic network, model, biological systems, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>BackgroundClustering molecular network is a typical method in system biology, which is effective in predicting protein complexes or functional modules. However, few studies have realized that biological molecules are spatial-temporally regulated to form a dynamic cellular network and only a subset of interactions take place at the same location in cells.ResultsIn this study, considering the subcellular localization of proteins, we first construct a co-localization human protein interaction network PIN and systematically investigate the relationship between subcellular localization and biological functions. After that, we propose a Locational and Topological Overlap Model LTOM to preprocess the co-localization PIN to identify functional modules. LTOM requires the topological overlaps, the common partners shared by two proteins, to be annotated in the same localization as the two proteins. We observed the model has better correspondence with the reference protein complexes and shows more relevance to cancers based on both human and yeast datasets and two clustering algorithms, ClusterONE and MCL.ConclusionTaking into consideration of protein localization and topological overlap can improve the performance of module detection from protein interaction networks.</div>','<b>Title: </b>Robust Optimal Reference-Tracking Design Method for Stochastic Synthetic Biology Systems: T-S Fuzzy Approach<br><b>Year: </b>2010<br><b>Author: </b> Bs. Chen and  Ch. Wu<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1109/TFUZZ.2010.2070842"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IEEE TRANSACTIONS ON FUZZY SYSTEMS<br><b>Cluster-Keywords: </b><em>gene networks, synthetic gene, synthetic biology, regulatory networks, design method, robust synthetic, nonlinear stochastic, molecular noises, gene regulatory, interaction networks, external disturbances, robust stability, robust gene, host cell, gene circuit, cellular network, design specifications, design procedure, signal transduction, parameter fluctuations<div style="font-size:11px;"><br><b>Abstract: </b><em>At present, the development in the nascent field of synthetic gene networks is still difficult. Most newly created gene networks are nonfunctioning due to intrinsic parameter fluctuations, uncertain interactions with unknown molecules and external disturbances of intra and extracellular environments on the host cell. How to design a completely new gene network, that is to track some desired behaviors under these intrinsic and extrinsic disturbances on the host cell, is the most important topic in synthetic biology. In this study, the intrinsic parameter fluctuations, uncertain interactions with unknown molecules and environmental disturbances, are modeled into the nonlinear stochastic systems of synthetic gene networks in vivo. Four design specifications are introduced to guarantee the stochastic synthetic gene network, which can achieve robust optimal tracking of a desired reference model in spite of these intrinsic and extrinsic disturbances on the host cell. However, the robust optimal reference-tracking design problem of nonlinear synthetic gene networks is still hard to solve. In order to simplify the design procedure of the robust optimal nonlinear stochastic-tracking design for synthetic gene networks, the Takagi-Sugeno T-S fuzzy method is introduced to solve the nonlinear stochastic minimum-error-tracking design problem. Hence, the robust optimal reference-tracking design problem under four design specifications can be solved by the linear matrix inequality LMI-constrained optimization method using convex optimization techniques. Further, a simple design procedure is developed for synthetic gene networks to meet the four design specifications to achieve robust optimal reference tracking. Finally, an eigenvalue-shifted design method is also proposed as an expedient scheme to improve the stochastic optimal-tracking design method of synthetic gene oscillators.</div>','<b>Title: </b>Network legos: Building blocks of cellular wiring diagrams<br><b>Year: </b>2008<br><b>Author: </b> Tm. Murali and  Cg. Rivera<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1089/cmb.2007.0139"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF COMPUTATIONAL BIOLOGY<br><b>Cluster-Keywords: </b><em>systems biology, regulatory network, interaction networks, network model, gene regulatory, network, network biology, networks, gene network, response networks, network dynamics, dynamic model, cell networks, gene expression, biological networks, system dynamics, dynamic network, model, biological systems, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Publicly available datasets provide detailed and large-scale information on multiple types of molecular interaction networks in a number of model organisms. The wiring diagrams composed of these interaction networks capture a static view of cellular state. An important challenge in systems biology is obtaining a dynamic perspective on these networks by integrating them with gene expression measurements taken under multiple conditions. We present a top-down computational approach to identify building blocks of molecular interaction networks by: i integrating gene expression measurements for a particular disease state e. g., leukemia or experimental condition e. g., treatment with growth serum with molecular interactions to reveal an active network, which is the network of interactions active in the cell in that disease state or condition; and ii systematically combining active networks computed for different experimental conditions using set-theoretic formulae to reveal network legos, which are modules of coherently interacting genes and gene products in the wiring diagram. We propose efficient methods to compute active networks, systematically mine candidate legos, assess the statistical significance of these candidates, arrange them in a directed acyclic graph DAG, and exploit the structure of the DAG to identify true network legos. We describe methods to assess the stability of our computations to changes in the input and to recover active networks by composing network legos. We analyze two human datasets using our method. A comparison of three leukemias demonstrates how a biologist can use our system to identify specific differences between these diseases. A larger-scale analysis of 13 distinct stresses illustrates our ability to compute the building blocks of the interaction networks activated in response to these stresses. Source code implementing our algorithms is available under version 2 of the GNU General Public License at http://bioinformatics.cs.vt.edu/similar to murali/software/network-lego.</div>','<b>Title: </b>STABILITY OF METABOLIC NETWORKS VIA LINEAR-IN-FLUX-EXPRESSIONS<br><b>Year: </b>2019<br><b>Author: </b> Nj. Merrill,  Zm. An,  St. Mcquade,  F. Garin,  K. Azer,  Re. Abrams and  B. Piccoli<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3934/nhm.2019006"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>NETWORKS AND HETEROGENEOUS MEDIA<br><b>Cluster-Keywords: </b><em>reaction networks, reaction systems, metabolic networks, chemical reaction, network structure, systems biology, biological systems, biological networks, biochemical reaction, reaction rates, differential equations, reaction graph, complex networks, ssi modules, networks, life systems, ordinary differential, cellular networks, reaction, network flows<div style="font-size:11px;"><br><b>Abstract: </b><em>The methodology named LIFE Linear-in-Flux-Expressions was developed with the purpose of simulating and analyzing large metabolic systems. With LIFE, the number of model parameters is reduced by accounting for correlations among the parameters of the system. Perturbation analysis on LIFE systems results in less overall variability of the system, leading to results that more closely resemble empirical data. These systems can be associated to graphs, and characteristics of the graph give insight into the dynamics of the system.This work addresses two main problems: 1. for fixed metabolite levels, find all fluxes for which the metabolite levels are an equilibrium, and 2. for fixed fluxes, find all metabolite levels which are equilibria for the system. We characterize the set of solutions for both problems, and show general results relating stability of systems to the structure of the associated graph. We show that there is a structure of the graph necessary for stable dynamics. Along with these general results, we show how stability analysis from the fields of network flows, compartmental systems, control theory and Markov chains apply to LIFE systems.</div>','<b>Title: </b>Cell differentiation processes as spatial networks: Identifying four-dimensional structure in embryogenesis<br><b>Year: </b>2018<br><b>Author: </b> B. Alicea and  R. Gordon<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.biosystems.2018.09.009"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOSYSTEMS<br><b>Cluster-Keywords: </b><em>complex networks, complex systems, network theory, network structure, biological network, social network, network science, dynamic networks, network systems, network model, network topology, systems biology, network, network analysis, system dynamics, networks, realworld networks, network automata, epidemic network, model network<div style="font-size:11px;"><br><b>Abstract: </b><em>One overarching principle of eukaroytic development is the generative spatial emergence and self-organization of cell populations. As cells divide and differentiate, they and their descendents form a spatiotemporal explicit and increasingly compartmentalized complex system. Yet despite this comparmentalization, there is selective functional overlap between these structural components. While contemporary tools such as lineage trees and molecular signaling networks prvide a window into this complexity, they do not characterize embryogenesis as a global process. Using a four-dimensional spatial representation, major features of the developmental process are revealed. To establish the role of developmental mechanisms that turn a spherical embryo into a highly asymmetrical adult phenotype, we can map the outcomes of the cell division process to a complex network model. This representational model provides information about the top-down mechanisms relevant to the differentiation process. In a complementary manner, looking for phenomena such as superdiffusive positioning and sublineage-based anatomical clustering incorporates dynamic information to our parallel view of embryogenesis. Characterizing the spatial organization and geometry of embryos in this way allows for novel indicators of developmental patterns both within and between organisms.</div>','<b>Title: </b>A geometrical approach to control and controllability of nonlinear dynamical networks<br><b>Year: </b>2016<br><b>Author: </b> Lz. Wang,  Rq. Su,  Zg. Huang,  X. Wang,  Wx. Wang,  C. Grebogi and  Yc. Lai<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1038/ncomms11323"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>NATURE COMMUNICATIONS<br><b>Cluster-Keywords: </b><em>gene regulatory, regulatory network, systems biology, cell differentiation, cell fate, model systems, potential landscape, network grn, dynamical systems, melanocyte differentiation, network models, gene expression, network dynamics, differentiation genes, dynamical models, dynamical networks, cellular differentiation, stem cells, attractor network, developmental gene<div style="font-size:11px;"><br><b>Abstract: </b><em>In spite of the recent interest and advances in linear controllability of complex networks, controlling nonlinear network dynamics remains an outstanding problem. Here we develop an experimentally feasible control framework for nonlinear dynamical networks that exhibit multistability. The control objective is to apply parameter perturbation to drive the system from one attractor to another, assuming that the former is undesired and the latter is desired. To make our framework practically meaningful, we consider restricted parameter perturbation by imposing two constraints: it must be experimentally realizable and applied only temporarily. We introduce the concept of attractor network, which allows us to formulate a quantifiable controllability framework for nonlinear dynamical networks: a network is more controllable if the attractor network is more strongly connected. We test our control framework using examples from various models of experimental gene regulatory networks and demonstrate the beneficial role of noise in facilitating control.</div>','<b>Title: </b>Deciphering deterioration mechanisms of complex diseases based on the construction of dynamic networks and systems analysis<br><b>Year: </b>2015<br><b>Author: </b> Yy. Li,  Sq. Jin,  L. Lei,  Zs. Pan and  Xf. Zou<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1038/srep09283"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SCIENTIFIC REPORTS<br><b>Cluster-Keywords: </b><em>systems biology, dynamic network, early differentiation, complex diseases, stem cells, erythroid cells, system dynamics, network biomarkers, immune system, tcm syndrome, dnb genes, gene expression, dnb model, stem cell, expression dynamics, model system, mathematical model, model dynamics, dynamic model, sensitivity analysis<div style="font-size:11px;"><br><b>Abstract: </b><em>The early diagnosis and investigation of the pathogenic mechanisms of complex diseases are the most challenging problems in the fields of biology and medicine. Network-based systems biology is an important technique for the study of complex diseases. The present study constructed dynamic protein-protein interaction PPI networks to identify dynamical network biomarkers DNBs and analyze the underlying mechanisms of complex diseases from a systems level. We developed a model-based framework for the construction of a series of time-sequenced networks by integrating high-throughput gene expression data into PPI data. By combining the dynamic networks and molecular modules, we identified significant DNBs for four complex diseases, including influenza caused by either H3N2 or H1N1, acute lung injury and type 2 diabetes mellitus, which can serve as warning signals for disease deterioration. Function and pathway analyses revealed that the identified DNBs were significantly enriched during key events in early disease development. Correlation and information flow analyses revealed that DNBs effectively discriminated between different disease processes and that dysfunctional regulation and disproportional information flow may contribute to the increased disease severity. This study provides a general paradigm for revealing the deterioration mechanisms of complex diseases and offers new insights into their early diagnoses.</div>','<b>Title: </b>Dynamical robustness of biological networks with hierarchical distribution of time scales<br><b>Year: </b>2007<br><b>Author: </b> An. Gorban and  O. Radulescu<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1049/iet-syb:20060083"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IET SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>dna damage, signalling networks, network dynamics, pmdm system, systems biology, systems dynamics, biological systems, single cell, kinase inhibitors, signaling network, biological networks, cellular automata, cell cycle, oscillation system, temporal dynamics, signaling pathway, individual cells, signaling processes, dynamic model, damage response<div style="font-size:11px;"><br><b>Abstract: </b><em>Concepts of distributed robustness and r-robustness proposed by biologists to explain a variety of stability phenomena in molecular biology are analysed. Then, the robustness of the relaxation time using a chemical reaction description of genetic and signalling networks is discussed. First, the following result for linear networks is obtained: for large multiscale systems with hierarchical distribution of time scales, the variance of the inverse relaxation time as well as the variance of the stationary rate is much lower than the variance of the separate constants. Moreover, it can tend to 0 faster than 1/n, where n is the number of reactions. Similar phenomena are valid in the nonlinear case as well. As a numerical illustration, a model of signalling network is used for the important transcription factor NF kappa B.</div>','<b>Title: </b>Inflammation Following Traumatic Brain Injury in Humans: Insights from Data-Driven and Mechanistic Models into Survival and Death<br><b>Year: </b>2016<br><b>Author: </b> A. Abboud,  Q. Mi,  A. Puccio,  D. Okonkwo,  M. Bulige,  G. Constantine and  Y. Vodovotz<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3389/fphar.2016.00342"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>FRONTIERS IN PHARMACOLOGY<br><b>Cluster-Keywords: </b><em>inflammatory response, network model, mechanistic models, lde model, injured patients, critically injured, prediction model, tbi survivors, computational modeling, model abm, impaired astrocytes, model predicts, metabolic dynamics, significant differences, twoprocess model, agent-based model, coupled model, model, inflammatory network, computational models<div style="font-size:11px;"><br><b>Abstract: </b><em>Inflammation induced by traumatic brain injury TBI is a complex mediator of morbidity and mortality. We have previously demonstrated the utility of both data-driven and mechanistic models in settings of traumatic injury. We hypothesized that differential dynamic inflammation programs characterize TBI survivors vs. non-survivors, and sought to leverage computational modeling to derive novel insights into this life/death bifurcation. Thirteen inflammatory cytokines and chemokines were determined using Luminex TM in serial cerebrospinal fluid CSF samples from 31 TBI patients over 5 days. In this cohort, 5 were non-survivors Glasgow Outcome Scale EGOS score = 1 and 26 were survivors GOS > 1. A Pearson correlation analysis of initial injury Glasgow Coma Scale GCS vs. GOS suggested that survivors and non survivors had distinct clinical response trajectories to injury. Statistically significant differences in interleukin IL-4, IL-5, IL-6, IL-8, IL-13, and tumor necrosis factor-alpha INF-alpha were observed between TBI survivors vs. non-survivors over 5 days. Principal Component Analysis and Dynamic Bayesian Network inference suggested differential roles of chemokines, INF-alpha, IL-6, and IL-10, based upon which an ordinary differential equation model of TBI was generated. This model was calibrated separately to the time course data of TBI survivors vs. non-survivors as a function of initial GCS. Analysis of parameter values in ensembles of simulations from these models suggested differences in microglial and damage responses in TBI survivors vs. non-survivors. These studies suggest the utility of combined data-driven and mechanistic models in the context of human TBI.</div>','<b>Title: </b>Modeling gene regulatory network motifs using statecharts<br><b>Year: </b>2012<br><b>Author: </b> F. Fioravanti,  M. Helmer-Citterich and  E. Nardelli<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1471-2105-13-S4-S20"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>network motifs, regulatory network, gene regulatory, systems biology, gene network, bifan motif, complex systems, model comparison, network modeling, complex gene, biological systems, dynamic systems, interaction network, feedforward loop, modeling gene, intelligent systems, modeling approach, biological pathways, motif model, model motifs<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Gene regulatory networks are widely used by biologists to describe the interactions among genes, proteins and other components at the intra-cellular level. Recently, a great effort has been devoted to give gene regulatory networks a formal semantics based on existing computational frameworks. For this purpose, we consider Statecharts, which are a modular, hierarchical and executable formal model widely used to represent software systems. We use Statecharts for modeling small and recurring patterns of interactions in gene regulatory networks, called motifs.Results: We present an improved method for modeling gene regulatory network motifs using Statecharts and we describe the successful modeling of several motifs, including those which could not be modeled or whose models could not be distinguished using the method of a previous proposal. We model motifs in an easy and intuitive way by taking advantage of the visual features of Statecharts. Our modeling approach is able to simulate some interesting temporal properties of gene regulatory network motifs: the delay in the activation and the deactivation of the "output" gene in the coherent type-1 feedforward loop, the pulse in the incoherent type-1 feedforward loop, the bistability nature of double positive and double negative feedback loops, the oscillatory behavior of the negative feedback loop, and the "lock-in" effect of positive autoregulation.Conclusions: We present a Statecharts-based approach for the modeling of gene regulatory network motifs in biological systems. The basic motifs used to build more complex networks that is, simple regulation, reciprocal regulation, feedback loop, feedforward loop, and autoregulation can be faithfully described and their temporal dynamics can be analyzed.</div>','<b>Title: </b>Relationships between actin regulatory mechanisms and measurable state variables<br><b>Year: </b>2007<br><b>Author: </b> M. Bindschadler and  Jl. Mcgrath<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s10439-007-9267-0"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ANNALS OF BIOMEDICAL ENGINEERING<br><b>Cluster-Keywords: </b><em>interaction network, actin filament, network model, filament network, network structures, actin network, protein interaction, actin dynamics, network, membrane protein, network dynamics, model dynamic, cell, model, network approach, protein complex, computational model, actin, adhesion dynamics, cell dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>In this report we extend our recent mathematical formulation of the actin cycle model Bindschadler et al. Biophys. J. 86 2004 2720 to predict the influence of key regulatory mechanisms on network-scale state variables estimable in live cell experiments. Specifically, we examine the influence of regulation by cofilin, profilin, capping protein and proteins that adjust filament number through nucleation and/or filament severing, on the higher order variables of average filament length, polymer fraction, and filament turnover rate. Importantly, we find that severing/nucleation, the acceleration of ADP-subunit disassembly by cofilin, and the catalytic and shuttle functions of profilin have signature effects on the higher order state variables. In this way, measurement of the state variables in live cells can allow inference of regulatory mechanisms underlying changes in cell state. Our results compare favorably to published data for endothelial cells undergoing a transition from non-motile confluent cells to highly motile subconfluent cells. The extension of our model to higher order state variables allows us to investigate other important issues such as the distinction between basic and higher order measures of filament dynamics, the influence of thymosin beta 4 on network state variables, the interplay between thymosin beta 4 and profilin, and the synergystic effects of cofilin and profilin.</div>','<b>Title: </b>Context-specific flow through the MEK/ERK module produces cell- and ligand-specific patterns of ERK single and double phosphorylation<br><b>Year: </b>2016<br><b>Author: </b> N. Iwamoto,  La. DAlessandro,  S. Depner,  B. Hahn,  Ba. Kramer,  P. Lucarelli,  A. Vlasov,  M. Stepath,  Me. Bohm,  D. Deharde,  G. Damm,  D. Seehofer,  Wd. Lehmann,  U. Klingmuller and  M. Schilling<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1126/scisignal.aab1967"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SCIENCE SIGNALING<br><b>Cluster-Keywords: </b><em>feedback loops, signaling network, positive feedback, network model, negative feedback, biochemical networks, signaling pathways, signal transduction, systems biology, kinase erk, protein kinase, transduction networks, dynamic model, emt network, erk pathway, cell fate, cell death, biochemical reaction, coupled feedback, impulse response<div style="font-size:11px;"><br><b>Abstract: </b><em>The same pathway, such as the mitogen-activated protein kinase MAPK pathway, can produce different cellular responses, depending on stimulus or cell type. We examined the phosphorylation dynamics of the MAPK kinase MEK and its targets extracellular signal-regulated kinase 1 and 2 ERK1/2 in primary hepatocytes and the transformed keratinocyte cell line HaCaT A5 exposed to either hepatocyte growth factor or interleukin-6. By combining quantitative mass spectrometry with dynamic modeling, we elucidated network structures for the reversible threonine and tyrosine phosphorylation of ERK in both cell types. In addition to differences in the phosphorylation and dephosphorylation reactions, the HaCaT network model required two feedback mechanisms, which, as the experimental data suggested, involved the induction of the dual-specificity phosphatase DUSP6 and the scaffold paxillin. We assayed and modeled the accumulation of the double-phosphorylated and active form of ERK1/2, as well as the dynamics of the changes in the monophosphorylated forms of ERK1/2. Modeling the differences in the dynamics of the changes in the distributions of the phosphorylated forms of ERK1/2 suggested that different amounts of MEK activity triggered context-specific responses, with primary hepatocytes favoring the formation of double-phosphorylated ERK1/2 and HaCaT A5 cells that produce both the threonine-phosphorylated and the double-phosphorylated form. These differences in phosphorylation distributions explained the threshold, sensitivity, and saturation of the ERK response. We extended the findings of differential ERK phosphorylation profiles to five additional cultured cell systems and matched liver tumor and normal tissue, which revealed context-specific patterns of the various forms of phosphorylated ERK.</div>','<b>Title: </b>Modeling of Receptor Tyrosine Kinase Signaling: Computational and Experimental Protocols<br><b>Year: </b>2017<br><b>Author: </b> D. Fey,  E. Aksamitiene,  A. Kiyatkin and  Bn. Kholodenko<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/978-1-4939-7154-127"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>KINASE SIGNALING NETWORKS<br><b>Cluster-Keywords: </b><em>signaling network, systems biology, signal transduction, signaling pathways, receptor tyrosine, erbb signaling, erbb receptor, receptor signaling, intracellular signaling, mathematical models, cell signaling, computational model, factor receptor, receptor activation, growth factor, signaling models, networks signaling, receptor system, receptors systems, downstream signaling<div style="font-size:11px;"><br><b>Abstract: </b><em>The advent of systems biology has convincingly demonstrated that the integration of experiments and dynamic modelling is a powerful approach to understand the cellular network biology. Here we present experimental and computational protocols that are necessary for applying this integrative approach to the quantitative studies of receptor tyrosine kinase RTK signaling networks. Signaling by RTKs controls multiple cellular processes, including the regulation of cell survival, motility, proliferation, differentiation, glucose metabolism, and apoptosis. We describe methods of model building and training on experimentally obtained quantitative datasets, as well as experimental methods of obtaining quantitative dose-response and temporal dependencies of protein phosphorylation and activities. The presented methods make possible 1 both the fine-grained modeling of complex signaling dynamics and identification of salient, course-grained network structures such as feedback loops that bring about intricate dynamics, and 2 experimental validation of dynamic models.</div>','<b>Title: </b>Parameter Estimation and Model Selection in Computational Biology<br><b>Year: </b>2010<br><b>Author: </b> G. Lillacci and  M. Khammash<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pcbi.1000696"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS COMPUTATIONAL BIOLOGY<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>A central challenge in computational modeling of biological systems is the determination of the model parameters. Typically, only a fraction of the parameters such as kinetic rate constants are experimentally measured, while the rest are often fitted. The fitting process is usually based on experimental time course measurements of observables, which are used to assign parameter values that minimize some measure of the error between these measurements and the corresponding model prediction. The measurements, which can come from immunoblotting assays, fluorescent markers, etc., tend to be very noisy and taken at a limited number of time points. In this work we present a new approach to the problem of parameter selection of biological models. We show how one can use a dynamic recursive estimator, known as extended Kalman filter, to arrive at estimates of the model parameters. The proposed method follows. First, we use a variation of the Kalman filter that is particularly well suited to biological applications to obtain a first guess for the unknown parameters. Secondly, we employ an a posteriori identifiability test to check the reliability of the estimates. Finally, we solve an optimization problem to refine the first guess in case it should not be accurate enough. The final estimates are guaranteed to be statistically consistent with the measurements. Furthermore, we show how the same tools can be used to discriminate among alternate models of the same biological process. We demonstrate these ideas by applying our methods to two examples, namely a model of the heat shock response in E. coli, and a model of a synthetic gene regulation system. The methods presented are quite general and may be applied to a wide class of biological systems where noisy measurements are used for parameter estimation or model selection.</div>','<b>Title: </b>PERSISTENCE RESULTS FOR CHEMICAL REACTION NETWORKS WITH TIME-DEPENDENT KINETICS AND NO GLOBAL CONSERVATION LAWS<br><b>Year: </b>2011<br><b>Author: </b> D. Angeli,  P. De Leenheer and  Ed. Sontag<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1137/090779401"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SIAM JOURNAL ON APPLIED MATHEMATICS<br><b>Cluster-Keywords: </b><em>reaction networks, reaction systems, metabolic networks, chemical reaction, network structure, systems biology, biological systems, biological networks, biochemical reaction, reaction rates, differential equations, reaction graph, complex networks, ssi modules, networks, life systems, ordinary differential, cellular networks, reaction, network flows<div style="font-size:11px;"><br><b>Abstract: </b><em>New checkable criteria for persistence of chemical reaction networks are proposed, which extend and complement existing ones. The new results allow the consideration of reaction rates which are time-varying, thus incorporating the effects of external signals, and also relax the assumption of existence of global conservation laws, thus allowing for inflows  production and outflows degradation. For time-invariant networks, parameter-dependent conditions for persistence of certain classes of networks are provided. As an illustration, two networks arising in the systems biology literature are analyzed, namely a hypoxia network and an apoptosis network.</div>','<b>Title: </b>Quantifying dynamic mechanisms of auto-regulation in Escherichia coli with synthetic promoter in response to varying external phosphate levels<br><b>Year: </b>2019<br><b>Author: </b> C. Uluseker,  J. Torres-Bacete,  Jl. Garcia,  Mm. Hanczyc,  J. Nogales and  O. Kahramanogullari<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1038/s41598-018-38223-w"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SCIENTIFIC REPORTS<br><b>Cluster-Keywords: </b><em>gene regulatory, systems biology, synthetic biology, regulatory network, gene network, gene expression, system model, synthetic network, network dynamics, systems theory, synthetic gene, genetic network, model predicts, dynamical systems, time delays, translational time, regulation models, oscillatory expression, network structure, biological systems<div style="font-size:11px;"><br><b>Abstract: </b><em>Escherichia coli have developed one of the most efficient regulatory response mechanisms to phosphate starvation. The machinery involves a cascade with a two-component system TCS that relays the external signal to the genetic circuit, resulting in a feedback response. Achieving a quantitative understanding of this system has implications in synthetic biology and biotechnology, for example, in applications for wastewater treatment. To this aim, we present a computational model and experimental results with a detailed description of the TCS, consisting of PhoR and PhoB, together with the mechanisms of gene expression. The model is parameterised within the feasible range, and fitted to the dynamic response of our experimental data on PhoB as well as PhoA, the product of this network that is used in alkaline phosphatase production. Deterministic and stochastic simulations with our model predict the regulation dynamics in higher external phosphate concentrations while reproducing the experimental observations. In a cycle of simulations and experimental verification, our model predicts and explores phenotypes with various synthetic promoter designs that can optimise the inorganic phosphate intake in E. coli. Sensitivity analysis demonstrates that the Pho-controlled genes have a significant influence over the phosphate response. Together with experimental findings, our model should thus provide insights for the investigations on engineering new sensors and regulators for living technologies.</div>','<b>Title: </b>Reverse engineering gene regulatory networks: Coupling an optimization algorithm with a parameter identification technique<br><b>Year: </b>2014<br><b>Author: </b> Yt. Hsiao and  Wp. Lee<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1471-2105-15-S15-S8"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>regulatory networks, gene regulatory, gene expression, network inference, expression data, network structure, gene networks, networks grns, regulatory genes, boolean networks, bayesian network, network reconstruction, networks gene, network dynamics, gene regulation, inference method, systems biology, time series, infer gene, network<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: To infer gene regulatory networks from time series gene profiles, two important tasks that are related to biological systems must be undertaken. One task is to determine a valid network structure that has topological properties that can influence the network dynamics profoundly. The other task is to optimize the network parameters to minimize the accumulated discrepancy between the gene expression data and the values produced by the inferred network model. Though the above two tasks must be conducted simultaneously, most existing work addresses only one of the tasks.Results: We propose an iterative approach that couples parameter identification and parameter optimization techniques, to address the two tasks simultaneously during network inference. This approach first identifies the most influential parameters against internal perturbations; this identification is based on sensitivity measurements. Then, a hybrid GA-PSO optimization method infers parameters in accordance with their criticalities. The proposed approach has been applied to several datasets, including subsets of the SOS DNA repair system in E. coli, the Rat central nervous system CNS, and the protein glycosylation system of yeast S. cerevisiae. The result and analysis show that our approach can infer solutions to satisfy both the requirements of network structure and network behavior.Conclusions: Network structure is an important though challenging issue to address in inferring sophisticated networks with biological details. In need of prior structural knowledge, we turn to measure parameter sensitivity instead to account for the network structure in an indirect way. By developing an integrated approach for considering both the network structure and behavior in the inference process, we can successfully infer critical gene interactions as well as valid time expression profiles.</div>','<b>Title: </b>Switch-like Transitions Insulate Network Motifs to Modularize Biological Networks<br><b>Year: </b>2016<br><b>Author: </b> O. Atay,  A. Doncic and  Jm. Skotheim<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.cels.2016.06.010"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>CELL SYSTEMS<br><b>Cluster-Keywords: </b><em>blood flow, blood cells, flow rate, blood pumps, cells, cell, results, patients, stem cells, methods, blood, patients blood, flow, patients methods, patients objectives, cell viability, objectives, results cells, blood pressure, cell culture<div style="font-size:11px;"><br><b>Abstract: </b><em>Cellular decisions are made by complex networks that are difficult to analyze. Although it is common to analyze smaller sub-networks known as network motifs, it is unclear whether this is valid, because these motifs are embedded in complex larger networks. Here, we address the general question of modularity by examining the S. cerevisiae pheromone response. We demonstrate that the feedforward motif controlling the cell-cycle inhibitor Far1 is insulated from cell-cycle dynamics by the positive feedback switch that drives reentry to the cell cycle. Before cells switch on positive feedback, the feedforward motif model predicts the behavior of the larger network. Conversely, after the switch, the feedforward motif is dismantled and has no discernable effect on the cell cycle. When insulation is broken, the feedforward motif no longer predicts network behavior. This work illustrates how, despite the inter-connectivity of networks, the activity of motifs can be insulated by switches that generate well-defined cellular states.</div>','<b>Title: </b>Equilibria and control of metabolic networks with enhancers and inhibitors<br><b>Year: </b>2019<br><b>Author: </b> Zm. An,  N. Merrill,  St. Mcquade and  B. Piccoli<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3934/mine.2019.3.648"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MATHEMATICS IN ENGINEERING<br><b>Cluster-Keywords: </b><em>reaction networks, reaction systems, metabolic networks, chemical reaction, network structure, systems biology, biological systems, biological networks, biochemical reaction, reaction rates, differential equations, reaction graph, complex networks, ssi modules, networks, life systems, ordinary differential, cellular networks, reaction, network flows<div style="font-size:11px;"><br><b>Abstract: </b><em>Linear-In-Flux-Expressions briefly LIFE methodology models metabolism by using correlations among fluxes of metabolic networks and reducing the number of model parameters. These correlations are calculated for an equilibrium state and developed to include tools from the fields of network flows, compartmental systems, Markov chains, and control theory. LIFE methodology was developed with pharmacology simulators in mind, and the present study advances this goal, by focusing on the control of metabolic networks and inclusion of enhancers and inhibitors. We consider two control problems on metabolic networks: 1. The optimization of intakes from the outside environment to drive the system to a desired state, and 2. The inclusion of inhibitors and enhancers and their optimization. Simulations are included to test the approach on these more complex networks.</div>','<b>Title: </b>Stochastic modelling of the eukaryotic heat shock response<br><b>Year: </b>2010<br><b>Author: </b> A. Mizera and  B. Gambin<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.jtbi.2010.04.029"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF THEORETICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>regulatory networks, stochastic simulation, gene networks, reaction networks, systems biology, stochastic model, network models, complex systems, networks stochastic, gene regulatory, biological systems, stochastic dynamics, differential equations, protein networks, cell cycle, continuous model, interaction networks, systems chemistry, time delays, biochemical reaction<div style="font-size:11px;"><br><b>Abstract: </b><em>The heat shock response HSR is a highly evolutionarily conserved defence mechanism allowing the cell to promptly react to elevated temperature conditions and other forms of stress. It has been subject to intense research for at least two main reasons. First, it is considered a promising candidate for deciphering the engineering principles underlying regulatory networks. Second, heat shock proteins main actors of the HSR play crucial role in many fundamental cellular processes. Therefore, profound understanding of the heat shock response would have far-reaching ramifications for the cell biology.Recently, a new deterministic model of the eukaryotic heat shock response has been proposed in the literature. It is very attractive since it consists of only the minimum number of components required by any functional regulatory network, while yet being capable of biological validation. However, it admits small molecule populations of some of the considered metabolites. In this paper a stochastic model corresponding to the deterministic one is constructed and the outcomes of these two models are confronted. The aim with this comparison is to show that, in the case of the heat shock response, the approximation of a discrete system with a continuous model is a reasonable approach. This is not always the truth, especially when the numbers of molecules of the considered species are small. By making the effort of performing and analysing 1000 stochastic simulations, we investigate the range of behaviour the stochastic model is likely to exhibit. We demonstrate that the obtained results agree well with the dynamics displayed by the continuous model, which strengthens the trust in the deterministic description. A proof of the existence and uniqueness of the stationary distribution of the Markov chain underlying the stochastic model is given. Moreover, the obtained view of the stochastic dynamics and the performed comparison to the outcome of the continuous formulation provide more insight into the dynamics of the heat shock response mechanism. C 2010 Elsevier Ltd. All rights reserved.</div>','<b>Title: </b>Mapping global sensitivity of cellular network dynamics: sensitivity heat maps and a global summation law<br><b>Year: </b>2008<br><b>Author: </b> Da. Rand<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1098/rsif.2008.0084.focus"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF THE ROYAL SOCIETY INTERFACE<br><b>Cluster-Keywords: </b><em>sensitivity analysis, systems biology, robustness analysis, network model, network inference, regulatory network, systems analysis, gene expression, parametric uncertainty, biological networks, mathematical models, biochemical networks, biological functions, cellular networks, biological system, parametric sensitivity, dynamical models, dynamic system, networks analysis, gene regulatory<div style="font-size:11px;"><br><b>Abstract: </b><em>The dynamical systems arising from gene regulatory, signalling and metabolic networks are strongly nonlinear, have high-dimensional state spaces and depend on large numbers of parameters. Understanding the relation between the structure and the function for such systems is a considerable challenge. We need tools to identify key points of regulation, illuminate such issues as robustness and control and aid in the design of experiments. Here, I tackle this by developing new techniques for sensitivity analysis. In particular, I show how to globally analyse the sensitivity of a complex system by means of two new graphical objects: the sensitivity heat map and the parameter sensitivity spectrum. The approach to sensitivity analysis is global in the sense that it studies the variation in the whole of the models solution rather than focusing on output variables one at a time, as in classical sensitivity analysis. This viewpoint relies on the discovery of local geometric rigidity for such systems, the mathematical insight that makes a practicable approach to such problems feasible for highly complex systems. In addition, we demonstrate a new summation theorem that substantially generalizes previous results for oscillatory and other dynamical phenomena. This theorem can be interpreted as a mathematical law stating the need for a balance between fragility and robustness in such systems.</div>','<b>Title: </b>An energetic reformulation of kinetic rate laws enables scalable parameter estimation for biochemical networks<br><b>Year: </b>2019<br><b>Author: </b> Jc. Mason and  Mw. Covert<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.jtbi.2018.10.041"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF THEORETICAL BIOLOGY<br><b>Cluster-Keywords: </b><em>systems biology, network models, signaling network, parameter estimation, dynamic models, biological systems, mathematical models, metabolic network, parameter sensitivities, computational model, gene circuit, circuit models, experimental data, metabolic models, model system, scalable parameter, model parameters, parameter space, biomolecular networks, model<div style="font-size:11px;"><br><b>Abstract: </b><em>The technology for building functionally complete or whole-cell biological simulations is rapidly developing. However, the predictive capabilities of these simulations are hindered by the availability of parameter values, which are often difficult or even impossible to obtain experimentally and must therefore be estimated. Using E. colis glycolytic network as a model system, we describe and apply a new method which can estimate the values of all the systems 102 parameters - fit to observations from studies of proteomics, metabolomics, enzyme kinetics and chemical energetics - and find that the resulting metabolic models are not only well-fit, but also dynamically stable. An analysis of how well parameter values in the network were determined by the training data revealed that over 80 of the parameter values were not well-specified. Moreover, the distribution of well-determined values was biased to a specific part of the network and against certain types of experimental data. Our results also suggest that perturbing the functional, energetic space of parameters rather than traditional metabolic parameters is a superior strategy for exploring the space of biological dynamics. The estimated parameter values matched both training data and previously withheld validation data within an order of magnitude for over 85 of the data points; notably, the area of greatest frustration in the network was also the most fully determined. Finally, our estimation method showed that fidelity to physiological observations such as network response time is enforced at the cost of fit to molecular parameter values. In summary, our reformulation enables estimation of accurate, biologically relevant parameters, generates insight into the biology of the simulated network, and appears generalizable to any biochemical network - potentially including whole-cell models. C 2018 The Authors. Published by Elsevier Ltd.</div>','<b>Title: </b>Prediction of Key Factor Controlling G1/S Phase in the Mammalian Cell Cycle Using System Analysis<br><b>Year: </b>2008<br><b>Author: </b> Y. Tashima,  H. Hamada,  M. Okamoto and  T. Hanai<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1263/jbb.106.368"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF BIOSCIENCE AND BIOENGINEERING<br><b>Cluster-Keywords: </b><em>cell cycle, mammalian cell, cdk network, cell proliferation, cell, cells cell, regulatory network, detailed model, cell size, molecular network, skeleton model, cycle arrest, cycle progression, cell division, cell density, kinases cdks, cycle, phase transition, cycle control, network driving<div style="font-size:11px;"><br><b>Abstract: </b><em>Control of the G1/S phase transition of the cell cycle contributes to the maintenance of homeostasis. It is well known that the disruption of the cell cycle is related to cell transformation and carcinogenesis. The G1/S phase transition involves a network of components that includes cyclins, cyclin-dependent kinases, and other proteins. Numerical simulation techniques and system analysis are expected to become powerful tools for investigating complex biological networks. To reach this goal, we designed a mathematical model of the G1/S phase transition. Using our model, we conducted a numerical simulation and comprehensive system analyses of this phase of the cell cycle. In this way, we were able to predict the key factors involved in the control of the G1/S transition.</div>','<b>Title: </b>Initial state perturbations as a validation method for data-driven fuzzy models of cellular networks<br><b>Year: </b>2018<br><b>Author: </b> L. Magdevska,  M. Mraz,  N. Zimic and  M. Moskon<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/s12859-018-2366-0"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>petri net, petri nets, bayesian networks, biological networks, systems biology, regulatory networks, dynamic bayesian, net model, gene regulatory, network data, biological systems, biological data, network analysis, network models, biochemical networks, time series, network structure, signal transduction, net models, dbn model<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Data-driven methods that automatically learn relations between attributes from given data are a popular tool for building mathematical models in computational biology. Since measurements are prone to errors, approaches dealing with uncertain data are especially suitable for this task. Fuzzy models are one such approach, but they contain a large amount of parameters and are thus susceptible to over-fitting. Validation methods that help detect over-fitting are therefore needed to eliminate inaccurate models.Results: We propose a method to enlarge the validation datasets on which a fuzzy dynamic model of a cellular network can be tested. We apply our method to two data-driven dynamic models of the MAPK signalling pathway and two models of the mammalian circadian clock. We show that random initial state perturbations can drastically increase the mean error of predictions of an inaccurate computational model, while keeping errors of predictions of accurate models small.Conclusions: With the improvement of validation methods, fuzzy models are becoming more accurate and are thus likely to gain new applications. This field of research is promising not only because fuzzy models can cope with uncertainty, but also because their run time is short compared to conventional modelling methods that are nowadays used in systems biology.</div>','<b>Title: </b>A skeleton model for the network of cyclin-dependent kinases driving the mammalian cell cycle<br><b>Year: </b>2011<br><b>Author: </b> C. Gerard and  A. Goldbeter<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1098/rsfs.2010.0008"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>INTERFACE FOCUS<br><b>Cluster-Keywords: </b><em>cell cycle, mammalian cell, cdk network, cell proliferation, cell, cells cell, regulatory network, detailed model, cell size, molecular network, skeleton model, cycle arrest, cycle progression, cell division, cell density, kinases cdks, cycle, phase transition, cycle control, network driving<div style="font-size:11px;"><br><b>Abstract: </b><em>We previously proposed a detailed, 39-variable model for the network of cyclin-dependent kinases Cdks that controls progression along the successive phases of the mammalian cell cycle. Here, we propose a skeleton, 5-variable model for the Cdk network that can be seen as the backbone of the more detailed model for the mammalian cell cycle. In the presence of sufficient amounts of growth factor, the skeleton model also passes from a stable steady state to sustained oscillations of the various cyclin/Cdk complexes. This transition corresponds to the switch from quiescence to cell proliferation. Sequential activation of the cyclin/Cdk complexes allows the ordered progression along the G1, S, G2 and M phases of the cell cycle. The 5-variable model can also account for the existence of a restriction point in G1, and for endoreplication. Like the detailed model, it contains multiple oscillatory circuits and can display complex oscillatory behaviour such as quasi-periodic oscillations and chaos. We compare the dynamical properties of the skeleton model with those of the more detailed model for the mammalian cell cycle.</div>','<b>Title: </b>Improving BDD-based attractor detection for synchronous Boolean networks<br><b>Year: </b>2016<br><b>Author: </b> Qx. Yuan,  Hy. Qu,  J. Pang and  A. Mizera<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1007/s11432-016-5594-9"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>SCIENCE CHINA-INFORMATION SCIENCES<br><b>Cluster-Keywords: </b><em>boolean network, boolean functions, regulatory networks, canalizing functions, network models, gene regulatory, boolean control, control networks, biological networks, boolean models, update functions, interaction networks, large network, regulatory functions, canalizing boolean, genetic regulatory, systems biology, driver nodes, networks bcns, biological systems<div style="font-size:11px;"><br><b>Abstract: </b><em>Boolean networks are an important formalism for modelling biological systems and have attracted much attention in recent years. An important challenge in Boolean networks is to exhaustively find attractors, which represent steady states of a biological network. In this paper, we propose a new approach to improve the efficiency of BDD-based attractor detection. Our approach includes a monolithic algorithm for small networks, an enumerative strategy to deal with large networks, a method to accelerate attractor detection based on an analysis of the network structure, and two heuristics on ordering BDD variables. We demonstrate the performance of our approach on a number of examples and on a realistic model of apoptosis in hepatocytes. We compare it with one existing technique in the literature.</div>','<b>Title: </b>Robust H-infinity-stabilization design in gene networks under Stochastic molecular noises: Fuzzy-interpolation approach<br><b>Year: </b>2008<br><b>Author: </b> Bs. Chen,  Yt. Chang and  Yc. Wang<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1109/TSMCB.2007.906975"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IEEE TRANSACTIONS ON SYSTEMS MAN AND CYBERNETICS PART B-CYBERNETICS<br><b>Cluster-Keywords: </b><em>gene networks, synthetic gene, synthetic biology, regulatory networks, design method, robust synthetic, nonlinear stochastic, molecular noises, gene regulatory, interaction networks, external disturbances, robust stability, robust gene, host cell, gene circuit, cellular network, design specifications, design procedure, signal transduction, parameter fluctuations<div style="font-size:11px;"><br><b>Abstract: </b><em>Molecular noises in gene networks come from intrinsic fluctuations, transmitted noise from upstream genes, and the global noise affecting all genes. Knowledge of molecular noise filtering in gene networks is crucial to understand the signal processing in gene networks and to design noise-tolerant gene circuits for synthetic biology. A nonlinear stochastic dynamic model is proposed in describing a gene network under intrinsic molecular fluctuations and extrinsic molecular noises. The stochastic molecular-noise-processing scheme of gene regulatory networks for attenuating these molecular noises is investigated from the nonlinear robust stabilization and filtering perspective. In order to improve the robust stability and noise filtering, a robust gene circuit design for gene networks is proposed based on the nonlinear robust H. stochastic stabilization and filtering scheme, which needs to solve a nonlinear Hamilton-Jacobi inequality. However, in order to avoid solving these complicated nonlinear stabilization and filtering problems, a fuzzy approximation method is employed to interpolate several linear stochastic gene networks at different operation points via fuzzy bases to approximate the nonlinear stochastic gene network. In this situation, the method of linear matrix inequality technique could be employed to simplify the gene circuit design problems to improve robust stability and molecular-noise-filtering ability of gene networks to overcome intrinsic molecular fluctuations and extrinsic molecular noises.</div>','<b>Title: </b>Cerebral: Visualizing Multiple Experimental Conditions on a Graph with Biological Context<br><b>Year: </b>2008<br><b>Author: </b> A. Barsky,  T. Munzner,  J. Gardy and  R. Kincaid<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1109/TVCG.2008.117"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IEEE TRANSACTIONS ON VISUALIZATION AND COMPUTER GRAPHICS<br><b>Cluster-Keywords: </b><em>gene regulatory, systems biology, regulatory networks, gene network, stem cell, network model, transcription factors, gene expression, cell transcriptomes, network analysis, data analysis, stem cells, network grnsight, cell transformation, weighted network, gene ontology, networks grns, graph model, graph layout, endothelial cell<div style="font-size:11px;"><br><b>Abstract: </b><em>Systems biologists use interaction graphs to model the behavior of biological systems at the molecular level. In an iterative process, such biologists observe the reactions of living cells under various experimental conditions, view the results in the context of the interaction graph, and then propose changes to the graph model. These graphs serve as a form of dynamic knowledge representation of the biological system being studied and evolve as new insight is gained from the experimental data. While numerous graph layout and drawing packages are available, these tools did not fully meet the needs of our immunologist collaborators. In this paper, we describe the data information display needs of these immunologists and translate them into design decisions. These decisions led us to create Cerebral, a system that uses a biologically guided graph layout and incorporates experimental data directly into the graph display. Small multiple views of different experimental conditions and a data-driven parallel coordinates view enable correlations between experimental conditions to be analyzed at the same time that the data is viewed in the graph context. This combination of coordinated views allows the biologist to view the data from many different perspectives simultaneously. To illustrate the typical analysis tasks performed, we analyze two datasets using Cerebral. Based on feedback from our collaborators we conclude that Cerebral is a valuable tool for analyzing experimental data in the context of an interaction graph model.</div>','<b>Title: </b>Multi-objective optimization framework to obtain model-based guidelines for tuning biological synthetic devices: an adaptive network case<br><b>Year: </b>2016<br><b>Author: </b> Y. Boada,  G. Reynoso-Meza,  J. Pico and  A. Vignoni<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/s12918-016-0269-0"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>gene circuits, synthetic biology, circuit behavior, biological parts, synthetic gene, circuit design, genetic circuits, cell types, metabolite dynamics, mathematical model, designed circuit, control circuit, biological systems, circuit models, synthetic circuit, feedback circuits, biological circuits, design models, metabolic pathways, circuit parameters<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Model based design plays a fundamental role in synthetic biology. Exploiting modularity, i.e. using biological parts and interconnecting them to build new and more complex biological circuits is one of the key issues. In this context, mathematical models have been used to generate predictions of the behavior of the designed device. Designers not only want the ability to predict the circuit behavior once all its components have been determined, but also to help on the design and selection of its biological parts, i.e. to provide guidelines for the experimental implementation. This is tantamount to obtaining proper values of the model parameters, for the circuit behavior results from the interplay between model structure and parameters tuning. However, determining crisp values for parameters of the involved parts is not a realistic approach. Uncertainty is ubiquitous to biology, and the characterization of biological parts is not exempt from it. Moreover, the desired dynamical behavior for the designed circuit usually results from a trade-off among several goals to be optimized.Results: We propose the use of a multi-objective optimization tuning framework to get a model-based set of guidelines for the selection of the kinetic parameters required to build a biological device with desired behavior. The design criteria are encoded in the formulation of the objectives and optimization problem itself. As a result, on the one hand the designer obtains qualitative regions/intervals of values of the circuit parameters giving rise to the predefined circuit behavior; on the other hand, he obtains useful information for its guidance in the implementation process. These parameters are chosen so that they can effectively be tuned at the wet-lab, i.e. they are effective biological tuning knobs. To show the proposed approach, the methodology is applied to the design of a well known biological circuit: a genetic incoherent feed-forward circuit showing adaptive behavior.Conclusion: The proposed multi-objective optimization design framework is able to provide effective guidelines to tune biological parameters so as to achieve a desired circuit behavior. Moreover, it is easy to analyze the impact of the context on the synthetic device to be designed. That is, one can analyze how the presence of a downstream load influences the performance of the designed circuit, and take it into account.</div>','<b>Title: </b>Quantitative multivariate analysis of dynamic multicellular morphogenic trajectories<br><b>Year: </b>2015<br><b>Author: </b> De. White,  Jb. Sylvester,  Tj. Levario,  H. Lu,  Jt. Streelman,  Tc. Mcdevitt and  Ml. Kemp<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1039/c5ib00072f"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>INTEGRATIVE BIOLOGY<br><b>Cluster-Keywords: </b><em>stem cell, stem cells, systems biology, mathematical models, cell biology, network model, signaling pathways, cell fate, computational models, cancer cells, gene expression, network dynamics, embryonic stem, signaling network, prostate cancer, mathematical modeling, psychological stress, apoptosis signaling, computational systems, molecular mechanisms<div style="font-size:11px;"><br><b>Abstract: </b><em>Interrogating fundamental cell biology principles that govern tissue morphogenesis is critical to better understanding of developmental biology and engineering novel multicellular systems. Recently, functional micro-tissues derived from pluripotent embryonic stem cell ESC aggregates have provided novel platforms for experimental investigation; however elucidating the factors directing emergent spatial phenotypic patterns remains a significant challenge. Computational modelling techniques offer a unique complementary approach to probe mechanisms regulating morphogenic processes and provide a wealth of spatio-temporal data, but quantitative analysis of simulations and comparison to experimental data is extremely difficult. Quantitative descriptions of spatial phenomena across multiple systems and scales would enable unprecedented comparisons of computational simulations with experimental systems, thereby leveraging the inherent power of computational methods to interrogate the mechanisms governing emergent properties of multicellular biology. To address these challenges, we developed a portable pattern recognition pipeline consisting of: the conversion of cellular images into networks, extraction of novel features via network analysis, and generation of morphogenic trajectories. This novel methodology enabled the quantitative description of morphogenic pattern trajectories that could be compared across diverse systems: computational modelling of multicellular structures, differentiation of stem cell aggregates, and gastrulation of cichlid fish. Moreover, this method identified novel spatio-temporal features associated with different stages of embryo gastrulation, and elucidated a complex paracrine mechanism capable of explaining spatiotemporal pattern kinetic differences in ESC aggregates of different sizes.</div>','<b>Title: </b>Towards a Dynamic Interaction Network of Life to unify and expand the evolutionary theory<br><b>Year: </b>2018<br><b>Author: </b> E. Bapteste and  P. Huneman<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/s12915-018-0531-6"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC BIOLOGY<br><b>Cluster-Keywords: </b><em>network motifs, regulatory network, gene regulatory, systems biology, gene network, bifan motif, complex systems, model comparison, network modeling, complex gene, biological systems, dynamic systems, interaction network, feedforward loop, modeling gene, intelligent systems, modeling approach, biological pathways, motif model, model motifs<div style="font-size:11px;"><br><b>Abstract: </b><em>The classic Darwinian theory and the Synthetic evolutionary theory and then linear models, while invaluable to study the origins and evolution of species, are not primarily designed to model the evolution of organisations, typically that of ecosystems, nor that of processes. How could evolutionary theory better explain the evolution of biological complexity and diversity Inclusive network based analyses of dynamic systems could retrace interactions between related or unrelated components. This theoretical shift from a Tree of Life to a Dynamic Interaction Network of Life, which is supported by diverse molecular, cellular, microbiological, organismal, ecological and evolutionary studies, would further unify evolutionary biology.</div>','<b>Title: </b>A genetic algorithm-based boolean delay model of intracellular signal transduction in inflammation<br><b>Year: </b>2011<br><b>Author: </b> Cc. Kang,  Yj. Chuang,  Kc. Tung,  Cc. Chao,  Cy. Tang,  Sc. Peng and  Dsh. Wong<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1471-2105-12-S1-S17"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>boolean network, network models, qualitative models, systems biology, mathematical modeling, boolean models, signaling networks, dynamic models, dynamic modeling, signaling dynamics, logic models, dynamic network, mathematical models, boolean modeling, signal transduction, regulatory networks, drug resistance, biological networks, network approach, network modeling<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Signal transduction is the major mechanism through which cells transmit external stimuli to evoke intracellular biochemical responses. Understanding relationship between external stimuli and corresponding cellular responses, as well as the subsequent effects on downstream genes, is a major challenge in systems biology. Thus, a systematic approach to integrate experimental data and qualitative knowledge to identify the physiological consequences of environmental stimuli is needed.Results: In present study, we employed a genetic algorithm-based Boolean model to represent NF kappa B signaling pathway. We were able to capture feedback and crosstalk characteristics to enhance our understanding on the acute and chronic inflammatory response. Key network components affecting the response dynamics were identified.Conclusions: We designed an effective algorithm to elucidate the process of immune response using comprehensive knowledge about network structure and limited experimental data on dynamic responses. This approach can potentially be implemented for large-scale analysis on cellular processes and organism behaviors.</div>','<b>Title: </b>Proteomic and protein interaction network analysis of human T lymphocytes during cell-cycle entry<br><b>Year: </b>2012<br><b>Author: </b> Sj. Orr,  Dr. Boutz,  R. Wang,  C. Chronis,  Nc. Lea,  T. Thayaparan,  E. Hamilton,  H. Milewicz,  E. Blanc,  Gj. Mufti,  Em. Marcotte and  Nsb. Thomas<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1038/msb.2012.5"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MOLECULAR SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>cell cycle, mammalian cell, cdk network, cell proliferation, cell, cells cell, regulatory network, detailed model, cell size, molecular network, skeleton model, cycle arrest, cycle progression, cell division, cell density, kinases cdks, cycle, phase transition, cycle control, network driving<div style="font-size:11px;"><br><b>Abstract: </b><em>Regulating the transition of cells such as T lymphocytes from quiescence G0 into an activated, proliferating state involves initiation of cellular programs resulting in entry into the cell cycle proliferation, the growth cycle blastogenesis, cell size and effector functional activation. We show the first proteomic analysis of protein interaction networks activated during entry into the first cell cycle from G0. We also provide proof of principle that blastogenesis and proliferation programs are separable in primary human Tcells. We employed a proteomic profiling method to identify large-scale changes in chromatin/nuclear matrix-bound and unbound proteins in human T lymphocytes during the transition from G0 into the first cell cycle and mapped them to form functionally annotated, dynamic protein interaction networks. Inhibiting the induction of two proteins involved in two of the most significantly upregulated cellular processes, ribosome biogenesis eIF6 and hnRNA splicing SF3B2/SF3B4, showed, respectively, that human T cells can enter the cell cycle without growing in size, or increase in size without entering the cell cycle. Molecular Systems Biology 8: 573; published online 13 March 2012; doi:10.1038/msb.2012.5 Subject Categories: proteomics; cell cycle</div>','<b>Title: </b>Modeling of the U1 snRNP assembly pathway in alternative splicing in human cells using Petri nets<br><b>Year: </b>2009<br><b>Author: </b> J. Kielbassa,  R. Bortfeldt,  S. Schuster and  I. Koch<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.compbiolchem.2008.07.022"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>COMPUTATIONAL BIOLOGY AND CHEMISTRY<br><b>Cluster-Keywords: </b><em>petri net, petri nets, bayesian networks, biological networks, systems biology, regulatory networks, dynamic bayesian, net model, gene regulatory, network data, biological systems, biological data, network analysis, network models, biochemical networks, time series, network structure, signal transduction, net models, dbn model<div style="font-size:11px;"><br><b>Abstract: </b><em>The investigation of spliceosomal processes is currently a topic of intense research in molecular biology. In the molecular mechanism of alternative splicing, a multi-protein-RNA complex - the spliceosome - plays a crucial role. To understand the biological processes of alternative splicing, it is essential to comprehend the biogenesis of the spliceosome.In this paper, we propose the first abstract model of the regulatory assembly pathway of the human spliceosomal subunit U1. Using Petri nets, we describe its highly ordered assembly that takes place in a stepwise manner.Petri net theory represents a mathematical formalism to model and analyze systems with concurrent processes at different abstraction levels with the possibility to combine them into a uniform description language. There exist many approaches to determine static and dynamic properties of Petri nets, which can be applied to analyze biochemical systems. In addition, Petri net tools usually provide intuitively understandable graphical network representations, which facilitate the dialog between experimentalists and theoreticians.Our Petri net model covers binding, transport, signaling, and covalent modification processes. Through the computation of structural and behavioral Petri net properties and their interpretation in biological terms, we validate our model and use it to get a better understanding of the complex processes of the assembly pathway.We can explain the basic network behavior, using minimal T-invariants which represent special pathways through the network. We find linear as well as cyclic pathways. We determine the P-invariants that represent conserved moieties in a network. The simulation of the net demonstrates the importance of the stability of complexes during the maturation pathway. We can show that complexes that dissociate too fast, hinder the formation of the complete U1 snRNP. c 2008 Elsevier Ltd. All rights reserved.</div>','<b>Title: </b>Boolean Biology: Introducing Boolean Networks and Finite Dynamical Systems Models to Biology and Mathematics Courses<br><b>Year: </b>2011<br><b>Author: </b> R. Robeva,  B. Kirkwood and  R. Davies<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1051/mmnp/20116603"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MATHEMATICAL MODELLING OF NATURAL PHENOMENA<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Since the release of the Bio 2010 report in 2003, significant emphasis has been placed on initiating changes in the way undergraduate biology and mathematics courses are taught and on creating new educational materials to facilitate those changes. Quantitative approaches, including mathematical models, are now considered critical for the education of the next generation of biologists. In response, mathematics departments across the country have initiated changes to their introductory calculus sequence, adding models, projects, and examples from the life sciences, or offering specialized calculus courses for biology majors. However, calculus-based models and techniques from those courses have been slow to propagate into the traditional general biology courses. And although modern biology has generated exciting opportunities for applications of a broad spectrum of mathematical theories, the impact on the undergraduate mathematics courses outside of the calculus/ordinary differential equations sequence has been minimal at best. Thus, the limited interdisciplinary cross-over between the undergraduate mathematics and biology curricula has remained stubbornly stagnant despite ongoing calls for integrated approaches.We think that this phenomenon is due primarily to a lack of appropriate non-calculus-based interdisciplinary educational materials rather than inaccessibility of the essential underlying mathematical and biology concepts. Here we present a module that uses Boolean network models of the lac operon regulatory mechanism as an introduction to the conceptual importance of mathematical models and their analysis. No mathematics background beyond high school algebra is necessary to construct the model, which makes the approach particularly appropriate for introductory biology and mathematics courses. Initially the module focuses on modeling via Boolean logic, Boolean algebra, discrete dynamical systems, and directed graphs. The analysis of the model, however, leads to interesting advanced mathematical questions involving polynomial ideals and algebraic varieties that are beyond the mathematical proficiency of most biology students but are of interest in advanced level abstract algebra courses. These questions can also be used to map a path toward further research-level problems. All computations are carried out by computational algebra systems and the advanced mathematical theory implemented by the software can be covered in detail in abstract algebra courses with mathematics students. Biology students and students in lower level mathematic courses, on the other hand, can view the implementation as a "blackbox" and focus on the interpretation and the implications of the output.The module is available from the authors for classroom testing and adoption.</div>','<b>Title: </b>Mitochondrial Variability as a Source of Extrinsic Cellular Noise<br><b>Year: </b>2012<br><b>Author: </b> Ig. Johnston,  B. Gaal,  Rp. Das Neves,  T. Enver,  Fj. Iborra and  Ns. Jones<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pcbi.1002416"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS COMPUTATIONAL BIOLOGY<br><b>Cluster-Keywords: </b><em>gene expression, gene network, stochastic model, gene regulatory, systems biology, expression systems, gene regulation, network activity, expression dynamics, noise control, regulatory network, deterministic models, network components, stochastic modeling, regulation system, escherichia coli, regulatory system, synthetic gene, regulatory genes, gene<div style="font-size:11px;"><br><b>Abstract: </b><em>We present a study investigating the role of mitochondrial variability in generating noise in eukaryotic cells. Noise in cellular physiology plays an important role in many fundamental cellular processes, including transcription, translation, stem cell differentiation and response to medication, but the specific random influences that affect these processes have yet to be clearly elucidated. Here we present a mechanism by which variability in mitochondrial volume and functionality, along with cell cycle dynamics, is linked to variability in transcription rate and hence has a profound effect on downstream cellular processes. Our model mechanism is supported by an appreciable volume of recent experimental evidence, and we present the results of several new experiments with which our model is also consistent. We find that noise due to mitochondrial variability can sometimes dominate over other extrinsic noise sources such as cell cycle asynchronicity and can significantly affect large-scale observable properties such as cell cycle length and gene expression levels. We also explore two recent regulatory network-based models for stem cell differentiation, and find that extrinsic noise in transcription rate causes appreciable variability in the behaviour of these model systems. These results suggest that mitochondrial and transcriptional variability may be an important mechanism influencing a large variety of cellular processes and properties.</div>','<b>Title: </b>A Dynamical method to estimate gene regulatory networks using time-series data<br><b>Year: </b>2015<br><b>Author: </b> Cy. Tu<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1002/cplx.21585"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>COMPLEXITY<br><b>Cluster-Keywords: </b><em>gene regulatory, gene expression, regulatory networks, expression data, regulatory pathway, regulatory genes, network dynamics, microarray data, genetic networks, transcription factor, models network, target gene, upstream regulatory, network topology, pathway network, gene functional, expression profile, transcriptional network, dynamic model, transcriptional regulatory<div style="font-size:11px;"><br><b>Abstract: </b><em>We propose an improved discrete dynamical system model to reconstruct the gene regulatory network GRN, then estimate the variable topology using discrete-time autosynchronization and predict the expression rate under the same condition. Although our method adopts a small number of sample time points to estimate the GRN, it could discern not only the role of the activator or repressor for each specific regulator, but also the regulatory ability of the regulator to the transcription rate of the target gene. Several examples are illustrated to verify that this method is feasible and effective for modeling the GRN and predicting the expression profile in the next cell cycle, the expression profile in the interval between two sample time points or the deficiency data. Additionally, this method provides a general tool for topology estimation of discrete-time dynamical networks. c 2014 Wiley Periodicals, Inc.</div>','<b>Title: </b>pathPSA: A Dynamical Pathway-Based Parametric Sensitivity Analysis<br><b>Year: </b>2014<br><b>Author: </b> Tm. Perumal and  R. Gunawan<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1021/ie403277d"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>INDUSTRIAL  ENGINEERING CHEMISTRY RESEARCH<br><b>Cluster-Keywords: </b><em>sensitivity analysis, systems biology, robustness analysis, network model, network inference, regulatory network, systems analysis, gene expression, parametric uncertainty, biological networks, mathematical models, biochemical networks, biological functions, cellular networks, biological system, parametric sensitivity, dynamical models, dynamic system, networks analysis, gene regulatory<div style="font-size:11px;"><br><b>Abstract: </b><em>Normal functioning of biological systems relies on coordinated activities of biomolecules participating in a complex network of biological interactions. As human intuition is often incapable in analyzing how biological functions emerge from such complex interactions, the use of mathematical models and systems analysis tools has become critical in understanding biological system behavior. While biological functions have been associated with the connectivity structure and pathways of the network and the kinetics of the processes involved, most model analyses address each of these aspects separately. In contrast, we present a novel sensitivity analysis, called pathway-based parametric sensitivity analysis pathPSA, which combines the analysis of both network structure and kinetics. Unlike existing sensitivity analyses, pathPSA relies on perturbing the kinetics of pathways in the network, using persistent perturbations or impulse perturbations at varying time. Consequently, the sensitivity coefficients can give insights on the dominant pathways in a network and any transient shift in the rate controlling mechanism. The efficacy of pathPSA is demonstrated through an application to understand competing signaling pathways in programmed cell death network.</div>','<b>Title: </b>Chronic hepatitis B: dynamic change in Traditional Chinese Medicine syndrome by dynamic network biomarkers<br><b>Year: </b>2019<br><b>Author: </b> Yy. Lu,  Zy. Fang,  T. Zeng,  My. Li,  Ql. Chen,  H. Zhang,  Qm. Zhou,  Yy. Hu,  Ln. Chen and  Sb. Su<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/s13020-019-0275-4"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>CHINESE MEDICINE<br><b>Cluster-Keywords: </b><em>systems biology, dynamic network, early differentiation, complex diseases, stem cells, erythroid cells, system dynamics, network biomarkers, immune system, tcm syndrome, dnb genes, gene expression, dnb model, stem cell, expression dynamics, model system, mathematical model, model dynamics, dynamic model, sensitivity analysis<div style="font-size:11px;"><br><b>Abstract: </b><em>Background In traditional Chinese medicine TCM clinical practice, TCM syndromes help to understand human homeostasis and guide individualized treatment. However, the TCM syndrome changes with disease progression, of which the scientific basis and mechanism remain unclear. Methods To demonstrate the underlying mechanism of dynamic changes in the TCM syndrome, we applied a dynamic network biomarker DNB algorithm to obtain the DNBs of changes in the TCM syndrome, based on the transcriptomic data of patients with chronic hepatitis B and typical TCM syndromes, including healthy controls and patients with liver-gallbladder dampness-heat syndrome LGDHS, liver-depression spleen-deficiency syndrome LDSDS, and liver-kidney yin-deficiency syndrome LKYDS. The DNB model exploits collective fluctuations and correlations of the observed genes, then diagnoses the critical state. Results Our results showed that the DNBs of TCM syndromes were comprised of 52 genes and the tipping point occurred at the LDSDS stage. Meanwhile, there were numerous differentially expressed genes between LGDHS and LKYDS, which highlighted the drastic changes before and after the tipping point, implying the 52 DNBs could serve as early-warning signals of the upcoming change in the TCM syndrome. Next, we validated DNBs by cytokine profiling and isobaric tags for relative and absolute quantitation iTRAQ. The results showed that PLG plasminogen and coagulation factor XII F12 were significantly expressed during the progression of TCM syndrome from LGDHS to LKYDS. Conclusions This study provides a scientific understanding of changes in the TCM syndrome. During this process, the cytokine system was involved all the time. The DNBs PLG and F12 were confirmed to significantly change during TCM-syndrome progression and indicated a potential value of DNBs in auxiliary diagnosis of TCM syndrome in CHB. Trial registration Identifier: NCT03189992. Registered on June 4, 2017. Retrospectively registered http://www.clinicaltrials.gov</div>','<b>Title: </b>EXTINCTIONS IN HETEROGENEOUS ENVIRONMENTS AND THE EVOLUTION OF MODULARITY<br><b>Year: </b>2009<br><b>Author: </b> N. Kashtan,  M. Parter,  E. Dekel,  Ae. Mayo and  U. Alon<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1111/j.1558-5646.2009.00684.x"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>EVOLUTION<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Extinctions of local subpopulations are common events in nature. Here, we ask whether such extinctions can affect the design of biological networks within organisms over evolutionary timescales. We study the impact of extinction events on modularity of biological systems, a common architectural principle found on multiple scales in biology. As a model system, we use networks that evolve toward goals specified as desired input-output relationships. We use an extinction-recolonization model, in which metapopulations occupy and migrate between different localities. Each locality displays a different environmental condition goal, but shares the same set of subgoals with other localities. We find that in the absence of extinction events, the evolved computational networks are typically highly optimal for their localities with a nonmodular structure. In contrast, when local populations go extinct from time to time, we find that the evolved networks are modular in structure. Modular circuitry is selected because of its ability to adapt rapidly to the conditions of the free niche following an extinction event. This rapid adaptation is mainly achieved through genetic recombination of modules between immigrants from neighboring local populations. This study suggests, therefore, that extinctions in heterogeneous environments promote the evolution of modular biological network structure, allowing local populations to effectively recombine their modules to recolonize niches.</div>','<b>Title: </b>Petri net modelling of gene regulation of the Duchenne muscular dystrophy<br><b>Year: </b>2008<br><b>Author: </b> G. Stefanie,  S. Astrid,  J. Ackermann and  I. Koch<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.biosystems.2008.02.005"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOSYSTEMS<br><b>Cluster-Keywords: </b><em>petri net, petri nets, bayesian networks, biological networks, systems biology, regulatory networks, dynamic bayesian, net model, gene regulatory, network data, biological systems, biological data, network analysis, network models, biochemical networks, time series, network structure, signal transduction, net models, dbn model<div style="font-size:11px;"><br><b>Abstract: </b><em>Searching for therapeutic strategies for Duchenne muscular dystrophy, it is of great interest to understand the responsible molecular pathways down-stream of dystrophin completely. For this reason we have performed real-time PCR experiments to compare mRNA expression levels of relevant genes in tissues of affected patients and controls. To bring experimental data in context with the underlying pathway theoretical models are needed. Modelling of biological processes in the cell at higher description levels is still an open problem in the field of systems biology. In this paper, a new application of Petri net theory is presented to model gene regulatory processes of Duchenne muscular dystrophy.We have developed a Petri net model, which is based mainly on own experimental and literature data. We distinguish between up- and down-regulated states of gene expression. The analysis of the model comprises the computation of structural and dynamic properties with focus on a thorough T-invariant analysis, including clustering techniques and the decomposition of the network into maximal common transition sets MCT-sets, which can be interpreted as functionally related building blocks. All possible pathways, which reflect the complex net behaviour in dependence of different gene expression patterns, are discussed. We introduce Mauritius maps of T-invariants, which enable, for example, theoretical knockout analysis.The resulted model serves as basis for a better understanding of pathological processes, and thereby for planning next experimental steps in searching for new therapeutic possibilities.Availability: Free availability of the Petri net editor and animator Snoopy and the clustering tool PInA via http://www-dssz.informatik.tu-cottbus.de/similar to wwwdssz/. The Petri net models used can be accessed via http://www.tfh-berlin.de/bi/duchenne/. c 2008 Elsevier Ireland Ltd. All rights reserved.</div>','<b>Title: </b>Maximizing capture of gene co-expression relationships through pre-clustering of input expression samples: an Arabidopsis case study<br><b>Year: </b>2013<br><b>Author: </b> Fa. Feltus,  Sp. Ficklin,  Sm. Gibson and  Mc. Smith<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1752-0509-7-44"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>systems biology, regulatory network, interaction networks, network model, gene regulatory, network, network biology, networks, gene network, response networks, network dynamics, dynamic model, cell networks, gene expression, biological networks, system dynamics, dynamic network, model, biological systems, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: In genomics, highly relevant gene interaction co-expression networks have been constructed by finding significant pair-wise correlations between genes in expression datasets. These networks are then mined to elucidate biological function at the polygenic level. In some cases networks may be constructed from input samples that measure gene expression under a variety of different conditions, such as for different genotypes, environments, disease states and tissues. When large sets of samples are obtained from public repositories it is often unmanageable to associate samples into condition-specific groups, and combining samples from various conditions has a negative effect on network size. A fixed significance threshold is often applied also limiting the size of the final network. Therefore, we propose pre-clustering of input expression samples to approximate condition-specific grouping of samples and individual network construction of each group as a means for dynamic significance thresholding. The net effect is increase sensitivity thus maximizing the total co-expression relationships in the final co-expression network compendium.Results: A total of 86 Arabidopsis thaliana co-expression networks were constructed after k-means partitioning of 7,105 publicly available ATH1 Affymetrix microarray samples. We term each pre-sorted network a Gene Interaction Layer GIL. Random Matrix Theory RMT, an un-supervised thresholding method, was used to threshold each of the 86 networks independently, effectively providing a dynamic non-global threshold for the network. The overall gene count across all GILs reached 19,588 genes 94.7 measured gene coverage and 558,022 unique co-expression relationships. In comparison, network construction without pre-sorting of input samples yielded only 3,297 genes 15.9 and 129,134 relationships. in the global network.Conclusions: Here we show that pre-clustering of microarray samples helps approximate condition-specific networks and allows for dynamic thresholding using un-supervised methods. Because RMT ensures only highly significant interactions are kept, the GIL compendium consists of 558,022 unique high quality A. thaliana co-expression relationships across almost all of the measurable genes on the ATH1 array. For A. thaliana, these networks represent the largest compendium to date of significant gene co-expression relationships, and are a means to explore complex pathway, polygenic, and pleiotropic relationships for this focal model plant. The networks can be explored at sysbio.genome.clemson.edu. Finally, this method is applicable to any large expression profile collection for any organism and is best suited where a knowledge-independent network construction method is desired.</div>','<b>Title: </b>Systems biology to predict blood function<br><b>Year: </b>2009<br><b>Author: </b> Sl. Diamond<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1111/j.1538-7836.2009.03463.x"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF THROMBOSIS AND HAEMOSTASIS<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Systems biology seeks to provide a quantitative framework to understand blood as a reactive biological fluid whose function is dictated by prevailing haemodynamics, vessel wall characteristics, plateletmetabolism, numerous coagulation factors in plasma, and small molecules released during thrombosis. The hierarchical nature of thrombosis requires analysis of adhesive bond dynamics of activated platelets captured from a flow field to a growing thrombus boundary along with the simultaneous assembly of the coagulation pathway. Several kinetic models of protease cascades have been developed. A full bottom-up model of platelet intracellular metabolism is now available to simulate the metabolism of resting platelets and platelets exposed to activators. Monte Carlo algorithms can finally accommodate platelet reaction, dispersion, and convection for full simulation of platelet deposition and clotting under flow. For clinical applications, the systems biology prediction of patient-specific pharmacological response requires the final assembly of platelet intracellular metabolism models with coagulation protease network models.</div>','<b>Title: </b>From systems biology to dynamical neuropharmacology: proposal for a new methodology<br><b>Year: </b>2006<br><b>Author: </b> P. Erdi,  T. Kiss,  J. Toth,  B. Ujfalussy and  L. Zalanyi<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1049/ip-syb:20050091"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>IEE PROCEEDINGS SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>information systems, gene expression, gene network, systems theory, system models, systems biology, systems information, network model, genetic network, mathematical models, dynamic systems, dynamic network, systemic model, network topology, information mechanisms, complex systems, systems complex, relational models, models main, information model<div style="font-size:11px;"><br><b>Abstract: </b><em>The concepts and methods of systems biology are extended to neuropharmacology in order to test and design drugs for the treatment of neurological and psychiatric disorders. Computational modelling by integrating compartmental neural modelling techniques and detailed kinetic descriptions of pharmacological modulation of tran smitter-receptor interaction is offered as a method to test the electrophysiological and behavioural effects of putative drugs. Even more, an inverse method is suggested as a method for controlling a neural system to realise a prescribed temporal pattern. In particular, as an application of the proposed new methodology, a computational platform is offered to analyse the generation and pharmacological modulation of theta rhythm related to anxiety.</div>','<b>Title: </b>Mathematical Modeling of the Dynamics of Shoot-Root Interactions and Resource Partitioning in Plant Growth<br><b>Year: </b>2015<br><b>Author: </b> C. Feller,  P. Favre,  A. Janka,  Sc. Zeeman,  Jp. Gabriel and  D. Reinhardt<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0127905"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Plants are highly plastic in their potential to adapt to changing environmental conditions. For example, they can selectively promote the relative growth of the root and the shoot in response to limiting supply of mineral nutrients and light, respectively, a phenomenon that is referred to as balanced growth or functional equilibrium. To gain insight into the regulatory network that controls this phenomenon, we took a systems biology approach that combines experimental work with mathematical modeling. We developed a mathematical model representing the activities of the root nutrient and water uptake and the shoot photosynthesis, and their interactions through the exchange of the substrates sugar and phosphate Pi. The model has been calibrated and validated with two independent experimental data sets obtained with Petunia hybrida. It involves a realistic environment with a day-and-night cycle, which necessitated the introduction of a transitory carbohydrate storage pool and an endogenous clock for coordination of metabolism with the environment. Our main goal was to grasp the dynamic adaptation of shoot: root ratio as a result of changes in light and Pi supply. The results of our study are in agreement with balanced growth hypothesis, suggesting that plants maintain a functional equilibrium between shoot and root activity based on differential growth of these two compartments. Furthermore, our results indicate that resource partitioning can be understood as the emergent property of many local physiological processes in the shoot and the root without explicit partitioning functions. Based on its encouraging predictive power, the model will be further developed as a tool to analyze resource partitioning in shoot and root crops.</div>','<b>Title: </b>Exploring signal transduction in heteromultimeric protein based on energy dissipation model<br><b>Year: </b>2015<br><b>Author: </b> Cw. Ma,  Zl. Xiu and  Ap. Zeng<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1080/07391102.2013.855145"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF BIOMOLECULAR STRUCTURE  DYNAMICS<br><b>Cluster-Keywords: </b><em>signaling network, systems biology, signal transduction, signaling pathways, receptor tyrosine, erbb signaling, erbb receptor, receptor signaling, intracellular signaling, mathematical models, cell signaling, computational model, factor receptor, receptor activation, growth factor, signaling models, networks signaling, receptor system, receptors systems, downstream signaling<div style="font-size:11px;"><br><b>Abstract: </b><em>Dynamic intersubunit interactions are key elements in the regulation of many biological systems. A better understanding of how subunits interact with each other and how their interactions are related to dynamic protein structure is a fundamental task in biology. In this paper, a heteromultimeric allosteric protein, Corynebacterium glutamicum aspartokinase, is used as a model system to explore the signal transduction involved in intersubunit interactions and allosteric communication with an emphasis on the intersubunit signaling process. For this purpose, energy dissipation simulation and network construction are conducted for each subunit and the whole protein. Comparison with experimental results shows that the new approach is able to predict all the mutation sites that have been experimentally proved to desensitize allosteric regulation of the enzyme. Additionally, analysis revealed that the function of the effector threonine is to facilitate the binding of the two subunits without contributing to the allosteric communication. During the allosteric regulation upon the binding of the effector lysine, signals can be transferred from the beta-subunit to the catalytic site of the alpha-subunit through both a direct way of intersubunit signal transduction, and an indirect way: first, to the regulatory region of the alpha-subunit by intersubunit signal transduction and then to the catalytic region by intramolecular signal transduction. Therefore, the new approach is able to illustrate the diversity of the underlying mechanisms when the strength of feedback inhibition by the effectors is modulated, providing useful information that has potential applications in engineering heteromultimeric allosteric regulation.</div>','<b>Title: </b>A graphical and computational modeling platform for biological pathways<br><b>Year: </b>2018<br><b>Author: </b> A. Livigni,  L. OHara,  Me. Polak,  T. Angus,  Dw. Wright,  Lb. Smith and  Tc. Freeman<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1038/nprot.2017.144"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>NATURE PROTOCOLS<br><b>Cluster-Keywords: </b><em>petri net, petri nets, bayesian networks, biological networks, systems biology, regulatory networks, dynamic bayesian, net model, gene regulatory, network data, biological systems, biological data, network analysis, network models, biochemical networks, time series, network structure, signal transduction, net models, dbn model<div style="font-size:11px;"><br><b>Abstract: </b><em>A major endeavor of systems biology is the construction of graphical and computational models of biological pathways as a means to better understand their structure and function. Here, we present a protocol for a biologist-friendly graphical modeling scheme that facilitates the construction of detailed network diagrams, summarizing the components of a biological pathway such as proteins and biochemicals and illustrating how they interact. These diagrams can then be used to simulate activity flow through a pathway, thereby modeling its dynamic behavior. The protocol is divided into four sections: i assembly of network diagrams using the modified Edinburgh Pathway Notation mEPN scheme and yEd network editing software with pathway information obtained from published literature and databases of molecular interaction data; ii parameterization of the pathway model within yEd through the placement of tokens on the basis of the known or imputed amount or activity of a component; iii model testing through visualization and quantitative analysis of the movement of tokens through the pathway, using the network analysis tool Graphia Professional and iv optimization of model parameterization and experimentation. This is the first modeling approach that combines a sophisticated notation scheme for depicting biological events at the molecular level with a Petri net-based flow simulation algorithm and a powerful visualization engine with which to observe the dynamics of the system being modeled. Unlike many mathematical approaches to modeling pathways, it does not require the construction of a series of equations or rate constants for model parameterization. Depending on a models complexity and the availability of information, its construction can take days to months, and, with refinement, possibly years. However, once assembled and parameterized, a simulation run, even on a large model, typically takes only seconds. Models constructed using this approach provide a means of knowledge management, information exchange and, through the computation simulation of their dynamic activity, generation and testing of hypotheses, as well as prediction of a systems behavior when perturbed.</div>','<b>Title: </b>Network-level analysis of light adaptation in rod cells under normal and altered conditions<br><b>Year: </b>2009<br><b>Author: </b> D. DellOrco,  H. Schmidt,  S. Mariani and  F. Fanelli<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1039/b908123b"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MOLECULAR BIOSYSTEMS<br><b>Cluster-Keywords: </b><em>systems biology, dynamic network, early differentiation, complex diseases, stem cells, erythroid cells, system dynamics, network biomarkers, immune system, tcm syndrome, dnb genes, gene expression, dnb model, stem cell, expression dynamics, model system, mathematical model, model dynamics, dynamic model, sensitivity analysis<div style="font-size:11px;"><br><b>Abstract: </b><em>Photoreceptor cells finely adjust their sensitivity and electrical response according to changes in light stimuli as a direct consequence of the feedback and regulation mechanisms in the phototransduction cascade. In this study, we employed a systems biology approach to develop a dynamic model of vertebrate rod phototransduction that accounts for the details of the underlying biochemistry. Following a bottom-up strategy, we first reproduced the results of a robust model developed by Hamer et al. Vis. Neurosci., 2005, 224, 417, and then added a number of additional cascade reactions including: a explicit reactions to simulate the interaction between the activated effector and the regulator of G-protein signalling RGS; b a reaction for the reformation of the G-protein from separate subunits; c a reaction for rhodopsin R reconstitution from the association of the opsin apoprotein with the 11-cis-retinal chromophore; d reactions for the slow activation of the cascade by opsin. The extended network structure successfully reproduced a number of experimental conditions that were inaccessible to prior models. With a single set of parameters the model was able to predict qualitative and quantitative features of rod photoresponses to light stimuli ranging over five orders of magnitude, in normal and altered conditions, including genetic manipulations of the cascade components. In particular, the model reproduced the salient dynamic features of the rod from Rpe65-/- animals, a well established model for Leber congenital amaurosis and vitamin A deficiency. The results of this study suggest that a systems-level approach can help to unravel the adaptation mechanisms in normal and in disease-associated conditions on a molecular basis.</div>','<b>Title: </b>NETWORK AUTOMATA: COUPLING STRUCTURE AND FUNCTION IN DYNAMIC NETWORKS<br><b>Year: </b>2011<br><b>Author: </b> Dmd. Smith,  Jp. Onnela,  Cf. Lee,  Md. Fricker and  Nf. Johnson<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1142/S0219525911003050"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ADVANCES IN COMPLEX SYSTEMS<br><b>Cluster-Keywords: </b><em>complex networks, complex systems, network theory, network structure, biological network, social network, network science, dynamic networks, network systems, network model, network topology, systems biology, network, network analysis, system dynamics, networks, realworld networks, network automata, epidemic network, model network<div style="font-size:11px;"><br><b>Abstract: </b><em>We introduce Network Automata, a framework which couples the topological evolution of a network to its structure. To demonstrate its implementation we describe a simple model which exhibits behavior similar to the "Game of Life" before recasting some simple, well-known network models as Network Automata. We then introduce Functional Network Automata which are useful for dealing with networks in which the topology evolves according to some specified microscopic rules and, simultaneously, there is a dynamic process taking place on the network that both depends on its structure but is also capable of modifying it. It is a generic framework for modeling systems in which network structure, dynamics, and function are interrelated. At the practical level, this framework allows for precise, unambiguous implementation of the microscopic rules involved in such systems. To demonstrate the approach, we develop a family of simple biologically inspired models of fungal growth.</div>','<b>Title: </b>SNAP: A General-Purpose Network Analysis and Graph-Mining Library<br><b>Year: </b>2016<br><b>Author: </b> J. Leskovec and  R. Sosic<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1145/2898361"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ACM TRANSACTIONS ON INTELLIGENT SYSTEMS AND TECHNOLOGY<br><b>Cluster-Keywords: </b><em>complex networks, complex systems, network theory, network structure, biological network, social network, network science, dynamic networks, network systems, network model, network topology, systems biology, network, network analysis, system dynamics, networks, realworld networks, network automata, epidemic network, model network<div style="font-size:11px;"><br><b>Abstract: </b><em>Large networks are becoming a widely used abstraction for studying complex systems in a broad set of disciplines, ranging from social-network analysis to molecular biology and neuroscience. Despite an increasing need to analyze and manipulate large networks, only a limited number of tools are available for this task.Here, we describe the Stanford Network Analysis Platform SNAP, a general-purpose, high-performance system that provides easy-to-use, high-level operations for analysis and manipulation of large networks. We present SNAP functionality, describe its implementational details, and give performance benchmarks. SNAP has been developed for single big-memory machines, and it balances the trade-off between maximum performance, compact in-memory graph representation, and the ability to handle dynamic graphs in which nodes and edges are being added or removed over time. SNAP can process massive networks with hundreds of millions of nodes and billions of edges. SNAP offers over 140 different graph algorithms that can efficiently manipulate large graphs, calculate structural properties, generate regular and random graphs, and handle attributes and metadata on nodes and edges. Besides being able to handle large graphs, an additional strength of SNAP is that networks and their attributes are fully dynamic; they can be modified during the computation at low cost. SNAP is provided as an open-source library in C as well as a module in Python.We also describe the Stanford Large Network Dataset, a set of social and information real-world networks and datasets, which we make publicly available. The collection is a complementary resource to our SNAP software and is widely used for development and benchmarking of graph analytics algorithms.</div>','<b>Title: </b>Analysis of Information Flows in Interaction Networks: Implication for Drug Discovery and Pharmacological Research<br><b>Year: </b>2011<br><b>Author: </b> Af. Fliri,  Wt. Loging and  Ra. VolkmannLink: No link<br><b>Journal: </b>DISCOVERY MEDICINE<br><b>Cluster-Keywords: </b><em>stem cell, stem cells, systems biology, mathematical models, cell biology, network model, signaling pathways, cell fate, computational models, cancer cells, gene expression, network dynamics, embryonic stem, signaling network, prostate cancer, mathematical modeling, psychological stress, apoptosis signaling, computational systems, molecular mechanisms<div style="font-size:11px;"><br><b>Abstract: </b><em>Frequent failures of experimental medicines in clinical trials question current concepts for predicting drug-effects in the human body. Improving the probability for success in drug discovery requires a better understanding of cause-effect relationships at the organism, organ, tissue, cellular, and molecular levels, each having a different degree of complexity. Despite the longstanding realization that clinical and preclinical drug-effect information needs to be integrated for generating more accurate forecasts of drug-effects, a road map for linking these disparate sources of information currently does not exist. This review focuses on a possible approach for obtaining these relationships by analyzing causes and effects on the basis of the topology of network interaction systems that process information at the cellular and organ system levels. Discovery Medicine 11 57: 133-143, February 2011</div>','<b>Title: </b>Graphical Modeling Tools for Systems Biology<br><b>Year: </b>2015<br><b>Author: </b> R. Gostner,  B. Baldacci,  Mj. Morine and  C. Priami<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1145/2633461"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ACM COMPUTING SURVEYS<br><b>Cluster-Keywords: </b><em>systems biology, network models, signaling network, parameter estimation, dynamic models, biological systems, mathematical models, metabolic network, parameter sensitivities, computational model, gene circuit, circuit models, experimental data, metabolic models, model system, scalable parameter, model parameters, parameter space, biomolecular networks, model<div style="font-size:11px;"><br><b>Abstract: </b><em>Modeling biological systems to understand their mechanistic behavior is an important activity in molecular systems biology. Mathematical modeling typically requires deep mathematical or computing knowledge, and this limits the spread of modeling tools among biologists. Graphical modeling languages have been introduced to minimize this limit. Here, we survey the main graphical formalisms supported by software tools available to model biological systems with a primary focus on their usability, within the framework of modeling reaction pathways with two-dimensional 2D possibly nested compartments. Considering the main characteristics of the surveyed formalisms, we synthesise a new proposal Style and report the results of an online survey conducted among biologists to assess usability of available graphical formalisms. We consider this proposal a guideline developed from what we learned in the survey, which can inform development of graphical formalisms to model reaction pathways in 2D space.</div>','<b>Title: </b>FOUNDATIONS FOR MODELING THE DYNAMICS OF GENE REGULATORY NETWORKS: A MULTILEVEL-PERSPECTIVE REVIEW<br><b>Year: </b>2014<br><b>Author: </b> I. Sanchez-Osorio,  F. Ramos,  P. Mayorga and  E. Dantan<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1142/S0219720013300037"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>JOURNAL OF BIOINFORMATICS AND COMPUTATIONAL BIOLOGY<br><b>Cluster-Keywords: </b><em>reaction networks, network model, systems biology, biochemical reaction, stochastic reaction, model reduction, stochastic model, network dynamics, stochastic biochemical, stochastic simulation, system size, chemical reaction, gene network, elementary reactions, biochemical networks, parameter identification, parameter inference, cell populations, small system, linear noise<div style="font-size:11px;"><br><b>Abstract: </b><em>A promising alternative for unraveling the principles under which the dynamic interactions among genes lead to cellular phenotypes relies on mathematical and computational models at different levels of abstraction, from the molecular level of protein-DNA interactions to the system level of functional relationships among genes. This review article presents, under a bottom-up perspective, a hierarchy of approaches to modeling gene regulatory network dynamics, from microscopic descriptions at the single-molecule level in the spatial context of an individual cell to macroscopic models providing phenomenological descriptions at the population-average level. The reviewed modeling approaches include Molecular Dynamics, Particle-Based Brownian Dynamics, the Master Equation approach, Ordinary Differential Equations, and the Boolean logic abstraction. Each of these frameworks is motivated by a particular biological context and the nature of the insight being pursued. The setting of gene network dynamic models from such frameworks involves assumptions and mathematical artifacts often ignored by the non-specialist. This article aims at providing an entry point for biologists new to the field and computer scientists not acquainted with some recent biophysically-inspired models of gene regulation. The connections promoting intuition between different abstraction levels and the role that approximations play in the modeling process are highlighted throughout the paper.</div>','<b>Title: </b>Mathematical model of TGF-beta signalling: feedback coupling is consistent with signal switching<br><b>Year: </b>2017<br><b>Author: </b> S. Khatibi,  Hj. Zhu,  J. Wagner,  Cw. Tan,  Jh. Manton and  Aw. Burgess<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/s12918-017-0421-5"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Transforming growth factor beta TGF-beta signalling regulates the development of embryos and tissue homeostasis in adults. In conjunction with other oncogenic changes, long-term perturbation of TGF-beta signalling is associated with cancer metastasis. Although TGF-beta signalling can be complex, many of the signalling components are well defined, so it is possible to develop mathematical models of TGF-beta signalling using reduction and scaling methods. The parameterization of our TGF-beta signalling model is consistent with experimental data.Results: We developed our mathematical model for the TGF-beta signalling pathway, i.e. the RF-model of TGF-beta signalling, using the "rapid equilibrium assumption" to reduce the network of TGF-beta signalling reactions based on the time scales of the individual reactions. By adding time-delayed positive feedback to the inherent time-delayed negative feedback for TGF-beta signalling. We were able to simulate the sigmoidal, switch-like behaviour observed for the concentration dependence of long-term >3 hours TGF-beta stimulation. Computer simulations revealed the vital role of the coupling of the positive and negative feedback loops on the regulation of the TGF-beta signalling system. The incorporation of time-delays for the negative feedback loop improved the accuracy, stability and robustness of the model. This model reproduces both the short-term and long-term switching responses for the intracellular signalling pathways at different TGF-beta concentrations. We have tested the model against experimental data from MEF mouse embryonic fibroblasts WT, SV40-immortalized MEFs and Gp130F/F MEFs. The predictions from the RF-model are consistent with the experimental data.Conclusions: Signalling feedback loops are required to model TGF-beta signal transduction and its effects on normal and cancer cells. We focus on the effects of time-delayed feedback loops and their coupling to ligand stimulation in this system. The model was simplified and reduced to its key components using standard methods and the rapid equilibrium assumption. We detected differences in short-term and long-term signal switching. The results from the RF-model compare well with experimental data and predict the dynamics of TGF-beta signalling in cancer cells with different mutations.</div>','<b>Title: </b>A sequestration feedback determines dynamics and temperature entrainment of the KaiABC circadian clock<br><b>Year: </b>2010<br><b>Author: </b> C. Brettschneider,  Rj. Rose,  S. Hertel,  Im. Axmann,  Ajr. Heck and  M. Kollmann<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1038/msb.2010.44"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>MOLECULAR SYSTEMS BIOLOGY<br><b>Cluster-Keywords: </b><em>circadian rhythms, systems biology, biological clock, circadian clock, biological rhythms, hpa axis, model, clock network, mathematical models, biological systems, dynamic systems, ode models, biology models, single cell, network, model parameters, experimental data, models, systems, dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>The circadian rhythm of the cyanobacterium Synechococcus elongatus is controlled by three proteins, KaiA, KaiB, and KaiC. In a test tube, these proteins form complexes of various stoichiometry and the average phosphorylation level of KaiC exhibits robust circadian oscillations in the presence of ATP. Using mathematical modeling, we were able to reproduce quantitatively the experimentally observed phosphorylation dynamics of the KaiABC clockwork in vitro. We thereby identified a highly non-linear feedback loop through KaiA inactivation as the key synchronization mechanism of KaiC phosphorylation. By using the novel method of native mass spectrometry, we confirm the theoretically predicted complex formation dynamics and show that inactivation of KaiA is a consequence of sequestration by KaiC hexamers and KaiBC complexes. To test further the predictive power of the mathematical model, we reproduced the observed phase synchronization dynamics on entrainment by temperature cycles. Our model gives strong evidence that the underlying entrainment mechanism arises from a temperature-dependent change in the abundance of KaiAC and KaiBC complexes. Molecular Systems Biology 6: 389; published online 13 July 2010; doi:10.1038/msb.2010.44 Subject Categories: simulation and data analysis; signal transduction</div>','<b>Title: </b>Dynamical modeling of uncertain interaction-based genomic networks<br><b>Year: </b>2015<br><b>Author: </b> Dn. Mohsenizadeh,  Jp. Hua,  M. Bittner and  Er. Dougherty<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1471-2105-16-S13-S3"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>noise<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: Most dynamical models for genomic networks are built upon two current methodologies, one process-based and the other based on Boolean-type networks. Both are problematic when it comes to experimental design purposes in the laboratory. The first approach requires a comprehensive knowledge of the parameters involved in all biological processes a priori, whereas the results from the second method may not have a biological correspondence and thus cannot be tested in the laboratory. Moreover, the current methods cannot readily utilize existing curated knowledge databases and do not consider uncertainty in the knowledge. Therefore, a new methodology is needed that can generate a dynamical model based on available biological data, assuming uncertainty, while the results from experimental design can be examined in the laboratory.Results: We propose a new methodology for dynamical modeling of genomic networks that can utilize the interaction knowledge provided in public databases. The model assigns discrete states for physical entities, sets priorities among interactions based on information provided in the database, and updates each interaction based on associated node states. Whenever uncertainty in dynamics arises, it explores all possible outcomes. By using the proposed model, biologists can study regulation networks that are too complex for manual analysis.Conclusions: The proposed approach can be effectively used for constructing dynamical models of interaction-based genomic networks without requiring a complete knowledge of all parameters affecting the network dynamics, and thus based on a small set of available data.</div>','<b>Title: </b>Modeling Logic Gene Networks by Means of Probabilistic Dynamic P Systems<br><b>Year: </b>2013<br><b>Author: </b> L. Valencia-Cabrera,  M. Garcia-Quismondo,  Mj. Perez-Jimenez,  Ys. Su,  H. Yu and  Lq. PanLink: No link<br><b>Journal: </b>INTERNATIONAL JOURNAL OF UNCONVENTIONAL COMPUTING<br><b>Cluster-Keywords: </b><em>coupling strength, synchronization transitions, timeperiodic coupling, time delay, logic network, distribution network, network model, oscillatory networks, neuron networks, water distribution, stochastic resonance, gene networks, multiple synchronization, firing synchronization, neuronal networks, firing behaviors, logical networks, results show, regulatory networks, cell polarization<div style="font-size:11px;"><br><b>Abstract: </b><em>Gene regulatory networks are useful models for biologists to understand the interactions among genes in living organisms. In this work, we consider a special class of gene regulatory networks, called logical networks, where the "links" among genes are inferred by logic analysis of phylogenetic profiles. We provide a formalization of logic networks, and introduce a membrane computing model for reconstructing logical networks and reproducing their dynamics. An example logical network is simulated by using software P-Lingua and MeCoSim. The simulation result shows that membrane computing is a versatile framework for modeling different biological scenarios.</div>','<b>Title: </b>FINDING AND ANALYZING THE MINIMUM SET OF DRIVER NODES IN CONTROL OF BOOLEAN NETWORKS<br><b>Year: </b>2016<br><b>Author: </b> Wp. Hou,  T. Tamura,  Wk. Ching and  T. Akutsu<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1142/S0219525916500065"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>ADVANCES IN COMPLEX SYSTEMS<br><b>Cluster-Keywords: </b><em>boolean network, boolean functions, regulatory networks, canalizing functions, network models, gene regulatory, boolean control, control networks, biological networks, boolean models, update functions, interaction networks, large network, regulatory functions, canalizing boolean, genetic regulatory, systems biology, driver nodes, networks bcns, biological systems<div style="font-size:11px;"><br><b>Abstract: </b><em>We study the minimum number of driver nodes control of which leads a Boolean network BN from an initial state to a target state in a specified number of time steps. We show that the problem is NP-hard and present an integer linear programming-based method that solves the problem exactly. We mathematically analyze the average size of the minimum set of driver nodes for random Boolean networks with bounded in-degree and with a small number of time steps. The results of computational experiments using randomly generated BNs show good agreements with theoretical analyses. A further examination in realistic BNs demonstrates the efficiency and generality of our theoretical analyses.</div>','<b>Title: </b>The ensemble approach to understand genetic regulatory networks<br><b>Year: </b>2004<br><b>Author: </b> S. Kauffman<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.physa.2004.05.018"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PHYSICA A-STATISTICAL MECHANICS AND ITS APPLICATIONS<br><b>Cluster-Keywords: </b><em>boolean networks, regulatory networks, network model, bayesian networks, model networks, systems biology, random boolean, real cells, immune system, genetic regulatory, networks, biochemical network, fluorescence images, cellular network, gene regulatory, cell differentiation, network, biological systems, ensemble approach, cell dynamics<div style="font-size:11px;"><br><b>Abstract: </b><em>Understanding the genetic regulatory network comprising genes, RNA, proteins and the network connections and dynamical control rules among them, is a major task of contemporary systems biology. I focus here on the use of the ensemble approach to find one or more well-defined ensembles of model networks whose statistical features match those of real cells and organisms. Such ensembles should help to explain and predict features of real cells and organisms. C 2004 Published by Elsevier B.V.</div>','<b>Title: </b>Simulated Evolution of Signal Transduction Networks<br><b>Year: </b>2012<br><b>Author: </b> M. Mobashir,  B. Schraven and  T. Beyer<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0050905"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>systems biology, regulatory network, interaction networks, network model, gene regulatory, network, network biology, networks, gene network, response networks, network dynamics, dynamic model, cell networks, gene expression, biological networks, system dynamics, dynamic network, model, biological systems, signaling networks<div style="font-size:11px;"><br><b>Abstract: </b><em>Signal transduction is the process of routing information inside cells when receiving stimuli from their environment that modulate the behavior and function. In such biological processes, the receptors, after receiving the corresponding signals, activate a number of biomolecules which eventually transduce the signal to the nucleus. The main objective of our work is to develop a theoretical approach which will help to better understand the behavior of signal transduction networks due to changes in kinetic parameters and network topology. By using an evolutionary algorithm, we designed a mathematical model which performs basic signaling tasks similar to the signaling process of living cells. We use a simple dynamical model of signaling networks of interacting proteins and their complexes. We study the evolution of signaling networks described by mass-action kinetics. The fitness of the networks is determined by the number of signals detected out of a series of signals with varying strength. The mutations include changes in the reaction rate and network topology. We found that stronger interactions and addition of new nodes lead to improved evolved responses. The strength of the signal does not play any role in determining the response type. This model will help to understand the dynamic behavior of the proteins involved in signaling pathways. It will also help to understand the robustness of the kinetics of the output response upon changes in the rate of reactions and the topology of the network.</div>','<b>Title: </b>An empirical Bayesian approach for model-based inference of cellular signaling networks<br><b>Year: </b>2009<br><b>Author: </b> Dj. Klinke<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1186/1471-2105-10-371"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BMC BIOINFORMATICS<br><b>Cluster-Keywords: </b><em>signaling network, systems biology, signal transduction, signaling pathways, receptor tyrosine, erbb signaling, erbb receptor, receptor signaling, intracellular signaling, mathematical models, cell signaling, computational model, factor receptor, receptor activation, growth factor, signaling models, networks signaling, receptor system, receptors systems, downstream signaling<div style="font-size:11px;"><br><b>Abstract: </b><em>Background: A common challenge in systems biology is to infer mechanistic descriptions of biological process given limited observations of a biological system. Mathematical models are frequently used to represent a belief about the causal relationships among proteins within a signaling network. Bayesian methods provide an attractive framework for inferring the validity of those beliefs in the context of the available data. However, efficient sampling of high-dimensional parameter space and appropriate convergence criteria provide barriers for implementing an empirical Bayesian approach. The objective of this study was to apply an Adaptive Markov chain Monte Carlo technique to a typical study of cellular signaling pathways.Results: As an illustrative example, a kinetic model for the early signaling events associated with the epidermal growth factor EGF signaling network was calibrated against dynamic measurements observed in primary rat hepatocytes. A convergence criterion, based upon the Gelman-Rubin potential scale reduction factor, was applied to the model predictions. The posterior distributions of the parameters exhibited complicated structure, including significant covariance between specific parameters and a broad range of variance among the parameters. The model predictions, in contrast, were narrowly distributed and were used to identify areas of agreement among a collection of experimental studies.Conclusion: In summary, an empirical Bayesian approach was developed for inferring the confidence that one can place in a particular model that describes signal transduction mechanisms and for inferring inconsistencies in experimental measurements.</div>','<b>Title: </b>Mesoscopic Models of Neurotransmission as Intermediates between Disease Simulators and Tools for Discovering Design Principles<br><b>Year: </b>2012<br><b>Author: </b> Eo. Voit,  Z. Qui and  S. Kikuchi<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1055/s-0032-1304653"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PHARMACOPSYCHIATRY<br><b>Cluster-Keywords: </b><em>systems biology, dynamic models, biological systems, model, dynamic systems, computational systems, network model, experimental data, model parameters, systems, systems model, models, pathway systems, data, biochemical pathway, dynamics model, biochemical networks, computational models, network, system<div style="font-size:11px;"><br><b>Abstract: </b><em>Two grand challenges have been declared as premier goals of computational systems biology. The first is the discovery of network motifs and design principles that help us understand and rationalize why biological systems are organized in the manner we encounter them rather than in a different fashion. The second goal is the development of computational models supporting the investigation of complex systems, in particular, as simulation platforms in personalized medicine and predictive health. Interestingly, most published systems models in biology contain between a handful and a few dozen variables. They are usually too complicated for systemic analyses of organizing principles, but they are at the same time too coarse to allow reliable simulations of diseases. While it may thus appear that the modeling efforts of the past have missed the declared targets of systems biology, we argue in this article that midsized mesoscopic models are excellent starting points for pursuing both goals in computational systems biology.</div>','<b>Title: </b>A Semiquantitative Framework for Gene Regulatory Networks: Increasing the Time and Quantitative Resolution of Boolean Networks<br><b>Year: </b>2015<br><b>Author: </b> J. Kerkhofs and  L. Geris<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1371/journal.pone.0130033"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PLOS ONE<br><b>Cluster-Keywords: </b><em>boolean network, network models, qualitative models, systems biology, mathematical modeling, boolean models, signaling networks, dynamic models, dynamic modeling, signaling dynamics, logic models, dynamic network, mathematical models, boolean modeling, signal transduction, regulatory networks, drug resistance, biological networks, network approach, network modeling<div style="font-size:11px;"><br><b>Abstract: </b><em>Boolean models have been instrumental in predicting general features of gene networks and more recently also as explorative tools in specific biological applications. In this study we introduce a basic quantitative and a limited time resolution to a discrete  Boolean framework. Quantitative resolution is improved through the employ of normalized variables in unison with an additive approach. Increased time resolution stems from the introduction of two distinct priority classes. Through the implementation of a previously published chondrocyte network and T helper cell network, we show that this addition of quantitative and time resolution broadens the scope of biological behaviour that can be captured by the models. Specifically, the quantitative resolution readily allows models to discern qualitative differences in dosage response to growth factors. The limited time resolution, in turn, can influence the reachability of attractors, delineating the likely long term system behaviour. Importantly, the information required for implementation of these features, such as the nature of an interaction, is typically obtainable from the literature. Nonetheless, a trade-off is always present between additional computational cost of this approach and the likelihood of extending the models scope. Indeed, in some cases the inclusion of these features does not yield additional insight. This framework, incorporating increased and readily available time and semiquantitative resolution, can help in substantiating the litmus test of dynamics for gene networks, firstly by excluding unlikely dynamics and secondly by refining falsifiable predictions on qualitative behaviour.</div>','<b>Title: </b>Dynamic Sequence Specific Constraint-Based Modeling of Cell-Free Protein Synthesis<br><b>Year: </b>2018<br><b>Author: </b> D. Dai,  N. Horvath and  J. Varner<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.3390/pr6080132"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>PROCESSES<br><b>Cluster-Keywords: </b><em>metabolic network, metabolic model, dynamic flux, growth rate, genomescale metabolic, flux balance, balance analysis, metabolic flux, metabolic pathways, dynamic model, metabolic reactions, cellular growth, regulatory network, systems biology, optimal growth, metabolic, modeling metabolic, growth conditions, dynamic analysis, cell growth<div style="font-size:11px;"><br><b>Abstract: </b><em>Cell-free protein expression has emerged as an important approach in systems and synthetic biology, and a promising technology for personalized point of care medicine. Cell-free systems derived from crude whole cell extracts have shown remarkable utility as a protein synthesis technology. However, if cell-free platforms for on-demand biomanufacturing are to become a reality, the performance limits of these systems must be defined and optimized. Toward this goal, we modeled E. coli cell-free protein expression using a sequence specific dynamic constraint-based approach in which metabolite measurements were directly incorporated into the flux estimation problem. A cell-free metabolic network was constructed by removing growth associated reactions from the iAF1260 reconstruction of K-12 MG1655 E. coli. Sequence specific descriptions of transcription and translation processes were then added to this metabolic network to describe protein production. A linear programming problem was then solved over short time intervals to estimate metabolic fluxes through the augmented cell-free network, subject to material balances, time rate of change and metabolite measurement constraints. The approach captured the biphasic cell-free production of a model protein, chloramphenicol acetyltransferase. Flux variability analysis suggested that cell-free metabolism was potentially robust; for example, the rate of protein production could be met by flux through the glycolytic, pentose phosphate, or the Entner-Doudoroff pathways. Variation of the metabolite constraints revealed central carbon metabolites, specifically upper glycolysis, tricarboxylic acid TCA cycle, and pentose phosphate, to be the most effective at training a predictive model, while energy and amino acid measurements were less effective. Irrespective of the measurement set, the metabolic fluxes for the most part remained unidentifiable. These findings suggested dynamic constraint-based modeling could aid in the design of cell-free protein expression experiments for metabolite prediction, but the flux estimation problem remains challenging. Furthermore, while we modeled the cell-free production of only a single protein in this study, the sequence specific dynamic constraint-based modeling approach presented here could be extended to multi-protein synthetic circuits, RNA circuits or even small molecule production.</div>','<b>Title: </b>Towards building the silicon cell: A modular approach<br><b>Year: </b>2006<br><b>Author: </b> Jl. Snoep,  F. Bruggeman,  Bg. Olivier and  Hv. Westerhoff<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1016/j.biosystems.2005.07.006"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>BIOSYSTEMS<br><b>Cluster-Keywords: </b><em>kinetic models, metabolic networks, kinetic parameters, dynamic models, systems biology, network model, dynamic metabolic, biochemical network, rate constants, enzyme kinetics, kinetic data, adjustable parameters, metabolic model, parameter estimation, dynamic optimization, model selection, mathematical model, kinetic network, model response, model<div style="font-size:11px;"><br><b>Abstract: </b><em>Systems Biology aims to understand quantitatively how properties of biological systems can be understood as functions of the characteristics of, and interactions between their macromolecular components. Whereas, traditional biochemistry focused on isolation and characterization Of cellular components, the challenge for Systems Biology lies in integration of this knowledge and the knowledge about molecular interactions. Computer models play an important role in this integration. We here discuss an approach with which we aim to link kinetic models on small pails of metabolism together, so as to form detailed kinetic models of larger chunks of metabolism, and ultimately of the entire living cell. Specifically, we will discuss techniques that can be used to model a sub-network in isolation of a larger network of which it is a part, while still maintaining the dynamics of the larger complete network. We will start by Outlining the JWS online system, the silicon cell project, and the type of models we propose. JWS online is a model repository, which can be used for the storage, simulation and analysis of kinetic models. We advocate to integrate a top-down approach, where measurements on the complete system are used to derive fluxes in a detailed structural model, with a bottom-up approach, consisting of the integration of molecular mechanism-based detailed kinetic models into the structural model. C 2005 Elsevier Ireland Ltd. All rights reserved.</div>','<b>Title: </b>Regeneration in an evolutionarily primitive brain - the planarian Dugesia japonica model<br><b>Year: </b>2011<br><b>Author: </b> Y. Umesono,  J. Tasaki,  K. Nishimura,  T. Inoue and  K. Agata<br><b>Link: </b> <a target="blank"  href="https://doi.org/10.1111/j.1460-9568.2011.07819.x"> Link registered: Click to get to a clickable link </a><br><b>Journal: </b>EUROPEAN JOURNAL OF NEUROSCIENCE<br><b>Cluster-Keywords: </b><em>stem cell, stem cells, systems biology, mathematical models, cell biology, network model, signaling pathways, cell fate, computational models, cancer cells, gene expression, network dynamics, embryonic stem, signaling network, prostate cancer, mathematical modeling, psychological stress, apoptosis signaling, computational systems, molecular mechanisms<div style="font-size:11px;"><br><b>Abstract: </b><em>A unique aspect of planarians is that they can regenerate a brain from somatic pluripotent stem cells called neoblasts, which have the ability to produce themselves self-renew and to give rise to all missing cell types during regeneration. Recent molecular studies have revealed that the planarian brain is composed of many distinct neuronal populations, which are evolutionarily and functionally conserved ones, and acts as an information-processing center to elicit distinct behavioral traits depending on a variety of signals arising from the external environment. How can planarians regenerate such a brain On the basis of our recent findings, here we review the cellular and molecular mechanisms that regulate the stem cell dynamics involved in the brain regeneration of the planarian Dugesia japonica. Our findings suggest the possible value of in vivo planarian studies for guiding regenerative medicine to treat neurodegenerative diseases via interlinking stem cell biology and regeneration biology.</div>'],
s: [1.3979400086720377,0.47712125471966244,1.2041199826559248,0.8450980400142568,0.9542425094393249,1.5797835966168101,1.591064607026499,0.3010299956639812,1.255272505103306,0.47712125471966244,0.0,1.5314789170422551,1.0791812460476249,0.8450980400142568,1.0413926851582251,0.9030899869919435,2.673020907128896,1.5185139398778875,1.146128035678238,0.6020599913279624,0.47712125471966244,1.3222192947339193,1.146128035678238,1.0413926851582251,1.3424226808222062,1.0791812460476249,2.3483048630481607,1.7853298350107671,0.9030899869919435,0.6989700043360189,1.0,0.0,1.4471580313422192,0.9030899869919435,0.3010299956639812,0.6989700043360189,1.3010299956639813,1.7781512503836436,1.3424226808222062,0.0,1.255272505103306,0.0,1.1760912590556813,1.3222192947339193,1.3222192947339193,1.1760912590556813,1.146128035678238,1.0413926851582251,1.2041199826559248,1.0,0.47712125471966244,0.3010299956639812,0.6020599913279624,2.4771212547196626,0.3010299956639812,0.0,0.0,2.0,1.2041199826559248,0.0,0.7781512503836436,0.0,1.2304489213782739,1.1139433523068367,1.4771212547196624,0.9030899869919435,1.255272505103306,0.6989700043360189,0.9030899869919435,0.9542425094393249,1.2787536009528289,1.1760912590556813,1.6020599913279623,0.0,1.7853298350107671,1.4313637641589874,0.3010299956639812,1.8325089127062364,0.3010299956639812,1.414973347970818,0.6020599913279624,0.0,1.5314789170422551,0.9030899869919435,0.9030899869919435,1.4471580313422192,2.0453229787866576,0.8450980400142568,0.8450980400142568,1.0791812460476249,0.7781512503836436,1.4313637641589874,1.1760912590556813,0.3010299956639812,1.0,0.6989700043360189,1.462397997898956,0.0,1.0,2.361727836017593,2.2227164711475833,0.0,1.1760912590556813,0.8450980400142568,1.2787536009528289,0.8450980400142568,2.662757831681574,1.5185139398778875,0.6020599913279624,1.2041199826559248,0.3010299956639812,0.9030899869919435,1.4471580313422192,0.6020599913279624,1.0413926851582251,0.8450980400142568,1.8920946026904804,1.414973347970818,1.0413926851582251,0.9542425094393249,0.6989700043360189,1.255272505103306,1.3010299956639813,1.0,1.662757831681574,1.724275869600789,1.3010299956639813,0.0,1.4913616938342726,0.9030899869919435,0.6020599913279624,1.146128035678238,1.7160033436347992,0.3010299956639812,0.0,0.0,1.662757831681574,0.9542425094393249,0.6020599913279624,1.0413926851582251,0.0,0.3010299956639812,0.0,1.8750612633917,1.1760912590556813,0.47712125471966244,0.9030899869919435,1.5797835966168101,0.9542425094393249,0.7781512503836436,0.3010299956639812,1.0791812460476249,1.1139433523068367,0.6020599913279624,0.0,1.5440680443502757,1.1139433523068367,1.3617278360175928,1.2304489213782739,0.3010299956639812,0.7781512503836436,1.1760912590556813,0.9030899869919435,1.9395192526186185,1.0791812460476249,1.8976270912904414,1.0,0.3010299956639812,0.47712125471966244,0.9030899869919435,1.414973347970818,1.8692317197309762,1.3222192947339193,0.8450980400142568,0.7781512503836436,1.0,0.47712125471966244,0.6020599913279624,0.8450980400142568,1.3222192947339193,0.3010299956639812,0.6020599913279624,1.6232492903979006,0.9542425094393249,1.4913616938342726,1.0,1.1760912590556813,2.357934847000454,0.7781512503836436,1.0,0.0,0.0,1.0413926851582251,1.6020599913279623,0.9030899869919435,1.146128035678238,0.3010299956639812,0.6989700043360189,1.255272505103306,0.3010299956639812,0.9030899869919435,1.0791812460476249,2.3502480183341627,0.7781512503836436,1.0,0.47712125471966244,2.0086001717619175,0.9030899869919435,0.47712125471966244,1.4771212547196624,2.3384564936046046,1.7160033436347992,0.47712125471966244,0.8450980400142568,0.0,1.1139433523068367,1.462397997898956,0.7781512503836436,0.8450980400142568,0.6020599913279624,1.5314789170422551,0.7781512503836436,0.8450980400142568,1.724275869600789,1.462397997898956,0.3010299956639812,0.47712125471966244,0.6989700043360189,0.47712125471966244,0.3010299956639812,1.0791812460476249,1.255272505103306,0.9542425094393249,1.8388490907372552,0.9542425094393249,1.462397997898956,1.462397997898956,0.6989700043360189,0.3010299956639812,0.6989700043360189,0.0,1.6989700043360187,0.0,0.6020599913279624,1.0413926851582251,0.0,1.4471580313422192,1.1760912590556813,0.6020599913279624,1.7323937598229686,1.5797835966168101,1.1139433523068367,0.9542425094393249,1.1139433523068367,0.47712125471966244,0.0,1.3979400086720377,0.0,0.9542425094393249,0.0,0.3010299956639812,0.9542425094393249,1.462397997898956,0.7781512503836436,1.146128035678238,1.591064607026499,0.7781512503836436,1.6901960800285136,0.7781512503836436,0.9030899869919435,1.6989700043360187,1.1760912590556813,0.0,1.7634279935629373,1.1760912590556813,0.0,1.1760912590556813,0.0,1.146128035678238,0.6989700043360189,0.47712125471966244,1.146128035678238,0.6020599913279624,1.2787536009528289,2.5428254269591797,0.0,1.5185139398778875,1.505149978319906,0.6020599913279624,0.3010299956639812,0.7781512503836436,0.6020599913279624,0.6020599913279624,1.568201724066995,1.4771212547196624,0.0,1.8195439355418688,1.724275869600789,1.1760912590556813,0.3010299956639812,1.4771212547196624,1.6901960800285136,0.0,0.7781512503836436,1.4771212547196624,0.47712125471966244,1.0,0.0,0.6020599913279624,0.3010299956639812,1.5563025007672873,1.7708520116421442,1.8808135922807914,0.6020599913279624,0.0,1.505149978319906,1.6434526764861874,0.6989700043360189,3.2873537727147464,1.0,0.6989700043360189,0.7781512503836436,0.0,0.0,1.380211241711606,0.8450980400142568,0.0,0.9030899869919435,1.3222192947339193,0.9030899869919435,1.0413926851582251,0.8450980400142568,1.4771212547196624,0.6020599913279624,0.47712125471966244,0.8450980400142568,0.47712125471966244,1.9084850188786497,1.4771212547196624,0.47712125471966244,1.1139433523068367,0.0,1.380211241711606,1.8388490907372552,0.3010299956639812,0.6989700043360189,0.6989700043360189,0.6989700043360189,0.0,2.1492191126553797,0.0,0.47712125471966244,0.9542425094393249,0.9030899869919435,1.2041199826559248,1.1760912590556813,0.0,1.380211241711606,1.1760912590556813,0.3010299956639812,1.7853298350107671,1.3617278360175928,1.0413926851582251,0.0,0.6989700043360189,0.0,1.380211241711606,0.9542425094393249,0.6020599913279624,1.7781512503836436,0.47712125471966244,1.6020599913279623,0.47712125471966244,1.5185139398778875,1.662757831681574,1.255272505103306,1.2041199826559248,1.0791812460476249,1.6989700043360187,0.3010299956639812,0.0,1.146128035678238,0.8450980400142568,0.47712125471966244,1.146128035678238,1.0791812460476249,1.380211241711606,2.100370545117563,0.7781512503836436,0.8450980400142568,0.9030899869919435,0.0,0.0,0.9542425094393249,0.3010299956639812,0.9542425094393249,0.47712125471966244,1.4313637641589874,1.255272505103306,0.3010299956639812,1.2304489213782739,0.3010299956639812,1.3617278360175928,1.6720978579357175,1.1760912590556813,1.1139433523068367,0.8450980400142568,0.9542425094393249,0.6020599913279624,0.6989700043360189,0.0,1.9731278535996986,0.3010299956639812,1.3222192947339193,1.6812412373755872,0.6989700043360189,2.250420002308894,1.6989700043360187,2.1367205671564067,0.6989700043360189,0.6989700043360189,0.47712125471966244,1.505149978319906,1.806179973983887,1.2304489213782739,0.3010299956639812,1.0413926851582251,1.968482948553935,0.6989700043360189,1.2787536009528289,0.6989700043360189,0.6020599913279624,1.505149978319906,1.146128035678238,1.4313637641589874,1.3617278360175928,1.3010299956639813,0.6989700043360189,0.0,1.5185139398778875,2.0211892990699383,0.8450980400142568,0.6989700043360189,1.462397997898956,1.5563025007672873,0.47712125471966244,0.3010299956639812,0.6020599913279624,1.2304489213782739,0.0,1.0791812460476249,1.6232492903979006,1.146128035678238,0.7781512503836436,0.47712125471966244,1.3010299956639813,0.7781512503836436,1.146128035678238,1.4771212547196624,0.9542425094393249,0.6989700043360189,0.0,0.7781512503836436,1.1139433523068367,1.0791812460476249,1.0791812460476249,2.164352855784437,0.6989700043360189,1.8573324964312685,1.0413926851582251,0.9030899869919435,1.2304489213782739,1.0791812460476249,1.1139433523068367,0.8450980400142568,1.3424226808222062,0.9030899869919435,1.6334684555795864,2.9556877503135057,1.414973347970818,1.0791812460476249,0.0,1.9867717342662448,0.9542425094393249,1.2787536009528289,1.4771212547196624,0.7781512503836436,0.3010299956639812,0.6989700043360189,1.0413926851582251,1.9590413923210936,1.255272505103306,0.47712125471966244,1.1760912590556813,1.3222192947339193,0.9030899869919435,0.9030899869919435,0.47712125471966244,0.6020599913279624,1.3010299956639813,1.3979400086720377,1.2787536009528289,1.1760912590556813,0.47712125471966244,0.6989700043360189,1.5797835966168101,0.0,1.3617278360175928,1.5563025007672873,0.7781512503836436,0.6020599913279624,1.4771212547196624,1.2787536009528289,0.6989700043360189,0.47712125471966244,1.4771212547196624,0.7781512503836436,1.3979400086720377,1.6127838567197355,1.3979400086720377,1.5185139398778875,1.255272505103306,1.1139433523068367,0.0,0.7781512503836436,1.1139433523068367,0.9030899869919435,0.7781512503836436,0.6020599913279624,0.6989700043360189,0.7781512503836436,1.3222192947339193,1.3617278360175928,0.0,1.505149978319906,1.1139433523068367,1.2304489213782739,0.9030899869919435,0.0,1.146128035678238,1.0791812460476249,1.255272505103306,0.8450980400142568,1.0791812460476249,1.7323937598229686,0.6020599913279624,0.9030899869919435,1.3010299956639813,1.255272505103306,1.2304489213782739,0.47712125471966244,0.7781512503836436,1.255272505103306,0.47712125471966244,1.3424226808222062,0.8450980400142568,1.4471580313422192,1.2304489213782739,1.380211241711606,1.2041199826559248,1.505149978319906,0.47712125471966244,0.7781512503836436,1.1760912590556813,0.3010299956639812,0.6020599913279624,1.462397997898956,1.3010299956639813,1.3979400086720377,1.0,0.9030899869919435,0.3010299956639812,1.6334684555795864,0.3010299956639812,1.5797835966168101,1.4313637641589874,0.9542425094393249,0.7781512503836436,2.155336037465062,0.8450980400142568,1.4771212547196624,0.3010299956639812,1.255272505103306,1.0791812460476249,1.1139433523068367,0.0,0.8450980400142568,0.7781512503836436,1.3617278360175928,0.3010299956639812,0.7781512503836436,1.0791812460476249,0.9030899869919435,1.1760912590556813,0.0,1.662757831681574,1.2304489213782739,1.0413926851582251,0.3010299956639812,0.47712125471966244,1.380211241711606,1.9395192526186185,0.8450980400142568,0.3010299956639812,1.5314789170422551,0.8450980400142568,2.0086001717619175,1.0791812460476249,0.9030899869919435,2.513217600067939,1.9395192526186185,1.3424226808222062,0.47712125471966244,0.3010299956639812,0.8450980400142568,0.6989700043360189,1.5314789170422551,1.380211241711606,1.2304489213782739,0.7781512503836436,0.9030899869919435,1.4313637641589874,1.1760912590556813,1.4471580313422192,0.6989700043360189,1.2787536009528289,0.0,0.0,0.47712125471966244,0.6989700043360189,1.0791812460476249,2.164352855784437,1.3010299956639813,1.0791812460476249,1.414973347970818,1.1760912590556813,1.1760912590556813,1.1760912590556813,1.1139433523068367,0.47712125471966244,1.7075701760979363,0.6989700043360189,1.6232492903979006,0.7781512503836436,1.2041199826559248,0.47712125471966244,0.9030899869919435,1.3979400086720377,0.8450980400142568,0.6989700043360189,0.0,0.0,0.8450980400142568,0.0,0.6989700043360189,1.3222192947339193,1.4913616938342726,0.47712125471966244,0.0,0.3010299956639812,1.6232492903979006,1.4771212547196624,1.0413926851582251,0.6020599913279624,0.47712125471966244,1.0413926851582251,1.6020599913279623,0.47712125471966244,0.47712125471966244,0.8450980400142568,2.113943352306837,1.2041199826559248,0.7781512503836436,2.2095150145426308,0.6989700043360189,0.0,1.6989700043360187,1.6020599913279623,1.462397997898956,0.6020599913279624,0.0,1.1760912590556813,0.47712125471966244,0.6989700043360189,1.7853298350107671,0.47712125471966244,1.6127838567197355,0.8450980400142568,0.0,0.8450980400142568,0.0,0.0,1.0,0.0,0.3010299956639812,0.3010299956639812,1.8195439355418688,0.9030899869919435,0.9030899869919435,1.7781512503836436,2.1702617153949575,1.2787536009528289,0.6020599913279624,0.3010299956639812,1.0791812460476249,0.0,1.146128035678238,1.3424226808222062,0.47712125471966244,0.9542425094393249,0.7781512503836436,0.3010299956639812,1.5440680443502757,0.6020599913279624,0.7781512503836436,0.47712125471966244,0.47712125471966244,0.3010299956639812,1.5314789170422551,0.3010299956639812,1.255272505103306,0.7781512503836436,0.9030899869919435,0.0,1.0,1.7708520116421442,1.255272505103306,1.0791812460476249,0.0,0.3010299956639812,1.146128035678238,0.0,1.2304489213782739,1.3010299956639813,1.0791812460476249,1.1139433523068367,0.6020599913279624,0.6020599913279624,1.146128035678238,0.0,0.0,1.1139433523068367,1.568201724066995,1.1139433523068367,0.7781512503836436,0.47712125471966244,0.47712125471966244,0.3010299956639812,1.3010299956639813,0.0,0.9542425094393249,1.6901960800285136,1.9242792860618816,1.5797835966168101,1.1139433523068367,1.0791812460476249,0.6020599913279624,0.3010299956639812,1.0413926851582251,1.4771212547196624,1.3979400086720377,0.6989700043360189,1.0,0.9030899869919435,1.0,0.47712125471966244,0.0,0.3010299956639812,1.146128035678238,0.9542425094393249,2.545307116465824,1.3222192947339193,1.3979400086720377,1.3424226808222062,0.47712125471966244,1.0791812460476249,1.2041199826559248,0.6989700043360189,0.0,0.9542425094393249,1.0413926851582251,0.9030899869919435,0.8450980400142568,0.7781512503836436,1.2787536009528289,0.3010299956639812,0.9542425094393249,0.8450980400142568,0.6020599913279624,0.7781512503836436,2.2576785748691846,1.1760912590556813,1.3617278360175928,1.4471580313422192,1.3617278360175928,1.3222192947339193,0.47712125471966244,0.47712125471966244,0.47712125471966244,0.8450980400142568,0.47712125471966244,2.0791812460476247,0.6020599913279624,1.380211241711606,0.6020599913279624,1.792391689498254,0.6989700043360189,0.7781512503836436,1.2041199826559248,0.47712125471966244,1.4471580313422192,1.4313637641589874,1.0,1.7634279935629373,0.8450980400142568,1.7634279935629373,1.6020599913279623,1.0791812460476249,2.0492180226701815,2.230448921378274,0.9030899869919435,1.6532125137753437,2.0086001717619175,0.47712125471966244,0.7781512503836436,1.9294189257142926,0.0,1.1139433523068367,1.146128035678238,1.7781512503836436,0.7781512503836436,0.0,1.1139433523068367,0.8450980400142568,0.0,0.0,0.8450980400142568,0.0,2.214843848047698,1.4771212547196624,1.0413926851582251,0.0,0.47712125471966244,0.8450980400142568,0.0,0.0,1.4313637641589874,1.2304489213782739,0.7781512503836436,0.3010299956639812,1.6334684555795864,0.7781512503836436,0.6020599913279624,0.9542425094393249,1.1760912590556813,1.0,0.6020599913279624,0.3010299956639812,0.6020599913279624,2.423245873936808,1.4913616938342726,1.3010299956639813,0.0,0.0,0.6989700043360189,1.0413926851582251,0.0,0.3010299956639812,0.6020599913279624,1.7323937598229686,1.462397997898956,1.6989700043360187,1.6901960800285136,0.9542425094393249,1.0791812460476249,2.733999286538387,1.8512583487190752,0.8450980400142568,1.0791812460476249,0.47712125471966244,2.089905111439398,1.3617278360175928,1.1139433523068367,0.6020599913279624,0.47712125471966244,1.3424226808222062,1.0413926851582251,0.7781512503836436,0.3010299956639812,1.3617278360175928,1.146128035678238,0.3010299956639812,0.0,0.0,2.184691430817599,1.4313637641589874,1.3222192947339193,1.6334684555795864,1.2787536009528289,1.0,1.380211241711606,1.2304489213782739,0.3010299956639812,0.47712125471966244,2.484299839346786,2.0718820073061255,0.8450980400142568,0.3010299956639812,0.8450980400142568,1.4313637641589874,1.919078092376074,0.47712125471966244,1.2041199826559248,1.9867717342662448,0.0,1.7781512503836436,0.47712125471966244,0.7781512503836436,1.1139433523068367,2.123851640967086,0.0,0.8450980400142568,0.6020599913279624,1.1139433523068367,0.47712125471966244,0.9542425094393249,0.7781512503836436,0.0,1.414973347970818,1.3979400086720377,1.146128035678238,0.7781512503836436,0.6020599913279624,0.3010299956639812,2.5658478186735176,0.6989700043360189,1.3617278360175928,0.47712125471966244,0.8450980400142568,1.3010299956639813,0.6020599913279624,0.9030899869919435,2.089905111439398,1.4913616938342726,0.9030899869919435,0.47712125471966244,0.9030899869919435,1.1139433523068367,0.6989700043360189,1.146128035678238,0.47712125471966244,0.47712125471966244,0.6989700043360189,1.2041199826559248,1.146128035678238,0.7781512503836436,1.4471580313422192,0.47712125471966244,0.9030899869919435,1.3979400086720377,1.2041199826559248,2.03342375548695,1.3617278360175928,1.8388490907372552,1.2787536009528289,0.6989700043360189,1.1760912590556813,1.2041199826559248,0.6989700043360189,0.9030899869919435,1.7853298350107671,1.1760912590556813,1.2304489213782739,1.1139433523068367,0.0,1.5185139398778875,1.146128035678238,1.5440680443502757,0.8450980400142568,0.6989700043360189,0.8450980400142568,2.0293837776852097,1.3010299956639813,1.2304489213782739,1.7853298350107671,1.6434526764861874,0.3010299956639812,1.0,0.6989700043360189,0.0,1.2787536009528289,0.0,0.9030899869919435,0.0,1.2787536009528289,0.47712125471966244,0.47712125471966244,1.7781512503836436,0.47712125471966244,0.0,0.8450980400142568,0.47712125471966244,0.6020599913279624,1.9637878273455553,0.6020599913279624,2.2095150145426308,0.0,0.9542425094393249,0.3010299956639812,0.9030899869919435,1.7481880270062005,0.0,0.0,0.7781512503836436,1.3222192947339193,0.0,1.9138138523837167,0.8450980400142568,1.3010299956639813,1.255272505103306,1.1139433523068367,1.5314789170422551,1.3979400086720377,1.414973347970818,1.724275869600789,1.863322860120456,1.414973347970818,0.9542425094393249,1.9030899869919435,1.3222192947339193,1.2041199826559248,1.146128035678238,0.47712125471966244,0.47712125471966244,0.47712125471966244,1.7160033436347992,0.8450980400142568,0.0,0.3010299956639812,1.6232492903979006,1.4913616938342726,1.146128035678238,1.146128035678238,1.0,0.9030899869919435,1.1139433523068367,0.0,2.2013971243204513,1.380211241711606,0.0,0.6020599913279624,0.9030899869919435,0.0,0.7781512503836436,1.6901960800285136,0.0,1.1760912590556813,0.0,1.6532125137753437,0.6989700043360189,1.6901960800285136,1.806179973983887,1.2787536009528289,0.9030899869919435,0.3010299956639812,0.6020599913279624,1.255272505103306,1.2787536009528289,0.3010299956639812,1.6720978579357175,0.47712125471966244,0.6020599913279624,0.0,1.4313637641589874,1.6532125137753437,1.255272505103306,1.2041199826559248,1.0791812460476249,1.0791812460476249,0.6020599913279624,0.6989700043360189,1.4771212547196624,0.8450980400142568,2.012837224705172,0.3010299956639812,1.0413926851582251,0.7781512503836436,0.47712125471966244,1.6020599913279623,0.9030899869919435,0.6020599913279624,0.6020599913279624,1.4913616938342726,1.0413926851582251,1.6020599913279623,1.2041199826559248,0.6989700043360189,0.3010299956639812,1.9030899869919435,1.5314789170422551],
colors: [
{
r: [31,247,255,176,106,149,91,233,193,228,151,31,0,156,23,0,172,233,176,176,175,80,23,156,34,176,233,87,176,184,255,176,184,255,91,176,151,216,87,216,233,91,176,149,15,176,255,0,0,149,91,255,23,80,232,149,233,0,151,91,87,0,228,116,96,175,156,47,255,255,176,87,34,184,176,193,57,28,149,47,96,255,184,102,80,176,91,163,233,193,184,175,151,216,15,0,47,96,102,34,106,1,80,87,232,176,87,156,31,176,28,216,103,23,0,176,151,176,176,23,176,15,0,0,28,34,193,34,103,106,232,247,47,176,114,176,15,91,255,240,0,255,103,15,103,176,0,232,228,47,176,47,47,31,28,91,176,15,87,0,87,31,163,176,255,176,80,102,0,87,130,247,193,242,80,34,130,47,176,0,34,102,0,91,176,242,176,193,176,203,116,15,165,0,176,163,233,0,1,255,203,203,193,156,184,176,255,255,107,149,31,91,47,240,165,15,151,184,255,96,31,255,247,255,216,233,176,242,0,116,106,233,87,193,176,130,87,176,176,176,151,43,80,184,28,203,233,176,255,156,228,96,247,255,31,102,193,57,176,255,28,242,176,34,149,0,233,34,176,149,151,240,242,103,149,184,233,247,232,233,34,176,151,175,106,15,0,216,151,184,57,156,80,103,107,114,193,176,23,216,106,91,96,184,87,232,34,255,176,176,149,0,176,47,23,176,255,156,255,28,247,156,114,1,1,176,0,228,57,156,149,91,0,176,0,176,114,156,255,149,0,151,247,163,176,47,102,34,176,247,31,151,165,43,247,255,28,176,31,193,107,87,255,176,23,216,106,233,228,96,203,149,28,193,149,87,0,0,31,0,149,0,216,80,184,233,176,176,149,247,156,255,47,216,28,216,15,176,228,149,91,203,255,0,228,23,116,91,28,232,103,156,43,176,149,176,240,193,15,116,255,184,175,0,165,184,107,34,103,47,184,247,176,255,15,176,96,149,0,242,193,255,87,34,57,228,28,163,233,255,149,203,149,28,96,91,247,114,106,255,0,176,176,102,184,163,23,0,176,203,216,176,233,176,87,116,163,176,255,176,184,203,106,80,87,31,176,247,176,47,233,184,176,1,255,34,247,28,23,247,247,255,87,103,149,176,240,96,28,156,203,31,91,151,193,116,102,176,176,247,233,255,31,176,255,1,87,193,216,255,176,172,176,47,156,255,87,233,151,149,193,228,176,57,247,176,31,233,156,240,255,23,232,151,176,233,193,103,0,232,240,151,31,151,184,15,15,240,87,15,149,15,31,176,149,47,247,34,28,176,176,176,31,31,43,114,255,43,176,176,31,176,57,176,149,149,203,163,34,107,43,0,216,31,34,0,165,156,203,87,1,47,232,28,102,23,23,228,23,176,151,151,15,193,165,87,232,242,216,247,176,176,184,31,87,233,156,23,176,233,87,176,106,31,255,255,28,31,151,176,0,163,255,255,255,176,23,130,0,172,91,15,0,23,255,176,116,47,247,28,107,149,23,43,156,165,163,57,233,15,176,255,103,0,87,34,255,91,176,28,149,87,255,87,176,43,130,240,0,233,91,0,0,176,255,91,184,116,193,34,151,233,23,149,31,34,28,130,23,130,184,31,31,114,149,103,31,102,151,240,28,184,103,255,15,0,57,15,176,31,228,255,0,130,247,23,28,34,149,102,176,103,233,184,31,172,28,149,43,156,255,149,172,31,43,176,80,149,114,0,57,184,87,232,149,87,175,176,130,91,116,0,233,216,43,15,175,203,156,176,31,176,151,102,184,15,176,31,0,31,149,184,255,0,255,23,233,175,91,114,255,242,31,216,247,34,114,15,87,255,247,0,116,15,176,176,172,163,23,255,87,163,149,15,116,255,163,0,28,232,0,176,175,0,87,232,15,176,156,57,184,156,43,43,175,102,176,176,47,176,149,232,31,0,151,102,156,0,149,176,57,130,103,102,255,103,1,31,87,216,87,242,102,96,156,247,247,193,0,87,255,176,0,176,175,228,87,103,91,31,176,156,23,28,156,0,176,172,0,156,165,107,1,165,28,149,23,176,176,116,0,163,0,255,233,15,91,0,156,15,203,203,91,228,43,0,184,15,163,172,193,255,149,0,233,0,232,103,176,176,233,156,156,31,176,91,255,151,176,28,247,176,176,233,1,107,176,23,255,233,193,203,23,176,176,247,149,116,28,165,151,57,176,31,176,193,228,184,116,176,232,28,130,184,34,193,255,176,34,0,28,15,43,228,176,87,149,165,193,233,149,15,255,156,0,47,1,31,255,247,151,255,172,193,176,0,193,23,176,176,176,31,149,176,31,176,31,130,28,216,163,0,114,91,184,165,255,176,130,57,247,0,130,149,175,102,107,149,107,151,176,228,242,47,91,0,107,149,176,87,106,175,163,176,149,31,176,80,176,255,149,163,28,28,47,102,255,176,255,176,172,151,151,31,255,156,0,103,23,47],
g: [0,129,151,156,62,57,138,202,114,107,230,0,0,184,13,0,115,202,156,156,131,8,13,184,161,156,202,48,156,8,114,156,8,114,68,156,230,135,1,135,202,138,156,57,175,156,119,0,0,158,138,151,13,8,0,57,202,0,230,138,1,41,107,59,4,131,184,31,114,67,156,48,161,189,156,65,57,214,57,31,4,151,8,79,8,156,138,124,202,65,189,131,230,135,175,0,31,4,79,161,62,135,8,48,0,156,1,184,0,156,214,135,70,13,115,156,230,156,156,13,156,175,115,0,214,161,114,161,70,62,0,129,31,156,255,156,175,138,151,132,0,119,70,175,70,156,0,0,107,31,156,31,31,0,214,138,156,175,48,115,48,0,124,156,114,156,8,79,124,48,0,129,65,131,8,161,0,31,156,115,161,79,115,138,156,131,156,65,156,141,59,175,114,115,156,124,202,124,135,119,141,141,65,184,8,156,67,114,130,57,0,68,31,132,114,175,230,8,67,4,0,114,129,119,135,202,156,131,0,59,62,202,1,114,156,0,1,156,156,156,230,190,8,189,214,141,202,156,114,184,107,4,129,67,0,79,65,57,156,119,214,131,156,161,158,0,202,161,156,57,230,132,131,70,57,8,202,129,0,202,161,156,230,131,62,175,0,135,230,8,57,186,8,70,130,255,65,156,13,135,62,138,4,8,1,0,161,151,156,156,158,0,156,31,13,156,151,184,67,214,129,184,255,135,135,156,115,107,57,186,57,68,0,156,0,156,255,186,151,158,41,230,129,124,156,31,79,161,156,129,0,230,114,190,129,151,214,156,0,65,130,1,151,156,13,135,62,202,107,4,141,158,214,114,158,48,115,0,0,115,57,41,135,8,8,202,156,156,57,129,186,67,31,135,214,135,175,156,107,57,68,141,114,124,107,13,59,138,214,0,70,186,190,156,158,156,132,65,175,59,114,189,131,41,114,189,130,161,70,31,189,129,156,114,175,156,4,158,41,131,65,67,48,161,57,107,214,124,202,119,158,141,57,214,4,138,129,255,62,114,0,156,156,79,8,124,13,0,156,141,135,156,202,156,48,59,124,156,67,156,8,141,62,8,48,0,156,129,156,31,202,189,156,135,119,161,129,214,13,129,129,119,48,70,57,156,132,4,214,184,141,0,138,230,114,59,79,156,156,129,202,67,0,156,119,135,1,114,135,114,156,115,156,31,184,67,1,202,230,158,114,107,156,57,129,156,0,202,184,132,151,13,0,230,156,202,65,70,41,0,132,230,0,230,8,175,175,132,1,175,158,175,0,156,158,31,129,161,214,156,156,156,0,0,190,255,114,190,156,156,0,156,57,156,57,158,141,124,161,130,190,0,135,0,161,124,114,186,141,1,135,31,0,214,79,13,13,107,13,156,230,230,175,65,114,1,0,131,135,129,156,156,8,0,1,202,186,13,156,202,1,156,62,0,151,114,214,0,230,156,124,124,67,114,119,156,13,0,0,115,68,175,124,13,114,156,59,31,129,214,130,158,13,190,186,114,124,57,202,175,156,151,70,0,48,161,114,138,156,214,158,48,114,1,156,190,0,132,0,202,68,115,124,156,114,138,189,59,65,161,230,202,13,57,0,161,214,0,13,0,189,0,0,255,158,70,0,79,230,132,214,8,70,67,175,0,57,175,156,0,107,151,0,0,129,13,214,161,158,79,156,70,202,189,0,115,214,158,190,186,114,57,115,0,190,156,8,158,255,115,57,189,48,0,57,1,131,156,0,138,59,124,202,135,190,175,131,141,186,156,0,156,230,79,8,175,156,0,0,0,57,8,151,0,114,13,202,131,68,255,119,131,0,135,129,161,255,175,1,67,129,41,59,175,156,156,115,124,13,119,1,124,57,175,59,119,124,0,214,0,124,156,131,41,48,0,175,156,186,57,8,186,190,190,131,79,156,156,31,156,158,0,0,41,230,79,184,0,158,156,57,0,70,79,114,70,135,0,48,135,48,131,79,4,186,129,129,65,0,1,67,156,124,156,131,107,1,70,138,0,156,184,13,214,186,41,156,115,0,184,114,130,135,114,214,57,13,156,156,59,0,124,115,114,202,175,138,0,184,175,141,141,138,107,190,41,189,175,124,115,114,114,158,0,202,124,0,70,156,156,202,184,184,0,156,68,119,230,156,214,129,156,156,202,135,130,156,13,119,202,114,141,13,156,156,129,57,59,214,114,230,57,156,0,156,114,107,189,59,156,0,214,0,189,161,65,67,156,161,124,214,175,190,107,156,1,57,114,114,202,158,175,67,184,115,31,135,0,114,129,230,67,115,65,156,41,65,13,156,156,156,0,158,156,0,156,0,0,214,135,124,115,255,68,8,114,151,156,0,57,129,124,0,57,131,79,130,158,130,230,156,107,131,31,68,0,130,158,156,48,62,131,124,156,158,0,156,8,156,151,158,124,214,214,31,79,119,156,114,156,115,230,230,0,151,184,0,70,13,31],
b: [232,228,170,132,0,255,51,0,135,1,214,232,156,68,204,156,104,0,132,132,104,0,204,68,0,132,0,0,132,249,255,132,249,255,241,132,214,1,170,1,0,51,132,255,172,132,0,156,156,253,51,170,204,0,0,255,0,156,214,51,170,247,1,100,63,104,68,0,255,161,132,0,0,172,132,0,6,158,255,0,63,170,249,0,0,132,51,0,0,0,172,104,214,1,172,156,0,63,0,0,0,0,0,0,0,132,170,68,232,132,158,1,0,204,98,132,214,132,132,204,132,172,98,156,158,0,135,0,0,0,0,228,0,132,255,132,172,51,170,208,156,0,0,172,0,132,156,0,1,0,132,0,0,232,158,51,132,172,0,98,0,232,0,132,255,132,0,0,226,0,12,228,0,199,0,0,12,0,132,98,0,0,98,51,132,199,132,0,132,184,100,172,104,98,132,0,0,226,0,0,184,184,0,68,249,132,161,255,130,255,232,241,0,208,104,172,214,249,161,63,232,255,228,0,1,0,132,199,156,100,0,0,170,135,132,12,170,132,132,132,214,126,0,172,158,184,0,132,255,68,1,63,228,161,232,0,0,6,132,0,158,199,132,0,253,156,0,0,132,255,214,208,199,0,255,249,0,228,0,0,0,132,214,104,0,172,156,1,214,249,6,209,0,0,130,255,0,132,204,1,0,51,63,249,170,0,0,170,132,132,253,156,132,0,204,132,170,68,161,158,228,68,255,0,0,132,98,1,6,209,255,241,156,132,156,132,255,209,170,253,247,214,228,0,132,0,0,0,132,228,232,214,104,126,228,170,158,132,232,0,130,170,170,132,204,1,0,0,1,63,184,253,158,135,253,0,98,156,232,98,255,247,1,0,249,0,132,132,255,228,209,161,0,1,158,1,172,132,1,255,241,184,255,226,1,204,100,51,158,0,0,209,126,132,253,132,208,0,172,100,255,172,104,247,104,172,130,0,0,0,172,228,132,255,172,132,63,253,247,199,0,161,0,0,6,1,158,0,0,0,253,184,255,158,63,51,228,255,0,255,156,132,132,0,249,0,204,156,132,184,1,132,0,132,0,100,0,132,161,132,249,184,0,0,0,232,132,228,132,0,0,172,132,0,0,0,228,158,204,228,228,0,0,0,255,132,208,63,158,68,184,232,51,214,135,100,0,132,132,228,0,161,232,132,0,0,170,135,1,255,132,104,132,0,68,161,170,0,214,253,135,1,132,6,228,132,232,0,68,208,170,204,0,214,132,0,0,0,247,0,208,214,232,214,249,172,172,208,170,172,253,172,232,132,253,0,228,0,158,132,132,132,232,232,126,255,255,126,132,132,232,132,6,132,255,253,184,0,0,130,126,156,1,232,0,226,104,209,184,170,0,0,0,158,0,204,204,1,204,132,214,214,172,0,104,170,0,199,1,228,132,132,249,232,170,0,209,204,132,0,170,132,0,232,170,255,158,232,214,132,226,0,161,255,0,132,204,12,156,104,241,172,226,204,255,132,100,0,228,158,130,253,204,126,209,104,0,6,0,172,132,170,0,156,0,0,255,51,132,158,253,0,255,170,132,126,12,208,156,0,241,98,226,132,255,51,172,100,0,0,214,0,204,255,232,0,158,12,204,12,172,232,232,255,253,0,232,0,214,208,158,249,0,161,172,156,6,172,132,232,1,170,156,12,228,204,158,0,253,0,132,0,0,172,232,104,158,253,126,209,255,255,104,232,126,132,0,253,255,98,6,172,0,0,255,170,104,132,12,51,100,226,0,1,126,172,104,184,209,132,232,132,214,0,249,172,132,232,156,232,255,249,170,156,255,204,0,104,241,255,0,199,232,1,228,0,255,172,170,161,228,247,100,172,132,132,104,0,204,0,170,0,255,172,100,0,0,156,158,0,226,132,104,247,0,0,172,132,209,6,249,209,126,126,104,0,132,132,0,132,253,0,232,247,214,0,68,156,253,132,6,12,0,0,255,0,0,232,0,1,0,199,0,63,209,228,228,0,156,170,161,132,226,132,104,1,170,0,51,232,132,68,204,158,209,247,132,104,156,68,104,130,0,104,158,255,204,132,132,100,156,0,98,255,0,172,51,156,68,172,184,184,51,1,126,247,172,172,0,104,135,255,253,156,0,226,0,0,132,132,0,68,68,232,132,241,0,214,132,158,228,132,132,0,0,130,132,204,0,0,135,184,204,132,132,228,255,100,158,104,214,6,132,232,132,135,1,172,100,132,0,158,12,172,0,0,161,132,0,226,158,172,126,1,132,170,255,104,135,0,253,172,161,68,98,0,0,232,255,228,214,161,104,0,132,247,0,204,132,132,132,232,253,132,232,132,232,12,158,1,0,98,255,241,249,104,170,132,12,6,228,226,12,255,104,0,130,253,130,214,132,1,199,0,241,156,130,253,132,0,0,104,0,132,253,232,132,0,132,170,253,0,158,158,0,0,0,132,255,132,104,214,214,232,170,68,156,0,204,0],
},
]},
};

    </script>
    
    <script>
      class Faerun {
        constructor() {
          this.body = document.getElementsByTagName('body')[0];

          this.selectedItems = [];
          this.selectedIndicators = [];
          this.selectedCurrent = [];
          this.scatterMeta = [{"categorical": [true], "fog_intensity": 1, "has_legend": false, "interactive": true, "is_range": [false], "label_index": [0], "legend": [[]], "legend_title": ["econ"], "mapping": {"c": "c", "cs": "cs", "labels": "labels", "s": "s", "x": "x", "y": "y", "z": "z"}, "max_c": [53.0], "max_legend_label": ["53.00"], "max_point_size": 30, "min_c": [-1.0], "min_legend_label": ["-1.00"], "name": "econ", "ondblclick": [null], "point_scale": 3.2, "selected_labels": [null], "series_title": [null], "shader": "circle", "title_index": [0]}];
          this.treeMeta = [];
          this.seriesState = {};
          this.el = {};

          this.currentPoint = null;

          this.lore = null;
          this.clearColorHex = '#2e1413';
          this.clearColor = null;
          this.view = 'front';
          this.antiAliasing = true;
          this.alphaBlending = true;

          this.treeHelpers = [];
          this.pointHelpers = [];
          this.octreeHelpers = [];
          this.coordinatesHelper = null;

          this.min = [Number.MAX_VALUE, Number.MAX_VALUE, Number.MAX_VALUE];
          this.max = [-Number.MAX_VALUE, -Number.MAX_VALUE, -Number.MAX_VALUE];
          this.maxRadius = -Number.MAX_VALUE;

          this.octreeHelpers = [];
          this.ohIndexToPhName = [];
          this.ohIndexToPhIndex = [];
          this.phIndexMap = {};
          this.ohIndexMap = {};

          this.coords = {
            show: false,
            grid: false,
            ticks: true,
            tickCount: 10,
            tickLength: 2.0,
            color: '#888888',
            box: false,
            offset: 5.0
          };

          this.legend = {
            show: false,
            title: 'Legend'
          };

          this.el = Faerun.bindElements();

          this.scatterMeta.forEach(s => {
            this.seriesState[s.name] = 0;
          });

          this.clearColor = Lore.Core.Color.fromHex(this.clearColorHex);
          this.alphaBlending = (this.view === 'free' ? false : true) || this.alphaBlending;

          this.initLore();
          this.initTreeHelpers();
          this.initPointHelpers();
          this.initCoords();
          this.initAxes();
          this.initView();
          this.initEvents();
          this.renderLegend();
        }

        initLore() {
          this.lore = Lore.init('lore', {
            antialiasing: this.antiAliasing,
            clearColor: this.clearColorHex,
            alphaBlending: this.alphaBlending
          });
        }

        initTreeHelpers() {
          this.treeMeta.forEach(t => {
            let th = new Lore.Helpers.TreeHelper(this.lore, t.name, 'tree');
            th.setXYZHexS(data[t.name].x, data[t.name].y, data[t.name].z, t.color);
            th.setFog([this.clearColor.components[0], this.clearColor.components[1], 
                       this.clearColor.components[2], this.clearColor.components[3]],
                      t.fog_intensity);
            this.treeHelpers.push(th);
          });
        }

        initPointHelpers() {
          this.scatterMeta.forEach(s => {           
            let ph = new Lore.Helpers.PointHelper(
              this.lore, s.name, s.shader, { maxPointSize: s.max_point_size }
            );

            ph.setXYZRGBS(data[s.name].x, data[s.name].y, data[s.name].z, 
                          data[s.name]['colors'][0].r, data[s.name]['colors'][0].g, 
                          data[s.name]['colors'][0].b, data[s.name].s);

            ph.setPointScale(s.point_scale);
            ph.setFog([this.clearColor.components[0], this.clearColor.components[1],
                       this.clearColor.components[2], this.clearColor.components[3]],
                      s.fog_intensity)

            this.phIndexMap[s.name] = this.pointHelpers.length;
            this.pointHelpers.push(ph);

            this.min[0] = Faerun.getMin(data[s.name].x, this.min[0]);
            this.min[1] = Faerun.getMin(data[s.name].y, this.min[1]);
            this.min[2] = Faerun.getMin(data[s.name].z, this.min[2]);
            this.max[0] = Faerun.getMax(data[s.name].x, this.max[0]);
            this.max[1] = Faerun.getMax(data[s.name].y, this.max[1]);
            this.max[2] = Faerun.getMax(data[s.name].z, this.max[2]);
            this.maxRadius = ph.getMaxRadius();
    
            if (s.interactive && data[s.name].labels) {
              this.octreeHelpers.push(
                new Lore.Helpers.OctreeHelper(this.lore, 'Octree_' + s.name, 'tree', ph)
              );

              this.ohIndexMap[s.name] = this.octreeHelpers.length - 1;
              this.ohIndexToPhName.push(s.name);
              this.ohIndexToPhIndex.push(this.phIndexMap[s.name]);
            }
          });
        }

        initCoords() {
          if (!this.coords.show) return;

          let min = [0, 0, 0];
          let max = [0, 0, 0];

          for (var i = 0; i < 3; i++) {
            min[i] = this.min[i] - this.coords.offset;
            max[i] = this.max[i] + this.coords.offset;
          }

          this.coordinatesHelper = new Lore.Helpers.CoordinatesHelper(this.lore, 'Coordinates', 'coordinates', {
            position: new Lore.Math.Vector3f(min[0], min[1], min[2]),
            axis: {
              x: {
                length: max[0] - min[0],
                color: Lore.Core.Color.fromHex(this.coords.color)
              },
              y: {
                length: max[1] - min[1],
                color: Lore.Core.Color.fromHex(this.coords.color)
              },
              z: {
                length: max[2] - min[2],
                color: Lore.Core.Color.fromHex(this.coords.color)
              }
            },
            ticks: {
              enabled: this.coords.ticks,
              x: {
                length: this.coords.tickLength,
                color: Lore.Core.Color.fromHex(this.coords.color),
                count: this.coords.tickCount
              },
              y: {
                length: this.coords.tickLength,
                color: Lore.Core.Color.fromHex(this.coords.color),
                count: this.coords.tickCount
              },
              z: {
                length: this.coords.tickLength,
                color: Lore.Core.Color.fromHex(this.coords.color),
                count: this.coords.tickCount
              }
            },
            box: {
              enabled: this.coords.box,
              x: {
                color: Lore.Core.Color.fromHex(this.coords.color)
              },
              y: {
                color: Lore.Core.Color.fromHex(this.coords.color)
              },
              z: {
                color: Lore.Core.Color.fromHex(this.coords.color)
              }
            }
          });
        }

        initAxes() {
          // Wait for DOM to get ready
          setTimeout(() => {
            this.updateTitle(true);
            this.updateXAxis(true);
            this.updateYAxis(true);
          }, 500);
        }

        initView() {
          let center = new Lore.Math.Vector3f(
              (this.max[0] + this.min[0]) / 2.0, 
              (this.max[1] + this.min[1]) / 2.0, 
              (this.max[2] + this.min[2]) / 2.0
            );
          this.lore.controls.setLookAt(center);
          this.lore.controls.setRadius(this.maxRadius + 100);
          this.lore.controls.setView(0.9, -0.5)
          this.lore.controls.setViewByName(this.view);
        }

        initEvents() {
          this.lore.controls.addEventListener('updated', () => {
            // Update the position / content of the annotations every time
            // the view changes
            this.updateTitle();
            this.updateYAxis();
            this.updateXAxis();
            this.updateSelectedIndicators();
          });

          Lore.Helpers.OctreeHelper.joinHoveredChanged(this.octreeHelpers, e => {
            let phName = this.ohIndexToPhName[e.source];
            if (e.e && data[phName].labels) {
              let fullLabel = data[phName].labels[e.e.index];
              let labelIndex = this.scatterMeta[this.ohIndexToPhIndex[e.source]]
                                  .label_index[this.seriesState[phName]];

              let rgbColor = this.pointHelpers[e.source].getColor(e.e.index);
              let hexColor = Lore.Core.Color.rgbToHex(rgbColor[0], rgbColor[1], rgbColor[2]);

              this.currentPoint = {
                index: e.e.index,
                fullLabel: fullLabel,
                source: phName,
                label: fullLabel.split('__')[labelIndex],
                color: hexColor
              }
              
              this.setTipContent();
              this.el.tip.classList.add('show');

              let pointSize = this.pointHelpers[e.source].getPointSize() / window.devicePixelRatio;
              let x = e.e.screenPosition[0];
              let y = e.e.screenPosition[1];

              this.el.hoverIndicator.style.width = pointSize + 'px';
              this.el.hoverIndicator.style.height = pointSize + 'px';
              this.el.hoverIndicator.style.left = (x - pointSize / 2.0 - 1) + 'px';
              this.el.hoverIndicator.style.top = (y - pointSize / 2.0 - 1) + 'px';

              this.el.hoverIndicator.classList.add('show');
            } else {
              this.currentPoint = null;
              this.el.tip.classList.remove('show');
              this.el.hoverIndicator.classList.remove('show');
            }
          });

          Lore.Helpers.OctreeHelper.joinSelectedChanged(this.octreeHelpers, items => {
            this.selectedItems = items;
            this.updateSelected();
          });

          Lore.Helpers.OctreeHelper.joinReselected(this.octreeHelpers, item => {
            this.updateSelected(
              this.getSelectedIndex(item[0].source, item[0].item.e.index)
            );
          });

          // Event listeners
          this.el.selectedPrev.addEventListener('click', e => {
            e.preventDefault();
            this.updateSelected(this.selectedCurrent - 1);
            return false;
          });

          this.el.selectedNext.addEventListener('click', e => {
            e.preventDefault();
            this.updateSelected(this.selectedCurrent + 1);
            return false;
          });

          document.addEventListener('dblclick', e => {
            if (this.currentPoint) {
              var index = this.currentPoint.index;
              var labels = this.currentPoint.label.split('__');
              var source = this.currentPoint.source;
              eval(this.scatterMeta[this.phIndexMap[source]].ondblclick[this.seriesState[source]]);
            }
          });

          document.addEventListener('mousemove', e => {
            let x = e.clientX;
            let y = e.clientY;

            if (x > window.innerWidth - this.el.tip.offsetWidth - 20) {
              x -= this.el.tip.offsetWidth;
            } else {
              x += 10;
            }

            if (y > window.innerHeight - this.el.tip.offsetHeight - 20) {
              y -= this.el.tip.offsetHeight;
            } else {
              y += 10;
            }

            if (this.el.tip) {
              this.el.tip.style.top = y + 'px';
              this.el.tip.style.left = x + 'px';
            }
          });

          this.el.selectedToggle.addEventListener('click', e => {
            this.el.selectedContainer.classList.toggle('hide');
            if (this.el.selectedContainer.classList.contains('hide'))
              this.el.selectedToggle.innerHTML = '<i class="fas fa-toggle-off"></i>';
            else
              this.el.selectedToggle.innerHTML = '<i class="fas fa-toggle-on"></i>';
          });
          
          this.el.showControls.addEventListener('click', e => {
            this.el.moreControls.classList.toggle('hide');
            e.preventDefault();
            return false;
          });

          this.el.search.addEventListener('click', e => {
            this.search();
            e.preventDefault();
            return false;
          });

          window.addEventListener('keydown', e => {
            if ((e.keyCode == 114) || (e.ctrlKey && e.keyCode == 70)) {
              this.search();
              e.preventDefault();
              return false;
            }
          })
        }

        setTipContent() {
          this.el.tipText.innerHTML = this.currentPoint.label;
          this.el.tip.style.borderColor = this.currentPoint.color;
        }

        setSelectedContent(fullLabel, labelIndex, selectedLabels) {
          this.el.selectedContainer.innerHTML = '';
          fullLabel.forEach((l, i) => {
            if (i === labelIndex) return;
            if (selectedLabels && selectedLabels[i]) {
              this.el.selectedContainer.appendChild(
                Faerun.createElement('div', { classes: 'label', content: selectedLabels[i] })
              );
            }
            this.el.selectedContainer.appendChild(
              Faerun.createElement('div', { classes: 'content', content: l })
            );
          });

          // Update the indicator
          this.updateSelectedIndicators();
        }

        renderLegend() {
          if (!this.legend.show) return;

          let legend = document.getElementById('legend');
          
          if (legend) this.body.removeChild(legend);

          legend = Faerun.createElement('div', { id: 'legend' });
          this.body.appendChild(legend)
          
          if (this.legend.title && this.legend.title !== '')
            legend.appendChild(Faerun.createElement('h2', { content: 'Legend' }));
          
          let container = Faerun.createElement('div', { classes: 'container' });
          legend.appendChild(container);

          this.scatterMeta.forEach(s => {
            let index = this.seriesState[s.name];
            if (s.has_legend) {
              let legendSection = []
              if (!s.is_range[index]) {
                s.legend[index].forEach(v => {
                  legendSection.push(Faerun.createElement('div', { classes: 'legend-element' }, [
                    Faerun.createColorBox(v[0]),
                    Faerun.createElement('div', { classes: 'legend-label', content: v[1] }),
                  ]))
                })
              } else {
                legendSection.push(Faerun.createElement('div', { classes: 'legend-element-range' }, [
                  ...Faerun.createColorScale(s.legend[index]),
                  Faerun.createElement('div', { 
                    classes: 'legend-label max', 
                    content: s.max_legend_label[index] 
                  }),
                  Faerun.createElement('div', {
                    classes: 'legend-label min',
                    content: s.min_legend_label[index] 
                  })
                ]))
              }

              let series = [];
              for (var i = 0; i < s.series_title.length; i++) {
                series.push(
                  Faerun.createElement('option', { 
                    value: i, 
                    content: s.series_title[i], 
                    selected: i === index
                  })
                );
              }


              let sectionHeader = Faerun.createElement(
                'h3', { content: s.legend_title[index] }
              );
              sectionHeader.addEventListener('click', e => {
                this.toggleLegendSection(s.name);
              });

              let seriesSelector = Faerun.createElement(
                'select', 
                { 
                  id: `select-${s.name}`, 
                  classes: 'series-selector',
                  'data-name': s.name,
                  hidden: s.series_title.length < 2,
                }, 
                [ ...series ]
              );
              seriesSelector.addEventListener('change', e => {
                let value = document.getElementById(`select-${s.name}`).value;
                this.changeSeries(value, s.name);
              });


              container.appendChild(
                Faerun.createElement(
                  'div', { id: `legend-${s.name}`, 
                  classes: 'legend-section', 
                  'data-name': `${s.name}` },
                  [ sectionHeader, seriesSelector, ...legendSection ]
                )
              );
            }
          });
        }

        toggleLegendSection(name) {
          let section = document.getElementById('legend-' + name);
          let geometry = this.pointHelpers[this.phIndexMap[name]].geometry;
          let isVisible = geometry.isVisible;

          if (isVisible) {
            geometry.isVisible = false;
            section.style.opacity = 0.5;
          } else {
            geometry.isVisible = true;
            section.style.opacity = 1.0;
          }
        }

        getSelectedIndex(source, index) {
          let selectedIndex = null;
          this.selectedItems.forEach((item, i) => {
            if (item.source == source && item.item.index == index) {
              selectedIndex = i;
              return;
            }
          });
          return selectedIndex;
        }

        updateSelected(current = -1) {
          let n = this.selectedItems.length
          // Hide the container if no items are selected
          if (n === 0) {
            this.el.selected.style.display = 'none';
            return;
          } else {
            this.el.selected.style.display = 'block';
          }

          if (current < 0) current = n - 1;
          if (current >= n) current = 0;
          this.selectedCurrent = current;

          let item = this.selectedItems[current];

          let phIndex = this.ohIndexToPhIndex[item.source];
          let meta = this.scatterMeta[phIndex];
          let phName = this.ohIndexToPhName[item.source];
          let seriesState = this.seriesState[phName];

          let fullLabel = data[phName].labels[item.item.index].split('__');

          let labelIndex = meta.label_index[seriesState];
          let titleIndex = meta.title_index[seriesState];
          let selectedLabels = meta.selected_labels[seriesState];

          this.el.selectedCurrent.innerHTML = current + 1;
          this.el.selectedTotal.innerHTML = n;
          this.el.selectedTitle.innerHTML = fullLabel[titleIndex];

          this.setSelectedContent(fullLabel, labelIndex, selectedLabels);

          // Remove all indicators
          this.selectedIndicators.forEach(indicator => {
            indicator.element.parentElement.removeChild(indicator.element);
          });
          this.selectedIndicators.length = 0;

          // Add the indicator for this object
          let indicatorElement = Faerun.createElement(
            'div', 
            { classes: 'selected-indicator' },
            [
              Faerun.createElement('div', { classes: 'crosshair-x' }),
              Faerun.createElement('div', { classes: 'crosshair-y' })
            ]
          );

          this.body.appendChild(indicatorElement);
          this.selectedIndicators.push({
            element: indicatorElement,
            index: item.item.index,
            ohIndex: item.source,
            phIndex: phIndex
          });
          this.updateSelectedIndicators();
        }

        updateSelectedIndicators() {
          this.selectedIndicators.forEach(indicator => {
            let pointSize = this.pointHelpers[indicator.phIndex].getPointSize();
            let screenPosition = this.octreeHelpers[indicator.ohIndex]
                                     .getScreenPosition(indicator.index);
            
            // Make the crosshairs larger than the point
            pointSize = Faerun.getMax([pointSize / window.devicePixelRatio, 10 / window.devicePixelRatio]);
            pointSize *= 1.25;
            let halfPointSize = pointSize / 2.0;
            indicator.element.style.left = (screenPosition[0] - halfPointSize) + 'px';
            indicator.element.style.top = (screenPosition[1] - halfPointSize) + 'px';
            indicator.element.style.width = pointSize + 'px';
            indicator.element.style.height = pointSize + 'px';
          });
        }

        updateTitle(first = false) {
          if (this.el.title === undefined) return;

          let bb = this.el.title.getBoundingClientRect();
          let scenePosition = new Lore.Math.Vector3f(
            this.min[0], this.min[1], 
            this.min[2]
          );

          let screenPosition = this.lore.controls.camera.sceneToScreen(scenePosition, this.lore);
          

          let maxscenePosition = new Lore.Math.Vector3f(
            this.max[0], this.max[1], 
            this.min[2]
          );

          let maxscreenPosition = this.lore.controls.camera.sceneToScreen(maxscenePosition, this.lore);        
          // this.el.title.style.left = (screenPosition[0] - (bb.width / 2.0)) + 'px';
          // console.log(this.min);

          // console.log(scenePosition);
          console.log(screenPosition[1]);
          console.log(maxscreenPosition[1]);

          this.el.title.style.left = (screenPosition[0]) + 'px';
          this.el.title.style.width = (maxscreenPosition[0]-screenPosition[0]) + 'px';

          this.el.title.style.top = (maxscreenPosition[1]) + 'px';
          // this.el.title.style.height = (screenPosition[1]-maxscreenPosition[1]+2) + 'px';




          //maxscreenPosition[1]+screenPosition[1]
          // this.el.title.style.right = (screenPosition[0]) + 'px';          
          // this.el.title.style.bottom = (screenPosition[1]) + 'px';
          // this.el.title.style.bottom = (screenPosition[1]) + 'px';          
          // this.el.title.style.top = (screenPosition[1]) + 'px';          

          // this.el.title.style.left = (screenPosition[0] - 200) + 'px';
          // this.el.title.style.right = (screenPosition[0] + 200) + 'px';

          // this.el.title.style.top = (screenPosition[1]) + 'px';

          // this.el.title.style.right = screenPosition[1] + 'px';
          // this.el.title.style.bottom = screenPosition[0] + 'px';

          // console.log(screenPosition);
          // this.el.title.style.top = (screenPosition[1] - bb.height-500) + 'px';

          if (first) this.el.title.classList.add('show');
        }

        updateXAxis(first = false) {
          if (this.el.xAxis === undefined) return;
          
          let bb = this.el.xAxis.getBoundingClientRect();
          let scenePosition = new Lore.Math.Vector3f(
            (this.min[0] + this.min[0]) / 2.0, this.min[1], 
            (this.min[2] + this.min[2]) / 2.0
          );

          let screenPosition = this.lore.controls.camera.sceneToScreen(scenePosition, this.lore);
          
          this.el.xAxis.style.left = (screenPosition[0] - (bb.width / 2.0)) + 'px';
          this.el.xAxis.style.top = (screenPosition[1]) + 'px';

          if (first) this.el.xAxis.classList.add('show');
        }

        updateYAxis(first = false) {
          if (this.el.yAxis === undefined) return;
            
          let bb = this.el.yAxis.getBoundingClientRect();
          let scenePosition = new Lore.Math.Vector3f(
            this.min[0], (this.min[1] + this.min[1]) / 2.0, 
            (this.min[2] + this.min[2]) / 2.0
          );
          
          let screenPosition = this.lore.controls.camera.sceneToScreen(scenePosition, this.lore);
          
          this.el.yAxis.style.left = (screenPosition[0] - bb.height) + 'px';
          this.el.yAxis.style.top = (screenPosition[1] - bb.width / 2.0) + 'px';

          if (first) this.el.yAxis.classList.add('show');
        }

        changeSeries(value, name) {
          value = parseInt(value);
          this.seriesState[name] = value;
          this.renderLegend();

          this.pointHelpers[this.phIndexMap[name]].setRGBFromArrays(
            data[name]['colors'][value].r, 
            data[name]['colors'][value].g, 
            data[name]['colors'][value].b
          );
        }

        search() {
          let searchTerm = prompt('Please enter a serch term. Please note, that the search is very sensitive to punctuation and spelling, so searches with very few words work best. As I don\'t want to violate the WoS-user agreement, only very basic search functions are allowed.');
          if (!searchTerm) return;

          let results = {}
          let re = new RegExp(searchTerm, 'i');
          Object.keys(data).forEach(name => {
            if (!('labels' in data[name])) return;
            results[name] = []
            data[name]['labels'].forEach((label, i) => {
              if (re.test(label))
                results[name].push(i);
            });
          });

          for (const [name, indices] of Object.entries(results)) {
            if (!name in this.ohIndexMap) return;
            indices.forEach(index => {
              this.octreeHelpers[this.ohIndexMap[name]].addSelected(index);
            });
          }
        }

        static createColorBox(value) {
          return Faerun.createElement(
            'div', 
            { 
              classes: 'color-box', 
              style: `background-color: rgba(${value[0] * 255}, ${value[1] * 255 }, ${value[2] * 255 }, ${value[3] });
                      border-color: rgba(${value[0] * 255 }, ${value[1] * 255 }, ${value[2] * 255 }, ${value[3] })`
            }
          );
        }

        static createColorScale(values) {
          let scale = [];

          values.forEach(value => {
            scale.push(
              Faerun.createElement(
                'div', 
                { 
                  classes: 'color-stripe', 
                  style: `background-color: rgba(${value[0][0] * 255}, ${value[0][1] * 255}, ${value[0][2] * 255}, ${value[0][3]});
                          border-color: rgba(${value[0][0] * 255}, ${value[0][1] * 255}, ${value[0][2] * 255}, ${value[0][3]})`,
                  alt: value[1]
                }
              ),
            )
          });

          return scale;
        }
        
        static createElement(tag, values, children) {
          let element = document.createElement(tag);

          for (const key of Object.keys(values)) {
            if (key === 'classes')
              element.classList.add(...values[key].split(' '));
            else if (key === 'content')
              element.innerHTML = values[key];
            else if (key === 'hidden') {
              if (values[key])
                element.setAttribute('hidden', true);
            }
            else if (key === 'selected') {
              if (values[key])
                element.setAttribute('selected', true);
            }
            else
              element.setAttribute(key, values[key]);
          }

          if (children) {
            if (Array.isArray(children)) {
              children.forEach(child => {
                element.appendChild(child);
              })
            } else {
              element.appendChild(children);
            }
          }

          return element;
        }

        static bindElements() {
          let result = {};
          document.querySelectorAll('[data-bind]').forEach(e => {
            result[e.getAttribute('data-bind')] = e;
          });
          return result;
        }

        static getMin(arr, other = Number.MAX_VALUE) {
          let m = Number.MAX_VALUE;
          for (var i = 0; i < arr.length; i++)
              if (arr[i] < m) m = arr[i];
          
          if (m < other) return m;
          return other;
        }

        static getMax(arr, other = -Number.MAX_VALUE) {
          let m = -Number.MAX_VALUE;
          for (var i = 0; i < arr.length; i++)
              if (arr[i] > m) m = arr[i];
          
          if (m > other) return m;
          return other;
        }
      }

      let f = new Faerun();
    </script>
    <script>
    	
$( window ).ready(function() {

setTimeout (function() {
$('#loader').remove();

}, 1300);

});



    </script>
  </body>
</html>